-- phpMyAdmin SQL Dump
-- version 5.0.2
-- https://www.phpmyadmin.net/
--
-- Host: 127.0.0.1:3306
-- Generation Time: May 07, 2021 at 07:27 PM
-- Server version: 8.0.21
-- PHP Version: 7.4.9

SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
START TRANSACTION;
SET time_zone = "+00:00";


/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8mb4 */;

--
-- Database: `biblio`
--

-- --------------------------------------------------------

--
-- Table structure for table `article`
--

DROP TABLE IF EXISTS `article`;
CREATE TABLE IF NOT EXISTS `article` (
  `id_article` int NOT NULL AUTO_INCREMENT,
  `origin` varchar(100) NOT NULL,
  `num_access` varchar(255) NOT NULL,
  `title` text NOT NULL,
  `abstract` text NOT NULL,
  `year` year NOT NULL,
  `journal` varchar(255) NOT NULL,
  `pmcid` varchar(50) DEFAULT NULL,
  `status` int DEFAULT NULL,
  `html_xml` mediumtext,
  `general_note` mediumtext,
  `user` int DEFAULT NULL,
  PRIMARY KEY (`id_article`),
  KEY `id_user` (`user`),
  KEY `status` (`status`)
) ENGINE=InnoDB AUTO_INCREMENT=80 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;

--
-- Dumping data for table `article`
--

INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(1, 'pubmed', '31198441', 'The - gene cluster of  encodes GH43- and GH62-α-l-arabinofuranosidases with complementary modes of action.', 'The α-l-arabinofuranosidases (α-l-ABFs) are exoenzymes involved in the hydrolysis of α-l-arabinosyl linkages in plant cell wall polysaccharides. They play a crucial role in the degradation of arabinoxylan and arabinan and they are used in many biotechnological applications. Analysis of the genome of  showed that putative cellulosomal α-l-ABFs are exclusively encoded by the - gene cluster, a large 32-kb gene cluster. Indeed, among the 14 Xyl-Doc enzymes encoded by this gene cluster, 6 are predicted to be α-l-ABFs belonging to the CAZyme families GH43 and GH62.The biochemical characterization of these six Xyl-Doc enzymes revealed that four of them are α-l-ABFs. GH43-1229 (RcAbf43A) which belongs to the subfamily 16 of the GH43, encoded by the gene at locus Ccel_1229, has a low specific activity on natural substrates and can cleave off arabinose decorations located at arabinoxylan chain extremities. GH43-1233 (RcAbf43A), the product of the gene at locus Ccel_1233, belonging to subfamily 10 of the GH43, can convert the double arabinose decorations present on arabinoxylan into single O2- or O3-linked decorations with high velocity ( = 16.6 ± 0.6 s). This enzyme acts in synergy with GH62-1234 (RcAbf62A), the product of the gene at locus Ccel_1234, a GH62 α-l-ABF which hydrolyzes α-(1 → 3) or α-(1 → 2)-arabinosyl linkages present on polysaccharides and arabinoxylooligosaccharides monodecorated. Finally, a bifunctional enzyme, GH62-CE6-1240 (RcAbf62BAxe6), encoded by the gene at locus Ccel_1240, which contains a GH62-α-l-ABF module and a carbohydrate esterase (CE6) module, catalyzes deacylation of plant cell wall polymers and cleavage of arabinosyl mono-substitutions. These enzymes are also active on arabinan, a component of the type I rhamnogalacturonan, showing their involvement in pectin degradation.Arabinofuranosyl decorations on arabinoxylan and pectin strongly inhibit the action of xylan-degrading enzymes and pectinases. α-l-ABFs encoded by the - gene cluster of  can remove all the decorations present in the backbone of arabinoxylan and arabinan, act synergistically, and, thus, play a crucial role in the degradation of plant cell wall polysaccharides.', 2019, 'Biotechnology for biofuels', 'PMC6556953', 3, NULL, NULL, 1),
(2, 'pubmed', '33476380', 'A novel dextranase gene from the marine bacterium Bacillus aquimaris S5 and its expression and characteristics.', 'Dextranase specifically hydrolyzes dextran and is used to produce functional isomalto-saccharide prebiotics. Moreover, dextranase is used as an additive in mouthwash to remove dental plaque. We cloned and expressed the dextranase gene of the marine bacterium Bacillus aquimaris S5. The length of the BaDex gene was 1788 bp, encoding 573 amino acids. Using bioinformatics to predict and analyze the amino acid sequence of BaDex, we found the isoelectric point and instability coefficient to be 4.55 and 29.22, respectively. The average hydrophilicity (GRAVY) was -0.662. The secondary structure of BaDex consisted of 145 alpha helices, accounting for 25.31% of the protein; 126 extended strands, accounting for 21.99%; and 282 random coils, accounting for 49.21%. The 3D structure of the BaDex protein was predicted and simulated using SWISS-MODEL, and BaDex was classified as a Glycoside Hydrolase Family 66 protein. The optimal temperature and pH for BaDex activity were 40°C and 6.0, respectively. The hydrolysates had excellent antioxidant activity, and 8 U/mL of BaDex could remove 80% of dental plaque in MBRC experiment. This recombinant protein thus has great promise for applications in the food and pharmaceutical industries.', 2021, 'FEMS microbiology letters', '', 3, NULL, NULL, 2),
(3, 'pubmed', '33465378', 'Structure-function analysis of pectate lyase Pel3 reveals essential facets of protein recognition by the bacterial type 2 secretion system.', 'The Type II Secretion System (T2SS) transports fully folded proteins of various functions and structures through the outer membrane of Gram-negative bacteria. The molecular mechanisms of substrate recruitment by T2SS remain elusive but a prevailing view is that the secretion determinants could be of a structural nature. The phytopathogenic γ-Proteobacteria, Pectobacterium carotovorum and Dickeya dadantii, secrete similar sets of homologous plant cell wall degrading enzymes, mainly pectinases, by similar T2SSs, called Out. However, the orthologous pectate lyases Pel3 and PelI from these bacteria, which share 67% of sequence identity, are not secreted by the counterpart T2SS of each bacterium, indicating a fine-tuned control of protein recruitment. To identify the related secretion determinants, we first performed a structural characterization and comparison of Pel3 with PelI using X-ray crystallography. Then, to assess the biological relevance of the observed structural variations, we conducted a loop-substitution analysis of Pel3 combined with secretion assays. We showed that there is not one element with a definite secondary structure but several distant and structurally flexible loop regions that are essential for the secretion of Pel3 and that these loop regions act together as a composite secretion signal. Interestingly, depending on the crystal contacts, one of these key secretion determinants undergoes disorder-to-order transitions that could reflect its transient structuration upon the contact with the appropriate T2SS components. We hypothesize that such T2SS-induced structuration of some intrinsically disordered zones of secretion substrates could be part of the recruitment mechanism used by T2SS.', 2021, 'The Journal of biological chemistry', 'PMC7949064', 4, NULL, NULL, 1),
(4, 'pubmed', '33469030', 'Degradation of complex arabinoxylans by human colonic Bacteroidetes.', 'Some Bacteroidetes and other human colonic bacteria can degrade arabinoxylans, common polysaccharides found in dietary fiber. Previous work has identified gene clusters (polysaccharide-utilization loci, PULs) for degradation of simple arabinoxylans. However, the degradation of complex arabinoxylans (containing side chains such as ferulic acid, a phenolic compound) is poorly understood. Here, we identify a PUL that encodes multiple esterases for degradation of complex arabinoxylans in Bacteroides species. The PUL is specifically upregulated in the presence of complex arabinoxylans. We characterize some of the esterases biochemically and structurally, and show that they release ferulic acid from complex arabinoxylans. Growth of four different colonic Bacteroidetes members, including Bacteroides intestinalis, on complex arabinoxylans results in accumulation of ferulic acid, a compound known to have antioxidative and immunomodulatory properties.', 2021, 'Nature communications', 'PMC7815789', 4, NULL, NULL, 5),
(5, 'pubmed', '33838183', 'Functional diversity of three tandem C-terminal carbohydrate-binding modules of a β-mannanase.', 'Carbohydrate active enzymes, such as those involved in plant cell wall and storage polysaccharide biosynthesis and deconstruction, often contain repeating non-catalytic carbohydrate binding modules (CBMs) to compensate for low-affinity binding typical of protein-carbohydrate interactions. The bacterium Saccharophagus degradans produces an endo-β-mannanase of glycoside hydrolase family 5 subfamily 8 with three phylogenetically distinct family 10 CBMs located C-terminally from the catalytic domain (SdGH5_8-CBM10x3). However, the functional roles and cooperativity of these CBM domains in polysaccharide binding is not clear. To learn more we studied the full-length enzyme, three stepwise CBM10 truncations, and green fluorescent protein fusions of the individual CBM10s and all three domains together by pull-down assays, affinity gel electrophoresis, and activity assays. Only the C-terminal CBM10-3 was found to bind strongly to microcrystalline cellulose (dissociation constant, K = 1.48 μM). CBM10-3 and CBM10-2 bound galactomannan with similar affinity (K = 0.2-0.4 mg/ml), but CBM10-1 had 20-fold lower affinity for this substrate. CBM10 truncations barely affected specific activity on carob galactomannan and konjac glucomannan. Full-length SdGH5_8-CBM10x3 was two-fold more active on the highly galactose-decorated viscous guar gum galactomannan and crystalline ivory nut mannan at high enzyme concentrations, but the specific activity was 4- to 9-fold reduced at low enzyme and substrate concentrations compared to the enzyme lacking CBM10-2 and -3. Comparison of activity and binding data for the different enzyme forms indicates unproductive and productive polysaccharide binding to occur. We conclude that the C-terminal-most CBM10-3 secures firm binding, with contribution from CBM10-2, which with CBM10-1 also provides spatial flexibility.', 2021, 'The Journal of biological chemistry', '', 3, NULL, NULL, 4),
(6, 'pubmed', '33837197', 'Discovery of fungal oligosaccharide-oxidising flavo-enzymes with previously unknown substrates, redox-activity profiles and interplay with LPMOs.', 'Oxidative plant cell-wall processing enzymes are of great importance in biology and biotechnology. Yet, our insight into the functional interplay amongst such oxidative enzymes remains limited. Here, a phylogenetic analysis of the auxiliary activity 7 family (AA7), currently harbouring oligosaccharide flavo-oxidases, reveals a striking abundance of AA7-genes in phytopathogenic fungi and Oomycetes. Expression of five fungal enzymes, including three from unexplored clades, expands the AA7-substrate range and unveils a cellooligosaccharide dehydrogenase activity, previously unknown within AA7. Sequence and structural analyses identify unique signatures distinguishing the strict dehydrogenase clade from canonical AA7 oxidases. The discovered dehydrogenase directly is able to transfer electrons to an AA9 lytic polysaccharide monooxygenase (LPMO) and fuel cellulose degradation by LPMOs without exogenous reductants. The expansion of redox-profiles and substrate range highlights the functional diversity within AA7 and sets the stage for harnessing AA7 dehydrogenases to fine-tune LPMO activity in biotechnological conversion of plant feedstocks.', 2021, 'Nature communications', '', 1, NULL, NULL, 2),
(7, 'pubmed', '33686095', 'Clinical evidence of the role of Methanobrevibacter smithii in severe acute malnutrition.', 'Gut microbial dysbiosis has been shown to be an instrumental factor in severe acute malnutrition (SAM) and particularly, the absence of Methanobrevibacter smithii, a key player in energy harvest. Nevertheless, it remains unknown whether this absence reflects an immaturity or a loss of the microbiota. In order to assess that, we performed a case-control study in Mali using a propensity score weighting approach. The presence of M. smithii was tested using quantitative PCR on faeces collected from SAM children at inclusion and at discharge when possible or at day 15 for controls. M. smithii was highly significantly associated with the absence of SAM, detected in 40.9% controls but only in 4.2% cases (p < 0.0001). The predictive positive value for detection of M. smithii gradually increased with age in controls while decreasing in cases. Among children providing two samples with a negative first sample, no SAM children became positive, while this proportion was 2/4 in controls (p = 0.0015). This data suggests that gut dysbiosis in SAM is not an immaturity but rather features a loss of M. smithii. The addition of M. smithii as a probiotic may thus represent an important addition to therapeutic approaches to restore gut symbiosis.', 2021, 'Scientific reports', 'PMC7940396', 1, NULL, NULL, 4),
(8, 'pubmed', '33571448', 'Strain-level functional variation in the human gut microbiota based on bacterial binding to artificial food particles.', 'Greater understanding of the spatial relationships between members of the human gut microbiota and available nutrients is needed to gain deeper insights about community dynamics and expressed functions. Therefore, we generated a panel of artificial food particles with each type composed of microscopic paramagnetic beads coated with a fluorescent barcode and one of 60 different dietary or host glycan preparations. Analysis of 160 Bacteroides and Parabacteroides strains disclosed diverse strain-specific and glycan-specific binding phenotypes. We identified carbohydrate structures that correlated with binding by specific bacterial strains in vitro and noted strain-specific differences in the catabolism of glycans that mediate adhesion. Mixed in vitro cultures revealed that these adhesion phenotypes are maintained in more complex communities. Additionally, orally administering glycan beads to gnotobiotic mice confirmed specificity in glycan binding. This approach should facilitate analyses of how strains occupying the same physical niche interact, and it should advance the development of synbiotics, more nutritious foods, and microbiota-based diagnostics.', 2021, 'Cell host & microbe', '', 2, NULL, NULL, 1),
(9, 'pubmed', '33538380', 'Gene family expansions and transcriptome signatures uncover fungal adaptations to wood decay.', 'Because they comprise some of the most efficient wood-decayers, Polyporales fungi impact carbon cycling in forest environment. Despite continuous discoveries on the enzymatic machinery involved in wood decomposition, the vision on their evolutionary adaptation to wood decay and genome diversity remains incomplete. We combined the genome sequence information from 50 Polyporales species, including 26 newly sequenced genomes and sought for genomic and functional adaptations to wood decay through the analysis of genome composition and transcriptome responses to different carbon sources. The genomes of Polyporales from different phylogenetic clades showed poor conservation in macrosynteny, indicative of genome rearrangements. We observed different gene family expansion/contraction histories for plant cell wall degrading enzymes in core polyporoids and phlebioids and captured expansions for genes involved in signalling and regulation in the lineages of white rotters. Furthermore, we identified conserved cupredoxins, thaumatin-like proteins and lytic polysaccharide monooxygenases with a yet uncharacterized appended module as new candidate players in wood decomposition. Given the current need for enzymatic toolkits dedicated to the transformation of renewable carbon sources, the observed genomic diversity among Polyporales strengthens the relevance of mining Polyporales biodiversity to understand the molecular mechanisms of wood decay.', 2021, 'Environmental microbiology', '', 3, NULL, NULL, 5),
(10, 'pubmed', '33460770', 'Characterization of three bacterial glycoside hydrolase family 9 endoglucanases with different modular architectures isolated from a compost metagenome.', 'Environmental bacteria express a wide diversity of glycoside hydrolases (GH). Screening and characterization of GH from metagenomic sources provides an insight into biomass degradation strategies of non-cultivated prokaryotes.In the present report, we screened a compost metagenome for lignocellulolytic activities and identified six genes encoding enzymes belonging to family GH9 (GH9a-f). Three of these enzymes (GH9b, GH9d and GH9e) were successfully expressed and characterized.A phylogenetic analysis of the catalytic domain of pro- and eukaryotic GH9 enzymes suggested the existence of two major subgroups. Bacterial GH9s displayed a wide variety of modular architectures and those harboring an N-terminal Ig-like domain, such as GH9b and GH9d, segregated from the remainder. We purified and characterized GH9 endoglucanases from both subgroups and examined their stabilities, substrate specificities and product profiles. GH9e exhibited an original hydrolysis pattern, liberating an elevated proportion of oligosaccharides longer than cellobiose. All of the enzymes exhibited processive behavior and a synergistic action on crystalline cellulose. Synergy was also evidenced between GH9d and a GH48 enzyme identified from the same metagenome.The characterized GH9 enzymes displayed different modular architectures and distinct substrate and product profiles. The presence of a cellulose binding domain was shown to be necessary for binding and digestion of insoluble cellulosic substrates, but not for processivity.The identification of six GH9 enzymes from a compost metagenome and the functional variety of three characterized members highlight the importance of this enzyme family in bacterial biomass deconstruction.', 2021, 'Biochimica et biophysica acta. General subjects', '', 2, NULL, NULL, 3),
(11, 'pubmed', '33268895', 'Galactosaminogalactan activates the inflammasome to provide host protection.', 'Inflammasomes are important sentinels of innate immune defence that are activated in response to diverse stimuli, including pathogen-associated molecular patterns (PAMPs). Activation of the inflammasome provides host defence against aspergillosis, which is a major health concern for patients who are immunocompromised. However, the Aspergillus fumigatus PAMPs that are responsible for inflammasome activation are not known. Here we show that the polysaccharide galactosaminogalactan (GAG) of A. fumigatus is a PAMP that activates the NLRP3 inflammasome. The binding of GAG to ribosomal proteins inhibited cellular translation machinery, and thus activated the NLRP3 inflammasome. The galactosamine moiety bound to ribosomal proteins and blocked cellular translation, which triggered activation of the NLRP3 inflammasome. In mice, a GAG-deficient Aspergillus mutant (Δgt4c) did not elicit protective activation of the inflammasome, and this strain exhibited enhanced virulence. Moreover, administration of GAG protected mice from colitis induced by dextran sulfate sodium in an inflammasome-dependent manner. Thus, ribosomes connect the sensing of this fungal PAMP to the activation of an innate immune response.', 2020, 'Nature', '', 2, NULL, NULL, 1),
(12, 'pubmed', '33211093', 'Genomic Analysis Enlightens Agaricales Lifestyle Evolution and Increasing Peroxidase Diversity.', 'As actors of global carbon cycle, Agaricomycetes (Basidiomycota) have developed complex enzymatic machineries that allow them to decompose all plant polymers, including lignin. Among them, saprotrophic Agaricales are characterized by an unparalleled diversity of habitats and lifestyles. Comparative analysis of 52 Agaricomycetes genomes (14 of them sequenced de novo) reveals that Agaricales possess a large diversity of hydrolytic and oxidative enzymes for lignocellulose decay. Based on the gene families with the predicted highest evolutionary rates-namely cellulose-binding CBM1, glycoside hydrolase GH43, lytic polysaccharide monooxygenase AA9, class-II peroxidases, glucose-methanol-choline oxidase/dehydrogenases, laccases, and unspecific peroxygenases-we reconstructed the lifestyles of the ancestors that led to the extant lignocellulose-decomposing Agaricomycetes. The changes in the enzymatic toolkit of ancestral Agaricales are correlated with the evolution of their ability to grow not only on wood but also on leaf litter and decayed wood, with grass-litter decomposers as the most recent eco-physiological group. In this context, the above families were analyzed in detail in connection with lifestyle diversity. Peroxidases appear as a central component of the enzymatic toolkit of saprotrophic Agaricomycetes, consistent with their essential role in lignin degradation and high evolutionary rates. This includes not only expansions/losses in peroxidase genes common to other basidiomycetes but also the widespread presence in Agaricales (and Russulales) of new peroxidases types not found in wood-rotting Polyporales, and other Agaricomycetes orders. Therefore, we analyzed the peroxidase evolution in Agaricomycetes by ancestral-sequence reconstruction revealing several major evolutionary pathways and mapped the appearance of the different enzyme types in a time-calibrated species tree.', 2021, 'Molecular biology and evolution', '', 4, NULL, NULL, 2),
(13, 'pubmed', '33199373', 'Identification of the molecular determinants driving the substrate specificity of fungal lytic polysaccharide monooxygenases (LPMOs).', 'Understanding enzymatic breakdown of plant biomass is crucial to develop nature-inspired biotechnological processes. Lytic polysaccharide monooxygenases (LPMOs) are microbial enzymes secreted by fungal saprotrophs involved in carbon recycling. LPMOs modify biomass by oxidatively cleaving polysaccharides thereby enhancing the efficiency of glycoside hydrolases. Fungal AA9 LPMOs are active on cellulose but some members also display activity on hemicelluloses and/or oligosaccharides. Although the active site subsites are well defined for a few model LPMOs, the molecular determinants driving broad substrate specificity are still not easily predictable. Based on bioinformatic clustering and sequence alignments, we selected seven fungal AA9 LPMOs that differ in the amino-acid residues constituting their subsites. Investigation of their substrate specificities revealed that all these LPMOs are active on cellulose and cello-oligosaccharides, as well as plant cell wall-derived hemicellulosic polysaccharides and carry out C4 oxidative cleavage. The product profiles from cello-oligosaccharides degradation suggests that the subtle differences in amino acids sequence within the substrate-binding loop regions lead to different preferred binding modes. Our functional analyses allowed us to probe the molecular determinants of substrate binding within two AA9 LPMO sub-clusters. Many wood-degrading fungal species rich in AA9 genes have at least one AA9 enzyme with structural loop features that allow recognition of short β-(1,4)-linked glucan chains. Time-course monitoring of these AA9 LPMOs on cello-oligosaccharides also provides a useful model system for mechanistic studies of LPMO catalysis. These results are valuable for the understanding of LPMO contribution to wood decaying process in nature and for the development of sustainable biorefineries.', 2020, 'The Journal of biological chemistry', 'PMC7949027', 2, NULL, NULL, 5),
(14, 'pubmed', '30897810', 'Expression, Purification and Characterization of Chondroitinase AC II from Marine Bacterium  sp. CS01.', 'Chondroitinase (ChSase), a type of glycosaminoglycan (GAG) lyase, can degrade chondroitin sulfate (CS) to unsaturate oligosaccharides, with various functional activities. In this study, ChSase AC II from a newly isolated marine bacterium  sp. CS01 was cloned, expressed in  X33, purified, and characterized. ChSase AC II, with a molecular weight of approximately 100 kDa and a specific activity of 18.7 U/mg, showed the highest activity at 37 °C and pH 6.5 and maintained stability at a broad range of pH (5⁻7.5) and temperature (below 35 °C). The enzyme activity was increased in the presence of Mn and was strongly inhibited by Hg. Moreover, the kinetic parameters of ChSase AC II against CS-A, CS-C, and HA were determined. TLC and ESI-MS analysis of the degradation products indicated that ChSase AC II displayed an exolytic action mode and completely hydrolyzed three substrates into oligosaccharides with low degrees of polymerization (DPs). All these features make ChSase AC II a promising candidate for the full use of GAG to produce oligosaccharides.', 2019, 'Marine drugs', 'PMC6471956', 1, NULL, NULL, 3),
(17, 'pubmed', '52', 'Evidence of the involvement of a 50S ribosomal protein in several active sites.', 'The functional role of the Bacillus stearothermophilus 50S ribosomal protein B-L3 (probably homologous to the Escherichia coli protein L2) was examined by chemical modification. The complex [B-L3-23S RNA] was photooxidized in the presence of rose bengal and the modified protein incorporated by reconstitution into 50S ribosomal subunits containing all other unmodified components. Particles containing photooxidized B-L3 are defective in several functional assays, including (1) poly(U)-directed poly(Phe) synthesis, (2) peptidyltransferase activity, (3) ability to associate with a [30S-poly(U)-Phe-tRNA] complex, and (4) binding of elongation factor G and GTP. The rates of loss of the partial functional activities during photooxidation of B-L3 indicate that at least two independent inactivating events are occurring, a faster one, involving oxidation of one or more histidine residues, affecting peptidyltransferase and subunit association activities and a slower one affecting EF-G binding. Therefore the protein B-L3 has one or more histidine residues which are essential for peptidyltransferase and subunit association, and another residue which is essential for EF-G-GTP binding. B-L3 may be the ribosomal peptidyltransferase protein, or a part of the active site, and may contribute functional groups to the other active sites as well.', 1975, 'Biochemistry', '', 4, NULL, NULL, 4);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(18, 'pubmed', '33188211', 'Microbiota-directed fibre activates both targeted and secondary metabolic shifts in the distal gut.', 'Beneficial modulation of the gut microbiome has high-impact implications not only in humans, but also in livestock that sustain our current societal needs. In this context, we have tailored an acetylated galactoglucomannan (AcGGM) fibre to match unique enzymatic capabilities of Roseburia and Faecalibacterium species, both renowned butyrate-producing gut commensals. Here, we test the accuracy of AcGGM within the complex endogenous gut microbiome of pigs, wherein we resolve 355 metagenome-assembled genomes together with quantitative metaproteomes. In AcGGM-fed pigs, both target populations differentially express AcGGM-specific polysaccharide utilization loci, including novel, mannan-specific esterases that are critical to its deconstruction. However, AcGGM-inclusion also manifests a \"butterfly effect\", whereby numerous metabolic changes and interdependent cross-feeding pathways occur in neighboring non-mannanolytic populations that produce short-chain fatty acids. Our findings show how intricate structural features and acetylation patterns of dietary fibre can be customized to specific bacterial populations, with potential to create greater modulatory effects at large.', 2020, 'Nature communications', 'PMC7666174', 2, '<h1 class=\"content-title\">Microbiota-directed fibre activates both targeted and secondary metabolic shifts in the distal gut</h1><br><span class=\"contrib-group fm-author\"><a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Michalak%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm139884799365728\" co-class=\"co-affbox\">Leszek Michalak</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Gaby%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm139884796760736\" co-class=\"co-affbox\">John Christian Gaby</a>,<sup><img src=\"https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/corrauth.gif\" alt=\"corresponding author\"></sup><sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Lagos%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm139884799979488\" co-class=\"co-affbox\">Leidy Lagos</a>,<sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=La%20Rosa%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm139884799977408\" co-class=\"co-affbox\">Sabina Leanti La Rosa</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Hvidsten%20TR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm139884799974592\" co-class=\"co-affbox\">Torgeir R. Hvidsten</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=T%26%23x000e9%3Btard-Jones%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm139884797178496\" co-class=\"co-affbox\">Catherine Tétard-Jones</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Willats%20WG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm139884797176416\" co-class=\"co-affbox\">William G. T. Willats</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Terrapon%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm139884797174384\" co-class=\"co-affbox\">Nicolas Terrapon</a>,<sup>4,</sup><sup>5</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Lombard%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm139884797170688\" co-class=\"co-affbox\">Vincent Lombard</a>,<sup>4,</sup><sup>5</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Henrissat%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm139884797167872\" co-class=\"co-affbox\">Bernard Henrissat</a>,<sup>4,</sup><sup>5,</sup><sup>6</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Dr%26%23x000f6%3Bge%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm139884797163392\" co-class=\"co-affbox\">Johannes Dröge</a>,<sup>7</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Arntzen%20M%26%23x000d8%3B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm139884797161264\" co-class=\"co-affbox\">Magnus Øverlie Arntzen</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Hagen%20LH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm139884876600000\" co-class=\"co-affbox\">Live Heldal Hagen</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=%26%23x000d8%3Bverland%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm139884876597184\" co-class=\"co-affbox\">Margareth Øverland</a>,<sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Pope%20PB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm139884876595152\" co-class=\"co-affbox\">Phillip B. Pope</a>,<sup><img src=\"https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/corrauth.gif\" alt=\"corresponding author\"></sup><sup>#</sup><sup>1,</sup><sup>2</sup> and  <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Westereng%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm139884876590688\" co-class=\"co-affbox\">Bjørge Westereng</a><sup><img src=\"https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/corrauth.gif\" alt=\"corresponding author\"></sup><sup>#</sup><sup>1</sup></span><span style=\"display:none\" class=\"contrib-group aff-tip\"><span id=\"_co_idm139884799365728\"><h3 class=\"no_margin\">Leszek Michalak</h3><p><sup>1</sup>Faculty of Chemistry, Biotechnology and Food Science, Norwegian University of Life Sciences, 1432 Ås, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Michalak%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Leszek Michalak</a></span></span><span id=\"_co_idm139884796760736\"><h3 class=\"no_margin\">John Christian Gaby</h3><p><sup>1</sup>Faculty of Chemistry, Biotechnology and Food Science, Norwegian University of Life Sciences, 1432 Ås, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Gaby%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">John Christian Gaby</a></span></span><span id=\"_co_idm139884799979488\"><h3 class=\"no_margin\">Leidy Lagos</h3><p><sup>2</sup>Faculty of Biosciences, Norwegian University of Life Sciences, 1432 Ås, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Lagos%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Leidy Lagos</a></span></span><span id=\"_co_idm139884799977408\"><h3 class=\"no_margin\">Sabina Leanti La Rosa</h3><p><sup>1</sup>Faculty of Chemistry, Biotechnology and Food Science, Norwegian University of Life Sciences, 1432 Ås, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=La%20Rosa%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Sabina Leanti La Rosa</a></span></span><span id=\"_co_idm139884799974592\"><h3 class=\"no_margin\">Torgeir R. Hvidsten</h3><p><sup>1</sup>Faculty of Chemistry, Biotechnology and Food Science, Norwegian University of Life Sciences, 1432 Ås, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Hvidsten%20TR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Torgeir R. Hvidsten</a></span></span><span id=\"_co_idm139884797178496\"><h3 class=\"no_margin\">Catherine Tétard-Jones</h3><p><sup>3</sup>School of Natural and Environmental Sciences, Newcastle University, Newcastle upon Tyne, UK </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=T%26%23x000e9%3Btard-Jones%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Catherine Tétard-Jones</a></span></span><span id=\"_co_idm139884797176416\"><h3 class=\"no_margin\">William G. T. Willats</h3><p><sup>3</sup>School of Natural and Environmental Sciences, Newcastle University, Newcastle upon Tyne, UK </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Willats%20WG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">William G. T. Willats</a></span></span><span id=\"_co_idm139884797174384\"><h3 class=\"no_margin\">Nicolas Terrapon</h3><p><sup>4</sup>Centre National de la Recherche Scientifique, Aix-Marseille Université, UMR7257 Marseille, France </p><p><sup>5</sup>Institut National de la Recherche Agronomique, USC1048 Architecture et Fonction des Macromolécules Biologiques, Marseille, France </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Terrapon%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Nicolas Terrapon</a></span></span><span id=\"_co_idm139884797170688\"><h3 class=\"no_margin\">Vincent Lombard</h3><p><sup>4</sup>Centre National de la Recherche Scientifique, Aix-Marseille Université, UMR7257 Marseille, France </p><p><sup>5</sup>Institut National de la Recherche Agronomique, USC1048 Architecture et Fonction des Macromolécules Biologiques, Marseille, France </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Lombard%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Vincent Lombard</a></span></span><span id=\"_co_idm139884797167872\"><h3 class=\"no_margin\">Bernard Henrissat</h3><p><sup>4</sup>Centre National de la Recherche Scientifique, Aix-Marseille Université, UMR7257 Marseille, France </p><p><sup>5</sup>Institut National de la Recherche Agronomique, USC1048 Architecture et Fonction des Macromolécules Biologiques, Marseille, France </p><p><sup>6</sup>Department of Biological Sciences, King Abdulaziz University, Jeddah, Saudi Arabia </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Henrissat%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Bernard Henrissat</a></span></span><span id=\"_co_idm139884797163392\"><h3 class=\"no_margin\">Johannes Dröge</h3><p><sup>7</sup>Department for Mathematical Sciences, Chalmers University of Technology, Gothenburg, Sweden </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Dr%26%23x000f6%3Bge%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Johannes Dröge</a></span></span><span id=\"_co_idm139884797161264\"><h3 class=\"no_margin\">Magnus Øverlie Arntzen</h3><p><sup>1</sup>Faculty of Chemistry, Biotechnology and Food Science, Norwegian University of Life Sciences, 1432 Ås, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Arntzen%20M%26%23x000d8%3B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Magnus Øverlie Arntzen</a></span></span><span id=\"_co_idm139884876600000\"><h3 class=\"no_margin\">Live Heldal Hagen</h3><p><sup>1</sup>Faculty of Chemistry, Biotechnology and Food Science, Norwegian University of Life Sciences, 1432 Ås, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Hagen%20LH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Live Heldal Hagen</a></span></span><span id=\"_co_idm139884876597184\"><h3 class=\"no_margin\">Margareth Øverland</h3><p><sup>2</sup>Faculty of Biosciences, Norwegian University of Life Sciences, 1432 Ås, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=%26%23x000d8%3Bverland%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Margareth Øverland</a></span></span><span id=\"_co_idm139884876595152\"><h3 class=\"no_margin\">Phillip B. Pope</h3><p><sup>1</sup>Faculty of Chemistry, Biotechnology and Food Science, Norwegian University of Life Sciences, 1432 Ås, Norway </p><p><sup>2</sup>Faculty of Biosciences, Norwegian University of Life Sciences, 1432 Ås, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Pope%20PB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Phillip B. Pope</a></span></span><span id=\"_co_idm139884876590688\"><h3 class=\"no_margin\">Bjørge Westereng</h3><p><sup>1</sup>Faculty of Chemistry, Biotechnology and Food Science, Norwegian University of Life Sciences, 1432 Ås, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Westereng%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Bjørge Westereng</a></span></span></span><span class=\"fm-panel half_rhythm\"><span class=\"togglers<br><span id=\" idm=\"\" 1\"=\"\" lang=\"en\"><h2 class=\"head no_bottom_margin\" id=\"idm\n1title\"><mark id=\"mark_2021-05-07-01-53-52\" style=\"background-color: #ffff00;\"><a id=\"link_2021-05-07-01-53-52\" class=\"note\" data-bs-toggle=\"popover\" data-bs-trigger=\"hover focus\" data-placement=\"bottom\" data-bs-html=\"true\" title=\"\" data-bs-content=\"Your annotation<hr class=\'sep\'>1 Replies\" onclick=\"annotateShow(\'2021-05-07-01-53-52\')\" data-bs-original-title=\"[2021-05-07-01-53-52] Eddy IKHLEF\">Abstract</a></mark></h2><!--article-meta--><span><p id=\"Par1\" class=\"p p-first-last\">Beneficial modulation of the gut m<mark id=\"mark_2021-05-07-02-20-07\" style=\"background-color: #ffff00;\"><a id=\"link_2021-05-07-02-20-07\" class=\"note\" data-bs-toggle=\"popover\" data-bs-trigger=\"hover focus\" data-placement=\"bottom\" data-bs-html=\"true\" title=\"\" data-bs-content=\"Your annotation<hr class=\'sep d-none\'>\" onclick=\"annotateShow(\'2021-05-07-02-20-07\')\" data-bs-original-title=\"[2021-05-07-02-20-07] Eddy IKHLEF\">icrobiome has high-imp</a></mark>act implications not only in humans, but also in livestock that sustain our current societal needs. In this context, we have tailored an acetylated galactoglucomannan (AcGGM) fibre to match unique enzymatic capabilities of <em>Roseburia</em> and <em>Faecalibacterium</em> species, both renowned butyrate-producing gut commensals. Here, we test the accuracy of AcGGM within the complex endogenous gut microbiome of pigs, wherein we resolve 355 metagenome-assembled genomes together with quantitative metaproteomes. In AcGGM-fed pigs, both target populations differentially express AcGGM-specific polysaccharide utilization loci, including novel, mannan-specific esterases that are critical to its deconstruction. However, AcGGM-inclusion also manifests a “butterfly effect”, whereby numerous metabolic changes and interdependent cross-feeding pathways occur in neighboring non-mannanolytic populations that produce short-chain fatty acids. Our findings show how intricate structural features and acetylation patterns of dietary fibre can be customized to specific bacterial populations, with potential to create greater modulatory effects at large.</p></span><span class=\"sec\"><strong class=\"kwd-title\">Subject terms: </strong><span class=\"kwd-text\">Bioinformatics, Metagenomics, Microbiome</span></span></span><span id=\"s\n1\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\n1title\">Introduction</h2><p id=\"Par3\" class=\"p p-first\">Microbiota-directed foods (MDFs) have emerged as a strategy to modulate the gut microbiome, as the diet has distinct and rapid effects on microbiome composition and function<sup><sup><a href=\"#CR1\" rid=\"CR1\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780187\">1</a></sup>,<sup><a href=\"#CR2\" rid=\"CR2\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780194\">2</a></sup></sup>. MDFs by definition are not broadly metabolized, but rather elicit a targeted metabolic response in specific indigenous microbiota that confers benefits to their host. This in itself presents a challenge; as many newly identified MDF target organisms, such as beneficial butyrate-producing (i.e., butyrogenic) <em>Roseburia</em> and <em>Faecalibacterium</em> spp.<sup><sup><a href=\"#CR3\" rid=\"CR3\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780192\">3</a></sup>,<sup><a href=\"#CR4\" rid=\"CR4\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780236\">4</a></sup></sup>, have broad metabolic capabilities that are shared with the vast majority of fiber-fermenting microbiota in the gut ecosystem. Nevertheless, recent studies have revealed intimate connections between the enzymatic and mechanistic features of microorganisms and the glycan structures of the fibers they consume<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780230\">5</a></sup>,<sup><a href=\"#CR6\" rid=\"CR6\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780193\">6</a></sup></sup>, which creates new conceptual MDF targets. This is exemplified by discoveries of sophisticated polysaccharide-degrading apparatuses that enable certain microbiota to consume fiber in a ‘selfish’ manner, whereby complex glycan structures (such as β-mannans) are cleaved into large oligosaccharides at the cell surface, which is subsequently transported into the cell and depolymerized into monomeric sugars<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780188\">5</a></sup>,<sup><a href=\"#CR7\" rid=\"CR7\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780244\">7</a></sup>,<sup><a href=\"#CR8\" rid=\"CR8\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780195\">8</a></sup></sup>. Such a mechanism restricts the release of sugars into the ecosystem for neighboring scavenging populations, thus giving a selective metabolic advantage to the selfish degrader in the presence of these highly complex glycans.</p><p id=\"Par4\"><mark id=\"mark_2021-05-07-07-04-26\" style=\"background-color: #ffff00;\"><a id=\"link_2021-05-07-07-04-26\" class=\"note\" data-bs-toggle=\"popover\" data-bs-trigger=\"hover focus\" data-placement=\"bottom\" data-bs-html=\"true\" title=\"\" data-bs-content=\"Your annotation<hr class=\'sep\'>1 Replies\" onclick=\"annotateShow(\'2021-05-07-07-04-26\')\" data-bs-original-title=\"[2021-05-07-07-04-26] Eddy IKHLEF\" aria-describedby=\"popover128918\">Beta-mannans are present in human </a></mark>and livestock diets, and depending on their plant origins, can be decorated with varying amounts of acetylation that protect the fiber from enzymatic degradation<sup><sup><a href=\"#CR9\" rid=\"CR9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780218\">9</a></sup></sup>. We recently demonstrated that the human gut commensal <em>Roseburia intestinalis</em> encodes a mannan-specific polysaccharide utilization locus (PUL), and ‘selfishly’ converts highly complex mannan substrates to butyrate<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780208\">5</a></sup></sup>. Within this mannan PUL, a carbohydrate esterase (CE) family 2 (<em>Ri</em>CE2) removes 3-<em>O</em>-, and 6-<em>O</em>-acetylations on mannan, whereas a novel CE family 17 (<em>Ri</em>CE17) removes the axially oriented 2-<em>O</em>-acetylations<sup><sup><a href=\"#CR9\" rid=\"CR9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780220\">9</a></sup></sup>, which are distinctive features found in limited mannan moieties and inaccessible to most of the characterized bacterial esterases present in the gut microbiome. Closer genome examinations have revealed that putative CE2/CE17-containing mannan PULs are in fact prominent within many butyrate-producers including <em>Roseburia</em> spp., <em>Faecalibacterium prausnitzii</em>, <em>Ruminococcus gnavus</em>, <em>Coprococcus eutactus</em> and <em>Butyrivibrio fibrisolvens</em><sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780205\">5</a></sup>,<sup><a href=\"#CR10\" rid=\"CR10\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780249\">10</a></sup></sup>. It is well known that the metabolic attributes of these populations are highly desirable in the gastrointestinal tract, and that their depletion is implicated in colorectal cancer, Crohn’s disease, inflammatory bowel syndrome, ulcerative colitis, forms of dermatitis, and several other diseases<sup><sup><a href=\"#CR11\" rid=\"CR11\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780238\">11</a></sup>,<sup><a href=\"#CR12\" rid=\"CR12\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780223\">12</a></sup></sup>. These collective findings thus raised the question: could a custom MDF fiber that was tailored to match these specialized enzymatic capabilities selectively engage butyrate-producers in a complex microbiome ecosystem?</p><p id=\"Par5\">2-<em>O-</em>acetylated mannans are found in a limited number of characterized western dietary fiber sources (i.e., tomatoes<sup><sup><a href=\"#CR13\" rid=\"CR13\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780186\">13</a></sup></sup> and coffee<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780197\">5</a></sup></sup>), however, 2-<em>O-</em>acetylations are present in acetylated galactoglucomannan (AcGGM), which is the main hemicellulose in the secondary cell wall of Norway spruce (<em>Picea abies</em>)<sup><sup><a href=\"#CR14\" rid=\"CR14\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780215\">14</a></sup></sup>. We have utilized a controlled steam explosion (SE), followed by ultrafiltration (UF) fractionation to extract complex AcGGM from spruce wood. Processing conditions were selected to tailor the fiber with a high degree of galactose branching’s and 2-<em>O-</em>, 3-<em>O-</em> and 6-<em>O-</em>acetylations<sup><sup><a href=\"#CR15\" rid=\"CR15\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780171\">15</a></sup></sup>, which is amenable to inclusion as an MDF in animal feed production. Previously, the MDF concept has matched polysaccharides with <em>Bacteroides</em>-encoded PULs to demonstrate the creation of exclusive metabolic niches<sup><sup><a href=\"#CR1\" rid=\"CR1\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780204\">1</a></sup>,<sup><a href=\"#CR16\" rid=\"CR16\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780229\">16</a></sup></sup>. In particular, Shepherd et al.<sup><sup><a href=\"#CR16\" rid=\"CR16\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780234\">16</a></sup></sup> engrafted exogenous strains in mice via their rare PUL-encoded enzymatic capabilities, whereas Patnode et al.<sup><sup><a href=\"#CR1\" rid=\"CR1\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780250\">1</a></sup></sup> illustrated that bioactive carbohydrates were found to target particular <em>Bacteroides</em> species in a defined consortium of 20 human gut microbial species in gnotobiotic mice. However, what is less understood from MDF studies to date, is (1) can the MDF concept be applied to target indigenous populations within a complex endogenous microbiome, and (2) what are the broader secondary community effects if a targeted population is stimulated, i.e., are new niches created and/or existing ones closed?</p><p id=\"Par6\" class=\"p p-last\">Here, we test whether our AcGGM fiber can specifically target beneficial Firmicutes species <em>Roseburia</em> and <em>Faecalibacterium</em> within a ‘real-world’ gut ecosystem that consists of 100–1000’s of different species. To evaluate this, we analyze the gut microbiomes of weaned piglets fed diets containing varying AcGGM levels over a 28-day period that extended from their first meal after sow separation until an adapted, fiber-degrading microbiome was established. Using metagenomics, we monitor temporal changes in the microbiome and phylogenetically and functionally resolve the genomes of indigenous microbiota. In parallel, our detailed quantitative metaproteomic and carbohydrate microarray analyses reveal the metabolic and enzymatic responses of the different microbiota to the varying AcGGM exposure. We demonstrate how the activity of specific beneficial microbiota can be directly stimulated while simultaneously deciphering the secondary, trophic effects on other populations and metabolic niches, with both aspects having broader implications for developing strategies to effectively modulate the gut microbiome.</p></span><span id=\"s\n2\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\n2title\">Results</h2><span id=\"s\n3\" class=\"sec sec-first\"><h3 id=\"s\n3title\">Production of highly complex dietary mannan fibers from wood</h3><p id=\"Par7\" class=\"p p-first-last\">Spruce galactoglucomannan consists of a backbone of β-(1,4)-linked mannose and glucose residues, decorated with α-(1,6) linked galactose branching, and a large degree of esterification of the mannose residues by 2-<em>O</em>- and 3-<em>O</em>- and 6-<em>O</em>- acetylations<sup><sup><a href=\"#CR14\" rid=\"CR14\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780209\">14</a></sup></sup> (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig1/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\" co-legend-rid=\"lgnd_Fig1\"><span>1a</span></a>). A crucial part of this study was the development of an efficient, large-scale extraction process entailing SE as well as ultra- and nanofiltration, which ultimately provided high quantities at high purity whilst not damaging the complexity of the AcGGM fiber (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig1/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\" co-legend-rid=\"lgnd_Fig1\"><span>1b, c</span></a>). A total of 700 kg of dry Norway spruce chips was processed using SE at conditions corresponding to a combined severity factor (<em>R</em>′<sub>0</sub>) of 1.70. We produced 50 kg of oligo/polysaccharides for feed production (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig1/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\" co-legend-rid=\"lgnd_Fig1\"><span>1c–e</span></a>), with a monosaccharide (Man:Glc:Gal) ratio of 4:1:0.6. The degree of polymerization (DP) of the AcGGM fiber ranged from β-manno-oligosaccharides with DP of 2–10 to manno-polysaccharides (DP of ≥11), with both exhibiting degrees of acetylation (DA = 0.35). Crucially, this DA value was higher than previous iterations of the fiber (DA = 0.28)<sup><sup><a href=\"#CR7\" rid=\"CR7\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780202\">7</a></sup></sup>, and its acetylation patterns (previously determined<sup><sup><a href=\"#CR9\" rid=\"CR9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780228\">9</a></sup></sup>) matched the enzymatic capabilities of mannan PULs encoded in human gut <em>Roseburia</em> and <em>Faecalibacterium</em> spp.<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780200\">5</a></sup>,<sup><a href=\"#CR10\" rid=\"CR10\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780198\">10</a></sup></sup>. We, therefore, predicted that our AcGGM fiber would match representatives of the same populations that are indigenous to porcine gut ecosystems<sup><sup><a href=\"#CR17\" rid=\"CR17\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780246\">17</a></sup>,<sup><a href=\"#CR18\" rid=\"CR18\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_915796626\">18</a></sup></sup> (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig1/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\" co-legend-rid=\"lgnd_Fig1\"><span>1f</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig1\" co-legend-rid=\"lgnd_Fig1\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig1/\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139884798147728\" class=\"figure\"></span></a><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7666174_41467_2020_19585_Fig1_HTML.jpg\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.\nObject name is 41467_2020_19585_Fig1_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7666174/bin/41467_2020_19585_Fig1_HTML.jpg\"></a></span><span id=\"largeobj_idm139884798147728\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig1/?report=objectonly\">Open in a separate window</a></span><span class=\"icnblk_cntnt\" id=\"lgnd_Fig1\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig1/\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\">Fig. 1</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Schematic representation and graphic illustration of the production pipeline for Norway spruce AcGGM.</strong><p id=\"__p1\"><strong>a</strong> Wood chips were milled to increase the surface area exposed for hydrothermal extraction and washing the released manno-oligosaccharides. Spruce wood contains long-chained galactoglucomannan (DP200-400), which is highly acetylated (DA ~ 0.65), predominantly with 2-<em>O</em> and 3-<em>O</em> acetylations<sup><sup><a href=\"#CR66\" rid=\"CR66\" class=\" bibr popnode\">66</a></sup></sup>. <strong>b</strong> During hydrothermal pretreatment, the release of acetic acid promoted hydrolysis of glycosidic bonds, thereby relinquishing monosaccharides, oligosaccharides, and other breakdown products. <strong>c</strong> Ultrafiltration retained the longer, complex oligosaccharides and discarded the monosaccharides, acetic acid, and other steam explosion byproducts. <strong>d</strong> The purified mannan was incorporated into feed pellets at varying inclusion levels, produced by a conventional feed pelleting process. <strong>e</strong> The growth performance experiment and a feeding trial were conducted in a randomized block design, with four inclusion levels of AcGGM. <strong>f</strong> Multi-omic approaches were used to analyze the porcine gut microbiome in response to AcGGM and determine if indigenous mannan PULs matched the glycan structure of the AcGGM fiber. Glycosidic bonds between the β-(1,4)-mannose and glucose units in the backbone of AcGGM require hydrolysis by glycoside hydrolases (GH) from families GH5 and GH26. GH36 and GH27 α-galactosidases are required to remove the α-(1,6)-galactose decorations. Single mannose and glucose units are removed from the non-reducing end of the oligosaccharides by enzymes from families GH1, GH2, GH3, GH113, and GH130, while mannose in the reducing end can be removed by GH113 family mannosidases. 3-<em>O</em>- and 6-<em>O</em>-acetylations on mannan are removed by esterases from family CE2. A unique feature of particular beta-mannans is the axially oriented 2-<em>O</em>-acetylation on mannose, which is the prevalent form of acetylation present on AcGGM used in this study. 2-<em>O</em>-acetylations are removed by esterases homologous to the RiCE17 mannan-specific esterase from <em>Roseburia intestinalis</em>, which was recently characterized by our group<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780189\">5</a></sup></sup>.</p></span></span></span></span><span id=\"s\n4\" class=\"sec\"><h3 id=\"s\n4title\">AcGGM altered the gut microbiome of weaned piglets</h3><p id=\"Par8\" class=\"p p-first\">We previously demonstrated that varieties of AcGGM can be metabolized by pure cultures of <em>Roseburia intestinalis</em> L1-82<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780256\">5</a></sup></sup>, in vitro enrichments with human gut butyrate-producers<sup><sup><a href=\"#CR10\" rid=\"CR10\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780224\">10</a></sup></sup> and ‘mini-microbiota’ within gnotobiotic mice<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780253\">5</a></sup></sup>. Here, we wanted to test our AcGGM fiber’s accuracy and ability to elicit a specific response in indigenous representatives of our target populations within a highly complex and competitive endogenous microbiome. In total, four separate cohorts of twelve weaned piglets were given a pelleted basal feed semi-ad libitum, which contained either 0% (control), 1, 2 or 4% AcGGM to additionally determine the level necessary to elicit an effect on both the host and its microbiome. We chose a dietary supplementation strategy, which is commonly applied to piglets during the post-weaning period to offset their particularly low feed intake and enhance gut health and growth performance<sup><sup><a href=\"#CR19\" rid=\"CR19\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780175\">19</a></sup></sup>. Cautious measures were made to eliminate potential pen biases (see details in materials and methods), and fecal samples, as well as animal performance metrics, were taken before AcGGM administration (when piglets were assigned to pens), and subsequently at days 7, 14, 21 and 27 during the feeding trial. On day 28, the piglets were sacrificed and host gut tissue and digesta samples taken from all intestinal regions (duodenum, jejunum, ileum, cecum, and colon) for down-stream analysis (Supplementary Table&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">1</a></sup> and Supplementary Data&nbsp;<sup><a href=\"#MOESM4\" rid=\"MOESM4\" class=\" media\">1</a></sup>). Measurements of major short-chain fatty acids (SCFAs) in the cecum and colon showed a trend of incremental increases of absolute and relative butyrate levels as AcGGM levels were increased (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig2/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig2\" rid-ob=\"ob-Fig2\" co-legend-rid=\"lgnd_Fig2\"><span>2</span></a>). However, despite SCFA data suggesting that AcGGM inclusion promotes butyrogenic fermentation, it also showed that there was no statistically significant increase in total SCFA levels (Supplementary Data&nbsp;<sup><a href=\"#MOESM4\" rid=\"MOESM4\" class=\" media\">1</a></sup>). Similarly, the levels of propionic acid were not affected by AcGGM inclusion (Supplementary Data&nbsp;<sup><a href=\"#MOESM4\" rid=\"MOESM4\" class=\" media\">1</a></sup>). While changes in SCFA and microbiome composition (Figs.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig2/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig2\" rid-ob=\"ob-Fig2\" co-legend-rid=\"lgnd_Fig2\"><span>2</span></a> and <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig3/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span style=\"position: relative;text-decoration:none;\">​<span class=\"figpopup-sensitive-area\" style=\"left: -1.5em;\">and3)</span></span><span>3</span></a>) resulted from AcGGM inclusion, we observed no adverse effects on the host’s physiology, with the average weight, feed conversion ratio, blood cell composition, T cell population, and colon morphology not differing between the control and AcGGM treatments (Supplementary Fig.&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">1</a></sup> and Supplementary Data&nbsp;<sup><a href=\"#MOESM4\" rid=\"MOESM4\" class=\" media\">1</a></sup>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig2\" co-legend-rid=\"lgnd_Fig2\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig2/\" rid-figpopup=\"Fig2\" rid-ob=\"ob-Fig2\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139884803744688\" class=\"figure\"></span></a><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7666174_41467_2020_19585_Fig2_HTML.jpg\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.\nObject name is 41467_2020_19585_Fig2_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7666174/bin/41467_2020_19585_Fig2_HTML.jpg\"></a></span><span id=\"largeobj_idm139884803744688\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig2/?report=objectonly\">Open in a separate window</a></span><span class=\"icnblk_cntnt\" id=\"lgnd_Fig2\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig2/\" rid-figpopup=\"Fig2\" rid-ob=\"ob-Fig2\">Fig. 2</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Butyrate detected in the cecum and colon digesta of pigs fed the four different AcGGM diets with varying inclusion levels (0–4%).</strong><p id=\"__p2\"><strong>a</strong> display absolute levels and <strong>b</strong> relative levels of butyrate. Raw SCFA measurements are presented in Supplementary Data&nbsp;<sup><a href=\"#MOESM4\" rid=\"MOESM4\" class=\" media\">1</a></sup>. Bars represent median and boxes interquartile range determined from 12 animals analyzed per dietary group. Data were analyzed using a two-tailed <em>t</em>-test, with <em>p</em>-values indicated between pigs fed either the control or 4% AcGGM diets. The boxes span the 25th–75th percentiles with the central bars being the medians. Whiskers extend maximum up to 1.5× the interquartile range (IQR) or, when all values are within 1.5× IQR, then the whisker extends to the most extreme data point.</p></span></span></span><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig3\" co-legend-rid=\"lgnd_Fig3\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig3/\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139884837283936\" class=\"figure\"></span></a><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7666174_41467_2020_19585_Fig3_HTML.jpg\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.\nObject name is 41467_2020_19585_Fig3_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7666174/bin/41467_2020_19585_Fig3_HTML.jpg\"></a></span><span id=\"largeobj_idm139884837283936\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig3/?report=objectonly\">Open in a separate window</a></span><span class=\"icnblk_cntnt\" id=\"lgnd_Fig3\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig3/\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\">Fig. 3</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Effect of AcGGM-containing diets on the pig gut microbiome.</strong><p id=\"__p3\">16S rRNA gene amplicon and shotgun metagenomic analysis was used to monitor the effect the AcGGM fiber had on the gut microbiome structure of weaned piglets, determined from 12 animals analyzed per dietary group. Both 16S rRNA and MAG relative abundance data were analyzed using a two-tailed <em>t</em>-test, with significant increases (red) and decreases (blue) indicated between pigs fed either the control or AcGGM diets (* denotes <em>p</em> &lt; 0.05, ** denotes <em>p</em> &lt; 0.001). The boxes span the 25th–75th percentiles with the central bars being the medians. Whiskers extend maximum up to 1.5× the interquartile range (IQR) or, when all values are within 1.5× IQR, then the whisker extends to the most extreme data point. <strong>a</strong> Ordination plots of Bray–Curtis distances between microbial communities from pigs feed either the control or AcGGM diets (at varying inclusion levels: 1, 2 or 4%, 12 animals per dietary group). Samples were collected at day 28 of the feeding trials from various sections of the small and large intestine. The AcGGM-effect was more pronounced in the fiber-fermenting distal regions (cecum, colon) of the gut. <strong>b</strong> Relative 16S rRNA gene abundance of <em>Roseburia</em>-affiliated phylotypes (agglomerated at the genus level) in either cecum or colon samples collected from pigs fed AcGGM diets with varying inclusion levels (cecum: 0% vs 4% <em>p</em> = 0.0004, colon: 0% vs 4% <em>p</em> = 0.034). MAG relative abundances were determined from colon samples only, collected from pigs fed either the control or 4% AcGGM diets (0% vs 4% <em>p</em>-value: MAG041 = 0.002, MAG179 = 0.028, MAG261 = 0.001, MAG297 = 0.014, MAG269 = 0.035). Despite being suspected as an active mannan degrader, 16S rRNA gene abundance of <em>Roseburia</em>-affiliated phylotypes decreased with increasing % AcGGM. However, MAG relative abundance analysis of <em>Roseburia</em> populations showed that specific phylotypes that encoded AcGGM-specific PULs (indicated by green text) were indeed stimulated by the AcGGM fiber. <strong>c</strong> The relative 16S rRNA gene abundance of the genus <em>Faecalibacterium</em> was enhanced by the inclusion of AcGGM (cecum: 0% vs 2% <em>p</em> = 0.008, 0% vs 4% <em>p</em> = 2.11E−05, colon: 0% vs 2% <em>p</em> = 0.046, 0% vs 4% <em>p</em> = 4.64E−06), as were affiliated MAGs in the colon (0% vs 4% <em>p</em>-value: MAG243 = 5.84E−06). The same trends were observed for phylotypes affiliated to <em>Catenibacterium</em> (<strong>d</strong> cecum: 0% vs 2% <em>p</em> = 0.006, 0% vs 4% <em>p</em> = 1.12E−05, colon: 0% vs 2% <em>p</em> = 0.0007, 0% vs 4% <em>p</em> = 1.08E−06, MAG048 = 1.52E−06), Dialister (<strong>e</strong> cecum: 0% vs 2% <em>p</em> = 0.013, 0% vs 4% <em>p</em> = 3.29E−09, colon: 0% vs 1% <em>p</em> = 0.032, 0% vs 4% <em>p</em> = 1.82E−07, MAG150 = 2.07E−09), Megasphaera (<strong>f</strong> cecum: 0% vs 4% <em>p</em> = 0.031, colon: 0% vs 4% MAG053 = 0.0011) and Prevotella group 9 (<strong>g</strong> cecum: 0% vs 2% <em>p</em> = 0.003, 0% vs 4% <em>p</em> = 4.71E−07, colon: 0 vs 2% <em>p</em> = 0.010, 0% vs 4% <em>p</em> = 3.96E−06, MAG045 = 0.008, MAG196 = 0.010, MAG138 = 0.0018, MAG285 = 0.0024, MAG137 = 6.99E−05, MAG191 = 0.0014). Due to space constraints, only a partial representation of Prevotella-affiliated MAG abundances is illustrated in <strong>g</strong>.</p></span></span></span><p id=\"Par9\">Monitoring of temporal microbiome changes using 16S rRNA gene analysis over the month-long trial indicated that the inclusion of AcGGM into the piglets feed caused a pronounced shift in the microbiome structural composition from the 21st day of the trial onwards (Supplementary Figs.&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">2b</a></sup> and <sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3</a></sup>). Spatial changes were also examined at the final sampling day and showed typical patterns, whereby the structure of the gut microbiome varied by gut region (Supplementary Figs.&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">2a</a></sup>, <sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3</a></sup>, and <sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">4</a></sup>). As expected, the AcGGM-effect was more pronounced in the fiber-fermenting distal regions (cecum, colon) of the gut, where the relative abundance of hundreds of phylotypes was observed to change (adjusted <em>p</em> &lt; 0.05) in response to varying inclusion levels (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig3/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3a</span></a> and Supplementary Data&nbsp;<sup><a href=\"#MOESM5\" rid=\"MOESM5\" class=\" media\">2</a></sup>). To determine the effect AcGGM had on microbiome function, we also closely examined the 355 metagenome-assembled genomes (MAGs, Supplementary Fig.&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">5a</a></sup>) that were reconstructed from metagenomic data generated from the colon samples of each pig fed the control and 4% AcGGM diets (Supplementary Table&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">2</a></sup>). The taxonomic affiliation of the MAGs was calculated via GTDB-Tk (Supplementary Data&nbsp;<sup><a href=\"#MOESM6\" rid=\"MOESM6\" class=\" media\">3</a></sup>), while phylogeny was inferred from a concatenated ribosomal protein tree (Newick format available in Supplementary Data&nbsp;<sup><a href=\"#MOESM7\" rid=\"MOESM7\" class=\" media\">4</a></sup>) that was constructed using MAGs from this study and 293 closely related reference genomes.</p><p id=\"Par10\" class=\"p p-last\">Our target butyrogenic populations produced mixed results, whereby the 16S rRNA gene relative abundance of <em>Faecalibacterium</em> affiliated phylotypes increased in response to increasing levels of AcGGM (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig3/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3c</span></a> and Supplementary Fig.&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3c</a></sup>), whereas <em>Roseburia</em>-affiliated phylotypes seemingly decreased (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig3/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3b</span></a> and Supplementary Fig.&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3b</a></sup>). However, a detailed analysis of <em>Roseburia-</em>affiliated MAGs (GTDB-Tk assigned as <em>Agathobacter</em>, see Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig4/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4</span></a>) showed that specific phylotypes that encoded AcGGM-specific PULs were indeed stimulated by the AcGGM fiber (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig3/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3b</span></a>). Reputable fiber-fermenting populations affiliated to <em>Prevotella</em><sup><sup><a href=\"#CR20\" rid=\"CR20\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780235\">20</a></sup></sup> also showed varying responses (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig3/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3g</span></a>), with 16S rRNA gene relative abundance of individual phylotypes increasing from 4 to 12% between the control and 4% AcGGM inclusion in both colon and cecum (Supplementary Fig.&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3a</a></sup>). Interestingly, relative abundance estimates of both 16S rRNA gene OTUs and MAGs indicated that phylotypes affiliated to non-fiber-degrading taxa, such as <em>Catenibacterium</em><sup><sup><a href=\"#CR21\" rid=\"CR21\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_915796625\">21</a></sup></sup>, <em>Dialister</em><sup><sup><a href=\"#CR22\" rid=\"CR22\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780177\">22</a></sup></sup>, and <em>Megasphaera</em>, demonstrated some of the highest dose-dependent increases in relative abundance in response to AcGGM (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig3/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3d–f</span></a>), indicating that other underlying factors are likely dictating microbiome structure, besides fiber degradation.</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig4\" co-legend-rid=\"lgnd_Fig4\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig4/\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139884800926576\" class=\"figure\"></span></a><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7666174_41467_2020_19585_Fig4_HTML.jpg\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.\nObject name is 41467_2020_19585_Fig4_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7666174/bin/41467_2020_19585_Fig4_HTML.jpg\"></a></span><span id=\"largeobj_idm139884800926576\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig4/?report=objectonly\">Open in a separate window</a></span><span class=\"icnblk_cntnt\" id=\"lgnd_Fig4\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig4/\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\">Fig. 4</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Genome-centric metaproteomic analysis of colon samples collected from pigs fed either the control of 4% AcGGM diet.</strong><p id=\"__p4\"><strong>a</strong> Principle Component Analysis of metaproteomes generated from randomly selected colon samples collected from pigs fed either the control (C1-4) or 4% AcGGM (M1-4) diet. A clear separation was observed between the two diets, highlighting that the detected proteins in the colon microbiomes from piglets fed the control, and 4% AcGGM diets were distinct. <strong>b</strong> Hierarchal clustering and heatmap of detected protein group abundance profiles. Rows are scaled so that red represents the highest abundance for that protein group and blue the lowest. Five different clusters were observed, with protein groups differentially detected in AcGGM-fed pigs (M1-4: red), control pigs (C1-4: blue), all pigs (M1-4 + C1-4: purple), AcGGM-fed pigs plus one control (M1-4 + C4: brown) and only in individual pigs (Individual: green). <strong>c</strong> Phylogeny and metaproteomic detection of 355 MAGs sampled from the colon of weaned piglets. This maximum likelihood tree is based on an alignment of 22 concatenated ribosomal proteins from the 355 MAGs reconstructed in this study from 24 colon metagenomes (12 control pigs, 12 fed 4% AcGGM diet), in addition to 239 reference genomes closely related to the MAGs. Branches are shaded with color to highlight phylum-level affiliations (see legend). Colored bars on the outside of the tree depict the average number of unique and shared protein groups detected for each MAG in four randomly selected samples that were analyzed from pigs fed either the control- (gray) or 4% AcGGM diet (green). Total number of proteins for each MAG in each pig is detailed in Supplementary Data&nbsp;<sup><a href=\"#MOESM8\" rid=\"MOESM8\" class=\" media\">5</a></sup>. Purple circles on the inside of the tree represent nodes with bootstrap support ≥70%, relative to size. MAG041 and MAG243 were found to encode CE2/CE17-containing mannan PULs (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig5/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span>5</span></a>) are indicated by *. All MAGs depicted in Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig6/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\" co-legend-rid=\"lgnd_Fig6\"><span>6</span></a> (blue circles) and Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig7/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span>7</span></a> (red circles) are listed, with average detected protein counts in both diets, which metaproteomic expression cluster they are enriched in (part <strong>b</strong>, adjusted <em>p</em>-values are indicated: * denotes <em>p</em> &lt; 0.05, ** denotes <em>p</em> &lt; 0.001, exact values listed in Supplementary Data&nbsp;<sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">7</a></sup>), and their MAG taxonomic affiliation determined via GTDB-Tk. MAG enrichment analysis was performed using the hypergeometric distribution function phyper in R with the false discovery rate controlled at 5% using the function <em>p</em>.adjust with method = ‘BH’. Recently reclassified <em>Roseburia</em> species are denoted as Agathobacter. The full tree in Newick format is provided in Supplementary Data&nbsp;<sup><a href=\"#MOESM7\" rid=\"MOESM7\" class=\" media\">4</a></sup>.</p></span></span><span id=\"s\n5\" class=\"sec\"><h3 id=\"s\n5title\">Targeted mannan PULs were detected in the colon of AcGGM-fed pigs</h3><p id=\"Par11\" class=\"p p-first\">Because our primary goal was to elucidate whether our target, butyrogenic populations were activated in response to AcGGM, we annotated MAGs affiliated to butyrate-producers and conducted the metaproteomic analysis with label-free quantification (LFQ) on randomly selected colon samples from four control and four 4% AcGGM-fed pigs (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig4/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4a, b</span></a>), and mapped 8515 detected protein groups back to our MAGs to identify functionally active populations (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig4/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4c</span></a>) (Supplementary Data&nbsp;5 and <sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). Community-wide analysis of the MAG genetic content (Supplementary Fig.&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">5b</a></sup>) from each sample and clustering analysis of their detected proteins (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig4/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4a, b</span></a>) further supported our 16S rRNA gene analysis, reiterating that the microbiomes from piglets fed the control and 4% AcGGM diets were distinct.</p><p id=\"Par12\">Our MAG-centric multi-omic approach gave clear indications as to what effect the AcGGM fiber had on putative butyrogenic <em>Roseburia</em> and <em>Faecalibacterium</em> populations in the distal gut of pigs. Fifteen MAGs clustered with representative <em>Roseburia</em> spp. genomes (Figs.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig3/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3b</span></a> and <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig4/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span style=\"position: relative;text-decoration:none;\">​<span class=\"figpopup-sensitive-area\" style=\"left: -1.5em;\">and4),</span></span><span>4</span></a>), which reflected the multiple <em>Roseburia</em>-affiliated phylotypes that were predicted with our 16S rRNA gene analysis (Supplementary Fig.&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3b</a></sup> and Supplementary Data&nbsp;<sup><a href=\"#MOESM5\" rid=\"MOESM5\" class=\" media\">2</a></sup>). In general, the relative abundance of <em>Roseburia</em>-affiliated MAGs (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig3/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3b</span></a>) and detected proteins (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig4/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4c</span></a>) were observed at either static or lower levels in AcGGM-fed pigs, reiterating our initial 16S rRNA gene observations that AcGGM negatively affected <em>Roseburia</em> populations (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig3/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3b</span></a>). However, one specific <em>Roseburia</em>-affiliated population (MAG041) was detected at significantly higher abundance levels (<em>p</em> = 0.0016) (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig3/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3b</span></a>), and its detected proteins were enriched (adj. <em>p</em> = 0.0034) in the 4% AcGGM pig samples compared to the control (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig4/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4b, c</span></a> and Supplementary Data&nbsp;<sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">7</a></sup>).</p><p id=\"Par13\">Closer examination of MAG041 revealed a putative CE2/CE17-containing mannan-degrading PUL that was absent in the other <em>Roseburia</em>-affiliated MAGs and was differentially expressed in the AcGGM diet (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig5/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span>5</span></a>). Importantly, the MAG041 mannan PUL exhibited gene synteny to the <em>R. intestinalis</em> strain L1-82 PUL whose biochemical properties we recently characterized in detail<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780255\">5</a></sup></sup> (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig5/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span>5</span></a>). The predicted multi-modular mannanase (CBM27-GH26-CBM23) in the MAG041 mannan PUL is homologous to the GH26 in <em>R. intestinalis</em> L1-82 (48% identity over 87% of the sequence), and can be presumed to fulfill the same function—‘selfishly’ breaking down AcGGM fibers at the cell surface prior to intracellular transport. Besides the detection of GH26 and esterases in AcGGM-fed pigs, other mannan-specific enzymes also responded to the dietary shift within the MAG041 mannan PUL, including a phosphoglucomutase, a multiple-sugar binding protein, a GH130.1 4-<em>O</em>-β-<span class=\"small-caps\">d</span>-mannosyl-<span class=\"small-caps\">d</span>-glucose phosphorylase and a GH130.2 β-1,4-manno-oligosaccharide phosphorylase (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig5/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span>5</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig5\" co-legend-rid=\"lgnd_Fig5\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig5/\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139884797373888\" class=\"figure\"></span></a><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7666174_41467_2020_19585_Fig5_HTML.jpg\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.\nObject name is 41467_2020_19585_Fig5_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7666174/bin/41467_2020_19585_Fig5_HTML.jpg\"></a></span><span id=\"largeobj_idm139884797373888\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig5/?report=objectonly\">Open in a separate window</a></span><span class=\"icnblk_cntnt\" id=\"lgnd_Fig5\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig5/\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\">Fig. 5</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Metaproteomic detection of CE2/CE17-containing mannan PULs encoded in <em>Roseburia</em>- (MAG041) and <em>Faecalibacterium</em>- (MAG243) affiliated MAGs in pigs fed with either the control or 4% AcGGM diet.</strong><p id=\"__p5\">Predicted gene organization and annotated gene function is color-coded and largely derived from the previous biochemical and structural characterization of the mannan degradation cluster (characterized genes indicated with *) in <em>R. intestinalis</em> L1-82<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780247\">5</a></sup></sup>. Gene synteny and identity % between mannan PULs found in <em>R. intestinalis</em> L1-82, MAG041 and MAG243 are indicated in gray boxes. Heat maps above detected enzymes show the LFQ detection levels for the four replicates sampled in control and 4% AcGGM-fed pigs. LFQ values of proteins from both clusters are in Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>. The predicted multi-modular mannanase (CBM27-GH26-CBM23) from MAG041 was the only extracellular protein in the locus, and the only extracellular mannanase expressed in response to AcGGM inclusion.</p></span></span></span><p id=\"Par14\" class=\"p p-last\">In contrast to <em>Roseburia</em>-affiliated MAGs, only one MAG clustered with <em>F. prausnitzii</em> (MAG243, Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig4/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4</span></a> and Supplementary Data&nbsp;<sup><a href=\"#MOESM6\" rid=\"MOESM6\" class=\" media\">3</a></sup>), implying that the multiple phylotypes that were predicted with our 16S rRNA gene data (Supplementary Fig.&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3c</a></sup> and Supplementary Data&nbsp;<sup><a href=\"#MOESM5\" rid=\"MOESM5\" class=\" media\">2</a></sup>) encode high genome similarity and coverage and thus likely co-assembled into a representative population-level MAG. The relative abundance of MAG243 in the distal gut of pigs fed 4% AcGGM increased ~5-fold to ~1.24% (<em>p</em> = 5.84E−06) (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig3/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3c</span></a>), while metaproteomic hierarchical clustering analysis showed that MAG243 proteins were enriched in the same samples (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig4/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4b, c</span></a> and Supplementary Data&nbsp;<sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">7</a></sup>). Similar to MAG041, MAG243 was found to encode a CE2/CE17-containing mannan PUL, which was broadly detectable in the presence of AcGGM but absent in the control samples (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig5/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span>5</span></a> and Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). While the MAG243 mannan PUL contained two GH130 manno-oligophosphorylases, a mannose 6-phosphate isomerase, phosphoglucomutase, and two carbohydrate esterase (CE17 and CE2), it lacked a GH26 mannanase representative, which suggests that <em>F. prausnitzii</em> is likely preferentially targeting the shorter acetylated manno-oligosaccharides that form part of the AcGGM structure (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig1/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\" co-legend-rid=\"lgnd_Fig1\"><span>1</span></a>). In addition to the mannan PULs of MAG041 and MAG243 being activated in AcGGM-fed pigs, their butyrogenic pathways were also detected at high levels, based on label-free quantification (LFQ) scores of detected proteins (Figs.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig6/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\" co-legend-rid=\"lgnd_Fig6\"><span>6</span></a>, <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig7/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span style=\"position: relative;text-decoration:none;\">​<span class=\"figpopup-sensitive-area\" style=\"left: -0.5em;\">,7</span></span><span>7</span></a> and Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>), suggesting that both populations can convert mannan to butyrate (Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig6\" co-legend-rid=\"lgnd_Fig6\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig6/\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139884802511776\" class=\"figure\"></span></a><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7666174_41467_2020_19585_Fig6_HTML.jpg\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.\nObject name is 41467_2020_19585_Fig6_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7666174/bin/41467_2020_19585_Fig6_HTML.jpg\"></a></span><span id=\"largeobj_idm139884802511776\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig6/?report=objectonly\">Open in a separate window</a></span><span class=\"icnblk_cntnt\" id=\"lgnd_Fig6\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig6/\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\">Fig. 6</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>The differing metabolisms of butyrate-producers in the colon microbiome of pigs fed AcGGM.</strong><p id=\"__p6\">A plethora of MAGs affiliated with putative butyrate-producers was recovered from the pig colon metagenome, which varied in their fiber-degrading capacity and detected activity in either the control (C) or 4% AcGGM (M) diets. Here, we present a representative selection of populations that: encoded specific AcGGM-degrading capabilities (MAG041 and MAG243), were abundant across both diets with no apparent effect (MAG292), were significantly enriched in pigs fed the control diet (MAG133, MAG324, and MAG269), and encoded an alternative pathway for butyrate production (via Butyrate kinase) (MAG324). GTBD-Tk inferred taxonomy (recently reclassified <em>Roseburia</em> species are denoted as <em>Agathobacter</em>), MAG relative abundance (control: black circles, AcGGM: green circles, <em>p</em>-value: MAG041 = 0.002, MAG243 = 5.84E−06, MAG292 = 0.98, MAG133 = 0.051, MAG324 = 0.120, MAG269 = 0.035) and MAG-centric metaproteomic enrichment analysis is indicated (see Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig4/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4c</span></a>, significant differences denoted by adjusted <em>p</em>-values: *<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.001, MAG041 = 3.4E−03, MAG243 = 8.5E−19, MAG292 = 5.2E−06, MAG133 = 2.8E−18, MAG324 = 4.9E−02, MAG269 = 1.9E−05). MAG enrichment analysis was performed using the hypergeometric distribution function phyper in R with the false discovery rate controlled at 5% using the function <em>p</em>.adjust with method = ‘BH’. CAZymes (GH, CE, and GT) involved in the catabolism of the listed polysaccharides are grouped according to box color and their specific activities are described on <a href=\"http://www.cazy.org\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">www.cazy.org</a>. Enzymes associated with acetate and butyrate metabolic pathways are listed as EC (enzyme commission) numbers. Corresponding functions can be found on <a href=\"https://www.genome.jp/kegg/kegg2.html\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://www.genome.jp/kegg/kegg2.html</a>.</p></span></span><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig7\" co-legend-rid=\"lgnd_Fig7\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig7/\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139884802505120\" class=\"figure\"></span></a><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7666174_41467_2020_19585_Fig7_HTML.jpg\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.\nObject name is 41467_2020_19585_Fig7_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7666174/bin/41467_2020_19585_Fig7_HTML.jpg\"></a></span><span id=\"largeobj_idm139884802505120\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig7/?report=objectonly\">Open in a separate window</a></span><span class=\"icnblk_cntnt\" id=\"lgnd_Fig7\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig7/\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\">Fig. 7</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Selected metabolic features of the porcine colon microbiome in response to AcGGM dietary intervention, as inferred from genome and proteome comparisons.</strong><p id=\"__p7\">The different metabolic pathways (fiber deconstruction, glycolysis, pentose-phosphate pathway, and SCFA production) are displayed for each population MAG. Graphical representation of pathways, enzymes, CAZymes, and cellular features are based on functional annotations that are depicted as numbered or abbreviated gene boxes, which are additionally listed in Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>. Metaproteomic analysis (detected genes and enzyme complexes) is highlighted according to the different LFQ values of the detected proteins sampled at a single time-point (28 days) either from animals fed the control or 4% AcGGM diet (green perforated boxes). The main dietary fibers (starches, xylans, glucans, and mannans), SCFAs (butyrate, acetate, propionate), and intermediates like lactate and succinate are represented by large colored arrows. GTBD-Tk inferred taxonomy is included (recently reclassified <em>Roseburia</em> species are denoted as Agathobacter). Gene names and abbreviations are also provided in Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>.</p></span></span><span id=\"s\n6\" class=\"sec\"><h3 id=\"s\n6title\">Specific removal of acetylations is key to access AcGGM</h3><p id=\"Par15\" class=\"p p-first-last\">A crucial step in the utilization of mannans as an energy source is the deacetylation of 2-<em>O</em>-, 3-<em>O</em>- and 6-<em>O</em>-mannose residues, which allows the subsequent breakdown of the sugar-containing backbone of the fiber. In <em>R. intestinalis</em> L1-82, AcGGM deacetylation occurs via the synergistic actions of two carbohydrate esterases (<em>Ri</em>CE2 and <em>Ri</em>CE17) that exert complementary specificities<sup><sup><a href=\"#CR9\" rid=\"CR9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780203\">9</a></sup></sup>. MAG041 and MAG243 both encoded CE2 homologs within their mannan PULs, sharing 63 and 31% identity (respectively) to <em>Ri</em>CE2, which has demonstrated activity on 3-<em>O</em>-, (4-<em>O</em>-) and 6-<em>O</em>-acetylations, and is mannan specific<sup><sup><a href=\"#CR9\" rid=\"CR9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780191\">9</a></sup></sup>. For CE17, MAG041 and MAG243 homologs shared 65 and 46% identity (respectively) with <em>Ri</em>CE17, including the active site residues and the aromatic stacking tryptophan (Trp326), which in <em>Ri</em>CE17 are associated with 2-<em>O</em>-acetylation specificity<sup><sup><a href=\"#CR9\" rid=\"CR9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780196\">9</a></sup></sup>. Broader screens of our MAG data revealed other CE2/CE17-containing PULs within Firmicute-affiliated MAGs from the pig colon microbiome (Supplementary Fig.&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">6</a></sup>), however, aside from MAG041 and MAG243, they originated from populations whose MAG relative abundance was very low (&lt;0.05%) and metabolic activity was undetectable via metaproteomics in any of the control or AcGGM diets (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig4/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4</span></a>). Finally, the differential proteomic detection of MAG041 and MAG243 CEs in pigs fed AcGGM diets (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig5/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span>5</span></a>), strengthened our hypothesis that both these populations can accommodate the unique features of the AcGGM fiber and are actively engaging in its utilization in vivo.</p></span><span id=\"s\n7\" class=\"sec sec-last\"><h3 id=\"s\n7title\">AcGGM also causes an effect in non-target populations</h3><p id=\"Par16\" class=\"p p-first\">MDFs studies to date have eloquently highlighted that metabolic symmetry between individual fibers and microbiota can be used to stimulate specific populations<sup><sup><a href=\"#CR1\" rid=\"CR1\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780245\">1</a></sup>,<sup><a href=\"#CR16\" rid=\"CR16\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780179\">16</a></sup></sup>. However, deeper microbiome effects that result from a target species being enriched via MDFs are poorly understood. Although the specificity of the AcGGM fiber matched selected mechanistic features of our target populations, our data showed that AcGGM dietary intervention reverberated further down the microbial trophic networks that support the conversion of dietary fiber into keystone SCFAs that are of nutritional value to the host animal. In particular, we observed varying effects upon different non-AcGGM-degrading butyrogenic populations belonging to clostridial cluster XIVa (Lachnospiraceae: <em>Roseburia</em> spp., <em>Eubacterium rectale</em>, <em>Butyrivibrio</em> and <em>Pseudobutyrivibrio</em>) and cluster IV (Ruminococcaceae: <em>Faecalibacterium prausnitzii</em>-related bacteria)<sup><sup><a href=\"#CR23\" rid=\"CR23\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780199\">23</a></sup></sup> (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig6/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\" co-legend-rid=\"lgnd_Fig6\"><span>6</span></a> and Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). Specific populations that were closely related to AcGGM-degrading MAG041 were found to be either unaffected (MAG292) or metabolically suppressed (MAG133) in the presence of 4% AcGGM. Indeed, detected proteins from MAG133 were enriched in pigs fed the control diet (adj. <em>p</em> = 2.8E−18, Supplementary Data&nbsp;<sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">7</a></sup>), and enzymes associated with butyrate production were largely undetected in 4% AcGGM-fed pigs (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig6/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\" co-legend-rid=\"lgnd_Fig6\"><span>6</span></a> and Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). In most cases, non-AcGGM degrading, butyrogenic populations were seemingly utilizing sugars found in starch, arabinoxylan, and/or arabinogalactan fibers that were detected in the basal feed components using Micro Array Polymer Profiling (MAPP; Supplementary Data&nbsp;<sup><a href=\"#MOESM11\" rid=\"MOESM11\" class=\" media\">8</a></sup>). A broader analysis of our omic data identified multiple abundant populations with similar metabolic capabilities, suggesting that suppressed butyrogenic populations were possibly being outcompeted by Prevotella-affiliated populations such as MAG285 (see below). While SCFA measurements indicated an increase in relative butyrate levels in AcGGM-fed pigs (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig2/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig2\" rid-ob=\"ob-Fig2\" co-legend-rid=\"lgnd_Fig2\"><span>2b</span></a>), the indirect effects of our MDF on this dynamic functional group are likely dampening an even greater influence of AcGGM-stimulated <em>Roseburia</em> and <em>Faecalibacterium</em> populations upon the broader microbiome and host animal (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig6/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\" co-legend-rid=\"lgnd_Fig6\"><span>6</span></a>).</p><p id=\"Par17\">Besides butyrate-producers, MAG-centric metaproteomic enrichment analysis showed that fiber-degrading <em>Prevotella</em>-affiliated populations experienced contrasting effects as a result of AcGGM inclusion. For example, MAG191 was found to account for the highest levels of detectable proteins in our datasets (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig4/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4c</span></a> and Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>), which were enriched in clusters differently detected in the presence of 4% AcGGM (adj. <em>p</em> = 0.0023, Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). Pathway annotation of AcGGM-enriched <em>Prevotella</em> populations (such as MAG191, MAG196, MAG285, see Figs.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig4/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4c</span></a>, <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig7/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span style=\"position: relative;text-decoration:none;\">​<span class=\"figpopup-sensitive-area\" style=\"left: -0.5em;\">,7,</span></span><span>7</span></a>, and Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>, <sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">7</a></sup>) indicated active metabolism of dietary fibers such arabinoxylans, starch, glucans (e.g., cellobiose), α-galactans, and mannose sugars (detected via MAPP analysis, Supplementary Data&nbsp;<sup><a href=\"#MOESM11\" rid=\"MOESM11\" class=\" media\">8</a></sup>) as well as acetate, succinate and/or propionate production, which were all detected with higher LFQ scores in AcGGM-fed pigs (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig7/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span>7</span></a> and Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). However, many CAZymes and fermentation enzymes from the aforementioned MAGs were also detected across both diets (albeit at varying LFQ values), while other <em>Prevotella</em> populations were specifically enriched in pigs fed the control diet (i.e., MAG034: adj. <em>p</em> = 2.0E−04, Supplementary Data&nbsp;<sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">7</a></sup>), suggesting that <em>Prevotella</em>-driven baseline consumption of basal feed fiber was occurring irrespective of AcGGM inclusion (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig7/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span>7</span></a> and Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>).</p><p id=\"Par18\">Several mannan-targeting PULs were identified in <em>Prevotella</em>-affiliated MAGs that were configured in an archetypical ‘Bacteroidetes-format’, which combines outer-membrane transport and carbohydrate-binding SusC/D-like proteins as well as CAZymes<sup><sup><a href=\"#CR24\" rid=\"CR24\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780221\">24</a></sup></sup> (Supplementary Fig.&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">6a</a></sup>). In particular, a PUL recovered from MAG196 encoded predicted SusC/D-like proteins, mannanases (GH26, GH5_7), mannosyl-phosphorylases (GH130), and an esterase, although neither the mannanases nor the esterase was detected in the metaproteomes recovered from the AcGGM-fed pigs (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig7/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span>7</span></a>, Supplementary Fig.&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">6</a></sup> and Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). In addition, we speculate that MAG196 and MAG191 are perhaps capable of metabolizing elements of the AcGGM fiber such as the α-galactose side-chain or deacetylated manno-oligosaccharides, which was inferred via detected GH36 and GH130 representatives (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig7/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span>7</span></a>, Supplementary Fig.&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">6</a></sup> and Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>).</p><p id=\"Par19\">Dietary inclusion of AcGGM also unexpectedly led to pronounced metabolic shifts in numerous phenotypically and phylogenetically diverse microbial populations that are not normally associated with fiber hydrolysis. Mirroring our 16S rRNA gene analysis, MAGs affiliated to the genera <em>Dialister</em> (MAG150), <em>Catenibacterium</em> (MAG048), and <em>Megasphaera</em> (MAG053) were the three highest enriched populations in expression clusters differentially detected in the AcGGM diet (adj. <em>p</em> &lt; 0.001), although none were found to encode CE2/CE17-containing mannan PULs (Figs.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig3/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3</span></a>–<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig4/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4</span></a>, <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig7/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span style=\"position: relative;text-decoration:none;\">​<span class=\"figpopup-sensitive-area\" style=\"left: -0.5em;\">,7</span></span><span>7</span></a> and Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). In particular, the MAG048 proteome included a putative sugar phosphotransferase system (PTS) and GH1 phospho-β-glucosidases (EC 3.2.1.86) that are predicted to catalyze the phosphorylation of disaccharides (such as cellobiose and mannobiose<sup><sup><a href=\"#CR25\" rid=\"CR25\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780226\">25</a></sup>,<sup><a href=\"#CR26\" rid=\"CR26\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780207\">26</a></sup></sup>) and hydrolyze the PTS-transported sugars into <span class=\"small-caps\">d</span>-glucose and <span class=\"small-caps\">d</span>-glucose 6-phosphate. Concomitantly, glycolysis, and acetogenesis pathways from MAG048 were also highly detected in AcGGM-fed pigs (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig7/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span>7</span></a> and Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). Together, these data suggest that certain <em>Catenibacterium</em> populations are advantageously consuming AcGGM-derived disaccharides that were either available as a fraction of the original AcGGM fiber preparation, have been generated via the actions of other fiber-degrading populations, or have become available via new ecological niches that have been created via the AcGGM-derived structural shifts in the microbiome.</p><p id=\"Par20\" class=\"p p-last\">Both MAG053 (<em>Megasphaera</em>) and MAG150 (<em>Dialister</em>) were predicted via our multi-omics approach to metabolize SCFAs such as lactate and succinate that were generated endogenously by <em>Lactobacillus-</em> (i.e., MAG013) and <em>Prevotella-</em> (i.e., MAG285) affiliated populations, and produce butyrate and propionate, respectively (Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig7/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span>7</span></a> and Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). Neither of these cross-species relationships had any evidence of being directly influenced by AcGGM and are presumably being driven by underlying availabilities of fermentation intermediates (e.g., lactate and succinate) resulting from as-yet uncharacterized niche fluctuations. Collectively, these data highlight that while a highly tailored MDF can directly activate enzymatically equipped populations in an endogenous microbiome, they can also inadvertently alter the metabolisms of other non-mannanolytic populations creating effects that should be quantified to ascertain the true modulation potential of an MDF. The community dynamics that drive fiber-fermentation is not well understood and is likely a contributory factor that restricts broader MDF development.</p></span><span id=\"s\n8\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\n8title\">Discussion</h2><p id=\"Par21\" class=\"p p-first\">Here, we have advanced the MDF concept by showing that the defined relationship between fiber and microbe can be used to directly target specific microbiota in highly complex and competitive gut ecosystems. This was achieved by matching the structural features of our characterized AcGGM fiber to a specific genetic locus in our target microbiota, for which there exists comprehensive biochemical information pertaining to the encoded enzymatic functions.</p><p id=\"Par22\" class=\"p p-last\">Using integrated multi-omics analysis, we illustrate the impact of the AcGGM structural configuration on microbial uptake and metabolism within the distal regions of the porcine digestive tract, with the key driver of AcGGM selectivity being the presence of acetylations of mannan, as well as carbohydrate composition and size<sup><sup><a href=\"#CR27\" rid=\"CR27\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780251\">27</a></sup></sup>. Preserving the complexity of AcGGM resulted in a highly specific shift in the composition of the colon microbiome from weaned piglets, with increases in the relative butyrate:acetate ratios and no adverse effect on host growth performance or health status. We showed that the AcGGM fiber activated a metabolic response in specific <em>Roseburia</em> and <em>Faecalibacterium</em> populations in vivo, as it did in the previous in vitro experiments<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780176\">5</a></sup>,<sup><a href=\"#CR10\" rid=\"CR10\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780211\">10</a></sup></sup>, with both populations expressing proteins from unique CE2/CE17-containing mannan PULs that are homologous to a biochemically characterized representative in <em>R. intestinalis</em> L1-82. In conclusion, our data provide a foundation for modulatory strategies to design and match custom dietary fibers to unique enzymatic features of their target organisms. However, they also underscore the fact that the greater network of interconnected metabolic exchanges and trophic structures inherent to the gut microbiome is highly susceptible to minor dietary interventions. Ultimately, further research is required to determine whether AcGGM MDFs can confer improvements to animal performance metrics (such as growth and disease resistance) in the context of full-scale livestock production.</p></span><span id=\"s\n9\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\n9title\">Methods</h2><span id=\"s\n10\" class=\"sec sec-first\"><h3 id=\"s\n10title\">Animals, diets, and experimental design</h3><p id=\"Par23\" class=\"p p-first\">Animal care protocols and experimental procedures were approved by the Norwegian Animal Research Authority, approval no. 17/9496, FOTS ID 11,314, and treated according to institutional guidelines. A total of 48 crossbred piglets (Landrace × Yorkshire), 24 male and 24 female, with average initial body weight (BW) of 9.8 ± 0.5 kg and weaned at 28 days of age, were sorted by litter, sex, and weight and randomly divided into twelve pens of four animals, with three pens fed each of the four diets. The piglets in each pen were trained to feed at individual stations containing their respective feed. The animals were housed in an environmentally controlled facility with plastic flooring and a mechanical ventilation system. The temperature of the room was maintained at 22 °C.</p><p id=\"Par24\" class=\"p p-last\">Piglets were fed cereal-based diets containing increasing levels of AcGGM diets (1, 2, and 4%). Inclusion levels were selected based on in-house experience from previous feeding trials and communication with the feed industry, where an inclusion level of 1–4% of MDF in feed would be realistic. The highest inclusion level at 4% was selected in order to test a dose response. Diets were pelleted with a 3 mm diameter feed formulated to meet the requirements for indispensable amino acids and all other nutrients (NRC, 2012). The composition of diets is listed in Supplementary Table&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3</a></sup>. Pigs were fed semi-ad libitum twice a day at a feeding level equal to about 5% of body weight. To evaluate growth performance, the BW of each pig was recorded at the beginning and once a week. Feed consumption was recorded for each individual pig during the experiment to calculate individual weight gain and feed intake. After each meal, feed leftovers were registered, dried, and subtracted from the total feed intake.</p></span><span id=\"s\n11\" class=\"sec\"><h3 id=\"s\n11title\">Production of AcGGM</h3><p id=\"Par25\" class=\"p p-first-last\">AcGGM oligosaccharides for the feeding trial were produced from Norway spruce chips milled with a hammer mill to &lt;2 mm size. Wood chips were then steam-exploded on a pilot-scale steam explosion rig (100 L reactor vessel) at the Norwegian University of Life Sciences (NMBU). The steam explosion was conducted in batches of approximately 6 kg dry matter, 14.5 bar pressure (equivalent to 200 °C), with 10 min residence time. The pH in the collected biomass slurry after the steam explosion was ~3.7, which corresponds to a combined severity factor R′<sub>0</sub> = 1.70 for the process. The combined severity (as per<sup><sup><a href=\"#CR28\" rid=\"CR28\" class=\" bibr popnode\">28</a></sup></sup>)was calculated by <em>R</em>′<sub>0</sub> = (10<sup>−pH</sup>) × (<em>t</em> × <em>e</em>(<sup>Texp−100)/14.75</sup>). Steam-exploded wood was collected in 50 L plastic buckets that were topped up with hot (~70 °C) water. The slurry was transferred to a 60 L cider press (Speidel, Germany) and the liquid fraction was pressed out. Milled wood was collected, soaked in hot water again, and pressed for the second time. The liquid fraction was collected and recirculated through a bag filter 50 µm pore WE50P2VWR (Allied filter systems, England) partly filled with the wood particles as a filter aid. Once free of floating wood particles, the liquid fraction of hemicellulose was filtered through a 5-kDa spiral wound Polysulphone/polyethersulphone ultrafiltration membrane, GR99PE polyester (Alfa Laval, Denmark) that was deliberately fouled to prevent larger oligosaccharides from running through the permeate, using a GEA pilot-scale filtration system Model L (GEA, Denmark). The fraction retained by the membrane was concentrated by nanofiltration using a TriSep XN 45, which had a higher efficiency for permeating water. The filtrate was further concentrated by vacuum evaporation (set to 65 °C) and the concentrate was freeze-dried and homogenized with a grain mill. The final product consisted of 0.9% rhamnose, 2.7% arabinose, 13.7% xylose, 58.9% mannose, 14.9% glucose, and 9.4% galactose (determined by gas chromatography as alditol acetates after sulfuric acid hydrolysis<sup><sup><a href=\"#CR29\" rid=\"CR29\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780214\">29</a></sup></sup>). AcGGM contained 0.73 % ash and 2.4% protein (quantified from total nitrogen by the Kjeldahl method). The Man:Glc:Gal ratio in the mannan was 4:1:0.6, and the DA = 0.35<sup><sup><a href=\"#CR9\" rid=\"CR9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780257\">9</a></sup></sup> (determined by acetate release from NaOH treated AcGGM by the same method as described in the SCFA section below). Moreover, previous two-dimensional (2D) <sup>1</sup>H-<sup>13</sup>C heteronuclear single quantum correlation (HSQC) NMR analysis of the AcGGM used in this study, showed the distribution of acetylations on mannose units, indicating that spruce mannan hydrolysate without enzyme addition contained prevalently 2-<em>O</em>-, some 3-<em>O</em>-acetylations, and a lower degree of 4-<em>O</em>- and 6-<em>O</em>-acetylations<sup><sup><a href=\"#CR9\" rid=\"CR9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780216\">9</a></sup></sup>. The dry matter content was determined by drying 0.2 g of the sample at 105 °C for 20 h. The remaining sample was burned at 600 °C for 24 h in an oven (Carbolite, Sheffield, England) to determine ash content. All measurements were performed in triplicates.</p></span><span id=\"s\n12\" class=\"sec\"><h3 id=\"s\n12title\">Fecal scoring</h3><p id=\"Par26\" class=\"p p-first-last\">During the experiment, fecal consistency was assessed using a scoring system developed by Pedersen and Toft<sup><sup><a href=\"#CR30\" rid=\"CR30\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780172\">30</a></sup></sup> to improve and help standardize current protocols for clinical characterization of fecal consistency. The scoring was based on the following 4 consistency categories: score 1 = firm and shaped, score 2 = soft and shaped, score 3 = loose, and score 4 = watery. Samples with a score 3 or 4 are considered diarrheic. Daily fecal scores for each pen were recorded throughout the trial.</p></span><span id=\"s\n13\" class=\"sec\"><h3 id=\"s\n13title\">pH measurements</h3><p id=\"Par27\" class=\"p p-first-last\">The pH of digesta samples from duodenum, jejunum, ileum, cecum, and colon were measured immediately after slaughter. Samples were placed in universal containers and pH measurements made using an Inolab pH7110 pH meter (WTW, Germany).</p></span><span id=\"s\n14\" class=\"sec\"><h3 id=\"s\n14title\">Blood sampling and flow cytometry</h3><p id=\"Par28\" class=\"p p-first\">Blood samples were collected from the same six piglets per diet at 0, 7, and 27 feeding days. The blood samples were taken 1–2 h post-prandial by venipuncture in the jugular vein while pigs were kept on their backs. Non-heparinized and K<sub>3</sub>EDTA vacuum tubes (Beckman Dickson Vacutainer System) were used to recollect serum and whole blood. The serum was isolated immediately by centrifugation at 1500 × <em>g</em> at 4 °C for 15 min. Serum samples were split in PCR-tubes (200 µL) and stored at −80 °C until analysis. Hematological and clinical analyses were performed with an Advia<sup>®</sup> 2120 Hematology System using Advia 2120 MultiSpecies System Software and clinical chemistry analyses were performed with Advia 1800 Chemistry System (both from Siemens AG Healthcare Sector).</p><p id=\"Par29\" class=\"p p-last\">For flow cytometry analysis, whole blood was diluted 1:1 in RPMI 1640 and kept on ice until single-cell isolation. For the isolation of peripheral blood mononuclear cells (PBMCs) blood was purified by centrifugation in a Ficoll gradient (Kreuzer et al. 2012). Then, isolated PBMCs were incubated with Fixable Yellow Dead Cell Stain Kit (Life Technologies, ThermoFisher Scientific Inc.) followed by primary monoclonal antibodies (mAbs), brief incubation with 30% normal pig serum to block Fc-receptors, and finally fluorescence-labeled secondary antibodies (Abcam plc, UK). To detect the intracellular CD3 epitope, surface-labeled cells were permeabilized with Intracellular Fixation and Permeabilization Buffer Set (eBioscience, Affymetrix Inc.) according to the manufacturer’s instructions. Labeled cells were analyzed on a Gallios Flow Cytometer (Beckman Coulter, Inc.) and data were processed using Kaluza 1.5 software (both Beckman Coulter, Inc.). Cell gates were designed to select for single and viable mononuclear cells. Defined markers were used to identify the different immune subpopulations. For monocytes, antibodies against CD45, CD3, CD14, CD163, and MHCII were used (dilution listed in ‘Reporting summary’). To analyze regulatory T cells (T reg) the following antibodies were used: CD45, CD3, TCR γ/δ, CD4, CD8, FOXp3, and CD25, while CD45, CD8, NKp46, CD4, CD8, Ki67, and CD27 were used to identify T and NK cells (dilution listed in ‘Reporting summary’). The gating strategy used for flow cytometric data is presented in Supplementary Fig.&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">7</a></sup>.</p></span><span id=\"s\n15\" class=\"sec\"><h3 id=\"s\n15title\">Analysis of serum cytokines: MULTIPLEX</h3><p id=\"Par30\" class=\"p p-first-last\">Expression of GMCSF, IFNG, IL-1A, IL1B, IL-1RA, IL-2, IL-4, IL-6, IL-8, IL-10, IL-12, IL-18, and TNFα were measured in serum samples using MILLIPLEX MAP Porcine Cytokine and Chemokine Magnetic Bead Panel - Immunology Multiplex Assay (Merck Millipore) following the manufacturer instructions. The measurement was performed using a Bio-Plex MAGPIX Multiplex Reader (Bio-Rad).</p></span><span id=\"s\n16\" class=\"sec\"><h3 id=\"s\n16title\">Small intestine morphology</h3><p id=\"Par31\" class=\"p p-first-last\">The samples of the small intestine were collected on days 0 and 28 for the determination of intestinal morphology and integrity. Intestinal morphological measurements included the following indices: villus height (VH), crypt depth (CD), and VH:CD. The mean values of VH, CD, and their ratio were calculated. Histology evaluation was performed by the Veterinary Histophalogy Center, VeHiCe, Chile.</p></span><span id=\"s\n17\" class=\"sec\"><h3 id=\"s\n17title\">SCFA analysis</h3><p id=\"Par32\" class=\"p p-first-last\">Samples of digesta from the duodenum, jejunum, ileum, cecum, and colon of individual pigs were collected for SCFA analysis. 250 mg or 250 µL of a sample, depending on the source site were mixed 1:1 with 4 mM H<sub>2</sub>SO<sub>4</sub>, homogenized by shaking at room temperature for 1 h, and centrifuged at 12,000 × <em>g</em> for 10 min. The supernatant was collected with a syringe and filtered through a 0.22 µm pore syringe filter. Samples were stored at −20 °C and centrifuged at 12,000 × <em>g</em> before transferring aliquots into HPLC vials for analysis. SCFA content was analyzed by HPLC using a REZEX ROA-Organic Acid H + (Phenomenex, Torrance, California, USA) 300 × 7.8 mm ion exclusion column, isocratic elution with 0.6 mL/min 4 mM H<sub>2</sub>SO<sub>4</sub> at 65 °C. Eluting analytes were detected by UV at 210 nm. The same HPLC method was used to determine the degree of acetylation, by measuring the acetate released from samples of AcGGM dissolved in 100 mM KOH and relating the values to the mannose content of the sample. All data were analyzed using a two-tailed <em>t-</em>test.</p></span><span id=\"s\n18\" class=\"sec\"><h3 id=\"s\n18title\">Micro array polymer profiling (MAPP)</h3><p id=\"Par33\" class=\"p p-first-last\">All cell-wall preparations were analyzed similarly to Moller et al.<sup><sup><a href=\"#CR31\" rid=\"CR31\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780185\">31</a></sup></sup>. Briefly, cell-wall glycans were sequentially extracted from 10 mg of each sample by incubation with 2 μg/mL cellulase, pH 8.9 in 20 mM tris buffer for 16 h, 45 °C to release residual glycans bound to cellulose. Each extract was mixed 50/50 with glycerol buffer (55.2% glycerol, 44% water, 0.8% Triton X-100) and spotted with four 5-fold dilutions and two technical replicates onto a nitrocellulose membrane with a pore size of 0.45 μm (Whatman, Maidstone, UK) using an Arrayjet Sprint (Arrayjey, Roslin, UK). The arrays were probed using antibodies from the JIM, LM (Plant Probes), BioSupplies, series and a Carbohydrate Binding Module (CBM, NZYTech) detected using anti-rat (JIM, LM), anti-mouse (BioSupplies) or anti-His (CBM) secondary antibodies conjugated to alkaline phosphatase (Sigma Aldrich)<sup><sup><a href=\"#CR32\" rid=\"CR32\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780173\">32</a></sup></sup>. Microarrays were incubated with 5% milk protein in TBS (tris buffered saline) to block non-probe binding sites, followed by incubation with primary antibody for 1.5 h. Arrays were washed in TBS, then incubated with secondary antibodies for 1.5 h. After washing with TBS, arrays were developed using an NBT/BCIP substrate to visualize sample spots that contained glycan epitopes that the probes bound to. The NBT/BCIP substrate contained Nitro Blue Tetrazolium (NBT) and 5-bromo-4-chloro-3-indolylphosphate p-toludine salt (BCIP) in Tris buffer (0.5 mM MgCl<sub>2</sub>, 100 mM diethanol-amine, pH 9.5). The arrays were scanned using a desktop scanner (CanoScan 8800 F, Canon) at 2400 dpi, and the intensity of NBT/BCIP stained spots quantified using Array-Pro Analyzer 6.3 (Media Cybernetics). This produces data for the normalized intensity of spots relative to the background intensity of the nitrocellulose membrane (to control for background staining). We used this data to create a heatmap, for average spot intensity across biological replicates and sample dilutions.</p></span><span id=\"s\n19\" class=\"sec\"><h3 id=\"s\n19title\">Microbial sampling</h3><p id=\"Par34\" class=\"p p-first-last\">Fecal samples were collected from six piglets per experimental group (<em>n</em> = 12) at days 0, 7, 14, 21, and 27 post-weaning. At the end of the trial, all piglets (<em>n</em> = 48) were sacrificed, and distinct samples were collected from the lumen of the duodenum, jejunum, ileum, cecum, and colon. Samples were obtained within the first 15 min after the piglets were sacrificed and the samples were flash-frozen in liquid nitrogen and stored at −80 °C until DNA extraction.</p></span><span id=\"s\n20\" class=\"sec\"><h3 id=\"s\n20title\">DNA extraction</h3><p id=\"Par35\" class=\"p p-first-last\">DNA was extracted with a MagAttract PowerMicrobiome DNA/RNA Kit (MO BIO Laboratories Inc., Carlsbad, CA, USA) according to the manufacturer instructions, except for the bead-beating step where we used a FastPrep-96 Homogenizer (MP Biomedicals LLC., Santa Ana, CA, USA) at maximum intensity for a total of 2 min in 4 pulses of 30 s with a 5 min cooling period between each pulse. A KingFisher Flex DNA extraction robot was used for the automated steps of the protocol. The extracted nucleic acids were quantified with a Qubit Fluorimeter and the Qubit dsDNA BR Assay Kit (ThermoFisher Scientific, Waltham, MA, USA) and stored at −80 °C.</p></span><span id=\"s\n21\" class=\"sec\"><h3 id=\"s\n21title\">16S amplicon sequencing and analysis</h3><p id=\"Par36\" class=\"p p-first\">16S amplicon sequence data was obtained for all fecal and intestinal samples. The V3-V4 region of the 16S rRNA gene was PCR amplified using the primers Pro341F and Pro805R (Supplementary Table&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">4</a></sup>), to which the MiSeq adaptors were additionally incorporated on the 5′ ends<sup><sup><a href=\"#CR33\" rid=\"CR33\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780241\">33</a></sup></sup>. The 25 µL PCR reactions consisted of 1× iProof High-Fidelity Master Mix (Bio-Rad, Hercules, CA, USA), 0.25 µM primers, and 5 ng template DNA. PCR thermal cycling began with a hot start step at 98 °C for 180 s and was followed by 25 cycles of 98 °C denaturation for 30 s, 55 °C annealing for 30 s, and 72 °C extension for 30 s, followed by a final, 300 s extension step at 72 °C. Amplicons were individually purified with AMPure XP beads (Beckman Coulter, Indianapolis, IN, USA) and indexed with the Nextera XT Index Kit v2 (Illumina, San Diego, CA, USA) according to the Illumina protocol for 16S rRNA gene sequencing analysis. Next, equal volumes from each indexing reaction were pooled together, and the pool was purified with AMPure XP beads. The purified amplicon pool was then quantified with a Qubit Fluorimeter, diluted, mixed with 15% PhiX Control v3 (Illumina), and denatured according to the aforementioned Illumina protocol. The denatured library was sequenced on the Illumina MiSeq platform using the MiSeq Reagent Kit v3 (600 cycles). Data were output from the sequencer as demultiplexed FASTQ format files.</p><p id=\"Par37\" class=\"p p-last\">Processing of the data was done with a combination of standalone programs, QIIME<sup><sup><a href=\"#CR34\" rid=\"CR34\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780222\">34</a></sup></sup> MOTHUR<sup><sup><a href=\"#CR35\" rid=\"CR35\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780225\">35</a></sup></sup>, and the R package Phyloseq<sup><sup><a href=\"#CR36\" rid=\"CR36\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780213\">36</a></sup></sup>. To process the data, the paired-end reads for each sample were merged with PEAR<sup><sup><a href=\"#CR37\" rid=\"CR37\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780240\">37</a></sup></sup>, specifying a minimum assembly length 400, maximum assembly length 575, minimum overlap 50, and no statistical test. Then, PRINSEQ<sup><sup><a href=\"#CR38\" rid=\"CR38\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780184\">38</a></sup></sup> version 0.20.4 was used to filter low quality reads by requiring a minimum quality score of 10 for all bases and a minimum mean quality of 30. Primer sequences were trimmed in MOTHUR version 1.36.1, and chimeric sequences were identified and filtered out using QIIME version 1.9.1. Next, open reference OTU<sub>0.97</sub> clustering<sup><sup><a href=\"#CR39\" rid=\"CR39\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780178\">39</a></sup></sup> was performed with VSEARCH<sup><sup><a href=\"#CR40\" rid=\"CR40\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780258\">40</a></sup></sup> version 2.3.2 and the Silva database<sup><sup><a href=\"#CR41\" rid=\"CR41\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780182\">41</a></sup></sup> release 128 as the taxonomy reference. Then, the QIIME core diversity analysis script was run. Differentially abundant phylotypes were identified in both cecum and colon for the control vs. 4% AcGGM samples using both the MetagenomeSeq fitZIG and DESeq2 negative binomial algorithms via the QIIME wrapper. The OTU table, phylogenetic tree, representative sequences, and taxonomy from QIIME were incorporated along with the sample metadata into a Phyloseq version 1.22.3 object in R for data exploration and visualization.</p></span><span id=\"s\n22\" class=\"sec\"><h3 id=\"s\n22title\">Whole-metagenome sequencing and analysis</h3><p id=\"Par38\" class=\"p p-first-last\">Whole-metagenome sequencing was performed at the Norwegian Sequencing Centre on 2 lanes of the Illumina HiSeq 4000 to generate 2 × 150 paired-end reads. TruSeq PCR-free libraries were prepared for 12 control and 12 AcGGM (4%) samples from the colon. All 24 samples were run in both lanes to eliminate the potential for lane-specific sequencing bias. FASTQ format files were received from the sequencing center, and prior to assembly, these were quality filtered with Trimmomatic<sup><sup><a href=\"#CR42\" rid=\"CR42\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780248\">42</a></sup></sup> version 0.36 whereby TruSeq adaptor sequences were eliminated, sequences were required to have an average quality score above 20, leading and trailing bases with quality below 20 were removed, sequences with an average quality score below 15 in a 4-base sliding window were trimmed, and the minimum read length was required to be 36 bases. Individual sample assembly was accomplished with metaSPAdes<sup><sup><a href=\"#CR43\" rid=\"CR43\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780210\">43</a></sup></sup> version 3.11.1. MegaHIT<sup><sup><a href=\"#CR44\" rid=\"CR44\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780239\">44</a></sup></sup> version 1.1.3 was used for co-assembly of all 24 samples together as well as co-assembly of the 12 control samples together and the 12 4% AcGGM samples together. MetaBAT<sup><sup><a href=\"#CR45\" rid=\"CR45\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780212\">45</a></sup></sup> version 0.26.3 was used to bin the assemblies, and dRep<sup><sup><a href=\"#CR46\" rid=\"CR46\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780254\">46</a></sup></sup> version 2.0.5&nbsp;was used to dereplicate the multiple assembly and binning combinations to produce an optimal set of MAGs. MASH<sup><sup><a href=\"#CR47\" rid=\"CR47\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780242\">47</a></sup></sup> version 2.0 used to compare the similarity of the 24 metagenomes by calculating pairwise Jaccard distances which were imported into R for NMDS ordination and visualization. Completeness and contamination were determined for each MAG using CheckM<sup><sup><a href=\"#CR48\" rid=\"CR48\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780183\">48</a></sup></sup> version 1.0.7. Taxonomic classifications of MAGs were performed using GTDB-Tk<sup><sup><a href=\"#CR49\" rid=\"CR49\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_915796627\">49</a></sup></sup>, and relative abundance estimations were generated using CoverM (<a href=\"https://github.com/wwood/CoverM\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://github.com/wwood/CoverM</a>). Feature and functional annotation were completed with the Prokka pipeline<sup><sup><a href=\"#CR50\" rid=\"CR50\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780180\">50</a></sup></sup> version 1.12, and the predicted protein sequences from all 355 MAGs were concatenated to create the metaproteomics reference database. Resulting annotated open reading frames (ORFs) were retrieved, further annotated for CAZymes using the CAZy annotation pipeline with libraries from the July 2018 database release<sup><sup><a href=\"#CR51\" rid=\"CR51\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780201\">51</a></sup>,<sup><a href=\"#CR52\" rid=\"CR52\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780237\">52</a></sup></sup>, and subsequently used as a reference database for the metaproteomics (with the exception of glycosyltransferases).</p></span><span id=\"s\n23\" class=\"sec\"><h3 id=\"s\n23title\">Metaproteomics</h3><p id=\"Par39\" class=\"p p-first\">Proteins were extracted from each sample by the following method. An aliquot (1 g) of colon digesta from pigs fed either a control diet or a diet supplemented with 4% β-mannan was dissolved 1:1 (w/v) in 50 mM Tris-HCl, pH 8.4. Lysis was performed using a bead-beating approach whereby glass beads (size ≤ 106 µm) were added to the colon digesta slurry and cells were disrupted in 3 × 60 s cycles using a FastPrep24 (MP Biomedicals, Santa Ana, CA, USA). Debris was removed by centrifugation at 16,600 × <em>g</em> for 20 min and proteins were precipitated overnight in 16% ice-cold TCA. The next day, proteins were dissolved in 100 μL 50 mM TrisHCl, pH 8.4, and concentration was determined using the Bradford protein assay (Bradford Laboratories, USA) using bovine serum albumin as a standard. Fifty milligrams of protein were prepared in SDS sample buffer, separated by SDS-PAGE using an Any-kD Mini-PROTEAN gel (Bio-Rad Laboratories, Hercules, CA, USA), and stained using Coomassie Brilliant Blue R250. The gel was cut into 6 slices and reduced, alkylated, and digested<sup><sup><a href=\"#CR53\" rid=\"CR53\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780190\">53</a></sup></sup>. Prior to mass spectrometry, peptides were desalted using C<sub>18</sub> ZipTips (Merck Millipore, Darmstadt, Germany) according to the manufacturer’s instructions.</p><p id=\"Par40\">The peptides were analyzed by nanoLC-MS/MS as described previously, using a Q-Exactive hybrid quadrupole orbitrap mass spectrometer (Thermo Scientific, Bremen, Germany)<sup><sup><a href=\"#CR54\" rid=\"CR54\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780243\">54</a></sup></sup>, and the acquired raw data was analyzed using MaxQuant<sup><sup><a href=\"#CR55\" rid=\"CR55\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780232\">55</a></sup></sup> version 1.4.1.2. Proteins were quantified using the MaxLFQ algorithm<sup><sup><a href=\"#CR56\" rid=\"CR56\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_915796628\">56</a></sup></sup>. Data were searched against a sample-specific database (602.947 protein sequences), generated from the 355 metagenome-assembled genomes (MAGs), and against the genome of a pig (<em>Sus scrofa domesticus)</em> (40.708 sequences). In addition, common contaminants such as human keratins, trypsin, and bovine serum albumin were concatenated to the database as well as reversed sequences of all protein entries for estimation of false discovery rates. Protein N-terminal acetylation, oxidation of methionine, conversion of glutamine to pyroglutamic acid, and deamination of asparagine and glutamine were used as variable modifications, while carbamidomethylation of cysteine residues was used as a fixed modification. Trypsin was used as a digestion enzyme and two missed cleavages were allowed. All identifications were filtered in order to achieve a protein false discovery rate (FDR) of 1% using the target-decoy strategy. A total of 8515 protein groups (20,350 shared proteins) were considered valid, which required the protein group to be both identified and quantified in at least two replicates, and in addition, we required at least one unique peptide per protein and at least two peptides in total for every protein group. In cases where a protein group consisted of two or more homologs protein identifications, both unique and shared proteins are indicated (i.e., Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig4/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4c</span></a> and Supplementary Data&nbsp;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). The output from MaxQuant was further explored in Perseus version 1.6.0.7 where filtering, data transformation, and imputation were performed.</p><p id=\"Par41\" class=\"p p-last\">For metaproteomic enrichment analysis, missing values in the proteomics data were set to zero, and proteins with zero variance across the eight samples, or with a non-zero value in only one replicate animal in either the control or AcGGM treatment were removed. This resulted in 4562 unique protein groups assigned to 12,535 MAG ORFs (i.e., shared detected proteins). A Principle Component Analysis (PCA) was performed on the 4562 × 8 matrix using the R function <em>prcomp</em>. Hierarchal clustering dendrograms were constructed using the R functions <em>hclust</em> with Pearson correlation and <em>method</em> = <em>‘ward.D’</em>, and a heatmap was drawn using the function <em>heatmap.2</em> with <em>scale</em> = <em>‘row’</em>. The number of protein clusters were selected by visual inspection and the clusters were selected using the <em>cutree</em> function. Enzyme Commission (EC) annotation (Supplementary Data&nbsp;<sup><a href=\"#MOESM12\" rid=\"MOESM12\" class=\" media\">9</a></sup>) and MAG enrichment analysis (Supplementary Data&nbsp;<sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">7</a></sup>) was performed using the hypergeometric distribution function <em>phyper</em> in R with the false discovery rate controlled at 5% using the function <em>p</em>.adjust with method = ‘BH’.</p></span><span id=\"s\n24\" class=\"sec\"><h3 id=\"s\n24title\">Genome tree</h3><p id=\"Par42\" class=\"p p-first-last\">Phylogenetic analysis was performed using a block of 22 universal ribosomal proteins (30S ribosomal protein L1, L2, L4-L6, L10, L11, L14, L15, L18, and 50S ribosomal protein S3, S5, S7-S13, S15, S17, S19)<sup><sup><a href=\"#CR57\" rid=\"CR57\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780227\">57</a></sup>,<sup><a href=\"#CR58\" rid=\"CR58\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780217\">58</a></sup></sup>. In addition to the MAGs, we recruited 239 reference genomes for phylogenetic resolution. These genomes were selected based on preliminary examination of the assembled metagenome using metaQUAST<sup><sup><a href=\"#CR59\" rid=\"CR59\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780174\">59</a></sup></sup>. The reference genomes were annotated using the Prokka pipeline in the same manner as for the MAGs. All identified ribosomal protein sequences were aligned separately with MUSCLE v3.8.31<sup><sup><a href=\"#CR60\" rid=\"CR60\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780231\">60</a></sup></sup> and manually checked for duplications and misaligned sequences. Divergent regions and poorly aligned positions were further eliminated using GBlocks<sup><sup><a href=\"#CR61\" rid=\"CR61\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780181\">61</a></sup></sup>, and the refined alignment was concatenated using catfasta2phyml.pl (<a href=\"https://github.com/nylander/catfasta2phyml\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://github.com/nylander/catfasta2phyml</a>) with the parameter ‘-c’ to replace missing ribosomal proteins with gaps (-). The maximum likelihood-based phylogeny of the concatenated ribosomal proteins was inferred using RAxML version 8.2.12<sup><sup><a href=\"#CR62\" rid=\"CR62\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780219\">62</a></sup></sup> (raxmlHPC-SSE3 under PROTGAMMA distributed model with WAG substitution matrix) and support values determined using 100 bootstrap replicates. The tree was rooted to the Euryarchaeota phylum and visualized using iTOL<sup><sup><a href=\"#CR63\" rid=\"CR63\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780252\">63</a></sup></sup>. Clades of reference genomes with only distant phylogenetic relation to the MAGs were collapsed to refine the final tree in Fig.&nbsp;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/figure/Fig3/\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>4</span></a>. The complete tree is available in Newick format as Supplementary Data&nbsp;<sup><a href=\"#MOESM7\" rid=\"MOESM7\" class=\" media\">4</a></sup>.</p></span><span id=\"s\n25\" class=\"sec sec-last\"><h3 id=\"s\n25title\">Reporting summary</h3><p id=\"Par43\" class=\"p p-first-last\">Further information on research design is available in the&nbsp;<sup><a href=\"#MOESM13\" rid=\"MOESM13\" class=\" media\">Nature Research Reporting Summary</a></sup> linked to this article.</p></span></span><span id=\"s\n1\" class=\"tsec sec\"><a id=\"supplementary-material-sec\"></a><h2 class=\"head no_bottom_margin\" id=\"s\n1title\">Supplementary information</h2><!--/article/body/sec/--><span id=\"s\n26\" class=\"sec sec-first-last\"><span class=\"sec suppmat\" id=\"MOESM1\"><span class=\"sup-box half_rhythm\" id=\"idm139884796580480\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/bin/41467_2020_19585_MOESM1_ESM.pdf\" data-ga-action=\"click_feat_suppl\">Supplementary Information</a><sup>(1.6M, pdf)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM2\"><span class=\"sup-box half_rhythm\" id=\"idm139884796577664\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/bin/41467_2020_19585_MOESM2_ESM.pdf\" data-ga-action=\"click_feat_suppl\">Peer Review File</a><sup>(384K, pdf)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM3\"><span class=\"sup-box half_rhythm\" id=\"idm139884796574784\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/bin/41467_2020_19585_MOESM3_ESM.docx\" data-ga-action=\"click_feat_suppl\">Description of Additional Supplementary Files</a><sup>(16K, docx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM4\"><span class=\"sup-box half_rhythm\" id=\"idm139884796571936\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/bin/41467_2020_19585_MOESM4_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Data 1</a><sup>(130K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM5\"><span class=\"sup-box half_rhythm\" id=\"idm139884796569056\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/bin/41467_2020_19585_MOESM5_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Data 2</a><sup>(58K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM6\"><span class=\"sup-box half_rhythm\" id=\"idm139884796566176\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/bin/41467_2020_19585_MOESM6_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Data 3</a><sup>(1.0M, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM7\"><span class=\"sup-box half_rhythm\" id=\"idm139884796563296\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/bin/41467_2020_19585_MOESM7_ESM.txt\" data-ga-action=\"click_feat_suppl\">Supplementary Data 4</a><sup>(42K, txt)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM8\"><span class=\"sup-box half_rhythm\" id=\"idm139884796560416\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/bin/41467_2020_19585_MOESM8_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Data 5</a><sup>(63K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM9\"><span class=\"sup-box half_rhythm\" id=\"idm139884796557536\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/bin/41467_2020_19585_MOESM9_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Data 6</a><sup>(535K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM10\"><span class=\"sup-box half_rhythm\" id=\"idm139884796554656\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/bin/41467_2020_19585_MOESM10_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Data 7</a><sup>(16K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM11\"><span class=\"sup-box half_rhythm\" id=\"idm139884797129856\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/bin/41467_2020_19585_MOESM11_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Data 8</a><sup>(36K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM12\"><span class=\"sup-box half_rhythm\" id=\"idm139884797126976\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/bin/41467_2020_19585_MOESM12_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Data 9</a><sup>(39K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM13\"><span class=\"sup-box half_rhythm\" id=\"idm139884797124096\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7666174/bin/41467_2020_19585_MOESM13_ESM.pdf\" data-ga-action=\"click_feat_suppl\">Reporting Summary</a><sup>(298K, pdf)</sup></span></span></span></span><span id=\"idm139884797116912\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139884797116912title\">Acknowledgements</h2><span class=\"sec\"><p id=\"__p25\">We are grateful for support from The Research Council of Norway (Bionær program 244259, 208674/F50, 270038, FRIPRO program, PBP: 250479), as well as the European Research Commission Starting Grant Fellowship (awarded to PBP; 336355 - MicroDE). The sequencing service was provided by the Norwegian Sequencing Centre (<a href=\"http://www.sequencing.uio.no\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">www.sequencing.uio.no</a>), a national technology platform hosted by the University of Oslo and supported by the ‘Functional Genomics’ and ‘Infrastructure’ programs of the Research Council of Norway and the Southeastern Regional Health Authorities. We thank Sivert Dæhli for performing the detailed analysis of degree of polymerization of the AcGGM.</p></span></span><span id=\"idm139884797114320\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139884797114320title\">Author contributions</h2><p id=\"__p26\">L.M., J.C.G., P.B.P. and B.W. conceived the study, performed the primary analysis of the data and wrote the paper (with input from all authors). L.M., J.C.G., L.L., S.L.L.R., L.H.H., T.R.H., P.B.P. and B.W. designed the figures. L.L. and M.Ø. designed, performed and analyzed the animal experiments. S.L.L.R., L.H.H., M.Ø.A., T.R.H. and J.D. generated the data and/or contributed to the data analyses. W.G.T.W. and C.T.-J. generated and analyzed MAPP data. N.T., V.L. and B.H. annotated and curated the MAGs and identified carbohydrate-active enzymes.</p></span><span id=\"idm139884797112496\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139884797112496title\">Data availability</h2><p id=\"__p27\">All sequencing reads have been deposited at the NCBI sequence read archive under BioProject <a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA574295\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA574295</a>, with specific numbers listed in Supplementary Table&nbsp;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">2</a></sup>. All annotated MAGs are publicly available via 10.6084/m9.figshare.9816581<sup><sup><a href=\"#CR64\" rid=\"CR64\" class=\" bibr popnode\">64</a></sup></sup>. The proteomics data have been deposited to the ProteomeXchange Consortium (<a href=\"http://proteomecentral.proteomexchange.org\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://proteomecentral.proteomexchange.org</a>) via the PRIDE partner repository<sup><sup><a href=\"#CR65\" rid=\"CR65\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780206\">65</a></sup></sup> with the dataset identifier <a href=\"https://www.ebi.ac.uk/pride/archive/projects/PXD015757\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PXD015757</a>.</p></span><span id=\"idm139884797104576\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139884797104576title\">Code availability</h2><p id=\"__p28\">The code used to perform the metaproteomic enrichment analysis is available at <a href=\"https://gitlab.com/hvidsten-lab/michalak\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://gitlab.com/hvidsten-lab/michalak</a>.</p></span><span id=\"FPar1\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"FPar1title\">Competing interests</h2><p id=\"Par44\">The authors declare no competing interests.</p></span><span id=\"idm139884797122336\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139884797122336title\">Footnotes</h2><!--back/fn-group--><span class=\"fm-sec half_rhythm small\"><p class=\"fn sec\" id=\"idm139884797122208\"></p><p id=\"__p21\" class=\"p p-first-last\"><strong>Peer review information</strong>\n<em>Nature Communications</em> thanks David Bolam and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available.</p><p></p><p class=\"fn sec\" id=\"idm139884797120384\"></p><p id=\"__p22\" class=\"p p-first-last\"><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p><p></p><p class=\"fn sec\" id=\"idm139884797119280\"></p><p id=\"__p23\" class=\"p p-first-last\">These authors contributed equally: Phillip B. Pope, Bjørge Westereng.</p><p></p></span></span><br><div class=\"ref-list-sec sec\" id=\"reference-list\"><div class=\"ref-cit-blk half_rhythm\" id=\"CR1\">1. <span class=\"element-citation\">Patnode ML, et al.  Interspecies competition impacts targeted manipulation of human gut bacteria by fiber-derived glycans. <span><span class=\"ref-journal\">Cell. </span>2019;<span class=\"ref-vol\">179</span>:59–73. doi: 10.1016/j.cell.2019.08.011.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6760872\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=6760872\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n31539500\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.cell.2019.08.011\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Interspecies+competition+impacts+targeted+manipulation+of+human+gut+bacteria+by+fiber-derived+glycans&amp;author=ML+Patnode&amp;volume=179&amp;publication_year=2019&amp;pages=59-73&amp;pmid=31539500&amp;doi=10.1016/j.cell.2019.08.011&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR2\">2. <span class=\"element-citation\">Blanton LV, Barratt MJ, Charbonneau MR, Ahmed T, Gordon JI. Childhood undernutrition, the gut microbiota, and microbiota-directed therapeutics. <span><span class=\"ref-journal\">Science. </span>2016;<span class=\"ref-vol\">352</span>:1533. doi: 10.1126/science.aad9359.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n27339978\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1126%2Fscience.aad9359\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Childhood+undernutrition,+the+gut+microbiota,+and+microbiota-directed+therapeutics&amp;author=LV+Blanton&amp;author=MJ+Barratt&amp;author=MR+Charbonneau&amp;author=T+Ahmed&amp;author=JI+Gordon&amp;volume=352&amp;publication_year=2016&amp;pages=1533&amp;pmid=27339978&amp;doi=10.1126/science.aad9359&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR3\">3. <span class=\"element-citation\">Gibson GR, et al.  Expert consensus document: The International Scientific Association for Probiotics and Prebiotics (ISAPP) consensus statement on the definition and scope of prebiotics. <span><span class=\"ref-journal\">Nat. Rev. Gastroenterol. Hepatol. </span>2017;<span class=\"ref-vol\">14</span>:491–502. doi: 10.1038/nrgastro.2017.75.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n28611480\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnrgastro.2017.75\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Gastroenterol.+Hepatol.&amp;title=Expert+consensus+document:+The+International+Scientific+Association+for+Probiotics+and+Prebiotics+(ISAPP)+consensus+statement+on+the+definition+and+scope+of+prebiotics&amp;author=GR+Gibson&amp;volume=14&amp;publication_year=2017&amp;pages=491-502&amp;pmid=28611480&amp;doi=10.1038/nrgastro.2017.75&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR4\">4. <span class=\"element-citation\">Sanders ME, Merenstein DJ, Reid G, Gibson GR, Rastall RA. Probiotics and prebiotics in intestinal health and disease: from biology to the clinic. <span><span class=\"ref-journal\">Nat. Rev. Gastroenterol. Hepatol. </span>2019;<span class=\"ref-vol\">16</span>:605–616. doi: 10.1038/s41575-019-0173-3.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n31296969\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41575-019-0173-3\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Gastroenterol.+Hepatol.&amp;title=Probiotics+and+prebiotics+in+intestinal+health+and+disease:+from+biology+to+the+clinic&amp;author=ME+Sanders&amp;author=DJ+Merenstein&amp;author=G+Reid&amp;author=GR+Gibson&amp;author=RA+Rastall&amp;volume=16&amp;publication_year=2019&amp;pages=605-616&amp;pmid=31296969&amp;doi=10.1038/s41575-019-0173-3&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR5\">5. <span class=\"element-citation\">La Rosa SL, et al.  The human gut Firmicute <em>Roseburia intestinalis</em> is a primary degrader of dietary β-mannans. <span><span class=\"ref-journal\">Nat. Commun. </span>2019;<span class=\"ref-vol\">10</span>:905. doi: 10.1038/s41467-019-08812-y.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6385246\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=6385246\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30796211\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41467-019-08812-y\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=The+human+gut+Firmicute+Roseburia+intestinalis+is+a+primary+degrader+of+dietary+β-mannans&amp;author=SL+La+Rosa&amp;volume=10&amp;publication_year=2019&amp;pages=905&amp;pmid=30796211&amp;doi=10.1038/s41467-019-08812-y&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR6\">6. <span class=\"element-citation\">Larsbrink J, Rogers T, Hemsworth G. A discrete genetic locus confers xyloglucan metabolism in select human gut Bacteroidetes. <span><span class=\"ref-journal\">Nature. </span>2014;<span class=\"ref-vol\">506</span>:498–502. doi: 10.1038/nature12907.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4282169\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=4282169\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24463512\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnature12907\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+discrete+genetic+locus+confers+xyloglucan+metabolism+in+select+human+gut+Bacteroidetes&amp;author=J+Larsbrink&amp;author=T+Rogers&amp;author=G+Hemsworth&amp;volume=506&amp;publication_year=2014&amp;pages=498-502&amp;pmid=24463512&amp;doi=10.1038/nature12907&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR7\">7. <span class=\"element-citation\">Cuskin F, et al.  Human gut Bacteroidetes can utilize yeast mannan through a selfish mechanism. <span><span class=\"ref-journal\">Nature. </span>2015;<span class=\"ref-vol\">517</span>:165–169. doi: 10.1038/nature13995.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4978465\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=4978465\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n25567280\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnature13995\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Human+gut+Bacteroidetes+can+utilize+yeast+mannan+through+a+selfish+mechanism&amp;author=F+Cuskin&amp;volume=517&amp;publication_year=2015&amp;pages=165-169&amp;pmid=25567280&amp;doi=10.1038/nature13995&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR8\">8. <span class=\"element-citation\">Solden LM, et al.  Interspecies cross-feeding orchestrates carbon degradation in the rumen ecosystem. <span><span class=\"ref-journal\">Nat. Microbiol. </span>2018;<span class=\"ref-vol\">3</span>:1274–1284. doi: 10.1038/s41564-018-0225-4.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6784887\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=6784887\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30356154\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41564-018-0225-4\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Microbiol.&amp;title=Interspecies+cross-feeding+orchestrates+carbon+degradation+in+the+rumen+ecosystem&amp;author=LM+Solden&amp;volume=3&amp;publication_year=2018&amp;pages=1274-1284&amp;pmid=30356154&amp;doi=10.1038/s41564-018-0225-4&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR9\">9. <span class=\"element-citation\">Michalak L, et al.  A pair of esterases from a commensal gut bacterium remove acetylations from all positions on complex β-mannans. <span><span class=\"ref-journal\">Proc. Natl Acad. Sci. USA. </span>2020;<span class=\"ref-vol\">117</span>:7122–7130. doi: 10.1073/pnas.1915376117.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7132267\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=7132267\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n32170022\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1073%2Fpnas.1915376117\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=A+pair+of+esterases+from+a+commensal+gut+bacterium+remove+acetylations+from+all+positions+on+complex+β-mannans&amp;author=L+Michalak&amp;volume=117&amp;publication_year=2020&amp;pages=7122-7130&amp;pmid=32170022&amp;doi=10.1073/pnas.1915376117&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR10\">10. <span class=\"element-citation\">La Rosa SL, et al.  Wood-derived dietary fibers promote beneficial human gut microbiota. <span><span class=\"ref-journal\">mSphere. </span>2019;<span class=\"ref-vol\">4</span>:e00554–00518.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6344601\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=6344601\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30674645\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=mSphere&amp;title=Wood-derived+dietary+fibers+promote+beneficial+human+gut+microbiota&amp;author=SL+La+Rosa&amp;volume=4&amp;publication_year=2019&amp;pages=e00554-00518&amp;pmid=30674645&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR11\">11. <span class=\"element-citation\">Ferreira-Halder CV, Faria AVS, Andrade SS. Action and function of <em>Faecalibacterium prausnitzii</em> in health and disease. <span><span class=\"ref-journal\">Best. Pract. Res. Clin. Gastroenterol. </span>2017;<span class=\"ref-vol\">31</span>:643–648. doi: 10.1016/j.bpg.2017.09.011.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n29566907\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.bpg.2017.09.011\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Best.+Pract.+Res.+Clin.+Gastroenterol.&amp;title=Action+and+function+of+Faecalibacterium+prausnitzii+in+health+and+disease&amp;author=CV+Ferreira-Halder&amp;author=AVS+Faria&amp;author=SS+Andrade&amp;volume=31&amp;publication_year=2017&amp;pages=643-648&amp;pmid=29566907&amp;doi=10.1016/j.bpg.2017.09.011&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR12\">12. <span class=\"element-citation\">Lopez-Siles M, Duncan SH, Garcia-Gil LJ, Martinez-Medina M. <em>Faecalibacterium prausnitzii</em>: from microbiology to diagnostics and prognostics. <span><span class=\"ref-journal\">ISME J. </span>2017;<span class=\"ref-vol\">11</span>:841–852. doi: 10.1038/ismej.2016.176.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5364359\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=5364359\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n28045459\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fismej.2016.176\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=ISME+J.&amp;title=Faecalibacterium+prausnitzii:+from+microbiology+to+diagnostics+and+prognostics&amp;author=M+Lopez-Siles&amp;author=SH+Duncan&amp;author=LJ+Garcia-Gil&amp;author=M+Martinez-Medina&amp;volume=11&amp;publication_year=2017&amp;pages=841-852&amp;pmid=28045459&amp;doi=10.1038/ismej.2016.176&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR13\">13. <span class=\"element-citation\">Lahaye M, Quemener B, Causse M, Seymour GB. Hemicellulose fine structure is affected differently during ripening of tomato lines with contrasted texture. <span><span class=\"ref-journal\">Int. J. Biol. Macromol. </span>2012;<span class=\"ref-vol\">51</span>:462–470. doi: 10.1016/j.ijbiomac.2012.05.024.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n22698552\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.ijbiomac.2012.05.024\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int.+J.+Biol.+Macromol.&amp;title=Hemicellulose+fine+structure+is+affected+differently+during+ripening+of+tomato+lines+with+contrasted+texture&amp;author=M+Lahaye&amp;author=B+Quemener&amp;author=M+Causse&amp;author=GB+Seymour&amp;volume=51&amp;publication_year=2012&amp;pages=462-470&amp;pmid=22698552&amp;doi=10.1016/j.ijbiomac.2012.05.024&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR14\">14. <span class=\"element-citation\">Xu C, et al.  Acetylation and characterization of spruce (<em>Picea abies</em>) galactoglucomannans. <span><span class=\"ref-journal\">Carbohydr. Res. </span>2010;<span class=\"ref-vol\">345</span>:810–816. doi: 10.1016/j.carres.2010.01.007.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n20144827\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.carres.2010.01.007\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Carbohydr.+Res.&amp;title=Acetylation+and+characterization+of+spruce+(Picea+abies)+galactoglucomannans&amp;author=C+Xu&amp;volume=345&amp;publication_year=2010&amp;pages=810-816&amp;pmid=20144827&amp;doi=10.1016/j.carres.2010.01.007&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR15\">15. <span class=\"element-citation\">Michalak L, Knutsen SH, Aarum I, Westereng B. Effects of pH on steam explosion extraction of acetylated galactoglucomannan from Norway spruce. <span><span class=\"ref-journal\">Biotechnol. Biofuels. </span>2018;<span class=\"ref-vol\">11</span>:311. doi: 10.1186/s13068-018-1300-z.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6225635\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=6225635\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30455740\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2Fs13068-018-1300-z\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Biotechnol.+Biofuels&amp;title=Effects+of+pH+on+steam+explosion+extraction+of+acetylated+galactoglucomannan+from+Norway+spruce&amp;author=L+Michalak&amp;author=SH+Knutsen&amp;author=I+Aarum&amp;author=B+Westereng&amp;volume=11&amp;publication_year=2018&amp;pages=311&amp;pmid=30455740&amp;doi=10.1186/s13068-018-1300-z&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR16\">16. <span class=\"element-citation\">Shepherd ES, DeLoache WC, Pruss KM, Whitaker WR, Sonnenburg JL. An exclusive metabolic niche enables strain engraftment in the gut microbiota. <span><span class=\"ref-journal\">Nature. </span>2018;<span class=\"ref-vol\">557</span>:434–438. doi: 10.1038/s41586-018-0092-4.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6126907\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=6126907\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n29743671\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41586-018-0092-4\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+exclusive+metabolic+niche+enables+strain+engraftment+in+the+gut+microbiota&amp;author=ES+Shepherd&amp;author=WC+DeLoache&amp;author=KM+Pruss&amp;author=WR+Whitaker&amp;author=JL+Sonnenburg&amp;volume=557&amp;publication_year=2018&amp;pages=434-438&amp;pmid=29743671&amp;doi=10.1038/s41586-018-0092-4&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR17\">17. <span class=\"element-citation\">Chen L, et al.  The maturing development of gut microbiota in commercial piglets during the weaning transition. <span><span class=\"ref-journal\">Front. Microbiol. </span>2017;<span class=\"ref-vol\">8</span>:1688. doi: 10.3389/fmicb.2017.01688.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5591375\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=5591375\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n28928724\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3389%2Ffmicb.2017.01688\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front.+Microbiol.&amp;title=The+maturing+development+of+gut+microbiota+in+commercial+piglets+during+the+weaning+transition&amp;author=L+Chen&amp;volume=8&amp;publication_year=2017&amp;pages=1688&amp;pmid=28928724&amp;doi=10.3389/fmicb.2017.01688&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR18\">18. <span class=\"mixed-citation\">Holman, D. B., Brunelle, B. W., Trachsel, J. &amp; Allen, H. K. Meta-analysis to define a core microbiota in the swine gut. <em>MSystems</em><strong>2</strong>, e00004–17 (2017). <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5443231\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=5443231\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n28567446\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>]</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR19\">19. <span class=\"element-citation\">Rattigan R, et al.  Laminarin-rich extract improves growth performance, small intestinal morphology, gene expression of nutrient transporters and the large intestinal microbial composition of piglets during the critical post-weaning period. <span><span class=\"ref-journal\">Br. J. Nutr. </span>2020;<span class=\"ref-vol\">123</span>:255–263. doi: 10.1017/S0007114519002678.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n31640819\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1017%2FS0007114519002678\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Br.+J.+Nutr.&amp;title=Laminarin-rich+extract+improves+growth+performance,+small+intestinal+morphology,+gene+expression+of+nutrient+transporters+and+the+large+intestinal+microbial+composition+of+piglets+during+the+critical+post-weaning+period&amp;author=R+Rattigan&amp;volume=123&amp;publication_year=2020&amp;pages=255-263&amp;pmid=31640819&amp;doi=10.1017/S0007114519002678&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR20\">20. <span class=\"element-citation\">Precup G, Vodnar DC. Gut Prevotella as a possible biomarker of diet and its eubiotic versus dysbiotic roles: a comprehensive literature review. <span><span class=\"ref-journal\">Br. J. Nutr. </span>2019;<span class=\"ref-vol\">122</span>:131–140. doi: 10.1017/S0007114519000680.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30924428\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1017%2FS0007114519000680\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Br.+J.+Nutr.&amp;title=Gut+Prevotella+as+a+possible+biomarker+of+diet+and+its+eubiotic+versus+dysbiotic+roles:+a+comprehensive+literature+review&amp;author=G+Precup&amp;author=DC+Vodnar&amp;volume=122&amp;publication_year=2019&amp;pages=131-140&amp;pmid=30924428&amp;doi=10.1017/S0007114519000680&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR21\">21. <span class=\"element-citation\">Kageyama A, Benno Y. <em>Catenibacterium mitsuokai</em> gen. nov., sp. nov., a gram-positive anaerobic bacterium isolated from human faeces. <span><span class=\"ref-journal\">Int. J. Syst. Bacteriol. </span>2000;<span class=\"ref-vol\">50</span>:1595–1599. doi: 10.1099/00207713-50-4-1595.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n10939666\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1099%2F00207713-50-4-1595\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int.+J.+Syst.+Bacteriol.&amp;title=Catenibacterium+mitsuokai+gen.+nov.,+sp.+nov.,+a+gram-positive+anaerobic+bacterium+isolated+from+human+faeces&amp;author=A+Kageyama&amp;author=Y+Benno&amp;volume=50&amp;publication_year=2000&amp;pages=1595-1599&amp;doi=10.1099/00207713-50-4-1595&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR22\">22. <span class=\"element-citation\">Moore LV, Moore WE. <em>Oribaculum catoniae</em> gen. nov., sp. nov.; <em>Catonella morbi</em> gen. nov., sp. nov.; <em>Hallella seregens</em> gen. nov., sp. nov.; <em>Johnsonella ignava</em> gen. nov., sp. nov.; and <em>Dialister pneumosintes</em> gen. nov., comb. nov., nom. rev., Anaerobic gram-negative bacilli from the human gingival crevice. <span><span class=\"ref-journal\">Int J. Syst. Bacteriol. </span>1994;<span class=\"ref-vol\">44</span>:187–192. doi: 10.1099/00207713-44-2-187.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n8186083\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1099%2F00207713-44-2-187\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int+J.+Syst.+Bacteriol.&amp;title=Oribaculum+catoniae+gen.+nov.,+sp.+nov.;+Catonella+morbi+gen.+nov.,+sp.+nov.;+Hallella+seregens+gen.+nov.,+sp.+nov.;+Johnsonella+ignava+gen.+nov.,+sp.+nov.;+and+Dialister+pneumosintes+gen.+nov.,+comb.+nov.,+nom.+rev.,+Anaerobic+gram-negative+bacilli+from+the+human+gingival+crevice&amp;author=LV+Moore&amp;author=WE+Moore&amp;volume=44&amp;publication_year=1994&amp;pages=187-192&amp;pmid=8186083&amp;doi=10.1099/00207713-44-2-187&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR23\">23. <span class=\"element-citation\">Louis P, Young P, Holtrop G, Flint HJ. Diversity of human colonic butyrate-producing bacteria revealed by analysis of the butyryl-CoA:acetate CoA-transferase gene. <span><span class=\"ref-journal\">Environ. Microbiol. </span>2010;<span class=\"ref-vol\">12</span>:304–314. doi: 10.1111/j.1462-2920.2009.02066.x.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n19807780\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1462-2920.2009.02066.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Environ.+Microbiol&amp;title=Diversity+of+human+colonic+butyrate-producing+bacteria+revealed+by+analysis+of+the+butyryl-CoA:acetate+CoA-transferase+gene&amp;author=P+Louis&amp;author=P+Young&amp;author=G+Holtrop&amp;author=HJ+Flint&amp;volume=12&amp;publication_year=2010&amp;pages=304-314&amp;pmid=19807780&amp;doi=10.1111/j.1462-2920.2009.02066.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR24\">24. <span class=\"element-citation\">Martens EC, Koropatkin NM, Smith TJ, Gordon JI. Complex glycan catabolism by the human gut microbiota: the Bacteroidetes Sus-like paradigm. <span><span class=\"ref-journal\">J. Biol. Chem. </span>2009;<span class=\"ref-vol\">284</span>:24673–24677. doi: 10.1074/jbc.R109.022848.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2757170\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=2757170\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n19553672\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1074%2Fjbc.R109.022848\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Complex+glycan+catabolism+by+the+human+gut+microbiota:+the+Bacteroidetes+Sus-like+paradigm&amp;author=EC+Martens&amp;author=NM+Koropatkin&amp;author=TJ+Smith&amp;author=JI+Gordon&amp;volume=284&amp;publication_year=2009&amp;pages=24673-24677&amp;pmid=19553672&amp;doi=10.1074/jbc.R109.022848&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR25\">25. <span class=\"element-citation\">Panwar D, Kapoor M. Transcriptional analysis of galactomannooligosaccharides utilization by <em>Lactobacillus plantarum</em> WCFS1. <span><span class=\"ref-journal\">Food Microbiol. </span>2020;<span class=\"ref-vol\">86</span>:103336. doi: 10.1016/j.fm.2019.103336.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n31703861\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.fm.2019.103336\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Food+Microbiol.&amp;title=Transcriptional+analysis+of+galactomannooligosaccharides+utilization+by+Lactobacillus+plantarum+WCFS1&amp;author=D+Panwar&amp;author=M+Kapoor&amp;volume=86&amp;publication_year=2020&amp;pages=103336&amp;pmid=31703861&amp;doi=10.1016/j.fm.2019.103336&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR26\">26. <span class=\"element-citation\">Sadaie Y, Nakadate H, Fukui R, Yee LM, Asai K. Glucomannan utilization operon of <em>Bacillus subtilis</em>. <span><span class=\"ref-journal\">FEMS Microbiol. Lett. </span>2008;<span class=\"ref-vol\">279</span>:103–109. doi: 10.1111/j.1574-6968.2007.01018.x.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n18177310\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1574-6968.2007.01018.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=FEMS+Microbiol.+Lett.&amp;title=Glucomannan+utilization+operon+of+Bacillus+subtilis&amp;author=Y+Sadaie&amp;author=H+Nakadate&amp;author=R+Fukui&amp;author=LM+Yee&amp;author=K+Asai&amp;volume=279&amp;publication_year=2008&amp;pages=103-109&amp;pmid=18177310&amp;doi=10.1111/j.1574-6968.2007.01018.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR27\">27. <span class=\"element-citation\">Leth ML, et al.  Differential bacterial capture and transport preferences facilitate co-growth on dietary xylan in the human gut. <span><span class=\"ref-journal\">Nat. Microbiol. </span>2018;<span class=\"ref-vol\">3</span>:570–580. doi: 10.1038/s41564-018-0132-8.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n29610517\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41564-018-0132-8\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Microbiol.&amp;title=Differential+bacterial+capture+and+transport+preferences+facilitate+co-growth+on+dietary+xylan+in+the+human+gut&amp;author=ML+Leth&amp;volume=3&amp;publication_year=2018&amp;pages=570-580&amp;pmid=29610517&amp;doi=10.1038/s41564-018-0132-8&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR28\">28. <span class=\"element-citation\">Chum HL, Johnson DK, Black SK, Overend RP. Pretreatment catalyst effects and the combined severity parameter. <span><span class=\"ref-journal\">Appl. Biochem. Biotechnol. </span>1990;<span class=\"ref-vol\">24</span>:1–14. doi: 10.1007/BF02920229.</span> [<a href=\"//dx.doi.org/10.1007%2FBF02920229\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Biochem.+Biotechnol.&amp;title=Pretreatment+catalyst+effects+and+the+combined+severity+parameter&amp;author=HL+Chum&amp;author=DK+Johnson&amp;author=SK+Black&amp;author=RP+Overend&amp;volume=24&amp;publication_year=1990&amp;pages=1-14&amp;pmid=2353809&amp;doi=10.1007/BF02920229&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR29\">29. <span class=\"element-citation\">Vestby LK, Møretrø T, Ballance S, Langsrud S, Nesse LL. Survival potential of wild type cellulose deficient <em>Salmonella</em> from the feed industry. <span><span class=\"ref-journal\">BMC Vet. Res. </span>2009;<span class=\"ref-vol\">5</span>:43. doi: 10.1186/1746-6148-5-43.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2788542\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=2788542\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n19930629\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2F1746-6148-5-43\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Vet.+Res.&amp;title=Survival+potential+of+wild+type+cellulose+deficient+Salmonella+from+the+feed+industry&amp;author=LK+Vestby&amp;author=T+Møretrø&amp;author=S+Ballance&amp;author=S+Langsrud&amp;author=LL+Nesse&amp;volume=5&amp;publication_year=2009&amp;pages=43&amp;pmid=19930629&amp;doi=10.1186/1746-6148-5-43&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR30\">30. <span class=\"element-citation\">Pedersen KS, Toft N. Intra- and inter-observer agreement when using a descriptive classification scale for clinical assessment of faecal consistency in growing pigs. <span><span class=\"ref-journal\">Prev. Vet. Med. </span>2011;<span class=\"ref-vol\">98</span>:288–291. doi: 10.1016/j.prevetmed.2010.11.016.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n21185096\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.prevetmed.2010.11.016\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Prev.+Vet.+Med.&amp;title=Intra-+and+inter-observer+agreement+when+using+a+descriptive+classification+scale+for+clinical+assessment+of+faecal+consistency+in+growing+pigs&amp;author=KS+Pedersen&amp;author=N+Toft&amp;volume=98&amp;publication_year=2011&amp;pages=288-291&amp;pmid=21185096&amp;doi=10.1016/j.prevetmed.2010.11.016&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR31\">31. <span class=\"element-citation\">Moller I, et al.  High-throughput mapping of cell-wall polymers within and between plants using novel microarrays. <span><span class=\"ref-journal\">Plant J. </span>2007;<span class=\"ref-vol\">50</span>:1118–1128. doi: 10.1111/j.1365-313X.2007.03114.x.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n17565618\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1365-313X.2007.03114.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Plant+J.&amp;title=High-throughput+mapping+of+cell-wall+polymers+within+and+between+plants+using+novel+microarrays&amp;author=I+Moller&amp;volume=50&amp;publication_year=2007&amp;pages=1118-1128&amp;pmid=17565618&amp;doi=10.1111/j.1365-313X.2007.03114.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR32\">32. <span class=\"element-citation\">Pedersen HL, et al.  Versatile high resolution oligosaccharide microarrays for plant glycobiology and cell wall research. <span><span class=\"ref-journal\">J. Biol. Chem. </span>2012;<span class=\"ref-vol\">287</span>:39429–39438. doi: 10.1074/jbc.M112.396598.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3501085\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=3501085\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n22988248\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1074%2Fjbc.M112.396598\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Versatile+high+resolution+oligosaccharide+microarrays+for+plant+glycobiology+and+cell+wall+research&amp;author=HL+Pedersen&amp;volume=287&amp;publication_year=2012&amp;pages=39429-39438&amp;pmid=22988248&amp;doi=10.1074/jbc.M112.396598&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR33\">33. <span class=\"element-citation\">Takahashi S, Tomita J, Nishioka K, Hisada T, Nishijima M. Development of a prokaryotic universal primer for simultaneous analysis of Bacteria and Archaea using next-generation sequencing. <span><span class=\"ref-journal\">PLoS ONE. </span>2014;<span class=\"ref-vol\">9</span>:e105592. doi: 10.1371/journal.pone.0105592.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4140814\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=4140814\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n25144201\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0105592\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Development+of+a+prokaryotic+universal+primer+for+simultaneous+analysis+of+Bacteria+and+Archaea+using+next-generation+sequencing&amp;author=S+Takahashi&amp;author=J+Tomita&amp;author=K+Nishioka&amp;author=T+Hisada&amp;author=M+Nishijima&amp;volume=9&amp;publication_year=2014&amp;pages=e105592&amp;pmid=25144201&amp;doi=10.1371/journal.pone.0105592&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR34\">34. <span class=\"element-citation\">Caporaso JG, et al.  QIIME allows analysis of high-throughput community sequencing data. <span><span class=\"ref-journal\">Nat. Methods. </span>2010;<span class=\"ref-vol\">7</span>:335–336. doi: 10.1038/nmeth.f.303.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3156573\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=3156573\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n20383131\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnmeth.f.303\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=QIIME+allows+analysis+of+high-throughput+community+sequencing+data&amp;author=JG+Caporaso&amp;volume=7&amp;publication_year=2010&amp;pages=335-336&amp;pmid=20383131&amp;doi=10.1038/nmeth.f.303&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR35\">35. <span class=\"element-citation\">Schloss PD, et al.  Introducing mothur: open-source, platform-independent, community-supported software for describing and comparing microbial communities. <span><span class=\"ref-journal\">Appl. Environ. Microbiol. </span>2009;<span class=\"ref-vol\">75</span>:7537–7541. doi: 10.1128/AEM.01541-09.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2786419\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=2786419\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n19801464\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.01541-09\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microbiol.&amp;title=Introducing+mothur:+open-source,+platform-independent,+community-supported+software+for+describing+and+comparing+microbial+communities&amp;author=PD+Schloss&amp;volume=75&amp;publication_year=2009&amp;pages=7537-7541&amp;pmid=19801464&amp;doi=10.1128/AEM.01541-09&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR36\">36. <span class=\"element-citation\">McMurdie PJ, Holmes S. phyloseq: an R package for reproducible interactive analysis and graphics of microbiome census data. <span><span class=\"ref-journal\">PLoS ONE. </span>2013;<span class=\"ref-vol\">8</span>:e61217. doi: 10.1371/journal.pone.0061217.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3632530\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=3632530\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n23630581\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0061217\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=phyloseq:+an+R+package+for+reproducible+interactive+analysis+and+graphics+of+microbiome+census+data&amp;author=PJ+McMurdie&amp;author=S+Holmes&amp;volume=8&amp;publication_year=2013&amp;pages=e61217&amp;pmid=23630581&amp;doi=10.1371/journal.pone.0061217&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR37\">37. <span class=\"element-citation\">Zhang J, Kobert K, Flouri T, Stamatakis A. PEAR: a fast and accurate Illumina Paired-End reAd mergeR. <span><span class=\"ref-journal\">Bioinformatics. </span>2014;<span class=\"ref-vol\">30</span>:614–620. doi: 10.1093/bioinformatics/btt593.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3933873\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=3933873\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24142950\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtt593\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=PEAR:+a+fast+and+accurate+Illumina+Paired-End+reAd+mergeR&amp;author=J+Zhang&amp;author=K+Kobert&amp;author=T+Flouri&amp;author=A+Stamatakis&amp;volume=30&amp;publication_year=2014&amp;pages=614-620&amp;pmid=24142950&amp;doi=10.1093/bioinformatics/btt593&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR38\">38. <span class=\"element-citation\">Schmieder R, Edwards R. Quality control and preprocessing of metagenomic datasets. <span><span class=\"ref-journal\">Bioinformatics. </span>2011;<span class=\"ref-vol\">27</span>:863–864. doi: 10.1093/bioinformatics/btr026.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3051327\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=3051327\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n21278185\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtr026\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Quality+control+and+preprocessing+of+metagenomic+datasets&amp;author=R+Schmieder&amp;author=R+Edwards&amp;volume=27&amp;publication_year=2011&amp;pages=863-864&amp;pmid=21278185&amp;doi=10.1093/bioinformatics/btr026&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR39\">39. <span class=\"element-citation\">Rideout JR, et al.  Subsampled open-reference clustering creates consistent, comprehensive OTU definitions and scales to billions of sequences. <span><span class=\"ref-journal\">PeerJ. </span>2014;<span class=\"ref-vol\">2</span>:e545. doi: 10.7717/peerj.545.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4145071\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=4145071\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n25177538\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.7717%2Fpeerj.545\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=Subsampled+open-reference+clustering+creates+consistent,+comprehensive+OTU+definitions+and+scales+to+billions+of+sequences&amp;author=JR+Rideout&amp;volume=2&amp;publication_year=2014&amp;pages=e545&amp;pmid=25177538&amp;doi=10.7717/peerj.545&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR40\">40. <span class=\"element-citation\">Rognes T, Flouri T, Nichols B, Quince C, Mahe F. VSEARCH: a versatile open source tool for metagenomics. <span><span class=\"ref-journal\">PeerJ. </span>2016;<span class=\"ref-vol\">4</span>:e2584. doi: 10.7717/peerj.2584.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5075697\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=5075697\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n27781170\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.7717%2Fpeerj.2584\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=VSEARCH:+a+versatile+open+source+tool+for+metagenomics&amp;author=T+Rognes&amp;author=T+Flouri&amp;author=B+Nichols&amp;author=C+Quince&amp;author=F+Mahe&amp;volume=4&amp;publication_year=2016&amp;pages=e2584&amp;pmid=27781170&amp;doi=10.7717/peerj.2584&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR41\">41. <span class=\"element-citation\">Yilmaz P, et al.  The SILVA and “All-species Living Tree Project (LTP)” taxonomic frameworks. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2014;<span class=\"ref-vol\">42</span>:D643–648. doi: 10.1093/nar/gkt1209.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3965112\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=3965112\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24293649\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgkt1209\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+SILVA+and+“All-species+Living+Tree+Project+(LTP)”+taxonomic+frameworks&amp;author=P+Yilmaz&amp;volume=42&amp;publication_year=2014&amp;pages=D643-648&amp;pmid=24293649&amp;doi=10.1093/nar/gkt1209&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR42\">42. <span class=\"element-citation\">Bolger AM, Lohse M, Usadel B. Trimmomatic: a flexible trimmer for Illumina sequence data. <span><span class=\"ref-journal\">Bioinformatics. </span>2014;<span class=\"ref-vol\">30</span>:2114–2120. doi: 10.1093/bioinformatics/btu170.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4103590\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=4103590\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24695404\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtu170\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Trimmomatic:+a+flexible+trimmer+for+Illumina+sequence+data&amp;author=AM+Bolger&amp;author=M+Lohse&amp;author=B+Usadel&amp;volume=30&amp;publication_year=2014&amp;pages=2114-2120&amp;pmid=24695404&amp;doi=10.1093/bioinformatics/btu170&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR43\">43. <span class=\"element-citation\">Nurk S, Meleshko D, Korobeynikov A, Pevzner P. A. metaSPAdes: a new versatile metagenomic assembler. <span><span class=\"ref-journal\">Genome Res. </span>2017;<span class=\"ref-vol\">27</span>:824–834. doi: 10.1101/gr.213959.116.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5411777\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=5411777\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n28298430\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1101%2Fgr.213959.116\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=A.+metaSPAdes:+a+new+versatile+metagenomic+assembler&amp;author=S+Nurk&amp;author=D+Meleshko&amp;author=A+Korobeynikov&amp;author=P+Pevzner&amp;volume=27&amp;publication_year=2017&amp;pages=824-834&amp;pmid=28298430&amp;doi=10.1101/gr.213959.116&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR44\">44. <span class=\"element-citation\">Li D, Liu CM, Luo R, Sadakane K, Lam TW. MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph. <span><span class=\"ref-journal\">Bioinformatics. </span>2015;<span class=\"ref-vol\">31</span>:1674–1676. doi: 10.1093/bioinformatics/btv033.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n25609793\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtv033\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=MEGAHIT:+an+ultra-fast+single-node+solution+for+large+and+complex+metagenomics+assembly+via+succinct+de+Bruijn+graph&amp;author=D+Li&amp;author=CM+Liu&amp;author=R+Luo&amp;author=K+Sadakane&amp;author=TW+Lam&amp;volume=31&amp;publication_year=2015&amp;pages=1674-1676&amp;pmid=25609793&amp;doi=10.1093/bioinformatics/btv033&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR45\">45. <span class=\"element-citation\">Kang DD, Froula J, Egan R, Wang Z. MetaBAT, an efficient tool for accurately reconstructing single genomes from complex microbial communities. <span><span class=\"ref-journal\">PeerJ. </span>2015;<span class=\"ref-vol\">3</span>:e1165. doi: 10.7717/peerj.1165.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4556158\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=4556158\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n26336640\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.7717%2Fpeerj.1165\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=MetaBAT,+an+efficient+tool+for+accurately+reconstructing+single+genomes+from+complex+microbial+communities&amp;author=DD+Kang&amp;author=J+Froula&amp;author=R+Egan&amp;author=Z+Wang&amp;volume=3&amp;publication_year=2015&amp;pages=e1165&amp;pmid=26336640&amp;doi=10.7717/peerj.1165&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR46\">46. <span class=\"element-citation\">Olm MR, Brown CT, Brooks B, Banfield JF. dRep: a tool for fast and accurate genomic comparisons that enables improved genome recovery from metagenomes through de-replication. <span><span class=\"ref-journal\">ISME J. </span>2017;<span class=\"ref-vol\">11</span>:2864–2868. doi: 10.1038/ismej.2017.126.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5702732\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=5702732\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n28742071\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fismej.2017.126\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=ISME+J.&amp;title=dRep:+a+tool+for+fast+and+accurate+genomic+comparisons+that+enables+improved+genome+recovery+from+metagenomes+through+de-replication&amp;author=MR+Olm&amp;author=CT+Brown&amp;author=B+Brooks&amp;author=JF+Banfield&amp;volume=11&amp;publication_year=2017&amp;pages=2864-2868&amp;pmid=28742071&amp;doi=10.1038/ismej.2017.126&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR47\">47. <span class=\"element-citation\">Ondov BD, et al.  Mash: fast genome and metagenome distance estimation using MinHash. <span><span class=\"ref-journal\">Genome Biol. </span>2016;<span class=\"ref-vol\">17</span>:132. doi: 10.1186/s13059-016-0997-x.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4915045\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=4915045\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n27323842\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2Fs13059-016-0997-x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Biol.&amp;title=Mash:+fast+genome+and+metagenome+distance+estimation+using+MinHash&amp;author=BD+Ondov&amp;volume=17&amp;publication_year=2016&amp;pages=132&amp;pmid=27323842&amp;doi=10.1186/s13059-016-0997-x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR48\">48. <span class=\"element-citation\">Parks DH, Imelfort M, Skennerton CT, Hugenholtz P, Tyson GW. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. <span><span class=\"ref-journal\">Genome Res. </span>2015;<span class=\"ref-vol\">25</span>:1043–1055. doi: 10.1101/gr.186072.114.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4484387\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=4484387\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n25977477\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1101%2Fgr.186072.114\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=CheckM:+assessing+the+quality+of+microbial+genomes+recovered+from+isolates,+single+cells,+and+metagenomes&amp;author=DH+Parks&amp;author=M+Imelfort&amp;author=CT+Skennerton&amp;author=P+Hugenholtz&amp;author=GW+Tyson&amp;volume=25&amp;publication_year=2015&amp;pages=1043-1055&amp;pmid=25977477&amp;doi=10.1101/gr.186072.114&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR49\">49. <span class=\"mixed-citation\">Chaumeil, P. A., Mussig, A. J., Hugenholtz, P. &amp; Parks, D. H. GTDB-Tk: a toolkit to classify genomes with the Genome Taxonomy Database. <em>Bioinformatics</em>10.1093/bioinformatics/btz848 (2019). <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7703759\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=7703759\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n31730192\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>]</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR50\">50. <span class=\"element-citation\">Seemann T. Prokka: rapid prokaryotic genome annotation. <span><span class=\"ref-journal\">Bioinformatics. </span>2014;<span class=\"ref-vol\">30</span>:2068–2069. doi: 10.1093/bioinformatics/btu153.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24642063\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtu153\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Prokka:+rapid+prokaryotic+genome+annotation&amp;author=T+Seemann&amp;volume=30&amp;publication_year=2014&amp;pages=2068-2069&amp;pmid=24642063&amp;doi=10.1093/bioinformatics/btu153&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR51\">51. <span class=\"element-citation\">Terrapon N, Lombard V, Gilbert HJ, Henrissat B. Automatic prediction of polysaccharide utilization loci in Bacteroidetes species. <span><span class=\"ref-journal\">Bioinformatics. </span>2014;<span class=\"ref-vol\">31</span>:647–655. doi: 10.1093/bioinformatics/btu716.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n25355788\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtu716\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Automatic+prediction+of+polysaccharide+utilization+loci+in+Bacteroidetes+species&amp;author=N+Terrapon&amp;author=V+Lombard&amp;author=HJ+Gilbert&amp;author=B+Henrissat&amp;volume=31&amp;publication_year=2014&amp;pages=647-655&amp;pmid=25355788&amp;doi=10.1093/bioinformatics/btu716&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR52\">52. <span class=\"element-citation\">Lombard V, Golaconda Ramulu H, Drula E, Coutinho PM, Henrissat B. The carbohydrate-active enzymes database (CAZy) in 2013. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2014;<span class=\"ref-vol\">42</span>:D490–495. doi: 10.1093/nar/gkt1178.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3965031\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=3965031\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24270786\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgkt1178\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+carbohydrate-active+enzymes+database+(CAZy)+in+2013&amp;author=V+Lombard&amp;author=H+Golaconda+Ramulu&amp;author=E+Drula&amp;author=PM+Coutinho&amp;author=B+Henrissat&amp;volume=42&amp;publication_year=2014&amp;pages=D490-495&amp;pmid=24270786&amp;doi=10.1093/nar/gkt1178&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR53\">53. <span class=\"element-citation\">Arntzen MO, Karlskas IL, Skaugen M, Eijsink VG. Proteomic investigation of the response of <em>Enterococcus faecalis</em> V583 when cultivated in urine. <span><span class=\"ref-journal\">PLoS ONE. </span>2015;<span class=\"ref-vol\">10</span>:e0126694. doi: 10.1371/journal.pone.0126694.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4411035\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=4411035\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n25915650\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0126694\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Proteomic+investigation+of+the+response+of+Enterococcus+faecalis+V583+when+cultivated+in+urine&amp;author=MO+Arntzen&amp;author=IL+Karlskas&amp;author=M+Skaugen&amp;author=VG+Eijsink&amp;volume=10&amp;publication_year=2015&amp;pages=e0126694&amp;pmid=25915650&amp;doi=10.1371/journal.pone.0126694&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR54\">54. <span class=\"element-citation\">Arntzen MO, Karlskas IL, Skaugen M, Eijsink VG, Mathiesen G. Proteomic investigation of the response of <em>Enterococcus faecalis</em> V583 when cultivated in urine. <span><span class=\"ref-journal\">PLoS ONE. </span>2015;<span class=\"ref-vol\">10</span>:e0126694. doi: 10.1371/journal.pone.0126694.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4411035\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=4411035\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n25915650\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0126694\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Proteomic+investigation+of+the+response+of+Enterococcus+faecalis+V583+when+cultivated+in+urine&amp;author=MO+Arntzen&amp;author=IL+Karlskas&amp;author=M+Skaugen&amp;author=VG+Eijsink&amp;author=G+Mathiesen&amp;volume=10&amp;publication_year=2015&amp;pages=e0126694&amp;pmid=25915650&amp;doi=10.1371/journal.pone.0126694&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR55\">55. <span class=\"element-citation\">Cox J, Mann M. MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. <span><span class=\"ref-journal\">Nat. Biotechnol. </span>2008;<span class=\"ref-vol\">26</span>:1367–1372. doi: 10.1038/nbt.1511.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n19029910\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnbt.1511\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=MaxQuant+enables+high+peptide+identification+rates,+individualized+p.p.b.-range+mass+accuracies+and+proteome-wide+protein+quantification&amp;author=J+Cox&amp;author=M+Mann&amp;volume=26&amp;publication_year=2008&amp;pages=1367-1372&amp;pmid=19029910&amp;doi=10.1038/nbt.1511&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR56\">56. <span class=\"element-citation\">Cox J, et al.  Accurate proteome-wide label-free quantification by delayed normalization and maximal peptide ratio extraction, termed MaxLFQ. <span><span class=\"ref-journal\">Mol. Cell. Proteom. </span>2014;<span class=\"ref-vol\">13</span>:2513–2526. doi: 10.1074/mcp.M113.031591.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4159666\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=4159666\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24942700\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1074%2Fmcp.M113.031591\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Proteom.&amp;title=Accurate+proteome-wide+label-free+quantification+by+delayed+normalization+and+maximal+peptide+ratio+extraction,+termed+MaxLFQ&amp;author=J+Cox&amp;volume=13&amp;publication_year=2014&amp;pages=2513-2526&amp;doi=10.1074/mcp.M113.031591&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR57\">57. <span class=\"element-citation\">Hug LA, et al.  A new view of the tree of life. <span><span class=\"ref-journal\">Nat. Microbiol. </span>2016;<span class=\"ref-vol\">1</span>:16048. doi: 10.1038/nmicrobiol.2016.48.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n27572647\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnmicrobiol.2016.48\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Microbiol.&amp;title=A+new+view+of+the+tree+of+life&amp;author=LA+Hug&amp;volume=1&amp;publication_year=2016&amp;pages=16048&amp;pmid=27572647&amp;doi=10.1038/nmicrobiol.2016.48&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR58\">58. <span class=\"element-citation\">Wu D, Jospin G, Eisen JA. Systematic identification of gene families for use as “markers” for phylogenetic and phylogeny-driven ecological studies of bacteria and archaea and their major subgroups. <span><span class=\"ref-journal\">PLoS ONE. </span>2013;<span class=\"ref-vol\">8</span>:e77033. doi: 10.1371/journal.pone.0077033.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3798382\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=3798382\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24146954\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0077033\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Systematic+identification+of+gene+families+for+use+as+“markers”+for+phylogenetic+and+phylogeny-driven+ecological+studies+of+bacteria+and+archaea+and+their+major+subgroups&amp;author=D+Wu&amp;author=G+Jospin&amp;author=JA+Eisen&amp;volume=8&amp;publication_year=2013&amp;pages=e77033&amp;pmid=24146954&amp;doi=10.1371/journal.pone.0077033&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR59\">59. <span class=\"element-citation\">Mikheenko A, Saveliev V, Gurevich A. MetaQUAST: evaluation of metagenome assemblies. <span><span class=\"ref-journal\">Bioinformatics. </span>2016;<span class=\"ref-vol\">32</span>:1088–1090. doi: 10.1093/bioinformatics/btv697.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n26614127\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtv697\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=MetaQUAST:+evaluation+of+metagenome+assemblies&amp;author=A+Mikheenko&amp;author=V+Saveliev&amp;author=A+Gurevich&amp;volume=32&amp;publication_year=2016&amp;pages=1088-1090&amp;pmid=26614127&amp;doi=10.1093/bioinformatics/btv697&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR60\">60. <span class=\"element-citation\">Edgar RC. MUSCLE: multiple sequence alignment with high accuracy and high throughput. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2004;<span class=\"ref-vol\">32</span>:1792–1797. doi: 10.1093/nar/gkh340.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC390337\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=390337\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n15034147\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgkh340\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=MUSCLE:+multiple+sequence+alignment+with+high+accuracy+and+high+throughput&amp;author=RC+Edgar&amp;volume=32&amp;publication_year=2004&amp;pages=1792-1797&amp;pmid=15034147&amp;doi=10.1093/nar/gkh340&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR61\">61. <span class=\"element-citation\">Talavera G, Castresana J. Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments. <span><span class=\"ref-journal\">Syst. Biol. </span>2007;<span class=\"ref-vol\">56</span>:564–577. doi: 10.1080/10635150701472164.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n17654362\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1080%2F10635150701472164\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Syst.+Biol.&amp;title=Improvement+of+phylogenies+after+removing+divergent+and+ambiguously+aligned+blocks+from+protein+sequence+alignments&amp;author=G+Talavera&amp;author=J+Castresana&amp;volume=56&amp;publication_year=2007&amp;pages=564-577&amp;pmid=17654362&amp;doi=10.1080/10635150701472164&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR62\">62. <span class=\"element-citation\">Stamatakis A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. <span><span class=\"ref-journal\">Bioinformatics. </span>2014;<span class=\"ref-vol\">30</span>:1312–1313. doi: 10.1093/bioinformatics/btu033.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3998144\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=3998144\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24451623\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtu033\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=RAxML+version+8:+a+tool+for+phylogenetic+analysis+and+post-analysis+of+large+phylogenies&amp;author=A+Stamatakis&amp;volume=30&amp;publication_year=2014&amp;pages=1312-1313&amp;pmid=24451623&amp;doi=10.1093/bioinformatics/btu033&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR63\">63. <span class=\"element-citation\">Letunic I, Bork P. Interactive tree of life (iTOL) v3: an online tool for the display and annotation of phylogenetic and other trees. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2016;<span class=\"ref-vol\">44</span>:W242–W245. doi: 10.1093/nar/gkw290.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987883\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=4987883\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n27095192\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgkw290\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Interactive+tree+of+life+(iTOL)+v3:+an+online+tool+for+the+display+and+annotation+of+phylogenetic+and+other+trees&amp;author=I+Letunic&amp;author=P+Bork&amp;volume=44&amp;publication_year=2016&amp;pages=W242-W245&amp;pmid=27095192&amp;doi=10.1093/nar/gkw290&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR64\">64. <span class=\"mixed-citation\">Pope, P. B. Annotated MAG files from the pig gut microbiome. Preprint at 10.6084/m6089.figshare.9816581.v9816581 (2020).</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR65\">65. <span class=\"element-citation\">Vizcaino JA, et al.  The PRoteomics IDEntifications (PRIDE) database and associated tools: status in 2013. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2013;<span class=\"ref-vol\">41</span>:D1063–1069. doi: 10.1093/nar/gks1262.</span> <span class=\"nowrap\">[<a tabindex=\"0\" class=\"int-reflink\" data-bs-toggle=\"popover\" data-bs-trigger=\"focus\" data-bs-html=\"true\" data-bs-content=\"<div class=\'row justify-content-center text-center\'><div class=\'col\'><a target=\'_BLANK\' href=\'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531176\'> Read From<br>PubMedCentral </a></div><div class=\'col\'><a target=\'_BLANK\' href=\'./utils/insertAndGo.php?ORIGIN=pubmed&amp;ID=3531176\'> Read From<br>BiblioTool </a></div></div>\" data-bs-original-title=\"\" title=\"\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n23203882\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgks1262\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+PRoteomics+IDEntifications+(PRIDE)+database+and+associated+tools:+status+in+2013&amp;author=JA+Vizcaino&amp;volume=41&amp;publication_year=2013&amp;pages=D1063-1069&amp;pmid=23203882&amp;doi=10.1093/nar/gks1262&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR66\">66. <span class=\"element-citation\">Willför S, et al.  Characterisation of water-soluble galactoglucomannans from Norway spruce wood and thermomechanical pulp. <span><span class=\"ref-journal\">Carbohydr. Polym. </span>2003;<span class=\"ref-vol\">52</span>:175–187. doi: 10.1016/S0144-8617(02)00288-6.</span> [<a href=\"//dx.doi.org/10.1016%2FS0144-8617(02)00288-6\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Carbohydr.+Polym.&amp;title=Characterisation+of+water-soluble+galactoglucomannans+from+Norway+spruce+wood+and+thermomechanical+pulp&amp;author=S+Willför&amp;volume=52&amp;publication_year=2003&amp;pages=175-187&amp;doi=10.1016/S0144-8617(02)00288-6&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div></div><br>', NULL, 2);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(19, 'pubmed', '53', 'The interaction of phospholipase A2 with micellar interfaces. The role of the N-terminal region.', 'The localization of the previously postulated interface recognition site (IRS) in porcine pancreatic phospholipase A2, required for a specific interaction between the enzyme and organized lipid-water interfaces, was investigated by ultraviolet difference spectroscopy, by measurements of the intrinsic fluorescence of the unique Trp residue, and by protection experiments against specific tryptic hydrolysis. Using the enzymically nondegradable substrate analogues: CnH(2n+1)(0-)OOCH2CH2N+(CH3)3-(H,OH), it is shown that the rather hydrophobic N-terminal sequence of the enzyme, viz., Ala-Leu-Trp-Gln-Phe-Arg, is directly involved in the interaction with the lipid-water interface. Besides hydrophobic probably also polar interactions contribute to the binding process. At neutral or acidic pH the presence of a salt bridge between the N-terminal alpha-NH3+ group and a negatively charged side chain stablizes the interface recognition site and allows the enzyme to penetrate micellar surfaces, even in the absence of metal ion. At alkaline pH, interaction of the enzyme with micellar interfaces requires the presence of Ca2+ (Ba2+) ions.', 1975, 'Biochemistry', '', 1, NULL, NULL, 1),
(20, 'pubmed', '31712689', 'Light on the cell cycle of the non-photosynthetic bacterium Ramlibacter tataouinensis.', 'Ramlibacter tataouinensis TTB310, a non-photosynthetic betaproteobacterium isolated from a semi-arid region of southern Tunisia, forms both rods and cysts. Cysts are resistant to desiccation and divide when water and nutrients are available. Rods are motile and capable of dissemination. Due to the strong correlation between sunlight and desiccation, light is probably an important external signal for anticipating desiccating conditions. Six genes encoding potential light sensors were identified in strain TTB310. Two genes encode for bacteriophytochromes, while the four remaining genes encode for putative blue light receptors. We determined the spectral and photochemical properties of the two recombinant bacteriophytochromes RtBphP1 and RtBphP2. In both cases, they act as sensitive red light detectors. Cyst divisions and a complete cyst-rod-cyst cycle are the main processes in darkness, whereas rod divisions predominate in red or far-red light. Mutant phenotypes caused by the inactivation of genes encoding bacteriophytochromes or heme oxygenase clearly show that both bacteriophytochromes are involved in regulating the rod-rod division. This process could favor rapid rod divisions at sunrise, after dew formation but before the progressive onset of desiccation. Our study provides the first evidence of a light-based strategy evolved in a non-photosynthetic bacterium to exploit scarse water in a desert environment.', 2019, 'Scientific reports', 'PMC6848086', 1, NULL, NULL, 5),
(21, 'pubmed', '31624160', 'Complete Genome Sequences of Four  Strains Isolated from Metal- and Radionuclide-Rich Soils.', 'Here, we present the genome sequences of four  strains, which were isolated at different locations in Europe from metal- or radionuclide-rich soils. High-quality complete genome sequences were obtained with PacBio and Illumina data sets with an original two-step procedure.', 2019, 'Microbiology resource announcements', 'PMC6797525', 2, NULL, NULL, 3),
(24, 'pubmed', '56', 'Kinetic light scattering studies on the dissociation of hemoglobin from Lumbricus terrestris.', 'The kinetics of the pH-induced dissociation of the 3 X 10(6) mol wt hemoglobin from Lumbricus terrestris (the earthworm) have been studied in a light-scattering stopped-flow apparatus. The ligand dependent dissociation data were fit well by a simple sequential model. The data for CO and oxyhemoglobin are consistent with Hb12 leads to 2Hb6 leads to 12Hb. Methemoglobin at pH 7 appears to be hexameric and the dissociation is consistent with the model: Hb6 leads to 6Hb. In a sequential decay scheme for which light-scattering changes are monitored, the relative amounts of rapid and slow phase are determined by the rate constants as well as the molecular weights of intermediate species. Assignment of the hexameric intermediate is supported by an investigation of the sensitivity of the theoretical kinetic curves to the molecular weights of the intermediates. This assignment is further supported by the following: (1) the same model will fit the data for oxy- and CO-hemoglobin at all three temperatures (a 24-29-fold variation in rate constants), (2) evidence from electron microscopy shows hexameric forms, and (3) methemoglobin is apparently stable as a hexamer at pH 7. When CO replaces O2 as the ligand, the dissociation rate increases by a factor of four. The met is about 20 times faster than the initial oxyhemoglobin dissociation rate, but perhaps more relevant for comparing dissociation of the hexamer, the met rate was respectively 100 times and 500 times faster than that for the assumed hexameric forms of CO- and oxy-hemoglobin. The activation energies for the dodecamer to hexamer dissociation and for the dissociation of the hexamer to smaller forms were about 30 kcal/mol for oxy-, CO-, and methemoglobin.', 1975, 'Biochemistry', '', 2, NULL, NULL, 4),
(25, 'pubmed', '33674431', 'Novel 3--α-d-Galactosyl-α-l-Arabinofuranosidase for the Assimilation of Gum Arabic Arabinogalactan Protein in Bifidobacterium longum subsp. .', 'Gum arabic arabinogalactan (AG) protein (AGP) is a unique dietary fiber that is degraded and assimilated by only specific strains of  subsp.  Here, we identified a novel 3--α-d-galactosyl-α-l-arabinofuranosidase (GAfase) from  JCM7052 and classified it into glycoside hydrolase family 39 (GH39). GAfase released α-d-Gal-(1→3)-l-Ara and β-l-Ara-(1→3)-l-Ara from gum arabic AGP and β-l-Ara-(1→3)-l-Ara from larch AGP, and the α-d-Gal-(1→3)-l-Ara release activity was found to be 594-fold higher than that of β-l-Ara-(1→3)-l-Ara. The GAfase gene was part of a gene cluster that included genes encoding a GH36 α-galactosidase candidate and ABC transporters for the assimilation of the released α-d-Gal-(1→3)-l-Ara in  Notably, when α-d-Gal-(1→3)-l-Ara was removed from gum arabic AGP, it was assimilated by both  JCM7052 and the nonassimilative  JCM1217, suggesting that the removal of α-d-Gal-(1→3)-l-Ara from gum arabic AGP by GAfase permitted the cooperative action with type II AG degradative enzymes in  The present study provides new insight into the mechanism of gum arabic AGP degradation in  Bifidobacteria harbor numerous carbohydrate-active enzymes that degrade several dietary fibers in the gastrointestinal tract.  JCM7052 is known to exhibit the ability to assimilate gum arabic AGP, but the key enzyme involved in the degradation of gum arabic AGP remains unidentified. Here, we cloned and characterized a GH39 3--α-d-galactosyl-α-l-arabinofuranosidase (GAfase) from  JCM7052. The enzyme was responsible for the release of α-d-Gal-(1→3)-l-Ara and β-l-Ara-(1→3)-l-Ara from gum arabic AGP. The presence of a gene cluster including the GAfase gene is specifically observed in gum arabic AGP assimilative strains. However, GAfase carrier strains may affect GAfase noncarrier strains that express other type II AG degradative enzymes. These findings provide insights into the bifidogenic effect of gum arabic AGP.', 2021, 'Applied and environmental microbiology', '', 3, NULL, NULL, 6),
(26, 'pubmed', '33394289', 'Substrate Specificities of GH8, GH39, and GH52 β-xylosidases from Bacillus halodurans C-125 Toward Substituted Xylooligosaccharides.', 'Substrate specificities of glycoside hydrolase families 8 (Rex), 39 (BhXyl39), and 52 (BhXyl52) β-xylosidases from Bacillus halodurans C-125 were investigated. BhXyl39 hydrolyzed xylotriose most efficiently among the linear xylooligosaccharides. The activity decreased in the order of xylohexaose > xylopentaose > xylotetraose and it had little effect on xylobiose. In contrast, BhXyl52 hydrolyzed xylobiose and xylotriose most efficiently, and its activity decreased when the main chain became longer as follows: xylotetraose > xylopentaose > xylohexaose. Rex produced O-β-D-xylopyranosyl-(1 → 4)-[O-α-L-arabinofuranosyl-(1 → 3)]-O-β-D-xylopyranosyl-(1 → 4)-β-D-xylopyranose (AraXyl) and O-β-D-xylopyranosyl-(1 → 4)-[O-4-O-methyl-α-D-glucuronopyranosyl-(l → 2)]-β-D-xylopyranosyl-(1 → 4)-β-D-xylopyranose (MeGlcAXyl), which lost a xylose residue from the reducing end of O-β-D-xylopyranosyl-(1 → 4)-[O-α-L-arabinofuranosyl-(1 → 3)]-O-β-D-xylopyranosyl-(1 → 4)-β-D-xylopyranosyl-(1 → 4)-β-D-xylopyranose (AraXyl) and O-β-D-xylopyranosyl-(1 → 4)-[O-4-O-methyl-α-D-glucuronopyranosyl-(1 → 2)]-β-D-xylopyranosyl-(1 → 4)-β-D-xylopyranosyl-(1 → 4)-β-D-xylopyranose (MeGlcAXyl). It was considered that there is no space to accommodate side chains at subsite -1. BhXyl39 rapidly hydrolyzes the non-reducing-end xylose linkages of MeGlcAXyl, while the arabinose branch does not significantly affect the enzyme activity because it degrades AraXyl as rapidly as unmodified xylotetraose. The model structure suggested that BhXyl39 enhanced the activity for MeGlcAXyl by forming a hydrogen bond between glucuronic acid and Lys265. BhXyl52 did not hydrolyze AraXyl and MeGlcAXyl because it has a narrow substrate binding pocket and 2- and 3-hydroxyl groups of xylose at subsite +1 hydrogen bond to the enzyme.', 2021, 'Applied biochemistry and biotechnology', '', 4, NULL, NULL, 5),
(27, 'pubmed', '32880690', 'Characterization and functional analysis of two novel thermotolerant α-L-arabinofuranosidases belonging to glycoside hydrolase family 51 from Thielavia terrestris and family 62 from Eupenicillium parvum.', 'Arabinofuranose substitutions on xylan are known to interfere with enzymatic hydrolysis of this primary hemicellulose. In this work, two novel α-L-arabinofuranosidases (ABFs), TtABF51A from Thielavia terrestris and EpABF62C from Eupenicillium parvum, were characterized and functionally analyzed. From sequences analyses, TtABF51A and EpABF62C belong to glycoside hydrolase (GH) families 51 and 62, respectively. Recombinant TtABF51A showed high activity on 4-nitrophenyl-α-L-arabinofuranoside (83.39 U/mg), low-viscosity wheat arabinoxylan (WAX, 39.66 U/mg), high-viscosity rye arabinoxylan (RAX, 32.24 U/mg), and sugarbeet arabinan (25.69 U/mg), while EpABF62C preferred to degrade arabinoxylan. For EpABF62C, the rate of hydrolysis of RAX (94.10 U/mg) was 2.1 times that of WAX (45.46 U/mg). The optimal pH and reaction temperature for the two enzymes was between 4.0 and 4.5 and 65 °C, respectively. Calcium played an important role in the thermal stability of EpABF62C. TtABF51A and EpABF62C showed the highest thermal stabilities at pH 4.5 or 5.0, respectively. At their optimal pHs, TtABF51A and EpABF62C retained greater than 80% of their initial activities after incubation at 55 °C for 96 h or 144 h, respectively. H NMR analysis indicated that the two enzymes selectively removed arabinose linked to C-3 of mono-substituted xylose residues in WAX. Compared with the singular application of the GH10 xylanase EpXYN1 from E. parvum, co-digestions of WAX including TtABF51A and/or EpABF62C released 2.49, 3.38, and 4.81 times xylose or 3.38, 1.65, and 2.57 times of xylobiose, respectively. Meanwhile, the amount of arabinose released from WAX by TtABF51A with EpXYN1 was 2.11 times the amount with TtABF51A alone. KEY POINTS: • Two novel α-l-arabinofuranosidases (ABFs) displayed high thermal stability. • The thermal stability of GH62 family EpABF62C was dependent on calcium. • Buffer pH affects the thermal stability of the two ABFs. • Both ABFs enhance the hydrolysis of WAX by a GH10 xylanase.', 2020, 'Applied microbiology and biotechnology', 'PMC7502447', NULL, NULL, NULL, 1),
(28, 'pubmed', '32679333', 'Biochemical characterization of a novel halo/organic-solvents/final-products tolerant GH39 xylosidase from saline soil and its synergic action with xylanase.', 'Xylosidases with tolerance to high concentration of salts, organic solvents, and enzyme hydrolytic products are preferential for industrial application but were rarely reported. In this study, a novel xylosidase XYL21 belong to glycoside hydrolase 39 was characterized with optimal temperature of 45 °C and optimal pH of 5.50. Different to other GH39 xylosidases, XYL21 had excellent tolerance to salts, the activity of which is not inhibited but slightly increased in 0.50-1.50 M NaCl. It is also tolerant to organic solvents, especially retaining 105.18% relative activity even in the presence of 15.00% (v/v) ethanol. Moreover, XYL21 was insensitive to the final lignocellulose hydrolysis products including glucose, xylose, arabinose, mannose and galactose, which retains 111.36% and 53.49% relative activity in 0.30 and 0.90 M xylose, respectively. Further structural modeling analysis indicated that its excellent tolerance may be attributed to its high structural flexibility caused by the high proportion of random coils. Furthermore, XYL21 had a wide substrate specificity to catalyze xylan and xylo-oligosaccharides, and it significantly cooperated with xylanase to improve the hydrolysis efficiency with 1.52-fold. Considering these unique properties, XYL21 is a good candidate for both basic research and various potential industrial applications such as seafood processing and bioethanol production.', 2020, 'International journal of biological macromolecules', '', 4, NULL, NULL, 2),
(29, 'pubmed', '31304065', 'Production and purification of mannan oligosaccharide with epithelial tight junction enhancing activity.', 'Mannanan oligosaccharide (MOS) is well-known as effective supplement food for livestock to increase their nutrients absorption and health status, but the structure and identification of bioactive MOS remain unclear. In this study, MOS production was accomplished, using enzymatic hydrolysis of pretreated coconut meal substrate with recombinant mannanase.The mannanase gene was cloned from AE24, then expressed in BL21. Purified Mannanase exhibit stability over a wide range of pH and temperature from pH 6-8 and 4 °C to 70 °C, respectively. SEM analysis revealed that sonication could change the surface characteristic of copra meal, which gave better MOS yield, compared to untreated substrates. The separation and purification of each MOS were achieved using Biogel-P2 column chromatography. Determination of biological active MOS species was also investigated. T84 cells were cultured and treated with each of the purified MOS species to determine their tight junction enhancing activity.Scanning electron microscope imaging showed that pretreatment using sonication could disrupt the surface of copra meal better than grinding alone, which can improve the production of MOS. Pentamer of MOS (M5) significantly increased tight junction integration of T84 cells measured with TEER ( < 0.0001).', 2019, 'PeerJ', 'PMC6611449', 3, NULL, NULL, 3),
(30, 'pubmed', '54', 'Phospholipase A2 as a probe of phospholipid distribution in erythrocyte membranes. Factors influencing the apparent specificity of the reaction.', 'The action of snake venom phospholipases A2 in intact human erythrocytes was investigated in detail. The basis phospholipase from Agkistrodon halys blomhifii was found to induce both hydrolysis of membrane phospholipids and total cell hemolysis under certain experimental conditions. The hydrolytic action of the basic enzyme was found to consist of two sequential events: (a) hydrolysis of 70% of the total cell ph osphatidylcholine without any evident hemolysis; and (b) complete hydrolysis of the remaining phosphatidylcholine, followed closely by extensive phosphatidylethanolamine hydrolysis and finally with onset of hemolysis, attack on the phosphatidylserine. At pH 7.4 and 10 mM Ca2+ only stage (a) occurred. However, a slight elevation of the pH of incubation to pH 8.0 and/or inclusion of 40 mM Ca2+ in the reaction mixture caused both events (a) and (b) to occur. The addition of glucose limited the action of the enzyme to stage (a) under any reaction conditions. An investigation showed that enzymically induced hemolysis occurred under conditions where the intracellular ATP levels were lowered. Data are presented which suggest that stage (b) is mediated by in influx of Ca2+ into the cell when the levels of ATP are low. Interestingly the phosphllipase from Naja naja venom (Pakistan) yielded results similar to those observed with the basic enzyme from Agkistrodon venom. However, the enzyme from Crotalus adamanteus and the acidic enzyme also present in the Agkistrodon venom produced only slight hydrolysis or hemolysis under any of the conditions studied. Other species of erythrocytes, e.g., guinea pig, monkey, pig, and rat, were tested but only those from guinea pig behaved similarly to the human cells. Pig, monkey, and rat erythrocytes underwent very limited hydrolysis and hemolysis. It is evident that the use of these phospholipases to probe the localization of phospholipds in erythrocyte membranes must be approached with caution. Certain facets of this problem are discussed.', 1975, 'Biochemistry', '', 4, NULL, NULL, 1),
(31, 'pubmed', '32338906', 'Engineering of β-Glucosidase Bgl15 with Simultaneously Enhanced Glucose Tolerance and Thermostability To Improve Its Performance in High-Solid Cellulose Hydrolysis.', 'In this study, a Petri-dish-based double-layer high-throughput screening method was established to improve glucose tolerance of β-glucosidase Bgl15. Two beneficial mutations were identified, and the joint mutant 2R1 improved the half-maximal inhibitory concentration of glucose from 0.04 to 2.1 M. The crystal structure of 2R1 was subsequently determined at 2.7 Å. Structure analysis revealed that enhancement of glucose tolerance may be due to improved transglycosylation activity made possible by a hydrophobic binding site for glucose as an acceptor and more stringent control of a putative water channel. To further ameliorate the application potential of the enzyme, it was engineered to increase the half-life at 50 °C from 0.8 h (Bgl15) to 180 h (mutant 5R1). Furthermore, supplementation of 5R1 to the cellulase cocktail significantly improved glucose production from pretreated sugar cane bagasse by 38%. Consequently, this study provided an efficient approach to enhance glucose tolerance and generated a promising catalyst for cellulose saccharification.', 2020, 'Journal of agricultural and food chemistry', '', 1, NULL, NULL, 5),
(32, 'pubmed', '55', 'Subunit interactions in yeast glyceraldehyde-3-phosphate dehydrogenase.', 'The spontaneous inactivation of yeast glyceraldehyde-3-phosphate dehydrogenase was found to fit a simple two-state model at pH 8.5 and 25 degrees. The first step is a relatively rapid dissociation of the tetramer to dimers with the equilibrium largely in favor of the tetramer. In the absence of NAD+ the dimer inactivates irreversibly. The apoenzyme is quite stable with a half-life for complete activity loss proportional to the square root of the enzyme concentration. Perturbances of the protein structure (by pH, ionic strength, and specific salts), which have no effect on the tetrameric state of the molecule, result in an alteration of the cooperativity of NAD+ binding, the reactivity of the active-site sulfhydryl group, and the catalytic activity of the enzyme. Covalent modification of two of the four active-site sulfhydryl groups has profound effects on the enzymic activity which are mediated by changes in the subunit interactions. Sedimentation analysis and hybridization studies indicate that the interaction between subunits remains strong after covalent modification. Under normal physiological and equilibrium dialysis conditions the protein is a tetramer. Equilibrium dialysis studies of NAD+ binding to the enzyme at pH 8.5 and 25 degrees reveal a mixed cooperativity pattern. A model consistent with these observations and the observed half-of-the-sites reactivity is that of ligand induced sequential conformational changes which are transferred across strongly interacting subunit domains. Methods for distinguishing negatively cooperative binding patterns from mixtures of denatured enzyme and multiple species are discussed.', 1975, 'Biochemistry', '', 1, NULL, NULL, 3),
(33, 'pubmed', '28917914', 'Structure, enzymatic transformation, anticancer activity of fucoidan and sulphated fucooligosaccharides from Sargassum horneri.', 'Structure and anticancer activity of fucoidan from Sargassum horneri and from products of its enzymatic transformation were investigated. A gene that encodes fucoidanase ffa1 in the marine bacteria F. algae was identified, cloned and the protein (FFA1) was produced in Escherichia coli. The mass of the gene product FFA1 is 111kDa. Sequence analysis has revealed that fucoidanase FFA1 belongs to the GH107 (CAZy) family. Recombinant fucoidanase FFA1 was used to produce fucooligosaccharides. Structure of 5 sulphated oligosaccharides with polymerization degree 4-10 was established by NMR-spectroscopy. The fucoidan extracted from S. horneri is almost pure fucan. The main chain of the fucoidan is established to consist mostly of the repeating →3-α-l-Fucp(2SO)-1→4-α-l-Fucp(2,3SO)-1→ fragment, with insertions of →3-α-l-Fucp(2,4SO)-1→ fragment. Unsulphated side chains with the α-l-Fucp-1→2-α-l-Fucp-1→ structure connect to the main one at the C4 of monosaccharide residue.', 2017, 'Carbohydrate polymers', '', 1, NULL, NULL, 2),
(34, 'pubmed', '33208778', 'Conversion of levoglucosan into glucose by the coordination of four enzymes through oxidation, elimination, hydration, and reduction.', 'Levoglucosan (LG) is an anhydrosugar produced through glucan pyrolysis and is widely found in nature. We previously isolated an LG-utilizing thermophile, Bacillus smithii S-2701M, and suggested that this bacterium may have a metabolic pathway from LG to glucose, initiated by LG dehydrogenase (LGDH). Here, we completely elucidated the metabolic pathway of LG involving three novel enzymes in addition to LGDH. In the S-2701M genome, three genes expected to be involved in the LG metabolism were found in the vicinity of the LGDH gene locus. These four genes including LGDH gene (lgdA, lgdB1, lgdB2, and lgdC) were expressed in Escherichia coli and purified to obtain functional recombinant proteins. Thin layer chromatography analyses of the reactions with the combination of the four enzymes elucidated the following metabolic pathway: LgdA (LGDH) catalyzes 3-dehydrogenation of LG to produce 3-keto-LG, which undergoes β-elimination of 3-keto-LG by LgdB1, followed by hydration to produce 3-keto-D-glucose by LgdB2; next, LgdC reduces 3-keto-D-glucose to glucose. This sequential reaction mechanism resembles that proposed for an enzyme belonging to glycoside hydrolase family 4, and results in the observational hydrolysis of LG into glucose with coordination of the four enzymes.', 2020, 'Scientific reports', 'PMC7676230', 2, NULL, NULL, 6),
(35, 'pubmed', '33108275', 'Specificity in glycosylation of multiple flagellins by the modular and cell cycle regulated glycosyltransferase FlmG.', 'How specificity is programmed into post-translational modification of proteins by glycosylation is poorly understood, especially for O-linked glycosylation systems. Here we reconstitute and dissect the substrate specificity underpinning the cytoplasmic O-glycosylation pathway that modifies all six flagellins, five structural and one regulatory paralog, in , a monopolarly flagellated alpha-proteobacterium. We characterize the biosynthetic pathway for the sialic acid-like sugar pseudaminic acid and show its requirement for flagellation, flagellin modification and efficient export. The cognate NeuB enzyme that condenses phosphoenolpyruvate with a hexose into pseudaminic acid is functionally interchangeable with other pseudaminic acid synthases. The previously unknown and cell cycle-regulated FlmG protein, a defining member of a new class of cytoplasmic O-glycosyltransferases, is required and sufficient for flagellin modification. The substrate specificity of FlmG is conferred by its N-terminal flagellin-binding domain. FlmG accumulates before the FlaF secretion chaperone, potentially timing flagellin modification, export, and assembly during the cell division cycle.', 2020, 'eLife', 'PMC7591256', 2, NULL, NULL, 6),
(36, 'pubmed', '33151668', 'Analysis of Four Chitin-Active Lytic Polysaccharide Monooxygenases from  Reveals Functional Variation.', 'Lytic polysaccharide monooxygenases (LPMOs) are redox-active enzymes that cleave insoluble polysaccharides by an oxidative reaction. In the present study, we have characterized four recombinant putative chitin-active LPMOs from  (LPMO10B, -C, -D, and -F) and evaluated their potential in enhancing hydrolysis of α- and β-chitin by three families of 18 chitinases of , ChiA, -B, and -C. All four recombinant LPMO10s showed oxidative activity toward both α- and β-chitin but exhibited different abilities to promote the release of chitobiose from chitin by chitinases depending on both the chitinase and the chitin type. These effects were observed under conditions where the amount of LPMO in the reaction was not rate-limiting, showing that the observed functional differences relate to different abilities of the LPMOs to interact with and act on the substrate. These results show that four seemingly similar LPMOs carrying out the same reaction, cleavage of chitin by C1 oxidation, may have different roles in natural chitin conversion, which provides a rationale for the multiplicity of these enzymes within the same organism. The ability of the LPMOs to act on more natural substrates was demonstrated by showing that LPMO10B improved chitin solubilization in dried powdered shrimp shells.', 2020, 'Journal of agricultural and food chemistry', '', 4, NULL, NULL, 6),
(37, 'pubmed', '33936332', 'Preschool autism services: A tale of two Canadian provinces and the implications for policy.', 'For children with autism spectrum disorder (ASD), a lifelong neurodevelopmental condition, assessment and treatment services vary widely across Canada-potentially creating inequities. To highlight this, the Preschool Autism Treatment Impact study compared children\'s services and outcomes in New Brunswick (NB) and Nova Scotia (NS). Diagnostic practices, service delivery models, wait times, and treatment approaches differed, as did children\'s 1-year outcomes and costs for families and the public sector. Considering NB and NS strengths, we suggest that an optimal system would include: rapid access to high-quality diagnostic and intervention services; adherence to research-informed practice guidelines; interventions to enhance parents\' skills and self-efficacy; and measures to minimize financial burdens for families. Our results also suggest that provinces/territories must do more to ensure equitable access to effective services, including sharing and reporting on national comparative data. Canadian children with ASD deserve access to effective and consistent services, no matter where they live.', 2021, 'Paediatrics & child health', 'PMC8077201', 4, NULL, NULL, 6),
(38, 'pubmed', '33905962', 'RELAY, ramucirumab plus erlotinib versus placebo plus erlotinib in patients with untreated, EGFR-mutated, metastatic non-small cell lung cancer: Europe/United States subset analysis.', 'In EGFR mutation-positive NSCLC, dual EGFR/VEGFR inhibition compared to EGFR alone increases anti-tumor efficacy. The Phase III RELAY trial demonstrated superior PFS for ramucirumab plus erlotinib (RAM + ERL) over placebo plus erlotinib (PBO + ERL) (HR 0.591 [95% CI 0.461-0.760], p<0.0001). EGFR mutated NSCLC is less prevalent in Western versus Asian patients. This prespecified analysis evaluates efficacy and safety of RAM + ERL in EU and US patients enrolled in RELAY.Patients were randomized 1:1 to ERL + RAM (10 mg/kg IV) or PBO Q2W. Treatment continued until unacceptable toxicity or progressive disease. Patients were stratified by geographic region (East Asia vs \"other\" [EU/US and Canada (EU/US)]). Objectives included PFS, ORR, DoR, OS, PFS2, safety and biomarker analysis.EU/US subset included 113/449 (25.9%) patients (58 RAM + ERL, 55 PBO + ERL). RAM + ERL improved PFS (20.6 vs 10.9 months, HR 0.605 [95% CI: 0.362-1.010]). ORR and DCR were similar, but median DoR was longer with RAM + ERL (18.0 vs 10.1 months, HR 0.527 [95% CI: 0.296-0.939]). OS and PFS2 were immature at data cut-off (censoring rates 81.0-81.8% and 67.3-79.3%, respectively). Most commonly reported Grade ≥3 TEAE for RAM + ERL was hypertension (17 [29.8%]) and for PBO + ERL, dermatitis acneiform (5 [9.1%]).EU/US subset analysis showed improved efficacy outcomes for RAM + ERL and a safety profile consistent with the overall population. Ramucirumab is a safe and effective addition to standard-of-care EGFR-TKI for EGFR mutation-positive metastatic NSCLC.', 2021, 'Cancer treatment and research communications', '', 1, NULL, NULL, 6),
(39, 'pubmed', '33894335', 'Pemetrexed Plus Platinum With or Without Pembrolizumab in Patients With Previously Untreated Metastatic Nonsquamous NSCLC: Protocol-Specified Final Analysis From KEYNOTE-189.', 'In the phase 3 KEYNOTE-189 study (NCT02578680), pembrolizumab plus pemetrexed and platinum-based chemotherapy (pemetrexed-platinum) significantly improved overall survival (OS) and progression-free survival (PFS) in patients with previously untreated metastatic nonsquamous NSCLC versus placebo plus pemetrexed-platinum. We report updated efficacy outcomes from the protocol-specified final analysis, including outcomes in patients who crossed over to pembrolizumab from pemetrexed-platinum and in patients who completed 35 cycles (approximately 2 years) of pembrolizumab.Eligible patients were randomized 2:1 to pembrolizumab 200 mg (n=410) or placebo (n=206) every 3 weeks (for up to 35 cycles, approximately 2 years) plus 4 cycles of pemetrexed (500 mg/m) and investigators\' choice of cisplatin (75 mg/m) or carboplatin (AUC 5 mg/ml/min) every 3 weeks, followed by pemetrexed until progression. Patients assigned to placebo plus pemetrexed-platinum could crossover to pembrolizumab upon progression if eligibility criteria were met. The primary endpoints were OS and PFS.After median follow-up of 31.0 months, pembrolizumab plus pemetrexed-platinum continued to improve OS (hazard ratio [HR], 0.56; 95% CI, 0.46‒0.69), and PFS (HR, 0.49; 95% CI, 0.41‒0.59) over placebo plus pemetrexed-platinum regardless of PD-L1 expression. ORR (48.3% versus 19.9%) and time to second/subsequent tumor progression on next-line treatment (PFS2; HR, 0.50; 95% CI, 0.41‒0.61) were improved in patients who received pembrolizumab plus pemetrexed-platinum. 84 patients (40.8%) from the placebo plus pemetrexed-platinum group crossed over to pembrolizumab on-study. Grade 3‒5 adverse events occurred in 72.1% of patients receiving pembrolizumab plus pemetrexed-platinum and 66.8% of patients receiving placebo plus pemetrexed-platinum. 56 patients completed 35 cycles (approximately 2 years) of pembrolizumab; ORR was 85.7% and 53 (94.6%) were alive at data cutoff.Pembrolizumab plus pemetrexed-platinum continued to show improved efficacy outcomes compared with placebo plus pemetrexed-platinum, with manageable toxicity. These findings support first-line pembrolizumab plus pemetrexed-platinum in patients with previously untreated metastatic nonsquamous NSCLC.', 2021, 'Annals of oncology : official journal of the European Society for Medical Oncology', '', 1, NULL, NULL, 6),
(40, 'pubmed', '33775558', 'Patient-Reported Outcomes with Durvalumab With or Without Tremelimumab Versus Standard Chemotherapy as First-Line Treatment of Metastatic Non-Small-Cell Lung Cancer (MYSTIC).', 'The phase 3 MYSTIC study of durvalumab ± tremelimumab versus chemotherapy in metastatic non-small-cell lung cancer (NSCLC) patients with tumor cell (TC) programmed cell death ligand 1 (PD-L1) expression ≥ 25% did not meet its primary endpoints. We report patient-reported outcomes (PROs).Treatment-naïve patients were randomized (1:1:1) to durvalumab, durvalumab + tremelimumab, or chemotherapy. PROs were assessed in patients with PD-L1 TC ≥ 25% using EORTC Quality of Life Questionnaire (QLQ)-C30/LC13. Changes from baseline (12 months) for prespecified PRO endpoints of interest were analyzed by mixed model for repeated measures (MMRM) and time to deterioration (TTD) by stratified log-rank tests.There were no between-arm differences in baseline PROs (N = 488). Between-arm differences in MMRM-adjusted mean changes from baseline favored at least one of the durvalumab-containing arms versus chemotherapy (nominal P < .01) for C30 fatigue: durvalumab (-9.5; 99% confidence interval [CI], -17.0 to -2.0), durvalumab + tremelimumab (-11.7; 99% CI, -19.4 to -4.1); and for C30 appetite loss: durvalumab (-11.9; 99% CI, -21.1 to -2.7). TTD was longer with at least one of the durvalumab-containing arms versus chemotherapy (nominal P < .01) for global health status/quality of life: durvalumab (hazard ratio [HR] = 0.7; 95% CI, 0.5-1.0), durvalumab + tremelimumab (HR = 0.7; 95% CI, 0.5-1.0); and for physical functioning: durvalumab (HR = 0.6; 95% CI, 0.4-0.8), durvalumab + tremelimumab (HR = 0.6; 95% CI, 0.5-0.9) (both C30); as well as for the key symptoms of dyspnea: durvalumab (HR = 0.6; 95% CI, 0.5-0.9), durvalumab + tremelimumab (HR = 0.7; 95% CI, 0.5-1.0) (both LC13); fatigue: durvalumab + tremelimumab (HR = 0.6; 95% CI, 0.4-0.8); and appetite loss: durvalumab (HR = 0.5; 95% CI, 0.4-0.7), durvalumab + tremelimumab (HR = 0.7; 95% CI, 0.5-0.9) (both C30).Durvalumab ± tremelimumab versus chemotherapy reduced symptom burden and improved TTD of PROs, suggesting it had no detrimental effects on quality of life in metastatic NSCLC patients.', 2021, 'Clinical lung cancer', '', 3, NULL, NULL, 6),
(41, 'pubmed', '33737345', 'Severity of COVID-19 in patients with lung cancer: evidence and challenges.', 'Cancer patients are highly vulnerable to SARS-CoV-2 infections due to frequent contacts with the healthcare system, immunocompromised state from cancer or its therapies, supportive medications such as steroids and most importantly their advanced age and comorbidities. Patients with lung cancer have consistently been reported to suffer from an increased risk of death compared with other cancers. This is possibly due to the combination of specific pathophysiological aspects, including underlying pulmonary compromise due to smoking history and the increased specific pressures on respiratory healthcare services caused by the related pandemic. Rationally and safely treating patients with lung cancer during the pandemic has become a continuous challenge over the last year. Deciding whether to offer, modify, postpone or even cancel treatments for this particular patient\'s population has become the crucial recurrent dilemma for lung cancer professionals. Chemotherapy, immunotherapy and targeted agents represent distinct risks factors in the context of COVID-19 that should be balanced with the short-term and long-term consequences of delaying cancer care. Despite the rapid and persistent trend of the pandemic, declared by WHO on March 11, 2020, and still ongoing at the time of writing (January 2021), various efforts were made by oncologists worldwide to understand the impact of COVID-19 on patients with cancer. Adapted recommendations of our evidence-based practice guidelines have been developed for all stakeholders. Different small and large-scale registries, such as the COVID-19 and Cancer Consortium (CCC19) and Thoracic Cancers International COVID-19 Collaboration quickly collected data, supporting cancer care decisions under the challenging circumstance created by the COVID-19 pandemic. Several recommendations were developed as guidance for prioritizing the various aspects of lung cancer care in order to mitigate the adverse effects of the COVID-19 healthcare crisis, potentially reducing the morbidity and mortality of our patients from COVID-19 and from cancer. These recommendations helped inform decisions about treatment of established disease, continuation of clinical research and lung cancer screening. In this review, we summarize available evidence regarding the direct and indirect impact of the COVID-19 pandemic on lung cancer care and patients.', 2021, 'Journal for immunotherapy of cancer', 'PMC7978268', 2, NULL, NULL, 3),
(42, 'pubmed', '33730652', 'Safety of pemetrexed plus platinum in combination with pembrolizumab for metastatic nonsquamous non-small cell lung cancer: A post hoc analysis of KEYNOTE-189.', 'This post hoc analysis assessed the safety of pemetrexed and platinum in combination with pembrolizumab, including time-to-onset and time-to-resolution of all-cause any-grade and grade ≥3 adverse events (AEs) and renal AEs.Patient-level data from KEYNOTE-189 were analyzed in the all-subjects-as-treated population (pembrolizumab arm, n = 405; placebo arm, n = 202), and among patients who received ≥5 cycles of pemetrexed (pemetrexed/pembrolizumab/platinum arm, n = 310; pemetrexed/placebo/platinum arm, n = 135). All-cause AEs were selected based on ≥2 % incidence from previously reported KEYNOTE-189 data and included neutropenia, febrile neutropenia, anemia, thrombocytopenia, asthenia, fatigue, dyspnea, diarrhea, nausea, vomiting, pneumonitis, and renal events. Descriptive statistics summarized all-cause AEs. Medians and interquartile ranges were used to examine time-to-onset and time-to-resolution. The data cutoff was November 8, 2017.In both treatment arms, most non-hematologic (nausea, vomiting, diarrhea, and asthenia), and hematologic (febrile neutropenia, thrombocytopenia, and neutropenia) grade ≥3 AEs with ≥2 % incidence had a median time-to-onset within the first 4 cycles, and a median time-to-resolution of within 2 weeks from onset. A small number of AEs had longer median time-to-onset (pneumonitis and fatigue) and median time-to-resolution (pneumonitis, fatigue, acute kidney injury, and anemia). Among patients who received ≥5 cycles of pemetrexed, the incidence of any-grade renal toxicity in the pemetrexed/pembrolizumab/platinum arm was 2.3 % in Cycles 1-4, 4.8 % in Cycles 5-8, 2.6 % in Cycles 9-12, and 2.5 % in Cycles ≥13; and, in the pemetrexed/placebo/platinum arm, 0.7 % in Cycles 1-4, 1.5 % in Cycles 5-8, 1.3 % in Cycles 9-12, and 2.0 % in Cycles ≥13.Pemetrexed/pembrolizumab/platinum has manageable toxicity with longer duration of treatment. While the incidence of renal toxicity was slightly higher in the pembrolizumab combination as compared to pemetrexed, the incidence did not increase in later treatment cycles. These results support the safe use of the KEYNOTE-189 regimen in clinical practice.NCT02578680 (clinicaltrials.gov).', 2021, 'Lung cancer (Amsterdam, Netherlands)', '', 3, NULL, NULL, 6),
(43, 'pubmed', '33638808', 'Targeted Therapy Approaches for MET Abnormalities in Non-Small Cell Lung Cancer.', 'The tyrosine kinase receptor mesenchymal epithelial transition (MET) is a proto-oncogene that, through the activation of the MET-hepatocyte growth factor (HGF) pathway, encodes a variety of biological processes, including cell development, proliferation, invasion, and migration. Abnormal activation of the MET pathway, occurring through MET protein overexpression, and gene amplification or mutation, can contribute to oncogenesis and has been implicated in non-small cell lung cancer (NSCLC). Though it is associated with poor clinical outcome in NSCLCs, MET overexpression and its role as a therapeutic target remains somewhat elusive due to discrepancies in its occurrence. Unlike MET overexpression, MET amplification has demonstrated a stronger potential as a biomarker for therapeutic treatment, with clinical data indicating a compelling connection between a high MET gene copy number and a high response rate to targeted therapies. However, MET exon 14 skipping mutations, occurring in 3%-4 % of lung adenocarcinomas, are of particular interest, as tumors harboring these mutations have shown a significant response to MET inhibitors. Following the discovery of MET as a potential therapeutic target, extensive clinical studies have proposed three approaches to targeting MET: (1) MET tyrosine kinase inhibitors (TKIs), including crizotinib, capmatinib, tepotinib, savolinitib, and cabozantinib; (2) MET or HGF monoclonal antibodies, including emibetuzumab and ficlatuzumab; and (3) MET or HGF antibody drug conjugates, including telisotuzumab. Herein, we discuss the relevant clinical trials, particularly focusing on the efficacy as well as the safety and tolerability of the treatment options, in the promising field of targeting MET in NSCLC.', 2021, 'Drugs', '', 1, NULL, NULL, 6),
(44, 'pubmed', '33613137', 'A Conceptual Framework for Integrated Community Care.', 'The various health and social care services provided in a given local area (i.e., place-based) must not only deliver primary care in proximity to the population, but act upstream on the social determinants of health. This type of care, when provided in a holistic and integrated manner, aims to improve the physical and mental health-but also the well-being and social capital-of individuals, families, groups and communities. This type of approach is known as Integrated Community Care (ICC).This article was developed from a non-systematic review of scientific and grey literature followed by a qualitative analysis and researcher reflections on ICC.The article presents the core concepts of ICC, namely temporality, local area, health care, social care, proximity and integration. These concepts are unpacked and a conceptual diagram is set forth to put the dynamic links between the concepts into perspective.The purpose of the article is to provide a conceptual clarification of ICC. Three examples of practise (from Switzerland, Quebec [Canada] and Italy) are used as illustrations to provide a better understanding of ICC and to open up horizons.', 2021, 'International journal of integrated care', '', 4, NULL, NULL, 6),
(45, 'pubmed', '33430321', 'In Silico Identification of Potential Druggable Binding Sites on CIN85 SH3 Domain.', 'Protein-protein interactions (PPIs) play a pivotal role in the regulation of many physiological processes. The dysfunction of some PPIs interactions led to the alteration of different biological pathways causing various diseases including cancer. In this context, the inhibition of PPIs represents an attractive strategy for the design of new antitumoral agents. In recent years, computational approaches were successfully used to study the interactions between proteins, providing useful hints for the design of small molecules able to modulate PPIs. Targeting PPIs presents several challenges mainly due to the large and flat binding surface that lack the typical binding pockets of traditional drug targets. Despite these hurdles, substantial progress has been made in the last decade resulting in the identification of PPI modulators where some of them even found clinical use. This study focuses on MUC1-CIN85 PPI which is involved in the migration and invasion of cancer cells. Particularly, we investigated the presence of druggable binding sites on the CIN85 surface which provided new insights for the structure-based design of novel MUC1-CIN85 PPI inhibitors as anti-metastatic agents.', 2021, 'International journal of molecular sciences', 'PMC7825788', 4, NULL, NULL, 6),
(46, 'pubmed', '33368222', 'Assessment of Autism Symptoms From 6 to 18 Months of Age Using the Autism Observation Scale for Infants in a Prospective High-Risk Cohort.', 'The objectives were to characterize behavioral signs of autism spectrum disorder (ASD) in younger siblings of diagnosed children (high-risk; HR) and examine classification features of the Autism Observation Scale for Infants (AOSI). Participants (501 HR and 180 low-risk [LR]) were assessed between 6 and 18 months using the AOSI and at age 3 for ASD diagnoses. Total AOSI scores differentiated HR infants later diagnosed with ASD starting at 12 months. ROC analyses identified 12- and 18-month cutoff scores associated with 0.52 sensitivity and 0.74 specificity and 0.73 sensitivity and 0.65 specificity, respectively. Although classification accuracy does not support use as a standalone screen, the AOSI identifies features associated with ASD starting at 6 months and differentiates HR infants with ASD by 12 months.', 2020, 'Child development', '', 1, NULL, NULL, 1),
(47, 'pubmed', '33185664', 'Diversification of Fungal Chitinases and Their Functional Differentiation in Histoplasma capsulatum.', 'Chitinases enzymatically hydrolyze chitin, a highly abundant and utilized polymer of N-acetyl-glucosamine. Fungi are a rich source of chitinases; however, the phylogenetic and functional diversity of fungal chitinases are not well understood. We surveyed fungal chitinases from 373 publicly available genomes, characterized domain architecture, and conducted phylogenetic analyses of the glycoside hydrolase (GH18) domain. This large-scale analysis does not support the previous division of fungal chitinases into three major clades (A, B, C) as chitinases previously assigned to the \"C\" clade are not resolved as distinct from the \"A\" clade. Fungal chitinase diversity was partly shaped by horizontal gene transfer, and at least one clade of bacterial origin occurs among chitinases previously assigned to the \"B\" clade. Furthermore, chitin-binding domains (including the LysM domain) do not define specific clades, but instead are found more broadly across clades of chitinases. To gain insight into biological function diversity, we characterized all eight chitinases (Cts) from the thermally dimorphic fungus, Histoplasma capsulatum: six A clade, one B clade, and one formerly classified C clade chitinases. Expression analyses showed variable induction of chitinase genes in the presence of chitin but preferential expression of CTS3 in the mycelial stage. Activity assays demonstrated that Cts1 (B-I), Cts2 (A-V), Cts3 (A-V), Cts4 (A-V) have endochitinase activities with varying degrees of chitobiosidase function. Cts6 (C-I) has activity consistent with N-acetyl-glucosaminidase exochitinase function and Cts8 (A-II) has chitobiase activity. These results suggest chitinase activity is variable even within subclades and that predictions of functionality require more sophisticated models.', 2021, 'Molecular biology and evolution', 'PMC8042737', 1, NULL, NULL, 6),
(48, 'pubmed', '31451726', 'Proteomic enzyme analysis of the marine fungus Paradendryphiella salina reveals alginate lyase as a minimal adaptation strategy for brown algae degradation.', 'We set out to investigate the genetic adaptations of the marine fungus Paradendryphiella salina CBS112865 for degradation of brown macroalgae. We performed whole genome and transcriptome sequencing and shotgun proteomic analysis of the secretome of P. salina grown on three species of brown algae and under carbon limitation. Genome comparison with closely related terrestrial fungi revealed that P. salina had a similar but reduced CAZyme profile relative to the terrestrial fungi except for the presence of three putative alginate lyases from Polysaccharide Lyase (PL) family 7 and a putative PL8 with similarity to ascomycete chondroitin AC lyases. Phylogenetic and homology analyses place the PL7 sequences amongst mannuronic acid specific PL7 proteins from marine bacteria. Recombinant expression, purification and characterization of one of the PL7 genes confirmed the specificity. Proteomic analysis of the P. salina secretome when growing on brown algae, revealed the PL7 and PL8 enzymes abundantly secreted together with enzymes necessary for degradation of laminarin, cellulose, lipids and peptides. Our findings indicate that the basic CAZyme repertoire of saprobic and plant pathogenic ascomycetes, with the addition of PL7 alginate lyases, provide P. salina with sufficient enzymatic capabilities to degrade several types of brown algae polysaccharides.', 2019, 'Scientific reports', 'PMC6710412', 2, NULL, NULL, 6);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(49, 'pubmed', '33281771', 'α-Galactosidase and Sucrose-Kinase Relationships in a Bi-functional AgaSK Enzyme Produced by the Human Gut Symbiont  E1.', 'Plant α-galactosides belonging to the raffinose family oligosaccharides (RFOs) and considered as prebiotics, are commonly degraded by α-galactosidases produced by the human gut microbiome. In this environment, the  E1 symbiont-well-known for various benefit-is able to produce an original AgaSK bifunctional enzyme. This enzyme contains an hydrolytic α-galactosidase domain linked to an ATP dependent extra-domain, specifically involved in the α-galactoside hydrolysis and the phosphorylation of the glucose, respectively. However, the multi-modular relationships between both catalytic domains remained hitherto unexplored and has been, consequently, herein investigated. Biochemical characterization of heterologously expressed enzymes either in full-form or in separated domains revealed similar kinetic parameters. These results were supported by molecular modeling studies performed on the whole enzyme in complex with different RFOs. Further enzymatic analysis associated with kinetic degradation of various substrates followed by high pressure anionic exchange chromatography revealed that catalytic efficiency decreased as the number of D-galactosyl moieties branched onto the oligosaccharide increased, suggesting a preference of AgaSK for RFO\'s short chains. A wide prevalence and abundance study on a human metagenomic library showed a high prevalence of the AgaSK encoding gene whatever the health status of the individuals. Finally, phylogeny and synteny studies suggested a limited spread by horizontal transfer of the clusters\' containing AgaSK to only few species of Firmicutes, highlighting the importance of these undispersed tandem activities in the human gut microbiome.', 2020, 'Frontiers in microbiology', '', 1, NULL, NULL, 6),
(50, 'pubmed', '33004077', 'Investigating host-microbiome interactions by droplet based microfluidics.', 'Despite the importance of the mucosal interface between microbiota and the host in gut homeostasis, little is known about the mechanisms of bacterial gut colonization, involving foraging for glycans produced by epithelial cells. The slow pace of progress toward understanding the underlying molecular mechanisms is largely due to the lack of efficient discovery tools, especially those targeting the uncultured fraction of the microbiota.Here, we introduce an ultra-high-throughput metagenomic approach based on droplet microfluidics, to screen fosmid libraries. Thousands of bacterial genomes can be covered in 1 h of work, with less than ten micrograms of substrate. Applied to the screening of the mucosal microbiota for β-N-acetylgalactosaminidase activity, this approach allowed the identification of pathways involved in the degradation of human gangliosides and milk oligosaccharides, the structural homologs of intestinal mucin glycans. These pathways, whose prevalence is associated with inflammatory bowel diseases, could be the result of horizontal gene transfers with Bacteroides species. Such pathways represent novel targets to study the microbiota-host interactions in the context of inflammatory bowel diseases, in which the integrity of the mucosal barrier is impaired.By compartmentalizing experiments inside microfluidic droplets, this method speeds up and miniaturizes by several orders of magnitude the screening process compared to conventional approaches, to capture entire metabolic pathways from metagenomic libraries. The method is compatible with all types of (meta)genomic libraries, and employs a commercially available flow cytometer instead of a custom-made sorting system to detect intracellular or extracellular enzyme activities. This versatile and generic workflow will accelerate experimental exploration campaigns in functional metagenomics and holobiomics studies, to further decipher host-microbiota relationships. Video Abstract.', 2020, 'Microbiome', 'PMC7531118', 2, NULL, NULL, 6),
(52, 'pubmed', '33683866', 'Recapitulating the Binding Affinity of Nrf2 for KEAP1 in a Cyclic Heptapeptide, Guided by NMR, X-ray Crystallography, and Machine Learning.', 'Macrocycles, including macrocyclic peptides, have shown promise for targeting challenging protein-protein interactions (PPIs). One PPI of high interest is between Kelch-like ECH-Associated Protein-1 (KEAP1) and Nuclear Factor (Erythroid-derived 2)-like 2 (Nrf2). Guided by X-ray crystallography, NMR, modeling, and machine learning, we show that the full 20 nM binding affinity of Nrf2 for KEAP1 can be recapitulated in a cyclic 7-mer peptide, c[()-β-homoAla-DPETGE]. This compound was identified from the Nrf2-derived linear peptide GDEETGE ( = 4.3 μM) solely by optimizing the conformation of the cyclic compound, without changing any KEAP1 interacting residue. X-ray crystal structures were determined for each linear and cyclic peptide variant bound to KEAP1. Despite large variations in affinity, no obvious differences in the conformation of the peptide binding residues or in the interactions they made with KEAP1 were observed. However, analysis of the X-ray structures by machine learning showed that locations of strain in the bound ligand could be identified through patterns of subangstrom distortions from the geometry observed for unstrained linear peptides. We show that optimizing the cyclic peptide affinity was driven partly through conformational preorganization associated with a proline substitution at position 78 and with the geometry of the noninteracting residue Asp77 and partly by decreasing strain in the ETGE motif itself. This approach may have utility in dissecting the trade-off between conformational preorganization and strain in other ligand-receptor systems. We also identify a pair of conserved hydrophobic residues flanking the core DxETGE motif which play a conformational role in facilitating the high-affinity binding of Nrf2 to KEAP1.', 2021, 'Journal of the American Chemical Society', '', 1, NULL, NULL, 6),
(53, 'pubmed', '33617586', 'Preliminary validity and reliability evidence of the Brief Antisocial Behavior Scale (B-ABS) in young adults from four countries.', 'The present research built on the Self-Reported Delinquency interview and the Antisocial Behavior Scale to develop an updated brief instrument to measure antisocial behavior. College students (n = 3188, 67.75% women) from the USA, Argentina, the Netherlands and Spain completed an online survey. Analyses that combined approaches from the Classical Test Theory and Item Response Theory were conducted to select the items for the brief version. Findings suggested that a 13-item Brief Antisocial Behavior Scale (B-ABS) fulfilled the high-quality criteria: salient factor loadings, adequate discrimination, variability in response endorsement, adequate fit based on infit/outfit values, nondifferent item functioning across the four participating countries, and Cronbach\'s alpha and ordinal omega coefficients higher than .70. The B-ABS scores generally significantly correlated with personality scores, mental health and marijuana outcomes, showing criterion-related validity evidence. Our overall findings suggest that B-ABS adequately assesses antisocial behavior in young adults from different countries/cultures.', 2021, 'PloS one', 'PMC7899364', 1, NULL, NULL, 6),
(57, 'pubmed', '33252248', 'The Role of Violent Video Game Exposure, Personality, and Deviant Peers in Aggressive Behaviors Among Adolescents: A Two-Wave Longitudinal Study.', 'The impact of violent video game exposure (VVGE) on aggressive behaviors has been extensively explored, but still remains controversial. Although some studies have shown slight detrimental short-term effects of VVGE, other studies have failed to find any consequence. In addition, the existence of long-lasting effects on aggressiveness, or their impact on adolescents, are still not well established. One limitation of most of these studies is that they do not control for other important risk variables for aggressive behaviors, such as personality and deviant peers, nor have they investigated the possible moderation role of these risk factors in the link between VVGE and aggression. Therefore, the main aim was to examine the additive and interactive role of VVGE, personality, and deviant peers in adolescent aggressive behaviors cross-sectionally and longitudinally. Many regression analyses and a cross-lagged autoregressive model were carried out. At both waves, aggressive behavior was predicted by having deviant peers and specific personality traits, especially low agreeableness. VVGE also presented a slight but significant effect at both waves, but it became nonsignificant when controlling for other variables. No long-term effects on the relation between VVGE and aggressive behaviors were found. Some moderation effects were consistently found at both waves: when participants reported having more deviant peers, the effects of VVGE and low agreeableness on aggressive behaviors significantly increased. These findings suggest that multiple biopsychosocial variables and their complex interplay need to be examined to gain a better understanding of the origin and expression of aggressive behavior.', 2021, 'Cyberpsychology, behavior and social networking', '', 1, NULL, NULL, 6),
(58, 'pubmed', '33106654', 'FoxO maintains a genuine muscle stem-cell quiescent state until geriatric age.', 'Tissue regeneration declines with ageing but little is known about whether this arises from changes in stem-cell heterogeneity. Here, in homeostatic skeletal muscle, we identify two quiescent stem-cell states distinguished by relative CD34 expression: CD34, with stemness properties (genuine state), and CD34, committed to myogenic differentiation (primed state). The genuine-quiescent state is unexpectedly preserved into later life, succumbing only in extreme old age due to the acquisition of primed-state traits. Niche-derived IGF1-dependent Akt activation debilitates the genuine stem-cell state by imposing primed-state features via FoxO inhibition. Interventions to neutralize Akt and promote FoxO activity drive a primed-to-genuine state conversion, whereas FoxO inactivation deteriorates the genuine state at a young age, causing regenerative failure of muscle, as occurs in geriatric mice. These findings reveal transcriptional determinants of stem-cell heterogeneity that resist ageing more than previously anticipated and are only lost in extreme old age, with implications for the repair of geriatric muscle.', 2020, 'Nature cell biology', '', 1, NULL, NULL, 6);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(70, 'pubmed', '31539500', 'Interspecies Competition Impacts Targeted Manipulation of Human Gut Bacteria by Fiber-Derived Glycans.', 'Development of microbiota-directed foods (MDFs) that selectively increase the abundance of beneficial human gut microbes, and their expressed functions, requires knowledge of both the bioactive components of MDFs and the mechanisms underlying microbe-microbe interactions. Here, gnotobiotic mice were colonized with a defined consortium of human-gut-derived bacterial strains and fed different combinations of 34 food-grade fibers added to a representative low-fiber diet consumed in the United States. Bioactive carbohydrates in fiber preparations targeting particular Bacteroides species were identified using community-wide quantitative proteomic analyses of bacterial gene expression coupled with forward genetic screens. Deliberate manipulation of community membership combined with administration of retrievable artificial food particles, consisting of paramagnetic microscopic beads coated with dietary polysaccharides, disclosed the contributions of targeted species to fiber degradation. Our approach, including the use of bead-based biosensors, defines nutrient-harvesting strategies that underlie, as well as alleviate, competition between Bacteroides and control the selectivity of MDF components.', 2019, 'Cell', 'PMC6760872', 1, '<h1 class=\"content-title\">Interspecies competition impacts targeted manipulation of human gut bacteria by fiber-derived glycans</h1><br><span class=\"contrib-group fm-author\"><a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Patnode%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701600144368\" co-class=\"co-affbox\">Michael L. Patnode</a>,<sup>1,</sup><sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Beller%20ZW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701607682080\" co-class=\"co-affbox\">Zachary W. Beller</a>,<sup>1,</sup><sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Han%20ND%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701607680032\" co-class=\"co-affbox\">Nathan D. Han</a>,<sup>1,</sup><sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Cheng%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701607677424\" co-class=\"co-affbox\">Jiye Cheng</a>,<sup>1,</sup><sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Peters%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701599012096\" co-class=\"co-affbox\">Samantha L. Peters</a>,<sup>7</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Terrapon%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701599010112\" co-class=\"co-affbox\">Nicolas Terrapon</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Henrissat%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701599563312\" co-class=\"co-affbox\">Bernard Henrissat</a>,<sup>3,</sup><sup>4</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Gall%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701599560592\" co-class=\"co-affbox\">Sophie Le Gall</a>,<sup>5</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Saulnier%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701599558608\" co-class=\"co-affbox\">Luc Saulnier</a>,<sup>5</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Hayashi%20DK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701607253552\" co-class=\"co-affbox\">David K. Hayashi</a>,<sup>6</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Meynier%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701607251568\" co-class=\"co-affbox\">Alexandra Meynier</a>,<sup>6</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Vinoy%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701607249584\" co-class=\"co-affbox\">Sophie Vinoy</a>,<sup>6</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Giannone%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701603847776\" co-class=\"co-affbox\">Richard J. Giannone</a>,<sup>7</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Hettich%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701603845792\" co-class=\"co-affbox\">Robert L. Hettich</a>,<sup>7</sup> and  <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Gordon%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701603843808\" co-class=\"co-affbox\">Jeffrey I. Gordon</a><sup>*,</sup><sup>1,</sup><sup>2</sup></span><span style=\"display:none\" class=\"contrib-group aff-tip\"><span id=\"_co_idm139701600144368\"><h3 class=\"no_margin\">Michael L. Patnode</h3><p><sup>1</sup>Edison Family Center for Genome Sciences and Systems Biology, Washington University School of Medicine St. Louis, MO 63110, USA</p><p><sup>2</sup>Center for Gut Microbiome and Nutrition Research, Washington University School of Medicine St. Louis, MO 63110, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Patnode%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Michael L. Patnode</a></span></span><span id=\"_co_idm139701607682080\"><h3 class=\"no_margin\">Zachary W. Beller</h3><p><sup>1</sup>Edison Family Center for Genome Sciences and Systems Biology, Washington University School of Medicine St. Louis, MO 63110, USA</p><p><sup>2</sup>Center for Gut Microbiome and Nutrition Research, Washington University School of Medicine St. Louis, MO 63110, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Beller%20ZW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Zachary W. Beller</a></span></span><span id=\"_co_idm139701607680032\"><h3 class=\"no_margin\">Nathan D. Han</h3><p><sup>1</sup>Edison Family Center for Genome Sciences and Systems Biology, Washington University School of Medicine St. Louis, MO 63110, USA</p><p><sup>2</sup>Center for Gut Microbiome and Nutrition Research, Washington University School of Medicine St. Louis, MO 63110, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Han%20ND%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Nathan D. Han</a></span></span><span id=\"_co_idm139701607677424\"><h3 class=\"no_margin\">Jiye Cheng</h3><p><sup>1</sup>Edison Family Center for Genome Sciences and Systems Biology, Washington University School of Medicine St. Louis, MO 63110, USA</p><p><sup>2</sup>Center for Gut Microbiome and Nutrition Research, Washington University School of Medicine St. Louis, MO 63110, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Cheng%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Jiye Cheng</a></span></span><span id=\"_co_idm139701599012096\"><h3 class=\"no_margin\">Samantha L. Peters</h3><p><sup>7</sup>Chemical Sciences Division, Oak Ridge National Laboratory, Oak Ridge, TN 37830, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Peters%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Samantha L. Peters</a></span></span><span id=\"_co_idm139701599010112\"><h3 class=\"no_margin\">Nicolas Terrapon</h3><p><sup>3</sup>Architecture et Fonction des Macromol&#x000e9;cules Biologiques, Centre National de la Recherche Scientifique and Aix-Marseille Universit&#x000e9; 13288 Marseille cedex 9, France</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Terrapon%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Nicolas Terrapon</a></span></span><span id=\"_co_idm139701599563312\"><h3 class=\"no_margin\">Bernard Henrissat</h3><p><sup>3</sup>Architecture et Fonction des Macromol&#x000e9;cules Biologiques, Centre National de la Recherche Scientifique and Aix-Marseille Universit&#x000e9; 13288 Marseille cedex 9, France</p><p><sup>4</sup>Department of Biological Sciences, King Abdulaziz University, Jeddah, Saudi Arabia</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Henrissat%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Bernard Henrissat</a></span></span><span id=\"_co_idm139701599560592\"><h3 class=\"no_margin\">Sophie Le Gall</h3><p><sup>5</sup>Institut National de la Recherche Agronomique, UR 1268 BIA (Biopolym&#x000e8;res, Interactions, Assemblages), 44316 Nantes cedex, France</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Gall%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Sophie Le Gall</a></span></span><span id=\"_co_idm139701599558608\"><h3 class=\"no_margin\">Luc Saulnier</h3><p><sup>5</sup>Institut National de la Recherche Agronomique, UR 1268 BIA (Biopolym&#x000e8;res, Interactions, Assemblages), 44316 Nantes cedex, France</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Saulnier%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Luc Saulnier</a></span></span><span id=\"_co_idm139701607253552\"><h3 class=\"no_margin\">David K. Hayashi</h3><p><sup>6</sup>Mondelez International, Deerfield, IL 60015, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Hayashi%20DK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">David K. Hayashi</a></span></span><span id=\"_co_idm139701607251568\"><h3 class=\"no_margin\">Alexandra Meynier</h3><p><sup>6</sup>Mondelez International, Deerfield, IL 60015, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Meynier%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Alexandra Meynier</a></span></span><span id=\"_co_idm139701607249584\"><h3 class=\"no_margin\">Sophie Vinoy</h3><p><sup>6</sup>Mondelez International, Deerfield, IL 60015, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Vinoy%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Sophie Vinoy</a></span></span><span id=\"_co_idm139701603847776\"><h3 class=\"no_margin\">Richard J. Giannone</h3><p><sup>7</sup>Chemical Sciences Division, Oak Ridge National Laboratory, Oak Ridge, TN 37830, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Giannone%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Richard J. Giannone</a></span></span><span id=\"_co_idm139701603845792\"><h3 class=\"no_margin\">Robert L. Hettich</h3><p><sup>7</sup>Chemical Sciences Division, Oak Ridge National Laboratory, Oak Ridge, TN 37830, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Hettich%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Robert L. Hettich</a></span></span><span id=\"_co_idm139701603843808\"><h3 class=\"no_margin\">Jeffrey I. Gordon</h3><p><sup>1</sup>Edison Family Center for Genome Sciences and Systems Biology, Washington University School of Medicine St. Louis, MO 63110, USA</p><p><sup>2</sup>Center for Gut Microbiome and Nutrition Research, Washington University School of Medicine St. Louis, MO 63110, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Gordon%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Jeffrey I. Gordon</a></span></span></span></span><span class=\"fm-panel half_rhythm\"><span class=\"togglers<br><span id=\"idm139701596865568\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139701596865568title\">Footnotes</h2><!--back/fn-group--><span class=\"fm-sec half_rhythm small\"><p class=\"fn sec\" id=\"FN3\"><p id=\"P85\" class=\"p p-first\">Declaration of interests:</p><p id=\"P86\" class=\"p p-last\">J.I.G. is a co-founder of Matatu, Inc., a company characterizing the role of diet-by-microbiota interactions in animal health. Elements of this report are the subject of patent applications that are currently being submitted.</p></p><p class=\"fn sec\" id=\"FN4\"><p id=\"P87\" class=\"p p-first-last\"><strong>Publisher\'s Disclaimer: </strong>This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.</p></p></span></span><br><div class=\"ref-list-sec sec\" id=\"reference-list\"><ul class=\"back-ref-list\" style=\"list-style-type:decimal;\"><li id=\"R1\"><span class=\"mixed-citation\">Adey A, Morrison HG, Asan Xun, X., Kitzman JO, Turner EH, Stackhouse B, MacKenzie AP, Caruccio NC, Zhang X, et al. (2010). <span class=\"ref-title\">Rapid, low-input, low-bias construction of shotgun fragment libraries by high-density in vitro transposition</span>. <span class=\"ref-journal\">Genome Biol</span>\n<span class=\"ref-vol\">11</span>, R119. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3046479/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21143862\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Rapid,+low-input,+low-bias+construction+of+shotgun+fragment+libraries+by+high-density+in+vitro+transposition&amp;author=A+Adey&amp;author=HG+Morrison&amp;author=Xun,+X.+Asan&amp;author=JO+Kitzman&amp;author=EH+Turner&amp;volume=11&amp;publication_year=2010&amp;pages=R119&amp;pmid=21143862&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R2\"><span class=\"mixed-citation\">Atmodjo MA, Hao Z, and Mohnen D (2013). <span class=\"ref-title\">Evolving views of pectin biosynthesis</span>. <span class=\"ref-journal\">Annu Rev Plant Biol</span>\n<span class=\"ref-vol\">64</span>, 747&#x02013;779. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23451775\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Plant+Biol&amp;title=Evolving+views+of+pectin+biosynthesis&amp;author=MA+Atmodjo&amp;author=Z+Hao&amp;author=D+Mohnen&amp;volume=64&amp;publication_year=2013&amp;pages=747-779&amp;pmid=23451775&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R3\"><span class=\"mixed-citation\">Bokulich NA, Subramanian S, Faith JJ, Gevers D, Gordon JI, Knight R, Mills DA, and Caporaso JG (2013). <span class=\"ref-title\">Quality-filtering vastly improves diversity estimates from Illumina amplicon sequencing</span>. <span class=\"ref-journal\">Nature Methods</span>\n<span class=\"ref-vol\">10</span>, 57&#x02013;59. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3531572/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23202435\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature+Methods&amp;title=Quality-filtering+vastly+improves+diversity+estimates+from+Illumina+amplicon+sequencing&amp;author=NA+Bokulich&amp;author=S+Subramanian&amp;author=JJ+Faith&amp;author=D+Gevers&amp;author=JI+Gordon&amp;volume=10&amp;publication_year=2013&amp;pages=57-59&amp;pmid=23202435&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R4\"><span class=\"mixed-citation\">Buffetto F, Cornuault V, Rydahl MG, Ropartz D, Alvarado C, Echasserieau V, Le Gall S, Bouchet B, Tranquet O, Verhertbruggen Y, et al. (2015). <span class=\"ref-title\">The Deconstruction of Pectic Rhamnogalacturonan I Unmasks the Occurrence of a Novel Arabinogalactan Oligosaccharide Epitope</span>. <span class=\"ref-journal\">Plant Cell Physiol</span>\n<span class=\"ref-vol\">56</span>, 2181&#x02013;2196. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26384432\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Plant+Cell+Physiol&amp;title=The+Deconstruction+of+Pectic+Rhamnogalacturonan+I+Unmasks+the+Occurrence+of+a+Novel+Arabinogalactan+Oligosaccharide+Epitope&amp;author=F+Buffetto&amp;author=V+Cornuault&amp;author=MG+Rydahl&amp;author=D+Ropartz&amp;author=C+Alvarado&amp;volume=56&amp;publication_year=2015&amp;pages=2181-2196&amp;pmid=26384432&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R5\"><span class=\"mixed-citation\">Clarkson SM, Giannone RJ, Kridelbaugh DM, Elkins JG, Guss AM, and Michener JK (2017). <span class=\"ref-title\">Construction and Optimization of a Heterologous Pathway for Protocatechuate Catabolism in Escherichia coli Enables Bioconversion of Model Aromatic Compounds</span>. <span class=\"ref-journal\">Applied and Environmental Microbiol</span>\n<span class=\"ref-vol\">83</span>, e01313&#x02013;01317. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5583500/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28733280\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Applied+and+Environmental+Microbiol&amp;title=Construction+and+Optimization+of+a+Heterologous+Pathway+for+Protocatechuate+Catabolism+in+Escherichia+coli+Enables+Bioconversion+of+Model+Aromatic+Compounds&amp;author=SM+Clarkson&amp;author=RJ+Giannone&amp;author=DM+Kridelbaugh&amp;author=JG+Elkins&amp;author=AM+Guss&amp;volume=83&amp;publication_year=2017&amp;pages=e01313-01317&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R6\"><span class=\"mixed-citation\">Desai MS, Seekatz AM, Koropatkin NM, Kamada N, Hickey CA, Wolter M, Pudlo NA, Kitamoto S, Terrapon N, Muller A, et al. (2016). <span class=\"ref-title\">A Dietary Fiber-Deprived Gut Microbiota Degrades the Colonic Mucus Barrier and Enhances Pathogen Susceptibility</span>. <span class=\"ref-journal\">Cell</span>\n<span class=\"ref-vol\">167</span>, 1339&#x02013;1353 e1321. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5131798/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27863247\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A+Dietary+Fiber-Deprived+Gut+Microbiota+Degrades+the+Colonic+Mucus+Barrier+and+Enhances+Pathogen+Susceptibility&amp;author=MS+Desai&amp;author=AM+Seekatz&amp;author=NM+Koropatkin&amp;author=N+Kamada&amp;author=CA+Hickey&amp;volume=167&amp;publication_year=2016&amp;pages=1339-1353+e1321&amp;pmid=27863247&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R7\"><span class=\"mixed-citation\">Edgar RC (2010). <span class=\"ref-title\">Search and clustering orders of magnitude faster than BLAST</span>. <span class=\"ref-journal\">Bioinformatics</span>\n<span class=\"ref-vol\">26</span>, 2460&#x02013;2461. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20709691\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Search+and+clustering+orders+of+magnitude+faster+than+BLAST&amp;author=RC+Edgar&amp;volume=26&amp;publication_year=2010&amp;pages=2460-2461&amp;pmid=20709691&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R8\"><span class=\"mixed-citation\">Englyst HN, and Cummings JH (1988). <span class=\"ref-title\">Improved Method for Measurement of Dietary Fiber as Non-Starch Polysaccharides in Plant Foods</span>. <span class=\"ref-journal\">J Assoc Official Analytical Chemists</span>\n<span class=\"ref-vol\">71</span>, 808&#x02013;814. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n2458334\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Assoc+Official+Analytical+Chemists&amp;title=Improved+Method+for+Measurement+of+Dietary+Fiber+as+Non-Starch+Polysaccharides+in+Plant+Foods&amp;author=HN+Englyst&amp;author=JH+Cummings&amp;volume=71&amp;publication_year=1988&amp;pages=808-814&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R9\"><span class=\"mixed-citation\">Faith JJ, Ahern PP, Ridaura VK, Cheng J, and Gordon JI (2014). <span class=\"ref-title\">Identifying gut microbe-host phenotype relationships using combinatorial communities in gnotobiotic mice</span>. <span class=\"ref-journal\">Sci Transl Med</span>\n<span class=\"ref-vol\">6</span>, 220ra211. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3973144/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24452263\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Sci+Transl+Med&amp;title=Identifying+gut+microbe-host+phenotype+relationships+using+combinatorial+communities+in+gnotobiotic+mice&amp;author=JJ+Faith&amp;author=PP+Ahern&amp;author=VK+Ridaura&amp;author=J+Cheng&amp;author=JI+Gordon&amp;volume=6&amp;publication_year=2014&amp;pages=220ra211&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R10\"><span class=\"mixed-citation\">Faith JJ, Guruge JL, Charbonneau M, Subramanian S, Seedorf H, Goodman AL, Clemente JC, Knight R, Heath AC, Leibel RL, et al. (2013). <span class=\"ref-title\">The long-term stability of the human gut microbiota</span>. <span class=\"ref-journal\">Science</span>\n<span class=\"ref-vol\">341</span>, 1237439. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3791589/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23828941\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+long-term+stability+of+the+human+gut+microbiota&amp;author=JJ+Faith&amp;author=JL+Guruge&amp;author=M+Charbonneau&amp;author=S+Subramanian&amp;author=H+Seedorf&amp;volume=341&amp;publication_year=2013&amp;pages=1237439&amp;pmid=23828941&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R11\"><span class=\"mixed-citation\">Filisetti-Cozzi TM, and Carpita NC (1991). <span class=\"ref-title\">Measurement of uronic acids without interference from neutral sugars</span>. <span class=\"ref-journal\">Anal Biochem</span>\n<span class=\"ref-vol\">197</span>, 157&#x02013;162. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n1952059\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Anal+Biochem&amp;title=Measurement+of+uronic+acids+without+interference+from+neutral+sugars&amp;author=TM+Filisetti-Cozzi&amp;author=NC+Carpita&amp;volume=197&amp;publication_year=1991&amp;pages=157-162&amp;pmid=1952059&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R12\"><span class=\"mixed-citation\">Fragiadakis GK, Smits SA, Sonnenburg ED, Van Treuren W, Reid G, Knight R, Manjurano A, Changalucha J, Dominguez-Bello MG, Leach J, et al. (2018). <span class=\"ref-title\">Links between environment, diet, and the hunter-gatherer microbiome</span>. <span class=\"ref-journal\">Gut Microbes</span>, 1&#x02013;12. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC6546328/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n30118385\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Gut+Microbes&amp;title=Links+between+environment,+diet,+and+the+hunter-gatherer+microbiome&amp;author=GK+Fragiadakis&amp;author=SA+Smits&amp;author=ED+Sonnenburg&amp;author=W+Van+Treuren&amp;author=G+Reid&amp;publication_year=2018&amp;pages=1-12&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R13\"><span class=\"mixed-citation\">Glenwright AJ, Pothula KR, Bhamidimarri SP, Chorev DS, Basle A, Firbank SJ, Zheng H, Robinson CV, Winterhalter M, Kleinekathofer U, Bolam DN, et al. (2017). <span class=\"ref-title\">Structural basis for nutrient acquisition by dominant members of the human gut microbiota</span>. <span class=\"ref-journal\">Nature</span>\n<span class=\"ref-vol\">541</span>, 407&#x02013;411. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5497811/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28077872\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structural+basis+for+nutrient+acquisition+by+dominant+members+of+the+human+gut+microbiota&amp;author=AJ+Glenwright&amp;author=KR+Pothula&amp;author=SP+Bhamidimarri&amp;author=DS+Chorev&amp;author=A+Basle&amp;volume=541&amp;publication_year=2017&amp;pages=407-411&amp;pmid=28077872&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R14\"><span class=\"mixed-citation\">Goodman AL, McNulty NP, Zhao Y, Leip D, Mitra RD, Lozupone CA, Knight R, and Gordon JI (2009). <span class=\"ref-title\">Identifying genetic determinants needed to establish a human gut symbiont in its habitat</span>. <span class=\"ref-journal\">Cell Host &#x00026; Microbe</span>\n<span class=\"ref-vol\">6</span>, 279&#x02013;289. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2895552/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19748469\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cell+Host+&#x00026;+Microbe&amp;title=Identifying+genetic+determinants+needed+to+establish+a+human+gut+symbiont+in+its+habitat&amp;author=AL+Goodman&amp;author=NP+McNulty&amp;author=Y+Zhao&amp;author=D+Leip&amp;author=RD+Mitra&amp;volume=6&amp;publication_year=2009&amp;pages=279-289&amp;pmid=19748469&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R15\"><span class=\"mixed-citation\">Hibberd MC, Wu M, Rodionov DA, Li X, Cheng J, Griffin NW, Barratt MJ, Giannone RJ, Hettich RL, Osterman AL, et al. (2017). <span class=\"ref-title\">The effects of micronutrient deficiencies on bacterial species from the human gut microbiota</span>. <span class=\"ref-journal\">Sci Transl Med</span>\n<span class=\"ref-vol\">9</span>. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5524138/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28515336\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Sci+Transl+Med&amp;title=The+effects+of+micronutrient+deficiencies+on+bacterial+species+from+the+human+gut+microbiota&amp;author=MC+Hibberd&amp;author=M+Wu&amp;author=DA+Rodionov&amp;author=X+Li&amp;author=J+Cheng&amp;volume=9&amp;publication_year=2017&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R16\"><span class=\"mixed-citation\">Hibbing ME, Fuqua C, Parsek MR, and Peterson SB (2010). <span class=\"ref-title\">Bacterial competition: surviving and thriving in the microbial jungle</span>. <span class=\"ref-journal\">Nature Reviews Microbiology</span>\n<span class=\"ref-vol\">8</span>, 15&#x02013;25. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2879262/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19946288\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature+Reviews+Microbiology&amp;title=Bacterial+competition:+surviving+and+thriving+in+the+microbial+jungle&amp;author=ME+Hibbing&amp;author=C+Fuqua&amp;author=MR+Parsek&amp;author=SB+Peterson&amp;volume=8&amp;publication_year=2010&amp;pages=15-25&amp;pmid=19946288&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R17\"><span class=\"mixed-citation\">Heiss C, Klutts JS, Wang Z, Doering TL, and Azadi P (2009). <span class=\"ref-title\">The structure of Cryptococcus neoformans galactoxylomannan contains beta-D-glucuronic acid</span>. <span class=\"ref-journal\">Carbohydr Res</span>\n<span class=\"ref-vol\">344</span>, 915&#x02013;920. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2695399/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19345342\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Carbohydr+Res&amp;title=The+structure+of+Cryptococcus+neoformans+galactoxylomannan+contains+beta-D-glucuronic+acid&amp;author=C+Heiss&amp;author=JS+Klutts&amp;author=Z+Wang&amp;author=TL+Doering&amp;author=P+Azadi&amp;volume=344&amp;publication_year=2009&amp;pages=915-920&amp;pmid=19345342&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R18\"><span class=\"mixed-citation\">Kotarski SF, and Salyers AA (1984). <span class=\"ref-title\">Isolation and characterization of outer membranes of Bacteroides thetaiotaomicron grown on different carbohydrates</span>. <span class=\"ref-journal\">J Bacteriol</span>\n<span class=\"ref-vol\">158</span>, 102&#x02013;109. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC215385/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n6715279\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Isolation+and+characterization+of+outer+membranes+of+Bacteroides+thetaiotaomicron+grown+on+different+carbohydrates&amp;author=SF+Kotarski&amp;author=AA+Salyers&amp;volume=158&amp;publication_year=1984&amp;pages=102-109&amp;pmid=6715279&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R19\"><span class=\"mixed-citation\">Law CW, Chen Y, Shi W, and Smyth GK (2014). <span class=\"ref-title\">voom: Precision weights unlock linear model analysis tools for RNA-seq read counts</span>. <span class=\"ref-journal\">Genome Biol</span>\n<span class=\"ref-vol\">15</span>, R29. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4053721/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24485249\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=voom:+Precision+weights+unlock+linear+model+analysis+tools+for+RNA-seq+read+counts&amp;author=CW+Law&amp;author=Y+Chen&amp;author=W+Shi&amp;author=GK+Smyth&amp;volume=15&amp;publication_year=2014&amp;pages=R29&amp;pmid=24485249&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R20\"><span class=\"mixed-citation\">Levigne S, Thomas M, Ralet MC, Quemener BC, and Thibault JF (2002). <span class=\"ref-title\">Determination of the degrees of methylation and acetylation of pectins using a C18 column and internal standards</span>. <span class=\"ref-journal\">Food Hydrocolloids</span>\n<span class=\"ref-vol\">16</span>, 547&#x02013;550. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Food+Hydrocolloids&amp;title=Determination+of+the+degrees+of+methylation+and+acetylation+of+pectins+using+a+C18+column+and+internal+standards&amp;author=S+Levigne&amp;author=M+Thomas&amp;author=MC+Ralet&amp;author=BC+Quemener&amp;author=JF+Thibault&amp;volume=16&amp;publication_year=2002&amp;pages=547-550&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R21\"><span class=\"mixed-citation\">Luis AS, Briggs J, Zhang X, Farnell B, Ndeh D, Labourel A, Basle A, Cartmell A, Terrapon N, Stott K, et al. (2018). <span class=\"ref-title\">Dietary pectic glycans are degraded by coordinated enzyme pathways in human colonic Bacteroides</span>. <span class=\"ref-journal\">Nature Microbiol</span>\n<span class=\"ref-vol\">3</span>, 210&#x02013;219. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5784806/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n29255254\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature+Microbiol&amp;title=Dietary+pectic+glycans+are+degraded+by+coordinated+enzyme+pathways+in+human+colonic+Bacteroides&amp;author=AS+Luis&amp;author=J+Briggs&amp;author=X+Zhang&amp;author=B+Farnell&amp;author=D+Ndeh&amp;volume=3&amp;publication_year=2018&amp;pages=210-219&amp;pmid=29255254&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R22\"><span class=\"mixed-citation\">Lynch JB, and Sonnenburg JL (2012). <span class=\"ref-title\">Prioritization of a plant polysaccharide over a mucus carbohydrate is enforced by a Bacteroides hybrid two-component system</span>. <span class=\"ref-journal\">Mol Microbiol</span>\n<span class=\"ref-vol\">85</span>, 478&#x02013;491. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3404733/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22686399\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol+Microbiol&amp;title=Prioritization+of+a+plant+polysaccharide+over+a+mucus+carbohydrate+is+enforced+by+a+Bacteroides+hybrid+two-component+system&amp;author=JB+Lynch&amp;author=JL+Sonnenburg&amp;volume=85&amp;publication_year=2012&amp;pages=478-491&amp;pmid=22686399&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R23\"><span class=\"mixed-citation\">Ma ZQ, Dasari S, Chambers MC, Litton MD, Sobecki SM, Zimmerman LJ, Halvey PJ, Schilling B, Drake PM, Gibson BW, et al. (2009). <span class=\"ref-title\">IDPicker 2.0: Improved protein assembly with high discrimination peptide identification filtering</span>. <span class=\"ref-journal\">J Proteome Res</span>\n<span class=\"ref-vol\">8</span>, 3872&#x02013;3881. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2810655/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19522537\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Proteome+Res&amp;title=IDPicker+2.0:+Improved+protein+assembly+with+high+discrimination+peptide+identification+filtering&amp;author=ZQ+Ma&amp;author=S+Dasari&amp;author=MC+Chambers&amp;author=MD+Litton&amp;author=SM+Sobecki&amp;volume=8&amp;publication_year=2009&amp;pages=3872-3881&amp;pmid=19522537&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R24\"><span class=\"mixed-citation\">Martens EC, Lowe EC, Chiang H, Pudlo NA, Wu M, McNulty NP, Abbott DW, Henrissat B, Gilbert HJ, Bolam DN, et al. (2011). <span class=\"ref-title\">Recognition and degradation of plant cell wall polysaccharides by two human gut symbionts</span>. <span class=\"ref-journal\">PLoS Biology</span>\n<span class=\"ref-vol\">9</span>, e1001221. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3243724/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22205877\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+Biology&amp;title=Recognition+and+degradation+of+plant+cell+wall+polysaccharides+by+two+human+gut+symbionts&amp;author=EC+Martens&amp;author=EC+Lowe&amp;author=H+Chiang&amp;author=NA+Pudlo&amp;author=M+Wu&amp;volume=9&amp;publication_year=2011&amp;pages=e1001221&amp;pmid=22205877&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R25\"><span class=\"mixed-citation\">McDonald D, Price MN, Goodrich J, Nawrocki EP, DeSantis TZ, Probst A, Andersen GL, Knight R, and Hugenholtz P (2012). <span class=\"ref-title\">An improved Greengenes taxonomy with explicit ranks for ecological and evolutionary analyses of bacteria and archaea</span>. <span class=\"ref-journal\">ISME J</span>\n<span class=\"ref-vol\">6</span>, 610&#x02013;618. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3280142/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22134646\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=ISME+J&amp;title=An+improved+Greengenes+taxonomy+with+explicit+ranks+for+ecological+and+evolutionary+analyses+of+bacteria+and+archaea&amp;author=D+McDonald&amp;author=MN+Price&amp;author=J+Goodrich&amp;author=EP+Nawrocki&amp;author=TZ+DeSantis&amp;volume=6&amp;publication_year=2012&amp;pages=610-618&amp;pmid=22134646&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R26\"><span class=\"mixed-citation\">McNulty NP, Wu M, Erickson AR, Pan C, Erickson BK, Martens EC, Pudlo NA, Muegge BD, Henrissat B, Hettich RL, et al. (2013). <span class=\"ref-title\">Effects of diet on resource utilization by a model human gut microbiota containing Bacteroides cellulosilyticus WH2, a symbiont with an extensive glycobiome</span>. <span class=\"ref-journal\">PLoS Biology</span>\n<span class=\"ref-vol\">11</span>, e1001637. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3747994/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23976882\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+Biology&amp;title=Effects+of+diet+on+resource+utilization+by+a+model+human+gut+microbiota+containing+Bacteroides+cellulosilyticus+WH2,+a+symbiont+with+an+extensive+glycobiome&amp;author=NP+McNulty&amp;author=M+Wu&amp;author=AR+Erickson&amp;author=C+Pan&amp;author=BK+Erickson&amp;volume=11&amp;publication_year=2013&amp;pages=e1001637&amp;pmid=23976882&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R27\"><span class=\"mixed-citation\">Menni C, Jackson MA, Pallister T, Steves CJ, Spector TD, and Valdes AM (2017). <span class=\"ref-title\">Gut microbiome diversity and high-fibre intake are related to lower long-term weight gain</span>. <span class=\"ref-journal\">Int J Obes</span>\n<span class=\"ref-vol\">41</span>, 1099&#x02013;1105. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5500185/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28286339\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int+J+Obes&amp;title=Gut+microbiome+diversity+and+high-fibre+intake+are+related+to+lower+long-term+weight+gain&amp;author=C+Menni&amp;author=MA+Jackson&amp;author=T+Pallister&amp;author=CJ+Steves&amp;author=TD+Spector&amp;volume=41&amp;publication_year=2017&amp;pages=1099-1105&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R28\"><span class=\"mixed-citation\">Pattathil S, Avci U, Miller JS, and Hahn MG (2012). <span class=\"ref-title\">Immunological approaches to plant cell wall and biomass characterization: Glycome Profiling</span>. <span class=\"ref-journal\">Methods Mol Biol</span>\n<span class=\"ref-vol\">908</span>, 61&#x02013;72. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22843389\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Methods+Mol+Biol&amp;title=Immunological+approaches+to+plant+cell+wall+and+biomass+characterization:+Glycome+Profiling&amp;author=S+Pattathil&amp;author=U+Avci&amp;author=JS+Miller&amp;author=MG+Hahn&amp;volume=908&amp;publication_year=2012&amp;pages=61-72&amp;pmid=22843389&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R29\"><span class=\"mixed-citation\">Pettolino FA, Walsh C, Fincher GB, and Bacic A (2012). <span class=\"ref-title\">Determining the polysaccharide composition of plant cell walls</span>. <span class=\"ref-journal\">Nature Protocols</span>\n<span class=\"ref-vol\">7</span>, 1590&#x02013;1607. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22864200\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature+Protocols&amp;title=Determining+the+polysaccharide+composition+of+plant+cell+walls&amp;author=FA+Pettolino&amp;author=C+Walsh&amp;author=GB+Fincher&amp;author=A+Bacic&amp;volume=7&amp;publication_year=2012&amp;pages=1590-1607&amp;pmid=22864200&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R30\"><span class=\"mixed-citation\">Porter NT, and Martens EC (2017). <span class=\"ref-title\">The critical roles of polysaccharides in gut microbial ecology and physiology</span>. <span class=\"ref-journal\">Annu Rev Microbiol</span>\n<span class=\"ref-vol\">71</span>, 349&#x02013;369. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28657886\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Microbiol&amp;title=The+critical+roles+of+polysaccharides+in+gut+microbial+ecology+and+physiology&amp;author=NT+Porter&amp;author=EC+Martens&amp;volume=71&amp;publication_year=2017&amp;pages=349-369&amp;pmid=28657886&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R31\"><span class=\"mixed-citation\">Ridaura VK, Faith JJ, Rey FE, Cheng J, Duncan AE, Kau AL, Griffin NW, Lombard V, Henrissat B, Bain JR, et al. (2013). <span class=\"ref-title\">Gut microbiota from twins discordant for obesity modulate metabolism in mice</span>. <span class=\"ref-journal\">Science</span>\n<span class=\"ref-vol\">341</span>, 1241214. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3829625/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24009397\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Gut+microbiota+from+twins+discordant+for+obesity+modulate+metabolism+in+mice&amp;author=VK+Ridaura&amp;author=JJ+Faith&amp;author=FE+Rey&amp;author=J+Cheng&amp;author=AE+Duncan&amp;volume=341&amp;publication_year=2013&amp;pages=1241214&amp;pmid=24009397&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R32\"><span class=\"mixed-citation\">Rogowski A, Briggs JA, Mortimer JC, Tryfona T, Terrapon N, Lowe EC, Basle A, Morland C, Day AM, Zheng H, et al. (2015). <span class=\"ref-title\">Glycan complexity dictates microbial resource allocation in the large intestine</span>. <span class=\"ref-journal\">Nature Commun</span>\n<span class=\"ref-vol\">6</span>, 7481. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4491172/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26112186\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature+Commun&amp;title=Glycan+complexity+dictates+microbial+resource+allocation+in+the+large+intestine&amp;author=A+Rogowski&amp;author=JA+Briggs&amp;author=JC+Mortimer&amp;author=T+Tryfona&amp;author=N+Terrapon&amp;volume=6&amp;publication_year=2015&amp;pages=7481&amp;pmid=26112186&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R33\"><span class=\"mixed-citation\">Santander J, Martin T, Loh A, Pohlenz C, Gatlin DM 3rd, and Curtiss R 3rd (2013). <span class=\"ref-title\">Mechanisms of intrinsic resistance to antimicrobial peptides of Edwardsiella ictaluri and its influence on fish gut inflammation and virulence</span>. <span class=\"ref-journal\">Microbiology</span>\n<span class=\"ref-vol\">159</span>, 1471&#x02013;1486. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4085987/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23676433\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Mechanisms+of+intrinsic+resistance+to+antimicrobial+peptides+of+Edwardsiella+ictaluri+and+its+influence+on+fish+gut+inflammation+and+virulence&amp;author=J+Santander&amp;author=T+Martin&amp;author=A+Loh&amp;author=C+Pohlenz&amp;author=DM+Gatlin&amp;volume=159&amp;publication_year=2013&amp;pages=1471-1486&amp;pmid=23676433&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R34\"><span class=\"mixed-citation\">Schwalm ND 3rd, Townsend GE 2nd, and Groisman EA (2016). <span class=\"ref-title\">Multiple signals govern utilization of a polysaccharide in the gut bacterium Bacteroides thetaiotaomicron</span>. <span class=\"ref-journal\">MBio</span>\n<span class=\"ref-vol\">7</span>. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5061871/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27729509\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=MBio&amp;title=Multiple+signals+govern+utilization+of+a+polysaccharide+in+the+gut+bacterium+Bacteroides+thetaiotaomicron&amp;author=ND+Schwalm&amp;author=GE+Townsend&amp;author=EA+Groisman&amp;volume=7&amp;publication_year=2016&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R35\"><span class=\"mixed-citation\">Shepherd ES, DeLoache WC, Pruss KM, Whitaker WR, and Sonnenburg JL (2018). <span class=\"ref-title\">An exclusive metabolic niche enables strain engraftment in the gut microbiota</span>. <span class=\"ref-journal\">Nature</span>\n<span class=\"ref-vol\">557</span>, 434&#x02013;438. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC6126907/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n29743671\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+exclusive+metabolic+niche+enables+strain+engraftment+in+the+gut+microbiota&amp;author=ES+Shepherd&amp;author=WC+DeLoache&amp;author=KM+Pruss&amp;author=WR+Whitaker&amp;author=JL+Sonnenburg&amp;volume=557&amp;publication_year=2018&amp;pages=434-438&amp;pmid=29743671&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R36\"><span class=\"mixed-citation\">Smyth GK (2004). <span class=\"ref-title\">Linear models and empirical bayes methods for assessing differential expression in microarray experiments</span>. <span class=\"ref-journal\">Stat Appl Genet Mol Biol</span>\n<span class=\"ref-vol\">3</span>, Article3. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n16646809\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Stat+Appl+Genet+Mol+Biol&amp;title=Linear+models+and+empirical+bayes+methods+for+assessing+differential+expression+in+microarray+experiments&amp;author=GK+Smyth&amp;volume=3&amp;publication_year=2004&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R37\"><span class=\"mixed-citation\">Sonnenburg ED, Smits SA, Tikhonov M, Higginbottom SK, Wingreen NS, and Sonnenburg JL (2016). <span class=\"ref-title\">Diet-induced extinctions in the gut microbiota compound over generations</span>. <span class=\"ref-journal\">Nature</span>\n<span class=\"ref-vol\">529</span>, 212&#x02013;215. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4850918/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26762459\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Diet-induced+extinctions+in+the+gut+microbiota+compound+over+generations&amp;author=ED+Sonnenburg&amp;author=SA+Smits&amp;author=M+Tikhonov&amp;author=SK+Higginbottom&amp;author=NS+Wingreen&amp;volume=529&amp;publication_year=2016&amp;pages=212-215&amp;pmid=26762459&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R38\"><span class=\"mixed-citation\">Sonnenburg ED, Zheng H, Joglekar P, Higginbottom SK, Firbank SJ, Bolam DN, and Sonnenburg JL (2010). <span class=\"ref-title\">Specificity of polysaccharide use in intestinal bacteroides species determines diet-induced microbiota alterations</span>. <span class=\"ref-journal\">Cell</span>\n<span class=\"ref-vol\">141</span>, 1241&#x02013;1252. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2900928/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20603004\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Specificity+of+polysaccharide+use+in+intestinal+bacteroides+species+determines+diet-induced+microbiota+alterations&amp;author=ED+Sonnenburg&amp;author=H+Zheng&amp;author=P+Joglekar&amp;author=SK+Higginbottom&amp;author=SJ+Firbank&amp;volume=141&amp;publication_year=2010&amp;pages=1241-1252&amp;pmid=20603004&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R39\"><span class=\"mixed-citation\">Tabb DL, Fernando CG, and Chambers MC (2007). <span class=\"ref-title\">MyriMatch: highly accurate tandem mass spectral peptide identification by multivariate hypergeometric analysis</span>. <span class=\"ref-journal\">J Proteome Res</span>\n<span class=\"ref-vol\">6</span>, 654&#x02013;661. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2525619/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17269722\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Proteome+Res&amp;title=MyriMatch:+highly+accurate+tandem+mass+spectral+peptide+identification+by+multivariate+hypergeometric+analysis&amp;author=DL+Tabb&amp;author=CG+Fernando&amp;author=MC+Chambers&amp;volume=6&amp;publication_year=2007&amp;pages=654-661&amp;pmid=17269722&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R40\"><span class=\"mixed-citation\">Tauzin AS, Laville E, Xiao Y, Nouaille S, Le Bourgeois P, Heux S, Portais JC, Monsan P, Martens EC, Potocki-Veronese G, et al. (2016). <span class=\"ref-title\">Functional characterization of a gene locus from an uncultured gut Bacteroides conferring xylo-oligosaccharides utilization to Escherichia coli</span>. <span class=\"ref-journal\">Mol Microbiol</span>\n<span class=\"ref-vol\">102</span>, 579&#x02013;592. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27573446\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol+Microbiol&amp;title=Functional+characterization+of+a+gene+locus+from+an+uncultured+gut+Bacteroides+conferring+xylo-oligosaccharides+utilization+to+Escherichia+coli&amp;author=AS+Tauzin&amp;author=E+Laville&amp;author=Y+Xiao&amp;author=S+Nouaille&amp;author=P+Le+Bourgeois&amp;volume=102&amp;publication_year=2016&amp;pages=579-592&amp;pmid=27573446&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R41\"><span class=\"mixed-citation\">Terrapon N, Lombard V, Drula E, Lapebie P, Al-Masaudi S, Gilbert HJ, and Henrissat B (2018). <span class=\"ref-title\">PULDB: the expanded database of Polysaccharide Utilization Loci</span>. <span class=\"ref-journal\">Nucleic Acids Res</span>\n<span class=\"ref-vol\">46</span>, D677&#x02013;D683. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5753385/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n29088389\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=PULDB:+the+expanded+database+of+Polysaccharide+Utilization+Loci&amp;author=N+Terrapon&amp;author=V+Lombard&amp;author=E+Drula&amp;author=P+Lapebie&amp;author=S+Al-Masaudi&amp;volume=46&amp;publication_year=2018&amp;pages=D677-D683&amp;pmid=29088389&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R42\"><span class=\"mixed-citation\">Thibault JF (1979). <span class=\"ref-title\">Automatisation du dosage des substances pectiques par la m&#x000e9;thode au m&#x000e9;tahydroxydiph&#x000e9;nyle</span>. <span class=\"ref-journal\">Lebensml Wiss Technol</span>\n<span class=\"ref-vol\">12</span>, 247&#x02013;251. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Lebensml+Wiss+Technol&amp;title=Automatisation+du+dosage+des+substances+pectiques+par+la+m&#x000e9;thode+au+m&#x000e9;tahydroxydiph&#x000e9;nyle&amp;author=JF+Thibault&amp;volume=12&amp;publication_year=1979&amp;pages=247-251&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R43\"><span class=\"mixed-citation\">Ting L, Cowley MJ, Hoon SL, Guilhaus M, Raftery MJ, and Cavicchioli R (2009). <span class=\"ref-title\">Normalization and statistical analysis of quantitative proteomics data generated by metabolic labeling</span>. <span class=\"ref-journal\">Mol Cell Proteomics</span>\n<span class=\"ref-vol\">8</span>, 2227&#x02013;2242. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2758752/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19605365\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Proteomics&amp;title=Normalization+and+statistical+analysis+of+quantitative+proteomics+data+generated+by+metabolic+labeling&amp;author=L+Ting&amp;author=MJ+Cowley&amp;author=SL+Hoon&amp;author=M+Guilhaus&amp;author=MJ+Raftery&amp;volume=8&amp;publication_year=2009&amp;pages=2227-2242&amp;pmid=19605365&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R44\"><span class=\"mixed-citation\">Wu M, McNulty NP, Rodionov DA, Khoroshkin MS, Griffin NW, Cheng J, Latreille P, Kerstetter RA, Terrapon N, Henrissat B, et al. (2015). <span class=\"ref-title\">Genetic determinants of in vivo fitness and diet responsiveness in multiple human gut Bacteroides</span>. <span class=\"ref-journal\">Science</span>\n<span class=\"ref-vol\">350</span>, aac5992. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4608238/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26430127\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genetic+determinants+of+in+vivo+fitness+and+diet+responsiveness+in+multiple+human+gut+Bacteroides&amp;author=M+Wu&amp;author=NP+McNulty&amp;author=DA+Rodionov&amp;author=MS+Khoroshkin&amp;author=NW+Griffin&amp;volume=350&amp;publication_year=2015&amp;pages=aac5992&amp;pmid=26430127&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R45\"><span class=\"mixed-citation\">Zeevi D, Korem T, Zmora N, Israeli D, Rothschild D, Weinberger A, Ben-Yacov O, Lador D, Avnit-Sagi T, Lotan-Pompan M, et al. (2015). <span class=\"ref-title\">Personalized Nutrition by Prediction of Glycemic Responses</span>. <span class=\"ref-journal\">Cell</span>\n<span class=\"ref-vol\">163</span>, 1079&#x02013;1094. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26590418\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Personalized+Nutrition+by+Prediction+of+Glycemic+Responses&amp;author=D+Zeevi&amp;author=T+Korem&amp;author=N+Zmora&amp;author=D+Israeli&amp;author=D+Rothschild&amp;volume=163&amp;publication_year=2015&amp;pages=1079-1094&amp;pmid=26590418&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R46\"><span class=\"mixed-citation\">Zhao L, Zhang F, Ding X, Wu G, Lam YY, Wang X, Fu H, Xue X, Lu C, Ma J, et al. (2018). <span class=\"ref-title\">Gut bacteria selectively promoted by dietary fibers alleviate type 2 diabetes</span>. <span class=\"ref-journal\">Science</span>\n<span class=\"ref-vol\">359</span>, 1151&#x02013;1156. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n29590046\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Gut+bacteria+selectively+promoted+by+dietary+fibers+alleviate+type+2+diabetes&amp;author=L+Zhao&amp;author=F+Zhang&amp;author=X+Ding&amp;author=G+Wu&amp;author=YY+Lam&amp;volume=359&amp;publication_year=2018&amp;pages=1151-1156&amp;pmid=29590046&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li></ul></div></div><br>', NULL, 1);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(71, 'pubmed', '31539500', 'Interspecies Competition Impacts Targeted Manipulation of Human Gut Bacteria by Fiber-Derived Glycans.', 'Development of microbiota-directed foods (MDFs) that selectively increase the abundance of beneficial human gut microbes, and their expressed functions, requires knowledge of both the bioactive components of MDFs and the mechanisms underlying microbe-microbe interactions. Here, gnotobiotic mice were colonized with a defined consortium of human-gut-derived bacterial strains and fed different combinations of 34 food-grade fibers added to a representative low-fiber diet consumed in the United States. Bioactive carbohydrates in fiber preparations targeting particular Bacteroides species were identified using community-wide quantitative proteomic analyses of bacterial gene expression coupled with forward genetic screens. Deliberate manipulation of community membership combined with administration of retrievable artificial food particles, consisting of paramagnetic microscopic beads coated with dietary polysaccharides, disclosed the contributions of targeted species to fiber degradation. Our approach, including the use of bead-based biosensors, defines nutrient-harvesting strategies that underlie, as well as alleviate, competition between Bacteroides and control the selectivity of MDF components.', 2019, 'Cell', 'PMC6760872', 1, NULL, NULL, 2),
(73, 'pubmed', '28928724', 'The Maturing Development of Gut Microbiota in Commercial Piglets during the Weaning Transition.', 'Early weaned piglets are vulnerable to diarrhea because of weaning stress and immaturity of intestinal tract. Compelling evidence suggests that gut microbiota is vital to host health. However, it is not well understood on the composition and succession of piglet gut microbiota during the weaning transition. In our two trials, total 17 commercial piglets were studied in a pig farm in Jiangxi Province, China. Fresh feces were collected for four times (10 days before weaned, weaned day, 10 days after weaned, 21 days after weaned) by rectal massage. Fecal bacterial composition was assessed by 16S rRNA gene V3-V4 regions sequencing by Illumina Miseq platform. The results showed that the gut microbiota of piglets shifted quickly after weaned and reached relatively stable level in 10 days after weaned. The alpha diversity increased significantly with the age of piglets. The microbiota of suckling piglets was mainly represented by , and . This pattern contrasted with that of  XIVa, and , which were major representative genera after weaned. In summary, we delineated the development of piglet gut microbiota during the weaning transition. This study helps us understand the maturing development of gut microbiota in commercial piglets.', 2017, 'Frontiers in microbiology', '', 1, NULL, NULL, 2);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(74, 'pubmed', '28567446', 'Meta-analysis To Define a Core Microbiota in the Swine Gut.', 'The swine gut microbiota encompasses a large and diverse population of bacteria that play a significant role in pig health. As such, a number of recent studies have utilized high-throughput sequencing of the 16S rRNA gene to characterize the composition and structure of the swine gut microbiota, often in response to dietary feed additives. It is important to determine which factors shape the composition of the gut microbiota among multiple studies and if certain bacteria are always present in the gut microbiota of swine, independently of study variables such as country of origin and experimental design. Therefore, we performed a meta-analysis using 20 publically available data sets from high-throughput 16S rRNA gene sequence studies of the swine gut microbiota. Next to the \"study\" itself, the gastrointestinal (GI) tract section that was sampled had the greatest effect on the composition and structure of the swine gut microbiota ( = 0.0001). Technical variation among studies, particularly the 16S rRNA gene hypervariable region sequenced, also significantly affected the composition of the swine gut microbiota ( = 0.0001). Despite this, numerous commonalities were discovered. Among fecal samples, the genera , , , and  and the RC9 gut group were found in 99% of all fecal samples. Additionally, , , , , , , the RC9 gut group, and  were shared by >90% of all GI samples, suggesting a so-called \"core\" microbiota for commercial swine worldwide.  The results of this meta-analysis demonstrate that \"study\" and GI sample location are the most significant factors in shaping the swine gut microbiota. However, in comparisons of results from different studies, some biological factors may be obscured by technical variation among studies. Nonetheless, there are some bacterial taxa that appear to form a core microbiota within the swine GI tract regardless of country of origin, diet, age, or breed. Thus, these results provide the framework for future studies to manipulate the swine gut microbiota for potential health benefits.', 0000, 'mSystems', 'PMC5443231', 1, '<h1 class=\"content-title\">Meta-analysis To Define a Core Microbiota in the Swine Gut</h1><br><span class=\"contrib-group fm-author\"><a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Holman%20DB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28567446\" class=\"affpopup\" co-rid=\"_co_idm140162580283744\" co-class=\"co-affbox\">Devin B. Holman</a>,<sup>a</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Brunelle%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28567446\" class=\"affpopup\" co-rid=\"_co_idm140162579596336\" co-class=\"co-affbox\">Brian W. Brunelle</a>,<sup>a</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Trachsel%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28567446\" class=\"affpopup\" co-rid=\"_co_idm140162585884096\" co-class=\"co-affbox\">Julian Trachsel</a>,<sup>a,</sup><sup>b</sup> and  <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Allen%20HK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28567446\" class=\"affpopup\" co-rid=\"_co_idm140162569374576\" co-class=\"co-affbox\">Heather K. Allen</a><sup><img src=\"https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/corrauth.gif\"\r\n alt=\"corresponding author\" /></sup><sup>a</sup></span><span style=\"display:none\" class=\"contrib-group aff-tip\"><span id=\"_co_idm140162580283744\"><h3 class=\"no_margin\">Devin B. Holman</h3><p><sup>a</sup>Food Safety and Enteric Pathogens Research Unit, National Animal Disease Center, Agricultural Research Service, United States Department of Agriculture, Ames, Iowa, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Holman%20DB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28567446\">Devin B. Holman</a></span></span><span id=\"_co_idm140162579596336\"><h3 class=\"no_margin\">Brian W. Brunelle</h3><p><sup>a</sup>Food Safety and Enteric Pathogens Research Unit, National Animal Disease Center, Agricultural Research Service, United States Department of Agriculture, Ames, Iowa, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Brunelle%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28567446\">Brian W. Brunelle</a></span></span><span id=\"_co_idm140162585884096\"><h3 class=\"no_margin\">Julian Trachsel</h3><p><sup>a</sup>Food Safety and Enteric Pathogens Research Unit, National Animal Disease Center, Agricultural Research Service, United States Department of Agriculture, Ames, Iowa, USA</p><p><sup>b</sup>Interdepartmental Microbiology Graduate Program, Iowa State University, Ames, Iowa, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Trachsel%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28567446\">Julian Trachsel</a></span></span><span id=\"_co_idm140162569374576\"><h3 class=\"no_margin\">Heather K. Allen</h3><p><sup>a</sup>Food Safety and Enteric Pathogens Research Unit, National Animal Disease Center, Agricultural Research Service, United States Department of Agriculture, Ames, Iowa, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Allen%20HK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28567446\">Heather K. Allen</a></span></span></span></span><span class=\"contrib-group half_rhythm fm-editor\">Holly Bik, <span class=\"fm-role\">Editor</span><span><span class=\"fm-affl\">Holly Bik, </span><span class=\"fm-affl\" lang=\"en\" id=\"idm140162583072544\"> University of California, Riverside;</span></span></span><span class=\"fm-panel half_rhythm\"><span class=\"togglers<br><span id=\"idm140162571806064\"><strong>Citation</strong> Holman DB, Brunelle BW, Trachsel J, Allen HK. 2017. Meta-analysis to define a core microbiota in the swine gut. mSystems 2:e00004-17. <a href=\"https://doi.org/10.1128/mSystems.00004-17\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://doi.org/10.1128/mSystems.00004-17</a>.</span></span><span class=\"fm-article-notes fm-panel hide half_rhythm\" id=\"idm140162577190064_an\" style=\"display:none\"><span class=\"fm-pubdate half_rhythm\">Received 2017 Jan 4; Accepted 2017 Apr 21.</span></span><span class=\"permissions fm-panel half_rhythm hide\" id=\"idm140162577190064_cpl\" style=\"display:none\"><span class=\"fm-copyright half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al.</span><span class=\"license half_rhythm\">This is an open-access article distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span><span id=\"pmclinksbox\" class=\"links-box whole_rhythm\"><span class=\"fm-panel\"><span>This article has been <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/citedby/\">cited by</a> other articles in PMC.</span></span></span></span><span class=\"sec\"></span><span id=\"ass-data\" class=\"tsec fm-sec whole_rhythm\" data-section=\"Featured_PMC_Datacitation\"><h2 class=\"nomenu\">Associated Data</h2><dl data-count=\"5\" class=\"box-data-suppmats whole_rhythm no_bottom_margin\"><dt> \r\n</dt><dd id=\"data-suppmats\"  \r\n><span class=\"half_rhythm\"><span class=\"caption half_rhythm no_bottom_margin\"><strong>TABLE&#x000a0;S1&#x000a0;</strong></span><p>The percentage of samples from each specific gastrointestinal location that had at least one 16S rRNA gene sequence from each of the individual genera identified. Genera are listed in descending order of overall relative abundance. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103st4.pdf\">TABLE&#x000a0;S1, PDF file, 1.1 MB</a>.</p><span class=\"caption half_rhythm no_bottom_margin\"><span class=\"small permissions\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al. <span class=\"license\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span></span><span class=\"half_rhythm\"><span class=\"caption half_rhythm no_bottom_margin\"><strong>FIG&#x000a0;S1&#x000a0;</strong></span><p>Principal-coordinate analysis plots of Bray-Curtis dissimilarities by study. The numbers in parentheses in the figure legend refer to the number of samples included for each study. The percentages of variation explained by the principal coordinates are indicated on the axes. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103sf1.tif\">FIG&#x000a0;S1, TIF file, 2.1 MB</a>.</p><span class=\"caption half_rhythm no_bottom_margin\"><span class=\"small permissions\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al. <span class=\"license\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span></span><span class=\"half_rhythm\"><span class=\"caption half_rhythm no_bottom_margin\"><strong>FIG&#x000a0;S2&#x000a0;</strong></span><p>Principal-coordinate analysis plot of the weighted UniFrac distances for all samples with associated metadata for the age of the pigs used in the study. Samples are colored by pig age in weeks. The numbers in parentheses for the figure legend refer to the number of samples at each week. The percentages of variation explained by the principal coordinates are indicated on the axes. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103sf2.tif\">FIG&#x000a0;S2, TIF file, 2.5 MB</a>.</p><span class=\"caption half_rhythm no_bottom_margin\"><span class=\"small permissions\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al. <span class=\"license\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span></span><span class=\"half_rhythm\"><span class=\"caption half_rhythm no_bottom_margin\"><strong>TABLE&#x000a0;S2&#x000a0;</strong></span><p>The percentage of fecal samples grouped by specific 16S rRNA gene hypervariable region sequenced that had at least one 16S rRNA gene sequence from each of the individual genera identified. Genera are listed in descending order of overall relative abundance. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103st5.pdf\">TABLE&#x000a0;S2, PDF file, 0.5 MB</a>.</p><span class=\"caption half_rhythm no_bottom_margin\"><span class=\"small permissions\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al. <span class=\"license\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span></span><span class=\"half_rhythm\"><span class=\"caption half_rhythm no_bottom_margin\"><strong>FIG&#x000a0;S3&#x000a0;</strong></span><p>Principal-coordinate analysis plots of weighted UniFrac distances (A), unweighted UniFrac distances (B), and Bray-Curtis dissimilarities (C) for studies investigating the effect of an antimicrobial on the swine fecal microbiota. Samples are colored by study and antimicrobial treatment. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103sf3.tif\">FIG&#x000a0;S3, TIF file, 1.7 MB</a>.</p><span class=\"caption half_rhythm no_bottom_margin\"><span class=\"small permissions\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al. <span class=\"license\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span></span></dd></dl></span><span id=\"idm140162573535312\" lang=\"en\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140162573535312title\" style=\"text-transform:uppercase\">ABSTRACT</h2><!--article-meta--><span><p id=\"__p3\" class=\"p p-first\">The swine gut microbiota encompasses a large and diverse population of bacteria that play a significant role in pig health. As such, a number of recent studies have utilized high-throughput sequencing of the 16S rRNA gene to characterize the composition and structure of the swine gut microbiota, often in response to dietary feed additives. It is important to determine which factors shape the composition of the gut microbiota among multiple studies and if certain bacteria are always present in the gut microbiota of swine, independently of study variables such as country of origin and experimental design. Therefore, we performed a meta-analysis using 20 publically available data sets from high-throughput 16S rRNA gene sequence studies of the swine gut microbiota. Next to the &#x0201c;study&#x0201d; itself, the gastrointestinal (GI) tract section that was sampled had the greatest effect on the composition and structure of the swine gut microbiota (<em>P</em> = 0.0001). Technical variation among studies, particularly the 16S rRNA gene hypervariable region sequenced, also significantly affected the composition of the swine gut microbiota (<em>P</em> = 0.0001). Despite this, numerous commonalities were discovered. Among fecal samples, the genera <em>Prevotella</em>, <em>Clostridium</em>, <em>Alloprevotella</em>, and <em>Ruminococcus</em> and the RC9 gut group were found in 99% of all fecal samples. Additionally, <em>Clostridium</em>, <em>Blautia</em>, <em>Lactobacillus</em>, <em>Prevotella</em>, <em>Ruminococcus</em>, <em>Roseburia</em>, the RC9 gut group, and <em>Subdoligranulum</em> were shared by &#x0003e;90% of all GI samples, suggesting a so-called &#x0201c;core&#x0201d; microbiota for commercial swine worldwide.</p><p id=\"__p4\" class=\"p p-last\"><strong>IMPORTANCE</strong> The results of this meta-analysis demonstrate that &#x0201c;study&#x0201d; and GI sample location are the most significant factors in shaping the swine gut microbiota. However, in comparisons of results from different studies, some biological factors may be obscured by technical variation among studies. Nonetheless, there are some bacterial taxa that appear to form a core microbiota within the swine GI tract regardless of country of origin, diet, age, or breed. Thus, these results provide the framework for future studies to manipulate the swine gut microbiota for potential health benefits.</p></span><span class=\"sec\"><strong class=\"kwd-title\">KEYWORDS: </strong><span class=\"kwd-text\">16S rRNA gene, bacteria, gut microbiome, gut microbiota, livestock, meta-analysis, microbial ecology, swine</span></span></span><span id=\"s0\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s0title\" style=\"text-transform:uppercase\">INTRODUCTION</h2><p id=\"__p5\" class=\"p p-first\">The mammalian gut microbiota is populated by an estimated 100 trillion bacterial cells that provide health benefits to the host through the production of short-chain fatty acids and vitamins from otherwise nondigestible food components, inhibition and prevention of colonization by pathogens, and development and maintenance of the host immune system (<a href=\"#B1\" rid=\"B1\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776876\">1</a>,<a href=\"#B2\" rid=\"B2\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776895\">&#x02013;</a><a href=\"#B3\" rid=\"B3\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776882\">3</a>). These properties also make the gut microbiota an attractive target for dietary interventions to improve the health and production of food-producing animals, including swine. Therefore, the characterization of the gut microbiota of swine has become an active area of research in recent years as the adaptation and availability of high-throughput sequencing methods have continued to expand.</p><p id=\"__p6\">The majority of swine gut microbiota studies to date have focused on sequencing specific hypervariable regions within the 16S rRNA gene. The myriad of studies that have sequenced the 16S rRNA gene can be readily aggregated and compared in a meta-analysis that may reveal patterns among individual studies working on similar sample types. Recent meta-analyses of the avian gut microbiota (<a href=\"#B4\" rid=\"B4\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776884\">4</a>) and the indoor microbiota (<a href=\"#B5\" rid=\"B5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776874\">5</a>) have reported that study-level variables, compared to biological factors such as sample type, have the greatest effect on the microbiota. In swine, researchers have examined the effect of a number of different feed additives on the swine gut microbiota, including antimicrobial agents (<a href=\"#B6\" rid=\"B6\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776902\">6</a>,<a href=\"#B7\" rid=\"B7\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776869\">&#x02013;</a><a href=\"#B12\" rid=\"B12\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776894\">12</a>), amylose/amylopectin (<a href=\"#B13\" rid=\"B13\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776845\">13</a>), calcium-phosphorus (<a href=\"#B14\" rid=\"B14\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776844\">14</a>), distillers&#x02019; dried grains with solubles (<a href=\"#B15\" rid=\"B15\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776901\">15</a>), and resistant starch (<a href=\"#B16\" rid=\"B16\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776879\">16</a>). The effect of challenge with <em>Escherichia coli</em> (<a href=\"#B17\" rid=\"B17\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776846\">17</a>) and <em>Salmonella</em> (<a href=\"#B18\" rid=\"B18\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776873\">18</a>) on the swine gut microbiota has also been investigated. All of those studies found various degrees of change in the swine gut microbiota as a result of these treatments. Aggregating and reanalyzing the data could reveal whether alterations observed in individual studies are still detectable when combined with data from similar studies or whether they would be lost due to technical variation between studies.</p><p id=\"__p7\" class=\"p p-last\">Our objectives were to characterize the swine gut microbiota using 16S rRNA gene high-throughput sequencing data from gastrointestinal (GI) samples from 20 publically available studies and to determine if a core gut microbiota, defined here as being present in &#x02265;90% of samples, exists in swine. We evaluated how technical variables, such as study design, country of origin, and hypervariable region sequenced, affect the observed swine gut microbiota.</p></span><span id=\"s1\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s1title\" style=\"text-transform:uppercase\">RESULTS AND DISCUSSION</h2><span id=\"s1.1\" class=\"sec sec-first\"><p></p><h3 id=\"s1.1title\" class=\"inline\">Meta-analysis characteristics. </h3><p id=\"__p8\" class=\"p p-first\">A total of 25,157,796 quality-filtered 16S rRNA gene sequences from 939 swine gastrointestinal samples were available for meta-analysis (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab1/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"tab1\" rid-ob=\"ob-tab1\" co-legend-rid=\"\"><span>Table&#x000a0;1</span></a>). Overall, 18,743,484 (74.5%) of these sequences were at least 97% similar to a sequence in the SILVA database. These sequences represented 25,182 bacterial operational taxonomic units (OTUs), 35 phyla, and 887 genera prior to random subsampling. It was necessary to randomly subsample to a level of 1,000 sequences per sample to retain as many samples as possible; however, as few as 100 sequences per sample have been shown to be sufficient when large differences between sample groups are present (<a href=\"#B19\" rid=\"B19\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776872\">19</a>). Not surprisingly, the majority of sequences available for meta-analysis were derived from swine feces, as these samples are the easiest to obtain from the GI tract and allow repeated sampling over time. Other regions of the swine GI tract, such as the ileal and colonic mucosa and digesta, have also been well represented among publically available sequences (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig1\" rid-ob=\"ob-fig1\" co-legend-rid=\"lgnd_fig1\"><span>Fig.&#x000a0;1</span></a>). Conversely, the number of samples from the duodenum and jejunum (digesta and mucosa) was relatively small (<em>n</em> &#x02264; 5 samples) and belonged to a single study. Therefore, these samples were not included in the linear discriminant analysis (LDA) effect size (LEfSe) described below. Sequences were also available from three different continents and 10 countries, although approximately half of these were from studies conducted in the United States. The age of the pigs used in each study ranged from~3&#x000a0;weeks (preweaning piglets) to ~24&#x000a0;weeks (slaughter age).</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"tab1\"><h3>TABLE&#x000a0;1&#x000a0;</h3><!--caption a7--><span class=\"caption\"><p id=\"__p9\">Details of studies included in the meta-analysis</p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140162571680672\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><thead><tr><th rowspan=\"1\" colspan=\"1\">Reference</th><th rowspan=\"1\" colspan=\"1\">No. of samples</th><th rowspan=\"1\" colspan=\"1\">Hypervariable region(s)</th><th rowspan=\"1\" colspan=\"1\">Sequencing platform</th><th rowspan=\"1\" colspan=\"1\">Avg % assigned sequences per sample</th><th rowspan=\"1\" colspan=\"1\">DNA extraction method</th><th rowspan=\"1\" colspan=\"1\">Sample type(s)</th><th rowspan=\"1\" colspan=\"1\">Study observation(s)</th><th rowspan=\"1\" colspan=\"1\">Age of pigs at sampling</th><th rowspan=\"1\" colspan=\"1\">Country(ies) of origin</th><th rowspan=\"1\" colspan=\"1\">Data availability (accession no.)</th></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">J. Trachsel, B. E. Bass, and H. K. Allen, unpublished</td><td rowspan=\"1\" colspan=\"1\">52</td><td rowspan=\"1\" colspan=\"1\">V1 to V3</td><td rowspan=\"1\" colspan=\"1\">Illumina MiSeq</td><td rowspan=\"1\" colspan=\"1\">76.2%</td><td rowspan=\"1\" colspan=\"1\">PowerSoil-htp 96-well soil DNA isolation kit</td><td rowspan=\"1\" colspan=\"1\">Ileum digesta, fecal</td><td rowspan=\"1\" colspan=\"1\">Subset of samples taken before heat stress treatment</td><td rowspan=\"1\" colspan=\"1\">7&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">United States</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA335425\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA335425</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B17\" rid=\"B17\" class=\" bibr popnode\">17</a></td><td rowspan=\"1\" colspan=\"1\">24</td><td rowspan=\"1\" colspan=\"1\">V1 to V2</td><td rowspan=\"1\" colspan=\"1\">Illumina MiSeq</td><td rowspan=\"1\" colspan=\"1\">88.9%</td><td rowspan=\"1\" colspan=\"1\">NA<sup>b</sup></td><td rowspan=\"1\" colspan=\"1\">Ileum digesta</td><td rowspan=\"1\" colspan=\"1\">Pigs challenged with <em>E.&#x000a0;coli</em> F4+ and fed IgG had a lower relative abundance of <em>Enterococcaceae</em> and <em>Streptococcaceae</em> and increased ileal diversity</td><td rowspan=\"1\" colspan=\"1\">6&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">Denmark</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA302730\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA302730</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B15\" rid=\"B15\" class=\" bibr popnode\">15</a><sup>a</sup></td><td rowspan=\"1\" colspan=\"1\">18</td><td rowspan=\"1\" colspan=\"1\">V4</td><td rowspan=\"1\" colspan=\"1\">Illumina MiSeq</td><td rowspan=\"1\" colspan=\"1\">75.4%</td><td rowspan=\"1\" colspan=\"1\">QIAamp DNA stool minikit</td><td rowspan=\"1\" colspan=\"1\">Colon digesta</td><td rowspan=\"1\" colspan=\"1\">Pigs fed DDGS<sup>c</sup> had a different microbial community structure</td><td rowspan=\"1\" colspan=\"1\">9&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">United States</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJEB11368/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJEB11368</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B45\" rid=\"B45\" class=\" bibr popnode\">45</a><sup>a</sup></td><td rowspan=\"1\" colspan=\"1\">40</td><td rowspan=\"1\" colspan=\"1\">V3 to V4</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">74.1%</td><td rowspan=\"1\" colspan=\"1\">QIAamp DNA stool minikit</td><td rowspan=\"1\" colspan=\"1\">Colon digesta, fecal</td><td rowspan=\"1\" colspan=\"1\">No differences in the gut microbiota between conventionally and organically raised pigs</td><td rowspan=\"1\" colspan=\"1\">24&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">Denmark, France, Italy, Sweden</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA266576\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA266576</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B66\" rid=\"B66\" class=\" bibr popnode\">66</a></td><td rowspan=\"1\" colspan=\"1\">39</td><td rowspan=\"1\" colspan=\"1\">V4</td><td rowspan=\"1\" colspan=\"1\">Illumina MiSeq</td><td rowspan=\"1\" colspan=\"1\">85.2%</td><td rowspan=\"1\" colspan=\"1\">PowerFecal DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Fecal</td><td rowspan=\"1\" colspan=\"1\">Difference between the fecal microbiota of black and white breed pigs</td><td rowspan=\"1\" colspan=\"1\">3&#x02013;12&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">South Korea</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA252973\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA252973</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B67\" rid=\"B67\" class=\" bibr popnode\">67</a></td><td rowspan=\"1\" colspan=\"1\">53</td><td rowspan=\"1\" colspan=\"1\">V1 to V3</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">83.3%</td><td rowspan=\"1\" colspan=\"1\">QIAamp DNA stool minikit</td><td rowspan=\"1\" colspan=\"1\">Colon, ileal digesta</td><td rowspan=\"1\" colspan=\"1\">Sow-reared and formula-fed piglets had a different gut microbiota</td><td rowspan=\"1\" colspan=\"1\">3&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">United States</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA266806\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA266806</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B13\" rid=\"B13\" class=\" bibr popnode\">13</a></td><td rowspan=\"1\" colspan=\"1\">20</td><td rowspan=\"1\" colspan=\"1\">V4</td><td rowspan=\"1\" colspan=\"1\">Illumina MiSeq</td><td rowspan=\"1\" colspan=\"1\">75.8%</td><td rowspan=\"1\" colspan=\"1\">Custom</td><td rowspan=\"1\" colspan=\"1\">Ileum, cecum digesta</td><td rowspan=\"1\" colspan=\"1\">No effect of high or low amylose/amylopectin ratio diets on the gut microbiota</td><td rowspan=\"1\" colspan=\"1\">21&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">China</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA281781\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA281781</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B68\" rid=\"B68\" class=\" bibr popnode\">68</a></td><td rowspan=\"1\" colspan=\"1\">19</td><td rowspan=\"1\" colspan=\"1\">V4</td><td rowspan=\"1\" colspan=\"1\">Illumina MiSeq</td><td rowspan=\"1\" colspan=\"1\">81.3%</td><td rowspan=\"1\" colspan=\"1\">PowerFecal DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Fecal</td><td rowspan=\"1\" colspan=\"1\">Differences in fecal microbiota by production phase</td><td rowspan=\"1\" colspan=\"1\">Piglets, growers, finishers, sows</td><td rowspan=\"1\" colspan=\"1\">South Korea</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA267233\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA267233</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B16\" rid=\"B16\" class=\" bibr popnode\">16</a></td><td rowspan=\"1\" colspan=\"1\">14</td><td rowspan=\"1\" colspan=\"1\">V3 to V5</td><td rowspan=\"1\" colspan=\"1\">Illumina MiSeq</td><td rowspan=\"1\" colspan=\"1\">73.2%</td><td rowspan=\"1\" colspan=\"1\">PowerSoil DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Cecum digesta</td><td rowspan=\"1\" colspan=\"1\">Minor differences in the cecal microbiota of pigs fed 70% enzymatically modified cornstarch or control starch</td><td rowspan=\"1\" colspan=\"1\">14&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">Austria</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJEB8696/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJEB8696</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B11\" rid=\"B11\" class=\" bibr popnode\">11</a></td><td rowspan=\"1\" colspan=\"1\">62</td><td rowspan=\"1\" colspan=\"1\">V4</td><td rowspan=\"1\" colspan=\"1\">Illumina MiSeq</td><td rowspan=\"1\" colspan=\"1\">80.6%</td><td rowspan=\"1\" colspan=\"1\">PowerFecal DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Fecal</td><td rowspan=\"1\" colspan=\"1\">No effect of diet with 0.2% chlortetracycline, sulfathiazole, and penicillin on the fecal microbiota</td><td rowspan=\"1\" colspan=\"1\">3&#x02013;12&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">South Korea</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA257009\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA257009</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B10\" rid=\"B10\" class=\" bibr popnode\">10</a></td><td rowspan=\"1\" colspan=\"1\">94</td><td rowspan=\"1\" colspan=\"1\">V4</td><td rowspan=\"1\" colspan=\"1\">Illumina MiSeq</td><td rowspan=\"1\" colspan=\"1\">51.9%</td><td rowspan=\"1\" colspan=\"1\">ZR fecal DNA MiniPrep (Zymo Research)</td><td rowspan=\"1\" colspan=\"1\">Fecal</td><td rowspan=\"1\" colspan=\"1\">Subtherapeutic tylosin, but not chlortetracycline, altered the fecal microbiota</td><td rowspan=\"1\" colspan=\"1\">3&#x02013;19&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">Canada</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA244928\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA244928</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B12\" rid=\"B12\" class=\" bibr popnode\">12</a></td><td rowspan=\"1\" colspan=\"1\">44</td><td rowspan=\"1\" colspan=\"1\">V1 to V3</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">87.6%</td><td rowspan=\"1\" colspan=\"1\">PowerSoil DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Ileal mucosa</td><td rowspan=\"1\" colspan=\"1\">No effect of in-feed chlortetracycline or high-complexity diets on the ileal mucosa microbiota</td><td rowspan=\"1\" colspan=\"1\">6&#x02013;11&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">Canada</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA257705\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA257705</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B7\" rid=\"B7\" class=\" bibr popnode\">7</a></td><td rowspan=\"1\" colspan=\"1\">80</td><td rowspan=\"1\" colspan=\"1\">V1 to V3</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">62.8%</td><td rowspan=\"1\" colspan=\"1\">Ultra-Clean soil DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Gastric mucosa, duodenum, jejunum, ileum cecum, colon digesta and mucosa, fecal</td><td rowspan=\"1\" colspan=\"1\">Reduced diversity in the ileum versus colon, ASP250-altered gut microbiota</td><td rowspan=\"1\" colspan=\"1\">15&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">United States</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA72355\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA72355</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B8\" rid=\"B8\" class=\" bibr popnode\">8</a></td><td rowspan=\"1\" colspan=\"1\">101</td><td rowspan=\"1\" colspan=\"1\">V1 to V3</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">54.1%</td><td rowspan=\"1\" colspan=\"1\">PowerSoil DNA isolation kit</td><td rowspan=\"1\" colspan=\"1\">Fecal</td><td rowspan=\"1\" colspan=\"1\">Carbadox resulted in early but not late alterations of the fecal microbiota</td><td rowspan=\"1\" colspan=\"1\">6&#x02013;15&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">United States</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/237795\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA237795</a>; <a href=\"https://www.ncbi.nlm.nih.gov/biosample/SAMN02645017\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">SAMN02645017</a> to <a href=\"https://www.ncbi.nlm.nih.gov/biosample/SAMN02645066\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">SAMN02645066</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B69\" rid=\"B69\" class=\" bibr popnode\">69</a></td><td rowspan=\"1\" colspan=\"1\">23</td><td rowspan=\"1\" colspan=\"1\">V1 to V3</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">47.3%</td><td rowspan=\"1\" colspan=\"1\">Ultra-Clean Fecal DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Fecal</td><td rowspan=\"1\" colspan=\"1\">The fecal microbiota of Landrace pigs differed that of from pigs of the Duroc breed</td><td rowspan=\"1\" colspan=\"1\">15&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">South Korea</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA263439\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA263439</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B14\" rid=\"B14\" class=\" bibr popnode\">14</a></td><td rowspan=\"1\" colspan=\"1\">92</td><td rowspan=\"1\" colspan=\"1\">V1 to V2</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">83.2%</td><td rowspan=\"1\" colspan=\"1\">PowerSoil DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Gastric, ileum, colon mucosa</td><td rowspan=\"1\" colspan=\"1\">Gut microbiota affected by Ca-P content and diet type</td><td rowspan=\"1\" colspan=\"1\">7&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">Austria</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJEB1576/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJEB1576</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B70\" rid=\"B70\" class=\" bibr popnode\">70</a></td><td rowspan=\"1\" colspan=\"1\">10</td><td rowspan=\"1\" colspan=\"1\">V3</td><td rowspan=\"1\" colspan=\"1\">Illumina HiSeq 2000</td><td rowspan=\"1\" colspan=\"1\">88.8%</td><td rowspan=\"1\" colspan=\"1\">QIAamp DNA stool minikit (Qiagen)</td><td rowspan=\"1\" colspan=\"1\">Cecum digesta</td><td rowspan=\"1\" colspan=\"1\">Fostering of piglets changes cecal microbiota</td><td rowspan=\"1\" colspan=\"1\">3&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">China</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA214736\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA214736</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B18\" rid=\"B18\" class=\" bibr popnode\">18</a></td><td rowspan=\"1\" colspan=\"1\">59</td><td rowspan=\"1\" colspan=\"1\">V1 to V3</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">80.7%</td><td rowspan=\"1\" colspan=\"1\">PowerSoil DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Fecal</td><td rowspan=\"1\" colspan=\"1\">Fecal microbiota differed between high- and low-<em>Salmonella</em>-shedder pigs</td><td rowspan=\"1\" colspan=\"1\">8&#x02013;11&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">United States</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA169497\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA169497</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B9\" rid=\"B9\" class=\" bibr popnode\">9</a></td><td rowspan=\"1\" colspan=\"1\">12</td><td rowspan=\"1\" colspan=\"1\">V3</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">67.5%</td><td rowspan=\"1\" colspan=\"1\">Powermax soil DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Fecal</td><td rowspan=\"1\" colspan=\"1\">In-feed ASP250 altered fecal microbiota</td><td rowspan=\"1\" colspan=\"1\">18&#x02013;20&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">United States</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA72355\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA72355</a>; <a href=\"https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP004660\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">SRP004660</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B6\" rid=\"B6\" class=\" bibr popnode\">6</a></td><td rowspan=\"1\" colspan=\"1\">83</td><td rowspan=\"1\" colspan=\"1\">V1 to V3</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">44.4%</td><td rowspan=\"1\" colspan=\"1\">Custom</td><td rowspan=\"1\" colspan=\"1\">Fecal</td><td rowspan=\"1\" colspan=\"1\">In-feed ASP250 altered fecal microbiota</td><td rowspan=\"1\" colspan=\"1\">3&#x02013;13&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">United States</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA72355\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA72355</a>; <a href=\"https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP008126\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">SRP008126</a></td></tr></tbody></table></span><span id=\"largeobj_idm140162571680672\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab1/?report=objectonly\">Open in a separate window</a></span><span class=\"tblwrap-foot\"><span id=\"ngtab1.2\"><sup>a</sup>Data are from studies that describe using commercial pigs.</span><span id=\"ngtab1.1\"><sup>b</sup>NA, not available.</span><span id=\"ngtab1.3\"><sup>c</sup>DDGS, distillers&#x02019; dried grains with solubles.</span></span></span><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"fig1\" co-legend-rid=\"lgnd_fig1\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig1/\" target=\"figure\" rid-figpopup=\"fig1\" rid-ob=\"ob-fig1\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140162575750784\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5443231_sys0031721030001.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is sys0031721030001.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5443231/bin/sys0031721030001.jpg\" /></a></span><span id=\"largeobj_idm140162575750784\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig1/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_fig1\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig1/\" target=\"figure\" rid-figpopup=\"fig1\" rid-ob=\"ob-fig1\">FIG&#x000a0;1&#x000a0;</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p13\">Diagram of the swine gastrointestinal tract with major sections indicated as well as direction of movement of digesta in the colon. Original collection sites are labeled on the drawing.</p></span></span></span></span><span id=\"s1.2\" class=\"sec\"><p></p><h3 id=\"s1.2title\" class=\"inline\">Identifying the core microbiota of the swine gastrointestinal tract. </h3><p id=\"__p14\" class=\"p p-first\">The idea of defining a core microbiota in the pig gut is intriguing as it may identify potential targets for dietary or therapeutic interventions in the swine production environment. Although a core microbiota may not exist in swine according to a strict definition that requires a particular taxon to be present in each sample, several phyla and genera were found in more than 90% of all GI samples. In the current meta-analysis, the <em>Firmicutes</em> and <em>Bacteroidetes</em> phyla accounted for nearly 85% of the total 16S rRNA gene sequences among all gastrointestinal locations, with <em>Proteobacteria</em> being the only other phylum common to all GI samples (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\" co-legend-rid=\"lgnd_fig2\"><span>Fig.&#x000a0;2A</span></a>). No OTUs or genera were shared among all samples analyzed, but the genera <em>Clostridium</em>, <em>Blautia</em>, <em>Lactobacillus</em>, <em>Prevotella</em>, <em>Ruminococcus</em>, and <em>Roseburia</em>, the RC9 gut group, and <em>Subdoligranulum</em> were found in more than 90% of samples. <em>Prevotella</em> was also the most abundant genus among all genera that were identified (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\" co-legend-rid=\"lgnd_fig2\"><span>Fig.&#x000a0;2B</span></a>; see also <a href=\"#tabS1\" rid=\"tabS1\" class=\" supplementary-material\">Table&#x000a0;S1</a>&#x000a0;in the supplemental material). In addition, a single OTU classified as an uncultured member of the <em>Prevotella</em> was present in at least 75% of all GI samples.</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"fig2\" co-legend-rid=\"lgnd_fig2\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig2/\" target=\"figure\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140162577110112\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5443231_sys0031721030002.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is sys0031721030002.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5443231/bin/sys0031721030002.jpg\" /></a></span><span id=\"largeobj_idm140162577110112\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig2/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_fig2\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig2/\" target=\"figure\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\">FIG&#x000a0;2&#x000a0;</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p16\">Percent relative abundances of the three most abundant phyla (A) and 20 most abundant genera (B) by gastrointestinal tract sample type.</p></span></span></span><span class=\"sec suppmat\" id=\"tabS1\"><h4>TABLE&#x000a0;S1&#x000a0;</h4><p id=\"__p15\">The percentage of samples from each specific gastrointestinal location that had at least one 16S rRNA gene sequence from each of the individual genera identified. Genera are listed in descending order of overall relative abundance. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103st4.pdf\" id=\"tabs1\">TABLE&#x000a0;S1, PDF file, 1.1 MB</a>.</p><span class=\"small permissions half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al.<span class=\"license half_rhythm\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span><p id=\"__p17\">Gut locations and feces yielded discrete core microbiotas. In the fecal microbiota, more than 99% of fecal samples (12 studies) contained sequences from <em>Prevotella</em>, <em>Clostridium</em>, <em>Alloprevotella</em>, <em>Ruminococcus</em>, and the RC9 gut group. Certain genera were present in all samples from a particular GI tract location (<em>n</em> = &#x0003e;50 samples). These included <em>Prevotella</em> and <em>Blautia</em> in samples of colonic digesta (<em>n =</em> 91; 5 studies); <em>Anaerovibrio</em>, <em>Clostridium</em>, <em>Phascolarctobacterium</em>, <em>Ruminococcus</em>, <em>Sarcina</em>, and <em>Streptococcus</em> in cecal digesta (<em>n =</em> 52; 4 studies); and <em>Clostridium</em> in ileal mucosal samples (<em>n =</em> 91; 3 studies). <em>Clostridium</em> spp. were also present in all but one sample of ileal digesta (<em>n =</em> 81; 5 studies).</p><p id=\"__p18\">In commercial swine production, pigs are typically fed a cereal grain-based diet that is relatively high in carbohydrate content (<a href=\"#B20\" rid=\"B20\" class=\" bibr popnode\">20</a>). Meanwhile, <em>Prevotella</em> spp. have been reported to be associated with dietary carbohydrates in humans (<a href=\"#B21\" rid=\"B21\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776891\">21</a>, <a href=\"#B22\" rid=\"B22\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776885\">22</a>), producing polysaccharidases such as glucanase, mannanase, and xylanase (<a href=\"#B23\" rid=\"B23\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776862\">23</a>). Members of <em>Prevotella</em> are also relatively abundant in the rumen and/or gut microbiota of other animals such as cattle (<a href=\"#B24\" rid=\"B24\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776892\">24</a>), chimpanzees (<a href=\"#B25\" rid=\"B25\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776871\">25</a>), goats (<a href=\"#B26\" rid=\"B26\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776886\">26</a>), and sheep (<a href=\"#B27\" rid=\"B27\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776898\">27</a>), attesting to the ubiquity of this genus in the mammalian gut. <em>Clostridium</em>, <em>Blautia</em>, and <em>Ruminococcus</em> are all members of the <em>Clostridiales</em> order, and, similarly to <em>Prevotella</em>, are widely found in the mammalian gut (<a href=\"#B28\" rid=\"B28\" class=\" bibr popnode\">28</a>). These genera can produce butyrate, a short-chain fatty acid (SCFA), most often from acetate (also an SCFA) via the butyryl-coenzyme A (CoA):acetate CoA-transferase pathway (<a href=\"#B29\" rid=\"B29\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776878\">29</a>). <em>Prevotella</em> spp., which produce acetate in the gut, are thereby able to provide a source of energy for butyrate-producing bacteria (<a href=\"#B8\" rid=\"B8\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776880\">8</a>). Importantly, butyrate decreases inflammation in the gut of the host, and cells in the intestinal epithelium can use it as an energy source (<a href=\"#B30\" rid=\"B30\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776861\">30</a>).</p><p id=\"__p19\" class=\"p p-last\">The RC9 gut group, which belongs to the <em>Rikenellaceae</em> family, was identified in 90.5% of all GI samples and was among the most abundant genera in the lower GI tract (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\" co-legend-rid=\"lgnd_fig2\"><span>Fig.&#x000a0;2B</span></a>; <a href=\"#tabS1\" rid=\"tabS1\" class=\" supplementary-material\">Table&#x000a0;S1</a>). This finding is noteworthy as its presence in the swine gut microbiota was reported only recently (<a href=\"#B31\" rid=\"B31\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776866\">31</a>, <a href=\"#B32\" rid=\"B32\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776887\">32</a>). The <em>Rikenellaceae</em> family itself is a relatively new taxonomic classification within the <em>Bacteroidales</em> order, with only three genera currently described (<a href=\"#B33\" rid=\"B33\" class=\" bibr popnode\">33</a>). Similarly to <em>Prevotella</em>, also a member of the <em>Bacteroidales</em>, some bacteria in the <em>Rikenellaceae</em> family are acetate producers (<a href=\"#B34\" rid=\"B34\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776904\">34</a>), suggesting a high level of functional redundancy for this metabolite in the swine gut microbiome.</p></span><span id=\"s1.3\" class=\"sec\"><p></p><h3 id=\"s1.3title\" class=\"inline\">Factors affecting the swine gut microbiota. </h3><p id=\"__p20\" class=\"p p-first\">We determined which variables most strongly affected the structure of the swine gut microbiota using a permutational multivariate analysis of variance (PERMANOVA) test of the unweighted and weighted UniFrac distances and Bray-Curtis dissimilarities (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab2/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"tab2\" rid-ob=\"ob-tab2\" co-legend-rid=\"\"><span>Table&#x000a0;2</span></a>). Among all distance metrics, each metadata category tested was significantly associated with the microbial community structure. However, this finding is likely a result of the heterogeneity of dispersion within each group in each metadata category, as shown by the permutational analysis of multivariate dispersions (PERMDISP; <em>P</em> &#x0003c; 0.05). Overall, &#x0201c;study&#x0201d; was the most important factor affecting the swine gut microbiota, as it explained the greatest variation in the data set (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3</span></a>; also see <a href=\"#figS1\" rid=\"figS1\" class=\" supplementary-material\">Fig.&#x000a0;S1</a>&#x000a0;in the supplemental material). The GI location that was sampled was also strongly associated with the structure of the microbiota as assessed using weighted UniFrac distances (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3A</span></a>); however, this finding was not replicated in the results obtained with respect to the unweighted UniFrac distances and Bray-Curtis dissimilarities (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3B</span></a> and <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span style=\"position: relative;text-decoration:none;\">&#x200B;<span class=\"figpopup-sensitive-area\" style=\"left: -1.5em;\">andC).</span></span><span>C</span></a>). Therefore, researchers could reasonably expect that the taxa that are relatively abundant in their swine gut samples are similar to those in samples taken from the same GI location of different pigs compared with those found in samples obtained from different GI locations.</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"tab2\"><h3>TABLE&#x000a0;2&#x000a0;</h3><!--caption a7--><span class=\"caption\"><p id=\"__p22\">Factors associated with the community structure of the swine gut microbiota as measured using PERMANOVA with the adonis function (9,999 permutations) of the weighted and unweighted UniFrac distances and Bray-Curtis dissimilarities<sup>a</sup></p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140162584427712\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><thead><tr><th rowspan=\"3\" colspan=\"1\">Parameter</th><th colspan=\"9\" rowspan=\"1\">Value<hr /></th></tr><tr><th colspan=\"3\" rowspan=\"1\">Weighted UniFrac<hr /></th><th colspan=\"3\" rowspan=\"1\">Unweighted UniFrac<hr /></th><th colspan=\"3\" rowspan=\"1\">Bray-Curtis<hr /></th></tr><tr><th rowspan=\"1\" colspan=\"1\">Pseudo-F ratio</th><th rowspan=\"1\" colspan=\"1\"><em>R</em><sup>2</sup></th><th rowspan=\"1\" colspan=\"1\"><em>P</em> value</th><th rowspan=\"1\" colspan=\"1\">Pseudo-F ratio</th><th rowspan=\"1\" colspan=\"1\"><em>R</em><sup>2</sup></th><th rowspan=\"1\" colspan=\"1\"><em>P</em> value</th><th rowspan=\"1\" colspan=\"1\">Pseudo-F ratio</th><th rowspan=\"1\" colspan=\"1\"><em>R</em><sup>2</sup></th><th rowspan=\"1\" colspan=\"1\"><em>P</em> value</th></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">Study</td><td rowspan=\"1\" colspan=\"1\">28.2</td><td rowspan=\"1\" colspan=\"1\">0.37</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">18.6</td><td rowspan=\"1\" colspan=\"1\">0.28</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">24.4</td><td rowspan=\"1\" colspan=\"1\">0.34</td><td rowspan=\"1\" colspan=\"1\">0.0001</td></tr><tr><td rowspan=\"1\" colspan=\"1\">GI sampling location</td><td rowspan=\"1\" colspan=\"1\">35.4</td><td rowspan=\"1\" colspan=\"1\">0.29</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">12.9</td><td rowspan=\"1\" colspan=\"1\">0.13</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">12.1</td><td rowspan=\"1\" colspan=\"1\">0.13</td><td rowspan=\"1\" colspan=\"1\">0.0001</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Age</td><td rowspan=\"1\" colspan=\"1\">13.9</td><td rowspan=\"1\" colspan=\"1\">0.21</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">8.5</td><td rowspan=\"1\" colspan=\"1\">0.14</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">10.6</td><td rowspan=\"1\" colspan=\"1\">0.16</td><td rowspan=\"1\" colspan=\"1\">0.0001</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Country of origin</td><td rowspan=\"1\" colspan=\"1\">25.4</td><td rowspan=\"1\" colspan=\"1\">0.18</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">15.6</td><td rowspan=\"1\" colspan=\"1\">0.12</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">20.4</td><td rowspan=\"1\" colspan=\"1\">0.15</td><td rowspan=\"1\" colspan=\"1\">0.0001</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Hypervariable region sequenced</td><td rowspan=\"1\" colspan=\"1\">34.5</td><td rowspan=\"1\" colspan=\"1\">0.16</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">22.7</td><td rowspan=\"1\" colspan=\"1\">0.11</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">18.9</td><td rowspan=\"1\" colspan=\"1\">0.13</td><td rowspan=\"1\" colspan=\"1\">0.0001</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Sequencing platform</td><td rowspan=\"1\" colspan=\"1\">20.4</td><td rowspan=\"1\" colspan=\"1\">0.04</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">19.0</td><td rowspan=\"1\" colspan=\"1\">0.04</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">25.6</td><td rowspan=\"1\" colspan=\"1\">0.05</td><td rowspan=\"1\" colspan=\"1\">0.0001</td></tr></tbody></table></span><span id=\"largeobj_idm140162584427712\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab2/?report=objectonly\">Open in a separate window</a></span><span class=\"tblwrap-foot\"><span id=\"ngtab2.1\"><sup>a</sup>For the age category, one study was excluded due to a lack of information about the age of the pigs used. PERMANOVA, permutational multivariate analysis of variance.</span></span></span><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"fig3\" co-legend-rid=\"lgnd_fig3\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig3/\" target=\"figure\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140162575154608\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5443231_sys0031721030003.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is sys0031721030003.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5443231/bin/sys0031721030003.jpg\" /></a></span><span id=\"largeobj_idm140162575154608\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig3/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_fig3\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig3/\" target=\"figure\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\">FIG&#x000a0;3&#x000a0;</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p24\">Principal-coordinate analysis plots of weighted UniFrac distances (A), unweighted UniFrac distances (B), and Bray-Curtis dissimilarities (C) classified by gastrointestinal tract sample type. The percentages of variation explained by the principal coordinates are indicated on the axes.</p></span></span></span><span class=\"sec suppmat\" id=\"figS1\"><h4>FIG&#x000a0;S1&#x000a0;</h4><p id=\"__p21\">Principal-coordinate analysis plots of Bray-Curtis dissimilarities by study. The numbers in parentheses in the figure legend refer to the number of samples included for each study. The percentages of variation explained by the principal coordinates are indicated on the axes. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103sf1.tif\" id=\"figs1\">FIG&#x000a0;S1, TIF file, 2.1 MB</a>.</p><span class=\"small permissions half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al.<span class=\"license half_rhythm\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span><p id=\"__p25\">We evaluated the effect of age on the swine gut microbiota using samples from the 19 studies (of 20) that provided specific information for pig age (<em>n =</em> 920; <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab2/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"tab2\" rid-ob=\"ob-tab2\" co-legend-rid=\"\"><span>Table&#x000a0;2</span></a>; <a href=\"#figS2\" rid=\"figS2\" class=\" supplementary-material\">Fig.&#x000a0;S2</a>). Although the effect of the age of the pigs sampled was not as large as the effects of study and GI sampling location, it did explain more variation in the data set than the remaining three metadata categories for the weighted UniFrac distances. It should be noted that pig age, like several of the other metadata categories, is also associated with &#x0201c;study,&#x0201d; as many studies sampled from pigs at only one particular time point. In addition, the 3-week-old category included both preweaned and postweaned pigs, and weaning is known to result in significant changes to the swine gut microbiota (<a href=\"#B10\" rid=\"B10\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776850\">10</a>, <a href=\"#B35\" rid=\"B35\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776854\">35</a>).</p><span class=\"sec suppmat\" id=\"figS2\"><h4>FIG&#x000a0;S2&#x000a0;</h4><p id=\"__p26\">Principal-coordinate analysis plot of the weighted UniFrac distances for all samples with associated metadata for the age of the pigs used in the study. Samples are colored by pig age in weeks. The numbers in parentheses for the figure legend refer to the number of samples at each week. The percentages of variation explained by the principal coordinates are indicated on the axes. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103sf2.tif\" id=\"figs2\">FIG&#x000a0;S2, TIF file, 2.5 MB</a>.</p><span class=\"small permissions half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al.<span class=\"license half_rhythm\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span><p id=\"__p27\">Genera that were enriched in specific GI samples were identified using linear discriminant analysis (LDA) effect size (LEfSe) (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig4\" rid-ob=\"ob-fig4\" co-legend-rid=\"lgnd_fig4\"><span>Fig.&#x000a0;4</span></a>). Among the more notable genera, <em>Lactobacillus</em> was more abundant in the gastric mucosa, <em>Prevotella</em>, <em>Helicobacter</em>, and <em>Campylobacter</em> in the colonic mucosa, <em>Clostridium</em> in the ileal mucosa, <em>Alloprevotella</em> in the cecal mucosa, <em>Bacteroides</em> in the colon, and <em>Treponema</em> in fecal samples. These differences in the relative abundances of certain genera along the gastrointestinal tract likely reflect the physiological conditions at each section. For example, the pH is significantly lower in the stomach than at any other GI location and <em>Lactobacillus</em> spp. are relatively acid tolerant, possibly explaining their enrichment in this location (<a href=\"#B36\" rid=\"B36\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776896\">36</a>). The lactobacilli are also able to attach to the epithelial and mucosal layers, where they can form biofilm-like communities, thereby aiding their persistence (<a href=\"#B37\" rid=\"B37\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776889\">37</a>). <em>Lactobacillus</em> spp. were present among all types of GI samples; therefore, the population of lactobacilli in the stomach may serve as a source of these bacteria for other gut locations (<a href=\"#B38\" rid=\"B38\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776888\">38</a>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"fig4\" co-legend-rid=\"lgnd_fig4\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig4/\" target=\"figure\" rid-figpopup=\"fig4\" rid-ob=\"ob-fig4\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140162573451328\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5443231_sys0031721030004.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is sys0031721030004.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5443231/bin/sys0031721030004.jpg\" /></a></span><span id=\"largeobj_idm140162573451328\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig4/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_fig4\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig4/\" target=\"figure\" rid-figpopup=\"fig4\" rid-ob=\"ob-fig4\">FIG&#x000a0;4&#x000a0;</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p28\">Differentially abundant genera in each gastrointestinal tract sample type as assessed using linear discriminant analysis (LDA) with effect size (LEfSe) measurements. Only those genera with an LDA score (log<sub>10</sub>) of &#x0003e;4.0 are displayed. Samples of duodenum and jejunum mucosa and digesta were excluded from analysis as there were fewer than five samples for each.</p></span></span></span><p id=\"__p29\">Comparisons of alpha-diversity measures also revealed significant differences among samples from different GI locations, including feces (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab3/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"tab3\" rid-ob=\"ob-tab3\" co-legend-rid=\"\"><span>Table&#x000a0;3</span></a>). In particular, there was a clear demarcation between upper and lower GI samples, with significantly higher diversity, richness, and evenness in the lower GI samples (<em>P</em> &#x0003c; 0.05). Within the lower GI tract, fecal and cecal mucosa samples were the most rich in terms of the number of OTUs and phylogenetic diversity (PD). These observations are similar to those reported in the longitudinal swine gut microbiota study conducted by Looft et al. (<a href=\"#B7\" rid=\"B7\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776848\">7</a>). The only significant difference among upper GI tract sections was observed between the samples of ileal digesta and mucosa, where the mucosal samples had greater phylogenetic diversity as determined using PD whole tree. These results are expected, given that transit is faster in the upper GI tract than in the lower GI tract, thereby limiting adhesion and colonization of bacteria in the upper GI tract (<a href=\"#B39\" rid=\"B39\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776883\">39</a>). Also, competition for nutrients between the host and bacteria is greater in the upper GI tract (<a href=\"#B40\" rid=\"B40\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776847\">40</a>).</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"tab3\"><h3>TABLE&#x000a0;3&#x000a0;</h3><!--caption a7--><span class=\"caption\"><p id=\"__p30\">Alpha-diversity measures for each gastrointestinal location<sup>a</sup></p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140162571636048\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><thead><tr><th rowspan=\"2\" colspan=\"1\">GI sample type</th><th colspan=\"5\" rowspan=\"1\">Values<hr /></th></tr><tr><th rowspan=\"1\" colspan=\"1\">No. of OTUs</th><th rowspan=\"1\" colspan=\"1\">Phylogenetic diversity</th><th rowspan=\"1\" colspan=\"1\">Shannon index</th><th rowspan=\"1\" colspan=\"1\">Simpson&#x02019;s reciprocal index</th><th rowspan=\"1\" colspan=\"1\">Equitability</th></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">Gastric mucosa (<em>n</em> = 36)</td><td rowspan=\"1\" colspan=\"1\">135 &#x000b1; 66c</td><td rowspan=\"1\" colspan=\"1\">26.9 &#x000b1; 12.8c</td><td rowspan=\"1\" colspan=\"1\">3.04 &#x000b1; 1.02d\n</td><td rowspan=\"1\" colspan=\"1\">12 &#x000b1; 10.2b</td><td rowspan=\"1\" colspan=\"1\">0.62 &#x000b1; 0.15cd</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Duodenum (<em>n</em> = 3)</td><td rowspan=\"1\" colspan=\"1\">92 &#x000b1; 13c</td><td rowspan=\"1\" colspan=\"1\">14 &#x000b1; 1.7d</td><td rowspan=\"1\" colspan=\"1\">1.87 &#x000b1; 0.98d\n</td><td rowspan=\"1\" colspan=\"1\">3.7 &#x000b1; 3.7b</td><td rowspan=\"1\" colspan=\"1\">0.41 &#x000b1; 0.21d</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Duodenal mucosa (<em>n</em> = 2)</td><td rowspan=\"1\" colspan=\"1\">120 &#x000b1; 0c</td><td rowspan=\"1\" colspan=\"1\">23.2 &#x000b1; 2.7cd</td><td rowspan=\"1\" colspan=\"1\">2.85 &#x000b1; 0.02d\n</td><td rowspan=\"1\" colspan=\"1\">6.4 &#x000b1; 0.3b</td><td rowspan=\"1\" colspan=\"1\">0.59 &#x000b1; 0.00cd</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Jejunum (<em>n</em> = 4)</td><td rowspan=\"1\" colspan=\"1\">118 &#x000b1; 16c</td><td rowspan=\"1\" colspan=\"1\">18.2 &#x000b1; 4.2cd</td><td rowspan=\"1\" colspan=\"1\">3.06 &#x000b1; 0.40cd</td><td rowspan=\"1\" colspan=\"1\">9.4 &#x000b1; 4.3b</td><td rowspan=\"1\" colspan=\"1\">0.64 &#x000b1; 0.07bcd</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Jejunal mucosa (<em>n</em> = 3)</td><td rowspan=\"1\" colspan=\"1\">126 &#x000b1; 25c</td><td rowspan=\"1\" colspan=\"1\">21.4 &#x000b1; 7.2cd</td><td rowspan=\"1\" colspan=\"1\">3.22 &#x000b1; 0.27bcd</td><td rowspan=\"1\" colspan=\"1\">11.4 &#x000b1; 2.7b</td><td rowspan=\"1\" colspan=\"1\">0.67 &#x000b1; 0.03abc</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Ileum (<em>n</em> = 81)</td><td rowspan=\"1\" colspan=\"1\">94 &#x000b1; 42c</td><td rowspan=\"1\" colspan=\"1\">15.8 &#x000b1; 6.3d</td><td rowspan=\"1\" colspan=\"1\">2.72 &#x000b1; 0.89d</td><td rowspan=\"1\" colspan=\"1\">10.2 &#x000b1; 8.3b</td><td rowspan=\"1\" colspan=\"1\">0.60 &#x000b1; 0.14cd</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Ileal mucosa (<em>n</em> = 91)</td><td rowspan=\"1\" colspan=\"1\">128 &#x000b1; 54c</td><td rowspan=\"1\" colspan=\"1\">24.1 &#x000b1; 9.6c</td><td rowspan=\"1\" colspan=\"1\">3.02 &#x000b1; 0.86d</td><td rowspan=\"1\" colspan=\"1\">11.5 &#x000b1; 9.8b</td><td rowspan=\"1\" colspan=\"1\">0.62 &#x000b1; 0.14cd</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Cecum (<em>n</em> = 52)</td><td rowspan=\"1\" colspan=\"1\">213 &#x000b1; 81b</td><td rowspan=\"1\" colspan=\"1\">35.7 &#x000b1; 10.3b</td><td rowspan=\"1\" colspan=\"1\">4.02 &#x000b1; 0.83bc</td><td rowspan=\"1\" colspan=\"1\">30.3 &#x000b1; 24.5a</td><td rowspan=\"1\" colspan=\"1\">0.75 &#x000b1; 0.11b</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Cecal mucosa (<em>n</em> = 18)</td><td rowspan=\"1\" colspan=\"1\">282 &#x000b1; 51a</td><td rowspan=\"1\" colspan=\"1\">42.9 &#x000b1; 7.3ab</td><td rowspan=\"1\" colspan=\"1\">4.61 &#x000b1; 0.43a</td><td rowspan=\"1\" colspan=\"1\">42.8 &#x000b1; 18.5a</td><td rowspan=\"1\" colspan=\"1\">0.82 &#x000b1; 0.05a</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Colon (<em>n</em> = 91)</td><td rowspan=\"1\" colspan=\"1\">221 &#x000b1; 66b</td><td rowspan=\"1\" colspan=\"1\">37.7 &#x000b1; 9b</td><td rowspan=\"1\" colspan=\"1\">4.23 &#x000b1; 0.53ab</td><td rowspan=\"1\" colspan=\"1\">33.2 &#x000b1; 22.6a</td><td rowspan=\"1\" colspan=\"1\">0.79 &#x000b1; 0.06ab</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Colonic mucosa (<em>n</em> = 48)</td><td rowspan=\"1\" colspan=\"1\">210 &#x000b1; 90b</td><td rowspan=\"1\" colspan=\"1\">37.2 &#x000b1; 12b</td><td rowspan=\"1\" colspan=\"1\">4.05 &#x000b1; 0.74abc</td><td rowspan=\"1\" colspan=\"1\">30.9 &#x000b1; 26.9a</td><td rowspan=\"1\" colspan=\"1\">0.76 &#x000b1; 0.09ab</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Fecal (<em>n</em> = 510)</td><td rowspan=\"1\" colspan=\"1\">255 &#x000b1; 73a</td><td rowspan=\"1\" colspan=\"1\">44.1 &#x000b1; 11.7a</td><td rowspan=\"1\" colspan=\"1\">4.41 &#x000b1; 0.65a</td><td rowspan=\"1\" colspan=\"1\">40.6 &#x000b1; 29.2a</td><td rowspan=\"1\" colspan=\"1\">0.80 &#x000b1; 0.08a</td></tr></tbody></table></span><span id=\"largeobj_idm140162571636048\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab3/?report=objectonly\">Open in a separate window</a></span><span class=\"tblwrap-foot\"><span id=\"ngtab3.1\"><sup>a</sup>Different lowercase letters in each column indicate significant differences (<em>P</em> &#x0003c; 0.05). Data represent means &#x000b1; standard deviations.</span></span></span><p id=\"__p32\" class=\"p p-last\">Although it is reassuring that GI sampling location is a strong predictor of the swine gut microbial community structure, study-level effects are also significant, as evidenced above. The importance of technical variation among studies has been documented in recent meta-analyses of the avian gut microbiota (<a href=\"#B4\" rid=\"B4\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776860\">4</a>) and the indoor microbiota (<a href=\"#B5\" rid=\"B5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776857\">5</a>), where study was the most important driver of the microbiota. In the present meta-analysis, a wide range of ages were included for the pigs, as well as different diets, pig breeds, and administered treatments (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab1/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"tab1\" rid-ob=\"ob-tab1\" co-legend-rid=\"\"><span>Table&#x000a0;1</span></a>). In addition, at least seven different DNA extraction methods were used and six separate 16S rRNA gene hypervariable regions were sequenced on three different sequencing platforms. The use of standard or uniform DNA extraction, 16S rRNA gene library preparation, and sequencing protocols would likely help reduce some of this study-to-study variation. With that said, eliminating all sources of technical variation in a single study may prove difficult as even technical replicates from different sequencing centers can differ significantly (<a href=\"#B41\" rid=\"B41\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776881\">41</a>).</p></span><span id=\"s1.4\" class=\"sec\"><p></p><h3 id=\"s1.4title\" class=\"inline\">Effect of 16S rRNA gene hypervariable region used for sequencing. </h3><p id=\"__p33\" class=\"p p-first\">The choice of which 16S rRNA gene hypervariable region(s) to sequence is an important consideration for any 16S-based microbial community study as even so-called &#x0201c;universal primers&#x0201d; differ greatly in their amplification of certain bacterial groups (<a href=\"#B42\" rid=\"B42\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776900\">42</a>). To determine the effect of different 16S hypervariable region sequences on the characterization of the swine gut microbiota, we analyzed only fecal samples to eliminate one of the largest sources of variability among samples (i.e., the GI sample type). When unweighted and weighted UniFrac distances and Bray-Curtis dissimilarities were plotted using PCoA, samples clustered by which hypervariable region was sequenced, although the amount of variation explained was smaller for the weighted UniFrac distances and there was more overlap between samples sequenced with different hypervariable regions (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig5\" rid-ob=\"ob-fig5\" co-legend-rid=\"lgnd_fig5\"><span>Fig.&#x000a0;5</span></a>). In the fecal samples, the study effect was still stronger than the effect of the hypervariable region sequenced with PERMANOVA (adonis <em>R</em><sup>2</sup>&#x000a0;values of 0.31, 0.28, and 0.36 for weighted and unweighted UniFrac distances and Bray-Curtis dissimilarities, respectively). However, this is to be expected given that each study sequenced only one specific 16S rRNA gene hypervariable region. Fecal samples that were sequenced using primers that targeted the V4 region did have more phylogenetic diversity (PD whole tree) and a higher Simpson&#x02019;s reciprocal value than those that were sequenced using primers that targeted the other 16S hypervariable regions analyzed, although the richness and the Shannon index data were not significantly different (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab4/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"tab4\" rid-ob=\"ob-tab4\" co-legend-rid=\"\"><span>Table&#x000a0;4</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"fig5\" co-legend-rid=\"lgnd_fig5\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig5/\" target=\"figure\" rid-figpopup=\"fig5\" rid-ob=\"ob-fig5\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140162575363392\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5443231_sys0031721030005.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is sys0031721030005.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5443231/bin/sys0031721030005.jpg\" /></a></span><span id=\"largeobj_idm140162575363392\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig5/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_fig5\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig5/\" target=\"figure\" rid-figpopup=\"fig5\" rid-ob=\"ob-fig5\">FIG&#x000a0;5&#x000a0;</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p34\">Principal-coordinate analysis plots of weighted UniFrac distances (A), unweighted UniFrac distances (B), and Bray-Curtis dissimilarities (C) classified by hypervariable region sequenced for fecal samples only. Percentages of variation explained by the principal coordinates are indicated on the axes, and the <em>R</em>&#x000a0;values on each plot indicate the dissimilarities between the hypervariable region groups.</p></span></span></span><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"tab4\"><h3>TABLE&#x000a0;4&#x000a0;</h3><!--caption a7--><span class=\"caption\"><p id=\"__p35\">Alpha-diversity measures by hypervariable region sequenced for fecal samples only<sup>a</sup></p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140162573733824\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><thead><tr><th rowspan=\"2\" colspan=\"1\">Hypervariable region(s)</th><th colspan=\"5\" rowspan=\"1\">Value(s)<hr /></th></tr><tr><th rowspan=\"1\" colspan=\"1\">No. of OTUs</th><th rowspan=\"1\" colspan=\"1\">Phylogenetic diversity</th><th rowspan=\"1\" colspan=\"1\">Shannon index</th><th rowspan=\"1\" colspan=\"1\">Simpson&#x02019;s reciprocal index</th><th rowspan=\"1\" colspan=\"1\">Equitability</th></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">V1 to V3 (<em>n</em> = 270)</td><td rowspan=\"1\" colspan=\"1\">253 &#x000b1; 49</td><td rowspan=\"1\" colspan=\"1\">42.6 &#x000b1; 6.9a</td><td rowspan=\"1\" colspan=\"1\">4.46 &#x000b1; 0.44</td><td rowspan=\"1\" colspan=\"1\">37.6 &#x000b1; 23a</td><td rowspan=\"1\" colspan=\"1\">0.81 &#x000b1; 0.06a</td></tr><tr><td rowspan=\"1\" colspan=\"1\">V3 (<em>n</em> = 12)</td><td rowspan=\"1\" colspan=\"1\">232 &#x000b1; 35</td><td rowspan=\"1\" colspan=\"1\">42.1 &#x000b1; 5.7a</td><td rowspan=\"1\" colspan=\"1\">4.29 &#x000b1; 0.4</td><td rowspan=\"1\" colspan=\"1\">29 &#x000b1; 12.5a</td><td rowspan=\"1\" colspan=\"1\">0.79 &#x000b1; 0.06b</td></tr><tr><td rowspan=\"1\" colspan=\"1\">V3 and V4 (<em>n</em> = 14)</td><td rowspan=\"1\" colspan=\"1\">236 &#x000b1; 50</td><td rowspan=\"1\" colspan=\"1\">39.6 &#x000b1; 7.1a</td><td rowspan=\"1\" colspan=\"1\">4.26 &#x000b1; 0.69</td><td rowspan=\"1\" colspan=\"1\">30.3 &#x000b1; 17.5a</td><td rowspan=\"1\" colspan=\"1\">0.78 &#x000b1; 0.1b</td></tr><tr><td rowspan=\"1\" colspan=\"1\">V4 (<em>n</em> = 214)</td><td rowspan=\"1\" colspan=\"1\">259 &#x000b1; 97</td><td rowspan=\"1\" colspan=\"1\">46.3 &#x000b1; 15.9b</td><td rowspan=\"1\" colspan=\"1\">4.35 &#x000b1; 0.85</td><td rowspan=\"1\" colspan=\"1\">45.7 &#x000b1; 36.1b</td><td rowspan=\"1\" colspan=\"1\">0.79 &#x000b1; 0.1b</td></tr></tbody></table></span><span id=\"largeobj_idm140162573733824\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab4/?report=objectonly\">Open in a separate window</a></span><span class=\"tblwrap-foot\"><span id=\"ngtab4.1\"><sup>a</sup>Different lowercase letters in each column indicate significant differences. Data represent means &#x000b1; standard deviations.</span></span></span><p id=\"__p37\">Comparing the region V1 to V3 and region V4 sequences from fecal samples using LEfSe, the region V1 to V3 sequences contained a higher proportion of the <em>Firmicutes</em> phylum and <em>Blautia</em> genus (LDA score [log<sub>10</sub>] = &#x0003e;4.0). Sequences classified at the phylum level as <em>Spirochaetae</em> and at the genus level as <em>Bacteroides</em>, <em>Lactobacillus</em>, and <em>Treponema</em> were more abundant in fecal samples that were sequenced using the V4 region. Another notable difference at the genus level among the different hypervariable regions was the nearly complete absence of <em>Bifidobacterium</em> in fecal samples from studies using the region V1 to V3 and region V3 16S primer sets (1 sample of 288; <a href=\"#tabS2\" rid=\"tabS2\" class=\" supplementary-material\">Table&#x000a0;S2</a>). In contrast, 74.3% of fecal samples that were sequenced using the V4 hypervariable region and 93% of fecal samples using region V3 to V4 had at least one <em>Bifidobacterium</em> 16S rRNA gene sequence. The generic 27F primer sequence (5&#x02032;-AGAGTTTGATCCTGGCTCAG-3&#x02032;) which is often used to amplify regions V1 to V3 has been previously reported to underestimate the relative abundance of bifidobacteria unless modifications are made to the primer sequence (<a href=\"#B43\" rid=\"B43\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776863\">43</a>, <a href=\"#B44\" rid=\"B44\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776890\">44</a>). Although the relative abundance of <em>Bifidobacterium</em> in fecal samples remained low (&#x0003c;0.35%) in the present meta-analysis, this may be an important consideration for researchers interested in this particular genus.</p><span class=\"sec suppmat\" id=\"tabS2\"><h4>TABLE&#x000a0;S2&#x000a0;</h4><p id=\"__p38\">The percentage of fecal samples grouped by specific 16S rRNA gene hypervariable region sequenced that had at least one 16S rRNA gene sequence from each of the individual genera identified. Genera are listed in descending order of overall relative abundance. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103st5.pdf\" id=\"tabs2\">TABLE&#x000a0;S2, PDF file, 0.5 MB</a>.</p><span class=\"small permissions half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al.<span class=\"license half_rhythm\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span><p id=\"__p39\" class=\"p p-last\">Primers targeting the V4 region, however, poorly amplify sequences from <em>Spirochaeta</em> and <em>Desulfitibacter</em>, as these genera were identified in only 12.1% and 0.5% of fecal samples that were sequenced with V4 primers, compared to 91.5% and 77.8% of fecal samples analyzed with region V1 to V3 primers, respectively (<a href=\"#tabS2\" rid=\"tabS2\" class=\" supplementary-material\">Table&#x000a0;S2</a>). Although only 14 fecal samples from a single study were available for analysis of region V3 to V4 (<a href=\"#B45\" rid=\"B45\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776851\">45</a>), <em>Treponema</em> spp. were completely absent from these samples, an unexpected finding given the ubiquity of this genus among fecal samples sequenced using other 16S primers; therefore, those results may reflect the primer set used rather than the hypervariable region sequenced.</p></span><span id=\"s1.5\" class=\"sec\"><p></p><h3 id=\"s1.5title\" class=\"inline\">Studies including antimicrobial administration. </h3><p id=\"__p40\" class=\"p p-first\">Antimicrobials are frequently used in swine production for the purpose of growth promotion, disease prevention, or disease treatment (<a href=\"#B46\" rid=\"B46\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776877\">46</a>). However, their usage in pigs may help maintain a reservoir of antimicrobial-resistant bacteria and resistance determinants (<a href=\"#B47\" rid=\"B47\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776868\">47</a>, <a href=\"#B48\" rid=\"B48\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776853\">48</a>). The development of alternatives to antimicrobials in swine requires a better understanding of how the microbiota is affected by these agents (<a href=\"#B49\" rid=\"B49\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776852\">49</a>, <a href=\"#B50\" rid=\"B50\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776867\">50</a>). As 16S rRNA gene sequences were available from six separate studies that investigated the effect of antimicrobial use on the swine fecal microbiota, we combined these studies to determine if any pattern would emerge that was associated with the use of any type of antimicrobial agent in swine (<a href=\"#B6\" rid=\"B6\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776864\">6</a>,<a href=\"#B7\" rid=\"B7\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776897\">&#x02013;</a><a href=\"#B11\" rid=\"B11\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776859\">11</a>). Samples once again clustered by study rather than by antimicrobial use, as nonantimicrobial control samples were not separated from antimicrobial treatment samples within each study (<a href=\"#figS3\" rid=\"figS3\" class=\" supplementary-material\">Fig.&#x000a0;S3</a>). Also, no genera were significantly different between pigs that received antimicrobials and those that did not (LDA [log<sub>10</sub>] score, &#x0003c;4.0). This result is perhaps not surprising given that there is a strong study effect and that antimicrobials with different mechanisms of action and dosages were used in each study. Each study also reported findings that are not necessarily in agreement with each other (<a href=\"#B51\" rid=\"B51\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776865\">51</a>). For example, the <em>Bacteroidetes</em> phylum has been reported to be relatively less abundant at specific time points in pigs fed tylosin (<a href=\"#B10\" rid=\"B10\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776875\">10</a>) and ASP250 (<a href=\"#B49\" rid=\"B49\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776855\">49</a>). However, carbadox treatment has been shown to actually increase the proportion of <em>Bacteroidetes</em> during the early phase of administration (<a href=\"#B8\" rid=\"B8\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776893\">8</a>).</p><span class=\"sec suppmat\" id=\"figS3\"><h4>FIG&#x000a0;S3&#x000a0;</h4><p id=\"__p41\">Principal-coordinate analysis plots of weighted UniFrac distances (A), unweighted UniFrac distances (B), and Bray-Curtis dissimilarities (C) for studies investigating the effect of an antimicrobial on the swine fecal microbiota. Samples are colored by study and antimicrobial treatment. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103sf3.tif\" id=\"figs3\">FIG&#x000a0;S3, TIF file, 1.7 MB</a>.</p><span class=\"small permissions half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al.<span class=\"license half_rhythm\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span><p id=\"__p42\" class=\"p p-last\">These findings emphasize the need for additional research into the effects of antimicrobials on the swine gut microbiota to better predict the response of the microbiota to these agents and to aid in the development of alternatives. The use of larger numbers of animals, longitudinal sampling, and the reporting of pig mass and other production attributes in each trial would all improve our understanding of the effect that antimicrobial agents have on gut microbiota of swine. It is also possible that any beneficial health or production effects observed in the pig are a result of changes in areas of the GI tract that preclude repeated sampling such as the ileum or cecum or that directly impact the host animal rather than the microbiota.</p></span><span id=\"s1.6\" class=\"sec sec-last\"><p></p><h3 id=\"s1.6title\" class=\"inline\">Conclusions. </h3><p id=\"__p43\" class=\"p p-first-last\">The results from our meta-analysis demonstrate that after &#x0201c;study&#x0201d; effects, GI tract location is the strongest predictor of the swine gut microbiota composition. In particular, the relatively abundant taxa are similar among samples from the same GI location, independently of other experimental variables. However, technical variation between individual studies may mask other biological differences among swine gut samples. For example, it was not possible to identify any consistent patterns related to antimicrobial use when six studies that included an antimicrobial treatment were analyzed together. The specificity of 16S rRNA gene primers and the hypervariable region that is sequenced also have a large effect on the analysis of the microbiota, as demonstrated in fecal samples from studies using different 16S regions. Nonetheless, a portion of the gut microbiota is shared among at least 90% of GI samples regardless of experimental variables, and this includes <em>Clostridium</em>, <em>Blautia</em>, <em>Lactobacillus</em>, <em>Prevotella</em>, <em>Ruminococcus</em>, <em>Roseburia</em>, the RC9 gut group, and <em>Subdoligranulum</em> at the genus level. These genera represent bacteria that are well adapted to the swine gut and may serve as potential markers of a typical swine gut microbiota. The inclusion of comprehensive metadata to accompany sequencing data sets would enhance future meta-analyses and allow researchers to compare their results directly with those from similar studies.</p></span></span><span id=\"s2\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s2title\" style=\"text-transform:uppercase\">MATERIALS AND METHODS</h2><span id=\"s2.1\" class=\"sec sec-first\"><p></p><h3 id=\"s2.1title\" class=\"inline\">Data acquisition and study inclusion criteria. </h3><p id=\"__p44\" class=\"p p-first-last\">A total of 20 studies were included in the meta-analysis and are described in <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab1/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"tab1\" rid-ob=\"ob-tab1\" co-legend-rid=\"\"><span>Table&#x000a0;1</span></a>. Studies were identified through a literature search of the Short Read Archive (SRA; <a href=\"http://www.ncbi.nlm.nih.gov/sra\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.ncbi.nlm.nih.gov/sra</a>), NCBI PubMed (<a href=\"http://www.ncbi.nlm.nih.gov/pubmed\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.ncbi.nlm.nih.gov/pubmed</a>), and Google Scholar (<a href=\"http://scholar.google.com\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://scholar.google.com</a>). To be included in the meta-analysis, all studies were required (i) to have been sampled from the swine GI tract, (ii) to have used high-throughput sequencing of the 16S rRNA gene, (iii) to have associated metadata and quality score files, and (iv) to have provided data that were publically available before 31&#x000a0;March 2016. Sequence files from each study were downloaded from the SRA or European Nucleotide Archive (ENA; <a href=\"http://www.ebi.ac.uk/ena\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.ebi.ac.uk/ena</a>).</p></span><span id=\"s2.2\" class=\"sec sec-last\"><p></p><h3 id=\"s2.2title\" class=\"inline\">Processing of 16S rRNA gene sequences. </h3><p id=\"__p45\" class=\"p p-first\">All downloaded 16S rRNA gene sequences were processed using the QIIME software package (v. 1.9.1) (<a href=\"#B52\" rid=\"B52\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776870\">52</a>). When paired-end sequence files were provided from Illumina MiSeq or HiSeq 2000-based studies, reads were joined using fastq-join (<a href=\"#B53\" rid=\"B53\" class=\" bibr popnode\">53</a>) with an allowed maximum difference of 15% and a minimum overlap of 35&#x000a0;bp. Sequences derived from the 454 FLX platform were converted from the fastq file format to fasta and qual files, and sequences were subsequently quality filtered using the <em>split_libraries.py</em> command with a Phred score cutoff of 25, a minimum read length of 200&#x000a0;bp, and a maximum homopolymer run of 6&#x000a0;bp. Illumina MiSeq and HiSeq 2000 reads were quality filtered using the <em>split_libraries_fastq.py</em> command with a Phred score cutoff of 25, reads were truncated following three consecutive base calls with a Phred score of &#x0003c;25, and reads with &#x0003c;75% of the original read length following truncation were discarded. The maximum length of each read was set based on the expected read length, as reads spanning regions V1 to V3, for example, are significantly longer than those corresponding to the V4 region.</p><p id=\"__p46\">The quality-filtered 16S rRNA gene sequences were clustered into operational taxonomic units (OTUs) at 97% similarity using a closed-reference approach, the SILVA release 119 database (<a href=\"#B54\" rid=\"B54\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776856\">54</a>), and USEARCH v. 6.1.544 (<a href=\"#B55\" rid=\"B55\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776849\">55</a>), with reverse strand matching enabled. In this method, reads that failed to match one of the 16S rRNA gene sequences in the SILVA database at 97% similarity were discarded. Depending upon the hypervariable region targeted, some 16S rRNA gene primers also amplify archaeal 16S rRNA gene sequences. Therefore, all sequences classified as <em>Archaea</em>, as well as chloroplast, mitochondrial, and eukaryotic sequences, were removed prior to downstream analysis. All samples were randomly subsampled to a level of 1,000 sequences per sample to account for uneven sequencing depth among samples and studies.</p><p id=\"__p47\">Samples were categorized based on sample type, treatment, diet, country of origin, age, sequencing platform, and 16S rRNA hypervariable region sequenced, where available. Only information about sample type, country of origin, sequencing platform, and hypervariable region sequenced was available for each sample. Linear discriminant analysis effect size (LEfSe) was used to determine which genera were significantly enriched in each sample type. Genera that were relatively more abundant in a particular sample group were identified by LEfSe using the Kruskal-Wallis test (<em>P</em> &#x0003c; 0.05), and the effect size of each of these genera was estimated using linear discriminant analysis (<a href=\"#B56\" rid=\"B56\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776899\">56</a>). An LDA score (log<sub>10</sub>) of 4.0 was used as the cutoff for identifying differentially abundant genera.</p><p id=\"__p48\" class=\"p p-last\">The between-sample (beta) diversity was assessed using the unweighted and weighted UniFrac (<a href=\"#B57\" rid=\"B57\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776858\">57</a>) distances and Bray-Curtis (<a href=\"#B58\" rid=\"B58\" class=\" bibr popnode\">58</a>) dissimilarities. Principal-coordinate analysis (PCoA) was used to visualize these distances using Emperor (<a href=\"#B59\" rid=\"B59\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776903\">59</a>). Permutational multivariate analysis of variance (PERMANOVA) using the adonis function (<a href=\"#B60\" rid=\"B60\" class=\" bibr popnode\">60</a>) with 9,999 permutations was implemented in QIIME to analyze the unweighted and weighted UniFrac distances and the Bray-Curtis dissimilarities for each gastrointestinal location, country of origin, hypervariable region, sequencing platform used, and study. Pig age was also assessed but with one study removed due to a lack of metadata. Permutational analysis of multivariate dispersions (PERMDISP) was used to test the homogeneity of dispersion for each metadata category as implemented in QIIME using the betadisper function of vegan (<a href=\"#B60\" rid=\"B60\" class=\" bibr popnode\">60</a>). The within-sample (alpha) diversity, richness, and evenness were calculated within QIIME using the Shannon index (<a href=\"#B61\" rid=\"B61\" class=\" bibr popnode\">61</a>), phylogenetic diversity (PD whole tree) (<a href=\"#B62\" rid=\"B62\" class=\" bibr popnode\">62</a>), Simpson reciprocal index (<a href=\"#B63\" rid=\"B63\" class=\" bibr popnode\">63</a>), and equitability (evenness) index. These metrics, as well as the metric of hypervariable region sequenced for fecal samples only, were compared among gastrointestinal sample types using a two-way ANOVA in R (v. 3.2.5) (<a href=\"#B64\" rid=\"B64\" class=\" bibr popnode\">64</a>) with hypervariable region and sample type as the independent factors, followed by Tukey&#x02019;s honestly significant difference (HSD) <em>post hoc</em> pairwise comparison test (agricolae package [<a href=\"#B65\" rid=\"B65\" class=\" bibr popnode\">65</a>]). All results were considered significant at <em>P</em> values of &#x0003c;0.05.</p></span></span><span id=\"idm140162575841136\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140162575841136title\" style=\"text-transform:uppercase\">ACKNOWLEDGMENTS</h2><span class=\"sec\"><p id=\"__p49\">We are grateful for Mike Marti&#x02019;s graphic design talents for the creation of <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig1\" rid-ob=\"ob-fig1\" co-legend-rid=\"lgnd_fig1\"><span>Fig.&#x000a0;1</span></a>.</p><p id=\"__p50\">D.B.H. obtained the sequence data, performed the analyses, and wrote the manuscript. B.W.B., D.B.H., and H.K.A. conceived the meta-analysis. B.W.B., H.K.A., and J.T. helped in the design and draft of the manuscript and provided critical feedback.</p><p id=\"__p51\">Mention of trade names or commercial products in this publication is solely for the purpose of providing specific information and does not imply recommendation or endorsement by the US Department of the Agriculture. The USDA is an equal opportunity provider and employer.</p></span></span><br><div class=\"ref-list-sec sec\" id=\"reference-list\"><div class=\"ref-cit-blk half_rhythm\" id=\"B1\">1. <span class=\"mixed-citation\">Brestoff JR, Artis D. \n2013. \n<span class=\"ref-title\">Commensal bacteria at the interface of host metabolism and the immune system</span>. <span class=\"ref-journal\">Nat Immunol</span>\n<span class=\"ref-vol\">14</span>:676&#x02013;684. doi:10.1038/ni.2640. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4013146/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23778795\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fni.2640\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat+Immunol&amp;title=Commensal+bacteria+at+the+interface+of+host+metabolism+and+the+immune+system&amp;author=JR+Brestoff&amp;author=D+Artis&amp;volume=14&amp;publication_year=2013&amp;pages=676-684&amp;pmid=23778795&amp;doi=10.1038/ni.2640&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B2\">2. <span class=\"mixed-citation\">Tremaroli V, B&#x000e4;ckhed F. \n2012. \n<span class=\"ref-title\">Functional interactions between the gut microbiota and host metabolism</span>. <span class=\"ref-journal\">Nature</span>\n<span class=\"ref-vol\">489</span>:242&#x02013;249. doi:10.1038/nature11552. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22972297\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnature11552\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Functional+interactions+between+the+gut+microbiota+and+host+metabolism&amp;author=V+Tremaroli&amp;author=F+B&#x000e4;ckhed&amp;volume=489&amp;publication_year=2012&amp;pages=242-249&amp;pmid=22972297&amp;doi=10.1038/nature11552&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B3\">3. <span class=\"mixed-citation\">Lall&#x000e8;s JP, Bosi P, Smidt H, Stokes CR. \n2007. \n<span class=\"ref-title\">Nutritional management of gut health in pigs around weaning</span>. <span class=\"ref-journal\">Proc Nutr Soc</span>\n<span class=\"ref-vol\">66</span>:260&#x02013;268. doi:10.1017/S0029665107005484. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17466106\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1017%2FS0029665107005484\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc+Nutr+Soc&amp;title=Nutritional+management+of+gut+health+in+pigs+around+weaning&amp;author=JP+Lall&#x000e8;s&amp;author=P+Bosi&amp;author=H+Smidt&amp;author=CR+Stokes&amp;volume=66&amp;publication_year=2007&amp;pages=260-268&amp;pmid=17466106&amp;doi=10.1017/S0029665107005484&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B4\">4. <span class=\"mixed-citation\">Waite DW, Taylor MW. \n2014. \n<span class=\"ref-title\">Characterizing the avian gut microbiota: membership, driving influences, and potential function</span>. <span class=\"ref-journal\">Front Microbiol</span>\n<span class=\"ref-vol\">5</span>:223. doi:10.3389/fmicb.2014.00223. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4032936/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24904538\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3389%2Ffmicb.2014.00223\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front+Microbiol&amp;title=Characterizing+the+avian+gut+microbiota:+membership,+driving+influences,+and+potential+function&amp;author=DW+Waite&amp;author=MW+Taylor&amp;volume=5&amp;publication_year=2014&amp;pages=223&amp;pmid=24904538&amp;doi=10.3389/fmicb.2014.00223&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B5\">5. <span class=\"mixed-citation\">Adams RI, Bateman AC, Bik HM, Meadow JF. \n2015. \n<span class=\"ref-title\">Microbiota of the indoor environment: a meta-analysis</span>. <span class=\"ref-journal\">Microbiome</span>\n<span class=\"ref-vol\">3</span>:49. doi:10.1186/s40168-015-0108-3. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4604073/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26459172\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2Fs40168-015-0108-3\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Microbiota+of+the+indoor+environment:+a+meta-analysis&amp;author=RI+Adams&amp;author=AC+Bateman&amp;author=HM+Bik&amp;author=JF+Meadow&amp;volume=3&amp;publication_year=2015&amp;pages=49&amp;pmid=26459172&amp;doi=10.1186/s40168-015-0108-3&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B6\">6. <span class=\"mixed-citation\">Allen HK, Looft T, Bayles DO, Humphrey S, Levine UY, Alt D, Stanton TB. \n2011. \n<span class=\"ref-title\">Antibiotics in feed induce prophages in swine fecal microbiomes</span>. <span class=\"ref-journal\">mBio</span>\n<span class=\"ref-vol\">2</span>:e00260-11. doi:10.1128/mBio.00260-11. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3225969/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22128350\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FmBio.00260-11\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Antibiotics+in+feed+induce+prophages+in+swine+fecal+microbiomes&amp;author=HK+Allen&amp;author=T+Looft&amp;author=DO+Bayles&amp;author=S+Humphrey&amp;author=UY+Levine&amp;volume=2&amp;publication_year=2011&amp;pages=e00260-11&amp;pmid=22128350&amp;doi=10.1128/mBio.00260-11&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B7\">7. <span class=\"mixed-citation\">Looft T, Allen HK, Cantarel BL, Levine UY, Bayles DO, Alt DP, Henrissat B, Stanton TB. \n2014. \n<span class=\"ref-title\">Bacteria, phages and pigs: the effects of in-feed antibiotics on the microbiome at different gut locations</span>. <span class=\"ref-journal\">ISME J</span>\n<span class=\"ref-vol\">8</span>:1566&#x02013;1576. doi:10.1038/ismej.2014.12. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4817603/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24522263\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fismej.2014.12\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=ISME+J&amp;title=Bacteria,+phages+and+pigs:+the+effects+of+in-feed+antibiotics+on+the+microbiome+at+different+gut+locations&amp;author=T+Looft&amp;author=HK+Allen&amp;author=BL+Cantarel&amp;author=UY+Levine&amp;author=DO+Bayles&amp;volume=8&amp;publication_year=2014&amp;pages=1566-1576&amp;pmid=24522263&amp;doi=10.1038/ismej.2014.12&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B8\">8. <span class=\"mixed-citation\">Looft T, Allen HK, Casey TA, Alt DP, Stanton TB. \n2014. \n<span class=\"ref-title\">Carbadox has both temporary and lasting effects on the swine gut microbiota</span>. <span class=\"ref-journal\">Front Microbiol</span>\n<span class=\"ref-vol\">5</span>:276. doi:10.3389/fmicb.2014.00276. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4050737/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24959163\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3389%2Ffmicb.2014.00276\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front+Microbiol&amp;title=Carbadox+has+both+temporary+and+lasting+effects+on+the+swine+gut+microbiota&amp;author=T+Looft&amp;author=HK+Allen&amp;author=TA+Casey&amp;author=DP+Alt&amp;author=TB+Stanton&amp;volume=5&amp;publication_year=2014&amp;pages=276&amp;pmid=24959163&amp;doi=10.3389/fmicb.2014.00276&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B9\">9. <span class=\"mixed-citation\">Looft T, Johnson TA, Allen HK, Bayles DO, Alt DP, Stedtfeld RD, Sul WJ, Stedtfeld TM, Chai B, Cole JR, Hashsham SA, Tiedje JM, Stanton TB. \n2012. \n<span class=\"ref-title\">In-feed antibiotic effects on the swine intestinal microbiome</span>. <span class=\"ref-journal\">Proc Natl Acad Sci U S A</span>\n<span class=\"ref-vol\">109</span>:1691&#x02013;1696. doi:10.1073/pnas.1120238109. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3277147/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22307632\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1073%2Fpnas.1120238109\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=In-feed+antibiotic+effects+on+the+swine+intestinal+microbiome&amp;author=T+Looft&amp;author=TA+Johnson&amp;author=HK+Allen&amp;author=DO+Bayles&amp;author=DP+Alt&amp;volume=109&amp;publication_year=2012&amp;pages=1691-1696&amp;pmid=22307632&amp;doi=10.1073/pnas.1120238109&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B10\">10. <span class=\"mixed-citation\">Holman DB, Ch&#x000e9;nier MR. \n2014. \n<span class=\"ref-title\">Temporal changes and the effect of subtherapeutic concentrations of antibiotics in the gut microbiota of swine</span>. <span class=\"ref-journal\">FEMS Microbiol Ecol</span>\n<span class=\"ref-vol\">90</span>:599&#x02013;608. doi:10.1111/1574-6941.12419. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25187398\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2F1574-6941.12419\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=FEMS+Microbiol+Ecol&amp;title=Temporal+changes+and+the+effect+of+subtherapeutic+concentrations+of+antibiotics+in+the+gut+microbiota+of+swine&amp;author=DB+Holman&amp;author=MR+Ch&#x000e9;nier&amp;volume=90&amp;publication_year=2014&amp;pages=599-608&amp;pmid=25187398&amp;doi=10.1111/1574-6941.12419&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B11\">11. <span class=\"mixed-citation\">Unno T, Kim JM, Guevarra RB, Nguyen SG. \n2015. \n<span class=\"ref-title\">Effects of antibiotic growth promoter and characterization of ecological succession in swine gut microbiota</span>. <span class=\"ref-journal\">J Microbiol Biotechnol</span>\n<span class=\"ref-vol\">25</span>:431&#x02013;438. doi:10.4014/jmb.1408.08063. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25370726\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.4014%2Fjmb.1408.08063\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Microbiol+Biotechnol&amp;title=Effects+of+antibiotic+growth+promoter+and+characterization+of+ecological+succession+in+swine+gut+microbiota&amp;author=T+Unno&amp;author=JM+Kim&amp;author=RB+Guevarra&amp;author=SG+Nguyen&amp;volume=25&amp;publication_year=2015&amp;pages=431-438&amp;pmid=25370726&amp;doi=10.4014/jmb.1408.08063&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B12\">12. <span class=\"mixed-citation\">Levesque CL, Hooda S, Swanson KS, de Lange K. \n2014. \n<span class=\"ref-title\">Alterations in ileal mucosa bacteria related to diet complexity and growth performance in young pigs</span>. <span class=\"ref-journal\">PLoS One</span>\n<span class=\"ref-vol\">9</span>:e108472. doi:10.1371/journal.pone.0108472. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4172762/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25247930\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0108472\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Alterations+in+ileal+mucosa+bacteria+related+to+diet+complexity+and+growth+performance+in+young+pigs&amp;author=CL+Levesque&amp;author=S+Hooda&amp;author=KS+Swanson&amp;author=K+de+Lange&amp;volume=9&amp;publication_year=2014&amp;pages=e108472&amp;pmid=25247930&amp;doi=10.1371/journal.pone.0108472&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B13\">13. <span class=\"mixed-citation\">Luo YH, Yang C, Wright AD, He J, Chen DW. \n2015. \n<span class=\"ref-title\">Responses in ileal and cecal bacteria to low and high amylose/amylopectin ratio diets in growing pigs</span>. <span class=\"ref-journal\">Appl Microbiol Biotechnol</span>\n<span class=\"ref-vol\">99</span>:10627&#x02013;10638. doi:10.1007/s00253-015-6917-2. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26318448\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs00253-015-6917-2\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl+Microbiol+Biotechnol&amp;title=Responses+in+ileal+and+cecal+bacteria+to+low+and+high+amylose/amylopectin+ratio+diets+in+growing+pigs&amp;author=YH+Luo&amp;author=C+Yang&amp;author=AD+Wright&amp;author=J+He&amp;author=DW+Chen&amp;volume=99&amp;publication_year=2015&amp;pages=10627-10638&amp;pmid=26318448&amp;doi=10.1007/s00253-015-6917-2&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B14\">14. <span class=\"mixed-citation\">Mann E, Schmitz-Esser S, Zebeli Q, Wagner M, Ritzmann M, Metzler-Zebeli BU. \n2014. \n<span class=\"ref-title\">Mucosa-associated bacterial microbiome of the gastrointestinal tract of weaned pigs and dynamics linked to dietary calcium-phosphorus</span>. <span class=\"ref-journal\">PLoS One</span>\n<span class=\"ref-vol\">9</span>:e86950. doi:10.1371/journal.pone.0086950. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3900689/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24466298\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0086950\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Mucosa-associated+bacterial+microbiome+of+the+gastrointestinal+tract+of+weaned+pigs+and+dynamics+linked+to+dietary+calcium-phosphorus&amp;author=E+Mann&amp;author=S+Schmitz-Esser&amp;author=Q+Zebeli&amp;author=M+Wagner&amp;author=M+Ritzmann&amp;volume=9&amp;publication_year=2014&amp;pages=e86950&amp;pmid=24466298&amp;doi=10.1371/journal.pone.0086950&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B15\">15. <span class=\"mixed-citation\">Burrough ER, Arruda BL, Patience JF, Plummer PJ. \n2015. \n<span class=\"ref-title\">Alterations in the colonic microbiota of pigs associated with feeding distillers dried grains with solubles</span>. <span class=\"ref-journal\">PLoS One</span>\n<span class=\"ref-vol\">10</span>:e0141337. doi:10.1371/journal.pone.0141337. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4640664/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26555787\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0141337\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Alterations+in+the+colonic+microbiota+of+pigs+associated+with+feeding+distillers+dried+grains+with+solubles&amp;author=ER+Burrough&amp;author=BL+Arruda&amp;author=JF+Patience&amp;author=PJ+Plummer&amp;volume=10&amp;publication_year=2015&amp;pages=e0141337&amp;pmid=26555787&amp;doi=10.1371/journal.pone.0141337&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B16\">16. <span class=\"mixed-citation\">Metzler-Zebeli BU, Schmitz-Esser S, Mann E, Gr&#x000fc;ll D, Molnar T, Zebeli Q. \n2015. \n<span class=\"ref-title\">Adaptation of the cecal bacterial microbiome of growing pigs in response to resistant starch type 4</span>. <span class=\"ref-journal\">Appl Environ Microbiol</span>\n<span class=\"ref-vol\">81</span>:8489&#x02013;8499. doi:10.1128/AEM.02756-15. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4644661/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26431973\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.02756-15\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl+Environ+Microbiol&amp;title=Adaptation+of+the+cecal+bacterial+microbiome+of+growing+pigs+in+response+to+resistant+starch+type+4&amp;author=BU+Metzler-Zebeli&amp;author=S+Schmitz-Esser&amp;author=E+Mann&amp;author=D+Gr&#x000fc;ll&amp;author=T+Molnar&amp;volume=81&amp;publication_year=2015&amp;pages=8489-8499&amp;pmid=26431973&amp;doi=10.1128/AEM.02756-15&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B17\">17. <span class=\"mixed-citation\">Hedegaard CJ, Strube ML, Hansen MB, Lindved BK, Lihme A, Boye M, Heegaard PM. \n2016. \n<span class=\"ref-title\">Natural pig plasma immunoglobulins have antibacterial effects: potential for use as feed supplement for treatment of intestinal infections in pigs</span>. <span class=\"ref-journal\">PLoS One</span>\n<span class=\"ref-vol\">11</span>:e0147373. doi:10.1371/journal.pone.0147373. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4744083/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26824607\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0147373\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Natural+pig+plasma+immunoglobulins+have+antibacterial+effects:+potential+for+use+as+feed+supplement+for+treatment+of+intestinal+infections+in+pigs&amp;author=CJ+Hedegaard&amp;author=ML+Strube&amp;author=MB+Hansen&amp;author=BK+Lindved&amp;author=A+Lihme&amp;volume=11&amp;publication_year=2016&amp;pages=e0147373&amp;pmid=26824607&amp;doi=10.1371/journal.pone.0147373&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B18\">18. <span class=\"mixed-citation\">Bearson SM, Allen HK, Bearson BL, Looft T, Brunelle BW, Kich JD, Tuggle CK, Bayles DO, Alt D, Levine UY, Stanton TB. \n2013. \n<span class=\"ref-title\">Profiling the gastrointestinal microbiota in response to <em>Salmonella</em>: low versus high <em>Salmonella</em> shedding in the natural porcine host</span>. <span class=\"ref-journal\">Infect Genet Evol</span>\n<span class=\"ref-vol\">16</span>:330&#x02013;340. doi:10.1016/j.meegid.2013.03.022. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23535116\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.meegid.2013.03.022\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Infect+Genet+Evol&amp;title=Profiling+the+gastrointestinal+microbiota+in+response+to+Salmonella:+low+versus+high+Salmonella+shedding+in+the+natural+porcine+host&amp;author=SM+Bearson&amp;author=HK+Allen&amp;author=BL+Bearson&amp;author=T+Looft&amp;author=BW+Brunelle&amp;volume=16&amp;publication_year=2013&amp;pages=330-340&amp;pmid=23535116&amp;doi=10.1016/j.meegid.2013.03.022&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B19\">19. <span class=\"mixed-citation\">Kuczynski J, Liu Z, Lozupone C, McDonald D, Fierer N, Knight R. \n2010. \n<span class=\"ref-title\">Microbial community resemblance methods differ in their ability to detect biologically relevant patterns</span>. <span class=\"ref-journal\">Nat Methods</span>\n<span class=\"ref-vol\">7</span>:813&#x02013;819. doi:10.1038/nmeth.1499. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2948603/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20818378\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnmeth.1499\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=Microbial+community+resemblance+methods+differ+in+their+ability+to+detect+biologically+relevant+patterns&amp;author=J+Kuczynski&amp;author=Z+Liu&amp;author=C+Lozupone&amp;author=D+McDonald&amp;author=N+Fierer&amp;volume=7&amp;publication_year=2010&amp;pages=813-819&amp;pmid=20818378&amp;doi=10.1038/nmeth.1499&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B20\">20. <span class=\"mixed-citation\">NRC \n2012. \n<span class=\"ref-journal\">Nutrient requirements of swine</span>, 11th ed. <span class=\"ref-journal\">The National</span>\nAcademies Press, Washington, DC. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Nutrient+requirements+of+swine&amp;title=The+National&amp;publication_year=2012&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B21\">21. <span class=\"mixed-citation\">Wu GD, Chen J, Hoffmann C, Bittinger K, Chen YY, Keilbaugh SA, Bewtra M, Knights D, Walters WA, Knight R, Sinha R, Gilroy E, Gupta K, Baldassano R, Nessel L, Li H, Bushman FD, Lewis JD. \n2011. \n<span class=\"ref-title\">Linking long-term dietary patterns with gut microbial enterotypes</span>. <span class=\"ref-journal\">Science</span>\n<span class=\"ref-vol\">334</span>:105&#x02013;108. doi:10.1126/science.1208344. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3368382/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21885731\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1126%2Fscience.1208344\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Linking+long-term+dietary+patterns+with+gut+microbial+enterotypes&amp;author=GD+Wu&amp;author=J+Chen&amp;author=C+Hoffmann&amp;author=K+Bittinger&amp;author=YY+Chen&amp;volume=334&amp;publication_year=2011&amp;pages=105-108&amp;pmid=21885731&amp;doi=10.1126/science.1208344&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B22\">22. <span class=\"mixed-citation\">De Filippis F, Pellegrini N, Vannini L, Jeffery IB, La Storia A, Laghi L, Serrazanetti DI, Di Cagno R, Ferrocino I, Lazzi C, Turroni S, Cocolin L, Brigidi P, Neviani E, Gobbetti M, O&#x02019;Toole PW, Ercolini D. \n2016. \n<span class=\"ref-title\">High-level adherence to a Mediterranean diet beneficially impacts the gut microbiota and associated metabolome</span>. <span class=\"ref-journal\">Gut</span>\n<span class=\"ref-vol\">65</span>:1812&#x02013;1821. doi:10.1136/gutjnl-2015-309957. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26416813\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1136%2Fgutjnl-2015-309957\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=High-level+adherence+to+a+Mediterranean+diet+beneficially+impacts+the+gut+microbiota+and+associated+metabolome&amp;author=F+De+Filippis&amp;author=N+Pellegrini&amp;author=L+Vannini&amp;author=IB+Jeffery&amp;author=A+La+Storia&amp;volume=65&amp;publication_year=2016&amp;pages=1812-1821&amp;pmid=26416813&amp;doi=10.1136/gutjnl-2015-309957&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B23\">23. <span class=\"mixed-citation\">Flint HJ, Bayer EA. \n2008. \n<span class=\"ref-title\">Plant cell wall breakdown by anaerobic microorganisms from the mammalian digestive tract</span>. <span class=\"ref-journal\">Ann N Y Acad Sci</span>\n<span class=\"ref-vol\">1125</span>:280&#x02013;288. doi:10.1196/annals.1419.022. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n18378598\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1196%2Fannals.1419.022\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Ann+N+Y+Acad+Sci&amp;title=Plant+cell+wall+breakdown+by+anaerobic+microorganisms+from+the+mammalian+digestive+tract&amp;author=HJ+Flint&amp;author=EA+Bayer&amp;volume=1125&amp;publication_year=2008&amp;pages=280-288&amp;pmid=18378598&amp;doi=10.1196/annals.1419.022&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B24\">24. <span class=\"mixed-citation\">Kim M, Kim J, Kuehn LA, Bono JL, Berry ED, Kalchayanand N, Freetly HC, Benson AK, Wells JE. \n2014. \n<span class=\"ref-title\">Investigation of bacterial diversity in the feces of cattle fed different diets</span>. <span class=\"ref-journal\">J Anim Sci</span>\n<span class=\"ref-vol\">92</span>:683&#x02013;694. doi:10.2527/jas.2013-6841. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24352967\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.2527%2Fjas.2013-6841\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Anim+Sci&amp;title=Investigation+of+bacterial+diversity+in+the+feces+of+cattle+fed+different+diets&amp;author=M+Kim&amp;author=J+Kim&amp;author=LA+Kuehn&amp;author=JL+Bono&amp;author=ED+Berry&amp;volume=92&amp;publication_year=2014&amp;pages=683-694&amp;pmid=24352967&amp;doi=10.2527/jas.2013-6841&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B25\">25. <span class=\"mixed-citation\">Moeller AH, Degnan PH, Pusey AE, Wilson ML, Hahn BH, Ochman H. \n2012. \n<span class=\"ref-title\">Chimpanzees and humans harbour compositionally similar gut enterotypes</span>. <span class=\"ref-journal\">Nat Commun</span>\n<span class=\"ref-vol\">3</span>:1179. doi:10.1038/ncomms2159. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3520023/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23149725\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fncomms2159\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=Chimpanzees+and+humans+harbour+compositionally+similar+gut+enterotypes&amp;author=AH+Moeller&amp;author=PH+Degnan&amp;author=AE+Pusey&amp;author=ML+Wilson&amp;author=BH+Hahn&amp;volume=3&amp;publication_year=2012&amp;pages=1179&amp;pmid=23149725&amp;doi=10.1038/ncomms2159&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B26\">26. <span class=\"mixed-citation\">Huo W, Zhu W, Mao S. \n2014. \n<span class=\"ref-title\">Impact of subacute ruminal acidosis on the diversity of liquid and solid-associated bacteria in the rumen of goats</span>. <span class=\"ref-journal\">World J Microbiol Biotechnol</span>\n<span class=\"ref-vol\">30</span>:669&#x02013;680. doi:10.1007/s11274-013-1489-8. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24068532\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs11274-013-1489-8\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=World+J+Microbiol+Biotechnol&amp;title=Impact+of+subacute+ruminal+acidosis+on+the+diversity+of+liquid+and+solid-associated+bacteria+in+the+rumen+of+goats&amp;author=W+Huo&amp;author=W+Zhu&amp;author=S+Mao&amp;volume=30&amp;publication_year=2014&amp;pages=669-680&amp;pmid=24068532&amp;doi=10.1007/s11274-013-1489-8&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B27\">27. <span class=\"mixed-citation\">Castro-Carrera T, Toral PG, Frutos P, McEwan NR, Herv&#x000e1;s G, Abecia L, Pinloche E, Girdwood SE, Belenguer A. \n2014. \n<span class=\"ref-title\">Rumen bacterial community evaluated by 454 pyrosequencing and terminal restriction fragment length polymorphism analyses in dairy sheep fed marine algae</span>. <span class=\"ref-journal\">J Dairy Sci</span>\n<span class=\"ref-vol\">97</span>:1661&#x02013;1669. doi:10.3168/jds.2013-7243. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24440247\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3168%2Fjds.2013-7243\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Dairy+Sci&amp;title=Rumen+bacterial+community+evaluated+by+454+pyrosequencing+and+terminal+restriction+fragment+length+polymorphism+analyses+in+dairy+sheep+fed+marine+algae&amp;author=T+Castro-Carrera&amp;author=PG+Toral&amp;author=P+Frutos&amp;author=NR+McEwan&amp;author=G+Herv&#x000e1;s&amp;volume=97&amp;publication_year=2014&amp;pages=1661-1669&amp;pmid=24440247&amp;doi=10.3168/jds.2013-7243&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B28\">28. <span class=\"mixed-citation\">Biddle A, Stewart L, Blanchard J, Leschine S. \n2013. \n<span class=\"ref-title\">Untangling the genetic basis of fibrolytic specialization by <em>Lachnospiraceae</em> and <em>Ruminococcaceae</em> in diverse gut communities</span>. <span class=\"ref-journal\">Diversity</span>\n<span class=\"ref-vol\">5</span>:627&#x02013;640. doi:10.3390/d5030627. [<a href=\"//dx.doi.org/10.3390%2Fd5030627\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Diversity&amp;title=Untangling+the+genetic+basis+of+fibrolytic+specialization+by+Lachnospiraceae+and+Ruminococcaceae+in+diverse+gut+communities&amp;author=A+Biddle&amp;author=L+Stewart&amp;author=J+Blanchard&amp;author=S+Leschine&amp;volume=5&amp;publication_year=2013&amp;pages=627-640&amp;doi=10.3390/d5030627&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B29\">29. <span class=\"mixed-citation\">Vital M, Howe AC, Tiedje JM. \n2014. \n<span class=\"ref-title\">Revealing the bacterial butyrate synthesis pathways by analyzing (meta) genomic data</span>. <span class=\"ref-journal\">mBio</span>\n<span class=\"ref-vol\">5</span>:e00889-14. doi:10.1128/mBio.00889-14. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3994512/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24757212\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FmBio.00889-14\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Revealing+the+bacterial+butyrate+synthesis+pathways+by+analyzing+(meta)+genomic+data&amp;author=M+Vital&amp;author=AC+Howe&amp;author=JM+Tiedje&amp;volume=5&amp;publication_year=2014&amp;pages=e00889-14&amp;pmid=24757212&amp;doi=10.1128/mBio.00889-14&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B30\">30. <span class=\"mixed-citation\">Hamer HM, Jonkers D, Venema K, Vanhoutvin S, Troost FJ, Brummer RJ. \n2008. \n<span class=\"ref-title\">Review article: the role of butyrate on colonic function</span>. <span class=\"ref-journal\">Aliment Pharmacol Ther</span>\n<span class=\"ref-vol\">27</span>:104&#x02013;119. doi:10.1111/j.1365-2036.2007.03562.x. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17973645\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1365-2036.2007.03562.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Aliment+Pharmacol+Ther&amp;title=Review+article:+the+role+of+butyrate+on+colonic+function&amp;author=HM+Hamer&amp;author=D+Jonkers&amp;author=K+Venema&amp;author=S+Vanhoutvin&amp;author=FJ+Troost&amp;volume=27&amp;publication_year=2008&amp;pages=104-119&amp;pmid=17973645&amp;doi=10.1111/j.1365-2036.2007.03562.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B31\">31. <span class=\"mixed-citation\">Tran TH, Boudry C, Everaert N, Th&#x000e9;wis A, Portetelle D, Daube G, Nezer C, Taminiau B, Bindelle J. \n2016. \n<span class=\"ref-title\">Adding mucins to an <em>in vitro</em> batch fermentation model of the large intestine induces changes in microbial population isolated from porcine feces depending on the substrate</span>. <span class=\"ref-journal\">FEMS Microbiol Ecol</span>\n<span class=\"ref-vol\">92</span>:fiv165. doi:10.1093/femsec/fiv165. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26691596\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Ffemsec%2Ffiv165\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=FEMS+Microbiol+Ecol&amp;title=Adding+mucins+to+an+in+vitro+batch+fermentation+model+of+the+large+intestine+induces+changes+in+microbial+population+isolated+from+porcine+feces+depending+on+the+substrate&amp;author=TH+Tran&amp;author=C+Boudry&amp;author=N+Everaert&amp;author=A+Th&#x000e9;wis&amp;author=D+Portetelle&amp;volume=92&amp;publication_year=2016&amp;pages=fiv165&amp;pmid=26691596&amp;doi=10.1093/femsec/fiv165&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B32\">32. <span class=\"mixed-citation\">Sun Y, Zhou L, Fang L, Su Y, Zhu W. \n2015. \n<span class=\"ref-title\">Responses in colonic microbial community and gene expression of pigs to a long-term high resistant starch diet</span>. <span class=\"ref-journal\">Front Microbiol</span>\n<span class=\"ref-vol\">6</span>:877. doi:10.3389/fmicb.2015.00877. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4548152/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26379652\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3389%2Ffmicb.2015.00877\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front+Microbiol&amp;title=Responses+in+colonic+microbial+community+and+gene+expression+of+pigs+to+a+long-term+high+resistant+starch+diet&amp;author=Y+Sun&amp;author=L+Zhou&amp;author=L+Fang&amp;author=Y+Su&amp;author=W+Zhu&amp;volume=6&amp;publication_year=2015&amp;pages=877&amp;pmid=26379652&amp;doi=10.3389/fmicb.2015.00877&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B33\">33. <span class=\"mixed-citation\">Graf J.\n2014. \n<span class=\"ref-title\">The family <em>Rikenellaceae</em></span>, p 857&#x02013;859. <em>In</em>\n<span class=\"ref-journal\">The prokaryotes</span>. Springer, Berlin, Germany. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=The+prokaryotes&amp;author=J+Graf&amp;publication_year=2014&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B34\">34. <span class=\"mixed-citation\">Su XL, Tian Q, Zhang J, Yuan XZ, Shi XS, Guo RB, Qiu YL. \n2014. \n<span class=\"ref-title\"><em>Acetobacteroides hydrogenigenes</em> gen. nov., sp. nov., an anaerobic hydrogen-producing bacterium in the family <em>Rikenellaceae</em> isolated from a reed swamp</span>. <span class=\"ref-journal\">Int J Syst Evol Microbiol</span>\n<span class=\"ref-vol\">64</span>:2986&#x02013;2991. doi:10.1099/ijs.0.063917-0. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24899658\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1099%2Fijs.0.063917-0\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int+J+Syst+Evol+Microbiol&amp;title=Acetobacteroides+hydrogenigenes+gen.+nov.,+sp.+nov.,+an+anaerobic+hydrogen-producing+bacterium+in+the+family+Rikenellaceae+isolated+from+a+reed+swamp&amp;author=XL+Su&amp;author=Q+Tian&amp;author=J+Zhang&amp;author=XZ+Yuan&amp;author=XS+Shi&amp;volume=64&amp;publication_year=2014&amp;pages=2986-2991&amp;pmid=24899658&amp;doi=10.1099/ijs.0.063917-0&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B35\">35. <span class=\"mixed-citation\">Frese SA, Parker K, Calvert CC, Mills DA. \n2015. \n<span class=\"ref-title\">Diet shapes the gut microbiome of pigs during nursing and weaning</span>. <span class=\"ref-journal\">Microbiome</span>\n<span class=\"ref-vol\">3</span>:28. doi:10.1186/s40168-015-0091-8. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4499176/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26167280\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2Fs40168-015-0091-8\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Diet+shapes+the+gut+microbiome+of+pigs+during+nursing+and+weaning&amp;author=SA+Frese&amp;author=K+Parker&amp;author=CC+Calvert&amp;author=DA+Mills&amp;volume=3&amp;publication_year=2015&amp;pages=28&amp;pmid=26167280&amp;doi=10.1186/s40168-015-0091-8&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B36\">36. <span class=\"mixed-citation\">van Winsen RL, Urlings BA, Lipman LJ, Snijders JM, Keuzenkamp D, Verheijden JH, van Knapen F. \n2001. \n<span class=\"ref-title\">Effect of fermented feed on the microbial population of the gastrointestinal tracts of pigs</span>. <span class=\"ref-journal\">Appl Environ Microbiol</span>\n<span class=\"ref-vol\">67</span>:3071&#x02013;3076. doi:10.1128/AEM.67.7.3071-3076.2001. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC92983/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n11425724\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.67.7.3071-3076.2001\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl+Environ+Microbiol&amp;title=Effect+of+fermented+feed+on+the+microbial+population+of+the+gastrointestinal+tracts+of+pigs&amp;author=RL+van+Winsen&amp;author=BA+Urlings&amp;author=LJ+Lipman&amp;author=JM+Snijders&amp;author=D+Keuzenkamp&amp;volume=67&amp;publication_year=2001&amp;pages=3071-3076&amp;pmid=11425724&amp;doi=10.1128/AEM.67.7.3071-3076.2001&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B37\">37. <span class=\"mixed-citation\">Walter J.\n2008. \n<span class=\"ref-title\">Ecological role of lactobacilli in the gastrointestinal tract: implications for fundamental and biomedical research</span>. <span class=\"ref-journal\">Appl Environ Microbiol</span>\n<span class=\"ref-vol\">74</span>:4985&#x02013;4996. doi:10.1128/AEM.00753-08. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2519286/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n18539818\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.00753-08\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl+Environ+Microbiol&amp;title=Ecological+role+of+lactobacilli+in+the+gastrointestinal+tract:+implications+for+fundamental+and+biomedical+research&amp;author=J+Walter&amp;volume=74&amp;publication_year=2008&amp;pages=4985-4996&amp;pmid=18539818&amp;doi=10.1128/AEM.00753-08&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B38\">38. <span class=\"mixed-citation\">Tannock GW.\n2004. \n<span class=\"ref-title\">A special fondness for lactobacilli</span>. <span class=\"ref-journal\">Appl Environ Microbiol</span>\n<span class=\"ref-vol\">70</span>:3189&#x02013;3194. doi:10.1128/AEM.70.6.3189-3194.2004. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC427720/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n15184111\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.70.6.3189-3194.2004\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl+Environ+Microbiol&amp;title=A+special+fondness+for+lactobacilli&amp;author=GW+Tannock&amp;volume=70&amp;publication_year=2004&amp;pages=3189-3194&amp;pmid=15184111&amp;doi=10.1128/AEM.70.6.3189-3194.2004&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B39\">39. <span class=\"mixed-citation\">Heo JM, Opapeju FO, Pluske JR, Kim JC, Hampson DJ, Nyachoti CM. \n2013. \n<span class=\"ref-title\">Gastrointestinal health and function in weaned pigs: a review of feeding strategies to control post-weaning diarrhoea without using in-feed antimicrobial compounds</span>. <span class=\"ref-journal\">J Anim Physiol Anim Nutr</span>\n<span class=\"ref-vol\">97</span>:207&#x02013;237. doi:10.1111/j.1439-0396.2012.01284.x. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22416941\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1439-0396.2012.01284.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Anim+Physiol+Anim+Nutr&amp;title=Gastrointestinal+health+and+function+in+weaned+pigs:+a+review+of+feeding+strategies+to+control+post-weaning+diarrhoea+without+using+in-feed+antimicrobial+compounds&amp;author=JM+Heo&amp;author=FO+Opapeju&amp;author=JR+Pluske&amp;author=JC+Kim&amp;author=DJ+Hampson&amp;volume=97&amp;publication_year=2013&amp;pages=207-237&amp;doi=10.1111/j.1439-0396.2012.01284.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B40\">40. <span class=\"mixed-citation\">Leser TD, M&#x000f8;lbak L. \n2009. \n<span class=\"ref-title\">Better living through microbial action: the benefits of the mammalian gastrointestinal microbiota on the host</span>. <span class=\"ref-journal\">Environ Microbiol</span>\n<span class=\"ref-vol\">11</span>:2194&#x02013;2206. doi:10.1111/j.1462-2920.2009.01941.x. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19737302\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1462-2920.2009.01941.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Environ+Microbiol&amp;title=Better+living+through+microbial+action:+the+benefits+of+the+mammalian+gastrointestinal+microbiota+on+the+host&amp;author=TD+Leser&amp;author=L+M&#x000f8;lbak&amp;volume=11&amp;publication_year=2009&amp;pages=2194-2206&amp;pmid=19737302&amp;doi=10.1111/j.1462-2920.2009.01941.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B41\">41. <span class=\"mixed-citation\">Hiergeist A, Reischl U; Priority Program 1656 Intestinal Microbiota Consortium/ quality assessment participants, Gessner A. \n2016. \n<span class=\"ref-title\">Multicenter quality assessment of 16S ribosomal DNA-sequencing for microbiome analyses reveals high inter-center variability</span>. <span class=\"ref-journal\">Int J Med Microbiol</span>\n<span class=\"ref-vol\">306</span>:334&#x02013;342. doi:10.1016/j.ijmm.2016.03.005. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27052158\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.ijmm.2016.03.005\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int+J+Med+Microbiol&amp;title=Multicenter+quality+assessment+of+16S+ribosomal+DNA-sequencing+for+microbiome+analyses+reveals+high+inter-center+variability&amp;author=A+Hiergeist&amp;author=U+Reischl&amp;author=A+Gessner&amp;volume=306&amp;publication_year=2016&amp;pages=334-342&amp;pmid=27052158&amp;doi=10.1016/j.ijmm.2016.03.005&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B42\">42. <span class=\"mixed-citation\">Klindworth A, Pruesse E, Schweer T, Peplies J, Quast C, Horn M, Gl&#x000f6;ckner FO. \n2013. \n<span class=\"ref-title\">Evaluation of general 16S ribosomal RNA gene PCR primers for classical and next-generation sequencing-based diversity studies</span>. <span class=\"ref-journal\">Nucleic Acids Res</span>\n<span class=\"ref-vol\">41</span>:e1. doi:10.1093/nar/gks808. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3592464/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22933715\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgks808\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Evaluation+of+general+16S+ribosomal+RNA+gene+PCR+primers+for+classical+and+next-generation+sequencing-based+diversity+studies&amp;author=A+Klindworth&amp;author=E+Pruesse&amp;author=T+Schweer&amp;author=J+Peplies&amp;author=C+Quast&amp;volume=41&amp;publication_year=2013&amp;pages=e1&amp;pmid=22933715&amp;doi=10.1093/nar/gks808&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B43\">43. <span class=\"mixed-citation\">Hayashi H, Sakamoto M, Benno Y. \n2004. \n<span class=\"ref-title\">Evaluation of three different forward primers by terminal restriction fragment length polymorphism analysis for determination of fecal <em>Bifidobacterium</em> spp. in healthy subjects</span>. <span class=\"ref-journal\">Microbiol Immunol</span>\n<span class=\"ref-vol\">48</span>:1&#x02013;6. doi:10.1111/j.1348-0421.2004.tb03481.x. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n14734852\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1348-0421.2004.tb03481.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Microbiol+Immunol&amp;title=Evaluation+of+three+different+forward+primers+by+terminal+restriction+fragment+length+polymorphism+analysis+for+determination+of+fecal+Bifidobacterium+spp.+in+healthy+subjects&amp;author=H+Hayashi&amp;author=M+Sakamoto&amp;author=Y+Benno&amp;volume=48&amp;publication_year=2004&amp;pages=1-6&amp;pmid=14734852&amp;doi=10.1111/j.1348-0421.2004.tb03481.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B44\">44. <span class=\"mixed-citation\">Walker AW, Martin JC, Scott P, Parkhill J, Flint HJ, Scott KP. \n2015. \n<span class=\"ref-title\">16S rRNA gene-based profiling of the human infant gut microbiota is strongly influenced by sample processing and PCR primer choice</span>. <span class=\"ref-journal\">Microbiome</span>\n<span class=\"ref-vol\">3</span>:26. doi:10.1186/s40168-015-0087-4. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4482049/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26120470\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2Fs40168-015-0087-4\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=16S+rRNA+gene-based+profiling+of+the+human+infant+gut+microbiota+is+strongly+influenced+by+sample+processing+and+PCR+primer+choice&amp;author=AW+Walker&amp;author=JC+Martin&amp;author=P+Scott&amp;author=J+Parkhill&amp;author=HJ+Flint&amp;volume=3&amp;publication_year=2015&amp;pages=26&amp;pmid=26120470&amp;doi=10.1186/s40168-015-0087-4&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B45\">45. <span class=\"mixed-citation\">Gerzova L, Babak V, Sedlar K, Faldynova M, Videnska P, Cejkova D, Jensen AN, Denis M, Kerouanton A, Ricci A, Cibin V, &#x000d6;sterberg J, Rychlik I. \n2015. \n<span class=\"ref-title\">Characterization of antibiotic resistance gene abundance and microbiota composition in feces of organic and conventional pigs from four EU countries</span>. <span class=\"ref-journal\">PLoS One</span>\n<span class=\"ref-vol\">10</span>:e0132892. doi:10.1371/journal.pone.0132892. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4517930/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26218075\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0132892\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Characterization+of+antibiotic+resistance+gene+abundance+and+microbiota+composition+in+feces+of+organic+and+conventional+pigs+from+four+EU+countries&amp;author=L+Gerzova&amp;author=V+Babak&amp;author=K+Sedlar&amp;author=M+Faldynova&amp;author=P+Videnska&amp;volume=10&amp;publication_year=2015&amp;pages=e0132892&amp;pmid=26218075&amp;doi=10.1371/journal.pone.0132892&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B46\">46. <span class=\"mixed-citation\">Barton MD.\n2014. \n<span class=\"ref-title\">Impact of antibiotic use in the swine industry</span>. <span class=\"ref-journal\">Curr Opin Microbiol</span>\n<span class=\"ref-vol\">19</span>:9&#x02013;15. doi:10.1016/j.mib.2014.05.017. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24959754\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.mib.2014.05.017\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Microbiol&amp;title=Impact+of+antibiotic+use+in+the+swine+industry&amp;author=MD+Barton&amp;volume=19&amp;publication_year=2014&amp;pages=9-15&amp;pmid=24959754&amp;doi=10.1016/j.mib.2014.05.017&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B47\">47. <span class=\"mixed-citation\">Allen HK, Stanton TB. \n2014. \n<span class=\"ref-title\">Altered egos: antibiotic effects on food animal microbiomes</span>. <span class=\"ref-journal\">Annu Rev Microbiol</span>\n<span class=\"ref-vol\">68</span>:297&#x02013;315. doi:10.1146/annurev-micro-091213-113052. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25002091\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1146%2Fannurev-micro-091213-113052\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Microbiol&amp;title=Altered+egos:+antibiotic+effects+on+food+animal+microbiomes&amp;author=HK+Allen&amp;author=TB+Stanton&amp;volume=68&amp;publication_year=2014&amp;pages=297-315&amp;pmid=25002091&amp;doi=10.1146/annurev-micro-091213-113052&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B48\">48. <span class=\"mixed-citation\">Holman DB, Ch&#x000e9;nier MR. \n2013. \n<span class=\"ref-title\">Impact of subtherapeutic administration of tylosin and chlortetracycline on antimicrobial resistance in farrow-to-finish swine</span>. <span class=\"ref-journal\">FEMS Microbiol Ecol</span>\n<span class=\"ref-vol\">85</span>:1&#x02013;13. doi:10.1111/1574-6941.12093. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23397987\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2F1574-6941.12093\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=FEMS+Microbiol+Ecol&amp;title=Impact+of+subtherapeutic+administration+of+tylosin+and+chlortetracycline+on+antimicrobial+resistance+in+farrow-to-finish+swine&amp;author=DB+Holman&amp;author=MR+Ch&#x000e9;nier&amp;volume=85&amp;publication_year=2013&amp;pages=1-13&amp;pmid=23397987&amp;doi=10.1111/1574-6941.12093&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B49\">49. <span class=\"mixed-citation\">Allen HK, Levine UY, Looft T, Bandrick M, Casey TA. \n2013. \n<span class=\"ref-title\">Treatment, promotion, commotion: antibiotic alternatives in food-producing animals</span>. <span class=\"ref-journal\">Trends Microbiol</span>\n<span class=\"ref-vol\">21</span>:114&#x02013;119. doi:10.1016/j.tim.2012.11.001. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23473629\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.tim.2012.11.001\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Trends+Microbiol&amp;title=Treatment,+promotion,+commotion:+antibiotic+alternatives+in+food-producing+animals&amp;author=HK+Allen&amp;author=UY+Levine&amp;author=T+Looft&amp;author=M+Bandrick&amp;author=TA+Casey&amp;volume=21&amp;publication_year=2013&amp;pages=114-119&amp;pmid=23473629&amp;doi=10.1016/j.tim.2012.11.001&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B50\">50. <span class=\"mixed-citation\">Allen HK, Trachsel J, Looft T, Casey TA. \n2014. \n<span class=\"ref-title\">Finding alternatives to antibiotics</span>. <span class=\"ref-journal\">Ann N Y Acad Sci</span>\n<span class=\"ref-vol\">1323</span>:91&#x02013;100. doi:10.1111/nyas.12468. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24953233\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fnyas.12468\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Ann+N+Y+Acad+Sci&amp;title=Finding+alternatives+to+antibiotics&amp;author=HK+Allen&amp;author=J+Trachsel&amp;author=T+Looft&amp;author=TA+Casey&amp;volume=1323&amp;publication_year=2014&amp;pages=91-100&amp;pmid=24953233&amp;doi=10.1111/nyas.12468&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B51\">51. <span class=\"mixed-citation\">Holman DB, Ch&#x000e9;nier MR. \n2015. \n<span class=\"ref-title\">Antimicrobial use in swine production and its effect on the swine gut microbiota and antimicrobial resistance</span>. <span class=\"ref-journal\">Can J Microbiol</span>\n<span class=\"ref-vol\">61</span>:785&#x02013;798. doi:10.1139/cjm-2015-0239. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26414105\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1139%2Fcjm-2015-0239\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Can+J+Microbiol&amp;title=Antimicrobial+use+in+swine+production+and+its+effect+on+the+swine+gut+microbiota+and+antimicrobial+resistance&amp;author=DB+Holman&amp;author=MR+Ch&#x000e9;nier&amp;volume=61&amp;publication_year=2015&amp;pages=785-798&amp;pmid=26414105&amp;doi=10.1139/cjm-2015-0239&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B52\">52. <span class=\"mixed-citation\">Caporaso JG, Kuczynski J, Stombaugh J, Bittinger K, Bushman FD, Costello EK, Fierer N, Pe&#x000f1;a AG, Goodrich JK, Gordon JI, Huttley GA, Kelley ST, Knights D, Koenig JE, Ley RE, Lozupone CA, McDonald D, Muegge BD, Pirrung M, Reeder J, Sevinsky JR, Turnbaugh PJ, Walters WA, Widmann J, Yatsunenko T, Zaneveld J, Knight R. \n2010. \n<span class=\"ref-title\">QIIME allows analysis of high-throughput community sequencing data</span>. <span class=\"ref-journal\">Nat Methods</span>\n<span class=\"ref-vol\">7</span>:335&#x02013;336. doi:10.1038/nmeth.f.303. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3156573/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20383131\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnmeth.f.303\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=QIIME+allows+analysis+of+high-throughput+community+sequencing+data&amp;author=JG+Caporaso&amp;author=J+Kuczynski&amp;author=J+Stombaugh&amp;author=K+Bittinger&amp;author=FD+Bushman&amp;volume=7&amp;publication_year=2010&amp;pages=335-336&amp;pmid=20383131&amp;doi=10.1038/nmeth.f.303&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B53\">53. <span class=\"mixed-citation\">Aronesty E.\n2013. \n<span class=\"ref-title\">Comparison of sequencing utility programs</span>. <span class=\"ref-journal\">Open Bioinforma J</span>\n<span class=\"ref-vol\">7</span>:1&#x02013;8. doi:10.2174/1875036201307010001. [<a href=\"//dx.doi.org/10.2174%2F1875036201307010001\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Open+Bioinforma+J&amp;title=Comparison+of+sequencing+utility+programs&amp;author=E+Aronesty&amp;volume=7&amp;publication_year=2013&amp;pages=1-8&amp;doi=10.2174/1875036201307010001&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B54\">54. <span class=\"mixed-citation\">Quast C, Pruesse E, Yilmaz P, Gerken J, Schweer T, Yarza P, Peplies J, Gl&#x000f6;ckner FO. \n2013. \n<span class=\"ref-title\">The SILVA ribosomal RNA gene database project: improved data processing and Web-based tools</span>. <span class=\"ref-journal\">Nucleic Acids Res</span>\n<span class=\"ref-vol\">41</span>:D590&#x02013;D596. doi:10.1093/nar/gks1219. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3531112/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23193283\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgks1219\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+SILVA+ribosomal+RNA+gene+database+project:+improved+data+processing+and+Web-based+tools&amp;author=C+Quast&amp;author=E+Pruesse&amp;author=P+Yilmaz&amp;author=J+Gerken&amp;author=T+Schweer&amp;volume=41&amp;publication_year=2013&amp;pages=D590-D596&amp;pmid=23193283&amp;doi=10.1093/nar/gks1219&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B55\">55. <span class=\"mixed-citation\">Edgar RC.\n2010. \n<span class=\"ref-title\">Search and clustering orders of magnitude faster than BLAST</span>. <span class=\"ref-journal\">Bioinformatics</span>\n<span class=\"ref-vol\">26</span>:2460&#x02013;2461. doi:10.1093/bioinformatics/btq461. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20709691\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtq461\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Search+and+clustering+orders+of+magnitude+faster+than+BLAST&amp;author=RC+Edgar&amp;volume=26&amp;publication_year=2010&amp;pages=2460-2461&amp;pmid=20709691&amp;doi=10.1093/bioinformatics/btq461&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B56\">56. <span class=\"mixed-citation\">Segata N, Izard J, Waldron L, Gevers D, Miropolsky L, Garrett WS, Huttenhower C. \n2011. \n<span class=\"ref-title\">Metagenomic biomarker discovery and explanation</span>. <span class=\"ref-journal\">Genome Biol</span>\n<span class=\"ref-vol\">12</span>:R60. doi:10.1186/gb-2011-12-6-r60. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3218848/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21702898\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2Fgb-2011-12-6-r60\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Metagenomic+biomarker+discovery+and+explanation&amp;author=N+Segata&amp;author=J+Izard&amp;author=L+Waldron&amp;author=D+Gevers&amp;author=L+Miropolsky&amp;volume=12&amp;publication_year=2011&amp;pages=R60&amp;pmid=21702898&amp;doi=10.1186/gb-2011-12-6-r60&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B57\">57. <span class=\"mixed-citation\">Lozupone C, Knight R. \n2005. \n<span class=\"ref-title\">UniFrac: a new phylogenetic method for comparing microbial communities</span>. <span class=\"ref-journal\">Appl Environ Microbiol</span>\n<span class=\"ref-vol\">71</span>:8228&#x02013;8235. doi:10.1128/AEM.71.12.8228-8235.2005. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC1317376/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n16332807\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.71.12.8228-8235.2005\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl+Environ+Microbiol&amp;title=UniFrac:+a+new+phylogenetic+method+for+comparing+microbial+communities&amp;author=C+Lozupone&amp;author=R+Knight&amp;volume=71&amp;publication_year=2005&amp;pages=8228-8235&amp;pmid=16332807&amp;doi=10.1128/AEM.71.12.8228-8235.2005&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B58\">58. <span class=\"mixed-citation\">Bray JR, Curtis JT. \n1957. \n<span class=\"ref-title\">An ordination of the upland forest communities of southern Wisconsin</span>. <span class=\"ref-journal\">Ecol Monogr</span>\n<span class=\"ref-vol\">27</span>:325&#x02013;349. doi:10.2307/1942268. [<a href=\"//dx.doi.org/10.2307%2F1942268\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Ecol+Monogr&amp;title=An+ordination+of+the+upland+forest+communities+of+southern+Wisconsin&amp;author=JR+Bray&amp;author=JT+Curtis&amp;volume=27&amp;publication_year=1957&amp;pages=325-349&amp;doi=10.2307/1942268&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B59\">59. <span class=\"mixed-citation\">V&#x000e1;zquez-Baeza Y, Pirrung M, Gonzalez A, Knight R. \n2013. \n<span class=\"ref-title\">Emperor: a tool for visualizing high-throughput microbial community data</span>. <span class=\"ref-journal\">Gigascience</span>\n<span class=\"ref-vol\">2</span>:16. doi:10.1186/2047-217X-2-16. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4076506/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24280061\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2F2047-217X-2-16\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=Emperor:+a+tool+for+visualizing+high-throughput+microbial+community+data&amp;author=Y+V&#x000e1;zquez-Baeza&amp;author=M+Pirrung&amp;author=A+Gonzalez&amp;author=R+Knight&amp;volume=2&amp;publication_year=2013&amp;pages=16&amp;pmid=24280061&amp;doi=10.1186/2047-217X-2-16&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B60\">60. <span class=\"mixed-citation\">Oksanen J, Blanchet FG, Kindt R, Legendre P, Minchin PR, O&#x02019;hara R, Simpson GL, Solymos P, Stevens MHH, Wagner H. \n2013. \n<span class=\"ref-journal\">Package &#x0201c;vegan&#x0201d;</span>. Community ecology package, version 2\n<a href=\"https://cran.r-project.org/web/packages/vegan/vegan.pdf\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://cran.r-project.org/web/packages/vegan/vegan.pdf</a> <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Package+&#x0201c;vegan&#x0201d;&amp;author=J+Oksanen&amp;author=FG+Blanchet&amp;author=R+Kindt&amp;author=P+Legendre&amp;author=PR+Minchin&amp;publication_year=2013&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B61\">61. <span class=\"mixed-citation\">Shannon CE.\n1948. \n<span class=\"ref-title\">A mathematical theory of communication</span>. <span class=\"ref-journal\">Bell Syst Tech J</span>\n<span class=\"ref-vol\">27</span>:379&#x02013;423. doi:10.1002/j.1538-7305.1948.tb01338.x. [<a href=\"//dx.doi.org/10.1002%2Fj.1538-7305.1948.tb01338.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bell+Syst+Tech+J&amp;title=A+mathematical+theory+of+communication&amp;author=CE+Shannon&amp;volume=27&amp;publication_year=1948&amp;pages=379-423&amp;doi=10.1002/j.1538-7305.1948.tb01338.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B62\">62. <span class=\"mixed-citation\">Faith DP.\n1992. \n<span class=\"ref-title\">Conservation evaluation and phylogenetic diversity</span>. <span class=\"ref-journal\">Biol Conserv</span>\n<span class=\"ref-vol\">61</span>:1&#x02013;10. doi:10.1016/0006-3207(92)91201-3. [<a href=\"//dx.doi.org/10.1016%2F0006-3207(92)91201-3\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Biol+Conserv&amp;title=Conservation+evaluation+and+phylogenetic+diversity&amp;author=DP+Faith&amp;volume=61&amp;publication_year=1992&amp;pages=1-10&amp;doi=10.1016/0006-3207(92)91201-3&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B63\">63. <span class=\"mixed-citation\">Simpson EH.\n1949. \n<span class=\"ref-title\">Measurement of diversity</span>. <span class=\"ref-journal\">Nature</span>\n<span class=\"ref-vol\">163</span>:688&#x02013;688. doi:10.1038/163688a0. [<a href=\"//dx.doi.org/10.1038%2F163688a0\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Measurement+of+diversity&amp;author=EH+Simpson&amp;volume=163&amp;publication_year=1949&amp;pages=688-688&amp;doi=10.1038/163688a0&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B64\">64. <span class=\"mixed-citation\">R Development Core Team \n2013. \n<span class=\"ref-journal\">R: a language and environment for statistical computing</span>. The R Foundation for Statistical Computing, Vienna, Austria: \n<a href=\"http://www.R-project.org/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.R-project.org/</a>. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=R:+a+language+and+environment+for+statistical+computing&amp;publication_year=2013&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B65\">65. <span class=\"mixed-citation\">De Mendiburu F.\n2014. \n<span class=\"ref-journal\">Agricolae: statistical procedures for agricultural research</span>. R package version 1, p\n1&#x02013;6. <a href=\"https://CRAN.R-project.org/package=agricolae\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://CRAN.R-project.org/package=agricolae</a>. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Agricolae:+statistical+procedures+for+agricultural+research&amp;author=F+De+Mendiburu&amp;publication_year=2014&amp;pages=1-6&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B66\">66. <span class=\"mixed-citation\">Guevarra RB, Kim J, Nguyen SG, Unno T. \n2015. \n<span class=\"ref-title\">Comparison of fecal microbial communities between white and black pigs</span>. <span class=\"ref-journal\">J Appl Biol Chem</span>\n<span class=\"ref-vol\">58</span>:369&#x02013;375. doi:10.3839/jabc.2015.058. [<a href=\"//dx.doi.org/10.3839%2Fjabc.2015.058\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Appl+Biol+Chem&amp;title=Comparison+of+fecal+microbial+communities+between+white+and+black+pigs&amp;author=RB+Guevarra&amp;author=J+Kim&amp;author=SG+Nguyen&amp;author=T+Unno&amp;volume=58&amp;publication_year=2015&amp;pages=369-375&amp;doi=10.3839/jabc.2015.058&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B67\">67. <span class=\"mixed-citation\">Hoeflinger JL, Kashtanov DO, Cox SB, Dowd SE, Jouni ZE, Donovan SM, Miller MJ. \n2015. \n<span class=\"ref-title\">Characterization of the intestinal lactobacilli community following galactooligosaccharides and polydextrose supplementation in the neonatal piglet</span>. <span class=\"ref-journal\">PLoS One</span>\n<span class=\"ref-vol\">10</span>:e0135494. doi:10.1371/journal.pone.0135494. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4537252/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26275147\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0135494\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Characterization+of+the+intestinal+lactobacilli+community+following+galactooligosaccharides+and+polydextrose+supplementation+in+the+neonatal+piglet&amp;author=JL+Hoeflinger&amp;author=DO+Kashtanov&amp;author=SB+Cox&amp;author=SE+Dowd&amp;author=ZE+Jouni&amp;volume=10&amp;publication_year=2015&amp;pages=e0135494&amp;pmid=26275147&amp;doi=10.1371/journal.pone.0135494&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B68\">68. <span class=\"mixed-citation\">Kim J, Nguyen SG, Guevarra RB, Lee I, Unno T. \n2015. \n<span class=\"ref-title\">Analysis of swine fecal microbiota at various growth stages</span>. <span class=\"ref-journal\">Arch Microbiol</span>\n<span class=\"ref-vol\">197</span>:753&#x02013;759. doi:10.1007/s00203-015-1108-1. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25832348\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs00203-015-1108-1\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Arch+Microbiol&amp;title=Analysis+of+swine+fecal+microbiota+at+various+growth+stages&amp;author=J+Kim&amp;author=SG+Nguyen&amp;author=RB+Guevarra&amp;author=I+Lee&amp;author=T+Unno&amp;volume=197&amp;publication_year=2015&amp;pages=753-759&amp;pmid=25832348&amp;doi=10.1007/s00203-015-1108-1&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B69\">69. <span class=\"mixed-citation\">Pajarillo EA, Chae JP, Balolong MP, Kim HB, Seo KS, Kang DK. \n2014. \n<span class=\"ref-title\">Pyrosequencing-based analysis of fecal microbial communities in three purebred pig lines</span>. <span class=\"ref-journal\">J Microbiol</span>\n<span class=\"ref-vol\">52</span>:646&#x02013;651. doi:10.1007/s12275-014-4270-2. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25047525\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs12275-014-4270-2\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Microbiol&amp;title=Pyrosequencing-based+analysis+of+fecal+microbial+communities+in+three+purebred+pig+lines&amp;author=EA+Pajarillo&amp;author=JP+Chae&amp;author=MP+Balolong&amp;author=HB+Kim&amp;author=KS+Seo&amp;volume=52&amp;publication_year=2014&amp;pages=646-651&amp;pmid=25047525&amp;doi=10.1007/s12275-014-4270-2&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B70\">70. <span class=\"mixed-citation\">Xian L, Li Y, Jiang Z, Ma J, Jin L, Chen L, Zhou C, Zhang J, Liu Y, Zhu L, Jiang A, Tang G, Xiao J, Wang J, Zeng B, Li M, Li X. \n2014. \n<span class=\"ref-title\">Alterations in cecal microbiota of Jinhua piglets fostered by a Yorkshire sow</span>. <span class=\"ref-journal\">Chin Sci Bull</span>\n<span class=\"ref-vol\">59</span>:4304&#x02013;4311. doi:10.1007/s11434-014-0532-y. [<a href=\"//dx.doi.org/10.1007%2Fs11434-014-0532-y\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Chin+Sci+Bull&amp;title=Alterations+in+cecal+microbiota+of+Jinhua+piglets+fostered+by+a+Yorkshire+sow&amp;author=L+Xian&amp;author=Y+Li&amp;author=Z+Jiang&amp;author=J+Ma&amp;author=L+Jin&amp;volume=59&amp;publication_year=2014&amp;pages=4304-4311&amp;doi=10.1007/s11434-014-0532-y&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div></div><br>', NULL, 2);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(75, 'pubmed', '23203882', 'The PRoteomics IDEntifications (PRIDE) database and associated tools: status in 2013.', 'The PRoteomics IDEntifications (PRIDE, http://www.ebi.ac.uk/pride) database at the European Bioinformatics Institute is one of the most prominent data repositories of mass spectrometry (MS)-based proteomics data. Here, we summarize recent developments in the PRIDE database and related tools. First, we provide up-to-date statistics in data content, splitting the figures by groups of organisms and species, including peptide and protein identifications, and post-translational modifications. We then describe the tools that are part of the PRIDE submission pipeline, especially the recently developed PRIDE Converter 2 (new submission tool) and PRIDE Inspector (visualization and analysis tool). We also give an update about the integration of PRIDE with other MS proteomics resources in the context of the ProteomeXchange consortium. Finally, we briefly review the quality control efforts that are ongoing at present and outline our future plans.', 2013, 'Nucleic acids research', 'PMC3531176', 2, NULL, NULL, 2),
(79, 'pubmed', '33879305', 'Self-reported DSM-5 Anxiety Severity Measures: Evidence of Validity and Reliability in Spanish youths.', 'Very few studies about the psychometric properties of the Anxiety Severity Measures (ASM) proposed in the DSM-5 exist, and none in Spanish-speaking populations. Thus, the aim of the present study was to provide validity and reliability evidence for the Spanish versions of the Agoraphobia, Social Anxiety, Separation Anxiety, Panic, Generalized Anxiety and Specific Phobia Severity measures.Participants included 567 Spaniards (mean age=21.26, SD=3.61; 68.3% women). We performed Exploratory and Confirmatory Factor Analyses to test the structure of the scales, Differential Item Functioning (DIF) by sex, Cronbach\'s alpha and ordinal omega to test reliability, and the Pearson correlations between the ASM and different outcomes to provide evidence for convergent/discriminant (internalizing/externalizing symptoms) and criterion validity (satisfaction, quality of life and personality).Structural analyses supported a one-factor solution for all of the ASM except for the Specific Phobia scale, which was also the only scale that exhibited DIF. Reliability indices ranked from .82 to .93. All six scales showed stronger associations with internalizing than externalizing measures and were also negatively related to criterion measures.The Spanish version of ASM is suitable for assessing anxiety-related symptoms, except the Specific Phobia Scale, which requires further examination.', 2021, 'Psicothema', '', 2, NULL, NULL, 2);

-- --------------------------------------------------------

--
-- Table structure for table `authors`
--

DROP TABLE IF EXISTS `authors`;
CREATE TABLE IF NOT EXISTS `authors` (
  `id_author` int NOT NULL AUTO_INCREMENT,
  `name` varchar(255) NOT NULL,
  `firstname` varchar(255) NOT NULL,
  `position` int DEFAULT NULL,
  PRIMARY KEY (`id_author`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;

-- --------------------------------------------------------

--
-- Table structure for table `author_article`
--

DROP TABLE IF EXISTS `author_article`;
CREATE TABLE IF NOT EXISTS `author_article` (
  `id_author` int NOT NULL,
  `id_article` int NOT NULL,
  KEY `id_author` (`id_author`),
  KEY `id_article` (`id_article`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;

-- --------------------------------------------------------

--
-- Table structure for table `note`
--

DROP TABLE IF EXISTS `note`;
CREATE TABLE IF NOT EXISTS `note` (
  `id_article` int NOT NULL,
  `id_user` int NOT NULL,
  `note` enum('1','2','3','4','5') NOT NULL,
  KEY `id_article` (`id_article`),
  KEY `id_user` (`id_user`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;

--
-- Dumping data for table `note`
--

INSERT INTO `note` (`id_article`, `id_user`, `note`) VALUES
(27, 2, '3'),
(18, 2, '2');

-- --------------------------------------------------------

--
-- Table structure for table `status`
--

DROP TABLE IF EXISTS `status`;
CREATE TABLE IF NOT EXISTS `status` (
  `id_status` int NOT NULL AUTO_INCREMENT,
  `name_status` varchar(15) NOT NULL,
  PRIMARY KEY (`id_status`)
) ENGINE=InnoDB AUTO_INCREMENT=5 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;

--
-- Dumping data for table `status`
--

INSERT INTO `status` (`id_status`, `name_status`) VALUES
(1, 'undefined'),
(2, 'to_treat'),
(3, 'treat'),
(4, 'reject');

-- --------------------------------------------------------

--
-- Table structure for table `user`
--

DROP TABLE IF EXISTS `user`;
CREATE TABLE IF NOT EXISTS `user` (
  `id_user` int NOT NULL AUTO_INCREMENT,
  `name_user` varchar(15) NOT NULL,
  `email` varchar(255) DEFAULT NULL,
  `password` varchar(200) DEFAULT NULL,
  `profile` enum('expert','assistant') DEFAULT NULL,
  PRIMARY KEY (`id_user`)
) ENGINE=InnoDB AUTO_INCREMENT=24 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;

--
-- Dumping data for table `user`
--

INSERT INTO `user` (`id_user`, `name_user`, `email`, `password`, `profile`) VALUES
(1, 'Nicolas', 'nicolas@mail.com', '1234', NULL),
(2, 'Eddy IKHLEF', 'eddy.ikhlef@protonmail.com', '$2y$10$HWiiDwRMwF06bRvN0k2EdO3Y1/EqosajkHcyIAzd3Ylsfp1RkJZqe', 'expert'),
(3, 'Vincent', 'vincent@mail.com', '1234', NULL),
(4, 'Marie-Line', 'marieline@mail.com', '1234', NULL),
(5, 'Pedro', 'pedro@gmail.com', '1234', NULL),
(6, 'thierry', 'thierry.galliano@etu.univ-amu.fr', '$2y$10$R0HIU0GkdXConABOnNuTZePLhGu7bgMqLGR/aZHbkMoSCdo6jG4Em', 'expert');

--
-- Constraints for dumped tables
--

--
-- Constraints for table `article`
--
ALTER TABLE `article`
  ADD CONSTRAINT `article_ibfk_1` FOREIGN KEY (`status`) REFERENCES `status` (`id_status`);

--
-- Constraints for table `author_article`
--
ALTER TABLE `author_article`
  ADD CONSTRAINT `author_article_ibfk_1` FOREIGN KEY (`id_author`) REFERENCES `authors` (`id_author`),
  ADD CONSTRAINT `author_article_ibfk_2` FOREIGN KEY (`id_article`) REFERENCES `article` (`id_article`);

--
-- Constraints for table `note`
--
ALTER TABLE `note`
  ADD CONSTRAINT `note_ibfk_1` FOREIGN KEY (`id_article`) REFERENCES `article` (`id_article`),
  ADD CONSTRAINT `note_ibfk_2` FOREIGN KEY (`id_user`) REFERENCES `user` (`id_user`);
COMMIT;

/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
