-- phpMyAdmin SQL Dump
-- version 4.9.5deb2
-- https://www.phpmyadmin.net/
--
-- Hôte : localhost:3306
-- Généré le : mar. 18 mai 2021 à 13:46
-- Version du serveur :  10.3.29-MariaDB-0ubuntu0.20.04.1
-- Version de PHP : 7.4.3

SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
SET AUTOCOMMIT = 0;
START TRANSACTION;
SET time_zone = "+00:00";


/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8mb4 */;

--
-- Base de données : `biblio`
--

-- --------------------------------------------------------

--
-- Structure de la table `article`
--

CREATE TABLE `article` (
  `id_article` int(11) NOT NULL,
  `origin` varchar(100) NOT NULL,
  `num_access` varchar(255) NOT NULL,
  `title` text NOT NULL,
  `abstract` text NOT NULL,
  `year` year(4) NOT NULL,
  `journal` varchar(255) NOT NULL,
  `pmcid` varchar(50) DEFAULT NULL,
  `status` int(11) DEFAULT NULL,
  `html_xml` mediumtext DEFAULT NULL,
  `general_note` mediumtext DEFAULT NULL,
  `user` int(11) DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;

--
-- Déchargement des données de la table `article`
--

INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(1, 'pubmed', '33465378', 'Structure-function analysis of pectate lyase Pel3 reveals essential facets of protein recognition by the bacterial type 2 secretion system.', 'The Type II Secretion System (T2SS) transports fully folded proteins of various functions and structures through the outer membrane of Gram-negative bacteria. The molecular mechanisms of substrate recruitment by T2SS remain elusive but a prevailing view is that the secretion determinants could be of a structural nature. The phytopathogenic γ-Proteobacteria, Pectobacterium carotovorum and Dickeya dadantii, secrete similar sets of homologous plant cell wall degrading enzymes, mainly pectinases, by similar T2SSs, called Out. However, the orthologous pectate lyases Pel3 and PelI from these bacteria, which share 67% of sequence identity, are not secreted by the counterpart T2SS of each bacterium, indicating a fine-tuned control of protein recruitment. To identify the related secretion determinants, we first performed a structural characterization and comparison of Pel3 with PelI using X-ray crystallography. Then, to assess the biological relevance of the observed structural variations, we conducted a loop-substitution analysis of Pel3 combined with secretion assays. We showed that there is not one element with a definite secondary structure but several distant and structurally flexible loop regions that are essential for the secretion of Pel3 and that these loop regions act together as a composite secretion signal. Interestingly, depending on the crystal contacts, one of these key secretion determinants undergoes disorder-to-order transitions that could reflect its transient structuration upon the contact with the appropriate T2SS components. We hypothesize that such T2SS-induced structuration of some intrinsically disordered zones of secretion substrates could be part of the recruitment mechanism used by T2SS.', 2021, 'The Journal of biological chemistry', 'PMC7949064', 2, '<h1 class=\"content-title\">Structure&#x02013;function analysis of pectate lyase Pel3 reveals essential facets of protein recognition by the bacterial type 2 secretion system</h1><br><span class=\"contrib-group fm-author\"><a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Pineau%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33465378\" id=\"au1\" class=\"affpopup\" co-rid=\"_co_idm140048951726784\" co-class=\"co-affbox\">Camille Pineau</a>,<sup>1,</sup><sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Guschinskaya%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33465378\" id=\"au2\" class=\"affpopup\" co-rid=\"_co_idm140048945032032\" co-class=\"co-affbox\">Natalia Guschinskaya</a>,<sup>1,</sup><sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Gon%26%23x000e7%3Balves%20IR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33465378\" id=\"au3\" class=\"affpopup\" co-rid=\"_co_idm140048938756320\" co-class=\"co-affbox\">Isabelle R. Gon&#x000e7;alves</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Ruaudel%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33465378\" id=\"au4\" class=\"affpopup\" co-rid=\"_co_idm140048945015936\" co-class=\"co-affbox\">Florence Ruaudel</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Robert%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33465378\" id=\"au5\" class=\"affpopup\" co-rid=\"_co_idm140048951955216\" co-class=\"co-affbox\">Xavier Robert</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Gouet%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33465378\" id=\"au6\" class=\"affpopup\" co-rid=\"_co_idm140048951952992\" co-class=\"co-affbox\">Patrice Gouet</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Ballut%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33465378\" id=\"au7\" class=\"affpopup\" co-rid=\"_co_idm140048942684736\" co-class=\"co-affbox\">Lionel Ballut</a>,<sup>3,</sup><sup>&#x02217;</sup> and  <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Shevchik%20VE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33465378\" id=\"au8\" class=\"affpopup\" co-rid=\"_co_idm140048951327328\" co-class=\"co-affbox\">Vladimir E. Shevchik</a><sup>1,</sup><sup>2,</sup><sup>&#x02217;</sup></span><span style=\"display:none\" class=\"contrib-group aff-tip\"><span id=\"_co_idm140048951726784\"><h3 class=\"no_margin\">Camille Pineau</h3><p><sup>1</sup>Microbiologie Adaptation et Pathog&#x000e9;nie, Univ Lyon, Universit&#x000e9; de Lyon 1, UMR5240 CNRS, Villeurbanne, France</p><p><sup>2</sup>Microbiologie Adaptation et Pathog&#x000e9;nie, Univ Lyon, INSA Lyon, UMR5240, Villeurbanne, France</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Pineau%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33465378\">Camille Pineau</a></span></span><span id=\"_co_idm140048945032032\"><h3 class=\"no_margin\">Natalia Guschinskaya</h3><p><sup>1</sup>Microbiologie Adaptation et Pathog&#x000e9;nie, Univ Lyon, Universit&#x000e9; de Lyon 1, UMR5240 CNRS, Villeurbanne, France</p><p><sup>2</sup>Microbiologie Adaptation et Pathog&#x000e9;nie, Univ Lyon, INSA Lyon, UMR5240, Villeurbanne, France</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Guschinskaya%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33465378\">Natalia Guschinskaya</a></span></span><span id=\"_co_idm140048938756320\"><h3 class=\"no_margin\">Isabelle R. Gon&#x000e7;alves</h3><p><sup>1</sup>Microbiologie Adaptation et Pathog&#x000e9;nie, Univ Lyon, Universit&#x000e9; de Lyon 1, UMR5240 CNRS, Villeurbanne, France</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Gon%26%23x000e7%3Balves%20IR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33465378\">Isabelle R. Gon&#x000e7;alves</a></span></span><span id=\"_co_idm140048945015936\"><h3 class=\"no_margin\">Florence Ruaudel</h3><p><sup>1</sup>Microbiologie Adaptation et Pathog&#x000e9;nie, Univ Lyon, Universit&#x000e9; de Lyon 1, UMR5240 CNRS, Villeurbanne, France</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Ruaudel%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33465378\">Florence Ruaudel</a></span></span><span id=\"_co_idm140048951955216\"><h3 class=\"no_margin\">Xavier Robert</h3><p><sup>3</sup>Molecular Microbiology and Structural Biochemistry, Univ Lyon, UMR5086 CNRS, Lyon, France</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Robert%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33465378\">Xavier Robert</a></span></span><span id=\"_co_idm140048951952992\"><h3 class=\"no_margin\">Patrice Gouet</h3><p><sup>3</sup>Molecular Microbiology and Structural Biochemistry, Univ Lyon, UMR5086 CNRS, Lyon, France</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Gouet%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33465378\">Patrice Gouet</a></span></span><span id=\"_co_idm140048942684736\"><h3 class=\"no_margin\">Lionel Ballut</h3><p><sup>3</sup>Molecular Microbiology and Structural Biochemistry, Univ Lyon, UMR5086 CNRS, Lyon, France</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Ballut%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33465378\">Lionel Ballut</a></span></span><span id=\"_co_idm140048951327328\"><h3 class=\"no_margin\">Vladimir E. Shevchik</h3><p><sup>1</sup>Microbiologie Adaptation et Pathog&#x000e9;nie, Univ Lyon, Universit&#x000e9; de Lyon 1, UMR5240 CNRS, Villeurbanne, France</p><p><sup>2</sup>Microbiologie Adaptation et Pathog&#x000e9;nie, Univ Lyon, INSA Lyon, UMR5240, Villeurbanne, France</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Shevchik%20VE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33465378\">Vladimir E. Shevchik</a></span></span></span></span><span class=\"fm-panel half_rhythm\"><span class=\"togglers<br><span id=\"idm\n0010\" lang=\"en\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm\n0010title\">Abstract</h2><!--article-meta--><span><p id=\"__p1\" class=\"p p-first-last\">The type II secretion system (T2SS) transports fully folded proteins of various functions and structures through the outer membrane of Gram-negative bacteria. The molecular mechanisms of substrate recruitment by T2SS remain elusive but a prevailing view is that the secretion determinants could be of a structural nature. The phytopathogenic &#x003b3;-proteobacteria, <em>Pectobacterium carotovorum</em> and <em>Dickeya dadantii</em>, secrete similar sets of homologous plant cell wall degrading enzymes, mainly pectinases, by similar T2SSs, called Out. However, the orthologous pectate lyases Pel3 and PelI from these bacteria, which share 67% of sequence identity, are not secreted by the counterpart T2SS of each bacterium, indicating a fine-tuned control of protein recruitment. To identify the related secretion determinants, we first performed a structural characterization and comparison of Pel3 with PelI using X-ray crystallography. Then, to assess the biological relevance of the observed structural variations, we conducted a loop-substitution analysis of Pel3 combined with secretion assays. We showed that there is not one element with a definite secondary structure but several distant and structurally flexible loop regions that are essential for the secretion of Pel3 and that these loop regions act together as a composite secretion signal. Interestingly, depending on the crystal contacts, one of these key secretion determinants undergoes disorder-to-order transitions that could reflect its transient structuration upon the contact with the appropriate T2SS components. We hypothesize that such T2SS-induced structuration of some intrinsically disordered zones of secretion substrates could be part of the recruitment mechanism used by T2SS.</p></span><span class=\"sec\"><strong class=\"kwd-title\">Keywords: </strong><span class=\"kwd-text\">type II secretion system (T2SS), protein secretion, bacterial pathogenesis, protein structure, structure&#x02013;function, virulence factor, pectinase, plant pathogen, <em>Dickeya</em>, <em>Pectobacterium</em></span></span><span class=\"sec\"><strong class=\"kwd-title\">Abbreviations: </strong><span class=\"kwd-text\">CD, catalytic domain; GSP, general secretory pathway; IPTG, isopropyl-&#x003b2;-<span class=\"small-caps\">d</span>-thiogalactopyranoside; Pel, pectate lyase; PL-3, polysaccharide lyases of family 3; r.m.s.d., root-mean-square deviation; T2SS, type 2 secretion system; T3SS, type 3 secretion system</span></span></span><span id=\"idm140048952160176\" class=\"tsec sec\"><h2 class=\"headless nomenu\"></h2><p id=\"p0010\" class=\"p p-first\">The Gram-negative bacteria possess a multilayer cell envelope composed of the inner membrane surrounding the cytoplasm and the outer membrane facing the external medium. The two membranes delimit an extracytoplasmic compartment, the periplasm, which contains a peptidoglycan layer (<a href=\"#bib1\" rid=\"bib1\" class=\" bibr popnode\">1</a>, <a href=\"#bib2\" rid=\"bib2\" class=\" bibr popnode\">2</a>). To ensure the selective transport of proteins and other macromolecules across this complex cell envelope, bacteria have evolved several specialized cell machineries (<a href=\"#bib3\" rid=\"bib3\" class=\" bibr popnode\">3</a>, <a href=\"#bib4\" rid=\"bib4\" class=\" bibr popnode\">4</a>, <a href=\"#bib5\" rid=\"bib5\" class=\" bibr popnode\">5</a>, <a href=\"#bib6\" rid=\"bib6\" class=\" bibr popnode\">6</a>). The type II secretion system (T2SS) is widespread among Gram-negative bacteria, and it is used to secrete fully folded proteins, usually lytic enzymes and toxins, from the periplasm into the external medium or host tissue (<a href=\"#bib7\" rid=\"bib7\" class=\" bibr popnode\">7</a>, <a href=\"#bib8\" rid=\"bib8\" class=\" bibr popnode\">8</a>, <a href=\"#bib9\" rid=\"bib9\" class=\" bibr popnode\">9</a>, <a href=\"#bib10\" rid=\"bib10\" class=\" bibr popnode\">10</a>). For instance, the phytopathogenic &#x003b3;-proteobacteria, <em>Pectobacterium carotovorum</em> and <em>Dickeya dadantii</em>, cause soft rot disease in a variety of plants through the action of several pectinases secreted by the T2SS, called Out (<a href=\"#bib11\" rid=\"bib11\" class=\" bibr popnode\">11</a>, <a href=\"#bib12\" rid=\"bib12\" class=\" bibr popnode\">12</a>). The T2SS is a sophisticated transenvelope scaffold that is composed of at least 12 conserved core elements, generically called GspC to GspO (for <span class=\"underline\">G</span>eneral <span class=\"underline\">S</span>ecretory <span class=\"underline\">P</span>athway) or, more specifically, OutC to OutO for <em>Pectobacterium</em> and <em>Dickeya</em>. An inner membrane platform, formed by GspC, L, M, and F, interacts with the cytoplasmic ATPase GspE. GspE is thought to energize the assembly of the periplasmic pseudopilus, composed of GspG, H, I, J, and K, which participates in the translocation of folded exoproteins through the proteinaceous channel formed by the outer membrane secretin GspD (<a href=\"#bib8\" rid=\"bib8\" class=\" bibr popnode\">8</a>, <a href=\"#bib9\" rid=\"bib9\" class=\" bibr popnode\">9</a>, <a href=\"#bib10\" rid=\"bib10\" class=\" bibr popnode\">10</a>).</p><p id=\"p0015\">The folded nature of proteins secreted by T2SS, together with an apparent absence of any common linear sequences, led to a widely held hypothesis that the T2SS secretion determinants could be of a structural nature (<a href=\"#bib13\" rid=\"bib13\" class=\" bibr popnode\">13</a>, <a href=\"#bib14\" rid=\"bib14\" class=\" bibr popnode\">14</a>, <a href=\"#bib15\" rid=\"bib15\" class=\" bibr popnode\">15</a>, <a href=\"#bib16\" rid=\"bib16\" class=\" bibr popnode\">16</a>). The T2SS is very versatile, and depending on the bacteria, it allows the secretion of up to 20 sequence-unrelated and structurally dissimilar proteins (<a href=\"#bib11\" rid=\"bib11\" class=\" bibr popnode\">11</a>, <a href=\"#bib17\" rid=\"bib17\" class=\" bibr popnode\">17</a>, <a href=\"#bib18\" rid=\"bib18\" class=\" bibr popnode\">18</a>, <a href=\"#bib19\" rid=\"bib19\" class=\" bibr popnode\">19</a>). On the other hand, the same T2SS restricts the secretion of very similar orthologous proteins from other species. For example, the T2SS of <em>D.&#x000a0;dadantii</em> secretes more than 15 different proteins, but it can discriminate between its own pectate lyase PelI and the Pel3 from <em>P.&#x000a0;carotovorum</em>, although they share 67% of sequence identity (<a href=\"#bib20\" rid=\"bib20\" class=\" bibr popnode\">20</a>). Recently, we have exploited this species-specific secretion to study the molecular mechanisms of substrate recognition by T2SS (<a href=\"#bib21\" rid=\"bib21\" class=\" bibr popnode\">21</a>).</p><p id=\"p0020\">We have shown that PelI interacts with two T2SS components, the inner membrane GspC and the outer membrane GspD (<a href=\"#bib21\" rid=\"bib21\" class=\" bibr popnode\">21</a>). In addition, we have found that an exposed 9 residue-long region, loop 3 of PelI, acts as a specific secretion signal that controls protein recruitment by the T2SS. The interaction of this loop with the dedicated domains of GspC and GspD is essential for the T2SS to discriminate between the cognate substrate PelI and heterologous Pel3. Furthermore, these data suggest that some other zones of PelI could also be involved in protein recruitment by the T2SS, indicating that this process is more complex than simply the recognition of a single loop (<a href=\"#bib21\" rid=\"bib21\" class=\" bibr popnode\">21</a>).</p><p id=\"p0025\">Previous studies have largely benefited from the high-resolution structures available for the pectate lyase PelI and for the periplasmic domains of the GspC and GspD components (<a href=\"#bib22\" rid=\"bib22\" class=\" bibr popnode\">22</a>, <a href=\"#bib23\" rid=\"bib23\" class=\" bibr popnode\">23</a>, <a href=\"#bib24\" rid=\"bib24\" class=\" bibr popnode\">24</a>, <a href=\"#bib25\" rid=\"bib25\" class=\" bibr popnode\">25</a>). However, the structure of Pel3 from <em>P.&#x000a0;carotovorum</em> was still unresolved. It can be expected that the orthologous Pel3 and PelI would share a similar overall topology, but fine specific structural features may also exist since Pel3 is not recognized by the T2SS of <em>D.&#x000a0;dadantii</em>. Therefore, we undertook a structural characterization of Pel3 to identify such potential secretion determinants and to carry out a rational design and construction of the Pel3 variants that could be secreted by <em>D.&#x000a0;dadantii</em>.</p><p id=\"p0030\" class=\"p\">We reveal that Pel3 shares the same general topology as PelI, consisting of an N-terminal domain of a fibronectin type III fold (Fn3) linked to a catalytic domain (CD) that adopts a parallel &#x003b2;-helix topology. Whereas the core structure of both Fn3 and CD is well conserved in Pel3 and PelI, significant differences were observed for several exposed loop regions, indicating that these are putative secretion determinants. To test this hypothesis, we systematically substituted such divergent zones of Pel3 with those from PelI and then assessed the secretion of the generated hybrids in <em>D.&#x000a0;dadantii</em>. We have demonstrated that, in addition to loop 3 of Fn3, several other loop regions are essential for secretion. Some of them are spatially close to the loop 3 and could together constitute a composite secretion determinant, whereas others are more distant and may act as independent secretion signals. These data suggest that the proteins secreted by T2SS pass through a multifaceted control that monitors the adequacy of several secretion determinants. Remarkably, structural analysis of Pel3 reveals that the key secretion signal, loop 3 of Fn3, is present in the crystals in different conformations. Such conformational transitions could reflect a transient structuration of loop 3 when in contact with an appropriate T2SS component. We hypothesize that such T2SS-induced structuration of some intrinsically disordered zones of the protein to be secreted is part of the recruitment mechanism used by T2SS.</p></span><span id=\"s\n1\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\n1title\">Results</h2><span id=\"s\n1.1\" class=\"sec sec-first\"><h3 id=\"s\n1.1title\">Overall fold of the pectate lyase Pel3</h3><p id=\"p0035\" class=\"p p-first-last\">We crystallized the full-length Pel3 in two types of monoclinic crystals with one or two monomers in the asymmetric unit, named Pel3<sub>1m</sub> and Pel3<sub>2m</sub>, and solved the structures at 1.8 and 2.1&#x000a0;&#x000c5; resolution, respectively (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Table&#x000a0;S1</a>). The structures are very similar, with the same fold and domain arrangement, as testified by an r.m.s. deviation of 1.6&#x000a0;&#x000c5; on all C&#x003b1; pairs. Pel3 adopts a compact pear-like overall shape composed of two unequal domains, a small fibronectin type III domain (Fn3) (residues 1&#x02013;109) and a large catalytic domain that has a &#x003b2;-helix fold (residues 120&#x02013;347) (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig1\" rid-ob=\"ob-fig1\" co-legend-rid=\"lgnd_fig1\"><span>Fig.&#x000a0;1</span></a>). The two domains are linked by a decapeptide segment (residues 110&#x02013;119), which is observed in the electron density map of Pel3<sub>1m</sub>, and have the same respective orientation in Pel3<sub>1m</sub> and Pel3<sub>2m</sub>. When superimposed onto the structure of the orthologous PelI from <em>D.&#x000a0;dadantii</em>, the same domain composition and overall shape are observed, with an r.m.s. deviation of 2&#x000a0;&#x000c5; on all C&#x003b1; pairs (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig1\" rid-ob=\"ob-fig1\" co-legend-rid=\"lgnd_fig1\"><span>Fig.&#x000a0;1</span></a><em>A</em>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"fig1\" co-legend-rid=\"lgnd_fig1\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig1/\" target=\"figure\" rid-figpopup=\"fig1\" rid-ob=\"ob-fig1\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140048993371984\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7949064_gr1.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is gr1.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7949064/bin/gr1.jpg\" /></a></span><span id=\"largeobj_idm140048993371984\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig1/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_fig1\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig1/\" target=\"figure\" rid-figpopup=\"fig1\" rid-ob=\"ob-fig1\">Figure&#x000a0;1</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p2\"><strong>Structure-sequence comparison of the pectate lyases Pel3 and PelI.</strong><em>A</em>, superimposition of Pel3<sub>1m</sub> (<em>blue</em>, PDB 4U4B) and PelI (<em>orange</em>, PDB 3B4N) structures in ribbon representation. Bound calcium and sulfate ions are shown for Pel3. The disordered interdomain linker is not visible in PelI structure and is depicted by a <em>dashed line</em>. <em>B</em>, structure-based sequence alignment of Pel3 and PelI with secondary structure elements. The identical and similar residues are shown on <em>red</em> background.</p></span></span></span></span><span id=\"s\n1.2\" class=\"sec\"><h3 id=\"s\n1.2title\">Organization of the catalytic domain</h3><p id=\"p0040\" class=\"p p-first\">The pectate lyases (EC 4.2.2.2 and EC 4.2.2.9) catalyze the cleavage of polymeric &#x003b1;-1-4-linked polygalacturonic acid within the pectin component of the plant cell wall, leaving an unsaturated C<sub>4</sub>&#x02013;C<sub>5</sub> bond at the newly formed nonreducing end (<a href=\"#bib24\" rid=\"bib24\" class=\" bibr popnode\">24</a>). Pectate lyases have been classified into five structurally and phylogenetically unrelated families of polysaccharide lyases (PL-1, 2, 3, 9, and 10) (<a href=\"http://www.cazy.org\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">www.cazy.org</a>, (<a href=\"#bib26\" rid=\"bib26\" class=\" bibr popnode\">26</a>)). The catalytic domain of Pel3 adopts the general &#x003b2;-helix fold typical of the structurally characterized proteins from the PL-3 family: it is made up of eight right-handed coils stacked on top of one another. Each coil consists of three consecutive strand-turn motifs, termed PBn.m-Tn.m, where n is 1 to 3 and m is 1 to 8 (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig1\" rid-ob=\"ob-fig1\" co-legend-rid=\"lgnd_fig1\"><span>Fig.&#x000a0;1</span></a>). Five disulfide bonds are well conserved between PelI and Pel3 (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S1</a>): three of them (124/137, #1, 180/185, #3, and 312/337, #5) reinforce the stability of the &#x003b2;-solenoid fold while the two other disulfide bonds (146/196, #2 and 257/260, #4) fasten the positions of the extended loops. The hydrophobic core of the Pel3 &#x003b2;-helix is stabilized by a series of hydrophobic interactions between the inward-pointing side chains of several aliphatic residues, Ile, Val, and Leu. These residues are organized into two regular ladders, extending along the whole length of the &#x003b2;-helix, at the &#x003b2;-strands PB1 and PB3, respectively (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S2</a>). These aliphatic interstrand stackings are well conserved among the PL-3 family members, while in some proteins, Phe is present instead of aliphatic residues (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\" co-legend-rid=\"lgnd_fig2\"><span>Fig.&#x000a0;2</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"fig2\" co-legend-rid=\"lgnd_fig2\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig2/\" target=\"figure\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140048945210400\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7949064_gr2.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is gr2.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7949064/bin/gr2.jpg\" /></a></span><span id=\"largeobj_idm140048945210400\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig2/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_fig2\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig2/\" target=\"figure\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\">Figure&#x000a0;2</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p3\"><strong>Family sequence alignment of PL-3 representative examples</strong>. Shown are: <em>P.&#x000a0;carotovorum</em> Pel3 (<a id=\"__tag_931268556\" class=\"tag_hotlink\" href=\"/protein/Q47465\" ref=\"/protein/Q47465\"><span class=\"tag-json\" style=\"display:none\">{\"type\":\"entrez-protein\",\"attrs\":{\"text\":\"Q47465\",\"term_id\":\"75347342\",\"term_text\":\"Q47465\"}}</span>Q47465</a>); <em>D.&#x000a0;dadantii</em> PelI (<a id=\"__tag_931268560\" class=\"tag_hotlink\" href=\"/protein/O50325\" ref=\"/protein/O50325\"><span class=\"tag-json\" style=\"display:none\">{\"type\":\"entrez-protein\",\"attrs\":{\"text\":\"O50325\",\"term_id\":\"75426367\",\"term_text\":\"O50325\"}}</span>O50325</a>); <em>Heterodera schachtii</em> Pel1 (A3F5C0); <em>H.&#x000a0;schachtii</em> Pel2 (A3F5B9); <em>Clavibacter michiganensis</em> Pel (A0A2S5VX08); <em>Bursaphelenchus xylophilus</em> BxPel1 (<a id=\"__tag_931268559\" class=\"tag_hotlink\" href=\"/protein/Q33CQ4\" ref=\"/protein/Q33CQ4\"><span class=\"tag-json\" style=\"display:none\">{\"type\":\"entrez-protein\",\"attrs\":{\"text\":\"Q33CQ4\",\"term_id\":\"122128956\",\"term_text\":\"Q33CQ4\"}}</span>Q33CQ4</a>); <em>Meloidogyne incognita</em> Pel (<a id=\"__tag_931268558\" class=\"tag_hotlink\" href=\"/protein/Q7YW99\" ref=\"/protein/Q7YW99\"><span class=\"tag-json\" style=\"display:none\">{\"type\":\"entrez-protein\",\"attrs\":{\"text\":\"Q7YW99\",\"term_id\":\"74923669\",\"term_text\":\"Q7YW99\"}}</span>Q7YW99</a>); <em>Actinoplanes awajinensis</em> Pel (A0A0X3V446); <em>Verticillium dahliae</em> (G2WR80); <em>Bacillus sp.</em> KSM-P15 Pel (<a id=\"__tag_931268555\" class=\"tag_hotlink\" href=\"/protein/Q9RHW0\" ref=\"/protein/Q9RHW0\"><span class=\"tag-json\" style=\"display:none\">{\"type\":\"entrez-protein\",\"attrs\":{\"text\":\"Q9RHW0\",\"term_id\":\"75420417\",\"term_text\":\"Q9RHW0\"}}</span>Q9RHW0</a>) and <em>Pectobac</em><em>terium carotovorum</em> HrpW (A0A221TA52), representative of various phylogenetic groups (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig4\" rid-ob=\"ob-fig4\" co-legend-rid=\"lgnd_fig4\"><span>Fig. 5</span></a>). In brackets are the corresponding UniProtKB codes. The secondary structure elements are shown for <em>P.&#x000a0;carotovorum</em> Pel3 (PDB 4U4B), <em>D.&#x000a0;dadantii</em> PelI (PDB 3B4N), and <em>Bacillus sp.</em> KSM-P15 Pel (PDB 1EE6), in <em>blue</em>, <em>orange</em>, and <em>green</em>, respectively. The residue numbering is shown for Pel3. The residues conserved in the PL-3 family members are in <em>red</em>. The residues of two hydrophobic ladders and an Asn stacking are shown with the <em>orange</em>, <em>red</em>, and <em>green triangles</em>, respectively. <em>Red asterisks</em> indicate the catalytic residues. The cysteine residues are highlighted in <em>yellow</em> and the disulfide bonds of Pel3 are numbered from 1 to 5 with <em>green numbers</em>. The figure was generated with the ESPript server (<a href=\"#bib57\" rid=\"bib57\" class=\" bibr popnode\">57</a>).</p></span></span></span><p id=\"p0045\">The Pel3 catalytic site is nearly identical to that of PelI and similar to those of two other structurally characterized PL-3 members from <em>Bacillus</em> sp. KSM-P15 and from <em>Caldicellulosiruptor bescii</em> (PDB entries 1EE6 and 3T9G, respectively) (<a href=\"#bib27\" rid=\"bib27\" class=\" bibr popnode\">27</a>, <a href=\"#bib28\" rid=\"bib28\" class=\" bibr popnode\">28</a>). It carries the same invariant residues with Lys227 acting as the catalytic base and Lys252 and Arg255 implicated in the binding of the substrate (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\" co-legend-rid=\"lgnd_fig2\"><span>Fig.&#x000a0;2</span></a> and <a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S3</a>). In the catalytic site of Pel3<sub>1m</sub>, a sulfate ion was detected, which forms salt bridges with Lys227, Lys252, and Arg255 mimicking the hydroxyl groups of the natural substrate, polygalacturonic acid (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S4</a>). In addition, a structural calcium ion was present in the monomer B of Pel3<sub>2m</sub> where it coordinates the main-chain carbonyl O of Ile195 and Pro221, the side-chain O&#x003b4; atom of Asp194 and Asn216, and two water molecules, which complete the pentagonal bipyramidal geometry (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S3</a><em>A</em>). Superimposition of monomers A and B of Pel3<sub>2m</sub> indicated that the Ca<sup>2+</sup> ion seems to be necessary for a proper fold of loop T3.4. Indeed, in the monomer B, T3.4 forms a lid over the bound Ca<sup>2+</sup>, whereas this turn is highly destabilized, in the monomer A, in the absence of calcium (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S5</a>). Interestingly, this arrangement is specific to Pel3 and PelI as the loop forming T3.4 is absent from other pectate lyases of the PL-3 family (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\" co-legend-rid=\"lgnd_fig2\"><span>Fig.&#x000a0;2</span></a> and <a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S3</a>).</p><p id=\"p0050\" class=\"p p-last\">Superimposition of the catalytic domains of Pel3 and PelI reveals some subtle structural differences that are essentially restricted to the extended loops formed by T3.1, T3.2, T3.4, and T1.8 (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3</span></a><em>A</em>). These zones have a low degree of sequence conservation and show variable arrangements in Pel3 and PelI. Notably, these loops are also differently arranged in the various crystal forms of Pel3, indicating their high intrinsic flexibility (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S6</a>). Such properties could be consistent with the formation of specific secretion patterns for cognate T2SS.</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"fig3\" co-legend-rid=\"lgnd_fig3\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/\" target=\"figure\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140048954797344\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7949064_gr3.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is gr3.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7949064/bin/gr3.jpg\" /></a></span><span id=\"largeobj_idm140048954797344\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_fig3\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/\" target=\"figure\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\">Figure&#x000a0;3</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p4\"><strong>Loop substitution analysis between Pel3 and PelI reveals several secretion-relevant protein zones.</strong><em>A</em>, structural variations between Pel3 and PelI loop regions: superimposition of Pel3 (<em>blue</em>) and PelI (<em>orange</em>) in ribbon representation shown in transparency. Loops T1.8, T3.1, T3.2, and T3.4 from the catalytic domain and loops 3 and 5 from the Fn3 domain used for chimeric Pel3/PelI constructs are shown in solid mode. The superimposition is made by using only the catalytic domains as templates. <em>B</em>, schematics of the used Pel3/PelI hybrids and their secretion efficiencies: zones from Pel3 are in <em>blue</em> and those from PelI are in <em>red</em>. The precise amino acid sequences are shown on <a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S10</a>. <em>C</em>, secretion assays with the Pel3/PelI hybrids. The Pel3/PelI variants shown in panel <em>B</em> were expressed from a plasmid in <em>D.&#x000a0;dadantii</em> A5159 <em>pelI</em>. Then, culture supernatants (S) and cells (C) were separated on SDS-PAGE and analyzed by immunoblotting with antibodies raised against pectate lyases PelI and PelD. The chromosome-expressed PelD is used as a positive secretion control to verify if secretion of other T2SS substrates is not affected by the expression of Pel3/PelI hybrids. The histogram shows the percentage of Pel3/PelI hybrids secreted in the culture supernatant. Each bar shows the percentage of secreted hybrid relative to the whole amount of the corresponding hybrid detected in culture supernatant and cells, S/(S&#x000a0;+ C). The bars are positioned below the corresponding gel lines. The bars represent the mean values&#x000a0;&#x000b1; SD of three independent experiments.</p></span></span></span></span><span id=\"s\n1.3\" class=\"sec\"><h3 id=\"s\n1.3title\">Fn3 domain</h3><p id=\"p0055\" class=\"p p-first\">The N-terminal domain of Pel3 has a seven-stranded Fn3 fold very similar to that of the Fn3 domain of PelI, with an r.m.s. deviation of 1.0&#x000a0;&#x000c5; on all C&#x003b1; pairs. It is composed of two antiparallel &#x003b2;-sheets packed against each other (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig1\" rid-ob=\"ob-fig1\" co-legend-rid=\"lgnd_fig1\"><span>Fig.&#x000a0;1</span></a>). Except for Pel3, PelI, and a few close homologs from &#x003b3;-proteobacteria, other characterized representatives of the PL-3 family do not possess an Fn3-like domain. Some PL-3 members carry, instead, a carbohydrate-binding module (Firmicutes and Fungi) or a ricin B-like lectin domain (Actinobacteria), while the Nematode&#x02019;s pectate lyases usually consist of the catalytic&#x000a0;PL-3 domain only (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Table&#x000a0;S2</a>). The Fn3 domain does not affect the catalytic activity of PelI and its biological function remains unclear (<a href=\"#bib24\" rid=\"bib24\" class=\" bibr popnode\">24</a>). Fn3-like modules have also been identified in some other carbohydrate-active enzymes suggesting its possible implication in plant cell wall degradation (<a href=\"#bib29\" rid=\"bib29\" class=\" bibr popnode\">29</a>, <a href=\"#bib30\" rid=\"bib30\" class=\" bibr popnode\">30</a>, <a href=\"#bib31\" rid=\"bib31\" class=\" bibr popnode\">31</a>).</p><p id=\"p0060\" class=\"p p-last\">The Fn3 topology is, however, very common in eukaryotes, and it has been found in about 2% of all animal proteins (<a href=\"#bib32\" rid=\"bib32\" class=\" bibr popnode\">32</a>). Consistent with these observations, a search for structural homologs of Fn3 Pel3, using the Dali server (<a href=\"#bib33\" rid=\"bib33\" class=\" bibr popnode\">33</a>), shows that the highest scoring hits are the Fn3 domains from eukaryotic proteins (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S7</a> and <a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Table&#x000a0;S3</a>). In contrast, a few characterized Fn3 domains from bacterial carbohydrate-active enzymes are structurally divergent from Fn3 Pel3. Regardless of the level of structural similarity of the considered Fn3 domains, their sequence identity is low (less than 15% for the best hits) and only a few aromatic and/or hydrophobic core-forming residues are conserved across these Fn3 domains (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S7</a>). In addition, the loop/turn regions vary widely, even between the orthologous Fn3 domains of Pel3 and PelI where the sequence identity is very poor (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig1\" rid-ob=\"ob-fig1\" co-legend-rid=\"lgnd_fig1\"><span>Fig.&#x000a0;1</span></a><em>B</em> and <a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S7</a>). This could be consistent with the specific functions of these zones. For instance, in some eukaryotic Fn3 domains, loop regions have been shown to constitute binding sites for cognate protein partners (<a href=\"#bib34\" rid=\"bib34\" class=\" bibr popnode\">34</a>, <a href=\"#bib35\" rid=\"bib35\" class=\" bibr popnode\">35</a>, <a href=\"#bib36\" rid=\"bib36\" class=\" bibr popnode\">36</a>).</p></span><span id=\"s\n1.4\" class=\"sec\"><h3 id=\"s\n1.4title\">Varied structuration of the key secretion signal, loop 3 of Fn3 Pel3</h3><p id=\"p0065\" class=\"p p-first\">Comparison of the three monomer structures of Pel3 observed in the different crystals shows that loop 3 of Fn3, located between strands &#x003b2;C and &#x003b2;C&#x02019;, adopts distinct conformations depending on the intra- and intermolecular contacts. Notably, in the Pel3<sub>2m</sub> monomer A, no electronic density corresponding to loop 3 could be detected, indicating that it is completely disordered (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig4\" rid-ob=\"ob-fig4\" co-legend-rid=\"lgnd_fig4\"><span>Fig. 4</span></a><em>A</em>). However, monomer B presents a well-structured loop with a 3<sub>10</sub>-helix (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig4\" rid-ob=\"ob-fig4\" co-legend-rid=\"lgnd_fig4\"><span>Fig.&#x000a0;4</span></a><em>B</em>). The helix seems to be stabilized by Q59, which is sandwiched between two tryptophan residues, W89 from the same monomer B and W89&#x02217; from monomer A. A similar conformation with a 3<sub>10</sub>-helix is observed in Pel3<sub>1m</sub>, where loop 3 is stabilized, by two hydrogen bonds, with another monomer and by a polar-&#x003c0; interaction between Q59 and W89 in the same monomer (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig4\" rid-ob=\"ob-fig4\" co-legend-rid=\"lgnd_fig4\"><span>Fig. 4</span></a><em>C</em> and <a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S8</a><em>B</em>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"fig4\" co-legend-rid=\"lgnd_fig4\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig4/\" target=\"figure\" rid-figpopup=\"fig4\" rid-ob=\"ob-fig4\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140048954484672\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7949064_gr4.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is gr4.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7949064/bin/gr4.jpg\" /></a></span><span id=\"largeobj_idm140048954484672\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig4/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_fig4\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig4/\" target=\"figure\" rid-figpopup=\"fig4\" rid-ob=\"ob-fig4\">Figure&#x000a0;4</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p5\"><strong>Variable structural organization of loop 3 Fn3 and interdomain linker in Pel3.</strong><em>A</em>&#x02013;<em>C</em>, organization of the loop 3 Fn3 observed respectively, in Pel3<sub>2m</sub> (PDB 4U49) monomer A (<em>A</em>) and monomer B (<em>B</em>) and Pel3<sub>1m</sub> (PDB 4U4B) (<em>C</em>). <em>D</em>&#x02013;<em>F</em>, close-up view of the protein zones surrounded in <em>A</em>&#x02013;<em>C</em>, respectively, showing crystal contacts with neighboring Pel3 monomers (in <em>green</em>) as well as phi and psi angles of residues Gln59, Ala60, and Gly61 forming a 3<sub>10</sub>-helix of Pel3 loop&#x000a0;3.</p></span></span></span><p id=\"p0070\" class=\"p p-last\">Interestingly, a similar cation&#x02013;&#x003c0; interaction is observed in the crystal of PelI, where R60 interacts with W87 and the loop 3 Fn3 is neatly ordered in a 3<sub>10</sub>-helix (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S8</a><em>A</em>). In PelI, the 3<sub>10</sub>-helix is additionally stabilized by a salt bridge between D58 and K328&#x02217;, as well by a hydrogen bond between N56 and Y325&#x02217;. Since the loop 3 of Fn3 has been shown to interact with the cognate T2SS (<a href=\"#bib21\" rid=\"bib21\" class=\" bibr popnode\">21</a>), it is tempting to hypothesize that such transient interactions with the T2SS components could play a similar structuring role during the Pel3/PelI recruitment by the secretion system.</p></span><span id=\"s\n1.5\" class=\"sec\"><h3 id=\"s\n1.5title\">Fn3/CD interdomain interface</h3><p id=\"p0075\" class=\"p p-first\">The two domains of Pel3 form a closed structure with a buried surface area of 880&#x000a0;&#x000c5;<sup>2</sup> stabilized by a series of hydrogen bonds and ionic interactions (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S9</a>). The Fn3 side of the interdomain interface includes several residues of strands &#x003b2;A, &#x003b2;B, and &#x003b2;E. On the CD side of the interface, the residues Asn166, Asn189, and Asn243 form a &#x0201c;Velcro&#x0201d;-like motif that offers a series of ionic and polar groups stabilizing the Fn3/CD interface. These residues form the turns T2 at coils 2, 3, and 5 of the &#x003b2;-helix, and they are well conserved among the PL-3 family members as a part of an Asn stacking (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\" co-legend-rid=\"lgnd_fig2\"><span>Fig.&#x000a0;2</span></a> and <a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S2</a>). In Pel3 and PelI, the Fn3 domain covers these residues of the CD until the protein is secreted from the periplasm by the T2SS. Once secreted <em>in planta</em>, the Fn3 domain of Pel3/PelI is cleaved off from the catalytic domain by the bacterial proteases (<a href=\"#bib37\" rid=\"bib37\" class=\" bibr popnode\">37</a>).</p><p id=\"p0080\" class=\"p p-last\">A similar compact organization is observed for PelI, with an interdomain buried contact surface of 750&#x000a0;&#x000c5;<sup>2</sup> (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S9</a>). The interdomain interface has a similar orientation in Pel3 and PelI and most interacting residue pairs are conserved, though some specific contacts are more prominent in PelI. For instance, a salt bridge K29/E190 is present in the PelI interface but absent in Pel3 (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S9</a>). These subtle differences in organization of the interface cause some displacements of the Fn3 and CD domains relative to each other, in the Pel3 and PelI core structures. Indeed, when superimposing the catalytic domains of Pel3 and PelI, their respective Fn3 domains and, more specifically, the loops 3 and 5 are obviously shifted (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3</span></a><em>A</em>). Since the loop 3 of Fn3 has been previously identified as a key secretion signal, these differences could, in turn, affect recognition of the protein by the T2SS.</p></span><span id=\"s\n1.6\" class=\"sec\"><h3 id=\"s\n1.6title\">Substitution analysis of the Fn3 domain</h3><p id=\"p0085\" class=\"p p-first-last\">Comparison of Pel3 and PelI revealed that their core structures composed of &#x003b2;-strands are highly conserved, whereas the arrangement of several extended loop regions of both Fn3 and CD varies significantly (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3</span></a><em>A</em>). To examine whether these regions could act as specific secretion signals, first, we systematically substituted these zones of Fn3 Pel3 with those from PelI and then assessed secretion of the generated hybrids in <em>D.&#x000a0;dadantii</em>. As was expected from the previous study (<a href=\"#bib21\" rid=\"bib21\" class=\" bibr popnode\">21</a>), the h16 hybrid, carrying the Fn3 of Pel3 and the CD of PelI, was not secreted by <em>D.&#x000a0;dadantii</em> (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3</span></a>, <em>B</em> and <em>C</em>). Substitution of the loop 3 slowly improved secretion of the resulting hybrid h16-3 (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3</span></a>, <em>B</em> and <em>C</em>), indicating the importance of this loop but suggesting that some other zones&#x000a0;of Fn3 are also necessary for secretion. Additional substitutions of loop 2 and 4 did not have any visible effect (h16-3/2 and h16-3/4) but the introduction of the loop 5 from PelI significantly enhanced secretion of the resulting hybrid h16-3/5 (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3</span></a>, <em>B</em> and <em>C</em>). Since the substitution of loop 5 alone (h16-5) improved secretion at a similar moderate level as with h16-3, the addition of these two substitutions in h16-3/5 has a clear synergistic effect. The loops 3 and 5 are adjacent in the Fn3 structure and could constitute a continuous binding interface, acting as a joint secretion determinant.</p></span><span id=\"s\n1.7\" class=\"sec\"><h3 id=\"s\n1.7title\">Substitution analysis of the catalytic domain</h3><p id=\"p0090\" class=\"p p-first-last\">To identify potential secretion determinants located at the catalytic domain, a similar loop-substitution strategy was employed. Hybrid h18, carrying a secretion-relevant portion of Fn3 PelI and the CD from Pel3, was barely secreted by <em>D.&#x000a0;dadantii</em>, which corroborates the presence of specific secretion signals within the CD. To search for such presumed secretion determinants, h18 was used as a framework to introduce the selected zones from the PelI CD (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3</span></a>, <em>B</em> and <em>C</em>). Structural comparison of Pel3 and PelI shows several extended loop regions of CD with variable arrangements and sequences, namely T3.1, T3.2, T3.4, and T1.8 (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3</span></a><em>A</em> and <a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S6</a>). The substitution of loop T1.8 obviously improved secretion of the resulting hybrid h18-C (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3</span></a>, <em>B</em> and <em>C</em>, compare h18-C and h18). The subsequent introduction of T3.4 from PelI further enhanced secretion of h18-B/C (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3</span></a>, <em>B</em> and <em>C</em>, compare h18-B/C with h18-C), indicating the importance of these two loops for protein secretion. On the&#x000a0;other hand, the substitution of the turn T3.1 led to the degradation of the resulting hybrid h35 (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3</span></a>, <em>B</em> and <em>C</em>). It seems that insertion of heterologous turn T3.1 could provoke some steric hindrances, thus destabilizing the overall protein fold. Collectively, these data show that two elements of the CD, T3.4 and T1.8, could constitute two additional species-specific secretion signals of Pel3/PelI. Furthermore, since these loops are located close enough to each other (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3</span></a><em>A</em>), it is possible that they form a common binding interface with an appropriate T2SS component.</p></span><span id=\"s\n1.8\" class=\"sec\"><h3 id=\"s\n1.8title\">Role of the Fn3/CD interdomain interface</h3><p id=\"p0095\" class=\"p p-first\">The secretion signals identified herein, notably the loops 3 and T1.8, are located at diametrically opposite sides of the Pel3/PelI core structure, respectively, at the Fn3 and the catalytic domain. These two domains interact <em>via</em> a large interdomain interface, which might still disassemble during the course of secretion. In this case, the secretion signals located at the Fn3 and the CD would be displaced with respect to each other. To test this hypothesis, the Fn3/CD interdomain interface of PelI was locked by an artificial disulfide bond. The resulting PelI variant, carrying the double cysteine substitution, T70C/N240C, was efficiently secreted (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S11</a>). Since disulfide bonds are formed in the periplasm prior to protein recruitment by the T2SS (<a href=\"#bib38\" rid=\"bib38\" class=\" bibr popnode\">38</a>), this indicates that such a permanently locked PelI variant is very compatible with the T2SS. It seems plausible, therefore, that during secretion of the native protein, Fn3 and CD could also remain bound together as in the crystal structure. In this way, the reciprocal positioning of various secretion signals with respect to one another (<em>e.g.</em>, loop 3 of Fn3 and loop T1.8 of CD) would be preserved during protein recruitment by the T2SS.</p><p id=\"p0100\" class=\"p p-last\">To determine whether the simultaneous presence of both sets of secretion signals, from Fn3 and CD, is essential for Pel3/PelI secretion, we constructed a truncated PelI, PelIstp, lacking the C-terminal half of the catalytic domain and the related loop T1.8 (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3</span></a>). This derivative was not secreted suggesting that protein recruitment by T2SS necessitates multisite recognition from the Fn3 and CD domains.</p></span><span id=\"s\n1.9\" class=\"sec sec-last\"><h3 id=\"s\n1.9title\">Pel3 phylogeny</h3><p id=\"p0105\" class=\"p p-first\">The catalytic domain of Pel3 belongs to the PL-3 family of polysaccharide lyases found in various groups of bacteria, fungi, oomycetes, and plant-associated nematodes (<a href=\"http://www.cazy.org\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">www.cazy.org</a>, (<a href=\"#bib26\" rid=\"bib26\" class=\" bibr popnode\">26</a>)). The bacterial PL-3 enzymes do not form a monophyletic clade but are interspersed between four distant groups (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig5\" rid-ob=\"ob-fig5\" co-legend-rid=\"lgnd_fig5\"><span>Fig. 5</span></a>). <em>Pectobacterium</em>, <em>Dickeya</em>, and some other plant-pathogenic Proteobacteria possess two phylogenetically distant classes of PL-3, a Pel3/PelI orthologous group and HrpW-like proteins. HrpW is a T3SS component that binds the pectin and facilitates the passage of the T3SS needle through the plant cell wall (<a href=\"#bib39\" rid=\"bib39\" class=\" bibr popnode\">39</a>, <a href=\"#bib40\" rid=\"bib40\" class=\" bibr popnode\">40</a>). The proteobacterial HrpWs carry a threonine in place of the residue equivalent to the K252 of Pel3 (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig5\" rid-ob=\"ob-fig5\" co-legend-rid=\"lgnd_fig5\"><span>Fig. 5</span></a>). Interestingly, the substitutions equivalent to K252T/S/R and R255Q/C have occurred independently in several PL-3 groups (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig5\" rid-ob=\"ob-fig5\" co-legend-rid=\"lgnd_fig5\"><span>Fig. 5</span></a> and <a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Table&#x000a0;S2</a>). The residues equivalent to K252 and R255 of Pel3 are involved in the substrate binding (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S4</a>) and their substitutions could allow for a better adaptation to a particular plant cell wall composition. For instance, the substitution equivalent to K252R permits enzymatic activity at a high concentration of calcium, conditions specific to the middle lamella (<a href=\"#bib24\" rid=\"bib24\" class=\" bibr popnode\">24</a>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"fig5\" co-legend-rid=\"lgnd_fig5\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig5/\" target=\"figure\" rid-figpopup=\"fig5\" rid-ob=\"ob-fig5\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140048954761440\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7949064_gr5.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is gr5.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7949064/bin/gr5.jpg\" /></a></span><span id=\"largeobj_idm140048954761440\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig5/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_fig5\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig5/\" target=\"figure\" rid-figpopup=\"fig5\" rid-ob=\"ob-fig5\">Figure&#x000a0;5</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p6\"><strong>Phylogenetic tree of Pel3 and the PL-3 family proteins.</strong> Phylogenetic tree topology was obtained by using bootstrapped maximum-likelihood approach with PhyML (<a href=\"#bib55\" rid=\"bib55\" class=\" bibr popnode\">55</a>) as described in &#x0201c;<a href=\"#sec4\" rid=\"s\n4\" class=\" sec\">Experimental procedures</a>&#x0201d; and visualized with iTOL software (<a href=\"#bib56\" rid=\"bib56\" class=\" bibr popnode\">56</a>). Bootstrap values &#x02265;70 are indicated below the branches. Colored branches and boxes show the PL-3 protein groups from Bacteria (<em>blue</em>), Nematodes (<em>green</em>), Fungi (<em>orange</em>), and Oomycetes (<em>magenta</em>). <em>Red circles</em> show the occurrence of disulfide bonds, numbered from #1 to 5, as in Pel3. <em>Green</em> and <em>blue triangles</em> indicate the substitutions of substrate-binding residues equivalent to K252T/S and K252R of Pel3, respectively. Full species names and some other features of the proteins used are listed in the <a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Table&#x000a0;S2</a>. The sequences used for family sequence alignment (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\" co-legend-rid=\"lgnd_fig2\"><span>Fig.&#x000a0;2</span></a>) are indicated with the <em>red stars</em>.</p></span></span></span><p id=\"p0110\">HrpW proteins form a large cluster together with the group of PL-3 enzymes originating from Firmicutes and some Proteobacteria (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig5\" rid-ob=\"ob-fig5\" co-legend-rid=\"lgnd_fig5\"><span>Fig. 5</span></a>). These proteins have no, or very occasionally one, disulfide bond, equivalent to #3 of Pel3. In contrast, all the other PL-3 groups possess the disulfide bonds equivalent to #3, 4, and 5 of Pel3 (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig5\" rid-ob=\"ob-fig5\" co-legend-rid=\"lgnd_fig5\"><span>Fig. 5</span></a> and <a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Table&#x000a0;S2</a>). It seems likely that the HrpW/Firmicutes cluster was separated from the other groups before these disulfide bonds evolved in ancestral PL-3 proteins. The proteins of the HrpW/Firmicutes cluster carry a well-ordered, regular &#x003b2;-helix that is further stabilized by an Asn ladder extending along the whole length of the &#x003b2;-helix (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S2</a>). Generally, the interstrand Asn stacking acts as a glue, sticking together neighboring strands, both in parallel &#x003b2;-helices and in amyloid fibrils (<a href=\"#bib41\" rid=\"bib41\" class=\" bibr popnode\">41</a>). Only a part of this Asn ladder is conserved in Pel3 and in many other PL-3 members (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\" co-legend-rid=\"lgnd_fig2\"><span>Fig.&#x000a0;2</span></a> and <a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S2</a>). Furthermore, the seventh and eighth coils of these proteins are significantly deformed by the presence of some additional loops, helices, and strands (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S2</a>). In these proteins, the disulfide bonds #3 and #5 act as a sort of a molecular clasp, reinforcing the basic &#x003b2;-helix framework, while #4 fastens the loop T1.6 that faces the catalytic groove and carries substrate-binding residues (<a href=\"#bib24\" rid=\"bib24\" class=\" bibr popnode\">24</a>). Interestingly, the disulfide bond equivalent to #2 of Pel3 seems to have appeared independently in two distant groups, Fungi/Oomycetes and Proteobacterial PelI/Pel3, respectively (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig5\" rid-ob=\"ob-fig5\" co-legend-rid=\"lgnd_fig5\"><span>Fig. 5</span></a>). This might corroborate the importance of this bond for the stabilization of protein structure (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S5</a>).</p><p id=\"p0115\" class=\"p p-last\">The disulfide bond equivalent to #1 of Pel3 (C124-C136) is only present in Pel3, PelI, and a few very close orthologs from &#x003b3;-proteobacteria, all secreted by the T2SS. These pectate lyases are preceded by an N-terminal Fn3 domain, another exclusive feature of the Pel3/PelI group. The disulfide bond #1 attaches the beginning of the &#x003b2;-solenoid to the interdomain linker, thus maintaining the correct orientation of the Fn3 domain, which is essential for secretion. Finally, another secretion determinant, the extended loop T3.4, is also an exclusive feature of Pel3/PelI-like pectate lyases from &#x003b3;-proteobacteria carrying a T2SS (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\" co-legend-rid=\"lgnd_fig2\"><span>Figs.&#x000a0;2</span></a> and <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig4\" rid-ob=\"ob-fig4\" co-legend-rid=\"lgnd_fig4\"><span style=\"position: relative;text-decoration:none;\">&#x200B;<span class=\"figpopup-sensitive-area\" style=\"left: -1.5em;\">and4).</span></span><span>4</span></a>). These data suggest a possible coevolution of Pel3/PelI orthologs for secretion by the T2SS.</p></span></span><span id=\"s\n2\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\n2title\">Discussion</h2><p id=\"p0120\" class=\"p p-first\">In order to establish the specific determinants of the type 2 secretion, we performed a structural characterization of the pectate lyase Pel3 from <em>P.&#x000a0;carotovorum</em> and compared it with its orthologous counterpart, PelI from <em>D.&#x000a0;dadantii</em>. Then, to assess the biological relevance of the observed structural variations, we carried out a loop-substitution analysis combined with secretion assays.</p><p id=\"p0125\">The loop substitution, or grafting, approach is largely employed with the Fn3 scaffold to engineer artificial nonantibody binders of varying specificity (<a href=\"#bib42\" rid=\"bib42\" class=\" bibr popnode\">42</a>, <a href=\"#bib43\" rid=\"bib43\" class=\" bibr popnode\">43</a>). To generate the binding interface of interest, the loops of selected binding specificity are grafted in place of the native loops into a basic Fn3 framework. We showed the adequacy of this technique with the Fn3 Pel3 domain, in which the two neighboring loops 3 and 5 have been identified as constituting a <em>bona fide</em> secretion signal. This approach also allowed us to identify two loop regions of the catalytic domain (T1.8 and T3.4) acting as another specific secretion determinant(s). Collectively, these data argue for a multisite recognition of Pel3/PelI by the T2SS. In agreement with this proposal, a truncated PelI derivative, lacking the C-terminal half of CD and carrying a reduced set of secretion signals, was not secreted. Some previous studies have also suggested that T2SS substrates possess two or more distantly located secretion signals (<a href=\"#bib13\" rid=\"bib13\" class=\" bibr popnode\">13</a>, <a href=\"#bib15\" rid=\"bib15\" class=\" bibr popnode\">15</a>, <a href=\"#bib44\" rid=\"bib44\" class=\" bibr popnode\">44</a>). In this study, only the elements divergent between the two homologous proteins, Pel3 and PelI, were examined. Therefore, it could not be excluded that some other structural features, such as overall folding or prevalence of certain secondary elements (<em>e.g.</em>, beta-strands) could be important or directly implicated in substrate recruitment by T2SS.</p><p id=\"p0130\">Remarkably, loops 3 of Fn3 and T1.8 of CD are very distant from one another and located on the diametrically opposite sides of the Pel3 (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3</span></a><em>A</em>). Apparently, they could not constitute a continuous binding interface with only one domain of the dedicated T2SS component. Instead, they may be recognized through a coordinated interaction with at least two T2SS components, or domains, in a pincer-like mode. Alternatively, various secretion signals could interact successively with appropriate T2SS components during the course of secretion. In the latter case, the positioning of these signals relative to each other would probably vary. Indeed, it seems possible that during the protein recruitment and passage through the T2SS, the Fn3 and CD domains could move relative to each other, thus displacing the respective secretion signals. However, a covalent attachment of the two domains does not prevent secretion of PelI<sup>70C/240C</sup>, demonstrating that the presumed movements of these two domains, and the related secretion signals, are not essential. This example supports a spatially coordinated pincer-like recognition of the secretion signals located at Fn3 and CD. Interestingly, the superimposition of Pel3 and PelI shows that their respective loops 3 and 5 of Fn3 are apparently displaced in respect to the cognate loop T1.8 of CD (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3</span></a><em>A</em>). This indicates that a lack of Pel3 recognition by the <em>D.&#x000a0;dadantii</em> Out system could be partly due to an improper positioning of these secretion signals relative to each other.</p><p id=\"p0135\">One of the most intriguing observations we made, when comparing the Pel3 and PelI structures, is the presence of a very labile and rare secondary structure in the Fn3 loop3, namely a 3<sub>10</sub>-helix. This helix is located at the exact position that plays a key role in protein recruitment by the T2SS and allows it to discriminate between the Pel3 and PelI. As supporting evidence, when we compared the loop 3 structures in Pel3 and PelI, we observed the presence of an acidic residue in PelI, D58, turned toward the solvent, and replaced by an alanine in Pel3 (A60). This led to the conclusion that the charge difference could be an essential part of the discrimination process. Consistent with this, D58A substitution prevents secretion of PelI by <em>D.&#x000a0;dadantii</em> (<a href=\"#bib21\" rid=\"bib21\" class=\" bibr popnode\">21</a>).</p><p id=\"p0140\">However, a closer inspection of these secondary structures revealed that a more subtle mechanism could be implicated in the recruitment of PelI/Pel3 by cognate T2SS. Indeed, in the different available structures of Pel3, the loop 3 of Fn3 undergoes significant structural variations, from the absence of any stable secondary structure, in the monomer A of Pel3<sub>2m</sub>, to a well-structured 3<sub>10</sub>-helix, in the monomer B of Pel3<sub>2m</sub> and in Pel3<sub>1m</sub> (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig4\" rid-ob=\"ob-fig4\" co-legend-rid=\"lgnd_fig4\"><span>Fig. 4</span></a>). Stabilization of this labile secondary structure seems to be dependent on interactions with protein partners and neighboring residues. Notably, polar&#x02013;&#x003c0; interactions between Q59 and W89 residues stabilize the 3<sub>10</sub>-helix in the monomer B of Pel3<sub>2m</sub> and in Pel3<sub>1m</sub> (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig4\" rid-ob=\"ob-fig4\" co-legend-rid=\"lgnd_fig4\"><span>Fig. 4</span></a>). Therefore, it is tempting to suggest that, in the course of secretion, loop 3 Fn3 could also undergo transient structuration as a result of contact with the appropriate T2SS components.</p><p id=\"p0145\">Interestingly, the Fn3 domain of Pel3/PelI shares more structural similarity with the Fn3 modules from eukaryotic signaling proteins than with those from bacterial carbohydrate-active enzymes. For instance, the Fn3 domains of Pel3 and of human <span class=\"underline\">C</span>ell adhesion molecule <span class=\"underline\">D</span>ownregulated by <span class=\"underline\">O</span>ncogenes (CDO) (PDB 3N1F) adopt very similar overall folds with an r.m.s. deviation of 2.0&#x000a0;&#x000c5; (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S7</a>). Fn3 domains from eukaryotic proteins are involved in various protein&#x02013;protein interactions by combining the residues from two or three neighboring loops in the binding interface. For example, the loops <em>CD</em> and <em>EF</em> of human Fn3 CDO (equivalent to loops 3 and 5 of Fn3 Pel3) together form the partner-binding interface with hedgehog signaling protein, Hh (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S12</a>) (<a href=\"#bib34\" rid=\"bib34\" class=\" bibr popnode\">34</a>, <a href=\"#bib35\" rid=\"bib35\" class=\" bibr popnode\">35</a>). In addition, depending on the contacts, these loops of Fn3 CDO have been visualized either as unstructured or as carrying a 3<sub>10</sub>-helix supporting the functional relevance of such structuration. Here we show that loops 3 and 5 of Fn3 Pel3 also act together and could constitute a common binding interface with a cognate T2SS component. Taking into account a contact-dependent structuration of the loop 3 Fn3 Pel3 into a 3<sub>10</sub>-helix, these exciting structural similarities suggest some parallels in the interaction mechanisms of these Fn3 domains. Of note is that the eukaryotic Fn3 domains often bind disordered regions of interacting proteins (<a href=\"#bib34\" rid=\"bib34\" class=\" bibr popnode\">34</a>, <a href=\"#bib35\" rid=\"bib35\" class=\" bibr popnode\">35</a>, <a href=\"#bib36\" rid=\"bib36\" class=\" bibr popnode\">36</a>). Intriguingly, many T2SS components carry disordered regions in the key positions and their disorder-to-order transitions seem to be functionally relevant (<a href=\"#bib8\" rid=\"bib8\" class=\" bibr popnode\">8</a>). Finally, the two other secretion determinants identified in the catalytic domain of Pel3/PelI, namely T3.4 and T1.8, are also variable loop regions, extended over the conserved &#x003b2;-helix domain, and readily accessible for any external contact with cognate T2SS components.</p><p id=\"p0150\" class=\"p p-last\">Interestingly, inspection of high-resolution structures of some other T2SS-secreted proteins reveals the presence of some structurally variable zones on otherwise stably folded proteins. For instance, pectate lyase PelC, secreted by the T2SS of <em>Dickeya</em>, is composed of a single &#x003b2;-helix domain decorated with several short &#x003b1;-helices and loops. PelC belongs to the PL-1 family and is sequence dissimilar and unrelated to Pel3/PelI (<a href=\"http://www.cazy.org\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">www.cazy.org</a> (<a href=\"#bib26\" rid=\"bib26\" class=\" bibr popnode\">26</a>)). Comparison of several crystal structures, available for PelC, reveals some important structural transitions of several exposed protein zones namely loop-to-helix and loop-to-strand (<a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Fig.&#x000a0;S13</a>). Such behavior in crystal does not necessarily reflect what could happen <em>in&#x000a0;vivo</em>, but these structural transitions are in line with our hypothesis.</p></span><span id=\"s\n3\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\n3title\">Conclusion</h2><p id=\"p0155\" class=\"p p-first\">The exact nature of the T2S signal is still unclear. Consistent with the highly structured nature of T2S substrates, acquired prior to their recruitment, and the absence of conserved amino acid sequences, a widely shared hypothesis is that such signals could be of a structural nature. However, even the proteins secreted by the same T2SS adopt very different folds, thus complicating the identification of such putative common structural determinants. This study shows that there is not an element with a definite secondary structure but instead there are some unstructured, or intrinsically disordered, segments of the otherwise well-structured protein that could act as a composite secretion signal.</p><p id=\"p0160\" class=\"p p-last\">Indeed, it is tempting to speculate that, as observed with the loop 3 Fn3 in the different crystal forms, a contact-induced transient structuration of such intrinsically disordered loop regions could constitute a general mechanism of protein recruitment by the T2SS. This interesting hypothesis is worthy of further exploration with other secreted proteins.</p></span><span id=\"s\n4\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\n4title\">Experimental procedures</h2><span id=\"s\n4.1\" class=\"sec sec-first\"><h3 id=\"s\n4.1title\">Strains, plasmids, construction of mutant proteins, and secretion assays</h3><p id=\"p0165\" class=\"p p-first-last\"><em>E.&#x000a0;coli</em> NM522 and BL21(DE3) strains (Stratagene) were used for DNA cloning and production of recombinant proteins, respectively. <em>D.&#x000a0;dadantii</em> A3756 <em>pelI:: uidA-nptI</em> (Km<sup>R</sup>) strain (<a href=\"#bib21\" rid=\"bib21\" class=\" bibr popnode\">21</a>) was used for secretion assays. The plasmids and primers used are listed in <a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">Tables&#x000a0;S4</a> and <a href=\"#appsec1\" rid=\"appsec1\" class=\" sec\">S5</a>, respectively. Single and multiple amino acid substitutions were generated by site-directed mutagenesis using PrimeSTAR Max DNA Polymerase (Takara Bio). Sequence of mutant genes was checked in both directions (Eurofins Genomics). To verify if the generated Pel3-PelI hybrid proteins are properly folded and therefore, stable enough in the periplasm, their stability (<em>i.e.</em>, persistence in the cells during a prolonged culture incubation) has been tested in the <em>out</em> minus strain of <em>D.&#x000a0;dadantii</em> by immunoblotting with anti-PelI antibodies that cross-react with Pel3 and Pel3-PelI variants (<a href=\"#bib21\" rid=\"bib21\" class=\" bibr popnode\">21</a>). Only the hybrids, detected at a level higher than that of Pel3, were retained for secretion assays. Secretion of generated Pel3 variants was estimated by immunoblotting as reported (<a href=\"#bib21\" rid=\"bib21\" class=\" bibr popnode\">21</a>). In this order, <em>D.&#x000a0;dadantii pelI</em> mutant cells carrying pBS plasmid with a Pel3 variant of&#x000a0;interest were grown in Luria broth supplemented with 100&#x000a0;&#x003bc;g/ml ampicillin, 0.2% sodium galacturonate, and 0.1% glycerol at 28 &#x000b0;C for 14 to 16&#x000a0;h, till early stationary phase. Culture supernatant containing secreted proteins was separated from cells by centrifugation at 10,000<em>g</em> for 3&#x000a0;min, and both fractions were loaded onto 12% polyacrylamide gel electrophoresis (SDS-PAGE) in an equivalent of 0.04 OD<sub>600</sub>&#x000a0;cells. The proteins were next transferred onto Immobilon P membrane (Merck) and probed with rabbit antisera directed against the pectate lyases PelI and PelD (<a href=\"#bib20\" rid=\"bib20\" class=\" bibr popnode\">20</a>) and secondary goat anti-rabbit antibodies conjugated to peroxidase (Sigma-Aldrich). PelD was used as a positive secretion control to check if secretion of other T2SS substrates is not affected by expression of Pel3/PelI hybrids. All the secretion assays were repeated at least three times, with two biological replicates (independent cultures). Blots were developed with Luminata forte substrate (Millipore) and chemiluminescence signal was recorded and quantified with Vilber fusion FX6 imaging system (Vilber Lourmat) or exposed to Hyperfilm (GE Healthcare). For each Pel3/PelI hybrid, the percentage of secreted protein relative to the sum of the corresponding hybrid protein in culture supernatant and cells was estimated as S/(S&#x000a0;+ C). Six to 10 scan quantifications from three independent experiments were performed with each hybrid, and the mean values&#x000a0;&#x000b1; SD were shown on <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7949064/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Figure&#x000a0;3</span></a><em>C</em>.</p></span><span id=\"s\n4.2\" class=\"sec\"><h3 id=\"s\n4.2title\">Protein purification</h3><p id=\"p0170\" class=\"p p-first-last\">The sequence of the mature Pel3 (residues Asp22 to Leu347) was fused to that of the PelB signal peptide onto pET-20b(+) vector (Novagen) and expressed in the periplasm of <em>E.&#x000a0;coli</em> BL21 (DE3) cells (Stratagene). The bacteria were grown in Luria broth supplemented with 150&#x000a0;&#x003bc;g/ml ampicillin at 30 &#x000b0;C to an OD<sub>600</sub> of 0.6 next, induced with 1&#x000a0;mM IPTG (isopropyl-&#x003b2;-<span class=\"small-caps\">d</span>-thiogalactopyranoside), and grown for an additional 3&#x000a0;h. The cells were frozen at&#x000a0;&#x02212;80 &#x000b0;C and the mature Pel3 protein (signal peptide-less) was extracted by three cycles of freezing&#x02013;thawing as described previously (<a href=\"#bib45\" rid=\"bib45\" class=\" bibr popnode\">45</a>, <a href=\"#bib46\" rid=\"bib46\" class=\" bibr popnode\">46</a>). Pel3 was next precipitated with ammonium sulfate (60&#x02013;90% step saturation) for 24&#x000a0;h at 4 &#x000b0;C. After centrifugation at 5000<em>g</em> for 30&#x000a0;min, the protein pellet was solubilized in 20&#x000a0;mM sodium phosphate buffer pH 7, 0.1&#x000a0;mM EDTA, 1.7&#x000a0;M ammonium sulfate (Buffer A) and loaded onto TSKgel-Phenyl column (Tosoh Bioscience) equilibrated with the same buffer. The column was next washed with Buffer A and protein was eluted by applying a gradient of ammonium sulfate from 1.7 to 0.8&#x000a0;M in 20&#x000a0;mM sodium phosphate buffer pH 7, 0.1&#x000a0;mM EDTA. Fractions containing Pel3 were collected and subjected to ion-exchange chromatography using Hi-Trap Sulfopropyl-Sepharose column (GE Healthcare) equilibrated in 20&#x000a0;mM sodium acetate buffer pH 5, 0.1&#x000a0;mM EDTA (Buffer B). The column was washed with buffer B and Pel3 protein was eluted by charge inversion with 20&#x000a0;mM Tris buffer pH 8.8, 0.1&#x000a0;mM EDTA by applying a NaCl gradient from 0 to 100&#x000a0;mM. Pure protein fractions were concentrated with Vivaspin devices (Sartorius) to 20&#x000a0;mg/ml and used in crystallization trials.</p></span><span id=\"s\n4.3\" class=\"sec\"><h3 id=\"s\n4.3title\">Crystallization</h3><p id=\"p0175\" class=\"p p-first-last\">Crystallization screening was carried out at 293K (vapor-diffusion by the sitting-drop method), with commercially available crystallization kits. For screening, a mosquito crystallization robot from TTP Labtech was employed (150&#x000a0;nl&#x000a0;+ 150&#x000a0;nl drops equilibrated against 70&#x000a0;&#x003bc;l). Pel3 crystals suitable for X-ray analysis grew within 3&#x000a0;months either in a solution containing 1&#x000a0;M succinic acid, 0.1&#x000a0;M HEPES pH 7.0, 1% (m/v) PEG MME 2000 (4U4B) or 0.2&#x000a0;M lithium acetate, 20% (w/v) PEG3350 (4U49). For cryoprotection, crystals were soaked in a reservoir solution supplemented with 15% (v/v) ethylene glycol during 2 to 5&#x000a0;min. X-ray diffraction data were collected at beamline X06DA at the SLS (Paul Scherrer Institute, Switzerland), at a wavelength of 0.979&#x000a0;&#x000c5; for 4U49 (Pel3<sub>2m</sub>), or at beamline ID29 at the ESRF (Grenoble, France), at a wavelength of 0.939&#x000a0;&#x000c5; for 4U4B (Pel3<sub>1m</sub>). Pel3 crystals diffracted X-rays to 2.10&#x000a0;&#x000c5; resolution for 4U4B and 1.80&#x000a0;&#x000c5; resolution for 4U49. They were both indexed and scaled with the program XDS. The crystal structure of Pel3 was solved by the molecular replacement method with the program PHENIX-AutoBuild (<a href=\"#bib47\" rid=\"bib47\" class=\" bibr popnode\">47</a>) using the refined structure of PelI from <em>D.&#x000a0;dadantii</em> (PDB 3B4N) as search model. The structure was then refined with the program PHENIX-Phaser and visualized with COOT software (<a href=\"#bib48\" rid=\"bib48\" class=\" bibr popnode\">48</a>). The refined structure was validated with PROCHECK (<a href=\"#bib49\" rid=\"bib49\" class=\" bibr popnode\">49</a>) before depositing.</p></span><span id=\"s\n4.4\" class=\"sec sec-last\"><h3 id=\"s\n4.4title\">Phylogenetic analysis</h3><p id=\"p0180\" class=\"p p-first-last\">Candidate homologs were searched at a protein level by using BLAST (<a href=\"#bib50\" rid=\"bib50\" class=\" bibr popnode\">50</a>) and Pel3 as a query sequence. Search was done against the UniProtKB database. Three different BLAST searches were done, respectively, against Bacteria, Fungi, and Nematodes. All sequences that aligned at least on 80% of the length of the Pel3 catalytic domain were kept for multiple sequence alignment. Sequences were then clustered with CD-HIT in order to keep a significant number of sequences for each group. Some other sequences were then manually added, namely two oomycota sequences from NCBI database (entries XP_024585360_1 and XP_024581403_1), since only two oomycota representatives are available in UniprotKB database (September 2020) and three sequences corresponding to the available protein structures from the PL-3 family (entries <a id=\"__tag_931268557\" class=\"tag_hotlink\" href=\"/protein/O50325\" ref=\"/protein/O50325\"><span class=\"tag-json\" style=\"display:none\">{\"type\":\"entrez-protein\",\"attrs\":{\"text\":\"O50325\",\"term_id\":\"75426367\",\"term_text\":\"O50325\"}}</span>O50325</a>, B9MKT4, and <a id=\"__tag_931268554\" class=\"tag_hotlink\" href=\"/protein/Q9RHW0\" ref=\"/protein/Q9RHW0\"><span class=\"tag-json\" style=\"display:none\">{\"type\":\"entrez-protein\",\"attrs\":{\"text\":\"Q9RHW0\",\"term_id\":\"75420417\",\"term_text\":\"Q9RHW0\"}}</span>Q9RHW0</a>, corresponding to the PDB entries 3B4N, 3T9G, and 1EE6, respectively). Multiple sequence alignments were done with MAFFT program (<a href=\"#bib51\" rid=\"bib51\" class=\" bibr popnode\">51</a>) and conserved blocks were selected by using BMGE 1.12 (<a href=\"#bib52\" rid=\"bib52\" class=\" bibr popnode\">52</a>), using the BLOSUM30 (<a href=\"#bib53\" rid=\"bib53\" class=\" bibr popnode\">53</a>) matrix with both programs. In total, 154 sites were kept for further analysis after the character trimming method performed by BMGE (<a href=\"#bib52\" rid=\"bib52\" class=\" bibr popnode\">52</a>). Phylogenetic analyses were performed, with the LG model and a gamma correction, using two approaches: a Bayesian method with mrBayes (<a href=\"#bib54\" rid=\"bib54\" class=\" bibr popnode\">54</a>) and a bootstrapped maximum-likelihood approach with PhyML (<a href=\"#bib55\" rid=\"bib55\" class=\" bibr popnode\">55</a>). By default, 100,000 generations were run when using mrBayes, and we ran 100 bootstrap replicates when using PhyML. Phylogenetic tree was generated and visualized with iTOL software (<a href=\"#bib56\" rid=\"bib56\" class=\" bibr popnode\">56</a>).</p></span></span><span id=\"s\n5\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\n5title\">Data availability</h2><p id=\"p0185\" class=\"p p-first-last\">All data described in the article are contained within the article. Pel3 structure coordinates have been deposited at the Protein Data Bank in Europe (PDBe) database with accession codes 4U49 and 4U4B. Strains and plasmids described in this article are available upon request from Vladimir Shevchik (<a href=\"http://mailto:vladimir.shevchik@insa-lyon.fr\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">vladimir.shevchik@insa-lyon.fr</a>).</p></span><span id=\"s\n1\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\n1title\">Conflict of interest</h2><p id=\"p0195\" class=\"p p-first-last\">The authors declare that they have no conflicts of interest with the contents of this article.</p></span><span id=\"ack0010\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"ack0010title\">Acknowledgments</h2><span class=\"sec\"><p id=\"p0200\">We are grateful to Yvan Rahbe for help with phylogenetic analysis. We thank Guy Condemine and other team members for comments and suggestions. We are grateful to Valerie James for critical review and English proofreading.</p><span id=\"s\n7\" class=\"sec\"><h3 id=\"s\n7title\">Author contributions</h3><p id=\"p0205\" class=\"p p-first-last\">L. B., C. P., and V. E. S. conceived and designed the experiments described in the paper. C. P., N. G., F. R., and V. E. S. performed all the experiments. L. B. and I. G. performed phylogenetic analysis; X. R., P. G., and L. B. performed structural data collection and analysis. L. B. and V. E. S. wrote the article.</p></span><span id=\"s\n8\" class=\"sec sec-last\"><h3 id=\"s\n8title\">Funding and additional information</h3><p id=\"p0210\" class=\"p p-first-last\">This research was supported by funding from CNRS (Centre National de la Recherche Scientifique) and a grant from ANR (Agence Nationale de la Recherche) SYNERGY_T2SS ANR-19-CE11-0020.</p></span></span></span><span id=\"idm140048952160688\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140048952160688title\">Notes</h2><p id=\"misc0010\">Edited by Joseph Jez</p></span><br><div class=\"ref-list-sec sec\" id=\"reference-list\"><div class=\"ref-cit-blk half_rhythm\" id=\"bib1\">1. <span class=\"element-citation\" id=\"sref1\">Silhavy T.J., Kahne D., Walker S. The bacterial cell envelope. <span><span class=\"ref-journal\">Cold Spring Harb. Perspect. Biol. </span>2010;<span class=\"ref-vol\">2</span></span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC2857177/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n20452953\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cold+Spring+Harb.+Perspect.+Biol.&amp;title=The+bacterial+cell+envelope&amp;author=T.J.+Silhavy&amp;author=D.+Kahne&amp;author=S.+Walker&amp;volume=2&amp;publication_year=2010&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib2\">2. <span class=\"element-citation\" id=\"sref2\">Miller S.I., Salama N.R. The gram-negative bacterial periplasm: Size matters. <span><span class=\"ref-journal\">PLoS Biol. </span>2018;<span class=\"ref-vol\">16</span></span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC5771553/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n29342145\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+Biol.&amp;title=The+gram-negative+bacterial+periplasm:+Size+matters&amp;author=S.I.+Miller&amp;author=N.R.+Salama&amp;volume=16&amp;publication_year=2018&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib3\">3. <span class=\"element-citation\" id=\"sref3\">Costa T.R., Felisberto-Rodrigues C., Meir A., Prevost M.S., Redzej A., Trokter M., Waksman G. Secretion systems in gram-negative bacteria: Structural and mechanistic insights. <span><span class=\"ref-journal\">Nat. Rev. Microbiol. </span>2015;<span class=\"ref-vol\">13</span>:343–359.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n25978706\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Microbiol.&amp;title=Secretion+systems+in+gram-negative+bacteria:+Structural+and+mechanistic+insights&amp;author=T.R.+Costa&amp;author=C.+Felisberto-Rodrigues&amp;author=A.+Meir&amp;author=M.S.+Prevost&amp;author=A.+Redzej&amp;volume=13&amp;publication_year=2015&amp;pages=343-359&amp;pmid=25978706&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib4\">4. <span class=\"element-citation\" id=\"sref4\">Sherman D.J., Xie R., Taylor R.J., George A.H., Okuda S., Foster P.J., Needleman D.J., Kahne D. Lipopolysaccharide is transported to the cell surface by a membrane-to-membrane protein bridge. <span><span class=\"ref-journal\">Science. </span>2018;<span class=\"ref-vol\">359</span>:798–801.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC5858563/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n29449493\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Lipopolysaccharide+is+transported+to+the+cell+surface+by+a+membrane-to-membrane+protein+bridge&amp;author=D.J.+Sherman&amp;author=R.+Xie&amp;author=R.J.+Taylor&amp;author=A.H.+George&amp;author=S.+Okuda&amp;volume=359&amp;publication_year=2018&amp;pages=798-801&amp;pmid=29449493&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib5\">5. <span class=\"element-citation\" id=\"sref5\">Ekiert D.C., Bhabha G., Isom G.L., Greenan G., Ovchinnikov S., Henderson I.R., Cox J.S., Vale R.D. Architectures of lipid transport systems for the bacterial outer membrane. <span><span class=\"ref-journal\">Cell. </span>2017;<span class=\"ref-vol\">169</span>:273–285.e217.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC5467742/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n28388411\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Architectures+of+lipid+transport+systems+for+the+bacterial+outer+membrane&amp;author=D.C.+Ekiert&amp;author=G.+Bhabha&amp;author=G.L.+Isom&amp;author=G.+Greenan&amp;author=S.+Ovchinnikov&amp;volume=169&amp;publication_year=2017&amp;pages=273-285.e217&amp;pmid=28388411&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib6\">6. <span class=\"element-citation\" id=\"sref6\">Dong C., Beis K., Nesper J., Brunkan-Lamontagne A.L., Clarke B.R., Whitfield C., Naismith J.H. Wza the translocon for E. coli capsular polysaccharides defines a new class of membrane protein. <span><span class=\"ref-journal\">Nature. </span>2006;<span class=\"ref-vol\">444</span>:226–229.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3315050/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n17086202\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Wza+the+translocon+for+E.+coli+capsular+polysaccharides+defines+a+new+class+of+membrane+protein&amp;author=C.+Dong&amp;author=K.+Beis&amp;author=J.+Nesper&amp;author=A.L.+Brunkan-Lamontagne&amp;author=B.R.+Clarke&amp;volume=444&amp;publication_year=2006&amp;pages=226-229&amp;pmid=17086202&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib7\">7. <span class=\"element-citation\" id=\"sref7\">Cianciotto N.P., White R.C. Expanding role of type II secretion in bacterial pathogenesis and beyond. <span><span class=\"ref-journal\">Infect. Immun. </span>2017;<span class=\"ref-vol\">85</span></span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC5400843/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n28264910\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Infect.+Immun.&amp;title=Expanding+role+of+type+II+secretion+in+bacterial+pathogenesis+and+beyond&amp;author=N.P.+Cianciotto&amp;author=R.C.+White&amp;volume=85&amp;publication_year=2017&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib8\">8. <span class=\"element-citation\" id=\"sref8\">Gu S., Shevchik V.E., Shaw R., Pickersgill R.W., Garnett J.A. The role of intrinsic disorder and dynamics in the assembly and function of the type II secretion system. <span><span class=\"ref-journal\">Biochim. Biophys. Acta. </span>2017;<span class=\"ref-vol\">1865</span>:1255–1266.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n28733198\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Biochim.+Biophys.+Acta&amp;title=The+role+of+intrinsic+disorder+and+dynamics+in+the+assembly+and+function+of+the+type+II+secretion+system&amp;author=S.+Gu&amp;author=V.E.+Shevchik&amp;author=R.+Shaw&amp;author=R.W.+Pickersgill&amp;author=J.A.+Garnett&amp;volume=1865&amp;publication_year=2017&amp;pages=1255-1266&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib9\">9. <span class=\"element-citation\" id=\"sref9\">Korotkov K.V., Sandkvist M., Hol W.G. The type II secretion system: Biogenesis, molecular architecture and mechanism. <span><span class=\"ref-journal\">Nat. Rev. Microbiol. </span>2012;<span class=\"ref-vol\">10</span>:336–351.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3705712/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n22466878\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Microbiol.&amp;title=The+type+II+secretion+system:+Biogenesis,+molecular+architecture+and+mechanism&amp;author=K.V.+Korotkov&amp;author=M.+Sandkvist&amp;author=W.G.+Hol&amp;volume=10&amp;publication_year=2012&amp;pages=336-351&amp;pmid=22466878&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib10\">10. <span class=\"element-citation\" id=\"sref10\">Thomassin J.L., Santos Moreno J., Guilvout I., Tran Van Nhieu G., Francetic O. The trans-envelope architecture and function of the type 2 secretion system: New insights raising new questions. <span><span class=\"ref-journal\">Mol. Microbiol. </span>2017;<span class=\"ref-vol\">105</span>:211–226.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n28486768\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol.&amp;title=The+trans-envelope+architecture+and+function+of+the+type+2+secretion+system:+New+insights+raising+new+questions&amp;author=J.L.+Thomassin&amp;author=J.+Santos+Moreno&amp;author=I.+Guilvout&amp;author=G.+Tran+Van+Nhieu&amp;author=O.+Francetic&amp;volume=105&amp;publication_year=2017&amp;pages=211-226&amp;pmid=28486768&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib11\">11. <span class=\"element-citation\" id=\"sref11\">Hugouvieux-Cotte-Pattat N., Condemine G., Shevchik V.E. Bacterial pectate lyases, structural and functional diversity. <span><span class=\"ref-journal\">Environ. Microbiol. Rep. </span>2014;<span class=\"ref-vol\">6</span>:427–440.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n25646533\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Environ.+Microbiol.+Rep.&amp;title=Bacterial+pectate+lyases,+structural+and+functional+diversity&amp;author=N.+Hugouvieux-Cotte-Pattat&amp;author=G.+Condemine&amp;author=V.E.+Shevchik&amp;volume=6&amp;publication_year=2014&amp;pages=427-440&amp;pmid=25646533&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib12\">12. <span class=\"element-citation\" id=\"sref12\">Hugouvieux-Cotte-Pattat N., Condemine G., Gueguen E., Shevchik V.E.  <span class=\"ref-journal\">eLS.</span> John Wiley &#x00026; Sons, Ltd; New York, NY: 2020. Dickeya plant pathogens; pp. 1–10. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=eLS&amp;author=N.+Hugouvieux-Cotte-Pattat&amp;author=G.+Condemine&amp;author=E.+Gueguen&amp;author=V.E.+Shevchik&amp;publication_year=2020&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib13\">13. <span class=\"element-citation\" id=\"sref13\">Palom&#x000e4;ki T., Pickersgill R., Riekki R., Romantschuk M., Saarilahti H.T. A&#x000a0;putative three-dimensional targeting motif of polygalacturonase (PehA), a protein secreted through the type II (GSP) pathway in Erwinia carotovora. <span><span class=\"ref-journal\">Mol. Microbiol. </span>2002;<span class=\"ref-vol\">43</span>:585–596.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n11929517\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol.&amp;title=A&#x000a0;putative+three-dimensional+targeting+motif+of+polygalacturonase+(PehA),+a+protein+secreted+through+the+type+II+(GSP)+pathway+in+Erwinia+carotovora&amp;author=T.+Palom&#x000e4;ki&amp;author=R.+Pickersgill&amp;author=R.+Riekki&amp;author=M.+Romantschuk&amp;author=H.T.+Saarilahti&amp;volume=43&amp;publication_year=2002&amp;pages=585-596&amp;pmid=11929517&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib14\">14. <span class=\"element-citation\" id=\"sref14\">Filloux A. Secretion signal and protein targeting in bacteria: A biological puzzle. <span><span class=\"ref-journal\">J.&#x000a0;Bacteriol. </span>2010;<span class=\"ref-vol\">192</span>:3847–3849.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC2916383/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n20525826\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.&#x000a0;Bacteriol.&amp;title=Secretion+signal+and+protein+targeting+in+bacteria:+A+biological+puzzle&amp;author=A.+Filloux&amp;volume=192&amp;publication_year=2010&amp;pages=3847-3849&amp;pmid=20525826&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib15\">15. <span class=\"element-citation\" id=\"sref15\">Franceti&#x00107; O., Pugsley A.P. Towards the identification of type II secretion signals in a nonacylated variant of pullulanase from Klebsiella oxytoca. <span><span class=\"ref-journal\">J.&#x000a0;Bacteriol. </span>2005;<span class=\"ref-vol\">187</span>:7045–7055.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC1251600/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n16199575\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.&#x000a0;Bacteriol.&amp;title=Towards+the+identification+of+type+II+secretion+signals+in+a+nonacylated+variant+of+pullulanase+from+Klebsiella+oxytoca&amp;author=O.+Franceti&#x00107;&amp;author=A.P.+Pugsley&amp;volume=187&amp;publication_year=2005&amp;pages=7045-7055&amp;pmid=16199575&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib16\">16. <span class=\"element-citation\" id=\"sref16\">Hirst T.R., Holmgren J. Conformation of protein secreted across bacterial outer membranes: A study of enterotoxin translocation from Vibrio cholerae. <span><span class=\"ref-journal\">Proc. Natl. Acad. Sci. U. S. A. </span>1987;<span class=\"ref-vol\">84</span>:7418–7422.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC299307/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n3478701\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A.&amp;title=Conformation+of+protein+secreted+across+bacterial+outer+membranes:+A+study+of+enterotoxin+translocation+from+Vibrio+cholerae&amp;author=T.R.+Hirst&amp;author=J.+Holmgren&amp;volume=84&amp;publication_year=1987&amp;pages=7418-7422&amp;pmid=3478701&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib17\">17. <span class=\"element-citation\" id=\"sref17\">Sikora A.E., Zielke R.A., Lawrence D.A., Andrews P.C., Sandkvist M. Proteomic analysis of the Vibrio cholerae type II secretome reveals new proteins, including three related serine proteases. <span><span class=\"ref-journal\">J.&#x000a0;Biol. Chem. </span>2011;<span class=\"ref-vol\">286</span>:16555–16566.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3089498/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n21385872\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.&#x000a0;Biol.+Chem.&amp;title=Proteomic+analysis+of+the+Vibrio+cholerae+type+II+secretome+reveals+new+proteins,+including+three+related+serine+proteases&amp;author=A.E.+Sikora&amp;author=R.A.+Zielke&amp;author=D.A.+Lawrence&amp;author=P.C.+Andrews&amp;author=M.+Sandkvist&amp;volume=286&amp;publication_year=2011&amp;pages=16555-16566&amp;pmid=21385872&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib18\">18. <span class=\"element-citation\" id=\"sref18\">White R.C., Cianciotto N.P. Assessing the impact, genomics and evolution of type II secretion across a large, medically important genus: The Legionella type II secretion paradigm. <span><span class=\"ref-journal\">Microb. Genom. </span>2019;<span class=\"ref-vol\">5</span></span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC6617341/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n31166887\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Microb.+Genom.&amp;title=Assessing+the+impact,+genomics+and+evolution+of+type+II+secretion+across+a+large,+medically+important+genus:+The+Legionella+type+II+secretion+paradigm&amp;author=R.C.+White&amp;author=N.P.+Cianciotto&amp;volume=5&amp;publication_year=2019&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib19\">19. <span class=\"element-citation\" id=\"sref19\">Filloux A. Protein secretion systems in Pseudomonas aeruginosa: An essay on diversity, evolution, and function. <span><span class=\"ref-journal\">Front. Microbiol. </span>2011;<span class=\"ref-vol\">2</span>:155.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3140646/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n21811488\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front.+Microbiol.&amp;title=Protein+secretion+systems+in+Pseudomonas+aeruginosa:+An+essay+on+diversity,+evolution,+and+function&amp;author=A.+Filloux&amp;volume=2&amp;publication_year=2011&amp;pages=155&amp;pmid=21811488&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib20\">20. <span class=\"element-citation\" id=\"sref20\">Bouley J., Condemine G., Shevchik V.E. The PDZ domain of OutC and the N-terminal region of OutD determine the secretion specificity of the type II out pathway of Erwinia chrysanthemi. <span><span class=\"ref-journal\">J.&#x000a0;Mol. Biol. </span>2001;<span class=\"ref-vol\">308</span>:205–219.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n11327762\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.&#x000a0;Mol.+Biol.&amp;title=The+PDZ+domain+of+OutC+and+the+N-terminal+region+of+OutD+determine+the+secretion+specificity+of+the+type+II+out+pathway+of+Erwinia+chrysanthemi&amp;author=J.+Bouley&amp;author=G.+Condemine&amp;author=V.E.+Shevchik&amp;volume=308&amp;publication_year=2001&amp;pages=205-219&amp;pmid=11327762&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib21\">21. <span class=\"element-citation\" id=\"sref21\">Pineau C., Guschinskaya N., Robert X., Gouet P., Ballut L., Shevchik V.E. Substrate recognition by the bacterial type II secretion system: More than a simple interaction. <span><span class=\"ref-journal\">Mol. Microbiol. </span>2014;<span class=\"ref-vol\">94</span>:126–140.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n25098941\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol.&amp;title=Substrate+recognition+by+the+bacterial+type+II+secretion+system:+More+than+a+simple+interaction&amp;author=C.+Pineau&amp;author=N.+Guschinskaya&amp;author=X.+Robert&amp;author=P.+Gouet&amp;author=L.+Ballut&amp;volume=94&amp;publication_year=2014&amp;pages=126-140&amp;pmid=25098941&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib22\">22. <span class=\"element-citation\" id=\"sref22\">Korotkov K.V., Johnson T.L., Jobling M.G., Pruneda J., Pardon E., Heroux A., Turley S., Steyaert J., Holmes R.K., Sandkvist M., Hol W.G. Structural and functional studies on the interaction of GspC and GspD in the type II secretion system. <span><span class=\"ref-journal\">PLoS Pathog. </span>2011;<span class=\"ref-vol\">7</span></span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3169554/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n21931548\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog.&amp;title=Structural+and+functional+studies+on+the+interaction+of+GspC+and+GspD+in+the+type+II+secretion+system&amp;author=K.V.+Korotkov&amp;author=T.L.+Johnson&amp;author=M.G.+Jobling&amp;author=J.+Pruneda&amp;author=E.+Pardon&amp;volume=7&amp;publication_year=2011&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib23\">23. <span class=\"element-citation\" id=\"sref23\">Gu S., Kelly G., Wang X., Frenkiel T., Shevchik V.E., Pickersgill R.W. Solution structure of homology region (HR) domain of type II secretion system. <span><span class=\"ref-journal\">J.&#x000a0;Biol. Chem. </span>2012;<span class=\"ref-vol\">287</span>:9072–9080.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3308744/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n22253442\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.&#x000a0;Biol.+Chem.&amp;title=Solution+structure+of+homology+region+(HR)+domain+of+type+II+secretion+system&amp;author=S.+Gu&amp;author=G.+Kelly&amp;author=X.+Wang&amp;author=T.+Frenkiel&amp;author=V.E.+Shevchik&amp;volume=287&amp;publication_year=2012&amp;pages=9072-9080&amp;pmid=22253442&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib24\">24. <span class=\"element-citation\" id=\"sref24\">Creze C., Castang S., Derivery E., Haser R., Hugouvieux-Cotte-Pattat N., Shevchik V.E., Gouet P. The crystal structure of pectate lyase peli from soft rot pathogen Erwinia chrysanthemi in complex with its substrate. <span><span class=\"ref-journal\">J.&#x000a0;Biol. Chem. </span>2008;<span class=\"ref-vol\">283</span>:18260–18268.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n18430740\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.&#x000a0;Biol.+Chem.&amp;title=The+crystal+structure+of+pectate+lyase+peli+from+soft+rot+pathogen+Erwinia+chrysanthemi+in+complex+with+its+substrate&amp;author=C.+Creze&amp;author=S.+Castang&amp;author=E.+Derivery&amp;author=R.+Haser&amp;author=N.+Hugouvieux-Cotte-Pattat&amp;volume=283&amp;publication_year=2008&amp;pages=18260-18268&amp;pmid=18430740&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib25\">25. <span class=\"element-citation\" id=\"sref25\">Korotkov K.V., Pardon E., Steyaert J., Hol W.G. Crystal structure of the N-terminal domain of the secretin GspD from ETEC determined with the assistance of a nanobody. <span><span class=\"ref-journal\">Structure. </span>2009;<span class=\"ref-vol\">17</span>:255–265.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC2662362/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n19217396\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Crystal+structure+of+the+N-terminal+domain+of+the+secretin+GspD+from+ETEC+determined+with+the+assistance+of+a+nanobody&amp;author=K.V.+Korotkov&amp;author=E.+Pardon&amp;author=J.+Steyaert&amp;author=W.G.+Hol&amp;volume=17&amp;publication_year=2009&amp;pages=255-265&amp;pmid=19217396&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib26\">26. <span class=\"element-citation\" id=\"sref26\">Lombard V., Golaconda Ramulu H., Drula E., Coutinho P.M., Henrissat B. The carbohydrate-active enzymes database (CAZy) in 2013. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2014;<span class=\"ref-vol\">42</span>:D490–495.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3965031/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24270786\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+carbohydrate-active+enzymes+database+(CAZy)+in+2013&amp;author=V.+Lombard&amp;author=H.+Golaconda+Ramulu&amp;author=E.+Drula&amp;author=P.M.+Coutinho&amp;author=B.+Henrissat&amp;volume=42&amp;publication_year=2014&amp;pages=D490-495&amp;pmid=24270786&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib27\">27. <span class=\"element-citation\" id=\"sref27\">Alahuhta M., Chandrayan P., Kataeva I., Adams M.W., Himmel M.E., Lunin V.V. A&#x000a0;1.5 &#x000c5; resolution X-ray structure of the catalytic module of Caldicellulosiruptor bescii family 3 pectate lyase. <span><span class=\"ref-journal\">Acta Crystallogr.Sect. F Struct. Biol. Cryst. Commun. </span>2011;<span class=\"ref-vol\">67</span>:1498–1500.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3232124/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n22139151\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr.Sect.+F+Struct.+Biol.+Cryst.+Commun.&amp;title=A&#x000a0;1.5+&#x000c5;+resolution+X-ray+structure+of+the+catalytic+module+of+Caldicellulosiruptor+bescii+family+3+pectate+lyase&amp;author=M.+Alahuhta&amp;author=P.+Chandrayan&amp;author=I.+Kataeva&amp;author=M.W.+Adams&amp;author=M.E.+Himmel&amp;volume=67&amp;publication_year=2011&amp;pages=1498-1500&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib28\">28. <span class=\"element-citation\" id=\"sref28\">Akita M., Suzuki A., Kobayashi T., Ito S., Yamane T. The first structure of pectate lyase belonging to polysaccharide lyase family 3. <span><span class=\"ref-journal\">Acta Crystallogr. D Biol. Crystallogr. </span>2001;<span class=\"ref-vol\">57</span>:1786–1792.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n11717490\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr.+D+Biol.+Crystallogr.&amp;title=The+first+structure+of+pectate+lyase+belonging+to+polysaccharide+lyase+family+3&amp;author=M.+Akita&amp;author=A.+Suzuki&amp;author=T.+Kobayashi&amp;author=S.+Ito&amp;author=T.+Yamane&amp;volume=57&amp;publication_year=2001&amp;pages=1786-1792&amp;pmid=11717490&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib29\">29. <span class=\"element-citation\" id=\"sref29\">Little E., Bork P., Doolittle R.F. Tracing the spread of fibronectin type III domains in bacterial glycohydrolases. <span><span class=\"ref-journal\">J.&#x000a0;Mol. Evol. </span>1994;<span class=\"ref-vol\">39</span>:631–643.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n7528812\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.&#x000a0;Mol.+Evol.&amp;title=Tracing+the+spread+of+fibronectin+type+III+domains+in+bacterial+glycohydrolases&amp;author=E.+Little&amp;author=P.+Bork&amp;author=R.F.+Doolittle&amp;volume=39&amp;publication_year=1994&amp;pages=631-643&amp;pmid=7528812&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib30\">30. <span class=\"element-citation\" id=\"sref30\">Kataeva I.A., Seidel R.D., 3rd, Shah A., West L.T., Li X.L., Ljungdahl L.G. The fibronectin type 3-like repeat from the Clostridium thermocellum cellobiohydrolase CbhA promotes hydrolysis of cellulose by modifying its surface. <span><span class=\"ref-journal\">Appl. Environ. Microbiol. </span>2002;<span class=\"ref-vol\">68</span>:4292–4300.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC124122/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n12200278\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microbiol.&amp;title=The+fibronectin+type+3-like+repeat+from+the+Clostridium+thermocellum+cellobiohydrolase+CbhA+promotes+hydrolysis+of+cellulose+by+modifying+its+surface&amp;author=I.A.+Kataeva&amp;author=R.D.+Seidel&amp;author=A.+Shah&amp;author=L.T.+West&amp;author=X.L.+Li&amp;volume=68&amp;publication_year=2002&amp;pages=4292-4300&amp;pmid=12200278&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib31\">31. <span class=\"element-citation\" id=\"sref31\">Sidar A., Albuquerque E.D., Voshol G.P., Ram A.F.J., Vijgenboom E., Punt P.J. Carbohydrate binding modules: Diversity of domain architecture in amylases and cellulases from filamentous microorganisms. <span><span class=\"ref-journal\">Front. Bioeng. Biotechnol. </span>2020;<span class=\"ref-vol\">8</span>:871.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7410926/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n32850729\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front.+Bioeng.+Biotechnol.&amp;title=Carbohydrate+binding+modules:+Diversity+of+domain+architecture+in+amylases+and+cellulases+from+filamentous+microorganisms&amp;author=A.+Sidar&amp;author=E.D.+Albuquerque&amp;author=G.P.+Voshol&amp;author=A.F.J.+Ram&amp;author=E.+Vijgenboom&amp;volume=8&amp;publication_year=2020&amp;pages=871&amp;pmid=32850729&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib32\">32. <span class=\"element-citation\" id=\"sref32\">Bork P., Doolittle R.F. Proposed acquisition of an animal protein domain by bacteria. <span><span class=\"ref-journal\">Proc. Natl. Acad. Sci. U. S. A. </span>1992;<span class=\"ref-vol\">89</span>:8990–8994.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC50050/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n1409594\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A.&amp;title=Proposed+acquisition+of+an+animal+protein+domain+by+bacteria&amp;author=P.+Bork&amp;author=R.F.+Doolittle&amp;volume=89&amp;publication_year=1992&amp;pages=8990-8994&amp;pmid=1409594&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib33\">33. <span class=\"element-citation\" id=\"sref33\">Holm L., Laakso L.M. Dali server update. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2016;<span class=\"ref-vol\">44</span>:W351–355.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC4987910/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n27131377\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Dali+server+update&amp;author=L.+Holm&amp;author=L.M.+Laakso&amp;volume=44&amp;publication_year=2016&amp;pages=W351-355&amp;pmid=27131377&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib34\">34. <span class=\"element-citation\" id=\"sref34\">Kavran J.M., Ward M.D., Oladosu O.O., Mulepati S., Leahy D.J. All mammalian Hedgehog proteins interact with cell adhesion molecule, down-regulated by oncogenes (CDO) and brother of CDO (BOC) in a conserved manner. <span><span class=\"ref-journal\">J.&#x000a0;Biol. Chem. </span>2010;<span class=\"ref-vol\">285</span>:24584–24590.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC2915694/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n20519495\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.&#x000a0;Biol.+Chem.&amp;title=All+mammalian+Hedgehog+proteins+interact+with+cell+adhesion+molecule,+down-regulated+by+oncogenes+(CDO)+and+brother+of+CDO+(BOC)+in+a+conserved+manner&amp;author=J.M.+Kavran&amp;author=M.D.+Ward&amp;author=O.O.+Oladosu&amp;author=S.+Mulepati&amp;author=D.J.+Leahy&amp;volume=285&amp;publication_year=2010&amp;pages=24584-24590&amp;pmid=20519495&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib35\">35. <span class=\"element-citation\" id=\"sref35\">McLellan J.S., Zheng X., Hauk G., Ghirlando R., Beachy P.A., Leahy D.J. The mode of Hedgehog binding to Ihog homologues is not conserved across different phyla. <span><span class=\"ref-journal\">Nature. </span>2008;<span class=\"ref-vol\">455</span>:979–983.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC2679680/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n18794898\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+mode+of+Hedgehog+binding+to+Ihog+homologues+is+not+conserved+across+different+phyla&amp;author=J.S.+McLellan&amp;author=X.+Zheng&amp;author=G.+Hauk&amp;author=R.+Ghirlando&amp;author=P.A.+Beachy&amp;volume=455&amp;publication_year=2008&amp;pages=979-983&amp;pmid=18794898&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib36\">36. <span class=\"element-citation\" id=\"sref36\">Healey E.G., Bishop B., Elegheert J., Bell C.H., Padilla-Parra S., Siebold C. Repulsive guidance molecule is a structural bridge between neogenin and bone morphogenetic protein. <span><span class=\"ref-journal\">Nat. Struct. Mol. Biol. </span>2015;<span class=\"ref-vol\">22</span>:458–465.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC4456160/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n25938661\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Struct.+Mol.+Biol.&amp;title=Repulsive+guidance+molecule+is+a+structural+bridge+between+neogenin+and+bone+morphogenetic+protein&amp;author=E.G.+Healey&amp;author=B.+Bishop&amp;author=J.+Elegheert&amp;author=C.H.+Bell&amp;author=S.+Padilla-Parra&amp;volume=22&amp;publication_year=2015&amp;pages=458-465&amp;pmid=25938661&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib37\">37. <span class=\"element-citation\" id=\"sref37\">Shevchik V.E., Boccara M., Vedel R., Hugouvieux-Cotte-Pattat N. Processing of the pectate lyase PelI by extracellular proteases of Erwinia chrysanthemi 3937. <span><span class=\"ref-journal\">Mol. Microbiol. </span>1998;<span class=\"ref-vol\">29</span>:1459–1469.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n9781882\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol.&amp;title=Processing+of+the+pectate+lyase+PelI+by+extracellular+proteases+of+Erwinia+chrysanthemi+3937&amp;author=V.E.+Shevchik&amp;author=M.+Boccara&amp;author=R.+Vedel&amp;author=N.+Hugouvieux-Cotte-Pattat&amp;volume=29&amp;publication_year=1998&amp;pages=1459-1469&amp;pmid=9781882&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib38\">38. <span class=\"element-citation\" id=\"sref38\">Shevchik V.E., Bortoli-German I., Robert-Baudouy J., Robinet S., Barras F., Condemine G. Differential effect of dsbA and dsbC mutations on extracellular enzyme secretion in Erwinia chrysanthemi. <span><span class=\"ref-journal\">Mol. Microbiol. </span>1995;<span class=\"ref-vol\">16</span>:745–753.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n7476168\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol.&amp;title=Differential+effect+of+dsbA+and+dsbC+mutations+on+extracellular+enzyme+secretion+in+Erwinia+chrysanthemi&amp;author=V.E.+Shevchik&amp;author=I.+Bortoli-German&amp;author=J.+Robert-Baudouy&amp;author=S.+Robinet&amp;author=F.+Barras&amp;volume=16&amp;publication_year=1995&amp;pages=745-753&amp;pmid=7476168&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib39\">39. <span class=\"element-citation\" id=\"sref40\">Charkowski A.O., Alfano J.R., Preston G., Yuan J., He S.Y., Collmer A. The Pseudomonas syringae pv. tomato HrpW protein has domains similar to harpins and pectate lyases and can elicit the plant hypersensitive response and bind to pectate. <span><span class=\"ref-journal\">J.&#x000a0;Bacteriol. </span>1998;<span class=\"ref-vol\">180</span>:5211–5217.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC107559/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n9748456\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.&#x000a0;Bacteriol.&amp;title=The+Pseudomonas+syringae+pv.+tomato+HrpW+protein+has+domains+similar+to+harpins+and+pectate+lyases+and+can+elicit+the+plant+hypersensitive+response+and+bind+to+pectate&amp;author=A.O.+Charkowski&amp;author=J.R.+Alfano&amp;author=G.+Preston&amp;author=J.+Yuan&amp;author=S.Y.+He&amp;volume=180&amp;publication_year=1998&amp;pages=5211-5217&amp;pmid=9748456&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib40\">40. <span class=\"element-citation\" id=\"sref41\">Jin Q., He S.Y. Role of the Hrp pilus in type III protein secretion in Pseudomonas syringae. <span><span class=\"ref-journal\">Science. </span>2001;<span class=\"ref-vol\">294</span>:2556–2558.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n11752577\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Role+of+the+Hrp+pilus+in+type+III+protein+secretion+in+Pseudomonas+syringae&amp;author=Q.+Jin&amp;author=S.Y.+He&amp;volume=294&amp;publication_year=2001&amp;pages=2556-2558&amp;pmid=11752577&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib41\">41. <span class=\"element-citation\" id=\"sref42\">Tsai H.H., Gunasekaran K., Nussinov R. Sequence and structure analysis of parallel beta helices: Implication for constructing amyloid structural models. <span><span class=\"ref-journal\">Structure. </span>2006;<span class=\"ref-vol\">14</span>:1059–1072.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n16765899\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Sequence+and+structure+analysis+of+parallel+beta+helices:+Implication+for+constructing+amyloid+structural+models&amp;author=H.H.+Tsai&amp;author=K.+Gunasekaran&amp;author=R.+Nussinov&amp;volume=14&amp;publication_year=2006&amp;pages=1059-1072&amp;pmid=16765899&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib42\">42. <span class=\"element-citation\" id=\"sref43\">Chandler P.G., Buckle A.M. Development and differentiation in monobodies based on the fibronectin type 3 domain. <span><span class=\"ref-journal\">Cells. </span>2020;<span class=\"ref-vol\">9</span>:610.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7140400/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n32143310\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=Development+and+differentiation+in+monobodies+based+on+the+fibronectin+type+3+domain&amp;author=P.G.+Chandler&amp;author=A.M.+Buckle&amp;volume=9&amp;publication_year=2020&amp;pages=610&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib43\">43. <span class=\"element-citation\" id=\"sref44\">Koide S., Koide A., Lipov&#x00161;ek D. Target-binding proteins based on the 10th human fibronectin type III domain (<sup>1</sup>&#x02070;Fn3) <span><span class=\"ref-journal\">Methods Enzymol. </span>2012;<span class=\"ref-vol\">503</span>:135–156.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n22230568\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Methods+Enzymol.&amp;title=Target-binding+proteins+based+on+the+10th+human+fibronectin+type+III+domain+(1&#x02070;Fn3)&amp;author=S.+Koide&amp;author=A.+Koide&amp;author=D.+Lipov&#x00161;ek&amp;volume=503&amp;publication_year=2012&amp;pages=135-156&amp;pmid=22230568&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib44\">44. <span class=\"element-citation\" id=\"sref45\">Lindeberg M., Boyd C.M., Keen N.T., Collmer A. External loops at the C terminus of Erwinia chrysanthemi pectate lyase C are required for species-specific secretion through the out type II pathway. <span><span class=\"ref-journal\">J.&#x000a0;Bacteriol. </span>1998;<span class=\"ref-vol\">180</span>:1431–1437.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC107041/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n9515910\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.&#x000a0;Bacteriol.&amp;title=External+loops+at+the+C+terminus+of+Erwinia+chrysanthemi+pectate+lyase+C+are+required+for+species-specific+secretion+through+the+out+type+II+pathway&amp;author=M.+Lindeberg&amp;author=C.M.+Boyd&amp;author=N.T.+Keen&amp;author=A.+Collmer&amp;volume=180&amp;publication_year=1998&amp;pages=1431-1437&amp;pmid=9515910&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib45\">45. <span class=\"element-citation\" id=\"sref46\">Johnson B.H., Hecht M.H. Recombinant proteins can be isolated from E. coli cells by repeated cycles of freezing and thawing. <span><span class=\"ref-journal\">Biotechnology (N. Y.) </span>1994;<span class=\"ref-vol\">12</span>:1357–1360.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n7765566\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Biotechnology+(N.+Y.)&amp;title=Recombinant+proteins+can+be+isolated+from+E.+coli+cells+by+repeated+cycles+of+freezing+and+thawing&amp;author=B.H.+Johnson&amp;author=M.H.+Hecht&amp;volume=12&amp;publication_year=1994&amp;pages=1357-1360&amp;pmid=7765566&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib46\">46. <span class=\"element-citation\" id=\"sref47\">Castang S., Shevchik V.E., Hugouvieux-Cotte-Pattat N., Legrand P., Haser R., Gouet P. Crystallization of the pectate lyase PelI from Erwinia chrysanthemi and SAD phasing of a gold derivative. <span><span class=\"ref-journal\">Acta Crystallogr. D Biol. Crystallogr. </span>2004;<span class=\"ref-vol\">60</span>:190–192.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n14684926\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr.+D+Biol.+Crystallogr.&amp;title=Crystallization+of+the+pectate+lyase+PelI+from+Erwinia+chrysanthemi+and+SAD+phasing+of+a+gold+derivative&amp;author=S.+Castang&amp;author=V.E.+Shevchik&amp;author=N.+Hugouvieux-Cotte-Pattat&amp;author=P.+Legrand&amp;author=R.+Haser&amp;volume=60&amp;publication_year=2004&amp;pages=190-192&amp;pmid=14684926&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib47\">47. <span class=\"element-citation\" id=\"sref48\">Liebschner D., Afonine P.V., Baker M.L., Bunk&#x000f3;czi G., Chen V.B., Croll T.I., Hintze B., Hung L.W., Jain S., McCoy A.J., Moriarty N.W., Oeffner R.D., Poon B.K., Prisant M.G., Read R.J. Macromolecular structure determination using X-rays, neutrons and electrons: Recent developments in Phenix. <span><span class=\"ref-journal\">Acta Crystallogr. D Struct. Biol. </span>2019;<span class=\"ref-vol\">75</span>:861–877.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC6778852/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n31588918\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr.+D+Struct.+Biol.&amp;title=Macromolecular+structure+determination+using+X-rays,+neutrons+and+electrons:+Recent+developments+in+Phenix&amp;author=D.+Liebschner&amp;author=P.V.+Afonine&amp;author=M.L.+Baker&amp;author=G.+Bunk&#x000f3;czi&amp;author=V.B.+Chen&amp;volume=75&amp;publication_year=2019&amp;pages=861-877&amp;pmid=31588918&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib48\">48. <span class=\"element-citation\" id=\"sref49\">Emsley P., Cowtan K. Coot: Model-building tools for molecular graphics. <span><span class=\"ref-journal\">Acta Crystallogr. D Biol. Crystallogr. </span>2004;<span class=\"ref-vol\">60</span>:2126–2132.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n15572765\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Acta+Crystallogr.+D+Biol.+Crystallogr.&amp;title=Coot:+Model-building+tools+for+molecular+graphics&amp;author=P.+Emsley&amp;author=K.+Cowtan&amp;volume=60&amp;publication_year=2004&amp;pages=2126-2132&amp;pmid=15572765&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib49\">49. <span class=\"element-citation\" id=\"sref50\">Laskowski R.A., MacArthur M.W., Moss D.S., Thornton J.M. PROCHECK: A program to check the stereochemical quality of protein structures. <span><span class=\"ref-journal\">J.&#x000a0;Appl. Crystallogr. </span>1993;<span class=\"ref-vol\">26</span>:283–291.</span> <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.&#x000a0;Appl.+Crystallogr.&amp;title=PROCHECK:+A+program+to+check+the+stereochemical+quality+of+protein+structures&amp;author=R.A.+Laskowski&amp;author=M.W.+MacArthur&amp;author=D.S.+Moss&amp;author=J.M.+Thornton&amp;volume=26&amp;publication_year=1993&amp;pages=283-291&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib50\">50. <span class=\"element-citation\" id=\"sref51\">Altschul S.F., Madden T.L., Sch&#x000e4;ffer A.A., Zhang J., Zhang Z., Miller W., Lipman D.J. Gapped BLAST and PSI-BLAST: A new generation of protein database search programs. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>1997;<span class=\"ref-vol\">25</span>:3389–3402.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC146917/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n9254694\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Gapped+BLAST+and+PSI-BLAST:+A+new+generation+of+protein+database+search+programs&amp;author=S.F.+Altschul&amp;author=T.L.+Madden&amp;author=A.A.+Sch&#x000e4;ffer&amp;author=J.+Zhang&amp;author=Z.+Zhang&amp;volume=25&amp;publication_year=1997&amp;pages=3389-3402&amp;pmid=9254694&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib51\">51. <span class=\"element-citation\" id=\"sref52\">Katoh K., Misawa K., Kuma K., Miyata T. MAFFT: A novel method for rapid multiple sequence alignment based on fast Fourier transform. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2002;<span class=\"ref-vol\">30</span>:3059–3066.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC135756/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n12136088\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=MAFFT:+A+novel+method+for+rapid+multiple+sequence+alignment+based+on+fast+Fourier+transform&amp;author=K.+Katoh&amp;author=K.+Misawa&amp;author=K.+Kuma&amp;author=T.+Miyata&amp;volume=30&amp;publication_year=2002&amp;pages=3059-3066&amp;pmid=12136088&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib52\">52. <span class=\"element-citation\" id=\"sref53\">Criscuolo A., Gribaldo S. BMGE (block mapping and gathering with entropy): A new software for selection of phylogenetic informative regions from multiple sequence alignments. <span><span class=\"ref-journal\">BMC Evol. Biol. </span>2010;<span class=\"ref-vol\">10</span>:210.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3017758/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n20626897\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Evol.+Biol.&amp;title=BMGE+(block+mapping+and+gathering+with+entropy):+A+new+software+for+selection+of+phylogenetic+informative+regions+from+multiple+sequence+alignments&amp;author=A.+Criscuolo&amp;author=S.+Gribaldo&amp;volume=10&amp;publication_year=2010&amp;pages=210&amp;pmid=20626897&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib53\">53. <span class=\"element-citation\" id=\"sref54\">Henikoff S., Henikoff J.G. Amino acid substitution matrices from protein blocks. <span><span class=\"ref-journal\">Proc. Natl. Acad. Sci. U. S. A. </span>1992;<span class=\"ref-vol\">89</span>:10915–10919.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC50453/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n1438297\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A.&amp;title=Amino+acid+substitution+matrices+from+protein+blocks&amp;author=S.+Henikoff&amp;author=J.G.+Henikoff&amp;volume=89&amp;publication_year=1992&amp;pages=10915-10919&amp;pmid=1438297&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib54\">54. <span class=\"element-citation\" id=\"sref55\">Huelsenbeck J.P., Ronquist F. MRBAYES: Bayesian inference of phylogenetic trees. <span><span class=\"ref-journal\">Bioinformatics. </span>2001;<span class=\"ref-vol\">17</span>:754–755.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n11524383\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=MRBAYES:+Bayesian+inference+of+phylogenetic+trees&amp;author=J.P.+Huelsenbeck&amp;author=F.+Ronquist&amp;volume=17&amp;publication_year=2001&amp;pages=754-755&amp;pmid=11524383&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib55\">55. <span class=\"element-citation\" id=\"sref56\">Guindon S., Dufayard J.F., Lefort V., Anisimova M., Hordijk W., Gascuel O. New algorithms and methods to estimate maximum-likelihood phylogenies: Assessing the performance of PhyML 3.0. <span><span class=\"ref-journal\">Syst. Biol. </span>2010;<span class=\"ref-vol\">59</span>:307–321.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n20525638\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Syst.+Biol.&amp;title=New+algorithms+and+methods+to+estimate+maximum-likelihood+phylogenies:+Assessing+the+performance+of+PhyML+3.0&amp;author=S.+Guindon&amp;author=J.F.+Dufayard&amp;author=V.+Lefort&amp;author=M.+Anisimova&amp;author=W.+Hordijk&amp;volume=59&amp;publication_year=2010&amp;pages=307-321&amp;pmid=20525638&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib56\">56. <span class=\"element-citation\" id=\"sref57\">Letunic I., Bork P. Interactive tree of life (iTOL) v4: Recent updates and new developments. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2019;<span class=\"ref-vol\">47</span>:W256–W259.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC6602468/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30931475\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Interactive+tree+of+life+(iTOL)+v4:+Recent+updates+and+new+developments&amp;author=I.+Letunic&amp;author=P.+Bork&amp;volume=47&amp;publication_year=2019&amp;pages=W256-W259&amp;pmid=30931475&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"bib57\">57. <span class=\"element-citation\" id=\"sref58\">Robert X., Gouet P. Deciphering key features in protein structures with the new ENDscript server. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2014;<span class=\"ref-vol\">42</span>:W320–W324.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC4086106/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24753421\" ref=\"reftype=pubmed&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Deciphering+key+features+in+protein+structures+with+the+new+ENDscript+server&amp;author=X.+Robert&amp;author=P.+Gouet&amp;volume=42&amp;publication_year=2014&amp;pages=W320-W324&amp;pmid=24753421&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7949064&amp;issue-id=374627&amp;journal-id=567&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div></div><br>', NULL, 6);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(2, 'pubmed', '32338906', 'Engineering of β-Glucosidase Bgl15 with Simultaneously Enhanced Glucose Tolerance and Thermostability To Improve Its Performance in High-Solid Cellulose Hydrolysis.', 'In this study, a Petri-dish-based double-layer high-throughput screening method was established to improve glucose tolerance of β-glucosidase Bgl15. Two beneficial mutations were identified, and the joint mutant 2R1 improved the half-maximal inhibitory concentration of glucose from 0.04 to 2.1 M. The crystal structure of 2R1 was subsequently determined at 2.7 Å. Structure analysis revealed that enhancement of glucose tolerance may be due to improved transglycosylation activity made possible by a hydrophobic binding site for glucose as an acceptor and more stringent control of a putative water channel. To further ameliorate the application potential of the enzyme, it was engineered to increase the half-life at 50 °C from 0.8 h (Bgl15) to 180 h (mutant 5R1). Furthermore, supplementation of 5R1 to the cellulase cocktail significantly improved glucose production from pretreated sugar cane bagasse by 38%. Consequently, this study provided an efficient approach to enhance glucose tolerance and generated a promising catalyst for cellulose saccharification.', 2020, 'Journal of agricultural and food chemistry', '', 1, NULL, NULL, 6),
(3, 'pubmed', '31198441', 'The - gene cluster of  encodes GH43- and GH62-α-l-arabinofuranosidases with complementary modes of action.', 'The α-l-arabinofuranosidases (α-l-ABFs) are exoenzymes involved in the hydrolysis of α-l-arabinosyl linkages in plant cell wall polysaccharides. They play a crucial role in the degradation of arabinoxylan and arabinan and they are used in many biotechnological applications. Analysis of the genome of  showed that putative cellulosomal α-l-ABFs are exclusively encoded by the - gene cluster, a large 32-kb gene cluster. Indeed, among the 14 Xyl-Doc enzymes encoded by this gene cluster, 6 are predicted to be α-l-ABFs belonging to the CAZyme families GH43 and GH62.The biochemical characterization of these six Xyl-Doc enzymes revealed that four of them are α-l-ABFs. GH43-1229 (RcAbf43A) which belongs to the subfamily 16 of the GH43, encoded by the gene at locus Ccel_1229, has a low specific activity on natural substrates and can cleave off arabinose decorations located at arabinoxylan chain extremities. GH43-1233 (RcAbf43A), the product of the gene at locus Ccel_1233, belonging to subfamily 10 of the GH43, can convert the double arabinose decorations present on arabinoxylan into single O2- or O3-linked decorations with high velocity ( = 16.6 ± 0.6 s). This enzyme acts in synergy with GH62-1234 (RcAbf62A), the product of the gene at locus Ccel_1234, a GH62 α-l-ABF which hydrolyzes α-(1 → 3) or α-(1 → 2)-arabinosyl linkages present on polysaccharides and arabinoxylooligosaccharides monodecorated. Finally, a bifunctional enzyme, GH62-CE6-1240 (RcAbf62BAxe6), encoded by the gene at locus Ccel_1240, which contains a GH62-α-l-ABF module and a carbohydrate esterase (CE6) module, catalyzes deacylation of plant cell wall polymers and cleavage of arabinosyl mono-substitutions. These enzymes are also active on arabinan, a component of the type I rhamnogalacturonan, showing their involvement in pectin degradation.Arabinofuranosyl decorations on arabinoxylan and pectin strongly inhibit the action of xylan-degrading enzymes and pectinases. α-l-ABFs encoded by the - gene cluster of  can remove all the decorations present in the backbone of arabinoxylan and arabinan, act synergistically, and, thus, play a crucial role in the degradation of plant cell wall polysaccharides.', 2019, 'Biotechnology for biofuels', 'PMC6556953', 3, NULL, NULL, 6),
(4, 'pubmed', '32308737', 'Investigation of a thermostable multi-domain xylanase-glucuronoyl esterase enzyme from  incorporating multiple carbohydrate-binding modules.', 'Efficient degradation of lignocellulosic biomass has become a major bottleneck in industrial processes which attempt to use biomass as a carbon source for the production of biofuels and materials. To make the most effective use of the source material, both the hemicellulosic as well as cellulosic parts of the biomass should be targeted, and as such both hemicellulases and cellulases are important enzymes in biorefinery processes. Using thermostable versions of these enzymes can also prove beneficial in biomass degradation, as they can be expected to act faster than mesophilic enzymes and the process can also be improved by lower viscosities at higher temperatures, as well as prevent the introduction of microbial contamination.This study presents the investigation of the thermostable, dual-function xylanase-glucuronoyl esterase enzyme Xyn10C-GE15A from the hyperthermophilic bacterium . Biochemical characterization of the enzyme was performed, including assays for establishing the melting points for the different protein domains, activity assays for the two catalytic domains, as well as binding assays for the multiple carbohydrate-binding domains present in Xyn10C-GE15A. Although the enzyme domains are naturally linked together, when added separately to biomass, the expected boosting of the xylanase action was not seen. This lack of intramolecular synergy might suggest, together with previous data, that increased xylose release is not the main beneficial trait given by glucuronoyl esterases.Due to its thermostability, Xyn10C-GE15A is a promising candidate for industrial processes, with both catalytic domains exhibiting melting temperatures over 70 °C. Of particular interest is the glucuronoyl esterase domain, as it represents the first studied thermostable enzyme displaying this activity.', 2020, 'Biotechnology for biofuels', 'PMC7151638', 4, NULL, NULL, 6),
(5, 'pubmed', '33538380', 'Gene family expansions and transcriptome signatures uncover fungal adaptations to wood decay.', 'Because they comprise some of the most efficient wood-decayers, Polyporales fungi impact carbon cycling in forest environment. Despite continuous discoveries on the enzymatic machinery involved in wood decomposition, the vision on their evolutionary adaptation to wood decay and genome diversity remains incomplete. We combined the genome sequence information from 50 Polyporales species, including 26 newly sequenced genomes and sought for genomic and functional adaptations to wood decay through the analysis of genome composition and transcriptome responses to different carbon sources. The genomes of Polyporales from different phylogenetic clades showed poor conservation in macrosynteny, indicative of genome rearrangements. We observed different gene family expansion/contraction histories for plant cell wall degrading enzymes in core polyporoids and phlebioids and captured expansions for genes involved in signalling and regulation in the lineages of white rotters. Furthermore, we identified conserved cupredoxins, thaumatin-like proteins and lytic polysaccharide monooxygenases with a yet uncharacterized appended module as new candidate players in wood decomposition. Given the current need for enzymatic toolkits dedicated to the transformation of renewable carbon sources, the observed genomic diversity among Polyporales strengthens the relevance of mining Polyporales biodiversity to understand the molecular mechanisms of wood decay.', 2021, 'Environmental microbiology', '', 1, NULL, NULL, 4),
(6, 'pubmed', '33435491', 'Fungal Treatment for the Valorization of Technical Soda Lignin.', 'Technical lignins produced as a by-product in biorefinery processes represent a potential source of renewable carbon. In consideration of the possibilities of the industrial transformation of this substrate into various valuable bio-based molecules, the biological deconstruction of a technical soda lignin by filamentous fungi was investigated. The ability of three basidiomycetes (,  and ) to modify this material, the resultant structural and chemical changes, and the secreted proteins during growth on this substrate were investigated. The three fungi could grow on the technical lignin alone, and the growth rate increased when the media were supplemented with glucose or maltose. The proteomic analysis of the culture supernatants after three days of growth revealed the secretion of numerous Carbohydrate-Active Enzymes (CAZymes). The secretomic profiles varied widely between the strains and the presence of technical lignin alone triggered the early secretion of many lignin-acting oxidoreductases. The secretomes were notably rich in glycoside hydrolases and HO-producing auxiliary activity enzymes with copper radical oxidases being induced on lignin for all strains. The lignin treatment by fungi modified both the soluble and insoluble lignin fractions. A significant decrease in the amount of soluble higher molar mass compounds was observed in the case of . This strain was also responsible for the modification of the lower molar mass compounds of the lignin insoluble fraction and a 40% decrease in the thioacidolysis yield. The similarity in the activities of  and  in modifying the functional groups of the technical lignin were observed, the results suggest that the lignin has undergone structural changes, or at least changes in its composition, and pave the route for the utilization of filamentous fungi to functionalize technical lignins and produce the enzymes of interest for biorefinery applications.', 2021, 'Journal of fungi (Basel, Switzerland)', 'PMC7827817', 1, NULL, NULL, 6),
(7, 'pubmed', '33211093', 'Genomic Analysis Enlightens Agaricales Lifestyle Evolution and Increasing Peroxidase Diversity.', 'As actors of global carbon cycle, Agaricomycetes (Basidiomycota) have developed complex enzymatic machineries that allow them to decompose all plant polymers, including lignin. Among them, saprotrophic Agaricales are characterized by an unparalleled diversity of habitats and lifestyles. Comparative analysis of 52 Agaricomycetes genomes (14 of them sequenced de novo) reveals that Agaricales possess a large diversity of hydrolytic and oxidative enzymes for lignocellulose decay. Based on the gene families with the predicted highest evolutionary rates-namely cellulose-binding CBM1, glycoside hydrolase GH43, lytic polysaccharide monooxygenase AA9, class-II peroxidases, glucose-methanol-choline oxidase/dehydrogenases, laccases, and unspecific peroxygenases-we reconstructed the lifestyles of the ancestors that led to the extant lignocellulose-decomposing Agaricomycetes. The changes in the enzymatic toolkit of ancestral Agaricales are correlated with the evolution of their ability to grow not only on wood but also on leaf litter and decayed wood, with grass-litter decomposers as the most recent eco-physiological group. In this context, the above families were analyzed in detail in connection with lifestyle diversity. Peroxidases appear as a central component of the enzymatic toolkit of saprotrophic Agaricomycetes, consistent with their essential role in lignin degradation and high evolutionary rates. This includes not only expansions/losses in peroxidase genes common to other basidiomycetes but also the widespread presence in Agaricales (and Russulales) of new peroxidases types not found in wood-rotting Polyporales, and other Agaricomycetes orders. Therefore, we analyzed the peroxidase evolution in Agaricomycetes by ancestral-sequence reconstruction revealing several major evolutionary pathways and mapped the appearance of the different enzyme types in a time-calibrated species tree.', 2021, 'Molecular biology and evolution', '', 1, NULL, NULL, 1),
(8, 'pubmed', '33077886', 'Niche differentiation and evolution of the wood decay machinery in the invasive fungus Serpula lacrymans.', 'Ecological niche breadth and the mechanisms facilitating its evolution are fundamental to understanding adaptation to changing environments, persistence of generalist and specialist lineages and the formation of new species. Woody substrates are structurally complex resources utilized by organisms with specialized decay machinery. Wood-decaying fungi represent ideal model systems to study evolution of niche breadth, as they vary greatly in their host range and preferred decay stage of the substrate. In order to dissect the genetic basis for niche specialization in the invasive brown rot fungus Serpula lacrymans, we used phenotyping and integrative analysis of phylogenomic and transcriptomic data to compare this species to wild relatives in the Serpulaceae with a range of specialist to generalist decay strategies. Our results indicate specialist species have rewired regulatory networks active during wood decay towards decreased reliance on enzymatic machinery, and therefore nitrogen-intensive decay components. This shift was likely accompanied with adaptation to a narrow tree line habitat and switch to a pioneer decomposer strategy, both requiring rapid colonization of a nitrogen-limited substrate. Among substrate specialists with narrow niches, we also found evidence for pathways facilitating reversal to generalism, highlighting how evolution may move along different axes of niche space.', 2021, 'The ISME journal', 'PMC8027034', 1, NULL, NULL, 3);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(9, 'pubmed', '33046698', 'Large-scale genome sequencing of mycorrhizal fungi provides insights into the early evolution of symbiotic traits.', 'Mycorrhizal fungi are mutualists that play crucial roles in nutrient acquisition in terrestrial ecosystems. Mycorrhizal symbioses arose repeatedly across multiple lineages of Mucoromycotina, Ascomycota, and Basidiomycota. Considerable variation exists in the capacity of mycorrhizal fungi to acquire carbon from soil organic matter. Here, we present a combined analysis of 135 fungal genomes from 73 saprotrophic, endophytic and pathogenic species, and 62 mycorrhizal species, including 29 new mycorrhizal genomes. This study samples ecologically dominant fungal guilds for which there were previously no symbiotic genomes available, including ectomycorrhizal Russulales, Thelephorales and Cantharellales. Our analyses show that transitions from saprotrophy to symbiosis involve (1) widespread losses of degrading enzymes acting on lignin and cellulose, (2) co-option of genes present in saprotrophic ancestors to fulfill new symbiotic functions, (3) diversification of novel, lineage-specific symbiosis-induced genes, (4) proliferation of transposable elements and (5) divergent genetic innovations underlying the convergent origins of the ectomycorrhizal guild.', 2020, 'Nature communications', 'PMC7550596', 2, '<h1 class=\"content-title\">Large-scale genome sequencing of mycorrhizal fungi provides insights into the early evolution of symbiotic traits</h1><br><span class=\"contrib-group fm-author\"><a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Miyauchi%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716228127600\" co-class=\"co-affbox\">Shingo Miyauchi</a>,<sup>#</sup><sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Kiss%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716134900144\" co-class=\"co-affbox\">Enik&#x00151; Kiss</a>,<sup>#</sup><sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Kuo%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716147902144\" co-class=\"co-affbox\">Alan Kuo</a>,<sup>#</sup><sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Drula%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716136057216\" co-class=\"co-affbox\">Elodie Drula</a>,<sup>4</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Kohler%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716142223344\" co-class=\"co-affbox\">Annegret Kohler</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=S%26%23x000e1%3Bnchez-Garc%26%23x000ed%3Ba%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716150741008\" co-class=\"co-affbox\">Marisol S&#x000e1;nchez-Garc&#x000ed;a</a>,<sup>5</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Morin%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716152135600\" co-class=\"co-affbox\">Emmanuelle Morin</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Andreopoulos%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716192751456\" co-class=\"co-affbox\">Bill Andreopoulos</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Barry%20KW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716145758960\" co-class=\"co-affbox\">Kerrie W. Barry</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Bonito%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716146524576\" co-class=\"co-affbox\">Gregory Bonito</a>,<sup>6</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Bu%26%23x000e9%3Be%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716139564240\" co-class=\"co-affbox\">Marc Bu&#x000e9;e</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Carver%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716140279152\" co-class=\"co-affbox\">Akiko Carver</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Chen%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716145097856\" co-class=\"co-affbox\">Cindy Chen</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Cichocki%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716190752224\" co-class=\"co-affbox\">Nicolas Cichocki</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Clum%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716134964096\" co-class=\"co-affbox\">Alicia Clum</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Culley%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716146596960\" co-class=\"co-affbox\">David Culley</a>,<sup>7</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Crous%20PW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716146634528\" co-class=\"co-affbox\">Pedro W. Crous</a>,<sup>8</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Fauchery%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716143760544\" co-class=\"co-affbox\">Laure Fauchery</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Girlanda%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716154423264\" co-class=\"co-affbox\">Mariangela Girlanda</a>,<sup>9</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Hayes%20RD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716144620384\" co-class=\"co-affbox\">Richard D. Hayes</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=K%26%23x000e9%3Bri%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716157979152\" co-class=\"co-affbox\">Zs&#x000f3;fia K&#x000e9;ri</a>,<sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=LaButti%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716149677744\" co-class=\"co-affbox\">Kurt LaButti</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Lipzen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716157395504\" co-class=\"co-affbox\">Anna Lipzen</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Lombard%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716141959984\" co-class=\"co-affbox\">Vincent Lombard</a>,<sup>4</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Magnuson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716228294528\" co-class=\"co-affbox\">Jon Magnuson</a>,<sup>7</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Maillard%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716139635360\" co-class=\"co-affbox\">Fran&#x000e7;ois Maillard</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Murat%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716150986960\" co-class=\"co-affbox\">Claude Murat</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Nolan%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716139980112\" co-class=\"co-affbox\">Matt Nolan</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Ohm%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716151825440\" co-class=\"co-affbox\">Robin A. Ohm</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Pangilinan%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716135769616\" co-class=\"co-affbox\">Jasmyn Pangilinan</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Pereira%20Md%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716148756560\" co-class=\"co-affbox\">Ma&#x000ed;ra de Freitas Pereira</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Perotto%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716145148448\" co-class=\"co-affbox\">Silvia Perotto</a>,<sup>9</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Peter%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716142844832\" co-class=\"co-affbox\">Martina Peter</a>,<sup>10</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Pfister%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716228320320\" co-class=\"co-affbox\">Stephanie Pfister</a>,<sup>10</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Riley%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716145024000\" co-class=\"co-affbox\">Robert Riley</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Sitrit%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716141552928\" co-class=\"co-affbox\">Yaron Sitrit</a>,<sup>11</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Stielow%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716151478240\" co-class=\"co-affbox\">J. Benjamin Stielow</a>,<sup>8</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Sz%26%23x000f6%3Bll%26%23x00151%3Bsi%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716145972176\" co-class=\"co-affbox\">Gergely Sz&#x000f6;ll&#x00151;si</a>,<sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=%26%23x0017d%3Bif%26%23x0010d%3B%26%23x000e1%3Bkov%26%23x000e1%3B%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716146644496\" co-class=\"co-affbox\">Lucia &#x0017d;if&#x0010d;&#x000e1;kov&#x000e1;</a>,<sup>12</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=%26%23x00160%3Btursov%26%23x000e1%3B%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716155280800\" co-class=\"co-affbox\">Martina &#x00160;tursov&#x000e1;</a>,<sup>12</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Spatafora%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716150252896\" co-class=\"co-affbox\">Joseph W. Spatafora</a>,<sup>13</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Tedersoo%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716145983648\" co-class=\"co-affbox\">Leho Tedersoo</a>,<sup>14</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Vaario%20LM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716154333168\" co-class=\"co-affbox\">Lu-Min Vaario</a>,<sup>15</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Yamada%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716142447616\" co-class=\"co-affbox\">Akiyoshi Yamada</a>,<sup>16</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Yan%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716153489712\" co-class=\"co-affbox\">Mi Yan</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Wang%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716138936752\" co-class=\"co-affbox\">Pengfei Wang</a>,<sup>17</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Xu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716157773584\" co-class=\"co-affbox\">Jianping Xu</a>,<sup>18</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Bruns%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716137413920\" co-class=\"co-affbox\">Tom Bruns</a>,<sup>19</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Baldrian%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716138390000\" co-class=\"co-affbox\">Petr Baldrian</a>,<sup>12</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Vilgalys%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716152916272\" co-class=\"co-affbox\">Rytas Vilgalys</a>,<sup>20</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Dunand%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716135954448\" co-class=\"co-affbox\">Christophe Dunand</a>,<sup>21</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Henrissat%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716150168544\" co-class=\"co-affbox\">Bernard Henrissat</a>,<sup>4,</sup><sup>22,</sup><sup>23</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Grigoriev%20IV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716151639936\" co-class=\"co-affbox\">Igor V. Grigoriev</a>,<sup>3,</sup><sup>19</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Hibbett%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716139049376\" co-class=\"co-affbox\">David Hibbett</a>,<sup>5</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Nagy%20LG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716156121680\" co-class=\"co-affbox\">L&#x000e1;szl&#x000f3; G. Nagy</a>,<sup>2</sup> and  <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Martin%20FM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\" class=\"affpopup\" co-rid=\"_co_idm139716155526240\" co-class=\"co-affbox\">Francis M. Martin</a><sup><img src=\"https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/corrauth.gif\"\n alt=\"corresponding author\" /></sup><sup>1,</sup><sup>24</sup></span><span style=\"display:none\" class=\"contrib-group aff-tip\"><span id=\"_co_idm139716228127600\"><h3 class=\"no_margin\">Shingo Miyauchi</h3><p><sup>1</sup>Universit&#x000e9; de Lorraine, Institut national de recherche pour l&#x02019;agriculture, l&#x02019;alimentation et l&#x02019; environnement, UMR Interactions Arbres/Microorganismes, Centre INRAE Grand Est-Nancy, 54280 Champenoux, France </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Miyauchi%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Shingo Miyauchi</a></span></span><span id=\"_co_idm139716134900144\"><h3 class=\"no_margin\">Enik&#x00151; Kiss</h3><p><sup>2</sup>Synthetic and Systems Biology Unit, Biological Research Centre, 6726 Szeged, Hungary </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Kiss%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Enik&#x00151; Kiss</a></span></span><span id=\"_co_idm139716147902144\"><h3 class=\"no_margin\">Alan Kuo</h3><p><sup>3</sup>US Department of Energy Joint Genome Institute, Lawrence Berkeley National Laboratory, Berkeley, CA USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Kuo%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Alan Kuo</a></span></span><span id=\"_co_idm139716136057216\"><h3 class=\"no_margin\">Elodie Drula</h3><p><sup>4</sup>INRAE, USC1408 Architecture et Fonction des Macromol&#x000e9;cules Biologiques, 13009 Marseille, France </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Drula%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Elodie Drula</a></span></span><span id=\"_co_idm139716142223344\"><h3 class=\"no_margin\">Annegret Kohler</h3><p><sup>1</sup>Universit&#x000e9; de Lorraine, Institut national de recherche pour l&#x02019;agriculture, l&#x02019;alimentation et l&#x02019; environnement, UMR Interactions Arbres/Microorganismes, Centre INRAE Grand Est-Nancy, 54280 Champenoux, France </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Kohler%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Annegret Kohler</a></span></span><span id=\"_co_idm139716150741008\"><h3 class=\"no_margin\">Marisol S&#x000e1;nchez-Garc&#x000ed;a</h3><p><sup>5</sup>Biology Department, Clark University, Lasry Center for Bioscience, 950 Main Street, Worcester, MA 01610 USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=S%26%23x000e1%3Bnchez-Garc%26%23x000ed%3Ba%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Marisol S&#x000e1;nchez-Garc&#x000ed;a</a></span></span><span id=\"_co_idm139716152135600\"><h3 class=\"no_margin\">Emmanuelle Morin</h3><p><sup>1</sup>Universit&#x000e9; de Lorraine, Institut national de recherche pour l&#x02019;agriculture, l&#x02019;alimentation et l&#x02019; environnement, UMR Interactions Arbres/Microorganismes, Centre INRAE Grand Est-Nancy, 54280 Champenoux, France </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Morin%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Emmanuelle Morin</a></span></span><span id=\"_co_idm139716192751456\"><h3 class=\"no_margin\">Bill Andreopoulos</h3><p><sup>3</sup>US Department of Energy Joint Genome Institute, Lawrence Berkeley National Laboratory, Berkeley, CA USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Andreopoulos%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Bill Andreopoulos</a></span></span><span id=\"_co_idm139716145758960\"><h3 class=\"no_margin\">Kerrie W. Barry</h3><p><sup>3</sup>US Department of Energy Joint Genome Institute, Lawrence Berkeley National Laboratory, Berkeley, CA USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Barry%20KW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Kerrie W. Barry</a></span></span><span id=\"_co_idm139716146524576\"><h3 class=\"no_margin\">Gregory Bonito</h3><p><sup>6</sup>Plant Soil and Microbial Sciences, Michigan State University, East Lansing, MI 48824 USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Bonito%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Gregory Bonito</a></span></span><span id=\"_co_idm139716139564240\"><h3 class=\"no_margin\">Marc Bu&#x000e9;e</h3><p><sup>1</sup>Universit&#x000e9; de Lorraine, Institut national de recherche pour l&#x02019;agriculture, l&#x02019;alimentation et l&#x02019; environnement, UMR Interactions Arbres/Microorganismes, Centre INRAE Grand Est-Nancy, 54280 Champenoux, France </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Bu%26%23x000e9%3Be%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Marc Bu&#x000e9;e</a></span></span><span id=\"_co_idm139716140279152\"><h3 class=\"no_margin\">Akiko Carver</h3><p><sup>3</sup>US Department of Energy Joint Genome Institute, Lawrence Berkeley National Laboratory, Berkeley, CA USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Carver%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Akiko Carver</a></span></span><span id=\"_co_idm139716145097856\"><h3 class=\"no_margin\">Cindy Chen</h3><p><sup>3</sup>US Department of Energy Joint Genome Institute, Lawrence Berkeley National Laboratory, Berkeley, CA USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Chen%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Cindy Chen</a></span></span><span id=\"_co_idm139716190752224\"><h3 class=\"no_margin\">Nicolas Cichocki</h3><p><sup>1</sup>Universit&#x000e9; de Lorraine, Institut national de recherche pour l&#x02019;agriculture, l&#x02019;alimentation et l&#x02019; environnement, UMR Interactions Arbres/Microorganismes, Centre INRAE Grand Est-Nancy, 54280 Champenoux, France </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Cichocki%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Nicolas Cichocki</a></span></span><span id=\"_co_idm139716134964096\"><h3 class=\"no_margin\">Alicia Clum</h3><p><sup>3</sup>US Department of Energy Joint Genome Institute, Lawrence Berkeley National Laboratory, Berkeley, CA USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Clum%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Alicia Clum</a></span></span><span id=\"_co_idm139716146596960\"><h3 class=\"no_margin\">David Culley</h3><p><sup>7</sup>Chemical &#x00026; Biological Processes Development Group, Pacific Northwest National Laboratory, Richland, WA USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Culley%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">David Culley</a></span></span><span id=\"_co_idm139716146634528\"><h3 class=\"no_margin\">Pedro W. Crous</h3><p><sup>8</sup>Westerdijk Fungal Biodiversity Institute, Uppsalalaan 8, 3584 CT Utrecht, Netherlands </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Crous%20PW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Pedro W. Crous</a></span></span><span id=\"_co_idm139716143760544\"><h3 class=\"no_margin\">Laure Fauchery</h3><p><sup>1</sup>Universit&#x000e9; de Lorraine, Institut national de recherche pour l&#x02019;agriculture, l&#x02019;alimentation et l&#x02019; environnement, UMR Interactions Arbres/Microorganismes, Centre INRAE Grand Est-Nancy, 54280 Champenoux, France </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Fauchery%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Laure Fauchery</a></span></span><span id=\"_co_idm139716154423264\"><h3 class=\"no_margin\">Mariangela Girlanda</h3><p><sup>9</sup>Department of Life Sciences and Systems Biology, University of Torino, Viale Mattioli 25, 10125 Torino, Italy </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Girlanda%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Mariangela Girlanda</a></span></span><span id=\"_co_idm139716144620384\"><h3 class=\"no_margin\">Richard D. Hayes</h3><p><sup>3</sup>US Department of Energy Joint Genome Institute, Lawrence Berkeley National Laboratory, Berkeley, CA USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Hayes%20RD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Richard D. Hayes</a></span></span><span id=\"_co_idm139716157979152\"><h3 class=\"no_margin\">Zs&#x000f3;fia K&#x000e9;ri</h3><p><sup>2</sup>Synthetic and Systems Biology Unit, Biological Research Centre, 6726 Szeged, Hungary </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=K%26%23x000e9%3Bri%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Zs&#x000f3;fia K&#x000e9;ri</a></span></span><span id=\"_co_idm139716149677744\"><h3 class=\"no_margin\">Kurt LaButti</h3><p><sup>3</sup>US Department of Energy Joint Genome Institute, Lawrence Berkeley National Laboratory, Berkeley, CA USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=LaButti%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Kurt LaButti</a></span></span><span id=\"_co_idm139716157395504\"><h3 class=\"no_margin\">Anna Lipzen</h3><p><sup>3</sup>US Department of Energy Joint Genome Institute, Lawrence Berkeley National Laboratory, Berkeley, CA USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Lipzen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Anna Lipzen</a></span></span><span id=\"_co_idm139716141959984\"><h3 class=\"no_margin\">Vincent Lombard</h3><p><sup>4</sup>INRAE, USC1408 Architecture et Fonction des Macromol&#x000e9;cules Biologiques, 13009 Marseille, France </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Lombard%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Vincent Lombard</a></span></span><span id=\"_co_idm139716228294528\"><h3 class=\"no_margin\">Jon Magnuson</h3><p><sup>7</sup>Chemical &#x00026; Biological Processes Development Group, Pacific Northwest National Laboratory, Richland, WA USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Magnuson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Jon Magnuson</a></span></span><span id=\"_co_idm139716139635360\"><h3 class=\"no_margin\">Fran&#x000e7;ois Maillard</h3><p><sup>1</sup>Universit&#x000e9; de Lorraine, Institut national de recherche pour l&#x02019;agriculture, l&#x02019;alimentation et l&#x02019; environnement, UMR Interactions Arbres/Microorganismes, Centre INRAE Grand Est-Nancy, 54280 Champenoux, France </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Maillard%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Fran&#x000e7;ois Maillard</a></span></span><span id=\"_co_idm139716150986960\"><h3 class=\"no_margin\">Claude Murat</h3><p><sup>1</sup>Universit&#x000e9; de Lorraine, Institut national de recherche pour l&#x02019;agriculture, l&#x02019;alimentation et l&#x02019; environnement, UMR Interactions Arbres/Microorganismes, Centre INRAE Grand Est-Nancy, 54280 Champenoux, France </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Murat%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Claude Murat</a></span></span><span id=\"_co_idm139716139980112\"><h3 class=\"no_margin\">Matt Nolan</h3><p><sup>3</sup>US Department of Energy Joint Genome Institute, Lawrence Berkeley National Laboratory, Berkeley, CA USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Nolan%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Matt Nolan</a></span></span><span id=\"_co_idm139716151825440\"><h3 class=\"no_margin\">Robin A. Ohm</h3><p><sup>3</sup>US Department of Energy Joint Genome Institute, Lawrence Berkeley National Laboratory, Berkeley, CA USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Ohm%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Robin A. Ohm</a></span></span><span id=\"_co_idm139716135769616\"><h3 class=\"no_margin\">Jasmyn Pangilinan</h3><p><sup>3</sup>US Department of Energy Joint Genome Institute, Lawrence Berkeley National Laboratory, Berkeley, CA USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Pangilinan%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Jasmyn Pangilinan</a></span></span><span id=\"_co_idm139716148756560\"><h3 class=\"no_margin\">Ma&#x000ed;ra de Freitas Pereira</h3><p><sup>1</sup>Universit&#x000e9; de Lorraine, Institut national de recherche pour l&#x02019;agriculture, l&#x02019;alimentation et l&#x02019; environnement, UMR Interactions Arbres/Microorganismes, Centre INRAE Grand Est-Nancy, 54280 Champenoux, France </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Pereira%20Md%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Ma&#x000ed;ra de Freitas Pereira</a></span></span><span id=\"_co_idm139716145148448\"><h3 class=\"no_margin\">Silvia Perotto</h3><p><sup>9</sup>Department of Life Sciences and Systems Biology, University of Torino, Viale Mattioli 25, 10125 Torino, Italy </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Perotto%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Silvia Perotto</a></span></span><span id=\"_co_idm139716142844832\"><h3 class=\"no_margin\">Martina Peter</h3><p><sup>10</sup>Swiss Federal Institute for Forest, Snow and Landscape Research WSL, Zuercherstrasse 111, 8903 Birmensdorf, Switzerland </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Peter%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Martina Peter</a></span></span><span id=\"_co_idm139716228320320\"><h3 class=\"no_margin\">Stephanie Pfister</h3><p><sup>10</sup>Swiss Federal Institute for Forest, Snow and Landscape Research WSL, Zuercherstrasse 111, 8903 Birmensdorf, Switzerland </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Pfister%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Stephanie Pfister</a></span></span><span id=\"_co_idm139716145024000\"><h3 class=\"no_margin\">Robert Riley</h3><p><sup>3</sup>US Department of Energy Joint Genome Institute, Lawrence Berkeley National Laboratory, Berkeley, CA USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Riley%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Robert Riley</a></span></span><span id=\"_co_idm139716141552928\"><h3 class=\"no_margin\">Yaron Sitrit</h3><p><sup>11</sup>The Jacob Blaustein Institutes for Desert Research, Bergman Campus, Ben-Gurion University of The Negev, Beer-Sheva, Israel </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Sitrit%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Yaron Sitrit</a></span></span><span id=\"_co_idm139716151478240\"><h3 class=\"no_margin\">J. Benjamin Stielow</h3><p><sup>8</sup>Westerdijk Fungal Biodiversity Institute, Uppsalalaan 8, 3584 CT Utrecht, Netherlands </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Stielow%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">J. Benjamin Stielow</a></span></span><span id=\"_co_idm139716145972176\"><h3 class=\"no_margin\">Gergely Sz&#x000f6;ll&#x00151;si</h3><p><sup>2</sup>Synthetic and Systems Biology Unit, Biological Research Centre, 6726 Szeged, Hungary </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Sz%26%23x000f6%3Bll%26%23x00151%3Bsi%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Gergely Sz&#x000f6;ll&#x00151;si</a></span></span><span id=\"_co_idm139716146644496\"><h3 class=\"no_margin\">Lucia &#x0017d;if&#x0010d;&#x000e1;kov&#x000e1;</h3><p><sup>12</sup>Laboratory of Environmental Microbiology, Institute of Microbiology of the Czech Academy of Sciences, Videnska 1083, 14220 Praha 4, Czech Republic </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=%26%23x0017d%3Bif%26%23x0010d%3B%26%23x000e1%3Bkov%26%23x000e1%3B%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Lucia &#x0017d;if&#x0010d;&#x000e1;kov&#x000e1;</a></span></span><span id=\"_co_idm139716155280800\"><h3 class=\"no_margin\">Martina &#x00160;tursov&#x000e1;</h3><p><sup>12</sup>Laboratory of Environmental Microbiology, Institute of Microbiology of the Czech Academy of Sciences, Videnska 1083, 14220 Praha 4, Czech Republic </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=%26%23x00160%3Btursov%26%23x000e1%3B%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Martina &#x00160;tursov&#x000e1;</a></span></span><span id=\"_co_idm139716150252896\"><h3 class=\"no_margin\">Joseph W. Spatafora</h3><p><sup>13</sup>Department Botany &#x00026; Plant Pathology, Oregon State University, Corvallis, OR USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Spatafora%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Joseph W. Spatafora</a></span></span><span id=\"_co_idm139716145983648\"><h3 class=\"no_margin\">Leho Tedersoo</h3><p><sup>14</sup>Natural History Museum, University of Tartu, 14a Ravila, 50411 Tartu, Estonia </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Tedersoo%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Leho Tedersoo</a></span></span><span id=\"_co_idm139716154333168\"><h3 class=\"no_margin\">Lu-Min Vaario</h3><p><sup>15</sup>Department of Forest Sciences, University of Helsinki, Helsinki, Finland </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Vaario%20LM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Lu-Min Vaario</a></span></span><span id=\"_co_idm139716142447616\"><h3 class=\"no_margin\">Akiyoshi Yamada</h3><p><sup>16</sup>Institute of Mountain Science, Faculty of Agriculture, Shinshu University, Minami-minowa, Kami-ina, Nagano, 399-4598 Japan </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Yamada%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Akiyoshi Yamada</a></span></span><span id=\"_co_idm139716153489712\"><h3 class=\"no_margin\">Mi Yan</h3><p><sup>3</sup>US Department of Energy Joint Genome Institute, Lawrence Berkeley National Laboratory, Berkeley, CA USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Yan%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Mi Yan</a></span></span><span id=\"_co_idm139716138936752\"><h3 class=\"no_margin\">Pengfei Wang</h3><p><sup>17</sup>Department of Key Laboratory, The 2nd Affiliated Hospital of Kunming Medical University, 374 Dian Mian Road, Kunming, 650101 Yunnan China </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Wang%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Pengfei Wang</a></span></span><span id=\"_co_idm139716157773584\"><h3 class=\"no_margin\">Jianping Xu</h3><p><sup>18</sup>Department of Biology, McMaster University, 1280 Main St. West, Hamilton, ON L8S 4K1 Canada </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Xu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Jianping Xu</a></span></span><span id=\"_co_idm139716137413920\"><h3 class=\"no_margin\">Tom Bruns</h3><p><sup>19</sup>Department of Plant and Microbial Biology, University of California &#x02013; Berkeley, Berkeley, CA USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Bruns%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Tom Bruns</a></span></span><span id=\"_co_idm139716138390000\"><h3 class=\"no_margin\">Petr Baldrian</h3><p><sup>12</sup>Laboratory of Environmental Microbiology, Institute of Microbiology of the Czech Academy of Sciences, Videnska 1083, 14220 Praha 4, Czech Republic </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Baldrian%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Petr Baldrian</a></span></span><span id=\"_co_idm139716152916272\"><h3 class=\"no_margin\">Rytas Vilgalys</h3><p><sup>20</sup>Department of Biology, Duke University, Durham, NC 27708 USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Vilgalys%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Rytas Vilgalys</a></span></span><span id=\"_co_idm139716135954448\"><h3 class=\"no_margin\">Christophe Dunand</h3><p><sup>21</sup>Laboratoire de Recherche en Sciences V&#x000e9;g&#x000e9;tales, Universit&#x000e9; de Toulouse, CNRS, UPS, Toulouse, France </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Dunand%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Christophe Dunand</a></span></span><span id=\"_co_idm139716150168544\"><h3 class=\"no_margin\">Bernard Henrissat</h3><p><sup>4</sup>INRAE, USC1408 Architecture et Fonction des Macromol&#x000e9;cules Biologiques, 13009 Marseille, France </p><p><sup>22</sup>Architecture et Fonction des Macromol&#x000e9;cules Biologiques (AFMB), CNRS, Aix-Marseille Univ., 13009 Marseille, France </p><p><sup>23</sup>Department of Biological Sciences, King Abdulaziz University, Jeddah, Saudi Arabia </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Henrissat%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Bernard Henrissat</a></span></span><span id=\"_co_idm139716151639936\"><h3 class=\"no_margin\">Igor V. Grigoriev</h3><p><sup>3</sup>US Department of Energy Joint Genome Institute, Lawrence Berkeley National Laboratory, Berkeley, CA USA </p><p><sup>19</sup>Department of Plant and Microbial Biology, University of California &#x02013; Berkeley, Berkeley, CA USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Grigoriev%20IV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Igor V. Grigoriev</a></span></span><span id=\"_co_idm139716139049376\"><h3 class=\"no_margin\">David Hibbett</h3><p><sup>5</sup>Biology Department, Clark University, Lasry Center for Bioscience, 950 Main Street, Worcester, MA 01610 USA </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Hibbett%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">David Hibbett</a></span></span><span id=\"_co_idm139716156121680\"><h3 class=\"no_margin\">L&#x000e1;szl&#x000f3; G. Nagy</h3><p><sup>2</sup>Synthetic and Systems Biology Unit, Biological Research Centre, 6726 Szeged, Hungary </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Nagy%20LG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">L&#x000e1;szl&#x000f3; G. Nagy</a></span></span><span id=\"_co_idm139716155526240\"><h3 class=\"no_margin\">Francis M. Martin</h3><p><sup>1</sup>Universit&#x000e9; de Lorraine, Institut national de recherche pour l&#x02019;agriculture, l&#x02019;alimentation et l&#x02019; environnement, UMR Interactions Arbres/Microorganismes, Centre INRAE Grand Est-Nancy, 54280 Champenoux, France </p><p><sup>24</sup>Beijing Advanced Innovation Centre for Tree Breeding by Molecular Design (BAIC-TBMD), Institute of Microbiology, Beijing Forestry University, Tsinghua East Road Haidian District, Beijing, China </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Martin%20FM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33046698\">Francis M. Martin</a></span></span></span></span><span class=\"fm-panel half_rhythm\"><span class=\"togglers<br><span id=\"idm\n1\" lang=\"en\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm\n1title\">Abstract</h2><!--article-meta--><span><p id=\"Par1\" class=\"p p-first-last\">Mycorrhizal fungi are mutualists that play crucial roles in nutrient acquisition in terrestrial ecosystems. Mycorrhizal symbioses arose repeatedly across multiple lineages of Mucoromycotina, Ascomycota, and Basidiomycota. Considerable variation exists in the capacity of mycorrhizal fungi to acquire carbon from soil organic matter. Here, we present a combined analysis of 135 fungal genomes from 73 saprotrophic, endophytic and pathogenic species, and 62 mycorrhizal species, including 29 new mycorrhizal genomes. This study samples ecologically dominant fungal guilds for which there were previously no symbiotic genomes available, including ectomycorrhizal Russulales, Thelephorales and Cantharellales. Our analyses show that transitions from saprotrophy to symbiosis involve (1) widespread losses of degrading enzymes acting on lignin and cellulose, (2) co-option of genes present in saprotrophic ancestors to fulfill new symbiotic functions, (3) diversification of novel, lineage-specific symbiosis-induced genes, (4) proliferation of transposable elements and (5) divergent genetic innovations underlying the convergent origins of the ectomycorrhizal guild.</p></span><span class=\"sec\"><strong class=\"kwd-title\">Subject terms: </strong><span class=\"kwd-text\">Microbial ecology, Evolutionary developmental biology</span></span></span><span id=\"s\n1\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\n1title\">Introduction</h2><p id=\"Par3\" class=\"p p-first\">Mycorrhizal fungi are central to the evolution, biology, and physiology of land plants because they promote plant growth by facilitating the acquisition of scarce and essential nutrients, such as phosphorus and nitrogen<sup><sup><a href=\"#CR1\" rid=\"CR1\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273260\">1</a></sup>&#x02013;<sup><a href=\"#CR3\" rid=\"CR3\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273223\">3</a></sup></sup>. They are also major drivers of carbon sequestration and they have a well-documented impact on the composition of microbial and plant communities<sup><sup><a href=\"#CR1\" rid=\"CR1\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273318\">1</a></sup>,<sup><a href=\"#CR2\" rid=\"CR2\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273231\">2</a></sup></sup>. The most ubiquitous classes of mycorrhizal symbioses are ectomycorrhiza, arbuscular mycorrhiza, orchid mycorrhiza, and ericoid mycorrhiza<sup><sup><a href=\"#CR4\" rid=\"CR4\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273322\">4</a></sup>,<sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273259\">5</a></sup></sup>. Each class is classified based on host plant and characteristic symbiotic structures. Although mycorrhizal fungi are highly diverse in terms of their evolutionary history, the independent evolution of similar symbiotic morphological structures, and physiological traits in divergent fungal taxa provides a striking example of convergent evolution<sup><sup><a href=\"#CR3\" rid=\"CR3\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273255\">3</a></sup>&#x02013;<sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273279\">5</a></sup></sup>. Although unique and common traits in mycorrhizal symbioses have recently been reviewed<sup><sup><a href=\"#CR2\" rid=\"CR2\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273226\">2</a></sup></sup>, molecular mechanisms underlying these convergent phenotypes remain largely undetermined<sup><sup><a href=\"#CR1\" rid=\"CR1\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273265\">1</a></sup>,<sup><a href=\"#CR3\" rid=\"CR3\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273280\">3</a></sup>&#x02013;<sup><a href=\"#CR8\" rid=\"CR8\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273261\">8</a></sup></sup>.</p><p id=\"Par4\">Prior comparisons of genomes from ectomycorrhizal, orchid and ericoid mycorrhizal fungi, wood decayers and soil decomposers have elucidated the mechanisms of several transitions from saprotrophy to mutualism in Dikarya<sup><sup><a href=\"#CR9\" rid=\"CR9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273285\">9</a></sup>&#x02013;<sup><a href=\"#CR16\" rid=\"CR16\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273222\">16</a></sup></sup>. These analyses have shown that multiple lineages of ectomycorrhizal fungi have lost most genes encoding lignocellulose&#x02010;degrading enzymes present in their saprotrophic ancestors, explaining the reduced capacity of ectomycorrhizal fungi to acquire C complexed in soil organic matter (SOM) and plant cell walls<sup><sup><a href=\"#CR17\" rid=\"CR17\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273283\">17</a></sup></sup> and, as a consequence, their increasing dependence on the host plant sugars. The diversity of trophic states in extant ectomycorrhizal fungi may be a consequence of their multiple origins from saprotrophic ancestors with varied decay capabilities, including white and brown rot wood decayers, and soil and litter decomposers<sup><sup><a href=\"#CR3\" rid=\"CR3\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273270\">3</a></sup>&#x02013;<sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273243\">5</a></sup></sup>. However, the extent to which ectomycorrhizal fungi make use of their secreted plant cell wall degrading enzymes (PCWDEs) and microbial cell wall degrading enzymes (MCWDE) to decay or decompose SOM is not well understood<sup><sup><a href=\"#CR17\" rid=\"CR17\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273220\">17</a></sup>&#x02013;<sup><a href=\"#CR24\" rid=\"CR24\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273271\">24</a></sup></sup>.</p><p id=\"Par5\">Despite their ecological prominence, much remains to be learned about the evolution and functional diversification of mycorrhizal symbionts and the crucial acquisitions that allow colonization of and nutrient exchange with plants<sup><sup><a href=\"#CR3\" rid=\"CR3\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273249\">3</a></sup>,<sup><a href=\"#CR25\" rid=\"CR25\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273228\">25</a></sup></sup>. Here, we present a combined analysis of 135 fungal genomes from 73 saprotrophic, endophytic and pathogenic fungal species, and 62 mycorrhizal fungal species, including 29 new mycorrhizal genomes. This study approximately doubles the number of published genomes of mycorrhizal fungi, and it samples major groups, for which there were previously no symbiotic genomes available, including Russulales, Thelephorales, Phallomycetidae, and Cantharellales. These groups are important, because they (1) are often ecologically dominant (Russulales and Thelephorales), (2) represent early diverging clades for which no ectomycorrhizal genomes were previously available (Phallomycetidae and Cantharellales), and (3) include groups that arose before (Cantharellales) or after the origin of ligninolytic peroxidases (class II POD) implicated in white rot<sup><sup><a href=\"#CR26\" rid=\"CR26\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273286\">26</a></sup></sup>.</p><p id=\"Par6\" class=\"p p-last\">This dataset presents an opportunity to carry out a broader analysis of the evolution of saprotrophic capabilities than that we previously attempted in our large-scale comparative analysis of Agaricomycetidae<sup><sup><a href=\"#CR12\" rid=\"CR12\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273237\">12</a></sup></sup>. In the present study, we hypothesize that the evolutionary mechanisms in action in Agaricomycetidae can be traced back to the early diverging clades of ectomycorrhizal fungi. To assess gains of ectomycorrhizal lifestyle traits, we discuss the fundamental adaptations that underlie convergent evolution of ectomycorrhizal fungi, including the loss of some metabolic functions, such as PCWDEs, and the acquisition of small secreted effector-like proteins that may facilitate the accommodation of symbiotic fungi within their host plants. We also compare additional symbiosis-related functional traits, such as nitrogen and phosphate acquisition. Finally, we investigate the age distribution of symbiosis-upregulated genes across a phylogenetically representative set of ectomycorrhizal fungi, using a phylostratigraphic approach to resolve lineage-specific and conserved elements of symbiotic transcriptomes. We show that transitions from saprotrophy to ectomycorrhizal symbiosis involve widespread losses of degrading enzymes acting on lignocellulose, co-option of genes present in saprotrophic ancestors to fulfill novel symbiotic functions, diversification of lineage-specific symbiosis-induced genes, proliferation of transposable elements (TEs), and divergent genetic innovations underlying the convergent origins of the ectomycorrhizal guild.</p></span><span id=\"s\n2\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\n2title\">Results</h2><span id=\"s\n3\" class=\"sec sec-first\"><h3 id=\"s\n3title\">Main features of mycorrhizal genomes</h3><p id=\"Par7\" class=\"p p-first-last\">We compared 62 draft genomes from mycorrhizal fungi, including 29 newly released genomes, and predicted 9344&#x02013;31,291 protein-coding genes per species (see &#x0201c;Methods&#x0201d;, Supplementary Information and Supplementary Data&#x000a0;<sup><a href=\"#MOESM5\" rid=\"MOESM5\" class=\" media\">1</a></sup>). This set includes new genomes from the early diverging fungal clades in the Russulales, Thelephorales, Phallomycetidae, and Cantharellales (Basidiomycota), and Helotiales and Pezizales (Ascomycota). We combined these mycorrhizal fungal genomes with 73 fungal genomes from wood decayers, soil/litter saprotrophs, and root endophytes (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\" co-legend-rid=\"lgnd_Fig1\"><span>1</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM6\" rid=\"MOESM6\" class=\" media\">2</a></sup>). There was little variation in the completeness of the gene repertoires, based on Benchmarking Universal Single-Copy Orthologs (BUSCO) analysis (coefficient of variation, c.v.&#x02009;=&#x02009;7.98), despite variation in assembly contiguity (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\" co-legend-rid=\"lgnd_Fig1\"><span>1</span></a>). Genome size varied greatly within each phylum, with genomes of mycorrhizal fungi being larger than those of saprotrophic species (Figs.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\" co-legend-rid=\"lgnd_Fig1\"><span>1</span></a> and <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig2\" rid-ob=\"ob-Fig2\" co-legend-rid=\"lgnd_Fig2\"><span style=\"position: relative;text-decoration:none;\">&#x200B;<span class=\"figpopup-sensitive-area\" style=\"left: -1.5em;\">and2,</span></span><span>2</span></a>, and Supplementary Data&#x000a0;<sup><a href=\"#MOESM6\" rid=\"MOESM6\" class=\" media\">2</a></sup>; <em>P</em>&#x02009;&#x0003c;&#x02009;0.05, generalized least squares with the Brownian motion model (GLS)). Glomeromycotina had exceptionally large genomes (Figs.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\" co-legend-rid=\"lgnd_Fig1\"><span>1</span></a> and <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig2\" rid-ob=\"ob-Fig2\" co-legend-rid=\"lgnd_Fig2\"><span style=\"position: relative;text-decoration:none;\">&#x200B;<span class=\"figpopup-sensitive-area\" style=\"left: -1.5em;\">and2,</span></span><span>2</span></a>, and Supplementary Data&#x000a0;<sup><a href=\"#MOESM6\" rid=\"MOESM6\" class=\" media\">2</a></sup>), with <em>Gigaspora rosea</em> having the largest genome (567&#x02009;Mb) among the 135 fungi compared<sup><sup><a href=\"#CR27\" rid=\"CR27\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273238\">27</a></sup></sup>.</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig1\" co-legend-rid=\"lgnd_Fig1\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig1/\" target=\"figure\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139716154461744\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7550596_41467_2020_18795_Fig1_HTML.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2020_18795_Fig1_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7550596/bin/41467_2020_18795_Fig1_HTML.jpg\" /></a></span><span id=\"largeobj_idm139716154461744\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig1/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_Fig1\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig1/\" target=\"figure\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\">Fig. 1</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>General features of the 135 fungal genomes analyzed.</strong><p id=\"__p1\">Genome and assembly features. The different lifestyles (e.g., ectomycorrhiza) are color coded, see bottom panel. The dotted lines show median values. Genome&#x000a0;(Mb): genome size in Mb, TE content&#x000a0;(%): coverage (%) of transposable elements (TE) in assemblies, Genes&#x000a0;(K): number of predicted genes, Secreted&#x000a0;(K): number of predicted secreted proteins, Scaffolds&#x000a0;(K): number of scaffolds, L50&#x000a0;(Mb): N50 length, BUSCO&#x000a0;(%): genome completeness in % based on BUSCO survey. Source data Fig.&#x000a0;<sup><a href=\"#MOESM21\" rid=\"MOESM21\" class=\" media\">1</a></sup>.</p></span></span></span><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig2\" co-legend-rid=\"lgnd_Fig2\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig2/\" target=\"figure\" rid-figpopup=\"Fig2\" rid-ob=\"ob-Fig2\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139716155702368\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7550596_41467_2020_18795_Fig2_HTML.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2020_18795_Fig2_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7550596/bin/41467_2020_18795_Fig2_HTML.jpg\" /></a></span><span id=\"largeobj_idm139716155702368\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig2/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_Fig2\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig2/\" target=\"figure\" rid-figpopup=\"Fig2\" rid-ob=\"ob-Fig2\">Fig. 2</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>The distribution of genome size (in Mb) and repeat element coverage (%) for each lifestyles.</strong><p id=\"__p2\">The boxes represent median, upper, and lower quartiles with the whiskers showing minimal and maximal values, and outliers in circle. The small dots show single observations. The number of species per lifestyle is as follows: ectomycorrhizal fungi (<em>n</em>&#x02009;=&#x02009;45), arbuscular mycorrhizal fungi (<em>n</em>&#x02009;=&#x02009;10), ericoid mycorrhizal fungi (<em>n</em>&#x02009;=&#x02009;4), orchid mycorrhizal fungi (<em>n</em>&#x02009;=&#x02009;3), endophytes (<em>n</em>&#x02009;=&#x02009;5), pathogens (<em>n</em>&#x02009;=&#x02009;14), soil/litter saprotrophs (<em>n</em>&#x02009;=&#x02009;32), and wood decayers (<em>n</em>&#x02009;=&#x02009;17). Source data Fig.&#x000a0;<sup><a href=\"#MOESM21\" rid=\"MOESM21\" class=\" media\">1</a></sup>.</p></span></span></span></span><span id=\"s\n4\" class=\"sec\"><h3 id=\"s\n4title\">TEs in mycorrhizal genomes</h3><p id=\"Par8\" class=\"p p-first-last\">The main driver of genome inflation appeared to be repeat content (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM7\" rid=\"MOESM7\" class=\" media\">3</a></sup>; <em>P</em>&#x02009;&#x0003c;&#x02009;0.05 GLS), such as long terminal repeat retrotransposons (LTRs), which ranged from 0.01 to 46.4% of the assembly (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM7\" rid=\"MOESM7\" class=\" media\">3</a></sup>). The distribution of TE categories varies between ectomycorrhizal fungal taxa even within the same fungal orders or genera (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3</span></a>), indicating that invasions by different TE families took place independently in different clades. However, the total TE coverage in genomes of ectomycorrhizal Ascomycota and Glomeromycota was significantly higher than in Basidiomycota (<em>P</em>&#x02009;&#x0003c;&#x02009;0.05, GLS; Supplementary Data&#x000a0;<sup><a href=\"#MOESM7\" rid=\"MOESM7\" class=\" media\">3</a></sup>). In Ascomycota, the most abundant TE families are LTRs, such as <em>Gypsy</em> and <em>Copia</em>, and non-LTR I, whereas in Basidiomycota, <em>Gypsy</em> and <em>Copia</em> LTRs, <em>Tad1</em>, helitrons, and Zizuptons are abundant (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3</span></a>). Lifestyle has a higher impact than phylogeny on TE coverage with a significantly higher TE content in ectomycorrhizal symbionts compared to other lifestyles for several TE categories (14&#x02013;35% of total variances, <em>P</em> value &#x0003c;0.05; PERMANOVA) (Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">1a, b</a></sup>, and Supplementary Data&#x000a0;<sup><a href=\"#MOESM7\" rid=\"MOESM7\" class=\" media\">3</a></sup> and <sup><a href=\"#MOESM8\" rid=\"MOESM8\" class=\" media\">4</a></sup>). In Agaricales, a large proportion of LTR insertions occurred during the past 5 million years (Mya) in <em>Amanita rubescens</em>, <em>Laccaria bicolor</em>, <em>Laccaria amethystina</em>, and <em>Cortinarius glaucopus</em>, while the major bursts are more ancient in <em>Tricholoma matsutake</em> (Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">1c</a></sup>). These LTRs have been proliferating over the past 5&#x02013;6&#x02009;Mya for <em>Tuber melanosporum</em>, <em> Tuber&#x000a0;aestivum</em>, and <em>Choiromyces venosus</em>, while LTR accumulation in <em>Tuber magnatum</em> occurred between 6 and 14&#x02009;Mya (ref. <sup><sup><a href=\"#CR16\" rid=\"CR16\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273305\">16</a></sup></sup>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig3\" co-legend-rid=\"lgnd_Fig3\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig3/\" target=\"figure\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139716139111168\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7550596_41467_2020_18795_Fig3_HTML.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2020_18795_Fig3_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7550596/bin/41467_2020_18795_Fig3_HTML.jpg\" /></a></span><span id=\"largeobj_idm139716139111168\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig3/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_Fig3\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig3/\" target=\"figure\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\">Fig. 3</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Distribution and coverage (%) of transposable elements families in genome assemblies.</strong><p id=\"__p3\">The bubble size is proportional to the coverage of each of TE family (% indicated inside bubbles). The bars show the total coverage per genome. See also Supplementary Data&#x000a0;<sup><a href=\"#MOESM6\" rid=\"MOESM6\" class=\" media\">2</a></sup>&#x02013;<sup><a href=\"#MOESM8\" rid=\"MOESM8\" class=\" media\">4</a></sup>. Source data are provided as a Source data Fig.&#x000a0;<sup><a href=\"#MOESM21\" rid=\"MOESM21\" class=\" media\">1</a></sup>.</p></span></span></span></span><span id=\"s\n5\" class=\"sec\"><h3 id=\"s\n5title\">Phylogeny, orthologous, and paralogous genes</h3><p id=\"Par9\" class=\"p p-first\">Reconstructed phylogenetic relationships and estimated divergence dates are generally consistent with the dates recovered by previous studies. For example, we estimated the age of the most recent common ancestor (MRCA) of the Agaricomycetidae to be 132&#x02009;Mya (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4a</span></a>), while it was estimated to be 149 or 125&#x02009;Mya in phylogenomic analyses by Floudas et al.<sup><sup><a href=\"#CR26\" rid=\"CR26\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273309\">26</a></sup></sup> and Kohler et al.<sup><sup><a href=\"#CR12\" rid=\"CR12\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273316\">12</a></sup></sup>, but 191&#x02013;176&#x02009;Mya in a multigene megaphylogeny by Varga et al.<sup><sup><a href=\"#CR28\" rid=\"CR28\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273266\">28</a></sup></sup>. We estimated the ages of the MRCAs of Agaricomycetes, Agaricales, Polyporales, Russulales, and Boletales to be 280, 116, 104, 88, and 82&#x02009;Mya (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4a</span></a>), respectively. We estimated the MRCA of Pezizomycotina at 326&#x02009;Mya (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4b</span></a>), while Floudas et al.<sup><sup><a href=\"#CR26\" rid=\"CR26\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273315\">26</a></sup></sup> obtained a mean age of 344&#x02009;Mya.</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig4\" co-legend-rid=\"lgnd_Fig4\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig4/\" target=\"figure\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139716136343088\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7550596_41467_2020_18795_Fig4_HTML.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2020_18795_Fig4_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7550596/bin/41467_2020_18795_Fig4_HTML.jpg\" /></a></span><span id=\"largeobj_idm139716136343088\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig4/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_Fig4\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig4/\" target=\"figure\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\">Fig. 4</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Evolution of gene families encoding PCWDEs in Basidiomycota and Ascomycota.</strong><p id=\"__p4\"><strong>a</strong>&#x000a0;Basidiomycota. <strong>b</strong>&#x000a0;Ascomycota.&#x000a0;Fungal taxa are displayed according to their phylogeny (left panel). Bubbles with numbers at the tree nodes represent the total number of genes coding for total PCWDEs (intracellular and secreted) for ancestral nodes determined by COMPARE. The bubble size is proportional to the number of PCWDE genes. Heat maps contain the number of genes coding for substrate-specific PCWDEs for the extant species. Relative abundance of genes is represented by a color scale, from the minimum (blue) to maximum (red) number of copies per species. Ectomycorrhizal fungi are framed with a black line. See also Supplementary Data&#x000a0;<sup><a href=\"#MOESM12\" rid=\"MOESM12\" class=\" media\">8</a></sup> and Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">5</a></sup>. The tree is a chronogram estimated with r8s on the basis of a maximum likelihood phylogeny inferred with RAxML. The geological timescale (in million year) is indicated at the bottom. Source data are provided as a Source data Fig.&#x000a0;<sup><a href=\"#MOESM21\" rid=\"MOESM21\" class=\" media\">2</a></sup>.</p></span></span></span><p id=\"Par10\" class=\"p p-last\">We inferred gene families from the predicted proteomes using gene family clustering. A total of 68,923 and 129,258 gene families were identified in Ascomycota and Basidiomycota, respectively, and were used to infer orthology and paralogy (Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">5</a></sup>). The species in our datasets contained substantial novelty in gene content. Species-specific genes ranged from 994 in <em>T. melanosporum</em> to 39,410 in <em>Mycena galopus</em>, for a total of 251,392 and 606,378 taxon-specific genes in Ascomycota and Basidiomycota, respectively.The large number of species-specific genes in <em>M. galopus</em> is the result of a series of striking expansion of gene families.</p></span><span id=\"s\n6\" class=\"sec\"><h3 id=\"s\n6title\">Functional gene categories encoded by mycorrhizal genomes</h3><p id=\"Par11\" class=\"p p-first-last\">Hierarchical clustering of the presence and abundance of different Pfam protein domains identified genome-wide patterns of functional domain content among some of these fungi (Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">2a</a></sup>). Arbuscular mycorrhizal fungi clustered together with Pfam categories showing a substantial differential abundance in genes encoding proteins with NUDIX, tetratricopeptide repeat, BTB/POZ, Sel1 repeat, ubiquitin, and high-mobility group box domains, corroborating our previous study<sup><sup><a href=\"#CR27\" rid=\"CR27\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273230\">27</a></sup></sup>. This comparison also emphasizes some of the unique aspects of the genomes from the ectomycorrhizal ascomycete <em>Acephala macrosclerotiorum</em> (Helotiaceae) and ericoid mycorrhizal fungi (e.g., <em>Oidiodendron maius</em>, <em>Meliniomyces</em> species), such as the expansion of genes encoding proteins with FAD and AMP-binding domains, aldehyde dehydrogenases and sugar transporters. They share their Pfam pattern with soil saprotrophic ascomycete species, such as <em>Chalara longipes</em> and the dark septate endophyte (DSE) <em>Phialocephala scopiformis</em>. Ectomycorrhizal fungi in Basidiomycota are grouped with soil saprotrophs and wood decayers, and displayed no specific pattern in their primary and secondary metabolism gene repertoires that may explain their symbiosis-related ability. Similarly, hierarchical clustering of the presence and abundance of the different membrane transporters and transcriptional regulators revealed no specific pattern(s) for ectomycorrhizal fungi (Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">2b, c</a></sup>).</p></span><span id=\"s\n7\" class=\"sec\"><h3 id=\"s\n7title\">Predicted secretomes of saprotrophs and symbiotrophs</h3><p id=\"Par12\" class=\"p p-first-last\">As secreted proteins play a key role in SOM decomposition and symbiosis development, we compared predicted secretomes, including carbohydrate-active enzymes (CAZymes), lipases, proteases, and other secreted proteins, such as effector-like small secreted proteins (SSPs; Supplementary Data&#x000a0;<sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">6</a></sup>). The number of genes encoding secreted proteins represents 4.6&#x02013;5.7% of the total protein repertoire for Basidiomycota and Ascomycota, respectively (Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3</a></sup>). The proportions of secreted proteins within the different protein categories were consistent in both phyla (Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3</a></sup>, and Supplementary Data&#x000a0;<sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">6</a></sup> and <sup><a href=\"#MOESM11\" rid=\"MOESM11\" class=\" media\">7</a></sup>). About half of the secreted proteins were SSPs (Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3a</a></sup> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM11\" rid=\"MOESM11\" class=\" media\">7c</a></sup>). Only secreted CAZymes and SSPs showed significant differences in relative abundance among lifestyles (<em>P</em>&#x02009;&#x0003c;&#x02009;0.01; generalized Campbell and Skillings procedure; Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3</a></sup> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM12\" rid=\"MOESM12\" class=\" media\">8</a></sup>), with orchid and ericoid mycorrhizal symbionts possessing the largest CAZyme repertoires. On the other hand, the average number of secreted proteases and lipases is similar in saprotrophs and symbiotrophs. No expansion of gene families coding for secreted proteases, secreted phosphatases, and phytases were found in ectomycorrhizal fungi (Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3c</a></sup>).</p></span><span id=\"s\n8\" class=\"sec\"><h3 id=\"s\n8title\">Losses of PCWDEs</h3><p id=\"Par13\" class=\"p p-first\">In Basidiomycota, ectomycorrhizal species contain significantly fewer secreted CAZymes acting on cellulose, hemicellulose, pectins, lignin, suberins, and tannins (<em>P</em>&#x02009;&#x0003c;&#x02009;0.01, the generalized Campbell and Skillings procedure; Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span>5</span></a>, Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3c</a></sup> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">6</a></sup>) than all other ecological guilds, i.e., lifestyles. They are also reduced in secreted auxiliary activity (AA) enzymes (including class II PODs, laccases, lytic polysaccharide monooxygenases (LPMO)), carbohydrate esterases (CE), polysaccharide lyases (PL), and associated cellulose-binding modules (e.g., CBM1; Supplementary Figs.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3</a></sup>&#x02013;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">5</a></sup>, and Supplementary Data&#x000a0;<sup><a href=\"#MOESM11\" rid=\"MOESM11\" class=\" media\">7</a></sup> and <sup><a href=\"#MOESM12\" rid=\"MOESM12\" class=\" media\">8</a></sup>). Notably, there is no invertase GH32 gene in their genome, except for <em>Cantharellus anzutake</em>, implying that ectomycorrhizal basidiomycetes are unable to use sucrose directly from the plant. Except for <em>Acephala macrosclerotiorum</em>, the number of PCWDEs in ectomycorrhizal Ascomycota is lower than those of most of other groups, excluding yeasts and arbuscular mycorrhizal fungi (Figs.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4</span></a> and <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span style=\"position: relative;text-decoration:none;\">&#x200B;<span class=\"figpopup-sensitive-area\" style=\"left: -1.5em;\">and5,</span></span><span>5</span></a>, and Supplementary Figs.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3</a></sup>&#x02013;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">5</a></sup>). According to RedoxiBase (<a href=\"http://peroxibase.toulouse.inra.fr\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://peroxibase.toulouse.inra.fr</a>), none of the class II PODs of ectomycorrhizal species are ligninolytic (i.e., ligninolytic POD or LiP), except those of <em>Gautieria morchelliformis</em> (see below; Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">4</a></sup> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">6p</a></sup>). Below, we discuss the PCWDEs of ectomycorrhizal fungi, emphasizing newly sampled lineages with previously unknown decomposition capacity.</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig5\" co-legend-rid=\"lgnd_Fig5\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig5/\" target=\"figure\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139716141741152\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7550596_41467_2020_18795_Fig5_HTML.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2020_18795_Fig5_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7550596/bin/41467_2020_18795_Fig5_HTML.jpg\" /></a></span><span id=\"largeobj_idm139716141741152\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig5/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_Fig5\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig5/\" target=\"figure\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\">Fig. 5</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Distribution of key secreted PCWDEs in analyzed fungi.</strong><p id=\"__p5\">Bubbles with numbers contain the number of genes coding for a series of secreted PCWDEs needed for cellulose and lignin degradation. Taxa are color coded according to their lifestyle (see bottom panel). See also Supplementary Data&#x000a0;<sup><a href=\"#MOESM11\" rid=\"MOESM11\" class=\" media\">6a, j</a></sup>. Left panel, Ascomycota and Mucoromycota; right panel: Basidiomycota. Source data are provided as a Source data Fig.&#x000a0;<sup><a href=\"#MOESM21\" rid=\"MOESM21\" class=\" media\">3</a></sup>.</p></span></span></span><p id=\"Par14\">Cantharellales arose before the origin of ligninolytic class II POD<sup><sup><a href=\"#CR26\" rid=\"CR26\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273232\">26</a></sup>,<sup><a href=\"#CR28\" rid=\"CR28\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273276\">28</a></sup>&#x02013;<sup><a href=\"#CR30\" rid=\"CR30\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273281\">30</a></sup></sup>. Both saprotrophs (<em>Botryobasidium botryosum</em>, <em>Sistotrema</em> sp.) and orchid symbionts (<em>Tulasnella calospora</em>, <em>Ceratobasidium</em> sp.) in Cantharellales possess large sets of enzymes acting on cellulose, hemicellulose, and pectins, including LPMOs (AA9; Figs.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4</span></a> and <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span style=\"position: relative;text-decoration:none;\">&#x200B;<span class=\"figpopup-sensitive-area\" style=\"left: -1.5em;\">and5,</span></span><span>5</span></a>, and Supplementary Figs.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3</a></sup>&#x02013;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">5</a></sup>). We sequenced the first genomes of ectomycorrhizal Cantharellales, <em>Hydnum rufescens</em>, and <em>C. anzutake</em>, and found that they have highly reduced repertoires of secreted PCWDEs. In this regard, the ectomycorrhizal symbionts of Cantharellales are more similar to ectomycorrhizal Agaricales and Boletales than to orchid symbionts of Cantharellales. These results provide another independent example of convergent loss of PCWDEs in ectomycorrhizal lineages, and highlight the different decomposition capacities of two guilds of plant symbionts.</p><p id=\"Par15\">In contrast to Cantharellales, Phallomycetidae diverged shortly after the evolution of ligninolytic class II POD. The one previously published genome from this group, the saprotrophic &#x0201c;cannonball fungus&#x0201d;, <em>Sphaerobolus stellatus</em>, has an astonishingly large repertoire of PCWDEs, including 63 class II PODs. The two genomes of ectomycorrhizal Phallomycetidae reported here, <em>G. morchelliformis</em> (Gomphales) and <em>Hysterangium stoloniferum</em> (Hysterangiales)<sup><sup><a href=\"#CR31\" rid=\"CR31\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273314\">31</a></sup></sup>, have diverse PCWDEs acting on cellulose, hemicellulose, pectins, and lignin, with 31 and five genes encoding ligninolytic class II PODs, respectively (Figs.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4</span></a> and <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span style=\"position: relative;text-decoration:none;\">&#x200B;<span class=\"figpopup-sensitive-area\" style=\"left: -1.5em;\">and5,</span></span><span>5</span></a>, Supplementary Figs.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3</a></sup>&#x02013;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">5</a></sup> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM12\" rid=\"MOESM12\" class=\" media\">8</a></sup>). Although many <em>Ramaria</em> species (Gomphales) form ectomycorrhiza, the newly sequenced <em>Ramaria rubella</em> (subgenus <em>Lentoramaria</em>) is likely a litter decomposer, which is consistent with its possession of 13 class II PODs, 6 cellulose-acting LPMOs, and GH6 and GH7 cellobiohydrolases. Thus, a robust suite of PCWDEs appears to be a characteristic of both saprotrophs and ectomycorrhizal species in Phallomycetidae.</p><p id=\"Par16\">Ectomycorrhizal Russulales and Thelephorales, for which we report the first genomes, both have highly reduced complements of PCWDEs. The two Thelephorales species (<em>Thelephora terrestris</em> and <em>T. ganbajun</em>) resemble Boletales in having a highly reduced suite of the major enzymes acting on crystalline cellulose and lignin (three cellulose-acting LPMOs, no class II POD, and no GH6 or GH7). Ectomycorrhizal Russulales (<em>Lactarius quietus, Russula emetica</em>, and <em>Russula ochroleuca</em>) also have small repertoires of PCWDEs (no CBM1, one or two cellulose-acting LPMOs, no GH6 or GH7, no CE1, and no LiP), but they retain one atypical Mn POD gene (Supplementary Data&#x000a0;<sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">6p</a></sup>). In contrast, saprotrophic Russulales (e.g., <em>Lentinellus vulpinus</em>) have a typical white rot array of PCWDEs.</p><p id=\"Par17\" class=\"p p-last\">In Ascomycota, species in Tuberaceae followed the general symbiotroph trend, except <em>C. venosus</em>, which has a large complement of PCWDEs (Figs.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4a</span></a> and <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span style=\"position: relative;text-decoration:none;\">&#x200B;<span class=\"figpopup-sensitive-area\" style=\"left: -1.5em;\">and5,</span></span><span>5</span></a>, see ref. <sup><sup><a href=\"#CR16\" rid=\"CR16\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273282\">16</a></sup></sup>). Other ectomycorrhizal Pezizales, such as the newly sequenced <em>Wilcoxina mikolae</em> and <em>Trichophaea hybrida</em> (Pyronemataceae), also have few PCWDEs. The desert truffles <em>Kalaharituber pfeilii</em> and <em>Terfezia boudieri</em> (Pezizaceae) both have ectomycorrhiza-like restricted suites of PCWDEs, but they differ in the numbers of genes encoding cellulose-acting LPMOs (ten and one copies, respectively) and they encode one copy of GH32 invertase. Like other ectomycorrhizal species<em>, A. macrosclerotiorum</em> contained no class II PODs, but had a large set of PCWDEs (282 genes, including 17 cellulose-acting LPMOs, and genes encoding GH6, GH7, GH32, GH45, PL1, and PL3; Figs.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4</span></a> and <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span style=\"position: relative;text-decoration:none;\">&#x200B;<span class=\"figpopup-sensitive-area\" style=\"left: -1.5em;\">and5,</span></span><span>5</span></a>, Supplementary Figs.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3</a></sup>&#x02013;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">5</a></sup> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM12\" rid=\"MOESM12\" class=\" media\">8</a></sup>). Closely related taxa to <em>A. macrosclerotiorum</em> mainly contain DSEs, such as <em>P. scopiformis</em>, saprotrophs, and pathogens, which are characterized by a larger set of PCWDEs compared to ectomycorrhizal fungi. To assess whether this large repertoire of PCWDEs is expressed during symbiosis development, we carried out transcript profiling of <em>Pinus sylvestris</em>&#x02013;<em>A. macrosclerotiorum</em> ectomycorrhizas using RNA-Seq. The fungal symbiont develops a dense Hartig net within the root cortex, but no mantle sheath (Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">6a</a></sup>). Transcript profiling of ectomycorrhizas showed that most of the PCWDEs are not induced in symbiotic tissues (Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">6b</a></sup> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM20\" rid=\"MOESM20\" class=\" media\">16</a></sup>), suggesting a tight transcriptional control of the PCWDE gene expression.</p></span><span id=\"s\n9\" class=\"sec\"><h3 id=\"s\n9title\">MCWDEs are retained in ectomycorrhizal fungi</h3><p id=\"Par18\" class=\"p p-first-last\">We explored the genetic capabilities of the sequenced fungi to decompose microbial (i.e., bacterial and fungal) cell walls by comparing PCWDE to MCWDE repertoires (Figs.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4</span></a> and <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span style=\"position: relative;text-decoration:none;\">&#x200B;<span class=\"figpopup-sensitive-area\" style=\"left: -1.5em;\">and5,</span></span><span>5</span></a>, Supplementary Figs.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3</a></sup>&#x02013;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">5</a></sup> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM11\" rid=\"MOESM11\" class=\" media\">7</a></sup>). The proportion of MCWDE (acting on chitin, glucans, mannans, and peptidoglycans) in ectomycorrhizal fungi is similar to that in saprotrophs. Thus, ectomycorrhizal fungi have generally retained MCWDE genes (e.g., chitinases, &#x000df;-1,3-glucanases) although they have lost most PCWDEs (e.g., endo- and exocellulases). Ectomycorrhizal fungi may use secreted MCWDE to scavenge nitrogen compounds trapped in SOM (e.g., chitin) by selectively using these hydrolytic enzymes in addition to oxidative mechanisms<sup><sup><a href=\"#CR32\" rid=\"CR32\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273263\">32</a></sup></sup>. Some of these chitin-, glucan-, and mannan-active enzymes are likely involved in fungal cell wall remodeling during complex developmental processes, such as ectomycorrhiza and sporocarp formation<sup><sup><a href=\"#CR10\" rid=\"CR10\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273328\">10</a></sup>,<sup><a href=\"#CR11\" rid=\"CR11\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273258\">11</a></sup>,<sup><a href=\"#CR33\" rid=\"CR33\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273235\">33</a></sup></sup>.</p></span><span id=\"s\n10\" class=\"sec\"><h3 id=\"s\n10title\">Mycorrhizal development is driven by gene co-option</h3><p id=\"Par19\" class=\"p p-first\">We used a phylostratigraphic approach to characterize the evolutionary origins of ectomycorrhizal lineages on the basis of gene functions in extant organisms. We examined the age distribution of genes induced at different stages of ectomycorrhiza establishment, so-called symbiosis-induced genes, by defining phylogenetic ages (phylostrata), that correspond to internal nodes of the tree along the lineage leading from the root to the symbiotic species for which transcriptomic data are available (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig6/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\" co-legend-rid=\"lgnd_Fig6\"><span>6</span></a>, Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">7</a></sup> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM13\" rid=\"MOESM13\" class=\" media\">9</a></sup>). In Ascomycota and Basidiomycota, an average of 74% and 67% of the ectomycorrhiza-induced genes predated the evolution of ectomycorrhizal symbiosis, respectively (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig6/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\" co-legend-rid=\"lgnd_Fig6\"><span>6</span></a>). Approximately 6% and 18% of these genes were already present in the MRCAs of the Ascomycota and that of the Basidiomycota (e.g., for <em>L. bicolor</em>, hydrophobins, GH131, GH28, and CBM1_GH5), respectively (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig6/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\" co-legend-rid=\"lgnd_Fig6\"><span>6</span></a> and Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">7</a></sup>). No specific phylostrata was enriched in symbiosis-induced genes (Fisher&#x02019;s exact test, <em>P</em>&#x02009;&#x0003c;&#x02009;0.005; Supplementary Data&#x000a0;<sup><a href=\"#MOESM13\" rid=\"MOESM13\" class=\" media\">9b</a></sup>), suggesting that there was no distinguished period during ectomycorrhiza evolution caracterized by an excess number of gene birth events. These observations imply that symbiosis-induced genes have mostly been co-opted for ectomycorrhiza development during evolution from saprotrophic ancestors. However, the phylostratigraphic analysis also suggested that an average of 9&#x02013;22 % (in Ascomycota; Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig6/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\" co-legend-rid=\"lgnd_Fig6\"><span>6a</span></a>) and 19&#x02013;20 % (in Basidiomycota; Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig6/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\" co-legend-rid=\"lgnd_Fig6\"><span>6b</span></a>) of ectomycorrhiza-induced genes are restricted to specific mycorrhizal lineages. Given our current taxon sampling, it is difficult to distinguish genes that coincidentally evolved with mycorrhiza formation from species-specific orphan genes, except in the case of three clades that comprise more than a single ectomycorrhizal species, Boletales, the genus <em>Laccaria</em> and Tuberaceae, which comprise more than a single ectomycorrhizal species. In these clades, the proportion of symbiosis-induced genes that map to the origin of ectomycorrhiza showed a large variation, from 0 to 0.8% in the Boletales (two species), to 10% in <em>Laccaria</em>, and 14% in the Tuberaceae (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig6/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\" co-legend-rid=\"lgnd_Fig6\"><span>6</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig6\" co-legend-rid=\"lgnd_Fig6\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig6/\" target=\"figure\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139716148186176\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7550596_41467_2020_18795_Fig6_HTML.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2020_18795_Fig6_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7550596/bin/41467_2020_18795_Fig6_HTML.jpg\" /></a></span><span id=\"largeobj_idm139716148186176\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig6/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_Fig6\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig6/\" target=\"figure\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\">Fig. 6</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Phylostratigraphy for ectomycorrhiza-specific upregulated genes.</strong><p id=\"__p6\">Pie charts on the time-calibrated trees represent the number of gene clusters containing ectomycorrhiza-specific upregulated genes from the species compared. Numbers and percentage of genes mapping to phylostrata are shown on the right of the trees. Ectomycorrhizal species used for the comparison are in green boxes. Upregulated genes were selected according to FDR adjusted <em>P</em> value &#x0003c; 0.05. <strong>a</strong> Ascomycota, the fold change (FC) used for defining ectomycorrhiza-specific upregulated genes was FC&#x02009;&#x02265;&#x02009;5. <strong>b</strong> Basidiomycota, the FC used for defining ectomycorrhiza-specific upregulated genes was FC&#x02009;&#x02265;&#x02009;5. See Supplementary Data&#x000a0;<sup><a href=\"#MOESM13\" rid=\"MOESM13\" class=\" media\">9</a></sup> for the identified phylostrata containing ectomycorrhiza-upregulated genes and enrichment statistics. Source data are provided as a Source data Fig.&#x000a0;<sup><a href=\"#MOESM21\" rid=\"MOESM21\" class=\" media\">4</a></sup>.</p></span></span></span><p id=\"Par20\">To test our phylostratigraphic results, we assessed the evolutionary conservation of 5917 ectomycorrhiza-induced transcripts identified in ten different ectomycorrhizal interactions among the 135 studied fungal genomes (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig7/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span>7a</span></a>). We found that 15.2 % of the 5917 symbiosis-induced genes are shared by all species in our dataset (clusters IV and V). Most encode proteins involved in core metabolic or signaling functions. Genes from cluster III are conserved within Basidiomycota only, while genes from cluster VII are shared by Ascomycota only. Most of them have no known function (i.e., no&#x000a0;KOG domain). Altogether, 31.2% of these symbiosis-induced genes are species specific (cluster VI). Most of these genes encode proteins with unknown KOG functions or mycorrhiza-induced small secreted proteins (MiSSPs). The proportion of species-specific, ectomycorrhiza-induced genes is therefore substantial, as reported earlier<sup><sup><a href=\"#CR10\" rid=\"CR10\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273225\">10</a></sup>,<sup><a href=\"#CR12\" rid=\"CR12\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273256\">12</a></sup>,<sup><a href=\"#CR16\" rid=\"CR16\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273301\">16</a></sup></sup> and suggested by our phylostratigraphic analysis.</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig7\" co-legend-rid=\"lgnd_Fig7\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig7/\" target=\"figure\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139716145198432\" class=\"figure\"><img loading=\"lazy\" class=\"fig-image\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2020_18795_Fig7_HTML.jpg\" title=\"An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2020_18795_Fig7_HTML.jpg\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7550596/bin/41467_2020_18795_Fig7_HTML.jpg\" /></span><span id=\"largeobj_idm139716145198432\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig7/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_Fig7\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig7/\" target=\"figure\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\">Fig. 7</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Phylogenetic conservation of ectomycorrhiza-induced genes.</strong><p id=\"__p7\">We analyzed genes coding for ectomycorrhiza-induced proteins,&#x000a0;secreted proteins and ectomycorrhiza-induced secreted proteins among 135 fungi. Heat maps show BLASTP sequence similarity of 5917 ectomycorrhiza-induced proteins (<strong>a</strong>), 6669 genes coding for secreted proteins (<strong>b</strong>), and 1028 ectomycorrhiza-induced&#x000a0; secreted proteins (<strong>c</strong>) from ten ectomycorrhizal fungi, with available symbiotic transcriptomes among 135 genomes of Basidiomycota and Ascomycota. The heat maps depict a double-hierarchical clustering of protein sequences encoded by symbiosis-upregulated genes, genes coding for secreted proteins and genes coding for symbiosis-upregulated secreted proteins (rows, fold change&#x02009;&#x02265;&#x02009;5 in symbiotic tissues compared to free-living mycelium, false discovery rate-corrected <em>P</em>&#x02009;&#x02264;&#x02009;0.05); right panel, functional categories (KOG) are given for each cluster of sequences in % as bargrams. Clusters I&#x02013;VIII correspond to group of sequences sharing the same level of protein sequence similarity based on BlastP. The distribution of transcripts belonging to Ascomycota and Basidiomycota in each cluster is shown as pie charts. Data were visualized and clustered using R (package HeatPlus<sup><sup><a href=\"#CR97\" rid=\"CR97\" class=\" bibr popnode\">97</a></sup></sup>). The hierarchical clustering was done by using a Euclidian distance and Ward clustering method. Color scale on the left (white to red) shows the % of sequence identity according to BLASTP. The symbiosis-induced genes were retrieved from the ectomycorrhizal transcriptomes of ten species: <em>A. macrosclerotiorum</em> EW76-UTF0540, <em>A. muscaria</em> Koide, <em>C. geophilum</em> 1.58, <em>H. cylindrosporum</em> h7, <em>L. bicolor</em> S238N, <em>P. involutus</em> ATCC 200175, <em>P. croceum</em> F1598, <em>P. microcarpus</em> 441, <em>T. matsutake</em> 945, and <em>T. magnatum</em> (see Supplementary Data&#x000a0;<sup><a href=\"#MOESM14\" rid=\"MOESM14\" class=\" media\">10</a></sup>, <sup><a href=\"#MOESM17\" rid=\"MOESM17\" class=\" media\">13</a></sup> and <sup><a href=\"#MOESM18\" rid=\"MOESM18\" class=\" media\">14</a></sup>). GEO accession codes are provided in the &#x0201c;Methods/Phylostratigraphy&#x0201d; section. Source data are provided as a Source data Fig.&#x000a0;<sup><a href=\"#MOESM21\" rid=\"MOESM21\" class=\" media\">5</a></sup>.</p></span></span></span><p id=\"Par21\" class=\"p p-last\">Next, we examined whether the same or different gene families were co-opted in independent ectomycorrhizal lineages, i.e., whether co-option was convergent or divergent during fungal evolution. We quantified convergence by the extent of overlap among conserved ectomycorrhiza-induced gene families within independent lineages. Conserved ectomycorrhiza-induced genes showed little or no overlap among the analyzed species (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig6/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\" co-legend-rid=\"lgnd_Fig6\"><span>6</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM13\" rid=\"MOESM13\" class=\" media\">9</a></sup>). For example, there were only eight clusters of ectomycorrhiza-induced genes common to at least four Basidiomycota symbionts (Supplementary Data&#x000a0;<sup><a href=\"#MOESM13\" rid=\"MOESM13\" class=\" media\">9</a></sup>). Similarly, only 12 gene clusters were shared by at least three Ascomycota symbionts (Supplementary Data&#x000a0;<sup><a href=\"#MOESM13\" rid=\"MOESM13\" class=\" media\">9</a></sup>). This low overlap between co-opted genes suggests that independently evolved ectomycorrhizal lineages recruited different ancestral gene families for symbiosis, in addition to a likewise unique set of novel genes, that evolved after the origins of symbiosis<sup><sup><a href=\"#CR12\" rid=\"CR12\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273219\">12</a></sup></sup>.</p></span><span id=\"s\n11\" class=\"sec sec-last\"><h3 id=\"s\n11title\">Symbiosis-induced secreted proteins in symbionts</h3><p id=\"Par22\" class=\"p p-first-last\">The expression of genes encoding secreted and symbiosis-induced secreted proteins from ectomycorrhizal fungi was measured by RNA-Seq profiling in ten ectomycorrhizal associations (Supplementary Methods). By using a BLASTP-based analysis, we assessed the evolutionary conservation of the expressed 6669 secreted proteins (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig7/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span>7b</span></a>) and 1028 symbiosis-induced secreted proteins (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig7/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span>7c</span></a>) among 135 fungal species (Supplementary Datas&#x000a0;12,&#x000a0;<sup><a href=\"#MOESM17\" rid=\"MOESM17\" class=\" media\">13</a></sup> and <sup><a href=\"#MOESM18\" rid=\"MOESM18\" class=\" media\">14</a></sup>). A substantial proportion of the secreted proteins are species-specific SSPs (cluster III, Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/figure/Fig7/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span>7b</span></a>). In addition, we found that 38.1% of symbiosis-induced secreted proteins are shared by all species of fungi (clusters I and III). Most code for core metabolic or signaling functions and CAZymes. Genes from cluster II (9.3%) are conserved within Basidiomycota only. Genes from cluster IV (21.4%) are conserved mainly in Ascomycota, showed a lower similarity in Basidiomycota and are poorly conserved in Glomeromycota. They encode MiSSPs, CAZymes, proteins with unknown KOG function and proteins of signaling and metabolic pathways. Altogether, 31.2% of these 1028 symbiosis-induced genes are mostly species specific (clusters V&#x02013;VII). Most of these genes encode MiSSPs, proteins with unknown KOG functions and proteins of signaling pathways. The phylostratigraphic analysis of secreted proteins and MiSSPs corroborated these results (Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">7</a></sup>).</p></span></span><span id=\"s\n12\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\n12title\">Discussion</h2><p id=\"Par23\" class=\"p p-first\">After the origin of Pinaceae (ca. 200&#x02009;Mya), ectomycorrhizal symbiosis arose repeatedly, perhaps 80 times or more, across multiple lineages of Mucoromycotina (Endogonales), Ascomycota, and especially Basidiomycota<sup><sup><a href=\"#CR2\" rid=\"CR2\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273240\">2</a></sup>&#x02013;<sup><a href=\"#CR4\" rid=\"CR4\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273295\">4</a></sup>,<sup><a href=\"#CR8\" rid=\"CR8\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273291\">8</a></sup>,<sup><a href=\"#CR34\" rid=\"CR34\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273323\">34</a></sup>,<sup><a href=\"#CR35\" rid=\"CR35\" class=\" bibr popnode\">35</a></sup></sup>. The ancestors of ectomycorrhizal fungi are genetically and ecologically diverse, making this a superb example of convergent evolution<sup><sup><a href=\"#CR3\" rid=\"CR3\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273234\">3</a></sup>,<sup><a href=\"#CR7\" rid=\"CR7\" class=\" bibr popnode\">7</a></sup>,<sup><a href=\"#CR9\" rid=\"CR9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273250\">9</a></sup>,<sup><a href=\"#CR12\" rid=\"CR12\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273267\">12</a></sup></sup>. To assess the general shared properties of the lifestyle evolution and functional biology of ectomycorrhizal symbioses, we conducted a comparative analysis of 62 mycorrhizal and 73 non-mycorrhizal fungal species. Our dataset, the most comprehensive so far, includes several major fungal clades that have not been sampled previously for ectomycorrhizal genomes, i.e., Cantharellales, Phallomycetidae, Thelephorales, and Russulales. For the sake of comparison, it also includes genomes of arbuscular mycorrhizal fungi, orchid mycorrhizal fungi, and ericoid mycorrhizal fungi providing a unique opportunity to highlight differences and similarities between the major types of mycorrhizal symbioses.</p><p id=\"Par24\">Our analyses of early diverging clades of ectomycorrhizal fungi (e.g., Cantharellales) support the general view that transitions from saprotrophy to ectomycorrhizal symbiosis involve (1) widespread losses of PCWDEs acting on lignin, cellulose, hemicellulose, pectins, suberins, and tannins, (2) co-option of metabolic and signaling genes present in saprotrophic ancestors to fulfill new symbiotic functions, (3) diversification of novel, lineage-specific symbiosis-induced orphan genes, and (4) massive proliferation of TEs. In addition, they corroborated and extended our previous analyses of the genomes of arbuscular mycorrhizal fungi<sup><sup><a href=\"#CR27\" rid=\"CR27\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273269\">27</a></sup></sup> and ericoid mycorrhizal fungi<sup><sup><a href=\"#CR15\" rid=\"CR15\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273257\">15</a></sup></sup>. Despite the general trend toward losses of PCWDEs in ectomycorrhizal lineages in both Ascomycota and Basidiomycota, there is considerable diversity in the apparent decay capacities of ectomycorrhizal fungi. For example, in Agaricales (Agaricomycetidae), <em>C. glaucopus</em> possesses 12 recently duplicated copies of atypical class II POD genes, which may confer some ability to obtain nutrients from soil phenolic compounds, as previously suggested<sup><sup><a href=\"#CR36\" rid=\"CR36\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273273\">36</a></sup></sup>. However, secreted PODs can play a variety of additional roles, such as biosynthesis of cell wall melanins, detoxifying the immediate hyphae environment, or/and converting plant polymers into more oxidized, recalcitrant components of SOM. Similarly, the genome of <em>T. matsutake</em> (also Agaricales) encodes two GH7 cellobiohydrolases, in agreement with its known facultative saprotrophic ability<sup><sup><a href=\"#CR37\" rid=\"CR37\" class=\" bibr popnode\">37</a></sup></sup>. Ectomycorrhizal fungi that evolved from brown rot fungi in the Boletales (also Agaricomycetidae), such as <em>Paxillus involutus</em>, appear to have adapted the oxidative decomposition system from their saprotrophic ancestors to liberate N entrapped in decaying SOM<sup><sup><a href=\"#CR38\" rid=\"CR38\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273268\">38</a></sup>&#x02013;<sup><a href=\"#CR41\" rid=\"CR41\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273254\">41</a></sup></sup>. Their secreted proteases, LPMOs, and laccases may act in concert to decay available SOM compounds. Alternative hypotheses for the role of the remaining PCWDEs in ectomycorrhizal fungi include the remodeling of root cell walls during host colonization<sup><sup><a href=\"#CR42\" rid=\"CR42\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273293\">42</a></sup></sup>.</p><p id=\"Par25\">Among the newly sampled fungal groups, ectomycorrhizal Phallomycetidae and Cantharellales present highly variable suites of PCWDEs. Most striking is the ectomycorrhizal <em>G. morchelliformis</em> (Gomphales, Phallomycetidae), which has 25 class II manganese PODs (MnP) that may be involved in the decay of SOM and/or detoxification of soil polyphenolic compounds, as well as numerous enzymes active on cellulose and hemicellulose. The class II PODs of <em>G. morchelliformis</em> are members of a gene family that underwent expansion after the divergence of Auriculariales and other Agaricomycetes. In contrast, Cantharellales evolved prior to the diversification of class II LiP<sup><sup><a href=\"#CR30\" rid=\"CR30\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273253\">30</a></sup></sup>, and none of its species&#x02014;saprotrophic or mycorrhizal&#x02014;possesses these ligninolytic enzymes. In contrast to their saprotrophic cousin <em>Sistotrema</em> sp., <em>C. anzutake</em> and <em>H. rufescens</em> are ectomycorrhizal species of Cantharellales that lack not only class II LiP, but also many glycoside hydrolases acting on cellulose, hemicellulose and pectins, LPMOs, and CBM1 motifs that are needed for plant cell wall and SOM decomposition. In this regard, they resemble some of the most derived ectomycorrhizal Agaricomycetidae with highly reduced saprotrophic capabilities, such as species of Boletales. The ectomycorrhizal Russulales and Thelephorales that we sampled also conform to this model of a greatly reduced saprotrophic apparatus in prominent symbiotic fungi playing major roles in forest ecosystems.</p><p id=\"Par26\">Ectomycorrhizal fungi are not all depauperate in PCWDEs. Species such as <em>A. macrosclerotium</em> in the Leotiomycetes, may represent transitional steps from pure saprotrophy toward pure ectomycorrhizal symbiosis. Such taxa could be capable of facultative saprotrophy, i.e., SOM decomposition. As suggested for ericoid mycorrhizal fungi, which exhibit a large PCWDE repertoire<sup><sup><a href=\"#CR15\" rid=\"CR15\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273272\">15</a></sup></sup>, a dual saprotrophic/mutualistic habit may provide greater ecological flexibility and fitness under specific environmental conditions. Moreover, in such lineages, evolutionary reversal to saprotrophy might be possible (although that has not been demonstrated). We showed that the PCWDEs of <em>A. macrosclerotium</em> are downregulated at the transcriptional level during the symbiotic interaction to avoid triggering plant defense reactions and/or digesting host root tissues. This suggests that loss of genes encoding PCWDEs is a consequence of, but not a requirement for, the evolution of ectomycorrhizal mutualisms. What remains to be determined is why ectomycorrhizal fungi do not retain their ancestral dual lifestyle, and how the ectomycorrhizal associations become so intimate that saprotrophic capabilities are irretrievably lost. The lack of invertase and sucrose transporter genes in most ectomycorrhizal fungi means that they are unable to use apoplastic sucrose released by their host plant. They fully rely on their partner for their glucose supply, a mechanism reinforcing their dependence on the plant partner.</p><p id=\"Par27\">Our results, and those of prior studies<sup><sup><a href=\"#CR3\" rid=\"CR3\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273319\">3</a></sup>,<sup><a href=\"#CR14\" rid=\"CR14\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273284\">14</a></sup></sup>, suggest that the predominant mechanism for the transition from saprotrophy to ectomycorrhizal symbiosis appears to involve restricted secretion of hydrolytic enzymes acting on plant cell walls (whether via regulatory shifts or gene loss), which enables the symbiont to be accommodated in roots. An example of an ultimate adaptation to symbiosis is illustrated by the arbuscular mycorrhizal fungi that are obligate symbionts. They display the lowest repertoire of PCWDEs of the current set of sequenced mycorrhizal fungi. Unfortunately, no genomic data exist yet to support the origin of arbuscular mycorrhizal fungi from saprotrophic CAZyme-rich Mucoromycotina. In contrast, ericoid mycorrhizal fungi and orchid mycorrhizal fungi have the higher set of PCWDEs supporting their dual saprotrophic/symbiotic lifestyles<sup><sup><a href=\"#CR3\" rid=\"CR3\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273236\">3</a></sup>,<sup><a href=\"#CR15\" rid=\"CR15\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273320\">15</a></sup></sup>. We thus propose that the PCWDE repertoire is reflective of the age of the symbiosis along the saprotrophy to symbiosis continuum with the arbuscular mycorrhizal symbiosis emerging in the early Devonian (393&#x02013;419&#x02009;Mya), the ectomycorrhizal symbiosis during the Jurassic (ca. 200&#x02009;Mya) and mycorrhizal associations with Ericaceae species to the Cretaceous (ca. 117&#x02009;Mya)<sup><sup><a href=\"#CR4\" rid=\"CR4\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273331\">4</a></sup></sup>.</p><p id=\"Par28\">Further insight into the differences in gene composition between saprotrophic and symbiotic fungi was obtained by focusing on genes involved in adaptations to the host plant habitat in which symbionts thrive. Novel genes, such as effector-like MiSSPs, are likely required for symbiosis development (e.g., dampening of plant defense reactions<sup><sup><a href=\"#CR43\" rid=\"CR43\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273329\">43</a></sup>,<sup><a href=\"#CR44\" rid=\"CR44\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273298\">44</a></sup></sup>) and metabolism<sup><sup><a href=\"#CR45\" rid=\"CR45\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273221\">45</a></sup></sup>, and have evolved in every ectomycorrhizal lineage. Transcript profiling of several ectomycorrhizal interactions has shown that 7&#x02013;38% of genes that are upregulated during symbiosis are species-specific genes, i.e., they are restricted to a single mycorrhizal species. Among all symbiosis-upregulated genes, 8&#x02013;28% encode candidate secreted effector-like MiSSPs. Their functional analysis is hampered by the the lack of high-throughput genetic engineering techniques for ectomycorrhizal fungi, but some of these MiSSPs have been functionally characterized, including LbMiSSP7, LbMiSSP8, LbMiSSP7.6, and PaMiSSP10 (refs. <sup><sup><a href=\"#CR44\" rid=\"CR44\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273290\">44</a></sup>,<sup><a href=\"#CR46\" rid=\"CR46\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273239\">46</a></sup>&#x02013;<sup><a href=\"#CR48\" rid=\"CR48\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273299\">48</a></sup></sup>). The evolution of innovations, such as the control of the host plant immunity, development, and metabolism by these lineage-specific symbiotic effectors was likely a necessary step to colonized the new niches represented by tree roots. Some of the effector-like SSPs, but also structural SSPs like hydrophobins, may have evolved from SSPs used by the saprotrophic ancestors to communicate or compete in the soil and wood environments<sup>,<sup><a href=\"#CR46\" rid=\"CR46\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273313\">46</a></sup>,<sup><a href=\"#CR49\" rid=\"CR49\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273312\">49</a></sup></sup>.</p><p id=\"Par29\">Novel and recently evolved genes, including MiSSPs, are undoubtedly responsible for the particular attributes of individual mycorrhizal lineages. Indeed, arrays of ectomycorrhiza-induced genes are unique to each specific clade with between 14 and 39% of symbiosis-upregulated genes being species-specific genes with no known function<sup><sup><a href=\"#CR12\" rid=\"CR12\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273297\">12</a></sup></sup> (present study). At the same time, our phylostratigraphic analysis showed that a large proportion of genes that are upregulated in ectomycorrhizas arose long before the evolution of the mutualistic associations. In other words, numerous genes used by saprotrophic ancestors were co-opted for the symbiotic lifestyle. Important ecological traits, such as N and P acquisition traits (e.g., organic N- and P-degrading secreted enzymes) were already present in free-living saprotrophic ancestors of ectomycorrhizal symbionts. Despite the well-documented ability of ectomycorrhizal fungi to hydrolyze organic phosphate compounds and scavenge nitrogen through the degradation of litter proteins accumulating in soil litter, we found that ectomycorrhizal genomes generally contain a similar number or less copies of genes coding for secreted N- and P-targeting hydrolases than saprotrophs, pathogens, or ericoid mycorrhizal fungi.</p><p id=\"Par30\">One of the most striking genomic features of mycorrhizal fungi is their high content in TEs. A large proportion of the genomes of mycorrhizal fungi can be ascribed to past transposition, providing a major contribution to their genome landscape. It has been proposed that the lack of sexual recombination (i.e., asexual reproduction) can favor the uncontrolled proliferation of TEs<sup><sup><a href=\"#CR47\" rid=\"CR47\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273336\">47</a></sup></sup>. It may be true for arbuscular mycorrhizal fungi with no known sexual reproduction, but this hypothesis cannot stand for the ectomycorrhizal fungi investigated in the present study as they are known to produce ascocarps or basidiocarps. In biotrophic pathogens, such as <em>Blumeria graminis</em> f. sp. <em>hordei</em> and <em>B. graminis</em> f. sp. <em>tritici</em>, it has been proposed that burst(s) of TEs offer a template for rapid evolution of virulence genes by duplications, small-scale rearrangements, or deletions<sup><sup><a href=\"#CR48\" rid=\"CR48\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273248\">48</a></sup></sup>. It is tempting to speculate that the observed TE proliferation(s) play a key role in promoting the rapid evolution of symbiosis-related factors, such as MiSSPs.</p><p id=\"Par31\">Collectively, our results suggest that each independently evolved ectomycorrhizal lineage uses species-specific novel genes, as well as members of ancestral gene families to develop symbiotic interactions, indicating that divergent genetic innovations underlie the convergent origins of the ectomycorrhizal guild. Thus, the functional heterogeneity in symbiosis-related gene networks in ectomycorrhizal fungi is not only found among species-specific genes as evidenced for MiSSPs<sup><sup><a href=\"#CR3\" rid=\"CR3\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273274\">3</a></sup>,<sup><a href=\"#CR14\" rid=\"CR14\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273296\">14</a></sup>,<sup><a href=\"#CR43\" rid=\"CR43\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273262\">43</a></sup>&#x02013;<sup><a href=\"#CR45\" rid=\"CR45\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273242\">45</a></sup>,<sup><a href=\"#CR50\" rid=\"CR50\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273289\">50</a></sup>,<sup><a href=\"#CR51\" rid=\"CR51\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273303\">51</a></sup></sup>, but also in conserved genes (e.g., hydrophobins) that predate the gains of mycorrhizal symbiosis.</p><p id=\"Par32\" class=\"p p-last\">The large number of independently evolved ectomycorrhizal lineages raises questions as to why the emergence of this guild is so common in nature. Furthermore, the absence of known reversals to the saprotrophic lifestyle suggests that genetic traits underlying the transitions to ectomycorrhizal lifestyles represent fixed evolutionary transitions. Future studies should focus on conditions that may predispose some groups to evolve ectomycorrhizal symbioses, or other mechanisms that increase the likelihood of convergent evolution<sup><sup><a href=\"#CR52\" rid=\"CR52\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273325\">52</a></sup>&#x02013;<sup><a href=\"#CR55\" rid=\"CR55\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273224\">55</a></sup></sup>. The present study roughly doubles the number of published ectomycorrhizal genomes. Even greater sampling of key lineages, particularly early divergent&#x000a0; clades, is needed to elucidate the macro- and microevolutionary mechanisms that were responsible for the diversification of ectomycorrhizal fungi.</p></span><span id=\"s\n13\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\n13title\">Methods</h2><span id=\"s\n14\" class=\"sec sec-first\"><h3 id=\"s\n14title\">Strains and fungal material used for genome sequencing</h3><p id=\"Par33\" class=\"p p-first-last\">Fungal strains used for genome sequencing are described in Supplementary Data&#x000a0;<sup><a href=\"#MOESM5\" rid=\"MOESM5\" class=\" media\">1</a></sup>. Assignment of individual taxa to functional guilds was based on common categorizations in the literature<sup><sup><a href=\"#CR34\" rid=\"CR34\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273317\">34</a></sup>,<sup><a href=\"#CR56\" rid=\"CR56\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273251\">56</a></sup>,<sup><a href=\"#CR57\" rid=\"CR57\" class=\" bibr popnode\">57</a></sup></sup>. DNA was extracted with a modified cetyltrimethylammonium bromide protocol<sup><sup><a href=\"#CR12\" rid=\"CR12\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273308\">12</a></sup></sup>.</p></span><span id=\"s\n15\" class=\"sec\"><h3 id=\"s\n15title\">Genome sequencing, assembly, and annotation</h3><p id=\"Par34\" class=\"p p-first-last\">All sequencing, assembly, and annotation was performed at JGI. Genome sequencing was done with either Illumina or Pacific Biosciences (PacBio) technology, or both (&#x0201c;hybrid&#x0201d;). Genomes were assembled from Illumina reads using ALLPATHS-LG<sup><sup><a href=\"#CR58\" rid=\"CR58\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273275\">58</a></sup></sup>, followed by patching using PacBio with PBJelly<sup><sup><a href=\"#CR59\" rid=\"CR59\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273241\">59</a></sup></sup> in the case of hybrid assemblies. All other genomes were assembled from PacBio reads using Falcon<sup><sup><a href=\"#CR60\" rid=\"CR60\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273218\">60</a></sup></sup> or Celera Assembler<sup><sup><a href=\"#CR61\" rid=\"CR61\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273311\">61</a></sup></sup>. Mitochondrial genomes were assembled separately. All transcriptome sequencing was done with Illumina only, and subsequently assembled into putative transcripts using either Rnnotator<sup><sup><a href=\"#CR11\" rid=\"CR11\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273302\">11</a></sup></sup> or Trinity<sup><sup><a href=\"#CR62\" rid=\"CR62\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273327\">62</a></sup></sup>. Each genome was annotated using the JGI Annotation Pipeline<sup><sup><a href=\"#CR63\" rid=\"CR63\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273247\">63</a></sup>,<sup><a href=\"#CR64\" rid=\"CR64\" class=\" bibr popnode\">64</a></sup></sup>, aided by the transcriptome when available. Detailed methods are described in the Supplementary Methods.</p></span><span id=\"s\n16\" class=\"sec\"><h3 id=\"s\n16title\">Organismal phylogeny</h3><p id=\"Par35\" class=\"p p-first\">We assembled two datasets, one of Ascomycota that included 51 genomes and one of Basidiomycota that included 62 genomes. We opted for analyzing Ascomycota and Basidiomycota genomes separately, and only fungal genomes to improve computational efficiency and because the origins of the ectomycorrhizal symbioses are shallow evolutionary events, so we reasoned that additional resolution cannot be added by including non-fungal outgroup taxa. The computational step of similarity-based clustering, multiple sequence alignments, and tree inference are particularly sensitive to the phylogenetic breadth that the species cover. Analyzing Ascomycota and Basidiomycota separately improved the accuracy of our predictions, by not having to analyze as deep nodes of the tree as the Dikarya (MRCA of Ascomycota and Basidiomycota). We performed all-vs-all Blast with mpiBlast 1.6.0 (ref. <sup><sup><a href=\"#CR65\" rid=\"CR65\" class=\" bibr popnode\">65</a></sup></sup>) and clustered proteins into gene families by Hifix 1.0.5 (ref. <sup><sup><a href=\"#CR66\" rid=\"CR66\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273330\">66</a></sup></sup>), using default parameters. Clusters having 0.5<em>n</em>&#x02013;2<em>n</em> proteins (<em>n</em>&#x02009;=&#x02009;number of species) were aligned by PRANK140603 (ref. <sup><sup><a href=\"#CR67\" rid=\"CR67\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273288\">67</a></sup></sup>) with default settings, and trimmed with trim-Al 1.4.rev15 (ref. <sup><sup><a href=\"#CR68\" rid=\"CR68\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273335\">68</a></sup></sup>) to remove ambiguously aligned regions (with the argument -gt 0.1). Maximum likelihood (ML) gene trees were inferred using FastTree 2.1.10 (refs. <sup><sup><a href=\"#CR69\" rid=\"CR69\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273287\">69</a></sup>,<sup><a href=\"#CR70\" rid=\"CR70\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273277\">70</a></sup></sup>) under the Whelan and Goldman (WAG) model of protein evolution with gamma-distributed rate heterogeneity. We excluded gene trees with deep paralogs to identify single-copy gene. Gene trees of not strictly single-copy gene families were checked if the duplications are deep in the tree, or species specific. In those cases where the duplications were species specific, the gene was marked as suitable for phylogenetic inference and the protein that was closest to the root (in terms of patristic distance) for each species was chosen for inclusion in the phylogenetic analyses.</p><p id=\"Par36\">Single-copy genes were realigned using PRANK v. 150803 (ref. <sup><sup><a href=\"#CR67\" rid=\"CR67\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273229\">67</a></sup></sup>) with the default settings and one round of alignment improvement. Poorly aligned regions were removed using trim-Al v1.2 (ref. <sup><sup><a href=\"#CR68\" rid=\"CR68\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273233\">68</a></sup></sup>) with the -strict setting. The alignments were concatenated into a supermatrix using a custom Perl script, excluding single alignments that were &#x0003c;50 amino acids long. We included genes in the supermatrix only if they were present in &#x0003e;40 and &#x0003e;50 Ascomycota and Basidiomycota species, respectively. ML phylogenies were inferred using RAxML 8.2.4 (ref. <sup><sup><a href=\"#CR71\" rid=\"CR71\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273246\">71</a></sup></sup>) with 100 bootstrap replicates and a partitioned model, where each gene was treated as a separate partition. The PROTGAMMAWAG model was used for each partition. The resulting phylogenies were used in the molecular dating analyses.</p><p id=\"Par37\" class=\"p p-last\">In addition, we constructed an organismal phylogeny restricted to the 62 mycorrhizal fungi. Orthologous genes among the fungi were identified using FastOrtho with the parameters set to 50% identity, 50% coverage, and inflation 3.0 (ref. <sup><sup><a href=\"#CR72\" rid=\"CR72\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273333\">72</a></sup></sup>). Protein sequences were downloaded from MycoCosm (<a href=\"http://mycocosm.jgi.doe.gov\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">mycocosm.jgi.doe.gov</a>). Clusters with single-copy genes were identified and aligned with MAFFT 7.221 (ref. <sup><sup><a href=\"#CR73\" rid=\"CR73\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273307\">73</a></sup></sup>), ambiguous regions (containing gaps and poorly aligned) were eliminated, and single-gene alignments were concatenated with Gblocks 0.91b (ref. <sup><sup><a href=\"#CR74\" rid=\"CR74\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273252\">74</a></sup></sup>). A phylogenetic tree was constructed with RAxML 7.7.2 (ref. <sup><sup><a href=\"#CR75\" rid=\"CR75\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273332\">75</a></sup></sup>), the standard algorithm, the PROTGAMMAWAG model of sequence evolution, and 1000 bootstrap replicates.</p></span><span id=\"s\n17\" class=\"sec\"><h3 id=\"s\n17title\">Molecular dating</h3><p id=\"Par38\" class=\"p p-first-last\">In order to time calibrate the phylogenies, we used the penalized likelihood algorithm as implemented in r8s (ref. <sup><sup><a href=\"#CR76\" rid=\"CR76\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273245\">76</a></sup></sup>) with the POWELL optimization. To identify the appropriate smoothing parameters, we performed a cross-validation analysis for each dataset. For the Basidiomycota phylogeny, the fossil <em>Archaeomarasmius legetti</em> from the mid-Cretaceous (90-94&#x02009;Ma)<sup><sup><a href=\"#CR77\" rid=\"CR77\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273304\">77</a></sup></sup> was used to calibrate the node containing the suborder Marasmiineae, and the fossil of a suilloid ectomycorrhiza from the middle Eocene (50&#x02009;Mya)<sup><sup><a href=\"#CR78\" rid=\"CR78\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273227\">78</a></sup></sup> was used to calibrate the node containing the suborders Boletineae, Paxillineae, Sclerodermatineae, and Suillineae. For the Ascomycota phylogeny, the fossil <em>Paleopyrenomycites devonicus</em> from the early Devonian (~400&#x02009;Ma)<sup><sup><a href=\"#CR79\" rid=\"CR79\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273321\">79</a></sup></sup> was used to calibrate the split between Pezizomycotina and Saccharomycotina. The Basidiomycota supermatrix consisted of 940,059 sites from 1042 genes. All nodes were recovered with a bootstrap support of 96% or higher. The Ascomycota supermatrix consisted of 1,474,883 sites from 1432 genes. All nodes were recovered with bootstrap support of 100%.</p></span><span id=\"s\n18\" class=\"sec\"><h3 id=\"s\n18title\">Analysis of PCWDE evolution</h3><p id=\"Par39\" class=\"p p-first\">A comprehensive list of CAZyme annotations can be found in Supplementary Data 11 and&#x000a0;at the Mycorrhizal Fungi page at the JGI MycoCosm database by following this link: <a href=\"https://mycocosm.jgi.doe.gov/mycocosm/annotations/browser/cazy/summary;-6jBQE?p=Mycorrhizal_fungi\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://mycocosm.jgi.doe.gov/mycocosm/annotations/browser/cazy/summary;-6jBQE?p=Mycorrhizal_fungi</a>.</p><p id=\"Par40\">We investigated the evolution of gene families encoding CAZymes with a particular interest in PCWDEs, which include enzymes active on cellulose (various GH families, AA3_1, LPMO, and CBM1), lignin (AA1_1, AA1_3, AA2_class II POD, AA5_1, DyP, HTP, and OXO), and pectin (CE8, PL, and GH families; see also Supplementary Data&#x000a0;<sup><a href=\"#MOESM12\" rid=\"MOESM12\" class=\" media\">8</a></sup>). We first performed all-vs-all Blast using mpiBlast 1.6.0 (ref. <sup><sup><a href=\"#CR80\" rid=\"CR80\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273326\">80</a></sup></sup>) for 51 Ascomycota and 61 Basidiomycota proteomes, and clustered proteins into gene families using the Hifix 1.0.5 clustering method, with default parameters resulting in 243,784 and 408,904 clusters, respectively. Identification of PCWDE families was performed based on InterPro (IPR) domains in the similarity-based clustering (see above). IPR data from the JGI website were used to annotate proteins with IPR domains for all species. We considered clusters as PCWDE families if at least 50% of the proteins had the appropriate IPR domain. Further classification of PCWDE families was supported by BLASTP 2.6.0+ search<sup><sup><a href=\"#CR80\" rid=\"CR80\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273278\">80</a></sup></sup>, and validation by the CAZy annotation pipeline<sup><sup><a href=\"#CR81\" rid=\"CR81\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273294\">81</a></sup></sup>. Multiple sequence alignments were inferred using PRANK 140,603 with default settings, and trim-Al 1.4.rev15 was used for removing ambiguously aligned regions (with the argument -gt 0.1). ML gene trees of the identified PCWDE clusters were inferred using FastTree 2.1.10 under the WAG model of protein evolution with gamma-distributed rate heterogeneity. TreeFix 1.1.10 (ref. <sup><sup><a href=\"#CR82\" rid=\"CR82\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273324\">82</a></sup></sup>) was used for gene tree/species tree reconciliation (-m PROTGAMMAWAG, &#x02013;alpha 0.001, &#x02013;niter 300) with the genome-based ML species trees. Gene clusters containing less than four proteins were excluded from the analysis. For the reconstruction of gene duplication and loss history of PCWDEs, we used the COMPARE pipeline<sup><sup><a href=\"#CR54\" rid=\"CR54\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273264\">54</a></sup></sup>. Orthogroups were identified, and duplications and losses were inferred for each enzyme group across species trees based on Dollo parsimony. The ancestral gene copy numbers for every internal node were calculated by summing the mapped gains and losses over the species tree.</p><p id=\"Par41\" class=\"p p-last\">Class I and class II PODs were also curated by the RedoxiBase database (<a href=\"http://peroxibase.toulouse.inra.fr\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://peroxibase.toulouse.inra.fr</a>) to distinguish between LiP, versatile PODs, MnP, or atypical Class II PODs. The latter PODs have been annotated as basidiomycete sub-class B (CIIBB) or sub-class C (CIIBC)<sup><sup><a href=\"#CR83\" rid=\"CR83\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273292\">83</a></sup></sup>.</p></span><span id=\"s\n19\" class=\"sec\"><h3 id=\"s\n19title\">Phylostratigraphy</h3><p id=\"Par42\" class=\"p p-first-last\">Phylogenetic ages of ectomycorrhiza-induced genes were assigned based on the most phylogenetically distant orthologous gene in the containing gene family, using a phylostratigraphic approach<sup><sup><a href=\"#CR84\" rid=\"CR84\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273244\">84</a></sup></sup>. Ectomycorrhiza-induced transcripts were retrieved from RNA-seq-based transcriptome profilings: <em>Amanita muscaria</em>, <a id=\"__tag_886273341\" class=\"tag_hotlink\" href=\"/geo/query/acc.cgi?acc=GSE63867\" ref=\"/geo/query/acc.cgi?acc=GSE63867\"><span class=\"tag-json\" style=\"display:none\">{\"type\":\"entrez-geo\",\"attrs\":{\"text\":\"GSE63867\",\"term_id\":\"63867\"}}</span>GSE63867</a>; <em>Cenococcum geophilum</em>, <a id=\"__tag_886273342\" class=\"tag_hotlink\" href=\"/geo/query/acc.cgi?acc=GSE83909\" ref=\"/geo/query/acc.cgi?acc=GSE83909\"><span class=\"tag-json\" style=\"display:none\">{\"type\":\"entrez-geo\",\"attrs\":{\"text\":\"GSE83909\",\"term_id\":\"83909\"}}</span>GSE83909</a>; <em>Hebeloma cylindrosporum</em>, <a id=\"__tag_886273337\" class=\"tag_hotlink\" href=\"/geo/query/acc.cgi?acc=GSE63868\" ref=\"/geo/query/acc.cgi?acc=GSE63868\"><span class=\"tag-json\" style=\"display:none\">{\"type\":\"entrez-geo\",\"attrs\":{\"text\":\"GSE63868\",\"term_id\":\"63868\"}}</span>GSE63868</a>; <em>P. involutus</em>, <a id=\"__tag_886273340\" class=\"tag_hotlink\" href=\"/geo/query/acc.cgi?acc=GSE63924\" ref=\"/geo/query/acc.cgi?acc=GSE63924\"><span class=\"tag-json\" style=\"display:none\">{\"type\":\"entrez-geo\",\"attrs\":{\"text\":\"GSE63924\",\"term_id\":\"63924\"}}</span>GSE63924</a>; <em>Piloderma croceum</em>, <a id=\"__tag_886273339\" class=\"tag_hotlink\" href=\"/geo/query/acc.cgi?acc=GSE63925\" ref=\"/geo/query/acc.cgi?acc=GSE63925\"><span class=\"tag-json\" style=\"display:none\">{\"type\":\"entrez-geo\",\"attrs\":{\"text\":\"GSE63925\",\"term_id\":\"63925\"}}</span>GSE63925</a>; <em>T. magnatum</em>, <a id=\"__tag_886273338\" class=\"tag_hotlink\" href=\"/geo/query/acc.cgi?acc=GSE116692\" ref=\"/geo/query/acc.cgi?acc=GSE116692\"><span class=\"tag-json\" style=\"display:none\">{\"type\":\"entrez-geo\",\"attrs\":{\"text\":\"GSE116692\",\"term_id\":\"116692\"}}</span>GSE116692</a>; <em>A. macrosclerotiorum</em>, SRP130276 and SRP130279-82; <em>L. bicolor</em>, SRP164436-38, SRP164526, SRP164559, and SRP164564; <em>Pisolithus microcarpus</em>, SRP122806, SRP122812, SRP122818, SRP122826, SRP122829, and SRP122850; and <em>T. matsutake</em>, SRP103258). Details of the RNA-seq experiments are described on Gene Expression Omnibus (GEO), NCBI. Note that this pooled set of ectomycorrhiza-induced genes represent a heterogeneous set of symbiosis-induced transcripts from ectomycorrhizal roots from different fungal-tree associations sampled at different stages of development, from early to late stage of symbiosis establishment. We selected those having an expression fold change (FC)&#x02009;&#x0003e;&#x02009;5 and FC&#x02009;&#x0003e;&#x02009;2 (Supplementary Data&#x000a0;<sup><a href=\"#MOESM13\" rid=\"MOESM13\" class=\" media\">9</a></sup>). The phylogenetic trees of the Asco- and Basidiomycota were divided into phylostrata, based on the phylogenetic distance in terms of internal nodes from the species being examined. Each phylostratum corresponded to an ancestral species; genes were assigned to phylostrata based on the species distribution of orthogroups the studied genes belong to. Using this method, the ages of upregulated genes were defined and mapped to species trees. We thoroughly selected a subset of the sequenced genomes of species belonging to either the Ascomycota or the Basidiomycota. Running COMPARE on the present set of 135 Glomeromycotina, Ascomycota, and Basidiomycota genomes separately would have been prohibitively time consuming involving disproportionate costs.</p></span><span id=\"s\n20\" class=\"sec\"><h3 id=\"s\n20title\">Comparative analyses and annotation of functional categories</h3><p id=\"Par43\" class=\"p p-first\">Statistics of JGI genome assemblies (i.e., N50, number of genes and scaffolds, and genome size) were obtained from Mycocosm (<a href=\"http://mycocosm.jgi.doe.gov\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">mycocosm.jgi.doe.gov</a>). Genome completeness with single-copy orthologues was calculated using BUSCO v3.0.2, with odb9 using default parameters<sup><sup><a href=\"#CR85\" rid=\"CR85\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273310\">85</a></sup></sup>. Secretomes were predicted as described in Pellegrin et al.. We compared the different categories of secreted proteins (e.g., CAZymes, lipases, and proteases) according to their lifestyle. Note that the comparisons of genomes are based on analyses of haploid genomes. <em>P</em> values were estimated from nonparametric multiple comparisons based on the generalized Campbell and Skillings&#x02019; procedure with the function <em>gao_cs</em> in R package <em>nparcomp</em><sup><sup><a href=\"#CR86\" rid=\"CR86\" class=\" bibr popnode\">86</a></sup></sup>. Distributions with the mean of the total count and statistically significant ecological groups (lifestyles; <em>P</em>&#x02009;&#x0003c;&#x02009;0.05) were determined. Genomic features (e.g., genome size and TE coverage) were statistically compared between fungal lifestyles based on the GLS procedure with the Brownian motion model for random evolution, using R package <em>nlme</em> with the function <em>gls</em><sup><sup><a href=\"#CR87\" rid=\"CR87\" class=\" bibr popnode\">87</a></sup></sup>. Output files generated were combined and visualized with a series of custom R scripts, Proteomic Information Navigated Genomic Outlook (PRINGO), incorporating R graphic packages <em>ggplot2</em>, <em>ggtree</em>, <em>egg</em>, and <em>ggpubr</em><sup><sup><a href=\"#CR88\" rid=\"CR88\" class=\" bibr popnode\">88</a></sup>&#x02013;<sup><a href=\"#CR91\" rid=\"CR91\" class=\" bibr popnode\">91</a></sup></sup>. The coverage of TEs in genomes was calculated and visualized, using a custom pipeline named Transposon Identification Nominative Genome Overview (TINGO)<sup><sup><a href=\"#CR27\" rid=\"CR27\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273300\">27</a></sup></sup>.</p><p id=\"Par44\">We determined the proportion of variances in genomic features explained by fungal ecological groups, and phylogenetic distances of 61 and 51 species from Basidiomycota and Ascomycota, respectively. Phylogenetic trees were converted into evolutionary distances of species with R package <em>ape</em><sup><sup><a href=\"#CR92\" rid=\"CR92\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273306\">92</a></sup></sup>. Genomic features (i.e., TE coverage, genome size, and predicted secreted proteins) were converted into numerical distances for PERMANOVA according to the workflow using R package <em>vegan</em><sup><sup><a href=\"#CR93\" rid=\"CR93\" class=\" bibr popnode\">93</a></sup>,<sup><a href=\"#CR94\" rid=\"CR94\" class=\" bibr popnode\">94</a></sup></sup>. Significantly different variables were examined (<em>P</em> value &#x0003c;0.05; PERMANOVA; variables&#x02009;~&#x02009;phylogeny&#x02009;+&#x02009;ecology). Significant differences among the fungal ecological groups were tested using pair-wise PERMANOVA with R package RVAideMemoire<sup><sup><a href=\"#CR95\" rid=\"CR95\" class=\" bibr popnode\">95</a></sup></sup>.</p><p id=\"Par45\" class=\"p p-last\">Since the 5&#x02032; and 3&#x02032; LTR of LTR retrotransposons are identical upon insertion, we estimated their time since insertion using the number of substitutions that occur between the two LTRs as described in Castanera et al.<sup><sup><a href=\"#CR96\" rid=\"CR96\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_886273334\">96</a></sup></sup>.&#x000a0;The&#x000a0;list of softwares and R packages is provided in Supplementary Dataset 15.&#x000a0;</p></span><span id=\"s\n21\" class=\"sec sec-last\"><h3 id=\"s\n21title\">Reporting summary</h3><p id=\"Par46\" class=\"p p-first-last\">Further information on research design is available in the&#x000a0;<sup><a href=\"#MOESM3\" rid=\"MOESM3\" class=\" media\">Nature Research Reporting Summary</a></sup> linked to this article.</p></span></span><span id=\"s\n1\" class=\"tsec sec\"><a id=\"supplementary-material-sec\"></a><h2 class=\"head no_bottom_margin\" id=\"s\n1title\">Supplementary information</h2><!--/article/body/sec/--><span id=\"s\n22\" class=\"sec sec-first-last\"><span class=\"sec suppmat\" id=\"MOESM1\"><span class=\"sup-box half_rhythm\" id=\"idm139716192332016\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM1_ESM.pdf\" data-ga-action=\"click_feat_suppl\">Supplementary Information</a><sup>(12M, pdf)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM2\"><span class=\"sup-box half_rhythm\" id=\"idm139716136530480\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM2_ESM.pdf\" data-ga-action=\"click_feat_suppl\">Peer Review</a><sup>(542K, pdf)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM3\"><span class=\"sup-box half_rhythm\" id=\"idm139716135764192\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM3_ESM.pdf\" data-ga-action=\"click_feat_suppl\">Reporting Summary</a><sup>(137K, pdf)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM4\"><span class=\"sup-box half_rhythm\" id=\"idm139716136708640\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM4_ESM.pdf\" data-ga-action=\"click_feat_suppl\">Description of Additional Supplementary Files</a><sup>(12K, pdf)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM5\"><span class=\"sup-box half_rhythm\" id=\"idm139716142456816\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM5_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Dataset 1</a><sup>(19K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM6\"><span class=\"sup-box half_rhythm\" id=\"idm139716138214976\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM6_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Dataset 2</a><sup>(37K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM7\"><span class=\"sup-box half_rhythm\" id=\"idm139716139021664\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM7_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Dataset 3</a><sup>(52K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM8\"><span class=\"sup-box half_rhythm\" id=\"idm139716155482960\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM8_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Dataset 4</a><sup>(125K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM9\"><span class=\"sup-box half_rhythm\" id=\"idm139716147375360\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM9_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Dataset 5</a><sup>(16K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM10\"><span class=\"sup-box half_rhythm\" id=\"idm139716136128176\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM10_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Dataset 6</a><sup>(1007K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM11\"><span class=\"sup-box half_rhythm\" id=\"idm139716145076880\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM11_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Dataset 7</a><sup>(96K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM12\"><span class=\"sup-box half_rhythm\" id=\"idm139716228031696\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM12_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Dataset 8</a><sup>(44K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM13\"><span class=\"sup-box half_rhythm\" id=\"idm139716136251648\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM13_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Dataset 9</a><sup>(5.4M, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM14\"><span class=\"sup-box half_rhythm\" id=\"idm139716152939376\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM14_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Dataset 10</a><sup>(318K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM15\"><span class=\"sup-box half_rhythm\" id=\"idm139716135226272\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM15_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Dataset 11</a><sup>(1.3M, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM16\"><span class=\"sup-box half_rhythm\" id=\"idm139716192375920\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM16_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Dataset 12</a><sup>(625K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM17\"><span class=\"sup-box half_rhythm\" id=\"idm139716152795984\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM17_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Dataset 13</a><sup>(312K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM18\"><span class=\"sup-box half_rhythm\" id=\"idm139716148620048\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM18_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Dataset 14</a><sup>(68K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM19\"><span class=\"sup-box half_rhythm\" id=\"idm139716142235072\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM19_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Dataset 15</a><sup>(13K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM20\"><span class=\"sup-box half_rhythm\" id=\"idm139716228252304\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM20_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Dataset 16</a><sup>(21K, xlsx)</sup></span></span></span></span><span id=\"idm139716135962320\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139716135962320title\">Acknowledgements</h2><span class=\"sec\"><p id=\"__p34\">The project was funded by the U.S. Department of Energy (DOE) Joint Genome Institute, a DOE Office of Science User Facility, and supported by the Office of Science of the U.S. DOE under Contract No. DE-AC02-05CH11231 within the framework of the Mycorrhizal Genomics Initiative (CSP # 305), Metatranscriptomics of Forest Soil Ecosystems project (CSP # 570), and the 1000 Fungal Genome projects (CSP # 662 and 1974). This research was also supported by the Laboratory of Excellence ARBRE (ANR-11-LABX-0002-01), the Region Lorraine, the European Regional Development Fund, and the Plant&#x02013;Microbe Interfaces Scientific Focus Area in the Genomic Science Program, the Office of Biological and Environmental Research in the US DOE Office of Science. L.G.N. is funded by the Momentum Program of the Hungarian Academy of Sciences (Grant No. LP2019/13-2019) and the National Research, Development and Innovation office (Contract No: GINOP-2.3.2-15-2016-00052). G.S. received funding from the European Research Council under the European Union&#x02019;s Horizon 2020 research and innovation programme under grant agreement no. 714774 and the grant GINOP-2.3.2.-15-2016-00057. We also thank the many colleagues whom provided fungal strains and material, including Jan Colpaert, Pierre-Emmanuel Courty, Joanna F. Dames, Patricia Jargeat, Bj&#x000f6;rn Lindahl, Asuncion Morte, Jose-Eduardo Marque-Galvez, Andy Taylor, and Yukari Kuga.</p></span></span><span id=\"App1\" class=\"tsec sec\"><h2 class=\"headless nomenu\"></h2><!--/article/back/app-group/app/--><span id=\"s\n23\" class=\"sec sec-first\"><h3 id=\"s\n23title\">Source data</h3><p id=\"Par49\" class=\"p p-first-last\"><span class=\"sup-box\" id=\"MOESM21\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7550596/bin/41467_2020_18795_MOESM21_ESM.zip\" data-ga-action=\"click_feat_suppl\">Source Data</a><sup>(14M, zip)</sup></span></p></span></span><span id=\"idm139716121937776\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139716121937776title\">Author contributions</h2><p id=\"__p35\">F.M.M. conceived and coordinates the Mycorrhizal Genomics Initiative. F.M.M., L.G.N., D.H., and I.V.G. designed the project. I.V.G. coordinated genome sequencing and annotation at JGI A.Kuo, B.A., K.W.B., A.Carver, A.C.l., C.C., A.L., K.L., M.N., R.A.O., R.D.H., R.R., J.P., and M.Y. performed genome and transcriptome sequencing, assembly and annotation at J.G.I. E.D., V.L., and B.H. performed CAZyme annotations. S.M., F.M., F.M.M., and M.B. analyzed CAZyme distribution. E.K., A.Kohler, L.G.N., and F.M.M. performed analyses of PCWDE evolution, phylostratigraphic studies and analyses of convergence. C.D. annotated peroxidases. S.M. performed comparative genome analyses with the help of E.M. and C.M., and developed scripts for the visual display of data. M.S.G. and D.H. constructed dated trees. M.F.P., M.P., S.P., and L.M.V. performed mycorrhization experiments for RNA-Seq. A.Kohler and E.M. analyzed transcriptome data. Z.K. and G.S. performed clustering analyses using HiFix and Silix.&#x000a0;A.Y., B.S., C.M., D.C., F.M., L.F., G.B., J.B.S., J.M., J.W.S., J.X., L.T., L.Z., M.B., M.G., M.P., M.S., M.Y., N.C., P.B., P.W., P.W.C., R.V., S.P, T.B. and Y.S. provided biological material, DNA and RNA for sequencing or genomes.&#x000a0; F.M.M., and S.M. wrote the manuscript with the help of D.H. and L.G.N.</p></span><span id=\"idm139716138258176\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139716138258176title\">Data availability</h2><p id=\"__p36\">Genome assemblies and annotations for the organisms used in this study are available via the JGI fungal genome portal MycoCosm (<a href=\"http://mycocosm.jgi.doe.gov\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">mycocosm.jgi.doe.gov</a>). In addition, the newly sequenced genome assemblies and annotations have been deposited to GenBank (see Supplementary Data&#x000a0;<sup><a href=\"#MOESM5\" rid=\"MOESM5\" class=\" media\">1</a></sup> for accession codes/BioProjects). The complete transcriptome datasets are available at the GEO at the National Center for Biotechnology Information (<a href=\"http://www.ncbi.nlm.nih.gov/geo/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.ncbi.nlm.nih.gov/geo/</a>). The accession codes for accessing the data deposited at GEO database are provided in the &#x0201c;Methods/Phylostratigraphy&#x0201d; section and in the caption of Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">6</a></sup>. All other data supporting the findings of this study are available within the article and its Supplementary information files or are available from the corresponding authors upon request.&#x000a0;<sup><a href=\"#MOESM21\" rid=\"MOESM21\" class=\" media\">Source data</a></sup> are provided with this paper.</p></span><span id=\"idm139716135074128\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139716135074128title\">Code availability</h2><p id=\"__p37\">PRINGO R scripts are deposited at GitHub: <a href=\"https://github.com/ShingoMiyauchi/PRINGO\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://github.com/ShingoMiyauchi/PRINGO</a>. The program COMPARE is also available at Github: <a href=\"https://github.com/laszlognagy/COMPARE\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://github.com/laszlognagy/COMPARE</a>.</p></span><span id=\"FPar1\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"FPar1title\">Competing interests</h2><p id=\"Par47\">The authors declare no competing interests.</p></span><br><div class=\"ref-list-sec sec\" id=\"reference-list\"><div class=\"ref-cit-blk half_rhythm\" id=\"CR1\">1. <span class=\"element-citation\">Van der, Heijden MGA, Martin FM, Selosse M-A, Sanders IR. Mycorrhizal ecology and evolution: the past, the present, and the future. <span><span class=\"ref-journal\">New Phytol. </span>2015;<span class=\"ref-vol\">205</span>:1406–1423. doi: 10.1111/nph.13288.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n25639293\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fnph.13288\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=New+Phytol.&amp;title=Mycorrhizal+ecology+and+evolution:+the+past,+the+present,+and+the+future&amp;author=MGA+Van+der,+Heijden&amp;author=FM+Martin&amp;author=M-A+Selosse&amp;author=IR+Sanders&amp;volume=205&amp;publication_year=2015&amp;pages=1406-1423&amp;pmid=25639293&amp;doi=10.1111/nph.13288&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR2\">2. <span class=\"element-citation\">Genre A, Lanfranco L, Perotto S, Bonfante P. Unique and common traits in mycorrhizal symbioses. <span><span class=\"ref-journal\">Nat. Rev. Microbiol. </span>2020 doi: 10.1038/s41579-020-0402-3.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n32694620\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41579-020-0402-3\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Microbiol.&amp;title=Unique+and+common+traits+in+mycorrhizal+symbioses&amp;author=A+Genre&amp;author=L+Lanfranco&amp;author=S+Perotto&amp;author=P+Bonfante&amp;publication_year=2020&amp;pmid=32694620&amp;doi=10.1038/s41579-020-0402-3&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR3\">3. <span class=\"element-citation\">Martin F, Kohler A, Murat C, Veneault-Fourrey C, Hibbett DS. Unearthing the roots of ectomycorrhizal symbioses. <span><span class=\"ref-journal\">Nat. Rev. Microbiol. </span>2016;<span class=\"ref-vol\">14</span>:760–773. doi: 10.1038/nrmicro.2016.149.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n27795567\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnrmicro.2016.149\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Microbiol.&amp;title=Unearthing+the+roots+of+ectomycorrhizal+symbioses&amp;author=F+Martin&amp;author=A+Kohler&amp;author=C+Murat&amp;author=C+Veneault-Fourrey&amp;author=DS+Hibbett&amp;volume=14&amp;publication_year=2016&amp;pages=760-773&amp;pmid=27795567&amp;doi=10.1038/nrmicro.2016.149&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR4\">4. <span class=\"element-citation\">Strullu&#x02010;Derrien C, Selosse M-A, Kenrick P, Martin FM. The origin and evolution of mycorrhizal symbioses: from palaeomycology to phylogenomics. <span><span class=\"ref-journal\">New Phytol. </span>2018;<span class=\"ref-vol\">220</span>:1012–1030. doi: 10.1111/nph.15076.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n29573278\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fnph.15076\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=New+Phytol.&amp;title=The+origin+and+evolution+of+mycorrhizal+symbioses:+from+palaeomycology+to+phylogenomics&amp;author=C+Strullu&#x02010;Derrien&amp;author=M-A+Selosse&amp;author=P+Kenrick&amp;author=FM+Martin&amp;volume=220&amp;publication_year=2018&amp;pages=1012-1030&amp;pmid=29573278&amp;doi=10.1111/nph.15076&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR5\">5. <span class=\"element-citation\">Brundrett MC, Tedersoo L. Evolutionary history of mycorrhizal symbioses and global host plant diversity. <span><span class=\"ref-journal\">New Phytol. </span>2018;<span class=\"ref-vol\">220</span>:1108–1115. doi: 10.1111/nph.14976.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n29355963\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fnph.14976\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=New+Phytol.&amp;title=Evolutionary+history+of+mycorrhizal+symbioses+and+global+host+plant+diversity&amp;author=MC+Brundrett&amp;author=L+Tedersoo&amp;volume=220&amp;publication_year=2018&amp;pages=1108-1115&amp;pmid=29355963&amp;doi=10.1111/nph.14976&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR6\">6. <span class=\"element-citation\">Matheny PB, et al.  Out of the Palaeotropics? Historical biogeography and diversification of the cosmopolitan ectomycorrhizal mushroom family Inocybaceae. <span><span class=\"ref-journal\">J. Biogeography. </span>2009;<span class=\"ref-vol\">36</span>:577–592. doi: 10.1111/j.1365-2699.2008.02055.x.</span> [<a href=\"//dx.doi.org/10.1111%2Fj.1365-2699.2008.02055.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Biogeography&amp;title=Out+of+the+Palaeotropics?+Historical+biogeography+and+diversification+of+the+cosmopolitan+ectomycorrhizal+mushroom+family+Inocybaceae&amp;author=PB+Matheny&amp;volume=36&amp;publication_year=2009&amp;pages=577-592&amp;doi=10.1111/j.1365-2699.2008.02055.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR7\">7. <span class=\"element-citation\">Tedersoo L, Smith ME. Ectomycorrhizal fungal lineages: detection of four new groups and notes on consistent recognition of ectomycorrhizal taxa in high-throughput sequencing studies. <span><span class=\"ref-journal\">Ecol. Stud. </span>2017;<span class=\"ref-vol\">230</span>:125–142. doi: 10.1007/978-3-319-56363-3_6.</span> [<a href=\"//dx.doi.org/10.1007%2F978-3-319-56363-3_6\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Ecol.+Stud.&amp;title=Ectomycorrhizal+fungal+lineages:+detection+of+four+new+groups+and+notes+on+consistent+recognition+of+ectomycorrhizal+taxa+in+high-throughput+sequencing+studies&amp;author=L+Tedersoo&amp;author=ME+Smith&amp;volume=230&amp;publication_year=2017&amp;pages=125-142&amp;doi=10.1007/978-3-319-56363-3_6&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR8\">8. <span class=\"element-citation\">Lutzoni F, et al.  Contemporaneous radiations of fungi and plants linked to symbiosis. <span><span class=\"ref-journal\">Nat. Commun. </span>2018;<span class=\"ref-vol\">9</span>:5451. doi: 10.1038/s41467-018-07849-9.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC6303338/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30575731\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41467-018-07849-9\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Contemporaneous+radiations+of+fungi+and+plants+linked+to+symbiosis&amp;author=F+Lutzoni&amp;volume=9&amp;publication_year=2018&amp;pages=5451&amp;pmid=30575731&amp;doi=10.1038/s41467-018-07849-9&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR9\">9. <span class=\"element-citation\">Chang Y, et al.  Phylogenomics of Endogonaceae and evolution of mycorrhizas within Mucoromycota. <span><span class=\"ref-journal\">New Phytol. </span>2019;<span class=\"ref-vol\">222</span>:511–525. doi: 10.1111/nph.15613.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30485448\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fnph.15613\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=New+Phytol.&amp;title=Phylogenomics+of+Endogonaceae+and+evolution+of+mycorrhizas+within+Mucoromycota&amp;author=Y+Chang&amp;volume=222&amp;publication_year=2019&amp;pages=511-525&amp;pmid=30485448&amp;doi=10.1111/nph.15613&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR10\">10. <span class=\"element-citation\">Martin F, et al.  The genome of <em>Laccaria bicolor</em> provides insights into mycorrhizal symbiosis. <span><span class=\"ref-journal\">Nature. </span>2008;<span class=\"ref-vol\">452</span>:88–92. doi: 10.1038/nature06556.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n18322534\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnature06556\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+genome+of+Laccaria+bicolor+provides+insights+into+mycorrhizal+symbiosis&amp;author=F+Martin&amp;volume=452&amp;publication_year=2008&amp;pages=88-92&amp;pmid=18322534&amp;doi=10.1038/nature06556&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR11\">11. <span class=\"element-citation\">Martin F, et al.  P&#x000e9;rigord black truffle genome uncovers evolutionary origins and mechanisms of symbiosis. <span><span class=\"ref-journal\">Nature. </span>2010;<span class=\"ref-vol\">464</span>:1033–1038. doi: 10.1038/nature08867.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n20348908\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnature08867\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=P&#x000e9;rigord+black+truffle+genome+uncovers+evolutionary+origins+and+mechanisms+of+symbiosis&amp;author=F+Martin&amp;volume=464&amp;publication_year=2010&amp;pages=1033-1038&amp;pmid=20348908&amp;doi=10.1038/nature08867&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR12\">12. <span class=\"element-citation\">Kohler A, et al.  Convergent losses of decay mechanisms and rapid turnover of symbiosis genes in mycorrhizal mutualists. <span><span class=\"ref-journal\">Nat. Genet. </span>2015;<span class=\"ref-vol\">47</span>:410–415. doi: 10.1038/ng.3223.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n25706625\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fng.3223\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Convergent+losses+of+decay+mechanisms+and+rapid+turnover+of+symbiosis+genes+in+mycorrhizal+mutualists&amp;author=A+Kohler&amp;volume=47&amp;publication_year=2015&amp;pages=410-415&amp;pmid=25706625&amp;doi=10.1038/ng.3223&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR13\">13. <span class=\"element-citation\">Peter M, et al.  Ectomycorrhizal ecology is imprinted in the genome of the dominant symbiotic fungus <em>Cenococcum geophilum</em>. <span><span class=\"ref-journal\">Nat. Commun. </span>2016;<span class=\"ref-vol\">7</span>:12662. doi: 10.1038/ncomms12662.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC5023957/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n27601008\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fncomms12662\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Ectomycorrhizal+ecology+is+imprinted+in+the+genome+of+the+dominant+symbiotic+fungus+Cenococcum+geophilum&amp;author=M+Peter&amp;volume=7&amp;publication_year=2016&amp;pages=12662&amp;pmid=27601008&amp;doi=10.1038/ncomms12662&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR14\">14. <span class=\"element-citation\">Hess JI, et al.  Rapid divergence of genome architectures following the origin of an ectomycorrhizal symbiosis in the genus <em>Amanita</em>. <span><span class=\"ref-journal\">Mol. Biol. Evol. </span>2018;<span class=\"ref-vol\">35</span>:2786–2804.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC6231487/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30239843\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Biol.+Evol.&amp;title=Rapid+divergence+of+genome+architectures+following+the+origin+of+an+ectomycorrhizal+symbiosis+in+the+genus+Amanita&amp;author=JI+Hess&amp;volume=35&amp;publication_year=2018&amp;pages=2786-2804&amp;pmid=30239843&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR15\">15. <span class=\"element-citation\">Martino E, et al.  Comparative genomics and transcriptomics depict ericoid mycorrhizal fungi as versatile saprotrophs and plant mutualists. <span><span class=\"ref-journal\">New Phytol. </span>2018;<span class=\"ref-vol\">217</span>:1213–1229. doi: 10.1111/nph.14974.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n29315638\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fnph.14974\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=New+Phytol.&amp;title=Comparative+genomics+and+transcriptomics+depict+ericoid+mycorrhizal+fungi+as+versatile+saprotrophs+and+plant+mutualists&amp;author=E+Martino&amp;volume=217&amp;publication_year=2018&amp;pages=1213-1229&amp;pmid=29315638&amp;doi=10.1111/nph.14974&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR16\">16. <span class=\"element-citation\">Murat C, et al.  Pezizomycetes genomes reveal the molecular basis of ectomycorrhizal truffle lifestyle. <span><span class=\"ref-journal\">Nat. Ecol. Evol. </span>2018;<span class=\"ref-vol\">2</span>:1956–1965. doi: 10.1038/s41559-018-0710-4.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30420746\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41559-018-0710-4\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Ecol.+Evol.&amp;title=Pezizomycetes+genomes+reveal+the+molecular+basis+of+ectomycorrhizal+truffle+lifestyle&amp;author=C+Murat&amp;volume=2&amp;publication_year=2018&amp;pages=1956-1965&amp;pmid=30420746&amp;doi=10.1038/s41559-018-0710-4&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR17\">17. <span class=\"element-citation\">Pellitier PT, Zak DR. Ectomycorrhizal fungi and the enzymatic liberation of nitrogen from soil organic matter: why evolutionary history matters. <span><span class=\"ref-journal\">New Phytol. </span>2018;<span class=\"ref-vol\">217</span>:68–73. doi: 10.1111/nph.14598.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n29193221\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fnph.14598\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=New+Phytol.&amp;title=Ectomycorrhizal+fungi+and+the+enzymatic+liberation+of+nitrogen+from+soil+organic+matter:+why+evolutionary+history+matters&amp;author=PT+Pellitier&amp;author=DR+Zak&amp;volume=217&amp;publication_year=2018&amp;pages=68-73&amp;pmid=29193221&amp;doi=10.1111/nph.14598&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR18\">18. <span class=\"element-citation\">Frey SD. Mycorrhizal fungi as mediators of soil organic matter dynamics. <span><span class=\"ref-journal\">Annu. Rev. Ecol. Evol. Syst. </span>2019;<span class=\"ref-vol\">50</span>:237–259. doi: 10.1146/annurev-ecolsys-110617-062331.</span> [<a href=\"//dx.doi.org/10.1146%2Fannurev-ecolsys-110617-062331\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Ecol.+Evol.+Syst.&amp;title=Mycorrhizal+fungi+as+mediators+of+soil+organic+matter+dynamics&amp;author=SD+Frey&amp;volume=50&amp;publication_year=2019&amp;pages=237-259&amp;doi=10.1146/annurev-ecolsys-110617-062331&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR19\">19. <span class=\"element-citation\">Clemmensen KE, et al.  Roots and associated fungi drive long-term carbon sequestration in boreal forest. <span><span class=\"ref-journal\">Science. </span>2013;<span class=\"ref-vol\">339</span>:1615–1618. doi: 10.1126/science.1231923.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n23539604\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1126%2Fscience.1231923\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Roots+and+associated+fungi+drive+long-term+carbon+sequestration+in+boreal+forest&amp;author=KE+Clemmensen&amp;volume=339&amp;publication_year=2013&amp;pages=1615-1618&amp;pmid=23539604&amp;doi=10.1126/science.1231923&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR20\">20. <span class=\"element-citation\">Lindahl BD, et al.  Spatial separation of litter decomposition and mycorrhizal nitrogen uptake in a boreal forest. <span><span class=\"ref-journal\">New Phytol. </span>2007;<span class=\"ref-vol\">173</span>:611–620. doi: 10.1111/j.1469-8137.2006.01936.x.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n17244056\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1469-8137.2006.01936.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=New+Phytol.&amp;title=Spatial+separation+of+litter+decomposition+and+mycorrhizal+nitrogen+uptake+in+a+boreal+forest&amp;author=BD+Lindahl&amp;volume=173&amp;publication_year=2007&amp;pages=611-620&amp;pmid=17244056&amp;doi=10.1111/j.1469-8137.2006.01936.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR21\">21. <span class=\"element-citation\">Akroume E, et al.  First evidences that the ectomycorrhizal fungus <em>Paxillus involutus</em> mobilizes nitrogen and carbon from saprotrophic fungus necromass. <span><span class=\"ref-journal\">Environ. Microbiol. </span>2019;<span class=\"ref-vol\">21</span>:197–208. doi: 10.1111/1462-2920.14440.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30307107\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2F1462-2920.14440\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Environ.+Microbiol.&amp;title=First+evidences+that+the+ectomycorrhizal+fungus+Paxillus+involutus+mobilizes+nitrogen+and+carbon+from+saprotrophic+fungus+necromass&amp;author=E+Akroume&amp;volume=21&amp;publication_year=2019&amp;pages=197-208&amp;pmid=30307107&amp;doi=10.1111/1462-2920.14440&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR22\">22. <span class=\"element-citation\">Maillard F, Schilling J, Andrews E, Schreiner KM, Kennedy P. Functional convergence in the decomposition of fungal necromass in soil and wood. <span><span class=\"ref-journal\">FEMS Microbiol. Ecol. </span>2020;<span class=\"ref-vol\">96</span>:fiz209. doi: 10.1093/femsec/fiz209.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n31868883\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Ffemsec%2Ffiz209\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=FEMS+Microbiol.+Ecol.&amp;title=Functional+convergence+in+the+decomposition+of+fungal+necromass+in+soil+and+wood&amp;author=F+Maillard&amp;author=J+Schilling&amp;author=E+Andrews&amp;author=KM+Schreiner&amp;author=P+Kennedy&amp;volume=96&amp;publication_year=2020&amp;pages=fiz209&amp;pmid=31868883&amp;doi=10.1093/femsec/fiz209&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR23\">23. <span class=\"element-citation\">Koide RT, Sharda JN, Herr JR, Malcolm GM. Ectomycorrhizal fungi and the biotrophy&#x02013;saprotrophy continuum. <span><span class=\"ref-journal\">New Phytol. </span>2008;<span class=\"ref-vol\">178</span>:230–233. doi: 10.1111/j.1469-8137.2008.02401.x.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n18312537\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1469-8137.2008.02401.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=New+Phytol.&amp;title=Ectomycorrhizal+fungi+and+the+biotrophy&#x02013;saprotrophy+continuum&amp;author=RT+Koide&amp;author=JN+Sharda&amp;author=JR+Herr&amp;author=GM+Malcolm&amp;volume=178&amp;publication_year=2008&amp;pages=230-233&amp;pmid=18312537&amp;doi=10.1111/j.1469-8137.2008.02401.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR24\">24. <span class=\"element-citation\">Sipos G, et al.  Genome expansion and lineage-specific genetic innovations in the forest pathogenic fungi <em>Armillaria</em>. <span><span class=\"ref-journal\">Nat. Ecol. Evol. </span>2017;<span class=\"ref-vol\">1</span>:1931–1941. doi: 10.1038/s41559-017-0347-8.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n29085064\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41559-017-0347-8\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Ecol.+Evol.&amp;title=Genome+expansion+and+lineage-specific+genetic+innovations+in+the+forest+pathogenic+fungi+Armillaria&amp;author=G+Sipos&amp;volume=1&amp;publication_year=2017&amp;pages=1931-1941&amp;pmid=29085064&amp;doi=10.1038/s41559-017-0347-8&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR25\">25. <span class=\"element-citation\">Tedersoo L, Bahram M. Mycorrhizal types differ in ecophysiology and alter plant nutrition and soil processes. <span><span class=\"ref-journal\">Biol. Rev. </span>2019;<span class=\"ref-vol\">94</span>:1857–1880. doi: 10.1111/brv.12538.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n31270944\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fbrv.12538\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Biol.+Rev.&amp;title=Mycorrhizal+types+differ+in+ecophysiology+and+alter+plant+nutrition+and+soil+processes&amp;author=L+Tedersoo&amp;author=M+Bahram&amp;volume=94&amp;publication_year=2019&amp;pages=1857-1880&amp;pmid=31270944&amp;doi=10.1111/brv.12538&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR26\">26. <span class=\"element-citation\">Floudas D, et al.  The Paleozoic origin of enzymatic lignin decomposition reconstructed from 31 fungal genomes. <span><span class=\"ref-journal\">Science. </span>2012;<span class=\"ref-vol\">336</span>:1715–1719. doi: 10.1126/science.1221748.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n22745431\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1126%2Fscience.1221748\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+Paleozoic+origin+of+enzymatic+lignin+decomposition+reconstructed+from+31+fungal+genomes&amp;author=D+Floudas&amp;volume=336&amp;publication_year=2012&amp;pages=1715-1719&amp;pmid=22745431&amp;doi=10.1126/science.1221748&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR27\">27. <span class=\"element-citation\">Morin E, et al.  Comparative genomics of <em>Rhizophagus irregularis</em>, <em>R. cerebriforme</em>, <em>R. diaphanus</em> and <em>Gigaspora rosea</em> highlights specific genetic features in Glomeromycotina. <span><span class=\"ref-journal\">New Phytol. </span>2019;<span class=\"ref-vol\">222</span>:1584–1598. doi: 10.1111/nph.15687.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30636349\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fnph.15687\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=New+Phytol.&amp;title=Comparative+genomics+of+Rhizophagus+irregularis,+R.+cerebriforme,+R.+diaphanus+and+Gigaspora+rosea+highlights+specific+genetic+features+in+Glomeromycotina&amp;author=E+Morin&amp;volume=222&amp;publication_year=2019&amp;pages=1584-1598&amp;pmid=30636349&amp;doi=10.1111/nph.15687&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR28\">28. <span class=\"element-citation\">Varga T, et al.  Megaphylogeny resolves global patterns of mushroom evolution. <span><span class=\"ref-journal\">Nat. Ecol. Evol. </span>2019;<span class=\"ref-vol\">3</span>:668–678. doi: 10.1038/s41559-019-0834-1.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC6443077/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30886374\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41559-019-0834-1\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Ecol.+Evol.&amp;title=Megaphylogeny+resolves+global+patterns+of+mushroom+evolution&amp;author=T+Varga&amp;volume=3&amp;publication_year=2019&amp;pages=668-678&amp;pmid=30886374&amp;doi=10.1038/s41559-019-0834-1&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR29\">29. <span class=\"element-citation\">Zak DR, et al.  Exploring the role of ectomycorrhizal fungi in soil carbon dynamics. <span><span class=\"ref-journal\">New Phytol. </span>2019;<span class=\"ref-vol\">223</span>:33–39. doi: 10.1111/nph.15679.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30636276\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fnph.15679\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=New+Phytol.&amp;title=Exploring+the+role+of+ectomycorrhizal+fungi+in+soil+carbon+dynamics&amp;author=DR+Zak&amp;volume=223&amp;publication_year=2019&amp;pages=33-39&amp;pmid=30636276&amp;doi=10.1111/nph.15679&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR30\">30. <span class=\"element-citation\">Nagy NG, et al.  Comparative genomics of early-diverging mushroom-forming fungi provides insights into the origins of lignocellulose decay capabilities. <span><span class=\"ref-journal\">Mol. Biol. Evol. </span>2016;<span class=\"ref-vol\">33</span>:959–970. doi: 10.1093/molbev/msv337.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n26659563\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fmolbev%2Fmsv337\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Biol.+Evol.&amp;title=Comparative+genomics+of+early-diverging+mushroom-forming+fungi+provides+insights+into+the+origins+of+lignocellulose+decay+capabilities&amp;author=NG+Nagy&amp;volume=33&amp;publication_year=2016&amp;pages=959-970&amp;pmid=26659563&amp;doi=10.1093/molbev/msv337&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR31\">31. <span class=\"element-citation\">Churchland C, Grayston SJ. Specificity of plant-microbe interactions in the tree mycorrhizosphere biome and consequences for soil C cycling. <span><span class=\"ref-journal\">Front. Microbiol. </span>2014;<span class=\"ref-vol\">5</span>:261. doi: 10.3389/fmicb.2014.00261.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC4042908/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24917855\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3389%2Ffmicb.2014.00261\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front.+Microbiol&amp;title=Specificity+of+plant-microbe+interactions+in+the+tree+mycorrhizosphere+biome+and+consequences+for+soil+C+cycling&amp;author=C+Churchland&amp;author=SJ+Grayston&amp;volume=5&amp;publication_year=2014&amp;pages=261&amp;pmid=24917855&amp;doi=10.3389/fmicb.2014.00261&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR32\">32. <span class=\"element-citation\">Shah F, et al.  Ectomycorrhizal fungi decompose soil organic matter using oxidative mechanisms adapted from saprotrophic ancestors. <span><span class=\"ref-journal\">New Phytol. </span>2016;<span class=\"ref-vol\">209</span>:1705–1719. doi: 10.1111/nph.13722.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC5061094/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n26527297\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fnph.13722\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=New+Phytol.&amp;title=Ectomycorrhizal+fungi+decompose+soil+organic+matter+using+oxidative+mechanisms+adapted+from+saprotrophic+ancestors&amp;author=F+Shah&amp;volume=209&amp;publication_year=2016&amp;pages=1705-1719&amp;pmid=26527297&amp;doi=10.1111/nph.13722&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR33\">33. <span class=\"element-citation\">Krizs&#x000e1;n K, et al.  Transcriptomic atlas of mushroom development reveals conserved genes behind complex multicellularity in fungi. <span><span class=\"ref-journal\">Proc. Natl Acad. Sci. USA. </span>2019;<span class=\"ref-vol\">116</span>:7409–7418. doi: 10.1073/pnas.1817822116.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC6462078/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30902897\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1073%2Fpnas.1817822116\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=Transcriptomic+atlas+of+mushroom+development+reveals+conserved+genes+behind+complex+multicellularity+in+fungi&amp;author=K+Krizs&#x000e1;n&amp;volume=116&amp;publication_year=2019&amp;pages=7409-7418&amp;pmid=30902897&amp;doi=10.1073/pnas.1817822116&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR34\">34. <span class=\"element-citation\">Tedersoo L, May TW, Smith ME. Ectomycorrhizal lifestyle in fungi: global diversity, distribution, and evolution of phylogenetic lineages. <span><span class=\"ref-journal\">Mycorrhiza. </span>2010;<span class=\"ref-vol\">20</span>:217–263. doi: 10.1007/s00572-009-0274-x.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n20191371\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs00572-009-0274-x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mycorrhiza&amp;title=Ectomycorrhizal+lifestyle+in+fungi:+global+diversity,+distribution,+and+evolution+of+phylogenetic+lineages&amp;author=L+Tedersoo&amp;author=TW+May&amp;author=ME+Smith&amp;volume=20&amp;publication_year=2010&amp;pages=217-263&amp;pmid=20191371&amp;doi=10.1007/s00572-009-0274-x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR35\">35. <span class=\"element-citation\">Tedersoo L, Brundrett M. Evolution of ectomycorrhizal symbiosis in plants. <span><span class=\"ref-journal\">Ecol. Stud. </span>2017;<span class=\"ref-vol\">230</span>:407–467. doi: 10.1007/978-3-319-56363-3_19.</span> [<a href=\"//dx.doi.org/10.1007%2F978-3-319-56363-3_19\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Ecol.+Stud.&amp;title=Evolution+of+ectomycorrhizal+symbiosis+in+plants&amp;author=L+Tedersoo&amp;author=M+Brundrett&amp;volume=230&amp;publication_year=2017&amp;pages=407-467&amp;doi=10.1007/978-3-319-56363-3_19&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR36\">36. <span class=\"element-citation\">B&#x000f6;deker ITM, et al.  Ectomycorrhizal <em>Cortinarius</em> species participate in enzymatic oxidation of humus in northern forest ecosystems. <span><span class=\"ref-journal\">New Phytol. </span>2014;<span class=\"ref-vol\">203</span>:245–256. doi: 10.1111/nph.12791.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24725281\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fnph.12791\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=New+Phytol.&amp;title=Ectomycorrhizal+Cortinarius+species+participate+in+enzymatic+oxidation+of+humus+in+northern+forest+ecosystems&amp;author=ITM+B&#x000f6;deker&amp;volume=203&amp;publication_year=2014&amp;pages=245-256&amp;pmid=24725281&amp;doi=10.1111/nph.12791&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR37\">37. <span class=\"element-citation\">Kusuda M, Ueda M, Miyatake K, Terashita T. Characterization of the carbohydrase productions of an ectomycorrhizal fungus, <em>Tricholoma matsutake</em>. <span><span class=\"ref-journal\">Mycoscience. </span>2008;<span class=\"ref-vol\">49</span>:291–297. doi: 10.1007/S10267-008-0423-7.</span> [<a href=\"//dx.doi.org/10.1007%2FS10267-008-0423-7\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mycoscience&amp;title=Characterization+of+the+carbohydrase+productions+of+an+ectomycorrhizal+fungus,+Tricholoma+matsutake&amp;author=M+Kusuda&amp;author=M+Ueda&amp;author=K+Miyatake&amp;author=T+Terashita&amp;volume=49&amp;publication_year=2008&amp;pages=291-297&amp;doi=10.1007/S10267-008-0423-7&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR38\">38. <span class=\"element-citation\">Rineau F, et al.  Carbon availability triggers the decomposition of plant litter and assimilation of nitrogen by an ectomycorrhizal fungus. <span><span class=\"ref-journal\">ISME J. </span>2013;<span class=\"ref-vol\">7</span>:2010–2022. doi: 10.1038/ismej.2013.91.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3965319/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n23788332\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fismej.2013.91\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=ISME+J.&amp;title=Carbon+availability+triggers+the+decomposition+of+plant+litter+and+assimilation+of+nitrogen+by+an+ectomycorrhizal+fungus&amp;author=F+Rineau&amp;volume=7&amp;publication_year=2013&amp;pages=2010-2022&amp;pmid=23788332&amp;doi=10.1038/ismej.2013.91&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR39\">39. <span class=\"element-citation\">Lindahl BD, Tunlid A. Ectomycorrhizal fungi&#x02014;potential organic matter decomposers, yet not saprotrophs. <span><span class=\"ref-journal\">New Phytol. </span>2015;<span class=\"ref-vol\">205</span>:1443–1447. doi: 10.1111/nph.13201.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n25524234\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fnph.13201\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=New+Phytol.&amp;title=Ectomycorrhizal+fungi&#x02014;potential+organic+matter+decomposers,+yet+not+saprotrophs&amp;author=BD+Lindahl&amp;author=A+Tunlid&amp;volume=205&amp;publication_year=2015&amp;pages=1443-1447&amp;pmid=25524234&amp;doi=10.1111/nph.13201&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR40\">40. <span class=\"element-citation\">Op De Beeck M, Troein C, Peterson C, Persson P, Tunlid A. Fenton reaction facilitates organic nitrogen acquisition by an ectomycorrhizal fungus. <span><span class=\"ref-journal\">New Phytol. </span>2018;<span class=\"ref-vol\">218</span>:335–343. doi: 10.1111/nph.14971.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC5873446/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n29297591\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fnph.14971\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=New+Phytol.&amp;title=Fenton+reaction+facilitates+organic+nitrogen+acquisition+by+an+ectomycorrhizal+fungus&amp;author=M+Op+De+Beeck&amp;author=C+Troein&amp;author=C+Peterson&amp;author=P+Persson&amp;author=A+Tunlid&amp;volume=218&amp;publication_year=2018&amp;pages=335-343&amp;pmid=29297591&amp;doi=10.1111/nph.14971&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR41\">41. <span class=\"element-citation\">Nicol&#x000e1;s C, et al.  The soil organic matter decomposition mechanisms in ectomycorrhizal fungi are tuned for liberating soil organic nitrogen. <span><span class=\"ref-journal\">ISME J. </span>2019;<span class=\"ref-vol\">13</span>:977–988. doi: 10.1038/s41396-018-0331-6.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC6461840/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30538275\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41396-018-0331-6\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=ISME+J.&amp;title=The+soil+organic+matter+decomposition+mechanisms+in+ectomycorrhizal+fungi+are+tuned+for+liberating+soil+organic+nitrogen&amp;author=C+Nicol&#x000e1;s&amp;volume=13&amp;publication_year=2019&amp;pages=977-988&amp;pmid=30538275&amp;doi=10.1038/s41396-018-0331-6&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR42\">42. <span class=\"element-citation\">Zhang F, et al.  The ectomycorrhizal basidiomycete <em>Laccaria bicolor</em> releases a secreted &#x003b2;&#x02010;1,4 endoglucanase that plays a key role in symbiosis development. <span><span class=\"ref-journal\">New Phytol. </span>2018;<span class=\"ref-vol\">220</span>:1309–1321. doi: 10.1111/nph.15113.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n29624684\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fnph.15113\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=New+Phytol.&amp;title=The+ectomycorrhizal+basidiomycete+Laccaria+bicolor+releases+a+secreted+&#x003b2;&#x02010;1,4+endoglucanase+that+plays+a+key+role+in+symbiosis+development&amp;author=F+Zhang&amp;volume=220&amp;publication_year=2018&amp;pages=1309-1321&amp;pmid=29624684&amp;doi=10.1111/nph.15113&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR43\">43. <span class=\"element-citation\">Plett JM, et al.  Effector MiSSP7 of the mutualistic fungus <em>Laccaria bicolor</em> stabilizes the <em>Populus</em> JAZ6 protein and represses jasmonic acid (JA) responsive genes. <span><span class=\"ref-journal\">Proc. Natl Acad. Sci. USA. </span>2014;<span class=\"ref-vol\">111</span>:8299–8304. doi: 10.1073/pnas.1322671111.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC4050555/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24847068\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1073%2Fpnas.1322671111\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=Effector+MiSSP7+of+the+mutualistic+fungus+Laccaria+bicolor+stabilizes+the+Populus+JAZ6+protein+and+represses+jasmonic+acid+(JA)+responsive+genes&amp;author=JM+Plett&amp;volume=111&amp;publication_year=2014&amp;pages=8299-8304&amp;pmid=24847068&amp;doi=10.1073/pnas.1322671111&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR44\">44. <span class=\"element-citation\">Kang H, et al.  The small secreted effector protein MiSSP7.6 of <em>Laccaria bicolor</em> is required for the establishment of ectomycorrhizal symbiosis. <span><span class=\"ref-journal\">Environ. Microbiol. </span>2020;<span class=\"ref-vol\">22</span>:1435–1446. doi: 10.1111/1462-2920.14959.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n32090429\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2F1462-2920.14959\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Environ.+Microbiol.&amp;title=The+small+secreted+effector+protein+MiSSP7.6+of+Laccaria+bicolor+is+required+for+the+establishment+of+ectomycorrhizal+symbiosis&amp;author=H+Kang&amp;volume=22&amp;publication_year=2020&amp;pages=1435-1446&amp;pmid=32090429&amp;doi=10.1111/1462-2920.14959&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR45\">45. <span class=\"mixed-citation\">Plett, J. M. et al. Mycorrhizal effector PaMiSSP10b alters polyamine biosynthesis in <em>Eucalyptus</em> root cells and promotes root colonization. <em>New Phytol</em>. (2020, in the press). [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n32562507\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>]</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR46\">46. <span class=\"element-citation\">Alm&#x000e1;si, et al.  Comparative genomics reveals unique wood&#x02010;decay strategies and fruiting body development in the Schizophyllaceae. <span><span class=\"ref-journal\">New Phytol. </span>2019;<span class=\"ref-vol\">224</span>:902–915. doi: 10.1111/nph.16032.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n31257601\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fnph.16032\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=New+Phytol.&amp;title=Comparative+genomics+reveals+unique+wood&#x02010;decay+strategies+and+fruiting+body+development+in+the+Schizophyllaceae&amp;author=+Alm&#x000e1;si&amp;volume=224&amp;publication_year=2019&amp;pages=902-915&amp;pmid=31257601&amp;doi=10.1111/nph.16032&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR47\">47. <span class=\"element-citation\">Schurko AM, Neiman M, Logsdon JM. Signs of sex: what we know and how we know it. <span><span class=\"ref-journal\">Trends Ecol. Evol. </span>2009;<span class=\"ref-vol\">24</span>:208–217. doi: 10.1016/j.tree.2008.11.010.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n19282047\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.tree.2008.11.010\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Trends+Ecol.+Evol.&amp;title=Signs+of+sex:+what+we+know+and+how+we+know+it&amp;author=AM+Schurko&amp;author=M+Neiman&amp;author=JM+Logsdon&amp;volume=24&amp;publication_year=2009&amp;pages=208-217&amp;pmid=19282047&amp;doi=10.1016/j.tree.2008.11.010&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR48\">48. <span class=\"element-citation\">Frantzeskakis L, et al.  Signatures of host specialization and a recent transposable element burst in the dynamic one-speed genome of the fungal barley powdery mildew pathogen. <span><span class=\"ref-journal\">BMC Genomics. </span>2018;<span class=\"ref-vol\">19</span>:27. doi: 10.1186/s12864-018-4750-6.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC5964911/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n29788921\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2Fs12864-018-4750-6\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Genomics&amp;title=Signatures+of+host+specialization+and+a+recent+transposable+element+burst+in+the+dynamic+one-speed+genome+of+the+fungal+barley+powdery+mildew+pathogen&amp;author=L+Frantzeskakis&amp;volume=19&amp;publication_year=2018&amp;pages=27&amp;pmid=29306326&amp;doi=10.1186/s12864-018-4750-6&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR49\">49. <span class=\"mixed-citation\">Pellegrin, C., Morin, E., Martin, F. M. &#x00026; Veneault-Fourrey, C. Comparative analysis of secretomes from ectomycorrhizal fungi with an emphasis on small-secreted proteins. <em>Front. Microbiol</em>. <strong>6</strong>, 1278 (2015). <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC4649063/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n26635749\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>]</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR50\">50. <span class=\"element-citation\">Pellegrin C, et al.  <em>Laccaria bicolor</em> MiSSP8 is a small&#x02010;secreted protein decisive for the establishment of the ectomycorrhizal symbiosis. <span><span class=\"ref-journal\">Environ. Microbiol. </span>2019;<span class=\"ref-vol\">21</span>:3765–3779. doi: 10.1111/1462-2920.14727.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n31260142\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2F1462-2920.14727\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Environ.+Microbiol.&amp;title=Laccaria+bicolor+MiSSP8+is+a+small&#x02010;secreted+protein+decisive+for+the+establishment+of+the+ectomycorrhizal+symbiosis&amp;author=C+Pellegrin&amp;volume=21&amp;publication_year=2019&amp;pages=3765-3779&amp;pmid=31260142&amp;doi=10.1111/1462-2920.14727&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR51\">51. <span class=\"element-citation\">Liao HL, Chen Y, Vilgalys R. Metatranscriptomic study of common and host-specific patterns of gene expression between pines and their symbiotic ectomycorrhizal fungi in the genus <em>Suillus</em>. <span><span class=\"ref-journal\">PLOS Genet. </span>2018;<span class=\"ref-vol\">14</span>:e1007742. doi: 10.1371/journal.pgen.1007742.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC6195257/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30339692\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pgen.1007742\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLOS+Genet.&amp;title=Metatranscriptomic+study+of+common+and+host-specific+patterns+of+gene+expression+between+pines+and+their+symbiotic+ectomycorrhizal+fungi+in+the+genus+Suillus&amp;author=HL+Liao&amp;author=Y+Chen&amp;author=R+Vilgalys&amp;volume=14&amp;publication_year=2018&amp;pages=e1007742&amp;pmid=30339692&amp;doi=10.1371/journal.pgen.1007742&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR52\">52. <span class=\"element-citation\">Soltis DE, et al.  Chloroplast gene sequence data suggest a single origin of the predisposition for symbiotic nitrogen fixation in angiosperms. <span><span class=\"ref-journal\">Proc. Natl Acad. Sci. USA. </span>1995;<span class=\"ref-vol\">92</span>:2647–2651. doi: 10.1073/pnas.92.7.2647.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC42275/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n7708699\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1073%2Fpnas.92.7.2647\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=Chloroplast+gene+sequence+data+suggest+a+single+origin+of+the+predisposition+for+symbiotic+nitrogen+fixation+in+angiosperms&amp;author=DE+Soltis&amp;volume=92&amp;publication_year=1995&amp;pages=2647-2651&amp;pmid=7708699&amp;doi=10.1073/pnas.92.7.2647&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR53\">53. <span class=\"element-citation\">Werner GDA, Cornwell WK, Sprent JI, Kattge J, Kiers ET. A single evolutionary innovation drives the deep evolution of symbiotic N<sub>2</sub>-fixation in angiosperms. <span><span class=\"ref-journal\">Nat. Commun. </span>2014;<span class=\"ref-vol\">5</span>:4087. doi: 10.1038/ncomms5087.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC4059933/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24912610\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fncomms5087\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=A+single+evolutionary+innovation+drives+the+deep+evolution+of+symbiotic+N2-fixation+in+angiosperms&amp;author=GDA+Werner&amp;author=WK+Cornwell&amp;author=JI+Sprent&amp;author=J+Kattge&amp;author=ET+Kiers&amp;volume=5&amp;publication_year=2014&amp;pages=4087&amp;pmid=24912610&amp;doi=10.1038/ncomms5087&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR54\">54. <span class=\"element-citation\">Nagy LG, et al.  Latent homology and convergent regulatory evolution underlies the repeated emergence of yeasts. <span><span class=\"ref-journal\">Nat. Commun. </span>2014;<span class=\"ref-vol\">5</span>:4471. doi: 10.1038/ncomms5471.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n25034666\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fncomms5471\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Latent+homology+and+convergent+regulatory+evolution+underlies+the+repeated+emergence+of+yeasts&amp;author=LG+Nagy&amp;volume=5&amp;publication_year=2014&amp;pages=4471&amp;pmid=25034666&amp;doi=10.1038/ncomms5471&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR55\">55. <span class=\"element-citation\">Nagy LG, Kov&#x000e1;cs GM, Krizs&#x000e1;n K. Complex multicellularity in fungi: evolutionary convergence, single origin, or both? <span><span class=\"ref-journal\">Biol. Rev. </span>2018;<span class=\"ref-vol\">93</span>:1778–1794. doi: 10.1111/brv.12418.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n29675836\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fbrv.12418\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Biol.+Rev.&amp;title=Complex+multicellularity+in+fungi:+evolutionary+convergence,+single+origin,+or+both?&amp;author=LG+Nagy&amp;author=GM+Kov&#x000e1;cs&amp;author=K+Krizs&#x000e1;n&amp;volume=93&amp;publication_year=2018&amp;pages=1778-1794&amp;pmid=29675836&amp;doi=10.1111/brv.12418&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR56\">56. <span class=\"element-citation\">Tedersoo L, et al.  Global diversity and geography of soil fungi. <span><span class=\"ref-journal\">Science. </span>2014;<span class=\"ref-vol\">346</span>:1256688. doi: 10.1126/science.1256688.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n25430773\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1126%2Fscience.1256688\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Global+diversity+and+geography+of+soil+fungi&amp;author=L+Tedersoo&amp;volume=346&amp;publication_year=2014&amp;pages=1256688&amp;pmid=25430773&amp;doi=10.1126/science.1256688&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR57\">57. <span class=\"element-citation\">Nguyen NH, et al.  FUNGuild: an open annotation tool for parsing fungal community datasets by ecological guild. <span><span class=\"ref-journal\">Fungal Ecol. </span>2016;<span class=\"ref-vol\">20</span>:241–248. doi: 10.1016/j.funeco.2015.06.006.</span> [<a href=\"//dx.doi.org/10.1016%2Fj.funeco.2015.06.006\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Fungal+Ecol.&amp;title=FUNGuild:+an+open+annotation+tool+for+parsing+fungal+community+datasets+by+ecological+guild&amp;author=NH+Nguyen&amp;volume=20&amp;publication_year=2016&amp;pages=241-248&amp;doi=10.1016/j.funeco.2015.06.006&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR58\">58. <span class=\"element-citation\">Gnerre S, et al.  High-quality draft assemblies of mammalian genomes from massively parallel sequence data. <span><span class=\"ref-journal\">Proc. Natl Acad. Sci. USA. </span>2011;<span class=\"ref-vol\">108</span>:1513–1518. doi: 10.1073/pnas.1017351108.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3029755/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n21187386\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1073%2Fpnas.1017351108\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=High-quality+draft+assemblies+of+mammalian+genomes+from+massively+parallel+sequence+data&amp;author=S+Gnerre&amp;volume=108&amp;publication_year=2011&amp;pages=1513-1518&amp;pmid=21187386&amp;doi=10.1073/pnas.1017351108&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR59\">59. <span class=\"element-citation\">English AC, et al.  Mind the gap: upgrading genomes with Pacific Biosciences RS long-read sequencing technology. <span><span class=\"ref-journal\">PLoS ONE. </span>2012;<span class=\"ref-vol\">7</span>:e47768. doi: 10.1371/journal.pone.0047768.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3504050/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n23185243\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0047768\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Mind+the+gap:+upgrading+genomes+with+Pacific+Biosciences+RS+long-read+sequencing+technology&amp;author=AC+English&amp;volume=7&amp;publication_year=2012&amp;pages=e47768&amp;pmid=23185243&amp;doi=10.1371/journal.pone.0047768&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR60\">60. <span class=\"element-citation\">Chin C-S, et al.  Phased diploid genome assembly with single-molecule real-time sequencing. <span><span class=\"ref-journal\">Nat. Methods. </span>2016;<span class=\"ref-vol\">13</span>:1050–1054. doi: 10.1038/nmeth.4035.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC5503144/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n27749838\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnmeth.4035\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Phased+diploid+genome+assembly+with+single-molecule+real-time+sequencing&amp;author=C-S+Chin&amp;volume=13&amp;publication_year=2016&amp;pages=1050-1054&amp;pmid=27749838&amp;doi=10.1038/nmeth.4035&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR61\">61. <span class=\"element-citation\">Koren S, et al.  Hybrid error correction and <em>de novo</em> assembly of single-molecule sequencing reads. <span><span class=\"ref-journal\">Nat. Biotechnol. </span>2012;<span class=\"ref-vol\">30</span>:693–700. doi: 10.1038/nbt.2280.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3707490/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n22750884\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnbt.2280\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=Hybrid+error+correction+and+de+novo+assembly+of+single-molecule+sequencing+reads&amp;author=S+Koren&amp;volume=30&amp;publication_year=2012&amp;pages=693-700&amp;pmid=22750884&amp;doi=10.1038/nbt.2280&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR62\">62. <span class=\"element-citation\">Grabherr MG, et al.  Full-length transcriptome assembly from RNA-Seq data without a reference genome. <span><span class=\"ref-journal\">Nat. Biotechnol. </span>2011;<span class=\"ref-vol\">29</span>:644–652. doi: 10.1038/nbt.1883.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3571712/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n21572440\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnbt.1883\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=Full-length+transcriptome+assembly+from+RNA-Seq+data+without+a+reference+genome&amp;author=MG+Grabherr&amp;volume=29&amp;publication_year=2011&amp;pages=644-652&amp;pmid=21572440&amp;doi=10.1038/nbt.1883&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR63\">63. <span class=\"element-citation\">Grigoriev IV, et al.  MycoCosm portal: gearing up for 1000 fungal genomes. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2014;<span class=\"ref-vol\">42</span>:D699–D704. doi: 10.1093/nar/gkt1183.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3965089/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24297253\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgkt1183\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=MycoCosm+portal:+gearing+up+for+1000+fungal+genomes&amp;author=IV+Grigoriev&amp;volume=42&amp;publication_year=2014&amp;pages=D699-D704&amp;pmid=24297253&amp;doi=10.1093/nar/gkt1183&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR64\">64. <span class=\"mixed-citation\">Kuo, A., Bushnell, B. &#x00026; Grigoriev, I. V. in Ecological Genomics of Fungi (ed Martin, F.), Advances In Botanical Research, 1&#x02013;52 (Elsevier Academic Press, Cambridge, United Kingdom, 2014).</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR65\">65. <span class=\"mixed-citation\">Darling, A. E. et al. The design, implementation, and evaluation of mpiBLAST. <em>ClusterWorld Conference and Expo and the 4</em><sup><em>th</em></sup><em>International Conference on Linux Clusters</em> (The HPC Revolution, CA, USA, 2003).</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR66\">66. <span class=\"element-citation\">Miele V, et al.  High-quality sequence clustering guided by network topology and multiple alignment likelihood. <span><span class=\"ref-journal\">Bioinformatics. </span>2012;<span class=\"ref-vol\">28</span>:1078–1085. doi: 10.1093/bioinformatics/bts098.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n22368255\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbts098\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=High-quality+sequence+clustering+guided+by+network+topology+and+multiple+alignment+likelihood&amp;author=V+Miele&amp;volume=28&amp;publication_year=2012&amp;pages=1078-1085&amp;pmid=22368255&amp;doi=10.1093/bioinformatics/bts098&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR67\">67. <span class=\"element-citation\">L&#x000f6;ytynoja A, Goldman N. Phylogeny-aware Gap placement prevents errors in sequence alignment and evolutionary analysis. <span><span class=\"ref-journal\">Science. </span>2008;<span class=\"ref-vol\">320</span>:1632–1635. doi: 10.1126/science.1158395.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n18566285\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1126%2Fscience.1158395\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Phylogeny-aware+Gap+placement+prevents+errors+in+sequence+alignment+and+evolutionary+analysis&amp;author=A+L&#x000f6;ytynoja&amp;author=N+Goldman&amp;volume=320&amp;publication_year=2008&amp;pages=1632-1635&amp;pmid=18566285&amp;doi=10.1126/science.1158395&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR68\">68. <span class=\"element-citation\">Capella-Gutierrez S, Silla-Martinez JM, Gabaldon T. trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses. <span><span class=\"ref-journal\">Bioinformatics. </span>2009;<span class=\"ref-vol\">25</span>:1972–1973. doi: 10.1093/bioinformatics/btp348.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC2712344/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n19505945\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtp348\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=trimAl:+a+tool+for+automated+alignment+trimming+in+large-scale+phylogenetic+analyses&amp;author=S+Capella-Gutierrez&amp;author=JM+Silla-Martinez&amp;author=T+Gabaldon&amp;volume=25&amp;publication_year=2009&amp;pages=1972-1973&amp;pmid=19505945&amp;doi=10.1093/bioinformatics/btp348&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR69\">69. <span class=\"element-citation\">Price MN, Dehal PS, Arkin AP. FastTree: Computing large minimum evolution trees with profiles instead of a distance matrix. <span><span class=\"ref-journal\">Mol. Biol. Evol. </span>2009;<span class=\"ref-vol\">26</span>:1641–1650. doi: 10.1093/molbev/msp077.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC2693737/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n19377059\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fmolbev%2Fmsp077\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Biol.+Evol.&amp;title=FastTree:+Computing+large+minimum+evolution+trees+with+profiles+instead+of+a+distance+matrix&amp;author=MN+Price&amp;author=PS+Dehal&amp;author=AP+Arkin&amp;volume=26&amp;publication_year=2009&amp;pages=1641-1650&amp;pmid=19377059&amp;doi=10.1093/molbev/msp077&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR70\">70. <span class=\"mixed-citation\">L&#x000f6;ytynoja, A. In <em>Methods in molecular biology</em>, Vol. 1079, 155&#x02013;170 (Human Press, Clifton, N.J., 2014). [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24170401\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>]</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR71\">71. <span class=\"element-citation\">Stamatakis A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. <span><span class=\"ref-journal\">Bioinformatics. </span>2014;<span class=\"ref-vol\">30</span>:1312–1313. doi: 10.1093/bioinformatics/btu033.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3998144/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24451623\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtu033\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=RAxML+version+8:+a+tool+for+phylogenetic+analysis+and+post-analysis+of+large+phylogenies&amp;author=A+Stamatakis&amp;volume=30&amp;publication_year=2014&amp;pages=1312-1313&amp;pmid=24451623&amp;doi=10.1093/bioinformatics/btu033&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR72\">72. <span class=\"element-citation\">Wattam AR, et al.  PATRIC, the bacterial bioinformatics database and analysis resource. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2014;<span class=\"ref-vol\">42</span>:D581–D591. doi: 10.1093/nar/gkt1099.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3965095/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24225323\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgkt1099\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=PATRIC,+the+bacterial+bioinformatics+database+and+analysis+resource&amp;author=AR+Wattam&amp;volume=42&amp;publication_year=2014&amp;pages=D581-D591&amp;pmid=24225323&amp;doi=10.1093/nar/gkt1099&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR73\">73. <span class=\"element-citation\">Katoh K, Standley DM. MAFFT: multiple sequence alignment software version 7: improvements in performance and usability. <span><span class=\"ref-journal\">Mol. Biol. Evol. </span>2013;<span class=\"ref-vol\">30</span>:772–780. doi: 10.1093/molbev/mst010.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3603318/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n23329690\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fmolbev%2Fmst010\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Biol.+Evol.&amp;title=MAFFT:+multiple+sequence+alignment+software+version+7:+improvements+in+performance+and+usability&amp;author=K+Katoh&amp;author=DM+Standley&amp;volume=30&amp;publication_year=2013&amp;pages=772-780&amp;pmid=23329690&amp;doi=10.1093/molbev/mst010&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR74\">74. <span class=\"element-citation\">Cruickshank R. Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. <span><span class=\"ref-journal\">Mol. Biol. Evol. </span>2000;<span class=\"ref-vol\">17</span>:540–552. doi: 10.1093/oxfordjournals.molbev.a026334.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n10742046\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Foxfordjournals.molbev.a026334\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Biol.+Evol.&amp;title=Selection+of+conserved+blocks+from+multiple+alignments+for+their+use+in+phylogenetic+analysis&amp;author=R+Cruickshank&amp;volume=17&amp;publication_year=2000&amp;pages=540-552&amp;pmid=10742046&amp;doi=10.1093/oxfordjournals.molbev.a026334&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR75\">75. <span class=\"element-citation\">Stamatakis A. RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models. <span><span class=\"ref-journal\">Bioinformatics. </span>2006;<span class=\"ref-vol\">22</span>:2688e2690. doi: 10.1093/bioinformatics/btl446.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n16928733\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtl446\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=RAxML-VI-HPC:+maximum+likelihood-based+phylogenetic+analyses+with+thousands+of+taxa+and+mixed+models&amp;author=A+Stamatakis&amp;volume=22&amp;publication_year=2006&amp;pages=2688e2690&amp;pmid=16928733&amp;doi=10.1093/bioinformatics/btl446&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR76\">76. <span class=\"element-citation\">Sanderson MJ. r8s: inferring absolute rates of molecular evolution and divergence times in the absence of a molecular clock. <span><span class=\"ref-journal\">Bioinformatics. </span>2003;<span class=\"ref-vol\">19</span>:301–302. doi: 10.1093/bioinformatics/19.2.301.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n12538260\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2F19.2.301\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=r8s:+inferring+absolute+rates+of+molecular+evolution+and+divergence+times+in+the+absence+of+a+molecular+clock&amp;author=MJ+Sanderson&amp;volume=19&amp;publication_year=2003&amp;pages=301-302&amp;pmid=12538260&amp;doi=10.1093/bioinformatics/19.2.301&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR77\">77. <span class=\"element-citation\">Hibbett DS, Grimaldi D, Donoghue MJ. Fossil mushrooms from Miocene and Cretaceous ambers and the evolution of Homobasidiomycetes. <span><span class=\"ref-journal\">Am. J. Bot. </span>1997;<span class=\"ref-vol\">84</span>:981–991. doi: 10.2307/2446289.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n21708653\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.2307%2F2446289\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Am.+J.+Bot.&amp;title=Fossil+mushrooms+from+Miocene+and+Cretaceous+ambers+and+the+evolution+of+Homobasidiomycetes&amp;author=DS+Hibbett&amp;author=D+Grimaldi&amp;author=MJ+Donoghue&amp;volume=84&amp;publication_year=1997&amp;pages=981-991&amp;pmid=21708653&amp;doi=10.2307/2446289&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR78\">78. <span class=\"element-citation\">Lepage BA, Currah RS, Stockey RA, Rothwell GW. Fossil ectomycorrhizae from the Middle Eocene. <span><span class=\"ref-journal\">Am. J. Bot. </span>1997;<span class=\"ref-vol\">84</span>:410–412. doi: 10.2307/2446014.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n21708594\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.2307%2F2446014\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Am.+J.+Bot.&amp;title=Fossil+ectomycorrhizae+from+the+Middle+Eocene&amp;author=BA+Lepage&amp;author=RS+Currah&amp;author=RA+Stockey&amp;author=GW+Rothwell&amp;volume=84&amp;publication_year=1997&amp;pages=410-412&amp;pmid=21708594&amp;doi=10.2307/2446014&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR79\">79. <span class=\"element-citation\">Taylor TN, Hass H, Kerp H, Krings M, Hanlin RT. Perithecial ascomycetes from the 400 million year old Rhynie chert: an example of ancestral polymorphism. <span><span class=\"ref-journal\">Mycologia. </span>2005;<span class=\"ref-vol\">97</span>:269–285. doi: 10.1080/15572536.2006.11832862.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n16389979\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1080%2F15572536.2006.11832862\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mycologia&amp;title=Perithecial+ascomycetes+from+the+400+million+year+old+Rhynie+chert:+an+example+of+ancestral+polymorphism&amp;author=TN+Taylor&amp;author=H+Hass&amp;author=H+Kerp&amp;author=M+Krings&amp;author=RT+Hanlin&amp;volume=97&amp;publication_year=2005&amp;pages=269-285&amp;pmid=16389979&amp;doi=10.1080/15572536.2006.11832862&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR80\">80. <span class=\"element-citation\">Altschul SF, et al.  Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>1997;<span class=\"ref-vol\">25</span>:3389–3402. doi: 10.1093/nar/25.17.3389.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC146917/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n9254694\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2F25.17.3389\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Gapped+BLAST+and+PSI-BLAST:+a+new+generation+of+protein+database+search+programs&amp;author=SF+Altschul&amp;volume=25&amp;publication_year=1997&amp;pages=3389-3402&amp;pmid=9254694&amp;doi=10.1093/nar/25.17.3389&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR81\">81. <span class=\"element-citation\">Lombard V, Golaconda Ramulu H, Drula E, Coutinho PM, Henrissat B. The carbohydrate-active enzymes database (CAZy) in 2013. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2014;<span class=\"ref-vol\">42</span>:D490–D495. doi: 10.1093/nar/gkt1178.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3965031/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24270786\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgkt1178\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+carbohydrate-active+enzymes+database+(CAZy)+in+2013&amp;author=V+Lombard&amp;author=H+Golaconda+Ramulu&amp;author=E+Drula&amp;author=PM+Coutinho&amp;author=B+Henrissat&amp;volume=42&amp;publication_year=2014&amp;pages=D490-D495&amp;pmid=24270786&amp;doi=10.1093/nar/gkt1178&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR82\">82. <span class=\"element-citation\">Wu Y-C, Rasmussen MD, Bansal MS, Kellis M. TreeFix: statistically informed gene tree error correction using species trees. <span><span class=\"ref-journal\">Syst. Biol. </span>2013;<span class=\"ref-vol\">62</span>:110–120. doi: 10.1093/sysbio/sys076.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3526801/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n22949484\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fsysbio%2Fsys076\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Syst.+Biol.&amp;title=TreeFix:+statistically+informed+gene+tree+error+correction+using+species+trees&amp;author=Y-C+Wu&amp;author=MD+Rasmussen&amp;author=MS+Bansal&amp;author=M+Kellis&amp;volume=62&amp;publication_year=2013&amp;pages=110-120&amp;pmid=22949484&amp;doi=10.1093/sysbio/sys076&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR83\">83. <span class=\"element-citation\">Math&#x000e9; C, Fawal N, Roux C, Dunand C. <em>In silico</em> definition of new ligninolytic peroxidase sub-classes in fungi and putative relation to fungal life style. <span><span class=\"ref-journal\">Sci. Rep. </span>2019;<span class=\"ref-vol\">9</span>:20373. doi: 10.1038/s41598-019-56774-4.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC6937255/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n31889110\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41598-019-56774-4\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=In+silico+definition+of+new+ligninolytic+peroxidase+sub-classes+in+fungi+and+putative+relation+to+fungal+life+style&amp;author=C+Math&#x000e9;&amp;author=N+Fawal&amp;author=C+Roux&amp;author=C+Dunand&amp;volume=9&amp;publication_year=2019&amp;pages=20373&amp;pmid=31889110&amp;doi=10.1038/s41598-019-56774-4&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR84\">84. <span class=\"element-citation\">Domazet-Lo&#x00161;o T, Brajkovi&#x00107; J, Tautz D. A phylostratigraphy approach to uncover the genomic history of major adaptations in metazoan lineages. <span><span class=\"ref-journal\">Trends Genet. </span>2007;<span class=\"ref-vol\">23</span>:533–539. doi: 10.1016/j.tig.2007.08.014.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n18029048\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.tig.2007.08.014\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Trends+Genet.&amp;title=A+phylostratigraphy+approach+to+uncover+the+genomic+history+of+major+adaptations+in+metazoan+lineages&amp;author=T+Domazet-Lo&#x00161;o&amp;author=J+Brajkovi&#x00107;&amp;author=D+Tautz&amp;volume=23&amp;publication_year=2007&amp;pages=533-539&amp;pmid=18029048&amp;doi=10.1016/j.tig.2007.08.014&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR85\">85. <span class=\"element-citation\">Sima FA, Waterhouse RM, Ioannidis P, Kriventseva EV, Zdobnov EM. Genome analysis BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs. <span><span class=\"ref-journal\">Bioinformatics. </span>2015;<span class=\"ref-vol\">31</span>:3210–3212. doi: 10.1093/bioinformatics/btv351.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n26059717\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtv351\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Genome+analysis+BUSCO:+assessing+genome+assembly+and+annotation+completeness+with+single-copy+orthologs&amp;author=FA+Sima&amp;author=RM+Waterhouse&amp;author=P+Ioannidis&amp;author=EV+Kriventseva&amp;author=EM+Zdobnov&amp;volume=31&amp;publication_year=2015&amp;pages=3210-3212&amp;pmid=26059717&amp;doi=10.1093/bioinformatics/btv351&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR86\">86. <span class=\"mixed-citation\">Konietschke, F. Simultane Konfidenzintervalle fuer nichtparametrische relative Kontrasteffekte. PhD thesis, University of Goettingen (2009).</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR87\">87. <span class=\"mixed-citation\">Pinheiro, J., Bates, D., DebRoy, S., Sarkar, D. &#x00026; R Core Team. nlme: Linear and Nonlinear Mixed Effects Models<em>. R package version 3.1&#x02013;137</em><a href=\"https://CRAN.R-project.org/package=nlme\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://CRAN.R-project.org/package=nlme</a> (2018).</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR88\">88. <span class=\"mixed-citation\">Auguie, B. <em>egg:</em><a href=\"https://cran.r-project.org/web/packages/egg/index.html\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://cran.r-project.org/web/packages/egg/index.html</a> (2017).</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR89\">89. <span class=\"element-citation\">Yu G, Smith DK, Zhu H, Guan Y, Lam TT-Y. ggtree: an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data. <span><span class=\"ref-journal\">Methods Ecol. Evol. </span>2017;<span class=\"ref-vol\">8</span>:28–36. doi: 10.1111/2041-210X.12628.</span> [<a href=\"//dx.doi.org/10.1111%2F2041-210X.12628\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Methods+Ecol.+Evol.&amp;title=ggtree:+an+R+package+for+visualization+and+annotation+of+phylogenetic+trees+with+their+covariates+and+other+associated+data&amp;author=G+Yu&amp;author=DK+Smith&amp;author=H+Zhu&amp;author=Y+Guan&amp;author=TT-Y+Lam&amp;volume=8&amp;publication_year=2017&amp;pages=28-36&amp;doi=10.1111/2041-210X.12628&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR90\">90. <span class=\"element-citation\">Wickham H.  <span class=\"ref-journal\">ggplot2: Elegant Graphics for Data Analysis.</span> New York: Springer; 2009.  <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=ggplot2:+Elegant+Graphics+for+Data+Analysis&amp;author=H+Wickham&amp;publication_year=2009&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR91\">91. <span class=\"mixed-citation\">Kassambara, A. ggpubr R package: ggplot2-based publication ready plots<em>. R package version 0.2.5.999</em><a href=\"https://rpkgs.datanovia.com/ggpubr/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://rpkgs.datanovia.com/ggpubr/</a> (2020).</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR92\">92. <span class=\"element-citation\">Paradis E, Schliep K. ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R. <span><span class=\"ref-journal\">Bioinformatics. </span>2018;<span class=\"ref-vol\">35</span>:526–528. doi: 10.1093/bioinformatics/bty633.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30016406\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbty633\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=ape+5.0:+an+environment+for+modern+phylogenetics+and+evolutionary+analyses+in+R&amp;author=E+Paradis&amp;author=K+Schliep&amp;volume=35&amp;publication_year=2018&amp;pages=526-528&amp;doi=10.1093/bioinformatics/bty633&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR93\">93. <span class=\"mixed-citation\">Mensy, F. Detecting the effect of biological categories on genome composition. <a href=\"https://github.com/fantin-mesny/Effect-Of-Biological-Categories-On-Genomes-Composition\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://github.com/fantin-mesny/Effect-Of-Biological-Categories-On-Genomes-Composition</a> (2020).</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR94\">94. <span class=\"mixed-citation\">Oksanen, J. et al. Vegan: community ecology package. <em>R package version 2.5&#x02013;6</em><a href=\"https://CRAN.R-project.org/package=vegan\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://CRAN.R-project.org/package=vegan</a> (2019).</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR95\">95. <span class=\"mixed-citation\">Herv&#x000e9;, M. RVAideMemoire: testing and plotting procedures for biostatistics. <em>R package version 0.9-75</em><a href=\"https://CRAN.R-project.org/package=RVAideMemoire\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://CRAN.R-project.org/package=RVAideMemoire</a> (2020).</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR96\">96. <span class=\"element-citation\">Castanera R, et al.  Transposable elements versus the fungal genome: Impact on whole-genome architecture and transcriptional profiles. <span><span class=\"ref-journal\">PLoS Genet. </span>2016;<span class=\"ref-vol\">12</span>:e1006108. doi: 10.1371/journal.pgen.1006108.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC4905642/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n27294409\" ref=\"reftype=pubmed&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pgen.1006108\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Transposable+elements+versus+the+fungal+genome:+Impact+on+whole-genome+architecture+and+transcriptional+profiles&amp;author=R+Castanera&amp;volume=12&amp;publication_year=2016&amp;pages=e1006108&amp;pmid=27294409&amp;doi=10.1371/journal.pgen.1006108&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR97\">97. <span class=\"mixed-citation\">Ploner, A. Heatplus: Heatmaps with row and/or column covariates and colored clusters. R package version 2.34.0, <a href=\"https://github.com/alexploner/Heatplus\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7550596&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://github.com/alexploner/Heatplus</a> (2020).</span></div></div><br>', NULL, 6);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(10, 'pubmed', '32916905', 'Characterization of the CAZy Repertoire from the Marine-Derived Fungus  in Relation to Saline Conditions.', 'Even if the ocean represents a large part of Earth\'s surface, only a few studies describe marine-derived fungi compared to their terrestrial homologues. In this ecosystem, marine-derived fungi have had to adapt to the salinity and to the plant biomass composition. This articles studies the growth of five marine isolates and the tuning of lignocellulolytic activities under different conditions, including the salinity. A de novo transcriptome sequencing and assembly were used in combination with a proteomic approach to characterize the Carbohydrate Active Enzymes (CAZy) repertoire of one of these strains. Following these approaches,  was selected for its adapted growth on xylan in saline conditions, its high xylanase activity, and its improved laccase activities in seagrass-containing cultures with salt. De novo transcriptome sequencing and assembly indicated the presence of 51 putative lignocellulolytic enzymes. Its secretome composition was studied in detail when the fungus was grown on either a terrestrial or a marine substrate, under saline and non-saline conditions. Proteomic analysis of the four  secretomes revealed a minimal suite of extracellular enzymes for plant biomass degradation and highlighted potential enzyme targets to be further studied for their adaptation to salts and for potential biotechnological applications.', 2020, 'Marine drugs', 'PMC7551824', 2, '<h1 class=\"content-title\">Characterization of the CAZy Repertoire from the Marine-Derived Fungus <em>Stemphylium lucomagnoense</em> in Relation to Saline Conditions</h1><br><span class=\"contrib-group fm-author\"><a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Ben%20Ali%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\" class=\"affpopup\" co-rid=\"_co_idm140580356941920\" co-class=\"co-affbox\">Wissal Ben Ali</a>,<sup>1,</sup><sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Navarro%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\" class=\"affpopup\" co-rid=\"_co_idm140580356925872\" co-class=\"co-affbox\">David Navarro</a>,<sup>1,</sup><sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Kumar%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\" class=\"affpopup\" co-rid=\"_co_idm140580356922896\" co-class=\"co-affbox\">Abhishek Kumar</a>,<sup>4,</sup><sup>5</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Drula%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\" class=\"affpopup\" co-rid=\"_co_idm140580356919056\" co-class=\"co-affbox\">Elodie Drula</a>,<sup>1,</sup><sup>6,</sup><sup>7</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Turb%26%23x000e9%3B-Doan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\" class=\"affpopup\" co-rid=\"_co_idm140580356915248\" co-class=\"co-affbox\">Annick Turb&#x000e9;-Doan</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Correia%20LO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\" class=\"affpopup\" co-rid=\"_co_idm140580356913104\" co-class=\"co-affbox\">Lydie Oliveira Correia</a>,<sup>8</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Baumberger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\" class=\"affpopup\" co-rid=\"_co_idm140580356910896\" co-class=\"co-affbox\">St&#x000e9;phanie Baumberger</a>,<sup>9</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Bertrand%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\" class=\"affpopup\" co-rid=\"_co_idm140580356908640\" co-class=\"co-affbox\">Emmanuel Bertrand</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Faulds%20CB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\" class=\"affpopup\" co-rid=\"_co_idm140580356906304\" co-class=\"co-affbox\">Craig B. Faulds</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Henrissat%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\" class=\"affpopup\" co-rid=\"_co_idm140580356904016\" co-class=\"co-affbox\">Bernard Henrissat</a>,<sup>6,</sup><sup>7,</sup><sup>10</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Sciara%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\" class=\"affpopup\" co-rid=\"_co_idm140580356899840\" co-class=\"co-affbox\">Giuliano Sciara</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Mechichi%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\" class=\"affpopup\" co-rid=\"_co_idm140580356897552\" co-class=\"co-affbox\">Tahar Mechichi</a>,<sup>2</sup> and  <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Record%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\" class=\"affpopup\" co-rid=\"_co_idm140580356895264\" co-class=\"co-affbox\">Eric Record</a><sup>1,</sup><sup>*</sup></span><span style=\"display:none\" class=\"contrib-group aff-tip\"><span id=\"_co_idm140580356941920\"><h3 class=\"no_margin\">Wissal Ben Ali</h3><p><sup>1</sup>Biodiversit&#x000e9; et Biotechnologie Fongiques, Aix-Marseille Universit&#x000e9;, INRAE, UMR1163, 13288 Marseille, France; <a href=\"mailto:dev@null\" data-email=\"rf.uma-vinu.ute@ILA-NEB.lassiw\" class=\"oemail\">rf.uma-vinu.ute@ILA-NEB.lassiw</a> (W.B.A.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@orravan.divad\" class=\"oemail\">rf.earni@orravan.divad</a> (D.N.); <a href=\"mailto:dev@null\" data-email=\"rf.srm-vinu.bmfa@alurd.eidole\" class=\"oemail\">rf.srm-vinu.bmfa@alurd.eidole</a> (E.D.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@naod.kcinna\" class=\"oemail\">rf.earni@naod.kcinna</a> (A.T.-D.); <a href=\"mailto:dev@null\" data-email=\"rF.umA-vinU@dnartreB.leunammE\" class=\"oemail\">rF.umA-vinU@dnartreB.leunammE</a> (E.B.); <a href=\"mailto:dev@null\" data-email=\"rF.umA-vinU@sdluaF.giarC\" class=\"oemail\">rF.umA-vinU@sdluaF.giarC</a> (C.B.F.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@araics.onailuig\" class=\"oemail\">rf.earni@araics.onailuig</a> (G.S.)</p><p><sup>2</sup>Laboratoire de Biochimie et de G&#x000e9;nie Enzymatique des Lipases, Ecole Nationale d&#x02019;Ing&#x000e9;nieurs de Sfax, Universit&#x000e9; de Sfax, Sfax 3029, Tunisia; <a href=\"mailto:dev@null\" data-email=\"nt.unr.sine@ihcihcem.rahat\" class=\"oemail\">nt.unr.sine@ihcihcem.rahat</a></p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Ben%20Ali%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\">Wissal Ben Ali</a></span></span><span id=\"_co_idm140580356925872\"><h3 class=\"no_margin\">David Navarro</h3><p><sup>1</sup>Biodiversit&#x000e9; et Biotechnologie Fongiques, Aix-Marseille Universit&#x000e9;, INRAE, UMR1163, 13288 Marseille, France; <a href=\"mailto:dev@null\" data-email=\"rf.uma-vinu.ute@ILA-NEB.lassiw\" class=\"oemail\">rf.uma-vinu.ute@ILA-NEB.lassiw</a> (W.B.A.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@orravan.divad\" class=\"oemail\">rf.earni@orravan.divad</a> (D.N.); <a href=\"mailto:dev@null\" data-email=\"rf.srm-vinu.bmfa@alurd.eidole\" class=\"oemail\">rf.srm-vinu.bmfa@alurd.eidole</a> (E.D.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@naod.kcinna\" class=\"oemail\">rf.earni@naod.kcinna</a> (A.T.-D.); <a href=\"mailto:dev@null\" data-email=\"rF.umA-vinU@dnartreB.leunammE\" class=\"oemail\">rF.umA-vinU@dnartreB.leunammE</a> (E.B.); <a href=\"mailto:dev@null\" data-email=\"rF.umA-vinU@sdluaF.giarC\" class=\"oemail\">rF.umA-vinU@sdluaF.giarC</a> (C.B.F.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@araics.onailuig\" class=\"oemail\">rf.earni@araics.onailuig</a> (G.S.)</p><p><sup>3</sup>INRAE, Aix-Marseille Universit&#x000e9;, UMR1163, CIRM-CF, 13288 Marseille, France</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Navarro%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\">David Navarro</a></span></span><span id=\"_co_idm140580356922896\"><h3 class=\"no_margin\">Abhishek Kumar</h3><p><sup>4</sup>Institute of Bioinformatics, International Technology Park, Bangalore 560066, India; <a href=\"mailto:dev@null\" data-email=\"moc.liamg@ramukkehsihba.kehsihba\" class=\"oemail\">moc.liamg@ramukkehsihba.kehsihba</a></p><p><sup>5</sup>Manipal Academy of Higher Education, Manipal 576104, India</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Kumar%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\">Abhishek Kumar</a></span></span><span id=\"_co_idm140580356919056\"><h3 class=\"no_margin\">Elodie Drula</h3><p><sup>1</sup>Biodiversit&#x000e9; et Biotechnologie Fongiques, Aix-Marseille Universit&#x000e9;, INRAE, UMR1163, 13288 Marseille, France; <a href=\"mailto:dev@null\" data-email=\"rf.uma-vinu.ute@ILA-NEB.lassiw\" class=\"oemail\">rf.uma-vinu.ute@ILA-NEB.lassiw</a> (W.B.A.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@orravan.divad\" class=\"oemail\">rf.earni@orravan.divad</a> (D.N.); <a href=\"mailto:dev@null\" data-email=\"rf.srm-vinu.bmfa@alurd.eidole\" class=\"oemail\">rf.srm-vinu.bmfa@alurd.eidole</a> (E.D.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@naod.kcinna\" class=\"oemail\">rf.earni@naod.kcinna</a> (A.T.-D.); <a href=\"mailto:dev@null\" data-email=\"rF.umA-vinU@dnartreB.leunammE\" class=\"oemail\">rF.umA-vinU@dnartreB.leunammE</a> (E.B.); <a href=\"mailto:dev@null\" data-email=\"rF.umA-vinU@sdluaF.giarC\" class=\"oemail\">rF.umA-vinU@sdluaF.giarC</a> (C.B.F.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@araics.onailuig\" class=\"oemail\">rf.earni@araics.onailuig</a> (G.S.)</p><p><sup>6</sup>Architecture et Fonction des Macromol&#x000e9;cules Biologiques, Centre National de la Recherche Scientifique, Aix-Marseille Universit&#x000e9;, 13288 Marseille, France; <a href=\"mailto:dev@null\" data-email=\"rF.srM-vinU.bmfA@tassirneH.dranreB\" class=\"oemail\">rF.srM-vinU.bmfA@tassirneH.dranreB</a></p><p><sup>7</sup>USC AFMB, Institut National de Recherche Agronomique, 13288 Marseille, France</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Drula%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\">Elodie Drula</a></span></span><span id=\"_co_idm140580356915248\"><h3 class=\"no_margin\">Annick Turb&#x000e9;-Doan</h3><p><sup>1</sup>Biodiversit&#x000e9; et Biotechnologie Fongiques, Aix-Marseille Universit&#x000e9;, INRAE, UMR1163, 13288 Marseille, France; <a href=\"mailto:dev@null\" data-email=\"rf.uma-vinu.ute@ILA-NEB.lassiw\" class=\"oemail\">rf.uma-vinu.ute@ILA-NEB.lassiw</a> (W.B.A.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@orravan.divad\" class=\"oemail\">rf.earni@orravan.divad</a> (D.N.); <a href=\"mailto:dev@null\" data-email=\"rf.srm-vinu.bmfa@alurd.eidole\" class=\"oemail\">rf.srm-vinu.bmfa@alurd.eidole</a> (E.D.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@naod.kcinna\" class=\"oemail\">rf.earni@naod.kcinna</a> (A.T.-D.); <a href=\"mailto:dev@null\" data-email=\"rF.umA-vinU@dnartreB.leunammE\" class=\"oemail\">rF.umA-vinU@dnartreB.leunammE</a> (E.B.); <a href=\"mailto:dev@null\" data-email=\"rF.umA-vinU@sdluaF.giarC\" class=\"oemail\">rF.umA-vinU@sdluaF.giarC</a> (C.B.F.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@araics.onailuig\" class=\"oemail\">rf.earni@araics.onailuig</a> (G.S.)</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Turb%26%23x000e9%3B-Doan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\">Annick Turb&#x000e9;-Doan</a></span></span><span id=\"_co_idm140580356913104\"><h3 class=\"no_margin\">Lydie Oliveira Correia</h3><p><sup>8</sup>AgroParisTech, Micalis Institute, PAPPSO, Universit&#x000e9; Paris-Saclay, INRAE, 78350 Jouy-en-Josas, France; <a href=\"mailto:dev@null\" data-email=\"rf.earni@aierroc-arievilo.eidyl\" class=\"oemail\">rf.earni@aierroc-arievilo.eidyl</a></p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Correia%20LO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\">Lydie Oliveira Correia</a></span></span><span id=\"_co_idm140580356910896\"><h3 class=\"no_margin\">St&#x000e9;phanie Baumberger</h3><p><sup>9</sup>Institut Jean-Pierre Bourgin, INRAE, AgroParisTech, CNRS, Universit&#x000e9; Paris-Saclay, 78000 Versailles, France; <a href=\"mailto:dev@null\" data-email=\"rf.earni@regrebmuab.einahpets\" class=\"oemail\">rf.earni@regrebmuab.einahpets</a></p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Baumberger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\">St&#x000e9;phanie Baumberger</a></span></span><span id=\"_co_idm140580356908640\"><h3 class=\"no_margin\">Emmanuel Bertrand</h3><p><sup>1</sup>Biodiversit&#x000e9; et Biotechnologie Fongiques, Aix-Marseille Universit&#x000e9;, INRAE, UMR1163, 13288 Marseille, France; <a href=\"mailto:dev@null\" data-email=\"rf.uma-vinu.ute@ILA-NEB.lassiw\" class=\"oemail\">rf.uma-vinu.ute@ILA-NEB.lassiw</a> (W.B.A.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@orravan.divad\" class=\"oemail\">rf.earni@orravan.divad</a> (D.N.); <a href=\"mailto:dev@null\" data-email=\"rf.srm-vinu.bmfa@alurd.eidole\" class=\"oemail\">rf.srm-vinu.bmfa@alurd.eidole</a> (E.D.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@naod.kcinna\" class=\"oemail\">rf.earni@naod.kcinna</a> (A.T.-D.); <a href=\"mailto:dev@null\" data-email=\"rF.umA-vinU@dnartreB.leunammE\" class=\"oemail\">rF.umA-vinU@dnartreB.leunammE</a> (E.B.); <a href=\"mailto:dev@null\" data-email=\"rF.umA-vinU@sdluaF.giarC\" class=\"oemail\">rF.umA-vinU@sdluaF.giarC</a> (C.B.F.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@araics.onailuig\" class=\"oemail\">rf.earni@araics.onailuig</a> (G.S.)</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Bertrand%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\">Emmanuel Bertrand</a></span></span><span id=\"_co_idm140580356906304\"><h3 class=\"no_margin\">Craig B. Faulds</h3><p><sup>1</sup>Biodiversit&#x000e9; et Biotechnologie Fongiques, Aix-Marseille Universit&#x000e9;, INRAE, UMR1163, 13288 Marseille, France; <a href=\"mailto:dev@null\" data-email=\"rf.uma-vinu.ute@ILA-NEB.lassiw\" class=\"oemail\">rf.uma-vinu.ute@ILA-NEB.lassiw</a> (W.B.A.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@orravan.divad\" class=\"oemail\">rf.earni@orravan.divad</a> (D.N.); <a href=\"mailto:dev@null\" data-email=\"rf.srm-vinu.bmfa@alurd.eidole\" class=\"oemail\">rf.srm-vinu.bmfa@alurd.eidole</a> (E.D.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@naod.kcinna\" class=\"oemail\">rf.earni@naod.kcinna</a> (A.T.-D.); <a href=\"mailto:dev@null\" data-email=\"rF.umA-vinU@dnartreB.leunammE\" class=\"oemail\">rF.umA-vinU@dnartreB.leunammE</a> (E.B.); <a href=\"mailto:dev@null\" data-email=\"rF.umA-vinU@sdluaF.giarC\" class=\"oemail\">rF.umA-vinU@sdluaF.giarC</a> (C.B.F.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@araics.onailuig\" class=\"oemail\">rf.earni@araics.onailuig</a> (G.S.)</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Faulds%20CB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\">Craig B. Faulds</a></span></span><span id=\"_co_idm140580356904016\"><h3 class=\"no_margin\">Bernard Henrissat</h3><p><sup>6</sup>Architecture et Fonction des Macromol&#x000e9;cules Biologiques, Centre National de la Recherche Scientifique, Aix-Marseille Universit&#x000e9;, 13288 Marseille, France; <a href=\"mailto:dev@null\" data-email=\"rF.srM-vinU.bmfA@tassirneH.dranreB\" class=\"oemail\">rF.srM-vinU.bmfA@tassirneH.dranreB</a></p><p><sup>7</sup>USC AFMB, Institut National de Recherche Agronomique, 13288 Marseille, France</p><p><sup>10</sup>Department of Biological Sciences, King Abdulaziz University, Jeddah 21589, Saudi Arabia</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Henrissat%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\">Bernard Henrissat</a></span></span><span id=\"_co_idm140580356899840\"><h3 class=\"no_margin\">Giuliano Sciara</h3><p><sup>1</sup>Biodiversit&#x000e9; et Biotechnologie Fongiques, Aix-Marseille Universit&#x000e9;, INRAE, UMR1163, 13288 Marseille, France; <a href=\"mailto:dev@null\" data-email=\"rf.uma-vinu.ute@ILA-NEB.lassiw\" class=\"oemail\">rf.uma-vinu.ute@ILA-NEB.lassiw</a> (W.B.A.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@orravan.divad\" class=\"oemail\">rf.earni@orravan.divad</a> (D.N.); <a href=\"mailto:dev@null\" data-email=\"rf.srm-vinu.bmfa@alurd.eidole\" class=\"oemail\">rf.srm-vinu.bmfa@alurd.eidole</a> (E.D.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@naod.kcinna\" class=\"oemail\">rf.earni@naod.kcinna</a> (A.T.-D.); <a href=\"mailto:dev@null\" data-email=\"rF.umA-vinU@dnartreB.leunammE\" class=\"oemail\">rF.umA-vinU@dnartreB.leunammE</a> (E.B.); <a href=\"mailto:dev@null\" data-email=\"rF.umA-vinU@sdluaF.giarC\" class=\"oemail\">rF.umA-vinU@sdluaF.giarC</a> (C.B.F.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@araics.onailuig\" class=\"oemail\">rf.earni@araics.onailuig</a> (G.S.)</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Sciara%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\">Giuliano Sciara</a></span></span><span id=\"_co_idm140580356897552\"><h3 class=\"no_margin\">Tahar Mechichi</h3><p><sup>2</sup>Laboratoire de Biochimie et de G&#x000e9;nie Enzymatique des Lipases, Ecole Nationale d&#x02019;Ing&#x000e9;nieurs de Sfax, Universit&#x000e9; de Sfax, Sfax 3029, Tunisia; <a href=\"mailto:dev@null\" data-email=\"nt.unr.sine@ihcihcem.rahat\" class=\"oemail\">nt.unr.sine@ihcihcem.rahat</a></p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Mechichi%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\">Tahar Mechichi</a></span></span><span id=\"_co_idm140580356895264\"><h3 class=\"no_margin\">Eric Record</h3><p><sup>1</sup>Biodiversit&#x000e9; et Biotechnologie Fongiques, Aix-Marseille Universit&#x000e9;, INRAE, UMR1163, 13288 Marseille, France; <a href=\"mailto:dev@null\" data-email=\"rf.uma-vinu.ute@ILA-NEB.lassiw\" class=\"oemail\">rf.uma-vinu.ute@ILA-NEB.lassiw</a> (W.B.A.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@orravan.divad\" class=\"oemail\">rf.earni@orravan.divad</a> (D.N.); <a href=\"mailto:dev@null\" data-email=\"rf.srm-vinu.bmfa@alurd.eidole\" class=\"oemail\">rf.srm-vinu.bmfa@alurd.eidole</a> (E.D.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@naod.kcinna\" class=\"oemail\">rf.earni@naod.kcinna</a> (A.T.-D.); <a href=\"mailto:dev@null\" data-email=\"rF.umA-vinU@dnartreB.leunammE\" class=\"oemail\">rF.umA-vinU@dnartreB.leunammE</a> (E.B.); <a href=\"mailto:dev@null\" data-email=\"rF.umA-vinU@sdluaF.giarC\" class=\"oemail\">rF.umA-vinU@sdluaF.giarC</a> (C.B.F.); <a href=\"mailto:dev@null\" data-email=\"rf.earni@araics.onailuig\" class=\"oemail\">rf.earni@araics.onailuig</a> (G.S.)</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n?term=Record%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=32916905\">Eric Record</a></span></span></span></span><span class=\"fm-panel half_rhythm\"><span class=\"togglers<br><span id=\"idm140580391459968\" lang=\"en\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140580391459968title\">Abstract</h2><!--article-meta--><span><p id=\"__p1\" class=\"p p-first-last\">Even if the ocean represents a large part of Earth&#x02019;s surface, only a few studies describe marine-derived fungi compared to their terrestrial homologues. In this ecosystem, marine-derived fungi have had to adapt to the salinity and to the plant biomass composition. This articles studies the growth of five marine isolates and the tuning of lignocellulolytic activities under different conditions, including the salinity. A de novo transcriptome sequencing and assembly were used in combination with a proteomic approach to characterize the Carbohydrate Active Enzymes (CAZy) repertoire of one of these strains. Following these approaches, <em>Stemphylium lucomagnoense</em> was selected for its adapted growth on xylan in saline conditions, its high xylanase activity, and its improved laccase activities in seagrass-containing cultures with salt. De novo transcriptome sequencing and assembly indicated the presence of 51 putative lignocellulolytic enzymes. Its secretome composition was studied in detail when the fungus was grown on either a terrestrial or a marine substrate, under saline and non-saline conditions. Proteomic analysis of the four <em>S. lucomagnoense</em> secretomes revealed a minimal suite of extracellular enzymes for plant biomass degradation and highlighted potential enzyme targets to be further studied for their adaptation to salts and for potential biotechnological applications.</p></span><span class=\"sec\"><strong class=\"kwd-title\">Keywords: </strong><span class=\"kwd-text\">marine-derived fungus, <em>Stemphylium lucomagnoense</em>, saline adaptation, lignocellulose-degrading enzymes, secretome</span></span></span><span id=\"s\n1-marinedrugs-18-00461\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\n1-marinedrugs-18-00461title\">1. Introduction</h2><p id=\"__p2\" class=\"p p-first\">Oceans, which represent approximately 70.8% of Earth&#x02019;s surface, are not characterized by a high diversity of living organisms when compared with lands. Indeed, Grosberg et al. [<a href=\"#B1-marinedrugs-18-00461\" rid=\"B1-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514707\">1</a>] reported that over the 1.5 million known species of macroscopic organisms on Earth, about 15% of species are found in oceans, 5% in freshwater, and a larger portion (approximately 80%) in terrestrial environments. Concerning the fungal kingdom, the same picture can be drawn, as about 1000 fungal species have been found in oceans and in freshwater, while the terrestrial habitats would account for more than 10<sup>6</sup> different species [<a href=\"#B1-marinedrugs-18-00461\" rid=\"B1-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514721\">1</a>]. A few studies have focused on the exploration of marine fungal diversity, and the number of described species is about a few hundred&#x02014;Kohlmeyer et al. being one of the first pioneers in their isolation and identification [<a href=\"#B2-marinedrugs-18-00461\" rid=\"B2-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514729\">2</a>]. Many fungi that are found in oceans are also found in terrestrial environments, e.g., <em>Fusarium</em>, <em>Aspergillus</em>, <em>Trichoderma</em>, or <em>Penicillium</em> [<a href=\"#B3-marinedrugs-18-00461\" rid=\"B3-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514673\">3</a>]. This statement was illustrated by the work of Schoch et al. (2009), showing by a phylogenetic approach, including obligate marine species within the Dothideomycetes, that several have recently transitioned from terrestrial ancestors and that the corresponding transitions have occurred for multiple cases [<a href=\"#B4-marinedrugs-18-00461\" rid=\"B4-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514684\">4</a>]. For this reason, for fungi isolated from aquatic environments, the term &#x0201c;marine-derived fungi&#x0201d; is often preferred and used in the literature, as their classification is not always well-established, although many efforts were made over the years [<a href=\"#B5-marinedrugs-18-00461\" rid=\"B5-marinedrugs-18-00461\" class=\" bibr popnode\">5</a>]. From an ecological perspective, marine-derived fungi have been classified as obligate marine fungi, for those growing exclusively in a marine habitat, and facultative marine fungi, for those isolated from the freshwater or terrestrial origin, and also from the marine environment. [<a href=\"#B6-marinedrugs-18-00461\" rid=\"B6-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514734\">6</a>,<a href=\"#B7-marinedrugs-18-00461\" rid=\"B7-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514679\">7</a>]. Marine-derived fungi have been shown to be present in various habitats, such as decayed mangrove woods, algae, salt marshes, seagrasses, deep-sea sediments, sponges, and fishes [<a href=\"#B8-marinedrugs-18-00461\" rid=\"B8-marinedrugs-18-00461\" class=\" bibr popnode\">8</a>]. Within these marine environments, fungal communities have adapted to diverse environmental conditions, very different from terrestrial ones, and notably high salinity, alkaline pH, low temperature, and eventually high pressure. For such a reason, their enzymes are expected to be of great interest for more profitable and ecological industrial bioprocesses, operated, for example, in non-pure waters at neutral or basic pH [<a href=\"#B9-marinedrugs-18-00461\" rid=\"B9-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514723\">9</a>]. As such, marine fungi have been screened for retrieving original enzymes among amylases, chitinases, cellulases, xylanases, laccases, and many other potential biocatalysts [<a href=\"#B10-marinedrugs-18-00461\" rid=\"B10-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514700\">10</a>]. In general, it was showed that their biochemical properties differ from those of catalysts isolated from corresponding strains isolated on lands, for instance, presenting activity in alkaline, low temperature, and saline conditions [<a href=\"#B10-marinedrugs-18-00461\" rid=\"B10-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514717\">10</a>]. Marine-derived enzymes, as already assessed for their homologues from terrestrial fungi, could be of great interest in industrial sectors, such as the pulp and paper, tanning, and textile industries, whose processes involve high saline and alkaline conditions.</p><p id=\"__p3\">To survive under saline conditions, marine-derived fungi have likely developed physiological adaptations facilitating cellular homeostasis in these environments and secreted enzymatic machineries, active on biomass in marine habitats [<a href=\"#B11-marinedrugs-18-00461\" rid=\"B11-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514735\">11</a>]. However, only a few studies describe the potential of marine-derived fungi to produce lignocellulolytic enzymes, which are key players of organic matter degradation in the sea [<a href=\"#B10-marinedrugs-18-00461\" rid=\"B10-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514653\">10</a>,<a href=\"#B11-marinedrugs-18-00461\" rid=\"B11-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514704\">11</a>,<a href=\"#B12-marinedrugs-18-00461\" rid=\"B12-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514691\">12</a>]. A few of these strains have been studied at the genomic level, such as <em>Scopulariopsis brevicaulis</em> [<a href=\"#B13-marinedrugs-18-00461\" rid=\"B13-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514716\">13</a>], <em>Cadophora malorum</em> [<a href=\"#B14-marinedrugs-18-00461\" rid=\"B14-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514683\">14</a>] <em>Calcarisporium</em> sp., and <em>Pestalotiopsis</em> sp. [<a href=\"#B15-marinedrugs-18-00461\" rid=\"B15-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514693\">15</a>]. For instance, it has been suggested that the marine-derived fungi <em>Pestalotiopsis</em> sp. and <em>S. brevicaulis</em> have conserved a metabolism relying on the breakdown of terrestrial plant biomass, while <em>Calcarisporium</em> sp. has adopted a different enzyme repertoire specialized into algal and animal biomass degradation. The latter reflects a lifestyle-oriented either towards parasitism or endophytic growth, or towards the necrotrophic use of algae or animals.</p><p id=\"__p4\">Few studies tried to gain insight into the adaptation of fungal enzymatic cocktails to degrade either terrestrial or marine complex carbon sources in saline conditions. Within the context of plant biomass biorefinery, wheat straw has already been used as a model terrestrial substrate to induce the production of fungal ligninolytic enzymes and their identification by a proteomic approach [<a href=\"#B16-marinedrugs-18-00461\" rid=\"B16-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514715\">16</a>]. As a model for marine substrate, we selected is <em>Cymodocea nodosa</em>, an obligate underwater seagrass belonging to the family <em>Cymodoceaceae</em>, which plays essential roles in nutrient cycles, in preventing coastline erosion, and in providing habitats and hatcheries to marine life [<a href=\"#B17-marinedrugs-18-00461\" rid=\"B17-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514648\">17</a>]. It naturally grows in Mediterranean coasts, as well as in the sites where we collected the five fungal isolates of our study. Finally, although micro-organisms drive the marine carbon cycle and subtle equilibria exist between microbial metabolism and carbon sequestration in coastal ecosystems [<a href=\"#B18-marinedrugs-18-00461\" rid=\"B18-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514694\">18</a>], only a few examples can be found in the literature addressing the association of fungi to coastal plants [<a href=\"#B19-marinedrugs-18-00461\" rid=\"B19-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514676\">19</a>], genomic analysis of marine-derived fungi [<a href=\"#B13-marinedrugs-18-00461\" rid=\"B13-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514678\">13</a>,<a href=\"#B14-marinedrugs-18-00461\" rid=\"B14-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514658\">14</a>,<a href=\"#B15-marinedrugs-18-00461\" rid=\"B15-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514656\">15</a>], and proteomic studies of marine fungal enzymes involved in plant biomass degradation [<a href=\"#B12-marinedrugs-18-00461\" rid=\"B12-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514710\">12</a>,<a href=\"#B20-marinedrugs-18-00461\" rid=\"B20-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514674\">20</a>,<a href=\"#B21-marinedrugs-18-00461\" rid=\"B21-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514660\">21</a>].</p><p id=\"__p5\" class=\"p p-last\">The main objectives of our work were to get insight into lignocellulose degradation by marine-derived fungi and to assess whether and how fungal enzymatic machineries adapt to saline conditions. In a previous study, five marine-derived fungal strains were isolated from Tunisian coasts and identified by a phylogenetic approach, and the adaptation of laccase-like activities in their secretome was assessed, in response to several factors, including salt conditions [<a href=\"#B22-marinedrugs-18-00461\" rid=\"B22-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514702\">22</a>]. In this work, we study the growth of the five isolates and the tuning of lignocellulolytic activities under different conditions. <em>S. lucomagnoense</em> was then selected to follow the adaptation of its secretome composition when the fungus was grown on either a terrestrial (wheat straw) or a marine (seagrass) substrate, under saline and non-saline conditions.</p></span><span id=\"s\n2-marinedrugs-18-00461\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\n2-marinedrugs-18-00461title\">2. Results</h2><span id=\"s\n2dot1-marinedrugs-18-00461\" class=\"sec sec-first\"><h3 id=\"s\n2dot1-marinedrugs-18-00461title\">2.1. Effect of Carbon Sources and Sea Salt on Fungal Growth</h3><p id=\"__p6\" class=\"p p-first\">The radial growth of five fungal strains isolated from marine environments [<a href=\"#B22-marinedrugs-18-00461\" rid=\"B22-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514709\">22</a>], namely, <em>Stemphylium lucomagnoense, Aspergillus nidulans,</em> and three <em>Trichoderma asperellum</em> strains (called <em>T. asperellum</em>1 (Tas 1), <em>T. asperellum</em>2 (Tas 2) and <em>T. asperellum</em>3 (Tas 3)), was evaluated. We followed and compared fungal growth on two different carbon sources, xylan, and carboxymethylcellulose (CMC), in the presence or absence of sea salt (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f001/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"marinedrugs-18-00461-f001\" rid-ob=\"ob-marinedrugs-18-00461-f001\" co-legend-rid=\"lgnd_marinedrugs-18-00461-f001\"><span>Figure 1</span></a>). The experiments were performed in triplicate for subsequent statistical analysis of data.</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"marinedrugs-18-00461-f001\" co-legend-rid=\"lgnd_marinedrugs-18-00461-f001\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f001/\" target=\"figure\" rid-figpopup=\"marinedrugs-18-00461-f001\" rid-ob=\"ob-marinedrugs-18-00461-f001\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140580392215888\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7551824_marinedrugs-18-00461-g001.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is marinedrugs-18-00461-g001.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7551824/bin/marinedrugs-18-00461-g001.jpg\" /></a></span><span id=\"largeobj_idm140580392215888\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f001/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_marinedrugs-18-00461-f001\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f001/\" target=\"figure\" rid-figpopup=\"marinedrugs-18-00461-f001\" rid-ob=\"ob-marinedrugs-18-00461-f001\">Figure 1</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p41\">The radial growth rate of five isolated fungi: Three <em>Trichoderma asperellum strains</em> (<em>Tas</em>1, 2 and 3), <em>Stemphylium lucomagnoense</em> (<em>Slu</em>), and <em>Aspergillus nidulans</em> (<em>Ani</em>). Growth is measured on two different carbon sources, either xylan or carboxymethylcellulose (CMC), in the absence or presence of sea salt. Groups with different letters exhibit significantly different growth rates at a 95% confidence interval according to a Kruskhal&#x02013;Wallis rank sum test, followed by a Dunn posthoc pairwise comparison test.</p></span></span></span><p id=\"__p7\" class=\"p p-last\">It can be seen that these strains were able to grow similarly on xylan in the absence and the presence of sea salt. However, while a slight reduction of fungal growth was observed for the three <em>T. asperellum</em> strains (from 0.63 cm/day to 0.49 cm/day) and <em>A. nidulans</em> (from 0.16 cm/day to 0.13 cm/day), <em>S. lucomagnoense</em> were not significantly affected by the salinity of the medium (0.27 cm/day). The five strains were also able to grow on CMC as the sole source of carbon, in the absence or presence of sea salt. Whereas for the three <em>T. asperellum</em> strains, no significant difference was observed in non-saline versus saline conditions, salt inhibited growth of <em>A. nidulans</em>, and improved that of <em>S. lucomagnoense</em>, from 0.29 cm/day to 0.52 cm/day.</p></span><span id=\"s\n2dot2-marinedrugs-18-00461\" class=\"sec\"><h3 id=\"s\n2dot2-marinedrugs-18-00461title\">2.2. Effect of Carbon Sources and Sea Salt on Lignocellulolytic Activities</h3><p id=\"__p8\" class=\"p p-first\">In parallel, lignocellulolytic activities were assessed for liquid cultures of the five strains grown in either non-saline (RHI medium) or saline (RHI plus 3% <em>w</em>/<em>v</em> sea salt) conditions on two different carbon sources, wheat straw, and seagrass. The potential towards the hydrolysis of azo-xylan (endo-xylanase activity) and azo-CMC (endo-cellulase activity), as well as towards the oxidation of 2,2&#x02032;-azino-bis(3-&#x000e9;thylbenzothiazoline-6-sulphonique) (ABTS) (laccase-like activity), was assessed for secretomes from fungi grown for nine days on either wheat straw or seagrass (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f002/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"marinedrugs-18-00461-f002\" rid-ob=\"ob-marinedrugs-18-00461-f002\" co-legend-rid=\"lgnd_marinedrugs-18-00461-f002\"><span>Figure 2</span></a>). The experiments were performed in triplicate for subsequent statistical analysis of data.</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"marinedrugs-18-00461-f002\" co-legend-rid=\"lgnd_marinedrugs-18-00461-f002\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f002/\" target=\"figure\" rid-figpopup=\"marinedrugs-18-00461-f002\" rid-ob=\"ob-marinedrugs-18-00461-f002\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140580392205712\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7551824_marinedrugs-18-00461-g002.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is marinedrugs-18-00461-g002.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7551824/bin/marinedrugs-18-00461-g002.jpg\" /></a></span><span id=\"largeobj_idm140580392205712\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f002/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_marinedrugs-18-00461-f002\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f002/\" target=\"figure\" rid-figpopup=\"marinedrugs-18-00461-f002\" rid-ob=\"ob-marinedrugs-18-00461-f002\">Figure 2</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p42\">Screening of lignocellulolytic activities in the supernatant of liquid cultures of five isolated fungi (<em>Trichoderma asperellum</em>, <em>Tas</em>1, 2, and 3 <em>Stemphylium lucomagnoense</em>, <em>Slu</em> and <em>Aspergillus nidulans</em>, <em>Ani</em>). Enzymatic activity was tested after nine days of growth on either wheat straw or seagrass, in either non-saline (RHI, black bars) or saline (RHI-salt with 3% sea salt, grey bars). Xylanase, cellulase, and laccase-like activities were assessed spectrophotometrically, by following azo-xylan (<strong>A</strong>) and azo-CMC (<strong>B</strong>), hydrolysis at 590 nm, and ABTS oxidation (<strong>C</strong>) at 420 nm. For each activity, groups with different letters exhibit significantly different activity at a 95% confidence interval according to a Kruskhal&#x02013;Wallis rank sum test, followed by a Dunn posthoc pairwise comparison test. The results of Dunn tests are to be found in the <a href=\"#app1-marinedrugs-18-00461\" rid=\"app1-marinedrugs-18-00461\" class=\" app\">supplementary data (Table S1)</a>.</p></span></span></span><p id=\"__p9\" class=\"p p-last\">Differences in xylanase activities (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f002/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"marinedrugs-18-00461-f002\" rid-ob=\"ob-marinedrugs-18-00461-f002\" co-legend-rid=\"lgnd_marinedrugs-18-00461-f002\"><span>Figure 2</span></a>A) were not pronounced when comparing cultures on wheat straw and seagrass, but saline conditions generally inhibited xylanase activities, as observed for laccase-like activities. When grown on wheat straw, <em>T. asperellum</em> strains showed the highest activities, followed by <em>A. nidulans</em> and <em>S. lucomagnoense</em>, respectively. On the other hand, azo-xylan hydrolysis from seagrass cultures of <em>S. lucomagnoense</em> was by far the most efficient. For cellulase activities, no significant differences could be seen, probably because the standard deviations were high in our test conditions. It would appear that cellulase activities were higher in cultures grown on seagrass in non-saline conditions (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f002/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"marinedrugs-18-00461-f002\" rid-ob=\"ob-marinedrugs-18-00461-f002\" co-legend-rid=\"lgnd_marinedrugs-18-00461-f002\"><span>Figure 2</span></a>B), and that <em>T. asperellum</em> and <em>A. nidulans</em>, showed the highest ones For laccase-like activities (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f002/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"marinedrugs-18-00461-f002\" rid-ob=\"ob-marinedrugs-18-00461-f002\" co-legend-rid=\"lgnd_marinedrugs-18-00461-f002\"><span>Figure 2</span></a>C), results show that globally in saline condition, the culture medium reduced the secretome activities for all strains except for <em>S. lucomagnoense</em> cultured on seagrass, confirming our previous results [<a href=\"#B22-marinedrugs-18-00461\" rid=\"B22-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514671\">22</a>]. In general, the three <em>T. asperellum</em> strains showed the highest activities, and <em>A. nidulans</em> the lowest. In addition, laccase-like activities were much higher for cultures grown on wheat straw.</p></span><span id=\"s\n2dot3-marinedrugs-18-00461\" class=\"sec\"><h3 id=\"s\n2dot3-marinedrugs-18-00461title\">2.3. cDNA Library Construction and Sequencing, and Annotation of the Lignocellulolytic Enzyme Set from the Stemphylium Lucomagnoense Proteomes</h3><p id=\"__p10\" class=\"p p-first\">A more detailed analysis of secreted enzymes in response to growth conditions was conducted on one strain only. In our search of a secreted enzymatic machinery with original features, with a focus on enzymes with novel properties, <em>S. lucomagnoense</em> was selected based on: (i) Its growth response in saline conditions that was either unaffected or improved on xylan and CMC, respectively (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f001/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"marinedrugs-18-00461-f001\" rid-ob=\"ob-marinedrugs-18-00461-f001\" co-legend-rid=\"lgnd_marinedrugs-18-00461-f001\"><span>Figure 1</span></a>); (ii) its highest xylanase activity among the five strains from cultures on seagrass; both in the presence and absence of sea salt (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f002/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"marinedrugs-18-00461-f002\" rid-ob=\"ob-marinedrugs-18-00461-f002\" co-legend-rid=\"lgnd_marinedrugs-18-00461-f002\"><span>Figure 2</span></a>A); and (iii) its secreted laccase-like activity which was found to be slightly improved in cultures on seagrass with sea salt (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f002/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"marinedrugs-18-00461-f002\" rid-ob=\"ob-marinedrugs-18-00461-f002\" co-legend-rid=\"lgnd_marinedrugs-18-00461-f002\"><span>Figure 2</span></a>C). These characteristics, in fact, point to the potential retrieval of novel microbial and enzymatic features of interest for industrial applications, deriving from <em>S. lucomagnoense</em> adaptation to the marine environment. As no genomic data are available for this fungus, a representative cDNA library was constructed and sequenced using the approach developed for the mangrove fungus, <em>Pestalotiopsis</em> sp. [<a href=\"#B12-marinedrugs-18-00461\" rid=\"B12-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514720\">12</a>]. Briefly, the transcriptome (total RNAs were extracted from the various growth condition sand pooled together before sequencing) was sequenced to prepare the cDNA library (replacing the genome) to identify the proteins obtained from the proteome and not to quantify the expression of the various genes. In addition, the cDNA library was normalized so that the read counts of each transcript could not be determined. The datasets of this study were deposited under National Center for Biotechnology Information (NCBI) BioSample accession ID: SAMN15897915 with corresponding NCBI BioProject accession ID: PRJNA659110. The corresponding proteins were identified using the CAZy database (<a href=\"http://www.cazy.org\">www.cazy.org</a>), and the resulting protein database was used to identify, by mass-matching, the proteins obtained from a proteomic mass spectrometry analysis.</p><p id=\"__p11\">To capture the largest possible number of transcripts, a normalized cDNA library was prepared from total RNAs isolated from pooled samples taken after 2, 4, 6, 8, and 10 days of culture for each condition. This library was sequenced using the Illumina technology (2 &#x000d7; 150 bp) (Illumina, San Diego, CA, USA) with an average of 300 million paired-end sequences. After the initial quality assessments, de novo transcriptomic assembly was achieved using the de novo assembler suite from the CLCBio Genomic Workbench version 11 (<a href=\"https://digitalinsights.qiagen.com\">https://digitalinsights.qiagen.com</a>) (Qiagen, Hilden, Allemagne). In this de novo transcriptomic assembly, a total of 520,000,000 reads were assembled in 350,657 contigs. The N50 for the contigs is 504 bp, and an average length of 316 bp (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/table/marinedrugs-18-00461-t001/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"marinedrugs-18-00461-t001\" rid-ob=\"ob-marinedrugs-18-00461-t001\" co-legend-rid=\"\"><span>Table 1</span></a>). The assembled contigs sequences are available at GitHub URL as <a href=\"https://github.com/drabhishekkumar/Stemphylium-lucomagnoense-transcriptomics\">https://github.com/drabhishekkumar/Stemphylium-lucomagnoense-transcriptomics</a>.</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"marinedrugs-18-00461-t001\"><h3>Table 1</h3><!--caption a7--><span class=\"caption\"><p id=\"__p45\">Summary of assembled contig statistics for de novo transcriptome of marine <em>Stemphylium lucomagnoense.</em></p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140580392192192\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><thead><tr><th align=\"left\" valign=\"middle\" style=\"border-top:solid thin;border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">\n</th><th align=\"left\" valign=\"middle\" style=\"border-top:solid thin;border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">Assembled Contig Statistics</th></tr></thead><tbody><tr><td align=\"left\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">Total length of contigs</td><td align=\"left\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">110,739,726 bp</td></tr><tr><td align=\"left\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">Total number of contigs</td><td align=\"left\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">350,657</td></tr><tr><td align=\"left\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">N25 stats</td><td align=\"left\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">25% of total contig length is contained in the 5614 contigs &#x0003e; = 2983 bp</td></tr><tr><td align=\"left\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">N50 stats</td><td align=\"left\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">50% of total contig length is contained in the 26,298 contigs &#x0003e; = 504 bp</td></tr><tr><td align=\"left\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">N75 stats</td><td align=\"left\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">75% of total contig length is contained in the 142,056 contigs &#x0003e; = 176 bp</td></tr><tr><td align=\"left\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">Total GC count</td><td align=\"left\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">59,818,129 bp</td></tr><tr><td align=\"left\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">% GC</td><td align=\"left\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">54.02</td></tr></tbody></table></span><span id=\"largeobj_idm140580392192192\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/table/marinedrugs-18-00461-t001/?report=objectonly\">Open in a separate window</a></span></span><p id=\"__p12\" class=\"p p-last\">In order to obtain an expert and accurate annotation of the lignocellulolytic enzymes present in the transcriptome, translated transcripts were compared to the entries in the CAZy database (<a href=\"http://www.cazy.org\">www.cazy.org</a>). Using 1D electrophoresis followed by LC-MS/MS analysis, 51 secreted <em>S. lucomagnoense</em> proteins were identified as belonging to one of the CAZy classes. CAZymes, including glycoside hydrolases (GHs), carbohydrate esterases (Ces), polysaccharide lyases (PLs), carbohydrate-binding modules (CBMs), glycosyl transferases (GTs), and auxiliary activities (Aas), are, in fact, microbial enzymes involved in complex carbohydrate breakdown and assembly, and in particular in lignocellulose degradation and transformation. Among the <em>S. lucomagnoense</em> CAZymes, 38 GHs, 3 Ces, 1 PLs, and 7 Aas and 13 CBMs, appended to an enzymatic module or not, were identified, including multi-modular proteins (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/table/marinedrugs-18-00461-t002/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"marinedrugs-18-00461-t002\" rid-ob=\"ob-marinedrugs-18-00461-t002\" co-legend-rid=\"\"><span>Table 2</span></a>).</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"marinedrugs-18-00461-t002\"><h3>Table 2</h3><!--caption a7--><span class=\"caption\"><p id=\"__p46\">CAZymes identified in <em>Stemphylium lucomagnoense</em> secretomes grown on either seagrass or wheat straw, in saline and non-saline conditions.</p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140580392161184\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><thead><tr><th rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-top:solid thin\" colspan=\"1\">Accession Number</th><th colspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-top:solid thin\" rowspan=\"1\">Number of Total Spectra </th><th colspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-top:solid thin\" rowspan=\"1\">Number of Unique Peptides </th><th rowspan=\"2\" align=\"left\" valign=\"middle\" style=\"border-top:solid thin\" colspan=\"1\">Predicted Protein Functions</th><th rowspan=\"2\" align=\"left\" valign=\"middle\" style=\"border-top:solid thin\" colspan=\"1\">Predicted Family<br />(Subfamily) *</th><th rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-top:solid thin\" colspan=\"1\">Induction</th></tr><tr><th align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W/Ws</th><th align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">S/Ss</th><th align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W/Ws</th><th align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">S/Ss</th></tr></thead><tbody><tr><td align=\"left\" valign=\"middle\" style=\"border-top:solid thin\" rowspan=\"1\" colspan=\"1\">APMZ2_prot13181</td><td align=\"center\" valign=\"middle\" style=\"border-top:solid thin\" rowspan=\"1\" colspan=\"1\">222/101</td><td align=\"center\" valign=\"middle\" style=\"border-top:solid thin\" rowspan=\"1\" colspan=\"1\">26/13</td><td align=\"center\" valign=\"middle\" style=\"border-top:solid thin\" rowspan=\"1\" colspan=\"1\">75/43</td><td align=\"center\" valign=\"middle\" style=\"border-top:solid thin\" rowspan=\"1\" colspan=\"1\">16/10</td><td align=\"left\" valign=\"middle\" style=\"border-top:solid thin\" rowspan=\"1\" colspan=\"1\">Glucoamylase</td><td align=\"left\" valign=\"middle\" style=\"border-top:solid thin\" rowspan=\"1\" colspan=\"1\">GH15-CBM20</td><td align=\"center\" valign=\"middle\" style=\"border-top:solid thin\" rowspan=\"1\" colspan=\"1\">S/Ss W/Ws</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot912</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">110/53</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">17/6</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">41/30</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">14/6</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Copper radical oxidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">AA5_1</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">S/Ss W/Ws</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot15973</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">92/56</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">15/10</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">33/25</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">9/8</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b2;-1,3-glucanase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH55</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">S/Ss W/Ws</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot2472</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">58/16</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">10/7</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">24/13</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">9/7</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b1;-amylase </td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH13_1</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">S/Ss W/Ws</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot14594</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">50/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">31/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b2;-1,4-xylanase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH10</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">S W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot3103</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">36/8</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">7/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">15/7</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">6/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b1;-mannosidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH47</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">S W/Ws</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot1893</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">34/5</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">28/5</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b1;-<span class=\"small-caps\">l</span>-rhamnosidase </td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH78</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">S W/Ws</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot27112</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">33/16</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">23/15</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Glucose/methanol/choline oxidoreductase (GMC)</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">AA3_2</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W/Ws</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot4629</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">29/18</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">5/8</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">20/13</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">5/6</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Pyrroloquinoline quinone-dependent oxidoreductase </td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">AA12</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">S/Ss W/Ws</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot29015</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">28/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">22/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b1;-glucosidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH31</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot1520</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">20/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">18/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b2;-1,4-glucosidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH3</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">S W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot21697</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">15/2</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">12/2</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b2;-<span class=\"small-caps\">d</span>-glucosaminidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH20</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W/Ws</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot29106</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">15/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">12/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b1;-glucosidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH31</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot275</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">13/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">10/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Chitin deacetylase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">CBM18-CE4-CBM18-CBM18</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">S W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot24323</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">12/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">11/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Glucose/methanol/choline oxidoreductase (GMC)</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">AA3_2</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">S W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot2902</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">12/5</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">4/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b1;,&#x003b1;-trehalase </td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH37</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W/Ws</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot3532</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">11/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">9/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b2;-1,4-glucosidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH3</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot3750</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">11/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">9/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b2;-1,4-glucosidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH3</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot15280</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">11/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">10/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b2;-1,4-glucosidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH3</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot8136</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">10/3</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">8/3</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Rhamnogalacturonyl hydrolase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH105</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">S W/Ws</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot26178</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">9/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">8/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Endo/Exo-&#x003b2;-1,4-glucanase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH55</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W/S</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot15779</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">9/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">5/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b1;-mannosidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH47</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot27204</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">8/8</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/2</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">7/7</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/2</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Glucoamylase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH15-CBM20</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Ss W/Ws</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot7619</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">8/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">8/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Feruloyl esterase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">CE1</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">S W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot18739</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">8/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">6/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Carbohydrate Binding</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">CBM50-CBM50-CBM50</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot8482</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">7/4</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">7/4</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b2;-1,3-1,4-glucan endo-1,3- &#x003b2;-glucosidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH17</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W/Ws</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot17304</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">7/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">6/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b2;-1,3-galactanase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH43_24-CBM35</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">S W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot11667</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">7/4</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">7/4</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b2;-glucuronidase/heparanase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH79</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W/Ws</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot17211</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">7/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">7/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b2;-<span class=\"small-caps\">l</span>-arabinofuranosidase </td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH142</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot21080</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">6/5</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">5/5</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Lytic polysaccharide mono-oxygenase (LPMO)</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">AA11</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W/Ws</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot9402</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">6/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">6/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b1;-<span class=\"small-caps\">l</span>-arabinofuranosidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH51</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot13544</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">6/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b1;-mannosidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH47</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot21512</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">5/2</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">4/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b2;-<span class=\"small-caps\">d</span>-glucosaminidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH20</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot23159</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">5/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">5/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Carbohydrate-binding</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">CBM50</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot27520</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">5/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">5/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b2;-1,4-glucosidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH1</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot7701</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">4/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/2</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">4/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/2</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Chitin deacetylase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">CE4-CBM18-CBM18</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">S/Ss W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot6391</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">4/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">4/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Lacto-<em>N</em>-biosidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH136</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot20074</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">4/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">4/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Lacto-<em>N</em>-biosidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH136</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot9251</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">4/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">4/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Chitinase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH18</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot29026</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">4/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">4/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Carbohydrate esterase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">CE15</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot1535</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">4/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b2;-1,3-1,4-glucan endo-1,3-&#x003b2;-glucosidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH17</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot16320</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b1;-glucosidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH31</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">S W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot5721</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b1;-<span class=\"small-caps\">l</span>-rhamnosidase </td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH78</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot21603</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b1;-<span class=\"small-caps\">l</span>-arabinofuranosidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH43_26</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot411</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b2;-1,4-xylanase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">CBM1-GH10</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot2229</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Glucose/methanol/choline oxidoreductase (GMC)</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">AA3_2</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot19857</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Dextranase/isopullulanase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH49</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot23037</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b1;-<span class=\"small-caps\">l-</span>arabinofuranosidase/&#x003b2;-<span class=\"small-caps\">d</span>-xylosidase/&#x003b1;-<span class=\"small-caps\">l</span>-arabinanase/&#x003b2;-<span class=\"small-caps\">d</span>-galactofuranosidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH43_22-GH43_34</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot3876</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x003b2;-<span class=\"small-caps\">d</span>-glucosaminidase</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">GH20</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">W</td></tr><tr><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">APMZ2_prot11475</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/6</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/6</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">0/0</td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Glucooligosaccharide oxidase </td><td align=\"left\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">AA7</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Ws</td></tr><tr><td align=\"left\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">APMZ2_prot21161</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">0/3</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">0/3</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">0/3</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">0/3</td><td align=\"left\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">Alginate lyase</td><td align=\"left\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">PL7_4</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">Ss Ws</td></tr></tbody></table></span><span id=\"largeobj_idm140580392161184\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/table/marinedrugs-18-00461-t002/?report=objectonly\">Open in a separate window</a></span><span class=\"tblwrap-foot\"><span id=\"idm140580391597968\"><p id=\"__p47\" class=\"p p-first-last\">W wheat straw, Ws wheat straw with 3% sea salt, S seagrass, Ss seagrass with 3% sea salt, GH glycoside hydrolase, PL polysaccharide lyase, AA auxiliary activities, CE carbohydrate esterase, CBM carbohydrate-binding module, GMC glucose/methanol/choline oxidoreductase, LPMO lytic polysaccharide mono-oxygenase. * Families were assigned according to the CAZy database.</p></span></span></span></span><span id=\"s\n2dot4-marinedrugs-18-00461\" class=\"sec sec-last\"><h3 id=\"s\n2dot4-marinedrugs-18-00461title\">2.4. Enzyme Distribution in Secretomes from Cultures on Wheat Straw and Seagrass, in Saline and Non-Saline Conditions</h3><p id=\"__p13\" class=\"p p-first\">To gain insight into the <em>S. lucomagnoense</em> enzyme machinery acting on lignocellulose, the distribution of enzymes retrieved by proteomic analysis in the four different growth conditions was determined. This analysis was carried out on four specific secretomes, i.e., <em>S. lucomagnoense</em> cultures produced on either wheat straw (as model terrestrial plant biomass substrate) or seagrass (as model marine substrate), with or without 3% sea salt (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f003/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"marinedrugs-18-00461-f003\" rid-ob=\"ob-marinedrugs-18-00461-f003\" co-legend-rid=\"lgnd_marinedrugs-18-00461-f003\"><span>Figure 3</span></a>). Out of the 51 identified proteins, 50 (98%) and 17 (33%) proteins were produced on wheat straw in non-saline and saline conditions, respectively, showing a negative pressure of sea salt on protein production (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f003/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"marinedrugs-18-00461-f003\" rid-ob=\"ob-marinedrugs-18-00461-f003\" co-legend-rid=\"lgnd_marinedrugs-18-00461-f003\"><span>Figure 3</span></a>). For seagrass cultures, overall, even fewer proteins were identified in the secretomes produced in non-saline and saline conditions, i.e., 17 (33%) and 8 (15.6%), respectively. Five CAZymes were produced in all the tested conditions (10%).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"marinedrugs-18-00461-f003\" co-legend-rid=\"lgnd_marinedrugs-18-00461-f003\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f003/\" target=\"figure\" rid-figpopup=\"marinedrugs-18-00461-f003\" rid-ob=\"ob-marinedrugs-18-00461-f003\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140580392201808\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7551824_marinedrugs-18-00461-g003.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is marinedrugs-18-00461-g003.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7551824/bin/marinedrugs-18-00461-g003.jpg\" /></a></span><span id=\"largeobj_idm140580392201808\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f003/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_marinedrugs-18-00461-f003\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f003/\" target=\"figure\" rid-figpopup=\"marinedrugs-18-00461-f003\" rid-ob=\"ob-marinedrugs-18-00461-f003\">Figure 3</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p43\">Venn diagram representing numbers of CAZymes identified in <em>Stemphylium lucomagnoense</em> secretomes grown on either seagrass (S/Ss) or wheat straw (W/Ws), in saline (Ss/Ws) and non-saline (S/W) conditions.</p></span></span></span><p id=\"__p14\">In more detail, all the identified proteins were produced in cultures grown on wheat straw, and only 19 (37%) on seagrass (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f003/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"marinedrugs-18-00461-f003\" rid-ob=\"ob-marinedrugs-18-00461-f003\" co-legend-rid=\"lgnd_marinedrugs-18-00461-f003\"><span>Figure 3</span></a>). No protein was exclusively found from cultures grown on seagrass, while 25 (49%) were exclusively found in wheat straw cultures, for instance, one member of family GH31 (APMZ2_prot29015) and three of family GH3 (APMZ2_prot3532, 3750, 15280) for the most abundant proteins (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/table/marinedrugs-18-00461-t002/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"marinedrugs-18-00461-t002\" rid-ob=\"ob-marinedrugs-18-00461-t002\" co-legend-rid=\"\"><span>Table 2</span></a>). For proteins identified in secretomes grown on wheat straw, 34 (66.7%) were exclusively identified in non-saline cultures, as for instance, one member of the family GH78 (APMZ2_prot5721). For seagrass cultures, only two proteins were exclusively identified in saline conditions. Only one protein, PL7_4 (APMZ2_prot21161), was exclusively identified in saline conditions from cultures grown on both wheat straw and seagrass.</p><p id=\"__p15\">Considering the CAZyme family repartitions in cultures grown on wheat straw, the production of GHs was drastically inhibited with 37 to 11 members, and a lower pressure was observed on Aas (7 to 5) (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f004/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"marinedrugs-18-00461-f004\" rid-ob=\"ob-marinedrugs-18-00461-f004\" co-legend-rid=\"lgnd_marinedrugs-18-00461-f004\"><span>Figure 4</span></a>). Of the two CE (carbohydrate esterase) members found in wheat straw cultures, none were found in saline conditions. Related to seagrass cultures, the ratio was equivalent to 11 GH (glycoside hydrolase) members against 4 found in non-saline versus saline conditions. In addition, one-third of the Aas was found in saline conditions. The unique CE member identified in seagrass cultures was found in both conditions. The only CAZy member found exclusively in saline conditions is the PL member already cited (APMZ2_prot21161).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"marinedrugs-18-00461-f004\" co-legend-rid=\"lgnd_marinedrugs-18-00461-f004\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f004/\" target=\"figure\" rid-figpopup=\"marinedrugs-18-00461-f004\" rid-ob=\"ob-marinedrugs-18-00461-f004\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140580392196400\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7551824_marinedrugs-18-00461-g004.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is marinedrugs-18-00461-g004.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\n/PMC7551824/bin/marinedrugs-18-00461-g004.jpg\" /></a></span><span id=\"largeobj_idm140580392196400\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f004/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_marinedrugs-18-00461-f004\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/figure/marinedrugs-18-00461-f004/\" target=\"figure\" rid-figpopup=\"marinedrugs-18-00461-f004\" rid-ob=\"ob-marinedrugs-18-00461-f004\">Figure 4</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p44\">Distribution of CAZymes in <em>Stemphylium lucomagnoense</em> secretomes grown on either wheat straw in non-saline (<strong>A</strong>) and saline (<strong>B</strong>) conditions, or on seagrass in non-saline (<strong>C</strong>) and saline (<strong>D</strong>) conditions. &#x0201c;Others&#x0201d; include enzymatic activates and proteins, such as PLs and CBMs.</p></span></span></span><p id=\"__p16\">Cellulases are a wide and heterogeneous group of enzymes, encompassing a large number of phylogenetically diverse proteins families. In <em>S. lucomagnoense</em> secretomes, only GH1 (APMZ2_prot27520) and GH3 (APMZ2_prot1520, 3532, 3750, 15280) accessory enzymes to cellulose degradation were found, but no members of families GH5, 6, 7 and 12 for instance, unlike normally found in most terrestrial lignocellulose degrading fungi. The five GH1 and GH3 enzymes were mainly produced when <em>S. lucomagnoense</em> was grown on wheat straw, with the exception of protein 1520, which was also found when seagrass was used as a growth substrate. No cellulases were identified in saline conditions though. Besides cellulases, other carbohydrate degrading enzymes were present in the secretome of <em>S. lucomagnoense</em>, such as GH18 members. Family GH18, generally found in filamentous fungi, includes chitinases and can be appended to carbohydrate-binding modules (CBMs) of family CBM18. Modules of family CBM18 have been demonstrated to bind to chitin and to be involved in fungal processes, such as cell wall degradation and modification, spore germination, tip growth, hyphae branching, spore differentiation, and autolysis [<a href=\"#B23-marinedrugs-18-00461\" rid=\"B23-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514688\">23</a>]. In the <em>S. lucomagnoense</em> proteome, the CBM18 module was found in two multi-modular proteins (APMZ2_prot275 and 7701), fused to carbohydrate esterase domains (CE4). Family CE4 contains putative chitin deacetylases, which are known to be involved in cell-wall chitosan biosynthesis in Mucorales, by action in tandem with chitin synthases of the glycoside transferase family GT2 [<a href=\"#B24-marinedrugs-18-00461\" rid=\"B24-marinedrugs-18-00461\" class=\" bibr popnode\">24</a>]. Strikingly, another carbohydrate-binding module, CBM50, was found as a unique module or in tandem as a trimodular protein (APMZ2_prot23159 and 18739, respectively). CBM50 is a module of approximately 50 residues, generally grafted to various enzymes that cleave chitin or peptidoglycan, including families GH18, 19, 23&#x02013;25, and 73. When not associated with other enzymatic modules, CBM50, like CBM18, is involved in the recognition or binding of chitin and prevents hydrolysis of the fungal cell wall by plant chitinases, therefore interfering with chitin-triggered host immunity [<a href=\"#B25-marinedrugs-18-00461\" rid=\"B25-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514669\">25</a>].</p><p id=\"__p17\">When considering candidate hemicellulases, two putative xylanases were identified as members of family GH10 (APMZ2_prot14594 and 411), appended to a carbohydrate-binding module of family CBM1, and one &#x003b1;-<span class=\"small-caps\">l</span>-arabinofuranosidase (family GH51, APMZ2_prot9402). Family GH43 was also particularly well represented in the <em>S. lucomagnoense</em> secretomes, with four representatives. This family includes &#x003b1;-<span class=\"small-caps\">l</span>-arabinofuranosidase, &#x003b2;-<span class=\"small-caps\">d</span>-xylosidase, &#x003b1;-<span class=\"small-caps\">l</span>-arabinanase, &#x003b2;-<span class=\"small-caps\">d</span>-galactosidase, and many other enzymes involved in hemicellulose and pectin debranching and degradation [<a href=\"#B26-marinedrugs-18-00461\" rid=\"B26-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514665\">26</a>]. Among the four GH43 representatives, the most abundant was APMZ2_prot17304 (GH43_24-CBM35), which is a hypothetical &#x003b2;-1,3-galactosidase fused to a CBM35 module, known to bind 1,3-&#x003b2;-<span class=\"small-caps\">d</span>-galactose. The second most abundant representative of the family in <em>S. lucomagnoense</em> belongs to subfamily GH43_26 (APMZ2_prot21603), and the two-last retrieved GH43 domains were associated with a bimodular protein, GH43_22-GH43_34 (APMZ2_prot23037). GH43_34 modules are most associated with another GH43 domain, but the resulting fusion proteins are not characterized, although bacterial GH43_34 members have been described as &#x003b1;-<span class=\"small-caps\">l</span>-arabinofuranosidases [<a href=\"#B27-marinedrugs-18-00461\" rid=\"B27-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514708\">27</a>]. All <em>S. lucomagnoense</em> enzymes involved in hemicellulose degradation are also found mainly in cultures grown on wheat straw, with the exception of two proteins (APMZ2_prot14594 and 17304, i.e., the most abundant hemicellulases in the analyzed secretomes), which were found both on seagrass and wheat straw.</p><p id=\"__p18\">Among Auxiliary Activities (Aas), a copper radical oxidase (CRO, APMZ2_prot912) was produced in a relatively large amount as compared to other representatives in all the culture conditions tested. This protein is predicted to be phylogenetically related to glyoxal oxidases (GLOXs, CAZy subfamily AA5_1) [<a href=\"#B28-marinedrugs-18-00461\" rid=\"B28-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514732\">28</a>]. CROs are metalloenzymes containing one copper metal ion. While GLOX enzymes have been biochemically characterized [<a href=\"#B29-marinedrugs-18-00461\" rid=\"B29-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514680\">29</a>,<a href=\"#B30-marinedrugs-18-00461\" rid=\"B30-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514719\">30</a>], other copper radical oxidases (CRO1 to CRO6) are still uncharacterized proteins of unknown function [<a href=\"#B31-marinedrugs-18-00461\" rid=\"B31-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514711\">31</a>]. Two more proteins well expressed from <em>S. lucomagnoense</em> (APMZ2_prot27712 and 24323) were identified as putative aryl alcohol oxidases (AAOs) (CAZy subfamily AA3_2). These two proteins respectively appeared in cultures grown wheat straw either with or without sea salt, or on both wheat straw and seagrass. The AA3 family is still not completely characterized, and recently 3 AAO-like enzymes were shown to act as aryl alcohol quinone oxidoreductases (AAQOs), displaying a preference for quinones as electron acceptors, and lower reactivity towards molecular oxygen as typically observed for AAO [<a href=\"#B32-marinedrugs-18-00461\" rid=\"B32-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514667\">32</a>]. In addition, glucose oxidases and dehydrogenases were found in subfamily AA3_2, and as such, further biochemical characterization will be required to precisely annotate these two <em>S. lucomagnoense</em> enzymes. A hypothetical pyrroloquinoline quinone (PQQ)-dependent oxidoreductase (AA12, APMZ2_prot4629) was also identified in <em>S. lucomagnoense</em> secretomes. PQQ-dependent enzymes were only recently found and described in fungi. They catalyzed the oxidation of various sugars, and their three-dimensional structure was recently elucidated [<a href=\"#B33-marinedrugs-18-00461\" rid=\"B33-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514730\">33</a>,<a href=\"#B34-marinedrugs-18-00461\" rid=\"B34-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514647\">34</a>]. One member of this family, the PQQ-dependent pyranose dehydrogenase from <em>Coprinopsis cinerea</em> (<em>Cc</em>PDH), was demonstrated to sustain the catalytic cycle of a lytic polysaccharide mono-oxygenase (LMPO) [<a href=\"#B35-marinedrugs-18-00461\" rid=\"B35-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514706\">35</a>]. In fact, LPMOs can complete their catalytic cycle and catalyze the oxidative cleavage of glycosidic bonds only if external electron donors are available [<a href=\"#B36-marinedrugs-18-00461\" rid=\"B36-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514666\">36</a>,<a href=\"#B37-marinedrugs-18-00461\" rid=\"B37-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514689\">37</a>]. As an example, it was shown that <em>Cc</em>PDH activates the C-1-oxidizing LPMO 9F and the C-4-oxidizing LPMO 9C from <em>Neurospora crassa</em> by direct electron transfer from the AA12 and through the AA8 (c cytochrome) domain [<a href=\"#B35-marinedrugs-18-00461\" rid=\"B35-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514698\">35</a>]. In our study, the AA12 protein was found in all conditions tested, and one LMPO (CAZy family AA11, APMZ2_prot21080) was also identified in secretomes produced on wheat straw in either saline or non-saline conditions. One AA11 LMPO was demonstrated to be involved in the hydrolysis of chitin chains, and C-1 oxidative cleavage has been demonstrated for a member of this family from <em>Aspergillus oryzae</em> [<a href=\"#B38-marinedrugs-18-00461\" rid=\"B38-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514659\">38</a>]. Chitin is present in the exoskeleton of arthropods, as well as in the fungal cell wall. The function and the enzymatic partners of AA11 enzymes in marine fungi are still to be elucidated. However, AA3_2 and AA12 enzymes found in <em>S. lucomagnoense</em> secretomes are reasonably good candidates [<a href=\"#B37-marinedrugs-18-00461\" rid=\"B37-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514668\">37</a>].</p><p id=\"__p19\" class=\"p p-last\">Among the most abundant proteins, a member of family GH55 (APMZ2_prot 15973) was identified. Members of this family include endo/exo-&#x003b2;-1,3-glucanases, which have a general role in morphogenesis, and more specifically, in cell wall softening during conidial formation or fruiting body development [<a href=\"#B39-marinedrugs-18-00461\" rid=\"B39-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514690\">39</a>,<a href=\"#B40-marinedrugs-18-00461\" rid=\"B40-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514657\">40</a>]. They were also shown to possess potential antifungal activity with a degradative effect on cell walls [<a href=\"#B41-marinedrugs-18-00461\" rid=\"B41-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514701\">41</a>]. GH55 enzymes display a broad range of specific activities, including laminarin hydrolase activities. Laminarin is a storage glucan found in brown algae (<em>Phaeophyceae</em>) and is formed of a glucan chain, made of &#x003b2;-1,3-linked glucose, with &#x003b2;-1,6 branches. It represents 10&#x02013;45% of microalgal biomass, and as such, a key component of food chains and the carbon cycle in the oceans [<a href=\"#B42-marinedrugs-18-00461\" rid=\"B42-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514672\">42</a>]. Not surprisingly, these enzymatic activities have already been found in marine-derived fungi [<a href=\"#B43-marinedrugs-18-00461\" rid=\"B43-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514726\">43</a>], as well as amylase activities [<a href=\"#B44-marinedrugs-18-00461\" rid=\"B44-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514651\">44</a>]. These activities are also present in <em>S. lucomagnoense</em> secretome, with a GH15-CBM20 glucoamylase (carrying a starch binding module, APMZ2_prot13181) and a GH13_1 &#x003b2;-amylase (APMZ2_prot2472). To conclude, the less expressed <em>S. lucomagnoense</em> protein is a member of family PL7 (APMZ2_prot21161) identified in cultures grown on both wheat straw and seagrass, but exclusively in saline conditions. This enzyme is a putative alginate lyase, an important enzyme used by marine-derived fungi to degrade brown algae biomass [<a href=\"#B20-marinedrugs-18-00461\" rid=\"B20-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514703\">20</a>].</p></span></span><span id=\"s\n3-marinedrugs-18-00461\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\n3-marinedrugs-18-00461title\">3. Discussion</h2><p id=\"__p20\" class=\"p p-first\">The objective of the present study was to provide new insight into the lignocellulose enzyme machinery of marine-derived fungi, to understand how they adapted to process biomass in saline conditions. For this purpose, we screened five fungal strains isolated from the Tunisian coast [<a href=\"#B22-marinedrugs-18-00461\" rid=\"B22-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514654\">22</a>]. Based on a microbial approach and on enzyme activity screening, we selected <em>S. lucomagnoense</em> for its adapted growth on xylan in saline conditions, its higher xylanase activity, and its improved laccase (seagrass-containing cultures) and cellulase (wheat straw-containing cultures) activities in the presence of sea salt. The screening of marine-derived isolates showed that the five strains can use xylan and cellulose for growth, and that they secreted the corresponding activities in the extracellular medium to degrade and use these substrates. Other marine-derived fungi, such as <em>Cadophora</em> sp. TS 2, <em>Emericellopsis</em> sp. TS 11 and <em>Pseudogymnoascus</em> sp. TS 12 isolated from the deep-sea sponge <em>Stelletta normani</em>, were previously demonstrated to possess lignocellulolytic properties without striking differences in their growth profiles, on either CMC or xylan [<a href=\"#B11-marinedrugs-18-00461\" rid=\"B11-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514677\">11</a>]. This study showed no effect on fungal growth for sodium chloride up to 0.5 M, but growth inhibition occurred for NaCl concentrations above 1 M. Wang et al. (2016) [<a href=\"#B45-marinedrugs-18-00461\" rid=\"B45-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514685\">45</a>] also studied the effect of various substrates on the growth of six marine-derived fungi: <em>Calcarisporium</em> sp. KF525, <em>Tritirachium</em> sp. LF562, <em>Bartalinia robillardoides</em> LF550, <em>Penicillium pinophilum</em> LF458, <em>Scopulariopsis brevicaulis</em> LF580 and <em>Pestalotiopsis</em> sp. KF079. For almost all fungal strains, grown in 3% sea salt, a preference for xylan over cellulose was generally showed, unlike in our results. Batista-Garcia (2017) [<a href=\"#B11-marinedrugs-18-00461\" rid=\"B11-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514670\">11</a>] also showed no impact on cellulase and xylanase activities at 0.5 M NaCl. At higher salt concentrations, however, the two enzyme profiles diverged, showing strong inhibition of cellulases, and different behavior of xylanases: Strong inhibition of xylanases from <em>Cadophora</em> sp. TS, slight activation of those from <em>Pseudogymnoascus</em> sp. TS 12, and activation between 1 and 2 M NaCl of xylanases from <em>Emericellopsis</em> sp. TS 11, suggesting that each strain adopted a different enzymatic strategy for degrading plant biomass in saline conditions. Arfi et al. [<a href=\"#B12-marinedrugs-18-00461\" rid=\"B12-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514718\">12</a>] reported that sea salt (3% <em>w</em>/<em>v</em>) has a negative effect on cellulase and xylanase activities of the mangrove fungus <em>Pestalotiopsis</em> sp. NCi6 without effect on fungal growth rate. Even with 6% salt, a 2-day-latency phase was observed&#x02014;but without impact on the final growth rate. Arfi et al. [<a href=\"#B12-marinedrugs-18-00461\" rid=\"B12-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514662\">12</a>] suggested that this fungal ability to metabolize the carbohydrate substrates was not affected by salt. Finally, other studies of xylanase production or of xylanase activities of cell-free supernatants could be found from other marine-derived fungi as from <em>Trichoderma pleuroticola</em> 08&#x000c7;K001 strain isolated from Mediterranean coastal sediments [<a href=\"#B46-marinedrugs-18-00461\" rid=\"B46-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514712\">46</a>], but a few reports are available for cellulases [<a href=\"#B47-marinedrugs-18-00461\" rid=\"B47-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514650\">47</a>]. Several reports previously focused on laccase-like activities, summarized in a recent review on marine-derived micro-organisms [<a href=\"#B48-marinedrugs-18-00461\" rid=\"B48-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514687\">48</a>], and one specifically dedicated to marine-derived fungi [<a href=\"#B10-marinedrugs-18-00461\" rid=\"B10-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514697\">10</a>]. For instance, Bonugli-Santos et al. reported laccase activities from <em>Marasmus</em> sp., <em>Peniophora</em> sp. And <em>Tinctoporellus</em> [<a href=\"#B49-marinedrugs-18-00461\" rid=\"B49-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514652\">49</a>], but they pursued no further characterization. They also optimized the production of laccases from <em>Mucor racemosus</em> with an optimum at 4 M NaCl [<a href=\"#B50-marinedrugs-18-00461\" rid=\"B50-marinedrugs-18-00461\" class=\" bibr popnode\">50</a>]. In contrast, a laccase from <em>Trematosphaeria mangrovei</em>, isolated from mangroves, was inhibited (50%) by 1 mM NaCl only [<a href=\"#B51-marinedrugs-18-00461\" rid=\"B51-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514663\">51</a>]. Finally, saline conditions enhanced the activities of two laccases, the purified forms of the marine-derived fungus <em>Pestalotiopsis</em> sp., for sea salt concentration up to 5% [<a href=\"#B52-marinedrugs-18-00461\" rid=\"B52-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514696\">52</a>].</p><p id=\"__p21\">Assaying enzyme activities in fungal secretomes to study the fungal adaptation to saline environments has limitations, as the method reflects the global activity of potentially diverse enzymatic machineries possessing different properties. To better understand the adaptation of <em>S. lucomagnoense</em> at the enzyme level, we used proteomic analysis, as previously done for the mangrove fungus <em>Pestalotiopsis</em> sp. NCi6 [<a href=\"#B12-marinedrugs-18-00461\" rid=\"B12-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514686\">12</a>]. This mangrove fungus belongs to the order <em>Pleosporales</em> [<a href=\"#B17-marinedrugs-18-00461\" rid=\"B17-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514731\">17</a>,<a href=\"#B53-marinedrugs-18-00461\" rid=\"B53-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514725\">53</a>] (Dothideomycetes), which includes marine-derived fungi and a majority of terrestrial species [<a href=\"#B4-marinedrugs-18-00461\" rid=\"B4-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514682\">4</a>]. We first sequenced and assembled the transcriptome of <em>Pestalotiopsis</em> sp. Transcribed genes were identified and functionally annotated using the CAZy database [<a href=\"#B54-marinedrugs-18-00461\" rid=\"B54-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514664\">54</a>] and used to identify secreted enzymes by matching LC-MS/MS data.</p><p id=\"__p22\">The correlation between the CAZyme repertoire identified in secretomes and growth phenotypes is not always clear, and the absence or incomplete transcriptomic and proteomic data might be claimed responsible in these specific cases, yet comparing proteomes from the four culture conditions provides an interesting insight into the adaptation on fungal secretomes to saline conditions. As an example, higher xylanase activities were found in secretomes from cultures in non-saline conditions, on both wheat straw and seagrass, indicating inhibition by sea salt added to the culture medium. Accordingly, APMZ2_prot14594 and prot411 (family GH10) are identified in proteomic data from these conditions (the latter on wheat straw only). A similar scenario was found for three GH43 xylanases. Intriguingly, xylanase activities were higher in cultures grown on seagrass, but the number of spectra was lower (for APMZ2_prot14594 and prot17304), suggesting lower protein secretion, but higher enzyme activity in saline conditions. To check this hypothesis, gene cloning and recombinant protein purification would be required, followed by enzyme characterization to understand the effect of salt on enzymatic activity. Using this approach, in a previous study on two laccases from <em>Pestalotiopsis</em> sp. KF079, isolated from the Baltic sea mudflats, we observed concomitant enzyme activation in saline conditions and the presence of highly charged, negative electrostatic protein surfaces [<a href=\"#B52-marinedrugs-18-00461\" rid=\"B52-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514675\">52</a>], as studied at the molecular level for a cellobiohydrolase (GH7) from the marine wood borer <em>Limnoria quadripunctata</em> [<a href=\"#B55-marinedrugs-18-00461\" rid=\"B55-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514714\">55</a>]. Other hemicellulases were identified, such as the feruloyl esterase (APMZ2_prot7619 produced in both carbon sources without salt) as the two other hemicellulases candidates (APMZ2_prot17211 and APMZ2_prot29026) that were identified only on wheat straw cultures without salt.</p><p id=\"__p23\">Cellulase activities of <em>S. lucomagnoense</em> were rather low compared to other marine-derived isolates screened in our study. Only one and four members of families GH1 and GH3, enzymes accessory to cellulose degradation, respectively, were identified in secretomes. No representatives of GH families 5, 6, 7, and 12 were found, unlike usually seen with terrestrial fungi. Four members were only found in cultures without salt of <em>S. lucomagnoense</em> grown on wheat straw, and one (APMZ2_prot1520) from both substrates, meaning that the cellulolytic ability of the fungus is limited or that these families contain cellulases never described before. Considering the second hypothesis, the cloning of the corresponding genes and the characterization of the recombinant enzymes will be required. In line with these data, cellulase activity was mainly detected in cultures without sea salt on wheat straw, and less, on seagrass only. Although a similarly low cellulase activity was found in cultures on wheat straw in saline conditions, no candidate cellulase was identified in the secretome. This possibly points to the sensitivity limit of our combined transcriptomic-proteomic approach, because of undetected transcript domains, as previously hypothesized in our previous study [<a href=\"#B12-marinedrugs-18-00461\" rid=\"B12-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514661\">12</a>].</p><p id=\"__p24\">Although laccase-like activity was detected in <em>S. lucomagnoense</em> secretomes from cultures grown on either wheat straw or seagrass, in any secretome, neither laccase-like (AA1) nor peroxidase-like (AA2) enzymes, also able to oxidize ABTS, were identified. This finding suggests that the oxidase responsible for ABTS oxidation was undetected, maybe for the reason just cited above as several transcripts were identified as encoding putative laccases in <em>S. lucomagnoense</em> transcriptome. Analyzing fungal secretion of Auxiliary Activities, the number of candidates was higher in wheat-containing cultures, but a large part was also found in seagrass-containing cultures. The number of identified proteins was reduced in saline conditions, but still, more than 50% of them were found, suggesting that <em>S. lucomagnoense</em> can cope with marine mimicking conditions. Something similar was also seen in the work of Arfi et al. [<a href=\"#B12-marinedrugs-18-00461\" rid=\"B12-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514724\">12</a>] on the mangrove fungus <em>Pestalotiopsis sp</em>. In that work, we only tested the effect of saline conditions on fungal physiology, while in our study, we combined the salt and the carbon substrate effects. Two enzymes, APMZ2_prot912 (family 5_1, CRO) and APMZ2_prot4629 (family AA12, pyrroloquinoline-quinone (PQQ)-dependant oxidoreductase), were found in all conditions tested at pretty high yields. Recently, it was shown that a (PQQ)-dependent pyranose dehydrogenase (family AA12) from <em>Coprinopsis cinerea</em> was able to activate a family LPMO from <em>Neurospora crassa</em> [<a href=\"#B35-marinedrugs-18-00461\" rid=\"B35-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514649\">35</a>]. One candidate LPMO (APMZ2_prot21080, family AA11), was also identified in <em>S. lucomagnoense</em> cultures grown on wheat straw, both in the presence and absence of sea salt. Although further investigation will be necessary to demonstrate the potential synergy between the two <em>S. lucomagnoense</em> enzymes, like for terrestrial fungi, fungal cellulose degradation seems to rely on LMPOs, which are copper oxidases classified among the AA section of the CAZy database. In a previous study, we demonstrated that an LPMO was produced from <em>Pestalotiopsis</em> sp. NCi6 was able to cleave polymeric cellulose in the presence of up to 6% sea salt [<a href=\"#B56-marinedrugs-18-00461\" rid=\"B56-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514695\">56</a>]. Salt resistant enzymes are interesting potential biocatalysts as they might achieve saccharification and other industrial processes in seawater, with improved ecological footprints.</p><p id=\"__p25\">Focusing on family AA3_2 (aryl alcohol and glucose oxidases and dehydrogenases), we can notice different enzymatic profiles in different growth conditions: APMZ2_prot27112 is only produced on wheat straw, with or without salt; APMZ2_prot24323 is secreted on seagrass without salt, as well as on wheat straw with salt; APMZ2_prot2229 only on wheat straw without salt. This diversity in the enzymatic response of <em>S. lucomagnoense</em> within a specific enzyme family illustrates a putative adaptation to the growth conditions: Nature of the carbon source and physico-chemical conditions, such as salt. More detailed studies will be necessary for in-depth understanding of this adaptation at the enzyme level.</p><p id=\"__p26\">The four most secreted proteins (APMZ2_prot13181, 912, 15973, 2472) were produced in all the conditions investigated. They correspond to GH55 (&#x003b2;-1,3-glucanase), probably involved in fungal morphology, GH15-CBM20 (glucoamylase fused to a starch binding module), and GH13_1 (&#x003b2;-amylase). Similar enzymatic activities were already found to be highly expressed in other secretomes, such as described for the coprophilous ascomycete <em>Podospora anserina,</em> cultivated on wheat straw, Avicel, sugar beet pulp, and for the corn pathogen <em>Fusarium verticillioides</em> [<a href=\"#B57-marinedrugs-18-00461\" rid=\"B57-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514727\">57</a>,<a href=\"#B58-marinedrugs-18-00461\" rid=\"B58-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514705\">58</a>]. These enzymatic activities were instead absent in secretomes from the white-rot basidiomycete <em>Phanerochaete chrysosporium</em> [<a href=\"#B59-marinedrugs-18-00461\" rid=\"B59-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514681\">59</a>]. Although further experiments will be needed to understand their role in <em>S. lucomagnoense</em> physiology, these constitutively secreted proteins might play a central metabolic role. One protein only was found for both substrates exclusively in saline conditions: APMZ2_prot21161, the less abundant enzyme revealed by our proteomic analysis, a putative alginate lyase (CAZy subfamily PL7_4). Such an enzymatic activity was already demonstrated to be key for the marine-derived fungus <em>Paradendryphiella salina</em> to degrade several types of brown algae polysaccharides [<a href=\"#B20-marinedrugs-18-00461\" rid=\"B20-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514728\">20</a>], despite a minimal CAZyme repertoire, which is a typical feature of saprobic and plant pathogenic ascomycetes. In a previous work, we also found four putative PL7_4 genes in the genome of a marine-derived fungus, <em>Calcarisporium</em> sp. KF525, also pointing to a possible capacity to degrade algal biomass [<a href=\"#B15-marinedrugs-18-00461\" rid=\"B15-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514692\">15</a>]. For comparison purposes, the marine-derived fungus <em>Pestalotiopsis</em> sp. KF079, isolated in the same environment [<a href=\"#B15-marinedrugs-18-00461\" rid=\"B15-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514713\">15</a>], and <em>Scopulariopsis brevicaulis</em> [<a href=\"#B13-marinedrugs-18-00461\" rid=\"B13-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514655\">13</a>] adopted instead a different and larger CAZy repertoire, enabling the breakdown of terrestrial plant biomass.</p><p id=\"__p27\" class=\"p p-last\">In conclusion, our work highlights the potential of proteomic analysis of marine-derived fungi to gain insight into microbial metabolism and adaptation to the marine environment. To achieve this goal, further characterization of several novel enzymes identified in <em>S. lucomagnoense</em> secretomes will be necessary, i.e., cloning and overexpressing the related genes in appropriate hosts, characterizing the recombinant enzymes from the kinetics to the 3D-structure, and elucidating the molecular determinants responsible for the properties of these enzymes. Finally, the results of our study contribute to more distant fields of research, such as the study of the carbon cycle and carbon sequestration, the understanding of microbial communities, and the discovery of novel biocatalysts for industrial applications.</p></span><span id=\"s\n4-marinedrugs-18-00461\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\n4-marinedrugs-18-00461title\">4. Materials and Methods</h2><span id=\"s\n4dot1-marinedrugs-18-00461\" class=\"sec sec-first\"><h3 id=\"s\n4dot1-marinedrugs-18-00461title\">4.1. Radial Growth Rate Determination</h3><p id=\"__p28\" class=\"p p-first-last\">The specific radial growth rate of five isolated fungal strains (<em>T. asperellum</em>1, <em>T. asperellum</em>2, <em>T. asperellum</em>3, <em>S. lucomagnoense,</em> and <em>A. nidulans</em>) was determined on Petri dishes, by inoculating 6 mm plugs obtained from one-month-old fungal pre-cultures grown in Vogel&#x02019;s medium [<a href=\"#B60-marinedrugs-18-00461\" rid=\"B60-marinedrugs-18-00461\" class=\" bibr popnode\">60</a>] into new plates containing the same medium supplemented with either 2% (<em>w</em>/<em>v</em>) carboxymethycellulose (CMC) (Sigma-Aldrich, Saint-Louis, MO, USA) or 2% (<em>w</em>/<em>v</em>) xylan (Sigma-Aldrich, Saint-Louis, MO, USA) as sole extra carbon sources. The effect on the growth of saline conditions was determined by adding 3% (<em>w</em>/<em>v</em>) sea salt to the growth medium of half of the samples. The growth rate at 30 &#x000b0;C was expressed as cm/day. The diameter of the colony around the inoculation plug was measured every 24 h for 10 days. Experiments were performed in triplicate for subsequent statistical analysis of data.</p></span><span id=\"s\n4dot2-marinedrugs-18-00461\" class=\"sec\"><h3 id=\"s\n4dot2-marinedrugs-18-00461title\">4.2. Screening of Lignocellulolytic Activities on Wheat Straw and Seagrass</h3><p id=\"__p29\" class=\"p p-first\">The culture supernatant of five strains grown on RHI medium (containing per liter: 18 g Bacto Agar, 1 g malt extract, 1 g yeast extract, and 40 g substrate) [<a href=\"#B12-marinedrugs-18-00461\" rid=\"B12-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514699\">12</a>] at pH 5.5 was used to follow the presence of lignocellulolytic enzymes in fungal secretomes grown on two different plant biomasses. Wheat straw was used as the terrestrial reference substrate, because it is an important agro-industrial residue and because it was previously demonstrated to be a good inducer of lignocellulolytic enzymes [<a href=\"#B57-marinedrugs-18-00461\" rid=\"B57-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514722\">57</a>]. Seagrass (<em>Cymodocea nodosa</em>) was the chosen marine substrate, as it was found in the same area where the marine strains were collected. The two plant substrates were washed with distilled water and then kept for 24 h in a 70 &#x000b0;C oven for drying, then grinded using an analytical mill (IKA A11 basic, Sigma-Aldricht, Saint-Louis, MO, USA) and the resulting fragments were then sieved with a metal mesh for a size between 0.2 and 0.8 mm. They were finally sterilized twice for 1 h at 110 &#x000b0;C. Each strain was inoculated in three 250 mL baffled Erlenmeyers (three biological replicates) containing 50 mL of medium and incubated at 30 &#x000b0;C in the dark under agitation at 50 rpm (Infors, Massy, France). The inoculum for each Erlenmeyer consisted of two agar plugs (6 mm diameter), cut from the growing edge of a plate stock culture, homogenized in 1 mL of water in Lysing Matrix A tubes with a FastPrep Instrument (MP Biomedicals, Illkirch-Graffenstaden, France) at 4 m s<sup>&#x02212;1</sup> for 15 s. Depending on the experiment, the &#x02018;RHI&#x02019; medium was supplemented with 3% (<em>w</em>/<em>v</em>) of sea salt (Sigma-Aldrich, Saint-Louis, MO, USA).</p><p id=\"__p30\" class=\"p p-last\">Enzymatic activities were assessed by collecting 1 mL of each culture supernatant every 2 days over 12 days. After centrifugation at 10,000&#x000d7; <em>g</em> for 5 min, the supernatant was collected and used in several enzymatic assays (two technical replicates). For each culture condition, the measurement was performed in triplicates (biological replicates). The laccase-like activity was tested by monitoring the oxidation of 2,2&#x02032;-azino-bis (3-ethylbenzothiazoline-6-sulphonic acid) (ABTS; Sigma-Aldrich, Saint-Louis, MO, USA). Twenty microliters of culture supernatant were added to 130 &#x000b5;L of 0.1% ABTS in 100 mM citrate-phosphate buffer pH 5, and optical density was measured at 420 nm after 30 min of incubation at 30 &#x000b0;C. Endo-1,4-&#x003b2;-<span class=\"small-caps\">d</span>-glucanase (endo-cellulase) (Megazyme, Libios, Vindry-sur-Turdine, France) and endo-1,4-&#x003b2;-<span class=\"small-caps\">d</span>-xylanase (endo-xylanase) (Megazyme, Libios, Vindry-sur-Turdine, France) activities were assessed by measuring, respectively, the hydrolysis of 1% (<em>w</em>/<em>v</em>) azo-CMC (Megazyme, Libios, Vindry-sur-Turdine, France) and 1% (<em>w</em>/<em>v</em>) azo-xylan (birchwood, Megazyme, Libios, Vindry-sur-Turdine, France) in water. Twenty microliters of culture supernatant were added to 130 &#x000b5;L of each azo-substrate solution and incubated 30 min at 30 &#x000b0;C. 375 &#x000b5;L of 95% (<em>v</em>/<em>v</em>) ethanol was then added, in order to stop the enzymatic reaction by protein denaturation/precipitation, and after centrifugation at 5000&#x000d7; <em>g</em> for 10 min, the corresponding supernatant was collected and optical density measured at 590 nm.</p></span><span id=\"s\n4dot3-marinedrugs-18-00461\" class=\"sec\"><h3 id=\"s\n4dot3-marinedrugs-18-00461title\">4.3. Statistical Analysis of Data</h3><p id=\"__p31\" class=\"p p-first-last\">The statistical analysis of data on radial growth rate was carried out using the linear regression model in R to compute the radial growth rate and its associated 95% confidence interval. The statistical analysis on cellulase, xylanase, and laccase-like activities were carried out with non- parametric Kruskal&#x02013;Wallis test [<a href=\"#B61-marinedrugs-18-00461\" rid=\"B61-marinedrugs-18-00461\" class=\" bibr popnode\">61</a>] (<em>p</em> &#x0003c; 0.05), followed by post hoc Dunn pairwise comparison test [<a href=\"#B62-marinedrugs-18-00461\" rid=\"B62-marinedrugs-18-00461\" class=\" bibr popnode\">62</a>] when differences were previously found significant using R version 3.6.3 and Rstudio version 1.2.5.033.</p></span><span id=\"s\n4dot4-marinedrugs-18-00461\" class=\"sec\"><h3 id=\"s\n4dot4-marinedrugs-18-00461title\">4.4. RNA Extraction, cDNA Library Construction, Sequencing, Assembling and Annotation</h3><p id=\"__p32\" class=\"p p-first\"><em>S. lucomagnoense</em> was cultivated in 50 mL of liquid medium RHI, with or without 3% (<em>w</em>/<em>v</em>) sea salt, on either wheat straw or seagrass, for 10 days at 30 &#x000b0;C and 110 rpm. For each condition, 18 baffled Erlenmeyers were inoculated with 1 mL of homogenized mycelium from a 5-day-old pre-culture grown on liquid malt medium (malt extract 18 gL<sup>&#x02212;1</sup> in water) at 30 &#x000b0;C. Every second day, 3 Erlenmeyers for each condition were sacrificed. All the samples were pooled, and the resulting mycelia were carefully collected with a spatula to separate them from the culture supernatant. The supernatant was pooled for further proteomic analysis. RNA was extracted from the mycelia using the TRIzol reagent (Invitrogen, ThermoFisher Scientific, Madison, WI, USA) according to the manufacturer&#x02019;s instructions. After purification, total RNA was quantified using a NanoDrop spectrophotometer (ThermoFisher Scientific, Madison, WI, USA) and further purified by LiCl precipitation. RNA samples from all sampling days were pooled to collect the total transcriptome corresponding to the 10-day cultures, and quality was assessed by capillary electrophoresis using a BioAnalyzer (Agilent, Santa Clara, CA, USA).</p><p id=\"__p33\" class=\"p p-last\">The pooled RNA sample was sent to Fasteris SA (Geneva, Switzerland) to prepare cDNA libraries and sequence it. Briefly, mRNAs were purified from total RNA using the Dynabeads mRNA purification kit (Invitrogen, ThermoFisher Scientific, Madison, WI, USA), and the DNA library was prepared using the Illumina TruSeq Stranded mRNA Library Prep kit (Illumina, Evry, France). The resulting library was then normalized using the Duplex Specific Nuclease protocol (Evrogen, Euromedex, Souffelweyersheim, France) and sequenced using the Illumina HiSeq-4000 technology (2 &#x000d7; 150 bp) (Illumina, San Diego, CA, USA). Sequencing reads were assembled using the de novo assembler suite of CLCBio Genomic workbench version 11 (<a href=\"https://digitalinsights.qiagen.com\">https://digitalinsights.qiagen.com</a>), with default parameters for cDNA. Transcripts were converted into open reading frames and proteins by TransDecoder (<a href=\"https://github.com/TransDecoder/TransDecoder\">https://github.com/TransDecoder/TransDecoder</a>). Identical proteins were clustered with a clustering cut-off of 90%.</p></span><span id=\"s\n4dot5-marinedrugs-18-00461\" class=\"sec\"><h3 id=\"s\n4dot5-marinedrugs-18-00461title\">4.5. Preparation of Stemphylium Lucomagnoense Secretomes</h3><p id=\"__p34\" class=\"p p-first-last\">For each culture condition (wheat straw or seagrass, with or without sea salt), the supernatants collected for three biological replicates as described in the previous section were centrifuged at 8000 rpm for 10 min at 4 &#x000b0;C and were filtered in three steps on glass microfiber filters through 2.7, 1.6 and 0.7 &#x003bc;m (GF/D, A and F filters, respectively, GE Healthcare Life Sciences, Whatman<sup>TM</sup>, ThermoFisher Scientific, Madison, WI, USA) and then on 0.4 and 0.2 &#x003bc;m PES membranes (Acrodisc<sup>&#x000ae;</sup>, Pall Corporation, Saint-Germain-en-Laye, France). Fifteen milliliters of each culture condition were concentrated three times using 10 kDa cut-off Vivaspin concentrators (Sartorius, Les Ulis, France) and then dialyzed against 50 mM sodium acetate buffer pH 5. The protein concentration in the obtained secretome samples was determined by the Bradford method [<a href=\"#B63-marinedrugs-18-00461\" rid=\"B63-marinedrugs-18-00461\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_885514733\">63</a>] using bovine serum albumin (BSA) as a standard.</p></span><span id=\"s\n4dot6-marinedrugs-18-00461\" class=\"sec sec-last\"><h3 id=\"s\n4dot6-marinedrugs-18-00461title\">4.6. Proteomic Analysis of Secretomes</h3><p id=\"__p35\" class=\"p p-first-last\">Short migrations (0.5 cm) of the secretome samples (previous section, 15 &#x003bc;g total protein) were performed by SDS-PAGE on pre-casted polyacrylamide gels (4 to 12% Bis-Tris Mini Gels, Invitrogen, ThermoFisher Scientific, Madison, WI, USA). Gels were stained with Coomassie Brilliant Blue (BioRad, Marnes-la-Coquette, France), and each one-dimensional electrophoresis lane was cut into two (2 mm wide) slices of gel. Protein identification was performed at the PAPPSO platform facility (&#x0201c;Plate-forme d&#x02019;Analyse Prot&#x000e9;omique de Paris Sud-Ouest&#x0201d;, France). In-gel digestion was carried out according to a standard trypsinolysis protocol. Gel pieces were washed twice with 10% acetic acid in 40% ethanol, and then with 50 mM sodium bicarbonate (NH<sub>4</sub>HCO<sub>3</sub>) in 50% acetonitrile (ACN) and incubated in the presence of 10 mM dithiothreitol (DTT) for 30 min at 56 &#x000b0;C. After cooling, the supernatant was removed, and the samples were incubated with 55 mM iodoacetamide at room temperature in the dark for 1 h. Digestion was performed for 8 h at 37 &#x000b0;C with 200 ng of modified trypsin (Promega, Charbonni&#x000e8;res-les-Bains, France) dissolved in 50 mM NH<sub>4</sub>CO<sub>3</sub>. Tryptic peptides were first extracted with 50% (<em>v</em>/<em>v</em>) can, 0.5% (<em>v</em>/<em>v</em>) trifluoroacetic acid (TFA), and then with pure ACN. Peptide extracts were dried in a vacuum speed concentrator (Thermo Fisher Scientific, Villebon sur Yvette, France) and suspended in 20 &#x003bc;L of 2% (<em>v</em>/<em>v</em>) ACN and 0.08% (<em>v</em>/<em>v</em>) TFA. HPLC was performed on an Ultimate 3000 RSLC system (Thermo Fisher Scientific, Madison, WI, USA). Trypsic digestion products were first concentrated and desalted on a pre-column cartridge (PepMap 100 C18, 0.3 &#x000d7; 5 mm, Dionex, Thermo Fisher Scientific, Madison, WI, USA) with 0,08% TFA in 2% ACN at 20 &#x003bc;L/min for 3 min. The pre-column cartridge was connected to the separating column (C18, 0.075 &#x000d7; 500 mm, Thermo Fisher Scientific, Madison, WI, USA), and the peptides were eluted with a linear gradient from 5 to 35% ACN in 0.1% HCOOH for 155 min at 300 nL/min. On-line analysis of peptides was performed with a Tribrid orbitrap fusion lumos mass spectrometer (Thermo Fisher Scientific, Madison, WI, USA), using a nanoelectrospray ion source. Peptide ions were analyzed using Xcalibur 2.1 (Thermo Scientific, Villebon sur Yvette, France) with the following data-dependent acquisition steps: (step 1) full MS scan (mass-to-charge ratio (m/z) 400 to 1400, resolution 70,000) and (step 2) MS/MS (normalized collision energy = 30%, resolution 15,000). The raw mass data were first converted to mzXML format with the MS convert software (ProteoWizard v 3.0.8934) (ProteoWizard, Palo Alto, CA, USA). Protein identification was performed querying MS/MS data against databases, together with an in-house contaminant database, using the X!Tandem pipeline software (version 0.2.34) with the following parameters: One trypsin missed cleavage allowed the alkylation of cysteine and conditional oxidation of methionine, precursor and fragment ion set at 10 ppm and 5 Da, respectively. A refined search was added with similar parameters, except that semi-tryptic peptides, possible N-term acetylation, and histidine mono- and dimethylations were searched. All peptides matched with an E-value lower than 0.05 were parsed with X!Tandem pipeline software. Proteins identified with at least two unique peptides and a log (E-value) lower than &#x02212;2.6 were validated.</p></span></span><span id=\"app1-marinedrugs-18-00461\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"app1-marinedrugs-18-00461title\">Supplementary Materials</h2><!--/article/back/app-group/app/--><p id=\"__p36\">The following are available online at <a href=\"https://www.mdpi.com/1660-3397/18/9/461/s1\">https://www.mdpi.com/1660-3397/18/9/461/s1</a>, Table S1: Statistical analysis results.</p><span class=\"sec suppmat\" id=\"marinedrugs-18-00461-s001\"><span class=\"sup-box half_rhythm\" id=\"idm140580395482592\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7551824/bin/marinedrugs-18-00461-s001.pdf\" data-ga-action=\"click_feat_suppl\">Click here for additional data file.</a><sup>(179K, pdf)</sup></span></span></span><span id=\"idm140580395480864\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140580395480864title\">Author Contributions</h2><p id=\"__p38\">Conceptualization, T.M. and E.R.; Methodology D.N., A.K., A.T.-D., L.O.C., E.B., and S.B.; Software, A.K. and E.D.; Investigation, W.B.A., D.N., A.T.-D., L.O.C., S.B.; Data Curation, W.B.A., A.K., B.H., E.D. and E.R.; Writing &#x00026; original draft preparation, W.B.A. and E.R.; Writing, review &#x00026; editing, A.K., E.B., C.B.F., G.S., B.H., T.M. and E.R.; Supervision T.M. and E.R.; Funding Acquisition, T.M. and E.R. All authors have read and agreed to the published version of the manuscript.</p></span><span id=\"idm140580395479376\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140580395479376title\">Funding</h2><p id=\"__p39\">This research was funded by PHC-Utique (grant number: 39274UH 2018-2020, 18G0817) for financial support. AK is recipient of Ramalingaswami Re-Retry Faculty Fellowship (Grant; BT/RLF/Re-entry/38/2017).</p></span><span id=\"idm140580395478128\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140580395478128title\">Conflicts of Interest</h2><p id=\"__p40\">The authors declare there is no conflict of interest.</p></span><br><div class=\"ref-list-sec sec\" id=\"reference-list\"><div class=\"ref-cit-blk half_rhythm\" id=\"B1-marinedrugs-18-00461\">1. <span class=\"element-citation\">Grosberg R.K., Vermeij G.J., Wainwright P.C. Biodiversity in water and on land. <span><span class=\"ref-journal\">Curr. Biol. </span>2012;<span class=\"ref-vol\">22</span>:R900–R903. doi: 10.1016/j.cub.2012.09.050.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n23137680\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.cub.2012.09.050\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Curr.+Biol.&amp;title=Biodiversity+in+water+and+on+land&amp;author=R.K.+Grosberg&amp;author=G.J.+Vermeij&amp;author=P.C.+Wainwright&amp;volume=22&amp;publication_year=2012&amp;pages=R900-R903&amp;pmid=23137680&amp;doi=10.1016/j.cub.2012.09.050&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B2-marinedrugs-18-00461\">2. <span class=\"element-citation\">Kohlmeyer J., Kohlmeyer E. Marine fungi from tropical America and Africa. <span><span class=\"ref-journal\">Mycologia. </span>1971;<span class=\"ref-vol\">63</span>:831–861. doi: 10.1080/00275514.1971.12019172.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n4939076\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1080%2F00275514.1971.12019172\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mycologia&amp;title=Marine+fungi+from+tropical+America+and+Africa&amp;author=J.+Kohlmeyer&amp;author=E.+Kohlmeyer&amp;volume=63&amp;publication_year=1971&amp;pages=831-861&amp;pmid=4939076&amp;doi=10.1080/00275514.1971.12019172&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B3-marinedrugs-18-00461\">3. <span class=\"element-citation\">Amend A., Burgaud G., Cunliffe M., Edgcomb V.P., Ettinger C.L., Guti&#x000e9;rrez M.H., Heitman J., Hom E.F.Y., Ianiri G., Jones A.C., et al.  Fungi in the marine environment: Open questions and unsolved problems. <span><span class=\"ref-journal\">mBio. </span>2019;<span class=\"ref-vol\">10</span>:e01189-18. doi: 10.1128/mBio.01189-18.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC6401481/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30837337\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FmBio.01189-18\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Fungi+in+the+marine+environment:+Open+questions+and+unsolved+problems&amp;author=A.+Amend&amp;author=G.+Burgaud&amp;author=M.+Cunliffe&amp;author=V.P.+Edgcomb&amp;author=C.L.+Ettinger&amp;volume=10&amp;publication_year=2019&amp;pages=e01189-18&amp;pmid=30837337&amp;doi=10.1128/mBio.01189-18&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B4-marinedrugs-18-00461\">4. <span class=\"element-citation\">Schoch C.L., Crous P.W., Groenewald J.Z., Boehm E.W.A., Burgess T.I., De Gruyter J., De Hoog G.S., Dixon L.J., Grube M., Gueidan C. A class-wide phylogenetic assessment of Dothideomycetes. <span><span class=\"ref-journal\">Stud. Mycol. </span>2009;<span class=\"ref-vol\">64</span>:1–15. doi: 10.3114/sim.2009.64.01.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC2816964/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n20169021\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3114%2Fsim.2009.64.01\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Stud.+Mycol.&amp;title=A+class-wide+phylogenetic+assessment+of+Dothideomycetes&amp;author=C.L.+Schoch&amp;author=P.W.+Crous&amp;author=J.Z.+Groenewald&amp;author=E.W.A.+Boehm&amp;author=T.I.+Burgess&amp;volume=64&amp;publication_year=2009&amp;pages=1-15&amp;pmid=20169021&amp;doi=10.3114/sim.2009.64.01&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B5-marinedrugs-18-00461\">5. <span class=\"element-citation\">Jones E.G.  Marine fungi and fungal-like organisms. In: Pang K.-L., editor. <span class=\"ref-journal\">Marine and Freshwater Botany.</span> Walter De Gruyter Incorporated; Berlin, Germany: 2012.  <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Marine+and+Freshwater+Botany&amp;author=E.G.+Jones&amp;publication_year=2012&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B6-marinedrugs-18-00461\">6. <span class=\"element-citation\">Kohlmeyer J., Volkmann-Kohlmeyer B. Fungi from coral reefs: A commentary. <span><span class=\"ref-journal\">Mycol. Res. </span>2003;<span class=\"ref-vol\">107</span>:385–387. doi: 10.1017/S0953756203227775.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n12825508\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1017%2FS0953756203227775\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mycol.+Res.&amp;title=Fungi+from+coral+reefs:+A+commentary&amp;author=J.+Kohlmeyer&amp;author=B.+Volkmann-Kohlmeyer&amp;volume=107&amp;publication_year=2003&amp;pages=385-387&amp;doi=10.1017/S0953756203227775&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B7-marinedrugs-18-00461\">7. <span class=\"element-citation\">Li Q., Wang G. Diversity of fungal isolates from three Hawaiian marine sponges. <span><span class=\"ref-journal\">Microbiol Res. </span>2009;<span class=\"ref-vol\">164</span>:233–241. doi: 10.1016/j.micres.2007.07.002.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n17681460\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.micres.2007.07.002\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Microbiol+Res.&amp;title=Diversity+of+fungal+isolates+from+three+Hawaiian+marine+sponges&amp;author=Q.+Li&amp;author=G.+Wang&amp;volume=164&amp;publication_year=2009&amp;pages=233-241&amp;pmid=17681460&amp;doi=10.1016/j.micres.2007.07.002&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B8-marinedrugs-18-00461\">8. <span class=\"element-citation\">Jones E.G. Are there more marine fungi to be described? <span><span class=\"ref-journal\">Bot. Mar. </span>2011;<span class=\"ref-vol\">54</span>:343–354. doi: 10.1515/bot.2011.043.</span> [<a href=\"//dx.doi.org/10.1515%2Fbot.2011.043\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bot.+Mar.&amp;title=Are+there+more+marine+fungi+to+be+described?&amp;author=E.G.+Jones&amp;volume=54&amp;publication_year=2011&amp;pages=343-354&amp;doi=10.1515/bot.2011.043&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B9-marinedrugs-18-00461\">9. <span class=\"element-citation\">Poli A., Finore I., Romano I., Gioiello A., Lama L., Nicolaus B. Microbial diversity in extreme marine habitats and their biomolecules. <span><span class=\"ref-journal\">Microorganisms. </span>2017;<span class=\"ref-vol\">5</span>:25.  doi: 10.3390/microorganisms5020025.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC5488096/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n28509857\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3390%2Fmicroorganisms5020025\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Microorganisms&amp;title=Microbial+diversity+in+extreme+marine+habitats+and+their+biomolecules&amp;author=A.+Poli&amp;author=I.+Finore&amp;author=I.+Romano&amp;author=A.+Gioiello&amp;author=L.+Lama&amp;volume=5&amp;publication_year=2017&amp;pages=25&amp;doi=10.3390/microorganisms5020025&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B10-marinedrugs-18-00461\">10. <span class=\"element-citation\">Bonugli-Santos R.C., Dos Santos Vasconcelos M.R., Passarini M.R.Z., Vieira G.A.L., Lopes V.C.P., Mainardi P.H., Dos Santos J.A., de Azevedo Duarte L., Otero I.V.R., da Silva Yoshida A.M., et al.  Marine-derived fungi: Diversity of enzymes and biotechnological applications. <span><span class=\"ref-journal\">Front. Microbiol. </span>2015;<span class=\"ref-vol\">6</span>:269. doi: 10.3389/fmicb.2015.00269.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC4392690/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n25914680\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3389%2Ffmicb.2015.00269\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front.+Microbiol.&amp;title=Marine-derived+fungi:+Diversity+of+enzymes+and+biotechnological+applications&amp;author=R.C.+Bonugli-Santos&amp;author=M.R.+Dos+Santos+Vasconcelos&amp;author=M.R.Z.+Passarini&amp;author=G.A.L.+Vieira&amp;author=V.C.P.+Lopes&amp;volume=6&amp;publication_year=2015&amp;pages=269&amp;pmid=25914680&amp;doi=10.3389/fmicb.2015.00269&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B11-marinedrugs-18-00461\">11. <span class=\"element-citation\">Batista-Garc&#x000ed;a R.A., Sutton T., Jackson S.A., Tovar-Herrera O.E., Balc&#x000e1;zar-L&#x000f3;pez E., del Rayo Sanchez-Carbente M., S&#x000e1;nchez-Reyes A., Dobson A.D., Folch-Mallol J.L. Characterization of lignocellulolytic activities from fungi isolated from the deep-sea sponge <em>Stelletta normani</em>. <span><span class=\"ref-journal\">PLoS ONE. </span>2017;<span class=\"ref-vol\">12</span>:e0173750.  doi: 10.1371/journal.pone.0173750.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC5365110/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n28339473\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0173750\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Characterization+of+lignocellulolytic+activities+from+fungi+isolated+from+the+deep-sea+sponge+Stelletta+normani&amp;author=R.A.+Batista-Garc&#x000ed;a&amp;author=T.+Sutton&amp;author=S.A.+Jackson&amp;author=O.E.+Tovar-Herrera&amp;author=E.+Balc&#x000e1;zar-L&#x000f3;pez&amp;volume=12&amp;publication_year=2017&amp;pages=e0173750&amp;pmid=28339473&amp;doi=10.1371/journal.pone.0173750&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B12-marinedrugs-18-00461\">12. <span class=\"element-citation\">Arfi Y., Chevret D., Henrissat B., Berrin J.-G., Levasseur A., Record E. Characterization of salt-adapted secreted lignocellulolytic enzymes from the mangrove fungus <em>Pestalotiopsis</em> sp. <span><span class=\"ref-journal\">Nat. Commun. </span>2013;<span class=\"ref-vol\">4</span>:1810. doi: 10.1038/ncomms2850.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n23651998\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fncomms2850\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Characterization+of+salt-adapted+secreted+lignocellulolytic+enzymes+from+the+mangrove+fungus+Pestalotiopsis+sp.&amp;author=Y.+Arfi&amp;author=D.+Chevret&amp;author=B.+Henrissat&amp;author=J.-G.+Berrin&amp;author=A.+Levasseur&amp;volume=4&amp;publication_year=2013&amp;pages=1810&amp;pmid=23651998&amp;doi=10.1038/ncomms2850&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B13-marinedrugs-18-00461\">13. <span class=\"element-citation\">Kumar A., Henrissat B., Arvas M., Syed M.F., Thieme N., Benz J.P., S&#x000f8;rensen J.L., Record E., P&#x000f6;ggeler S., Kempken F. <em>De novo</em> assembly and genome analyses of the marine-derived <em>Scopulariopsis brevicaulis</em> strain LF580 unravels life-style traits and anticancerous scopularide biosynthetic gene cluster. <span><span class=\"ref-journal\">PLoS ONE. </span>2015;<span class=\"ref-vol\">10</span>:e0140398.  doi: 10.1371/journal.pone.0140398.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC4624724/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n26505484\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0140398\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=De+novo+assembly+and+genome+analyses+of+the+marine-derived+Scopulariopsis+brevicaulis+strain+LF580+unravels+life-style+traits+and+anticancerous+scopularide+biosynthetic+gene+cluster&amp;author=A.+Kumar&amp;author=B.+Henrissat&amp;author=M.+Arvas&amp;author=M.F.+Syed&amp;author=N.+Thieme&amp;volume=10&amp;publication_year=2015&amp;pages=e0140398&amp;pmid=26505484&amp;doi=10.1371/journal.pone.0140398&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B14-marinedrugs-18-00461\">14. <span class=\"element-citation\">R&#x000e9;dou V., Kumar A., Hainaut M., Henrissat B., Record E., Barbier G., Burgaud G. Draft genome sequence of the deep-sea ascomycetous filamentous fungus <em>Cadophora malorum</em> Mo12 from the Mid-Atlantic Ridge reveals its biotechnological potential. <span><span class=\"ref-journal\">Genome Announc. </span>2016;<span class=\"ref-vol\">4</span>:e00467-16. doi: 10.1128/genomeA.00467-16.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC4939777/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n27389260\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FgenomeA.00467-16\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Announc.&amp;title=Draft+genome+sequence+of+the+deep-sea+ascomycetous+filamentous+fungus+Cadophora+malorum+Mo12+from+the+Mid-Atlantic+Ridge+reveals+its+biotechnological+potential&amp;author=V.+R&#x000e9;dou&amp;author=A.+Kumar&amp;author=M.+Hainaut&amp;author=B.+Henrissat&amp;author=E.+Record&amp;volume=4&amp;publication_year=2016&amp;pages=e00467-16&amp;pmid=27389260&amp;doi=10.1128/genomeA.00467-16&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B15-marinedrugs-18-00461\">15. <span class=\"element-citation\">Kumar A., S&#x000f8;rensen J.L., Hansen F.T., Arvas M., Syed M.F., Hassan L., Benz J.P., Record E., Henrissat B., P&#x000f6;ggeler S. Genome sequencing and analyses of two marine fungi from the North Sea unraveled a plethora of novel biosynthetic gene clusters. <span><span class=\"ref-journal\">Sci. Rep. </span>2018;<span class=\"ref-vol\">8</span>:10187. doi: 10.1038/s41598-018-28473-z.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC6033941/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n29976990\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41598-018-28473-z\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=Genome+sequencing+and+analyses+of+two+marine+fungi+from+the+North+Sea+unraveled+a+plethora+of+novel+biosynthetic+gene+clusters&amp;author=A.+Kumar&amp;author=J.L.+S&#x000f8;rensen&amp;author=F.T.+Hansen&amp;author=M.+Arvas&amp;author=M.F.+Syed&amp;volume=8&amp;publication_year=2018&amp;pages=10187&amp;pmid=29976990&amp;doi=10.1038/s41598-018-28473-z&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B16-marinedrugs-18-00461\">16. <span class=\"element-citation\">Couturier M., Navarro D., Oliv&#x000e9; C., Chevret D., Haon M., Favel A., Lesage-Meessen L., Henrissat B., Coutinho P.M., Berrin J.-G. Post-genomic analyses of fungal lignocellulosic biomass degradation reveal the unexpected potential of the plant pathogen <em>Ustilago maydis</em>. <span><span class=\"ref-journal\">BMC Genom. </span>2012;<span class=\"ref-vol\">13</span>:57.  doi: 10.1186/1471-2164-13-57.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3298532/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n22300648\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2F1471-2164-13-57\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Genom.&amp;title=Post-genomic+analyses+of+fungal+lignocellulosic+biomass+degradation+reveal+the+unexpected+potential+of+the+plant+pathogen+Ustilago+maydis&amp;author=M.+Couturier&amp;author=D.+Navarro&amp;author=C.+Oliv&#x000e9;&amp;author=D.+Chevret&amp;author=M.+Haon&amp;volume=13&amp;publication_year=2012&amp;pages=57&amp;doi=10.1186/1471-2164-13-57&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B17-marinedrugs-18-00461\">17. <span class=\"element-citation\">Ettinger C.L., Eisen J.A. Characterization of the mycobiome of the seagrass, Zostera marina, reveals putative associations with marine chytrids. <span><span class=\"ref-journal\">Front. Microbiol. </span>2019;<span class=\"ref-vol\">10</span>:2476. doi: 10.3389/fmicb.2019.02476.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC6842960/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n31749781\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3389%2Ffmicb.2019.02476\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front.+Microbiol.&amp;title=Characterization+of+the+mycobiome+of+the+seagrass,+Zostera+marina,+reveals+putative+associations+with+marine+chytrids&amp;author=C.L.+Ettinger&amp;author=J.A.+Eisen&amp;volume=10&amp;publication_year=2019&amp;pages=2476&amp;pmid=31749781&amp;doi=10.3389/fmicb.2019.02476&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B18-marinedrugs-18-00461\">18. <span class=\"element-citation\">Trevathan-Tackett S.M., Thomson A.C.G., Ralph P.J., Macreadie P.I. Fresh carbon inputs to seagrass sediments induce variable microbial priming responses. <span><span class=\"ref-journal\">Sci. Total Environ. </span>2018;<span class=\"ref-vol\">621</span>:663–669. doi: 10.1016/j.scitotenv.2017.11.193.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n29197285\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.scitotenv.2017.11.193\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Sci.+Total+Environ.&amp;title=Fresh+carbon+inputs+to+seagrass+sediments+induce+variable+microbial+priming+responses&amp;author=S.M.+Trevathan-Tackett&amp;author=A.C.G.+Thomson&amp;author=P.J.+Ralph&amp;author=P.I.+Macreadie&amp;volume=621&amp;publication_year=2018&amp;pages=663-669&amp;pmid=29197285&amp;doi=10.1016/j.scitotenv.2017.11.193&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B19-marinedrugs-18-00461\">19. <span class=\"element-citation\">Hurtado-McCormick V., Kahlke T., Petrou K., Jeffries T., Ralph P.J., Seymour J.R. Regional and microenvironmental scale characterization of the <em>Zostera muelleri</em> seagrass microbiome. <span><span class=\"ref-journal\">Fron. Microbiol. </span>2019;<span class=\"ref-vol\">10</span>:1011. doi: 10.3389/fmicb.2019.01011.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC6527750/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n31139163\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3389%2Ffmicb.2019.01011\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Fron.+Microbiol.&amp;title=Regional+and+microenvironmental+scale+characterization+of+the+Zostera+muelleri+seagrass+microbiome&amp;author=V.+Hurtado-McCormick&amp;author=T.+Kahlke&amp;author=K.+Petrou&amp;author=T.+Jeffries&amp;author=P.J.+Ralph&amp;volume=10&amp;publication_year=2019&amp;pages=1011&amp;doi=10.3389/fmicb.2019.01011&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B20-marinedrugs-18-00461\">20. <span class=\"element-citation\">Pilgaard B., Wilkens C., Herbst F.-A., Vuillemin M., Rhein-Knudsen N., Meyer A.S., Lange L. Proteomic enzyme analysis of the marine fungus <em>Paradendryphiella salina</em> reveals alginate lyase as a minimal adaptation strategy for brown algae degradation. <span><span class=\"ref-journal\">Sci. Rep. </span>2019;<span class=\"ref-vol\">9</span>:12338. doi: 10.1038/s41598-019-48823-9.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC6710412/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n31451726\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41598-019-48823-9\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=Proteomic+enzyme+analysis+of+the+marine+fungus+Paradendryphiella+salina+reveals+alginate+lyase+as+a+minimal+adaptation+strategy+for+brown+algae+degradation&amp;author=B.+Pilgaard&amp;author=C.+Wilkens&amp;author=F.-A.+Herbst&amp;author=M.+Vuillemin&amp;author=N.+Rhein-Knudsen&amp;volume=9&amp;publication_year=2019&amp;pages=12338&amp;pmid=31451726&amp;doi=10.1038/s41598-019-48823-9&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B21-marinedrugs-18-00461\">21. <span class=\"element-citation\">Orsi W.D., Richards T.A., Francis W.R. Predicted microbial secretomes and their target substrates in marine sediment. <span><span class=\"ref-journal\">Nat. Microbiol. </span>2018;<span class=\"ref-vol\">3</span>:32–37. doi: 10.1038/s41564-017-0047-9.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n29062087\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41564-017-0047-9\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Microbiol.&amp;title=Predicted+microbial+secretomes+and+their+target+substrates+in+marine+sediment&amp;author=W.D.+Orsi&amp;author=T.A.+Richards&amp;author=W.R.+Francis&amp;volume=3&amp;publication_year=2018&amp;pages=32-37&amp;pmid=29062087&amp;doi=10.1038/s41564-017-0047-9&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B22-marinedrugs-18-00461\">22. <span class=\"element-citation\">Ben Ali W., Chaduli D., Navarro D., Lechat C., Turb&#x000e9;-Doan A., Bertrand E., Faulds C.B., Sciara G., Lesage-Meessen L., Record E., et al.  Screening of five marine-derived fungal strains for their potential to produce oxidases with laccase activities suitable for biotechnological applications. <span><span class=\"ref-journal\">BMC Biotechnol. </span>2020;<span class=\"ref-vol\">20</span>:27.  doi: 10.1186/s12896-020-00617-y.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7218534/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n32398071\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2Fs12896-020-00617-y\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Biotechnol.&amp;title=Screening+of+five+marine-derived+fungal+strains+for+their+potential+to+produce+oxidases+with+laccase+activities+suitable+for+biotechnological+applications&amp;author=W.+Ben+Ali&amp;author=D.+Chaduli&amp;author=D.+Navarro&amp;author=C.+Lechat&amp;author=A.+Turb&#x000e9;-Doan&amp;volume=20&amp;publication_year=2020&amp;pages=27&amp;pmid=32398071&amp;doi=10.1186/s12896-020-00617-y&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B23-marinedrugs-18-00461\">23. <span class=\"element-citation\">Adams D.J. Fungal cell wall chitinases and glucanases. <span><span class=\"ref-journal\">Microbiology. </span>2004;<span class=\"ref-vol\">150</span>:2029–2035. doi: 10.1099/mic.0.26980-0.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n15256547\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1099%2Fmic.0.26980-0\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Fungal+cell+wall+chitinases+and+glucanases&amp;author=D.J.+Adams&amp;volume=150&amp;publication_year=2004&amp;pages=2029-2035&amp;pmid=15256547&amp;doi=10.1099/mic.0.26980-0&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B24-marinedrugs-18-00461\">24. <span class=\"element-citation\">Davis L.L., Bartnicki-Garcia S. Chitosan synthesis by the tandem action of chitin synthetase and chitin deacetylase from <em>Mucor rouxii</em>. <span><span class=\"ref-journal\">Biochemistry. </span>1984;<span class=\"ref-vol\">23</span>:1065–1073. doi: 10.1021/bi00301a005.</span> [<a href=\"//dx.doi.org/10.1021%2Fbi00301a005\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Chitosan+synthesis+by+the+tandem+action+of+chitin+synthetase+and+chitin+deacetylase+from+Mucor+rouxii&amp;author=L.L.+Davis&amp;author=S.+Bartnicki-Garcia&amp;volume=23&amp;publication_year=1984&amp;pages=1065-1073&amp;doi=10.1021/bi00301a005&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B25-marinedrugs-18-00461\">25. <span class=\"element-citation\">Volk H., Marton K., Flaj&#x00161;man M., Radi&#x00161;ek S., Tian H., Hein I., Podlipnik &#x0010c;., Thomma B.P., Ko&#x00161;melj K., Javornik B. Chitin-binding protein of <em>Verticillium nonalfalfae</em> disguises fungus from plant chitinases and suppresses chitin-triggered host immunity. <span><span class=\"ref-journal\">Mol. Plant. Microbe Interact. </span>2019;<span class=\"ref-vol\">32</span>:1378–1390. doi: 10.1094/MPMI-03-19-0079-R.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n31063047\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1094%2FMPMI-03-19-0079-R\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Plant.+Microbe+Interact.&amp;title=Chitin-binding+protein+of+Verticillium+nonalfalfae+disguises+fungus+from+plant+chitinases+and+suppresses+chitin-triggered+host+immunity&amp;author=H.+Volk&amp;author=K.+Marton&amp;author=M.+Flaj&#x00161;man&amp;author=S.+Radi&#x00161;ek&amp;author=H.+Tian&amp;volume=32&amp;publication_year=2019&amp;pages=1378-1390&amp;pmid=31063047&amp;doi=10.1094/MPMI-03-19-0079-R&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B26-marinedrugs-18-00461\">26. <span class=\"element-citation\">Mewis K., Lenfant N., Lombard V., Henrissat B. Dividing the large glycoside hydrolase family 43 into subfamilies: A motivation for detailed enzyme characterization. <span><span class=\"ref-journal\">Appl. Environ. Microb. </span>2016;<span class=\"ref-vol\">82</span>:1686–1692. doi: 10.1128/AEM.03453-15.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC4784025/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n26729713\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.03453-15\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microb.&amp;title=Dividing+the+large+glycoside+hydrolase+family+43+into+subfamilies:+A+motivation+for+detailed+enzyme+characterization&amp;author=K.+Mewis&amp;author=N.+Lenfant&amp;author=V.+Lombard&amp;author=B.+Henrissat&amp;volume=82&amp;publication_year=2016&amp;pages=1686-1692&amp;pmid=26729713&amp;doi=10.1128/AEM.03453-15&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B27-marinedrugs-18-00461\">27. <span class=\"element-citation\">Sakka M., Yamada K., Kitamura T., Kunitake E., Kimura T., Sakka K. The modular arabinanolytic enzyme Abf43A-Abf43B-Abf43C from <em>Ruminiclostridium josui</em> consists of three GH43 modules classified in different subfamilies. <span><span class=\"ref-journal\">Enzyme Microb. Tech. </span>2019;<span class=\"ref-vol\">124</span>:23–31. doi: 10.1016/j.enzmictec.2019.01.011.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30797476\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.enzmictec.2019.01.011\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Enzyme+Microb.+Tech.&amp;title=The+modular+arabinanolytic+enzyme+Abf43A-Abf43B-Abf43C+from+Ruminiclostridium+josui+consists+of+three+GH43+modules+classified+in+different+subfamilies&amp;author=M.+Sakka&amp;author=K.+Yamada&amp;author=T.+Kitamura&amp;author=E.+Kunitake&amp;author=T.+Kimura&amp;volume=124&amp;publication_year=2019&amp;pages=23-31&amp;pmid=30797476&amp;doi=10.1016/j.enzmictec.2019.01.011&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B28-marinedrugs-18-00461\">28. <span class=\"element-citation\">Whittaker M.M., Kersten P.J., Cullen D., Whittaker J.W. Identification of catalytic residues in glyoxal oxidase by targeted mutagenesis. <span><span class=\"ref-journal\">J. Biol. Chem. </span>1999;<span class=\"ref-vol\">274</span>:36226–36232. doi: 10.1074/jbc.274.51.36226.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n10593910\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1074%2Fjbc.274.51.36226\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Identification+of+catalytic+residues+in+glyoxal+oxidase+by+targeted+mutagenesis&amp;author=M.M.+Whittaker&amp;author=P.J.+Kersten&amp;author=D.+Cullen&amp;author=J.W.+Whittaker&amp;volume=274&amp;publication_year=1999&amp;pages=36226-36232&amp;pmid=10593910&amp;doi=10.1074/jbc.274.51.36226&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B29-marinedrugs-18-00461\">29. <span class=\"element-citation\">Daou M., Piumi F., Cullen D., Record E., Faulds C.B. Heterologous production and characterization of two glyoxal oxidases from <em>Pycnoporus cinnabarinus</em>. <span><span class=\"ref-journal\">Appl. Environ. Microb. </span>2016;<span class=\"ref-vol\">82</span>:4867–4875. doi: 10.1128/AEM.00304-16.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC4968546/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n27260365\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.00304-16\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microb.&amp;title=Heterologous+production+and+characterization+of+two+glyoxal+oxidases+from+Pycnoporus+cinnabarinus&amp;author=M.+Daou&amp;author=F.+Piumi&amp;author=D.+Cullen&amp;author=E.+Record&amp;author=C.B.+Faulds&amp;volume=82&amp;publication_year=2016&amp;pages=4867-4875&amp;doi=10.1128/AEM.00304-16&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B30-marinedrugs-18-00461\">30. <span class=\"element-citation\">Whittaker M.M., Kersten P.J., Nakamura N., San ders-Loehr J., Schweizer E.S., Whittaker J.W. Glyoxal oxidase from <em>Phanerochaete chrysosporium</em> is a new radical-copper oxidase. <span><span class=\"ref-journal\">J. Biol. Chem. </span>1996;<span class=\"ref-vol\">271</span>:681–687. doi: 10.1074/jbc.271.2.681.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n8557673\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1074%2Fjbc.271.2.681\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Glyoxal+oxidase+from+Phanerochaete+chrysosporium+is+a+new+radical-copper+oxidase&amp;author=M.M.+Whittaker&amp;author=P.J.+Kersten&amp;author=N.+Nakamura&amp;author=J.+San+ders-Loehr&amp;author=E.S.+Schweizer&amp;volume=271&amp;publication_year=1996&amp;pages=681-687&amp;pmid=8557673&amp;doi=10.1074/jbc.271.2.681&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B31-marinedrugs-18-00461\">31. <span class=\"element-citation\">Wymelenberg A.V., Sabat G., Mozuch M., Kersten P.J., Cullen D., Blanchette R.A. Structure, organization, and transcriptional regulation of a family of copper radical oxidase genes in the lignin-degrading basidiomycete <em>Phanerochaete chrysosporium</em>. <span><span class=\"ref-journal\">Appl. Environ. Microb. </span>2006;<span class=\"ref-vol\">72</span>:4871–4877. doi: 10.1128/AEM.00375-06.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC1489383/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n16820482\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.00375-06\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microb.&amp;title=Structure,+organization,+and+transcriptional+regulation+of+a+family+of+copper+radical+oxidase+genes+in+the+lignin-degrading+basidiomycete+Phanerochaete+chrysosporium&amp;author=A.V.+Wymelenberg&amp;author=G.+Sabat&amp;author=M.+Mozuch&amp;author=P.J.+Kersten&amp;author=D.+Cullen&amp;volume=72&amp;publication_year=2006&amp;pages=4871-4877&amp;doi=10.1128/AEM.00375-06&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B32-marinedrugs-18-00461\">32. <span class=\"element-citation\">Mathieu Y., Piumi F., Valli R., Aramburu J.C., Ferreira P., Faulds C.B., Record E. Activities of secreted aryl alcohol quinone oxidoreductases from <em>Pycnoporus cinnabarinus</em> provide insights into fungal degradation of plant biomass. <span><span class=\"ref-journal\">Appl. Environ. Microb. </span>2016;<span class=\"ref-vol\">82</span>:2411–2423. doi: 10.1128/AEM.03761-15.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC4959486/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n26873317\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.03761-15\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microb.&amp;title=Activities+of+secreted+aryl+alcohol+quinone+oxidoreductases+from+Pycnoporus+cinnabarinus+provide+insights+into+fungal+degradation+of+plant+biomass&amp;author=Y.+Mathieu&amp;author=F.+Piumi&amp;author=R.+Valli&amp;author=J.C.+Aramburu&amp;author=P.+Ferreira&amp;volume=82&amp;publication_year=2016&amp;pages=2411-2423&amp;pmid=26873317&amp;doi=10.1128/AEM.03761-15&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B33-marinedrugs-18-00461\">33. <span class=\"element-citation\">Turbe-Doan A., Record E., Lombard V., Kumar R., Levasseur A., Henrissat B., Garron M.-L. <em>Trichoderma reesei</em> dehydrogenase, a pyrroloquinoline quinone-dependent member of auxiliary activity family 12 of the carbohydrate-active Enzymes database: Functional and structural characterization. <span><span class=\"ref-journal\">Appl. Environ. Microb. </span>2019;<span class=\"ref-vol\">85</span>:e00964-19. doi: 10.1128/AEM.00964-19.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC6881810/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n31604773\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.00964-19\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microb.&amp;title=Trichoderma+reesei+dehydrogenase,+a+pyrroloquinoline+quinone-dependent+member+of+auxiliary+activity+family+12+of+the+carbohydrate-active+Enzymes+database:+Functional+and+structural+characterization&amp;author=A.+Turbe-Doan&amp;author=E.+Record&amp;author=V.+Lombard&amp;author=R.+Kumar&amp;author=A.+Levasseur&amp;volume=85&amp;publication_year=2019&amp;pages=e00964-19&amp;pmid=31604773&amp;doi=10.1128/AEM.00964-19&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B34-marinedrugs-18-00461\">34. <span class=\"element-citation\">Takeda K., Ishida T., Yoshida M., Samejima M., Ohno H., Igarashi K., Nakamura N. Crystal structure of the catalytic and cytochrome b domains in a eukaryotic pyrroloquinoline quinone-dependent dehydrogenase. <span><span class=\"ref-journal\">Appl. Environ. Microb. </span>2019;<span class=\"ref-vol\">85</span> doi: 10.1128/AEM.01692-19.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC6881789/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n31604769\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.01692-19\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microb.&amp;title=Crystal+structure+of+the+catalytic+and+cytochrome+b+domains+in+a+eukaryotic+pyrroloquinoline+quinone-dependent+dehydrogenase&amp;author=K.+Takeda&amp;author=T.+Ishida&amp;author=M.+Yoshida&amp;author=M.+Samejima&amp;author=H.+Ohno&amp;volume=85&amp;publication_year=2019&amp;pmid=31604769&amp;doi=10.1128/AEM.01692-19&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B35-marinedrugs-18-00461\">35. <span class=\"element-citation\">V&#x000e1;rnai A., Umezawa K., Yoshida M., Eijsink V.G. The pyrroloquinoline-quinone-dependent pyranose dehydrogenase from <em>Coprinopsis cinerea</em> drives lytic polysaccharide monooxygenase action. <span><span class=\"ref-journal\">Appl. Environ. Microb. </span>2018;<span class=\"ref-vol\">84</span>:e00156-18. doi: 10.1128/AEM.00156-18.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC5960967/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n29602785\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.00156-18\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microb.&amp;title=The+pyrroloquinoline-quinone-dependent+pyranose+dehydrogenase+from+Coprinopsis+cinerea+drives+lytic+polysaccharide+monooxygenase+action&amp;author=A.+V&#x000e1;rnai&amp;author=K.+Umezawa&amp;author=M.+Yoshida&amp;author=V.G.+Eijsink&amp;volume=84&amp;publication_year=2018&amp;pages=e00156-18&amp;doi=10.1128/AEM.00156-18&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B36-marinedrugs-18-00461\">36. <span class=\"element-citation\">Netea M.G., Joosten L.A.B., Latz E., Mills K.H.G., Natoli G., Stunnenberg H.G., O&#x02019;Neill L.A.J., Xavier R.J. Trained immunity: A program of innate immune memory in health and disease. <span><span class=\"ref-journal\">Science. </span>2016;<span class=\"ref-vol\">352</span>:aaf1098. doi: 10.1126/science.aaf1098.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC5087274/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n27102489\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1126%2Fscience.aaf1098\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Trained+immunity:+A+program+of+innate+immune+memory+in+health+and+disease&amp;author=M.G.+Netea&amp;author=L.A.B.+Joosten&amp;author=E.+Latz&amp;author=K.H.G.+Mills&amp;author=G.+Natoli&amp;volume=352&amp;publication_year=2016&amp;pages=aaf1098&amp;pmid=27102489&amp;doi=10.1126/science.aaf1098&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B37-marinedrugs-18-00461\">37. <span class=\"element-citation\">Garajova S., Mathieu Y., Beccia M.R., Bennati-Granier C., Biaso F., Fanuel M., Ropartz D., Guigliarelli B., Record E., Rogniaux H. Single-domain flavoenzymes trigger lytic polysaccharide monooxygenases for oxidative degradation of cellulose. <span><span class=\"ref-journal\">Sci. Rep. </span>2016;<span class=\"ref-vol\">6</span>:28276. doi: 10.1038/srep28276.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC4911613/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n27312718\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fsrep28276\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=Single-domain+flavoenzymes+trigger+lytic+polysaccharide+monooxygenases+for+oxidative+degradation+of+cellulose&amp;author=S.+Garajova&amp;author=Y.+Mathieu&amp;author=M.R.+Beccia&amp;author=C.+Bennati-Granier&amp;author=F.+Biaso&amp;volume=6&amp;publication_year=2016&amp;pages=28276&amp;pmid=27312718&amp;doi=10.1038/srep28276&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B38-marinedrugs-18-00461\">38. <span class=\"element-citation\">Hemsworth G.R., Henrissat B., Davies G.J., Walton P.H. Discovery and characterization of a new family of lytic polysaccharide monooxygenases. <span><span class=\"ref-journal\">Nat. Chem. Biol. </span>2014;<span class=\"ref-vol\">10</span>:122–126. doi: 10.1038/nchembio.1417.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC4274766/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24362702\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnchembio.1417\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Chem.+Biol.&amp;title=Discovery+and+characterization+of+a+new+family+of+lytic+polysaccharide+monooxygenases&amp;author=G.R.+Hemsworth&amp;author=B.+Henrissat&amp;author=G.J.+Davies&amp;author=P.H.+Walton&amp;volume=10&amp;publication_year=2014&amp;pages=122-126&amp;pmid=24362702&amp;doi=10.1038/nchembio.1417&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B39-marinedrugs-18-00461\">39. <span class=\"element-citation\">Tao Y., Xie B., Yang Z., Chen Z., Chen B., Deng Y., Jiang Y., van Peer A.F. Identification and expression analysis of a new glycoside hydrolase family 55 exo-&#x003b2;-1,3-glucanase-encoding gene in <em>Volvariella volvacea</em> suggests a role in fruiting body development. <span><span class=\"ref-journal\">Gene. </span>2013;<span class=\"ref-vol\">527</span>:154–160. doi: 10.1016/j.gene.2013.05.071.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n23751305\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.gene.2013.05.071\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Identification+and+expression+analysis+of+a+new+glycoside+hydrolase+family+55+exo-&#x003b2;-1,3-glucanase-encoding+gene+in+Volvariella+volvacea+suggests+a+role+in+fruiting+body+development&amp;author=Y.+Tao&amp;author=B.+Xie&amp;author=Z.+Yang&amp;author=Z.+Chen&amp;author=B.+Chen&amp;volume=527&amp;publication_year=2013&amp;pages=154-160&amp;pmid=23751305&amp;doi=10.1016/j.gene.2013.05.071&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B40-marinedrugs-18-00461\">40. <span class=\"element-citation\">Mouyna I., Hartl L., Latg&#x000e9; J.-P. &#x003b2;-1,3-glucan modifying enzymes in <em>Aspergillus fumigatus</em>. <span><span class=\"ref-journal\">Front. Microbiol. </span>2013;<span class=\"ref-vol\">4</span>:81. doi: 10.3389/fmicb.2013.00081.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3627985/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n23616783\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3389%2Ffmicb.2013.00081\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front.+Microbiol.&amp;title=&#x003b2;-1,3-glucan+modifying+enzymes+in+Aspergillus+fumigatus&amp;author=I.+Mouyna&amp;author=L.+Hartl&amp;author=J.-P.+Latg&#x000e9;&amp;volume=4&amp;publication_year=2013&amp;pages=81&amp;pmid=23616783&amp;doi=10.3389/fmicb.2013.00081&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B41-marinedrugs-18-00461\">41. <span class=\"element-citation\">Jiang C., Song J., Cong H., Zhang J., Yang Q. Expression and characterization of a novel antifungal Exo-&#x003b2;-1,3-glucanase from <em>Chaetomium cupreum</em>. <span><span class=\"ref-journal\">Appl. Biochem. Biotech. </span>2017;<span class=\"ref-vol\">182</span>:261–275. doi: 10.1007/s12010-016-2325-z.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n27854040\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs12010-016-2325-z\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Biochem.+Biotech.&amp;title=Expression+and+characterization+of+a+novel+antifungal+Exo-&#x003b2;-1,3-glucanase+from+Chaetomium+cupreum&amp;author=C.+Jiang&amp;author=J.+Song&amp;author=H.+Cong&amp;author=J.+Zhang&amp;author=Q.+Yang&amp;volume=182&amp;publication_year=2017&amp;pages=261-275&amp;doi=10.1007/s12010-016-2325-z&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B42-marinedrugs-18-00461\">42. <span class=\"element-citation\">Becker S., Tebben J., Coffinet S., Wiltshire K., Iversen M.H., Harder T., Hinrichs K.-U., Hehemann J.-H. Laminarin is a major molecule in the marine carbon cycle. <span><span class=\"ref-journal\">Proc. Natl. Acad. Sci. USA. </span>2020;<span class=\"ref-vol\">117</span>:6599–6607. doi: 10.1073/pnas.1917001117.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC7104365/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n32170018\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1073%2Fpnas.1917001117\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Laminarin+is+a+major+molecule+in+the+marine+carbon+cycle&amp;author=S.+Becker&amp;author=J.+Tebben&amp;author=S.+Coffinet&amp;author=K.+Wiltshire&amp;author=M.H.+Iversen&amp;volume=117&amp;publication_year=2020&amp;pages=6599-6607&amp;pmid=32170018&amp;doi=10.1073/pnas.1917001117&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B43-marinedrugs-18-00461\">43. <span class=\"element-citation\">Burtseva Y.V., Sova V., Mikhail P., Zvyagintseva T. Enzymes of carbohydrate metabolism of mycelial fungi from marine environments. &#x003b2;-l,3-glucanase of the marine fungus <em>Chaetomium indicum</em>. <span><span class=\"ref-journal\">Biochemistry. </span>2000;<span class=\"ref-vol\">65</span>:1175–1183.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n11092961\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Enzymes+of+carbohydrate+metabolism+of+mycelial+fungi+from+marine+environments.+&#x003b2;-l,3-glucanase+of+the+marine+fungus+Chaetomium+indicum&amp;author=Y.V.+Burtseva&amp;author=V.+Sova&amp;author=P.+Mikhail&amp;author=T.+Zvyagintseva&amp;volume=65&amp;publication_year=2000&amp;pages=1175-1183&amp;pmid=11092961&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B44-marinedrugs-18-00461\">44. <span class=\"element-citation\">Suriya J., Bharathiraja S., Krishnan M., Manivasagan P., Kim S.-K. Chapter Eight&#x02014;Marine Microbial Amylases: Properties and Applications. <span><span class=\"ref-journal\">Adv. Food Nutr. Res. </span>2016;<span class=\"ref-vol\">79</span>:161–177.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n27770860\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Adv.+Food+Nutr.+Res.&amp;title=Chapter+Eight&#x02014;Marine+Microbial+Amylases:+Properties+and+Applications&amp;author=J.+Suriya&amp;author=S.+Bharathiraja&amp;author=M.+Krishnan&amp;author=P.+Manivasagan&amp;author=S.-K.+Kim&amp;volume=79&amp;publication_year=2016&amp;pages=161-177&amp;pmid=27770860&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B45-marinedrugs-18-00461\">45. <span class=\"element-citation\">Wang Y., Barth D., Tamminen A., Wiebe M.G. Growth of marine fungi on polymeric substrates. <span><span class=\"ref-journal\">BMC Biotechnol. </span>2016;<span class=\"ref-vol\">16</span>:3.  doi: 10.1186/s12896-016-0233-5.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC4715362/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n26772742\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2Fs12896-016-0233-5\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Biotechnol.&amp;title=Growth+of+marine+fungi+on+polymeric+substrates&amp;author=Y.+Wang&amp;author=D.+Barth&amp;author=A.+Tamminen&amp;author=M.G.+Wiebe&amp;volume=16&amp;publication_year=2016&amp;pages=3&amp;pmid=26772742&amp;doi=10.1186/s12896-016-0233-5&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B46-marinedrugs-18-00461\">46. <span class=\"element-citation\">Korkmaz M.N., Ozdemir S.C., Uzel A. Xylanase production from marine derived <em>Trichoderma pleuroticola</em> 08&#x000c7;K001 strain isolated from mediterranean coastal sediments. <span><span class=\"ref-journal\">J. Basic. Microb. </span>2017;<span class=\"ref-vol\">57</span>:839–851. doi: 10.1002/jobm.201700135.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n28758291\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1002%2Fjobm.201700135\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Basic.+Microb.&amp;title=Xylanase+production+from+marine+derived+Trichoderma+pleuroticola+08&#x000c7;K001+strain+isolated+from+mediterranean+coastal+sediments&amp;author=M.N.+Korkmaz&amp;author=S.C.+Ozdemir&amp;author=A.+Uzel&amp;volume=57&amp;publication_year=2017&amp;pages=839-851&amp;pmid=28758291&amp;doi=10.1002/jobm.201700135&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B47-marinedrugs-18-00461\">47. <span class=\"element-citation\">Zeng S., Ling J., Ahmad M., Lin L., Zhang Y., Wang C. Culturable fungal diversity and cellulase production by mixed culture <em>Aspergillus</em> fungi from Sanya mangrove. <span><span class=\"ref-journal\">J. Gen. Appl. Microbiol. </span>2018;<span class=\"ref-vol\">64</span>:293–298. doi: 10.2323/jgam.2018.02.004.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30158368\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.2323%2Fjgam.2018.02.004\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Gen.+Appl.+Microbiol.&amp;title=Culturable+fungal+diversity+and+cellulase+production+by+mixed+culture+Aspergillus+fungi+from+Sanya+mangrove&amp;author=S.+Zeng&amp;author=J.+Ling&amp;author=M.+Ahmad&amp;author=L.+Lin&amp;author=Y.+Zhang&amp;volume=64&amp;publication_year=2018&amp;pages=293-298&amp;pmid=30158368&amp;doi=10.2323/jgam.2018.02.004&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B48-marinedrugs-18-00461\">48. <span class=\"element-citation\">Theerachat M., Guieysse D., Morel S., Remaud-Sim&#x000e9;on M., Chulalaksananukul W. Laccases from marine organisms and their applications in the biodegradation of toxic and environmental pollutants: A review. <span><span class=\"ref-journal\">Appl. Biochem. Biotech. </span>2019;<span class=\"ref-vol\">187</span>:583–611. doi: 10.1007/s12010-018-2829-9.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30009326\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs12010-018-2829-9\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Biochem.+Biotech.&amp;title=Laccases+from+marine+organisms+and+their+applications+in+the+biodegradation+of+toxic+and+environmental+pollutants:+A+review&amp;author=M.+Theerachat&amp;author=D.+Guieysse&amp;author=S.+Morel&amp;author=M.+Remaud-Sim&#x000e9;on&amp;author=W.+Chulalaksananukul&amp;volume=187&amp;publication_year=2019&amp;pages=583-611&amp;doi=10.1007/s12010-018-2829-9&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B49-marinedrugs-18-00461\">49. <span class=\"element-citation\">Bonugli-santos R.C., Durrant L.R., Sette L.D. Laccase activity and putative laccase genes in marine-derived basidiomycetes. <span><span class=\"ref-journal\">Fungal. Biol. </span>2010;<span class=\"ref-vol\">114</span>:863–872. doi: 10.1016/j.funbio.2010.08.003.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n20943196\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.funbio.2010.08.003\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Fungal.+Biol.&amp;title=Laccase+activity+and+putative+laccase+genes+in+marine-derived+basidiomycetes&amp;author=R.C.+Bonugli-santos&amp;author=L.R.+Durrant&amp;author=L.D.+Sette&amp;volume=114&amp;publication_year=2010&amp;pages=863-872&amp;pmid=20943196&amp;doi=10.1016/j.funbio.2010.08.003&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B50-marinedrugs-18-00461\">50. <span class=\"element-citation\">Bonugli-Santos R.C., Durrant L.R., da Silva M., Sette L.D. Production of laccase, manganese peroxidase and lignin peroxidase by Brazilian marine-derived fungi. <span><span class=\"ref-journal\">Enzym. Microb. Tech. </span>2010;<span class=\"ref-vol\">46</span>:32–37. doi: 10.1016/j.enzmictec.2009.07.014.</span> [<a href=\"//dx.doi.org/10.1016%2Fj.enzmictec.2009.07.014\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Enzym.+Microb.+Tech.&amp;title=Production+of+laccase,+manganese+peroxidase+and+lignin+peroxidase+by+Brazilian+marine-derived+fungi&amp;author=R.C.+Bonugli-Santos&amp;author=L.R.+Durrant&amp;author=M.+da+Silva&amp;author=L.D.+Sette&amp;volume=46&amp;publication_year=2010&amp;pages=32-37&amp;doi=10.1016/j.enzmictec.2009.07.014&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B51-marinedrugs-18-00461\">51. <span class=\"element-citation\">Atalla M.M., Zeinab H.K., Eman R.H., Amani A.Y., Abeer A.A. Characterization and kinetic properties of the purified <em>Trematosphaeria mangrovei</em> laccase enzyme. <span><span class=\"ref-journal\">Saudi J. Biol. Sci. </span>2013;<span class=\"ref-vol\">20</span>:373–381. doi: 10.1016/j.sjbs.2013.04.001.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3824134/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24235874\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.sjbs.2013.04.001\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Saudi+J.+Biol.+Sci.&amp;title=Characterization+and+kinetic+properties+of+the+purified+Trematosphaeria+mangrovei+laccase+enzyme&amp;author=M.M.+Atalla&amp;author=H.K.+Zeinab&amp;author=R.H.+Eman&amp;author=A.Y.+Amani&amp;author=A.A.+Abeer&amp;volume=20&amp;publication_year=2013&amp;pages=373-381&amp;pmid=24235874&amp;doi=10.1016/j.sjbs.2013.04.001&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B52-marinedrugs-18-00461\">52. <span class=\"element-citation\">Wikee S., Hatton J., Turb&#x000e9;-Doan A., Mathieu Y., Daou M., Lomascolo A., Kumar A., Lumyong S., Sciara G., Faulds C.B., et al.  Characterization and Dye Decolorization Potential of Two Laccases from the Marine-Derived Fungus <em>Pestalotiopsis</em> sp. <span><span class=\"ref-journal\">Int. J. Mol. Sci. </span>2019;<span class=\"ref-vol\">20</span>:1864.  doi: 10.3390/ijms20081864.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC6515530/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n30991752\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3390%2Fijms20081864\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int.+J.+Mol.+Sci.&amp;title=Characterization+and+Dye+Decolorization+Potential+of+Two+Laccases+from+the+Marine-Derived+Fungus+Pestalotiopsis+sp.&amp;author=S.+Wikee&amp;author=J.+Hatton&amp;author=A.+Turb&#x000e9;-Doan&amp;author=Y.+Mathieu&amp;author=M.+Daou&amp;volume=20&amp;publication_year=2019&amp;pages=1864&amp;pmid=30991752&amp;doi=10.3390/ijms20081864&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B53-marinedrugs-18-00461\">53. <span class=\"element-citation\">Woudenberg J.H.C., Hanse B., van Leeuwen G.C.M., Groenewald J.Z., Crous P.W. <em>Stemphylium</em> revisited. <span><span class=\"ref-journal\">Stud. Mycol. </span>2017;<span class=\"ref-vol\">87</span>:77–103. doi: 10.1016/j.simyco.2017.06.001.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC5480992/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n28663603\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.simyco.2017.06.001\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Stud.+Mycol.&amp;title=Stemphylium+revisited&amp;author=J.H.C.+Woudenberg&amp;author=B.+Hanse&amp;author=G.C.M.+van+Leeuwen&amp;author=J.Z.+Groenewald&amp;author=P.W.+Crous&amp;volume=87&amp;publication_year=2017&amp;pages=77-103&amp;pmid=28663603&amp;doi=10.1016/j.simyco.2017.06.001&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B54-marinedrugs-18-00461\">54. <span class=\"element-citation\">Lombard V., Golaconda Ramulu H., Drula E., Coutinho P.M., Henrissat B. The carbohydrate-active enzymes database (CAZy) in 2013. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2013;<span class=\"ref-vol\">42</span>:D490–D495. doi: 10.1093/nar/gkt1178.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3965031/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24270786\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgkt1178\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+carbohydrate-active+enzymes+database+(CAZy)+in+2013&amp;author=V.+Lombard&amp;author=H.+Golaconda+Ramulu&amp;author=E.+Drula&amp;author=P.M.+Coutinho&amp;author=B.+Henrissat&amp;volume=42&amp;publication_year=2013&amp;pages=D490-D495&amp;pmid=24270786&amp;doi=10.1093/nar/gkt1178&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B55-marinedrugs-18-00461\">55. <span class=\"element-citation\">Kern M., McGeehan J.E., Streeter S.D., Martin R.N.A., Besser K., Elias L., Eborall W., Malyon G.P., Payne C.M., Himmel M.E., et al.  Structural characterization of a unique marine animal family 7 cellobiohydrolase suggests a mechanism of cellulase salt tolerance. <span><span class=\"ref-journal\">Proc. Natl. Acad. Sci. USA. </span>2013;<span class=\"ref-vol\">110</span>:10189–10194. doi: 10.1073/pnas.1301502110.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC3690837/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n23733951\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1073%2Fpnas.1301502110\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Structural+characterization+of+a+unique+marine+animal+family+7+cellobiohydrolase+suggests+a+mechanism+of+cellulase+salt+tolerance&amp;author=M.+Kern&amp;author=J.E.+McGeehan&amp;author=S.D.+Streeter&amp;author=R.N.A.+Martin&amp;author=K.+Besser&amp;volume=110&amp;publication_year=2013&amp;pages=10189-10194&amp;pmid=23733951&amp;doi=10.1073/pnas.1301502110&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B56-marinedrugs-18-00461\">56. <span class=\"element-citation\">Patel I., Kracher D., Ma S., Garajova S., Haon M., Faulds C.B., Berrin J.-G., Ludwig R., Record E. Salt-responsive lytic polysaccharide monooxygenases from the mangrove fungus <em>Pestalotiopsis</em> sp. NCi6. <span><span class=\"ref-journal\">Biotechnol. Biofuels. </span>2016;<span class=\"ref-vol\">9</span>:108. doi: 10.1186/s13068-016-0520-3.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\n/PMC4875668/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n27213015\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2Fs13068-016-0520-3\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Biotechnol.+Biofuels&amp;title=Salt-responsive+lytic+polysaccharide+monooxygenases+from+the+mangrove+fungus+Pestalotiopsis+sp.+NCi6&amp;author=I.+Patel&amp;author=D.+Kracher&amp;author=S.+Ma&amp;author=S.+Garajova&amp;author=M.+Haon&amp;volume=9&amp;publication_year=2016&amp;pages=108&amp;pmid=27213015&amp;doi=10.1186/s13068-016-0520-3&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B57-marinedrugs-18-00461\">57. <span class=\"element-citation\">Poidevin L., Berrin J.-G., Bennati-Granier C., Levasseur A., Herpo&#x000eb;l-Gimbert I., Chevret D., Coutinho P.M., Henrissat B., Heiss-Blanquet S., Record E. Comparative analyses of <em>Podospora anserina</em> secretomes reveal a large array of lignocellulose-active enzymes. <span><span class=\"ref-journal\">Appl. Microbiol. Biot. </span>2014;<span class=\"ref-vol\">98</span>:7457–7469. doi: 10.1007/s00253-014-5698-3.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n24695830\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs00253-014-5698-3\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Microbiol.+Biot.&amp;title=Comparative+analyses+of+Podospora+anserina+secretomes+reveal+a+large+array+of+lignocellulose-active+enzymes&amp;author=L.+Poidevin&amp;author=J.-G.+Berrin&amp;author=C.+Bennati-Granier&amp;author=A.+Levasseur&amp;author=I.+Herpo&#x000eb;l-Gimbert&amp;volume=98&amp;publication_year=2014&amp;pages=7457-7469&amp;pmid=24695830&amp;doi=10.1007/s00253-014-5698-3&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B58-marinedrugs-18-00461\">58. <span class=\"element-citation\">Ravalason H., Grisel S., Chevret D., Favel A., Berrin J.-G., Sigoillot J.-C., Herpo&#x000eb;l-Gimbert I. <em>Fusarium verticillioides</em> secretome as a source of auxiliary enzymes to enhance saccharification of wheat straw. <span><span class=\"ref-journal\">Bioresour. Technol. </span>2012;<span class=\"ref-vol\">114</span>:589–596. doi: 10.1016/j.biortech.2012.03.009.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n22459963\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.biortech.2012.03.009\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioresour.+Technol.&amp;title=Fusarium+verticillioides+secretome+as+a+source+of+auxiliary+enzymes+to+enhance+saccharification+of+wheat+straw&amp;author=H.+Ravalason&amp;author=S.+Grisel&amp;author=D.+Chevret&amp;author=A.+Favel&amp;author=J.-G.+Berrin&amp;volume=114&amp;publication_year=2012&amp;pages=589-596&amp;pmid=22459963&amp;doi=10.1016/j.biortech.2012.03.009&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B59-marinedrugs-18-00461\">59. <span class=\"element-citation\">Ravalason H., Jan G., Moll&#x000e9; D., Pasco M., Coutinho P.M., Lapierre C., Pollet B., Bertaud F., Petit-Conil M., Grisel S., et al.  Secretome analysis of <em>Phanerochaete chrysosporium</em> strain CIRM-BRFM41 grown on softwood. <span><span class=\"ref-journal\">Appl. Microbiol. Biot. </span>2008;<span class=\"ref-vol\">80</span>:719. doi: 10.1007/s00253-008-1596-x.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n18654772\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs00253-008-1596-x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Microbiol.+Biot.&amp;title=Secretome+analysis+of+Phanerochaete+chrysosporium+strain+CIRM-BRFM41+grown+on+softwood&amp;author=H.+Ravalason&amp;author=G.+Jan&amp;author=D.+Moll&#x000e9;&amp;author=M.+Pasco&amp;author=P.M.+Coutinho&amp;volume=80&amp;publication_year=2008&amp;pages=719&amp;doi=10.1007/s00253-008-1596-x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B60-marinedrugs-18-00461\">60. <span class=\"element-citation\">Vogel H.J. A convenient growth medium for <em>Neurospora</em> (medium N) <span><span class=\"ref-journal\">Microb. Genet. Bull. </span>1956;<span class=\"ref-vol\">13</span>:42–43.</span> <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Microb.+Genet.+Bull.&amp;title=A+convenient+growth+medium+for+Neurospora+(medium+N)&amp;author=H.J.+Vogel&amp;volume=13&amp;publication_year=1956&amp;pages=42-43&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B61-marinedrugs-18-00461\">61. <span class=\"element-citation\">Kruskal W.H., Wallis W.A. Use of Ranks in One-Criterion Variance Analysis. <span><span class=\"ref-journal\">J. Am. Stat. Assoc. </span>1952;<span class=\"ref-vol\">47</span>:583–621. doi: 10.1080/01621459.1952.10483441.</span> [<a href=\"//dx.doi.org/10.1080%2F01621459.1952.10483441\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Am.+Stat.+Assoc.&amp;title=Use+of+Ranks+in+One-Criterion+Variance+Analysis&amp;author=W.H.+Kruskal&amp;author=W.A.+Wallis&amp;volume=47&amp;publication_year=1952&amp;pages=583-621&amp;doi=10.1080/01621459.1952.10483441&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B62-marinedrugs-18-00461\">62. <span class=\"element-citation\">Dunn O.J. Multiple comparisons using rank sums. <span><span class=\"ref-journal\">Technometrics. </span>1964;<span class=\"ref-vol\">6</span>:241–252. doi: 10.1080/00401706.1964.10490181.</span> [<a href=\"//dx.doi.org/10.1080%2F00401706.1964.10490181\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Technometrics&amp;title=Multiple+comparisons+using+rank+sums&amp;author=O.J.+Dunn&amp;volume=6&amp;publication_year=1964&amp;pages=241-252&amp;doi=10.1080/00401706.1964.10490181&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B63-marinedrugs-18-00461\">63. <span class=\"element-citation\">Bradford M.M. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. <span><span class=\"ref-journal\">Anal. Biochem. </span>1976;<span class=\"ref-vol\">72</span>:248–254. doi: 10.1016/0003-2697(76)90527-3.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\n942051\" ref=\"reftype=pubmed&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2F0003-2697(76)90527-3\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Anal.+Biochem.&amp;title=A+rapid+and+sensitive+method+for+the+quantitation+of+microgram+quantities+of+protein+utilizing+the+principle+of+protein-dye+binding&amp;author=M.M.+Bradford&amp;volume=72&amp;publication_year=1976&amp;pages=248-254&amp;pmid=942051&amp;doi=10.1016/0003-2697(76)90527-3&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7551824&amp;issue-id=366801&amp;journal-id=590&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div></div><br>', NULL, 6);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(11, 'pubmed', '32531032', 'Conserved white-rot enzymatic mechanism for wood decay in the Basidiomycota genus Pycnoporus.', 'White-rot (WR) fungi are pivotal decomposers of dead organic matter in forest ecosystems and typically use a large array of hydrolytic and oxidative enzymes to deconstruct lignocellulose. However, the extent of lignin and cellulose degradation may vary between species and wood type. Here, we combined comparative genomics, transcriptomics and secretome proteomics to identify conserved enzymatic signatures at the onset of wood-decaying activity within the Basidiomycota genus Pycnoporus. We observed a strong conservation in the genome structures and the repertoires of protein-coding genes across the four Pycnoporus species described to date, despite the species having distinct geographic distributions. We further analysed the early response of P. cinnabarinus, P. coccineus and P. sanguineus to diverse (ligno)-cellulosic substrates. We identified a conserved set of enzymes mobilized by the three species for breaking down cellulose, hemicellulose and pectin. The co-occurrence in the exo-proteomes of H2O2-producing enzymes with H2O2-consuming enzymes was a common feature of the three species, although each enzymatic partner displayed independent transcriptional regulation. Finally, cellobiose dehydrogenase-coding genes were systematically co-regulated with at least one AA9 lytic polysaccharide monooxygenase gene, indicative of enzymatic synergy in vivo. This study highlights a conserved core white-rot fungal enzymatic mechanism behind the wood-decaying process.', 2020, 'DNA research : an international journal for rapid publication of reports on genes and genomes', 'PMC7406137', 1, NULL, NULL, 3),
(12, 'pubmed', '31314937', 'Multi-omic analyses of exogenous nutrient bag decomposition by the black morel Morchella importuna reveal sustained carbon acquisition and transferring.', 'The black morel (Morchella importuna Kuo, O\'Donnell and Volk) was once an uncultivable wild mushroom, until the development of exogenous nutrient bag (ENB), making its agricultural production quite feasible and stable. To date, how the nutritional acquisition of the morel mycelium is fulfilled to trigger its fruiting remains unknown. To investigate the mechanisms involved in ENB decomposition, the genome of a cultivable morel strain (M. importuna SCYDJ1-A1) was sequenced and the genes coding for the decay apparatus were identified. Expression of the encoded carbohydrate-active enzymes (CAZymes) was then analyzed by metatranscriptomics and metaproteomics in combination with biochemical assays. The results show that a diverse set of hydrolytic and redox CAZymes secreted by the morel mycelium is the main force driving the substrate decomposition. Plant polysaccharides such as starch and cellulose present in ENB substrate (wheat grains plus rice husks) were rapidly degraded, whereas triglycerides were accumulated initially and consumed later. ENB decomposition led to a rapid increase in the organic carbon content in the surface soil of the mushroom bed, which was thereafter consumed during morel fruiting. In contrast to the high carbon consumption, no significant acquisition of nitrogen was observed. Our findings contribute to an increasingly detailed portrait of molecular features triggering morel fruiting.', 2019, 'Environmental microbiology', '', 2, NULL, NULL, 6),
(13, 'pubmed', '31275257', 'Investigating Host Microbiota Relationships Through Functional Metagenomics.', 'The human Intestinal mucus is formed by glycoproteins, the O- and N-linked glycans which constitute a crucial source of carbon for commensal gut bacteria, especially when deprived of dietary glycans of plant origin. In recent years, a dozen carbohydrate-active enzymes from cultivated mucin degraders have been characterized. But yet, considering the fact that uncultured species predominate in the human gut microbiota, these biochemical data are far from exhaustive. In this study, we used functional metagenomics to identify new metabolic pathways in uncultured bacteria involved in harvesting mucin glycans. First, we performed a high-throughput screening of a fosmid metagenomic library constructed from the ileum mucosa microbiota using chromogenic substrates. The screening resulted in the isolation of 124 clones producing activities crucial in the degradation of human O- and N-glycans, namely sialidases, β-D-N-acetyl-glucosaminidase, β-D-N-acetyl-galactosaminidase, and/or β-D-mannosidase. Thirteen of these clones were selected based on their diversified functional profiles and were further analyzed on a secondary screening. This step consisted of lectin binding assays to demonstrate the ability of the clones to degrade human intestinal mucus. In total, the structural modification of several mucin motifs, sialylated mucin ones in particular, was evidenced for nine clones. Sequencing their metagenomic loci highlighted complex catabolic pathways involving the complementary functions of glycan sensing, transport, hydrolysis, deacetylation, and deamination, which were sometimes associated with amino acid metabolism machinery. These loci are assigned to several  and  species highly prevalent and abundant in the gut microbiome and explain the metabolic flexibility of gut bacteria feeding both on dietary and human glycans.', 2019, 'Frontiers in microbiology', '', 1, NULL, NULL, 2),
(14, 'pubmed', '31257601', 'Comparative genomics reveals unique wood-decay strategies and fruiting body development in the Schizophyllaceae.', 'Agaricomycetes are fruiting body-forming fungi that produce some of the most efficient enzyme systems to degrade wood. Despite decades-long interest in their biology, the evolution and functional diversity of both wood-decay and fruiting body formation are incompletely known. We performed comparative genomic and transcriptomic analyses of wood-decay and fruiting body development in Auriculariopsis ampla and Schizophyllum commune (Schizophyllaceae), species with secondarily simplified morphologies, an enigmatic wood-decay strategy and weak pathogenicity to woody plants. The plant cell wall-degrading enzyme repertoires of Schizophyllaceae are transitional between those of white rot species and less efficient wood-degraders such as brown rot or mycorrhizal fungi. Rich repertoires of suberinase and tannase genes were found in both species, with tannases restricted to Agaricomycetes that preferentially colonize bark-covered wood, suggesting potential complementation of their weaker wood-decaying abilities and adaptations to wood colonization through the bark. Fruiting body transcriptomes revealed a high rate of divergence in developmental gene expression, but also several genes with conserved expression patterns, including novel transcription factors and small-secreted proteins, some of the latter which might represent fruiting body effectors. Taken together, our analyses highlighted novel aspects of wood-decay and fruiting body development in an important family of mushroom-forming fungi.', 2019, 'The New phytologist', '', 1, NULL, NULL, 2);

-- --------------------------------------------------------

--
-- Structure de la table `authors`
--

CREATE TABLE `authors` (
  `id_author` int(11) NOT NULL,
  `name` varchar(255) NOT NULL,
  `firstname` varchar(255) NOT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;

--
-- Déchargement des données de la table `authors`
--

INSERT INTO `authors` (`id_author`, `name`, `firstname`) VALUES
(1, 'Pineau', 'Camille'),
(2, 'Guschinskaya', 'Natalia'),
(3, 'Gonçalves', 'Isabelle'),
(4, 'Ruaudel', 'Florence'),
(5, 'Robert', 'Xavier'),
(6, 'Gouet', 'Patrice'),
(7, 'Ballut', 'Lionel'),
(8, 'Shevchik', 'Vladimir'),
(9, 'Cao', 'Lichuang'),
(10, 'Chen', 'Ran'),
(11, 'Huang', 'Xin'),
(12, 'Li', 'Shuifeng'),
(13, 'Zhang', 'Sufang'),
(14, 'Yang', 'Xiangpeng'),
(15, 'Qin', 'Zongmin'),
(16, 'Kong', 'Wei'),
(17, 'Xie', 'Wei'),
(18, 'Liu', 'Yuhuan'),
(19, 'Mroueh', 'Mohamed'),
(20, 'Aruanno', 'Marion'),
(21, 'Borne', 'Romain'),
(22, 'de', 'Philip'),
(23, 'Fierobe', 'Henri-Pierre'),
(24, 'Tardif', 'Chantal'),
(25, 'Pagès', 'Sandrine'),
(26, 'Krska', 'Daniel'),
(27, 'Larsbrink', 'Johan'),
(28, 'Hage', 'Hayat'),
(29, 'Miyauchi', 'Shingo'),
(30, 'Virágh', 'Máté'),
(31, 'Drula', 'Elodie'),
(32, 'Min', 'Byoungnam'),
(33, 'Chaduli', 'Delphine'),
(34, 'Navarro', 'David'),
(35, 'Favel', 'Anne'),
(36, 'Norest', 'Manon'),
(37, 'Lesage-Meessen', 'Laurence'),
(38, 'Bálint', 'Balázs'),
(39, 'Merényi', 'Zsolt'),
(40, 'de', 'Eugenio'),
(41, 'Morin', 'Emmanuelle'),
(42, 'Martínez', 'Angel'),
(43, 'Baldrian', 'Petr'),
(44, 'Štursová', 'Martina'),
(45, 'Martínez', 'María'),
(46, 'Novotny', 'Cenek'),
(47, 'Magnuson', 'Jon'),
(48, 'Spatafora', 'Joey'),
(49, 'Maurice', 'Sundy'),
(50, 'Pangilinan', 'Jasmyn'),
(51, 'Andreopoulos', 'Willian'),
(52, 'LaButti', 'Kurt'),
(53, 'Hundley', 'Hope'),
(54, 'Na', 'Hyunsoo'),
(55, 'Kuo', 'Alan'),
(56, 'Barry', 'Kerrie'),
(57, 'Lipzen', 'Anna'),
(58, 'Henrissat', 'Bernard'),
(59, 'Riley', 'Robert'),
(60, 'Ahrendt', 'Steven'),
(61, 'Nagy', 'László'),
(62, 'Grigoriev', 'Igor'),
(63, 'Martin', 'Francis'),
(64, 'Rosso', 'Marie-Noëlle'),
(65, 'Daou', 'Mariane'),
(66, 'Farfan', 'Soto'),
(67, 'Majira', 'Amel'),
(68, 'Cézard', 'Laurent'),
(69, 'Cottyn', 'Betty'),
(70, 'Pion', 'Florian'),
(71, 'Oliveira', 'Correia'),
(72, 'Record', 'Eric'),
(73, 'Raouche', 'Sana'),
(74, 'Baumberger', 'Stéphanie'),
(75, 'Faulds', 'Craig'),
(76, 'Ruiz-Dueñas', 'Francisco'),
(77, 'Barrasa', 'José'),
(78, 'Sánchez-García', 'Marisol'),
(79, 'Camarero', 'Susana'),
(80, 'Serrano', 'Ana'),
(81, 'Linde', 'Dolores'),
(82, 'Babiker', 'Rashid'),
(83, 'Ayuso-Fernández', 'Iván'),
(84, 'Pacheco', 'Remedios'),
(85, 'Padilla', 'Guillermo'),
(86, 'Ferreira', 'Patricia'),
(87, 'Barriuso', 'Jorge'),
(88, 'Kellner', 'Harald'),
(89, 'Castanera', 'Raúl'),
(90, 'Alfaro', 'Manuel'),
(91, 'Ramírez', 'Lucía'),
(92, 'Pisabarro', 'Antonio'),
(93, 'Andreopoulos', 'William'),
(94, 'Tritt', 'Andrew'),
(95, 'He', 'Guifen'),
(96, 'Yan', 'Mi'),
(97, 'Ng', 'Vivian'),
(98, 'Cullen', 'Daniel'),
(99, 'Hibbett', 'David'),
(100, 'Hess', 'Jaqueline'),
(101, 'Balasundaram', 'Sudhagar'),
(102, 'Bakkemo', 'Renee'),
(103, 'Högberg', 'Nils'),
(104, 'Eastwood', 'Daniel'),
(105, 'Skrede', 'Inger'),
(106, 'Kiss', 'Enikő'),
(107, 'Kohler', 'Annegret'),
(108, 'Andreopoulos', 'Bill'),
(109, 'Bonito', 'Gregory'),
(110, 'Buée', 'Marc'),
(111, 'Carver', 'Akiko'),
(112, 'Chen', 'Cindy'),
(113, 'Cichocki', 'Nicolas'),
(114, 'Clum', 'Alicia'),
(115, 'Culley', 'David'),
(116, 'Crous', 'Pedro'),
(117, 'Fauchery', 'Laure'),
(118, 'Girlanda', 'Mariangela'),
(119, 'Hayes', 'Richard'),
(120, 'Kéri', 'Zsófia'),
(121, 'Lombard', 'Vincent'),
(122, 'Maillard', 'François'),
(123, 'Murat', 'Claude'),
(124, 'Nolan', 'Matt'),
(125, 'Ohm', 'Robin'),
(126, 'Pereira', 'Maíra'),
(127, 'Perotto', 'Silvia'),
(128, 'Peter', 'Martina'),
(129, 'Pfister', 'Stephanie'),
(130, 'Sitrit', 'Yaron'),
(131, 'Stielow', 'J'),
(132, 'Szöllősi', 'Gergely'),
(133, 'Žifčáková', 'Lucia'),
(134, 'Spatafora', 'Joseph'),
(135, 'Tedersoo', 'Leho'),
(136, 'Vaario', 'Lu-Min'),
(137, 'Yamada', 'Akiyoshi'),
(138, 'Wang', 'Pengfei'),
(139, 'Xu', 'Jianping'),
(140, 'Bruns', 'Tom'),
(141, 'Vilgalys', 'Rytas'),
(142, 'Dunand', 'Christophe'),
(143, 'Ben', 'Ali'),
(144, 'Kumar', 'Abhishek'),
(145, 'Turbé-Doan', 'Annick'),
(146, 'Correia', 'Lydie'),
(147, 'Bertrand', 'Emmanuel'),
(148, 'Sciara', 'Giuliano'),
(149, 'Mechichi', 'Tahar'),
(150, 'Berrin', 'Jean-Guy'),
(151, 'Grisel', 'Sacha'),
(152, 'Haon', 'Mireille'),
(153, 'Piumi', 'François'),
(154, 'Levasseur', 'Anthony'),
(155, 'Lomascolo', 'Anne'),
(156, 'Chevret', 'Didier'),
(157, 'Daum', 'Chris'),
(158, 'Mariette', 'Jérôme'),
(159, 'Klopp', 'Christophe'),
(160, 'de', 'Vries'),
(161, 'Gathman', 'Allen'),
(162, 'Hainaut', 'Matthieu'),
(163, 'Hildén', 'Kristiina'),
(164, 'Kües', 'Ursula'),
(165, 'Lilly', 'Walt'),
(166, 'Mäkelä', 'Miia'),
(167, 'Morel-Rouhier', 'Mélanie'),
(168, 'Ram', 'Arthur'),
(169, 'Wösten', 'Han'),
(170, 'Tan', 'Hao'),
(171, 'Miao', 'Renyun'),
(172, 'Liu', 'Tianhai'),
(173, 'Zhang', 'Qiang'),
(174, 'Zhang', 'Bo'),
(175, 'Jiang', 'Lin'),
(176, 'Wang', 'Yong'),
(177, 'Xie', 'Liyuan'),
(178, 'Tang', 'Jie'),
(179, 'Li', 'Xiaolin'),
(180, 'Liu', 'Lixu'),
(181, 'Wang', 'Bo'),
(182, 'Huang', 'Zhongqian'),
(183, 'Gan', 'Bingcheng'),
(184, 'Peng', 'Weihong'),
(185, 'Laville', 'Elisabeth'),
(186, 'Perrier', 'Josette'),
(187, 'Bejar', 'Nada'),
(188, 'Maresca', 'Marc'),
(189, 'Esque', 'Jeremy'),
(190, 'Tauzin', 'Alexandra'),
(191, 'Bouhajja', 'Emna'),
(192, 'Leclerc', 'Marion'),
(193, 'Berdah', 'Stephane'),
(194, 'Di', 'Pasquale'),
(195, 'Robe', 'Patrick'),
(196, 'Potocki-Veronese', 'Gabrielle'),
(197, 'Almási', 'Éva'),
(198, 'Sahu', 'Neha'),
(199, 'Krizsán', 'Krisztina'),
(200, 'Kovács', 'Gábor'),
(201, 'Kiss', 'Brigitta'),
(202, 'Cseklye', 'Judit'),
(203, 'Nagy', 'István'),
(204, 'Chovatia', 'Mansi'),
(205, 'Adam', 'Catherine');

-- --------------------------------------------------------

--
-- Structure de la table `author_article`
--

CREATE TABLE `author_article` (
  `id_author` int(11) NOT NULL,
  `id_article` int(11) NOT NULL,
  `position` int(11) NOT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;

--
-- Déchargement des données de la table `author_article`
--

INSERT INTO `author_article` (`id_author`, `id_article`, `position`) VALUES
(1, 1, 1),
(2, 1, 2),
(3, 1, 3),
(4, 1, 4),
(5, 1, 5),
(6, 1, 6),
(7, 1, 7),
(8, 1, 8),
(9, 2, 1),
(10, 2, 2),
(11, 2, 3),
(12, 2, 4),
(13, 2, 5),
(14, 2, 6),
(15, 2, 7),
(16, 2, 8),
(17, 2, 9),
(18, 2, 10),
(19, 3, 1),
(20, 3, 2),
(21, 3, 3),
(22, 3, 4),
(23, 3, 5),
(24, 3, 6),
(25, 3, 7),
(26, 4, 1),
(27, 4, 2),
(28, 5, 1),
(29, 5, 2),
(30, 5, 3),
(31, 5, 4),
(32, 5, 5),
(33, 5, 6),
(34, 5, 7),
(35, 5, 8),
(36, 5, 9),
(37, 5, 10),
(38, 5, 11),
(39, 5, 12),
(40, 5, 13),
(41, 5, 14),
(42, 5, 15),
(43, 5, 16),
(44, 5, 17),
(45, 5, 18),
(46, 5, 19),
(47, 5, 20),
(48, 5, 21),
(49, 5, 22),
(50, 5, 23),
(51, 5, 24),
(52, 5, 25),
(53, 5, 26),
(54, 5, 27),
(55, 5, 28),
(56, 5, 29),
(57, 5, 30),
(58, 5, 31),
(59, 5, 32),
(60, 5, 33),
(61, 5, 34),
(62, 5, 35),
(63, 5, 36),
(64, 5, 37),
(65, 6, 1),
(66, 6, 2),
(67, 6, 3),
(68, 6, 4),
(69, 6, 5),
(70, 6, 6),
(34, 6, 7),
(71, 6, 8),
(31, 6, 9),
(72, 6, 10),
(73, 6, 11),
(74, 6, 12),
(75, 6, 13),
(76, 7, 1),
(77, 7, 2),
(78, 7, 3),
(79, 7, 4),
(29, 7, 5),
(80, 7, 6),
(81, 7, 7),
(82, 7, 8),
(31, 7, 9),
(83, 7, 10),
(84, 7, 11),
(85, 7, 12),
(86, 7, 13),
(87, 7, 14),
(88, 7, 15),
(89, 7, 16),
(90, 7, 17),
(91, 7, 18),
(92, 7, 19),
(59, 7, 20),
(55, 7, 21),
(93, 7, 22),
(52, 7, 23),
(50, 7, 24),
(94, 7, 25),
(57, 7, 26),
(95, 7, 27),
(96, 7, 28),
(97, 7, 29),
(62, 7, 30),
(98, 7, 31),
(63, 7, 32),
(64, 7, 33),
(58, 7, 34),
(99, 7, 35),
(42, 7, 36),
(100, 8, 1),
(101, 8, 2),
(102, 8, 3),
(31, 8, 4),
(58, 8, 5),
(103, 8, 6),
(104, 8, 7),
(105, 8, 8),
(29, 9, 1),
(106, 9, 2),
(55, 9, 3),
(31, 9, 4),
(107, 9, 5),
(78, 9, 6),
(41, 9, 7),
(108, 9, 8),
(56, 9, 9),
(109, 9, 10),
(110, 9, 11),
(111, 9, 12),
(112, 9, 13),
(113, 9, 14),
(114, 9, 15),
(115, 9, 16),
(116, 9, 17),
(117, 9, 18),
(118, 9, 19),
(119, 9, 20),
(120, 9, 21),
(52, 9, 22),
(57, 9, 23),
(121, 9, 24),
(47, 9, 25),
(122, 9, 26),
(123, 9, 27),
(124, 9, 28),
(125, 9, 29),
(50, 9, 30),
(126, 9, 31),
(127, 9, 32),
(128, 9, 33),
(129, 9, 34),
(59, 9, 35),
(130, 9, 36),
(131, 9, 37),
(132, 9, 38),
(133, 9, 39),
(44, 9, 40),
(134, 9, 41),
(135, 9, 42),
(136, 9, 43),
(137, 9, 44),
(96, 9, 45),
(138, 9, 46),
(139, 9, 47),
(140, 9, 48),
(43, 9, 49),
(141, 9, 50),
(142, 9, 51),
(58, 9, 52),
(62, 9, 53),
(99, 9, 54),
(61, 9, 55),
(63, 9, 56),
(143, 10, 1),
(34, 10, 2),
(144, 10, 3),
(31, 10, 4),
(145, 10, 5),
(146, 10, 6),
(74, 10, 7),
(147, 10, 8),
(75, 10, 9),
(58, 10, 10),
(148, 10, 11),
(149, 10, 12),
(72, 10, 13),
(29, 11, 1),
(28, 11, 2),
(31, 11, 3),
(37, 11, 4),
(150, 11, 5),
(34, 11, 6),
(35, 11, 7),
(33, 11, 8),
(151, 11, 9),
(152, 11, 10),
(153, 11, 11),
(154, 11, 12),
(155, 11, 13),
(60, 11, 14),
(56, 11, 15),
(52, 11, 16),
(156, 11, 17),
(157, 11, 18),
(158, 11, 19),
(159, 11, 20),
(98, 11, 21),
(160, 11, 22),
(161, 11, 23),
(162, 11, 24),
(58, 11, 25),
(163, 11, 26),
(164, 11, 27),
(165, 11, 28),
(57, 11, 29),
(166, 11, 30),
(42, 11, 31),
(167, 11, 32),
(41, 11, 33),
(50, 11, 34),
(168, 11, 35),
(169, 11, 36),
(76, 11, 37),
(59, 11, 38),
(72, 11, 39),
(62, 11, 40),
(64, 11, 41),
(170, 12, 1),
(107, 12, 2),
(171, 12, 3),
(172, 12, 4),
(173, 12, 5),
(174, 12, 6),
(175, 12, 7),
(176, 12, 8),
(177, 12, 9),
(178, 12, 10),
(179, 12, 11),
(180, 12, 12),
(62, 12, 13),
(157, 12, 14),
(52, 12, 15),
(57, 12, 16),
(55, 12, 17),
(41, 12, 18),
(31, 12, 19),
(58, 12, 20),
(181, 12, 21),
(182, 12, 22),
(183, 12, 23),
(184, 12, 24),
(63, 12, 25),
(185, 13, 1),
(186, 13, 2),
(187, 13, 3),
(188, 13, 4),
(189, 13, 5),
(190, 13, 6),
(191, 13, 7),
(192, 13, 8),
(31, 13, 9),
(58, 13, 10),
(193, 13, 11),
(194, 13, 12),
(195, 13, 13),
(196, 13, 14),
(197, 14, 1),
(198, 14, 2),
(199, 14, 3),
(38, 14, 4),
(200, 14, 5),
(201, 14, 6),
(202, 14, 7),
(31, 14, 8),
(58, 14, 9),
(203, 14, 10),
(204, 14, 11),
(205, 14, 12),
(52, 14, 13),
(57, 14, 14),
(59, 14, 15),
(62, 14, 16),
(61, 14, 17);

-- --------------------------------------------------------

--
-- Structure de la table `note`
--

CREATE TABLE `note` (
  `id_article` int(11) NOT NULL,
  `id_user` int(11) NOT NULL,
  `note` enum('1','2','3','4','5') NOT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;

-- --------------------------------------------------------

--
-- Structure de la table `prot_access_table`
--

CREATE TABLE `prot_access_table` (
  `id_article` int(11) NOT NULL,
  `prot_access` varchar(30) NOT NULL,
  `entry_id_cazy` int(11) DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;

--
-- Déchargement des données de la table `prot_access_table`
--

INSERT INTO `prot_access_table` (`id_article`, `prot_access`, `entry_id_cazy`) VALUES
(9, 'XP_038907285.1', NULL),
(9, 'XP_038907284.1', NULL),
(9, 'XP_038907283.1', NULL),
(9, 'XP_038907289.1', NULL),
(9, 'XP_038907288.1', NULL),
(9, 'XP_038907287.1', NULL),
(9, 'XP_038907286.1', NULL),
(9, 'XP_038907291.1', NULL),
(9, 'XP_038907290.1', NULL),
(9, 'XP_038907294.1', NULL),
(9, 'XP_038907293.1', NULL),
(9, 'XP_038907292.1', NULL),
(9, 'XP_038907295.1', NULL),
(9, 'XP_038907296.1', NULL),
(9, 'XP_038907297.1', NULL),
(9, 'XP_038907298.1', NULL),
(9, 'XP_038907299.1', NULL),
(9, 'XP_038907301.1', NULL),
(9, 'XP_038907300.1', NULL),
(9, 'XP_038907302.1', NULL),
(9, 'XP_038907303.1', NULL),
(9, 'XP_038907304.1', NULL),
(9, 'XP_038907311.1', NULL),
(9, 'XP_038907310.1', NULL),
(9, 'XP_038907309.1', NULL),
(9, 'XP_038907308.1', NULL),
(9, 'XP_038907307.1', NULL),
(9, 'XP_038907306.1', NULL),
(9, 'XP_038907305.1', NULL),
(9, 'XP_038907313.1', NULL),
(9, 'XP_038907312.1', NULL),
(9, 'XP_038907314.1', NULL),
(9, 'XP_038907315.1', NULL),
(9, 'XP_038907316.1', NULL),
(9, 'XP_038907318.1', NULL),
(9, 'XP_038907317.1', NULL),
(9, 'XP_038907319.1', NULL),
(9, 'XP_038907321.1', NULL),
(9, 'XP_038907320.1', NULL),
(9, 'XP_038907334.1', NULL),
(9, 'XP_038907333.1', NULL),
(9, 'XP_038907332.1', NULL),
(9, 'XP_038907331.1', NULL),
(9, 'XP_038907330.1', NULL),
(9, 'XP_038907329.1', NULL),
(9, 'XP_038907328.1', NULL),
(9, 'XP_038907327.1', NULL),
(9, 'XP_038907326.1', NULL),
(9, 'XP_038907325.1', NULL),
(9, 'XP_038907324.1', NULL),
(9, 'XP_038907323.1', NULL),
(9, 'XP_038907322.1', NULL),
(9, 'XP_038907337.1', NULL),
(9, 'XP_038907336.1', NULL),
(9, 'XP_038907335.1', NULL),
(9, 'XP_038907344.1', NULL),
(9, 'XP_038907343.1', NULL),
(9, 'XP_038907342.1', NULL),
(9, 'XP_038907341.1', NULL),
(9, 'XP_038907340.1', NULL),
(9, 'XP_038907339.1', NULL),
(9, 'XP_038907338.1', NULL),
(9, 'XP_038907346.1', NULL),
(9, 'XP_038907345.1', NULL),
(9, 'XP_038907347.1', NULL),
(9, 'XP_038907349.1', NULL),
(9, 'XP_038907348.1', NULL),
(9, 'XP_038907351.1', NULL),
(9, 'XP_038907350.1', NULL),
(9, 'XP_038907352.1', NULL),
(9, 'XP_038907354.1', NULL),
(9, 'XP_038907353.1', NULL),
(9, 'XP_038907361.1', NULL),
(9, 'XP_038907360.1', NULL),
(9, 'XP_038907359.1', NULL),
(9, 'XP_038907358.1', NULL),
(9, 'XP_038907357.1', NULL),
(9, 'XP_038907356.1', NULL),
(9, 'XP_038907355.1', NULL),
(9, 'XP_038907362.1', NULL),
(9, 'XP_038907368.1', NULL),
(9, 'XP_038907367.1', NULL),
(9, 'XP_038907366.1', NULL),
(9, 'XP_038907365.1', NULL),
(9, 'XP_038907364.1', NULL),
(9, 'XP_038907363.1', NULL),
(9, 'XP_038907371.1', NULL),
(9, 'XP_038907370.1', NULL),
(9, 'XP_038907369.1', NULL),
(9, 'XP_038907373.1', NULL),
(9, 'XP_038907372.1', NULL),
(9, 'XP_038907374.1', NULL),
(9, 'XP_038907378.1', NULL),
(9, 'XP_038907377.1', NULL),
(9, 'XP_038907376.1', NULL),
(9, 'XP_038907375.1', NULL),
(9, 'XP_038907380.1', NULL),
(9, 'XP_038907379.1', NULL),
(9, 'XP_038907381.1', NULL),
(9, 'XP_038907383.1', NULL),
(9, 'XP_038907382.1', NULL),
(9, 'XP_038907385.1', NULL),
(9, 'XP_038907384.1', NULL),
(9, 'XP_038907397.1', NULL),
(9, 'XP_038907396.1', NULL),
(9, 'XP_038907395.1', NULL),
(9, 'XP_038907394.1', NULL),
(9, 'XP_038907393.1', NULL),
(9, 'XP_038907392.1', NULL),
(9, 'XP_038907391.1', NULL),
(9, 'XP_038907390.1', NULL),
(9, 'XP_038907389.1', NULL),
(9, 'XP_038907388.1', NULL),
(9, 'XP_038907387.1', NULL),
(9, 'XP_038907386.1', NULL),
(9, 'XP_038907400.1', NULL),
(9, 'XP_038907399.1', NULL),
(9, 'XP_038907398.1', NULL),
(9, 'XP_038907401.1', NULL),
(9, 'XP_038907410.1', NULL),
(9, 'XP_038907409.1', NULL),
(9, 'XP_038907408.1', NULL),
(9, 'XP_038907407.1', NULL),
(9, 'XP_038907406.1', NULL),
(9, 'XP_038907405.1', NULL),
(9, 'XP_038907404.1', NULL),
(9, 'XP_038907403.1', NULL),
(9, 'XP_038907402.1', NULL),
(9, 'XP_038907411.1', NULL),
(9, 'XP_038907413.1', NULL),
(9, 'XP_038907412.1', NULL),
(9, 'XP_038907415.1', NULL),
(9, 'XP_038907414.1', NULL),
(9, 'XP_038907417.1', NULL),
(9, 'XP_038907416.1', NULL),
(9, 'XP_038907423.1', NULL),
(9, 'XP_038907422.1', NULL),
(9, 'XP_038907421.1', NULL),
(9, 'XP_038907420.1', NULL),
(9, 'XP_038907419.1', NULL),
(9, 'XP_038907418.1', NULL),
(9, 'XP_038907429.1', NULL),
(9, 'XP_038907428.1', NULL),
(9, 'XP_038907427.1', NULL),
(9, 'XP_038907426.1', NULL),
(9, 'XP_038907425.1', NULL),
(9, 'XP_038907424.1', NULL),
(9, 'XP_038907430.1', NULL),
(9, 'XP_038907432.1', NULL),
(9, 'XP_038907431.1', NULL),
(9, 'XP_038907433.1', NULL),
(9, 'XP_038907434.1', NULL),
(9, 'XP_038907443.1', NULL),
(9, 'XP_038907442.1', NULL),
(9, 'XP_038907441.1', NULL),
(9, 'XP_038907440.1', NULL),
(9, 'XP_038907439.1', NULL),
(9, 'XP_038907438.1', NULL),
(9, 'XP_038907437.1', NULL),
(9, 'XP_038907436.1', NULL),
(9, 'XP_038907435.1', NULL),
(9, 'XP_038907444.1', NULL),
(9, 'XP_038907451.1', NULL),
(9, 'XP_038907450.1', NULL),
(9, 'XP_038907449.1', NULL),
(9, 'XP_038907448.1', NULL),
(9, 'XP_038907447.1', NULL),
(9, 'XP_038907446.1', NULL),
(9, 'XP_038907445.1', NULL),
(9, 'XP_038907454.1', NULL),
(9, 'XP_038907453.1', NULL),
(9, 'XP_038907452.1', NULL),
(9, 'XP_038907455.1', NULL),
(9, 'XP_038907460.1', NULL),
(9, 'XP_038907459.1', NULL),
(9, 'XP_038907458.1', NULL),
(9, 'XP_038907457.1', NULL),
(9, 'XP_038907456.1', NULL),
(9, 'XP_038907463.1', NULL),
(9, 'XP_038907462.1', NULL),
(9, 'XP_038907461.1', NULL),
(9, 'XP_038907467.1', NULL),
(9, 'XP_038907466.1', NULL),
(9, 'XP_038907465.1', NULL),
(9, 'XP_038907464.1', NULL),
(9, 'XP_038907468.1', NULL),
(9, 'XP_038907469.1', NULL),
(9, 'XP_038907470.1', NULL),
(9, 'XP_038907473.1', NULL),
(9, 'XP_038907472.1', NULL),
(9, 'XP_038907471.1', NULL),
(9, 'XP_038907474.1', NULL),
(9, 'XP_038907482.1', NULL),
(9, 'XP_038907481.1', NULL),
(9, 'XP_038907480.1', NULL),
(9, 'XP_038907479.1', NULL),
(9, 'XP_038907478.1', NULL),
(9, 'XP_038907477.1', NULL),
(9, 'XP_038907476.1', NULL),
(9, 'XP_038907475.1', NULL),
(9, 'XP_038907489.1', NULL),
(9, 'XP_038907488.1', NULL),
(9, 'XP_038907487.1', NULL),
(9, 'XP_038907486.1', NULL),
(9, 'XP_038907485.1', NULL),
(9, 'XP_038907484.1', NULL),
(9, 'XP_038907483.1', NULL),
(9, 'XP_038907491.1', NULL),
(9, 'XP_038907490.1', NULL),
(9, 'XP_038907492.1', NULL),
(9, 'XP_038907493.1', NULL),
(9, 'XP_038907499.1', NULL),
(9, 'XP_038907498.1', NULL),
(9, 'XP_038907497.1', NULL),
(9, 'XP_038907496.1', NULL),
(9, 'XP_038907495.1', NULL),
(9, 'XP_038907494.1', NULL),
(9, 'XP_038907509.1', NULL),
(9, 'XP_038907508.1', NULL),
(9, 'XP_038907507.1', NULL),
(9, 'XP_038907506.1', NULL),
(9, 'XP_038907505.1', NULL),
(9, 'XP_038907504.1', NULL),
(9, 'XP_038907503.1', NULL),
(9, 'XP_038907502.1', NULL),
(9, 'XP_038907501.1', NULL),
(9, 'XP_038907500.1', NULL),
(9, 'XP_038907512.1', NULL),
(9, 'XP_038907511.1', NULL),
(9, 'XP_038907510.1', NULL),
(9, 'XP_038907515.1', NULL),
(9, 'XP_038907514.1', NULL),
(9, 'XP_038907513.1', NULL),
(9, 'XP_038907516.1', NULL),
(9, 'XP_038907519.1', NULL),
(9, 'XP_038907518.1', NULL),
(9, 'XP_038907517.1', NULL),
(9, 'XP_038907520.1', NULL),
(9, 'XP_038907521.1', NULL),
(9, 'XP_038907522.1', NULL),
(9, 'XP_038907533.1', NULL),
(9, 'XP_038907532.1', NULL),
(9, 'XP_038907531.1', NULL),
(9, 'XP_038907530.1', NULL),
(9, 'XP_038907529.1', NULL),
(9, 'XP_038907528.1', NULL),
(9, 'XP_038907527.1', NULL),
(9, 'XP_038907526.1', NULL),
(9, 'XP_038907525.1', NULL),
(9, 'XP_038907524.1', NULL),
(9, 'XP_038907523.1', NULL),
(9, 'XP_038907534.1', NULL),
(9, 'XP_038907540.1', NULL),
(9, 'XP_038907539.1', NULL),
(9, 'XP_038907538.1', NULL),
(9, 'XP_038907537.1', NULL),
(9, 'XP_038907536.1', NULL),
(9, 'XP_038907535.1', NULL),
(9, 'XP_038907561.1', NULL),
(9, 'XP_038907560.1', NULL),
(9, 'XP_038907559.1', NULL),
(9, 'XP_038907558.1', NULL),
(9, 'XP_038907557.1', NULL),
(9, 'XP_038907556.1', NULL),
(9, 'XP_038907555.1', NULL),
(9, 'XP_038907554.1', NULL),
(9, 'XP_038907553.1', NULL),
(9, 'XP_038907552.1', NULL),
(9, 'XP_038907551.1', NULL),
(9, 'XP_038907550.1', NULL),
(9, 'XP_038907549.1', NULL),
(9, 'XP_038907548.1', NULL),
(9, 'XP_038907547.1', NULL),
(9, 'XP_038907546.1', NULL),
(9, 'XP_038907545.1', NULL),
(9, 'XP_038907544.1', NULL),
(9, 'XP_038907543.1', NULL),
(9, 'XP_038907542.1', NULL),
(9, 'XP_038907541.1', NULL),
(9, 'XP_038907562.1', NULL),
(9, 'XP_038907565.1', NULL),
(9, 'XP_038907564.1', NULL),
(9, 'XP_038907563.1', NULL),
(9, 'XP_038907574.1', NULL),
(9, 'XP_038907573.1', NULL),
(9, 'XP_038907572.1', NULL),
(9, 'XP_038907571.1', NULL),
(9, 'XP_038907570.1', NULL),
(9, 'XP_038907569.1', NULL),
(9, 'XP_038907568.1', NULL),
(9, 'XP_038907567.1', NULL),
(9, 'XP_038907566.1', NULL),
(9, 'XP_038907575.1', NULL),
(9, 'XP_038907580.1', NULL),
(9, 'XP_038907579.1', NULL),
(9, 'XP_038907578.1', NULL),
(9, 'XP_038907577.1', NULL),
(9, 'XP_038907576.1', NULL),
(9, 'XP_038907581.1', NULL),
(9, 'XP_038907596.1', NULL),
(9, 'XP_038907595.1', NULL),
(9, 'XP_038907594.1', NULL),
(9, 'XP_038907593.1', NULL),
(9, 'XP_038907592.1', NULL),
(9, 'XP_038907591.1', NULL),
(9, 'XP_038907590.1', NULL),
(9, 'XP_038907589.1', NULL),
(9, 'XP_038907588.1', NULL),
(9, 'XP_038907587.1', NULL),
(9, 'XP_038907586.1', NULL),
(9, 'XP_038907585.1', NULL),
(9, 'XP_038907584.1', NULL),
(9, 'XP_038907583.1', NULL),
(9, 'XP_038907582.1', NULL),
(9, 'XP_038907606.1', NULL),
(9, 'XP_038907605.1', NULL),
(9, 'XP_038907604.1', NULL),
(9, 'XP_038907603.1', NULL),
(9, 'XP_038907602.1', NULL),
(9, 'XP_038907601.1', NULL),
(9, 'XP_038907600.1', NULL),
(9, 'XP_038907599.1', NULL),
(9, 'XP_038907598.1', NULL),
(9, 'XP_038907597.1', NULL),
(9, 'XP_038907611.1', NULL),
(9, 'XP_038907610.1', NULL),
(9, 'XP_038907609.1', NULL),
(9, 'XP_038907608.1', NULL),
(9, 'XP_038907607.1', NULL),
(9, 'XP_038907618.1', NULL),
(9, 'XP_038907617.1', NULL),
(9, 'XP_038907616.1', NULL),
(9, 'XP_038907615.1', NULL),
(9, 'XP_038907614.1', NULL),
(9, 'XP_038907613.1', NULL),
(9, 'XP_038907612.1', NULL),
(9, 'XP_038907632.1', NULL),
(9, 'XP_038907631.1', NULL),
(9, 'XP_038907630.1', NULL),
(9, 'XP_038907629.1', NULL),
(9, 'XP_038907628.1', NULL),
(9, 'XP_038907627.1', NULL),
(9, 'XP_038907626.1', NULL),
(9, 'XP_038907625.1', NULL),
(9, 'XP_038907624.1', NULL),
(9, 'XP_038907623.1', NULL),
(9, 'XP_038907622.1', NULL),
(9, 'XP_038907621.1', NULL),
(9, 'XP_038907620.1', NULL),
(9, 'XP_038907619.1', NULL),
(9, 'XP_038907634.1', NULL),
(9, 'XP_038907633.1', NULL),
(9, 'XP_038907635.1', NULL),
(9, 'XP_038907636.1', NULL),
(9, 'XP_038907640.1', NULL),
(9, 'XP_038907639.1', NULL),
(9, 'XP_038907638.1', NULL),
(9, 'XP_038907637.1', NULL),
(9, 'XP_038907646.1', NULL),
(9, 'XP_038907645.1', NULL),
(9, 'XP_038907644.1', NULL),
(9, 'XP_038907643.1', NULL),
(9, 'XP_038907642.1', NULL),
(9, 'XP_038907641.1', NULL),
(9, 'XP_038907647.1', NULL),
(9, 'XP_038907649.1', NULL),
(9, 'XP_038907648.1', NULL),
(9, 'XP_038907651.1', NULL),
(9, 'XP_038907650.1', NULL),
(9, 'XP_038907661.1', NULL),
(9, 'XP_038907660.1', NULL),
(9, 'XP_038907659.1', NULL),
(9, 'XP_038907658.1', NULL),
(9, 'XP_038907657.1', NULL),
(9, 'XP_038907656.1', NULL),
(9, 'XP_038907655.1', NULL),
(9, 'XP_038907654.1', NULL),
(9, 'XP_038907653.1', NULL),
(9, 'XP_038907652.1', NULL),
(9, 'XP_038907666.1', NULL),
(9, 'XP_038907665.1', NULL),
(9, 'XP_038907664.1', NULL),
(9, 'XP_038907663.1', NULL),
(9, 'XP_038907662.1', NULL),
(9, 'XP_038907669.1', NULL),
(9, 'XP_038907668.1', NULL),
(9, 'XP_038907667.1', NULL),
(9, 'XP_038907673.1', NULL),
(9, 'XP_038907672.1', NULL),
(9, 'XP_038907671.1', NULL),
(9, 'XP_038907670.1', NULL),
(9, 'XP_038907675.1', NULL),
(9, 'XP_038907674.1', NULL),
(9, 'XP_038907678.1', NULL),
(9, 'XP_038907677.1', NULL),
(9, 'XP_038907676.1', NULL),
(9, 'XP_038907680.1', NULL),
(9, 'XP_038907679.1', NULL),
(9, 'XP_038907686.1', NULL),
(9, 'XP_038907685.1', NULL),
(9, 'XP_038907684.1', NULL),
(9, 'XP_038907683.1', NULL),
(9, 'XP_038907682.1', NULL),
(9, 'XP_038907681.1', NULL),
(9, 'XP_038907687.1', NULL),
(9, 'XP_038907701.1', NULL),
(9, 'XP_038907700.1', NULL),
(9, 'XP_038907699.1', NULL),
(9, 'XP_038907698.1', NULL),
(9, 'XP_038907697.1', NULL),
(9, 'XP_038907696.1', NULL),
(9, 'XP_038907695.1', NULL),
(9, 'XP_038907694.1', NULL),
(9, 'XP_038907693.1', NULL),
(9, 'XP_038907692.1', NULL),
(9, 'XP_038907691.1', NULL),
(9, 'XP_038907690.1', NULL),
(9, 'XP_038907689.1', NULL),
(9, 'XP_038907688.1', NULL),
(9, 'XP_038907702.1', NULL),
(9, 'XP_038907705.1', NULL),
(9, 'XP_038907704.1', NULL),
(9, 'XP_038907703.1', NULL),
(9, 'XP_038907711.1', NULL),
(9, 'XP_038907710.1', NULL),
(9, 'XP_038907709.1', NULL),
(9, 'XP_038907708.1', NULL),
(9, 'XP_038907707.1', NULL),
(9, 'XP_038907706.1', NULL),
(9, 'XP_038907713.1', NULL),
(9, 'XP_038907712.1', NULL),
(9, 'XP_038907715.1', NULL),
(9, 'XP_038907714.1', NULL),
(9, 'XP_038907735.1', NULL),
(9, 'XP_038907734.1', NULL),
(9, 'XP_038907733.1', NULL),
(9, 'XP_038907732.1', NULL),
(9, 'XP_038907731.1', NULL),
(9, 'XP_038907730.1', NULL),
(9, 'XP_038907729.1', NULL),
(9, 'XP_038907728.1', NULL),
(9, 'XP_038907727.1', NULL),
(9, 'XP_038907726.1', NULL),
(9, 'XP_038907725.1', NULL),
(9, 'XP_038907724.1', NULL),
(9, 'XP_038907723.1', NULL),
(9, 'XP_038907722.1', NULL),
(9, 'XP_038907721.1', NULL),
(9, 'XP_038907720.1', NULL),
(9, 'XP_038907719.1', NULL),
(9, 'XP_038907718.1', NULL),
(9, 'XP_038907717.1', NULL),
(9, 'XP_038907716.1', NULL),
(9, 'XP_038907737.1', NULL),
(9, 'XP_038907736.1', NULL),
(9, 'XP_038907740.1', NULL),
(9, 'XP_038907739.1', NULL),
(9, 'XP_038907738.1', NULL),
(9, 'XP_038907757.1', NULL),
(9, 'XP_038907756.1', NULL),
(9, 'XP_038907755.1', NULL),
(9, 'XP_038907754.1', NULL),
(9, 'XP_038907753.1', NULL),
(9, 'XP_038907752.1', NULL),
(9, 'XP_038907751.1', NULL),
(9, 'XP_038907750.1', NULL),
(9, 'XP_038907749.1', NULL),
(9, 'XP_038907748.1', NULL),
(9, 'XP_038907747.1', NULL),
(9, 'XP_038907746.1', NULL),
(9, 'XP_038907745.1', NULL),
(9, 'XP_038907744.1', NULL),
(9, 'XP_038907743.1', NULL),
(9, 'XP_038907742.1', NULL),
(9, 'XP_038907741.1', NULL),
(9, 'XP_038907759.1', NULL),
(9, 'XP_038907758.1', NULL),
(9, 'XP_038907760.1', NULL),
(9, 'XP_038907767.1', NULL),
(9, 'XP_038907766.1', NULL),
(9, 'XP_038907765.1', NULL),
(9, 'XP_038907764.1', NULL),
(9, 'XP_038907763.1', NULL),
(9, 'XP_038907762.1', NULL),
(9, 'XP_038907761.1', NULL),
(9, 'XP_038907771.1', NULL),
(9, 'XP_038907770.1', NULL),
(9, 'XP_038907769.1', NULL),
(9, 'XP_038907768.1', NULL),
(9, 'XP_038907772.1', NULL),
(9, 'XP_038907773.1', NULL),
(9, 'XP_038907778.1', NULL),
(9, 'XP_038907777.1', NULL),
(9, 'XP_038907776.1', NULL),
(9, 'XP_038907775.1', NULL),
(9, 'XP_038907774.1', NULL),
(9, 'XP_038907781.1', NULL),
(9, 'XP_038907780.1', NULL),
(9, 'XP_038907779.1', NULL),
(9, 'XP_038907784.1', NULL),
(9, 'XP_038907783.1', NULL),
(9, 'XP_038907782.1', NULL),
(9, 'XP_038907799.1', NULL),
(9, 'XP_038907798.1', NULL),
(9, 'XP_038907797.1', NULL),
(9, 'XP_038907796.1', NULL),
(9, 'XP_038907795.1', NULL),
(9, 'XP_038907794.1', NULL),
(9, 'XP_038907793.1', NULL),
(9, 'XP_038907792.1', NULL),
(9, 'XP_038907791.1', NULL),
(9, 'XP_038907790.1', NULL),
(9, 'XP_038907789.1', NULL),
(9, 'XP_038907788.1', NULL),
(9, 'XP_038907787.1', NULL),
(9, 'XP_038907786.1', NULL),
(9, 'XP_038907785.1', NULL),
(9, 'XP_038907813.1', NULL),
(9, 'XP_038907812.1', NULL),
(9, 'XP_038907811.1', NULL),
(9, 'XP_038907810.1', NULL),
(9, 'XP_038907809.1', NULL),
(9, 'XP_038907808.1', NULL),
(9, 'XP_038907807.1', NULL),
(9, 'XP_038907806.1', NULL),
(9, 'XP_038907805.1', NULL),
(9, 'XP_038907804.1', NULL),
(9, 'XP_038907803.1', NULL),
(9, 'XP_038907802.1', NULL),
(9, 'XP_038907801.1', NULL),
(9, 'XP_038907800.1', NULL),
(9, 'XP_038907814.1', NULL),
(9, 'XP_038907819.1', NULL),
(9, 'XP_038907818.1', NULL),
(9, 'XP_038907817.1', NULL),
(9, 'XP_038907816.1', NULL),
(9, 'XP_038907815.1', NULL),
(9, 'XP_038907822.1', NULL),
(9, 'XP_038907821.1', NULL),
(9, 'XP_038907820.1', NULL),
(9, 'XP_038907827.1', NULL),
(9, 'XP_038907826.1', NULL),
(9, 'XP_038907825.1', NULL),
(9, 'XP_038907824.1', NULL),
(9, 'XP_038907823.1', NULL),
(9, 'XP_038907834.1', NULL),
(9, 'XP_038907833.1', NULL),
(9, 'XP_038907832.1', NULL),
(9, 'XP_038907831.1', NULL),
(9, 'XP_038907830.1', NULL),
(9, 'XP_038907829.1', NULL),
(9, 'XP_038907828.1', NULL),
(9, 'XP_038907846.1', NULL),
(9, 'XP_038907845.1', NULL),
(9, 'XP_038907844.1', NULL),
(9, 'XP_038907843.1', NULL),
(9, 'XP_038907842.1', NULL),
(9, 'XP_038907841.1', NULL),
(9, 'XP_038907840.1', NULL),
(9, 'XP_038907839.1', NULL),
(9, 'XP_038907838.1', NULL),
(9, 'XP_038907837.1', NULL),
(9, 'XP_038907836.1', NULL),
(9, 'XP_038907835.1', NULL),
(9, 'XP_038907854.1', NULL),
(9, 'XP_038907853.1', NULL),
(9, 'XP_038907852.1', NULL),
(9, 'XP_038907851.1', NULL),
(9, 'XP_038907850.1', NULL),
(9, 'XP_038907849.1', NULL),
(9, 'XP_038907848.1', NULL),
(9, 'XP_038907847.1', NULL),
(9, 'XP_038907855.1', NULL),
(9, 'XP_038907858.1', NULL),
(9, 'XP_038907857.1', NULL),
(9, 'XP_038907856.1', NULL),
(9, 'XP_038907871.1', NULL),
(9, 'XP_038907870.1', NULL),
(9, 'XP_038907869.1', NULL),
(9, 'XP_038907868.1', NULL),
(9, 'XP_038907867.1', NULL),
(9, 'XP_038907866.1', NULL),
(9, 'XP_038907865.1', NULL),
(9, 'XP_038907864.1', NULL),
(9, 'XP_038907863.1', NULL),
(9, 'XP_038907862.1', NULL),
(9, 'XP_038907861.1', NULL),
(9, 'XP_038907860.1', NULL),
(9, 'XP_038907859.1', NULL),
(9, 'XP_038907872.1', NULL),
(9, 'XP_038907876.1', NULL),
(9, 'XP_038907875.1', NULL),
(9, 'XP_038907874.1', NULL),
(9, 'XP_038907873.1', NULL),
(9, 'XP_038907889.1', NULL),
(9, 'XP_038907888.1', NULL),
(9, 'XP_038907887.1', NULL),
(9, 'XP_038907886.1', NULL),
(9, 'XP_038907885.1', NULL),
(9, 'XP_038907884.1', NULL),
(9, 'XP_038907883.1', NULL),
(9, 'XP_038907882.1', NULL),
(9, 'XP_038907881.1', NULL),
(9, 'XP_038907880.1', NULL),
(9, 'XP_038907879.1', NULL),
(9, 'XP_038907878.1', NULL),
(9, 'XP_038907877.1', NULL),
(9, 'XP_038907893.1', NULL),
(9, 'XP_038907892.1', NULL),
(9, 'XP_038907891.1', NULL),
(9, 'XP_038907890.1', NULL),
(9, 'XP_038907921.1', NULL),
(9, 'XP_038907920.1', NULL),
(9, 'XP_038907919.1', NULL),
(9, 'XP_038907918.1', NULL),
(9, 'XP_038907917.1', NULL),
(9, 'XP_038907916.1', NULL),
(9, 'XP_038907915.1', NULL),
(9, 'XP_038907914.1', NULL),
(9, 'XP_038907913.1', NULL),
(9, 'XP_038907912.1', NULL),
(9, 'XP_038907911.1', NULL),
(9, 'XP_038907910.1', NULL),
(9, 'XP_038907909.1', NULL),
(9, 'XP_038907908.1', NULL),
(9, 'XP_038907907.1', NULL),
(9, 'XP_038907906.1', NULL),
(9, 'XP_038907905.1', NULL),
(9, 'XP_038907904.1', NULL),
(9, 'XP_038907903.1', NULL),
(9, 'XP_038907902.1', NULL),
(9, 'XP_038907901.1', NULL),
(9, 'XP_038907900.1', NULL),
(9, 'XP_038907899.1', NULL),
(9, 'XP_038907898.1', NULL),
(9, 'XP_038907897.1', NULL),
(9, 'XP_038907896.1', NULL),
(9, 'XP_038907895.1', NULL),
(9, 'XP_038907894.1', NULL),
(9, 'XP_038907938.1', NULL),
(9, 'XP_038907937.1', NULL),
(9, 'XP_038907936.1', NULL),
(9, 'XP_038907935.1', NULL),
(9, 'XP_038907934.1', NULL),
(9, 'XP_038907933.1', NULL),
(9, 'XP_038907932.1', NULL),
(9, 'XP_038907931.1', NULL),
(9, 'XP_038907930.1', NULL),
(9, 'XP_038907929.1', NULL),
(9, 'XP_038907928.1', NULL),
(9, 'XP_038907927.1', NULL),
(9, 'XP_038907926.1', NULL),
(9, 'XP_038907925.1', NULL),
(9, 'XP_038907924.1', NULL),
(9, 'XP_038907923.1', NULL),
(9, 'XP_038907922.1', NULL),
(9, 'XP_038907939.1', NULL),
(9, 'XP_038907940.1', NULL),
(9, 'XP_038907956.1', NULL),
(9, 'XP_038907955.1', NULL),
(9, 'XP_038907954.1', NULL),
(9, 'XP_038907953.1', NULL),
(9, 'XP_038907952.1', NULL),
(9, 'XP_038907951.1', NULL),
(9, 'XP_038907950.1', NULL),
(9, 'XP_038907949.1', NULL),
(9, 'XP_038907948.1', NULL),
(9, 'XP_038907947.1', NULL),
(9, 'XP_038907946.1', NULL),
(9, 'XP_038907945.1', NULL),
(9, 'XP_038907944.1', NULL),
(9, 'XP_038907943.1', NULL),
(9, 'XP_038907942.1', NULL),
(9, 'XP_038907941.1', NULL),
(9, 'XP_038907959.1', NULL),
(9, 'XP_038907958.1', NULL),
(9, 'XP_038907957.1', NULL),
(9, 'XP_038907976.1', NULL),
(9, 'XP_038907975.1', NULL),
(9, 'XP_038907974.1', NULL),
(9, 'XP_038907973.1', NULL),
(9, 'XP_038907972.1', NULL),
(9, 'XP_038907971.1', NULL),
(9, 'XP_038907970.1', NULL),
(9, 'XP_038907969.1', NULL),
(9, 'XP_038907968.1', NULL),
(9, 'XP_038907967.1', NULL),
(9, 'XP_038907966.1', NULL),
(9, 'XP_038907965.1', NULL),
(9, 'XP_038907964.1', NULL),
(9, 'XP_038907963.1', NULL),
(9, 'XP_038907962.1', NULL),
(9, 'XP_038907961.1', NULL),
(9, 'XP_038907960.1', NULL),
(9, 'XP_038907996.1', NULL),
(9, 'XP_038907995.1', NULL),
(9, 'XP_038907994.1', NULL),
(9, 'XP_038907993.1', NULL),
(9, 'XP_038907992.1', NULL),
(9, 'XP_038907991.1', NULL),
(9, 'XP_038907990.1', NULL),
(9, 'XP_038907989.1', NULL),
(9, 'XP_038907988.1', NULL),
(9, 'XP_038907987.1', NULL),
(9, 'XP_038907986.1', NULL),
(9, 'XP_038907985.1', NULL),
(9, 'XP_038907984.1', NULL),
(9, 'XP_038907983.1', NULL),
(9, 'XP_038907982.1', NULL),
(9, 'XP_038907981.1', NULL),
(9, 'XP_038907980.1', NULL),
(9, 'XP_038907979.1', NULL),
(9, 'XP_038907978.1', NULL),
(9, 'XP_038907977.1', NULL),
(9, 'XP_038907997.1', NULL),
(9, 'XP_038908010.1', NULL),
(9, 'XP_038908009.1', NULL),
(9, 'XP_038908008.1', NULL),
(9, 'XP_038908007.1', NULL),
(9, 'XP_038908006.1', NULL),
(9, 'XP_038908005.1', NULL),
(9, 'XP_038908004.1', NULL),
(9, 'XP_038908003.1', NULL),
(9, 'XP_038908002.1', NULL),
(9, 'XP_038908001.1', NULL),
(9, 'XP_038908000.1', NULL),
(9, 'XP_038907999.1', NULL),
(9, 'XP_038907998.1', NULL),
(9, 'XP_038908019.1', NULL),
(9, 'XP_038908018.1', NULL),
(9, 'XP_038908017.1', NULL),
(9, 'XP_038908016.1', NULL),
(9, 'XP_038908015.1', NULL),
(9, 'XP_038908014.1', NULL),
(9, 'XP_038908013.1', NULL),
(9, 'XP_038908012.1', NULL),
(9, 'XP_038908011.1', NULL),
(9, 'XP_038908030.1', NULL),
(9, 'XP_038908029.1', NULL),
(9, 'XP_038908028.1', NULL),
(9, 'XP_038908027.1', NULL),
(9, 'XP_038908026.1', NULL),
(9, 'XP_038908025.1', NULL),
(9, 'XP_038908024.1', NULL),
(9, 'XP_038908023.1', NULL),
(9, 'XP_038908022.1', NULL),
(9, 'XP_038908021.1', NULL),
(9, 'XP_038908020.1', NULL),
(9, 'XP_038908049.1', NULL),
(9, 'XP_038908048.1', NULL),
(9, 'XP_038908047.1', NULL),
(9, 'XP_038908046.1', NULL),
(9, 'XP_038908045.1', NULL),
(9, 'XP_038908044.1', NULL),
(9, 'XP_038908043.1', NULL),
(9, 'XP_038908042.1', NULL),
(9, 'XP_038908041.1', NULL),
(9, 'XP_038908040.1', NULL),
(9, 'XP_038908039.1', NULL),
(9, 'XP_038908038.1', NULL),
(9, 'XP_038908037.1', NULL),
(9, 'XP_038908036.1', NULL),
(9, 'XP_038908035.1', NULL),
(9, 'XP_038908034.1', NULL),
(9, 'XP_038908033.1', NULL),
(9, 'XP_038908032.1', NULL),
(9, 'XP_038908031.1', NULL),
(9, 'XP_038908059.1', NULL),
(9, 'XP_038908058.1', NULL),
(9, 'XP_038908057.1', NULL),
(9, 'XP_038908056.1', NULL),
(9, 'XP_038908055.1', NULL),
(9, 'XP_038908054.1', NULL),
(9, 'XP_038908053.1', NULL),
(9, 'XP_038908052.1', NULL),
(9, 'XP_038908051.1', NULL),
(9, 'XP_038908050.1', NULL),
(9, 'XP_038908076.1', NULL),
(9, 'XP_038908075.1', NULL),
(9, 'XP_038908074.1', NULL),
(9, 'XP_038908073.1', NULL),
(9, 'XP_038908072.1', NULL),
(9, 'XP_038908071.1', NULL),
(9, 'XP_038908070.1', NULL),
(9, 'XP_038908069.1', NULL),
(9, 'XP_038908068.1', NULL),
(9, 'XP_038908067.1', NULL),
(9, 'XP_038908066.1', NULL),
(9, 'XP_038908065.1', NULL),
(9, 'XP_038908064.1', NULL),
(9, 'XP_038908063.1', NULL),
(9, 'XP_038908062.1', NULL),
(9, 'XP_038908061.1', NULL),
(9, 'XP_038908060.1', NULL),
(9, 'XP_038908098.1', NULL),
(9, 'XP_038908097.1', NULL),
(9, 'XP_038908096.1', NULL),
(9, 'XP_038908095.1', NULL),
(9, 'XP_038908094.1', NULL),
(9, 'XP_038908093.1', NULL),
(9, 'XP_038908092.1', NULL),
(9, 'XP_038908091.1', NULL),
(9, 'XP_038908090.1', NULL),
(9, 'XP_038908089.1', NULL),
(9, 'XP_038908088.1', NULL),
(9, 'XP_038908087.1', NULL),
(9, 'XP_038908086.1', NULL),
(9, 'XP_038908085.1', NULL),
(9, 'XP_038908084.1', NULL),
(9, 'XP_038908083.1', NULL),
(9, 'XP_038908082.1', NULL),
(9, 'XP_038908081.1', NULL),
(9, 'XP_038908080.1', NULL),
(9, 'XP_038908079.1', NULL),
(9, 'XP_038908078.1', NULL),
(9, 'XP_038908077.1', NULL),
(9, 'XP_038908128.1', NULL),
(9, 'XP_038908127.1', NULL),
(9, 'XP_038908126.1', NULL),
(9, 'XP_038908125.1', NULL),
(9, 'XP_038908124.1', NULL),
(9, 'XP_038908123.1', NULL),
(9, 'XP_038908122.1', NULL),
(9, 'XP_038908121.1', NULL),
(9, 'XP_038908120.1', NULL),
(9, 'XP_038908119.1', NULL),
(9, 'XP_038908118.1', NULL),
(9, 'XP_038908117.1', NULL),
(9, 'XP_038908116.1', NULL),
(9, 'XP_038908115.1', NULL),
(9, 'XP_038908114.1', NULL),
(9, 'XP_038908113.1', NULL),
(9, 'XP_038908112.1', NULL),
(9, 'XP_038908111.1', NULL),
(9, 'XP_038908110.1', NULL),
(9, 'XP_038908109.1', NULL),
(9, 'XP_038908108.1', NULL),
(9, 'XP_038908107.1', NULL),
(9, 'XP_038908106.1', NULL),
(9, 'XP_038908105.1', NULL),
(9, 'XP_038908104.1', NULL),
(9, 'XP_038908103.1', NULL),
(9, 'XP_038908102.1', NULL),
(9, 'XP_038908101.1', NULL),
(9, 'XP_038908100.1', NULL),
(9, 'XP_038908099.1', NULL),
(9, 'XP_038908130.1', NULL),
(9, 'XP_038908129.1', NULL),
(9, 'XP_038908154.1', NULL),
(9, 'XP_038908153.1', NULL),
(9, 'XP_038908152.1', NULL),
(9, 'XP_038908151.1', NULL),
(9, 'XP_038908150.1', NULL),
(9, 'XP_038908149.1', NULL),
(9, 'XP_038908148.1', NULL),
(9, 'XP_038908147.1', NULL),
(9, 'XP_038908146.1', NULL),
(9, 'XP_038908145.1', NULL),
(9, 'XP_038908144.1', NULL),
(9, 'XP_038908143.1', NULL),
(9, 'XP_038908142.1', NULL),
(9, 'XP_038908141.1', NULL),
(9, 'XP_038908140.1', NULL),
(9, 'XP_038908139.1', NULL),
(9, 'XP_038908138.1', NULL),
(9, 'XP_038908137.1', NULL),
(9, 'XP_038908136.1', NULL),
(9, 'XP_038908135.1', NULL),
(9, 'XP_038908134.1', NULL),
(9, 'XP_038908133.1', NULL),
(9, 'XP_038908132.1', NULL),
(9, 'XP_038908131.1', NULL),
(9, 'XP_038908157.1', NULL),
(9, 'XP_038908156.1', NULL),
(9, 'XP_038908155.1', NULL),
(9, 'XP_038908175.1', NULL),
(9, 'XP_038908174.1', NULL),
(9, 'XP_038908173.1', NULL),
(9, 'XP_038908172.1', NULL),
(9, 'XP_038908171.1', NULL),
(9, 'XP_038908170.1', NULL),
(9, 'XP_038908169.1', NULL),
(9, 'XP_038908168.1', NULL),
(9, 'XP_038908167.1', NULL),
(9, 'XP_038908166.1', NULL),
(9, 'XP_038908165.1', NULL),
(9, 'XP_038908164.1', NULL),
(9, 'XP_038908163.1', NULL),
(9, 'XP_038908162.1', NULL),
(9, 'XP_038908161.1', NULL),
(9, 'XP_038908160.1', NULL),
(9, 'XP_038908159.1', NULL),
(9, 'XP_038908158.1', NULL),
(9, 'XP_038908191.1', NULL),
(9, 'XP_038908190.1', NULL),
(9, 'XP_038908189.1', NULL),
(9, 'XP_038908188.1', NULL),
(9, 'XP_038908187.1', NULL),
(9, 'XP_038908186.1', NULL),
(9, 'XP_038908185.1', NULL),
(9, 'XP_038908184.1', NULL),
(9, 'XP_038908183.1', NULL),
(9, 'XP_038908182.1', NULL),
(9, 'XP_038908181.1', NULL),
(9, 'XP_038908180.1', NULL),
(9, 'XP_038908179.1', NULL),
(9, 'XP_038908178.1', NULL),
(9, 'XP_038908177.1', NULL),
(9, 'XP_038908176.1', NULL),
(9, 'XP_038908199.1', NULL),
(9, 'XP_038908198.1', NULL),
(9, 'XP_038908197.1', NULL),
(9, 'XP_038908196.1', NULL),
(9, 'XP_038908195.1', NULL),
(9, 'XP_038908194.1', NULL),
(9, 'XP_038908193.1', NULL),
(9, 'XP_038908192.1', NULL),
(9, 'XP_038908204.1', NULL),
(9, 'XP_038908203.1', NULL),
(9, 'XP_038908202.1', NULL),
(9, 'XP_038908201.1', NULL),
(9, 'XP_038908200.1', NULL),
(9, 'XP_038908221.1', NULL),
(9, 'XP_038908220.1', NULL),
(9, 'XP_038908219.1', NULL),
(9, 'XP_038908218.1', NULL),
(9, 'XP_038908217.1', NULL),
(9, 'XP_038908216.1', NULL),
(9, 'XP_038908215.1', NULL),
(9, 'XP_038908214.1', NULL),
(9, 'XP_038908213.1', NULL),
(9, 'XP_038908212.1', NULL),
(9, 'XP_038908211.1', NULL),
(9, 'XP_038908210.1', NULL),
(9, 'XP_038908209.1', NULL),
(9, 'XP_038908208.1', NULL),
(9, 'XP_038908207.1', NULL),
(9, 'XP_038908206.1', NULL),
(9, 'XP_038908205.1', NULL),
(9, 'XP_038908222.1', NULL),
(9, 'XP_038908259.1', NULL),
(9, 'XP_038908258.1', NULL),
(9, 'XP_038908257.1', NULL),
(9, 'XP_038908256.1', NULL),
(9, 'XP_038908255.1', NULL),
(9, 'XP_038908254.1', NULL),
(9, 'XP_038908253.1', NULL),
(9, 'XP_038908252.1', NULL),
(9, 'XP_038908251.1', NULL),
(9, 'XP_038908250.1', NULL),
(9, 'XP_038908249.1', NULL),
(9, 'XP_038908248.1', NULL),
(9, 'XP_038908247.1', NULL),
(9, 'XP_038908246.1', NULL),
(9, 'XP_038908245.1', NULL),
(9, 'XP_038908244.1', NULL),
(9, 'XP_038908243.1', NULL),
(9, 'XP_038908242.1', NULL),
(9, 'XP_038908241.1', NULL),
(9, 'XP_038908240.1', NULL),
(9, 'XP_038908239.1', NULL),
(9, 'XP_038908238.1', NULL),
(9, 'XP_038908237.1', NULL),
(9, 'XP_038908236.1', NULL),
(9, 'XP_038908235.1', NULL),
(9, 'XP_038908234.1', NULL),
(9, 'XP_038908233.1', NULL),
(9, 'XP_038908232.1', NULL),
(9, 'XP_038908231.1', NULL),
(9, 'XP_038908230.1', NULL),
(9, 'XP_038908229.1', NULL),
(9, 'XP_038908228.1', NULL),
(9, 'XP_038908227.1', NULL),
(9, 'XP_038908226.1', NULL),
(9, 'XP_038908225.1', NULL),
(9, 'XP_038908224.1', NULL),
(9, 'XP_038908223.1', NULL),
(9, 'XP_038908263.1', NULL),
(9, 'XP_038908262.1', NULL),
(9, 'XP_038908261.1', NULL),
(9, 'XP_038908260.1', NULL),
(9, 'XP_038908280.1', NULL),
(9, 'XP_038908279.1', NULL),
(9, 'XP_038908278.1', NULL),
(9, 'XP_038908277.1', NULL),
(9, 'XP_038908276.1', NULL),
(9, 'XP_038908275.1', NULL),
(9, 'XP_038908274.1', NULL),
(9, 'XP_038908273.1', NULL),
(9, 'XP_038908272.1', NULL),
(9, 'XP_038908271.1', NULL),
(9, 'XP_038908270.1', NULL),
(9, 'XP_038908269.1', NULL),
(9, 'XP_038908268.1', NULL),
(9, 'XP_038908267.1', NULL),
(9, 'XP_038908266.1', NULL),
(9, 'XP_038908265.1', NULL),
(9, 'XP_038908264.1', NULL),
(9, 'XP_038908302.1', NULL),
(9, 'XP_038908301.1', NULL),
(9, 'XP_038908300.1', NULL),
(9, 'XP_038908299.1', NULL),
(9, 'XP_038908298.1', NULL),
(9, 'XP_038908297.1', NULL),
(9, 'XP_038908296.1', NULL),
(9, 'XP_038908295.1', NULL),
(9, 'XP_038908294.1', NULL),
(9, 'XP_038908293.1', NULL),
(9, 'XP_038908292.1', NULL),
(9, 'XP_038908291.1', NULL),
(9, 'XP_038908290.1', NULL),
(9, 'XP_038908289.1', NULL),
(9, 'XP_038908288.1', NULL),
(9, 'XP_038908287.1', NULL),
(9, 'XP_038908286.1', NULL),
(9, 'XP_038908285.1', NULL),
(9, 'XP_038908284.1', NULL),
(9, 'XP_038908283.1', NULL),
(9, 'XP_038908282.1', NULL),
(9, 'XP_038908281.1', NULL),
(9, 'XP_038908328.1', NULL),
(9, 'XP_038908327.1', NULL),
(9, 'XP_038908326.1', NULL),
(9, 'XP_038908325.1', NULL),
(9, 'XP_038908324.1', NULL),
(9, 'XP_038908323.1', NULL),
(9, 'XP_038908322.1', NULL),
(9, 'XP_038908321.1', NULL),
(9, 'XP_038908320.1', NULL),
(9, 'XP_038908319.1', NULL),
(9, 'XP_038908318.1', NULL),
(9, 'XP_038908317.1', NULL),
(9, 'XP_038908316.1', NULL),
(9, 'XP_038908315.1', NULL),
(9, 'XP_038908314.1', NULL),
(9, 'XP_038908313.1', NULL),
(9, 'XP_038908312.1', NULL),
(9, 'XP_038908311.1', NULL),
(9, 'XP_038908310.1', NULL),
(9, 'XP_038908309.1', NULL),
(9, 'XP_038908308.1', NULL),
(9, 'XP_038908307.1', NULL),
(9, 'XP_038908306.1', NULL),
(9, 'XP_038908305.1', NULL),
(9, 'XP_038908304.1', NULL),
(9, 'XP_038908303.1', NULL),
(9, 'XP_038908362.1', NULL),
(9, 'XP_038908361.1', NULL),
(9, 'XP_038908360.1', NULL),
(9, 'XP_038908359.1', NULL),
(9, 'XP_038908358.1', NULL),
(9, 'XP_038908357.1', NULL),
(9, 'XP_038908356.1', NULL),
(9, 'XP_038908355.1', NULL),
(9, 'XP_038908354.1', NULL),
(9, 'XP_038908353.1', NULL),
(9, 'XP_038908352.1', NULL),
(9, 'XP_038908351.1', NULL),
(9, 'XP_038908350.1', NULL),
(9, 'XP_038908349.1', NULL),
(9, 'XP_038908348.1', NULL),
(9, 'XP_038908347.1', NULL),
(9, 'XP_038908346.1', NULL),
(9, 'XP_038908345.1', NULL),
(9, 'XP_038908344.1', NULL),
(9, 'XP_038908343.1', NULL),
(9, 'XP_038908342.1', NULL),
(9, 'XP_038908341.1', NULL),
(9, 'XP_038908340.1', NULL),
(9, 'XP_038908339.1', NULL),
(9, 'XP_038908338.1', NULL),
(9, 'XP_038908337.1', NULL),
(9, 'XP_038908336.1', NULL),
(9, 'XP_038908335.1', NULL),
(9, 'XP_038908334.1', NULL),
(9, 'XP_038908333.1', NULL),
(9, 'XP_038908332.1', NULL),
(9, 'XP_038908331.1', NULL),
(9, 'XP_038908330.1', NULL),
(9, 'XP_038908329.1', NULL),
(9, 'XP_038908364.1', NULL),
(9, 'XP_038908363.1', NULL),
(9, 'XP_038908367.1', NULL),
(9, 'XP_038908366.1', NULL),
(9, 'XP_038908365.1', NULL),
(9, 'XP_038908382.1', NULL),
(9, 'XP_038908381.1', NULL),
(9, 'XP_038908380.1', NULL),
(9, 'XP_038908379.1', NULL),
(9, 'XP_038908378.1', NULL),
(9, 'XP_038908377.1', NULL),
(9, 'XP_038908376.1', NULL),
(9, 'XP_038908375.1', NULL),
(9, 'XP_038908374.1', NULL),
(9, 'XP_038908373.1', NULL),
(9, 'XP_038908372.1', NULL),
(9, 'XP_038908371.1', NULL),
(9, 'XP_038908370.1', NULL),
(9, 'XP_038908369.1', NULL),
(9, 'XP_038908368.1', NULL),
(9, 'XP_038908407.1', NULL),
(9, 'XP_038908406.1', NULL),
(9, 'XP_038908405.1', NULL),
(9, 'XP_038908404.1', NULL),
(9, 'XP_038908403.1', NULL),
(9, 'XP_038908402.1', NULL),
(9, 'XP_038908401.1', NULL),
(9, 'XP_038908400.1', NULL),
(9, 'XP_038908399.1', NULL),
(9, 'XP_038908398.1', NULL),
(9, 'XP_038908397.1', NULL),
(9, 'XP_038908396.1', NULL),
(9, 'XP_038908395.1', NULL),
(9, 'XP_038908394.1', NULL),
(9, 'XP_038908393.1', NULL),
(9, 'XP_038908392.1', NULL),
(9, 'XP_038908391.1', NULL),
(9, 'XP_038908390.1', NULL),
(9, 'XP_038908389.1', NULL),
(9, 'XP_038908388.1', NULL),
(9, 'XP_038908387.1', NULL),
(9, 'XP_038908386.1', NULL),
(9, 'XP_038908385.1', NULL),
(9, 'XP_038908384.1', NULL),
(9, 'XP_038908383.1', NULL),
(9, 'XP_038908408.1', NULL),
(9, 'XP_038908441.1', NULL),
(9, 'XP_038908440.1', NULL),
(9, 'XP_038908439.1', NULL),
(9, 'XP_038908438.1', NULL),
(9, 'XP_038908437.1', NULL),
(9, 'XP_038908436.1', NULL),
(9, 'XP_038908435.1', NULL),
(9, 'XP_038908434.1', NULL),
(9, 'XP_038908433.1', NULL),
(9, 'XP_038908432.1', NULL),
(9, 'XP_038908431.1', NULL),
(9, 'XP_038908430.1', NULL),
(9, 'XP_038908429.1', NULL),
(9, 'XP_038908428.1', NULL),
(9, 'XP_038908427.1', NULL),
(9, 'XP_038908426.1', NULL),
(9, 'XP_038908425.1', NULL),
(9, 'XP_038908424.1', NULL),
(9, 'XP_038908423.1', NULL),
(9, 'XP_038908422.1', NULL),
(9, 'XP_038908421.1', NULL),
(9, 'XP_038908420.1', NULL),
(9, 'XP_038908419.1', NULL),
(9, 'XP_038908418.1', NULL),
(9, 'XP_038908417.1', NULL),
(9, 'XP_038908416.1', NULL),
(9, 'XP_038908415.1', NULL),
(9, 'XP_038908414.1', NULL),
(9, 'XP_038908413.1', NULL),
(9, 'XP_038908412.1', NULL),
(9, 'XP_038908411.1', NULL),
(9, 'XP_038908410.1', NULL),
(9, 'XP_038908409.1', NULL),
(9, 'XP_038908480.1', NULL),
(9, 'XP_038908479.1', NULL),
(9, 'XP_038908478.1', NULL),
(9, 'XP_038908477.1', NULL),
(9, 'XP_038908476.1', NULL),
(9, 'XP_038908475.1', NULL),
(9, 'XP_038908474.1', NULL),
(9, 'XP_038908473.1', NULL),
(9, 'XP_038908472.1', NULL),
(9, 'XP_038908471.1', NULL),
(9, 'XP_038908470.1', NULL),
(9, 'XP_038908469.1', NULL),
(9, 'XP_038908468.1', NULL),
(9, 'XP_038908467.1', NULL),
(9, 'XP_038908466.1', NULL),
(9, 'XP_038908465.1', NULL),
(9, 'XP_038908464.1', NULL),
(9, 'XP_038908463.1', NULL),
(9, 'XP_038908462.1', NULL),
(9, 'XP_038908461.1', NULL),
(9, 'XP_038908460.1', NULL),
(9, 'XP_038908459.1', NULL),
(9, 'XP_038908458.1', NULL),
(9, 'XP_038908457.1', NULL),
(9, 'XP_038908456.1', NULL),
(9, 'XP_038908455.1', NULL),
(9, 'XP_038908454.1', NULL),
(9, 'XP_038908453.1', NULL),
(9, 'XP_038908452.1', NULL),
(9, 'XP_038908451.1', NULL),
(9, 'XP_038908450.1', NULL),
(9, 'XP_038908449.1', NULL),
(9, 'XP_038908448.1', NULL),
(9, 'XP_038908447.1', NULL),
(9, 'XP_038908446.1', NULL),
(9, 'XP_038908445.1', NULL),
(9, 'XP_038908444.1', NULL),
(9, 'XP_038908443.1', NULL),
(9, 'XP_038908442.1', NULL),
(9, 'XP_038908496.1', NULL),
(9, 'XP_038908495.1', NULL),
(9, 'XP_038908494.1', NULL),
(9, 'XP_038908493.1', NULL),
(9, 'XP_038908492.1', NULL),
(9, 'XP_038908491.1', NULL),
(9, 'XP_038908490.1', NULL),
(9, 'XP_038908489.1', NULL),
(9, 'XP_038908488.1', NULL),
(9, 'XP_038908487.1', NULL),
(9, 'XP_038908486.1', NULL),
(9, 'XP_038908485.1', NULL),
(9, 'XP_038908484.1', NULL),
(9, 'XP_038908483.1', NULL),
(9, 'XP_038908482.1', NULL),
(9, 'XP_038908481.1', NULL),
(9, 'XP_038908516.1', NULL),
(9, 'XP_038908515.1', NULL),
(9, 'XP_038908514.1', NULL),
(9, 'XP_038908513.1', NULL),
(9, 'XP_038908512.1', NULL),
(9, 'XP_038908511.1', NULL),
(9, 'XP_038908510.1', NULL),
(9, 'XP_038908509.1', NULL),
(9, 'XP_038908508.1', NULL),
(9, 'XP_038908507.1', NULL),
(9, 'XP_038908506.1', NULL),
(9, 'XP_038908505.1', NULL),
(9, 'XP_038908504.1', NULL),
(9, 'XP_038908503.1', NULL),
(9, 'XP_038908502.1', NULL),
(9, 'XP_038908501.1', NULL),
(9, 'XP_038908500.1', NULL),
(9, 'XP_038908499.1', NULL),
(9, 'XP_038908498.1', NULL),
(9, 'XP_038908497.1', NULL),
(9, 'XP_038908523.1', NULL),
(9, 'XP_038908522.1', NULL),
(9, 'XP_038908521.1', NULL),
(9, 'XP_038908520.1', NULL),
(9, 'XP_038908519.1', NULL),
(9, 'XP_038908518.1', NULL),
(9, 'XP_038908517.1', NULL),
(9, 'XP_038908524.1', NULL),
(9, 'XP_038908560.1', NULL),
(9, 'XP_038908559.1', NULL),
(9, 'XP_038908558.1', NULL),
(9, 'XP_038908557.1', NULL),
(9, 'XP_038908556.1', NULL),
(9, 'XP_038908555.1', NULL),
(9, 'XP_038908554.1', NULL),
(9, 'XP_038908553.1', NULL),
(9, 'XP_038908552.1', NULL),
(9, 'XP_038908551.1', NULL),
(9, 'XP_038908550.1', NULL),
(9, 'XP_038908549.1', NULL),
(9, 'XP_038908548.1', NULL),
(9, 'XP_038908547.1', NULL),
(9, 'XP_038908546.1', NULL),
(9, 'XP_038908545.1', NULL),
(9, 'XP_038908544.1', NULL),
(9, 'XP_038908543.1', NULL),
(9, 'XP_038908542.1', NULL),
(9, 'XP_038908541.1', NULL),
(9, 'XP_038908540.1', NULL),
(9, 'XP_038908539.1', NULL),
(9, 'XP_038908538.1', NULL),
(9, 'XP_038908537.1', NULL),
(9, 'XP_038908536.1', NULL),
(9, 'XP_038908535.1', NULL),
(9, 'XP_038908534.1', NULL),
(9, 'XP_038908533.1', NULL),
(9, 'XP_038908532.1', NULL),
(9, 'XP_038908531.1', NULL),
(9, 'XP_038908530.1', NULL),
(9, 'XP_038908529.1', NULL),
(9, 'XP_038908528.1', NULL),
(9, 'XP_038908527.1', NULL),
(9, 'XP_038908526.1', NULL),
(9, 'XP_038908525.1', NULL),
(9, 'XP_038908577.1', NULL),
(9, 'XP_038908576.1', NULL),
(9, 'XP_038908575.1', NULL),
(9, 'XP_038908574.1', NULL),
(9, 'XP_038908573.1', NULL),
(9, 'XP_038908572.1', NULL),
(9, 'XP_038908571.1', NULL),
(9, 'XP_038908570.1', NULL),
(9, 'XP_038908569.1', NULL),
(9, 'XP_038908568.1', NULL),
(9, 'XP_038908567.1', NULL),
(9, 'XP_038908566.1', NULL),
(9, 'XP_038908565.1', NULL),
(9, 'XP_038908564.1', NULL),
(9, 'XP_038908563.1', NULL),
(9, 'XP_038908562.1', NULL),
(9, 'XP_038908561.1', NULL),
(9, 'XP_038908595.1', NULL),
(9, 'XP_038908594.1', NULL),
(9, 'XP_038908593.1', NULL),
(9, 'XP_038908592.1', NULL),
(9, 'XP_038908591.1', NULL),
(9, 'XP_038908590.1', NULL),
(9, 'XP_038908589.1', NULL),
(9, 'XP_038908588.1', NULL),
(9, 'XP_038908587.1', NULL),
(9, 'XP_038908586.1', NULL),
(9, 'XP_038908585.1', NULL),
(9, 'XP_038908584.1', NULL),
(9, 'XP_038908583.1', NULL),
(9, 'XP_038908582.1', NULL),
(9, 'XP_038908581.1', NULL),
(9, 'XP_038908580.1', NULL),
(9, 'XP_038908579.1', NULL),
(9, 'XP_038908578.1', NULL),
(9, 'XP_038908603.1', NULL),
(9, 'XP_038908602.1', NULL),
(9, 'XP_038908601.1', NULL),
(9, 'XP_038908600.1', NULL),
(9, 'XP_038908599.1', NULL),
(9, 'XP_038908598.1', NULL),
(9, 'XP_038908597.1', NULL),
(9, 'XP_038908596.1', NULL),
(9, 'XP_038908607.1', NULL),
(9, 'XP_038908606.1', NULL),
(9, 'XP_038908605.1', NULL),
(9, 'XP_038908604.1', NULL),
(9, 'XP_038908636.1', NULL),
(9, 'XP_038908635.1', NULL),
(9, 'XP_038908634.1', NULL),
(9, 'XP_038908633.1', NULL),
(9, 'XP_038908632.1', NULL),
(9, 'XP_038908631.1', NULL),
(9, 'XP_038908630.1', NULL),
(9, 'XP_038908629.1', NULL),
(9, 'XP_038908628.1', NULL),
(9, 'XP_038908627.1', NULL),
(9, 'XP_038908626.1', NULL),
(9, 'XP_038908625.1', NULL),
(9, 'XP_038908624.1', NULL),
(9, 'XP_038908623.1', NULL),
(9, 'XP_038908622.1', NULL),
(9, 'XP_038908621.1', NULL),
(9, 'XP_038908620.1', NULL),
(9, 'XP_038908619.1', NULL),
(9, 'XP_038908618.1', NULL),
(9, 'XP_038908617.1', NULL),
(9, 'XP_038908616.1', NULL),
(9, 'XP_038908615.1', NULL),
(9, 'XP_038908614.1', NULL),
(9, 'XP_038908613.1', NULL),
(9, 'XP_038908612.1', NULL),
(9, 'XP_038908611.1', NULL),
(9, 'XP_038908610.1', NULL),
(9, 'XP_038908609.1', NULL),
(9, 'XP_038908608.1', NULL),
(9, 'XP_038908645.1', NULL),
(9, 'XP_038908644.1', NULL),
(9, 'XP_038908643.1', NULL),
(9, 'XP_038908642.1', NULL),
(9, 'XP_038908641.1', NULL),
(9, 'XP_038908640.1', NULL),
(9, 'XP_038908639.1', NULL),
(9, 'XP_038908638.1', NULL),
(9, 'XP_038908637.1', NULL),
(9, 'XP_038908696.1', NULL),
(9, 'XP_038908695.1', NULL),
(9, 'XP_038908694.1', NULL),
(9, 'XP_038908693.1', NULL),
(9, 'XP_038908692.1', NULL),
(9, 'XP_038908691.1', NULL),
(9, 'XP_038908690.1', NULL),
(9, 'XP_038908689.1', NULL),
(9, 'XP_038908688.1', NULL),
(9, 'XP_038908687.1', NULL),
(9, 'XP_038908686.1', NULL),
(9, 'XP_038908685.1', NULL),
(9, 'XP_038908684.1', NULL),
(9, 'XP_038908683.1', NULL),
(9, 'XP_038908682.1', NULL),
(9, 'XP_038908681.1', NULL),
(9, 'XP_038908680.1', NULL),
(9, 'XP_038908679.1', NULL),
(9, 'XP_038908678.1', NULL),
(9, 'XP_038908677.1', NULL),
(9, 'XP_038908676.1', NULL),
(9, 'XP_038908675.1', NULL),
(9, 'XP_038908674.1', NULL),
(9, 'XP_038908673.1', NULL),
(9, 'XP_038908672.1', NULL),
(9, 'XP_038908671.1', NULL),
(9, 'XP_038908670.1', NULL),
(9, 'XP_038908669.1', NULL),
(9, 'XP_038908668.1', NULL),
(9, 'XP_038908667.1', NULL),
(9, 'XP_038908666.1', NULL),
(9, 'XP_038908665.1', NULL),
(9, 'XP_038908664.1', NULL),
(9, 'XP_038908663.1', NULL),
(9, 'XP_038908662.1', NULL),
(9, 'XP_038908661.1', NULL),
(9, 'XP_038908660.1', NULL),
(9, 'XP_038908659.1', NULL),
(9, 'XP_038908658.1', NULL),
(9, 'XP_038908657.1', NULL),
(9, 'XP_038908656.1', NULL),
(9, 'XP_038908655.1', NULL),
(9, 'XP_038908654.1', NULL),
(9, 'XP_038908653.1', NULL),
(9, 'XP_038908652.1', NULL),
(9, 'XP_038908651.1', NULL),
(9, 'XP_038908650.1', NULL),
(9, 'XP_038908649.1', NULL),
(9, 'XP_038908648.1', NULL),
(9, 'XP_038908647.1', NULL),
(9, 'XP_038908646.1', NULL),
(9, 'XP_038908713.1', NULL),
(9, 'XP_038908712.1', NULL),
(9, 'XP_038908711.1', NULL),
(9, 'XP_038908710.1', NULL),
(9, 'XP_038908709.1', NULL),
(9, 'XP_038908708.1', NULL),
(9, 'XP_038908707.1', NULL),
(9, 'XP_038908706.1', NULL),
(9, 'XP_038908705.1', NULL),
(9, 'XP_038908704.1', NULL),
(9, 'XP_038908703.1', NULL),
(9, 'XP_038908702.1', NULL),
(9, 'XP_038908701.1', NULL),
(9, 'XP_038908700.1', NULL),
(9, 'XP_038908699.1', NULL),
(9, 'XP_038908698.1', NULL),
(9, 'XP_038908697.1', NULL),
(9, 'XP_038908727.1', NULL),
(9, 'XP_038908726.1', NULL),
(9, 'XP_038908725.1', NULL),
(9, 'XP_038908724.1', NULL),
(9, 'XP_038908723.1', NULL),
(9, 'XP_038908722.1', NULL),
(9, 'XP_038908721.1', NULL),
(9, 'XP_038908720.1', NULL),
(9, 'XP_038908719.1', NULL),
(9, 'XP_038908718.1', NULL),
(9, 'XP_038908717.1', NULL),
(9, 'XP_038908716.1', NULL),
(9, 'XP_038908715.1', NULL),
(9, 'XP_038908714.1', NULL),
(9, 'XP_038908758.1', NULL),
(9, 'XP_038908757.1', NULL),
(9, 'XP_038908756.1', NULL),
(9, 'XP_038908755.1', NULL),
(9, 'XP_038908754.1', NULL),
(9, 'XP_038908753.1', NULL),
(9, 'XP_038908752.1', NULL),
(9, 'XP_038908751.1', NULL),
(9, 'XP_038908750.1', NULL),
(9, 'XP_038908749.1', NULL),
(9, 'XP_038908748.1', NULL),
(9, 'XP_038908747.1', NULL),
(9, 'XP_038908746.1', NULL),
(9, 'XP_038908745.1', NULL),
(9, 'XP_038908744.1', NULL),
(9, 'XP_038908743.1', NULL),
(9, 'XP_038908742.1', NULL),
(9, 'XP_038908741.1', NULL),
(9, 'XP_038908740.1', NULL),
(9, 'XP_038908739.1', NULL),
(9, 'XP_038908738.1', NULL),
(9, 'XP_038908737.1', NULL),
(9, 'XP_038908736.1', NULL),
(9, 'XP_038908735.1', NULL),
(9, 'XP_038908734.1', NULL),
(9, 'XP_038908733.1', NULL),
(9, 'XP_038908732.1', NULL),
(9, 'XP_038908731.1', NULL),
(9, 'XP_038908730.1', NULL),
(9, 'XP_038908729.1', NULL),
(9, 'XP_038908728.1', NULL),
(9, 'XP_038908774.1', NULL),
(9, 'XP_038908773.1', NULL),
(9, 'XP_038908772.1', NULL),
(9, 'XP_038908771.1', NULL),
(9, 'XP_038908770.1', NULL),
(9, 'XP_038908769.1', NULL),
(9, 'XP_038908768.1', NULL),
(9, 'XP_038908767.1', NULL),
(9, 'XP_038908766.1', NULL),
(9, 'XP_038908765.1', NULL),
(9, 'XP_038908764.1', NULL),
(9, 'XP_038908763.1', NULL),
(9, 'XP_038908762.1', NULL),
(9, 'XP_038908761.1', NULL),
(9, 'XP_038908760.1', NULL),
(9, 'XP_038908759.1', NULL),
(9, 'XP_038908776.1', NULL),
(9, 'XP_038908775.1', NULL),
(9, 'XP_038908788.1', NULL),
(9, 'XP_038908787.1', NULL),
(9, 'XP_038908786.1', NULL),
(9, 'XP_038908785.1', NULL),
(9, 'XP_038908784.1', NULL),
(9, 'XP_038908783.1', NULL),
(9, 'XP_038908782.1', NULL),
(9, 'XP_038908781.1', NULL),
(9, 'XP_038908780.1', NULL),
(9, 'XP_038908779.1', NULL),
(9, 'XP_038908778.1', NULL),
(9, 'XP_038908777.1', NULL),
(9, 'XP_038908799.1', NULL),
(9, 'XP_038908798.1', NULL),
(9, 'XP_038908797.1', NULL),
(9, 'XP_038908796.1', NULL),
(9, 'XP_038908795.1', NULL),
(9, 'XP_038908794.1', NULL),
(9, 'XP_038908793.1', NULL),
(9, 'XP_038908792.1', NULL),
(9, 'XP_038908791.1', NULL),
(9, 'XP_038908790.1', NULL),
(9, 'XP_038908789.1', NULL),
(9, 'XP_038908801.1', NULL),
(9, 'XP_038908800.1', NULL),
(9, 'XP_038908832.1', NULL),
(9, 'XP_038908831.1', NULL),
(9, 'XP_038908830.1', NULL),
(9, 'XP_038908829.1', NULL),
(9, 'XP_038908828.1', NULL),
(9, 'XP_038908827.1', NULL),
(9, 'XP_038908826.1', NULL),
(9, 'XP_038908825.1', NULL),
(9, 'XP_038908824.1', NULL),
(9, 'XP_038908823.1', NULL),
(9, 'XP_038908822.1', NULL),
(9, 'XP_038908821.1', NULL),
(9, 'XP_038908820.1', NULL),
(9, 'XP_038908819.1', NULL),
(9, 'XP_038908818.1', NULL),
(9, 'XP_038908817.1', NULL),
(9, 'XP_038908816.1', NULL),
(9, 'XP_038908815.1', NULL),
(9, 'XP_038908814.1', NULL),
(9, 'XP_038908813.1', NULL),
(9, 'XP_038908812.1', NULL),
(9, 'XP_038908811.1', NULL),
(9, 'XP_038908810.1', NULL),
(9, 'XP_038908809.1', NULL),
(9, 'XP_038908808.1', NULL),
(9, 'XP_038908807.1', NULL),
(9, 'XP_038908806.1', NULL),
(9, 'XP_038908805.1', NULL),
(9, 'XP_038908804.1', NULL),
(9, 'XP_038908803.1', NULL),
(9, 'XP_038908802.1', NULL),
(9, 'XP_038908844.1', NULL),
(9, 'XP_038908843.1', NULL),
(9, 'XP_038908842.1', NULL),
(9, 'XP_038908841.1', NULL),
(9, 'XP_038908840.1', NULL),
(9, 'XP_038908839.1', NULL),
(9, 'XP_038908838.1', NULL),
(9, 'XP_038908837.1', NULL),
(9, 'XP_038908836.1', NULL),
(9, 'XP_038908835.1', NULL),
(9, 'XP_038908834.1', NULL),
(9, 'XP_038908833.1', NULL),
(9, 'XP_038908873.1', NULL),
(9, 'XP_038908872.1', NULL),
(9, 'XP_038908871.1', NULL),
(9, 'XP_038908870.1', NULL),
(9, 'XP_038908869.1', NULL),
(9, 'XP_038908868.1', NULL),
(9, 'XP_038908867.1', NULL),
(9, 'XP_038908866.1', NULL),
(9, 'XP_038908865.1', NULL),
(9, 'XP_038908864.1', NULL),
(9, 'XP_038908863.1', NULL),
(9, 'XP_038908862.1', NULL),
(9, 'XP_038908861.1', NULL),
(9, 'XP_038908860.1', NULL),
(9, 'XP_038908859.1', NULL),
(9, 'XP_038908858.1', NULL),
(9, 'XP_038908857.1', NULL),
(9, 'XP_038908856.1', NULL),
(9, 'XP_038908855.1', NULL),
(9, 'XP_038908854.1', NULL),
(9, 'XP_038908853.1', NULL),
(9, 'XP_038908852.1', NULL),
(9, 'XP_038908851.1', NULL),
(9, 'XP_038908850.1', NULL),
(9, 'XP_038908849.1', NULL),
(9, 'XP_038908848.1', NULL),
(9, 'XP_038908847.1', NULL),
(9, 'XP_038908846.1', NULL),
(9, 'XP_038908845.1', NULL),
(9, 'XP_038908905.1', NULL),
(9, 'XP_038908904.1', NULL),
(9, 'XP_038908903.1', NULL),
(9, 'XP_038908902.1', NULL),
(9, 'XP_038908901.1', NULL),
(9, 'XP_038908900.1', NULL),
(9, 'XP_038908899.1', NULL),
(9, 'XP_038908898.1', NULL),
(9, 'XP_038908897.1', NULL),
(9, 'XP_038908896.1', NULL),
(9, 'XP_038908895.1', NULL),
(9, 'XP_038908894.1', NULL),
(9, 'XP_038908893.1', NULL),
(9, 'XP_038908892.1', NULL),
(9, 'XP_038908891.1', NULL),
(9, 'XP_038908890.1', NULL),
(9, 'XP_038908889.1', NULL),
(9, 'XP_038908888.1', NULL),
(9, 'XP_038908887.1', NULL),
(9, 'XP_038908886.1', NULL),
(9, 'XP_038908885.1', NULL),
(9, 'XP_038908884.1', NULL),
(9, 'XP_038908883.1', NULL),
(9, 'XP_038908882.1', NULL),
(9, 'XP_038908881.1', NULL),
(9, 'XP_038908880.1', NULL),
(9, 'XP_038908879.1', NULL),
(9, 'XP_038908878.1', NULL),
(9, 'XP_038908877.1', NULL),
(9, 'XP_038908876.1', NULL),
(9, 'XP_038908875.1', NULL),
(9, 'XP_038908874.1', NULL),
(9, 'XP_038908907.1', NULL),
(9, 'XP_038908906.1', NULL),
(9, 'XP_038908937.1', NULL),
(9, 'XP_038908936.1', NULL),
(9, 'XP_038908935.1', NULL),
(9, 'XP_038908934.1', NULL),
(9, 'XP_038908933.1', NULL),
(9, 'XP_038908932.1', NULL),
(9, 'XP_038908931.1', NULL),
(9, 'XP_038908930.1', NULL),
(9, 'XP_038908929.1', NULL),
(9, 'XP_038908928.1', NULL),
(9, 'XP_038908927.1', NULL),
(9, 'XP_038908926.1', NULL),
(9, 'XP_038908925.1', NULL),
(9, 'XP_038908924.1', NULL),
(9, 'XP_038908923.1', NULL),
(9, 'XP_038908922.1', NULL),
(9, 'XP_038908921.1', NULL),
(9, 'XP_038908920.1', NULL),
(9, 'XP_038908919.1', NULL),
(9, 'XP_038908918.1', NULL),
(9, 'XP_038908917.1', NULL),
(9, 'XP_038908916.1', NULL),
(9, 'XP_038908915.1', NULL),
(9, 'XP_038908914.1', NULL),
(9, 'XP_038908913.1', NULL),
(9, 'XP_038908912.1', NULL),
(9, 'XP_038908911.1', NULL),
(9, 'XP_038908910.1', NULL),
(9, 'XP_038908909.1', NULL),
(9, 'XP_038908908.1', NULL),
(9, 'XP_038908962.1', NULL),
(9, 'XP_038908961.1', NULL),
(9, 'XP_038908960.1', NULL),
(9, 'XP_038908959.1', NULL),
(9, 'XP_038908958.1', NULL),
(9, 'XP_038908957.1', NULL),
(9, 'XP_038908956.1', NULL),
(9, 'XP_038908955.1', NULL),
(9, 'XP_038908954.1', NULL),
(9, 'XP_038908953.1', NULL),
(9, 'XP_038908952.1', NULL),
(9, 'XP_038908951.1', NULL),
(9, 'XP_038908950.1', NULL),
(9, 'XP_038908949.1', NULL),
(9, 'XP_038908948.1', NULL),
(9, 'XP_038908947.1', NULL),
(9, 'XP_038908946.1', NULL),
(9, 'XP_038908945.1', NULL),
(9, 'XP_038908944.1', NULL),
(9, 'XP_038908943.1', NULL),
(9, 'XP_038908942.1', NULL),
(9, 'XP_038908941.1', NULL),
(9, 'XP_038908940.1', NULL),
(9, 'XP_038908939.1', NULL),
(9, 'XP_038908938.1', NULL),
(9, 'XP_038908963.1', NULL),
(9, 'XP_038908992.1', NULL),
(9, 'XP_038908991.1', NULL),
(9, 'XP_038908990.1', NULL),
(9, 'XP_038908989.1', NULL),
(9, 'XP_038908988.1', NULL),
(9, 'XP_038908987.1', NULL),
(9, 'XP_038908986.1', NULL),
(9, 'XP_038908985.1', NULL),
(9, 'XP_038908984.1', NULL),
(9, 'XP_038908983.1', NULL),
(9, 'XP_038908982.1', NULL),
(9, 'XP_038908981.1', NULL),
(9, 'XP_038908980.1', NULL),
(9, 'XP_038908979.1', NULL),
(9, 'XP_038908978.1', NULL),
(9, 'XP_038908977.1', NULL),
(9, 'XP_038908976.1', NULL),
(9, 'XP_038908975.1', NULL),
(9, 'XP_038908974.1', NULL),
(9, 'XP_038908973.1', NULL),
(9, 'XP_038908972.1', NULL),
(9, 'XP_038908971.1', NULL),
(9, 'XP_038908970.1', NULL),
(9, 'XP_038908969.1', NULL),
(9, 'XP_038908968.1', NULL),
(9, 'XP_038908967.1', NULL),
(9, 'XP_038908966.1', NULL),
(9, 'XP_038908965.1', NULL),
(9, 'XP_038908964.1', NULL),
(9, 'XP_038909028.1', NULL),
(9, 'XP_038909027.1', NULL),
(9, 'XP_038909026.1', NULL),
(9, 'XP_038909025.1', NULL),
(9, 'XP_038909024.1', NULL),
(9, 'XP_038909023.1', NULL),
(9, 'XP_038909022.1', NULL),
(9, 'XP_038909021.1', NULL),
(9, 'XP_038909020.1', NULL),
(9, 'XP_038909019.1', NULL),
(9, 'XP_038909018.1', NULL),
(9, 'XP_038909017.1', NULL),
(9, 'XP_038909016.1', NULL),
(9, 'XP_038909015.1', NULL),
(9, 'XP_038909014.1', NULL),
(9, 'XP_038909013.1', NULL),
(9, 'XP_038909012.1', NULL),
(9, 'XP_038909011.1', NULL),
(9, 'XP_038909010.1', NULL),
(9, 'XP_038909009.1', NULL),
(9, 'XP_038909008.1', NULL),
(9, 'XP_038909007.1', NULL),
(9, 'XP_038909006.1', NULL),
(9, 'XP_038909005.1', NULL),
(9, 'XP_038909004.1', NULL),
(9, 'XP_038909003.1', NULL),
(9, 'XP_038909002.1', NULL),
(9, 'XP_038909001.1', NULL),
(9, 'XP_038909000.1', NULL),
(9, 'XP_038908999.1', NULL),
(9, 'XP_038908998.1', NULL),
(9, 'XP_038908997.1', NULL),
(9, 'XP_038908996.1', NULL),
(9, 'XP_038908995.1', NULL),
(9, 'XP_038908994.1', NULL),
(9, 'XP_038908993.1', NULL),
(9, 'XP_038909048.1', NULL),
(9, 'XP_038909047.1', NULL),
(9, 'XP_038909046.1', NULL),
(9, 'XP_038909045.1', NULL),
(9, 'XP_038909044.1', NULL),
(9, 'XP_038909043.1', NULL),
(9, 'XP_038909042.1', NULL),
(9, 'XP_038909041.1', NULL),
(9, 'XP_038909040.1', NULL),
(9, 'XP_038909039.1', NULL),
(9, 'XP_038909038.1', NULL),
(9, 'XP_038909037.1', NULL),
(9, 'XP_038909036.1', NULL),
(9, 'XP_038909035.1', NULL),
(9, 'XP_038909034.1', NULL),
(9, 'XP_038909033.1', NULL),
(9, 'XP_038909032.1', NULL),
(9, 'XP_038909031.1', NULL),
(9, 'XP_038909030.1', NULL),
(9, 'XP_038909029.1', NULL),
(9, 'XP_038909050.1', NULL),
(9, 'XP_038909049.1', NULL),
(9, 'XP_038909076.1', NULL),
(9, 'XP_038909075.1', NULL),
(9, 'XP_038909074.1', NULL),
(9, 'XP_038909073.1', NULL),
(9, 'XP_038909072.1', NULL),
(9, 'XP_038909071.1', NULL),
(9, 'XP_038909070.1', NULL),
(9, 'XP_038909069.1', NULL),
(9, 'XP_038909068.1', NULL),
(9, 'XP_038909067.1', NULL),
(9, 'XP_038909066.1', NULL),
(9, 'XP_038909065.1', NULL),
(9, 'XP_038909064.1', NULL),
(9, 'XP_038909063.1', NULL),
(9, 'XP_038909062.1', NULL),
(9, 'XP_038909061.1', NULL),
(9, 'XP_038909060.1', NULL),
(9, 'XP_038909059.1', NULL),
(9, 'XP_038909058.1', NULL),
(9, 'XP_038909057.1', NULL),
(9, 'XP_038909056.1', NULL),
(9, 'XP_038909055.1', NULL),
(9, 'XP_038909054.1', NULL),
(9, 'XP_038909053.1', NULL),
(9, 'XP_038909052.1', NULL),
(9, 'XP_038909051.1', NULL),
(9, 'XP_038909085.1', NULL),
(9, 'XP_038909084.1', NULL),
(9, 'XP_038909083.1', NULL),
(9, 'XP_038909082.1', NULL),
(9, 'XP_038909081.1', NULL),
(9, 'XP_038909080.1', NULL),
(9, 'XP_038909079.1', NULL),
(9, 'XP_038909078.1', NULL),
(9, 'XP_038909077.1', NULL),
(9, 'XP_038909090.1', NULL),
(9, 'XP_038909089.1', NULL),
(9, 'XP_038909088.1', NULL),
(9, 'XP_038909087.1', NULL),
(9, 'XP_038909086.1', NULL),
(9, 'XP_038909117.1', NULL),
(9, 'XP_038909116.1', NULL),
(9, 'XP_038909115.1', NULL),
(9, 'XP_038909114.1', NULL),
(9, 'XP_038909113.1', NULL),
(9, 'XP_038909112.1', NULL),
(9, 'XP_038909111.1', NULL),
(9, 'XP_038909110.1', NULL),
(9, 'XP_038909109.1', NULL),
(9, 'XP_038909108.1', NULL),
(9, 'XP_038909107.1', NULL),
(9, 'XP_038909106.1', NULL),
(9, 'XP_038909105.1', NULL),
(9, 'XP_038909104.1', NULL),
(9, 'XP_038909103.1', NULL),
(9, 'XP_038909102.1', NULL),
(9, 'XP_038909101.1', NULL),
(9, 'XP_038909100.1', NULL),
(9, 'XP_038909099.1', NULL),
(9, 'XP_038909098.1', NULL),
(9, 'XP_038909097.1', NULL),
(9, 'XP_038909096.1', NULL),
(9, 'XP_038909095.1', NULL),
(9, 'XP_038909094.1', NULL),
(9, 'XP_038909093.1', NULL),
(9, 'XP_038909092.1', NULL),
(9, 'XP_038909091.1', NULL),
(9, 'XP_038909125.1', NULL),
(9, 'XP_038909124.1', NULL),
(9, 'XP_038909123.1', NULL),
(9, 'XP_038909122.1', NULL),
(9, 'XP_038909121.1', NULL),
(9, 'XP_038909120.1', NULL),
(9, 'XP_038909119.1', NULL),
(9, 'XP_038909118.1', NULL),
(9, 'XP_038909167.1', NULL),
(9, 'XP_038909166.1', NULL),
(9, 'XP_038909165.1', NULL),
(9, 'XP_038909164.1', NULL),
(9, 'XP_038909163.1', NULL);
INSERT INTO `prot_access_table` (`id_article`, `prot_access`, `entry_id_cazy`) VALUES
(9, 'XP_038909162.1', NULL),
(9, 'XP_038909161.1', NULL),
(9, 'XP_038909160.1', NULL),
(9, 'XP_038909159.1', NULL),
(9, 'XP_038909158.1', NULL),
(9, 'XP_038909157.1', NULL),
(9, 'XP_038909156.1', NULL),
(9, 'XP_038909155.1', NULL),
(9, 'XP_038909154.1', NULL),
(9, 'XP_038909153.1', NULL),
(9, 'XP_038909152.1', NULL),
(9, 'XP_038909151.1', NULL),
(9, 'XP_038909150.1', NULL),
(9, 'XP_038909149.1', NULL),
(9, 'XP_038909148.1', NULL),
(9, 'XP_038909147.1', NULL),
(9, 'XP_038909146.1', NULL),
(9, 'XP_038909145.1', NULL),
(9, 'XP_038909144.1', NULL),
(9, 'XP_038909143.1', NULL),
(9, 'XP_038909142.1', NULL),
(9, 'XP_038909141.1', NULL),
(9, 'XP_038909140.1', NULL),
(9, 'XP_038909139.1', NULL),
(9, 'XP_038909138.1', NULL),
(9, 'XP_038909137.1', NULL),
(9, 'XP_038909136.1', NULL),
(9, 'XP_038909135.1', NULL),
(9, 'XP_038909134.1', NULL),
(9, 'XP_038909133.1', NULL),
(9, 'XP_038909132.1', NULL),
(9, 'XP_038909131.1', NULL),
(9, 'XP_038909130.1', NULL),
(9, 'XP_038909129.1', NULL),
(9, 'XP_038909128.1', NULL),
(9, 'XP_038909127.1', NULL),
(9, 'XP_038909126.1', NULL),
(9, 'XP_038909197.1', NULL),
(9, 'XP_038909196.1', NULL),
(9, 'XP_038909195.1', NULL),
(9, 'XP_038909194.1', NULL),
(9, 'XP_038909193.1', NULL),
(9, 'XP_038909192.1', NULL),
(9, 'XP_038909191.1', NULL),
(9, 'XP_038909190.1', NULL),
(9, 'XP_038909189.1', NULL),
(9, 'XP_038909188.1', NULL),
(9, 'XP_038909187.1', NULL),
(9, 'XP_038909186.1', NULL),
(9, 'XP_038909185.1', NULL),
(9, 'XP_038909184.1', NULL),
(9, 'XP_038909183.1', NULL),
(9, 'XP_038909182.1', NULL),
(9, 'XP_038909181.1', NULL),
(9, 'XP_038909180.1', NULL),
(9, 'XP_038909179.1', NULL),
(9, 'XP_038909178.1', NULL),
(9, 'XP_038909177.1', NULL),
(9, 'XP_038909176.1', NULL),
(9, 'XP_038909175.1', NULL),
(9, 'XP_038909174.1', NULL),
(9, 'XP_038909173.1', NULL),
(9, 'XP_038909172.1', NULL),
(9, 'XP_038909171.1', NULL),
(9, 'XP_038909170.1', NULL),
(9, 'XP_038909169.1', NULL),
(9, 'XP_038909168.1', NULL),
(9, 'XP_038909222.1', NULL),
(9, 'XP_038909221.1', NULL),
(9, 'XP_038909220.1', NULL),
(9, 'XP_038909219.1', NULL),
(9, 'XP_038909218.1', NULL),
(9, 'XP_038909217.1', NULL),
(9, 'XP_038909216.1', NULL),
(9, 'XP_038909215.1', NULL),
(9, 'XP_038909214.1', NULL),
(9, 'XP_038909213.1', NULL),
(9, 'XP_038909212.1', NULL),
(9, 'XP_038909211.1', NULL),
(9, 'XP_038909210.1', NULL),
(9, 'XP_038909209.1', NULL),
(9, 'XP_038909208.1', NULL),
(9, 'XP_038909207.1', NULL),
(9, 'XP_038909206.1', NULL),
(9, 'XP_038909205.1', NULL),
(9, 'XP_038909204.1', NULL),
(9, 'XP_038909203.1', NULL),
(9, 'XP_038909202.1', NULL),
(9, 'XP_038909201.1', NULL),
(9, 'XP_038909200.1', NULL),
(9, 'XP_038909199.1', NULL),
(9, 'XP_038909198.1', NULL),
(9, 'XP_038909250.1', NULL),
(9, 'XP_038909249.1', NULL),
(9, 'XP_038909248.1', NULL),
(9, 'XP_038909247.1', NULL),
(9, 'XP_038909246.1', NULL),
(9, 'XP_038909245.1', NULL),
(9, 'XP_038909244.1', NULL),
(9, 'XP_038909243.1', NULL),
(9, 'XP_038909242.1', NULL),
(9, 'XP_038909241.1', NULL),
(9, 'XP_038909240.1', NULL),
(9, 'XP_038909239.1', NULL),
(9, 'XP_038909238.1', NULL),
(9, 'XP_038909237.1', NULL),
(9, 'XP_038909236.1', NULL),
(9, 'XP_038909235.1', NULL),
(9, 'XP_038909234.1', NULL),
(9, 'XP_038909233.1', NULL),
(9, 'XP_038909232.1', NULL),
(9, 'XP_038909231.1', NULL),
(9, 'XP_038909230.1', NULL),
(9, 'XP_038909229.1', NULL),
(9, 'XP_038909228.1', NULL),
(9, 'XP_038909227.1', NULL),
(9, 'XP_038909226.1', NULL),
(9, 'XP_038909225.1', NULL),
(9, 'XP_038909224.1', NULL),
(9, 'XP_038909223.1', NULL),
(9, 'XP_038909282.1', NULL),
(9, 'XP_038909281.1', NULL),
(9, 'XP_038909280.1', NULL),
(9, 'XP_038909279.1', NULL),
(9, 'XP_038909278.1', NULL),
(9, 'XP_038909277.1', NULL),
(9, 'XP_038909276.1', NULL),
(9, 'XP_038909275.1', NULL),
(9, 'XP_038909274.1', NULL),
(9, 'XP_038909273.1', NULL),
(9, 'XP_038909272.1', NULL),
(9, 'XP_038909271.1', NULL),
(9, 'XP_038909270.1', NULL),
(9, 'XP_038909269.1', NULL),
(9, 'XP_038909268.1', NULL),
(9, 'XP_038909267.1', NULL),
(9, 'XP_038909266.1', NULL),
(9, 'XP_038909265.1', NULL),
(9, 'XP_038909264.1', NULL),
(9, 'XP_038909263.1', NULL),
(9, 'XP_038909262.1', NULL),
(9, 'XP_038909261.1', NULL),
(9, 'XP_038909260.1', NULL),
(9, 'XP_038909259.1', NULL),
(9, 'XP_038909258.1', NULL),
(9, 'XP_038909257.1', NULL),
(9, 'XP_038909256.1', NULL),
(9, 'XP_038909255.1', NULL),
(9, 'XP_038909254.1', NULL),
(9, 'XP_038909253.1', NULL),
(9, 'XP_038909252.1', NULL),
(9, 'XP_038909251.1', NULL),
(9, 'XP_038909304.1', NULL),
(9, 'XP_038909303.1', NULL),
(9, 'XP_038909302.1', NULL),
(9, 'XP_038909301.1', NULL),
(9, 'XP_038909300.1', NULL),
(9, 'XP_038909299.1', NULL),
(9, 'XP_038909298.1', NULL),
(9, 'XP_038909297.1', NULL),
(9, 'XP_038909296.1', NULL),
(9, 'XP_038909295.1', NULL),
(9, 'XP_038909294.1', NULL),
(9, 'XP_038909293.1', NULL),
(9, 'XP_038909292.1', NULL),
(9, 'XP_038909291.1', NULL),
(9, 'XP_038909290.1', NULL),
(9, 'XP_038909289.1', NULL),
(9, 'XP_038909288.1', NULL),
(9, 'XP_038909287.1', NULL),
(9, 'XP_038909286.1', NULL),
(9, 'XP_038909285.1', NULL),
(9, 'XP_038909284.1', NULL),
(9, 'XP_038909283.1', NULL),
(9, 'XP_038909371.1', NULL),
(9, 'XP_038909370.1', NULL),
(9, 'XP_038909369.1', NULL),
(9, 'XP_038909368.1', NULL),
(9, 'XP_038909367.1', NULL),
(9, 'XP_038909366.1', NULL),
(9, 'XP_038909365.1', NULL),
(9, 'XP_038909364.1', NULL),
(9, 'XP_038909363.1', NULL),
(9, 'XP_038909362.1', NULL),
(9, 'XP_038909361.1', NULL),
(9, 'XP_038909360.1', NULL),
(9, 'XP_038909359.1', NULL),
(9, 'XP_038909358.1', NULL),
(9, 'XP_038909357.1', NULL),
(9, 'XP_038909356.1', NULL),
(9, 'XP_038909355.1', NULL),
(9, 'XP_038909354.1', NULL),
(9, 'XP_038909353.1', NULL),
(9, 'XP_038909352.1', NULL),
(9, 'XP_038909351.1', NULL),
(9, 'XP_038909350.1', NULL),
(9, 'XP_038909349.1', NULL),
(9, 'XP_038909348.1', NULL),
(9, 'XP_038909347.1', NULL),
(9, 'XP_038909346.1', NULL),
(9, 'XP_038909345.1', NULL),
(9, 'XP_038909344.1', NULL),
(9, 'XP_038909343.1', NULL),
(9, 'XP_038909342.1', NULL),
(9, 'XP_038909341.1', NULL),
(9, 'XP_038909340.1', NULL),
(9, 'XP_038909339.1', NULL),
(9, 'XP_038909338.1', NULL),
(9, 'XP_038909337.1', NULL),
(9, 'XP_038909336.1', NULL),
(9, 'XP_038909335.1', NULL),
(9, 'XP_038909334.1', NULL),
(9, 'XP_038909333.1', NULL),
(9, 'XP_038909332.1', NULL),
(9, 'XP_038909331.1', NULL),
(9, 'XP_038909330.1', NULL),
(9, 'XP_038909329.1', NULL),
(9, 'XP_038909328.1', NULL),
(9, 'XP_038909327.1', NULL),
(9, 'XP_038909326.1', NULL),
(9, 'XP_038909325.1', NULL),
(9, 'XP_038909324.1', NULL),
(9, 'XP_038909323.1', NULL),
(9, 'XP_038909322.1', NULL),
(9, 'XP_038909321.1', NULL),
(9, 'XP_038909320.1', NULL),
(9, 'XP_038909319.1', NULL),
(9, 'XP_038909318.1', NULL),
(9, 'XP_038909317.1', NULL),
(9, 'XP_038909316.1', NULL),
(9, 'XP_038909315.1', NULL),
(9, 'XP_038909314.1', NULL),
(9, 'XP_038909313.1', NULL),
(9, 'XP_038909312.1', NULL),
(9, 'XP_038909311.1', NULL),
(9, 'XP_038909310.1', NULL),
(9, 'XP_038909309.1', NULL),
(9, 'XP_038909308.1', NULL),
(9, 'XP_038909307.1', NULL),
(9, 'XP_038909306.1', NULL),
(9, 'XP_038909305.1', NULL),
(9, 'XP_038909421.1', NULL),
(9, 'XP_038909420.1', NULL),
(9, 'XP_038909419.1', NULL),
(9, 'XP_038909418.1', NULL),
(9, 'XP_038909417.1', NULL),
(9, 'XP_038909416.1', NULL),
(9, 'XP_038909415.1', NULL),
(9, 'XP_038909414.1', NULL),
(9, 'XP_038909413.1', NULL),
(9, 'XP_038909412.1', NULL),
(9, 'XP_038909411.1', NULL),
(9, 'XP_038909410.1', NULL),
(9, 'XP_038909409.1', NULL),
(9, 'XP_038909408.1', NULL),
(9, 'XP_038909407.1', NULL),
(9, 'XP_038909406.1', NULL),
(9, 'XP_038909405.1', NULL),
(9, 'XP_038909404.1', NULL),
(9, 'XP_038909403.1', NULL),
(9, 'XP_038909402.1', NULL),
(9, 'XP_038909401.1', NULL),
(9, 'XP_038909400.1', NULL),
(9, 'XP_038909399.1', NULL),
(9, 'XP_038909398.1', NULL),
(9, 'XP_038909397.1', NULL),
(9, 'XP_038909396.1', NULL),
(9, 'XP_038909395.1', NULL),
(9, 'XP_038909394.1', NULL),
(9, 'XP_038909393.1', NULL),
(9, 'XP_038909392.1', NULL),
(9, 'XP_038909391.1', NULL),
(9, 'XP_038909390.1', NULL),
(9, 'XP_038909389.1', NULL),
(9, 'XP_038909388.1', NULL),
(9, 'XP_038909387.1', NULL),
(9, 'XP_038909386.1', NULL),
(9, 'XP_038909385.1', NULL),
(9, 'XP_038909384.1', NULL),
(9, 'XP_038909383.1', NULL),
(9, 'XP_038909382.1', NULL),
(9, 'XP_038909381.1', NULL),
(9, 'XP_038909380.1', NULL),
(9, 'XP_038909379.1', NULL),
(9, 'XP_038909378.1', NULL),
(9, 'XP_038909377.1', NULL),
(9, 'XP_038909376.1', NULL),
(9, 'XP_038909375.1', NULL),
(9, 'XP_038909374.1', NULL),
(9, 'XP_038909373.1', NULL),
(9, 'XP_038909372.1', NULL),
(9, 'XP_038909450.1', NULL),
(9, 'XP_038909449.1', NULL),
(9, 'XP_038909448.1', NULL),
(9, 'XP_038909447.1', NULL),
(9, 'XP_038909446.1', NULL),
(9, 'XP_038909445.1', NULL),
(9, 'XP_038909444.1', NULL),
(9, 'XP_038909443.1', NULL),
(9, 'XP_038909442.1', NULL),
(9, 'XP_038909441.1', NULL),
(9, 'XP_038909440.1', NULL),
(9, 'XP_038909439.1', NULL),
(9, 'XP_038909438.1', NULL),
(9, 'XP_038909437.1', NULL),
(9, 'XP_038909436.1', NULL),
(9, 'XP_038909435.1', NULL),
(9, 'XP_038909434.1', NULL),
(9, 'XP_038909433.1', NULL),
(9, 'XP_038909432.1', NULL),
(9, 'XP_038909431.1', NULL),
(9, 'XP_038909430.1', NULL),
(9, 'XP_038909429.1', NULL),
(9, 'XP_038909428.1', NULL),
(9, 'XP_038909427.1', NULL),
(9, 'XP_038909426.1', NULL),
(9, 'XP_038909425.1', NULL),
(9, 'XP_038909424.1', NULL),
(9, 'XP_038909423.1', NULL),
(9, 'XP_038909422.1', NULL),
(9, 'XP_038909488.1', NULL),
(9, 'XP_038909487.1', NULL),
(9, 'XP_038909486.1', NULL),
(9, 'XP_038909485.1', NULL),
(9, 'XP_038909484.1', NULL),
(9, 'XP_038909483.1', NULL),
(9, 'XP_038909482.1', NULL),
(9, 'XP_038909481.1', NULL),
(9, 'XP_038909480.1', NULL),
(9, 'XP_038909479.1', NULL),
(9, 'XP_038909478.1', NULL),
(9, 'XP_038909477.1', NULL),
(9, 'XP_038909476.1', NULL),
(9, 'XP_038909475.1', NULL),
(9, 'XP_038909474.1', NULL),
(9, 'XP_038909473.1', NULL),
(9, 'XP_038909472.1', NULL),
(9, 'XP_038909471.1', NULL),
(9, 'XP_038909470.1', NULL),
(9, 'XP_038909469.1', NULL),
(9, 'XP_038909468.1', NULL),
(9, 'XP_038909467.1', NULL),
(9, 'XP_038909466.1', NULL),
(9, 'XP_038909465.1', NULL),
(9, 'XP_038909464.1', NULL),
(9, 'XP_038909463.1', NULL),
(9, 'XP_038909462.1', NULL),
(9, 'XP_038909461.1', NULL),
(9, 'XP_038909460.1', NULL),
(9, 'XP_038909459.1', NULL),
(9, 'XP_038909458.1', NULL),
(9, 'XP_038909457.1', NULL),
(9, 'XP_038909456.1', NULL),
(9, 'XP_038909455.1', NULL),
(9, 'XP_038909454.1', NULL),
(9, 'XP_038909453.1', NULL),
(9, 'XP_038909452.1', NULL),
(9, 'XP_038909451.1', NULL),
(9, 'XP_038909512.1', NULL),
(9, 'XP_038909511.1', NULL),
(9, 'XP_038909510.1', NULL),
(9, 'XP_038909509.1', NULL),
(9, 'XP_038909508.1', NULL),
(9, 'XP_038909507.1', NULL),
(9, 'XP_038909506.1', NULL),
(9, 'XP_038909505.1', NULL),
(9, 'XP_038909504.1', NULL),
(9, 'XP_038909503.1', NULL),
(9, 'XP_038909502.1', NULL),
(9, 'XP_038909501.1', NULL),
(9, 'XP_038909500.1', NULL),
(9, 'XP_038909499.1', NULL),
(9, 'XP_038909498.1', NULL),
(9, 'XP_038909497.1', NULL),
(9, 'XP_038909496.1', NULL),
(9, 'XP_038909495.1', NULL),
(9, 'XP_038909494.1', NULL),
(9, 'XP_038909493.1', NULL),
(9, 'XP_038909492.1', NULL),
(9, 'XP_038909491.1', NULL),
(9, 'XP_038909490.1', NULL),
(9, 'XP_038909489.1', NULL),
(9, 'XP_038909566.1', NULL),
(9, 'XP_038909565.1', NULL),
(9, 'XP_038909564.1', NULL),
(9, 'XP_038909563.1', NULL),
(9, 'XP_038909562.1', NULL),
(9, 'XP_038909561.1', NULL),
(9, 'XP_038909560.1', NULL),
(9, 'XP_038909559.1', NULL),
(9, 'XP_038909558.1', NULL),
(9, 'XP_038909557.1', NULL),
(9, 'XP_038909556.1', NULL),
(9, 'XP_038909555.1', NULL),
(9, 'XP_038909554.1', NULL),
(9, 'XP_038909553.1', NULL),
(9, 'XP_038909552.1', NULL),
(9, 'XP_038909551.1', NULL),
(9, 'XP_038909550.1', NULL),
(9, 'XP_038909549.1', NULL),
(9, 'XP_038909548.1', NULL),
(9, 'XP_038909547.1', NULL),
(9, 'XP_038909546.1', NULL),
(9, 'XP_038909545.1', NULL),
(9, 'XP_038909544.1', NULL),
(9, 'XP_038909543.1', NULL),
(9, 'XP_038909542.1', NULL),
(9, 'XP_038909541.1', NULL),
(9, 'XP_038909540.1', NULL),
(9, 'XP_038909539.1', NULL),
(9, 'XP_038909538.1', NULL),
(9, 'XP_038909537.1', NULL),
(9, 'XP_038909536.1', NULL),
(9, 'XP_038909535.1', NULL),
(9, 'XP_038909534.1', NULL),
(9, 'XP_038909533.1', NULL),
(9, 'XP_038909532.1', NULL),
(9, 'XP_038909531.1', NULL),
(9, 'XP_038909530.1', NULL),
(9, 'XP_038909529.1', NULL),
(9, 'XP_038909528.1', NULL),
(9, 'XP_038909527.1', NULL),
(9, 'XP_038909526.1', NULL),
(9, 'XP_038909525.1', NULL),
(9, 'XP_038909524.1', NULL),
(9, 'XP_038909523.1', NULL),
(9, 'XP_038909522.1', NULL),
(9, 'XP_038909521.1', NULL),
(9, 'XP_038909520.1', NULL),
(9, 'XP_038909519.1', NULL),
(9, 'XP_038909518.1', NULL),
(9, 'XP_038909517.1', NULL),
(9, 'XP_038909516.1', NULL),
(9, 'XP_038909515.1', NULL),
(9, 'XP_038909514.1', NULL),
(9, 'XP_038909513.1', NULL),
(9, 'XP_038909600.1', NULL),
(9, 'XP_038909599.1', NULL),
(9, 'XP_038909598.1', NULL),
(9, 'XP_038909597.1', NULL),
(9, 'XP_038909596.1', NULL),
(9, 'XP_038909595.1', NULL),
(9, 'XP_038909594.1', NULL),
(9, 'XP_038909593.1', NULL),
(9, 'XP_038909592.1', NULL),
(9, 'XP_038909591.1', NULL),
(9, 'XP_038909590.1', NULL),
(9, 'XP_038909589.1', NULL),
(9, 'XP_038909588.1', NULL),
(9, 'XP_038909587.1', NULL),
(9, 'XP_038909586.1', NULL),
(9, 'XP_038909585.1', NULL),
(9, 'XP_038909584.1', NULL),
(9, 'XP_038909583.1', NULL),
(9, 'XP_038909582.1', NULL),
(9, 'XP_038909581.1', NULL),
(9, 'XP_038909580.1', NULL),
(9, 'XP_038909579.1', NULL),
(9, 'XP_038909578.1', NULL),
(9, 'XP_038909577.1', NULL),
(9, 'XP_038909576.1', NULL),
(9, 'XP_038909575.1', NULL),
(9, 'XP_038909574.1', NULL),
(9, 'XP_038909573.1', NULL),
(9, 'XP_038909572.1', NULL),
(9, 'XP_038909571.1', NULL),
(9, 'XP_038909570.1', NULL),
(9, 'XP_038909569.1', NULL),
(9, 'XP_038909568.1', NULL),
(9, 'XP_038909567.1', NULL),
(9, 'XP_038909645.1', NULL),
(9, 'XP_038909644.1', NULL),
(9, 'XP_038909643.1', NULL),
(9, 'XP_038909642.1', NULL),
(9, 'XP_038909641.1', NULL),
(9, 'XP_038909640.1', NULL),
(9, 'XP_038909639.1', NULL),
(9, 'XP_038909638.1', NULL),
(9, 'XP_038909637.1', NULL),
(9, 'XP_038909636.1', NULL),
(9, 'XP_038909635.1', NULL),
(9, 'XP_038909634.1', NULL),
(9, 'XP_038909633.1', NULL),
(9, 'XP_038909632.1', NULL),
(9, 'XP_038909631.1', NULL),
(9, 'XP_038909630.1', NULL),
(9, 'XP_038909629.1', NULL),
(9, 'XP_038909628.1', NULL),
(9, 'XP_038909627.1', NULL),
(9, 'XP_038909626.1', NULL),
(9, 'XP_038909625.1', NULL),
(9, 'XP_038909624.1', NULL),
(9, 'XP_038909623.1', NULL),
(9, 'XP_038909622.1', NULL),
(9, 'XP_038909621.1', NULL),
(9, 'XP_038909620.1', NULL),
(9, 'XP_038909619.1', NULL),
(9, 'XP_038909618.1', NULL),
(9, 'XP_038909617.1', NULL),
(9, 'XP_038909616.1', NULL),
(9, 'XP_038909615.1', NULL),
(9, 'XP_038909614.1', NULL),
(9, 'XP_038909613.1', NULL),
(9, 'XP_038909612.1', NULL),
(9, 'XP_038909611.1', NULL),
(9, 'XP_038909610.1', NULL),
(9, 'XP_038909609.1', NULL),
(9, 'XP_038909608.1', NULL),
(9, 'XP_038909607.1', NULL),
(9, 'XP_038909606.1', NULL),
(9, 'XP_038909605.1', NULL),
(9, 'XP_038909604.1', NULL),
(9, 'XP_038909603.1', NULL),
(9, 'XP_038909602.1', NULL),
(9, 'XP_038909601.1', NULL),
(9, 'XP_038909647.1', NULL),
(9, 'XP_038909646.1', NULL),
(9, 'XP_038909651.1', NULL),
(9, 'XP_038909650.1', NULL),
(9, 'XP_038909649.1', NULL),
(9, 'XP_038909648.1', NULL),
(9, 'XP_038909660.1', NULL),
(9, 'XP_038909659.1', NULL),
(9, 'XP_038909658.1', NULL),
(9, 'XP_038909657.1', NULL),
(9, 'XP_038909656.1', NULL),
(9, 'XP_038909655.1', NULL),
(9, 'XP_038909654.1', NULL),
(9, 'XP_038909653.1', NULL),
(9, 'XP_038909652.1', NULL),
(9, 'XP_038909686.1', NULL),
(9, 'XP_038909685.1', NULL),
(9, 'XP_038909684.1', NULL),
(9, 'XP_038909683.1', NULL),
(9, 'XP_038909682.1', NULL),
(9, 'XP_038909681.1', NULL),
(9, 'XP_038909680.1', NULL),
(9, 'XP_038909679.1', NULL),
(9, 'XP_038909678.1', NULL),
(9, 'XP_038909677.1', NULL),
(9, 'XP_038909676.1', NULL),
(9, 'XP_038909675.1', NULL),
(9, 'XP_038909674.1', NULL),
(9, 'XP_038909673.1', NULL),
(9, 'XP_038909672.1', NULL),
(9, 'XP_038909671.1', NULL),
(9, 'XP_038909670.1', NULL),
(9, 'XP_038909669.1', NULL),
(9, 'XP_038909668.1', NULL),
(9, 'XP_038909667.1', NULL),
(9, 'XP_038909666.1', NULL),
(9, 'XP_038909665.1', NULL),
(9, 'XP_038909664.1', NULL),
(9, 'XP_038909663.1', NULL),
(9, 'XP_038909662.1', NULL),
(9, 'XP_038909661.1', NULL),
(9, 'XP_038909724.1', NULL),
(9, 'XP_038909723.1', NULL),
(9, 'XP_038909722.1', NULL),
(9, 'XP_038909721.1', NULL),
(9, 'XP_038909720.1', NULL),
(9, 'XP_038909719.1', NULL),
(9, 'XP_038909718.1', NULL),
(9, 'XP_038909717.1', NULL),
(9, 'XP_038909716.1', NULL),
(9, 'XP_038909715.1', NULL),
(9, 'XP_038909714.1', NULL),
(9, 'XP_038909713.1', NULL),
(9, 'XP_038909712.1', NULL),
(9, 'XP_038909711.1', NULL),
(9, 'XP_038909710.1', NULL),
(9, 'XP_038909709.1', NULL),
(9, 'XP_038909708.1', NULL),
(9, 'XP_038909707.1', NULL),
(9, 'XP_038909706.1', NULL),
(9, 'XP_038909705.1', NULL),
(9, 'XP_038909704.1', NULL),
(9, 'XP_038909703.1', NULL),
(9, 'XP_038909702.1', NULL),
(9, 'XP_038909701.1', NULL),
(9, 'XP_038909700.1', NULL),
(9, 'XP_038909699.1', NULL),
(9, 'XP_038909698.1', NULL),
(9, 'XP_038909697.1', NULL),
(9, 'XP_038909696.1', NULL),
(9, 'XP_038909695.1', NULL),
(9, 'XP_038909694.1', NULL),
(9, 'XP_038909693.1', NULL),
(9, 'XP_038909692.1', NULL),
(9, 'XP_038909691.1', NULL),
(9, 'XP_038909690.1', NULL),
(9, 'XP_038909689.1', NULL),
(9, 'XP_038909688.1', NULL),
(9, 'XP_038909687.1', NULL),
(9, 'XP_038909757.1', NULL),
(9, 'XP_038909756.1', NULL),
(9, 'XP_038909755.1', NULL),
(9, 'XP_038909754.1', NULL),
(9, 'XP_038909753.1', NULL),
(9, 'XP_038909752.1', NULL),
(9, 'XP_038909751.1', NULL),
(9, 'XP_038909750.1', NULL),
(9, 'XP_038909749.1', NULL),
(9, 'XP_038909748.1', NULL),
(9, 'XP_038909747.1', NULL),
(9, 'XP_038909746.1', NULL),
(9, 'XP_038909745.1', NULL),
(9, 'XP_038909744.1', NULL),
(9, 'XP_038909743.1', NULL),
(9, 'XP_038909742.1', NULL),
(9, 'XP_038909741.1', NULL),
(9, 'XP_038909740.1', NULL),
(9, 'XP_038909739.1', NULL),
(9, 'XP_038909738.1', NULL),
(9, 'XP_038909737.1', NULL),
(9, 'XP_038909736.1', NULL),
(9, 'XP_038909735.1', NULL),
(9, 'XP_038909734.1', NULL),
(9, 'XP_038909733.1', NULL),
(9, 'XP_038909732.1', NULL),
(9, 'XP_038909731.1', NULL),
(9, 'XP_038909730.1', NULL),
(9, 'XP_038909729.1', NULL),
(9, 'XP_038909728.1', NULL),
(9, 'XP_038909727.1', NULL),
(9, 'XP_038909726.1', NULL),
(9, 'XP_038909725.1', NULL),
(9, 'XP_038909819.1', NULL),
(9, 'XP_038909818.1', NULL),
(9, 'XP_038909817.1', NULL),
(9, 'XP_038909816.1', NULL),
(9, 'XP_038909815.1', NULL),
(9, 'XP_038909814.1', NULL),
(9, 'XP_038909813.1', NULL),
(9, 'XP_038909812.1', NULL),
(9, 'XP_038909811.1', NULL),
(9, 'XP_038909810.1', NULL),
(9, 'XP_038909809.1', NULL),
(9, 'XP_038909808.1', NULL),
(9, 'XP_038909807.1', NULL),
(9, 'XP_038909806.1', NULL),
(9, 'XP_038909805.1', NULL),
(9, 'XP_038909804.1', NULL),
(9, 'XP_038909803.1', NULL),
(9, 'XP_038909802.1', NULL),
(9, 'XP_038909801.1', NULL),
(9, 'XP_038909800.1', NULL),
(9, 'XP_038909799.1', NULL),
(9, 'XP_038909798.1', NULL),
(9, 'XP_038909797.1', NULL),
(9, 'XP_038909796.1', NULL),
(9, 'XP_038909795.1', NULL),
(9, 'XP_038909794.1', NULL),
(9, 'XP_038909793.1', NULL),
(9, 'XP_038909792.1', NULL),
(9, 'XP_038909791.1', NULL),
(9, 'XP_038909790.1', NULL),
(9, 'XP_038909789.1', NULL),
(9, 'XP_038909788.1', NULL),
(9, 'XP_038909787.1', NULL),
(9, 'XP_038909786.1', NULL),
(9, 'XP_038909785.1', NULL),
(9, 'XP_038909784.1', NULL),
(9, 'XP_038909783.1', NULL),
(9, 'XP_038909782.1', NULL),
(9, 'XP_038909781.1', NULL),
(9, 'XP_038909780.1', NULL),
(9, 'XP_038909779.1', NULL),
(9, 'XP_038909778.1', NULL),
(9, 'XP_038909777.1', NULL),
(9, 'XP_038909776.1', NULL),
(9, 'XP_038909775.1', NULL),
(9, 'XP_038909774.1', NULL),
(9, 'XP_038909773.1', NULL),
(9, 'XP_038909772.1', NULL),
(9, 'XP_038909771.1', NULL),
(9, 'XP_038909770.1', NULL),
(9, 'XP_038909769.1', NULL),
(9, 'XP_038909768.1', NULL),
(9, 'XP_038909767.1', NULL),
(9, 'XP_038909766.1', NULL),
(9, 'XP_038909765.1', NULL),
(9, 'XP_038909764.1', NULL),
(9, 'XP_038909763.1', NULL),
(9, 'XP_038909762.1', NULL),
(9, 'XP_038909761.1', NULL),
(9, 'XP_038909760.1', NULL),
(9, 'XP_038909759.1', NULL),
(9, 'XP_038909758.1', NULL),
(9, 'XP_038909850.1', NULL),
(9, 'XP_038909849.1', NULL),
(9, 'XP_038909848.1', NULL),
(9, 'XP_038909847.1', NULL),
(9, 'XP_038909846.1', NULL),
(9, 'XP_038909845.1', NULL),
(9, 'XP_038909844.1', NULL),
(9, 'XP_038909843.1', NULL),
(9, 'XP_038909842.1', NULL),
(9, 'XP_038909841.1', NULL),
(9, 'XP_038909840.1', NULL),
(9, 'XP_038909839.1', NULL),
(9, 'XP_038909838.1', NULL),
(9, 'XP_038909837.1', NULL),
(9, 'XP_038909836.1', NULL),
(9, 'XP_038909835.1', NULL),
(9, 'XP_038909834.1', NULL),
(9, 'XP_038909833.1', NULL),
(9, 'XP_038909832.1', NULL),
(9, 'XP_038909831.1', NULL),
(9, 'XP_038909830.1', NULL),
(9, 'XP_038909829.1', NULL),
(9, 'XP_038909828.1', NULL),
(9, 'XP_038909827.1', NULL),
(9, 'XP_038909826.1', NULL),
(9, 'XP_038909825.1', NULL),
(9, 'XP_038909824.1', NULL),
(9, 'XP_038909823.1', NULL),
(9, 'XP_038909822.1', NULL),
(9, 'XP_038909821.1', NULL),
(9, 'XP_038909820.1', NULL),
(9, 'XP_038909881.1', NULL),
(9, 'XP_038909880.1', NULL),
(9, 'XP_038909879.1', NULL),
(9, 'XP_038909878.1', NULL),
(9, 'XP_038909877.1', NULL),
(9, 'XP_038909876.1', NULL),
(9, 'XP_038909875.1', NULL),
(9, 'XP_038909874.1', NULL),
(9, 'XP_038909873.1', NULL),
(9, 'XP_038909872.1', NULL),
(9, 'XP_038909871.1', NULL),
(9, 'XP_038909870.1', NULL),
(9, 'XP_038909869.1', NULL),
(9, 'XP_038909868.1', NULL),
(9, 'XP_038909867.1', NULL),
(9, 'XP_038909866.1', NULL),
(9, 'XP_038909865.1', NULL),
(9, 'XP_038909864.1', NULL),
(9, 'XP_038909863.1', NULL),
(9, 'XP_038909862.1', NULL),
(9, 'XP_038909861.1', NULL),
(9, 'XP_038909860.1', NULL),
(9, 'XP_038909859.1', NULL),
(9, 'XP_038909858.1', NULL),
(9, 'XP_038909857.1', NULL),
(9, 'XP_038909856.1', NULL),
(9, 'XP_038909855.1', NULL),
(9, 'XP_038909854.1', NULL),
(9, 'XP_038909853.1', NULL),
(9, 'XP_038909852.1', NULL),
(9, 'XP_038909851.1', NULL),
(9, 'XP_038909933.1', NULL),
(9, 'XP_038909932.1', NULL),
(9, 'XP_038909931.1', NULL),
(9, 'XP_038909930.1', NULL),
(9, 'XP_038909929.1', NULL),
(9, 'XP_038909928.1', NULL),
(9, 'XP_038909927.1', NULL),
(9, 'XP_038909926.1', NULL),
(9, 'XP_038909925.1', NULL),
(9, 'XP_038909924.1', NULL),
(9, 'XP_038909923.1', NULL),
(9, 'XP_038909922.1', NULL),
(9, 'XP_038909921.1', NULL),
(9, 'XP_038909920.1', NULL),
(9, 'XP_038909919.1', NULL),
(9, 'XP_038909918.1', NULL),
(9, 'XP_038909917.1', NULL),
(9, 'XP_038909916.1', NULL),
(9, 'XP_038909915.1', NULL),
(9, 'XP_038909914.1', NULL),
(9, 'XP_038909913.1', NULL),
(9, 'XP_038909912.1', NULL),
(9, 'XP_038909911.1', NULL),
(9, 'XP_038909910.1', NULL),
(9, 'XP_038909909.1', NULL),
(9, 'XP_038909908.1', NULL),
(9, 'XP_038909907.1', NULL),
(9, 'XP_038909906.1', NULL),
(9, 'XP_038909905.1', NULL),
(9, 'XP_038909904.1', NULL),
(9, 'XP_038909903.1', NULL),
(9, 'XP_038909902.1', NULL),
(9, 'XP_038909901.1', NULL),
(9, 'XP_038909900.1', NULL),
(9, 'XP_038909899.1', NULL),
(9, 'XP_038909898.1', NULL),
(9, 'XP_038909897.1', NULL),
(9, 'XP_038909896.1', NULL),
(9, 'XP_038909895.1', NULL),
(9, 'XP_038909894.1', NULL),
(9, 'XP_038909893.1', NULL),
(9, 'XP_038909892.1', NULL),
(9, 'XP_038909891.1', NULL),
(9, 'XP_038909890.1', NULL),
(9, 'XP_038909889.1', NULL),
(9, 'XP_038909888.1', NULL),
(9, 'XP_038909887.1', NULL),
(9, 'XP_038909886.1', NULL),
(9, 'XP_038909885.1', NULL),
(9, 'XP_038909884.1', NULL),
(9, 'XP_038909883.1', NULL),
(9, 'XP_038909882.1', NULL),
(9, 'XP_038909958.1', NULL),
(9, 'XP_038909957.1', NULL),
(9, 'XP_038909956.1', NULL),
(9, 'XP_038909955.1', NULL),
(9, 'XP_038909954.1', NULL),
(9, 'XP_038909953.1', NULL),
(9, 'XP_038909952.1', NULL),
(9, 'XP_038909951.1', NULL),
(9, 'XP_038909950.1', NULL),
(9, 'XP_038909949.1', NULL),
(9, 'XP_038909948.1', NULL),
(9, 'XP_038909947.1', NULL),
(9, 'XP_038909946.1', NULL),
(9, 'XP_038909945.1', NULL),
(9, 'XP_038909944.1', NULL),
(9, 'XP_038909943.1', NULL),
(9, 'XP_038909942.1', NULL),
(9, 'XP_038909941.1', NULL),
(9, 'XP_038909940.1', NULL),
(9, 'XP_038909939.1', NULL),
(9, 'XP_038909938.1', NULL),
(9, 'XP_038909937.1', NULL),
(9, 'XP_038909936.1', NULL),
(9, 'XP_038909935.1', NULL),
(9, 'XP_038909934.1', NULL),
(9, 'XP_038909989.1', NULL),
(9, 'XP_038909988.1', NULL),
(9, 'XP_038909987.1', NULL),
(9, 'XP_038909986.1', NULL),
(9, 'XP_038909985.1', NULL),
(9, 'XP_038909984.1', NULL),
(9, 'XP_038909983.1', NULL),
(9, 'XP_038909982.1', NULL),
(9, 'XP_038909981.1', NULL),
(9, 'XP_038909980.1', NULL),
(9, 'XP_038909979.1', NULL),
(9, 'XP_038909978.1', NULL),
(9, 'XP_038909977.1', NULL),
(9, 'XP_038909976.1', NULL),
(9, 'XP_038909975.1', NULL),
(9, 'XP_038909974.1', NULL),
(9, 'XP_038909973.1', NULL),
(9, 'XP_038909972.1', NULL),
(9, 'XP_038909971.1', NULL),
(9, 'XP_038909970.1', NULL),
(9, 'XP_038909969.1', NULL),
(9, 'XP_038909968.1', NULL),
(9, 'XP_038909967.1', NULL),
(9, 'XP_038909966.1', NULL),
(9, 'XP_038909965.1', NULL),
(9, 'XP_038909964.1', NULL),
(9, 'XP_038909963.1', NULL),
(9, 'XP_038909962.1', NULL),
(9, 'XP_038909961.1', NULL),
(9, 'XP_038909960.1', NULL),
(9, 'XP_038909959.1', NULL),
(9, 'XP_038910032.1', NULL),
(9, 'XP_038910031.1', NULL),
(9, 'XP_038910030.1', NULL),
(9, 'XP_038910029.1', NULL),
(9, 'XP_038910028.1', NULL),
(9, 'XP_038910027.1', NULL),
(9, 'XP_038910026.1', NULL),
(9, 'XP_038910025.1', NULL),
(9, 'XP_038910024.1', NULL),
(9, 'XP_038910023.1', NULL),
(9, 'XP_038910022.1', NULL),
(9, 'XP_038910021.1', NULL),
(9, 'XP_038910020.1', NULL),
(9, 'XP_038910019.1', NULL),
(9, 'XP_038910018.1', NULL),
(9, 'XP_038910017.1', NULL),
(9, 'XP_038910016.1', NULL),
(9, 'XP_038910015.1', NULL),
(9, 'XP_038910014.1', NULL),
(9, 'XP_038910013.1', NULL),
(9, 'XP_038910012.1', NULL),
(9, 'XP_038910011.1', NULL),
(9, 'XP_038910010.1', NULL),
(9, 'XP_038910009.1', NULL),
(9, 'XP_038910008.1', NULL),
(9, 'XP_038910007.1', NULL),
(9, 'XP_038910006.1', NULL),
(9, 'XP_038910005.1', NULL),
(9, 'XP_038910004.1', NULL),
(9, 'XP_038910003.1', NULL),
(9, 'XP_038910002.1', NULL),
(9, 'XP_038910001.1', NULL),
(9, 'XP_038910000.1', NULL),
(9, 'XP_038909999.1', NULL),
(9, 'XP_038909998.1', NULL),
(9, 'XP_038909997.1', NULL),
(9, 'XP_038909996.1', NULL),
(9, 'XP_038909995.1', NULL),
(9, 'XP_038909994.1', NULL),
(9, 'XP_038909993.1', NULL),
(9, 'XP_038909992.1', NULL),
(9, 'XP_038909991.1', NULL),
(9, 'XP_038909990.1', NULL),
(9, 'XP_038910079.1', NULL),
(9, 'XP_038910078.1', NULL),
(9, 'XP_038910077.1', NULL),
(9, 'XP_038910076.1', NULL),
(9, 'XP_038910075.1', NULL),
(9, 'XP_038910074.1', NULL),
(9, 'XP_038910073.1', NULL),
(9, 'XP_038910072.1', NULL),
(9, 'XP_038910071.1', NULL),
(9, 'XP_038910070.1', NULL),
(9, 'XP_038910069.1', NULL),
(9, 'XP_038910068.1', NULL),
(9, 'XP_038910067.1', NULL),
(9, 'XP_038910066.1', NULL),
(9, 'XP_038910065.1', NULL),
(9, 'XP_038910064.1', NULL),
(9, 'XP_038910063.1', NULL),
(9, 'XP_038910062.1', NULL),
(9, 'XP_038910061.1', NULL),
(9, 'XP_038910060.1', NULL),
(9, 'XP_038910059.1', NULL),
(9, 'XP_038910058.1', NULL),
(9, 'XP_038910057.1', NULL),
(9, 'XP_038910056.1', NULL),
(9, 'XP_038910055.1', NULL),
(9, 'XP_038910054.1', NULL),
(9, 'XP_038910053.1', NULL),
(9, 'XP_038910052.1', NULL),
(9, 'XP_038910051.1', NULL),
(9, 'XP_038910050.1', NULL),
(9, 'XP_038910049.1', NULL),
(9, 'XP_038910048.1', NULL),
(9, 'XP_038910047.1', NULL),
(9, 'XP_038910046.1', NULL),
(9, 'XP_038910045.1', NULL),
(9, 'XP_038910044.1', NULL),
(9, 'XP_038910043.1', NULL),
(9, 'XP_038910042.1', NULL),
(9, 'XP_038910041.1', NULL),
(9, 'XP_038910040.1', NULL),
(9, 'XP_038910039.1', NULL),
(9, 'XP_038910038.1', NULL),
(9, 'XP_038910037.1', NULL),
(9, 'XP_038910036.1', NULL),
(9, 'XP_038910035.1', NULL),
(9, 'XP_038910034.1', NULL),
(9, 'XP_038910033.1', NULL),
(9, 'XP_038910102.1', NULL),
(9, 'XP_038910101.1', NULL),
(9, 'XP_038910100.1', NULL),
(9, 'XP_038910099.1', NULL),
(9, 'XP_038910098.1', NULL),
(9, 'XP_038910097.1', NULL),
(9, 'XP_038910096.1', NULL),
(9, 'XP_038910095.1', NULL),
(9, 'XP_038910094.1', NULL),
(9, 'XP_038910093.1', NULL),
(9, 'XP_038910092.1', NULL),
(9, 'XP_038910091.1', NULL),
(9, 'XP_038910090.1', NULL),
(9, 'XP_038910089.1', NULL),
(9, 'XP_038910088.1', NULL),
(9, 'XP_038910087.1', NULL),
(9, 'XP_038910086.1', NULL),
(9, 'XP_038910085.1', NULL),
(9, 'XP_038910084.1', NULL),
(9, 'XP_038910083.1', NULL),
(9, 'XP_038910082.1', NULL),
(9, 'XP_038910081.1', NULL),
(9, 'XP_038910080.1', NULL),
(9, 'XP_038910144.1', NULL),
(9, 'XP_038910143.1', NULL),
(9, 'XP_038910142.1', NULL),
(9, 'XP_038910141.1', NULL),
(9, 'XP_038910140.1', NULL),
(9, 'XP_038910139.1', NULL),
(9, 'XP_038910138.1', NULL),
(9, 'XP_038910137.1', NULL),
(9, 'XP_038910136.1', NULL),
(9, 'XP_038910135.1', NULL),
(9, 'XP_038910134.1', NULL),
(9, 'XP_038910133.1', NULL),
(9, 'XP_038910132.1', NULL),
(9, 'XP_038910131.1', NULL),
(9, 'XP_038910130.1', NULL),
(9, 'XP_038910129.1', NULL),
(9, 'XP_038910128.1', NULL),
(9, 'XP_038910127.1', NULL),
(9, 'XP_038910126.1', NULL),
(9, 'XP_038910125.1', NULL),
(9, 'XP_038910124.1', NULL),
(9, 'XP_038910123.1', NULL),
(9, 'XP_038910122.1', NULL),
(9, 'XP_038910121.1', NULL),
(9, 'XP_038910120.1', NULL),
(9, 'XP_038910119.1', NULL),
(9, 'XP_038910118.1', NULL),
(9, 'XP_038910117.1', NULL),
(9, 'XP_038910116.1', NULL),
(9, 'XP_038910115.1', NULL),
(9, 'XP_038910114.1', NULL),
(9, 'XP_038910113.1', NULL),
(9, 'XP_038910112.1', NULL),
(9, 'XP_038910111.1', NULL),
(9, 'XP_038910110.1', NULL),
(9, 'XP_038910109.1', NULL),
(9, 'XP_038910108.1', NULL),
(9, 'XP_038910107.1', NULL),
(9, 'XP_038910106.1', NULL),
(9, 'XP_038910105.1', NULL),
(9, 'XP_038910104.1', NULL),
(9, 'XP_038910103.1', NULL),
(9, 'XP_038910206.1', NULL),
(9, 'XP_038910205.1', NULL),
(9, 'XP_038910204.1', NULL),
(9, 'XP_038910203.1', NULL),
(9, 'XP_038910202.1', NULL),
(9, 'XP_038910201.1', NULL),
(9, 'XP_038910200.1', NULL),
(9, 'XP_038910199.1', NULL),
(9, 'XP_038910198.1', NULL),
(9, 'XP_038910197.1', NULL),
(9, 'XP_038910196.1', NULL),
(9, 'XP_038910195.1', NULL),
(9, 'XP_038910194.1', NULL),
(9, 'XP_038910193.1', NULL),
(9, 'XP_038910192.1', NULL),
(9, 'XP_038910191.1', NULL),
(9, 'XP_038910190.1', NULL),
(9, 'XP_038910189.1', NULL),
(9, 'XP_038910188.1', NULL),
(9, 'XP_038910187.1', NULL),
(9, 'XP_038910186.1', NULL),
(9, 'XP_038910185.1', NULL),
(9, 'XP_038910184.1', NULL),
(9, 'XP_038910183.1', NULL),
(9, 'XP_038910182.1', NULL),
(9, 'XP_038910181.1', NULL),
(9, 'XP_038910180.1', NULL),
(9, 'XP_038910179.1', NULL),
(9, 'XP_038910178.1', NULL),
(9, 'XP_038910177.1', NULL),
(9, 'XP_038910176.1', NULL),
(9, 'XP_038910175.1', NULL),
(9, 'XP_038910174.1', NULL),
(9, 'XP_038910173.1', NULL),
(9, 'XP_038910172.1', NULL),
(9, 'XP_038910171.1', NULL),
(9, 'XP_038910170.1', NULL),
(9, 'XP_038910169.1', NULL),
(9, 'XP_038910168.1', NULL),
(9, 'XP_038910167.1', NULL),
(9, 'XP_038910166.1', NULL),
(9, 'XP_038910165.1', NULL),
(9, 'XP_038910164.1', NULL),
(9, 'XP_038910163.1', NULL),
(9, 'XP_038910162.1', NULL),
(9, 'XP_038910161.1', NULL),
(9, 'XP_038910160.1', NULL),
(9, 'XP_038910159.1', NULL),
(9, 'XP_038910158.1', NULL),
(9, 'XP_038910157.1', NULL),
(9, 'XP_038910156.1', NULL),
(9, 'XP_038910155.1', NULL),
(9, 'XP_038910154.1', NULL),
(9, 'XP_038910153.1', NULL),
(9, 'XP_038910152.1', NULL),
(9, 'XP_038910151.1', NULL),
(9, 'XP_038910150.1', NULL),
(9, 'XP_038910149.1', NULL),
(9, 'XP_038910148.1', NULL),
(9, 'XP_038910147.1', NULL),
(9, 'XP_038910146.1', NULL),
(9, 'XP_038910145.1', NULL),
(9, 'XP_038910208.1', NULL),
(9, 'XP_038910207.1', NULL),
(9, 'XP_038910235.1', NULL),
(9, 'XP_038910234.1', NULL),
(9, 'XP_038910233.1', NULL),
(9, 'XP_038910232.1', NULL),
(9, 'XP_038910231.1', NULL),
(9, 'XP_038910230.1', NULL),
(9, 'XP_038910229.1', NULL),
(9, 'XP_038910228.1', NULL),
(9, 'XP_038910227.1', NULL),
(9, 'XP_038910226.1', NULL),
(9, 'XP_038910225.1', NULL),
(9, 'XP_038910224.1', NULL),
(9, 'XP_038910223.1', NULL),
(9, 'XP_038910222.1', NULL),
(9, 'XP_038910221.1', NULL),
(9, 'XP_038910220.1', NULL),
(9, 'XP_038910219.1', NULL),
(9, 'XP_038910218.1', NULL),
(9, 'XP_038910217.1', NULL),
(9, 'XP_038910216.1', NULL),
(9, 'XP_038910215.1', NULL),
(9, 'XP_038910214.1', NULL),
(9, 'XP_038910213.1', NULL),
(9, 'XP_038910212.1', NULL),
(9, 'XP_038910211.1', NULL),
(9, 'XP_038910210.1', NULL),
(9, 'XP_038910209.1', NULL),
(9, 'XP_038910262.1', NULL),
(9, 'XP_038910261.1', NULL),
(9, 'XP_038910260.1', NULL),
(9, 'XP_038910259.1', NULL),
(9, 'XP_038910258.1', NULL),
(9, 'XP_038910257.1', NULL),
(9, 'XP_038910256.1', NULL),
(9, 'XP_038910255.1', NULL),
(9, 'XP_038910254.1', NULL),
(9, 'XP_038910253.1', NULL),
(9, 'XP_038910252.1', NULL),
(9, 'XP_038910251.1', NULL),
(9, 'XP_038910250.1', NULL),
(9, 'XP_038910249.1', NULL),
(9, 'XP_038910248.1', NULL),
(9, 'XP_038910247.1', NULL),
(9, 'XP_038910246.1', NULL),
(9, 'XP_038910245.1', NULL),
(9, 'XP_038910244.1', NULL),
(9, 'XP_038910243.1', NULL),
(9, 'XP_038910242.1', NULL),
(9, 'XP_038910241.1', NULL),
(9, 'XP_038910240.1', NULL),
(9, 'XP_038910239.1', NULL),
(9, 'XP_038910238.1', NULL),
(9, 'XP_038910237.1', NULL),
(9, 'XP_038910236.1', NULL),
(9, 'XP_038910312.1', NULL),
(9, 'XP_038910311.1', NULL),
(9, 'XP_038910310.1', NULL),
(9, 'XP_038910309.1', NULL),
(9, 'XP_038910308.1', NULL),
(9, 'XP_038910307.1', NULL),
(9, 'XP_038910306.1', NULL),
(9, 'XP_038910305.1', NULL),
(9, 'XP_038910304.1', NULL),
(9, 'XP_038910303.1', NULL),
(9, 'XP_038910302.1', NULL),
(9, 'XP_038910301.1', NULL),
(9, 'XP_038910300.1', NULL),
(9, 'XP_038910299.1', NULL),
(9, 'XP_038910298.1', NULL),
(9, 'XP_038910297.1', NULL),
(9, 'XP_038910296.1', NULL),
(9, 'XP_038910295.1', NULL),
(9, 'XP_038910294.1', NULL),
(9, 'XP_038910293.1', NULL),
(9, 'XP_038910292.1', NULL),
(9, 'XP_038910291.1', NULL),
(9, 'XP_038910290.1', NULL),
(9, 'XP_038910289.1', NULL),
(9, 'XP_038910288.1', NULL),
(9, 'XP_038910287.1', NULL),
(9, 'XP_038910286.1', NULL),
(9, 'XP_038910285.1', NULL),
(9, 'XP_038910284.1', NULL),
(9, 'XP_038910283.1', NULL),
(9, 'XP_038910282.1', NULL),
(9, 'XP_038910281.1', NULL),
(9, 'XP_038910280.1', NULL),
(9, 'XP_038910279.1', NULL),
(9, 'XP_038910278.1', NULL),
(9, 'XP_038910277.1', NULL),
(9, 'XP_038910276.1', NULL),
(9, 'XP_038910275.1', NULL),
(9, 'XP_038910274.1', NULL),
(9, 'XP_038910273.1', NULL),
(9, 'XP_038910272.1', NULL),
(9, 'XP_038910271.1', NULL),
(9, 'XP_038910270.1', NULL),
(9, 'XP_038910269.1', NULL),
(9, 'XP_038910268.1', NULL),
(9, 'XP_038910267.1', NULL),
(9, 'XP_038910266.1', NULL),
(9, 'XP_038910265.1', NULL),
(9, 'XP_038910264.1', NULL),
(9, 'XP_038910263.1', NULL),
(9, 'XP_038910435.1', NULL),
(9, 'XP_038910434.1', NULL),
(9, 'XP_038910433.1', NULL),
(9, 'XP_038910432.1', NULL),
(9, 'XP_038910431.1', NULL),
(9, 'XP_038910430.1', NULL),
(9, 'XP_038910429.1', NULL),
(9, 'XP_038910428.1', NULL),
(9, 'XP_038910427.1', NULL),
(9, 'XP_038910426.1', NULL),
(9, 'XP_038910425.1', NULL),
(9, 'XP_038910424.1', NULL),
(9, 'XP_038910423.1', NULL),
(9, 'XP_038910422.1', NULL),
(9, 'XP_038910421.1', NULL),
(9, 'XP_038910420.1', NULL),
(9, 'XP_038910419.1', NULL),
(9, 'XP_038910418.1', NULL),
(9, 'XP_038910417.1', NULL),
(9, 'XP_038910416.1', NULL),
(9, 'XP_038910415.1', NULL),
(9, 'XP_038910414.1', NULL),
(9, 'XP_038910413.1', NULL),
(9, 'XP_038910412.1', NULL),
(9, 'XP_038910411.1', NULL),
(9, 'XP_038910410.1', NULL),
(9, 'XP_038910409.1', NULL),
(9, 'XP_038910408.1', NULL),
(9, 'XP_038910407.1', NULL),
(9, 'XP_038910406.1', NULL),
(9, 'XP_038910405.1', NULL),
(9, 'XP_038910404.1', NULL),
(9, 'XP_038910403.1', NULL),
(9, 'XP_038910402.1', NULL),
(9, 'XP_038910401.1', NULL),
(9, 'XP_038910400.1', NULL),
(9, 'XP_038910399.1', NULL),
(9, 'XP_038910398.1', NULL),
(9, 'XP_038910397.1', NULL),
(9, 'XP_038910396.1', NULL),
(9, 'XP_038910395.1', NULL),
(9, 'XP_038910394.1', NULL),
(9, 'XP_038910393.1', NULL),
(9, 'XP_038910392.1', NULL),
(9, 'XP_038910391.1', NULL),
(9, 'XP_038910390.1', NULL),
(9, 'XP_038910389.1', NULL),
(9, 'XP_038910388.1', NULL),
(9, 'XP_038910387.1', NULL),
(9, 'XP_038910386.1', NULL),
(9, 'XP_038910385.1', NULL),
(9, 'XP_038910384.1', NULL),
(9, 'XP_038910383.1', NULL),
(9, 'XP_038910382.1', NULL),
(9, 'XP_038910381.1', NULL),
(9, 'XP_038910380.1', NULL),
(9, 'XP_038910379.1', NULL),
(9, 'XP_038910378.1', NULL),
(9, 'XP_038910377.1', NULL),
(9, 'XP_038910376.1', NULL),
(9, 'XP_038910375.1', NULL),
(9, 'XP_038910374.1', NULL),
(9, 'XP_038910373.1', NULL),
(9, 'XP_038910372.1', NULL),
(9, 'XP_038910371.1', NULL),
(9, 'XP_038910370.1', NULL),
(9, 'XP_038910369.1', NULL),
(9, 'XP_038910368.1', NULL),
(9, 'XP_038910367.1', NULL),
(9, 'XP_038910366.1', NULL),
(9, 'XP_038910365.1', NULL),
(9, 'XP_038910364.1', NULL),
(9, 'XP_038910363.1', NULL),
(9, 'XP_038910362.1', NULL),
(9, 'XP_038910361.1', NULL),
(9, 'XP_038910360.1', NULL),
(9, 'XP_038910359.1', NULL),
(9, 'XP_038910358.1', NULL),
(9, 'XP_038910357.1', NULL),
(9, 'XP_038910356.1', NULL),
(9, 'XP_038910355.1', NULL),
(9, 'XP_038910354.1', NULL),
(9, 'XP_038910353.1', NULL),
(9, 'XP_038910352.1', NULL),
(9, 'XP_038910351.1', NULL),
(9, 'XP_038910350.1', NULL),
(9, 'XP_038910349.1', NULL),
(9, 'XP_038910348.1', NULL),
(9, 'XP_038910347.1', NULL),
(9, 'XP_038910346.1', NULL),
(9, 'XP_038910345.1', NULL),
(9, 'XP_038910344.1', NULL),
(9, 'XP_038910343.1', NULL),
(9, 'XP_038910342.1', NULL),
(9, 'XP_038910341.1', NULL),
(9, 'XP_038910340.1', NULL),
(9, 'XP_038910339.1', NULL),
(9, 'XP_038910338.1', NULL),
(9, 'XP_038910337.1', NULL),
(9, 'XP_038910336.1', NULL),
(9, 'XP_038910335.1', NULL),
(9, 'XP_038910334.1', NULL),
(9, 'XP_038910333.1', NULL),
(9, 'XP_038910332.1', NULL),
(9, 'XP_038910331.1', NULL),
(9, 'XP_038910330.1', NULL),
(9, 'XP_038910329.1', NULL),
(9, 'XP_038910328.1', NULL),
(9, 'XP_038910327.1', NULL),
(9, 'XP_038910326.1', NULL),
(9, 'XP_038910325.1', NULL),
(9, 'XP_038910324.1', NULL),
(9, 'XP_038910323.1', NULL),
(9, 'XP_038910322.1', NULL),
(9, 'XP_038910321.1', NULL),
(9, 'XP_038910320.1', NULL),
(9, 'XP_038910319.1', NULL),
(9, 'XP_038910318.1', NULL),
(9, 'XP_038910317.1', NULL),
(9, 'XP_038910316.1', NULL),
(9, 'XP_038910315.1', NULL),
(9, 'XP_038910314.1', NULL),
(9, 'XP_038910313.1', NULL),
(9, 'XP_038910538.1', NULL),
(9, 'XP_038910537.1', NULL),
(9, 'XP_038910536.1', NULL),
(9, 'XP_038910535.1', NULL),
(9, 'XP_038910534.1', NULL),
(9, 'XP_038910533.1', NULL),
(9, 'XP_038910532.1', NULL),
(9, 'XP_038910531.1', NULL),
(9, 'XP_038910530.1', NULL),
(9, 'XP_038910529.1', NULL),
(9, 'XP_038910528.1', NULL),
(9, 'XP_038910527.1', NULL),
(9, 'XP_038910526.1', NULL),
(9, 'XP_038910525.1', NULL),
(9, 'XP_038910524.1', NULL),
(9, 'XP_038910523.1', NULL),
(9, 'XP_038910522.1', NULL),
(9, 'XP_038910521.1', NULL),
(9, 'XP_038910520.1', NULL),
(9, 'XP_038910519.1', NULL),
(9, 'XP_038910518.1', NULL),
(9, 'XP_038910517.1', NULL),
(9, 'XP_038910516.1', NULL),
(9, 'XP_038910515.1', NULL),
(9, 'XP_038910514.1', NULL),
(9, 'XP_038910513.1', NULL),
(9, 'XP_038910512.1', NULL),
(9, 'XP_038910511.1', NULL),
(9, 'XP_038910510.1', NULL),
(9, 'XP_038910509.1', NULL),
(9, 'XP_038910508.1', NULL),
(9, 'XP_038910507.1', NULL),
(9, 'XP_038910506.1', NULL),
(9, 'XP_038910505.1', NULL),
(9, 'XP_038910504.1', NULL),
(9, 'XP_038910503.1', NULL),
(9, 'XP_038910502.1', NULL),
(9, 'XP_038910501.1', NULL),
(9, 'XP_038910500.1', NULL),
(9, 'XP_038910499.1', NULL),
(9, 'XP_038910498.1', NULL),
(9, 'XP_038910497.1', NULL),
(9, 'XP_038910496.1', NULL),
(9, 'XP_038910495.1', NULL),
(9, 'XP_038910494.1', NULL),
(9, 'XP_038910493.1', NULL),
(9, 'XP_038910492.1', NULL),
(9, 'XP_038910491.1', NULL),
(9, 'XP_038910490.1', NULL),
(9, 'XP_038910489.1', NULL),
(9, 'XP_038910488.1', NULL),
(9, 'XP_038910487.1', NULL),
(9, 'XP_038910486.1', NULL),
(9, 'XP_038910485.1', NULL),
(9, 'XP_038910484.1', NULL),
(9, 'XP_038910483.1', NULL),
(9, 'XP_038910482.1', NULL),
(9, 'XP_038910481.1', NULL),
(9, 'XP_038910480.1', NULL),
(9, 'XP_038910479.1', NULL),
(9, 'XP_038910478.1', NULL),
(9, 'XP_038910477.1', NULL),
(9, 'XP_038910476.1', NULL),
(9, 'XP_038910475.1', NULL),
(9, 'XP_038910474.1', NULL),
(9, 'XP_038910473.1', NULL),
(9, 'XP_038910472.1', NULL),
(9, 'XP_038910471.1', NULL),
(9, 'XP_038910470.1', NULL),
(9, 'XP_038910469.1', NULL),
(9, 'XP_038910468.1', NULL),
(9, 'XP_038910467.1', NULL),
(9, 'XP_038910466.1', NULL),
(9, 'XP_038910465.1', NULL),
(9, 'XP_038910464.1', NULL),
(9, 'XP_038910463.1', NULL),
(9, 'XP_038910462.1', NULL),
(9, 'XP_038910461.1', NULL),
(9, 'XP_038910460.1', NULL),
(9, 'XP_038910459.1', NULL),
(9, 'XP_038910458.1', NULL),
(9, 'XP_038910457.1', NULL),
(9, 'XP_038910456.1', NULL),
(9, 'XP_038910455.1', NULL),
(9, 'XP_038910454.1', NULL),
(9, 'XP_038910453.1', NULL),
(9, 'XP_038910452.1', NULL),
(9, 'XP_038910451.1', NULL),
(9, 'XP_038910450.1', NULL),
(9, 'XP_038910449.1', NULL),
(9, 'XP_038910448.1', NULL),
(9, 'XP_038910447.1', NULL),
(9, 'XP_038910446.1', NULL),
(9, 'XP_038910445.1', NULL),
(9, 'XP_038910444.1', NULL),
(9, 'XP_038910443.1', NULL),
(9, 'XP_038910442.1', NULL),
(9, 'XP_038910441.1', NULL),
(9, 'XP_038910440.1', NULL),
(9, 'XP_038910439.1', NULL),
(9, 'XP_038910438.1', NULL),
(9, 'XP_038910437.1', NULL),
(9, 'XP_038910436.1', NULL),
(9, 'XP_038910563.1', NULL),
(9, 'XP_038910562.1', NULL),
(9, 'XP_038910561.1', NULL),
(9, 'XP_038910560.1', NULL),
(9, 'XP_038910559.1', NULL),
(9, 'XP_038910558.1', NULL),
(9, 'XP_038910557.1', NULL),
(9, 'XP_038910556.1', NULL),
(9, 'XP_038910555.1', NULL),
(9, 'XP_038910554.1', NULL),
(9, 'XP_038910553.1', NULL),
(9, 'XP_038910552.1', NULL),
(9, 'XP_038910551.1', NULL),
(9, 'XP_038910550.1', NULL),
(9, 'XP_038910549.1', NULL),
(9, 'XP_038910548.1', NULL),
(9, 'XP_038910547.1', NULL),
(9, 'XP_038910546.1', NULL),
(9, 'XP_038910545.1', NULL),
(9, 'XP_038910544.1', NULL),
(9, 'XP_038910543.1', NULL),
(9, 'XP_038910542.1', NULL),
(9, 'XP_038910541.1', NULL),
(9, 'XP_038910540.1', NULL),
(9, 'XP_038910539.1', NULL),
(9, 'XP_038910633.1', NULL),
(9, 'XP_038910632.1', NULL),
(9, 'XP_038910631.1', NULL),
(9, 'XP_038910630.1', NULL),
(9, 'XP_038910629.1', NULL),
(9, 'XP_038910628.1', NULL),
(9, 'XP_038910627.1', NULL),
(9, 'XP_038910626.1', NULL),
(9, 'XP_038910625.1', NULL),
(9, 'XP_038910624.1', NULL),
(9, 'XP_038910623.1', NULL),
(9, 'XP_038910622.1', NULL),
(9, 'XP_038910621.1', NULL),
(9, 'XP_038910620.1', NULL),
(9, 'XP_038910619.1', NULL),
(9, 'XP_038910618.1', NULL),
(9, 'XP_038910617.1', NULL),
(9, 'XP_038910616.1', NULL),
(9, 'XP_038910615.1', NULL),
(9, 'XP_038910614.1', NULL),
(9, 'XP_038910613.1', NULL),
(9, 'XP_038910612.1', NULL),
(9, 'XP_038910611.1', NULL),
(9, 'XP_038910610.1', NULL),
(9, 'XP_038910609.1', NULL),
(9, 'XP_038910608.1', NULL),
(9, 'XP_038910607.1', NULL),
(9, 'XP_038910606.1', NULL),
(9, 'XP_038910605.1', NULL),
(9, 'XP_038910604.1', NULL),
(9, 'XP_038910603.1', NULL),
(9, 'XP_038910602.1', NULL),
(9, 'XP_038910601.1', NULL),
(9, 'XP_038910600.1', NULL),
(9, 'XP_038910599.1', NULL),
(9, 'XP_038910598.1', NULL),
(9, 'XP_038910597.1', NULL),
(9, 'XP_038910596.1', NULL),
(9, 'XP_038910595.1', NULL),
(9, 'XP_038910594.1', NULL),
(9, 'XP_038910593.1', NULL),
(9, 'XP_038910592.1', NULL),
(9, 'XP_038910591.1', NULL),
(9, 'XP_038910590.1', NULL),
(9, 'XP_038910589.1', NULL),
(9, 'XP_038910588.1', NULL),
(9, 'XP_038910587.1', NULL),
(9, 'XP_038910586.1', NULL),
(9, 'XP_038910585.1', NULL),
(9, 'XP_038910584.1', NULL),
(9, 'XP_038910583.1', NULL),
(9, 'XP_038910582.1', NULL),
(9, 'XP_038910581.1', NULL),
(9, 'XP_038910580.1', NULL),
(9, 'XP_038910579.1', NULL),
(9, 'XP_038910578.1', NULL),
(9, 'XP_038910577.1', NULL),
(9, 'XP_038910576.1', NULL),
(9, 'XP_038910575.1', NULL),
(9, 'XP_038910574.1', NULL),
(9, 'XP_038910573.1', NULL),
(9, 'XP_038910572.1', NULL),
(9, 'XP_038910571.1', NULL),
(9, 'XP_038910570.1', NULL),
(9, 'XP_038910569.1', NULL),
(9, 'XP_038910568.1', NULL),
(9, 'XP_038910567.1', NULL),
(9, 'XP_038910566.1', NULL),
(9, 'XP_038910565.1', NULL),
(9, 'XP_038910564.1', NULL),
(9, 'XP_038910667.1', NULL),
(9, 'XP_038910666.1', NULL),
(9, 'XP_038910665.1', NULL),
(9, 'XP_038910664.1', NULL),
(9, 'XP_038910663.1', NULL),
(9, 'XP_038910662.1', NULL),
(9, 'XP_038910661.1', NULL),
(9, 'XP_038910660.1', NULL),
(9, 'XP_038910659.1', NULL),
(9, 'XP_038910658.1', NULL),
(9, 'XP_038910657.1', NULL),
(9, 'XP_038910656.1', NULL),
(9, 'XP_038910655.1', NULL),
(9, 'XP_038910654.1', NULL),
(9, 'XP_038910653.1', NULL),
(9, 'XP_038910652.1', NULL),
(9, 'XP_038910651.1', NULL),
(9, 'XP_038910650.1', NULL),
(9, 'XP_038910649.1', NULL),
(9, 'XP_038910648.1', NULL),
(9, 'XP_038910647.1', NULL),
(9, 'XP_038910646.1', NULL),
(9, 'XP_038910645.1', NULL),
(9, 'XP_038910644.1', NULL),
(9, 'XP_038910643.1', NULL),
(9, 'XP_038910642.1', NULL),
(9, 'XP_038910641.1', NULL),
(9, 'XP_038910640.1', NULL),
(9, 'XP_038910639.1', NULL),
(9, 'XP_038910638.1', NULL),
(9, 'XP_038910637.1', NULL),
(9, 'XP_038910636.1', NULL),
(9, 'XP_038910635.1', NULL),
(9, 'XP_038910634.1', NULL),
(9, 'XP_038910716.1', NULL),
(9, 'XP_038910715.1', NULL),
(9, 'XP_038910714.1', NULL),
(9, 'XP_038910713.1', NULL),
(9, 'XP_038910712.1', NULL),
(9, 'XP_038910711.1', NULL),
(9, 'XP_038910710.1', NULL),
(9, 'XP_038910709.1', NULL),
(9, 'XP_038910708.1', NULL),
(9, 'XP_038910707.1', NULL),
(9, 'XP_038910706.1', NULL),
(9, 'XP_038910705.1', NULL),
(9, 'XP_038910704.1', NULL),
(9, 'XP_038910703.1', NULL),
(9, 'XP_038910702.1', NULL),
(9, 'XP_038910701.1', NULL),
(9, 'XP_038910700.1', NULL),
(9, 'XP_038910699.1', NULL),
(9, 'XP_038910698.1', NULL),
(9, 'XP_038910697.1', NULL),
(9, 'XP_038910696.1', NULL),
(9, 'XP_038910695.1', NULL),
(9, 'XP_038910694.1', NULL),
(9, 'XP_038910693.1', NULL),
(9, 'XP_038910692.1', NULL),
(9, 'XP_038910691.1', NULL),
(9, 'XP_038910690.1', NULL),
(9, 'XP_038910689.1', NULL),
(9, 'XP_038910688.1', NULL),
(9, 'XP_038910687.1', NULL),
(9, 'XP_038910686.1', NULL),
(9, 'XP_038910685.1', NULL),
(9, 'XP_038910684.1', NULL),
(9, 'XP_038910683.1', NULL),
(9, 'XP_038910682.1', NULL),
(9, 'XP_038910681.1', NULL),
(9, 'XP_038910680.1', NULL),
(9, 'XP_038910679.1', NULL),
(9, 'XP_038910678.1', NULL),
(9, 'XP_038910677.1', NULL),
(9, 'XP_038910676.1', NULL),
(9, 'XP_038910675.1', NULL),
(9, 'XP_038910674.1', NULL),
(9, 'XP_038910673.1', NULL),
(9, 'XP_038910672.1', NULL),
(9, 'XP_038910671.1', NULL),
(9, 'XP_038910670.1', NULL),
(9, 'XP_038910669.1', NULL),
(9, 'XP_038910668.1', NULL),
(9, 'XP_038910729.1', NULL),
(9, 'XP_038910728.1', NULL),
(9, 'XP_038910727.1', NULL),
(9, 'XP_038910726.1', NULL),
(9, 'XP_038910725.1', NULL),
(9, 'XP_038910724.1', NULL),
(9, 'XP_038910723.1', NULL),
(9, 'XP_038910722.1', NULL),
(9, 'XP_038910721.1', NULL),
(9, 'XP_038910720.1', NULL),
(9, 'XP_038910719.1', NULL),
(9, 'XP_038910718.1', NULL),
(9, 'XP_038910717.1', NULL),
(9, 'XP_038910762.1', NULL),
(9, 'XP_038910761.1', NULL),
(9, 'XP_038910760.1', NULL),
(9, 'XP_038910759.1', NULL),
(9, 'XP_038910758.1', NULL),
(9, 'XP_038910757.1', NULL),
(9, 'XP_038910756.1', NULL),
(9, 'XP_038910755.1', NULL),
(9, 'XP_038910754.1', NULL),
(9, 'XP_038910753.1', NULL),
(9, 'XP_038910752.1', NULL),
(9, 'XP_038910751.1', NULL),
(9, 'XP_038910750.1', NULL),
(9, 'XP_038910749.1', NULL),
(9, 'XP_038910748.1', NULL),
(9, 'XP_038910747.1', NULL),
(9, 'XP_038910746.1', NULL),
(9, 'XP_038910745.1', NULL),
(9, 'XP_038910744.1', NULL),
(9, 'XP_038910743.1', NULL),
(9, 'XP_038910742.1', NULL),
(9, 'XP_038910741.1', NULL),
(9, 'XP_038910740.1', NULL),
(9, 'XP_038910739.1', NULL),
(9, 'XP_038910738.1', NULL),
(9, 'XP_038910737.1', NULL),
(9, 'XP_038910736.1', NULL),
(9, 'XP_038910735.1', NULL),
(9, 'XP_038910734.1', NULL),
(9, 'XP_038910733.1', NULL),
(9, 'XP_038910732.1', NULL),
(9, 'XP_038910731.1', NULL),
(9, 'XP_038910730.1', NULL),
(9, 'XP_038910811.1', NULL),
(9, 'XP_038910810.1', NULL),
(9, 'XP_038910809.1', NULL),
(9, 'XP_038910808.1', NULL),
(9, 'XP_038910807.1', NULL),
(9, 'XP_038910806.1', NULL),
(9, 'XP_038910805.1', NULL),
(9, 'XP_038910804.1', NULL),
(9, 'XP_038910803.1', NULL),
(9, 'XP_038910802.1', NULL),
(9, 'XP_038910801.1', NULL),
(9, 'XP_038910800.1', NULL),
(9, 'XP_038910799.1', NULL),
(9, 'XP_038910798.1', NULL),
(9, 'XP_038910797.1', NULL),
(9, 'XP_038910796.1', NULL),
(9, 'XP_038910795.1', NULL),
(9, 'XP_038910794.1', NULL),
(9, 'XP_038910793.1', NULL),
(9, 'XP_038910792.1', NULL),
(9, 'XP_038910791.1', NULL),
(9, 'XP_038910790.1', NULL),
(9, 'XP_038910789.1', NULL),
(9, 'XP_038910788.1', NULL),
(9, 'XP_038910787.1', NULL),
(9, 'XP_038910786.1', NULL),
(9, 'XP_038910785.1', NULL),
(9, 'XP_038910784.1', NULL),
(9, 'XP_038910783.1', NULL),
(9, 'XP_038910782.1', NULL),
(9, 'XP_038910781.1', NULL),
(9, 'XP_038910780.1', NULL),
(9, 'XP_038910779.1', NULL),
(9, 'XP_038910778.1', NULL),
(9, 'XP_038910777.1', NULL),
(9, 'XP_038910776.1', NULL),
(9, 'XP_038910775.1', NULL),
(9, 'XP_038910774.1', NULL),
(9, 'XP_038910773.1', NULL),
(9, 'XP_038910772.1', NULL),
(9, 'XP_038910771.1', NULL),
(9, 'XP_038910770.1', NULL),
(9, 'XP_038910769.1', NULL),
(9, 'XP_038910768.1', NULL),
(9, 'XP_038910767.1', NULL),
(9, 'XP_038910766.1', NULL),
(9, 'XP_038910765.1', NULL),
(9, 'XP_038910764.1', NULL),
(9, 'XP_038910763.1', NULL),
(9, 'XP_038910836.1', NULL),
(9, 'XP_038910835.1', NULL),
(9, 'XP_038910834.1', NULL),
(9, 'XP_038910833.1', NULL),
(9, 'XP_038910832.1', NULL),
(9, 'XP_038910831.1', NULL),
(9, 'XP_038910830.1', NULL),
(9, 'XP_038910829.1', NULL),
(9, 'XP_038910828.1', NULL),
(9, 'XP_038910827.1', NULL),
(9, 'XP_038910826.1', NULL),
(9, 'XP_038910825.1', NULL),
(9, 'XP_038910824.1', NULL),
(9, 'XP_038910823.1', NULL),
(9, 'XP_038910822.1', NULL),
(9, 'XP_038910821.1', NULL),
(9, 'XP_038910820.1', NULL),
(9, 'XP_038910819.1', NULL),
(9, 'XP_038910818.1', NULL),
(9, 'XP_038910817.1', NULL),
(9, 'XP_038910816.1', NULL),
(9, 'XP_038910815.1', NULL),
(9, 'XP_038910814.1', NULL),
(9, 'XP_038910813.1', NULL),
(9, 'XP_038910812.1', NULL),
(9, 'XP_038910872.1', NULL),
(9, 'XP_038910871.1', NULL),
(9, 'XP_038910870.1', NULL),
(9, 'XP_038910869.1', NULL),
(9, 'XP_038910868.1', NULL),
(9, 'XP_038910867.1', NULL),
(9, 'XP_038910866.1', NULL),
(9, 'XP_038910865.1', NULL),
(9, 'XP_038910864.1', NULL),
(9, 'XP_038910863.1', NULL),
(9, 'XP_038910862.1', NULL),
(9, 'XP_038910861.1', NULL),
(9, 'XP_038910860.1', NULL),
(9, 'XP_038910859.1', NULL),
(9, 'XP_038910858.1', NULL),
(9, 'XP_038910857.1', NULL),
(9, 'XP_038910856.1', NULL),
(9, 'XP_038910855.1', NULL),
(9, 'XP_038910854.1', NULL),
(9, 'XP_038910853.1', NULL),
(9, 'XP_038910852.1', NULL),
(9, 'XP_038910851.1', NULL),
(9, 'XP_038910850.1', NULL),
(9, 'XP_038910849.1', NULL),
(9, 'XP_038910848.1', NULL),
(9, 'XP_038910847.1', NULL),
(9, 'XP_038910846.1', NULL),
(9, 'XP_038910845.1', NULL),
(9, 'XP_038910844.1', NULL),
(9, 'XP_038910843.1', NULL),
(9, 'XP_038910842.1', NULL),
(9, 'XP_038910841.1', NULL),
(9, 'XP_038910840.1', NULL),
(9, 'XP_038910839.1', NULL),
(9, 'XP_038910838.1', NULL),
(9, 'XP_038910837.1', NULL),
(9, 'XP_038910906.1', NULL),
(9, 'XP_038910905.1', NULL),
(9, 'XP_038910904.1', NULL),
(9, 'XP_038910903.1', NULL),
(9, 'XP_038910902.1', NULL),
(9, 'XP_038910901.1', NULL),
(9, 'XP_038910900.1', NULL),
(9, 'XP_038910899.1', NULL),
(9, 'XP_038910898.1', NULL),
(9, 'XP_038910897.1', NULL),
(9, 'XP_038910896.1', NULL),
(9, 'XP_038910895.1', NULL),
(9, 'XP_038910894.1', NULL),
(9, 'XP_038910893.1', NULL),
(9, 'XP_038910892.1', NULL),
(9, 'XP_038910891.1', NULL),
(9, 'XP_038910890.1', NULL),
(9, 'XP_038910889.1', NULL),
(9, 'XP_038910888.1', NULL),
(9, 'XP_038910887.1', NULL),
(9, 'XP_038910886.1', NULL),
(9, 'XP_038910885.1', NULL),
(9, 'XP_038910884.1', NULL),
(9, 'XP_038910883.1', NULL),
(9, 'XP_038910882.1', NULL),
(9, 'XP_038910881.1', NULL),
(9, 'XP_038910880.1', NULL),
(9, 'XP_038910879.1', NULL),
(9, 'XP_038910878.1', NULL),
(9, 'XP_038910877.1', NULL),
(9, 'XP_038910876.1', NULL),
(9, 'XP_038910875.1', NULL),
(9, 'XP_038910874.1', NULL),
(9, 'XP_038910873.1', NULL),
(9, 'XP_038910974.1', NULL),
(9, 'XP_038910973.1', NULL),
(9, 'XP_038910972.1', NULL),
(9, 'XP_038910971.1', NULL),
(9, 'XP_038910970.1', NULL),
(9, 'XP_038910969.1', NULL),
(9, 'XP_038910968.1', NULL),
(9, 'XP_038910967.1', NULL),
(9, 'XP_038910966.1', NULL),
(9, 'XP_038910965.1', NULL),
(9, 'XP_038910964.1', NULL),
(9, 'XP_038910963.1', NULL),
(9, 'XP_038910962.1', NULL),
(9, 'XP_038910961.1', NULL),
(9, 'XP_038910960.1', NULL),
(9, 'XP_038910959.1', NULL),
(9, 'XP_038910958.1', NULL),
(9, 'XP_038910957.1', NULL),
(9, 'XP_038910956.1', NULL),
(9, 'XP_038910955.1', NULL),
(9, 'XP_038910954.1', NULL),
(9, 'XP_038910953.1', NULL),
(9, 'XP_038910952.1', NULL),
(9, 'XP_038910951.1', NULL),
(9, 'XP_038910950.1', NULL),
(9, 'XP_038910949.1', NULL),
(9, 'XP_038910948.1', NULL),
(9, 'XP_038910947.1', NULL),
(9, 'XP_038910946.1', NULL),
(9, 'XP_038910945.1', NULL),
(9, 'XP_038910944.1', NULL),
(9, 'XP_038910943.1', NULL),
(9, 'XP_038910942.1', NULL),
(9, 'XP_038910941.1', NULL),
(9, 'XP_038910940.1', NULL),
(9, 'XP_038910939.1', NULL),
(9, 'XP_038910938.1', NULL),
(9, 'XP_038910937.1', NULL),
(9, 'XP_038910936.1', NULL),
(9, 'XP_038910935.1', NULL),
(9, 'XP_038910934.1', NULL),
(9, 'XP_038910933.1', NULL),
(9, 'XP_038910932.1', NULL),
(9, 'XP_038910931.1', NULL),
(9, 'XP_038910930.1', NULL),
(9, 'XP_038910929.1', NULL),
(9, 'XP_038910928.1', NULL),
(9, 'XP_038910927.1', NULL),
(9, 'XP_038910926.1', NULL),
(9, 'XP_038910925.1', NULL),
(9, 'XP_038910924.1', NULL),
(9, 'XP_038910923.1', NULL),
(9, 'XP_038910922.1', NULL),
(9, 'XP_038910921.1', NULL),
(9, 'XP_038910920.1', NULL),
(9, 'XP_038910919.1', NULL),
(9, 'XP_038910918.1', NULL),
(9, 'XP_038910917.1', NULL),
(9, 'XP_038910916.1', NULL),
(9, 'XP_038910915.1', NULL),
(9, 'XP_038910914.1', NULL),
(9, 'XP_038910913.1', NULL),
(9, 'XP_038910912.1', NULL),
(9, 'XP_038910911.1', NULL),
(9, 'XP_038910910.1', NULL),
(9, 'XP_038910909.1', NULL),
(9, 'XP_038910908.1', NULL),
(9, 'XP_038910907.1', NULL),
(9, 'XP_038911026.1', NULL),
(9, 'XP_038911025.1', NULL),
(9, 'XP_038911024.1', NULL),
(9, 'XP_038911023.1', NULL);
INSERT INTO `prot_access_table` (`id_article`, `prot_access`, `entry_id_cazy`) VALUES
(9, 'XP_038911022.1', NULL),
(9, 'XP_038911021.1', NULL),
(9, 'XP_038911020.1', NULL),
(9, 'XP_038911019.1', NULL),
(9, 'XP_038911018.1', NULL),
(9, 'XP_038911017.1', NULL),
(9, 'XP_038911016.1', NULL),
(9, 'XP_038911015.1', NULL),
(9, 'XP_038911014.1', NULL),
(9, 'XP_038911013.1', NULL),
(9, 'XP_038911012.1', NULL),
(9, 'XP_038911011.1', NULL),
(9, 'XP_038911010.1', NULL),
(9, 'XP_038911009.1', NULL),
(9, 'XP_038911008.1', NULL),
(9, 'XP_038911007.1', NULL),
(9, 'XP_038911006.1', NULL),
(9, 'XP_038911005.1', NULL),
(9, 'XP_038911004.1', NULL),
(9, 'XP_038911003.1', NULL),
(9, 'XP_038911002.1', NULL),
(9, 'XP_038911001.1', NULL),
(9, 'XP_038911000.1', NULL),
(9, 'XP_038910999.1', NULL),
(9, 'XP_038910998.1', NULL),
(9, 'XP_038910997.1', NULL),
(9, 'XP_038910996.1', NULL),
(9, 'XP_038910995.1', NULL),
(9, 'XP_038910994.1', NULL),
(9, 'XP_038910993.1', NULL),
(9, 'XP_038910992.1', NULL),
(9, 'XP_038910991.1', NULL),
(9, 'XP_038910990.1', NULL),
(9, 'XP_038910989.1', NULL),
(9, 'XP_038910988.1', NULL),
(9, 'XP_038910987.1', NULL),
(9, 'XP_038910986.1', NULL),
(9, 'XP_038910985.1', NULL),
(9, 'XP_038910984.1', NULL),
(9, 'XP_038910983.1', NULL),
(9, 'XP_038910982.1', NULL),
(9, 'XP_038910981.1', NULL),
(9, 'XP_038910980.1', NULL),
(9, 'XP_038910979.1', NULL),
(9, 'XP_038910978.1', NULL),
(9, 'XP_038910977.1', NULL),
(9, 'XP_038910976.1', NULL),
(9, 'XP_038910975.1', NULL),
(9, 'XP_038911068.1', NULL),
(9, 'XP_038911067.1', NULL),
(9, 'XP_038911066.1', NULL),
(9, 'XP_038911065.1', NULL),
(9, 'XP_038911064.1', NULL),
(9, 'XP_038911063.1', NULL),
(9, 'XP_038911062.1', NULL),
(9, 'XP_038911061.1', NULL),
(9, 'XP_038911060.1', NULL),
(9, 'XP_038911059.1', NULL),
(9, 'XP_038911058.1', NULL),
(9, 'XP_038911057.1', NULL),
(9, 'XP_038911056.1', NULL),
(9, 'XP_038911055.1', NULL),
(9, 'XP_038911054.1', NULL),
(9, 'XP_038911053.1', NULL),
(9, 'XP_038911052.1', NULL),
(9, 'XP_038911051.1', NULL),
(9, 'XP_038911050.1', NULL),
(9, 'XP_038911049.1', NULL),
(9, 'XP_038911048.1', NULL),
(9, 'XP_038911047.1', NULL),
(9, 'XP_038911046.1', NULL),
(9, 'XP_038911045.1', NULL),
(9, 'XP_038911044.1', NULL),
(9, 'XP_038911043.1', NULL),
(9, 'XP_038911042.1', NULL),
(9, 'XP_038911041.1', NULL),
(9, 'XP_038911040.1', NULL),
(9, 'XP_038911039.1', NULL),
(9, 'XP_038911038.1', NULL),
(9, 'XP_038911037.1', NULL),
(9, 'XP_038911036.1', NULL),
(9, 'XP_038911035.1', NULL),
(9, 'XP_038911034.1', NULL),
(9, 'XP_038911033.1', NULL),
(9, 'XP_038911032.1', NULL),
(9, 'XP_038911031.1', NULL),
(9, 'XP_038911030.1', NULL),
(9, 'XP_038911029.1', NULL),
(9, 'XP_038911028.1', NULL),
(9, 'XP_038911027.1', NULL),
(9, 'XP_038911093.1', NULL),
(9, 'XP_038911092.1', NULL),
(9, 'XP_038911091.1', NULL),
(9, 'XP_038911090.1', NULL),
(9, 'XP_038911089.1', NULL),
(9, 'XP_038911088.1', NULL),
(9, 'XP_038911087.1', NULL),
(9, 'XP_038911086.1', NULL),
(9, 'XP_038911085.1', NULL),
(9, 'XP_038911084.1', NULL),
(9, 'XP_038911083.1', NULL),
(9, 'XP_038911082.1', NULL),
(9, 'XP_038911081.1', NULL),
(9, 'XP_038911080.1', NULL),
(9, 'XP_038911079.1', NULL),
(9, 'XP_038911078.1', NULL),
(9, 'XP_038911077.1', NULL),
(9, 'XP_038911076.1', NULL),
(9, 'XP_038911075.1', NULL),
(9, 'XP_038911074.1', NULL),
(9, 'XP_038911073.1', NULL),
(9, 'XP_038911072.1', NULL),
(9, 'XP_038911071.1', NULL),
(9, 'XP_038911070.1', NULL),
(9, 'XP_038911069.1', NULL),
(9, 'XP_038911133.1', NULL),
(9, 'XP_038911132.1', NULL),
(9, 'XP_038911131.1', NULL),
(9, 'XP_038911130.1', NULL),
(9, 'XP_038911129.1', NULL),
(9, 'XP_038911128.1', NULL),
(9, 'XP_038911127.1', NULL),
(9, 'XP_038911126.1', NULL),
(9, 'XP_038911125.1', NULL),
(9, 'XP_038911124.1', NULL),
(9, 'XP_038911123.1', NULL),
(9, 'XP_038911122.1', NULL),
(9, 'XP_038911121.1', NULL),
(9, 'XP_038911120.1', NULL),
(9, 'XP_038911119.1', NULL),
(9, 'XP_038911118.1', NULL),
(9, 'XP_038911117.1', NULL),
(9, 'XP_038911116.1', NULL),
(9, 'XP_038911115.1', NULL),
(9, 'XP_038911114.1', NULL),
(9, 'XP_038911113.1', NULL),
(9, 'XP_038911112.1', NULL),
(9, 'XP_038911111.1', NULL),
(9, 'XP_038911110.1', NULL),
(9, 'XP_038911109.1', NULL),
(9, 'XP_038911108.1', NULL),
(9, 'XP_038911107.1', NULL),
(9, 'XP_038911106.1', NULL),
(9, 'XP_038911105.1', NULL),
(9, 'XP_038911104.1', NULL),
(9, 'XP_038911103.1', NULL),
(9, 'XP_038911102.1', NULL),
(9, 'XP_038911101.1', NULL),
(9, 'XP_038911100.1', NULL),
(9, 'XP_038911099.1', NULL),
(9, 'XP_038911098.1', NULL),
(9, 'XP_038911097.1', NULL),
(9, 'XP_038911096.1', NULL),
(9, 'XP_038911095.1', NULL),
(9, 'XP_038911094.1', NULL),
(9, 'XP_038911234.1', NULL),
(9, 'XP_038911233.1', NULL),
(9, 'XP_038911232.1', NULL),
(9, 'XP_038911231.1', NULL),
(9, 'XP_038911230.1', NULL),
(9, 'XP_038911229.1', NULL),
(9, 'XP_038911228.1', NULL),
(9, 'XP_038911227.1', NULL),
(9, 'XP_038911226.1', NULL),
(9, 'XP_038911225.1', NULL),
(9, 'XP_038911224.1', NULL),
(9, 'XP_038911223.1', NULL),
(9, 'XP_038911222.1', NULL),
(9, 'XP_038911221.1', NULL),
(9, 'XP_038911220.1', NULL),
(9, 'XP_038911219.1', NULL),
(9, 'XP_038911218.1', NULL),
(9, 'XP_038911217.1', NULL),
(9, 'XP_038911216.1', NULL),
(9, 'XP_038911215.1', NULL),
(9, 'XP_038911214.1', NULL),
(9, 'XP_038911213.1', NULL),
(9, 'XP_038911212.1', NULL),
(9, 'XP_038911211.1', NULL),
(9, 'XP_038911210.1', NULL),
(9, 'XP_038911209.1', NULL),
(9, 'XP_038911208.1', NULL),
(9, 'XP_038911207.1', NULL),
(9, 'XP_038911206.1', NULL),
(9, 'XP_038911205.1', NULL),
(9, 'XP_038911204.1', NULL),
(9, 'XP_038911203.1', NULL),
(9, 'XP_038911202.1', NULL),
(9, 'XP_038911201.1', NULL),
(9, 'XP_038911200.1', NULL),
(9, 'XP_038911199.1', NULL),
(9, 'XP_038911198.1', NULL),
(9, 'XP_038911197.1', NULL),
(9, 'XP_038911196.1', NULL),
(9, 'XP_038911195.1', NULL),
(9, 'XP_038911194.1', NULL),
(9, 'XP_038911193.1', NULL),
(9, 'XP_038911192.1', NULL),
(9, 'XP_038911191.1', NULL),
(9, 'XP_038911190.1', NULL),
(9, 'XP_038911189.1', NULL),
(9, 'XP_038911188.1', NULL),
(9, 'XP_038911187.1', NULL),
(9, 'XP_038911186.1', NULL),
(9, 'XP_038911185.1', NULL),
(9, 'XP_038911184.1', NULL),
(9, 'XP_038911183.1', NULL),
(9, 'XP_038911182.1', NULL),
(9, 'XP_038911181.1', NULL),
(9, 'XP_038911180.1', NULL),
(9, 'XP_038911179.1', NULL),
(9, 'XP_038911178.1', NULL),
(9, 'XP_038911177.1', NULL),
(9, 'XP_038911176.1', NULL),
(9, 'XP_038911175.1', NULL),
(9, 'XP_038911174.1', NULL),
(9, 'XP_038911173.1', NULL),
(9, 'XP_038911172.1', NULL),
(9, 'XP_038911171.1', NULL),
(9, 'XP_038911170.1', NULL),
(9, 'XP_038911169.1', NULL),
(9, 'XP_038911168.1', NULL),
(9, 'XP_038911167.1', NULL),
(9, 'XP_038911166.1', NULL),
(9, 'XP_038911165.1', NULL),
(9, 'XP_038911164.1', NULL),
(9, 'XP_038911163.1', NULL),
(9, 'XP_038911162.1', NULL),
(9, 'XP_038911161.1', NULL),
(9, 'XP_038911160.1', NULL),
(9, 'XP_038911159.1', NULL),
(9, 'XP_038911158.1', NULL),
(9, 'XP_038911157.1', NULL),
(9, 'XP_038911156.1', NULL),
(9, 'XP_038911155.1', NULL),
(9, 'XP_038911154.1', NULL),
(9, 'XP_038911153.1', NULL),
(9, 'XP_038911152.1', NULL),
(9, 'XP_038911151.1', NULL),
(9, 'XP_038911150.1', NULL),
(9, 'XP_038911149.1', NULL),
(9, 'XP_038911148.1', NULL),
(9, 'XP_038911147.1', NULL),
(9, 'XP_038911146.1', NULL),
(9, 'XP_038911145.1', NULL),
(9, 'XP_038911144.1', NULL),
(9, 'XP_038911143.1', NULL),
(9, 'XP_038911142.1', NULL),
(9, 'XP_038911141.1', NULL),
(9, 'XP_038911140.1', NULL),
(9, 'XP_038911139.1', NULL),
(9, 'XP_038911138.1', NULL),
(9, 'XP_038911137.1', NULL),
(9, 'XP_038911136.1', NULL),
(9, 'XP_038911135.1', NULL),
(9, 'XP_038911134.1', NULL),
(9, 'XP_038911303.1', NULL),
(9, 'XP_038911302.1', NULL),
(9, 'XP_038911301.1', NULL),
(9, 'XP_038911300.1', NULL),
(9, 'XP_038911299.1', NULL),
(9, 'XP_038911298.1', NULL),
(9, 'XP_038911297.1', NULL),
(9, 'XP_038911296.1', NULL),
(9, 'XP_038911295.1', NULL),
(9, 'XP_038911294.1', NULL),
(9, 'XP_038911293.1', NULL),
(9, 'XP_038911292.1', NULL),
(9, 'XP_038911291.1', NULL),
(9, 'XP_038911290.1', NULL),
(9, 'XP_038911289.1', NULL),
(9, 'XP_038911288.1', NULL),
(9, 'XP_038911287.1', NULL),
(9, 'XP_038911286.1', NULL),
(9, 'XP_038911285.1', NULL),
(9, 'XP_038911284.1', NULL),
(9, 'XP_038911283.1', NULL),
(9, 'XP_038911282.1', NULL),
(9, 'XP_038911281.1', NULL),
(9, 'XP_038911280.1', NULL),
(9, 'XP_038911279.1', NULL),
(9, 'XP_038911278.1', NULL),
(9, 'XP_038911277.1', NULL),
(9, 'XP_038911276.1', NULL),
(9, 'XP_038911275.1', NULL),
(9, 'XP_038911274.1', NULL),
(9, 'XP_038911273.1', NULL),
(9, 'XP_038911272.1', NULL),
(9, 'XP_038911271.1', NULL),
(9, 'XP_038911270.1', NULL),
(9, 'XP_038911269.1', NULL),
(9, 'XP_038911268.1', NULL),
(9, 'XP_038911267.1', NULL),
(9, 'XP_038911266.1', NULL),
(9, 'XP_038911265.1', NULL),
(9, 'XP_038911264.1', NULL),
(9, 'XP_038911263.1', NULL),
(9, 'XP_038911262.1', NULL),
(9, 'XP_038911261.1', NULL),
(9, 'XP_038911260.1', NULL),
(9, 'XP_038911259.1', NULL),
(9, 'XP_038911258.1', NULL),
(9, 'XP_038911257.1', NULL),
(9, 'XP_038911256.1', NULL),
(9, 'XP_038911255.1', NULL),
(9, 'XP_038911254.1', NULL),
(9, 'XP_038911253.1', NULL),
(9, 'XP_038911252.1', NULL),
(9, 'XP_038911251.1', NULL),
(9, 'XP_038911250.1', NULL),
(9, 'XP_038911249.1', NULL),
(9, 'XP_038911248.1', NULL),
(9, 'XP_038911247.1', NULL),
(9, 'XP_038911246.1', NULL),
(9, 'XP_038911245.1', NULL),
(9, 'XP_038911244.1', NULL),
(9, 'XP_038911243.1', NULL),
(9, 'XP_038911242.1', NULL),
(9, 'XP_038911241.1', NULL),
(9, 'XP_038911240.1', NULL),
(9, 'XP_038911239.1', NULL),
(9, 'XP_038911238.1', NULL),
(9, 'XP_038911237.1', NULL),
(9, 'XP_038911236.1', NULL),
(9, 'XP_038911235.1', NULL),
(9, 'XP_038911392.1', NULL),
(9, 'XP_038911391.1', NULL),
(9, 'XP_038911390.1', NULL),
(9, 'XP_038911389.1', NULL),
(9, 'XP_038911388.1', NULL),
(9, 'XP_038911387.1', NULL),
(9, 'XP_038911386.1', NULL),
(9, 'XP_038911385.1', NULL),
(9, 'XP_038911384.1', NULL),
(9, 'XP_038911383.1', NULL),
(9, 'XP_038911382.1', NULL),
(9, 'XP_038911381.1', NULL),
(9, 'XP_038911380.1', NULL),
(9, 'XP_038911379.1', NULL),
(9, 'XP_038911378.1', NULL),
(9, 'XP_038911377.1', NULL),
(9, 'XP_038911376.1', NULL),
(9, 'XP_038911375.1', NULL),
(9, 'XP_038911374.1', NULL),
(9, 'XP_038911373.1', NULL),
(9, 'XP_038911372.1', NULL),
(9, 'XP_038911371.1', NULL),
(9, 'XP_038911370.1', NULL),
(9, 'XP_038911369.1', NULL),
(9, 'XP_038911368.1', NULL),
(9, 'XP_038911367.1', NULL),
(9, 'XP_038911366.1', NULL),
(9, 'XP_038911365.1', NULL),
(9, 'XP_038911364.1', NULL),
(9, 'XP_038911363.1', NULL),
(9, 'XP_038911362.1', NULL),
(9, 'XP_038911361.1', NULL),
(9, 'XP_038911360.1', NULL),
(9, 'XP_038911359.1', NULL),
(9, 'XP_038911358.1', NULL),
(9, 'XP_038911357.1', NULL),
(9, 'XP_038911356.1', NULL),
(9, 'XP_038911355.1', NULL),
(9, 'XP_038911354.1', NULL),
(9, 'XP_038911353.1', NULL),
(9, 'XP_038911352.1', NULL),
(9, 'XP_038911351.1', NULL),
(9, 'XP_038911350.1', NULL),
(9, 'XP_038911349.1', NULL),
(9, 'XP_038911348.1', NULL),
(9, 'XP_038911347.1', NULL),
(9, 'XP_038911346.1', NULL),
(9, 'XP_038911345.1', NULL),
(9, 'XP_038911344.1', NULL),
(9, 'XP_038911343.1', NULL),
(9, 'XP_038911342.1', NULL),
(9, 'XP_038911341.1', NULL),
(9, 'XP_038911340.1', NULL),
(9, 'XP_038911339.1', NULL),
(9, 'XP_038911338.1', NULL),
(9, 'XP_038911337.1', NULL),
(9, 'XP_038911336.1', NULL),
(9, 'XP_038911335.1', NULL),
(9, 'XP_038911334.1', NULL),
(9, 'XP_038911333.1', NULL),
(9, 'XP_038911332.1', NULL),
(9, 'XP_038911331.1', NULL),
(9, 'XP_038911330.1', NULL),
(9, 'XP_038911329.1', NULL),
(9, 'XP_038911328.1', NULL),
(9, 'XP_038911327.1', NULL),
(9, 'XP_038911326.1', NULL),
(9, 'XP_038911325.1', NULL),
(9, 'XP_038911324.1', NULL),
(9, 'XP_038911323.1', NULL),
(9, 'XP_038911322.1', NULL),
(9, 'XP_038911321.1', NULL),
(9, 'XP_038911320.1', NULL),
(9, 'XP_038911319.1', NULL),
(9, 'XP_038911318.1', NULL),
(9, 'XP_038911317.1', NULL),
(9, 'XP_038911316.1', NULL),
(9, 'XP_038911315.1', NULL),
(9, 'XP_038911314.1', NULL),
(9, 'XP_038911313.1', NULL),
(9, 'XP_038911312.1', NULL),
(9, 'XP_038911311.1', NULL),
(9, 'XP_038911310.1', NULL),
(9, 'XP_038911309.1', NULL),
(9, 'XP_038911308.1', NULL),
(9, 'XP_038911307.1', NULL),
(9, 'XP_038911306.1', NULL),
(9, 'XP_038911305.1', NULL),
(9, 'XP_038911304.1', NULL),
(9, 'XP_038911524.1', NULL),
(9, 'XP_038911523.1', NULL),
(9, 'XP_038911522.1', NULL),
(9, 'XP_038911521.1', NULL),
(9, 'XP_038911520.1', NULL),
(9, 'XP_038911519.1', NULL),
(9, 'XP_038911518.1', NULL),
(9, 'XP_038911517.1', NULL),
(9, 'XP_038911516.1', NULL),
(9, 'XP_038911515.1', NULL),
(9, 'XP_038911514.1', NULL),
(9, 'XP_038911513.1', NULL),
(9, 'XP_038911512.1', NULL),
(9, 'XP_038911511.1', NULL),
(9, 'XP_038911510.1', NULL),
(9, 'XP_038911509.1', NULL),
(9, 'XP_038911508.1', NULL),
(9, 'XP_038911507.1', NULL),
(9, 'XP_038911506.1', NULL),
(9, 'XP_038911505.1', NULL),
(9, 'XP_038911504.1', NULL),
(9, 'XP_038911503.1', NULL),
(9, 'XP_038911502.1', NULL),
(9, 'XP_038911501.1', NULL),
(9, 'XP_038911500.1', NULL),
(9, 'XP_038911499.1', NULL),
(9, 'XP_038911498.1', NULL),
(9, 'XP_038911497.1', NULL),
(9, 'XP_038911496.1', NULL),
(9, 'XP_038911495.1', NULL),
(9, 'XP_038911494.1', NULL),
(9, 'XP_038911493.1', NULL),
(9, 'XP_038911492.1', NULL),
(9, 'XP_038911491.1', NULL),
(9, 'XP_038911490.1', NULL),
(9, 'XP_038911489.1', NULL),
(9, 'XP_038911488.1', NULL),
(9, 'XP_038911487.1', NULL),
(9, 'XP_038911486.1', NULL),
(9, 'XP_038911485.1', NULL),
(9, 'XP_038911484.1', NULL),
(9, 'XP_038911483.1', NULL),
(9, 'XP_038911482.1', NULL),
(9, 'XP_038911481.1', NULL),
(9, 'XP_038911480.1', NULL),
(9, 'XP_038911479.1', NULL),
(9, 'XP_038911478.1', NULL),
(9, 'XP_038911477.1', NULL),
(9, 'XP_038911476.1', NULL),
(9, 'XP_038911475.1', NULL),
(9, 'XP_038911474.1', NULL),
(9, 'XP_038911473.1', NULL),
(9, 'XP_038911472.1', NULL),
(9, 'XP_038911471.1', NULL),
(9, 'XP_038911470.1', NULL),
(9, 'XP_038911469.1', NULL),
(9, 'XP_038911468.1', NULL),
(9, 'XP_038911467.1', NULL),
(9, 'XP_038911466.1', NULL),
(9, 'XP_038911465.1', NULL),
(9, 'XP_038911464.1', NULL),
(9, 'XP_038911463.1', NULL),
(9, 'XP_038911462.1', NULL),
(9, 'XP_038911461.1', NULL),
(9, 'XP_038911460.1', NULL),
(9, 'XP_038911459.1', NULL),
(9, 'XP_038911458.1', NULL),
(9, 'XP_038911457.1', NULL),
(9, 'XP_038911456.1', NULL),
(9, 'XP_038911455.1', NULL),
(9, 'XP_038911454.1', NULL),
(9, 'XP_038911453.1', NULL),
(9, 'XP_038911452.1', NULL),
(9, 'XP_038911451.1', NULL),
(9, 'XP_038911450.1', NULL),
(9, 'XP_038911449.1', NULL),
(9, 'XP_038911448.1', NULL),
(9, 'XP_038911447.1', NULL),
(9, 'XP_038911446.1', NULL),
(9, 'XP_038911445.1', NULL),
(9, 'XP_038911444.1', NULL),
(9, 'XP_038911443.1', NULL),
(9, 'XP_038911442.1', NULL),
(9, 'XP_038911441.1', NULL),
(9, 'XP_038911440.1', NULL),
(9, 'XP_038911439.1', NULL),
(9, 'XP_038911438.1', NULL),
(9, 'XP_038911437.1', NULL),
(9, 'XP_038911436.1', NULL),
(9, 'XP_038911435.1', NULL),
(9, 'XP_038911434.1', NULL),
(9, 'XP_038911433.1', NULL),
(9, 'XP_038911432.1', NULL),
(9, 'XP_038911431.1', NULL),
(9, 'XP_038911430.1', NULL),
(9, 'XP_038911429.1', NULL),
(9, 'XP_038911428.1', NULL),
(9, 'XP_038911427.1', NULL),
(9, 'XP_038911426.1', NULL),
(9, 'XP_038911425.1', NULL),
(9, 'XP_038911424.1', NULL),
(9, 'XP_038911423.1', NULL),
(9, 'XP_038911422.1', NULL),
(9, 'XP_038911421.1', NULL),
(9, 'XP_038911420.1', NULL),
(9, 'XP_038911419.1', NULL),
(9, 'XP_038911418.1', NULL),
(9, 'XP_038911417.1', NULL),
(9, 'XP_038911416.1', NULL),
(9, 'XP_038911415.1', NULL),
(9, 'XP_038911414.1', NULL),
(9, 'XP_038911413.1', NULL),
(9, 'XP_038911412.1', NULL),
(9, 'XP_038911411.1', NULL),
(9, 'XP_038911410.1', NULL),
(9, 'XP_038911409.1', NULL),
(9, 'XP_038911408.1', NULL),
(9, 'XP_038911407.1', NULL),
(9, 'XP_038911406.1', NULL),
(9, 'XP_038911405.1', NULL),
(9, 'XP_038911404.1', NULL),
(9, 'XP_038911403.1', NULL),
(9, 'XP_038911402.1', NULL),
(9, 'XP_038911401.1', NULL),
(9, 'XP_038911400.1', NULL),
(9, 'XP_038911399.1', NULL),
(9, 'XP_038911398.1', NULL),
(9, 'XP_038911397.1', NULL),
(9, 'XP_038911396.1', NULL),
(9, 'XP_038911395.1', NULL),
(9, 'XP_038911394.1', NULL),
(9, 'XP_038911393.1', NULL),
(9, 'XP_038911608.1', NULL),
(9, 'XP_038911607.1', NULL),
(9, 'XP_038911606.1', NULL),
(9, 'XP_038911605.1', NULL),
(9, 'XP_038911604.1', NULL),
(9, 'XP_038911603.1', NULL),
(9, 'XP_038911602.1', NULL),
(9, 'XP_038911601.1', NULL),
(9, 'XP_038911600.1', NULL),
(9, 'XP_038911599.1', NULL),
(9, 'XP_038911598.1', NULL),
(9, 'XP_038911597.1', NULL),
(9, 'XP_038911596.1', NULL),
(9, 'XP_038911595.1', NULL),
(9, 'XP_038911594.1', NULL),
(9, 'XP_038911593.1', NULL),
(9, 'XP_038911592.1', NULL),
(9, 'XP_038911591.1', NULL),
(9, 'XP_038911590.1', NULL),
(9, 'XP_038911589.1', NULL),
(9, 'XP_038911588.1', NULL),
(9, 'XP_038911587.1', NULL),
(9, 'XP_038911586.1', NULL),
(9, 'XP_038911585.1', NULL),
(9, 'XP_038911584.1', NULL),
(9, 'XP_038911583.1', NULL),
(9, 'XP_038911582.1', NULL),
(9, 'XP_038911581.1', NULL),
(9, 'XP_038911580.1', NULL),
(9, 'XP_038911579.1', NULL),
(9, 'XP_038911578.1', NULL),
(9, 'XP_038911577.1', NULL),
(9, 'XP_038911576.1', NULL),
(9, 'XP_038911575.1', NULL),
(9, 'XP_038911574.1', NULL),
(9, 'XP_038911573.1', NULL),
(9, 'XP_038911572.1', NULL),
(9, 'XP_038911571.1', NULL),
(9, 'XP_038911570.1', NULL),
(9, 'XP_038911569.1', NULL),
(9, 'XP_038911568.1', NULL),
(9, 'XP_038911567.1', NULL),
(9, 'XP_038911566.1', NULL),
(9, 'XP_038911565.1', NULL),
(9, 'XP_038911564.1', NULL),
(9, 'XP_038911563.1', NULL),
(9, 'XP_038911562.1', NULL),
(9, 'XP_038911561.1', NULL),
(9, 'XP_038911560.1', NULL),
(9, 'XP_038911559.1', NULL),
(9, 'XP_038911558.1', NULL),
(9, 'XP_038911557.1', NULL),
(9, 'XP_038911556.1', NULL),
(9, 'XP_038911555.1', NULL),
(9, 'XP_038911554.1', NULL),
(9, 'XP_038911553.1', NULL),
(9, 'XP_038911552.1', NULL),
(9, 'XP_038911551.1', NULL),
(9, 'XP_038911550.1', NULL),
(9, 'XP_038911549.1', NULL),
(9, 'XP_038911548.1', NULL),
(9, 'XP_038911547.1', NULL),
(9, 'XP_038911546.1', NULL),
(9, 'XP_038911545.1', NULL),
(9, 'XP_038911544.1', NULL),
(9, 'XP_038911543.1', NULL),
(9, 'XP_038911542.1', NULL),
(9, 'XP_038911541.1', NULL),
(9, 'XP_038911540.1', NULL),
(9, 'XP_038911539.1', NULL),
(9, 'XP_038911538.1', NULL),
(9, 'XP_038911537.1', NULL),
(9, 'XP_038911536.1', NULL),
(9, 'XP_038911535.1', NULL),
(9, 'XP_038911534.1', NULL),
(9, 'XP_038911533.1', NULL),
(9, 'XP_038911532.1', NULL),
(9, 'XP_038911531.1', NULL),
(9, 'XP_038911530.1', NULL),
(9, 'XP_038911529.1', NULL),
(9, 'XP_038911528.1', NULL),
(9, 'XP_038911527.1', NULL),
(9, 'XP_038911526.1', NULL),
(9, 'XP_038911525.1', NULL),
(9, 'XP_038911706.1', NULL),
(9, 'XP_038911705.1', NULL),
(9, 'XP_038911704.1', NULL),
(9, 'XP_038911703.1', NULL),
(9, 'XP_038911702.1', NULL),
(9, 'XP_038911701.1', NULL),
(9, 'XP_038911700.1', NULL),
(9, 'XP_038911699.1', NULL),
(9, 'XP_038911698.1', NULL),
(9, 'XP_038911697.1', NULL),
(9, 'XP_038911696.1', NULL),
(9, 'XP_038911695.1', NULL),
(9, 'XP_038911694.1', NULL),
(9, 'XP_038911693.1', NULL),
(9, 'XP_038911692.1', NULL),
(9, 'XP_038911691.1', NULL),
(9, 'XP_038911690.1', NULL),
(9, 'XP_038911689.1', NULL),
(9, 'XP_038911688.1', NULL),
(9, 'XP_038911687.1', NULL),
(9, 'XP_038911686.1', NULL),
(9, 'XP_038911685.1', NULL),
(9, 'XP_038911684.1', NULL),
(9, 'XP_038911683.1', NULL),
(9, 'XP_038911682.1', NULL),
(9, 'XP_038911681.1', NULL),
(9, 'XP_038911680.1', NULL),
(9, 'XP_038911679.1', NULL),
(9, 'XP_038911678.1', NULL),
(9, 'XP_038911677.1', NULL),
(9, 'XP_038911676.1', NULL),
(9, 'XP_038911675.1', NULL),
(9, 'XP_038911674.1', NULL),
(9, 'XP_038911673.1', NULL),
(9, 'XP_038911672.1', NULL),
(9, 'XP_038911671.1', NULL),
(9, 'XP_038911670.1', NULL),
(9, 'XP_038911669.1', NULL),
(9, 'XP_038911668.1', NULL),
(9, 'XP_038911667.1', NULL),
(9, 'XP_038911666.1', NULL),
(9, 'XP_038911665.1', NULL),
(9, 'XP_038911664.1', NULL),
(9, 'XP_038911663.1', NULL),
(9, 'XP_038911662.1', NULL),
(9, 'XP_038911661.1', NULL),
(9, 'XP_038911660.1', NULL),
(9, 'XP_038911659.1', NULL),
(9, 'XP_038911658.1', NULL),
(9, 'XP_038911657.1', NULL),
(9, 'XP_038911656.1', NULL),
(9, 'XP_038911655.1', NULL),
(9, 'XP_038911654.1', NULL),
(9, 'XP_038911653.1', NULL),
(9, 'XP_038911652.1', NULL),
(9, 'XP_038911651.1', NULL),
(9, 'XP_038911650.1', NULL),
(9, 'XP_038911649.1', NULL),
(9, 'XP_038911648.1', NULL),
(9, 'XP_038911647.1', NULL),
(9, 'XP_038911646.1', NULL),
(9, 'XP_038911645.1', NULL),
(9, 'XP_038911644.1', NULL),
(9, 'XP_038911643.1', NULL),
(9, 'XP_038911642.1', NULL),
(9, 'XP_038911641.1', NULL),
(9, 'XP_038911640.1', NULL),
(9, 'XP_038911639.1', NULL),
(9, 'XP_038911638.1', NULL),
(9, 'XP_038911637.1', NULL),
(9, 'XP_038911636.1', NULL),
(9, 'XP_038911635.1', NULL),
(9, 'XP_038911634.1', NULL),
(9, 'XP_038911633.1', NULL),
(9, 'XP_038911632.1', NULL),
(9, 'XP_038911631.1', NULL),
(9, 'XP_038911630.1', NULL),
(9, 'XP_038911629.1', NULL),
(9, 'XP_038911628.1', NULL),
(9, 'XP_038911627.1', NULL),
(9, 'XP_038911626.1', NULL),
(9, 'XP_038911625.1', NULL),
(9, 'XP_038911624.1', NULL),
(9, 'XP_038911623.1', NULL),
(9, 'XP_038911622.1', NULL),
(9, 'XP_038911621.1', NULL),
(9, 'XP_038911620.1', NULL),
(9, 'XP_038911619.1', NULL),
(9, 'XP_038911618.1', NULL),
(9, 'XP_038911617.1', NULL),
(9, 'XP_038911616.1', NULL),
(9, 'XP_038911615.1', NULL),
(9, 'XP_038911614.1', NULL),
(9, 'XP_038911613.1', NULL),
(9, 'XP_038911612.1', NULL),
(9, 'XP_038911611.1', NULL),
(9, 'XP_038911610.1', NULL),
(9, 'XP_038911609.1', NULL),
(9, 'XP_038911781.1', NULL),
(9, 'XP_038911780.1', NULL),
(9, 'XP_038911779.1', NULL),
(9, 'XP_038911778.1', NULL),
(9, 'XP_038911777.1', NULL),
(9, 'XP_038911776.1', NULL),
(9, 'XP_038911775.1', NULL),
(9, 'XP_038911774.1', NULL),
(9, 'XP_038911773.1', NULL),
(9, 'XP_038911772.1', NULL),
(9, 'XP_038911771.1', NULL),
(9, 'XP_038911770.1', NULL),
(9, 'XP_038911769.1', NULL),
(9, 'XP_038911768.1', NULL),
(9, 'XP_038911767.1', NULL),
(9, 'XP_038911766.1', NULL),
(9, 'XP_038911765.1', NULL),
(9, 'XP_038911764.1', NULL),
(9, 'XP_038911763.1', NULL),
(9, 'XP_038911762.1', NULL),
(9, 'XP_038911761.1', NULL),
(9, 'XP_038911760.1', NULL),
(9, 'XP_038911759.1', NULL),
(9, 'XP_038911758.1', NULL),
(9, 'XP_038911757.1', NULL),
(9, 'XP_038911756.1', NULL),
(9, 'XP_038911755.1', NULL),
(9, 'XP_038911754.1', NULL),
(9, 'XP_038911753.1', NULL),
(9, 'XP_038911752.1', NULL),
(9, 'XP_038911751.1', NULL),
(9, 'XP_038911750.1', NULL),
(9, 'XP_038911749.1', NULL),
(9, 'XP_038911748.1', NULL),
(9, 'XP_038911747.1', NULL),
(9, 'XP_038911746.1', NULL),
(9, 'XP_038911745.1', NULL),
(9, 'XP_038911744.1', NULL),
(9, 'XP_038911743.1', NULL),
(9, 'XP_038911742.1', NULL),
(9, 'XP_038911741.1', NULL),
(9, 'XP_038911740.1', NULL),
(9, 'XP_038911739.1', NULL),
(9, 'XP_038911738.1', NULL),
(9, 'XP_038911737.1', NULL),
(9, 'XP_038911736.1', NULL),
(9, 'XP_038911735.1', NULL),
(9, 'XP_038911734.1', NULL),
(9, 'XP_038911733.1', NULL),
(9, 'XP_038911732.1', NULL),
(9, 'XP_038911731.1', NULL),
(9, 'XP_038911730.1', NULL),
(9, 'XP_038911729.1', NULL),
(9, 'XP_038911728.1', NULL),
(9, 'XP_038911727.1', NULL),
(9, 'XP_038911726.1', NULL),
(9, 'XP_038911725.1', NULL),
(9, 'XP_038911724.1', NULL),
(9, 'XP_038911723.1', NULL),
(9, 'XP_038911722.1', NULL),
(9, 'XP_038911721.1', NULL),
(9, 'XP_038911720.1', NULL),
(9, 'XP_038911719.1', NULL),
(9, 'XP_038911718.1', NULL),
(9, 'XP_038911717.1', NULL),
(9, 'XP_038911716.1', NULL),
(9, 'XP_038911715.1', NULL),
(9, 'XP_038911714.1', NULL),
(9, 'XP_038911713.1', NULL),
(9, 'XP_038911712.1', NULL),
(9, 'XP_038911711.1', NULL),
(9, 'XP_038911710.1', NULL),
(9, 'XP_038911709.1', NULL),
(9, 'XP_038911708.1', NULL),
(9, 'XP_038911707.1', NULL),
(9, 'XP_038911833.1', NULL),
(9, 'XP_038911832.1', NULL),
(9, 'XP_038911831.1', NULL),
(9, 'XP_038911830.1', NULL),
(9, 'XP_038911829.1', NULL),
(9, 'XP_038911828.1', NULL),
(9, 'XP_038911827.1', NULL),
(9, 'XP_038911826.1', NULL),
(9, 'XP_038911825.1', NULL),
(9, 'XP_038911824.1', NULL),
(9, 'XP_038911823.1', NULL),
(9, 'XP_038911822.1', NULL),
(9, 'XP_038911821.1', NULL),
(9, 'XP_038911820.1', NULL),
(9, 'XP_038911819.1', NULL),
(9, 'XP_038911818.1', NULL),
(9, 'XP_038911817.1', NULL),
(9, 'XP_038911816.1', NULL),
(9, 'XP_038911815.1', NULL),
(9, 'XP_038911814.1', NULL),
(9, 'XP_038911813.1', NULL),
(9, 'XP_038911812.1', NULL),
(9, 'XP_038911811.1', NULL),
(9, 'XP_038911810.1', NULL),
(9, 'XP_038911809.1', NULL),
(9, 'XP_038911808.1', NULL),
(9, 'XP_038911807.1', NULL),
(9, 'XP_038911806.1', NULL),
(9, 'XP_038911805.1', NULL),
(9, 'XP_038911804.1', NULL),
(9, 'XP_038911803.1', NULL),
(9, 'XP_038911802.1', NULL),
(9, 'XP_038911801.1', NULL),
(9, 'XP_038911800.1', NULL),
(9, 'XP_038911799.1', NULL),
(9, 'XP_038911798.1', NULL),
(9, 'XP_038911797.1', NULL),
(9, 'XP_038911796.1', NULL),
(9, 'XP_038911795.1', NULL),
(9, 'XP_038911794.1', NULL),
(9, 'XP_038911793.1', NULL),
(9, 'XP_038911792.1', NULL),
(9, 'XP_038911791.1', NULL),
(9, 'XP_038911790.1', NULL),
(9, 'XP_038911789.1', NULL),
(9, 'XP_038911788.1', NULL),
(9, 'XP_038911787.1', NULL),
(9, 'XP_038911786.1', NULL),
(9, 'XP_038911785.1', NULL),
(9, 'XP_038911784.1', NULL),
(9, 'XP_038911783.1', NULL),
(9, 'XP_038911782.1', NULL),
(9, 'XP_038911927.1', NULL),
(9, 'XP_038911926.1', NULL),
(9, 'XP_038911925.1', NULL),
(9, 'XP_038911924.1', NULL),
(9, 'XP_038911923.1', NULL),
(9, 'XP_038911922.1', NULL),
(9, 'XP_038911921.1', NULL),
(9, 'XP_038911920.1', NULL),
(9, 'XP_038911919.1', NULL),
(9, 'XP_038911918.1', NULL),
(9, 'XP_038911917.1', NULL),
(9, 'XP_038911916.1', NULL),
(9, 'XP_038911915.1', NULL),
(9, 'XP_038911914.1', NULL),
(9, 'XP_038911913.1', NULL),
(9, 'XP_038911912.1', NULL),
(9, 'XP_038911911.1', NULL),
(9, 'XP_038911910.1', NULL),
(9, 'XP_038911909.1', NULL),
(9, 'XP_038911908.1', NULL),
(9, 'XP_038911907.1', NULL),
(9, 'XP_038911906.1', NULL),
(9, 'XP_038911905.1', NULL),
(9, 'XP_038911904.1', NULL),
(9, 'XP_038911903.1', NULL),
(9, 'XP_038911902.1', NULL),
(9, 'XP_038911901.1', NULL),
(9, 'XP_038911900.1', NULL),
(9, 'XP_038911899.1', NULL),
(9, 'XP_038911898.1', NULL),
(9, 'XP_038911897.1', NULL),
(9, 'XP_038911896.1', NULL),
(9, 'XP_038911895.1', NULL),
(9, 'XP_038911894.1', NULL),
(9, 'XP_038911893.1', NULL),
(9, 'XP_038911892.1', NULL),
(9, 'XP_038911891.1', NULL),
(9, 'XP_038911890.1', NULL),
(9, 'XP_038911889.1', NULL),
(9, 'XP_038911888.1', NULL),
(9, 'XP_038911887.1', NULL),
(9, 'XP_038911886.1', NULL),
(9, 'XP_038911885.1', NULL),
(9, 'XP_038911884.1', NULL),
(9, 'XP_038911883.1', NULL),
(9, 'XP_038911882.1', NULL),
(9, 'XP_038911881.1', NULL),
(9, 'XP_038911880.1', NULL),
(9, 'XP_038911879.1', NULL),
(9, 'XP_038911878.1', NULL),
(9, 'XP_038911877.1', NULL),
(9, 'XP_038911876.1', NULL),
(9, 'XP_038911875.1', NULL),
(9, 'XP_038911874.1', NULL),
(9, 'XP_038911873.1', NULL),
(9, 'XP_038911872.1', NULL),
(9, 'XP_038911871.1', NULL),
(9, 'XP_038911870.1', NULL),
(9, 'XP_038911869.1', NULL),
(9, 'XP_038911868.1', NULL),
(9, 'XP_038911867.1', NULL),
(9, 'XP_038911866.1', NULL),
(9, 'XP_038911865.1', NULL),
(9, 'XP_038911864.1', NULL),
(9, 'XP_038911863.1', NULL),
(9, 'XP_038911862.1', NULL),
(9, 'XP_038911861.1', NULL),
(9, 'XP_038911860.1', NULL),
(9, 'XP_038911859.1', NULL),
(9, 'XP_038911858.1', NULL),
(9, 'XP_038911857.1', NULL),
(9, 'XP_038911856.1', NULL),
(9, 'XP_038911855.1', NULL),
(9, 'XP_038911854.1', NULL),
(9, 'XP_038911853.1', NULL),
(9, 'XP_038911852.1', NULL),
(9, 'XP_038911851.1', NULL),
(9, 'XP_038911850.1', NULL),
(9, 'XP_038911849.1', NULL),
(9, 'XP_038911848.1', NULL),
(9, 'XP_038911847.1', NULL),
(9, 'XP_038911846.1', NULL),
(9, 'XP_038911845.1', NULL),
(9, 'XP_038911844.1', NULL),
(9, 'XP_038911843.1', NULL),
(9, 'XP_038911842.1', NULL),
(9, 'XP_038911841.1', NULL),
(9, 'XP_038911840.1', NULL),
(9, 'XP_038911839.1', NULL),
(9, 'XP_038911838.1', NULL),
(9, 'XP_038911837.1', NULL),
(9, 'XP_038911836.1', NULL),
(9, 'XP_038911835.1', NULL),
(9, 'XP_038911834.1', NULL),
(9, 'XP_038911985.1', NULL),
(9, 'XP_038911984.1', NULL),
(9, 'XP_038911983.1', NULL),
(9, 'XP_038911982.1', NULL),
(9, 'XP_038911981.1', NULL),
(9, 'XP_038911980.1', NULL),
(9, 'XP_038911979.1', NULL),
(9, 'XP_038911978.1', NULL),
(9, 'XP_038911977.1', NULL),
(9, 'XP_038911976.1', NULL),
(9, 'XP_038911975.1', NULL),
(9, 'XP_038911974.1', NULL),
(9, 'XP_038911973.1', NULL),
(9, 'XP_038911972.1', NULL),
(9, 'XP_038911971.1', NULL),
(9, 'XP_038911970.1', NULL),
(9, 'XP_038911969.1', NULL),
(9, 'XP_038911968.1', NULL),
(9, 'XP_038911967.1', NULL),
(9, 'XP_038911966.1', NULL),
(9, 'XP_038911965.1', NULL),
(9, 'XP_038911964.1', NULL),
(9, 'XP_038911963.1', NULL),
(9, 'XP_038911962.1', NULL),
(9, 'XP_038911961.1', NULL),
(9, 'XP_038911960.1', NULL),
(9, 'XP_038911959.1', NULL),
(9, 'XP_038911958.1', NULL),
(9, 'XP_038911957.1', NULL),
(9, 'XP_038911956.1', NULL),
(9, 'XP_038911955.1', NULL),
(9, 'XP_038911954.1', NULL),
(9, 'XP_038911953.1', NULL),
(9, 'XP_038911952.1', NULL),
(9, 'XP_038911951.1', NULL),
(9, 'XP_038911950.1', NULL),
(9, 'XP_038911949.1', NULL),
(9, 'XP_038911948.1', NULL),
(9, 'XP_038911947.1', NULL),
(9, 'XP_038911946.1', NULL),
(9, 'XP_038911945.1', NULL),
(9, 'XP_038911944.1', NULL),
(9, 'XP_038911943.1', NULL),
(9, 'XP_038911942.1', NULL),
(9, 'XP_038911941.1', NULL),
(9, 'XP_038911940.1', NULL),
(9, 'XP_038911939.1', NULL),
(9, 'XP_038911938.1', NULL),
(9, 'XP_038911937.1', NULL),
(9, 'XP_038911936.1', NULL),
(9, 'XP_038911935.1', NULL),
(9, 'XP_038911934.1', NULL),
(9, 'XP_038911933.1', NULL),
(9, 'XP_038911932.1', NULL),
(9, 'XP_038911931.1', NULL),
(9, 'XP_038911930.1', NULL),
(9, 'XP_038911929.1', NULL),
(9, 'XP_038911928.1', NULL),
(9, 'XP_038912048.1', NULL),
(9, 'XP_038912047.1', NULL),
(9, 'XP_038912046.1', NULL),
(9, 'XP_038912045.1', NULL),
(9, 'XP_038912044.1', NULL),
(9, 'XP_038912043.1', NULL),
(9, 'XP_038912042.1', NULL),
(9, 'XP_038912041.1', NULL),
(9, 'XP_038912040.1', NULL),
(9, 'XP_038912039.1', NULL),
(9, 'XP_038912038.1', NULL),
(9, 'XP_038912037.1', NULL),
(9, 'XP_038912036.1', NULL),
(9, 'XP_038912035.1', NULL),
(9, 'XP_038912034.1', NULL),
(9, 'XP_038912033.1', NULL),
(9, 'XP_038912032.1', NULL),
(9, 'XP_038912031.1', NULL),
(9, 'XP_038912030.1', NULL),
(9, 'XP_038912029.1', NULL),
(9, 'XP_038912028.1', NULL),
(9, 'XP_038912027.1', NULL),
(9, 'XP_038912026.1', NULL),
(9, 'XP_038912025.1', NULL),
(9, 'XP_038912024.1', NULL),
(9, 'XP_038912023.1', NULL),
(9, 'XP_038912022.1', NULL),
(9, 'XP_038912021.1', NULL),
(9, 'XP_038912020.1', NULL),
(9, 'XP_038912019.1', NULL),
(9, 'XP_038912018.1', NULL),
(9, 'XP_038912017.1', NULL),
(9, 'XP_038912016.1', NULL),
(9, 'XP_038912015.1', NULL),
(9, 'XP_038912014.1', NULL),
(9, 'XP_038912013.1', NULL),
(9, 'XP_038912012.1', NULL),
(9, 'XP_038912011.1', NULL),
(9, 'XP_038912010.1', NULL),
(9, 'XP_038912009.1', NULL),
(9, 'XP_038912008.1', NULL),
(9, 'XP_038912007.1', NULL),
(9, 'XP_038912006.1', NULL),
(9, 'XP_038912005.1', NULL),
(9, 'XP_038912004.1', NULL),
(9, 'XP_038912003.1', NULL),
(9, 'XP_038912002.1', NULL),
(9, 'XP_038912001.1', NULL),
(9, 'XP_038912000.1', NULL),
(9, 'XP_038911999.1', NULL),
(9, 'XP_038911998.1', NULL),
(9, 'XP_038911997.1', NULL),
(9, 'XP_038911996.1', NULL),
(9, 'XP_038911995.1', NULL),
(9, 'XP_038911994.1', NULL),
(9, 'XP_038911993.1', NULL),
(9, 'XP_038911992.1', NULL),
(9, 'XP_038911991.1', NULL),
(9, 'XP_038911990.1', NULL),
(9, 'XP_038911989.1', NULL),
(9, 'XP_038911988.1', NULL),
(9, 'XP_038911987.1', NULL),
(9, 'XP_038911986.1', NULL),
(9, 'XP_038912197.1', NULL),
(9, 'XP_038912196.1', NULL),
(9, 'XP_038912195.1', NULL),
(9, 'XP_038912194.1', NULL),
(9, 'XP_038912193.1', NULL),
(9, 'XP_038912192.1', NULL),
(9, 'XP_038912191.1', NULL),
(9, 'XP_038912190.1', NULL),
(9, 'XP_038912189.1', NULL),
(9, 'XP_038912188.1', NULL),
(9, 'XP_038912187.1', NULL),
(9, 'XP_038912186.1', NULL),
(9, 'XP_038912185.1', NULL),
(9, 'XP_038912184.1', NULL),
(9, 'XP_038912183.1', NULL),
(9, 'XP_038912182.1', NULL),
(9, 'XP_038912181.1', NULL),
(9, 'XP_038912180.1', NULL),
(9, 'XP_038912179.1', NULL),
(9, 'XP_038912178.1', NULL),
(9, 'XP_038912177.1', NULL),
(9, 'XP_038912176.1', NULL),
(9, 'XP_038912175.1', NULL),
(9, 'XP_038912174.1', NULL),
(9, 'XP_038912173.1', NULL),
(9, 'XP_038912172.1', NULL),
(9, 'XP_038912171.1', NULL),
(9, 'XP_038912170.1', NULL),
(9, 'XP_038912169.1', NULL),
(9, 'XP_038912168.1', NULL),
(9, 'XP_038912167.1', NULL),
(9, 'XP_038912166.1', NULL),
(9, 'XP_038912165.1', NULL),
(9, 'XP_038912164.1', NULL),
(9, 'XP_038912163.1', NULL),
(9, 'XP_038912162.1', NULL),
(9, 'XP_038912161.1', NULL),
(9, 'XP_038912160.1', NULL),
(9, 'XP_038912159.1', NULL),
(9, 'XP_038912158.1', NULL),
(9, 'XP_038912157.1', NULL),
(9, 'XP_038912156.1', NULL),
(9, 'XP_038912155.1', NULL),
(9, 'XP_038912154.1', NULL),
(9, 'XP_038912153.1', NULL),
(9, 'XP_038912152.1', NULL),
(9, 'XP_038912151.1', NULL),
(9, 'XP_038912150.1', NULL),
(9, 'XP_038912149.1', NULL),
(9, 'XP_038912148.1', NULL),
(9, 'XP_038912147.1', NULL),
(9, 'XP_038912146.1', NULL),
(9, 'XP_038912145.1', NULL),
(9, 'XP_038912144.1', NULL),
(9, 'XP_038912143.1', NULL),
(9, 'XP_038912142.1', NULL),
(9, 'XP_038912141.1', NULL),
(9, 'XP_038912140.1', NULL),
(9, 'XP_038912139.1', NULL),
(9, 'XP_038912138.1', NULL),
(9, 'XP_038912137.1', NULL),
(9, 'XP_038912136.1', NULL),
(9, 'XP_038912135.1', NULL),
(9, 'XP_038912134.1', NULL),
(9, 'XP_038912133.1', NULL),
(9, 'XP_038912132.1', NULL),
(9, 'XP_038912131.1', NULL),
(9, 'XP_038912130.1', NULL),
(9, 'XP_038912129.1', NULL),
(9, 'XP_038912128.1', NULL),
(9, 'XP_038912127.1', NULL),
(9, 'XP_038912126.1', NULL),
(9, 'XP_038912125.1', NULL),
(9, 'XP_038912124.1', NULL),
(9, 'XP_038912123.1', NULL),
(9, 'XP_038912122.1', NULL),
(9, 'XP_038912121.1', NULL),
(9, 'XP_038912120.1', NULL),
(9, 'XP_038912119.1', NULL),
(9, 'XP_038912118.1', NULL),
(9, 'XP_038912117.1', NULL),
(9, 'XP_038912116.1', NULL),
(9, 'XP_038912115.1', NULL),
(9, 'XP_038912114.1', NULL),
(9, 'XP_038912113.1', NULL),
(9, 'XP_038912112.1', NULL),
(9, 'XP_038912111.1', NULL),
(9, 'XP_038912110.1', NULL),
(9, 'XP_038912109.1', NULL),
(9, 'XP_038912108.1', NULL),
(9, 'XP_038912107.1', NULL),
(9, 'XP_038912106.1', NULL),
(9, 'XP_038912105.1', NULL),
(9, 'XP_038912104.1', NULL),
(9, 'XP_038912103.1', NULL),
(9, 'XP_038912102.1', NULL),
(9, 'XP_038912101.1', NULL),
(9, 'XP_038912100.1', NULL),
(9, 'XP_038912099.1', NULL),
(9, 'XP_038912098.1', NULL),
(9, 'XP_038912097.1', NULL),
(9, 'XP_038912096.1', NULL),
(9, 'XP_038912095.1', NULL),
(9, 'XP_038912094.1', NULL),
(9, 'XP_038912093.1', NULL),
(9, 'XP_038912092.1', NULL),
(9, 'XP_038912091.1', NULL),
(9, 'XP_038912090.1', NULL),
(9, 'XP_038912089.1', NULL),
(9, 'XP_038912088.1', NULL),
(9, 'XP_038912087.1', NULL),
(9, 'XP_038912086.1', NULL),
(9, 'XP_038912085.1', NULL),
(9, 'XP_038912084.1', NULL),
(9, 'XP_038912083.1', NULL),
(9, 'XP_038912082.1', NULL),
(9, 'XP_038912081.1', NULL),
(9, 'XP_038912080.1', NULL),
(9, 'XP_038912079.1', NULL),
(9, 'XP_038912078.1', NULL),
(9, 'XP_038912077.1', NULL),
(9, 'XP_038912076.1', NULL),
(9, 'XP_038912075.1', NULL),
(9, 'XP_038912074.1', NULL),
(9, 'XP_038912073.1', NULL),
(9, 'XP_038912072.1', NULL),
(9, 'XP_038912071.1', NULL),
(9, 'XP_038912070.1', NULL),
(9, 'XP_038912069.1', NULL),
(9, 'XP_038912068.1', NULL),
(9, 'XP_038912067.1', NULL),
(9, 'XP_038912066.1', NULL),
(9, 'XP_038912065.1', NULL),
(9, 'XP_038912064.1', NULL),
(9, 'XP_038912063.1', NULL),
(9, 'XP_038912062.1', NULL),
(9, 'XP_038912061.1', NULL),
(9, 'XP_038912060.1', NULL),
(9, 'XP_038912059.1', NULL),
(9, 'XP_038912058.1', NULL),
(9, 'XP_038912057.1', NULL),
(9, 'XP_038912056.1', NULL),
(9, 'XP_038912055.1', NULL),
(9, 'XP_038912054.1', NULL),
(9, 'XP_038912053.1', NULL),
(9, 'XP_038912052.1', NULL),
(9, 'XP_038912051.1', NULL),
(9, 'XP_038912050.1', NULL),
(9, 'XP_038912049.1', NULL),
(9, 'XP_038912233.1', NULL),
(9, 'XP_038912232.1', NULL),
(9, 'XP_038912231.1', NULL),
(9, 'XP_038912230.1', NULL),
(9, 'XP_038912229.1', NULL),
(9, 'XP_038912228.1', NULL),
(9, 'XP_038912227.1', NULL),
(9, 'XP_038912226.1', NULL),
(9, 'XP_038912225.1', NULL),
(9, 'XP_038912224.1', NULL),
(9, 'XP_038912223.1', NULL),
(9, 'XP_038912222.1', NULL),
(9, 'XP_038912221.1', NULL),
(9, 'XP_038912220.1', NULL),
(9, 'XP_038912219.1', NULL),
(9, 'XP_038912218.1', NULL),
(9, 'XP_038912217.1', NULL),
(9, 'XP_038912216.1', NULL),
(9, 'XP_038912215.1', NULL),
(9, 'XP_038912214.1', NULL),
(9, 'XP_038912213.1', NULL),
(9, 'XP_038912212.1', NULL),
(9, 'XP_038912211.1', NULL),
(9, 'XP_038912210.1', NULL),
(9, 'XP_038912209.1', NULL),
(9, 'XP_038912208.1', NULL),
(9, 'XP_038912207.1', NULL),
(9, 'XP_038912206.1', NULL),
(9, 'XP_038912205.1', NULL),
(9, 'XP_038912204.1', NULL),
(9, 'XP_038912203.1', NULL),
(9, 'XP_038912202.1', NULL),
(9, 'XP_038912201.1', NULL),
(9, 'XP_038912200.1', NULL),
(9, 'XP_038912199.1', NULL),
(9, 'XP_038912198.1', NULL),
(9, 'XP_038912328.1', NULL),
(9, 'XP_038912327.1', NULL),
(9, 'XP_038912326.1', NULL),
(9, 'XP_038912325.1', NULL),
(9, 'XP_038912324.1', NULL),
(9, 'XP_038912323.1', NULL),
(9, 'XP_038912322.1', NULL),
(9, 'XP_038912321.1', NULL),
(9, 'XP_038912320.1', NULL),
(9, 'XP_038912319.1', NULL),
(9, 'XP_038912318.1', NULL),
(9, 'XP_038912317.1', NULL),
(9, 'XP_038912316.1', NULL),
(9, 'XP_038912315.1', NULL),
(9, 'XP_038912314.1', NULL),
(9, 'XP_038912313.1', NULL),
(9, 'XP_038912312.1', NULL),
(9, 'XP_038912311.1', NULL),
(9, 'XP_038912310.1', NULL),
(9, 'XP_038912309.1', NULL),
(9, 'XP_038912308.1', NULL),
(9, 'XP_038912307.1', NULL),
(9, 'XP_038912306.1', NULL),
(9, 'XP_038912305.1', NULL),
(9, 'XP_038912304.1', NULL),
(9, 'XP_038912303.1', NULL),
(9, 'XP_038912302.1', NULL),
(9, 'XP_038912301.1', NULL),
(9, 'XP_038912300.1', NULL),
(9, 'XP_038912299.1', NULL),
(9, 'XP_038912298.1', NULL),
(9, 'XP_038912297.1', NULL),
(9, 'XP_038912296.1', NULL),
(9, 'XP_038912295.1', NULL),
(9, 'XP_038912294.1', NULL),
(9, 'XP_038912293.1', NULL),
(9, 'XP_038912292.1', NULL),
(9, 'XP_038912291.1', NULL),
(9, 'XP_038912290.1', NULL),
(9, 'XP_038912289.1', NULL),
(9, 'XP_038912288.1', NULL),
(9, 'XP_038912287.1', NULL),
(9, 'XP_038912286.1', NULL),
(9, 'XP_038912285.1', NULL),
(9, 'XP_038912284.1', NULL),
(9, 'XP_038912283.1', NULL),
(9, 'XP_038912282.1', NULL),
(9, 'XP_038912281.1', NULL),
(9, 'XP_038912280.1', NULL),
(9, 'XP_038912279.1', NULL),
(9, 'XP_038912278.1', NULL),
(9, 'XP_038912277.1', NULL),
(9, 'XP_038912276.1', NULL),
(9, 'XP_038912275.1', NULL),
(9, 'XP_038912274.1', NULL),
(9, 'XP_038912273.1', NULL),
(9, 'XP_038912272.1', NULL),
(9, 'XP_038912271.1', NULL),
(9, 'XP_038912270.1', NULL),
(9, 'XP_038912269.1', NULL),
(9, 'XP_038912268.1', NULL),
(9, 'XP_038912267.1', NULL),
(9, 'XP_038912266.1', NULL),
(9, 'XP_038912265.1', NULL),
(9, 'XP_038912264.1', NULL),
(9, 'XP_038912263.1', NULL),
(9, 'XP_038912262.1', NULL),
(9, 'XP_038912261.1', NULL),
(9, 'XP_038912260.1', NULL),
(9, 'XP_038912259.1', NULL),
(9, 'XP_038912258.1', NULL),
(9, 'XP_038912257.1', NULL),
(9, 'XP_038912256.1', NULL),
(9, 'XP_038912255.1', NULL),
(9, 'XP_038912254.1', NULL),
(9, 'XP_038912253.1', NULL),
(9, 'XP_038912252.1', NULL),
(9, 'XP_038912251.1', NULL),
(9, 'XP_038912250.1', NULL),
(9, 'XP_038912249.1', NULL),
(9, 'XP_038912248.1', NULL),
(9, 'XP_038912247.1', NULL),
(9, 'XP_038912246.1', NULL),
(9, 'XP_038912245.1', NULL),
(9, 'XP_038912244.1', NULL),
(9, 'XP_038912243.1', NULL),
(9, 'XP_038912242.1', NULL),
(9, 'XP_038912241.1', NULL),
(9, 'XP_038912240.1', NULL),
(9, 'XP_038912239.1', NULL),
(9, 'XP_038912238.1', NULL),
(9, 'XP_038912237.1', NULL),
(9, 'XP_038912236.1', NULL),
(9, 'XP_038912235.1', NULL),
(9, 'XP_038912234.1', NULL),
(9, 'XP_038912384.1', NULL),
(9, 'XP_038912383.1', NULL),
(9, 'XP_038912382.1', NULL),
(9, 'XP_038912381.1', NULL),
(9, 'XP_038912380.1', NULL),
(9, 'XP_038912379.1', NULL),
(9, 'XP_038912378.1', NULL),
(9, 'XP_038912377.1', NULL),
(9, 'XP_038912376.1', NULL),
(9, 'XP_038912375.1', NULL),
(9, 'XP_038912374.1', NULL),
(9, 'XP_038912373.1', NULL),
(9, 'XP_038912372.1', NULL),
(9, 'XP_038912371.1', NULL),
(9, 'XP_038912370.1', NULL),
(9, 'XP_038912369.1', NULL),
(9, 'XP_038912368.1', NULL),
(9, 'XP_038912367.1', NULL),
(9, 'XP_038912366.1', NULL),
(9, 'XP_038912365.1', NULL),
(9, 'XP_038912364.1', NULL),
(9, 'XP_038912363.1', NULL),
(9, 'XP_038912362.1', NULL),
(9, 'XP_038912361.1', NULL),
(9, 'XP_038912360.1', NULL),
(9, 'XP_038912359.1', NULL),
(9, 'XP_038912358.1', NULL),
(9, 'XP_038912357.1', NULL),
(9, 'XP_038912356.1', NULL),
(9, 'XP_038912355.1', NULL),
(9, 'XP_038912354.1', NULL),
(9, 'XP_038912353.1', NULL),
(9, 'XP_038912352.1', NULL),
(9, 'XP_038912351.1', NULL),
(9, 'XP_038912350.1', NULL),
(9, 'XP_038912349.1', NULL),
(9, 'XP_038912348.1', NULL),
(9, 'XP_038912347.1', NULL),
(9, 'XP_038912346.1', NULL),
(9, 'XP_038912345.1', NULL),
(9, 'XP_038912344.1', NULL),
(9, 'XP_038912343.1', NULL),
(9, 'XP_038912342.1', NULL),
(9, 'XP_038912341.1', NULL),
(9, 'XP_038912340.1', NULL),
(9, 'XP_038912339.1', NULL),
(9, 'XP_038912338.1', NULL),
(9, 'XP_038912337.1', NULL),
(9, 'XP_038912336.1', NULL),
(9, 'XP_038912335.1', NULL),
(9, 'XP_038912334.1', NULL),
(9, 'XP_038912333.1', NULL),
(9, 'XP_038912332.1', NULL),
(9, 'XP_038912331.1', NULL),
(9, 'XP_038912330.1', NULL),
(9, 'XP_038912329.1', NULL),
(9, 'XP_038912446.1', NULL),
(9, 'XP_038912445.1', NULL),
(9, 'XP_038912444.1', NULL),
(9, 'XP_038912443.1', NULL),
(9, 'XP_038912442.1', NULL),
(9, 'XP_038912441.1', NULL),
(9, 'XP_038912440.1', NULL),
(9, 'XP_038912439.1', NULL),
(9, 'XP_038912438.1', NULL),
(9, 'XP_038912437.1', NULL),
(9, 'XP_038912436.1', NULL),
(9, 'XP_038912435.1', NULL),
(9, 'XP_038912434.1', NULL),
(9, 'XP_038912433.1', NULL),
(9, 'XP_038912432.1', NULL),
(9, 'XP_038912431.1', NULL),
(9, 'XP_038912430.1', NULL),
(9, 'XP_038912429.1', NULL),
(9, 'XP_038912428.1', NULL),
(9, 'XP_038912427.1', NULL),
(9, 'XP_038912426.1', NULL),
(9, 'XP_038912425.1', NULL),
(9, 'XP_038912424.1', NULL),
(9, 'XP_038912423.1', NULL),
(9, 'XP_038912422.1', NULL),
(9, 'XP_038912421.1', NULL),
(9, 'XP_038912420.1', NULL),
(9, 'XP_038912419.1', NULL),
(9, 'XP_038912418.1', NULL),
(9, 'XP_038912417.1', NULL),
(9, 'XP_038912416.1', NULL),
(9, 'XP_038912415.1', NULL),
(9, 'XP_038912414.1', NULL),
(9, 'XP_038912413.1', NULL),
(9, 'XP_038912412.1', NULL),
(9, 'XP_038912411.1', NULL),
(9, 'XP_038912410.1', NULL),
(9, 'XP_038912409.1', NULL),
(9, 'XP_038912408.1', NULL),
(9, 'XP_038912407.1', NULL),
(9, 'XP_038912406.1', NULL),
(9, 'XP_038912405.1', NULL),
(9, 'XP_038912404.1', NULL),
(9, 'XP_038912403.1', NULL),
(9, 'XP_038912402.1', NULL),
(9, 'XP_038912401.1', NULL),
(9, 'XP_038912400.1', NULL),
(9, 'XP_038912399.1', NULL),
(9, 'XP_038912398.1', NULL),
(9, 'XP_038912397.1', NULL),
(9, 'XP_038912396.1', NULL),
(9, 'XP_038912395.1', NULL),
(9, 'XP_038912394.1', NULL),
(9, 'XP_038912393.1', NULL),
(9, 'XP_038912392.1', NULL),
(9, 'XP_038912391.1', NULL),
(9, 'XP_038912390.1', NULL),
(9, 'XP_038912389.1', NULL),
(9, 'XP_038912388.1', NULL),
(9, 'XP_038912387.1', NULL),
(9, 'XP_038912386.1', NULL),
(9, 'XP_038912385.1', NULL),
(9, 'XP_038912602.1', NULL),
(9, 'XP_038912601.1', NULL),
(9, 'XP_038912600.1', NULL),
(9, 'XP_038912599.1', NULL),
(9, 'XP_038912598.1', NULL),
(9, 'XP_038912597.1', NULL),
(9, 'XP_038912596.1', NULL),
(9, 'XP_038912595.1', NULL),
(9, 'XP_038912594.1', NULL),
(9, 'XP_038912593.1', NULL),
(9, 'XP_038912592.1', NULL),
(9, 'XP_038912591.1', NULL),
(9, 'XP_038912590.1', NULL),
(9, 'XP_038912589.1', NULL),
(9, 'XP_038912588.1', NULL),
(9, 'XP_038912587.1', NULL),
(9, 'XP_038912586.1', NULL),
(9, 'XP_038912585.1', NULL),
(9, 'XP_038912584.1', NULL),
(9, 'XP_038912583.1', NULL),
(9, 'XP_038912582.1', NULL),
(9, 'XP_038912581.1', NULL),
(9, 'XP_038912580.1', NULL),
(9, 'XP_038912579.1', NULL),
(9, 'XP_038912578.1', NULL),
(9, 'XP_038912577.1', NULL),
(9, 'XP_038912576.1', NULL),
(9, 'XP_038912575.1', NULL),
(9, 'XP_038912574.1', NULL),
(9, 'XP_038912573.1', NULL),
(9, 'XP_038912572.1', NULL),
(9, 'XP_038912571.1', NULL),
(9, 'XP_038912570.1', NULL),
(9, 'XP_038912569.1', NULL),
(9, 'XP_038912568.1', NULL),
(9, 'XP_038912567.1', NULL),
(9, 'XP_038912566.1', NULL),
(9, 'XP_038912565.1', NULL),
(9, 'XP_038912564.1', NULL),
(9, 'XP_038912563.1', NULL),
(9, 'XP_038912562.1', NULL),
(9, 'XP_038912561.1', NULL),
(9, 'XP_038912560.1', NULL),
(9, 'XP_038912559.1', NULL),
(9, 'XP_038912558.1', NULL),
(9, 'XP_038912557.1', NULL),
(9, 'XP_038912556.1', NULL),
(9, 'XP_038912555.1', NULL),
(9, 'XP_038912554.1', NULL),
(9, 'XP_038912553.1', NULL),
(9, 'XP_038912552.1', NULL),
(9, 'XP_038912551.1', NULL),
(9, 'XP_038912550.1', NULL),
(9, 'XP_038912549.1', NULL),
(9, 'XP_038912548.1', NULL),
(9, 'XP_038912547.1', NULL),
(9, 'XP_038912546.1', NULL),
(9, 'XP_038912545.1', NULL),
(9, 'XP_038912544.1', NULL),
(9, 'XP_038912543.1', NULL),
(9, 'XP_038912542.1', NULL),
(9, 'XP_038912541.1', NULL),
(9, 'XP_038912540.1', NULL),
(9, 'XP_038912539.1', NULL),
(9, 'XP_038912538.1', NULL),
(9, 'XP_038912537.1', NULL),
(9, 'XP_038912536.1', NULL),
(9, 'XP_038912535.1', NULL),
(9, 'XP_038912534.1', NULL),
(9, 'XP_038912533.1', NULL),
(9, 'XP_038912532.1', NULL),
(9, 'XP_038912531.1', NULL),
(9, 'XP_038912530.1', NULL),
(9, 'XP_038912529.1', NULL),
(9, 'XP_038912528.1', NULL),
(9, 'XP_038912527.1', NULL),
(9, 'XP_038912526.1', NULL),
(9, 'XP_038912525.1', NULL),
(9, 'XP_038912524.1', NULL),
(9, 'XP_038912523.1', NULL),
(9, 'XP_038912522.1', NULL),
(9, 'XP_038912521.1', NULL),
(9, 'XP_038912520.1', NULL),
(9, 'XP_038912519.1', NULL),
(9, 'XP_038912518.1', NULL),
(9, 'XP_038912517.1', NULL),
(9, 'XP_038912516.1', NULL),
(9, 'XP_038912515.1', NULL),
(9, 'XP_038912514.1', NULL),
(9, 'XP_038912513.1', NULL),
(9, 'XP_038912512.1', NULL),
(9, 'XP_038912511.1', NULL),
(9, 'XP_038912510.1', NULL),
(9, 'XP_038912509.1', NULL),
(9, 'XP_038912508.1', NULL),
(9, 'XP_038912507.1', NULL),
(9, 'XP_038912506.1', NULL),
(9, 'XP_038912505.1', NULL),
(9, 'XP_038912504.1', NULL),
(9, 'XP_038912503.1', NULL),
(9, 'XP_038912502.1', NULL),
(9, 'XP_038912501.1', NULL),
(9, 'XP_038912500.1', NULL),
(9, 'XP_038912499.1', NULL),
(9, 'XP_038912498.1', NULL),
(9, 'XP_038912497.1', NULL),
(9, 'XP_038912496.1', NULL),
(9, 'XP_038912495.1', NULL),
(9, 'XP_038912494.1', NULL),
(9, 'XP_038912493.1', NULL),
(9, 'XP_038912492.1', NULL),
(9, 'XP_038912491.1', NULL),
(9, 'XP_038912490.1', NULL),
(9, 'XP_038912489.1', NULL),
(9, 'XP_038912488.1', NULL),
(9, 'XP_038912487.1', NULL),
(9, 'XP_038912486.1', NULL),
(9, 'XP_038912485.1', NULL),
(9, 'XP_038912484.1', NULL),
(9, 'XP_038912483.1', NULL),
(9, 'XP_038912482.1', NULL),
(9, 'XP_038912481.1', NULL),
(9, 'XP_038912480.1', NULL),
(9, 'XP_038912479.1', NULL),
(9, 'XP_038912478.1', NULL),
(9, 'XP_038912477.1', NULL),
(9, 'XP_038912476.1', NULL),
(9, 'XP_038912475.1', NULL),
(9, 'XP_038912474.1', NULL),
(9, 'XP_038912473.1', NULL),
(9, 'XP_038912472.1', NULL),
(9, 'XP_038912471.1', NULL),
(9, 'XP_038912470.1', NULL),
(9, 'XP_038912469.1', NULL),
(9, 'XP_038912468.1', NULL),
(9, 'XP_038912467.1', NULL),
(9, 'XP_038912466.1', NULL),
(9, 'XP_038912465.1', NULL),
(9, 'XP_038912464.1', NULL),
(9, 'XP_038912463.1', NULL),
(9, 'XP_038912462.1', NULL),
(9, 'XP_038912461.1', NULL),
(9, 'XP_038912460.1', NULL),
(9, 'XP_038912459.1', NULL),
(9, 'XP_038912458.1', NULL),
(9, 'XP_038912457.1', NULL),
(9, 'XP_038912456.1', NULL),
(9, 'XP_038912455.1', NULL),
(9, 'XP_038912454.1', NULL),
(9, 'XP_038912453.1', NULL),
(9, 'XP_038912452.1', NULL),
(9, 'XP_038912451.1', NULL),
(9, 'XP_038912450.1', NULL),
(9, 'XP_038912449.1', NULL),
(9, 'XP_038912448.1', NULL),
(9, 'XP_038912447.1', NULL),
(9, 'XP_038912749.1', NULL),
(9, 'XP_038912748.1', NULL),
(9, 'XP_038912747.1', NULL),
(9, 'XP_038912746.1', NULL),
(9, 'XP_038912745.1', NULL),
(9, 'XP_038912744.1', NULL),
(9, 'XP_038912743.1', NULL),
(9, 'XP_038912742.1', NULL),
(9, 'XP_038912741.1', NULL),
(9, 'XP_038912740.1', NULL),
(9, 'XP_038912739.1', NULL),
(9, 'XP_038912738.1', NULL),
(9, 'XP_038912737.1', NULL),
(9, 'XP_038912736.1', NULL),
(9, 'XP_038912735.1', NULL),
(9, 'XP_038912734.1', NULL),
(9, 'XP_038912733.1', NULL),
(9, 'XP_038912732.1', NULL),
(9, 'XP_038912731.1', NULL),
(9, 'XP_038912730.1', NULL),
(9, 'XP_038912729.1', NULL),
(9, 'XP_038912728.1', NULL),
(9, 'XP_038912727.1', NULL),
(9, 'XP_038912726.1', NULL),
(9, 'XP_038912725.1', NULL),
(9, 'XP_038912724.1', NULL),
(9, 'XP_038912723.1', NULL),
(9, 'XP_038912722.1', NULL),
(9, 'XP_038912721.1', NULL),
(9, 'XP_038912720.1', NULL),
(9, 'XP_038912719.1', NULL),
(9, 'XP_038912718.1', NULL),
(9, 'XP_038912717.1', NULL),
(9, 'XP_038912716.1', NULL),
(9, 'XP_038912715.1', NULL),
(9, 'XP_038912714.1', NULL),
(9, 'XP_038912713.1', NULL),
(9, 'XP_038912712.1', NULL),
(9, 'XP_038912711.1', NULL),
(9, 'XP_038912710.1', NULL),
(9, 'XP_038912709.1', NULL),
(9, 'XP_038912708.1', NULL),
(9, 'XP_038912707.1', NULL),
(9, 'XP_038912706.1', NULL),
(9, 'XP_038912705.1', NULL),
(9, 'XP_038912704.1', NULL),
(9, 'XP_038912703.1', NULL),
(9, 'XP_038912702.1', NULL),
(9, 'XP_038912701.1', NULL),
(9, 'XP_038912700.1', NULL),
(9, 'XP_038912699.1', NULL),
(9, 'XP_038912698.1', NULL),
(9, 'XP_038912697.1', NULL),
(9, 'XP_038912696.1', NULL),
(9, 'XP_038912695.1', NULL),
(9, 'XP_038912694.1', NULL),
(9, 'XP_038912693.1', NULL),
(9, 'XP_038912692.1', NULL),
(9, 'XP_038912691.1', NULL),
(9, 'XP_038912690.1', NULL),
(9, 'XP_038912689.1', NULL),
(9, 'XP_038912688.1', NULL),
(9, 'XP_038912687.1', NULL),
(9, 'XP_038912686.1', NULL),
(9, 'XP_038912685.1', NULL),
(9, 'XP_038912684.1', NULL),
(9, 'XP_038912683.1', NULL),
(9, 'XP_038912682.1', NULL),
(9, 'XP_038912681.1', NULL),
(9, 'XP_038912680.1', NULL),
(9, 'XP_038912679.1', NULL),
(9, 'XP_038912678.1', NULL),
(9, 'XP_038912677.1', NULL),
(9, 'XP_038912676.1', NULL),
(9, 'XP_038912675.1', NULL),
(9, 'XP_038912674.1', NULL),
(9, 'XP_038912673.1', NULL),
(9, 'XP_038912672.1', NULL),
(9, 'XP_038912671.1', NULL),
(9, 'XP_038912670.1', NULL),
(9, 'XP_038912669.1', NULL),
(9, 'XP_038912668.1', NULL),
(9, 'XP_038912667.1', NULL),
(9, 'XP_038912666.1', NULL),
(9, 'XP_038912665.1', NULL),
(9, 'XP_038912664.1', NULL),
(9, 'XP_038912663.1', NULL),
(9, 'XP_038912662.1', NULL),
(9, 'XP_038912661.1', NULL),
(9, 'XP_038912660.1', NULL),
(9, 'XP_038912659.1', NULL),
(9, 'XP_038912658.1', NULL),
(9, 'XP_038912657.1', NULL),
(9, 'XP_038912656.1', NULL),
(9, 'XP_038912655.1', NULL),
(9, 'XP_038912654.1', NULL),
(9, 'XP_038912653.1', NULL),
(9, 'XP_038912652.1', NULL),
(9, 'XP_038912651.1', NULL),
(9, 'XP_038912650.1', NULL),
(9, 'XP_038912649.1', NULL),
(9, 'XP_038912648.1', NULL),
(9, 'XP_038912647.1', NULL),
(9, 'XP_038912646.1', NULL),
(9, 'XP_038912645.1', NULL),
(9, 'XP_038912644.1', NULL),
(9, 'XP_038912643.1', NULL),
(9, 'XP_038912642.1', NULL),
(9, 'XP_038912641.1', NULL),
(9, 'XP_038912640.1', NULL),
(9, 'XP_038912639.1', NULL),
(9, 'XP_038912638.1', NULL),
(9, 'XP_038912637.1', NULL),
(9, 'XP_038912636.1', NULL),
(9, 'XP_038912635.1', NULL),
(9, 'XP_038912634.1', NULL),
(9, 'XP_038912633.1', NULL),
(9, 'XP_038912632.1', NULL),
(9, 'XP_038912631.1', NULL),
(9, 'XP_038912630.1', NULL),
(9, 'XP_038912629.1', NULL),
(9, 'XP_038912628.1', NULL),
(9, 'XP_038912627.1', NULL),
(9, 'XP_038912626.1', NULL),
(9, 'XP_038912625.1', NULL),
(9, 'XP_038912624.1', NULL),
(9, 'XP_038912623.1', NULL),
(9, 'XP_038912622.1', NULL),
(9, 'XP_038912621.1', NULL),
(9, 'XP_038912620.1', NULL),
(9, 'XP_038912619.1', NULL),
(9, 'XP_038912618.1', NULL),
(9, 'XP_038912617.1', NULL),
(9, 'XP_038912616.1', NULL),
(9, 'XP_038912615.1', NULL),
(9, 'XP_038912614.1', NULL),
(9, 'XP_038912613.1', NULL),
(9, 'XP_038912612.1', NULL),
(9, 'XP_038912611.1', NULL),
(9, 'XP_038912610.1', NULL),
(9, 'XP_038912609.1', NULL),
(9, 'XP_038912608.1', NULL),
(9, 'XP_038912607.1', NULL),
(9, 'XP_038912606.1', NULL),
(9, 'XP_038912605.1', NULL),
(9, 'XP_038912604.1', NULL),
(9, 'XP_038912603.1', NULL),
(9, 'XP_038912801.1', NULL),
(9, 'XP_038912800.1', NULL),
(9, 'XP_038912799.1', NULL),
(9, 'XP_038912798.1', NULL),
(9, 'XP_038912797.1', NULL),
(9, 'XP_038912796.1', NULL),
(9, 'XP_038912795.1', NULL),
(9, 'XP_038912794.1', NULL),
(9, 'XP_038912793.1', NULL),
(9, 'XP_038912792.1', NULL),
(9, 'XP_038912791.1', NULL),
(9, 'XP_038912790.1', NULL),
(9, 'XP_038912789.1', NULL),
(9, 'XP_038912788.1', NULL),
(9, 'XP_038912787.1', NULL),
(9, 'XP_038912786.1', NULL),
(9, 'XP_038912785.1', NULL),
(9, 'XP_038912784.1', NULL),
(9, 'XP_038912783.1', NULL),
(9, 'XP_038912782.1', NULL),
(9, 'XP_038912781.1', NULL),
(9, 'XP_038912780.1', NULL),
(9, 'XP_038912779.1', NULL),
(9, 'XP_038912778.1', NULL),
(9, 'XP_038912777.1', NULL),
(9, 'XP_038912776.1', NULL),
(9, 'XP_038912775.1', NULL),
(9, 'XP_038912774.1', NULL),
(9, 'XP_038912773.1', NULL),
(9, 'XP_038912772.1', NULL),
(9, 'XP_038912771.1', NULL),
(9, 'XP_038912770.1', NULL),
(9, 'XP_038912769.1', NULL),
(9, 'XP_038912768.1', NULL),
(9, 'XP_038912767.1', NULL),
(9, 'XP_038912766.1', NULL),
(9, 'XP_038912765.1', NULL),
(9, 'XP_038912764.1', NULL),
(9, 'XP_038912763.1', NULL),
(9, 'XP_038912762.1', NULL),
(9, 'XP_038912761.1', NULL),
(9, 'XP_038912760.1', NULL),
(9, 'XP_038912759.1', NULL),
(9, 'XP_038912758.1', NULL),
(9, 'XP_038912757.1', NULL),
(9, 'XP_038912756.1', NULL),
(9, 'XP_038912755.1', NULL),
(9, 'XP_038912754.1', NULL),
(9, 'XP_038912753.1', NULL),
(9, 'XP_038912752.1', NULL),
(9, 'XP_038912751.1', NULL),
(9, 'XP_038912750.1', NULL),
(9, 'XP_038912918.1', NULL),
(9, 'XP_038912917.1', NULL),
(9, 'XP_038912916.1', NULL),
(9, 'XP_038912915.1', NULL),
(9, 'XP_038912914.1', NULL),
(9, 'XP_038912913.1', NULL),
(9, 'XP_038912912.1', NULL),
(9, 'XP_038912911.1', NULL),
(9, 'XP_038912910.1', NULL),
(9, 'XP_038912909.1', NULL),
(9, 'XP_038912908.1', NULL),
(9, 'XP_038912907.1', NULL),
(9, 'XP_038912906.1', NULL),
(9, 'XP_038912905.1', NULL),
(9, 'XP_038912904.1', NULL),
(9, 'XP_038912903.1', NULL),
(9, 'XP_038912902.1', NULL),
(9, 'XP_038912901.1', NULL),
(9, 'XP_038912900.1', NULL),
(9, 'XP_038912899.1', NULL),
(9, 'XP_038912898.1', NULL),
(9, 'XP_038912897.1', NULL),
(9, 'XP_038912896.1', NULL),
(9, 'XP_038912895.1', NULL),
(9, 'XP_038912894.1', NULL);
INSERT INTO `prot_access_table` (`id_article`, `prot_access`, `entry_id_cazy`) VALUES
(9, 'XP_038912893.1', NULL),
(9, 'XP_038912892.1', NULL),
(9, 'XP_038912891.1', NULL),
(9, 'XP_038912890.1', NULL),
(9, 'XP_038912889.1', NULL),
(9, 'XP_038912888.1', NULL),
(9, 'XP_038912887.1', NULL),
(9, 'XP_038912886.1', NULL),
(9, 'XP_038912885.1', NULL),
(9, 'XP_038912884.1', NULL),
(9, 'XP_038912883.1', NULL),
(9, 'XP_038912882.1', NULL),
(9, 'XP_038912881.1', NULL),
(9, 'XP_038912880.1', NULL),
(9, 'XP_038912879.1', NULL),
(9, 'XP_038912878.1', NULL),
(9, 'XP_038912877.1', NULL),
(9, 'XP_038912876.1', NULL),
(9, 'XP_038912875.1', NULL),
(9, 'XP_038912874.1', NULL),
(9, 'XP_038912873.1', NULL),
(9, 'XP_038912872.1', NULL),
(9, 'XP_038912871.1', NULL),
(9, 'XP_038912870.1', NULL),
(9, 'XP_038912869.1', NULL),
(9, 'XP_038912868.1', NULL),
(9, 'XP_038912867.1', NULL),
(9, 'XP_038912866.1', NULL),
(9, 'XP_038912865.1', NULL),
(9, 'XP_038912864.1', NULL),
(9, 'XP_038912863.1', NULL),
(9, 'XP_038912862.1', NULL),
(9, 'XP_038912861.1', NULL),
(9, 'XP_038912860.1', NULL),
(9, 'XP_038912859.1', NULL),
(9, 'XP_038912858.1', NULL),
(9, 'XP_038912857.1', NULL),
(9, 'XP_038912856.1', NULL),
(9, 'XP_038912855.1', NULL),
(9, 'XP_038912854.1', NULL),
(9, 'XP_038912853.1', NULL),
(9, 'XP_038912852.1', NULL),
(9, 'XP_038912851.1', NULL),
(9, 'XP_038912850.1', NULL),
(9, 'XP_038912849.1', NULL),
(9, 'XP_038912848.1', NULL),
(9, 'XP_038912847.1', NULL),
(9, 'XP_038912846.1', NULL),
(9, 'XP_038912845.1', NULL),
(9, 'XP_038912844.1', NULL),
(9, 'XP_038912843.1', NULL),
(9, 'XP_038912842.1', NULL),
(9, 'XP_038912841.1', NULL),
(9, 'XP_038912840.1', NULL),
(9, 'XP_038912839.1', NULL),
(9, 'XP_038912838.1', NULL),
(9, 'XP_038912837.1', NULL),
(9, 'XP_038912836.1', NULL),
(9, 'XP_038912835.1', NULL),
(9, 'XP_038912834.1', NULL),
(9, 'XP_038912833.1', NULL),
(9, 'XP_038912832.1', NULL),
(9, 'XP_038912831.1', NULL),
(9, 'XP_038912830.1', NULL),
(9, 'XP_038912829.1', NULL),
(9, 'XP_038912828.1', NULL),
(9, 'XP_038912827.1', NULL),
(9, 'XP_038912826.1', NULL),
(9, 'XP_038912825.1', NULL),
(9, 'XP_038912824.1', NULL),
(9, 'XP_038912823.1', NULL),
(9, 'XP_038912822.1', NULL),
(9, 'XP_038912821.1', NULL),
(9, 'XP_038912820.1', NULL),
(9, 'XP_038912819.1', NULL),
(9, 'XP_038912818.1', NULL),
(9, 'XP_038912817.1', NULL),
(9, 'XP_038912816.1', NULL),
(9, 'XP_038912815.1', NULL),
(9, 'XP_038912814.1', NULL),
(9, 'XP_038912813.1', NULL),
(9, 'XP_038912812.1', NULL),
(9, 'XP_038912811.1', NULL),
(9, 'XP_038912810.1', NULL),
(9, 'XP_038912809.1', NULL),
(9, 'XP_038912808.1', NULL),
(9, 'XP_038912807.1', NULL),
(9, 'XP_038912806.1', NULL),
(9, 'XP_038912805.1', NULL),
(9, 'XP_038912804.1', NULL),
(9, 'XP_038912803.1', NULL),
(9, 'XP_038912802.1', NULL),
(9, 'XP_038913114.1', NULL),
(9, 'XP_038913113.1', NULL),
(9, 'XP_038913112.1', NULL),
(9, 'XP_038913111.1', NULL),
(9, 'XP_038913110.1', NULL),
(9, 'XP_038913109.1', NULL),
(9, 'XP_038913108.1', NULL),
(9, 'XP_038913107.1', NULL),
(9, 'XP_038913106.1', NULL),
(9, 'XP_038913105.1', NULL),
(9, 'XP_038913104.1', NULL),
(9, 'XP_038913103.1', NULL),
(9, 'XP_038913102.1', NULL),
(9, 'XP_038913101.1', NULL),
(9, 'XP_038913100.1', NULL),
(9, 'XP_038913099.1', NULL),
(9, 'XP_038913098.1', NULL),
(9, 'XP_038913097.1', NULL),
(9, 'XP_038913096.1', NULL),
(9, 'XP_038913095.1', NULL),
(9, 'XP_038913094.1', NULL),
(9, 'XP_038913093.1', NULL),
(9, 'XP_038913092.1', NULL),
(9, 'XP_038913091.1', NULL),
(9, 'XP_038913090.1', NULL),
(9, 'XP_038913089.1', NULL),
(9, 'XP_038913088.1', NULL),
(9, 'XP_038913087.1', NULL),
(9, 'XP_038913086.1', NULL),
(9, 'XP_038913085.1', NULL),
(9, 'XP_038913084.1', NULL),
(9, 'XP_038913083.1', NULL),
(9, 'XP_038913082.1', NULL),
(9, 'XP_038913081.1', NULL),
(9, 'XP_038913080.1', NULL),
(9, 'XP_038913079.1', NULL),
(9, 'XP_038913078.1', NULL),
(9, 'XP_038913077.1', NULL),
(9, 'XP_038913076.1', NULL),
(9, 'XP_038913075.1', NULL),
(9, 'XP_038913074.1', NULL),
(9, 'XP_038913073.1', NULL),
(9, 'XP_038913072.1', NULL),
(9, 'XP_038913071.1', NULL),
(9, 'XP_038913070.1', NULL),
(9, 'XP_038913069.1', NULL),
(9, 'XP_038913068.1', NULL),
(9, 'XP_038913067.1', NULL),
(9, 'XP_038913066.1', NULL),
(9, 'XP_038913065.1', NULL),
(9, 'XP_038913064.1', NULL),
(9, 'XP_038913063.1', NULL),
(9, 'XP_038913062.1', NULL),
(9, 'XP_038913061.1', NULL),
(9, 'XP_038913060.1', NULL),
(9, 'XP_038913059.1', NULL),
(9, 'XP_038913058.1', NULL),
(9, 'XP_038913057.1', NULL),
(9, 'XP_038913056.1', NULL),
(9, 'XP_038913055.1', NULL),
(9, 'XP_038913054.1', NULL),
(9, 'XP_038913053.1', NULL),
(9, 'XP_038913052.1', NULL),
(9, 'XP_038913051.1', NULL),
(9, 'XP_038913050.1', NULL),
(9, 'XP_038913049.1', NULL),
(9, 'XP_038913048.1', NULL),
(9, 'XP_038913047.1', NULL),
(9, 'XP_038913046.1', NULL),
(9, 'XP_038913045.1', NULL),
(9, 'XP_038913044.1', NULL),
(9, 'XP_038913043.1', NULL),
(9, 'XP_038913042.1', NULL),
(9, 'XP_038913041.1', NULL),
(9, 'XP_038913040.1', NULL),
(9, 'XP_038913039.1', NULL),
(9, 'XP_038913038.1', NULL),
(9, 'XP_038913037.1', NULL),
(9, 'XP_038913036.1', NULL),
(9, 'XP_038913035.1', NULL),
(9, 'XP_038913034.1', NULL),
(9, 'XP_038913033.1', NULL),
(9, 'XP_038913032.1', NULL),
(9, 'XP_038913031.1', NULL),
(9, 'XP_038913030.1', NULL),
(9, 'XP_038913029.1', NULL),
(9, 'XP_038913028.1', NULL),
(9, 'XP_038913027.1', NULL),
(9, 'XP_038913026.1', NULL),
(9, 'XP_038913025.1', NULL),
(9, 'XP_038913024.1', NULL),
(9, 'XP_038913023.1', NULL),
(9, 'XP_038913022.1', NULL),
(9, 'XP_038913021.1', NULL),
(9, 'XP_038913020.1', NULL),
(9, 'XP_038913019.1', NULL),
(9, 'XP_038913018.1', NULL),
(9, 'XP_038913017.1', NULL),
(9, 'XP_038913016.1', NULL),
(9, 'XP_038913015.1', NULL),
(9, 'XP_038913014.1', NULL),
(9, 'XP_038913013.1', NULL),
(9, 'XP_038913012.1', NULL),
(9, 'XP_038913011.1', NULL),
(9, 'XP_038913010.1', NULL),
(9, 'XP_038913009.1', NULL),
(9, 'XP_038913008.1', NULL),
(9, 'XP_038913007.1', NULL),
(9, 'XP_038913006.1', NULL),
(9, 'XP_038913005.1', NULL),
(9, 'XP_038913004.1', NULL),
(9, 'XP_038913003.1', NULL),
(9, 'XP_038913002.1', NULL),
(9, 'XP_038913001.1', NULL),
(9, 'XP_038913000.1', NULL),
(9, 'XP_038912999.1', NULL),
(9, 'XP_038912998.1', NULL),
(9, 'XP_038912997.1', NULL),
(9, 'XP_038912996.1', NULL),
(9, 'XP_038912995.1', NULL),
(9, 'XP_038912994.1', NULL),
(9, 'XP_038912993.1', NULL),
(9, 'XP_038912992.1', NULL),
(9, 'XP_038912991.1', NULL),
(9, 'XP_038912990.1', NULL),
(9, 'XP_038912989.1', NULL),
(9, 'XP_038912988.1', NULL),
(9, 'XP_038912987.1', NULL),
(9, 'XP_038912986.1', NULL),
(9, 'XP_038912985.1', NULL),
(9, 'XP_038912984.1', NULL),
(9, 'XP_038912983.1', NULL),
(9, 'XP_038912982.1', NULL),
(9, 'XP_038912981.1', NULL),
(9, 'XP_038912980.1', NULL),
(9, 'XP_038912979.1', NULL),
(9, 'XP_038912978.1', NULL),
(9, 'XP_038912977.1', NULL),
(9, 'XP_038912976.1', NULL),
(9, 'XP_038912975.1', NULL),
(9, 'XP_038912974.1', NULL),
(9, 'XP_038912973.1', NULL),
(9, 'XP_038912972.1', NULL),
(9, 'XP_038912971.1', NULL),
(9, 'XP_038912970.1', NULL),
(9, 'XP_038912969.1', NULL),
(9, 'XP_038912968.1', NULL),
(9, 'XP_038912967.1', NULL),
(9, 'XP_038912966.1', NULL),
(9, 'XP_038912965.1', NULL),
(9, 'XP_038912964.1', NULL),
(9, 'XP_038912963.1', NULL),
(9, 'XP_038912962.1', NULL),
(9, 'XP_038912961.1', NULL),
(9, 'XP_038912960.1', NULL),
(9, 'XP_038912959.1', NULL),
(9, 'XP_038912958.1', NULL),
(9, 'XP_038912957.1', NULL),
(9, 'XP_038912956.1', NULL),
(9, 'XP_038912955.1', NULL),
(9, 'XP_038912954.1', NULL),
(9, 'XP_038912953.1', NULL),
(9, 'XP_038912952.1', NULL),
(9, 'XP_038912951.1', NULL),
(9, 'XP_038912950.1', NULL),
(9, 'XP_038912949.1', NULL),
(9, 'XP_038912948.1', NULL),
(9, 'XP_038912947.1', NULL),
(9, 'XP_038912946.1', NULL),
(9, 'XP_038912945.1', NULL),
(9, 'XP_038912944.1', NULL),
(9, 'XP_038912943.1', NULL),
(9, 'XP_038912942.1', NULL),
(9, 'XP_038912941.1', NULL),
(9, 'XP_038912940.1', NULL),
(9, 'XP_038912939.1', NULL),
(9, 'XP_038912938.1', NULL),
(9, 'XP_038912937.1', NULL),
(9, 'XP_038912936.1', NULL),
(9, 'XP_038912935.1', NULL),
(9, 'XP_038912934.1', NULL),
(9, 'XP_038912933.1', NULL),
(9, 'XP_038912932.1', NULL),
(9, 'XP_038912931.1', NULL),
(9, 'XP_038912930.1', NULL),
(9, 'XP_038912929.1', NULL),
(9, 'XP_038912928.1', NULL),
(9, 'XP_038912927.1', NULL),
(9, 'XP_038912926.1', NULL),
(9, 'XP_038912925.1', NULL),
(9, 'XP_038912924.1', NULL),
(9, 'XP_038912923.1', NULL),
(9, 'XP_038912922.1', NULL),
(9, 'XP_038912921.1', NULL),
(9, 'XP_038912920.1', NULL),
(9, 'XP_038912919.1', NULL),
(9, 'XP_038913221.1', NULL),
(9, 'XP_038913220.1', NULL),
(9, 'XP_038913219.1', NULL),
(9, 'XP_038913218.1', NULL),
(9, 'XP_038913217.1', NULL),
(9, 'XP_038913216.1', NULL),
(9, 'XP_038913215.1', NULL),
(9, 'XP_038913214.1', NULL),
(9, 'XP_038913213.1', NULL),
(9, 'XP_038913212.1', NULL),
(9, 'XP_038913211.1', NULL),
(9, 'XP_038913210.1', NULL),
(9, 'XP_038913209.1', NULL),
(9, 'XP_038913208.1', NULL),
(9, 'XP_038913207.1', NULL),
(9, 'XP_038913206.1', NULL),
(9, 'XP_038913205.1', NULL),
(9, 'XP_038913204.1', NULL),
(9, 'XP_038913203.1', NULL),
(9, 'XP_038913202.1', NULL),
(9, 'XP_038913201.1', NULL),
(9, 'XP_038913200.1', NULL),
(9, 'XP_038913199.1', NULL),
(9, 'XP_038913198.1', NULL),
(9, 'XP_038913197.1', NULL),
(9, 'XP_038913196.1', NULL),
(9, 'XP_038913195.1', NULL),
(9, 'XP_038913194.1', NULL),
(9, 'XP_038913193.1', NULL),
(9, 'XP_038913192.1', NULL),
(9, 'XP_038913191.1', NULL),
(9, 'XP_038913190.1', NULL),
(9, 'XP_038913189.1', NULL),
(9, 'XP_038913188.1', NULL),
(9, 'XP_038913187.1', NULL),
(9, 'XP_038913186.1', NULL),
(9, 'XP_038913185.1', NULL),
(9, 'XP_038913184.1', NULL),
(9, 'XP_038913183.1', NULL),
(9, 'XP_038913182.1', NULL),
(9, 'XP_038913181.1', NULL),
(9, 'XP_038913180.1', NULL),
(9, 'XP_038913179.1', NULL),
(9, 'XP_038913178.1', NULL),
(9, 'XP_038913177.1', NULL),
(9, 'XP_038913176.1', NULL),
(9, 'XP_038913175.1', NULL),
(9, 'XP_038913174.1', NULL),
(9, 'XP_038913173.1', NULL),
(9, 'XP_038913172.1', NULL),
(9, 'XP_038913171.1', NULL),
(9, 'XP_038913170.1', NULL),
(9, 'XP_038913169.1', NULL),
(9, 'XP_038913168.1', NULL),
(9, 'XP_038913167.1', NULL),
(9, 'XP_038913166.1', NULL),
(9, 'XP_038913165.1', NULL),
(9, 'XP_038913164.1', NULL),
(9, 'XP_038913163.1', NULL),
(9, 'XP_038913162.1', NULL),
(9, 'XP_038913161.1', NULL),
(9, 'XP_038913160.1', NULL),
(9, 'XP_038913159.1', NULL),
(9, 'XP_038913158.1', NULL),
(9, 'XP_038913157.1', NULL),
(9, 'XP_038913156.1', NULL),
(9, 'XP_038913155.1', NULL),
(9, 'XP_038913154.1', NULL),
(9, 'XP_038913153.1', NULL),
(9, 'XP_038913152.1', NULL),
(9, 'XP_038913151.1', NULL),
(9, 'XP_038913150.1', NULL),
(9, 'XP_038913149.1', NULL),
(9, 'XP_038913148.1', NULL),
(9, 'XP_038913147.1', NULL),
(9, 'XP_038913146.1', NULL),
(9, 'XP_038913145.1', NULL),
(9, 'XP_038913144.1', NULL),
(9, 'XP_038913143.1', NULL),
(9, 'XP_038913142.1', NULL),
(9, 'XP_038913141.1', NULL),
(9, 'XP_038913140.1', NULL),
(9, 'XP_038913139.1', NULL),
(9, 'XP_038913138.1', NULL),
(9, 'XP_038913137.1', NULL),
(9, 'XP_038913136.1', NULL),
(9, 'XP_038913135.1', NULL),
(9, 'XP_038913134.1', NULL),
(9, 'XP_038913133.1', NULL),
(9, 'XP_038913132.1', NULL),
(9, 'XP_038913131.1', NULL),
(9, 'XP_038913130.1', NULL),
(9, 'XP_038913129.1', NULL),
(9, 'XP_038913128.1', NULL),
(9, 'XP_038913127.1', NULL),
(9, 'XP_038913126.1', NULL),
(9, 'XP_038913125.1', NULL),
(9, 'XP_038913124.1', NULL),
(9, 'XP_038913123.1', NULL),
(9, 'XP_038913122.1', NULL),
(9, 'XP_038913121.1', NULL),
(9, 'XP_038913120.1', NULL),
(9, 'XP_038913119.1', NULL),
(9, 'XP_038913118.1', NULL),
(9, 'XP_038913117.1', NULL),
(9, 'XP_038913116.1', NULL),
(9, 'XP_038913115.1', NULL),
(9, 'XP_038913305.1', NULL),
(9, 'XP_038913304.1', NULL),
(9, 'XP_038913303.1', NULL),
(9, 'XP_038913302.1', NULL),
(9, 'XP_038913301.1', NULL),
(9, 'XP_038913300.1', NULL),
(9, 'XP_038913299.1', NULL),
(9, 'XP_038913298.1', NULL),
(9, 'XP_038913297.1', NULL),
(9, 'XP_038913296.1', NULL),
(9, 'XP_038913295.1', NULL),
(9, 'XP_038913294.1', NULL),
(9, 'XP_038913293.1', NULL),
(9, 'XP_038913292.1', NULL),
(9, 'XP_038913291.1', NULL),
(9, 'XP_038913290.1', NULL),
(9, 'XP_038913289.1', NULL),
(9, 'XP_038913288.1', NULL),
(9, 'XP_038913287.1', NULL),
(9, 'XP_038913286.1', NULL),
(9, 'XP_038913285.1', NULL),
(9, 'XP_038913284.1', NULL),
(9, 'XP_038913283.1', NULL),
(9, 'XP_038913282.1', NULL),
(9, 'XP_038913281.1', NULL),
(9, 'XP_038913280.1', NULL),
(9, 'XP_038913279.1', NULL),
(9, 'XP_038913278.1', NULL),
(9, 'XP_038913277.1', NULL),
(9, 'XP_038913276.1', NULL),
(9, 'XP_038913275.1', NULL),
(9, 'XP_038913274.1', NULL),
(9, 'XP_038913273.1', NULL),
(9, 'XP_038913272.1', NULL),
(9, 'XP_038913271.1', NULL),
(9, 'XP_038913270.1', NULL),
(9, 'XP_038913269.1', NULL),
(9, 'XP_038913268.1', NULL),
(9, 'XP_038913267.1', NULL),
(9, 'XP_038913266.1', NULL),
(9, 'XP_038913265.1', NULL),
(9, 'XP_038913264.1', NULL),
(9, 'XP_038913263.1', NULL),
(9, 'XP_038913262.1', NULL),
(9, 'XP_038913261.1', NULL),
(9, 'XP_038913260.1', NULL),
(9, 'XP_038913259.1', NULL),
(9, 'XP_038913258.1', NULL),
(9, 'XP_038913257.1', NULL),
(9, 'XP_038913256.1', NULL),
(9, 'XP_038913255.1', NULL),
(9, 'XP_038913254.1', NULL),
(9, 'XP_038913253.1', NULL),
(9, 'XP_038913252.1', NULL),
(9, 'XP_038913251.1', NULL),
(9, 'XP_038913250.1', NULL),
(9, 'XP_038913249.1', NULL),
(9, 'XP_038913248.1', NULL),
(9, 'XP_038913247.1', NULL),
(9, 'XP_038913246.1', NULL),
(9, 'XP_038913245.1', NULL),
(9, 'XP_038913244.1', NULL),
(9, 'XP_038913243.1', NULL),
(9, 'XP_038913242.1', NULL),
(9, 'XP_038913241.1', NULL),
(9, 'XP_038913240.1', NULL),
(9, 'XP_038913239.1', NULL),
(9, 'XP_038913238.1', NULL),
(9, 'XP_038913237.1', NULL),
(9, 'XP_038913236.1', NULL),
(9, 'XP_038913235.1', NULL),
(9, 'XP_038913234.1', NULL),
(9, 'XP_038913233.1', NULL),
(9, 'XP_038913232.1', NULL),
(9, 'XP_038913231.1', NULL),
(9, 'XP_038913230.1', NULL),
(9, 'XP_038913229.1', NULL),
(9, 'XP_038913228.1', NULL),
(9, 'XP_038913227.1', NULL),
(9, 'XP_038913226.1', NULL),
(9, 'XP_038913225.1', NULL),
(9, 'XP_038913224.1', NULL),
(9, 'XP_038913223.1', NULL),
(9, 'XP_038913222.1', NULL),
(9, 'XP_038913474.1', NULL),
(9, 'XP_038913473.1', NULL),
(9, 'XP_038913472.1', NULL),
(9, 'XP_038913471.1', NULL),
(9, 'XP_038913470.1', NULL),
(9, 'XP_038913469.1', NULL),
(9, 'XP_038913468.1', NULL),
(9, 'XP_038913467.1', NULL),
(9, 'XP_038913466.1', NULL),
(9, 'XP_038913465.1', NULL),
(9, 'XP_038913464.1', NULL),
(9, 'XP_038913463.1', NULL),
(9, 'XP_038913462.1', NULL),
(9, 'XP_038913461.1', NULL),
(9, 'XP_038913460.1', NULL),
(9, 'XP_038913459.1', NULL),
(9, 'XP_038913458.1', NULL),
(9, 'XP_038913457.1', NULL),
(9, 'XP_038913456.1', NULL),
(9, 'XP_038913455.1', NULL),
(9, 'XP_038913454.1', NULL),
(9, 'XP_038913453.1', NULL),
(9, 'XP_038913452.1', NULL),
(9, 'XP_038913451.1', NULL),
(9, 'XP_038913450.1', NULL),
(9, 'XP_038913449.1', NULL),
(9, 'XP_038913448.1', NULL),
(9, 'XP_038913447.1', NULL),
(9, 'XP_038913446.1', NULL),
(9, 'XP_038913445.1', NULL),
(9, 'XP_038913444.1', NULL),
(9, 'XP_038913443.1', NULL),
(9, 'XP_038913442.1', NULL),
(9, 'XP_038913441.1', NULL),
(9, 'XP_038913440.1', NULL),
(9, 'XP_038913439.1', NULL),
(9, 'XP_038913438.1', NULL),
(9, 'XP_038913437.1', NULL),
(9, 'XP_038913436.1', NULL),
(9, 'XP_038913435.1', NULL),
(9, 'XP_038913434.1', NULL),
(9, 'XP_038913433.1', NULL),
(9, 'XP_038913432.1', NULL),
(9, 'XP_038913431.1', NULL),
(9, 'XP_038913430.1', NULL),
(9, 'XP_038913429.1', NULL),
(9, 'XP_038913428.1', NULL),
(9, 'XP_038913427.1', NULL),
(9, 'XP_038913426.1', NULL),
(9, 'XP_038913425.1', NULL),
(9, 'XP_038913424.1', NULL),
(9, 'XP_038913423.1', NULL),
(9, 'XP_038913422.1', NULL),
(9, 'XP_038913421.1', NULL),
(9, 'XP_038913420.1', NULL),
(9, 'XP_038913419.1', NULL),
(9, 'XP_038913418.1', NULL),
(9, 'XP_038913417.1', NULL),
(9, 'XP_038913416.1', NULL),
(9, 'XP_038913415.1', NULL),
(9, 'XP_038913414.1', NULL),
(9, 'XP_038913413.1', NULL),
(9, 'XP_038913412.1', NULL),
(9, 'XP_038913411.1', NULL),
(9, 'XP_038913410.1', NULL),
(9, 'XP_038913409.1', NULL),
(9, 'XP_038913408.1', NULL),
(9, 'XP_038913407.1', NULL),
(9, 'XP_038913406.1', NULL),
(9, 'XP_038913405.1', NULL),
(9, 'XP_038913404.1', NULL),
(9, 'XP_038913403.1', NULL),
(9, 'XP_038913402.1', NULL),
(9, 'XP_038913401.1', NULL),
(9, 'XP_038913400.1', NULL),
(9, 'XP_038913399.1', NULL),
(9, 'XP_038913398.1', NULL),
(9, 'XP_038913397.1', NULL),
(9, 'XP_038913396.1', NULL),
(9, 'XP_038913395.1', NULL),
(9, 'XP_038913394.1', NULL),
(9, 'XP_038913393.1', NULL),
(9, 'XP_038913392.1', NULL),
(9, 'XP_038913391.1', NULL),
(9, 'XP_038913390.1', NULL),
(9, 'XP_038913389.1', NULL),
(9, 'XP_038913388.1', NULL),
(9, 'XP_038913387.1', NULL),
(9, 'XP_038913386.1', NULL),
(9, 'XP_038913385.1', NULL),
(9, 'XP_038913384.1', NULL),
(9, 'XP_038913383.1', NULL),
(9, 'XP_038913382.1', NULL),
(9, 'XP_038913381.1', NULL),
(9, 'XP_038913380.1', NULL),
(9, 'XP_038913379.1', NULL),
(9, 'XP_038913378.1', NULL),
(9, 'XP_038913377.1', NULL),
(9, 'XP_038913376.1', NULL),
(9, 'XP_038913375.1', NULL),
(9, 'XP_038913374.1', NULL),
(9, 'XP_038913373.1', NULL),
(9, 'XP_038913372.1', NULL),
(9, 'XP_038913371.1', NULL),
(9, 'XP_038913370.1', NULL),
(9, 'XP_038913369.1', NULL),
(9, 'XP_038913368.1', NULL),
(9, 'XP_038913367.1', NULL),
(9, 'XP_038913366.1', NULL),
(9, 'XP_038913365.1', NULL),
(9, 'XP_038913364.1', NULL),
(9, 'XP_038913363.1', NULL),
(9, 'XP_038913362.1', NULL),
(9, 'XP_038913361.1', NULL),
(9, 'XP_038913360.1', NULL),
(9, 'XP_038913359.1', NULL),
(9, 'XP_038913358.1', NULL),
(9, 'XP_038913357.1', NULL),
(9, 'XP_038913356.1', NULL),
(9, 'XP_038913355.1', NULL),
(9, 'XP_038913354.1', NULL),
(9, 'XP_038913353.1', NULL),
(9, 'XP_038913352.1', NULL),
(9, 'XP_038913351.1', NULL),
(9, 'XP_038913350.1', NULL),
(9, 'XP_038913349.1', NULL),
(9, 'XP_038913348.1', NULL),
(9, 'XP_038913347.1', NULL),
(9, 'XP_038913346.1', NULL),
(9, 'XP_038913345.1', NULL),
(9, 'XP_038913344.1', NULL),
(9, 'XP_038913343.1', NULL),
(9, 'XP_038913342.1', NULL),
(9, 'XP_038913341.1', NULL),
(9, 'XP_038913340.1', NULL),
(9, 'XP_038913339.1', NULL),
(9, 'XP_038913338.1', NULL),
(9, 'XP_038913337.1', NULL),
(9, 'XP_038913336.1', NULL),
(9, 'XP_038913335.1', NULL),
(9, 'XP_038913334.1', NULL),
(9, 'XP_038913333.1', NULL),
(9, 'XP_038913332.1', NULL),
(9, 'XP_038913331.1', NULL),
(9, 'XP_038913330.1', NULL),
(9, 'XP_038913329.1', NULL),
(9, 'XP_038913328.1', NULL),
(9, 'XP_038913327.1', NULL),
(9, 'XP_038913326.1', NULL),
(9, 'XP_038913325.1', NULL),
(9, 'XP_038913324.1', NULL),
(9, 'XP_038913323.1', NULL),
(9, 'XP_038913322.1', NULL),
(9, 'XP_038913321.1', NULL),
(9, 'XP_038913320.1', NULL),
(9, 'XP_038913319.1', NULL),
(9, 'XP_038913318.1', NULL),
(9, 'XP_038913317.1', NULL),
(9, 'XP_038913316.1', NULL),
(9, 'XP_038913315.1', NULL),
(9, 'XP_038913314.1', NULL),
(9, 'XP_038913313.1', NULL),
(9, 'XP_038913312.1', NULL),
(9, 'XP_038913311.1', NULL),
(9, 'XP_038913310.1', NULL),
(9, 'XP_038913309.1', NULL),
(9, 'XP_038913308.1', NULL),
(9, 'XP_038913307.1', NULL),
(9, 'XP_038913306.1', NULL),
(9, 'XP_038913594.1', NULL),
(9, 'XP_038913593.1', NULL),
(9, 'XP_038913592.1', NULL),
(9, 'XP_038913591.1', NULL),
(9, 'XP_038913590.1', NULL),
(9, 'XP_038913589.1', NULL),
(9, 'XP_038913588.1', NULL),
(9, 'XP_038913587.1', NULL),
(9, 'XP_038913586.1', NULL),
(9, 'XP_038913585.1', NULL),
(9, 'XP_038913584.1', NULL),
(9, 'XP_038913583.1', NULL),
(9, 'XP_038913582.1', NULL),
(9, 'XP_038913581.1', NULL),
(9, 'XP_038913580.1', NULL),
(9, 'XP_038913579.1', NULL),
(9, 'XP_038913578.1', NULL),
(9, 'XP_038913577.1', NULL),
(9, 'XP_038913576.1', NULL),
(9, 'XP_038913575.1', NULL),
(9, 'XP_038913574.1', NULL),
(9, 'XP_038913573.1', NULL),
(9, 'XP_038913572.1', NULL),
(9, 'XP_038913571.1', NULL),
(9, 'XP_038913570.1', NULL),
(9, 'XP_038913569.1', NULL),
(9, 'XP_038913568.1', NULL),
(9, 'XP_038913567.1', NULL),
(9, 'XP_038913566.1', NULL),
(9, 'XP_038913565.1', NULL),
(9, 'XP_038913564.1', NULL),
(9, 'XP_038913563.1', NULL),
(9, 'XP_038913562.1', NULL),
(9, 'XP_038913561.1', NULL),
(9, 'XP_038913560.1', NULL),
(9, 'XP_038913559.1', NULL),
(9, 'XP_038913558.1', NULL),
(9, 'XP_038913557.1', NULL),
(9, 'XP_038913556.1', NULL),
(9, 'XP_038913555.1', NULL),
(9, 'XP_038913554.1', NULL),
(9, 'XP_038913553.1', NULL),
(9, 'XP_038913552.1', NULL),
(9, 'XP_038913551.1', NULL),
(9, 'XP_038913550.1', NULL),
(9, 'XP_038913549.1', NULL),
(9, 'XP_038913548.1', NULL),
(9, 'XP_038913547.1', NULL),
(9, 'XP_038913546.1', NULL),
(9, 'XP_038913545.1', NULL),
(9, 'XP_038913544.1', NULL),
(9, 'XP_038913543.1', NULL),
(9, 'XP_038913542.1', NULL),
(9, 'XP_038913541.1', NULL),
(9, 'XP_038913540.1', NULL),
(9, 'XP_038913539.1', NULL),
(9, 'XP_038913538.1', NULL),
(9, 'XP_038913537.1', NULL),
(9, 'XP_038913536.1', NULL),
(9, 'XP_038913535.1', NULL),
(9, 'XP_038913534.1', NULL),
(9, 'XP_038913533.1', NULL),
(9, 'XP_038913532.1', NULL),
(9, 'XP_038913531.1', NULL),
(9, 'XP_038913530.1', NULL),
(9, 'XP_038913529.1', NULL),
(9, 'XP_038913528.1', NULL),
(9, 'XP_038913527.1', NULL),
(9, 'XP_038913526.1', NULL),
(9, 'XP_038913525.1', NULL),
(9, 'XP_038913524.1', NULL),
(9, 'XP_038913523.1', NULL),
(9, 'XP_038913522.1', NULL),
(9, 'XP_038913521.1', NULL),
(9, 'XP_038913520.1', NULL),
(9, 'XP_038913519.1', NULL),
(9, 'XP_038913518.1', NULL),
(9, 'XP_038913517.1', NULL),
(9, 'XP_038913516.1', NULL),
(9, 'XP_038913515.1', NULL),
(9, 'XP_038913514.1', NULL),
(9, 'XP_038913513.1', NULL),
(9, 'XP_038913512.1', NULL),
(9, 'XP_038913511.1', NULL),
(9, 'XP_038913510.1', NULL),
(9, 'XP_038913509.1', NULL),
(9, 'XP_038913508.1', NULL),
(9, 'XP_038913507.1', NULL),
(9, 'XP_038913506.1', NULL),
(9, 'XP_038913505.1', NULL),
(9, 'XP_038913504.1', NULL),
(9, 'XP_038913503.1', NULL),
(9, 'XP_038913502.1', NULL),
(9, 'XP_038913501.1', NULL),
(9, 'XP_038913500.1', NULL),
(9, 'XP_038913499.1', NULL),
(9, 'XP_038913498.1', NULL),
(9, 'XP_038913497.1', NULL),
(9, 'XP_038913496.1', NULL),
(9, 'XP_038913495.1', NULL),
(9, 'XP_038913494.1', NULL),
(9, 'XP_038913493.1', NULL),
(9, 'XP_038913492.1', NULL),
(9, 'XP_038913491.1', NULL),
(9, 'XP_038913490.1', NULL),
(9, 'XP_038913489.1', NULL),
(9, 'XP_038913488.1', NULL),
(9, 'XP_038913487.1', NULL),
(9, 'XP_038913486.1', NULL),
(9, 'XP_038913485.1', NULL),
(9, 'XP_038913484.1', NULL),
(9, 'XP_038913483.1', NULL),
(9, 'XP_038913482.1', NULL),
(9, 'XP_038913481.1', NULL),
(9, 'XP_038913480.1', NULL),
(9, 'XP_038913479.1', NULL),
(9, 'XP_038913478.1', NULL),
(9, 'XP_038913477.1', NULL),
(9, 'XP_038913476.1', NULL),
(9, 'XP_038913475.1', NULL),
(9, 'XP_038913695.1', NULL),
(9, 'XP_038913694.1', NULL),
(9, 'XP_038913693.1', NULL),
(9, 'XP_038913692.1', NULL),
(9, 'XP_038913691.1', NULL),
(9, 'XP_038913690.1', NULL),
(9, 'XP_038913689.1', NULL),
(9, 'XP_038913688.1', NULL),
(9, 'XP_038913687.1', NULL),
(9, 'XP_038913686.1', NULL),
(9, 'XP_038913685.1', NULL),
(9, 'XP_038913684.1', NULL),
(9, 'XP_038913683.1', NULL),
(9, 'XP_038913682.1', NULL),
(9, 'XP_038913681.1', NULL),
(9, 'XP_038913680.1', NULL),
(9, 'XP_038913679.1', NULL),
(9, 'XP_038913678.1', NULL),
(9, 'XP_038913677.1', NULL),
(9, 'XP_038913676.1', NULL),
(9, 'XP_038913675.1', NULL),
(9, 'XP_038913674.1', NULL),
(9, 'XP_038913673.1', NULL),
(9, 'XP_038913672.1', NULL),
(9, 'XP_038913671.1', NULL),
(9, 'XP_038913670.1', NULL),
(9, 'XP_038913669.1', NULL),
(9, 'XP_038913668.1', NULL),
(9, 'XP_038913667.1', NULL),
(9, 'XP_038913666.1', NULL),
(9, 'XP_038913665.1', NULL),
(9, 'XP_038913664.1', NULL),
(9, 'XP_038913663.1', NULL),
(9, 'XP_038913662.1', NULL),
(9, 'XP_038913661.1', NULL),
(9, 'XP_038913660.1', NULL),
(9, 'XP_038913659.1', NULL),
(9, 'XP_038913658.1', NULL),
(9, 'XP_038913657.1', NULL),
(9, 'XP_038913656.1', NULL),
(9, 'XP_038913655.1', NULL),
(9, 'XP_038913654.1', NULL),
(9, 'XP_038913653.1', NULL),
(9, 'XP_038913652.1', NULL),
(9, 'XP_038913651.1', NULL),
(9, 'XP_038913650.1', NULL),
(9, 'XP_038913649.1', NULL),
(9, 'XP_038913648.1', NULL),
(9, 'XP_038913647.1', NULL),
(9, 'XP_038913646.1', NULL),
(9, 'XP_038913645.1', NULL),
(9, 'XP_038913644.1', NULL),
(9, 'XP_038913643.1', NULL),
(9, 'XP_038913642.1', NULL),
(9, 'XP_038913641.1', NULL),
(9, 'XP_038913640.1', NULL),
(9, 'XP_038913639.1', NULL),
(9, 'XP_038913638.1', NULL),
(9, 'XP_038913637.1', NULL),
(9, 'XP_038913636.1', NULL),
(9, 'XP_038913635.1', NULL),
(9, 'XP_038913634.1', NULL),
(9, 'XP_038913633.1', NULL),
(9, 'XP_038913632.1', NULL),
(9, 'XP_038913631.1', NULL),
(9, 'XP_038913630.1', NULL),
(9, 'XP_038913629.1', NULL),
(9, 'XP_038913628.1', NULL),
(9, 'XP_038913627.1', NULL),
(9, 'XP_038913626.1', NULL),
(9, 'XP_038913625.1', NULL),
(9, 'XP_038913624.1', NULL),
(9, 'XP_038913623.1', NULL),
(9, 'XP_038913622.1', NULL),
(9, 'XP_038913621.1', NULL),
(9, 'XP_038913620.1', NULL),
(9, 'XP_038913619.1', NULL),
(9, 'XP_038913618.1', NULL),
(9, 'XP_038913617.1', NULL),
(9, 'XP_038913616.1', NULL),
(9, 'XP_038913615.1', NULL),
(9, 'XP_038913614.1', NULL),
(9, 'XP_038913613.1', NULL),
(9, 'XP_038913612.1', NULL),
(9, 'XP_038913611.1', NULL),
(9, 'XP_038913610.1', NULL),
(9, 'XP_038913609.1', NULL),
(9, 'XP_038913608.1', NULL),
(9, 'XP_038913607.1', NULL),
(9, 'XP_038913606.1', NULL),
(9, 'XP_038913605.1', NULL),
(9, 'XP_038913604.1', NULL),
(9, 'XP_038913603.1', NULL),
(9, 'XP_038913602.1', NULL),
(9, 'XP_038913601.1', NULL),
(9, 'XP_038913600.1', NULL),
(9, 'XP_038913599.1', NULL),
(9, 'XP_038913598.1', NULL),
(9, 'XP_038913597.1', NULL),
(9, 'XP_038913596.1', NULL),
(9, 'XP_038913595.1', NULL),
(9, 'XP_038913814.1', NULL),
(9, 'XP_038913813.1', NULL),
(9, 'XP_038913812.1', NULL),
(9, 'XP_038913811.1', NULL),
(9, 'XP_038913810.1', NULL),
(9, 'XP_038913809.1', NULL),
(9, 'XP_038913808.1', NULL),
(9, 'XP_038913807.1', NULL),
(9, 'XP_038913806.1', NULL),
(9, 'XP_038913805.1', NULL),
(9, 'XP_038913804.1', NULL),
(9, 'XP_038913803.1', NULL),
(9, 'XP_038913802.1', NULL),
(9, 'XP_038913801.1', NULL),
(9, 'XP_038913800.1', NULL),
(9, 'XP_038913799.1', NULL),
(9, 'XP_038913798.1', NULL),
(9, 'XP_038913797.1', NULL),
(9, 'XP_038913796.1', NULL),
(9, 'XP_038913795.1', NULL),
(9, 'XP_038913794.1', NULL),
(9, 'XP_038913793.1', NULL),
(9, 'XP_038913792.1', NULL),
(9, 'XP_038913791.1', NULL),
(9, 'XP_038913790.1', NULL),
(9, 'XP_038913789.1', NULL),
(9, 'XP_038913788.1', NULL),
(9, 'XP_038913787.1', NULL),
(9, 'XP_038913786.1', NULL),
(9, 'XP_038913785.1', NULL),
(9, 'XP_038913784.1', NULL),
(9, 'XP_038913783.1', NULL),
(9, 'XP_038913782.1', NULL),
(9, 'XP_038913781.1', NULL),
(9, 'XP_038913780.1', NULL),
(9, 'XP_038913779.1', NULL),
(9, 'XP_038913778.1', NULL),
(9, 'XP_038913777.1', NULL),
(9, 'XP_038913776.1', NULL),
(9, 'XP_038913775.1', NULL),
(9, 'XP_038913774.1', NULL),
(9, 'XP_038913773.1', NULL),
(9, 'XP_038913772.1', NULL),
(9, 'XP_038913771.1', NULL),
(9, 'XP_038913770.1', NULL),
(9, 'XP_038913769.1', NULL),
(9, 'XP_038913768.1', NULL),
(9, 'XP_038913767.1', NULL),
(9, 'XP_038913766.1', NULL),
(9, 'XP_038913765.1', NULL),
(9, 'XP_038913764.1', NULL),
(9, 'XP_038913763.1', NULL),
(9, 'XP_038913762.1', NULL),
(9, 'XP_038913761.1', NULL),
(9, 'XP_038913760.1', NULL),
(9, 'XP_038913759.1', NULL),
(9, 'XP_038913758.1', NULL),
(9, 'XP_038913757.1', NULL),
(9, 'XP_038913756.1', NULL),
(9, 'XP_038913755.1', NULL),
(9, 'XP_038913754.1', NULL),
(9, 'XP_038913753.1', NULL),
(9, 'XP_038913752.1', NULL),
(9, 'XP_038913751.1', NULL),
(9, 'XP_038913750.1', NULL),
(9, 'XP_038913749.1', NULL),
(9, 'XP_038913748.1', NULL),
(9, 'XP_038913747.1', NULL),
(9, 'XP_038913746.1', NULL),
(9, 'XP_038913745.1', NULL),
(9, 'XP_038913744.1', NULL),
(9, 'XP_038913743.1', NULL),
(9, 'XP_038913742.1', NULL),
(9, 'XP_038913741.1', NULL),
(9, 'XP_038913740.1', NULL),
(9, 'XP_038913739.1', NULL),
(9, 'XP_038913738.1', NULL),
(9, 'XP_038913737.1', NULL),
(9, 'XP_038913736.1', NULL),
(9, 'XP_038913735.1', NULL),
(9, 'XP_038913734.1', NULL),
(9, 'XP_038913733.1', NULL),
(9, 'XP_038913732.1', NULL),
(9, 'XP_038913731.1', NULL),
(9, 'XP_038913730.1', NULL),
(9, 'XP_038913729.1', NULL),
(9, 'XP_038913728.1', NULL),
(9, 'XP_038913727.1', NULL),
(9, 'XP_038913726.1', NULL),
(9, 'XP_038913725.1', NULL),
(9, 'XP_038913724.1', NULL),
(9, 'XP_038913723.1', NULL),
(9, 'XP_038913722.1', NULL),
(9, 'XP_038913721.1', NULL),
(9, 'XP_038913720.1', NULL),
(9, 'XP_038913719.1', NULL),
(9, 'XP_038913718.1', NULL),
(9, 'XP_038913717.1', NULL),
(9, 'XP_038913716.1', NULL),
(9, 'XP_038913715.1', NULL),
(9, 'XP_038913714.1', NULL),
(9, 'XP_038913713.1', NULL),
(9, 'XP_038913712.1', NULL),
(9, 'XP_038913711.1', NULL),
(9, 'XP_038913710.1', NULL),
(9, 'XP_038913709.1', NULL),
(9, 'XP_038913708.1', NULL),
(9, 'XP_038913707.1', NULL),
(9, 'XP_038913706.1', NULL),
(9, 'XP_038913705.1', NULL),
(9, 'XP_038913704.1', NULL),
(9, 'XP_038913703.1', NULL),
(9, 'XP_038913702.1', NULL),
(9, 'XP_038913701.1', NULL),
(9, 'XP_038913700.1', NULL),
(9, 'XP_038913699.1', NULL),
(9, 'XP_038913698.1', NULL),
(9, 'XP_038913697.1', NULL),
(9, 'XP_038913696.1', NULL),
(9, 'XP_038913930.1', NULL),
(9, 'XP_038913929.1', NULL),
(9, 'XP_038913928.1', NULL),
(9, 'XP_038913927.1', NULL),
(9, 'XP_038913926.1', NULL),
(9, 'XP_038913925.1', NULL),
(9, 'XP_038913924.1', NULL),
(9, 'XP_038913923.1', NULL),
(9, 'XP_038913922.1', NULL),
(9, 'XP_038913921.1', NULL),
(9, 'XP_038913920.1', NULL),
(9, 'XP_038913919.1', NULL),
(9, 'XP_038913918.1', NULL),
(9, 'XP_038913917.1', NULL),
(9, 'XP_038913916.1', NULL),
(9, 'XP_038913915.1', NULL),
(9, 'XP_038913914.1', NULL),
(9, 'XP_038913913.1', NULL),
(9, 'XP_038913912.1', NULL),
(9, 'XP_038913911.1', NULL),
(9, 'XP_038913910.1', NULL),
(9, 'XP_038913909.1', NULL),
(9, 'XP_038913908.1', NULL),
(9, 'XP_038913907.1', NULL),
(9, 'XP_038913906.1', NULL),
(9, 'XP_038913905.1', NULL),
(9, 'XP_038913904.1', NULL),
(9, 'XP_038913903.1', NULL),
(9, 'XP_038913902.1', NULL),
(9, 'XP_038913901.1', NULL),
(9, 'XP_038913900.1', NULL),
(9, 'XP_038913899.1', NULL),
(9, 'XP_038913898.1', NULL),
(9, 'XP_038913897.1', NULL),
(9, 'XP_038913896.1', NULL),
(9, 'XP_038913895.1', NULL),
(9, 'XP_038913894.1', NULL),
(9, 'XP_038913893.1', NULL),
(9, 'XP_038913892.1', NULL),
(9, 'XP_038913891.1', NULL),
(9, 'XP_038913890.1', NULL),
(9, 'XP_038913889.1', NULL),
(9, 'XP_038913888.1', NULL),
(9, 'XP_038913887.1', NULL),
(9, 'XP_038913886.1', NULL),
(9, 'XP_038913885.1', NULL),
(9, 'XP_038913884.1', NULL),
(9, 'XP_038913883.1', NULL),
(9, 'XP_038913882.1', NULL),
(9, 'XP_038913881.1', NULL),
(9, 'XP_038913880.1', NULL),
(9, 'XP_038913879.1', NULL),
(9, 'XP_038913878.1', NULL),
(9, 'XP_038913877.1', NULL),
(9, 'XP_038913876.1', NULL),
(9, 'XP_038913875.1', NULL),
(9, 'XP_038913874.1', NULL),
(9, 'XP_038913873.1', NULL),
(9, 'XP_038913872.1', NULL),
(9, 'XP_038913871.1', NULL),
(9, 'XP_038913870.1', NULL),
(9, 'XP_038913869.1', NULL),
(9, 'XP_038913868.1', NULL),
(9, 'XP_038913867.1', NULL),
(9, 'XP_038913866.1', NULL),
(9, 'XP_038913865.1', NULL),
(9, 'XP_038913864.1', NULL),
(9, 'XP_038913863.1', NULL),
(9, 'XP_038913862.1', NULL),
(9, 'XP_038913861.1', NULL),
(9, 'XP_038913860.1', NULL),
(9, 'XP_038913859.1', NULL),
(9, 'XP_038913858.1', NULL),
(9, 'XP_038913857.1', NULL),
(9, 'XP_038913856.1', NULL),
(9, 'XP_038913855.1', NULL),
(9, 'XP_038913854.1', NULL),
(9, 'XP_038913853.1', NULL),
(9, 'XP_038913852.1', NULL),
(9, 'XP_038913851.1', NULL),
(9, 'XP_038913850.1', NULL),
(9, 'XP_038913849.1', NULL),
(9, 'XP_038913848.1', NULL),
(9, 'XP_038913847.1', NULL),
(9, 'XP_038913846.1', NULL),
(9, 'XP_038913845.1', NULL),
(9, 'XP_038913844.1', NULL),
(9, 'XP_038913843.1', NULL),
(9, 'XP_038913842.1', NULL),
(9, 'XP_038913841.1', NULL),
(9, 'XP_038913840.1', NULL),
(9, 'XP_038913839.1', NULL),
(9, 'XP_038913838.1', NULL),
(9, 'XP_038913837.1', NULL),
(9, 'XP_038913836.1', NULL),
(9, 'XP_038913835.1', NULL),
(9, 'XP_038913834.1', NULL),
(9, 'XP_038913833.1', NULL),
(9, 'XP_038913832.1', NULL),
(9, 'XP_038913831.1', NULL),
(9, 'XP_038913830.1', NULL),
(9, 'XP_038913829.1', NULL),
(9, 'XP_038913828.1', NULL),
(9, 'XP_038913827.1', NULL),
(9, 'XP_038913826.1', NULL),
(9, 'XP_038913825.1', NULL),
(9, 'XP_038913824.1', NULL),
(9, 'XP_038913823.1', NULL),
(9, 'XP_038913822.1', NULL),
(9, 'XP_038913821.1', NULL),
(9, 'XP_038913820.1', NULL),
(9, 'XP_038913819.1', NULL),
(9, 'XP_038913818.1', NULL),
(9, 'XP_038913817.1', NULL),
(9, 'XP_038913816.1', NULL),
(9, 'XP_038913815.1', NULL),
(9, 'XP_038914148.1', NULL),
(9, 'XP_038914147.1', NULL),
(9, 'XP_038914146.1', NULL),
(9, 'XP_038914145.1', NULL),
(9, 'XP_038914144.1', NULL),
(9, 'XP_038914143.1', NULL),
(9, 'XP_038914142.1', NULL),
(9, 'XP_038914141.1', NULL),
(9, 'XP_038914140.1', NULL),
(9, 'XP_038914139.1', NULL),
(9, 'XP_038914138.1', NULL),
(9, 'XP_038914137.1', NULL),
(9, 'XP_038914136.1', NULL),
(9, 'XP_038914135.1', NULL),
(9, 'XP_038914134.1', NULL),
(9, 'XP_038914133.1', NULL),
(9, 'XP_038914132.1', NULL),
(9, 'XP_038914131.1', NULL),
(9, 'XP_038914130.1', NULL),
(9, 'XP_038914129.1', NULL),
(9, 'XP_038914128.1', NULL),
(9, 'XP_038914127.1', NULL),
(9, 'XP_038914126.1', NULL),
(9, 'XP_038914125.1', NULL),
(9, 'XP_038914124.1', NULL),
(9, 'XP_038914123.1', NULL),
(9, 'XP_038914122.1', NULL),
(9, 'XP_038914121.1', NULL),
(9, 'XP_038914120.1', NULL),
(9, 'XP_038914119.1', NULL),
(9, 'XP_038914118.1', NULL),
(9, 'XP_038914117.1', NULL),
(9, 'XP_038914116.1', NULL),
(9, 'XP_038914115.1', NULL),
(9, 'XP_038914114.1', NULL),
(9, 'XP_038914113.1', NULL),
(9, 'XP_038914112.1', NULL),
(9, 'XP_038914111.1', NULL),
(9, 'XP_038914110.1', NULL),
(9, 'XP_038914109.1', NULL),
(9, 'XP_038914108.1', NULL),
(9, 'XP_038914107.1', NULL),
(9, 'XP_038914106.1', NULL),
(9, 'XP_038914105.1', NULL),
(9, 'XP_038914104.1', NULL),
(9, 'XP_038914103.1', NULL),
(9, 'XP_038914102.1', NULL),
(9, 'XP_038914101.1', NULL),
(9, 'XP_038914100.1', NULL),
(9, 'XP_038914099.1', NULL),
(9, 'XP_038914098.1', NULL),
(9, 'XP_038914097.1', NULL),
(9, 'XP_038914096.1', NULL),
(9, 'XP_038914095.1', NULL),
(9, 'XP_038914094.1', NULL),
(9, 'XP_038914093.1', NULL),
(9, 'XP_038914092.1', NULL),
(9, 'XP_038914091.1', NULL),
(9, 'XP_038914090.1', NULL),
(9, 'XP_038914089.1', NULL),
(9, 'XP_038914088.1', NULL),
(9, 'XP_038914087.1', NULL),
(9, 'XP_038914086.1', NULL),
(9, 'XP_038914085.1', NULL),
(9, 'XP_038914084.1', NULL),
(9, 'XP_038914083.1', NULL),
(9, 'XP_038914082.1', NULL),
(9, 'XP_038914081.1', NULL),
(9, 'XP_038914080.1', NULL),
(9, 'XP_038914079.1', NULL),
(9, 'XP_038914078.1', NULL),
(9, 'XP_038914077.1', NULL),
(9, 'XP_038914076.1', NULL),
(9, 'XP_038914075.1', NULL),
(9, 'XP_038914074.1', NULL),
(9, 'XP_038914073.1', NULL),
(9, 'XP_038914072.1', NULL),
(9, 'XP_038914071.1', NULL),
(9, 'XP_038914070.1', NULL),
(9, 'XP_038914069.1', NULL),
(9, 'XP_038914068.1', NULL),
(9, 'XP_038914067.1', NULL),
(9, 'XP_038914066.1', NULL),
(9, 'XP_038914065.1', NULL),
(9, 'XP_038914064.1', NULL),
(9, 'XP_038914063.1', NULL),
(9, 'XP_038914062.1', NULL),
(9, 'XP_038914061.1', NULL),
(9, 'XP_038914060.1', NULL),
(9, 'XP_038914059.1', NULL),
(9, 'XP_038914058.1', NULL),
(9, 'XP_038914057.1', NULL),
(9, 'XP_038914056.1', NULL),
(9, 'XP_038914055.1', NULL),
(9, 'XP_038914054.1', NULL),
(9, 'XP_038914053.1', NULL),
(9, 'XP_038914052.1', NULL),
(9, 'XP_038914051.1', NULL),
(9, 'XP_038914050.1', NULL),
(9, 'XP_038914049.1', NULL),
(9, 'XP_038914048.1', NULL),
(9, 'XP_038914047.1', NULL),
(9, 'XP_038914046.1', NULL),
(9, 'XP_038914045.1', NULL),
(9, 'XP_038914044.1', NULL),
(9, 'XP_038914043.1', NULL),
(9, 'XP_038914042.1', NULL),
(9, 'XP_038914041.1', NULL),
(9, 'XP_038914040.1', NULL),
(9, 'XP_038914039.1', NULL),
(9, 'XP_038914038.1', NULL),
(9, 'XP_038914037.1', NULL),
(9, 'XP_038914036.1', NULL),
(9, 'XP_038914035.1', NULL),
(9, 'XP_038914034.1', NULL),
(9, 'XP_038914033.1', NULL),
(9, 'XP_038914032.1', NULL),
(9, 'XP_038914031.1', NULL),
(9, 'XP_038914030.1', NULL),
(9, 'XP_038914029.1', NULL),
(9, 'XP_038914028.1', NULL),
(9, 'XP_038914027.1', NULL),
(9, 'XP_038914026.1', NULL),
(9, 'XP_038914025.1', NULL),
(9, 'XP_038914024.1', NULL),
(9, 'XP_038914023.1', NULL),
(9, 'XP_038914022.1', NULL),
(9, 'XP_038914021.1', NULL),
(9, 'XP_038914020.1', NULL),
(9, 'XP_038914019.1', NULL),
(9, 'XP_038914018.1', NULL),
(9, 'XP_038914017.1', NULL),
(9, 'XP_038914016.1', NULL),
(9, 'XP_038914015.1', NULL),
(9, 'XP_038914014.1', NULL),
(9, 'XP_038914013.1', NULL),
(9, 'XP_038914012.1', NULL),
(9, 'XP_038914011.1', NULL),
(9, 'XP_038914010.1', NULL),
(9, 'XP_038914009.1', NULL),
(9, 'XP_038914008.1', NULL),
(9, 'XP_038914007.1', NULL),
(9, 'XP_038914006.1', NULL),
(9, 'XP_038914005.1', NULL),
(9, 'XP_038914004.1', NULL),
(9, 'XP_038914003.1', NULL),
(9, 'XP_038914002.1', NULL),
(9, 'XP_038914001.1', NULL),
(9, 'XP_038914000.1', NULL),
(9, 'XP_038913999.1', NULL),
(9, 'XP_038913998.1', NULL),
(9, 'XP_038913997.1', NULL),
(9, 'XP_038913996.1', NULL),
(9, 'XP_038913995.1', NULL),
(9, 'XP_038913994.1', NULL),
(9, 'XP_038913993.1', NULL),
(9, 'XP_038913992.1', NULL),
(9, 'XP_038913991.1', NULL),
(9, 'XP_038913990.1', NULL),
(9, 'XP_038913989.1', NULL),
(9, 'XP_038913988.1', NULL),
(9, 'XP_038913987.1', NULL),
(9, 'XP_038913986.1', NULL),
(9, 'XP_038913985.1', NULL),
(9, 'XP_038913984.1', NULL),
(9, 'XP_038913983.1', NULL),
(9, 'XP_038913982.1', NULL),
(9, 'XP_038913981.1', NULL),
(9, 'XP_038913980.1', NULL),
(9, 'XP_038913979.1', NULL),
(9, 'XP_038913978.1', NULL),
(9, 'XP_038913977.1', NULL),
(9, 'XP_038913976.1', NULL),
(9, 'XP_038913975.1', NULL),
(9, 'XP_038913974.1', NULL),
(9, 'XP_038913973.1', NULL),
(9, 'XP_038913972.1', NULL),
(9, 'XP_038913971.1', NULL),
(9, 'XP_038913970.1', NULL),
(9, 'XP_038913969.1', NULL),
(9, 'XP_038913968.1', NULL),
(9, 'XP_038913967.1', NULL),
(9, 'XP_038913966.1', NULL),
(9, 'XP_038913965.1', NULL),
(9, 'XP_038913964.1', NULL),
(9, 'XP_038913963.1', NULL),
(9, 'XP_038913962.1', NULL),
(9, 'XP_038913961.1', NULL),
(9, 'XP_038913960.1', NULL),
(9, 'XP_038913959.1', NULL),
(9, 'XP_038913958.1', NULL),
(9, 'XP_038913957.1', NULL),
(9, 'XP_038913956.1', NULL),
(9, 'XP_038913955.1', NULL),
(9, 'XP_038913954.1', NULL),
(9, 'XP_038913953.1', NULL),
(9, 'XP_038913952.1', NULL),
(9, 'XP_038913951.1', NULL),
(9, 'XP_038913950.1', NULL),
(9, 'XP_038913949.1', NULL),
(9, 'XP_038913948.1', NULL),
(9, 'XP_038913947.1', NULL),
(9, 'XP_038913946.1', NULL),
(9, 'XP_038913945.1', NULL),
(9, 'XP_038913944.1', NULL),
(9, 'XP_038913943.1', NULL),
(9, 'XP_038913942.1', NULL),
(9, 'XP_038913941.1', NULL),
(9, 'XP_038913940.1', NULL),
(9, 'XP_038913939.1', NULL),
(9, 'XP_038913938.1', NULL),
(9, 'XP_038913937.1', NULL),
(9, 'XP_038913936.1', NULL),
(9, 'XP_038913935.1', NULL),
(9, 'XP_038913934.1', NULL),
(9, 'XP_038913933.1', NULL),
(9, 'XP_038913932.1', NULL),
(9, 'XP_038913931.1', NULL),
(9, 'XP_038914277.1', NULL),
(9, 'XP_038914276.1', NULL),
(9, 'XP_038914275.1', NULL),
(9, 'XP_038914274.1', NULL),
(9, 'XP_038914273.1', NULL),
(9, 'XP_038914272.1', NULL),
(9, 'XP_038914271.1', NULL),
(9, 'XP_038914270.1', NULL),
(9, 'XP_038914269.1', NULL),
(9, 'XP_038914268.1', NULL),
(9, 'XP_038914267.1', NULL),
(9, 'XP_038914266.1', NULL),
(9, 'XP_038914265.1', NULL),
(9, 'XP_038914264.1', NULL),
(9, 'XP_038914263.1', NULL),
(9, 'XP_038914262.1', NULL),
(9, 'XP_038914261.1', NULL),
(9, 'XP_038914260.1', NULL),
(9, 'XP_038914259.1', NULL),
(9, 'XP_038914258.1', NULL),
(9, 'XP_038914257.1', NULL),
(9, 'XP_038914256.1', NULL),
(9, 'XP_038914255.1', NULL),
(9, 'XP_038914254.1', NULL),
(9, 'XP_038914253.1', NULL),
(9, 'XP_038914252.1', NULL),
(9, 'XP_038914251.1', NULL),
(9, 'XP_038914250.1', NULL),
(9, 'XP_038914249.1', NULL),
(9, 'XP_038914248.1', NULL),
(9, 'XP_038914247.1', NULL),
(9, 'XP_038914246.1', NULL),
(9, 'XP_038914245.1', NULL),
(9, 'XP_038914244.1', NULL),
(9, 'XP_038914243.1', NULL),
(9, 'XP_038914242.1', NULL),
(9, 'XP_038914241.1', NULL),
(9, 'XP_038914240.1', NULL),
(9, 'XP_038914239.1', NULL),
(9, 'XP_038914238.1', NULL),
(9, 'XP_038914237.1', NULL),
(9, 'XP_038914236.1', NULL),
(9, 'XP_038914235.1', NULL),
(9, 'XP_038914234.1', NULL),
(9, 'XP_038914233.1', NULL),
(9, 'XP_038914232.1', NULL),
(9, 'XP_038914231.1', NULL),
(9, 'XP_038914230.1', NULL),
(9, 'XP_038914229.1', NULL),
(9, 'XP_038914228.1', NULL),
(9, 'XP_038914227.1', NULL),
(9, 'XP_038914226.1', NULL),
(9, 'XP_038914225.1', NULL),
(9, 'XP_038914224.1', NULL),
(9, 'XP_038914223.1', NULL),
(9, 'XP_038914222.1', NULL),
(9, 'XP_038914221.1', NULL),
(9, 'XP_038914220.1', NULL),
(9, 'XP_038914219.1', NULL),
(9, 'XP_038914218.1', NULL),
(9, 'XP_038914217.1', NULL),
(9, 'XP_038914216.1', NULL),
(9, 'XP_038914215.1', NULL),
(9, 'XP_038914214.1', NULL),
(9, 'XP_038914213.1', NULL),
(9, 'XP_038914212.1', NULL),
(9, 'XP_038914211.1', NULL),
(9, 'XP_038914210.1', NULL),
(9, 'XP_038914209.1', NULL),
(9, 'XP_038914208.1', NULL),
(9, 'XP_038914207.1', NULL),
(9, 'XP_038914206.1', NULL),
(9, 'XP_038914205.1', NULL),
(9, 'XP_038914204.1', NULL),
(9, 'XP_038914203.1', NULL),
(9, 'XP_038914202.1', NULL),
(9, 'XP_038914201.1', NULL),
(9, 'XP_038914200.1', NULL),
(9, 'XP_038914199.1', NULL),
(9, 'XP_038914198.1', NULL),
(9, 'XP_038914197.1', NULL),
(9, 'XP_038914196.1', NULL),
(9, 'XP_038914195.1', NULL),
(9, 'XP_038914194.1', NULL),
(9, 'XP_038914193.1', NULL),
(9, 'XP_038914192.1', NULL),
(9, 'XP_038914191.1', NULL),
(9, 'XP_038914190.1', NULL),
(9, 'XP_038914189.1', NULL),
(9, 'XP_038914188.1', NULL),
(9, 'XP_038914187.1', NULL),
(9, 'XP_038914186.1', NULL),
(9, 'XP_038914185.1', NULL),
(9, 'XP_038914184.1', NULL),
(9, 'XP_038914183.1', NULL),
(9, 'XP_038914182.1', NULL),
(9, 'XP_038914181.1', NULL),
(9, 'XP_038914180.1', NULL),
(9, 'XP_038914179.1', NULL),
(9, 'XP_038914178.1', NULL),
(9, 'XP_038914177.1', NULL),
(9, 'XP_038914176.1', NULL),
(9, 'XP_038914175.1', NULL),
(9, 'XP_038914174.1', NULL),
(9, 'XP_038914173.1', NULL),
(9, 'XP_038914172.1', NULL),
(9, 'XP_038914171.1', NULL),
(9, 'XP_038914170.1', NULL),
(9, 'XP_038914169.1', NULL),
(9, 'XP_038914168.1', NULL),
(9, 'XP_038914167.1', NULL),
(9, 'XP_038914166.1', NULL),
(9, 'XP_038914165.1', NULL),
(9, 'XP_038914164.1', NULL),
(9, 'XP_038914163.1', NULL),
(9, 'XP_038914162.1', NULL),
(9, 'XP_038914161.1', NULL),
(9, 'XP_038914160.1', NULL),
(9, 'XP_038914159.1', NULL),
(9, 'XP_038914158.1', NULL),
(9, 'XP_038914157.1', NULL),
(9, 'XP_038914156.1', NULL),
(9, 'XP_038914155.1', NULL),
(9, 'XP_038914154.1', NULL),
(9, 'XP_038914153.1', NULL),
(9, 'XP_038914152.1', NULL),
(9, 'XP_038914151.1', NULL),
(9, 'XP_038914150.1', NULL),
(9, 'XP_038914149.1', NULL),
(9, 'XP_038914382.1', NULL),
(9, 'XP_038914381.1', NULL),
(9, 'XP_038914380.1', NULL),
(9, 'XP_038914379.1', NULL),
(9, 'XP_038914378.1', NULL),
(9, 'XP_038914377.1', NULL),
(9, 'XP_038914376.1', NULL),
(9, 'XP_038914375.1', NULL),
(9, 'XP_038914374.1', NULL),
(9, 'XP_038914373.1', NULL),
(9, 'XP_038914372.1', NULL),
(9, 'XP_038914371.1', NULL),
(9, 'XP_038914370.1', NULL),
(9, 'XP_038914369.1', NULL),
(9, 'XP_038914368.1', NULL),
(9, 'XP_038914367.1', NULL),
(9, 'XP_038914366.1', NULL),
(9, 'XP_038914365.1', NULL),
(9, 'XP_038914364.1', NULL),
(9, 'XP_038914363.1', NULL),
(9, 'XP_038914362.1', NULL),
(9, 'XP_038914361.1', NULL),
(9, 'XP_038914360.1', NULL),
(9, 'XP_038914359.1', NULL),
(9, 'XP_038914358.1', NULL),
(9, 'XP_038914357.1', NULL),
(9, 'XP_038914356.1', NULL),
(9, 'XP_038914355.1', NULL),
(9, 'XP_038914354.1', NULL),
(9, 'XP_038914353.1', NULL),
(9, 'XP_038914352.1', NULL),
(9, 'XP_038914351.1', NULL),
(9, 'XP_038914350.1', NULL),
(9, 'XP_038914349.1', NULL),
(9, 'XP_038914348.1', NULL),
(9, 'XP_038914347.1', NULL),
(9, 'XP_038914346.1', NULL),
(9, 'XP_038914345.1', NULL),
(9, 'XP_038914344.1', NULL),
(9, 'XP_038914343.1', NULL),
(9, 'XP_038914342.1', NULL),
(9, 'XP_038914341.1', NULL),
(9, 'XP_038914340.1', NULL),
(9, 'XP_038914339.1', NULL),
(9, 'XP_038914338.1', NULL),
(9, 'XP_038914337.1', NULL),
(9, 'XP_038914336.1', NULL),
(9, 'XP_038914335.1', NULL),
(9, 'XP_038914334.1', NULL),
(9, 'XP_038914333.1', NULL),
(9, 'XP_038914332.1', NULL),
(9, 'XP_038914331.1', NULL),
(9, 'XP_038914330.1', NULL),
(9, 'XP_038914329.1', NULL),
(9, 'XP_038914328.1', NULL),
(9, 'XP_038914327.1', NULL),
(9, 'XP_038914326.1', NULL),
(9, 'XP_038914325.1', NULL),
(9, 'XP_038914324.1', NULL),
(9, 'XP_038914323.1', NULL),
(9, 'XP_038914322.1', NULL),
(9, 'XP_038914321.1', NULL),
(9, 'XP_038914320.1', NULL),
(9, 'XP_038914319.1', NULL),
(9, 'XP_038914318.1', NULL),
(9, 'XP_038914317.1', NULL),
(9, 'XP_038914316.1', NULL),
(9, 'XP_038914315.1', NULL),
(9, 'XP_038914314.1', NULL),
(9, 'XP_038914313.1', NULL),
(9, 'XP_038914312.1', NULL),
(9, 'XP_038914311.1', NULL),
(9, 'XP_038914310.1', NULL),
(9, 'XP_038914309.1', NULL),
(9, 'XP_038914308.1', NULL),
(9, 'XP_038914307.1', NULL),
(9, 'XP_038914306.1', NULL),
(9, 'XP_038914305.1', NULL),
(9, 'XP_038914304.1', NULL),
(9, 'XP_038914303.1', NULL),
(9, 'XP_038914302.1', NULL),
(9, 'XP_038914301.1', NULL),
(9, 'XP_038914300.1', NULL),
(9, 'XP_038914299.1', NULL),
(9, 'XP_038914298.1', NULL),
(9, 'XP_038914297.1', NULL),
(9, 'XP_038914296.1', NULL),
(9, 'XP_038914295.1', NULL),
(9, 'XP_038914294.1', NULL),
(9, 'XP_038914293.1', NULL),
(9, 'XP_038914292.1', NULL),
(9, 'XP_038914291.1', NULL),
(9, 'XP_038914290.1', NULL),
(9, 'XP_038914289.1', NULL),
(9, 'XP_038914288.1', NULL),
(9, 'XP_038914287.1', NULL),
(9, 'XP_038914286.1', NULL),
(9, 'XP_038914285.1', NULL),
(9, 'XP_038914284.1', NULL),
(9, 'XP_038914283.1', NULL),
(9, 'XP_038914282.1', NULL),
(9, 'XP_038914281.1', NULL),
(9, 'XP_038914280.1', NULL),
(9, 'XP_038914279.1', NULL),
(9, 'XP_038914278.1', NULL),
(9, 'XP_038914463.1', NULL),
(9, 'XP_038914462.1', NULL),
(9, 'XP_038914461.1', NULL),
(9, 'XP_038914460.1', NULL),
(9, 'XP_038914459.1', NULL),
(9, 'XP_038914458.1', NULL),
(9, 'XP_038914457.1', NULL),
(9, 'XP_038914456.1', NULL),
(9, 'XP_038914455.1', NULL),
(9, 'XP_038914454.1', NULL),
(9, 'XP_038914453.1', NULL),
(9, 'XP_038914452.1', NULL),
(9, 'XP_038914451.1', NULL),
(9, 'XP_038914450.1', NULL),
(9, 'XP_038914449.1', NULL),
(9, 'XP_038914448.1', NULL),
(9, 'XP_038914447.1', NULL),
(9, 'XP_038914446.1', NULL),
(9, 'XP_038914445.1', NULL),
(9, 'XP_038914444.1', NULL),
(9, 'XP_038914443.1', NULL),
(9, 'XP_038914442.1', NULL),
(9, 'XP_038914441.1', NULL),
(9, 'XP_038914440.1', NULL),
(9, 'XP_038914439.1', NULL),
(9, 'XP_038914438.1', NULL),
(9, 'XP_038914437.1', NULL),
(9, 'XP_038914436.1', NULL),
(9, 'XP_038914435.1', NULL),
(9, 'XP_038914434.1', NULL),
(9, 'XP_038914433.1', NULL),
(9, 'XP_038914432.1', NULL),
(9, 'XP_038914431.1', NULL),
(9, 'XP_038914430.1', NULL),
(9, 'XP_038914429.1', NULL),
(9, 'XP_038914428.1', NULL),
(9, 'XP_038914427.1', NULL),
(9, 'XP_038914426.1', NULL),
(9, 'XP_038914425.1', NULL),
(9, 'XP_038914424.1', NULL),
(9, 'XP_038914423.1', NULL),
(9, 'XP_038914422.1', NULL),
(9, 'XP_038914421.1', NULL),
(9, 'XP_038914420.1', NULL),
(9, 'XP_038914419.1', NULL),
(9, 'XP_038914418.1', NULL),
(9, 'XP_038914417.1', NULL),
(9, 'XP_038914416.1', NULL),
(9, 'XP_038914415.1', NULL),
(9, 'XP_038914414.1', NULL),
(9, 'XP_038914413.1', NULL),
(9, 'XP_038914412.1', NULL),
(9, 'XP_038914411.1', NULL),
(9, 'XP_038914410.1', NULL),
(9, 'XP_038914409.1', NULL),
(9, 'XP_038914408.1', NULL),
(9, 'XP_038914407.1', NULL),
(9, 'XP_038914406.1', NULL),
(9, 'XP_038914405.1', NULL),
(9, 'XP_038914404.1', NULL),
(9, 'XP_038914403.1', NULL),
(9, 'XP_038914402.1', NULL),
(9, 'XP_038914401.1', NULL),
(9, 'XP_038914400.1', NULL),
(9, 'XP_038914399.1', NULL),
(9, 'XP_038914398.1', NULL),
(9, 'XP_038914397.1', NULL),
(9, 'XP_038914396.1', NULL),
(9, 'XP_038914395.1', NULL),
(9, 'XP_038914394.1', NULL),
(9, 'XP_038914393.1', NULL),
(9, 'XP_038914392.1', NULL),
(9, 'XP_038914391.1', NULL),
(9, 'XP_038914390.1', NULL),
(9, 'XP_038914389.1', NULL),
(9, 'XP_038914388.1', NULL),
(9, 'XP_038914387.1', NULL),
(9, 'XP_038914386.1', NULL),
(9, 'XP_038914385.1', NULL),
(9, 'XP_038914384.1', NULL),
(9, 'XP_038914383.1', NULL),
(9, 'XP_038914623.1', NULL),
(9, 'XP_038914622.1', NULL),
(9, 'XP_038914621.1', NULL),
(9, 'XP_038914620.1', NULL),
(9, 'XP_038914619.1', NULL),
(9, 'XP_038914618.1', NULL),
(9, 'XP_038914617.1', NULL),
(9, 'XP_038914616.1', NULL),
(9, 'XP_038914615.1', NULL),
(9, 'XP_038914614.1', NULL),
(9, 'XP_038914613.1', NULL),
(9, 'XP_038914612.1', NULL),
(9, 'XP_038914611.1', NULL),
(9, 'XP_038914610.1', NULL),
(9, 'XP_038914609.1', NULL),
(9, 'XP_038914608.1', NULL),
(9, 'XP_038914607.1', NULL),
(9, 'XP_038914606.1', NULL),
(9, 'XP_038914605.1', NULL),
(9, 'XP_038914604.1', NULL),
(9, 'XP_038914603.1', NULL),
(9, 'XP_038914602.1', NULL),
(9, 'XP_038914601.1', NULL),
(9, 'XP_038914600.1', NULL),
(9, 'XP_038914599.1', NULL),
(9, 'XP_038914598.1', NULL),
(9, 'XP_038914597.1', NULL),
(9, 'XP_038914596.1', NULL),
(9, 'XP_038914595.1', NULL),
(9, 'XP_038914594.1', NULL),
(9, 'XP_038914593.1', NULL),
(9, 'XP_038914592.1', NULL),
(9, 'XP_038914591.1', NULL),
(9, 'XP_038914590.1', NULL),
(9, 'XP_038914589.1', NULL),
(9, 'XP_038914588.1', NULL),
(9, 'XP_038914587.1', NULL),
(9, 'XP_038914586.1', NULL),
(9, 'XP_038914585.1', NULL),
(9, 'XP_038914584.1', NULL),
(9, 'XP_038914583.1', NULL),
(9, 'XP_038914582.1', NULL),
(9, 'XP_038914581.1', NULL),
(9, 'XP_038914580.1', NULL),
(9, 'XP_038914579.1', NULL),
(9, 'XP_038914578.1', NULL),
(9, 'XP_038914577.1', NULL),
(9, 'XP_038914576.1', NULL),
(9, 'XP_038914575.1', NULL),
(9, 'XP_038914574.1', NULL),
(9, 'XP_038914573.1', NULL),
(9, 'XP_038914572.1', NULL),
(9, 'XP_038914571.1', NULL),
(9, 'XP_038914570.1', NULL),
(9, 'XP_038914569.1', NULL),
(9, 'XP_038914568.1', NULL),
(9, 'XP_038914567.1', NULL),
(9, 'XP_038914566.1', NULL),
(9, 'XP_038914565.1', NULL),
(9, 'XP_038914564.1', NULL),
(9, 'XP_038914563.1', NULL),
(9, 'XP_038914562.1', NULL),
(9, 'XP_038914561.1', NULL),
(9, 'XP_038914560.1', NULL),
(9, 'XP_038914559.1', NULL),
(9, 'XP_038914558.1', NULL),
(9, 'XP_038914557.1', NULL),
(9, 'XP_038914556.1', NULL),
(9, 'XP_038914555.1', NULL),
(9, 'XP_038914554.1', NULL),
(9, 'XP_038914553.1', NULL),
(9, 'XP_038914552.1', NULL),
(9, 'XP_038914551.1', NULL),
(9, 'XP_038914550.1', NULL),
(9, 'XP_038914549.1', NULL),
(9, 'XP_038914548.1', NULL),
(9, 'XP_038914547.1', NULL),
(9, 'XP_038914546.1', NULL),
(9, 'XP_038914545.1', NULL),
(9, 'XP_038914544.1', NULL),
(9, 'XP_038914543.1', NULL),
(9, 'XP_038914542.1', NULL),
(9, 'XP_038914541.1', NULL),
(9, 'XP_038914540.1', NULL),
(9, 'XP_038914539.1', NULL),
(9, 'XP_038914538.1', NULL),
(9, 'XP_038914537.1', NULL),
(9, 'XP_038914536.1', NULL),
(9, 'XP_038914535.1', NULL),
(9, 'XP_038914534.1', NULL),
(9, 'XP_038914533.1', NULL),
(9, 'XP_038914532.1', NULL),
(9, 'XP_038914531.1', NULL),
(9, 'XP_038914530.1', NULL),
(9, 'XP_038914529.1', NULL),
(9, 'XP_038914528.1', NULL),
(9, 'XP_038914527.1', NULL),
(9, 'XP_038914526.1', NULL),
(9, 'XP_038914525.1', NULL),
(9, 'XP_038914524.1', NULL),
(9, 'XP_038914523.1', NULL),
(9, 'XP_038914522.1', NULL),
(9, 'XP_038914521.1', NULL),
(9, 'XP_038914520.1', NULL),
(9, 'XP_038914519.1', NULL),
(9, 'XP_038914518.1', NULL),
(9, 'XP_038914517.1', NULL),
(9, 'XP_038914516.1', NULL),
(9, 'XP_038914515.1', NULL),
(9, 'XP_038914514.1', NULL),
(9, 'XP_038914513.1', NULL),
(9, 'XP_038914512.1', NULL),
(9, 'XP_038914511.1', NULL),
(9, 'XP_038914510.1', NULL),
(9, 'XP_038914509.1', NULL),
(9, 'XP_038914508.1', NULL),
(9, 'XP_038914507.1', NULL),
(9, 'XP_038914506.1', NULL),
(9, 'XP_038914505.1', NULL),
(9, 'XP_038914504.1', NULL),
(9, 'XP_038914503.1', NULL),
(9, 'XP_038914502.1', NULL),
(9, 'XP_038914501.1', NULL),
(9, 'XP_038914500.1', NULL),
(9, 'XP_038914499.1', NULL),
(9, 'XP_038914498.1', NULL),
(9, 'XP_038914497.1', NULL),
(9, 'XP_038914496.1', NULL),
(9, 'XP_038914495.1', NULL),
(9, 'XP_038914494.1', NULL),
(9, 'XP_038914493.1', NULL),
(9, 'XP_038914492.1', NULL),
(9, 'XP_038914491.1', NULL),
(9, 'XP_038914490.1', NULL),
(9, 'XP_038914489.1', NULL),
(9, 'XP_038914488.1', NULL),
(9, 'XP_038914487.1', NULL),
(9, 'XP_038914486.1', NULL),
(9, 'XP_038914485.1', NULL),
(9, 'XP_038914484.1', NULL),
(9, 'XP_038914483.1', NULL),
(9, 'XP_038914482.1', NULL),
(9, 'XP_038914481.1', NULL),
(9, 'XP_038914480.1', NULL),
(9, 'XP_038914479.1', NULL),
(9, 'XP_038914478.1', NULL),
(9, 'XP_038914477.1', NULL),
(9, 'XP_038914476.1', NULL),
(9, 'XP_038914475.1', NULL),
(9, 'XP_038914474.1', NULL),
(9, 'XP_038914473.1', NULL),
(9, 'XP_038914472.1', NULL),
(9, 'XP_038914471.1', NULL),
(9, 'XP_038914470.1', NULL),
(9, 'XP_038914469.1', NULL),
(9, 'XP_038914468.1', NULL),
(9, 'XP_038914467.1', NULL),
(9, 'XP_038914466.1', NULL),
(9, 'XP_038914465.1', NULL),
(9, 'XP_038914464.1', NULL),
(9, 'XP_038914747.1', NULL),
(9, 'XP_038914746.1', NULL),
(9, 'XP_038914745.1', NULL),
(9, 'XP_038914744.1', NULL),
(9, 'XP_038914743.1', NULL),
(9, 'XP_038914742.1', NULL),
(9, 'XP_038914741.1', NULL),
(9, 'XP_038914740.1', NULL),
(9, 'XP_038914739.1', NULL),
(9, 'XP_038914738.1', NULL),
(9, 'XP_038914737.1', NULL),
(9, 'XP_038914736.1', NULL),
(9, 'XP_038914735.1', NULL),
(9, 'XP_038914734.1', NULL),
(9, 'XP_038914733.1', NULL),
(9, 'XP_038914732.1', NULL),
(9, 'XP_038914731.1', NULL),
(9, 'XP_038914730.1', NULL),
(9, 'XP_038914729.1', NULL),
(9, 'XP_038914728.1', NULL),
(9, 'XP_038914727.1', NULL),
(9, 'XP_038914726.1', NULL),
(9, 'XP_038914725.1', NULL),
(9, 'XP_038914724.1', NULL),
(9, 'XP_038914723.1', NULL),
(9, 'XP_038914722.1', NULL),
(9, 'XP_038914721.1', NULL),
(9, 'XP_038914720.1', NULL),
(9, 'XP_038914719.1', NULL),
(9, 'XP_038914718.1', NULL),
(9, 'XP_038914717.1', NULL),
(9, 'XP_038914716.1', NULL),
(9, 'XP_038914715.1', NULL),
(9, 'XP_038914714.1', NULL),
(9, 'XP_038914713.1', NULL),
(9, 'XP_038914712.1', NULL),
(9, 'XP_038914711.1', NULL),
(9, 'XP_038914710.1', NULL),
(9, 'XP_038914709.1', NULL),
(9, 'XP_038914708.1', NULL),
(9, 'XP_038914707.1', NULL),
(9, 'XP_038914706.1', NULL),
(9, 'XP_038914705.1', NULL),
(9, 'XP_038914704.1', NULL),
(9, 'XP_038914703.1', NULL),
(9, 'XP_038914702.1', NULL),
(9, 'XP_038914701.1', NULL),
(9, 'XP_038914700.1', NULL),
(9, 'XP_038914699.1', NULL),
(9, 'XP_038914698.1', NULL),
(9, 'XP_038914697.1', NULL);
INSERT INTO `prot_access_table` (`id_article`, `prot_access`, `entry_id_cazy`) VALUES
(9, 'XP_038914696.1', NULL),
(9, 'XP_038914695.1', NULL),
(9, 'XP_038914694.1', NULL),
(9, 'XP_038914693.1', NULL),
(9, 'XP_038914692.1', NULL),
(9, 'XP_038914691.1', NULL),
(9, 'XP_038914690.1', NULL),
(9, 'XP_038914689.1', NULL),
(9, 'XP_038914688.1', NULL),
(9, 'XP_038914687.1', NULL),
(9, 'XP_038914686.1', NULL),
(9, 'XP_038914685.1', NULL),
(9, 'XP_038914684.1', NULL),
(9, 'XP_038914683.1', NULL),
(9, 'XP_038914682.1', NULL),
(9, 'XP_038914681.1', NULL),
(9, 'XP_038914680.1', NULL),
(9, 'XP_038914679.1', NULL),
(9, 'XP_038914678.1', NULL),
(9, 'XP_038914677.1', NULL),
(9, 'XP_038914676.1', NULL),
(9, 'XP_038914675.1', NULL),
(9, 'XP_038914674.1', NULL),
(9, 'XP_038914673.1', NULL),
(9, 'XP_038914672.1', NULL),
(9, 'XP_038914671.1', NULL),
(9, 'XP_038914670.1', NULL),
(9, 'XP_038914669.1', NULL),
(9, 'XP_038914668.1', NULL),
(9, 'XP_038914667.1', NULL),
(9, 'XP_038914666.1', NULL),
(9, 'XP_038914665.1', NULL),
(9, 'XP_038914664.1', NULL),
(9, 'XP_038914663.1', NULL),
(9, 'XP_038914662.1', NULL),
(9, 'XP_038914661.1', NULL),
(9, 'XP_038914660.1', NULL),
(9, 'XP_038914659.1', NULL),
(9, 'XP_038914658.1', NULL),
(9, 'XP_038914657.1', NULL),
(9, 'XP_038914656.1', NULL),
(9, 'XP_038914655.1', NULL),
(9, 'XP_038914654.1', NULL),
(9, 'XP_038914653.1', NULL),
(9, 'XP_038914652.1', NULL),
(9, 'XP_038914651.1', NULL),
(9, 'XP_038914650.1', NULL),
(9, 'XP_038914649.1', NULL),
(9, 'XP_038914648.1', NULL),
(9, 'XP_038914647.1', NULL),
(9, 'XP_038914646.1', NULL),
(9, 'XP_038914645.1', NULL),
(9, 'XP_038914644.1', NULL),
(9, 'XP_038914643.1', NULL),
(9, 'XP_038914642.1', NULL),
(9, 'XP_038914641.1', NULL),
(9, 'XP_038914640.1', NULL),
(9, 'XP_038914639.1', NULL),
(9, 'XP_038914638.1', NULL),
(9, 'XP_038914637.1', NULL),
(9, 'XP_038914636.1', NULL),
(9, 'XP_038914635.1', NULL),
(9, 'XP_038914634.1', NULL),
(9, 'XP_038914633.1', NULL),
(9, 'XP_038914632.1', NULL),
(9, 'XP_038914631.1', NULL),
(9, 'XP_038914630.1', NULL),
(9, 'XP_038914629.1', NULL),
(9, 'XP_038914628.1', NULL),
(9, 'XP_038914627.1', NULL),
(9, 'XP_038914626.1', NULL),
(9, 'XP_038914625.1', NULL),
(9, 'XP_038914624.1', NULL),
(9, 'XP_038915007.1', NULL),
(9, 'XP_038915006.1', NULL),
(9, 'XP_038915005.1', NULL),
(9, 'XP_038915004.1', NULL),
(9, 'XP_038915003.1', NULL),
(9, 'XP_038915002.1', NULL),
(9, 'XP_038915001.1', NULL),
(9, 'XP_038915000.1', NULL),
(9, 'XP_038914999.1', NULL),
(9, 'XP_038914998.1', NULL),
(9, 'XP_038914997.1', NULL),
(9, 'XP_038914996.1', NULL),
(9, 'XP_038914995.1', NULL),
(9, 'XP_038914994.1', NULL),
(9, 'XP_038914993.1', NULL),
(9, 'XP_038914992.1', NULL),
(9, 'XP_038914991.1', NULL),
(9, 'XP_038914990.1', NULL),
(9, 'XP_038914989.1', NULL),
(9, 'XP_038914988.1', NULL),
(9, 'XP_038914987.1', NULL),
(9, 'XP_038914986.1', NULL),
(9, 'XP_038914985.1', NULL),
(9, 'XP_038914984.1', NULL),
(9, 'XP_038914983.1', NULL),
(9, 'XP_038914982.1', NULL),
(9, 'XP_038914981.1', NULL),
(9, 'XP_038914980.1', NULL),
(9, 'XP_038914979.1', NULL),
(9, 'XP_038914978.1', NULL),
(9, 'XP_038914977.1', NULL),
(9, 'XP_038914976.1', NULL),
(9, 'XP_038914975.1', NULL),
(9, 'XP_038914974.1', NULL),
(9, 'XP_038914973.1', NULL),
(9, 'XP_038914972.1', NULL),
(9, 'XP_038914971.1', NULL),
(9, 'XP_038914970.1', NULL),
(9, 'XP_038914969.1', NULL),
(9, 'XP_038914968.1', NULL),
(9, 'XP_038914967.1', NULL),
(9, 'XP_038914966.1', NULL),
(9, 'XP_038914965.1', NULL),
(9, 'XP_038914964.1', NULL),
(9, 'XP_038914963.1', NULL),
(9, 'XP_038914962.1', NULL),
(9, 'XP_038914961.1', NULL),
(9, 'XP_038914960.1', NULL),
(9, 'XP_038914959.1', NULL),
(9, 'XP_038914958.1', NULL),
(9, 'XP_038914957.1', NULL),
(9, 'XP_038914956.1', NULL),
(9, 'XP_038914955.1', NULL),
(9, 'XP_038914954.1', NULL),
(9, 'XP_038914953.1', NULL),
(9, 'XP_038914952.1', NULL),
(9, 'XP_038914951.1', NULL),
(9, 'XP_038914950.1', NULL),
(9, 'XP_038914949.1', NULL),
(9, 'XP_038914948.1', NULL),
(9, 'XP_038914947.1', NULL),
(9, 'XP_038914946.1', NULL),
(9, 'XP_038914945.1', NULL),
(9, 'XP_038914944.1', NULL),
(9, 'XP_038914943.1', NULL),
(9, 'XP_038914942.1', NULL),
(9, 'XP_038914941.1', NULL),
(9, 'XP_038914940.1', NULL),
(9, 'XP_038914939.1', NULL),
(9, 'XP_038914938.1', NULL),
(9, 'XP_038914937.1', NULL),
(9, 'XP_038914936.1', NULL),
(9, 'XP_038914935.1', NULL),
(9, 'XP_038914934.1', NULL),
(9, 'XP_038914933.1', NULL),
(9, 'XP_038914932.1', NULL),
(9, 'XP_038914931.1', NULL),
(9, 'XP_038914930.1', NULL),
(9, 'XP_038914929.1', NULL),
(9, 'XP_038914928.1', NULL),
(9, 'XP_038914927.1', NULL),
(9, 'XP_038914926.1', NULL),
(9, 'XP_038914925.1', NULL),
(9, 'XP_038914924.1', NULL),
(9, 'XP_038914923.1', NULL),
(9, 'XP_038914922.1', NULL),
(9, 'XP_038914921.1', NULL),
(9, 'XP_038914920.1', NULL),
(9, 'XP_038914919.1', NULL),
(9, 'XP_038914918.1', NULL),
(9, 'XP_038914917.1', NULL),
(9, 'XP_038914916.1', NULL),
(9, 'XP_038914915.1', NULL),
(9, 'XP_038914914.1', NULL),
(9, 'XP_038914913.1', NULL),
(9, 'XP_038914912.1', NULL),
(9, 'XP_038914911.1', NULL),
(9, 'XP_038914910.1', NULL),
(9, 'XP_038914909.1', NULL),
(9, 'XP_038914908.1', NULL),
(9, 'XP_038914907.1', NULL),
(9, 'XP_038914906.1', NULL),
(9, 'XP_038914905.1', NULL),
(9, 'XP_038914904.1', NULL),
(9, 'XP_038914903.1', NULL),
(9, 'XP_038914902.1', NULL),
(9, 'XP_038914901.1', NULL),
(9, 'XP_038914900.1', NULL),
(9, 'XP_038914899.1', NULL),
(9, 'XP_038914898.1', NULL),
(9, 'XP_038914897.1', NULL),
(9, 'XP_038914896.1', NULL),
(9, 'XP_038914895.1', NULL),
(9, 'XP_038914894.1', NULL),
(9, 'XP_038914893.1', NULL),
(9, 'XP_038914892.1', NULL),
(9, 'XP_038914891.1', NULL),
(9, 'XP_038914890.1', NULL),
(9, 'XP_038914889.1', NULL),
(9, 'XP_038914888.1', NULL),
(9, 'XP_038914887.1', NULL),
(9, 'XP_038914886.1', NULL),
(9, 'XP_038914885.1', NULL),
(9, 'XP_038914884.1', NULL),
(9, 'XP_038914883.1', NULL),
(9, 'XP_038914882.1', NULL),
(9, 'XP_038914881.1', NULL),
(9, 'XP_038914880.1', NULL),
(9, 'XP_038914879.1', NULL),
(9, 'XP_038914878.1', NULL),
(9, 'XP_038914877.1', NULL),
(9, 'XP_038914876.1', NULL),
(9, 'XP_038914875.1', NULL),
(9, 'XP_038914874.1', NULL),
(9, 'XP_038914873.1', NULL),
(9, 'XP_038914872.1', NULL),
(9, 'XP_038914871.1', NULL),
(9, 'XP_038914870.1', NULL),
(9, 'XP_038914869.1', NULL),
(9, 'XP_038914868.1', NULL),
(9, 'XP_038914867.1', NULL),
(9, 'XP_038914866.1', NULL),
(9, 'XP_038914865.1', NULL),
(9, 'XP_038914864.1', NULL),
(9, 'XP_038914863.1', NULL),
(9, 'XP_038914862.1', NULL),
(9, 'XP_038914861.1', NULL),
(9, 'XP_038914860.1', NULL),
(9, 'XP_038914859.1', NULL),
(9, 'XP_038914858.1', NULL),
(9, 'XP_038914857.1', NULL),
(9, 'XP_038914856.1', NULL),
(9, 'XP_038914855.1', NULL),
(9, 'XP_038914854.1', NULL),
(9, 'XP_038914853.1', NULL),
(9, 'XP_038914852.1', NULL),
(9, 'XP_038914851.1', NULL),
(9, 'XP_038914850.1', NULL),
(9, 'XP_038914849.1', NULL),
(9, 'XP_038914848.1', NULL),
(9, 'XP_038914847.1', NULL),
(9, 'XP_038914846.1', NULL),
(9, 'XP_038914845.1', NULL),
(9, 'XP_038914844.1', NULL),
(9, 'XP_038914843.1', NULL),
(9, 'XP_038914842.1', NULL),
(9, 'XP_038914841.1', NULL),
(9, 'XP_038914840.1', NULL),
(9, 'XP_038914839.1', NULL),
(9, 'XP_038914838.1', NULL),
(9, 'XP_038914837.1', NULL),
(9, 'XP_038914836.1', NULL),
(9, 'XP_038914835.1', NULL),
(9, 'XP_038914834.1', NULL),
(9, 'XP_038914833.1', NULL),
(9, 'XP_038914832.1', NULL),
(9, 'XP_038914831.1', NULL),
(9, 'XP_038914830.1', NULL),
(9, 'XP_038914829.1', NULL),
(9, 'XP_038914828.1', NULL),
(9, 'XP_038914827.1', NULL),
(9, 'XP_038914826.1', NULL),
(9, 'XP_038914825.1', NULL),
(9, 'XP_038914824.1', NULL),
(9, 'XP_038914823.1', NULL),
(9, 'XP_038914822.1', NULL),
(9, 'XP_038914821.1', NULL),
(9, 'XP_038914820.1', NULL),
(9, 'XP_038914819.1', NULL),
(9, 'XP_038914818.1', NULL),
(9, 'XP_038914817.1', NULL),
(9, 'XP_038914816.1', NULL),
(9, 'XP_038914815.1', NULL),
(9, 'XP_038914814.1', NULL),
(9, 'XP_038914813.1', NULL),
(9, 'XP_038914812.1', NULL),
(9, 'XP_038914811.1', NULL),
(9, 'XP_038914810.1', NULL),
(9, 'XP_038914809.1', NULL),
(9, 'XP_038914808.1', NULL),
(9, 'XP_038914807.1', NULL),
(9, 'XP_038914806.1', NULL),
(9, 'XP_038914805.1', NULL),
(9, 'XP_038914804.1', NULL),
(9, 'XP_038914803.1', NULL),
(9, 'XP_038914802.1', NULL),
(9, 'XP_038914801.1', NULL),
(9, 'XP_038914800.1', NULL),
(9, 'XP_038914799.1', NULL),
(9, 'XP_038914798.1', NULL),
(9, 'XP_038914797.1', NULL),
(9, 'XP_038914796.1', NULL),
(9, 'XP_038914795.1', NULL),
(9, 'XP_038914794.1', NULL),
(9, 'XP_038914793.1', NULL),
(9, 'XP_038914792.1', NULL),
(9, 'XP_038914791.1', NULL),
(9, 'XP_038914790.1', NULL),
(9, 'XP_038914789.1', NULL),
(9, 'XP_038914788.1', NULL),
(9, 'XP_038914787.1', NULL),
(9, 'XP_038914786.1', NULL),
(9, 'XP_038914785.1', NULL),
(9, 'XP_038914784.1', NULL),
(9, 'XP_038914783.1', NULL),
(9, 'XP_038914782.1', NULL),
(9, 'XP_038914781.1', NULL),
(9, 'XP_038914780.1', NULL),
(9, 'XP_038914779.1', NULL),
(9, 'XP_038914778.1', NULL),
(9, 'XP_038914777.1', NULL),
(9, 'XP_038914776.1', NULL),
(9, 'XP_038914775.1', NULL),
(9, 'XP_038914774.1', NULL),
(9, 'XP_038914773.1', NULL),
(9, 'XP_038914772.1', NULL),
(9, 'XP_038914771.1', NULL),
(9, 'XP_038914770.1', NULL),
(9, 'XP_038914769.1', NULL),
(9, 'XP_038914768.1', NULL),
(9, 'XP_038914767.1', NULL),
(9, 'XP_038914766.1', NULL),
(9, 'XP_038914765.1', NULL),
(9, 'XP_038914764.1', NULL),
(9, 'XP_038914763.1', NULL),
(9, 'XP_038914762.1', NULL),
(9, 'XP_038914761.1', NULL),
(9, 'XP_038914760.1', NULL),
(9, 'XP_038914759.1', NULL),
(9, 'XP_038914758.1', NULL),
(9, 'XP_038914757.1', NULL),
(9, 'XP_038914756.1', NULL),
(9, 'XP_038914755.1', NULL),
(9, 'XP_038914754.1', NULL),
(9, 'XP_038914753.1', NULL),
(9, 'XP_038914752.1', NULL),
(9, 'XP_038914751.1', NULL),
(9, 'XP_038914750.1', NULL),
(9, 'XP_038914749.1', NULL),
(9, 'XP_038914748.1', NULL),
(9, 'XP_038915188.1', NULL),
(9, 'XP_038915187.1', NULL),
(9, 'XP_038915186.1', NULL),
(9, 'XP_038915185.1', NULL),
(9, 'XP_038915184.1', NULL),
(9, 'XP_038915183.1', NULL),
(9, 'XP_038915182.1', NULL),
(9, 'XP_038915181.1', NULL),
(9, 'XP_038915180.1', NULL),
(9, 'XP_038915179.1', NULL),
(9, 'XP_038915178.1', NULL),
(9, 'XP_038915177.1', NULL),
(9, 'XP_038915176.1', NULL),
(9, 'XP_038915175.1', NULL),
(9, 'XP_038915174.1', NULL),
(9, 'XP_038915173.1', NULL),
(9, 'XP_038915172.1', NULL),
(9, 'XP_038915171.1', NULL),
(9, 'XP_038915170.1', NULL),
(9, 'XP_038915169.1', NULL),
(9, 'XP_038915168.1', NULL),
(9, 'XP_038915167.1', NULL),
(9, 'XP_038915166.1', NULL),
(9, 'XP_038915165.1', NULL),
(9, 'XP_038915164.1', NULL),
(9, 'XP_038915163.1', NULL),
(9, 'XP_038915162.1', NULL),
(9, 'XP_038915161.1', NULL),
(9, 'XP_038915160.1', NULL),
(9, 'XP_038915159.1', NULL),
(9, 'XP_038915158.1', NULL),
(9, 'XP_038915157.1', NULL),
(9, 'XP_038915156.1', NULL),
(9, 'XP_038915155.1', NULL),
(9, 'XP_038915154.1', NULL),
(9, 'XP_038915153.1', NULL),
(9, 'XP_038915152.1', NULL),
(9, 'XP_038915151.1', NULL),
(9, 'XP_038915150.1', NULL),
(9, 'XP_038915149.1', NULL),
(9, 'XP_038915148.1', NULL),
(9, 'XP_038915147.1', NULL),
(9, 'XP_038915146.1', NULL),
(9, 'XP_038915145.1', NULL),
(9, 'XP_038915144.1', NULL),
(9, 'XP_038915143.1', NULL),
(9, 'XP_038915142.1', NULL),
(9, 'XP_038915141.1', NULL),
(9, 'XP_038915140.1', NULL),
(9, 'XP_038915139.1', NULL),
(9, 'XP_038915138.1', NULL),
(9, 'XP_038915137.1', NULL),
(9, 'XP_038915136.1', NULL),
(9, 'XP_038915135.1', NULL),
(9, 'XP_038915134.1', NULL),
(9, 'XP_038915133.1', NULL),
(9, 'XP_038915132.1', NULL),
(9, 'XP_038915131.1', NULL),
(9, 'XP_038915130.1', NULL),
(9, 'XP_038915129.1', NULL),
(9, 'XP_038915128.1', NULL),
(9, 'XP_038915127.1', NULL),
(9, 'XP_038915126.1', NULL),
(9, 'XP_038915125.1', NULL),
(9, 'XP_038915124.1', NULL),
(9, 'XP_038915123.1', NULL),
(9, 'XP_038915122.1', NULL),
(9, 'XP_038915121.1', NULL),
(9, 'XP_038915120.1', NULL),
(9, 'XP_038915119.1', NULL),
(9, 'XP_038915118.1', NULL),
(9, 'XP_038915117.1', NULL),
(9, 'XP_038915116.1', NULL),
(9, 'XP_038915115.1', NULL),
(9, 'XP_038915114.1', NULL),
(9, 'XP_038915113.1', NULL),
(9, 'XP_038915112.1', NULL),
(9, 'XP_038915111.1', NULL),
(9, 'XP_038915110.1', NULL),
(9, 'XP_038915109.1', NULL),
(9, 'XP_038915108.1', NULL),
(9, 'XP_038915107.1', NULL),
(9, 'XP_038915106.1', NULL),
(9, 'XP_038915105.1', NULL),
(9, 'XP_038915104.1', NULL),
(9, 'XP_038915103.1', NULL),
(9, 'XP_038915102.1', NULL),
(9, 'XP_038915101.1', NULL),
(9, 'XP_038915100.1', NULL),
(9, 'XP_038915099.1', NULL),
(9, 'XP_038915098.1', NULL),
(9, 'XP_038915097.1', NULL),
(9, 'XP_038915096.1', NULL),
(9, 'XP_038915095.1', NULL),
(9, 'XP_038915094.1', NULL),
(9, 'XP_038915093.1', NULL),
(9, 'XP_038915092.1', NULL),
(9, 'XP_038915091.1', NULL),
(9, 'XP_038915090.1', NULL),
(9, 'XP_038915089.1', NULL),
(9, 'XP_038915088.1', NULL),
(9, 'XP_038915087.1', NULL),
(9, 'XP_038915086.1', NULL),
(9, 'XP_038915085.1', NULL),
(9, 'XP_038915084.1', NULL),
(9, 'XP_038915083.1', NULL),
(9, 'XP_038915082.1', NULL),
(9, 'XP_038915081.1', NULL),
(9, 'XP_038915080.1', NULL),
(9, 'XP_038915079.1', NULL),
(9, 'XP_038915078.1', NULL),
(9, 'XP_038915077.1', NULL),
(9, 'XP_038915076.1', NULL),
(9, 'XP_038915075.1', NULL),
(9, 'XP_038915074.1', NULL),
(9, 'XP_038915073.1', NULL),
(9, 'XP_038915072.1', NULL),
(9, 'XP_038915071.1', NULL),
(9, 'XP_038915070.1', NULL),
(9, 'XP_038915069.1', NULL),
(9, 'XP_038915068.1', NULL),
(9, 'XP_038915067.1', NULL),
(9, 'XP_038915066.1', NULL),
(9, 'XP_038915065.1', NULL),
(9, 'XP_038915064.1', NULL),
(9, 'XP_038915063.1', NULL),
(9, 'XP_038915062.1', NULL),
(9, 'XP_038915061.1', NULL),
(9, 'XP_038915060.1', NULL),
(9, 'XP_038915059.1', NULL),
(9, 'XP_038915058.1', NULL),
(9, 'XP_038915057.1', NULL),
(9, 'XP_038915056.1', NULL),
(9, 'XP_038915055.1', NULL),
(9, 'XP_038915054.1', NULL),
(9, 'XP_038915053.1', NULL),
(9, 'XP_038915052.1', NULL),
(9, 'XP_038915051.1', NULL),
(9, 'XP_038915050.1', NULL),
(9, 'XP_038915049.1', NULL),
(9, 'XP_038915048.1', NULL),
(9, 'XP_038915047.1', NULL),
(9, 'XP_038915046.1', NULL),
(9, 'XP_038915045.1', NULL),
(9, 'XP_038915044.1', NULL),
(9, 'XP_038915043.1', NULL),
(9, 'XP_038915042.1', NULL),
(9, 'XP_038915041.1', NULL),
(9, 'XP_038915040.1', NULL),
(9, 'XP_038915039.1', NULL),
(9, 'XP_038915038.1', NULL),
(9, 'XP_038915037.1', NULL),
(9, 'XP_038915036.1', NULL),
(9, 'XP_038915035.1', NULL),
(9, 'XP_038915034.1', NULL),
(9, 'XP_038915033.1', NULL),
(9, 'XP_038915032.1', NULL),
(9, 'XP_038915031.1', NULL),
(9, 'XP_038915030.1', NULL),
(9, 'XP_038915029.1', NULL),
(9, 'XP_038915028.1', NULL),
(9, 'XP_038915027.1', NULL),
(9, 'XP_038915026.1', NULL),
(9, 'XP_038915025.1', NULL),
(9, 'XP_038915024.1', NULL),
(9, 'XP_038915023.1', NULL),
(9, 'XP_038915022.1', NULL),
(9, 'XP_038915021.1', NULL),
(9, 'XP_038915020.1', NULL),
(9, 'XP_038915019.1', NULL),
(9, 'XP_038915018.1', NULL),
(9, 'XP_038915017.1', NULL),
(9, 'XP_038915016.1', NULL),
(9, 'XP_038915015.1', NULL),
(9, 'XP_038915014.1', NULL),
(9, 'XP_038915013.1', NULL),
(9, 'XP_038915012.1', NULL),
(9, 'XP_038915011.1', NULL),
(9, 'XP_038915010.1', NULL),
(9, 'XP_038915009.1', NULL),
(9, 'XP_038915008.1', NULL),
(9, 'XP_038915310.1', NULL),
(9, 'XP_038915309.1', NULL),
(9, 'XP_038915308.1', NULL),
(9, 'XP_038915307.1', NULL),
(9, 'XP_038915306.1', NULL),
(9, 'XP_038915305.1', NULL),
(9, 'XP_038915304.1', NULL),
(9, 'XP_038915303.1', NULL),
(9, 'XP_038915302.1', NULL),
(9, 'XP_038915301.1', NULL),
(9, 'XP_038915300.1', NULL),
(9, 'XP_038915299.1', NULL),
(9, 'XP_038915298.1', NULL),
(9, 'XP_038915297.1', NULL),
(9, 'XP_038915296.1', NULL),
(9, 'XP_038915295.1', NULL),
(9, 'XP_038915294.1', NULL),
(9, 'XP_038915293.1', NULL),
(9, 'XP_038915292.1', NULL),
(9, 'XP_038915291.1', NULL),
(9, 'XP_038915290.1', NULL),
(9, 'XP_038915289.1', NULL),
(9, 'XP_038915288.1', NULL),
(9, 'XP_038915287.1', NULL),
(9, 'XP_038915286.1', NULL),
(9, 'XP_038915285.1', NULL),
(9, 'XP_038915284.1', NULL),
(9, 'XP_038915283.1', NULL),
(9, 'XP_038915282.1', NULL),
(9, 'XP_038915281.1', NULL),
(9, 'XP_038915280.1', NULL),
(9, 'XP_038915279.1', NULL),
(9, 'XP_038915278.1', NULL),
(9, 'XP_038915277.1', NULL),
(9, 'XP_038915276.1', NULL),
(9, 'XP_038915275.1', NULL),
(9, 'XP_038915274.1', NULL),
(9, 'XP_038915273.1', NULL),
(9, 'XP_038915272.1', NULL),
(9, 'XP_038915271.1', NULL),
(9, 'XP_038915270.1', NULL),
(9, 'XP_038915269.1', NULL),
(9, 'XP_038915268.1', NULL),
(9, 'XP_038915267.1', NULL),
(9, 'XP_038915266.1', NULL),
(9, 'XP_038915265.1', NULL),
(9, 'XP_038915264.1', NULL),
(9, 'XP_038915263.1', NULL),
(9, 'XP_038915262.1', NULL),
(9, 'XP_038915261.1', NULL),
(9, 'XP_038915260.1', NULL),
(9, 'XP_038915259.1', NULL),
(9, 'XP_038915258.1', NULL),
(9, 'XP_038915257.1', NULL),
(9, 'XP_038915256.1', NULL),
(9, 'XP_038915255.1', NULL),
(9, 'XP_038915254.1', NULL),
(9, 'XP_038915253.1', NULL),
(9, 'XP_038915252.1', NULL),
(9, 'XP_038915251.1', NULL),
(9, 'XP_038915250.1', NULL),
(9, 'XP_038915249.1', NULL),
(9, 'XP_038915248.1', NULL),
(9, 'XP_038915247.1', NULL),
(9, 'XP_038915246.1', NULL),
(9, 'XP_038915245.1', NULL),
(9, 'XP_038915244.1', NULL),
(9, 'XP_038915243.1', NULL),
(9, 'XP_038915242.1', NULL),
(9, 'XP_038915241.1', NULL),
(9, 'XP_038915240.1', NULL),
(9, 'XP_038915239.1', NULL),
(9, 'XP_038915238.1', NULL),
(9, 'XP_038915237.1', NULL),
(9, 'XP_038915236.1', NULL),
(9, 'XP_038915235.1', NULL),
(9, 'XP_038915234.1', NULL),
(9, 'XP_038915233.1', NULL),
(9, 'XP_038915232.1', NULL),
(9, 'XP_038915231.1', NULL),
(9, 'XP_038915230.1', NULL),
(9, 'XP_038915229.1', NULL),
(9, 'XP_038915228.1', NULL),
(9, 'XP_038915227.1', NULL),
(9, 'XP_038915226.1', NULL),
(9, 'XP_038915225.1', NULL),
(9, 'XP_038915224.1', NULL),
(9, 'XP_038915223.1', NULL),
(9, 'XP_038915222.1', NULL),
(9, 'XP_038915221.1', NULL),
(9, 'XP_038915220.1', NULL),
(9, 'XP_038915219.1', NULL),
(9, 'XP_038915218.1', NULL),
(9, 'XP_038915217.1', NULL),
(9, 'XP_038915216.1', NULL),
(9, 'XP_038915215.1', NULL),
(9, 'XP_038915214.1', NULL),
(9, 'XP_038915213.1', NULL),
(9, 'XP_038915212.1', NULL),
(9, 'XP_038915211.1', NULL),
(9, 'XP_038915210.1', NULL),
(9, 'XP_038915209.1', NULL),
(9, 'XP_038915208.1', NULL),
(9, 'XP_038915207.1', NULL),
(9, 'XP_038915206.1', NULL),
(9, 'XP_038915205.1', NULL),
(9, 'XP_038915204.1', NULL),
(9, 'XP_038915203.1', NULL),
(9, 'XP_038915202.1', NULL),
(9, 'XP_038915201.1', NULL),
(9, 'XP_038915200.1', NULL),
(9, 'XP_038915199.1', NULL),
(9, 'XP_038915198.1', NULL),
(9, 'XP_038915197.1', NULL),
(9, 'XP_038915196.1', NULL),
(9, 'XP_038915195.1', NULL),
(9, 'XP_038915194.1', NULL),
(9, 'XP_038915193.1', NULL),
(9, 'XP_038915192.1', NULL),
(9, 'XP_038915191.1', NULL),
(9, 'XP_038915190.1', NULL),
(9, 'XP_038915189.1', NULL),
(9, 'XP_038915387.1', NULL),
(9, 'XP_038915386.1', NULL),
(9, 'XP_038915385.1', NULL),
(9, 'XP_038915384.1', NULL),
(9, 'XP_038915383.1', NULL),
(9, 'XP_038915382.1', NULL),
(9, 'XP_038915381.1', NULL),
(9, 'XP_038915380.1', NULL),
(9, 'XP_038915379.1', NULL),
(9, 'XP_038915378.1', NULL),
(9, 'XP_038915377.1', NULL),
(9, 'XP_038915376.1', NULL),
(9, 'XP_038915375.1', NULL),
(9, 'XP_038915374.1', NULL),
(9, 'XP_038915373.1', NULL),
(9, 'XP_038915372.1', NULL),
(9, 'XP_038915371.1', NULL),
(9, 'XP_038915370.1', NULL),
(9, 'XP_038915369.1', NULL),
(9, 'XP_038915368.1', NULL),
(9, 'XP_038915367.1', NULL),
(9, 'XP_038915366.1', NULL),
(9, 'XP_038915365.1', NULL),
(9, 'XP_038915364.1', NULL),
(9, 'XP_038915363.1', NULL),
(9, 'XP_038915362.1', NULL),
(9, 'XP_038915361.1', NULL),
(9, 'XP_038915360.1', NULL),
(9, 'XP_038915359.1', NULL),
(9, 'XP_038915358.1', NULL),
(9, 'XP_038915357.1', NULL),
(9, 'XP_038915356.1', NULL),
(9, 'XP_038915355.1', NULL),
(9, 'XP_038915354.1', NULL),
(9, 'XP_038915353.1', NULL),
(9, 'XP_038915352.1', NULL),
(9, 'XP_038915351.1', NULL),
(9, 'XP_038915350.1', NULL),
(9, 'XP_038915349.1', NULL),
(9, 'XP_038915348.1', NULL),
(9, 'XP_038915347.1', NULL),
(9, 'XP_038915346.1', NULL),
(9, 'XP_038915345.1', NULL),
(9, 'XP_038915344.1', NULL),
(9, 'XP_038915343.1', NULL),
(9, 'XP_038915342.1', NULL),
(9, 'XP_038915341.1', NULL),
(9, 'XP_038915340.1', NULL),
(9, 'XP_038915339.1', NULL),
(9, 'XP_038915338.1', NULL),
(9, 'XP_038915337.1', NULL),
(9, 'XP_038915336.1', NULL),
(9, 'XP_038915335.1', NULL),
(9, 'XP_038915334.1', NULL),
(9, 'XP_038915333.1', NULL),
(9, 'XP_038915332.1', NULL),
(9, 'XP_038915331.1', NULL),
(9, 'XP_038915330.1', NULL),
(9, 'XP_038915329.1', NULL),
(9, 'XP_038915328.1', NULL),
(9, 'XP_038915327.1', NULL),
(9, 'XP_038915326.1', NULL),
(9, 'XP_038915325.1', NULL),
(9, 'XP_038915324.1', NULL),
(9, 'XP_038915323.1', NULL),
(9, 'XP_038915322.1', NULL),
(9, 'XP_038915321.1', NULL),
(9, 'XP_038915320.1', NULL),
(9, 'XP_038915319.1', NULL),
(9, 'XP_038915318.1', NULL),
(9, 'XP_038915317.1', NULL),
(9, 'XP_038915316.1', NULL),
(9, 'XP_038915315.1', NULL),
(9, 'XP_038915314.1', NULL),
(9, 'XP_038915313.1', NULL),
(9, 'XP_038915312.1', NULL),
(9, 'XP_038915311.1', NULL),
(9, 'XP_038915545.1', NULL),
(9, 'XP_038915544.1', NULL),
(9, 'XP_038915543.1', NULL),
(9, 'XP_038915542.1', NULL),
(9, 'XP_038915541.1', NULL),
(9, 'XP_038915540.1', NULL),
(9, 'XP_038915539.1', NULL),
(9, 'XP_038915538.1', NULL),
(9, 'XP_038915537.1', NULL),
(9, 'XP_038915536.1', NULL),
(9, 'XP_038915535.1', NULL),
(9, 'XP_038915534.1', NULL),
(9, 'XP_038915533.1', NULL),
(9, 'XP_038915532.1', NULL),
(9, 'XP_038915531.1', NULL),
(9, 'XP_038915530.1', NULL),
(9, 'XP_038915529.1', NULL),
(9, 'XP_038915528.1', NULL),
(9, 'XP_038915527.1', NULL),
(9, 'XP_038915526.1', NULL),
(9, 'XP_038915525.1', NULL),
(9, 'XP_038915524.1', NULL),
(9, 'XP_038915523.1', NULL),
(9, 'XP_038915522.1', NULL),
(9, 'XP_038915521.1', NULL),
(9, 'XP_038915520.1', NULL),
(9, 'XP_038915519.1', NULL),
(9, 'XP_038915518.1', NULL),
(9, 'XP_038915517.1', NULL),
(9, 'XP_038915516.1', NULL),
(9, 'XP_038915515.1', NULL),
(9, 'XP_038915514.1', NULL),
(9, 'XP_038915513.1', NULL),
(9, 'XP_038915512.1', NULL),
(9, 'XP_038915511.1', NULL),
(9, 'XP_038915510.1', NULL),
(9, 'XP_038915509.1', NULL),
(9, 'XP_038915508.1', NULL),
(9, 'XP_038915507.1', NULL),
(9, 'XP_038915506.1', NULL),
(9, 'XP_038915505.1', NULL),
(9, 'XP_038915504.1', NULL),
(9, 'XP_038915503.1', NULL),
(9, 'XP_038915502.1', NULL),
(9, 'XP_038915501.1', NULL),
(9, 'XP_038915500.1', NULL),
(9, 'XP_038915499.1', NULL),
(9, 'XP_038915498.1', NULL),
(9, 'XP_038915497.1', NULL),
(9, 'XP_038915496.1', NULL),
(9, 'XP_038915495.1', NULL),
(9, 'XP_038915494.1', NULL),
(9, 'XP_038915493.1', NULL),
(9, 'XP_038915492.1', NULL),
(9, 'XP_038915491.1', NULL),
(9, 'XP_038915490.1', NULL),
(9, 'XP_038915489.1', NULL),
(9, 'XP_038915488.1', NULL),
(9, 'XP_038915487.1', NULL),
(9, 'XP_038915486.1', NULL),
(9, 'XP_038915485.1', NULL),
(9, 'XP_038915484.1', NULL),
(9, 'XP_038915483.1', NULL),
(9, 'XP_038915482.1', NULL),
(9, 'XP_038915481.1', NULL),
(9, 'XP_038915480.1', NULL),
(9, 'XP_038915479.1', NULL),
(9, 'XP_038915478.1', NULL),
(9, 'XP_038915477.1', NULL),
(9, 'XP_038915476.1', NULL),
(9, 'XP_038915475.1', NULL),
(9, 'XP_038915474.1', NULL),
(9, 'XP_038915473.1', NULL),
(9, 'XP_038915472.1', NULL),
(9, 'XP_038915471.1', NULL),
(9, 'XP_038915470.1', NULL),
(9, 'XP_038915469.1', NULL),
(9, 'XP_038915468.1', NULL),
(9, 'XP_038915467.1', NULL),
(9, 'XP_038915466.1', NULL),
(9, 'XP_038915465.1', NULL),
(9, 'XP_038915464.1', NULL),
(9, 'XP_038915463.1', NULL),
(9, 'XP_038915462.1', NULL),
(9, 'XP_038915461.1', NULL),
(9, 'XP_038915460.1', NULL),
(9, 'XP_038915459.1', NULL),
(9, 'XP_038915458.1', NULL),
(9, 'XP_038915457.1', NULL),
(9, 'XP_038915456.1', NULL),
(9, 'XP_038915455.1', NULL),
(9, 'XP_038915454.1', NULL),
(9, 'XP_038915453.1', NULL),
(9, 'XP_038915452.1', NULL),
(9, 'XP_038915451.1', NULL),
(9, 'XP_038915450.1', NULL),
(9, 'XP_038915449.1', NULL),
(9, 'XP_038915448.1', NULL),
(9, 'XP_038915447.1', NULL),
(9, 'XP_038915446.1', NULL),
(9, 'XP_038915445.1', NULL),
(9, 'XP_038915444.1', NULL),
(9, 'XP_038915443.1', NULL),
(9, 'XP_038915442.1', NULL),
(9, 'XP_038915441.1', NULL),
(9, 'XP_038915440.1', NULL),
(9, 'XP_038915439.1', NULL),
(9, 'XP_038915438.1', NULL),
(9, 'XP_038915437.1', NULL),
(9, 'XP_038915436.1', NULL),
(9, 'XP_038915435.1', NULL),
(9, 'XP_038915434.1', NULL),
(9, 'XP_038915433.1', NULL),
(9, 'XP_038915432.1', NULL),
(9, 'XP_038915431.1', NULL),
(9, 'XP_038915430.1', NULL),
(9, 'XP_038915429.1', NULL),
(9, 'XP_038915428.1', NULL),
(9, 'XP_038915427.1', NULL),
(9, 'XP_038915426.1', NULL),
(9, 'XP_038915425.1', NULL),
(9, 'XP_038915424.1', NULL),
(9, 'XP_038915423.1', NULL),
(9, 'XP_038915422.1', NULL),
(9, 'XP_038915421.1', NULL),
(9, 'XP_038915420.1', NULL),
(9, 'XP_038915419.1', NULL),
(9, 'XP_038915418.1', NULL),
(9, 'XP_038915417.1', NULL),
(9, 'XP_038915416.1', NULL),
(9, 'XP_038915415.1', NULL),
(9, 'XP_038915414.1', NULL),
(9, 'XP_038915413.1', NULL),
(9, 'XP_038915412.1', NULL),
(9, 'XP_038915411.1', NULL),
(9, 'XP_038915410.1', NULL),
(9, 'XP_038915409.1', NULL),
(9, 'XP_038915408.1', NULL),
(9, 'XP_038915407.1', NULL),
(9, 'XP_038915406.1', NULL),
(9, 'XP_038915405.1', NULL),
(9, 'XP_038915404.1', NULL),
(9, 'XP_038915403.1', NULL),
(9, 'XP_038915402.1', NULL),
(9, 'XP_038915401.1', NULL),
(9, 'XP_038915400.1', NULL),
(9, 'XP_038915399.1', NULL),
(9, 'XP_038915398.1', NULL),
(9, 'XP_038915397.1', NULL),
(9, 'XP_038915396.1', NULL),
(9, 'XP_038915395.1', NULL),
(9, 'XP_038915394.1', NULL),
(9, 'XP_038915393.1', NULL),
(9, 'XP_038915392.1', NULL),
(9, 'XP_038915391.1', NULL),
(9, 'XP_038915390.1', NULL),
(9, 'XP_038915389.1', NULL),
(9, 'XP_038915388.1', NULL),
(9, 'XP_038915599.1', NULL),
(9, 'XP_038915598.1', NULL),
(9, 'XP_038915597.1', NULL),
(9, 'XP_038915596.1', NULL),
(9, 'XP_038915595.1', NULL),
(9, 'XP_038915594.1', NULL),
(9, 'XP_038915593.1', NULL),
(9, 'XP_038915592.1', NULL),
(9, 'XP_038915591.1', NULL),
(9, 'XP_038915590.1', NULL),
(9, 'XP_038915589.1', NULL),
(9, 'XP_038915588.1', NULL),
(9, 'XP_038915587.1', NULL),
(9, 'XP_038915586.1', NULL),
(9, 'XP_038915585.1', NULL),
(9, 'XP_038915584.1', NULL),
(9, 'XP_038915583.1', NULL),
(9, 'XP_038915582.1', NULL),
(9, 'XP_038915581.1', NULL),
(9, 'XP_038915580.1', NULL),
(9, 'XP_038915579.1', NULL),
(9, 'XP_038915578.1', NULL),
(9, 'XP_038915577.1', NULL),
(9, 'XP_038915576.1', NULL),
(9, 'XP_038915575.1', NULL),
(9, 'XP_038915574.1', NULL),
(9, 'XP_038915573.1', NULL),
(9, 'XP_038915572.1', NULL),
(9, 'XP_038915571.1', NULL),
(9, 'XP_038915570.1', NULL),
(9, 'XP_038915569.1', NULL),
(9, 'XP_038915568.1', NULL),
(9, 'XP_038915567.1', NULL),
(9, 'XP_038915566.1', NULL),
(9, 'XP_038915565.1', NULL),
(9, 'XP_038915564.1', NULL),
(9, 'XP_038915563.1', NULL),
(9, 'XP_038915562.1', NULL),
(9, 'XP_038915561.1', NULL),
(9, 'XP_038915560.1', NULL),
(9, 'XP_038915559.1', NULL),
(9, 'XP_038915558.1', NULL),
(9, 'XP_038915557.1', NULL),
(9, 'XP_038915556.1', NULL),
(9, 'XP_038915555.1', NULL),
(9, 'XP_038915554.1', NULL),
(9, 'XP_038915553.1', NULL),
(9, 'XP_038915552.1', NULL),
(9, 'XP_038915551.1', NULL),
(9, 'XP_038915550.1', NULL),
(9, 'XP_038915549.1', NULL),
(9, 'XP_038915548.1', NULL),
(9, 'XP_038915547.1', NULL),
(9, 'XP_038915546.1', NULL),
(9, 'XP_038915838.1', NULL),
(9, 'XP_038915837.1', NULL),
(9, 'XP_038915836.1', NULL),
(9, 'XP_038915835.1', NULL),
(9, 'XP_038915834.1', NULL),
(9, 'XP_038915833.1', NULL),
(9, 'XP_038915832.1', NULL),
(9, 'XP_038915831.1', NULL),
(9, 'XP_038915830.1', NULL),
(9, 'XP_038915829.1', NULL),
(9, 'XP_038915828.1', NULL),
(9, 'XP_038915827.1', NULL),
(9, 'XP_038915826.1', NULL),
(9, 'XP_038915825.1', NULL),
(9, 'XP_038915824.1', NULL),
(9, 'XP_038915823.1', NULL),
(9, 'XP_038915822.1', NULL),
(9, 'XP_038915821.1', NULL),
(9, 'XP_038915820.1', NULL),
(9, 'XP_038915819.1', NULL),
(9, 'XP_038915818.1', NULL),
(9, 'XP_038915817.1', NULL),
(9, 'XP_038915816.1', NULL),
(9, 'XP_038915815.1', NULL),
(9, 'XP_038915814.1', NULL),
(9, 'XP_038915813.1', NULL),
(9, 'XP_038915812.1', NULL),
(9, 'XP_038915811.1', NULL),
(9, 'XP_038915810.1', NULL),
(9, 'XP_038915809.1', NULL),
(9, 'XP_038915808.1', NULL),
(9, 'XP_038915807.1', NULL),
(9, 'XP_038915806.1', NULL),
(9, 'XP_038915805.1', NULL),
(9, 'XP_038915804.1', NULL),
(9, 'XP_038915803.1', NULL),
(9, 'XP_038915802.1', NULL),
(9, 'XP_038915801.1', NULL),
(9, 'XP_038915800.1', NULL),
(9, 'XP_038915799.1', NULL),
(9, 'XP_038915798.1', NULL),
(9, 'XP_038915797.1', NULL),
(9, 'XP_038915796.1', NULL),
(9, 'XP_038915795.1', NULL),
(9, 'XP_038915794.1', NULL),
(9, 'XP_038915793.1', NULL),
(9, 'XP_038915792.1', NULL),
(9, 'XP_038915791.1', NULL),
(9, 'XP_038915790.1', NULL),
(9, 'XP_038915789.1', NULL),
(9, 'XP_038915788.1', NULL),
(9, 'XP_038915787.1', NULL),
(9, 'XP_038915786.1', NULL),
(9, 'XP_038915785.1', NULL),
(9, 'XP_038915784.1', NULL),
(9, 'XP_038915783.1', NULL),
(9, 'XP_038915782.1', NULL),
(9, 'XP_038915781.1', NULL),
(9, 'XP_038915780.1', NULL),
(9, 'XP_038915779.1', NULL),
(9, 'XP_038915778.1', NULL),
(9, 'XP_038915777.1', NULL),
(9, 'XP_038915776.1', NULL),
(9, 'XP_038915775.1', NULL),
(9, 'XP_038915774.1', NULL),
(9, 'XP_038915773.1', NULL),
(9, 'XP_038915772.1', NULL),
(9, 'XP_038915771.1', NULL),
(9, 'XP_038915770.1', NULL),
(9, 'XP_038915769.1', NULL),
(9, 'XP_038915768.1', NULL),
(9, 'XP_038915767.1', NULL),
(9, 'XP_038915766.1', NULL),
(9, 'XP_038915765.1', NULL),
(9, 'XP_038915764.1', NULL),
(9, 'XP_038915763.1', NULL),
(9, 'XP_038915762.1', NULL),
(9, 'XP_038915761.1', NULL),
(9, 'XP_038915760.1', NULL),
(9, 'XP_038915759.1', NULL),
(9, 'XP_038915758.1', NULL),
(9, 'XP_038915757.1', NULL),
(9, 'XP_038915756.1', NULL),
(9, 'XP_038915755.1', NULL),
(9, 'XP_038915754.1', NULL),
(9, 'XP_038915753.1', NULL),
(9, 'XP_038915752.1', NULL),
(9, 'XP_038915751.1', NULL),
(9, 'XP_038915750.1', NULL),
(9, 'XP_038915749.1', NULL),
(9, 'XP_038915748.1', NULL),
(9, 'XP_038915747.1', NULL),
(9, 'XP_038915746.1', NULL),
(9, 'XP_038915745.1', NULL),
(9, 'XP_038915744.1', NULL),
(9, 'XP_038915743.1', NULL),
(9, 'XP_038915742.1', NULL),
(9, 'XP_038915741.1', NULL),
(9, 'XP_038915740.1', NULL),
(9, 'XP_038915739.1', NULL),
(9, 'XP_038915738.1', NULL),
(9, 'XP_038915737.1', NULL),
(9, 'XP_038915736.1', NULL),
(9, 'XP_038915735.1', NULL),
(9, 'XP_038915734.1', NULL),
(9, 'XP_038915733.1', NULL),
(9, 'XP_038915732.1', NULL),
(9, 'XP_038915731.1', NULL),
(9, 'XP_038915730.1', NULL),
(9, 'XP_038915729.1', NULL),
(9, 'XP_038915728.1', NULL),
(9, 'XP_038915727.1', NULL),
(9, 'XP_038915726.1', NULL),
(9, 'XP_038915725.1', NULL),
(9, 'XP_038915724.1', NULL),
(9, 'XP_038915723.1', NULL),
(9, 'XP_038915722.1', NULL),
(9, 'XP_038915721.1', NULL),
(9, 'XP_038915720.1', NULL),
(9, 'XP_038915719.1', NULL),
(9, 'XP_038915718.1', NULL),
(9, 'XP_038915717.1', NULL),
(9, 'XP_038915716.1', NULL),
(9, 'XP_038915715.1', NULL),
(9, 'XP_038915714.1', NULL),
(9, 'XP_038915713.1', NULL),
(9, 'XP_038915712.1', NULL),
(9, 'XP_038915711.1', NULL),
(9, 'XP_038915710.1', NULL),
(9, 'XP_038915709.1', NULL),
(9, 'XP_038915708.1', NULL),
(9, 'XP_038915707.1', NULL),
(9, 'XP_038915706.1', NULL),
(9, 'XP_038915705.1', NULL),
(9, 'XP_038915704.1', NULL),
(9, 'XP_038915703.1', NULL),
(9, 'XP_038915702.1', NULL),
(9, 'XP_038915701.1', NULL),
(9, 'XP_038915700.1', NULL),
(9, 'XP_038915699.1', NULL),
(9, 'XP_038915698.1', NULL),
(9, 'XP_038915697.1', NULL),
(9, 'XP_038915696.1', NULL),
(9, 'XP_038915695.1', NULL),
(9, 'XP_038915694.1', NULL),
(9, 'XP_038915693.1', NULL),
(9, 'XP_038915692.1', NULL),
(9, 'XP_038915691.1', NULL),
(9, 'XP_038915690.1', NULL),
(9, 'XP_038915689.1', NULL),
(9, 'XP_038915688.1', NULL),
(9, 'XP_038915687.1', NULL),
(9, 'XP_038915686.1', NULL),
(9, 'XP_038915685.1', NULL),
(9, 'XP_038915684.1', NULL),
(9, 'XP_038915683.1', NULL),
(9, 'XP_038915682.1', NULL),
(9, 'XP_038915681.1', NULL),
(9, 'XP_038915680.1', NULL),
(9, 'XP_038915679.1', NULL),
(9, 'XP_038915678.1', NULL),
(9, 'XP_038915677.1', NULL),
(9, 'XP_038915676.1', NULL),
(9, 'XP_038915675.1', NULL),
(9, 'XP_038915674.1', NULL),
(9, 'XP_038915673.1', NULL),
(9, 'XP_038915672.1', NULL),
(9, 'XP_038915671.1', NULL),
(9, 'XP_038915670.1', NULL),
(9, 'XP_038915669.1', NULL),
(9, 'XP_038915668.1', NULL),
(9, 'XP_038915667.1', NULL),
(9, 'XP_038915666.1', NULL),
(9, 'XP_038915665.1', NULL),
(9, 'XP_038915664.1', NULL),
(9, 'XP_038915663.1', NULL),
(9, 'XP_038915662.1', NULL),
(9, 'XP_038915661.1', NULL),
(9, 'XP_038915660.1', NULL),
(9, 'XP_038915659.1', NULL),
(9, 'XP_038915658.1', NULL),
(9, 'XP_038915657.1', NULL),
(9, 'XP_038915656.1', NULL),
(9, 'XP_038915655.1', NULL),
(9, 'XP_038915654.1', NULL),
(9, 'XP_038915653.1', NULL),
(9, 'XP_038915652.1', NULL),
(9, 'XP_038915651.1', NULL),
(9, 'XP_038915650.1', NULL),
(9, 'XP_038915649.1', NULL),
(9, 'XP_038915648.1', NULL),
(9, 'XP_038915647.1', NULL),
(9, 'XP_038915646.1', NULL),
(9, 'XP_038915645.1', NULL),
(9, 'XP_038915644.1', NULL),
(9, 'XP_038915643.1', NULL),
(9, 'XP_038915642.1', NULL),
(9, 'XP_038915641.1', NULL),
(9, 'XP_038915640.1', NULL),
(9, 'XP_038915639.1', NULL),
(9, 'XP_038915638.1', NULL),
(9, 'XP_038915637.1', NULL),
(9, 'XP_038915636.1', NULL),
(9, 'XP_038915635.1', NULL),
(9, 'XP_038915634.1', NULL),
(9, 'XP_038915633.1', NULL),
(9, 'XP_038915632.1', NULL),
(9, 'XP_038915631.1', NULL),
(9, 'XP_038915630.1', NULL),
(9, 'XP_038915629.1', NULL),
(9, 'XP_038915628.1', NULL),
(9, 'XP_038915627.1', NULL),
(9, 'XP_038915626.1', NULL),
(9, 'XP_038915625.1', NULL),
(9, 'XP_038915624.1', NULL),
(9, 'XP_038915623.1', NULL),
(9, 'XP_038915622.1', NULL),
(9, 'XP_038915621.1', NULL),
(9, 'XP_038915620.1', NULL),
(9, 'XP_038915619.1', NULL),
(9, 'XP_038915618.1', NULL),
(9, 'XP_038915617.1', NULL),
(9, 'XP_038915616.1', NULL),
(9, 'XP_038915615.1', NULL),
(9, 'XP_038915614.1', NULL),
(9, 'XP_038915613.1', NULL),
(9, 'XP_038915612.1', NULL),
(9, 'XP_038915611.1', NULL),
(9, 'XP_038915610.1', NULL),
(9, 'XP_038915609.1', NULL),
(9, 'XP_038915608.1', NULL),
(9, 'XP_038915607.1', NULL),
(9, 'XP_038915606.1', NULL),
(9, 'XP_038915605.1', NULL),
(9, 'XP_038915604.1', NULL),
(9, 'XP_038915603.1', NULL),
(9, 'XP_038915602.1', NULL),
(9, 'XP_038915601.1', NULL),
(9, 'XP_038915600.1', NULL),
(9, 'XP_038915967.1', NULL),
(9, 'XP_038915966.1', NULL),
(9, 'XP_038915965.1', NULL),
(9, 'XP_038915964.1', NULL),
(9, 'XP_038915963.1', NULL),
(9, 'XP_038915962.1', NULL),
(9, 'XP_038915961.1', NULL),
(9, 'XP_038915960.1', NULL),
(9, 'XP_038915959.1', NULL),
(9, 'XP_038915958.1', NULL),
(9, 'XP_038915957.1', NULL),
(9, 'XP_038915956.1', NULL),
(9, 'XP_038915955.1', NULL),
(9, 'XP_038915954.1', NULL),
(9, 'XP_038915953.1', NULL),
(9, 'XP_038915952.1', NULL),
(9, 'XP_038915951.1', NULL),
(9, 'XP_038915950.1', NULL),
(9, 'XP_038915949.1', NULL),
(9, 'XP_038915948.1', NULL),
(9, 'XP_038915947.1', NULL),
(9, 'XP_038915946.1', NULL),
(9, 'XP_038915945.1', NULL),
(9, 'XP_038915944.1', NULL),
(9, 'XP_038915943.1', NULL),
(9, 'XP_038915942.1', NULL),
(9, 'XP_038915941.1', NULL),
(9, 'XP_038915940.1', NULL),
(9, 'XP_038915939.1', NULL),
(9, 'XP_038915938.1', NULL),
(9, 'XP_038915937.1', NULL),
(9, 'XP_038915936.1', NULL),
(9, 'XP_038915935.1', NULL),
(9, 'XP_038915934.1', NULL),
(9, 'XP_038915933.1', NULL),
(9, 'XP_038915932.1', NULL),
(9, 'XP_038915931.1', NULL),
(9, 'XP_038915930.1', NULL),
(9, 'XP_038915929.1', NULL),
(9, 'XP_038915928.1', NULL),
(9, 'XP_038915927.1', NULL),
(9, 'XP_038915926.1', NULL),
(9, 'XP_038915925.1', NULL),
(9, 'XP_038915924.1', NULL),
(9, 'XP_038915923.1', NULL),
(9, 'XP_038915922.1', NULL),
(9, 'XP_038915921.1', NULL),
(9, 'XP_038915920.1', NULL),
(9, 'XP_038915919.1', NULL),
(9, 'XP_038915918.1', NULL),
(9, 'XP_038915917.1', NULL),
(9, 'XP_038915916.1', NULL),
(9, 'XP_038915915.1', NULL),
(9, 'XP_038915914.1', NULL),
(9, 'XP_038915913.1', NULL),
(9, 'XP_038915912.1', NULL),
(9, 'XP_038915911.1', NULL),
(9, 'XP_038915910.1', NULL),
(9, 'XP_038915909.1', NULL),
(9, 'XP_038915908.1', NULL),
(9, 'XP_038915907.1', NULL),
(9, 'XP_038915906.1', NULL),
(9, 'XP_038915905.1', NULL),
(9, 'XP_038915904.1', NULL),
(9, 'XP_038915903.1', NULL),
(9, 'XP_038915902.1', NULL),
(9, 'XP_038915901.1', NULL),
(9, 'XP_038915900.1', NULL),
(9, 'XP_038915899.1', NULL),
(9, 'XP_038915898.1', NULL),
(9, 'XP_038915897.1', NULL),
(9, 'XP_038915896.1', NULL),
(9, 'XP_038915895.1', NULL),
(9, 'XP_038915894.1', NULL),
(9, 'XP_038915893.1', NULL),
(9, 'XP_038915892.1', NULL),
(9, 'XP_038915891.1', NULL),
(9, 'XP_038915890.1', NULL),
(9, 'XP_038915889.1', NULL),
(9, 'XP_038915888.1', NULL),
(9, 'XP_038915887.1', NULL),
(9, 'XP_038915886.1', NULL),
(9, 'XP_038915885.1', NULL),
(9, 'XP_038915884.1', NULL),
(9, 'XP_038915883.1', NULL),
(9, 'XP_038915882.1', NULL),
(9, 'XP_038915881.1', NULL),
(9, 'XP_038915880.1', NULL),
(9, 'XP_038915879.1', NULL),
(9, 'XP_038915878.1', NULL),
(9, 'XP_038915877.1', NULL),
(9, 'XP_038915876.1', NULL),
(9, 'XP_038915875.1', NULL),
(9, 'XP_038915874.1', NULL),
(9, 'XP_038915873.1', NULL),
(9, 'XP_038915872.1', NULL),
(9, 'XP_038915871.1', NULL),
(9, 'XP_038915870.1', NULL),
(9, 'XP_038915869.1', NULL),
(9, 'XP_038915868.1', NULL),
(9, 'XP_038915867.1', NULL),
(9, 'XP_038915866.1', NULL),
(9, 'XP_038915865.1', NULL),
(9, 'XP_038915864.1', NULL),
(9, 'XP_038915863.1', NULL),
(9, 'XP_038915862.1', NULL),
(9, 'XP_038915861.1', NULL),
(9, 'XP_038915860.1', NULL),
(9, 'XP_038915859.1', NULL),
(9, 'XP_038915858.1', NULL),
(9, 'XP_038915857.1', NULL),
(9, 'XP_038915856.1', NULL),
(9, 'XP_038915855.1', NULL),
(9, 'XP_038915854.1', NULL),
(9, 'XP_038915853.1', NULL),
(9, 'XP_038915852.1', NULL),
(9, 'XP_038915851.1', NULL),
(9, 'XP_038915850.1', NULL),
(9, 'XP_038915849.1', NULL),
(9, 'XP_038915848.1', NULL),
(9, 'XP_038915847.1', NULL),
(9, 'XP_038915846.1', NULL),
(9, 'XP_038915845.1', NULL),
(9, 'XP_038915844.1', NULL),
(9, 'XP_038915843.1', NULL),
(9, 'XP_038915842.1', NULL),
(9, 'XP_038915841.1', NULL),
(9, 'XP_038915840.1', NULL),
(9, 'XP_038915839.1', NULL),
(9, 'XP_038916220.1', NULL),
(9, 'XP_038916219.1', NULL),
(9, 'XP_038916218.1', NULL),
(9, 'XP_038916217.1', NULL),
(9, 'XP_038916216.1', NULL),
(9, 'XP_038916215.1', NULL),
(9, 'XP_038916214.1', NULL),
(9, 'XP_038916213.1', NULL),
(9, 'XP_038916212.1', NULL),
(9, 'XP_038916211.1', NULL),
(9, 'XP_038916210.1', NULL),
(9, 'XP_038916209.1', NULL),
(9, 'XP_038916208.1', NULL),
(9, 'XP_038916207.1', NULL),
(9, 'XP_038916206.1', NULL),
(9, 'XP_038916205.1', NULL),
(9, 'XP_038916204.1', NULL),
(9, 'XP_038916203.1', NULL),
(9, 'XP_038916202.1', NULL),
(9, 'XP_038916201.1', NULL),
(9, 'XP_038916200.1', NULL),
(9, 'XP_038916199.1', NULL),
(9, 'XP_038916198.1', NULL),
(9, 'XP_038916197.1', NULL),
(9, 'XP_038916196.1', NULL),
(9, 'XP_038916195.1', NULL),
(9, 'XP_038916194.1', NULL),
(9, 'XP_038916193.1', NULL),
(9, 'XP_038916192.1', NULL),
(9, 'XP_038916191.1', NULL),
(9, 'XP_038916190.1', NULL),
(9, 'XP_038916189.1', NULL),
(9, 'XP_038916188.1', NULL),
(9, 'XP_038916187.1', NULL),
(9, 'XP_038916186.1', NULL),
(9, 'XP_038916185.1', NULL),
(9, 'XP_038916184.1', NULL),
(9, 'XP_038916183.1', NULL),
(9, 'XP_038916182.1', NULL),
(9, 'XP_038916181.1', NULL),
(9, 'XP_038916180.1', NULL),
(9, 'XP_038916179.1', NULL),
(9, 'XP_038916178.1', NULL),
(9, 'XP_038916177.1', NULL),
(9, 'XP_038916176.1', NULL),
(9, 'XP_038916175.1', NULL),
(9, 'XP_038916174.1', NULL),
(9, 'XP_038916173.1', NULL),
(9, 'XP_038916172.1', NULL),
(9, 'XP_038916171.1', NULL),
(9, 'XP_038916170.1', NULL),
(9, 'XP_038916169.1', NULL),
(9, 'XP_038916168.1', NULL),
(9, 'XP_038916167.1', NULL),
(9, 'XP_038916166.1', NULL),
(9, 'XP_038916165.1', NULL),
(9, 'XP_038916164.1', NULL),
(9, 'XP_038916163.1', NULL),
(9, 'XP_038916162.1', NULL),
(9, 'XP_038916161.1', NULL),
(9, 'XP_038916160.1', NULL),
(9, 'XP_038916159.1', NULL),
(9, 'XP_038916158.1', NULL),
(9, 'XP_038916157.1', NULL),
(9, 'XP_038916156.1', NULL),
(9, 'XP_038916155.1', NULL),
(9, 'XP_038916154.1', NULL),
(9, 'XP_038916153.1', NULL),
(9, 'XP_038916152.1', NULL),
(9, 'XP_038916151.1', NULL),
(9, 'XP_038916150.1', NULL),
(9, 'XP_038916149.1', NULL),
(9, 'XP_038916148.1', NULL),
(9, 'XP_038916147.1', NULL),
(9, 'XP_038916146.1', NULL),
(9, 'XP_038916145.1', NULL),
(9, 'XP_038916144.1', NULL),
(9, 'XP_038916143.1', NULL),
(9, 'XP_038916142.1', NULL),
(9, 'XP_038916141.1', NULL),
(9, 'XP_038916140.1', NULL),
(9, 'XP_038916139.1', NULL),
(9, 'XP_038916138.1', NULL),
(9, 'XP_038916137.1', NULL),
(9, 'XP_038916136.1', NULL),
(9, 'XP_038916135.1', NULL),
(9, 'XP_038916134.1', NULL),
(9, 'XP_038916133.1', NULL),
(9, 'XP_038916132.1', NULL),
(9, 'XP_038916131.1', NULL),
(9, 'XP_038916130.1', NULL),
(9, 'XP_038916129.1', NULL),
(9, 'XP_038916128.1', NULL),
(9, 'XP_038916127.1', NULL),
(9, 'XP_038916126.1', NULL),
(9, 'XP_038916125.1', NULL),
(9, 'XP_038916124.1', NULL),
(9, 'XP_038916123.1', NULL),
(9, 'XP_038916122.1', NULL),
(9, 'XP_038916121.1', NULL),
(9, 'XP_038916120.1', NULL),
(9, 'XP_038916119.1', NULL),
(9, 'XP_038916118.1', NULL),
(9, 'XP_038916117.1', NULL),
(9, 'XP_038916116.1', NULL),
(9, 'XP_038916115.1', NULL),
(9, 'XP_038916114.1', NULL),
(9, 'XP_038916113.1', NULL),
(9, 'XP_038916112.1', NULL),
(9, 'XP_038916111.1', NULL),
(9, 'XP_038916110.1', NULL),
(9, 'XP_038916109.1', NULL),
(9, 'XP_038916108.1', NULL),
(9, 'XP_038916107.1', NULL),
(9, 'XP_038916106.1', NULL),
(9, 'XP_038916105.1', NULL),
(9, 'XP_038916104.1', NULL),
(9, 'XP_038916103.1', NULL),
(9, 'XP_038916102.1', NULL),
(9, 'XP_038916101.1', NULL),
(9, 'XP_038916100.1', NULL),
(9, 'XP_038916099.1', NULL),
(9, 'XP_038916098.1', NULL),
(9, 'XP_038916097.1', NULL),
(9, 'XP_038916096.1', NULL),
(9, 'XP_038916095.1', NULL),
(9, 'XP_038916094.1', NULL),
(9, 'XP_038916093.1', NULL),
(9, 'XP_038916092.1', NULL),
(9, 'XP_038916091.1', NULL),
(9, 'XP_038916090.1', NULL),
(9, 'XP_038916089.1', NULL),
(9, 'XP_038916088.1', NULL),
(9, 'XP_038916087.1', NULL),
(9, 'XP_038916086.1', NULL),
(9, 'XP_038916085.1', NULL),
(9, 'XP_038916084.1', NULL),
(9, 'XP_038916083.1', NULL),
(9, 'XP_038916082.1', NULL),
(9, 'XP_038916081.1', NULL),
(9, 'XP_038916080.1', NULL),
(9, 'XP_038916079.1', NULL),
(9, 'XP_038916078.1', NULL),
(9, 'XP_038916077.1', NULL),
(9, 'XP_038916076.1', NULL),
(9, 'XP_038916075.1', NULL),
(9, 'XP_038916074.1', NULL),
(9, 'XP_038916073.1', NULL),
(9, 'XP_038916072.1', NULL),
(9, 'XP_038916071.1', NULL),
(9, 'XP_038916070.1', NULL),
(9, 'XP_038916069.1', NULL),
(9, 'XP_038916068.1', NULL),
(9, 'XP_038916067.1', NULL),
(9, 'XP_038916066.1', NULL),
(9, 'XP_038916065.1', NULL),
(9, 'XP_038916064.1', NULL),
(9, 'XP_038916063.1', NULL),
(9, 'XP_038916062.1', NULL),
(9, 'XP_038916061.1', NULL),
(9, 'XP_038916060.1', NULL),
(9, 'XP_038916059.1', NULL),
(9, 'XP_038916058.1', NULL),
(9, 'XP_038916057.1', NULL),
(9, 'XP_038916056.1', NULL),
(9, 'XP_038916055.1', NULL),
(9, 'XP_038916054.1', NULL),
(9, 'XP_038916053.1', NULL),
(9, 'XP_038916052.1', NULL),
(9, 'XP_038916051.1', NULL),
(9, 'XP_038916050.1', NULL),
(9, 'XP_038916049.1', NULL),
(9, 'XP_038916048.1', NULL),
(9, 'XP_038916047.1', NULL),
(9, 'XP_038916046.1', NULL),
(9, 'XP_038916045.1', NULL),
(9, 'XP_038916044.1', NULL),
(9, 'XP_038916043.1', NULL),
(9, 'XP_038916042.1', NULL),
(9, 'XP_038916041.1', NULL),
(9, 'XP_038916040.1', NULL),
(9, 'XP_038916039.1', NULL),
(9, 'XP_038916038.1', NULL),
(9, 'XP_038916037.1', NULL),
(9, 'XP_038916036.1', NULL),
(9, 'XP_038916035.1', NULL),
(9, 'XP_038916034.1', NULL),
(9, 'XP_038916033.1', NULL),
(9, 'XP_038916032.1', NULL),
(9, 'XP_038916031.1', NULL),
(9, 'XP_038916030.1', NULL),
(9, 'XP_038916029.1', NULL),
(9, 'XP_038916028.1', NULL),
(9, 'XP_038916027.1', NULL),
(9, 'XP_038916026.1', NULL),
(9, 'XP_038916025.1', NULL),
(9, 'XP_038916024.1', NULL),
(9, 'XP_038916023.1', NULL),
(9, 'XP_038916022.1', NULL),
(9, 'XP_038916021.1', NULL),
(9, 'XP_038916020.1', NULL),
(9, 'XP_038916019.1', NULL),
(9, 'XP_038916018.1', NULL),
(9, 'XP_038916017.1', NULL),
(9, 'XP_038916016.1', NULL),
(9, 'XP_038916015.1', NULL),
(9, 'XP_038916014.1', NULL),
(9, 'XP_038916013.1', NULL),
(9, 'XP_038916012.1', NULL),
(9, 'XP_038916011.1', NULL),
(9, 'XP_038916010.1', NULL),
(9, 'XP_038916009.1', NULL),
(9, 'XP_038916008.1', NULL),
(9, 'XP_038916007.1', NULL),
(9, 'XP_038916006.1', NULL),
(9, 'XP_038916005.1', NULL),
(9, 'XP_038916004.1', NULL),
(9, 'XP_038916003.1', NULL),
(9, 'XP_038916002.1', NULL),
(9, 'XP_038916001.1', NULL),
(9, 'XP_038916000.1', NULL),
(9, 'XP_038915999.1', NULL),
(9, 'XP_038915998.1', NULL),
(9, 'XP_038915997.1', NULL),
(9, 'XP_038915996.1', NULL),
(9, 'XP_038915995.1', NULL),
(9, 'XP_038915994.1', NULL),
(9, 'XP_038915993.1', NULL),
(9, 'XP_038915992.1', NULL),
(9, 'XP_038915991.1', NULL),
(9, 'XP_038915990.1', NULL),
(9, 'XP_038915989.1', NULL),
(9, 'XP_038915988.1', NULL),
(9, 'XP_038915987.1', NULL),
(9, 'XP_038915986.1', NULL),
(9, 'XP_038915985.1', NULL),
(9, 'XP_038915984.1', NULL),
(9, 'XP_038915983.1', NULL),
(9, 'XP_038915982.1', NULL),
(9, 'XP_038915981.1', NULL),
(9, 'XP_038915980.1', NULL),
(9, 'XP_038915979.1', NULL),
(9, 'XP_038915978.1', NULL),
(9, 'XP_038915977.1', NULL),
(9, 'XP_038915976.1', NULL),
(9, 'XP_038915975.1', NULL),
(9, 'XP_038915974.1', NULL),
(9, 'XP_038915973.1', NULL),
(9, 'XP_038915972.1', NULL),
(9, 'XP_038915971.1', NULL),
(9, 'XP_038915970.1', NULL),
(9, 'XP_038915969.1', NULL),
(9, 'XP_038915968.1', NULL),
(9, 'XP_038916386.1', NULL),
(9, 'XP_038916385.1', NULL),
(9, 'XP_038916384.1', NULL),
(9, 'XP_038916383.1', NULL),
(9, 'XP_038916382.1', NULL),
(9, 'XP_038916381.1', NULL),
(9, 'XP_038916380.1', NULL),
(9, 'XP_038916379.1', NULL),
(9, 'XP_038916378.1', NULL),
(9, 'XP_038916377.1', NULL),
(9, 'XP_038916376.1', NULL),
(9, 'XP_038916375.1', NULL),
(9, 'XP_038916374.1', NULL),
(9, 'XP_038916373.1', NULL),
(9, 'XP_038916372.1', NULL),
(9, 'XP_038916371.1', NULL),
(9, 'XP_038916370.1', NULL),
(9, 'XP_038916369.1', NULL),
(9, 'XP_038916368.1', NULL),
(9, 'XP_038916367.1', NULL),
(9, 'XP_038916366.1', NULL),
(9, 'XP_038916365.1', NULL),
(9, 'XP_038916364.1', NULL),
(9, 'XP_038916363.1', NULL),
(9, 'XP_038916362.1', NULL),
(9, 'XP_038916361.1', NULL),
(9, 'XP_038916360.1', NULL),
(9, 'XP_038916359.1', NULL),
(9, 'XP_038916358.1', NULL),
(9, 'XP_038916357.1', NULL),
(9, 'XP_038916356.1', NULL),
(9, 'XP_038916355.1', NULL),
(9, 'XP_038916354.1', NULL),
(9, 'XP_038916353.1', NULL),
(9, 'XP_038916352.1', NULL),
(9, 'XP_038916351.1', NULL),
(9, 'XP_038916350.1', NULL),
(9, 'XP_038916349.1', NULL),
(9, 'XP_038916348.1', NULL),
(9, 'XP_038916347.1', NULL),
(9, 'XP_038916346.1', NULL),
(9, 'XP_038916345.1', NULL),
(9, 'XP_038916344.1', NULL),
(9, 'XP_038916343.1', NULL),
(9, 'XP_038916342.1', NULL),
(9, 'XP_038916341.1', NULL),
(9, 'XP_038916340.1', NULL),
(9, 'XP_038916339.1', NULL),
(9, 'XP_038916338.1', NULL),
(9, 'XP_038916337.1', NULL),
(9, 'XP_038916336.1', NULL),
(9, 'XP_038916335.1', NULL),
(9, 'XP_038916334.1', NULL),
(9, 'XP_038916333.1', NULL),
(9, 'XP_038916332.1', NULL),
(9, 'XP_038916331.1', NULL),
(9, 'XP_038916330.1', NULL),
(9, 'XP_038916329.1', NULL),
(9, 'XP_038916328.1', NULL),
(9, 'XP_038916327.1', NULL),
(9, 'XP_038916326.1', NULL),
(9, 'XP_038916325.1', NULL),
(9, 'XP_038916324.1', NULL),
(9, 'XP_038916323.1', NULL),
(9, 'XP_038916322.1', NULL),
(9, 'XP_038916321.1', NULL),
(9, 'XP_038916320.1', NULL),
(9, 'XP_038916319.1', NULL),
(9, 'XP_038916318.1', NULL),
(9, 'XP_038916317.1', NULL),
(9, 'XP_038916316.1', NULL),
(9, 'XP_038916315.1', NULL),
(9, 'XP_038916314.1', NULL),
(9, 'XP_038916313.1', NULL),
(9, 'XP_038916312.1', NULL),
(9, 'XP_038916311.1', NULL),
(9, 'XP_038916310.1', NULL),
(9, 'XP_038916309.1', NULL),
(9, 'XP_038916308.1', NULL),
(9, 'XP_038916307.1', NULL),
(9, 'XP_038916306.1', NULL),
(9, 'XP_038916305.1', NULL),
(9, 'XP_038916304.1', NULL),
(9, 'XP_038916303.1', NULL),
(9, 'XP_038916302.1', NULL),
(9, 'XP_038916301.1', NULL),
(9, 'XP_038916300.1', NULL),
(9, 'XP_038916299.1', NULL),
(9, 'XP_038916298.1', NULL),
(9, 'XP_038916297.1', NULL),
(9, 'XP_038916296.1', NULL),
(9, 'XP_038916295.1', NULL),
(9, 'XP_038916294.1', NULL),
(9, 'XP_038916293.1', NULL),
(9, 'XP_038916292.1', NULL),
(9, 'XP_038916291.1', NULL),
(9, 'XP_038916290.1', NULL),
(9, 'XP_038916289.1', NULL),
(9, 'XP_038916288.1', NULL),
(9, 'XP_038916287.1', NULL),
(9, 'XP_038916286.1', NULL),
(9, 'XP_038916285.1', NULL),
(9, 'XP_038916284.1', NULL),
(9, 'XP_038916283.1', NULL),
(9, 'XP_038916282.1', NULL),
(9, 'XP_038916281.1', NULL),
(9, 'XP_038916280.1', NULL),
(9, 'XP_038916279.1', NULL),
(9, 'XP_038916278.1', NULL),
(9, 'XP_038916277.1', NULL),
(9, 'XP_038916276.1', NULL),
(9, 'XP_038916275.1', NULL),
(9, 'XP_038916274.1', NULL),
(9, 'XP_038916273.1', NULL),
(9, 'XP_038916272.1', NULL),
(9, 'XP_038916271.1', NULL),
(9, 'XP_038916270.1', NULL),
(9, 'XP_038916269.1', NULL),
(9, 'XP_038916268.1', NULL),
(9, 'XP_038916267.1', NULL),
(9, 'XP_038916266.1', NULL),
(9, 'XP_038916265.1', NULL),
(9, 'XP_038916264.1', NULL),
(9, 'XP_038916263.1', NULL),
(9, 'XP_038916262.1', NULL),
(9, 'XP_038916261.1', NULL),
(9, 'XP_038916260.1', NULL),
(9, 'XP_038916259.1', NULL),
(9, 'XP_038916258.1', NULL),
(9, 'XP_038916257.1', NULL),
(9, 'XP_038916256.1', NULL),
(9, 'XP_038916255.1', NULL),
(9, 'XP_038916254.1', NULL),
(9, 'XP_038916253.1', NULL),
(9, 'XP_038916252.1', NULL),
(9, 'XP_038916251.1', NULL),
(9, 'XP_038916250.1', NULL),
(9, 'XP_038916249.1', NULL),
(9, 'XP_038916248.1', NULL),
(9, 'XP_038916247.1', NULL),
(9, 'XP_038916246.1', NULL),
(9, 'XP_038916245.1', NULL),
(9, 'XP_038916244.1', NULL),
(9, 'XP_038916243.1', NULL),
(9, 'XP_038916242.1', NULL),
(9, 'XP_038916241.1', NULL),
(9, 'XP_038916240.1', NULL),
(9, 'XP_038916239.1', NULL),
(9, 'XP_038916238.1', NULL),
(9, 'XP_038916237.1', NULL),
(9, 'XP_038916236.1', NULL),
(9, 'XP_038916235.1', NULL),
(9, 'XP_038916234.1', NULL),
(9, 'XP_038916233.1', NULL),
(9, 'XP_038916232.1', NULL),
(9, 'XP_038916231.1', NULL),
(9, 'XP_038916230.1', NULL),
(9, 'XP_038916229.1', NULL),
(9, 'XP_038916228.1', NULL),
(9, 'XP_038916227.1', NULL),
(9, 'XP_038916226.1', NULL),
(9, 'XP_038916225.1', NULL),
(9, 'XP_038916224.1', NULL),
(9, 'XP_038916223.1', NULL),
(9, 'XP_038916222.1', NULL),
(9, 'XP_038916221.1', NULL),
(9, 'XP_038916676.1', NULL),
(9, 'XP_038916675.1', NULL),
(9, 'XP_038916674.1', NULL),
(9, 'XP_038916673.1', NULL),
(9, 'XP_038916672.1', NULL),
(9, 'XP_038916671.1', NULL),
(9, 'XP_038916670.1', NULL),
(9, 'XP_038916669.1', NULL),
(9, 'XP_038916668.1', NULL),
(9, 'XP_038916667.1', NULL),
(9, 'XP_038916666.1', NULL),
(9, 'XP_038916665.1', NULL),
(9, 'XP_038916664.1', NULL),
(9, 'XP_038916663.1', NULL),
(9, 'XP_038916662.1', NULL),
(9, 'XP_038916661.1', NULL),
(9, 'XP_038916660.1', NULL),
(9, 'XP_038916659.1', NULL),
(9, 'XP_038916658.1', NULL),
(9, 'XP_038916657.1', NULL),
(9, 'XP_038916656.1', NULL),
(9, 'XP_038916655.1', NULL),
(9, 'XP_038916654.1', NULL),
(9, 'XP_038916653.1', NULL),
(9, 'XP_038916652.1', NULL),
(9, 'XP_038916651.1', NULL),
(9, 'XP_038916650.1', NULL),
(9, 'XP_038916649.1', NULL),
(9, 'XP_038916648.1', NULL),
(9, 'XP_038916647.1', NULL),
(9, 'XP_038916646.1', NULL),
(9, 'XP_038916645.1', NULL),
(9, 'XP_038916644.1', NULL),
(9, 'XP_038916643.1', NULL),
(9, 'XP_038916642.1', NULL),
(9, 'XP_038916641.1', NULL),
(9, 'XP_038916640.1', NULL),
(9, 'XP_038916639.1', NULL),
(9, 'XP_038916638.1', NULL),
(9, 'XP_038916637.1', NULL),
(9, 'XP_038916636.1', NULL),
(9, 'XP_038916635.1', NULL),
(9, 'XP_038916634.1', NULL),
(9, 'XP_038916633.1', NULL),
(9, 'XP_038916632.1', NULL),
(9, 'XP_038916631.1', NULL),
(9, 'XP_038916630.1', NULL),
(9, 'XP_038916629.1', NULL),
(9, 'XP_038916628.1', NULL),
(9, 'XP_038916627.1', NULL),
(9, 'XP_038916626.1', NULL),
(9, 'XP_038916625.1', NULL),
(9, 'XP_038916624.1', NULL),
(9, 'XP_038916623.1', NULL),
(9, 'XP_038916622.1', NULL),
(9, 'XP_038916621.1', NULL),
(9, 'XP_038916620.1', NULL),
(9, 'XP_038916619.1', NULL),
(9, 'XP_038916618.1', NULL),
(9, 'XP_038916617.1', NULL),
(9, 'XP_038916616.1', NULL),
(9, 'XP_038916615.1', NULL),
(9, 'XP_038916614.1', NULL),
(9, 'XP_038916613.1', NULL),
(9, 'XP_038916612.1', NULL),
(9, 'XP_038916611.1', NULL),
(9, 'XP_038916610.1', NULL),
(9, 'XP_038916609.1', NULL),
(9, 'XP_038916608.1', NULL),
(9, 'XP_038916607.1', NULL),
(9, 'XP_038916606.1', NULL),
(9, 'XP_038916605.1', NULL),
(9, 'XP_038916604.1', NULL),
(9, 'XP_038916603.1', NULL),
(9, 'XP_038916602.1', NULL),
(9, 'XP_038916601.1', NULL),
(9, 'XP_038916600.1', NULL),
(9, 'XP_038916599.1', NULL),
(9, 'XP_038916598.1', NULL),
(9, 'XP_038916597.1', NULL),
(9, 'XP_038916596.1', NULL),
(9, 'XP_038916595.1', NULL),
(9, 'XP_038916594.1', NULL),
(9, 'XP_038916593.1', NULL),
(9, 'XP_038916592.1', NULL),
(9, 'XP_038916591.1', NULL),
(9, 'XP_038916590.1', NULL),
(9, 'XP_038916589.1', NULL),
(9, 'XP_038916588.1', NULL),
(9, 'XP_038916587.1', NULL),
(9, 'XP_038916586.1', NULL),
(9, 'XP_038916585.1', NULL),
(9, 'XP_038916584.1', NULL),
(9, 'XP_038916583.1', NULL),
(9, 'XP_038916582.1', NULL),
(9, 'XP_038916581.1', NULL),
(9, 'XP_038916580.1', NULL),
(9, 'XP_038916579.1', NULL),
(9, 'XP_038916578.1', NULL),
(9, 'XP_038916577.1', NULL),
(9, 'XP_038916576.1', NULL),
(9, 'XP_038916575.1', NULL),
(9, 'XP_038916574.1', NULL),
(9, 'XP_038916573.1', NULL),
(9, 'XP_038916572.1', NULL),
(9, 'XP_038916571.1', NULL),
(9, 'XP_038916570.1', NULL),
(9, 'XP_038916569.1', NULL),
(9, 'XP_038916568.1', NULL),
(9, 'XP_038916567.1', NULL),
(9, 'XP_038916566.1', NULL),
(9, 'XP_038916565.1', NULL),
(9, 'XP_038916564.1', NULL),
(9, 'XP_038916563.1', NULL),
(9, 'XP_038916562.1', NULL),
(9, 'XP_038916561.1', NULL),
(9, 'XP_038916560.1', NULL),
(9, 'XP_038916559.1', NULL),
(9, 'XP_038916558.1', NULL),
(9, 'XP_038916557.1', NULL),
(9, 'XP_038916556.1', NULL),
(9, 'XP_038916555.1', NULL),
(9, 'XP_038916554.1', NULL),
(9, 'XP_038916553.1', NULL),
(9, 'XP_038916552.1', NULL),
(9, 'XP_038916551.1', NULL),
(9, 'XP_038916550.1', NULL),
(9, 'XP_038916549.1', NULL),
(9, 'XP_038916548.1', NULL),
(9, 'XP_038916547.1', NULL),
(9, 'XP_038916546.1', NULL),
(9, 'XP_038916545.1', NULL),
(9, 'XP_038916544.1', NULL),
(9, 'XP_038916543.1', NULL),
(9, 'XP_038916542.1', NULL),
(9, 'XP_038916541.1', NULL);
INSERT INTO `prot_access_table` (`id_article`, `prot_access`, `entry_id_cazy`) VALUES
(9, 'XP_038916540.1', NULL),
(9, 'XP_038916539.1', NULL),
(9, 'XP_038916538.1', NULL),
(9, 'XP_038916537.1', NULL),
(9, 'XP_038916536.1', NULL),
(9, 'XP_038916535.1', NULL),
(9, 'XP_038916534.1', NULL),
(9, 'XP_038916533.1', NULL),
(9, 'XP_038916532.1', NULL),
(9, 'XP_038916531.1', NULL),
(9, 'XP_038916530.1', NULL),
(9, 'XP_038916529.1', NULL),
(9, 'XP_038916528.1', NULL),
(9, 'XP_038916527.1', NULL),
(9, 'XP_038916526.1', NULL),
(9, 'XP_038916525.1', NULL),
(9, 'XP_038916524.1', NULL),
(9, 'XP_038916523.1', NULL),
(9, 'XP_038916522.1', NULL),
(9, 'XP_038916521.1', NULL),
(9, 'XP_038916520.1', NULL),
(9, 'XP_038916519.1', NULL),
(9, 'XP_038916518.1', NULL),
(9, 'XP_038916517.1', NULL),
(9, 'XP_038916516.1', NULL),
(9, 'XP_038916515.1', NULL),
(9, 'XP_038916514.1', NULL),
(9, 'XP_038916513.1', NULL),
(9, 'XP_038916512.1', NULL),
(9, 'XP_038916511.1', NULL),
(9, 'XP_038916510.1', NULL),
(9, 'XP_038916509.1', NULL),
(9, 'XP_038916508.1', NULL),
(9, 'XP_038916507.1', NULL),
(9, 'XP_038916506.1', NULL),
(9, 'XP_038916505.1', NULL),
(9, 'XP_038916504.1', NULL),
(9, 'XP_038916503.1', NULL),
(9, 'XP_038916502.1', NULL),
(9, 'XP_038916501.1', NULL),
(9, 'XP_038916500.1', NULL),
(9, 'XP_038916499.1', NULL),
(9, 'XP_038916498.1', NULL),
(9, 'XP_038916497.1', NULL),
(9, 'XP_038916496.1', NULL),
(9, 'XP_038916495.1', NULL),
(9, 'XP_038916494.1', NULL),
(9, 'XP_038916493.1', NULL),
(9, 'XP_038916492.1', NULL),
(9, 'XP_038916491.1', NULL),
(9, 'XP_038916490.1', NULL),
(9, 'XP_038916489.1', NULL),
(9, 'XP_038916488.1', NULL),
(9, 'XP_038916487.1', NULL),
(9, 'XP_038916486.1', NULL),
(9, 'XP_038916485.1', NULL),
(9, 'XP_038916484.1', NULL),
(9, 'XP_038916483.1', NULL),
(9, 'XP_038916482.1', NULL),
(9, 'XP_038916481.1', NULL),
(9, 'XP_038916480.1', NULL),
(9, 'XP_038916479.1', NULL),
(9, 'XP_038916478.1', NULL),
(9, 'XP_038916477.1', NULL),
(9, 'XP_038916476.1', NULL),
(9, 'XP_038916475.1', NULL),
(9, 'XP_038916474.1', NULL),
(9, 'XP_038916473.1', NULL),
(9, 'XP_038916472.1', NULL),
(9, 'XP_038916471.1', NULL),
(9, 'XP_038916470.1', NULL),
(9, 'XP_038916469.1', NULL),
(9, 'XP_038916468.1', NULL),
(9, 'XP_038916467.1', NULL),
(9, 'XP_038916466.1', NULL),
(9, 'XP_038916465.1', NULL),
(9, 'XP_038916464.1', NULL),
(9, 'XP_038916463.1', NULL),
(9, 'XP_038916462.1', NULL),
(9, 'XP_038916461.1', NULL),
(9, 'XP_038916460.1', NULL),
(9, 'XP_038916459.1', NULL),
(9, 'XP_038916458.1', NULL),
(9, 'XP_038916457.1', NULL),
(9, 'XP_038916456.1', NULL),
(9, 'XP_038916455.1', NULL),
(9, 'XP_038916454.1', NULL),
(9, 'XP_038916453.1', NULL),
(9, 'XP_038916452.1', NULL),
(9, 'XP_038916451.1', NULL),
(9, 'XP_038916450.1', NULL),
(9, 'XP_038916449.1', NULL),
(9, 'XP_038916448.1', NULL),
(9, 'XP_038916447.1', NULL),
(9, 'XP_038916446.1', NULL),
(9, 'XP_038916445.1', NULL),
(9, 'XP_038916444.1', NULL),
(9, 'XP_038916443.1', NULL),
(9, 'XP_038916442.1', NULL),
(9, 'XP_038916441.1', NULL),
(9, 'XP_038916440.1', NULL),
(9, 'XP_038916439.1', NULL),
(9, 'XP_038916438.1', NULL),
(9, 'XP_038916437.1', NULL),
(9, 'XP_038916436.1', NULL),
(9, 'XP_038916435.1', NULL),
(9, 'XP_038916434.1', NULL),
(9, 'XP_038916433.1', NULL),
(9, 'XP_038916432.1', NULL),
(9, 'XP_038916431.1', NULL),
(9, 'XP_038916430.1', NULL),
(9, 'XP_038916429.1', NULL),
(9, 'XP_038916428.1', NULL),
(9, 'XP_038916427.1', NULL),
(9, 'XP_038916426.1', NULL),
(9, 'XP_038916425.1', NULL),
(9, 'XP_038916424.1', NULL),
(9, 'XP_038916423.1', NULL),
(9, 'XP_038916422.1', NULL),
(9, 'XP_038916421.1', NULL),
(9, 'XP_038916420.1', NULL),
(9, 'XP_038916419.1', NULL),
(9, 'XP_038916418.1', NULL),
(9, 'XP_038916417.1', NULL),
(9, 'XP_038916416.1', NULL),
(9, 'XP_038916415.1', NULL),
(9, 'XP_038916414.1', NULL),
(9, 'XP_038916413.1', NULL),
(9, 'XP_038916412.1', NULL),
(9, 'XP_038916411.1', NULL),
(9, 'XP_038916410.1', NULL),
(9, 'XP_038916409.1', NULL),
(9, 'XP_038916408.1', NULL),
(9, 'XP_038916407.1', NULL),
(9, 'XP_038916406.1', NULL),
(9, 'XP_038916405.1', NULL),
(9, 'XP_038916404.1', NULL),
(9, 'XP_038916403.1', NULL),
(9, 'XP_038916402.1', NULL),
(9, 'XP_038916401.1', NULL),
(9, 'XP_038916400.1', NULL),
(9, 'XP_038916399.1', NULL),
(9, 'XP_038916398.1', NULL),
(9, 'XP_038916397.1', NULL),
(9, 'XP_038916396.1', NULL),
(9, 'XP_038916395.1', NULL),
(9, 'XP_038916394.1', NULL),
(9, 'XP_038916393.1', NULL),
(9, 'XP_038916392.1', NULL),
(9, 'XP_038916391.1', NULL),
(9, 'XP_038916390.1', NULL),
(9, 'XP_038916389.1', NULL),
(9, 'XP_038916388.1', NULL),
(9, 'XP_038916387.1', NULL),
(9, 'XP_038916865.1', NULL),
(9, 'XP_038916864.1', NULL),
(9, 'XP_038916863.1', NULL),
(9, 'XP_038916862.1', NULL),
(9, 'XP_038916861.1', NULL),
(9, 'XP_038916860.1', NULL),
(9, 'XP_038916859.1', NULL),
(9, 'XP_038916858.1', NULL),
(9, 'XP_038916857.1', NULL),
(9, 'XP_038916856.1', NULL),
(9, 'XP_038916855.1', NULL),
(9, 'XP_038916854.1', NULL),
(9, 'XP_038916853.1', NULL),
(9, 'XP_038916852.1', NULL),
(9, 'XP_038916851.1', NULL),
(9, 'XP_038916850.1', NULL),
(9, 'XP_038916849.1', NULL),
(9, 'XP_038916848.1', NULL),
(9, 'XP_038916847.1', NULL),
(9, 'XP_038916846.1', NULL),
(9, 'XP_038916845.1', NULL),
(9, 'XP_038916844.1', NULL),
(9, 'XP_038916843.1', NULL),
(9, 'XP_038916842.1', NULL),
(9, 'XP_038916841.1', NULL),
(9, 'XP_038916840.1', NULL),
(9, 'XP_038916839.1', NULL),
(9, 'XP_038916838.1', NULL),
(9, 'XP_038916837.1', NULL),
(9, 'XP_038916836.1', NULL),
(9, 'XP_038916835.1', NULL),
(9, 'XP_038916834.1', NULL),
(9, 'XP_038916833.1', NULL),
(9, 'XP_038916832.1', NULL),
(9, 'XP_038916831.1', NULL),
(9, 'XP_038916830.1', NULL),
(9, 'XP_038916829.1', NULL),
(9, 'XP_038916828.1', NULL),
(9, 'XP_038916827.1', NULL),
(9, 'XP_038916826.1', NULL),
(9, 'XP_038916825.1', NULL),
(9, 'XP_038916824.1', NULL),
(9, 'XP_038916823.1', NULL),
(9, 'XP_038916822.1', NULL),
(9, 'XP_038916821.1', NULL),
(9, 'XP_038916820.1', NULL),
(9, 'XP_038916819.1', NULL),
(9, 'XP_038916818.1', NULL),
(9, 'XP_038916817.1', NULL),
(9, 'XP_038916816.1', NULL),
(9, 'XP_038916815.1', NULL),
(9, 'XP_038916814.1', NULL),
(9, 'XP_038916813.1', NULL),
(9, 'XP_038916812.1', NULL),
(9, 'XP_038916811.1', NULL),
(9, 'XP_038916810.1', NULL),
(9, 'XP_038916809.1', NULL),
(9, 'XP_038916808.1', NULL),
(9, 'XP_038916807.1', NULL),
(9, 'XP_038916806.1', NULL),
(9, 'XP_038916805.1', NULL),
(9, 'XP_038916804.1', NULL),
(9, 'XP_038916803.1', NULL),
(9, 'XP_038916802.1', NULL),
(9, 'XP_038916801.1', NULL),
(9, 'XP_038916800.1', NULL),
(9, 'XP_038916799.1', NULL),
(9, 'XP_038916798.1', NULL),
(9, 'XP_038916797.1', NULL),
(9, 'XP_038916796.1', NULL),
(9, 'XP_038916795.1', NULL),
(9, 'XP_038916794.1', NULL),
(9, 'XP_038916793.1', NULL),
(9, 'XP_038916792.1', NULL),
(9, 'XP_038916791.1', NULL),
(9, 'XP_038916790.1', NULL),
(9, 'XP_038916789.1', NULL),
(9, 'XP_038916788.1', NULL),
(9, 'XP_038916787.1', NULL),
(9, 'XP_038916786.1', NULL),
(9, 'XP_038916785.1', NULL),
(9, 'XP_038916784.1', NULL),
(9, 'XP_038916783.1', NULL),
(9, 'XP_038916782.1', NULL),
(9, 'XP_038916781.1', NULL),
(9, 'XP_038916780.1', NULL),
(9, 'XP_038916779.1', NULL),
(9, 'XP_038916778.1', NULL),
(9, 'XP_038916777.1', NULL),
(9, 'XP_038916776.1', NULL),
(9, 'XP_038916775.1', NULL),
(9, 'XP_038916774.1', NULL),
(9, 'XP_038916773.1', NULL),
(9, 'XP_038916772.1', NULL),
(9, 'XP_038916771.1', NULL),
(9, 'XP_038916770.1', NULL),
(9, 'XP_038916769.1', NULL),
(9, 'XP_038916768.1', NULL),
(9, 'XP_038916767.1', NULL),
(9, 'XP_038916766.1', NULL),
(9, 'XP_038916765.1', NULL),
(9, 'XP_038916764.1', NULL),
(9, 'XP_038916763.1', NULL),
(9, 'XP_038916762.1', NULL),
(9, 'XP_038916761.1', NULL),
(9, 'XP_038916760.1', NULL),
(9, 'XP_038916759.1', NULL),
(9, 'XP_038916758.1', NULL),
(9, 'XP_038916757.1', NULL),
(9, 'XP_038916756.1', NULL),
(9, 'XP_038916755.1', NULL),
(9, 'XP_038916754.1', NULL),
(9, 'XP_038916753.1', NULL),
(9, 'XP_038916752.1', NULL),
(9, 'XP_038916751.1', NULL),
(9, 'XP_038916750.1', NULL),
(9, 'XP_038916749.1', NULL),
(9, 'XP_038916748.1', NULL),
(9, 'XP_038916747.1', NULL),
(9, 'XP_038916746.1', NULL),
(9, 'XP_038916745.1', NULL),
(9, 'XP_038916744.1', NULL),
(9, 'XP_038916743.1', NULL),
(9, 'XP_038916742.1', NULL),
(9, 'XP_038916741.1', NULL),
(9, 'XP_038916740.1', NULL),
(9, 'XP_038916739.1', NULL),
(9, 'XP_038916738.1', NULL),
(9, 'XP_038916737.1', NULL),
(9, 'XP_038916736.1', NULL),
(9, 'XP_038916735.1', NULL),
(9, 'XP_038916734.1', NULL),
(9, 'XP_038916733.1', NULL),
(9, 'XP_038916732.1', NULL),
(9, 'XP_038916731.1', NULL),
(9, 'XP_038916730.1', NULL),
(9, 'XP_038916729.1', NULL),
(9, 'XP_038916728.1', NULL),
(9, 'XP_038916727.1', NULL),
(9, 'XP_038916726.1', NULL),
(9, 'XP_038916725.1', NULL),
(9, 'XP_038916724.1', NULL),
(9, 'XP_038916723.1', NULL),
(9, 'XP_038916722.1', NULL),
(9, 'XP_038916721.1', NULL),
(9, 'XP_038916720.1', NULL),
(9, 'XP_038916719.1', NULL),
(9, 'XP_038916718.1', NULL),
(9, 'XP_038916717.1', NULL),
(9, 'XP_038916716.1', NULL),
(9, 'XP_038916715.1', NULL),
(9, 'XP_038916714.1', NULL),
(9, 'XP_038916713.1', NULL),
(9, 'XP_038916712.1', NULL),
(9, 'XP_038916711.1', NULL),
(9, 'XP_038916710.1', NULL),
(9, 'XP_038916709.1', NULL),
(9, 'XP_038916708.1', NULL),
(9, 'XP_038916707.1', NULL),
(9, 'XP_038916706.1', NULL),
(9, 'XP_038916705.1', NULL),
(9, 'XP_038916704.1', NULL),
(9, 'XP_038916703.1', NULL),
(9, 'XP_038916702.1', NULL),
(9, 'XP_038916701.1', NULL),
(9, 'XP_038916700.1', NULL),
(9, 'XP_038916699.1', NULL),
(9, 'XP_038916698.1', NULL),
(9, 'XP_038916697.1', NULL),
(9, 'XP_038916696.1', NULL),
(9, 'XP_038916695.1', NULL),
(9, 'XP_038916694.1', NULL),
(9, 'XP_038916693.1', NULL),
(9, 'XP_038916692.1', NULL),
(9, 'XP_038916691.1', NULL),
(9, 'XP_038916690.1', NULL),
(9, 'XP_038916689.1', NULL),
(9, 'XP_038916688.1', NULL),
(9, 'XP_038916687.1', NULL),
(9, 'XP_038916686.1', NULL),
(9, 'XP_038916685.1', NULL),
(9, 'XP_038916684.1', NULL),
(9, 'XP_038916683.1', NULL),
(9, 'XP_038916682.1', NULL),
(9, 'XP_038916681.1', NULL),
(9, 'XP_038916680.1', NULL),
(9, 'XP_038916679.1', NULL),
(9, 'XP_038916678.1', NULL),
(9, 'XP_038916677.1', NULL),
(9, 'XP_038917163.1', NULL),
(9, 'XP_038917162.1', NULL),
(9, 'XP_038917161.1', NULL),
(9, 'XP_038917160.1', NULL),
(9, 'XP_038917159.1', NULL),
(9, 'XP_038917158.1', NULL),
(9, 'XP_038917157.1', NULL),
(9, 'XP_038917156.1', NULL),
(9, 'XP_038917155.1', NULL),
(9, 'XP_038917154.1', NULL),
(9, 'XP_038917153.1', NULL),
(9, 'XP_038917152.1', NULL),
(9, 'XP_038917151.1', NULL),
(9, 'XP_038917150.1', NULL),
(9, 'XP_038917149.1', NULL),
(9, 'XP_038917148.1', NULL),
(9, 'XP_038917147.1', NULL),
(9, 'XP_038917146.1', NULL),
(9, 'XP_038917145.1', NULL),
(9, 'XP_038917144.1', NULL),
(9, 'XP_038917143.1', NULL),
(9, 'XP_038917142.1', NULL),
(9, 'XP_038917141.1', NULL),
(9, 'XP_038917140.1', NULL),
(9, 'XP_038917139.1', NULL),
(9, 'XP_038917138.1', NULL),
(9, 'XP_038917137.1', NULL),
(9, 'XP_038917136.1', NULL),
(9, 'XP_038917135.1', NULL),
(9, 'XP_038917134.1', NULL),
(9, 'XP_038917133.1', NULL),
(9, 'XP_038917132.1', NULL),
(9, 'XP_038917131.1', NULL),
(9, 'XP_038917130.1', NULL),
(9, 'XP_038917129.1', NULL),
(9, 'XP_038917128.1', NULL),
(9, 'XP_038917127.1', NULL),
(9, 'XP_038917126.1', NULL),
(9, 'XP_038917125.1', NULL),
(9, 'XP_038917124.1', NULL),
(9, 'XP_038917123.1', NULL),
(9, 'XP_038917122.1', NULL),
(9, 'XP_038917121.1', NULL),
(9, 'XP_038917120.1', NULL),
(9, 'XP_038917119.1', NULL),
(9, 'XP_038917118.1', NULL),
(9, 'XP_038917117.1', NULL),
(9, 'XP_038917116.1', NULL),
(9, 'XP_038917115.1', NULL),
(9, 'XP_038917114.1', NULL),
(9, 'XP_038917113.1', NULL),
(9, 'XP_038917112.1', NULL),
(9, 'XP_038917111.1', NULL),
(9, 'XP_038917110.1', NULL),
(9, 'XP_038917109.1', NULL),
(9, 'XP_038917108.1', NULL),
(9, 'XP_038917107.1', NULL),
(9, 'XP_038917106.1', NULL),
(9, 'XP_038917105.1', NULL),
(9, 'XP_038917104.1', NULL),
(9, 'XP_038917103.1', NULL),
(9, 'XP_038917102.1', NULL),
(9, 'XP_038917101.1', NULL),
(9, 'XP_038917100.1', NULL),
(9, 'XP_038917099.1', NULL),
(9, 'XP_038917098.1', NULL),
(9, 'XP_038917097.1', NULL),
(9, 'XP_038917096.1', NULL),
(9, 'XP_038917095.1', NULL),
(9, 'XP_038917094.1', NULL),
(9, 'XP_038917093.1', NULL),
(9, 'XP_038917092.1', NULL),
(9, 'XP_038917091.1', NULL),
(9, 'XP_038917090.1', NULL),
(9, 'XP_038917089.1', NULL),
(9, 'XP_038917088.1', NULL),
(9, 'XP_038917087.1', NULL),
(9, 'XP_038917086.1', NULL),
(9, 'XP_038917085.1', NULL),
(9, 'XP_038917084.1', NULL),
(9, 'XP_038917083.1', NULL),
(9, 'XP_038917082.1', NULL),
(9, 'XP_038917081.1', NULL),
(9, 'XP_038917080.1', NULL),
(9, 'XP_038917079.1', NULL),
(9, 'XP_038917078.1', NULL),
(9, 'XP_038917077.1', NULL),
(9, 'XP_038917076.1', NULL),
(9, 'XP_038917075.1', NULL),
(9, 'XP_038917074.1', NULL),
(9, 'XP_038917073.1', NULL),
(9, 'XP_038917072.1', NULL),
(9, 'XP_038917071.1', NULL),
(9, 'XP_038917070.1', NULL),
(9, 'XP_038917069.1', NULL),
(9, 'XP_038917068.1', NULL),
(9, 'XP_038917067.1', NULL),
(9, 'XP_038917066.1', NULL),
(9, 'XP_038917065.1', NULL),
(9, 'XP_038917064.1', NULL),
(9, 'XP_038917063.1', NULL),
(9, 'XP_038917062.1', NULL),
(9, 'XP_038917061.1', NULL),
(9, 'XP_038917060.1', NULL),
(9, 'XP_038917059.1', NULL),
(9, 'XP_038917058.1', NULL),
(9, 'XP_038917057.1', NULL),
(9, 'XP_038917056.1', NULL),
(9, 'XP_038917055.1', NULL),
(9, 'XP_038917054.1', NULL),
(9, 'XP_038917053.1', NULL),
(9, 'XP_038917052.1', NULL),
(9, 'XP_038917051.1', NULL),
(9, 'XP_038917050.1', NULL),
(9, 'XP_038917049.1', NULL),
(9, 'XP_038917048.1', NULL),
(9, 'XP_038917047.1', NULL),
(9, 'XP_038917046.1', NULL),
(9, 'XP_038917045.1', NULL),
(9, 'XP_038917044.1', NULL),
(9, 'XP_038917043.1', NULL),
(9, 'XP_038917042.1', NULL),
(9, 'XP_038917041.1', NULL),
(9, 'XP_038917040.1', NULL),
(9, 'XP_038917039.1', NULL),
(9, 'XP_038917038.1', NULL),
(9, 'XP_038917037.1', NULL),
(9, 'XP_038917036.1', NULL),
(9, 'XP_038917035.1', NULL),
(9, 'XP_038917034.1', NULL),
(9, 'XP_038917033.1', NULL),
(9, 'XP_038917032.1', NULL),
(9, 'XP_038917031.1', NULL),
(9, 'XP_038917030.1', NULL),
(9, 'XP_038917029.1', NULL),
(9, 'XP_038917028.1', NULL),
(9, 'XP_038917027.1', NULL),
(9, 'XP_038917026.1', NULL),
(9, 'XP_038917025.1', NULL),
(9, 'XP_038917024.1', NULL),
(9, 'XP_038917023.1', NULL),
(9, 'XP_038917022.1', NULL),
(9, 'XP_038917021.1', NULL),
(9, 'XP_038917020.1', NULL),
(9, 'XP_038917019.1', NULL),
(9, 'XP_038917018.1', NULL),
(9, 'XP_038917017.1', NULL),
(9, 'XP_038917016.1', NULL),
(9, 'XP_038917015.1', NULL),
(9, 'XP_038917014.1', NULL),
(9, 'XP_038917013.1', NULL),
(9, 'XP_038917012.1', NULL),
(9, 'XP_038917011.1', NULL),
(9, 'XP_038917010.1', NULL),
(9, 'XP_038917009.1', NULL),
(9, 'XP_038917008.1', NULL),
(9, 'XP_038917007.1', NULL),
(9, 'XP_038917006.1', NULL),
(9, 'XP_038917005.1', NULL),
(9, 'XP_038917004.1', NULL),
(9, 'XP_038917003.1', NULL),
(9, 'XP_038917002.1', NULL),
(9, 'XP_038917001.1', NULL),
(9, 'XP_038917000.1', NULL),
(9, 'XP_038916999.1', NULL),
(9, 'XP_038916998.1', NULL),
(9, 'XP_038916997.1', NULL),
(9, 'XP_038916996.1', NULL),
(9, 'XP_038916995.1', NULL),
(9, 'XP_038916994.1', NULL),
(9, 'XP_038916993.1', NULL),
(9, 'XP_038916992.1', NULL),
(9, 'XP_038916991.1', NULL),
(9, 'XP_038916990.1', NULL),
(9, 'XP_038916989.1', NULL),
(9, 'XP_038916988.1', NULL),
(9, 'XP_038916987.1', NULL),
(9, 'XP_038916986.1', NULL),
(9, 'XP_038916985.1', NULL),
(9, 'XP_038916984.1', NULL),
(9, 'XP_038916983.1', NULL),
(9, 'XP_038916982.1', NULL),
(9, 'XP_038916981.1', NULL),
(9, 'XP_038916980.1', NULL),
(9, 'XP_038916979.1', NULL),
(9, 'XP_038916978.1', NULL),
(9, 'XP_038916977.1', NULL),
(9, 'XP_038916976.1', NULL),
(9, 'XP_038916975.1', NULL),
(9, 'XP_038916974.1', NULL),
(9, 'XP_038916973.1', NULL),
(9, 'XP_038916972.1', NULL),
(9, 'XP_038916971.1', NULL),
(9, 'XP_038916970.1', NULL),
(9, 'XP_038916969.1', NULL),
(9, 'XP_038916968.1', NULL),
(9, 'XP_038916967.1', NULL),
(9, 'XP_038916966.1', NULL),
(9, 'XP_038916965.1', NULL),
(9, 'XP_038916964.1', NULL),
(9, 'XP_038916963.1', NULL),
(9, 'XP_038916962.1', NULL),
(9, 'XP_038916961.1', NULL),
(9, 'XP_038916960.1', NULL),
(9, 'XP_038916959.1', NULL),
(9, 'XP_038916958.1', NULL),
(9, 'XP_038916957.1', NULL),
(9, 'XP_038916956.1', NULL),
(9, 'XP_038916955.1', NULL),
(9, 'XP_038916954.1', NULL),
(9, 'XP_038916953.1', NULL),
(9, 'XP_038916952.1', NULL),
(9, 'XP_038916951.1', NULL),
(9, 'XP_038916950.1', NULL),
(9, 'XP_038916949.1', NULL),
(9, 'XP_038916948.1', NULL),
(9, 'XP_038916947.1', NULL),
(9, 'XP_038916946.1', NULL),
(9, 'XP_038916945.1', NULL),
(9, 'XP_038916944.1', NULL),
(9, 'XP_038916943.1', NULL),
(9, 'XP_038916942.1', NULL),
(9, 'XP_038916941.1', NULL),
(9, 'XP_038916940.1', NULL),
(9, 'XP_038916939.1', NULL),
(9, 'XP_038916938.1', NULL),
(9, 'XP_038916937.1', NULL),
(9, 'XP_038916936.1', NULL),
(9, 'XP_038916935.1', NULL),
(9, 'XP_038916934.1', NULL),
(9, 'XP_038916933.1', NULL),
(9, 'XP_038916932.1', NULL),
(9, 'XP_038916931.1', NULL),
(9, 'XP_038916930.1', NULL),
(9, 'XP_038916929.1', NULL),
(9, 'XP_038916928.1', NULL),
(9, 'XP_038916927.1', NULL),
(9, 'XP_038916926.1', NULL),
(9, 'XP_038916925.1', NULL),
(9, 'XP_038916924.1', NULL),
(9, 'XP_038916923.1', NULL),
(9, 'XP_038916922.1', NULL),
(9, 'XP_038916921.1', NULL),
(9, 'XP_038916920.1', NULL),
(9, 'XP_038916919.1', NULL),
(9, 'XP_038916918.1', NULL),
(9, 'XP_038916917.1', NULL),
(9, 'XP_038916916.1', NULL),
(9, 'XP_038916915.1', NULL),
(9, 'XP_038916914.1', NULL),
(9, 'XP_038916913.1', NULL),
(9, 'XP_038916912.1', NULL),
(9, 'XP_038916911.1', NULL),
(9, 'XP_038916910.1', NULL),
(9, 'XP_038916909.1', NULL),
(9, 'XP_038916908.1', NULL),
(9, 'XP_038916907.1', NULL),
(9, 'XP_038916906.1', NULL),
(9, 'XP_038916905.1', NULL),
(9, 'XP_038916904.1', NULL),
(9, 'XP_038916903.1', NULL),
(9, 'XP_038916902.1', NULL),
(9, 'XP_038916901.1', NULL),
(9, 'XP_038916900.1', NULL),
(9, 'XP_038916899.1', NULL),
(9, 'XP_038916898.1', NULL),
(9, 'XP_038916897.1', NULL),
(9, 'XP_038916896.1', NULL),
(9, 'XP_038916895.1', NULL),
(9, 'XP_038916894.1', NULL),
(9, 'XP_038916893.1', NULL),
(9, 'XP_038916892.1', NULL),
(9, 'XP_038916891.1', NULL),
(9, 'XP_038916890.1', NULL),
(9, 'XP_038916889.1', NULL),
(9, 'XP_038916888.1', NULL),
(9, 'XP_038916887.1', NULL),
(9, 'XP_038916886.1', NULL),
(9, 'XP_038916885.1', NULL),
(9, 'XP_038916884.1', NULL),
(9, 'XP_038916883.1', NULL),
(9, 'XP_038916882.1', NULL),
(9, 'XP_038916881.1', NULL),
(9, 'XP_038916880.1', NULL),
(9, 'XP_038916879.1', NULL),
(9, 'XP_038916878.1', NULL),
(9, 'XP_038916877.1', NULL),
(9, 'XP_038916876.1', NULL),
(9, 'XP_038916875.1', NULL),
(9, 'XP_038916874.1', NULL),
(9, 'XP_038916873.1', NULL),
(9, 'XP_038916872.1', NULL),
(9, 'XP_038916871.1', NULL),
(9, 'XP_038916870.1', NULL),
(9, 'XP_038916869.1', NULL),
(9, 'XP_038916868.1', NULL),
(9, 'XP_038916867.1', NULL),
(9, 'XP_038916866.1', NULL),
(9, 'XP_038917413.1', NULL),
(9, 'XP_038917412.1', NULL),
(9, 'XP_038917411.1', NULL),
(9, 'XP_038917410.1', NULL),
(9, 'XP_038917409.1', NULL),
(9, 'XP_038917408.1', NULL),
(9, 'XP_038917407.1', NULL),
(9, 'XP_038917406.1', NULL),
(9, 'XP_038917405.1', NULL),
(9, 'XP_038917404.1', NULL),
(9, 'XP_038917403.1', NULL),
(9, 'XP_038917402.1', NULL),
(9, 'XP_038917401.1', NULL),
(9, 'XP_038917400.1', NULL),
(9, 'XP_038917399.1', NULL),
(9, 'XP_038917398.1', NULL),
(9, 'XP_038917397.1', NULL),
(9, 'XP_038917396.1', NULL),
(9, 'XP_038917395.1', NULL),
(9, 'XP_038917394.1', NULL),
(9, 'XP_038917393.1', NULL),
(9, 'XP_038917392.1', NULL),
(9, 'XP_038917391.1', NULL),
(9, 'XP_038917390.1', NULL),
(9, 'XP_038917389.1', NULL),
(9, 'XP_038917388.1', NULL),
(9, 'XP_038917387.1', NULL),
(9, 'XP_038917386.1', NULL),
(9, 'XP_038917385.1', NULL),
(9, 'XP_038917384.1', NULL),
(9, 'XP_038917383.1', NULL),
(9, 'XP_038917382.1', NULL),
(9, 'XP_038917381.1', NULL),
(9, 'XP_038917380.1', NULL),
(9, 'XP_038917379.1', NULL),
(9, 'XP_038917378.1', NULL),
(9, 'XP_038917377.1', NULL),
(9, 'XP_038917376.1', NULL),
(9, 'XP_038917375.1', NULL),
(9, 'XP_038917374.1', NULL),
(9, 'XP_038917373.1', NULL),
(9, 'XP_038917372.1', NULL),
(9, 'XP_038917371.1', NULL),
(9, 'XP_038917370.1', NULL),
(9, 'XP_038917369.1', NULL),
(9, 'XP_038917368.1', NULL),
(9, 'XP_038917367.1', NULL),
(9, 'XP_038917366.1', NULL),
(9, 'XP_038917365.1', NULL),
(9, 'XP_038917364.1', NULL),
(9, 'XP_038917363.1', NULL),
(9, 'XP_038917362.1', NULL),
(9, 'XP_038917361.1', NULL),
(9, 'XP_038917360.1', NULL),
(9, 'XP_038917359.1', NULL),
(9, 'XP_038917358.1', NULL),
(9, 'XP_038917357.1', NULL),
(9, 'XP_038917356.1', NULL),
(9, 'XP_038917355.1', NULL),
(9, 'XP_038917354.1', NULL),
(9, 'XP_038917353.1', NULL),
(9, 'XP_038917352.1', NULL),
(9, 'XP_038917351.1', NULL),
(9, 'XP_038917350.1', NULL),
(9, 'XP_038917349.1', NULL),
(9, 'XP_038917348.1', NULL),
(9, 'XP_038917347.1', NULL),
(9, 'XP_038917346.1', NULL),
(9, 'XP_038917345.1', NULL),
(9, 'XP_038917344.1', NULL),
(9, 'XP_038917343.1', NULL),
(9, 'XP_038917342.1', NULL),
(9, 'XP_038917341.1', NULL),
(9, 'XP_038917340.1', NULL),
(9, 'XP_038917339.1', NULL),
(9, 'XP_038917338.1', NULL),
(9, 'XP_038917337.1', NULL),
(9, 'XP_038917336.1', NULL),
(9, 'XP_038917335.1', NULL),
(9, 'XP_038917334.1', NULL),
(9, 'XP_038917333.1', NULL),
(9, 'XP_038917332.1', NULL),
(9, 'XP_038917331.1', NULL),
(9, 'XP_038917330.1', NULL),
(9, 'XP_038917329.1', NULL),
(9, 'XP_038917328.1', NULL),
(9, 'XP_038917327.1', NULL),
(9, 'XP_038917326.1', NULL),
(9, 'XP_038917325.1', NULL),
(9, 'XP_038917324.1', NULL),
(9, 'XP_038917323.1', NULL),
(9, 'XP_038917322.1', NULL),
(9, 'XP_038917321.1', NULL),
(9, 'XP_038917320.1', NULL),
(9, 'XP_038917319.1', NULL),
(9, 'XP_038917318.1', NULL),
(9, 'XP_038917317.1', NULL),
(9, 'XP_038917316.1', NULL),
(9, 'XP_038917315.1', NULL),
(9, 'XP_038917314.1', NULL),
(9, 'XP_038917313.1', NULL),
(9, 'XP_038917312.1', NULL),
(9, 'XP_038917311.1', NULL),
(9, 'XP_038917310.1', NULL),
(9, 'XP_038917309.1', NULL),
(9, 'XP_038917308.1', NULL),
(9, 'XP_038917307.1', NULL),
(9, 'XP_038917306.1', NULL),
(9, 'XP_038917305.1', NULL),
(9, 'XP_038917304.1', NULL),
(9, 'XP_038917303.1', NULL),
(9, 'XP_038917302.1', NULL),
(9, 'XP_038917301.1', NULL),
(9, 'XP_038917300.1', NULL),
(9, 'XP_038917299.1', NULL),
(9, 'XP_038917298.1', NULL),
(9, 'XP_038917297.1', NULL),
(9, 'XP_038917296.1', NULL),
(9, 'XP_038917295.1', NULL);

-- --------------------------------------------------------

--
-- Structure de la table `status`
--

CREATE TABLE `status` (
  `id_status` int(11) NOT NULL,
  `name_status` varchar(15) NOT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;

--
-- Déchargement des données de la table `status`
--

INSERT INTO `status` (`id_status`, `name_status`) VALUES
(1, 'undefined'),
(2, 'tasks'),
(3, 'treat'),
(4, 'reject');

-- --------------------------------------------------------

--
-- Structure de la table `user`
--

CREATE TABLE `user` (
  `id_user` int(11) NOT NULL,
  `name_user` varchar(15) NOT NULL,
  `email` varchar(255) DEFAULT NULL,
  `password` varchar(200) DEFAULT NULL,
  `profile` enum('expert','assistant') DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;

--
-- Déchargement des données de la table `user`
--

INSERT INTO `user` (`id_user`, `name_user`, `email`, `password`, `profile`) VALUES
(1, 'Nicolas', NULL, NULL, NULL),
(2, 'Elodie', NULL, NULL, NULL),
(3, 'Vincent', NULL, NULL, NULL),
(4, 'Marie-Line', NULL, NULL, NULL),
(5, 'Pedro', NULL, NULL, NULL),
(6, 'thierry', 'thierry.galliano@etu.univ-amu.fr', '$2y$10$R0HIU0GkdXConABOnNuTZePLhGu7bgMqLGR/aZHbkMoSCdo6jG4Em', 'expert');

--
-- Index pour les tables déchargées
--

--
-- Index pour la table `article`
--
ALTER TABLE `article`
  ADD PRIMARY KEY (`id_article`),
  ADD KEY `id_user` (`user`),
  ADD KEY `status` (`status`);

--
-- Index pour la table `authors`
--
ALTER TABLE `authors`
  ADD PRIMARY KEY (`id_author`);

--
-- Index pour la table `author_article`
--
ALTER TABLE `author_article`
  ADD KEY `id_author` (`id_author`),
  ADD KEY `id_article` (`id_article`);

--
-- Index pour la table `note`
--
ALTER TABLE `note`
  ADD KEY `id_article` (`id_article`),
  ADD KEY `id_user` (`id_user`);

--
-- Index pour la table `prot_access_table`
--
ALTER TABLE `prot_access_table`
  ADD KEY `id_article` (`id_article`);

--
-- Index pour la table `status`
--
ALTER TABLE `status`
  ADD PRIMARY KEY (`id_status`);

--
-- Index pour la table `user`
--
ALTER TABLE `user`
  ADD PRIMARY KEY (`id_user`);

--
-- AUTO_INCREMENT pour les tables déchargées
--

--
-- AUTO_INCREMENT pour la table `article`
--
ALTER TABLE `article`
  MODIFY `id_article` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=15;

--
-- AUTO_INCREMENT pour la table `authors`
--
ALTER TABLE `authors`
  MODIFY `id_author` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=206;

--
-- AUTO_INCREMENT pour la table `status`
--
ALTER TABLE `status`
  MODIFY `id_status` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=5;

--
-- AUTO_INCREMENT pour la table `user`
--
ALTER TABLE `user`
  MODIFY `id_user` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=24;

--
-- Contraintes pour les tables déchargées
--

--
-- Contraintes pour la table `article`
--
ALTER TABLE `article`
  ADD CONSTRAINT `article_ibfk_1` FOREIGN KEY (`status`) REFERENCES `status` (`id_status`);

--
-- Contraintes pour la table `author_article`
--
ALTER TABLE `author_article`
  ADD CONSTRAINT `author_article_ibfk_1` FOREIGN KEY (`id_author`) REFERENCES `authors` (`id_author`),
  ADD CONSTRAINT `author_article_ibfk_2` FOREIGN KEY (`id_article`) REFERENCES `article` (`id_article`);

--
-- Contraintes pour la table `note`
--
ALTER TABLE `note`
  ADD CONSTRAINT `note_ibfk_1` FOREIGN KEY (`id_article`) REFERENCES `article` (`id_article`),
  ADD CONSTRAINT `note_ibfk_2` FOREIGN KEY (`id_user`) REFERENCES `user` (`id_user`);

--
-- Contraintes pour la table `prot_access_table`
--
ALTER TABLE `prot_access_table`
  ADD CONSTRAINT `prot_access_table_ibfk_1` FOREIGN KEY (`id_article`) REFERENCES `article` (`id_article`);
COMMIT;

/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
