-- phpMyAdmin SQL Dump
-- version 5.0.2
-- https://www.phpmyadmin.net/
--
-- Host: 127.0.0.1:3306
-- Generation Time: May 10, 2021 at 06:32 AM
-- Server version: 8.0.21
-- PHP Version: 7.4.9

SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
START TRANSACTION;
SET time_zone = "+00:00";


/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8mb4 */;

--
-- Database: `biblio`
--

-- --------------------------------------------------------

--
-- Table structure for table `article`
--

DROP TABLE IF EXISTS `article`;
CREATE TABLE IF NOT EXISTS `article` (
  `id_article` int NOT NULL AUTO_INCREMENT,
  `origin` varchar(100) NOT NULL,
  `num_access` varchar(255) NOT NULL,
  `title` text NOT NULL,
  `abstract` text NOT NULL,
  `year` year NOT NULL,
  `journal` varchar(255) NOT NULL,
  `pmcid` varchar(50) DEFAULT NULL,
  `status` int DEFAULT NULL,
  `html_xml` mediumtext,
  `general_note` mediumtext,
  `user` int DEFAULT NULL,
  PRIMARY KEY (`id_article`),
  KEY `id_user` (`user`),
  KEY `status` (`status`)
) ENGINE=InnoDB AUTO_INCREMENT=86 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;

--
-- Dumping data for table `article`
--

INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(1, 'pubmed', '31198441', 'The - gene cluster of  encodes GH43- and GH62-α-l-arabinofuranosidases with complementary modes of action.', 'The α-l-arabinofuranosidases (α-l-ABFs) are exoenzymes involved in the hydrolysis of α-l-arabinosyl linkages in plant cell wall polysaccharides. They play a crucial role in the degradation of arabinoxylan and arabinan and they are used in many biotechnological applications. Analysis of the genome of  showed that putative cellulosomal α-l-ABFs are exclusively encoded by the - gene cluster, a large 32-kb gene cluster. Indeed, among the 14 Xyl-Doc enzymes encoded by this gene cluster, 6 are predicted to be α-l-ABFs belonging to the CAZyme families GH43 and GH62.The biochemical characterization of these six Xyl-Doc enzymes revealed that four of them are α-l-ABFs. GH43-1229 (RcAbf43A) which belongs to the subfamily 16 of the GH43, encoded by the gene at locus Ccel_1229, has a low specific activity on natural substrates and can cleave off arabinose decorations located at arabinoxylan chain extremities. GH43-1233 (RcAbf43A), the product of the gene at locus Ccel_1233, belonging to subfamily 10 of the GH43, can convert the double arabinose decorations present on arabinoxylan into single O2- or O3-linked decorations with high velocity ( = 16.6 ± 0.6 s). This enzyme acts in synergy with GH62-1234 (RcAbf62A), the product of the gene at locus Ccel_1234, a GH62 α-l-ABF which hydrolyzes α-(1 → 3) or α-(1 → 2)-arabinosyl linkages present on polysaccharides and arabinoxylooligosaccharides monodecorated. Finally, a bifunctional enzyme, GH62-CE6-1240 (RcAbf62BAxe6), encoded by the gene at locus Ccel_1240, which contains a GH62-α-l-ABF module and a carbohydrate esterase (CE6) module, catalyzes deacylation of plant cell wall polymers and cleavage of arabinosyl mono-substitutions. These enzymes are also active on arabinan, a component of the type I rhamnogalacturonan, showing their involvement in pectin degradation.Arabinofuranosyl decorations on arabinoxylan and pectin strongly inhibit the action of xylan-degrading enzymes and pectinases. α-l-ABFs encoded by the - gene cluster of  can remove all the decorations present in the backbone of arabinoxylan and arabinan, act synergistically, and, thus, play a crucial role in the degradation of plant cell wall polysaccharides.', 2019, 'Biotechnology for biofuels', 'PMC6556953', 3, NULL, NULL, 1),
(2, 'pubmed', '33476380', 'A novel dextranase gene from the marine bacterium Bacillus aquimaris S5 and its expression and characteristics.', 'Dextranase specifically hydrolyzes dextran and is used to produce functional isomalto-saccharide prebiotics. Moreover, dextranase is used as an additive in mouthwash to remove dental plaque. We cloned and expressed the dextranase gene of the marine bacterium Bacillus aquimaris S5. The length of the BaDex gene was 1788 bp, encoding 573 amino acids. Using bioinformatics to predict and analyze the amino acid sequence of BaDex, we found the isoelectric point and instability coefficient to be 4.55 and 29.22, respectively. The average hydrophilicity (GRAVY) was -0.662. The secondary structure of BaDex consisted of 145 alpha helices, accounting for 25.31% of the protein; 126 extended strands, accounting for 21.99%; and 282 random coils, accounting for 49.21%. The 3D structure of the BaDex protein was predicted and simulated using SWISS-MODEL, and BaDex was classified as a Glycoside Hydrolase Family 66 protein. The optimal temperature and pH for BaDex activity were 40°C and 6.0, respectively. The hydrolysates had excellent antioxidant activity, and 8 U/mL of BaDex could remove 80% of dental plaque in MBRC experiment. This recombinant protein thus has great promise for applications in the food and pharmaceutical industries.', 2021, 'FEMS microbiology letters', '', 3, NULL, NULL, 2),
(3, 'pubmed', '33465378', 'Structure-function analysis of pectate lyase Pel3 reveals essential facets of protein recognition by the bacterial type 2 secretion system.', 'The Type II Secretion System (T2SS) transports fully folded proteins of various functions and structures through the outer membrane of Gram-negative bacteria. The molecular mechanisms of substrate recruitment by T2SS remain elusive but a prevailing view is that the secretion determinants could be of a structural nature. The phytopathogenic γ-Proteobacteria, Pectobacterium carotovorum and Dickeya dadantii, secrete similar sets of homologous plant cell wall degrading enzymes, mainly pectinases, by similar T2SSs, called Out. However, the orthologous pectate lyases Pel3 and PelI from these bacteria, which share 67% of sequence identity, are not secreted by the counterpart T2SS of each bacterium, indicating a fine-tuned control of protein recruitment. To identify the related secretion determinants, we first performed a structural characterization and comparison of Pel3 with PelI using X-ray crystallography. Then, to assess the biological relevance of the observed structural variations, we conducted a loop-substitution analysis of Pel3 combined with secretion assays. We showed that there is not one element with a definite secondary structure but several distant and structurally flexible loop regions that are essential for the secretion of Pel3 and that these loop regions act together as a composite secretion signal. Interestingly, depending on the crystal contacts, one of these key secretion determinants undergoes disorder-to-order transitions that could reflect its transient structuration upon the contact with the appropriate T2SS components. We hypothesize that such T2SS-induced structuration of some intrinsically disordered zones of secretion substrates could be part of the recruitment mechanism used by T2SS.', 2021, 'The Journal of biological chemistry', 'PMC7949064', 4, NULL, NULL, 1),
(4, 'pubmed', '33469030', 'Degradation of complex arabinoxylans by human colonic Bacteroidetes.', 'Some Bacteroidetes and other human colonic bacteria can degrade arabinoxylans, common polysaccharides found in dietary fiber. Previous work has identified gene clusters (polysaccharide-utilization loci, PULs) for degradation of simple arabinoxylans. However, the degradation of complex arabinoxylans (containing side chains such as ferulic acid, a phenolic compound) is poorly understood. Here, we identify a PUL that encodes multiple esterases for degradation of complex arabinoxylans in Bacteroides species. The PUL is specifically upregulated in the presence of complex arabinoxylans. We characterize some of the esterases biochemically and structurally, and show that they release ferulic acid from complex arabinoxylans. Growth of four different colonic Bacteroidetes members, including Bacteroides intestinalis, on complex arabinoxylans results in accumulation of ferulic acid, a compound known to have antioxidative and immunomodulatory properties.', 2021, 'Nature communications', 'PMC7815789', 4, NULL, NULL, 5),
(5, 'pubmed', '33838183', 'Functional diversity of three tandem C-terminal carbohydrate-binding modules of a β-mannanase.', 'Carbohydrate active enzymes, such as those involved in plant cell wall and storage polysaccharide biosynthesis and deconstruction, often contain repeating non-catalytic carbohydrate binding modules (CBMs) to compensate for low-affinity binding typical of protein-carbohydrate interactions. The bacterium Saccharophagus degradans produces an endo-β-mannanase of glycoside hydrolase family 5 subfamily 8 with three phylogenetically distinct family 10 CBMs located C-terminally from the catalytic domain (SdGH5_8-CBM10x3). However, the functional roles and cooperativity of these CBM domains in polysaccharide binding is not clear. To learn more we studied the full-length enzyme, three stepwise CBM10 truncations, and green fluorescent protein fusions of the individual CBM10s and all three domains together by pull-down assays, affinity gel electrophoresis, and activity assays. Only the C-terminal CBM10-3 was found to bind strongly to microcrystalline cellulose (dissociation constant, K = 1.48 μM). CBM10-3 and CBM10-2 bound galactomannan with similar affinity (K = 0.2-0.4 mg/ml), but CBM10-1 had 20-fold lower affinity for this substrate. CBM10 truncations barely affected specific activity on carob galactomannan and konjac glucomannan. Full-length SdGH5_8-CBM10x3 was two-fold more active on the highly galactose-decorated viscous guar gum galactomannan and crystalline ivory nut mannan at high enzyme concentrations, but the specific activity was 4- to 9-fold reduced at low enzyme and substrate concentrations compared to the enzyme lacking CBM10-2 and -3. Comparison of activity and binding data for the different enzyme forms indicates unproductive and productive polysaccharide binding to occur. We conclude that the C-terminal-most CBM10-3 secures firm binding, with contribution from CBM10-2, which with CBM10-1 also provides spatial flexibility.', 2021, 'The Journal of biological chemistry', '', 3, NULL, NULL, 4),
(6, 'pubmed', '33837197', 'Discovery of fungal oligosaccharide-oxidising flavo-enzymes with previously unknown substrates, redox-activity profiles and interplay with LPMOs.', 'Oxidative plant cell-wall processing enzymes are of great importance in biology and biotechnology. Yet, our insight into the functional interplay amongst such oxidative enzymes remains limited. Here, a phylogenetic analysis of the auxiliary activity 7 family (AA7), currently harbouring oligosaccharide flavo-oxidases, reveals a striking abundance of AA7-genes in phytopathogenic fungi and Oomycetes. Expression of five fungal enzymes, including three from unexplored clades, expands the AA7-substrate range and unveils a cellooligosaccharide dehydrogenase activity, previously unknown within AA7. Sequence and structural analyses identify unique signatures distinguishing the strict dehydrogenase clade from canonical AA7 oxidases. The discovered dehydrogenase directly is able to transfer electrons to an AA9 lytic polysaccharide monooxygenase (LPMO) and fuel cellulose degradation by LPMOs without exogenous reductants. The expansion of redox-profiles and substrate range highlights the functional diversity within AA7 and sets the stage for harnessing AA7 dehydrogenases to fine-tune LPMO activity in biotechnological conversion of plant feedstocks.', 2021, 'Nature communications', '', 3, NULL, NULL, 2),
(7, 'pubmed', '33686095', 'Clinical evidence of the role of Methanobrevibacter smithii in severe acute malnutrition.', 'Gut microbial dysbiosis has been shown to be an instrumental factor in severe acute malnutrition (SAM) and particularly, the absence of Methanobrevibacter smithii, a key player in energy harvest. Nevertheless, it remains unknown whether this absence reflects an immaturity or a loss of the microbiota. In order to assess that, we performed a case-control study in Mali using a propensity score weighting approach. The presence of M. smithii was tested using quantitative PCR on faeces collected from SAM children at inclusion and at discharge when possible or at day 15 for controls. M. smithii was highly significantly associated with the absence of SAM, detected in 40.9% controls but only in 4.2% cases (p < 0.0001). The predictive positive value for detection of M. smithii gradually increased with age in controls while decreasing in cases. Among children providing two samples with a negative first sample, no SAM children became positive, while this proportion was 2/4 in controls (p = 0.0015). This data suggests that gut dysbiosis in SAM is not an immaturity but rather features a loss of M. smithii. The addition of M. smithii as a probiotic may thus represent an important addition to therapeutic approaches to restore gut symbiosis.', 2021, 'Scientific reports', 'PMC7940396', 1, NULL, NULL, 4),
(8, 'pubmed', '33571448', 'Strain-level functional variation in the human gut microbiota based on bacterial binding to artificial food particles.', 'Greater understanding of the spatial relationships between members of the human gut microbiota and available nutrients is needed to gain deeper insights about community dynamics and expressed functions. Therefore, we generated a panel of artificial food particles with each type composed of microscopic paramagnetic beads coated with a fluorescent barcode and one of 60 different dietary or host glycan preparations. Analysis of 160 Bacteroides and Parabacteroides strains disclosed diverse strain-specific and glycan-specific binding phenotypes. We identified carbohydrate structures that correlated with binding by specific bacterial strains in vitro and noted strain-specific differences in the catabolism of glycans that mediate adhesion. Mixed in vitro cultures revealed that these adhesion phenotypes are maintained in more complex communities. Additionally, orally administering glycan beads to gnotobiotic mice confirmed specificity in glycan binding. This approach should facilitate analyses of how strains occupying the same physical niche interact, and it should advance the development of synbiotics, more nutritious foods, and microbiota-based diagnostics.', 2021, 'Cell host & microbe', '', 2, NULL, NULL, 1),
(9, 'pubmed', '33538380', 'Gene family expansions and transcriptome signatures uncover fungal adaptations to wood decay.', 'Because they comprise some of the most efficient wood-decayers, Polyporales fungi impact carbon cycling in forest environment. Despite continuous discoveries on the enzymatic machinery involved in wood decomposition, the vision on their evolutionary adaptation to wood decay and genome diversity remains incomplete. We combined the genome sequence information from 50 Polyporales species, including 26 newly sequenced genomes and sought for genomic and functional adaptations to wood decay through the analysis of genome composition and transcriptome responses to different carbon sources. The genomes of Polyporales from different phylogenetic clades showed poor conservation in macrosynteny, indicative of genome rearrangements. We observed different gene family expansion/contraction histories for plant cell wall degrading enzymes in core polyporoids and phlebioids and captured expansions for genes involved in signalling and regulation in the lineages of white rotters. Furthermore, we identified conserved cupredoxins, thaumatin-like proteins and lytic polysaccharide monooxygenases with a yet uncharacterized appended module as new candidate players in wood decomposition. Given the current need for enzymatic toolkits dedicated to the transformation of renewable carbon sources, the observed genomic diversity among Polyporales strengthens the relevance of mining Polyporales biodiversity to understand the molecular mechanisms of wood decay.', 2021, 'Environmental microbiology', '', 3, NULL, NULL, 5),
(10, 'pubmed', '33460770', 'Characterization of three bacterial glycoside hydrolase family 9 endoglucanases with different modular architectures isolated from a compost metagenome.', 'Environmental bacteria express a wide diversity of glycoside hydrolases (GH). Screening and characterization of GH from metagenomic sources provides an insight into biomass degradation strategies of non-cultivated prokaryotes.In the present report, we screened a compost metagenome for lignocellulolytic activities and identified six genes encoding enzymes belonging to family GH9 (GH9a-f). Three of these enzymes (GH9b, GH9d and GH9e) were successfully expressed and characterized.A phylogenetic analysis of the catalytic domain of pro- and eukaryotic GH9 enzymes suggested the existence of two major subgroups. Bacterial GH9s displayed a wide variety of modular architectures and those harboring an N-terminal Ig-like domain, such as GH9b and GH9d, segregated from the remainder. We purified and characterized GH9 endoglucanases from both subgroups and examined their stabilities, substrate specificities and product profiles. GH9e exhibited an original hydrolysis pattern, liberating an elevated proportion of oligosaccharides longer than cellobiose. All of the enzymes exhibited processive behavior and a synergistic action on crystalline cellulose. Synergy was also evidenced between GH9d and a GH48 enzyme identified from the same metagenome.The characterized GH9 enzymes displayed different modular architectures and distinct substrate and product profiles. The presence of a cellulose binding domain was shown to be necessary for binding and digestion of insoluble cellulosic substrates, but not for processivity.The identification of six GH9 enzymes from a compost metagenome and the functional variety of three characterized members highlight the importance of this enzyme family in bacterial biomass deconstruction.', 2021, 'Biochimica et biophysica acta. General subjects', '', 2, NULL, NULL, 3),
(11, 'pubmed', '33268895', 'Galactosaminogalactan activates the inflammasome to provide host protection.', 'Inflammasomes are important sentinels of innate immune defence that are activated in response to diverse stimuli, including pathogen-associated molecular patterns (PAMPs). Activation of the inflammasome provides host defence against aspergillosis, which is a major health concern for patients who are immunocompromised. However, the Aspergillus fumigatus PAMPs that are responsible for inflammasome activation are not known. Here we show that the polysaccharide galactosaminogalactan (GAG) of A. fumigatus is a PAMP that activates the NLRP3 inflammasome. The binding of GAG to ribosomal proteins inhibited cellular translation machinery, and thus activated the NLRP3 inflammasome. The galactosamine moiety bound to ribosomal proteins and blocked cellular translation, which triggered activation of the NLRP3 inflammasome. In mice, a GAG-deficient Aspergillus mutant (Δgt4c) did not elicit protective activation of the inflammasome, and this strain exhibited enhanced virulence. Moreover, administration of GAG protected mice from colitis induced by dextran sulfate sodium in an inflammasome-dependent manner. Thus, ribosomes connect the sensing of this fungal PAMP to the activation of an innate immune response.', 2020, 'Nature', '', 2, NULL, NULL, 1),
(12, 'pubmed', '33211093', 'Genomic Analysis Enlightens Agaricales Lifestyle Evolution and Increasing Peroxidase Diversity.', 'As actors of global carbon cycle, Agaricomycetes (Basidiomycota) have developed complex enzymatic machineries that allow them to decompose all plant polymers, including lignin. Among them, saprotrophic Agaricales are characterized by an unparalleled diversity of habitats and lifestyles. Comparative analysis of 52 Agaricomycetes genomes (14 of them sequenced de novo) reveals that Agaricales possess a large diversity of hydrolytic and oxidative enzymes for lignocellulose decay. Based on the gene families with the predicted highest evolutionary rates-namely cellulose-binding CBM1, glycoside hydrolase GH43, lytic polysaccharide monooxygenase AA9, class-II peroxidases, glucose-methanol-choline oxidase/dehydrogenases, laccases, and unspecific peroxygenases-we reconstructed the lifestyles of the ancestors that led to the extant lignocellulose-decomposing Agaricomycetes. The changes in the enzymatic toolkit of ancestral Agaricales are correlated with the evolution of their ability to grow not only on wood but also on leaf litter and decayed wood, with grass-litter decomposers as the most recent eco-physiological group. In this context, the above families were analyzed in detail in connection with lifestyle diversity. Peroxidases appear as a central component of the enzymatic toolkit of saprotrophic Agaricomycetes, consistent with their essential role in lignin degradation and high evolutionary rates. This includes not only expansions/losses in peroxidase genes common to other basidiomycetes but also the widespread presence in Agaricales (and Russulales) of new peroxidases types not found in wood-rotting Polyporales, and other Agaricomycetes orders. Therefore, we analyzed the peroxidase evolution in Agaricomycetes by ancestral-sequence reconstruction revealing several major evolutionary pathways and mapped the appearance of the different enzyme types in a time-calibrated species tree.', 2021, 'Molecular biology and evolution', '', 4, NULL, NULL, 2),
(13, 'pubmed', '33199373', 'Identification of the molecular determinants driving the substrate specificity of fungal lytic polysaccharide monooxygenases (LPMOs).', 'Understanding enzymatic breakdown of plant biomass is crucial to develop nature-inspired biotechnological processes. Lytic polysaccharide monooxygenases (LPMOs) are microbial enzymes secreted by fungal saprotrophs involved in carbon recycling. LPMOs modify biomass by oxidatively cleaving polysaccharides thereby enhancing the efficiency of glycoside hydrolases. Fungal AA9 LPMOs are active on cellulose but some members also display activity on hemicelluloses and/or oligosaccharides. Although the active site subsites are well defined for a few model LPMOs, the molecular determinants driving broad substrate specificity are still not easily predictable. Based on bioinformatic clustering and sequence alignments, we selected seven fungal AA9 LPMOs that differ in the amino-acid residues constituting their subsites. Investigation of their substrate specificities revealed that all these LPMOs are active on cellulose and cello-oligosaccharides, as well as plant cell wall-derived hemicellulosic polysaccharides and carry out C4 oxidative cleavage. The product profiles from cello-oligosaccharides degradation suggests that the subtle differences in amino acids sequence within the substrate-binding loop regions lead to different preferred binding modes. Our functional analyses allowed us to probe the molecular determinants of substrate binding within two AA9 LPMO sub-clusters. Many wood-degrading fungal species rich in AA9 genes have at least one AA9 enzyme with structural loop features that allow recognition of short β-(1,4)-linked glucan chains. Time-course monitoring of these AA9 LPMOs on cello-oligosaccharides also provides a useful model system for mechanistic studies of LPMO catalysis. These results are valuable for the understanding of LPMO contribution to wood decaying process in nature and for the development of sustainable biorefineries.', 2020, 'The Journal of biological chemistry', 'PMC7949027', 2, NULL, NULL, 5),
(14, 'pubmed', '30897810', 'Expression, Purification and Characterization of Chondroitinase AC II from Marine Bacterium  sp. CS01.', 'Chondroitinase (ChSase), a type of glycosaminoglycan (GAG) lyase, can degrade chondroitin sulfate (CS) to unsaturate oligosaccharides, with various functional activities. In this study, ChSase AC II from a newly isolated marine bacterium  sp. CS01 was cloned, expressed in  X33, purified, and characterized. ChSase AC II, with a molecular weight of approximately 100 kDa and a specific activity of 18.7 U/mg, showed the highest activity at 37 °C and pH 6.5 and maintained stability at a broad range of pH (5⁻7.5) and temperature (below 35 °C). The enzyme activity was increased in the presence of Mn and was strongly inhibited by Hg. Moreover, the kinetic parameters of ChSase AC II against CS-A, CS-C, and HA were determined. TLC and ESI-MS analysis of the degradation products indicated that ChSase AC II displayed an exolytic action mode and completely hydrolyzed three substrates into oligosaccharides with low degrees of polymerization (DPs). All these features make ChSase AC II a promising candidate for the full use of GAG to produce oligosaccharides.', 2019, 'Marine drugs', 'PMC6471956', 1, NULL, NULL, 3),
(17, 'pubmed', '52', 'Evidence of the involvement of a 50S ribosomal protein in several active sites.', 'The functional role of the Bacillus stearothermophilus 50S ribosomal protein B-L3 (probably homologous to the Escherichia coli protein L2) was examined by chemical modification. The complex [B-L3-23S RNA] was photooxidized in the presence of rose bengal and the modified protein incorporated by reconstitution into 50S ribosomal subunits containing all other unmodified components. Particles containing photooxidized B-L3 are defective in several functional assays, including (1) poly(U)-directed poly(Phe) synthesis, (2) peptidyltransferase activity, (3) ability to associate with a [30S-poly(U)-Phe-tRNA] complex, and (4) binding of elongation factor G and GTP. The rates of loss of the partial functional activities during photooxidation of B-L3 indicate that at least two independent inactivating events are occurring, a faster one, involving oxidation of one or more histidine residues, affecting peptidyltransferase and subunit association activities and a slower one affecting EF-G binding. Therefore the protein B-L3 has one or more histidine residues which are essential for peptidyltransferase and subunit association, and another residue which is essential for EF-G-GTP binding. B-L3 may be the ribosomal peptidyltransferase protein, or a part of the active site, and may contribute functional groups to the other active sites as well.', 1975, 'Biochemistry', '', 4, NULL, NULL, 4);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(18, 'pubmed', '33188211', 'Microbiota-directed fibre activates both targeted and secondary metabolic shifts in the distal gut.', 'Beneficial modulation of the gut microbiome has high-impact implications not only in humans, but also in livestock that sustain our current societal needs. In this context, we have tailored an acetylated galactoglucomannan (AcGGM) fibre to match unique enzymatic capabilities of Roseburia and Faecalibacterium species, both renowned butyrate-producing gut commensals. Here, we test the accuracy of AcGGM within the complex endogenous gut microbiome of pigs, wherein we resolve 355 metagenome-assembled genomes together with quantitative metaproteomes. In AcGGM-fed pigs, both target populations differentially express AcGGM-specific polysaccharide utilization loci, including novel, mannan-specific esterases that are critical to its deconstruction. However, AcGGM-inclusion also manifests a \"butterfly effect\", whereby numerous metabolic changes and interdependent cross-feeding pathways occur in neighboring non-mannanolytic populations that produce short-chain fatty acids. Our findings show how intricate structural features and acetylation patterns of dietary fibre can be customized to specific bacterial populations, with potential to create greater modulatory effects at large.', 2020, 'Nature communications', 'PMC7666174', 2, '<h1 class=\"content-title\">Microbiota-directed fibre activates both targeted and secondary metabolic shifts in the distal gut</h1><br><span class=\"contrib-group fm-author\"><a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Michalak%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm140434883320176\" co-class=\"co-affbox\">Leszek Michalak</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Gaby%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm140434879736752\" co-class=\"co-affbox\">John Christian Gaby</a>,<sup><img src=\"https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/corrauth.gif\"\r\n alt=\"corresponding author\" /></sup><sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Lagos%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm140434876032848\" co-class=\"co-affbox\">Leidy Lagos</a>,<sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=La%20Rosa%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm140434873391728\" co-class=\"co-affbox\">Sabina Leanti La Rosa</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Hvidsten%20TR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm140434883497824\" co-class=\"co-affbox\">Torgeir R. Hvidsten</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=T%26%23x000e9%3Btard-Jones%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm140434859900688\" co-class=\"co-affbox\">Catherine T&#x000e9;tard-Jones</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Willats%20WG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm140434859898704\" co-class=\"co-affbox\">William G. T. Willats</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Terrapon%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm140434876274304\" co-class=\"co-affbox\">Nicolas Terrapon</a>,<sup>4,</sup><sup>5</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Lombard%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm140434879393696\" co-class=\"co-affbox\">Vincent Lombard</a>,<sup>4,</sup><sup>5</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Henrissat%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm140434879390976\" co-class=\"co-affbox\">Bernard Henrissat</a>,<sup>4,</sup><sup>5,</sup><sup>6</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Dr%26%23x000f6%3Bge%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm140434916541232\" co-class=\"co-affbox\">Johannes Dr&#x000f6;ge</a>,<sup>7</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Arntzen%20M%26%23x000d8%3B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm140434879541680\" co-class=\"co-affbox\">Magnus &#x000d8;verlie Arntzen</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Hagen%20LH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm140434860946816\" co-class=\"co-affbox\">Live Heldal Hagen</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=%26%23x000d8%3Bverland%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm140434860944096\" co-class=\"co-affbox\">Margareth &#x000d8;verland</a>,<sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Pope%20PB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm140434953429152\" co-class=\"co-affbox\">Phillip B. Pope</a>,<sup><img src=\"https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/corrauth.gif\"\r\n alt=\"corresponding author\" /></sup><sup>#</sup><sup>1,</sup><sup>2</sup> and  <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Westereng%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\" class=\"affpopup\" co-rid=\"_co_idm140434860734336\" co-class=\"co-affbox\">Bj&#x000f8;rge Westereng</a><sup><img src=\"https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/corrauth.gif\"\r\n alt=\"corresponding author\" /></sup><sup>#</sup><sup>1</sup></span><span style=\"display:none\" class=\"contrib-group aff-tip\"><span id=\"_co_idm140434883320176\"><h3 class=\"no_margin\">Leszek Michalak</h3><p><sup>1</sup>Faculty of Chemistry, Biotechnology and Food Science, Norwegian University of Life Sciences, 1432 &#x000c5;s, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Michalak%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Leszek Michalak</a></span></span><span id=\"_co_idm140434879736752\"><h3 class=\"no_margin\">John Christian Gaby</h3><p><sup>1</sup>Faculty of Chemistry, Biotechnology and Food Science, Norwegian University of Life Sciences, 1432 &#x000c5;s, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Gaby%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">John Christian Gaby</a></span></span><span id=\"_co_idm140434876032848\"><h3 class=\"no_margin\">Leidy Lagos</h3><p><sup>2</sup>Faculty of Biosciences, Norwegian University of Life Sciences, 1432 &#x000c5;s, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Lagos%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Leidy Lagos</a></span></span><span id=\"_co_idm140434873391728\"><h3 class=\"no_margin\">Sabina Leanti La Rosa</h3><p><sup>1</sup>Faculty of Chemistry, Biotechnology and Food Science, Norwegian University of Life Sciences, 1432 &#x000c5;s, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=La%20Rosa%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Sabina Leanti La Rosa</a></span></span><span id=\"_co_idm140434883497824\"><h3 class=\"no_margin\">Torgeir R. Hvidsten</h3><p><sup>1</sup>Faculty of Chemistry, Biotechnology and Food Science, Norwegian University of Life Sciences, 1432 &#x000c5;s, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Hvidsten%20TR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Torgeir R. Hvidsten</a></span></span><span id=\"_co_idm140434859900688\"><h3 class=\"no_margin\">Catherine T&#x000e9;tard-Jones</h3><p><sup>3</sup>School of Natural and Environmental Sciences, Newcastle University, Newcastle upon Tyne, UK </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=T%26%23x000e9%3Btard-Jones%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Catherine T&#x000e9;tard-Jones</a></span></span><span id=\"_co_idm140434859898704\"><h3 class=\"no_margin\">William G. T. Willats</h3><p><sup>3</sup>School of Natural and Environmental Sciences, Newcastle University, Newcastle upon Tyne, UK </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Willats%20WG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">William G. T. Willats</a></span></span><span id=\"_co_idm140434876274304\"><h3 class=\"no_margin\">Nicolas Terrapon</h3><p><sup>4</sup>Centre National de la Recherche Scientifique, Aix-Marseille Universit&#x000e9;, UMR7257 Marseille, France </p><p><sup>5</sup>Institut National de la Recherche Agronomique, USC1048 Architecture et Fonction des Macromol&#x000e9;cules Biologiques, Marseille, France </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Terrapon%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Nicolas Terrapon</a></span></span><span id=\"_co_idm140434879393696\"><h3 class=\"no_margin\">Vincent Lombard</h3><p><sup>4</sup>Centre National de la Recherche Scientifique, Aix-Marseille Universit&#x000e9;, UMR7257 Marseille, France </p><p><sup>5</sup>Institut National de la Recherche Agronomique, USC1048 Architecture et Fonction des Macromol&#x000e9;cules Biologiques, Marseille, France </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Lombard%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Vincent Lombard</a></span></span><span id=\"_co_idm140434879390976\"><h3 class=\"no_margin\">Bernard Henrissat</h3><p><sup>4</sup>Centre National de la Recherche Scientifique, Aix-Marseille Universit&#x000e9;, UMR7257 Marseille, France </p><p><sup>5</sup>Institut National de la Recherche Agronomique, USC1048 Architecture et Fonction des Macromol&#x000e9;cules Biologiques, Marseille, France </p><p><sup>6</sup>Department of Biological Sciences, King Abdulaziz University, Jeddah, Saudi Arabia </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Henrissat%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Bernard Henrissat</a></span></span><span id=\"_co_idm140434916541232\"><h3 class=\"no_margin\">Johannes Dr&#x000f6;ge</h3><p><sup>7</sup>Department for Mathematical Sciences, Chalmers University of Technology, Gothenburg, Sweden </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Dr%26%23x000f6%3Bge%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Johannes Dr&#x000f6;ge</a></span></span><span id=\"_co_idm140434879541680\"><h3 class=\"no_margin\">Magnus &#x000d8;verlie Arntzen</h3><p><sup>1</sup>Faculty of Chemistry, Biotechnology and Food Science, Norwegian University of Life Sciences, 1432 &#x000c5;s, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Arntzen%20M%26%23x000d8%3B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Magnus &#x000d8;verlie Arntzen</a></span></span><span id=\"_co_idm140434860946816\"><h3 class=\"no_margin\">Live Heldal Hagen</h3><p><sup>1</sup>Faculty of Chemistry, Biotechnology and Food Science, Norwegian University of Life Sciences, 1432 &#x000c5;s, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Hagen%20LH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Live Heldal Hagen</a></span></span><span id=\"_co_idm140434860944096\"><h3 class=\"no_margin\">Margareth &#x000d8;verland</h3><p><sup>2</sup>Faculty of Biosciences, Norwegian University of Life Sciences, 1432 &#x000c5;s, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=%26%23x000d8%3Bverland%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Margareth &#x000d8;verland</a></span></span><span id=\"_co_idm140434953429152\"><h3 class=\"no_margin\">Phillip B. Pope</h3><p><sup>1</sup>Faculty of Chemistry, Biotechnology and Food Science, Norwegian University of Life Sciences, 1432 &#x000c5;s, Norway </p><p><sup>2</sup>Faculty of Biosciences, Norwegian University of Life Sciences, 1432 &#x000c5;s, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Pope%20PB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Phillip B. Pope</a></span></span><span id=\"_co_idm140434860734336\"><h3 class=\"no_margin\">Bj&#x000f8;rge Westereng</h3><p><sup>1</sup>Faculty of Chemistry, Biotechnology and Food Science, Norwegian University of Life Sciences, 1432 &#x000c5;s, Norway </p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Westereng%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33188211\">Bj&#x000f8;rge Westereng</a></span></span></span></span><span class=\"fm-panel half_rhythm\"><span class=\"togglers<br><span id=\"idm\r\n1\" lang=\"en\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm\r\n1title\">Abstract</h2><!--article-meta--><span><p id=\"Par1\" class=\"p p-first-last\">Beneficial modulation of the gut microbiome has high-impact implications not only in humans, but also in livestock that sustain our current societal needs. In this context, we have tailored an acetylated galactoglucomannan (AcGGM) fibre to match unique enzymatic capabilities of <em>Roseburia</em> and <em>Faecalibacterium</em> species, both renowned butyrate-producing gut commensals. Here, we test the accuracy of AcGGM within the complex endogenous gut microbiome of pigs, wherein we resolve 355 metagenome-assembled genomes together with quantitative metaproteomes. In AcGGM-fed pigs, both target populations differentially express AcGGM-specific polysaccharide utilization loci, including novel, mannan-specific esterases that are critical to its deconstruction. However, AcGGM-inclusion also manifests a &#x0201c;butterfly effect&#x0201d;, whereby numerous metabolic changes and interdependent cross-feeding pathways occur in neighboring non-mannanolytic populations that produce short-chain fatty acids. Our findings show how intricate structural features and acetylation patterns of dietary fibre can be customized to specific bacterial populations, with potential to create greater modulatory effects at large.</p></span><span class=\"sec\"><strong class=\"kwd-title\">Subject terms: </strong><span class=\"kwd-text\">Bioinformatics, Metagenomics, Microbiome</span></span></span><span id=\"s\r\n1\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n1title\">Introduction</h2><p id=\"Par3\" class=\"p p-first\">Microbiota-directed foods (MDFs) have emerged as a strategy to modulate the gut microbiome, as the diet has distinct and rapid effects on microbiome composition and function<sup><sup><a href=\"#CR1\" rid=\"CR1\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780187\">1</a></sup>,<sup><a href=\"#CR2\" rid=\"CR2\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780194\">2</a></sup></sup>. MDFs by definition are not broadly metabolized, but rather elicit a targeted metabolic response in specific indigenous microbiota that confers benefits to their host. This in itself presents a challenge; as many newly identified MDF target organisms, such as beneficial butyrate-producing (i.e., butyrogenic) <em>Roseburia</em> and <em>Faecalibacterium</em> spp.<sup><sup><a href=\"#CR3\" rid=\"CR3\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780192\">3</a></sup>,<sup><a href=\"#CR4\" rid=\"CR4\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780236\">4</a></sup></sup>, have broad metabolic capabilities that are shared with the vast majority of fiber-fermenting microbiota in the gut ecosystem. Nevertheless, recent studies have revealed intimate connections between the enzymatic and mechanistic features of microorganisms and the glycan structures of the fibers they consume<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780230\">5</a></sup>,<sup><a href=\"#CR6\" rid=\"CR6\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780193\">6</a></sup></sup>, which creates new conceptual MDF targets. This is exemplified by discoveries of sophisticated polysaccharide-degrading apparatuses that enable certain microbiota to consume fiber in a &#x02018;selfish&#x02019; manner, whereby complex glycan structures (such as &#x003b2;-mannans) are cleaved into large oligosaccharides at the cell surface, which is subsequently transported into the cell and depolymerized into monomeric sugars<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780188\">5</a></sup>,<sup><a href=\"#CR7\" rid=\"CR7\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780244\">7</a></sup>,<sup><a href=\"#CR8\" rid=\"CR8\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780195\">8</a></sup></sup>. Such a mechanism restricts the release of sugars into the ecosystem for neighboring scavenging populations, thus giving a selective metabolic advantage to the selfish degrader in the presence of these highly complex glycans.</p><p id=\"Par4\">Beta-mannans are present in human and livestock diets, and depending on their plant origins, can be decorated with varying amounts of acetylation that protect the fiber from enzymatic degradation<sup><sup><a href=\"#CR9\" rid=\"CR9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780218\">9</a></sup></sup>. We recently demonstrated that the human gut commensal <em>Roseburia intestinalis</em> encodes a mannan-specific polysaccharide utilization locus (PUL), and &#x02018;selfishly&#x02019; converts highly complex mannan substrates to butyrate<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780208\">5</a></sup></sup>. Within this mannan PUL, a carbohydrate esterase (CE) family 2 (<em>Ri</em>CE2) removes 3-<em>O</em>-, and 6-<em>O</em>-acetylations on mannan, whereas a novel CE family 17 (<em>Ri</em>CE17) removes the axially oriented 2-<em>O</em>-acetylations<sup><sup><a href=\"#CR9\" rid=\"CR9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780220\">9</a></sup></sup>, which are distinctive features found in limited mannan moieties and inaccessible to most of the characterized bacterial esterases present in the gut microbiome. Closer genome examinations have revealed that putative CE2/CE17-containing mannan PULs are in fact prominent within many butyrate-producers including <em>Roseburia</em> spp., <em>Faecalibacterium prausnitzii</em>, <em>Ruminococcus gnavus</em>, <em>Coprococcus eutactus</em> and <em>Butyrivibrio fibrisolvens</em><sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780205\">5</a></sup>,<sup><a href=\"#CR10\" rid=\"CR10\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780249\">10</a></sup></sup>. It is well known that the metabolic attributes of these populations are highly desirable in the gastrointestinal tract, and that their depletion is implicated in colorectal cancer, Crohn&#x02019;s disease, inflammatory bowel syndrome, ulcerative colitis, forms of dermatitis, and several other diseases<sup><sup><a href=\"#CR11\" rid=\"CR11\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780238\">11</a></sup>,<sup><a href=\"#CR12\" rid=\"CR12\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780223\">12</a></sup></sup>. These collective findings thus raised the question: could a custom MDF fiber that was tailored to match these specialized enzymatic capabilities selectively engage butyrate-producers in a complex microbiome ecosystem?</p><p id=\"Par5\">2-<em>O-</em>acetylated mannans are found in a limited number of characterized western dietary fiber sources (i.e., tomatoes<sup><sup><a href=\"#CR13\" rid=\"CR13\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780186\">13</a></sup></sup> and coffee<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780197\">5</a></sup></sup>), however, 2-<em>O-</em>acetylations are present in acetylated galactoglucomannan (AcGGM), which is the main hemicellulose in the secondary cell wall of Norway spruce (<em>Picea abies</em>)<sup><sup><a href=\"#CR14\" rid=\"CR14\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780215\">14</a></sup></sup>. We have utilized a controlled steam explosion (SE), followed by ultrafiltration (UF) fractionation to extract complex AcGGM from spruce wood. Processing conditions were selected to tailor the fiber with a high degree of galactose branching&#x02019;s and 2-<em>O-</em>, 3-<em>O-</em> and 6-<em>O-</em>acetylations<sup><sup><a href=\"#CR15\" rid=\"CR15\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780171\">15</a></sup></sup>, which is amenable to inclusion as an MDF in animal feed production. Previously, the MDF concept has matched polysaccharides with <em>Bacteroides</em>-encoded PULs to demonstrate the creation of exclusive metabolic niches<sup><sup><a href=\"#CR1\" rid=\"CR1\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780204\">1</a></sup>,<sup><a href=\"#CR16\" rid=\"CR16\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780229\">16</a></sup></sup>. In particular, Shepherd et al.<sup><sup><a href=\"#CR16\" rid=\"CR16\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780234\">16</a></sup></sup> engrafted exogenous strains in mice via their rare PUL-encoded enzymatic capabilities, whereas Patnode et al.<sup><sup><a href=\"#CR1\" rid=\"CR1\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780250\">1</a></sup></sup> illustrated that bioactive carbohydrates were found to target particular <em>Bacteroides</em> species in a defined consortium of 20 human gut microbial species in gnotobiotic mice. However, what is less understood from MDF studies to date, is (1) can the MDF concept be applied to target indigenous populations within a complex endogenous microbiome, and (2) what are the broader secondary community effects if a targeted population is stimulated, i.e., are new niches created and/or existing ones closed?</p><p id=\"Par6\" class=\"p p-last\">Here, we test whether our AcGGM fiber can specifically target beneficial Firmicutes species <em>Roseburia</em> and <em>Faecalibacterium</em> within a &#x02018;real-world&#x02019; gut ecosystem that consists of 100&#x02013;1000&#x02019;s of different species. To evaluate this, we analyze the gut microbiomes of weaned piglets fed diets containing varying AcGGM levels over a 28-day period that extended from their first meal after sow separation until an adapted, fiber-degrading microbiome was established. Using metagenomics, we monitor temporal changes in the microbiome and phylogenetically and functionally resolve the genomes of indigenous microbiota. In parallel, our detailed quantitative metaproteomic and carbohydrate microarray analyses reveal the metabolic and enzymatic responses of the different microbiota to the varying AcGGM exposure. We demonstrate how the activity of specific beneficial microbiota can be directly stimulated while simultaneously deciphering the secondary, trophic effects on other populations and metabolic niches, with both aspects having broader implications for developing strategies to effectively modulate the gut microbiome.</p></span><span id=\"s\r\n2\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n2title\">Results</h2><span id=\"s\r\n3\" class=\"sec sec-first\"><h3 id=\"s\r\n3title\">Production of highly complex dietary mannan fibers from wood</h3><p id=\"Par7\" class=\"p p-first-last\">Spruce galactoglucomannan consists of a backbone of &#x003b2;-(1,4)-linked mannose and glucose residues, decorated with &#x003b1;-(1,6) linked galactose branching, and a large degree of esterification of the mannose residues by 2-<em>O</em>- and 3-<em>O</em>- and 6-<em>O</em>- acetylations<sup><sup><a href=\"#CR14\" rid=\"CR14\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780209\">14</a></sup></sup> (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\" co-legend-rid=\"lgnd_Fig1\"><span>1a</span></a>). A crucial part of this study was the development of an efficient, large-scale extraction process entailing SE as well as ultra- and nanofiltration, which ultimately provided high quantities at high purity whilst not damaging the complexity of the AcGGM fiber (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\" co-legend-rid=\"lgnd_Fig1\"><span>1b, c</span></a>). A total of 700&#x02009;kg of dry Norway spruce chips was processed using SE at conditions corresponding to a combined severity factor (<em>R</em>&#x02032;<sub>0</sub>) of 1.70. We produced 50&#x02009;kg of oligo/polysaccharides for feed production (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\" co-legend-rid=\"lgnd_Fig1\"><span>1c&#x02013;e</span></a>), with a monosaccharide (Man:Glc:Gal) ratio of 4:1:0.6. The degree of polymerization (DP) of the AcGGM fiber ranged from &#x003b2;-manno-oligosaccharides with DP of 2&#x02013;10 to manno-polysaccharides (DP of &#x02265;11), with both exhibiting degrees of acetylation (DA&#x02009;=&#x02009;0.35). Crucially, this DA value was higher than previous iterations of the fiber (DA&#x02009;=&#x02009;0.28)<sup><sup><a href=\"#CR7\" rid=\"CR7\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780202\">7</a></sup></sup>, and its acetylation patterns (previously determined<sup><sup><a href=\"#CR9\" rid=\"CR9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780228\">9</a></sup></sup>) matched the enzymatic capabilities of mannan PULs encoded in human gut <em>Roseburia</em> and <em>Faecalibacterium</em> spp.<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780200\">5</a></sup>,<sup><a href=\"#CR10\" rid=\"CR10\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780198\">10</a></sup></sup>. We, therefore, predicted that our AcGGM fiber would match representatives of the same populations that are indigenous to porcine gut ecosystems<sup><sup><a href=\"#CR17\" rid=\"CR17\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780246\">17</a></sup>,<sup><a href=\"#CR18\" rid=\"CR18\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_915796626\">18</a></sup></sup> (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\" co-legend-rid=\"lgnd_Fig1\"><span>1f</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig1\" co-legend-rid=\"lgnd_Fig1\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig1/\" target=\"figure\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140434860251856\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7666174_41467_2020_19585_Fig1_HTML.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2020_19585_Fig1_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC7666174/bin/41467_2020_19585_Fig1_HTML.jpg\" /></a></span><span id=\"largeobj_idm140434860251856\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig1/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_Fig1\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig1/\" target=\"figure\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\">Fig. 1</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Schematic representation and graphic illustration of the production pipeline for Norway spruce AcGGM.</strong><p id=\"__p1\"><strong>a</strong> Wood chips were milled to increase the surface area exposed for hydrothermal extraction and washing the released manno-oligosaccharides. Spruce wood contains long-chained galactoglucomannan (DP200-400), which is highly acetylated (DA&#x02009;~&#x02009;0.65), predominantly with 2-<em>O</em> and 3-<em>O</em> acetylations<sup><sup><a href=\"#CR66\" rid=\"CR66\" class=\" bibr popnode\">66</a></sup></sup>. <strong>b</strong> During hydrothermal pretreatment, the release of acetic acid promoted hydrolysis of glycosidic bonds, thereby relinquishing monosaccharides, oligosaccharides, and other breakdown products. <strong>c</strong> Ultrafiltration retained the longer, complex oligosaccharides and discarded the monosaccharides, acetic acid, and other steam explosion byproducts. <strong>d</strong> The purified mannan was incorporated into feed pellets at varying inclusion levels, produced by a conventional feed pelleting process. <strong>e</strong> The growth performance experiment and a feeding trial were conducted in a randomized block design, with four inclusion levels of AcGGM. <strong>f</strong> Multi-omic approaches were used to analyze the porcine gut microbiome in response to AcGGM and determine if indigenous mannan PULs matched the glycan structure of the AcGGM fiber. Glycosidic bonds between the &#x003b2;-(1,4)-mannose and glucose units in the backbone of AcGGM require hydrolysis by glycoside hydrolases (GH) from families GH5 and GH26. GH36 and GH27 &#x003b1;-galactosidases are required to remove the &#x003b1;-(1,6)-galactose decorations. Single mannose and glucose units are removed from the non-reducing end of the oligosaccharides by enzymes from families GH1, GH2, GH3, GH113, and GH130, while mannose in the reducing end can be removed by GH113 family mannosidases. 3-<em>O</em>- and 6-<em>O</em>-acetylations on mannan are removed by esterases from family CE2. A unique feature of particular beta-mannans is the axially oriented 2-<em>O</em>-acetylation on mannose, which is the prevalent form of acetylation present on AcGGM used in this study. 2-<em>O</em>-acetylations are removed by esterases homologous to the RiCE17 mannan-specific esterase from <em>Roseburia intestinalis</em>, which was recently characterized by our group<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780189\">5</a></sup></sup>.</p></span></span></span></span><span id=\"s\r\n4\" class=\"sec\"><h3 id=\"s\r\n4title\">AcGGM altered the gut microbiome of weaned piglets</h3><p id=\"Par8\" class=\"p p-first\">We previously demonstrated that varieties of AcGGM can be metabolized by pure cultures of <em>Roseburia intestinalis</em> L1-82<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780256\">5</a></sup></sup>, in vitro enrichments with human gut butyrate-producers<sup><sup><a href=\"#CR10\" rid=\"CR10\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780224\">10</a></sup></sup> and &#x02018;mini-microbiota&#x02019; within gnotobiotic mice<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780253\">5</a></sup></sup>. Here, we wanted to test our AcGGM fiber&#x02019;s accuracy and ability to elicit a specific response in indigenous representatives of our target populations within a highly complex and competitive endogenous microbiome. In total, four separate cohorts of twelve weaned piglets were given a pelleted basal feed semi-ad libitum, which contained either 0% (control), 1, 2 or 4% AcGGM to additionally determine the level necessary to elicit an effect on both the host and its microbiome. We chose a dietary supplementation strategy, which is commonly applied to piglets during the post-weaning period to offset their particularly low feed intake and enhance gut health and growth performance<sup><sup><a href=\"#CR19\" rid=\"CR19\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780175\">19</a></sup></sup>. Cautious measures were made to eliminate potential pen biases (see details in materials and methods), and fecal samples, as well as animal performance metrics, were taken before AcGGM administration (when piglets were assigned to pens), and subsequently at days 7, 14, 21 and 27 during the feeding trial. On day 28, the piglets were sacrificed and host gut tissue and digesta samples taken from all intestinal regions (duodenum, jejunum, ileum, cecum, and colon) for down-stream analysis (Supplementary Table&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">1</a></sup> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM4\" rid=\"MOESM4\" class=\" media\">1</a></sup>). Measurements of major short-chain fatty acids (SCFAs) in the cecum and colon showed a trend of incremental increases of absolute and relative butyrate levels as AcGGM levels were increased (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig2\" rid-ob=\"ob-Fig2\" co-legend-rid=\"lgnd_Fig2\"><span>2</span></a>). However, despite SCFA data suggesting that AcGGM inclusion promotes butyrogenic fermentation, it also showed that there was no statistically significant increase in total SCFA levels (Supplementary Data&#x000a0;<sup><a href=\"#MOESM4\" rid=\"MOESM4\" class=\" media\">1</a></sup>). Similarly, the levels of propionic acid were not affected by AcGGM inclusion (Supplementary Data&#x000a0;<sup><a href=\"#MOESM4\" rid=\"MOESM4\" class=\" media\">1</a></sup>). While changes in SCFA and microbiome composition (Figs.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig2\" rid-ob=\"ob-Fig2\" co-legend-rid=\"lgnd_Fig2\"><span>2</span></a> and <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span style=\"position: relative;text-decoration:none;\">&#x200B;<span class=\"figpopup-sensitive-area\" style=\"left: -1.5em;\">and3)</span></span><span>3</span></a>) resulted from AcGGM inclusion, we observed no adverse effects on the host&#x02019;s physiology, with the average weight, feed conversion ratio, blood cell composition, T cell population, and colon morphology not differing between the control and AcGGM treatments (Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">1</a></sup> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM4\" rid=\"MOESM4\" class=\" media\">1</a></sup>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig2\" co-legend-rid=\"lgnd_Fig2\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig2/\" target=\"figure\" rid-figpopup=\"Fig2\" rid-ob=\"ob-Fig2\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140434882382960\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7666174_41467_2020_19585_Fig2_HTML.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2020_19585_Fig2_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC7666174/bin/41467_2020_19585_Fig2_HTML.jpg\" /></a></span><span id=\"largeobj_idm140434882382960\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig2/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_Fig2\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig2/\" target=\"figure\" rid-figpopup=\"Fig2\" rid-ob=\"ob-Fig2\">Fig. 2</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Butyrate detected in the cecum and colon digesta of pigs fed the four different AcGGM diets with varying inclusion levels (0&#x02013;4%).</strong><p id=\"__p2\"><strong>a</strong> display absolute levels and <strong>b</strong> relative levels of butyrate. Raw SCFA measurements are presented in Supplementary Data&#x000a0;<sup><a href=\"#MOESM4\" rid=\"MOESM4\" class=\" media\">1</a></sup>. Bars represent median and boxes interquartile range determined from 12 animals analyzed per dietary group. Data were analyzed using a two-tailed <em>t</em>-test, with <em>p</em>-values indicated between pigs fed either the control or 4% AcGGM diets. The boxes span the 25th&#x02013;75th percentiles with the central bars being the medians. Whiskers extend maximum up to 1.5&#x000d7; the interquartile range (IQR) or, when all values are within 1.5&#x000d7; IQR, then the whisker extends to the most extreme data point.</p></span></span></span><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig3\" co-legend-rid=\"lgnd_Fig3\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig3/\" target=\"figure\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140434862688000\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7666174_41467_2020_19585_Fig3_HTML.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2020_19585_Fig3_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC7666174/bin/41467_2020_19585_Fig3_HTML.jpg\" /></a></span><span id=\"largeobj_idm140434862688000\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig3/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_Fig3\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig3/\" target=\"figure\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\">Fig. 3</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Effect of AcGGM-containing diets on the pig gut microbiome.</strong><p id=\"__p3\">16S rRNA gene amplicon and shotgun metagenomic analysis was used to monitor the effect the AcGGM fiber had on the gut microbiome structure of weaned piglets, determined from 12 animals analyzed per dietary group. Both 16S rRNA and MAG relative abundance data were analyzed using a two-tailed <em>t</em>-test, with significant increases (red) and decreases (blue) indicated between pigs fed either the control or AcGGM diets (* denotes <em>p</em>&#x02009;&#x0003c;&#x02009;0.05, ** denotes <em>p</em>&#x02009;&#x0003c;&#x02009;0.001). The boxes span the 25th&#x02013;75th percentiles with the central bars being the medians. Whiskers extend maximum up to 1.5&#x000d7; the interquartile range (IQR) or, when all values are within 1.5&#x000d7; IQR, then the whisker extends to the most extreme data point. <strong>a</strong> Ordination plots of Bray&#x02013;Curtis distances between microbial communities from pigs feed either the control or AcGGM diets (at varying inclusion levels: 1, 2 or 4%, 12 animals per dietary group). Samples were collected at day 28 of the feeding trials from various sections of the small and large intestine. The AcGGM-effect was more pronounced in the fiber-fermenting distal regions (cecum, colon) of the gut. <strong>b</strong> Relative 16S rRNA gene abundance of <em>Roseburia</em>-affiliated phylotypes (agglomerated at the genus level) in either cecum or colon samples collected from pigs fed AcGGM diets with varying inclusion levels (cecum: 0% vs 4% <em>p</em>&#x02009;=&#x02009;0.0004, colon: 0% vs 4% <em>p</em>&#x02009;=&#x02009;0.034). MAG relative abundances were determined from colon samples only, collected from pigs fed either the control or 4% AcGGM diets (0% vs 4% <em>p</em>-value: MAG041&#x02009;=&#x02009;0.002, MAG179&#x02009;=&#x02009;0.028, MAG261&#x02009;=&#x02009;0.001, MAG297&#x02009;=&#x02009;0.014, MAG269&#x02009;=&#x02009;0.035). Despite being suspected as an active mannan degrader, 16S rRNA gene abundance of <em>Roseburia</em>-affiliated phylotypes decreased with increasing % AcGGM. However, MAG relative abundance analysis of <em>Roseburia</em> populations showed that specific phylotypes that encoded AcGGM-specific PULs (indicated by green text) were indeed stimulated by the AcGGM fiber. <strong>c</strong> The relative 16S rRNA gene abundance of the genus <em>Faecalibacterium</em> was enhanced by the inclusion of AcGGM (cecum: 0% vs 2% <em>p</em>&#x02009;=&#x02009;0.008, 0% vs 4% <em>p</em>&#x02009;=&#x02009;2.11E&#x02212;05, colon: 0% vs 2% <em>p</em>&#x02009;=&#x02009;0.046, 0% vs 4% <em>p</em>&#x02009;=&#x02009;4.64E&#x02212;06), as were affiliated MAGs in the colon (0% vs 4% <em>p</em>-value: MAG243&#x02009;=&#x02009;5.84E&#x02212;06). The same trends were observed for phylotypes affiliated to <em>Catenibacterium</em> (<strong>d</strong> cecum: 0% vs 2% <em>p</em>&#x02009;=&#x02009;0.006, 0% vs 4% <em>p</em>&#x02009;=&#x02009;1.12E&#x02212;05, colon: 0% vs 2% <em>p</em>&#x02009;=&#x02009;0.0007, 0% vs 4% <em>p</em>&#x02009;=&#x02009;1.08E&#x02212;06, MAG048&#x02009;=&#x02009;1.52E&#x02212;06), Dialister (<strong>e</strong> cecum: 0% vs 2% <em>p</em>&#x02009;=&#x02009;0.013, 0% vs 4% <em>p</em>&#x02009;=&#x02009;3.29E&#x02212;09, colon: 0% vs 1% <em>p</em>&#x02009;=&#x02009;0.032, 0% vs 4% <em>p</em>&#x02009;=&#x02009;1.82E&#x02212;07, MAG150&#x02009;=&#x02009;2.07E&#x02212;09), Megasphaera (<strong>f</strong> cecum: 0% vs 4% <em>p</em>&#x02009;=&#x02009;0.031, colon: 0% vs 4% MAG053&#x02009;=&#x02009;0.0011) and Prevotella group 9 (<strong>g</strong> cecum: 0% vs 2% <em>p</em>&#x02009;=&#x02009;0.003, 0% vs 4% <em>p</em>&#x02009;=&#x02009;4.71E&#x02212;07, colon: 0 vs 2% <em>p</em>&#x02009;=&#x02009;0.010, 0% vs 4% <em>p</em>&#x02009;=&#x02009;3.96E&#x02212;06, MAG045&#x02009;=&#x02009;0.008, MAG196&#x02009;=&#x02009;0.010, MAG138&#x02009;=&#x02009;0.0018, MAG285&#x02009;=&#x02009;0.0024, MAG137&#x02009;=&#x02009;6.99E&#x02212;05, MAG191&#x02009;=&#x02009;0.0014). Due to space constraints, only a partial representation of Prevotella-affiliated MAG abundances is illustrated in <strong>g</strong>.</p></span></span></span><p id=\"Par9\">Monitoring of temporal microbiome changes using 16S rRNA gene analysis over the month-long trial indicated that the inclusion of AcGGM into the piglets feed caused a pronounced shift in the microbiome structural composition from the 21st day of the trial onwards (Supplementary Figs.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">2b</a></sup> and <sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3</a></sup>). Spatial changes were also examined at the final sampling day and showed typical patterns, whereby the structure of the gut microbiome varied by gut region (Supplementary Figs.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">2a</a></sup>, <sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3</a></sup>, and <sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">4</a></sup>). As expected, the AcGGM-effect was more pronounced in the fiber-fermenting distal regions (cecum, colon) of the gut, where the relative abundance of hundreds of phylotypes was observed to change (adjusted <em>p</em>&#x02009;&#x0003c;&#x02009;0.05) in response to varying inclusion levels (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3a</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM5\" rid=\"MOESM5\" class=\" media\">2</a></sup>). To determine the effect AcGGM had on microbiome function, we also closely examined the 355 metagenome-assembled genomes (MAGs, Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">5a</a></sup>) that were reconstructed from metagenomic data generated from the colon samples of each pig fed the control and 4% AcGGM diets (Supplementary Table&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">2</a></sup>). The taxonomic affiliation of the MAGs was calculated via GTDB-Tk (Supplementary Data&#x000a0;<sup><a href=\"#MOESM6\" rid=\"MOESM6\" class=\" media\">3</a></sup>), while phylogeny was inferred from a concatenated ribosomal protein tree (Newick format available in Supplementary Data&#x000a0;<sup><a href=\"#MOESM7\" rid=\"MOESM7\" class=\" media\">4</a></sup>) that was constructed using MAGs from this study and 293 closely related reference genomes.</p><p id=\"Par10\" class=\"p p-last\">Our target butyrogenic populations produced mixed results, whereby the 16S rRNA gene relative abundance of <em>Faecalibacterium</em> affiliated phylotypes increased in response to increasing levels of AcGGM (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3c</span></a> and Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3c</a></sup>), whereas <em>Roseburia</em>-affiliated phylotypes seemingly decreased (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3b</span></a> and Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3b</a></sup>). However, a detailed analysis of <em>Roseburia-</em>affiliated MAGs (GTDB-Tk assigned as <em>Agathobacter</em>, see Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4</span></a>) showed that specific phylotypes that encoded AcGGM-specific PULs were indeed stimulated by the AcGGM fiber (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3b</span></a>). Reputable fiber-fermenting populations affiliated to <em>Prevotella</em><sup><sup><a href=\"#CR20\" rid=\"CR20\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780235\">20</a></sup></sup> also showed varying responses (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3g</span></a>), with 16S rRNA gene relative abundance of individual phylotypes increasing from 4 to 12% between the control and 4% AcGGM inclusion in both colon and cecum (Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3a</a></sup>). Interestingly, relative abundance estimates of both 16S rRNA gene OTUs and MAGs indicated that phylotypes affiliated to non-fiber-degrading taxa, such as <em>Catenibacterium</em><sup><sup><a href=\"#CR21\" rid=\"CR21\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_915796625\">21</a></sup></sup>, <em>Dialister</em><sup><sup><a href=\"#CR22\" rid=\"CR22\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780177\">22</a></sup></sup>, and <em>Megasphaera</em>, demonstrated some of the highest dose-dependent increases in relative abundance in response to AcGGM (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3d&#x02013;f</span></a>), indicating that other underlying factors are likely dictating microbiome structure, besides fiber degradation.</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig4\" co-legend-rid=\"lgnd_Fig4\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig4/\" target=\"figure\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140434879423392\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7666174_41467_2020_19585_Fig4_HTML.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2020_19585_Fig4_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC7666174/bin/41467_2020_19585_Fig4_HTML.jpg\" /></a></span><span id=\"largeobj_idm140434879423392\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig4/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_Fig4\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig4/\" target=\"figure\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\">Fig. 4</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Genome-centric metaproteomic analysis of colon samples collected from pigs fed either the control of 4% AcGGM diet.</strong><p id=\"__p4\"><strong>a</strong> Principle Component Analysis of metaproteomes generated from randomly selected colon samples collected from pigs fed either the control (C1-4) or 4% AcGGM (M1-4) diet. A clear separation was observed between the two diets, highlighting that the detected proteins in the colon microbiomes from piglets fed the control, and 4% AcGGM diets were distinct. <strong>b</strong> Hierarchal clustering and heatmap of detected protein group abundance profiles. Rows are scaled so that red represents the highest abundance for that protein group and blue the lowest. Five different clusters were observed, with protein groups differentially detected in AcGGM-fed pigs (M1-4: red), control pigs (C1-4: blue), all pigs (M1-4&#x02009;+&#x02009;C1-4: purple), AcGGM-fed pigs plus one control (M1-4&#x02009;+&#x02009;C4: brown) and only in individual pigs (Individual: green). <strong>c</strong> Phylogeny and metaproteomic detection of 355 MAGs sampled from the colon of weaned piglets. This maximum likelihood tree is based on an alignment of 22 concatenated ribosomal proteins from the 355 MAGs reconstructed in this study from 24 colon metagenomes (12 control pigs, 12 fed 4% AcGGM diet), in addition to 239 reference genomes closely related to the MAGs. Branches are shaded with color to highlight phylum-level affiliations (see legend). Colored bars on the outside of the tree depict the average number of unique and shared protein groups detected for each MAG in four randomly selected samples that were analyzed from pigs fed either the control- (gray) or 4% AcGGM diet (green). Total number of proteins for each MAG in each pig is detailed in Supplementary Data&#x000a0;<sup><a href=\"#MOESM8\" rid=\"MOESM8\" class=\" media\">5</a></sup>. Purple circles on the inside of the tree represent nodes with bootstrap support &#x02265;70%, relative to size. MAG041 and MAG243 were found to encode CE2/CE17-containing mannan PULs (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span>5</span></a>) are indicated by *. All MAGs depicted in Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig6/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\" co-legend-rid=\"lgnd_Fig6\"><span>6</span></a> (blue circles) and Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig7/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span>7</span></a> (red circles) are listed, with average detected protein counts in both diets, which metaproteomic expression cluster they are enriched in (part <strong>b</strong>, adjusted <em>p</em>-values are indicated: * denotes <em>p</em>&#x02009;&#x0003c;&#x02009;0.05, ** denotes <em>p</em>&#x02009;&#x0003c;&#x02009;0.001, exact values listed in Supplementary Data&#x000a0;<sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">7</a></sup>), and their MAG taxonomic affiliation determined via GTDB-Tk. MAG enrichment analysis was performed using the hypergeometric distribution function phyper in R with the false discovery rate controlled at 5% using the function <em>p</em>.adjust with method&#x02009;=&#x02009;&#x02018;BH&#x02019;. Recently reclassified <em>Roseburia</em> species are denoted as Agathobacter. The full tree in Newick format is provided in Supplementary Data&#x000a0;<sup><a href=\"#MOESM7\" rid=\"MOESM7\" class=\" media\">4</a></sup>.</p></span></span></span></span><span id=\"s\r\n5\" class=\"sec\"><h3 id=\"s\r\n5title\">Targeted mannan PULs were detected in the colon of AcGGM-fed pigs</h3><p id=\"Par11\" class=\"p p-first\">Because our primary goal was to elucidate whether our target, butyrogenic populations were activated in response to AcGGM, we annotated MAGs affiliated to butyrate-producers and conducted the metaproteomic analysis with label-free quantification (LFQ) on randomly selected colon samples from four control and four 4% AcGGM-fed pigs (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4a, b</span></a>), and mapped 8515 detected protein groups back to our MAGs to identify functionally active populations (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4c</span></a>) (Supplementary Data&#x000a0;5 and <sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). Community-wide analysis of the MAG genetic content (Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">5b</a></sup>) from each sample and clustering analysis of their detected proteins (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4a, b</span></a>) further supported our 16S rRNA gene analysis, reiterating that the microbiomes from piglets fed the control and 4% AcGGM diets were distinct.</p><p id=\"Par12\">Our MAG-centric multi-omic approach gave clear indications as to what effect the AcGGM fiber had on putative butyrogenic <em>Roseburia</em> and <em>Faecalibacterium</em> populations in the distal gut of pigs. Fifteen MAGs clustered with representative <em>Roseburia</em> spp. genomes (Figs.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3b</span></a> and <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span style=\"position: relative;text-decoration:none;\">&#x200B;<span class=\"figpopup-sensitive-area\" style=\"left: -1.5em;\">and4),</span></span><span>4</span></a>), which reflected the multiple <em>Roseburia</em>-affiliated phylotypes that were predicted with our 16S rRNA gene analysis (Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3b</a></sup> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM5\" rid=\"MOESM5\" class=\" media\">2</a></sup>). In general, the relative abundance of <em>Roseburia</em>-affiliated MAGs (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3b</span></a>) and detected proteins (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4c</span></a>) were observed at either static or lower levels in AcGGM-fed pigs, reiterating our initial 16S rRNA gene observations that AcGGM negatively affected <em>Roseburia</em> populations (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3b</span></a>). However, one specific <em>Roseburia</em>-affiliated population (MAG041) was detected at significantly higher abundance levels (<em>p</em>&#x02009;=&#x02009;0.0016) (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3b</span></a>), and its detected proteins were enriched (adj. <em>p</em>&#x02009;=&#x02009;0.0034) in the 4% AcGGM pig samples compared to the control (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4b, c</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">7</a></sup>).</p><p id=\"Par13\">Closer examination of MAG041 revealed a putative CE2/CE17-containing mannan-degrading PUL that was absent in the other <em>Roseburia</em>-affiliated MAGs and was differentially expressed in the AcGGM diet (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span>5</span></a>). Importantly, the MAG041 mannan PUL exhibited gene synteny to the <em>R. intestinalis</em> strain L1-82 PUL whose biochemical properties we recently characterized in detail<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780255\">5</a></sup></sup> (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span>5</span></a>). The predicted multi-modular mannanase (CBM27-GH26-CBM23) in the MAG041 mannan PUL is homologous to the GH26 in <em>R. intestinalis</em> L1-82 (48% identity over 87% of the sequence), and can be presumed to fulfill the same function&#x02014;&#x02018;selfishly&#x02019; breaking down AcGGM fibers at the cell surface prior to intracellular transport. Besides the detection of GH26 and esterases in AcGGM-fed pigs, other mannan-specific enzymes also responded to the dietary shift within the MAG041 mannan PUL, including a phosphoglucomutase, a multiple-sugar binding protein, a GH130.1 4-<em>O</em>-&#x003b2;-<span class=\"small-caps\">d</span>-mannosyl-<span class=\"small-caps\">d</span>-glucose phosphorylase and a GH130.2 &#x003b2;-1,4-manno-oligosaccharide phosphorylase (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span>5</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig5\" co-legend-rid=\"lgnd_Fig5\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig5/\" target=\"figure\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140434879781408\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7666174_41467_2020_19585_Fig5_HTML.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2020_19585_Fig5_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC7666174/bin/41467_2020_19585_Fig5_HTML.jpg\" /></a></span><span id=\"largeobj_idm140434879781408\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig5/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_Fig5\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig5/\" target=\"figure\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\">Fig. 5</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Metaproteomic detection of CE2/CE17-containing mannan PULs encoded in <em>Roseburia</em>- (MAG041) and <em>Faecalibacterium</em>- (MAG243) affiliated MAGs in pigs fed with either the control or 4% AcGGM diet.</strong><p id=\"__p5\">Predicted gene organization and annotated gene function is color-coded and largely derived from the previous biochemical and structural characterization of the mannan degradation cluster (characterized genes indicated with *) in <em>R. intestinalis</em> L1-82<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780247\">5</a></sup></sup>. Gene synteny and identity % between mannan PULs found in <em>R. intestinalis</em> L1-82, MAG041 and MAG243 are indicated in gray boxes. Heat maps above detected enzymes show the LFQ detection levels for the four replicates sampled in control and 4% AcGGM-fed pigs. LFQ values of proteins from both clusters are in Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>. The predicted multi-modular mannanase (CBM27-GH26-CBM23) from MAG041 was the only extracellular protein in the locus, and the only extracellular mannanase expressed in response to AcGGM inclusion.</p></span></span></span><p id=\"Par14\" class=\"p p-last\">In contrast to <em>Roseburia</em>-affiliated MAGs, only one MAG clustered with <em>F. prausnitzii</em> (MAG243, Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM6\" rid=\"MOESM6\" class=\" media\">3</a></sup>), implying that the multiple phylotypes that were predicted with our 16S rRNA gene data (Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3c</a></sup> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM5\" rid=\"MOESM5\" class=\" media\">2</a></sup>) encode high genome similarity and coverage and thus likely co-assembled into a representative population-level MAG. The relative abundance of MAG243 in the distal gut of pigs fed 4% AcGGM increased ~5-fold to ~1.24% (<em>p</em>&#x02009;=&#x02009;5.84E&#x02212;06) (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3c</span></a>), while metaproteomic hierarchical clustering analysis showed that MAG243 proteins were enriched in the same samples (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4b, c</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">7</a></sup>). Similar to MAG041, MAG243 was found to encode a CE2/CE17-containing mannan PUL, which was broadly detectable in the presence of AcGGM but absent in the control samples (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span>5</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). While the MAG243 mannan PUL contained two GH130 manno-oligophosphorylases, a mannose 6-phosphate isomerase, phosphoglucomutase, and two carbohydrate esterase (CE17 and CE2), it lacked a GH26 mannanase representative, which suggests that <em>F. prausnitzii</em> is likely preferentially targeting the shorter acetylated manno-oligosaccharides that form part of the AcGGM structure (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig1\" rid-ob=\"ob-Fig1\" co-legend-rid=\"lgnd_Fig1\"><span>1</span></a>). In addition to the mannan PULs of MAG041 and MAG243 being activated in AcGGM-fed pigs, their butyrogenic pathways were also detected at high levels, based on label-free quantification (LFQ) scores of detected proteins (Figs.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig6/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\" co-legend-rid=\"lgnd_Fig6\"><span>6</span></a>, <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig7/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span style=\"position: relative;text-decoration:none;\">&#x200B;<span class=\"figpopup-sensitive-area\" style=\"left: -0.5em;\">,7</span></span><span>7</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>), suggesting that both populations can convert mannan to butyrate (Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig6\" co-legend-rid=\"lgnd_Fig6\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig6/\" target=\"figure\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140434875663328\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7666174_41467_2020_19585_Fig6_HTML.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2020_19585_Fig6_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC7666174/bin/41467_2020_19585_Fig6_HTML.jpg\" /></a></span><span id=\"largeobj_idm140434875663328\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig6/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_Fig6\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig6/\" target=\"figure\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\">Fig. 6</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>The differing metabolisms of butyrate-producers in the colon microbiome of pigs fed AcGGM.</strong><p id=\"__p6\">A plethora of MAGs affiliated with putative butyrate-producers was recovered from the pig colon metagenome, which varied in their fiber-degrading capacity and detected activity in either the control (C) or 4% AcGGM (M) diets. Here, we present a representative selection of populations that: encoded specific AcGGM-degrading capabilities (MAG041 and MAG243), were abundant across both diets with no apparent effect (MAG292), were significantly enriched in pigs fed the control diet (MAG133, MAG324, and MAG269), and encoded an alternative pathway for butyrate production (via Butyrate kinase) (MAG324). GTBD-Tk inferred taxonomy (recently reclassified <em>Roseburia</em> species are denoted as <em>Agathobacter</em>), MAG relative abundance (control: black circles, AcGGM: green circles, <em>p</em>-value: MAG041&#x02009;=&#x02009;0.002, MAG243&#x02009;=&#x02009;5.84E&#x02212;06, MAG292&#x02009;=&#x02009;0.98, MAG133&#x02009;=&#x02009;0.051, MAG324&#x02009;=&#x02009;0.120, MAG269&#x02009;=&#x02009;0.035) and MAG-centric metaproteomic enrichment analysis is indicated (see Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4c</span></a>, significant differences denoted by adjusted <em>p</em>-values: *<em>p</em>&#x02009;&#x0003c;&#x02009;0.05, **<em>p</em>&#x02009;&#x0003c;&#x02009;0.001, MAG041&#x02009;=&#x02009;3.4E&#x02212;03, MAG243&#x02009;=&#x02009;8.5E&#x02212;19, MAG292&#x02009;=&#x02009;5.2E&#x02212;06, MAG133&#x02009;=&#x02009;2.8E&#x02212;18, MAG324&#x02009;=&#x02009;4.9E&#x02212;02, MAG269&#x02009;=&#x02009;1.9E&#x02212;05). MAG enrichment analysis was performed using the hypergeometric distribution function phyper in R with the false discovery rate controlled at 5% using the function <em>p</em>.adjust with method&#x02009;=&#x02009;&#x02018;BH&#x02019;. CAZymes (GH, CE, and GT) involved in the catabolism of the listed polysaccharides are grouped according to box color and their specific activities are described on <a href=\"http://www.cazy.org\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">www.cazy.org</a>. Enzymes associated with acetate and butyrate metabolic pathways are listed as EC (enzyme commission) numbers. Corresponding functions can be found on <a href=\"https://www.genome.jp/kegg/kegg2.html\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://www.genome.jp/kegg/kegg2.html</a>.</p></span></span></span><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"Fig7\" co-legend-rid=\"lgnd_Fig7\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig7/\" target=\"figure\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140434882072752\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7666174_41467_2020_19585_Fig7_HTML.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 41467_2020_19585_Fig7_HTML.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC7666174/bin/41467_2020_19585_Fig7_HTML.jpg\" /></a></span><span id=\"largeobj_idm140434882072752\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig7/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_Fig7\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig7/\" target=\"figure\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\">Fig. 7</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Selected metabolic features of the porcine colon microbiome in response to AcGGM dietary intervention, as inferred from genome and proteome comparisons.</strong><p id=\"__p7\">The different metabolic pathways (fiber deconstruction, glycolysis, pentose-phosphate pathway, and SCFA production) are displayed for each population MAG. Graphical representation of pathways, enzymes, CAZymes, and cellular features are based on functional annotations that are depicted as numbered or abbreviated gene boxes, which are additionally listed in Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>. Metaproteomic analysis (detected genes and enzyme complexes) is highlighted according to the different LFQ values of the detected proteins sampled at a single time-point (28 days) either from animals fed the control or 4% AcGGM diet (green perforated boxes). The main dietary fibers (starches, xylans, glucans, and mannans), SCFAs (butyrate, acetate, propionate), and intermediates like lactate and succinate are represented by large colored arrows. GTBD-Tk inferred taxonomy is included (recently reclassified <em>Roseburia</em> species are denoted as Agathobacter). Gene names and abbreviations are also provided in Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>.</p></span></span></span></span><span id=\"s\r\n6\" class=\"sec\"><h3 id=\"s\r\n6title\">Specific removal of acetylations is key to access AcGGM</h3><p id=\"Par15\" class=\"p p-first-last\">A crucial step in the utilization of mannans as an energy source is the deacetylation of 2-<em>O</em>-, 3-<em>O</em>- and 6-<em>O</em>-mannose residues, which allows the subsequent breakdown of the sugar-containing backbone of the fiber. In <em>R. intestinalis</em> L1-82, AcGGM deacetylation occurs via the synergistic actions of two carbohydrate esterases (<em>Ri</em>CE2 and <em>Ri</em>CE17) that exert complementary specificities<sup><sup><a href=\"#CR9\" rid=\"CR9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780203\">9</a></sup></sup>. MAG041 and MAG243 both encoded CE2 homologs within their mannan PULs, sharing 63 and 31% identity (respectively) to <em>Ri</em>CE2, which has demonstrated activity on 3-<em>O</em>-, (4-<em>O</em>-) and 6-<em>O</em>-acetylations, and is mannan specific<sup><sup><a href=\"#CR9\" rid=\"CR9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780191\">9</a></sup></sup>. For CE17, MAG041 and MAG243 homologs shared 65 and 46% identity (respectively) with <em>Ri</em>CE17, including the active site residues and the aromatic stacking tryptophan (Trp326), which in <em>Ri</em>CE17 are associated with 2-<em>O</em>-acetylation specificity<sup><sup><a href=\"#CR9\" rid=\"CR9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780196\">9</a></sup></sup>. Broader screens of our MAG data revealed other CE2/CE17-containing PULs within Firmicute-affiliated MAGs from the pig colon microbiome (Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">6</a></sup>), however, aside from MAG041 and MAG243, they originated from populations whose MAG relative abundance was very low (&#x0003c;0.05%) and metabolic activity was undetectable via metaproteomics in any of the control or AcGGM diets (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4</span></a>). Finally, the differential proteomic detection of MAG041 and MAG243 CEs in pigs fed AcGGM diets (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig5\" rid-ob=\"ob-Fig5\" co-legend-rid=\"lgnd_Fig5\"><span>5</span></a>), strengthened our hypothesis that both these populations can accommodate the unique features of the AcGGM fiber and are actively engaging in its utilization in vivo.</p></span><span id=\"s\r\n7\" class=\"sec sec-last\"><h3 id=\"s\r\n7title\">AcGGM also causes an effect in non-target populations</h3><p id=\"Par16\" class=\"p p-first\">MDFs studies to date have eloquently highlighted that metabolic symmetry between individual fibers and microbiota can be used to stimulate specific populations<sup><sup><a href=\"#CR1\" rid=\"CR1\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780245\">1</a></sup>,<sup><a href=\"#CR16\" rid=\"CR16\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780179\">16</a></sup></sup>. However, deeper microbiome effects that result from a target species being enriched via MDFs are poorly understood. Although the specificity of the AcGGM fiber matched selected mechanistic features of our target populations, our data showed that AcGGM dietary intervention reverberated further down the microbial trophic networks that support the conversion of dietary fiber into keystone SCFAs that are of nutritional value to the host animal. In particular, we observed varying effects upon different non-AcGGM-degrading butyrogenic populations belonging to clostridial cluster XIVa (Lachnospiraceae: <em>Roseburia</em> spp., <em>Eubacterium rectale</em>, <em>Butyrivibrio</em> and <em>Pseudobutyrivibrio</em>) and cluster IV (Ruminococcaceae: <em>Faecalibacterium prausnitzii</em>-related bacteria)<sup><sup><a href=\"#CR23\" rid=\"CR23\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780199\">23</a></sup></sup> (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig6/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\" co-legend-rid=\"lgnd_Fig6\"><span>6</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). Specific populations that were closely related to AcGGM-degrading MAG041 were found to be either unaffected (MAG292) or metabolically suppressed (MAG133) in the presence of 4% AcGGM. Indeed, detected proteins from MAG133 were enriched in pigs fed the control diet (adj. <em>p</em>&#x02009;=&#x02009;2.8E&#x02212;18, Supplementary Data&#x000a0;<sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">7</a></sup>), and enzymes associated with butyrate production were largely undetected in 4% AcGGM-fed pigs (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig6/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\" co-legend-rid=\"lgnd_Fig6\"><span>6</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). In most cases, non-AcGGM degrading, butyrogenic populations were seemingly utilizing sugars found in starch, arabinoxylan, and/or arabinogalactan fibers that were detected in the basal feed components using Micro Array Polymer Profiling (MAPP; Supplementary Data&#x000a0;<sup><a href=\"#MOESM11\" rid=\"MOESM11\" class=\" media\">8</a></sup>). A broader analysis of our omic data identified multiple abundant populations with similar metabolic capabilities, suggesting that suppressed butyrogenic populations were possibly being outcompeted by Prevotella-affiliated populations such as MAG285 (see below). While SCFA measurements indicated an increase in relative butyrate levels in AcGGM-fed pigs (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig2\" rid-ob=\"ob-Fig2\" co-legend-rid=\"lgnd_Fig2\"><span>2b</span></a>), the indirect effects of our MDF on this dynamic functional group are likely dampening an even greater influence of AcGGM-stimulated <em>Roseburia</em> and <em>Faecalibacterium</em> populations upon the broader microbiome and host animal (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig6/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig6\" rid-ob=\"ob-Fig6\" co-legend-rid=\"lgnd_Fig6\"><span>6</span></a>).</p><p id=\"Par17\">Besides butyrate-producers, MAG-centric metaproteomic enrichment analysis showed that fiber-degrading <em>Prevotella</em>-affiliated populations experienced contrasting effects as a result of AcGGM inclusion. For example, MAG191 was found to account for the highest levels of detectable proteins in our datasets (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4c</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>), which were enriched in clusters differently detected in the presence of 4% AcGGM (adj. <em>p</em>&#x02009;=&#x02009;0.0023, Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). Pathway annotation of AcGGM-enriched <em>Prevotella</em> populations (such as MAG191, MAG196, MAG285, see Figs.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4c</span></a>, <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig7/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span style=\"position: relative;text-decoration:none;\">&#x200B;<span class=\"figpopup-sensitive-area\" style=\"left: -0.5em;\">,7,</span></span><span>7</span></a>, and Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>, <sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">7</a></sup>) indicated active metabolism of dietary fibers such arabinoxylans, starch, glucans (e.g., cellobiose), &#x003b1;-galactans, and mannose sugars (detected via MAPP analysis, Supplementary Data&#x000a0;<sup><a href=\"#MOESM11\" rid=\"MOESM11\" class=\" media\">8</a></sup>) as well as acetate, succinate and/or propionate production, which were all detected with higher LFQ scores in AcGGM-fed pigs (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig7/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span>7</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). However, many CAZymes and fermentation enzymes from the aforementioned MAGs were also detected across both diets (albeit at varying LFQ values), while other <em>Prevotella</em> populations were specifically enriched in pigs fed the control diet (i.e., MAG034: adj. <em>p</em>&#x02009;=&#x02009;2.0E&#x02212;04, Supplementary Data&#x000a0;<sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">7</a></sup>), suggesting that <em>Prevotella</em>-driven baseline consumption of basal feed fiber was occurring irrespective of AcGGM inclusion (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig7/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span>7</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>).</p><p id=\"Par18\">Several mannan-targeting PULs were identified in <em>Prevotella</em>-affiliated MAGs that were configured in an archetypical &#x02018;Bacteroidetes-format&#x02019;, which combines outer-membrane transport and carbohydrate-binding SusC/D-like proteins as well as CAZymes<sup><sup><a href=\"#CR24\" rid=\"CR24\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780221\">24</a></sup></sup> (Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">6a</a></sup>). In particular, a PUL recovered from MAG196 encoded predicted SusC/D-like proteins, mannanases (GH26, GH5_7), mannosyl-phosphorylases (GH130), and an esterase, although neither the mannanases nor the esterase was detected in the metaproteomes recovered from the AcGGM-fed pigs (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig7/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span>7</span></a>, Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">6</a></sup> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). In addition, we speculate that MAG196 and MAG191 are perhaps capable of metabolizing elements of the AcGGM fiber such as the &#x003b1;-galactose side-chain or deacetylated manno-oligosaccharides, which was inferred via detected GH36 and GH130 representatives (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig7/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span>7</span></a>, Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">6</a></sup> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>).</p><p id=\"Par19\">Dietary inclusion of AcGGM also unexpectedly led to pronounced metabolic shifts in numerous phenotypically and phylogenetically diverse microbial populations that are not normally associated with fiber hydrolysis. Mirroring our 16S rRNA gene analysis, MAGs affiliated to the genera <em>Dialister</em> (MAG150), <em>Catenibacterium</em> (MAG048), and <em>Megasphaera</em> (MAG053) were the three highest enriched populations in expression clusters differentially detected in the AcGGM diet (adj. <em>p</em>&#x02009;&#x0003c;&#x02009;0.001), although none were found to encode CE2/CE17-containing mannan PULs (Figs.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>3</span></a>&#x02013;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4</span></a>, <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig7/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span style=\"position: relative;text-decoration:none;\">&#x200B;<span class=\"figpopup-sensitive-area\" style=\"left: -0.5em;\">,7</span></span><span>7</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). In particular, the MAG048 proteome included a putative sugar phosphotransferase system (PTS) and GH1 phospho-&#x003b2;-glucosidases (EC 3.2.1.86) that are predicted to catalyze the phosphorylation of disaccharides (such as cellobiose and mannobiose<sup><sup><a href=\"#CR25\" rid=\"CR25\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780226\">25</a></sup>,<sup><a href=\"#CR26\" rid=\"CR26\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780207\">26</a></sup></sup>) and hydrolyze the PTS-transported sugars into <span class=\"small-caps\">d</span>-glucose and <span class=\"small-caps\">d</span>-glucose 6-phosphate. Concomitantly, glycolysis, and acetogenesis pathways from MAG048 were also highly detected in AcGGM-fed pigs (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig7/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span>7</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). Together, these data suggest that certain <em>Catenibacterium</em> populations are advantageously consuming AcGGM-derived disaccharides that were either available as a fraction of the original AcGGM fiber preparation, have been generated via the actions of other fiber-degrading populations, or have become available via new ecological niches that have been created via the AcGGM-derived structural shifts in the microbiome.</p><p id=\"Par20\" class=\"p p-last\">Both MAG053 (<em>Megasphaera</em>) and MAG150 (<em>Dialister</em>) were predicted via our multi-omics approach to metabolize SCFAs such as lactate and succinate that were generated endogenously by <em>Lactobacillus-</em> (i.e., MAG013) and <em>Prevotella-</em> (i.e., MAG285) affiliated populations, and produce butyrate and propionate, respectively (Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig7/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig7\" rid-ob=\"ob-Fig7\" co-legend-rid=\"lgnd_Fig7\"><span>7</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). Neither of these cross-species relationships had any evidence of being directly influenced by AcGGM and are presumably being driven by underlying availabilities of fermentation intermediates (e.g., lactate and succinate) resulting from as-yet uncharacterized niche fluctuations. Collectively, these data highlight that while a highly tailored MDF can directly activate enzymatically equipped populations in an endogenous microbiome, they can also inadvertently alter the metabolisms of other non-mannanolytic populations creating effects that should be quantified to ascertain the true modulation potential of an MDF. The community dynamics that drive fiber-fermentation is not well understood and is likely a contributory factor that restricts broader MDF development.</p></span></span><span id=\"s\r\n8\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n8title\">Discussion</h2><p id=\"Par21\" class=\"p p-first\">Here, we have advanced the MDF concept by showing that the defined relationship between fiber and microbe can be used to directly target specific microbiota in highly complex and competitive gut ecosystems. This was achieved by matching the structural features of our characterized AcGGM fiber to a specific genetic locus in our target microbiota, for which there exists comprehensive biochemical information pertaining to the encoded enzymatic functions.</p><p id=\"Par22\" class=\"p p-last\">Using integrated multi-omics analysis, we illustrate the impact of the AcGGM structural configuration on microbial uptake and metabolism within the distal regions of the porcine digestive tract, with the key driver of AcGGM selectivity being the presence of acetylations of mannan, as well as carbohydrate composition and size<sup><sup><a href=\"#CR27\" rid=\"CR27\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780251\">27</a></sup></sup>. Preserving the complexity of AcGGM resulted in a highly specific shift in the composition of the colon microbiome from weaned piglets, with increases in the relative butyrate:acetate ratios and no adverse effect on host growth performance or health status. We showed that the AcGGM fiber activated a metabolic response in specific <em>Roseburia</em> and <em>Faecalibacterium</em> populations in vivo, as it did in the previous in vitro experiments<sup><sup><a href=\"#CR5\" rid=\"CR5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780176\">5</a></sup>,<sup><a href=\"#CR10\" rid=\"CR10\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780211\">10</a></sup></sup>, with both populations expressing proteins from unique CE2/CE17-containing mannan PULs that are homologous to a biochemically characterized representative in <em>R. intestinalis</em> L1-82. In conclusion, our data provide a foundation for modulatory strategies to design and match custom dietary fibers to unique enzymatic features of their target organisms. However, they also underscore the fact that the greater network of interconnected metabolic exchanges and trophic structures inherent to the gut microbiome is highly susceptible to minor dietary interventions. Ultimately, further research is required to determine whether AcGGM MDFs can confer improvements to animal performance metrics (such as growth and disease resistance) in the context of full-scale livestock production.</p></span><span id=\"s\r\n9\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n9title\">Methods</h2><span id=\"s\r\n10\" class=\"sec sec-first\"><h3 id=\"s\r\n10title\">Animals, diets, and experimental design</h3><p id=\"Par23\" class=\"p p-first\">Animal care protocols and experimental procedures were approved by the Norwegian Animal Research Authority, approval no. 17/9496, FOTS ID 11,314, and treated according to institutional guidelines. A total of 48 crossbred piglets (Landrace&#x02009;&#x000d7;&#x02009;Yorkshire), 24 male and 24 female, with average initial body weight (BW) of 9.8&#x02009;&#x000b1;&#x02009;0.5&#x02009;kg and weaned at 28 days of age, were sorted by litter, sex, and weight and randomly divided into twelve pens of four animals, with three pens fed each of the four diets. The piglets in each pen were trained to feed at individual stations containing their respective feed. The animals were housed in an environmentally controlled facility with plastic flooring and a mechanical ventilation system. The temperature of the room was maintained at 22&#x02009;&#x000b0;C.</p><p id=\"Par24\" class=\"p p-last\">Piglets were fed cereal-based diets containing increasing levels of AcGGM diets (1, 2, and 4%). Inclusion levels were selected based on in-house experience from previous feeding trials and communication with the feed industry, where an inclusion level of 1&#x02013;4% of MDF in feed would be realistic. The highest inclusion level at 4% was selected in order to test a dose response. Diets were pelleted with a 3&#x02009;mm diameter feed formulated to meet the requirements for indispensable amino acids and all other nutrients (NRC, 2012). The composition of diets is listed in Supplementary Table&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">3</a></sup>. Pigs were fed semi-ad libitum twice a day at a feeding level equal to about 5% of body weight. To evaluate growth performance, the BW of each pig was recorded at the beginning and once a week. Feed consumption was recorded for each individual pig during the experiment to calculate individual weight gain and feed intake. After each meal, feed leftovers were registered, dried, and subtracted from the total feed intake.</p></span><span id=\"s\r\n11\" class=\"sec\"><h3 id=\"s\r\n11title\">Production of AcGGM</h3><p id=\"Par25\" class=\"p p-first-last\">AcGGM oligosaccharides for the feeding trial were produced from Norway spruce chips milled with a hammer mill to &#x0003c;2&#x02009;mm size. Wood chips were then steam-exploded on a pilot-scale steam explosion rig (100&#x02009;L reactor vessel) at the Norwegian University of Life Sciences (NMBU). The steam explosion was conducted in batches of approximately 6&#x02009;kg dry matter, 14.5&#x02009;bar pressure (equivalent to 200&#x02009;&#x000b0;C), with 10&#x02009;min residence time. The pH in the collected biomass slurry after the steam explosion was ~3.7, which corresponds to a combined severity factor R&#x02032;<sub>0</sub>&#x02009;=&#x02009;1.70 for the process. The combined severity (as per<sup><sup><a href=\"#CR28\" rid=\"CR28\" class=\" bibr popnode\">28</a></sup></sup>)was calculated by <em>R</em>&#x02032;<sub>0</sub>&#x02009;=&#x02009;(10<sup>&#x02212;pH</sup>)&#x02009;&#x000d7;&#x02009;(<em>t</em>&#x02009;&#x000d7;&#x02009;<em>e</em>(<sup>Texp&#x02212;100)/14.75</sup>). Steam-exploded wood was collected in 50&#x02009;L plastic buckets that were topped up with hot (~70&#x02009;&#x000b0;C) water. The slurry was transferred to a 60&#x02009;L cider press (Speidel, Germany) and the liquid fraction was pressed out. Milled wood was collected, soaked in hot water again, and pressed for the second time. The liquid fraction was collected and recirculated through a bag filter 50&#x02009;&#x000b5;m pore WE50P2VWR (Allied filter systems, England) partly filled with the wood particles as a filter aid. Once free of floating wood particles, the liquid fraction of hemicellulose was filtered through a 5-kDa spiral wound Polysulphone/polyethersulphone ultrafiltration membrane, GR99PE polyester (Alfa Laval, Denmark) that was deliberately fouled to prevent larger oligosaccharides from running through the permeate, using a GEA pilot-scale filtration system Model L (GEA, Denmark). The fraction retained by the membrane was concentrated by nanofiltration using a TriSep XN 45, which had a higher efficiency for permeating water. The filtrate was further concentrated by vacuum evaporation (set to 65&#x02009;&#x000b0;C) and the concentrate was freeze-dried and homogenized with a grain mill. The final product consisted of 0.9% rhamnose, 2.7% arabinose, 13.7% xylose, 58.9% mannose, 14.9% glucose, and 9.4% galactose (determined by gas chromatography as alditol acetates after sulfuric acid hydrolysis<sup><sup><a href=\"#CR29\" rid=\"CR29\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780214\">29</a></sup></sup>). AcGGM contained 0.73 % ash and 2.4% protein (quantified from total nitrogen by the Kjeldahl method). The Man:Glc:Gal ratio in the mannan was 4:1:0.6, and the DA&#x02009;=&#x02009;0.35<sup><sup><a href=\"#CR9\" rid=\"CR9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780257\">9</a></sup></sup> (determined by acetate release from NaOH treated AcGGM by the same method as described in the SCFA section below). Moreover, previous two-dimensional (2D) <sup>1</sup>H-<sup>13</sup>C heteronuclear single quantum correlation (HSQC) NMR analysis of the AcGGM used in this study, showed the distribution of acetylations on mannose units, indicating that spruce mannan hydrolysate without enzyme addition contained prevalently 2-<em>O</em>-, some 3-<em>O</em>-acetylations, and a lower degree of 4-<em>O</em>- and 6-<em>O</em>-acetylations<sup><sup><a href=\"#CR9\" rid=\"CR9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780216\">9</a></sup></sup>. The dry matter content was determined by drying 0.2&#x02009;g of the sample at 105&#x02009;&#x000b0;C for 20&#x02009;h. The remaining sample was burned at 600&#x02009;&#x000b0;C for 24&#x02009;h in an oven (Carbolite, Sheffield, England) to determine ash content. All measurements were performed in triplicates.</p></span><span id=\"s\r\n12\" class=\"sec\"><h3 id=\"s\r\n12title\">Fecal scoring</h3><p id=\"Par26\" class=\"p p-first-last\">During the experiment, fecal consistency was assessed using a scoring system developed by Pedersen and Toft<sup><sup><a href=\"#CR30\" rid=\"CR30\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780172\">30</a></sup></sup> to improve and help standardize current protocols for clinical characterization of fecal consistency. The scoring was based on the following 4 consistency categories: score 1&#x02009;=&#x02009;firm and shaped, score 2&#x02009;=&#x02009;soft and shaped, score 3&#x02009;=&#x02009;loose, and score 4&#x02009;=&#x02009;watery. Samples with a score 3 or 4 are considered diarrheic. Daily fecal scores for each pen were recorded throughout the trial.</p></span><span id=\"s\r\n13\" class=\"sec\"><h3 id=\"s\r\n13title\">pH measurements</h3><p id=\"Par27\" class=\"p p-first-last\">The pH of digesta samples from duodenum, jejunum, ileum, cecum, and colon were measured immediately after slaughter. Samples were placed in universal containers and pH measurements made using an Inolab pH7110 pH meter (WTW, Germany).</p></span><span id=\"s\r\n14\" class=\"sec\"><h3 id=\"s\r\n14title\">Blood sampling and flow cytometry</h3><p id=\"Par28\" class=\"p p-first\">Blood samples were collected from the same six piglets per diet at 0, 7, and 27 feeding days. The blood samples were taken 1&#x02013;2&#x02009;h post-prandial by venipuncture in the jugular vein while pigs were kept on their backs. Non-heparinized and K<sub>3</sub>EDTA vacuum tubes (Beckman Dickson Vacutainer System) were used to recollect serum and whole blood. The serum was isolated immediately by centrifugation at 1500&#x02009;&#x000d7;&#x02009;<em>g</em> at 4&#x02009;&#x000b0;C for 15&#x02009;min. Serum samples were split in PCR-tubes (200&#x02009;&#x000b5;L) and stored at &#x02212;80&#x02009;&#x000b0;C until analysis. Hematological and clinical analyses were performed with an Advia<sup>&#x000ae;</sup> 2120 Hematology System using Advia 2120 MultiSpecies System Software and clinical chemistry analyses were performed with Advia 1800 Chemistry System (both from Siemens AG Healthcare Sector).</p><p id=\"Par29\" class=\"p p-last\">For flow cytometry analysis, whole blood was diluted 1:1 in RPMI 1640 and kept on ice until single-cell isolation. For the isolation of peripheral blood mononuclear cells (PBMCs) blood was purified by centrifugation in a Ficoll gradient (Kreuzer et al. 2012). Then, isolated PBMCs were incubated with Fixable Yellow Dead Cell Stain Kit (Life Technologies, ThermoFisher Scientific Inc.) followed by primary monoclonal antibodies (mAbs), brief incubation with 30% normal pig serum to block Fc-receptors, and finally fluorescence-labeled secondary antibodies (Abcam plc, UK). To detect the intracellular CD3 epitope, surface-labeled cells were permeabilized with Intracellular Fixation and Permeabilization Buffer Set (eBioscience, Affymetrix Inc.) according to the manufacturer&#x02019;s instructions. Labeled cells were analyzed on a Gallios Flow Cytometer (Beckman Coulter, Inc.) and data were processed using Kaluza 1.5 software (both Beckman Coulter, Inc.). Cell gates were designed to select for single and viable mononuclear cells. Defined markers were used to identify the different immune subpopulations. For monocytes, antibodies against CD45, CD3, CD14, CD163, and MHCII were used (dilution listed in &#x02018;Reporting summary&#x02019;). To analyze regulatory T cells (T reg) the following antibodies were used: CD45, CD3, TCR &#x003b3;/&#x003b4;, CD4, CD8, FOXp3, and CD25, while CD45, CD8, NKp46, CD4, CD8, Ki67, and CD27 were used to identify T and NK cells (dilution listed in &#x02018;Reporting summary&#x02019;). The gating strategy used for flow cytometric data is presented in Supplementary Fig.&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">7</a></sup>.</p></span><span id=\"s\r\n15\" class=\"sec\"><h3 id=\"s\r\n15title\">Analysis of serum cytokines: MULTIPLEX</h3><p id=\"Par30\" class=\"p p-first-last\">Expression of GMCSF, IFNG, IL-1A, IL1B, IL-1RA, IL-2, IL-4, IL-6, IL-8, IL-10, IL-12, IL-18, and TNF&#x003b1; were measured in serum samples using MILLIPLEX MAP Porcine Cytokine and Chemokine Magnetic Bead Panel - Immunology Multiplex Assay (Merck Millipore) following the manufacturer instructions. The measurement was performed using a Bio-Plex MAGPIX Multiplex Reader (Bio-Rad).</p></span><span id=\"s\r\n16\" class=\"sec\"><h3 id=\"s\r\n16title\">Small intestine morphology</h3><p id=\"Par31\" class=\"p p-first-last\">The samples of the small intestine were collected on days 0 and 28 for the determination of intestinal morphology and integrity. Intestinal morphological measurements included the following indices: villus height (VH), crypt depth (CD), and VH:CD. The mean values of VH, CD, and their ratio were calculated. Histology evaluation was performed by the Veterinary Histophalogy Center, VeHiCe, Chile.</p></span><span id=\"s\r\n17\" class=\"sec\"><h3 id=\"s\r\n17title\">SCFA analysis</h3><p id=\"Par32\" class=\"p p-first-last\">Samples of digesta from the duodenum, jejunum, ileum, cecum, and colon of individual pigs were collected for SCFA analysis. 250&#x02009;mg or 250&#x02009;&#x000b5;L of a sample, depending on the source site were mixed 1:1 with 4&#x02009;mM H<sub>2</sub>SO<sub>4</sub>, homogenized by shaking at room temperature for 1&#x02009;h, and centrifuged at 12,000&#x02009;&#x000d7;&#x02009;<em>g</em> for 10&#x02009;min. The supernatant was collected with a syringe and filtered through a 0.22&#x02009;&#x000b5;m pore syringe filter. Samples were stored at &#x02212;20&#x02009;&#x000b0;C and centrifuged at 12,000&#x02009;&#x000d7;&#x02009;<em>g</em> before transferring aliquots into HPLC vials for analysis. SCFA content was analyzed by HPLC using a REZEX ROA-Organic Acid H&#x02009;+&#x02009;(Phenomenex, Torrance, California, USA) 300&#x02009;&#x000d7;&#x02009;7.8&#x02009;mm ion exclusion column, isocratic elution with 0.6&#x02009;mL/min 4&#x02009;mM H<sub>2</sub>SO<sub>4</sub> at 65&#x02009;&#x000b0;C. Eluting analytes were detected by UV at 210&#x02009;nm. The same HPLC method was used to determine the degree of acetylation, by measuring the acetate released from samples of AcGGM dissolved in 100&#x02009;mM KOH and relating the values to the mannose content of the sample. All data were analyzed using a two-tailed <em>t-</em>test.</p></span><span id=\"s\r\n18\" class=\"sec\"><h3 id=\"s\r\n18title\">Micro array polymer profiling (MAPP)</h3><p id=\"Par33\" class=\"p p-first-last\">All cell-wall preparations were analyzed similarly to Moller et al.<sup><sup><a href=\"#CR31\" rid=\"CR31\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780185\">31</a></sup></sup>. Briefly, cell-wall glycans were sequentially extracted from 10&#x02009;mg of each sample by incubation with 2&#x02009;&#x003bc;g/mL cellulase, pH 8.9 in 20&#x02009;mM tris buffer for 16&#x02009;h, 45&#x02009;&#x000b0;C to release residual glycans bound to cellulose. Each extract was mixed 50/50 with glycerol buffer (55.2% glycerol, 44% water, 0.8% Triton X-100) and spotted with four 5-fold dilutions and two technical replicates onto a nitrocellulose membrane with a pore size of 0.45&#x02009;&#x003bc;m (Whatman, Maidstone, UK) using an Arrayjet Sprint (Arrayjey, Roslin, UK). The arrays were probed using antibodies from the JIM, LM (Plant Probes), BioSupplies, series and a Carbohydrate Binding Module (CBM, NZYTech) detected using anti-rat (JIM, LM), anti-mouse (BioSupplies) or anti-His (CBM) secondary antibodies conjugated to alkaline phosphatase (Sigma Aldrich)<sup><sup><a href=\"#CR32\" rid=\"CR32\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780173\">32</a></sup></sup>. Microarrays were incubated with 5% milk protein in TBS (tris buffered saline) to block non-probe binding sites, followed by incubation with primary antibody for 1.5&#x02009;h. Arrays were washed in TBS, then incubated with secondary antibodies for 1.5&#x02009;h. After washing with TBS, arrays were developed using an NBT/BCIP substrate to visualize sample spots that contained glycan epitopes that the probes bound to. The NBT/BCIP substrate contained Nitro Blue Tetrazolium (NBT) and 5-bromo-4-chloro-3-indolylphosphate p-toludine salt (BCIP) in Tris buffer (0.5&#x02009;mM MgCl<sub>2</sub>, 100&#x02009;mM diethanol-amine, pH 9.5). The arrays were scanned using a desktop scanner (CanoScan 8800&#x02009;F, Canon) at 2400 dpi, and the intensity of NBT/BCIP stained spots quantified using Array-Pro Analyzer 6.3 (Media Cybernetics). This produces data for the normalized intensity of spots relative to the background intensity of the nitrocellulose membrane (to control for background staining). We used this data to create a heatmap, for average spot intensity across biological replicates and sample dilutions.</p></span><span id=\"s\r\n19\" class=\"sec\"><h3 id=\"s\r\n19title\">Microbial sampling</h3><p id=\"Par34\" class=\"p p-first-last\">Fecal samples were collected from six piglets per experimental group (<em>n</em>&#x02009;=&#x02009;12) at days 0, 7, 14, 21, and 27 post-weaning. At the end of the trial, all piglets (<em>n</em>&#x02009;=&#x02009;48) were sacrificed, and distinct samples were collected from the lumen of the duodenum, jejunum, ileum, cecum, and colon. Samples were obtained within the first 15&#x02009;min after the piglets were sacrificed and the samples were flash-frozen in liquid nitrogen and stored at &#x02212;80&#x02009;&#x000b0;C until DNA extraction.</p></span><span id=\"s\r\n20\" class=\"sec\"><h3 id=\"s\r\n20title\">DNA extraction</h3><p id=\"Par35\" class=\"p p-first-last\">DNA was extracted with a MagAttract PowerMicrobiome DNA/RNA Kit (MO BIO Laboratories Inc., Carlsbad, CA, USA) according to the manufacturer instructions, except for the bead-beating step where we used a FastPrep-96 Homogenizer (MP Biomedicals LLC., Santa Ana, CA, USA) at maximum intensity for a total of 2&#x02009;min in 4 pulses of 30&#x02009;s with a 5 min cooling period between each pulse. A KingFisher Flex DNA extraction robot was used for the automated steps of the protocol. The extracted nucleic acids were quantified with a Qubit Fluorimeter and the Qubit dsDNA BR Assay Kit (ThermoFisher Scientific, Waltham, MA, USA) and stored at &#x02212;80&#x02009;&#x000b0;C.</p></span><span id=\"s\r\n21\" class=\"sec\"><h3 id=\"s\r\n21title\">16S amplicon sequencing and analysis</h3><p id=\"Par36\" class=\"p p-first\">16S amplicon sequence data was obtained for all fecal and intestinal samples. The V3-V4 region of the 16S rRNA gene was PCR amplified using the primers Pro341F and Pro805R (Supplementary Table&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">4</a></sup>), to which the MiSeq adaptors were additionally incorporated on the 5&#x02032; ends<sup><sup><a href=\"#CR33\" rid=\"CR33\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780241\">33</a></sup></sup>. The 25&#x02009;&#x000b5;L PCR reactions consisted of 1&#x000d7; iProof High-Fidelity Master Mix (Bio-Rad, Hercules, CA, USA), 0.25&#x02009;&#x000b5;M primers, and 5&#x02009;ng template DNA. PCR thermal cycling began with a hot start step at 98&#x02009;&#x000b0;C for 180&#x02009;s and was followed by 25 cycles of 98&#x02009;&#x000b0;C denaturation for 30&#x02009;s, 55&#x02009;&#x000b0;C annealing for 30&#x02009;s, and 72&#x02009;&#x000b0;C extension for 30&#x02009;s, followed by a final, 300&#x02009;s extension step at 72&#x02009;&#x000b0;C. Amplicons were individually purified with AMPure XP beads (Beckman Coulter, Indianapolis, IN, USA) and indexed with the Nextera XT Index Kit v2 (Illumina, San Diego, CA, USA) according to the Illumina protocol for 16S rRNA gene sequencing analysis. Next, equal volumes from each indexing reaction were pooled together, and the pool was purified with AMPure XP beads. The purified amplicon pool was then quantified with a Qubit Fluorimeter, diluted, mixed with 15% PhiX Control v3 (Illumina), and denatured according to the aforementioned Illumina protocol. The denatured library was sequenced on the Illumina MiSeq platform using the MiSeq Reagent Kit v3 (600 cycles). Data were output from the sequencer as demultiplexed FASTQ format files.</p><p id=\"Par37\" class=\"p p-last\">Processing of the data was done with a combination of standalone programs, QIIME<sup><sup><a href=\"#CR34\" rid=\"CR34\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780222\">34</a></sup></sup> MOTHUR<sup><sup><a href=\"#CR35\" rid=\"CR35\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780225\">35</a></sup></sup>, and the R package Phyloseq<sup><sup><a href=\"#CR36\" rid=\"CR36\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780213\">36</a></sup></sup>. To process the data, the paired-end reads for each sample were merged with PEAR<sup><sup><a href=\"#CR37\" rid=\"CR37\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780240\">37</a></sup></sup>, specifying a minimum assembly length 400, maximum assembly length 575, minimum overlap 50, and no statistical test. Then, PRINSEQ<sup><sup><a href=\"#CR38\" rid=\"CR38\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780184\">38</a></sup></sup> version 0.20.4 was used to filter low quality reads by requiring a minimum quality score of 10 for all bases and a minimum mean quality of 30. Primer sequences were trimmed in MOTHUR version 1.36.1, and chimeric sequences were identified and filtered out using QIIME version 1.9.1. Next, open reference OTU<sub>0.97</sub> clustering<sup><sup><a href=\"#CR39\" rid=\"CR39\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780178\">39</a></sup></sup> was performed with VSEARCH<sup><sup><a href=\"#CR40\" rid=\"CR40\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780258\">40</a></sup></sup> version 2.3.2 and the Silva database<sup><sup><a href=\"#CR41\" rid=\"CR41\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780182\">41</a></sup></sup> release 128 as the taxonomy reference. Then, the QIIME core diversity analysis script was run. Differentially abundant phylotypes were identified in both cecum and colon for the control vs. 4% AcGGM samples using both the MetagenomeSeq fitZIG and DESeq2 negative binomial algorithms via the QIIME wrapper. The OTU table, phylogenetic tree, representative sequences, and taxonomy from QIIME were incorporated along with the sample metadata into a Phyloseq version 1.22.3 object in R for data exploration and visualization.</p></span><span id=\"s\r\n22\" class=\"sec\"><h3 id=\"s\r\n22title\">Whole-metagenome sequencing and analysis</h3><p id=\"Par38\" class=\"p p-first-last\">Whole-metagenome sequencing was performed at the Norwegian Sequencing Centre on 2 lanes of the Illumina HiSeq 4000 to generate 2&#x02009;&#x000d7;&#x02009;150 paired-end reads. TruSeq PCR-free libraries were prepared for 12 control and 12 AcGGM (4%) samples from the colon. All 24 samples were run in both lanes to eliminate the potential for lane-specific sequencing bias. FASTQ format files were received from the sequencing center, and prior to assembly, these were quality filtered with Trimmomatic<sup><sup><a href=\"#CR42\" rid=\"CR42\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780248\">42</a></sup></sup> version 0.36 whereby TruSeq adaptor sequences were eliminated, sequences were required to have an average quality score above 20, leading and trailing bases with quality below 20 were removed, sequences with an average quality score below 15 in a 4-base sliding window were trimmed, and the minimum read length was required to be 36 bases. Individual sample assembly was accomplished with metaSPAdes<sup><sup><a href=\"#CR43\" rid=\"CR43\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780210\">43</a></sup></sup> version 3.11.1. MegaHIT<sup><sup><a href=\"#CR44\" rid=\"CR44\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780239\">44</a></sup></sup> version 1.1.3 was used for co-assembly of all 24 samples together as well as co-assembly of the 12 control samples together and the 12 4% AcGGM samples together. MetaBAT<sup><sup><a href=\"#CR45\" rid=\"CR45\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780212\">45</a></sup></sup> version 0.26.3 was used to bin the assemblies, and dRep<sup><sup><a href=\"#CR46\" rid=\"CR46\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780254\">46</a></sup></sup> version 2.0.5&#x000a0;was used to dereplicate the multiple assembly and binning combinations to produce an optimal set of MAGs. MASH<sup><sup><a href=\"#CR47\" rid=\"CR47\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780242\">47</a></sup></sup> version 2.0 used to compare the similarity of the 24 metagenomes by calculating pairwise Jaccard distances which were imported into R for NMDS ordination and visualization. Completeness and contamination were determined for each MAG using CheckM<sup><sup><a href=\"#CR48\" rid=\"CR48\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780183\">48</a></sup></sup> version 1.0.7. Taxonomic classifications of MAGs were performed using GTDB-Tk<sup><sup><a href=\"#CR49\" rid=\"CR49\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_915796627\">49</a></sup></sup>, and relative abundance estimations were generated using CoverM (<a href=\"https://github.com/wwood/CoverM\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://github.com/wwood/CoverM</a>). Feature and functional annotation were completed with the Prokka pipeline<sup><sup><a href=\"#CR50\" rid=\"CR50\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780180\">50</a></sup></sup> version 1.12, and the predicted protein sequences from all 355 MAGs were concatenated to create the metaproteomics reference database. Resulting annotated open reading frames (ORFs) were retrieved, further annotated for CAZymes using the CAZy annotation pipeline with libraries from the July 2018 database release<sup><sup><a href=\"#CR51\" rid=\"CR51\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780201\">51</a></sup>,<sup><a href=\"#CR52\" rid=\"CR52\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780237\">52</a></sup></sup>, and subsequently used as a reference database for the metaproteomics (with the exception of glycosyltransferases).</p></span><span id=\"s\r\n23\" class=\"sec\"><h3 id=\"s\r\n23title\">Metaproteomics</h3><p id=\"Par39\" class=\"p p-first\">Proteins were extracted from each sample by the following method. An aliquot (1&#x02009;g) of colon digesta from pigs fed either a control diet or a diet supplemented with 4% &#x003b2;-mannan was dissolved 1:1 (w/v) in 50&#x02009;mM Tris-HCl, pH 8.4. Lysis was performed using a bead-beating approach whereby glass beads (size&#x02009;&#x02264;&#x02009;106&#x02009;&#x000b5;m) were added to the colon digesta slurry and cells were disrupted in 3&#x02009;&#x000d7;&#x02009;60&#x02009;s cycles using a FastPrep24 (MP Biomedicals, Santa Ana, CA, USA). Debris was removed by centrifugation at 16,600&#x02009;&#x000d7;&#x02009;<em>g</em> for 20&#x02009;min and proteins were precipitated overnight in 16% ice-cold TCA. The next day, proteins were dissolved in 100&#x02009;&#x003bc;L 50&#x02009;mM TrisHCl, pH 8.4, and concentration was determined using the Bradford protein assay (Bradford Laboratories, USA) using bovine serum albumin as a standard. Fifty milligrams of protein were prepared in SDS sample buffer, separated by SDS-PAGE using an Any-kD Mini-PROTEAN gel (Bio-Rad Laboratories, Hercules, CA, USA), and stained using Coomassie Brilliant Blue R250. The gel was cut into 6 slices and reduced, alkylated, and digested<sup><sup><a href=\"#CR53\" rid=\"CR53\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780190\">53</a></sup></sup>. Prior to mass spectrometry, peptides were desalted using C<sub>18</sub> ZipTips (Merck Millipore, Darmstadt, Germany) according to the manufacturer&#x02019;s instructions.</p><p id=\"Par40\">The peptides were analyzed by nanoLC-MS/MS as described previously, using a Q-Exactive hybrid quadrupole orbitrap mass spectrometer (Thermo Scientific, Bremen, Germany)<sup><sup><a href=\"#CR54\" rid=\"CR54\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780243\">54</a></sup></sup>, and the acquired raw data was analyzed using MaxQuant<sup><sup><a href=\"#CR55\" rid=\"CR55\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780232\">55</a></sup></sup> version 1.4.1.2. Proteins were quantified using the MaxLFQ algorithm<sup><sup><a href=\"#CR56\" rid=\"CR56\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_915796628\">56</a></sup></sup>. Data were searched against a sample-specific database (602.947 protein sequences), generated from the 355 metagenome-assembled genomes (MAGs), and against the genome of a pig (<em>Sus scrofa domesticus)</em> (40.708 sequences). In addition, common contaminants such as human keratins, trypsin, and bovine serum albumin were concatenated to the database as well as reversed sequences of all protein entries for estimation of false discovery rates. Protein N-terminal acetylation, oxidation of methionine, conversion of glutamine to pyroglutamic acid, and deamination of asparagine and glutamine were used as variable modifications, while carbamidomethylation of cysteine residues was used as a fixed modification. Trypsin was used as a digestion enzyme and two missed cleavages were allowed. All identifications were filtered in order to achieve a protein false discovery rate (FDR) of 1% using the target-decoy strategy. A total of 8515 protein groups (20,350 shared proteins) were considered valid, which required the protein group to be both identified and quantified in at least two replicates, and in addition, we required at least one unique peptide per protein and at least two peptides in total for every protein group. In cases where a protein group consisted of two or more homologs protein identifications, both unique and shared proteins are indicated (i.e., Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig4\" rid-ob=\"ob-Fig4\" co-legend-rid=\"lgnd_Fig4\"><span>4c</span></a> and Supplementary Data&#x000a0;<sup><a href=\"#MOESM9\" rid=\"MOESM9\" class=\" media\">6</a></sup>). The output from MaxQuant was further explored in Perseus version 1.6.0.7 where filtering, data transformation, and imputation were performed.</p><p id=\"Par41\" class=\"p p-last\">For metaproteomic enrichment analysis, missing values in the proteomics data were set to zero, and proteins with zero variance across the eight samples, or with a non-zero value in only one replicate animal in either the control or AcGGM treatment were removed. This resulted in 4562 unique protein groups assigned to 12,535 MAG ORFs (i.e., shared detected proteins). A Principle Component Analysis (PCA) was performed on the 4562&#x02009;&#x000d7;&#x02009;8 matrix using the R function <em>prcomp</em>. Hierarchal clustering dendrograms were constructed using the R functions <em>hclust</em> with Pearson correlation and <em>method</em>&#x02009;=&#x02009;<em>&#x02018;ward.D&#x02019;</em>, and a heatmap was drawn using the function <em>heatmap.2</em> with <em>scale</em>&#x02009;=&#x02009;<em>&#x02018;row&#x02019;</em>. The number of protein clusters were selected by visual inspection and the clusters were selected using the <em>cutree</em> function. Enzyme Commission (EC) annotation (Supplementary Data&#x000a0;<sup><a href=\"#MOESM12\" rid=\"MOESM12\" class=\" media\">9</a></sup>) and MAG enrichment analysis (Supplementary Data&#x000a0;<sup><a href=\"#MOESM10\" rid=\"MOESM10\" class=\" media\">7</a></sup>) was performed using the hypergeometric distribution function <em>phyper</em> in R with the false discovery rate controlled at 5% using the function <em>p</em>.adjust with method&#x02009;=&#x02009;&#x02018;BH&#x02019;.</p></span><span id=\"s\r\n24\" class=\"sec\"><h3 id=\"s\r\n24title\">Genome tree</h3><p id=\"Par42\" class=\"p p-first-last\">Phylogenetic analysis was performed using a block of 22 universal ribosomal proteins (30S ribosomal protein L1, L2, L4-L6, L10, L11, L14, L15, L18, and 50S ribosomal protein S3, S5, S7-S13, S15, S17, S19)<sup><sup><a href=\"#CR57\" rid=\"CR57\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780227\">57</a></sup>,<sup><a href=\"#CR58\" rid=\"CR58\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780217\">58</a></sup></sup>. In addition to the MAGs, we recruited 239 reference genomes for phylogenetic resolution. These genomes were selected based on preliminary examination of the assembled metagenome using metaQUAST<sup><sup><a href=\"#CR59\" rid=\"CR59\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780174\">59</a></sup></sup>. The reference genomes were annotated using the Prokka pipeline in the same manner as for the MAGs. All identified ribosomal protein sequences were aligned separately with MUSCLE v3.8.31<sup><sup><a href=\"#CR60\" rid=\"CR60\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780231\">60</a></sup></sup> and manually checked for duplications and misaligned sequences. Divergent regions and poorly aligned positions were further eliminated using GBlocks<sup><sup><a href=\"#CR61\" rid=\"CR61\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780181\">61</a></sup></sup>, and the refined alignment was concatenated using catfasta2phyml.pl (<a href=\"https://github.com/nylander/catfasta2phyml\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://github.com/nylander/catfasta2phyml</a>) with the parameter &#x02018;-c&#x02019; to replace missing ribosomal proteins with gaps (-). The maximum likelihood-based phylogeny of the concatenated ribosomal proteins was inferred using RAxML version 8.2.12<sup><sup><a href=\"#CR62\" rid=\"CR62\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780219\">62</a></sup></sup> (raxmlHPC-SSE3 under PROTGAMMA distributed model with WAG substitution matrix) and support values determined using 100 bootstrap replicates. The tree was rooted to the Euryarchaeota phylum and visualized using iTOL<sup><sup><a href=\"#CR63\" rid=\"CR63\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780252\">63</a></sup></sup>. Clades of reference genomes with only distant phylogenetic relation to the MAGs were collapsed to refine the final tree in Fig.&#x000a0;<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/figure/Fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"Fig3\" rid-ob=\"ob-Fig3\" co-legend-rid=\"lgnd_Fig3\"><span>4</span></a>. The complete tree is available in Newick format as Supplementary Data&#x000a0;<sup><a href=\"#MOESM7\" rid=\"MOESM7\" class=\" media\">4</a></sup>.</p></span><span id=\"s\r\n25\" class=\"sec sec-last\"><h3 id=\"s\r\n25title\">Reporting summary</h3><p id=\"Par43\" class=\"p p-first-last\">Further information on research design is available in the&#x000a0;<sup><a href=\"#MOESM13\" rid=\"MOESM13\" class=\" media\">Nature Research Reporting Summary</a></sup> linked to this article.</p></span></span><span id=\"s\r\n1\" class=\"tsec sec\"><a id=\"supplementary-material-sec\"></a><h2 class=\"head no_bottom_margin\" id=\"s\r\n1title\">Supplementary information</h2><!--/article/body/sec/--><span id=\"s\r\n26\" class=\"sec sec-first-last\"><span class=\"sec suppmat\" id=\"MOESM1\"><span class=\"sup-box half_rhythm\" id=\"idm140434860572992\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/bin/41467_2020_19585_MOESM1_ESM.pdf\" data-ga-action=\"click_feat_suppl\">Supplementary Information</a><sup>(1.6M, pdf)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM2\"><span class=\"sup-box half_rhythm\" id=\"idm140434860570144\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/bin/41467_2020_19585_MOESM2_ESM.pdf\" data-ga-action=\"click_feat_suppl\">Peer Review File</a><sup>(384K, pdf)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM3\"><span class=\"sup-box half_rhythm\" id=\"idm140434861584064\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/bin/41467_2020_19585_MOESM3_ESM.docx\" data-ga-action=\"click_feat_suppl\">Description of Additional Supplementary Files</a><sup>(16K, docx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM4\"><span class=\"sup-box half_rhythm\" id=\"idm140434861581216\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/bin/41467_2020_19585_MOESM4_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Data 1</a><sup>(130K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM5\"><span class=\"sup-box half_rhythm\" id=\"idm140434879672192\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/bin/41467_2020_19585_MOESM5_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Data 2</a><sup>(58K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM6\"><span class=\"sup-box half_rhythm\" id=\"idm140434859204544\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/bin/41467_2020_19585_MOESM6_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Data 3</a><sup>(1.0M, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM7\"><span class=\"sup-box half_rhythm\" id=\"idm140434859201664\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/bin/41467_2020_19585_MOESM7_ESM.txt\" data-ga-action=\"click_feat_suppl\">Supplementary Data 4</a><sup>(42K, txt)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM8\"><span class=\"sup-box half_rhythm\" id=\"idm140434953572992\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/bin/41467_2020_19585_MOESM8_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Data 5</a><sup>(63K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM9\"><span class=\"sup-box half_rhythm\" id=\"idm140434953570112\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/bin/41467_2020_19585_MOESM9_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Data 6</a><sup>(535K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM10\"><span class=\"sup-box half_rhythm\" id=\"idm140434859657008\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/bin/41467_2020_19585_MOESM10_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Data 7</a><sup>(16K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM11\"><span class=\"sup-box half_rhythm\" id=\"idm140434859654128\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/bin/41467_2020_19585_MOESM11_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Data 8</a><sup>(36K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM12\"><span class=\"sup-box half_rhythm\" id=\"idm140434873457152\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/bin/41467_2020_19585_MOESM12_ESM.xlsx\" data-ga-action=\"click_feat_suppl\">Supplementary Data 9</a><sup>(39K, xlsx)</sup></span></span><span class=\"sec suppmat\" id=\"MOESM13\"><span class=\"sup-box half_rhythm\" id=\"idm140434873454272\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7666174/bin/41467_2020_19585_MOESM13_ESM.pdf\" data-ga-action=\"click_feat_suppl\">Reporting Summary</a><sup>(298K, pdf)</sup></span></span></span></span><span id=\"idm140434881198592\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140434881198592title\">Acknowledgements</h2><span class=\"sec\"><p id=\"__p25\">We are grateful for support from The Research Council of Norway (Bion&#x000e6;r program 244259, 208674/F50, 270038, FRIPRO program, PBP: 250479), as well as the European Research Commission Starting Grant Fellowship (awarded to PBP; 336355 - MicroDE). The sequencing service was provided by the Norwegian Sequencing Centre (<a href=\"http://www.sequencing.uio.no\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">www.sequencing.uio.no</a>), a national technology platform hosted by the University of Oslo and supported by the &#x02018;Functional Genomics&#x02019; and &#x02018;Infrastructure&#x02019; programs of the Research Council of Norway and the Southeastern Regional Health Authorities. We thank Sivert D&#x000e6;hli for performing the detailed analysis of degree of polymerization of the AcGGM.</p></span></span><span id=\"idm140434881196000\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140434881196000title\">Author contributions</h2><p id=\"__p26\">L.M., J.C.G., P.B.P. and B.W. conceived the study, performed the primary analysis of the data and wrote the paper (with input from all authors). L.M., J.C.G., L.L., S.L.L.R., L.H.H., T.R.H., P.B.P. and B.W. designed the figures. L.L. and M.&#x000d8;. designed, performed and analyzed the animal experiments. S.L.L.R., L.H.H., M.&#x000d8;.A., T.R.H. and J.D. generated the data and/or contributed to the data analyses. W.G.T.W. and C.T.-J. generated and analyzed MAPP data. N.T., V.L. and B.H. annotated and curated the MAGs and identified carbohydrate-active enzymes.</p></span><span id=\"idm140434874422928\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140434874422928title\">Data availability</h2><p id=\"__p27\">All sequencing reads have been deposited at the NCBI sequence read archive under BioProject <a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA574295\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA574295</a>, with specific numbers listed in Supplementary Table&#x000a0;<sup><a href=\"#MOESM1\" rid=\"MOESM1\" class=\" media\">2</a></sup>. All annotated MAGs are publicly available via 10.6084/m9.figshare.9816581<sup><sup><a href=\"#CR64\" rid=\"CR64\" class=\" bibr popnode\">64</a></sup></sup>. The proteomics data have been deposited to the ProteomeXchange Consortium (<a href=\"http://proteomecentral.proteomexchange.org\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://proteomecentral.proteomexchange.org</a>) via the PRIDE partner repository<sup><sup><a href=\"#CR65\" rid=\"CR65\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_894780206\">65</a></sup></sup> with the dataset identifier <a href=\"https://www.ebi.ac.uk/pride/archive/projects/PXD015757\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PXD015757</a>.</p></span><span id=\"idm140434880223168\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140434880223168title\">Code availability</h2><p id=\"__p28\">The code used to perform the metaproteomic enrichment analysis is available at <a href=\"https://gitlab.com/hvidsten-lab/michalak\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://gitlab.com/hvidsten-lab/michalak</a>.</p></span><span id=\"FPar1\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"FPar1title\">Competing interests</h2><p id=\"Par44\">The authors declare no competing interests.</p></span><span id=\"idm140434875050272\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140434875050272title\">Footnotes</h2><!--back/fn-group--><span class=\"fm-sec half_rhythm small\"><p class=\"fn sec\" id=\"idm140434875050144\"><p id=\"__p21\" class=\"p p-first-last\"><strong>Peer review information</strong>\n<em>Nature Communications</em> thanks David Bolam and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available.</p></p><p class=\"fn sec\" id=\"idm140434875048320\"><p id=\"__p22\" class=\"p p-first-last\"><strong>Publisher&#x02019;s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></p><p class=\"fn sec\" id=\"idm140434875047216\"><p id=\"__p23\" class=\"p p-first-last\">These authors contributed equally: Phillip B. Pope, Bj&#x000f8;rge Westereng.</p></p></span></span><br><div class=\"ref-list-sec sec\" id=\"reference-list\"><div class=\"ref-cit-blk half_rhythm\" id=\"CR1\">1. <span class=\"element-citation\">Patnode ML, et al.  Interspecies competition impacts targeted manipulation of human gut bacteria by fiber-derived glycans. <span><span class=\"ref-journal\">Cell. </span>2019;<span class=\"ref-vol\">179</span>:59–73. doi: 10.1016/j.cell.2019.08.011.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC6760872/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n31539500\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.cell.2019.08.011\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Interspecies+competition+impacts+targeted+manipulation+of+human+gut+bacteria+by+fiber-derived+glycans&amp;author=ML+Patnode&amp;volume=179&amp;publication_year=2019&amp;pages=59-73&amp;pmid=31539500&amp;doi=10.1016/j.cell.2019.08.011&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR2\">2. <span class=\"element-citation\">Blanton LV, Barratt MJ, Charbonneau MR, Ahmed T, Gordon JI. Childhood undernutrition, the gut microbiota, and microbiota-directed therapeutics. <span><span class=\"ref-journal\">Science. </span>2016;<span class=\"ref-vol\">352</span>:1533. doi: 10.1126/science.aad9359.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27339978\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1126%2Fscience.aad9359\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Childhood+undernutrition,+the+gut+microbiota,+and+microbiota-directed+therapeutics&amp;author=LV+Blanton&amp;author=MJ+Barratt&amp;author=MR+Charbonneau&amp;author=T+Ahmed&amp;author=JI+Gordon&amp;volume=352&amp;publication_year=2016&amp;pages=1533&amp;pmid=27339978&amp;doi=10.1126/science.aad9359&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR3\">3. <span class=\"element-citation\">Gibson GR, et al.  Expert consensus document: The International Scientific Association for Probiotics and Prebiotics (ISAPP) consensus statement on the definition and scope of prebiotics. <span><span class=\"ref-journal\">Nat. Rev. Gastroenterol. Hepatol. </span>2017;<span class=\"ref-vol\">14</span>:491–502. doi: 10.1038/nrgastro.2017.75.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28611480\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnrgastro.2017.75\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Gastroenterol.+Hepatol.&amp;title=Expert+consensus+document:+The+International+Scientific+Association+for+Probiotics+and+Prebiotics+(ISAPP)+consensus+statement+on+the+definition+and+scope+of+prebiotics&amp;author=GR+Gibson&amp;volume=14&amp;publication_year=2017&amp;pages=491-502&amp;pmid=28611480&amp;doi=10.1038/nrgastro.2017.75&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR4\">4. <span class=\"element-citation\">Sanders ME, Merenstein DJ, Reid G, Gibson GR, Rastall RA. Probiotics and prebiotics in intestinal health and disease: from biology to the clinic. <span><span class=\"ref-journal\">Nat. Rev. Gastroenterol. Hepatol. </span>2019;<span class=\"ref-vol\">16</span>:605–616. doi: 10.1038/s41575-019-0173-3.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n31296969\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41575-019-0173-3\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Gastroenterol.+Hepatol.&amp;title=Probiotics+and+prebiotics+in+intestinal+health+and+disease:+from+biology+to+the+clinic&amp;author=ME+Sanders&amp;author=DJ+Merenstein&amp;author=G+Reid&amp;author=GR+Gibson&amp;author=RA+Rastall&amp;volume=16&amp;publication_year=2019&amp;pages=605-616&amp;pmid=31296969&amp;doi=10.1038/s41575-019-0173-3&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR5\">5. <span class=\"element-citation\">La Rosa SL, et al.  The human gut Firmicute <em>Roseburia intestinalis</em> is a primary degrader of dietary &#x003b2;-mannans. <span><span class=\"ref-journal\">Nat. Commun. </span>2019;<span class=\"ref-vol\">10</span>:905. doi: 10.1038/s41467-019-08812-y.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC6385246/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n30796211\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41467-019-08812-y\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=The+human+gut+Firmicute+Roseburia+intestinalis+is+a+primary+degrader+of+dietary+&#x003b2;-mannans&amp;author=SL+La+Rosa&amp;volume=10&amp;publication_year=2019&amp;pages=905&amp;pmid=30796211&amp;doi=10.1038/s41467-019-08812-y&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR6\">6. <span class=\"element-citation\">Larsbrink J, Rogers T, Hemsworth G. A discrete genetic locus confers xyloglucan metabolism in select human gut Bacteroidetes. <span><span class=\"ref-journal\">Nature. </span>2014;<span class=\"ref-vol\">506</span>:498–502. doi: 10.1038/nature12907.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4282169/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24463512\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnature12907\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+discrete+genetic+locus+confers+xyloglucan+metabolism+in+select+human+gut+Bacteroidetes&amp;author=J+Larsbrink&amp;author=T+Rogers&amp;author=G+Hemsworth&amp;volume=506&amp;publication_year=2014&amp;pages=498-502&amp;pmid=24463512&amp;doi=10.1038/nature12907&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR7\">7. <span class=\"element-citation\">Cuskin F, et al.  Human gut Bacteroidetes can utilize yeast mannan through a selfish mechanism. <span><span class=\"ref-journal\">Nature. </span>2015;<span class=\"ref-vol\">517</span>:165–169. doi: 10.1038/nature13995.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4978465/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25567280\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnature13995\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Human+gut+Bacteroidetes+can+utilize+yeast+mannan+through+a+selfish+mechanism&amp;author=F+Cuskin&amp;volume=517&amp;publication_year=2015&amp;pages=165-169&amp;pmid=25567280&amp;doi=10.1038/nature13995&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR8\">8. <span class=\"element-citation\">Solden LM, et al.  Interspecies cross-feeding orchestrates carbon degradation in the rumen ecosystem. <span><span class=\"ref-journal\">Nat. Microbiol. </span>2018;<span class=\"ref-vol\">3</span>:1274–1284. doi: 10.1038/s41564-018-0225-4.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC6784887/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n30356154\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41564-018-0225-4\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Microbiol.&amp;title=Interspecies+cross-feeding+orchestrates+carbon+degradation+in+the+rumen+ecosystem&amp;author=LM+Solden&amp;volume=3&amp;publication_year=2018&amp;pages=1274-1284&amp;pmid=30356154&amp;doi=10.1038/s41564-018-0225-4&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR9\">9. <span class=\"element-citation\">Michalak L, et al.  A pair of esterases from a commensal gut bacterium remove acetylations from all positions on complex &#x003b2;-mannans. <span><span class=\"ref-journal\">Proc. Natl Acad. Sci. USA. </span>2020;<span class=\"ref-vol\">117</span>:7122–7130. doi: 10.1073/pnas.1915376117.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7132267/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n32170022\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1073%2Fpnas.1915376117\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=A+pair+of+esterases+from+a+commensal+gut+bacterium+remove+acetylations+from+all+positions+on+complex+&#x003b2;-mannans&amp;author=L+Michalak&amp;volume=117&amp;publication_year=2020&amp;pages=7122-7130&amp;pmid=32170022&amp;doi=10.1073/pnas.1915376117&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR10\">10. <span class=\"element-citation\">La Rosa SL, et al.  Wood-derived dietary fibers promote beneficial human gut microbiota. <span><span class=\"ref-journal\">mSphere. </span>2019;<span class=\"ref-vol\">4</span>:e00554–00518.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC6344601/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n30674645\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=mSphere&amp;title=Wood-derived+dietary+fibers+promote+beneficial+human+gut+microbiota&amp;author=SL+La+Rosa&amp;volume=4&amp;publication_year=2019&amp;pages=e00554-00518&amp;pmid=30674645&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR11\">11. <span class=\"element-citation\">Ferreira-Halder CV, Faria AVS, Andrade SS. Action and function of <em>Faecalibacterium prausnitzii</em> in health and disease. <span><span class=\"ref-journal\">Best. Pract. Res. Clin. Gastroenterol. </span>2017;<span class=\"ref-vol\">31</span>:643–648. doi: 10.1016/j.bpg.2017.09.011.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n29566907\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.bpg.2017.09.011\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Best.+Pract.+Res.+Clin.+Gastroenterol.&amp;title=Action+and+function+of+Faecalibacterium+prausnitzii+in+health+and+disease&amp;author=CV+Ferreira-Halder&amp;author=AVS+Faria&amp;author=SS+Andrade&amp;volume=31&amp;publication_year=2017&amp;pages=643-648&amp;pmid=29566907&amp;doi=10.1016/j.bpg.2017.09.011&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR12\">12. <span class=\"element-citation\">Lopez-Siles M, Duncan SH, Garcia-Gil LJ, Martinez-Medina M. <em>Faecalibacterium prausnitzii</em>: from microbiology to diagnostics and prognostics. <span><span class=\"ref-journal\">ISME J. </span>2017;<span class=\"ref-vol\">11</span>:841–852. doi: 10.1038/ismej.2016.176.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5364359/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28045459\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fismej.2016.176\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=ISME+J.&amp;title=Faecalibacterium+prausnitzii:+from+microbiology+to+diagnostics+and+prognostics&amp;author=M+Lopez-Siles&amp;author=SH+Duncan&amp;author=LJ+Garcia-Gil&amp;author=M+Martinez-Medina&amp;volume=11&amp;publication_year=2017&amp;pages=841-852&amp;pmid=28045459&amp;doi=10.1038/ismej.2016.176&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR13\">13. <span class=\"element-citation\">Lahaye M, Quemener B, Causse M, Seymour GB. Hemicellulose fine structure is affected differently during ripening of tomato lines with contrasted texture. <span><span class=\"ref-journal\">Int. J. Biol. Macromol. </span>2012;<span class=\"ref-vol\">51</span>:462–470. doi: 10.1016/j.ijbiomac.2012.05.024.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22698552\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.ijbiomac.2012.05.024\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int.+J.+Biol.+Macromol.&amp;title=Hemicellulose+fine+structure+is+affected+differently+during+ripening+of+tomato+lines+with+contrasted+texture&amp;author=M+Lahaye&amp;author=B+Quemener&amp;author=M+Causse&amp;author=GB+Seymour&amp;volume=51&amp;publication_year=2012&amp;pages=462-470&amp;pmid=22698552&amp;doi=10.1016/j.ijbiomac.2012.05.024&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR14\">14. <span class=\"element-citation\">Xu C, et al.  Acetylation and characterization of spruce (<em>Picea abies</em>) galactoglucomannans. <span><span class=\"ref-journal\">Carbohydr. Res. </span>2010;<span class=\"ref-vol\">345</span>:810–816. doi: 10.1016/j.carres.2010.01.007.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20144827\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.carres.2010.01.007\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Carbohydr.+Res.&amp;title=Acetylation+and+characterization+of+spruce+(Picea+abies)+galactoglucomannans&amp;author=C+Xu&amp;volume=345&amp;publication_year=2010&amp;pages=810-816&amp;pmid=20144827&amp;doi=10.1016/j.carres.2010.01.007&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR15\">15. <span class=\"element-citation\">Michalak L, Knutsen SH, Aarum I, Westereng B. Effects of pH on steam explosion extraction of acetylated galactoglucomannan from Norway spruce. <span><span class=\"ref-journal\">Biotechnol. Biofuels. </span>2018;<span class=\"ref-vol\">11</span>:311. doi: 10.1186/s13068-018-1300-z.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC6225635/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n30455740\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2Fs13068-018-1300-z\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Biotechnol.+Biofuels&amp;title=Effects+of+pH+on+steam+explosion+extraction+of+acetylated+galactoglucomannan+from+Norway+spruce&amp;author=L+Michalak&amp;author=SH+Knutsen&amp;author=I+Aarum&amp;author=B+Westereng&amp;volume=11&amp;publication_year=2018&amp;pages=311&amp;pmid=30455740&amp;doi=10.1186/s13068-018-1300-z&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR16\">16. <span class=\"element-citation\">Shepherd ES, DeLoache WC, Pruss KM, Whitaker WR, Sonnenburg JL. An exclusive metabolic niche enables strain engraftment in the gut microbiota. <span><span class=\"ref-journal\">Nature. </span>2018;<span class=\"ref-vol\">557</span>:434–438. doi: 10.1038/s41586-018-0092-4.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC6126907/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n29743671\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41586-018-0092-4\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+exclusive+metabolic+niche+enables+strain+engraftment+in+the+gut+microbiota&amp;author=ES+Shepherd&amp;author=WC+DeLoache&amp;author=KM+Pruss&amp;author=WR+Whitaker&amp;author=JL+Sonnenburg&amp;volume=557&amp;publication_year=2018&amp;pages=434-438&amp;pmid=29743671&amp;doi=10.1038/s41586-018-0092-4&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR17\">17. <span class=\"element-citation\">Chen L, et al.  The maturing development of gut microbiota in commercial piglets during the weaning transition. <span><span class=\"ref-journal\">Front. Microbiol. </span>2017;<span class=\"ref-vol\">8</span>:1688. doi: 10.3389/fmicb.2017.01688.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5591375/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28928724\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3389%2Ffmicb.2017.01688\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front.+Microbiol.&amp;title=The+maturing+development+of+gut+microbiota+in+commercial+piglets+during+the+weaning+transition&amp;author=L+Chen&amp;volume=8&amp;publication_year=2017&amp;pages=1688&amp;pmid=28928724&amp;doi=10.3389/fmicb.2017.01688&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR18\">18. <span class=\"mixed-citation\">Holman, D. B., Brunelle, B. W., Trachsel, J. &#x00026; Allen, H. K. Meta-analysis to define a core microbiota in the swine gut. <em>MSystems</em><strong>2</strong>, e00004&#x02013;17 (2017). <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28567446\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>]</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR19\">19. <span class=\"element-citation\">Rattigan R, et al.  Laminarin-rich extract improves growth performance, small intestinal morphology, gene expression of nutrient transporters and the large intestinal microbial composition of piglets during the critical post-weaning period. <span><span class=\"ref-journal\">Br. J. Nutr. </span>2020;<span class=\"ref-vol\">123</span>:255–263. doi: 10.1017/S0007114519002678.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n31640819\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1017%2FS0007114519002678\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Br.+J.+Nutr.&amp;title=Laminarin-rich+extract+improves+growth+performance,+small+intestinal+morphology,+gene+expression+of+nutrient+transporters+and+the+large+intestinal+microbial+composition+of+piglets+during+the+critical+post-weaning+period&amp;author=R+Rattigan&amp;volume=123&amp;publication_year=2020&amp;pages=255-263&amp;pmid=31640819&amp;doi=10.1017/S0007114519002678&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR20\">20. <span class=\"element-citation\">Precup G, Vodnar DC. Gut Prevotella as a possible biomarker of diet and its eubiotic versus dysbiotic roles: a comprehensive literature review. <span><span class=\"ref-journal\">Br. J. Nutr. </span>2019;<span class=\"ref-vol\">122</span>:131–140. doi: 10.1017/S0007114519000680.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n30924428\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1017%2FS0007114519000680\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Br.+J.+Nutr.&amp;title=Gut+Prevotella+as+a+possible+biomarker+of+diet+and+its+eubiotic+versus+dysbiotic+roles:+a+comprehensive+literature+review&amp;author=G+Precup&amp;author=DC+Vodnar&amp;volume=122&amp;publication_year=2019&amp;pages=131-140&amp;pmid=30924428&amp;doi=10.1017/S0007114519000680&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR21\">21. <span class=\"element-citation\">Kageyama A, Benno Y. <em>Catenibacterium mitsuokai</em> gen. nov., sp. nov., a gram-positive anaerobic bacterium isolated from human faeces. <span><span class=\"ref-journal\">Int. J. Syst. Bacteriol. </span>2000;<span class=\"ref-vol\">50</span>:1595–1599. doi: 10.1099/00207713-50-4-1595.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n10939666\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1099%2F00207713-50-4-1595\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int.+J.+Syst.+Bacteriol.&amp;title=Catenibacterium+mitsuokai+gen.+nov.,+sp.+nov.,+a+gram-positive+anaerobic+bacterium+isolated+from+human+faeces&amp;author=A+Kageyama&amp;author=Y+Benno&amp;volume=50&amp;publication_year=2000&amp;pages=1595-1599&amp;doi=10.1099/00207713-50-4-1595&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR22\">22. <span class=\"element-citation\">Moore LV, Moore WE. <em>Oribaculum catoniae</em> gen. nov., sp. nov.; <em>Catonella morbi</em> gen. nov., sp. nov.; <em>Hallella seregens</em> gen. nov., sp. nov.; <em>Johnsonella ignava</em> gen. nov., sp. nov.; and <em>Dialister pneumosintes</em> gen. nov., comb. nov., nom. rev., Anaerobic gram-negative bacilli from the human gingival crevice. <span><span class=\"ref-journal\">Int J. Syst. Bacteriol. </span>1994;<span class=\"ref-vol\">44</span>:187–192. doi: 10.1099/00207713-44-2-187.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n8186083\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1099%2F00207713-44-2-187\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int+J.+Syst.+Bacteriol.&amp;title=Oribaculum+catoniae+gen.+nov.,+sp.+nov.;+Catonella+morbi+gen.+nov.,+sp.+nov.;+Hallella+seregens+gen.+nov.,+sp.+nov.;+Johnsonella+ignava+gen.+nov.,+sp.+nov.;+and+Dialister+pneumosintes+gen.+nov.,+comb.+nov.,+nom.+rev.,+Anaerobic+gram-negative+bacilli+from+the+human+gingival+crevice&amp;author=LV+Moore&amp;author=WE+Moore&amp;volume=44&amp;publication_year=1994&amp;pages=187-192&amp;pmid=8186083&amp;doi=10.1099/00207713-44-2-187&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR23\">23. <span class=\"element-citation\">Louis P, Young P, Holtrop G, Flint HJ. Diversity of human colonic butyrate-producing bacteria revealed by analysis of the butyryl-CoA:acetate CoA-transferase gene. <span><span class=\"ref-journal\">Environ. Microbiol. </span>2010;<span class=\"ref-vol\">12</span>:304–314. doi: 10.1111/j.1462-2920.2009.02066.x.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19807780\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1462-2920.2009.02066.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Environ.+Microbiol&amp;title=Diversity+of+human+colonic+butyrate-producing+bacteria+revealed+by+analysis+of+the+butyryl-CoA:acetate+CoA-transferase+gene&amp;author=P+Louis&amp;author=P+Young&amp;author=G+Holtrop&amp;author=HJ+Flint&amp;volume=12&amp;publication_year=2010&amp;pages=304-314&amp;pmid=19807780&amp;doi=10.1111/j.1462-2920.2009.02066.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR24\">24. <span class=\"element-citation\">Martens EC, Koropatkin NM, Smith TJ, Gordon JI. Complex glycan catabolism by the human gut microbiota: the Bacteroidetes Sus-like paradigm. <span><span class=\"ref-journal\">J. Biol. Chem. </span>2009;<span class=\"ref-vol\">284</span>:24673–24677. doi: 10.1074/jbc.R109.022848.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2757170/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19553672\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1074%2Fjbc.R109.022848\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Complex+glycan+catabolism+by+the+human+gut+microbiota:+the+Bacteroidetes+Sus-like+paradigm&amp;author=EC+Martens&amp;author=NM+Koropatkin&amp;author=TJ+Smith&amp;author=JI+Gordon&amp;volume=284&amp;publication_year=2009&amp;pages=24673-24677&amp;pmid=19553672&amp;doi=10.1074/jbc.R109.022848&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR25\">25. <span class=\"element-citation\">Panwar D, Kapoor M. Transcriptional analysis of galactomannooligosaccharides utilization by <em>Lactobacillus plantarum</em> WCFS1. <span><span class=\"ref-journal\">Food Microbiol. </span>2020;<span class=\"ref-vol\">86</span>:103336. doi: 10.1016/j.fm.2019.103336.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n31703861\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.fm.2019.103336\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Food+Microbiol.&amp;title=Transcriptional+analysis+of+galactomannooligosaccharides+utilization+by+Lactobacillus+plantarum+WCFS1&amp;author=D+Panwar&amp;author=M+Kapoor&amp;volume=86&amp;publication_year=2020&amp;pages=103336&amp;pmid=31703861&amp;doi=10.1016/j.fm.2019.103336&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR26\">26. <span class=\"element-citation\">Sadaie Y, Nakadate H, Fukui R, Yee LM, Asai K. Glucomannan utilization operon of <em>Bacillus subtilis</em>. <span><span class=\"ref-journal\">FEMS Microbiol. Lett. </span>2008;<span class=\"ref-vol\">279</span>:103–109. doi: 10.1111/j.1574-6968.2007.01018.x.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n18177310\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1574-6968.2007.01018.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=FEMS+Microbiol.+Lett.&amp;title=Glucomannan+utilization+operon+of+Bacillus+subtilis&amp;author=Y+Sadaie&amp;author=H+Nakadate&amp;author=R+Fukui&amp;author=LM+Yee&amp;author=K+Asai&amp;volume=279&amp;publication_year=2008&amp;pages=103-109&amp;pmid=18177310&amp;doi=10.1111/j.1574-6968.2007.01018.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR27\">27. <span class=\"element-citation\">Leth ML, et al.  Differential bacterial capture and transport preferences facilitate co-growth on dietary xylan in the human gut. <span><span class=\"ref-journal\">Nat. Microbiol. </span>2018;<span class=\"ref-vol\">3</span>:570–580. doi: 10.1038/s41564-018-0132-8.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n29610517\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fs41564-018-0132-8\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Microbiol.&amp;title=Differential+bacterial+capture+and+transport+preferences+facilitate+co-growth+on+dietary+xylan+in+the+human+gut&amp;author=ML+Leth&amp;volume=3&amp;publication_year=2018&amp;pages=570-580&amp;pmid=29610517&amp;doi=10.1038/s41564-018-0132-8&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR28\">28. <span class=\"element-citation\">Chum HL, Johnson DK, Black SK, Overend RP. Pretreatment catalyst effects and the combined severity parameter. <span><span class=\"ref-journal\">Appl. Biochem. Biotechnol. </span>1990;<span class=\"ref-vol\">24</span>:1–14. doi: 10.1007/BF02920229.</span> [<a href=\"//dx.doi.org/10.1007%2FBF02920229\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Biochem.+Biotechnol.&amp;title=Pretreatment+catalyst+effects+and+the+combined+severity+parameter&amp;author=HL+Chum&amp;author=DK+Johnson&amp;author=SK+Black&amp;author=RP+Overend&amp;volume=24&amp;publication_year=1990&amp;pages=1-14&amp;pmid=2353809&amp;doi=10.1007/BF02920229&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR29\">29. <span class=\"element-citation\">Vestby LK, M&#x000f8;retr&#x000f8; T, Ballance S, Langsrud S, Nesse LL. Survival potential of wild type cellulose deficient <em>Salmonella</em> from the feed industry. <span><span class=\"ref-journal\">BMC Vet. Res. </span>2009;<span class=\"ref-vol\">5</span>:43. doi: 10.1186/1746-6148-5-43.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2788542/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19930629\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2F1746-6148-5-43\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Vet.+Res.&amp;title=Survival+potential+of+wild+type+cellulose+deficient+Salmonella+from+the+feed+industry&amp;author=LK+Vestby&amp;author=T+M&#x000f8;retr&#x000f8;&amp;author=S+Ballance&amp;author=S+Langsrud&amp;author=LL+Nesse&amp;volume=5&amp;publication_year=2009&amp;pages=43&amp;pmid=19930629&amp;doi=10.1186/1746-6148-5-43&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR30\">30. <span class=\"element-citation\">Pedersen KS, Toft N. Intra- and inter-observer agreement when using a descriptive classification scale for clinical assessment of faecal consistency in growing pigs. <span><span class=\"ref-journal\">Prev. Vet. Med. </span>2011;<span class=\"ref-vol\">98</span>:288–291. doi: 10.1016/j.prevetmed.2010.11.016.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21185096\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.prevetmed.2010.11.016\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Prev.+Vet.+Med.&amp;title=Intra-+and+inter-observer+agreement+when+using+a+descriptive+classification+scale+for+clinical+assessment+of+faecal+consistency+in+growing+pigs&amp;author=KS+Pedersen&amp;author=N+Toft&amp;volume=98&amp;publication_year=2011&amp;pages=288-291&amp;pmid=21185096&amp;doi=10.1016/j.prevetmed.2010.11.016&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR31\">31. <span class=\"element-citation\">Moller I, et al.  High-throughput mapping of cell-wall polymers within and between plants using novel microarrays. <span><span class=\"ref-journal\">Plant J. </span>2007;<span class=\"ref-vol\">50</span>:1118–1128. doi: 10.1111/j.1365-313X.2007.03114.x.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17565618\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1365-313X.2007.03114.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Plant+J.&amp;title=High-throughput+mapping+of+cell-wall+polymers+within+and+between+plants+using+novel+microarrays&amp;author=I+Moller&amp;volume=50&amp;publication_year=2007&amp;pages=1118-1128&amp;pmid=17565618&amp;doi=10.1111/j.1365-313X.2007.03114.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR32\">32. <span class=\"element-citation\">Pedersen HL, et al.  Versatile high resolution oligosaccharide microarrays for plant glycobiology and cell wall research. <span><span class=\"ref-journal\">J. Biol. Chem. </span>2012;<span class=\"ref-vol\">287</span>:39429–39438. doi: 10.1074/jbc.M112.396598.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3501085/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22988248\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1074%2Fjbc.M112.396598\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Versatile+high+resolution+oligosaccharide+microarrays+for+plant+glycobiology+and+cell+wall+research&amp;author=HL+Pedersen&amp;volume=287&amp;publication_year=2012&amp;pages=39429-39438&amp;pmid=22988248&amp;doi=10.1074/jbc.M112.396598&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR33\">33. <span class=\"element-citation\">Takahashi S, Tomita J, Nishioka K, Hisada T, Nishijima M. Development of a prokaryotic universal primer for simultaneous analysis of Bacteria and Archaea using next-generation sequencing. <span><span class=\"ref-journal\">PLoS ONE. </span>2014;<span class=\"ref-vol\">9</span>:e105592. doi: 10.1371/journal.pone.0105592.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4140814/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25144201\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0105592\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Development+of+a+prokaryotic+universal+primer+for+simultaneous+analysis+of+Bacteria+and+Archaea+using+next-generation+sequencing&amp;author=S+Takahashi&amp;author=J+Tomita&amp;author=K+Nishioka&amp;author=T+Hisada&amp;author=M+Nishijima&amp;volume=9&amp;publication_year=2014&amp;pages=e105592&amp;pmid=25144201&amp;doi=10.1371/journal.pone.0105592&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR34\">34. <span class=\"element-citation\">Caporaso JG, et al.  QIIME allows analysis of high-throughput community sequencing data. <span><span class=\"ref-journal\">Nat. Methods. </span>2010;<span class=\"ref-vol\">7</span>:335–336. doi: 10.1038/nmeth.f.303.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3156573/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20383131\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnmeth.f.303\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=QIIME+allows+analysis+of+high-throughput+community+sequencing+data&amp;author=JG+Caporaso&amp;volume=7&amp;publication_year=2010&amp;pages=335-336&amp;pmid=20383131&amp;doi=10.1038/nmeth.f.303&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR35\">35. <span class=\"element-citation\">Schloss PD, et al.  Introducing mothur: open-source, platform-independent, community-supported software for describing and comparing microbial communities. <span><span class=\"ref-journal\">Appl. Environ. Microbiol. </span>2009;<span class=\"ref-vol\">75</span>:7537–7541. doi: 10.1128/AEM.01541-09.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2786419/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19801464\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.01541-09\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microbiol.&amp;title=Introducing+mothur:+open-source,+platform-independent,+community-supported+software+for+describing+and+comparing+microbial+communities&amp;author=PD+Schloss&amp;volume=75&amp;publication_year=2009&amp;pages=7537-7541&amp;pmid=19801464&amp;doi=10.1128/AEM.01541-09&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR36\">36. <span class=\"element-citation\">McMurdie PJ, Holmes S. phyloseq: an R package for reproducible interactive analysis and graphics of microbiome census data. <span><span class=\"ref-journal\">PLoS ONE. </span>2013;<span class=\"ref-vol\">8</span>:e61217. doi: 10.1371/journal.pone.0061217.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3632530/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23630581\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0061217\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=phyloseq:+an+R+package+for+reproducible+interactive+analysis+and+graphics+of+microbiome+census+data&amp;author=PJ+McMurdie&amp;author=S+Holmes&amp;volume=8&amp;publication_year=2013&amp;pages=e61217&amp;pmid=23630581&amp;doi=10.1371/journal.pone.0061217&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR37\">37. <span class=\"element-citation\">Zhang J, Kobert K, Flouri T, Stamatakis A. PEAR: a fast and accurate Illumina Paired-End reAd mergeR. <span><span class=\"ref-journal\">Bioinformatics. </span>2014;<span class=\"ref-vol\">30</span>:614–620. doi: 10.1093/bioinformatics/btt593.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3933873/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24142950\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtt593\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=PEAR:+a+fast+and+accurate+Illumina+Paired-End+reAd+mergeR&amp;author=J+Zhang&amp;author=K+Kobert&amp;author=T+Flouri&amp;author=A+Stamatakis&amp;volume=30&amp;publication_year=2014&amp;pages=614-620&amp;pmid=24142950&amp;doi=10.1093/bioinformatics/btt593&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR38\">38. <span class=\"element-citation\">Schmieder R, Edwards R. Quality control and preprocessing of metagenomic datasets. <span><span class=\"ref-journal\">Bioinformatics. </span>2011;<span class=\"ref-vol\">27</span>:863–864. doi: 10.1093/bioinformatics/btr026.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3051327/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21278185\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtr026\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Quality+control+and+preprocessing+of+metagenomic+datasets&amp;author=R+Schmieder&amp;author=R+Edwards&amp;volume=27&amp;publication_year=2011&amp;pages=863-864&amp;pmid=21278185&amp;doi=10.1093/bioinformatics/btr026&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR39\">39. <span class=\"element-citation\">Rideout JR, et al.  Subsampled open-reference clustering creates consistent, comprehensive OTU definitions and scales to billions of sequences. <span><span class=\"ref-journal\">PeerJ. </span>2014;<span class=\"ref-vol\">2</span>:e545. doi: 10.7717/peerj.545.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4145071/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25177538\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.7717%2Fpeerj.545\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=Subsampled+open-reference+clustering+creates+consistent,+comprehensive+OTU+definitions+and+scales+to+billions+of+sequences&amp;author=JR+Rideout&amp;volume=2&amp;publication_year=2014&amp;pages=e545&amp;pmid=25177538&amp;doi=10.7717/peerj.545&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR40\">40. <span class=\"element-citation\">Rognes T, Flouri T, Nichols B, Quince C, Mahe F. VSEARCH: a versatile open source tool for metagenomics. <span><span class=\"ref-journal\">PeerJ. </span>2016;<span class=\"ref-vol\">4</span>:e2584. doi: 10.7717/peerj.2584.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5075697/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27781170\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.7717%2Fpeerj.2584\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=VSEARCH:+a+versatile+open+source+tool+for+metagenomics&amp;author=T+Rognes&amp;author=T+Flouri&amp;author=B+Nichols&amp;author=C+Quince&amp;author=F+Mahe&amp;volume=4&amp;publication_year=2016&amp;pages=e2584&amp;pmid=27781170&amp;doi=10.7717/peerj.2584&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR41\">41. <span class=\"element-citation\">Yilmaz P, et al.  The SILVA and &#x0201c;All-species Living Tree Project (LTP)&#x0201d; taxonomic frameworks. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2014;<span class=\"ref-vol\">42</span>:D643–648. doi: 10.1093/nar/gkt1209.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3965112/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24293649\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgkt1209\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+SILVA+and+&#x0201c;All-species+Living+Tree+Project+(LTP)&#x0201d;+taxonomic+frameworks&amp;author=P+Yilmaz&amp;volume=42&amp;publication_year=2014&amp;pages=D643-648&amp;pmid=24293649&amp;doi=10.1093/nar/gkt1209&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR42\">42. <span class=\"element-citation\">Bolger AM, Lohse M, Usadel B. Trimmomatic: a flexible trimmer for Illumina sequence data. <span><span class=\"ref-journal\">Bioinformatics. </span>2014;<span class=\"ref-vol\">30</span>:2114–2120. doi: 10.1093/bioinformatics/btu170.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4103590/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24695404\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtu170\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Trimmomatic:+a+flexible+trimmer+for+Illumina+sequence+data&amp;author=AM+Bolger&amp;author=M+Lohse&amp;author=B+Usadel&amp;volume=30&amp;publication_year=2014&amp;pages=2114-2120&amp;pmid=24695404&amp;doi=10.1093/bioinformatics/btu170&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR43\">43. <span class=\"element-citation\">Nurk S, Meleshko D, Korobeynikov A, Pevzner P. A. metaSPAdes: a new versatile metagenomic assembler. <span><span class=\"ref-journal\">Genome Res. </span>2017;<span class=\"ref-vol\">27</span>:824–834. doi: 10.1101/gr.213959.116.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28298430\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1101%2Fgr.213959.116\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=A.+metaSPAdes:+a+new+versatile+metagenomic+assembler&amp;author=S+Nurk&amp;author=D+Meleshko&amp;author=A+Korobeynikov&amp;author=P+Pevzner&amp;volume=27&amp;publication_year=2017&amp;pages=824-834&amp;pmid=28298430&amp;doi=10.1101/gr.213959.116&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR44\">44. <span class=\"element-citation\">Li D, Liu CM, Luo R, Sadakane K, Lam TW. MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph. <span><span class=\"ref-journal\">Bioinformatics. </span>2015;<span class=\"ref-vol\">31</span>:1674–1676. doi: 10.1093/bioinformatics/btv033.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25609793\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtv033\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=MEGAHIT:+an+ultra-fast+single-node+solution+for+large+and+complex+metagenomics+assembly+via+succinct+de+Bruijn+graph&amp;author=D+Li&amp;author=CM+Liu&amp;author=R+Luo&amp;author=K+Sadakane&amp;author=TW+Lam&amp;volume=31&amp;publication_year=2015&amp;pages=1674-1676&amp;pmid=25609793&amp;doi=10.1093/bioinformatics/btv033&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR45\">45. <span class=\"element-citation\">Kang DD, Froula J, Egan R, Wang Z. MetaBAT, an efficient tool for accurately reconstructing single genomes from complex microbial communities. <span><span class=\"ref-journal\">PeerJ. </span>2015;<span class=\"ref-vol\">3</span>:e1165. doi: 10.7717/peerj.1165.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4556158/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26336640\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.7717%2Fpeerj.1165\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=MetaBAT,+an+efficient+tool+for+accurately+reconstructing+single+genomes+from+complex+microbial+communities&amp;author=DD+Kang&amp;author=J+Froula&amp;author=R+Egan&amp;author=Z+Wang&amp;volume=3&amp;publication_year=2015&amp;pages=e1165&amp;pmid=26336640&amp;doi=10.7717/peerj.1165&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR46\">46. <span class=\"element-citation\">Olm MR, Brown CT, Brooks B, Banfield JF. dRep: a tool for fast and accurate genomic comparisons that enables improved genome recovery from metagenomes through de-replication. <span><span class=\"ref-journal\">ISME J. </span>2017;<span class=\"ref-vol\">11</span>:2864–2868. doi: 10.1038/ismej.2017.126.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5702732/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28742071\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fismej.2017.126\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=ISME+J.&amp;title=dRep:+a+tool+for+fast+and+accurate+genomic+comparisons+that+enables+improved+genome+recovery+from+metagenomes+through+de-replication&amp;author=MR+Olm&amp;author=CT+Brown&amp;author=B+Brooks&amp;author=JF+Banfield&amp;volume=11&amp;publication_year=2017&amp;pages=2864-2868&amp;pmid=28742071&amp;doi=10.1038/ismej.2017.126&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR47\">47. <span class=\"element-citation\">Ondov BD, et al.  Mash: fast genome and metagenome distance estimation using MinHash. <span><span class=\"ref-journal\">Genome Biol. </span>2016;<span class=\"ref-vol\">17</span>:132. doi: 10.1186/s13059-016-0997-x.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4915045/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27323842\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2Fs13059-016-0997-x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Biol.&amp;title=Mash:+fast+genome+and+metagenome+distance+estimation+using+MinHash&amp;author=BD+Ondov&amp;volume=17&amp;publication_year=2016&amp;pages=132&amp;pmid=27323842&amp;doi=10.1186/s13059-016-0997-x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR48\">48. <span class=\"element-citation\">Parks DH, Imelfort M, Skennerton CT, Hugenholtz P, Tyson GW. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. <span><span class=\"ref-journal\">Genome Res. </span>2015;<span class=\"ref-vol\">25</span>:1043–1055. doi: 10.1101/gr.186072.114.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4484387/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25977477\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1101%2Fgr.186072.114\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=CheckM:+assessing+the+quality+of+microbial+genomes+recovered+from+isolates,+single+cells,+and+metagenomes&amp;author=DH+Parks&amp;author=M+Imelfort&amp;author=CT+Skennerton&amp;author=P+Hugenholtz&amp;author=GW+Tyson&amp;volume=25&amp;publication_year=2015&amp;pages=1043-1055&amp;pmid=25977477&amp;doi=10.1101/gr.186072.114&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR49\">49. <span class=\"mixed-citation\">Chaumeil, P. A., Mussig, A. J., Hugenholtz, P. &#x00026; Parks, D. H. GTDB-Tk: a toolkit to classify genomes with the Genome Taxonomy Database. <em>Bioinformatics</em>10.1093/bioinformatics/btz848 (2019). <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7703759/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n31730192\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>]</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR50\">50. <span class=\"element-citation\">Seemann T. Prokka: rapid prokaryotic genome annotation. <span><span class=\"ref-journal\">Bioinformatics. </span>2014;<span class=\"ref-vol\">30</span>:2068–2069. doi: 10.1093/bioinformatics/btu153.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24642063\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtu153\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Prokka:+rapid+prokaryotic+genome+annotation&amp;author=T+Seemann&amp;volume=30&amp;publication_year=2014&amp;pages=2068-2069&amp;pmid=24642063&amp;doi=10.1093/bioinformatics/btu153&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR51\">51. <span class=\"element-citation\">Terrapon N, Lombard V, Gilbert HJ, Henrissat B. Automatic prediction of polysaccharide utilization loci in Bacteroidetes species. <span><span class=\"ref-journal\">Bioinformatics. </span>2014;<span class=\"ref-vol\">31</span>:647–655. doi: 10.1093/bioinformatics/btu716.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25355788\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtu716\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Automatic+prediction+of+polysaccharide+utilization+loci+in+Bacteroidetes+species&amp;author=N+Terrapon&amp;author=V+Lombard&amp;author=HJ+Gilbert&amp;author=B+Henrissat&amp;volume=31&amp;publication_year=2014&amp;pages=647-655&amp;pmid=25355788&amp;doi=10.1093/bioinformatics/btu716&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR52\">52. <span class=\"element-citation\">Lombard V, Golaconda Ramulu H, Drula E, Coutinho PM, Henrissat B. The carbohydrate-active enzymes database (CAZy) in 2013. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2014;<span class=\"ref-vol\">42</span>:D490–495. doi: 10.1093/nar/gkt1178.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3965031/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24270786\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgkt1178\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+carbohydrate-active+enzymes+database+(CAZy)+in+2013&amp;author=V+Lombard&amp;author=H+Golaconda+Ramulu&amp;author=E+Drula&amp;author=PM+Coutinho&amp;author=B+Henrissat&amp;volume=42&amp;publication_year=2014&amp;pages=D490-495&amp;pmid=24270786&amp;doi=10.1093/nar/gkt1178&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR53\">53. <span class=\"element-citation\">Arntzen MO, Karlskas IL, Skaugen M, Eijsink VG. Proteomic investigation of the response of <em>Enterococcus faecalis</em> V583 when cultivated in urine. <span><span class=\"ref-journal\">PLoS ONE. </span>2015;<span class=\"ref-vol\">10</span>:e0126694. doi: 10.1371/journal.pone.0126694.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4411035/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25915650\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0126694\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Proteomic+investigation+of+the+response+of+Enterococcus+faecalis+V583+when+cultivated+in+urine&amp;author=MO+Arntzen&amp;author=IL+Karlskas&amp;author=M+Skaugen&amp;author=VG+Eijsink&amp;volume=10&amp;publication_year=2015&amp;pages=e0126694&amp;pmid=25915650&amp;doi=10.1371/journal.pone.0126694&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR54\">54. <span class=\"element-citation\">Arntzen MO, Karlskas IL, Skaugen M, Eijsink VG, Mathiesen G. Proteomic investigation of the response of <em>Enterococcus faecalis</em> V583 when cultivated in urine. <span><span class=\"ref-journal\">PLoS ONE. </span>2015;<span class=\"ref-vol\">10</span>:e0126694. doi: 10.1371/journal.pone.0126694.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4411035/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25915650\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0126694\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Proteomic+investigation+of+the+response+of+Enterococcus+faecalis+V583+when+cultivated+in+urine&amp;author=MO+Arntzen&amp;author=IL+Karlskas&amp;author=M+Skaugen&amp;author=VG+Eijsink&amp;author=G+Mathiesen&amp;volume=10&amp;publication_year=2015&amp;pages=e0126694&amp;pmid=25915650&amp;doi=10.1371/journal.pone.0126694&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR55\">55. <span class=\"element-citation\">Cox J, Mann M. MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. <span><span class=\"ref-journal\">Nat. Biotechnol. </span>2008;<span class=\"ref-vol\">26</span>:1367–1372. doi: 10.1038/nbt.1511.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19029910\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnbt.1511\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=MaxQuant+enables+high+peptide+identification+rates,+individualized+p.p.b.-range+mass+accuracies+and+proteome-wide+protein+quantification&amp;author=J+Cox&amp;author=M+Mann&amp;volume=26&amp;publication_year=2008&amp;pages=1367-1372&amp;pmid=19029910&amp;doi=10.1038/nbt.1511&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR56\">56. <span class=\"element-citation\">Cox J, et al.  Accurate proteome-wide label-free quantification by delayed normalization and maximal peptide ratio extraction, termed MaxLFQ. <span><span class=\"ref-journal\">Mol. Cell. Proteom. </span>2014;<span class=\"ref-vol\">13</span>:2513–2526. doi: 10.1074/mcp.M113.031591.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4159666/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24942700\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1074%2Fmcp.M113.031591\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Proteom.&amp;title=Accurate+proteome-wide+label-free+quantification+by+delayed+normalization+and+maximal+peptide+ratio+extraction,+termed+MaxLFQ&amp;author=J+Cox&amp;volume=13&amp;publication_year=2014&amp;pages=2513-2526&amp;doi=10.1074/mcp.M113.031591&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR57\">57. <span class=\"element-citation\">Hug LA, et al.  A new view of the tree of life. <span><span class=\"ref-journal\">Nat. Microbiol. </span>2016;<span class=\"ref-vol\">1</span>:16048. doi: 10.1038/nmicrobiol.2016.48.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27572647\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnmicrobiol.2016.48\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Microbiol.&amp;title=A+new+view+of+the+tree+of+life&amp;author=LA+Hug&amp;volume=1&amp;publication_year=2016&amp;pages=16048&amp;pmid=27572647&amp;doi=10.1038/nmicrobiol.2016.48&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR58\">58. <span class=\"element-citation\">Wu D, Jospin G, Eisen JA. Systematic identification of gene families for use as &#x0201c;markers&#x0201d; for phylogenetic and phylogeny-driven ecological studies of bacteria and archaea and their major subgroups. <span><span class=\"ref-journal\">PLoS ONE. </span>2013;<span class=\"ref-vol\">8</span>:e77033. doi: 10.1371/journal.pone.0077033.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3798382/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24146954\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0077033\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Systematic+identification+of+gene+families+for+use+as+&#x0201c;markers&#x0201d;+for+phylogenetic+and+phylogeny-driven+ecological+studies+of+bacteria+and+archaea+and+their+major+subgroups&amp;author=D+Wu&amp;author=G+Jospin&amp;author=JA+Eisen&amp;volume=8&amp;publication_year=2013&amp;pages=e77033&amp;pmid=24146954&amp;doi=10.1371/journal.pone.0077033&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR59\">59. <span class=\"element-citation\">Mikheenko A, Saveliev V, Gurevich A. MetaQUAST: evaluation of metagenome assemblies. <span><span class=\"ref-journal\">Bioinformatics. </span>2016;<span class=\"ref-vol\">32</span>:1088–1090. doi: 10.1093/bioinformatics/btv697.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26614127\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtv697\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=MetaQUAST:+evaluation+of+metagenome+assemblies&amp;author=A+Mikheenko&amp;author=V+Saveliev&amp;author=A+Gurevich&amp;volume=32&amp;publication_year=2016&amp;pages=1088-1090&amp;pmid=26614127&amp;doi=10.1093/bioinformatics/btv697&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR60\">60. <span class=\"element-citation\">Edgar RC. MUSCLE: multiple sequence alignment with high accuracy and high throughput. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2004;<span class=\"ref-vol\">32</span>:1792–1797. doi: 10.1093/nar/gkh340.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC390337/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n15034147\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgkh340\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=MUSCLE:+multiple+sequence+alignment+with+high+accuracy+and+high+throughput&amp;author=RC+Edgar&amp;volume=32&amp;publication_year=2004&amp;pages=1792-1797&amp;pmid=15034147&amp;doi=10.1093/nar/gkh340&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR61\">61. <span class=\"element-citation\">Talavera G, Castresana J. Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments. <span><span class=\"ref-journal\">Syst. Biol. </span>2007;<span class=\"ref-vol\">56</span>:564–577. doi: 10.1080/10635150701472164.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17654362\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1080%2F10635150701472164\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Syst.+Biol.&amp;title=Improvement+of+phylogenies+after+removing+divergent+and+ambiguously+aligned+blocks+from+protein+sequence+alignments&amp;author=G+Talavera&amp;author=J+Castresana&amp;volume=56&amp;publication_year=2007&amp;pages=564-577&amp;pmid=17654362&amp;doi=10.1080/10635150701472164&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR62\">62. <span class=\"element-citation\">Stamatakis A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. <span><span class=\"ref-journal\">Bioinformatics. </span>2014;<span class=\"ref-vol\">30</span>:1312–1313. doi: 10.1093/bioinformatics/btu033.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3998144/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24451623\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtu033\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=RAxML+version+8:+a+tool+for+phylogenetic+analysis+and+post-analysis+of+large+phylogenies&amp;author=A+Stamatakis&amp;volume=30&amp;publication_year=2014&amp;pages=1312-1313&amp;pmid=24451623&amp;doi=10.1093/bioinformatics/btu033&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR63\">63. <span class=\"element-citation\">Letunic I, Bork P. Interactive tree of life (iTOL) v3: an online tool for the display and annotation of phylogenetic and other trees. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2016;<span class=\"ref-vol\">44</span>:W242–W245. doi: 10.1093/nar/gkw290.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4987883/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27095192\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgkw290\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Interactive+tree+of+life+(iTOL)+v3:+an+online+tool+for+the+display+and+annotation+of+phylogenetic+and+other+trees&amp;author=I+Letunic&amp;author=P+Bork&amp;volume=44&amp;publication_year=2016&amp;pages=W242-W245&amp;pmid=27095192&amp;doi=10.1093/nar/gkw290&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR64\">64. <span class=\"mixed-citation\">Pope, P. B. Annotated MAG files from the pig gut microbiome. Preprint at 10.6084/m6089.figshare.9816581.v9816581 (2020).</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR65\">65. <span class=\"element-citation\">Vizcaino JA, et al.  The PRoteomics IDEntifications (PRIDE) database and associated tools: status in 2013. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2013;<span class=\"ref-vol\">41</span>:D1063–1069. doi: 10.1093/nar/gks1262.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3531176/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23203882\" ref=\"reftype=pubmed&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgks1262\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+PRoteomics+IDEntifications+(PRIDE)+database+and+associated+tools:+status+in+2013&amp;author=JA+Vizcaino&amp;volume=41&amp;publication_year=2013&amp;pages=D1063-1069&amp;pmid=23203882&amp;doi=10.1093/nar/gks1262&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"CR66\">66. <span class=\"element-citation\">Willf&#x000f6;r S, et al.  Characterisation of water-soluble galactoglucomannans from Norway spruce wood and thermomechanical pulp. <span><span class=\"ref-journal\">Carbohydr. Polym. </span>2003;<span class=\"ref-vol\">52</span>:175–187. doi: 10.1016/S0144-8617(02)00288-6.</span> [<a href=\"//dx.doi.org/10.1016%2FS0144-8617(02)00288-6\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Carbohydr.+Polym.&amp;title=Characterisation+of+water-soluble+galactoglucomannans+from+Norway+spruce+wood+and+thermomechanical+pulp&amp;author=S+Willf&#x000f6;r&amp;volume=52&amp;publication_year=2003&amp;pages=175-187&amp;doi=10.1016/S0144-8617(02)00288-6&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7666174&amp;issue-id=348897&amp;journal-id=2873&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div></div><br>', NULL, 2);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(19, 'pubmed', '53', 'The interaction of phospholipase A2 with micellar interfaces. The role of the N-terminal region.', 'The localization of the previously postulated interface recognition site (IRS) in porcine pancreatic phospholipase A2, required for a specific interaction between the enzyme and organized lipid-water interfaces, was investigated by ultraviolet difference spectroscopy, by measurements of the intrinsic fluorescence of the unique Trp residue, and by protection experiments against specific tryptic hydrolysis. Using the enzymically nondegradable substrate analogues: CnH(2n+1)(0-)OOCH2CH2N+(CH3)3-(H,OH), it is shown that the rather hydrophobic N-terminal sequence of the enzyme, viz., Ala-Leu-Trp-Gln-Phe-Arg, is directly involved in the interaction with the lipid-water interface. Besides hydrophobic probably also polar interactions contribute to the binding process. At neutral or acidic pH the presence of a salt bridge between the N-terminal alpha-NH3+ group and a negatively charged side chain stablizes the interface recognition site and allows the enzyme to penetrate micellar surfaces, even in the absence of metal ion. At alkaline pH, interaction of the enzyme with micellar interfaces requires the presence of Ca2+ (Ba2+) ions.', 1975, 'Biochemistry', '', 1, NULL, NULL, 1),
(20, 'pubmed', '31712689', 'Light on the cell cycle of the non-photosynthetic bacterium Ramlibacter tataouinensis.', 'Ramlibacter tataouinensis TTB310, a non-photosynthetic betaproteobacterium isolated from a semi-arid region of southern Tunisia, forms both rods and cysts. Cysts are resistant to desiccation and divide when water and nutrients are available. Rods are motile and capable of dissemination. Due to the strong correlation between sunlight and desiccation, light is probably an important external signal for anticipating desiccating conditions. Six genes encoding potential light sensors were identified in strain TTB310. Two genes encode for bacteriophytochromes, while the four remaining genes encode for putative blue light receptors. We determined the spectral and photochemical properties of the two recombinant bacteriophytochromes RtBphP1 and RtBphP2. In both cases, they act as sensitive red light detectors. Cyst divisions and a complete cyst-rod-cyst cycle are the main processes in darkness, whereas rod divisions predominate in red or far-red light. Mutant phenotypes caused by the inactivation of genes encoding bacteriophytochromes or heme oxygenase clearly show that both bacteriophytochromes are involved in regulating the rod-rod division. This process could favor rapid rod divisions at sunrise, after dew formation but before the progressive onset of desiccation. Our study provides the first evidence of a light-based strategy evolved in a non-photosynthetic bacterium to exploit scarse water in a desert environment.', 2019, 'Scientific reports', 'PMC6848086', 1, NULL, NULL, 5),
(21, 'pubmed', '31624160', 'Complete Genome Sequences of Four  Strains Isolated from Metal- and Radionuclide-Rich Soils.', 'Here, we present the genome sequences of four  strains, which were isolated at different locations in Europe from metal- or radionuclide-rich soils. High-quality complete genome sequences were obtained with PacBio and Illumina data sets with an original two-step procedure.', 2019, 'Microbiology resource announcements', 'PMC6797525', 2, NULL, NULL, 3),
(24, 'pubmed', '56', 'Kinetic light scattering studies on the dissociation of hemoglobin from Lumbricus terrestris.', 'The kinetics of the pH-induced dissociation of the 3 X 10(6) mol wt hemoglobin from Lumbricus terrestris (the earthworm) have been studied in a light-scattering stopped-flow apparatus. The ligand dependent dissociation data were fit well by a simple sequential model. The data for CO and oxyhemoglobin are consistent with Hb12 leads to 2Hb6 leads to 12Hb. Methemoglobin at pH 7 appears to be hexameric and the dissociation is consistent with the model: Hb6 leads to 6Hb. In a sequential decay scheme for which light-scattering changes are monitored, the relative amounts of rapid and slow phase are determined by the rate constants as well as the molecular weights of intermediate species. Assignment of the hexameric intermediate is supported by an investigation of the sensitivity of the theoretical kinetic curves to the molecular weights of the intermediates. This assignment is further supported by the following: (1) the same model will fit the data for oxy- and CO-hemoglobin at all three temperatures (a 24-29-fold variation in rate constants), (2) evidence from electron microscopy shows hexameric forms, and (3) methemoglobin is apparently stable as a hexamer at pH 7. When CO replaces O2 as the ligand, the dissociation rate increases by a factor of four. The met is about 20 times faster than the initial oxyhemoglobin dissociation rate, but perhaps more relevant for comparing dissociation of the hexamer, the met rate was respectively 100 times and 500 times faster than that for the assumed hexameric forms of CO- and oxy-hemoglobin. The activation energies for the dodecamer to hexamer dissociation and for the dissociation of the hexamer to smaller forms were about 30 kcal/mol for oxy-, CO-, and methemoglobin.', 1975, 'Biochemistry', '', 2, NULL, NULL, 4),
(25, 'pubmed', '33674431', 'Novel 3--α-d-Galactosyl-α-l-Arabinofuranosidase for the Assimilation of Gum Arabic Arabinogalactan Protein in Bifidobacterium longum subsp. .', 'Gum arabic arabinogalactan (AG) protein (AGP) is a unique dietary fiber that is degraded and assimilated by only specific strains of  subsp.  Here, we identified a novel 3--α-d-galactosyl-α-l-arabinofuranosidase (GAfase) from  JCM7052 and classified it into glycoside hydrolase family 39 (GH39). GAfase released α-d-Gal-(1→3)-l-Ara and β-l-Ara-(1→3)-l-Ara from gum arabic AGP and β-l-Ara-(1→3)-l-Ara from larch AGP, and the α-d-Gal-(1→3)-l-Ara release activity was found to be 594-fold higher than that of β-l-Ara-(1→3)-l-Ara. The GAfase gene was part of a gene cluster that included genes encoding a GH36 α-galactosidase candidate and ABC transporters for the assimilation of the released α-d-Gal-(1→3)-l-Ara in  Notably, when α-d-Gal-(1→3)-l-Ara was removed from gum arabic AGP, it was assimilated by both  JCM7052 and the nonassimilative  JCM1217, suggesting that the removal of α-d-Gal-(1→3)-l-Ara from gum arabic AGP by GAfase permitted the cooperative action with type II AG degradative enzymes in  The present study provides new insight into the mechanism of gum arabic AGP degradation in  Bifidobacteria harbor numerous carbohydrate-active enzymes that degrade several dietary fibers in the gastrointestinal tract.  JCM7052 is known to exhibit the ability to assimilate gum arabic AGP, but the key enzyme involved in the degradation of gum arabic AGP remains unidentified. Here, we cloned and characterized a GH39 3--α-d-galactosyl-α-l-arabinofuranosidase (GAfase) from  JCM7052. The enzyme was responsible for the release of α-d-Gal-(1→3)-l-Ara and β-l-Ara-(1→3)-l-Ara from gum arabic AGP. The presence of a gene cluster including the GAfase gene is specifically observed in gum arabic AGP assimilative strains. However, GAfase carrier strains may affect GAfase noncarrier strains that express other type II AG degradative enzymes. These findings provide insights into the bifidogenic effect of gum arabic AGP.', 2021, 'Applied and environmental microbiology', '', 3, NULL, NULL, 6),
(26, 'pubmed', '33394289', 'Substrate Specificities of GH8, GH39, and GH52 β-xylosidases from Bacillus halodurans C-125 Toward Substituted Xylooligosaccharides.', 'Substrate specificities of glycoside hydrolase families 8 (Rex), 39 (BhXyl39), and 52 (BhXyl52) β-xylosidases from Bacillus halodurans C-125 were investigated. BhXyl39 hydrolyzed xylotriose most efficiently among the linear xylooligosaccharides. The activity decreased in the order of xylohexaose > xylopentaose > xylotetraose and it had little effect on xylobiose. In contrast, BhXyl52 hydrolyzed xylobiose and xylotriose most efficiently, and its activity decreased when the main chain became longer as follows: xylotetraose > xylopentaose > xylohexaose. Rex produced O-β-D-xylopyranosyl-(1 → 4)-[O-α-L-arabinofuranosyl-(1 → 3)]-O-β-D-xylopyranosyl-(1 → 4)-β-D-xylopyranose (AraXyl) and O-β-D-xylopyranosyl-(1 → 4)-[O-4-O-methyl-α-D-glucuronopyranosyl-(l → 2)]-β-D-xylopyranosyl-(1 → 4)-β-D-xylopyranose (MeGlcAXyl), which lost a xylose residue from the reducing end of O-β-D-xylopyranosyl-(1 → 4)-[O-α-L-arabinofuranosyl-(1 → 3)]-O-β-D-xylopyranosyl-(1 → 4)-β-D-xylopyranosyl-(1 → 4)-β-D-xylopyranose (AraXyl) and O-β-D-xylopyranosyl-(1 → 4)-[O-4-O-methyl-α-D-glucuronopyranosyl-(1 → 2)]-β-D-xylopyranosyl-(1 → 4)-β-D-xylopyranosyl-(1 → 4)-β-D-xylopyranose (MeGlcAXyl). It was considered that there is no space to accommodate side chains at subsite -1. BhXyl39 rapidly hydrolyzes the non-reducing-end xylose linkages of MeGlcAXyl, while the arabinose branch does not significantly affect the enzyme activity because it degrades AraXyl as rapidly as unmodified xylotetraose. The model structure suggested that BhXyl39 enhanced the activity for MeGlcAXyl by forming a hydrogen bond between glucuronic acid and Lys265. BhXyl52 did not hydrolyze AraXyl and MeGlcAXyl because it has a narrow substrate binding pocket and 2- and 3-hydroxyl groups of xylose at subsite +1 hydrogen bond to the enzyme.', 2021, 'Applied biochemistry and biotechnology', '', 4, NULL, NULL, 5),
(27, 'pubmed', '32880690', 'Characterization and functional analysis of two novel thermotolerant α-L-arabinofuranosidases belonging to glycoside hydrolase family 51 from Thielavia terrestris and family 62 from Eupenicillium parvum.', 'Arabinofuranose substitutions on xylan are known to interfere with enzymatic hydrolysis of this primary hemicellulose. In this work, two novel α-L-arabinofuranosidases (ABFs), TtABF51A from Thielavia terrestris and EpABF62C from Eupenicillium parvum, were characterized and functionally analyzed. From sequences analyses, TtABF51A and EpABF62C belong to glycoside hydrolase (GH) families 51 and 62, respectively. Recombinant TtABF51A showed high activity on 4-nitrophenyl-α-L-arabinofuranoside (83.39 U/mg), low-viscosity wheat arabinoxylan (WAX, 39.66 U/mg), high-viscosity rye arabinoxylan (RAX, 32.24 U/mg), and sugarbeet arabinan (25.69 U/mg), while EpABF62C preferred to degrade arabinoxylan. For EpABF62C, the rate of hydrolysis of RAX (94.10 U/mg) was 2.1 times that of WAX (45.46 U/mg). The optimal pH and reaction temperature for the two enzymes was between 4.0 and 4.5 and 65 °C, respectively. Calcium played an important role in the thermal stability of EpABF62C. TtABF51A and EpABF62C showed the highest thermal stabilities at pH 4.5 or 5.0, respectively. At their optimal pHs, TtABF51A and EpABF62C retained greater than 80% of their initial activities after incubation at 55 °C for 96 h or 144 h, respectively. H NMR analysis indicated that the two enzymes selectively removed arabinose linked to C-3 of mono-substituted xylose residues in WAX. Compared with the singular application of the GH10 xylanase EpXYN1 from E. parvum, co-digestions of WAX including TtABF51A and/or EpABF62C released 2.49, 3.38, and 4.81 times xylose or 3.38, 1.65, and 2.57 times of xylobiose, respectively. Meanwhile, the amount of arabinose released from WAX by TtABF51A with EpXYN1 was 2.11 times the amount with TtABF51A alone. KEY POINTS: • Two novel α-l-arabinofuranosidases (ABFs) displayed high thermal stability. • The thermal stability of GH62 family EpABF62C was dependent on calcium. • Buffer pH affects the thermal stability of the two ABFs. • Both ABFs enhance the hydrolysis of WAX by a GH10 xylanase.', 2020, 'Applied microbiology and biotechnology', 'PMC7502447', NULL, NULL, NULL, 1),
(28, 'pubmed', '32679333', 'Biochemical characterization of a novel halo/organic-solvents/final-products tolerant GH39 xylosidase from saline soil and its synergic action with xylanase.', 'Xylosidases with tolerance to high concentration of salts, organic solvents, and enzyme hydrolytic products are preferential for industrial application but were rarely reported. In this study, a novel xylosidase XYL21 belong to glycoside hydrolase 39 was characterized with optimal temperature of 45 °C and optimal pH of 5.50. Different to other GH39 xylosidases, XYL21 had excellent tolerance to salts, the activity of which is not inhibited but slightly increased in 0.50-1.50 M NaCl. It is also tolerant to organic solvents, especially retaining 105.18% relative activity even in the presence of 15.00% (v/v) ethanol. Moreover, XYL21 was insensitive to the final lignocellulose hydrolysis products including glucose, xylose, arabinose, mannose and galactose, which retains 111.36% and 53.49% relative activity in 0.30 and 0.90 M xylose, respectively. Further structural modeling analysis indicated that its excellent tolerance may be attributed to its high structural flexibility caused by the high proportion of random coils. Furthermore, XYL21 had a wide substrate specificity to catalyze xylan and xylo-oligosaccharides, and it significantly cooperated with xylanase to improve the hydrolysis efficiency with 1.52-fold. Considering these unique properties, XYL21 is a good candidate for both basic research and various potential industrial applications such as seafood processing and bioethanol production.', 2020, 'International journal of biological macromolecules', '', 4, NULL, NULL, 2),
(29, 'pubmed', '31304065', 'Production and purification of mannan oligosaccharide with epithelial tight junction enhancing activity.', 'Mannanan oligosaccharide (MOS) is well-known as effective supplement food for livestock to increase their nutrients absorption and health status, but the structure and identification of bioactive MOS remain unclear. In this study, MOS production was accomplished, using enzymatic hydrolysis of pretreated coconut meal substrate with recombinant mannanase.The mannanase gene was cloned from AE24, then expressed in BL21. Purified Mannanase exhibit stability over a wide range of pH and temperature from pH 6-8 and 4 °C to 70 °C, respectively. SEM analysis revealed that sonication could change the surface characteristic of copra meal, which gave better MOS yield, compared to untreated substrates. The separation and purification of each MOS were achieved using Biogel-P2 column chromatography. Determination of biological active MOS species was also investigated. T84 cells were cultured and treated with each of the purified MOS species to determine their tight junction enhancing activity.Scanning electron microscope imaging showed that pretreatment using sonication could disrupt the surface of copra meal better than grinding alone, which can improve the production of MOS. Pentamer of MOS (M5) significantly increased tight junction integration of T84 cells measured with TEER ( < 0.0001).', 2019, 'PeerJ', 'PMC6611449', 3, NULL, NULL, 3),
(30, 'pubmed', '54', 'Phospholipase A2 as a probe of phospholipid distribution in erythrocyte membranes. Factors influencing the apparent specificity of the reaction.', 'The action of snake venom phospholipases A2 in intact human erythrocytes was investigated in detail. The basis phospholipase from Agkistrodon halys blomhifii was found to induce both hydrolysis of membrane phospholipids and total cell hemolysis under certain experimental conditions. The hydrolytic action of the basic enzyme was found to consist of two sequential events: (a) hydrolysis of 70% of the total cell ph osphatidylcholine without any evident hemolysis; and (b) complete hydrolysis of the remaining phosphatidylcholine, followed closely by extensive phosphatidylethanolamine hydrolysis and finally with onset of hemolysis, attack on the phosphatidylserine. At pH 7.4 and 10 mM Ca2+ only stage (a) occurred. However, a slight elevation of the pH of incubation to pH 8.0 and/or inclusion of 40 mM Ca2+ in the reaction mixture caused both events (a) and (b) to occur. The addition of glucose limited the action of the enzyme to stage (a) under any reaction conditions. An investigation showed that enzymically induced hemolysis occurred under conditions where the intracellular ATP levels were lowered. Data are presented which suggest that stage (b) is mediated by in influx of Ca2+ into the cell when the levels of ATP are low. Interestingly the phosphllipase from Naja naja venom (Pakistan) yielded results similar to those observed with the basic enzyme from Agkistrodon venom. However, the enzyme from Crotalus adamanteus and the acidic enzyme also present in the Agkistrodon venom produced only slight hydrolysis or hemolysis under any of the conditions studied. Other species of erythrocytes, e.g., guinea pig, monkey, pig, and rat, were tested but only those from guinea pig behaved similarly to the human cells. Pig, monkey, and rat erythrocytes underwent very limited hydrolysis and hemolysis. It is evident that the use of these phospholipases to probe the localization of phospholipds in erythrocyte membranes must be approached with caution. Certain facets of this problem are discussed.', 1975, 'Biochemistry', '', 4, NULL, NULL, 1),
(31, 'pubmed', '32338906', 'Engineering of β-Glucosidase Bgl15 with Simultaneously Enhanced Glucose Tolerance and Thermostability To Improve Its Performance in High-Solid Cellulose Hydrolysis.', 'In this study, a Petri-dish-based double-layer high-throughput screening method was established to improve glucose tolerance of β-glucosidase Bgl15. Two beneficial mutations were identified, and the joint mutant 2R1 improved the half-maximal inhibitory concentration of glucose from 0.04 to 2.1 M. The crystal structure of 2R1 was subsequently determined at 2.7 Å. Structure analysis revealed that enhancement of glucose tolerance may be due to improved transglycosylation activity made possible by a hydrophobic binding site for glucose as an acceptor and more stringent control of a putative water channel. To further ameliorate the application potential of the enzyme, it was engineered to increase the half-life at 50 °C from 0.8 h (Bgl15) to 180 h (mutant 5R1). Furthermore, supplementation of 5R1 to the cellulase cocktail significantly improved glucose production from pretreated sugar cane bagasse by 38%. Consequently, this study provided an efficient approach to enhance glucose tolerance and generated a promising catalyst for cellulose saccharification.', 2020, 'Journal of agricultural and food chemistry', '', 1, NULL, NULL, 5),
(32, 'pubmed', '55', 'Subunit interactions in yeast glyceraldehyde-3-phosphate dehydrogenase.', 'The spontaneous inactivation of yeast glyceraldehyde-3-phosphate dehydrogenase was found to fit a simple two-state model at pH 8.5 and 25 degrees. The first step is a relatively rapid dissociation of the tetramer to dimers with the equilibrium largely in favor of the tetramer. In the absence of NAD+ the dimer inactivates irreversibly. The apoenzyme is quite stable with a half-life for complete activity loss proportional to the square root of the enzyme concentration. Perturbances of the protein structure (by pH, ionic strength, and specific salts), which have no effect on the tetrameric state of the molecule, result in an alteration of the cooperativity of NAD+ binding, the reactivity of the active-site sulfhydryl group, and the catalytic activity of the enzyme. Covalent modification of two of the four active-site sulfhydryl groups has profound effects on the enzymic activity which are mediated by changes in the subunit interactions. Sedimentation analysis and hybridization studies indicate that the interaction between subunits remains strong after covalent modification. Under normal physiological and equilibrium dialysis conditions the protein is a tetramer. Equilibrium dialysis studies of NAD+ binding to the enzyme at pH 8.5 and 25 degrees reveal a mixed cooperativity pattern. A model consistent with these observations and the observed half-of-the-sites reactivity is that of ligand induced sequential conformational changes which are transferred across strongly interacting subunit domains. Methods for distinguishing negatively cooperative binding patterns from mixtures of denatured enzyme and multiple species are discussed.', 1975, 'Biochemistry', '', 1, NULL, NULL, 3),
(33, 'pubmed', '28917914', 'Structure, enzymatic transformation, anticancer activity of fucoidan and sulphated fucooligosaccharides from Sargassum horneri.', 'Structure and anticancer activity of fucoidan from Sargassum horneri and from products of its enzymatic transformation were investigated. A gene that encodes fucoidanase ffa1 in the marine bacteria F. algae was identified, cloned and the protein (FFA1) was produced in Escherichia coli. The mass of the gene product FFA1 is 111kDa. Sequence analysis has revealed that fucoidanase FFA1 belongs to the GH107 (CAZy) family. Recombinant fucoidanase FFA1 was used to produce fucooligosaccharides. Structure of 5 sulphated oligosaccharides with polymerization degree 4-10 was established by NMR-spectroscopy. The fucoidan extracted from S. horneri is almost pure fucan. The main chain of the fucoidan is established to consist mostly of the repeating →3-α-l-Fucp(2SO)-1→4-α-l-Fucp(2,3SO)-1→ fragment, with insertions of →3-α-l-Fucp(2,4SO)-1→ fragment. Unsulphated side chains with the α-l-Fucp-1→2-α-l-Fucp-1→ structure connect to the main one at the C4 of monosaccharide residue.', 2017, 'Carbohydrate polymers', '', 2, NULL, NULL, 2),
(34, 'pubmed', '33208778', 'Conversion of levoglucosan into glucose by the coordination of four enzymes through oxidation, elimination, hydration, and reduction.', 'Levoglucosan (LG) is an anhydrosugar produced through glucan pyrolysis and is widely found in nature. We previously isolated an LG-utilizing thermophile, Bacillus smithii S-2701M, and suggested that this bacterium may have a metabolic pathway from LG to glucose, initiated by LG dehydrogenase (LGDH). Here, we completely elucidated the metabolic pathway of LG involving three novel enzymes in addition to LGDH. In the S-2701M genome, three genes expected to be involved in the LG metabolism were found in the vicinity of the LGDH gene locus. These four genes including LGDH gene (lgdA, lgdB1, lgdB2, and lgdC) were expressed in Escherichia coli and purified to obtain functional recombinant proteins. Thin layer chromatography analyses of the reactions with the combination of the four enzymes elucidated the following metabolic pathway: LgdA (LGDH) catalyzes 3-dehydrogenation of LG to produce 3-keto-LG, which undergoes β-elimination of 3-keto-LG by LgdB1, followed by hydration to produce 3-keto-D-glucose by LgdB2; next, LgdC reduces 3-keto-D-glucose to glucose. This sequential reaction mechanism resembles that proposed for an enzyme belonging to glycoside hydrolase family 4, and results in the observational hydrolysis of LG into glucose with coordination of the four enzymes.', 2020, 'Scientific reports', 'PMC7676230', 2, NULL, NULL, 6),
(35, 'pubmed', '33108275', 'Specificity in glycosylation of multiple flagellins by the modular and cell cycle regulated glycosyltransferase FlmG.', 'How specificity is programmed into post-translational modification of proteins by glycosylation is poorly understood, especially for O-linked glycosylation systems. Here we reconstitute and dissect the substrate specificity underpinning the cytoplasmic O-glycosylation pathway that modifies all six flagellins, five structural and one regulatory paralog, in , a monopolarly flagellated alpha-proteobacterium. We characterize the biosynthetic pathway for the sialic acid-like sugar pseudaminic acid and show its requirement for flagellation, flagellin modification and efficient export. The cognate NeuB enzyme that condenses phosphoenolpyruvate with a hexose into pseudaminic acid is functionally interchangeable with other pseudaminic acid synthases. The previously unknown and cell cycle-regulated FlmG protein, a defining member of a new class of cytoplasmic O-glycosyltransferases, is required and sufficient for flagellin modification. The substrate specificity of FlmG is conferred by its N-terminal flagellin-binding domain. FlmG accumulates before the FlaF secretion chaperone, potentially timing flagellin modification, export, and assembly during the cell division cycle.', 2020, 'eLife', 'PMC7591256', 2, NULL, NULL, 6),
(36, 'pubmed', '33151668', 'Analysis of Four Chitin-Active Lytic Polysaccharide Monooxygenases from  Reveals Functional Variation.', 'Lytic polysaccharide monooxygenases (LPMOs) are redox-active enzymes that cleave insoluble polysaccharides by an oxidative reaction. In the present study, we have characterized four recombinant putative chitin-active LPMOs from  (LPMO10B, -C, -D, and -F) and evaluated their potential in enhancing hydrolysis of α- and β-chitin by three families of 18 chitinases of , ChiA, -B, and -C. All four recombinant LPMO10s showed oxidative activity toward both α- and β-chitin but exhibited different abilities to promote the release of chitobiose from chitin by chitinases depending on both the chitinase and the chitin type. These effects were observed under conditions where the amount of LPMO in the reaction was not rate-limiting, showing that the observed functional differences relate to different abilities of the LPMOs to interact with and act on the substrate. These results show that four seemingly similar LPMOs carrying out the same reaction, cleavage of chitin by C1 oxidation, may have different roles in natural chitin conversion, which provides a rationale for the multiplicity of these enzymes within the same organism. The ability of the LPMOs to act on more natural substrates was demonstrated by showing that LPMO10B improved chitin solubilization in dried powdered shrimp shells.', 2020, 'Journal of agricultural and food chemistry', '', 4, NULL, NULL, 6),
(37, 'pubmed', '33936332', 'Preschool autism services: A tale of two Canadian provinces and the implications for policy.', 'For children with autism spectrum disorder (ASD), a lifelong neurodevelopmental condition, assessment and treatment services vary widely across Canada-potentially creating inequities. To highlight this, the Preschool Autism Treatment Impact study compared children\'s services and outcomes in New Brunswick (NB) and Nova Scotia (NS). Diagnostic practices, service delivery models, wait times, and treatment approaches differed, as did children\'s 1-year outcomes and costs for families and the public sector. Considering NB and NS strengths, we suggest that an optimal system would include: rapid access to high-quality diagnostic and intervention services; adherence to research-informed practice guidelines; interventions to enhance parents\' skills and self-efficacy; and measures to minimize financial burdens for families. Our results also suggest that provinces/territories must do more to ensure equitable access to effective services, including sharing and reporting on national comparative data. Canadian children with ASD deserve access to effective and consistent services, no matter where they live.', 2021, 'Paediatrics & child health', 'PMC8077201', 4, NULL, NULL, 6),
(38, 'pubmed', '33905962', 'RELAY, ramucirumab plus erlotinib versus placebo plus erlotinib in patients with untreated, EGFR-mutated, metastatic non-small cell lung cancer: Europe/United States subset analysis.', 'In EGFR mutation-positive NSCLC, dual EGFR/VEGFR inhibition compared to EGFR alone increases anti-tumor efficacy. The Phase III RELAY trial demonstrated superior PFS for ramucirumab plus erlotinib (RAM + ERL) over placebo plus erlotinib (PBO + ERL) (HR 0.591 [95% CI 0.461-0.760], p<0.0001). EGFR mutated NSCLC is less prevalent in Western versus Asian patients. This prespecified analysis evaluates efficacy and safety of RAM + ERL in EU and US patients enrolled in RELAY.Patients were randomized 1:1 to ERL + RAM (10 mg/kg IV) or PBO Q2W. Treatment continued until unacceptable toxicity or progressive disease. Patients were stratified by geographic region (East Asia vs \"other\" [EU/US and Canada (EU/US)]). Objectives included PFS, ORR, DoR, OS, PFS2, safety and biomarker analysis.EU/US subset included 113/449 (25.9%) patients (58 RAM + ERL, 55 PBO + ERL). RAM + ERL improved PFS (20.6 vs 10.9 months, HR 0.605 [95% CI: 0.362-1.010]). ORR and DCR were similar, but median DoR was longer with RAM + ERL (18.0 vs 10.1 months, HR 0.527 [95% CI: 0.296-0.939]). OS and PFS2 were immature at data cut-off (censoring rates 81.0-81.8% and 67.3-79.3%, respectively). Most commonly reported Grade ≥3 TEAE for RAM + ERL was hypertension (17 [29.8%]) and for PBO + ERL, dermatitis acneiform (5 [9.1%]).EU/US subset analysis showed improved efficacy outcomes for RAM + ERL and a safety profile consistent with the overall population. Ramucirumab is a safe and effective addition to standard-of-care EGFR-TKI for EGFR mutation-positive metastatic NSCLC.', 2021, 'Cancer treatment and research communications', '', 1, NULL, NULL, 6),
(39, 'pubmed', '33894335', 'Pemetrexed Plus Platinum With or Without Pembrolizumab in Patients With Previously Untreated Metastatic Nonsquamous NSCLC: Protocol-Specified Final Analysis From KEYNOTE-189.', 'In the phase 3 KEYNOTE-189 study (NCT02578680), pembrolizumab plus pemetrexed and platinum-based chemotherapy (pemetrexed-platinum) significantly improved overall survival (OS) and progression-free survival (PFS) in patients with previously untreated metastatic nonsquamous NSCLC versus placebo plus pemetrexed-platinum. We report updated efficacy outcomes from the protocol-specified final analysis, including outcomes in patients who crossed over to pembrolizumab from pemetrexed-platinum and in patients who completed 35 cycles (approximately 2 years) of pembrolizumab.Eligible patients were randomized 2:1 to pembrolizumab 200 mg (n=410) or placebo (n=206) every 3 weeks (for up to 35 cycles, approximately 2 years) plus 4 cycles of pemetrexed (500 mg/m) and investigators\' choice of cisplatin (75 mg/m) or carboplatin (AUC 5 mg/ml/min) every 3 weeks, followed by pemetrexed until progression. Patients assigned to placebo plus pemetrexed-platinum could crossover to pembrolizumab upon progression if eligibility criteria were met. The primary endpoints were OS and PFS.After median follow-up of 31.0 months, pembrolizumab plus pemetrexed-platinum continued to improve OS (hazard ratio [HR], 0.56; 95% CI, 0.46‒0.69), and PFS (HR, 0.49; 95% CI, 0.41‒0.59) over placebo plus pemetrexed-platinum regardless of PD-L1 expression. ORR (48.3% versus 19.9%) and time to second/subsequent tumor progression on next-line treatment (PFS2; HR, 0.50; 95% CI, 0.41‒0.61) were improved in patients who received pembrolizumab plus pemetrexed-platinum. 84 patients (40.8%) from the placebo plus pemetrexed-platinum group crossed over to pembrolizumab on-study. Grade 3‒5 adverse events occurred in 72.1% of patients receiving pembrolizumab plus pemetrexed-platinum and 66.8% of patients receiving placebo plus pemetrexed-platinum. 56 patients completed 35 cycles (approximately 2 years) of pembrolizumab; ORR was 85.7% and 53 (94.6%) were alive at data cutoff.Pembrolizumab plus pemetrexed-platinum continued to show improved efficacy outcomes compared with placebo plus pemetrexed-platinum, with manageable toxicity. These findings support first-line pembrolizumab plus pemetrexed-platinum in patients with previously untreated metastatic nonsquamous NSCLC.', 2021, 'Annals of oncology : official journal of the European Society for Medical Oncology', '', 1, NULL, NULL, 6),
(40, 'pubmed', '33775558', 'Patient-Reported Outcomes with Durvalumab With or Without Tremelimumab Versus Standard Chemotherapy as First-Line Treatment of Metastatic Non-Small-Cell Lung Cancer (MYSTIC).', 'The phase 3 MYSTIC study of durvalumab ± tremelimumab versus chemotherapy in metastatic non-small-cell lung cancer (NSCLC) patients with tumor cell (TC) programmed cell death ligand 1 (PD-L1) expression ≥ 25% did not meet its primary endpoints. We report patient-reported outcomes (PROs).Treatment-naïve patients were randomized (1:1:1) to durvalumab, durvalumab + tremelimumab, or chemotherapy. PROs were assessed in patients with PD-L1 TC ≥ 25% using EORTC Quality of Life Questionnaire (QLQ)-C30/LC13. Changes from baseline (12 months) for prespecified PRO endpoints of interest were analyzed by mixed model for repeated measures (MMRM) and time to deterioration (TTD) by stratified log-rank tests.There were no between-arm differences in baseline PROs (N = 488). Between-arm differences in MMRM-adjusted mean changes from baseline favored at least one of the durvalumab-containing arms versus chemotherapy (nominal P < .01) for C30 fatigue: durvalumab (-9.5; 99% confidence interval [CI], -17.0 to -2.0), durvalumab + tremelimumab (-11.7; 99% CI, -19.4 to -4.1); and for C30 appetite loss: durvalumab (-11.9; 99% CI, -21.1 to -2.7). TTD was longer with at least one of the durvalumab-containing arms versus chemotherapy (nominal P < .01) for global health status/quality of life: durvalumab (hazard ratio [HR] = 0.7; 95% CI, 0.5-1.0), durvalumab + tremelimumab (HR = 0.7; 95% CI, 0.5-1.0); and for physical functioning: durvalumab (HR = 0.6; 95% CI, 0.4-0.8), durvalumab + tremelimumab (HR = 0.6; 95% CI, 0.5-0.9) (both C30); as well as for the key symptoms of dyspnea: durvalumab (HR = 0.6; 95% CI, 0.5-0.9), durvalumab + tremelimumab (HR = 0.7; 95% CI, 0.5-1.0) (both LC13); fatigue: durvalumab + tremelimumab (HR = 0.6; 95% CI, 0.4-0.8); and appetite loss: durvalumab (HR = 0.5; 95% CI, 0.4-0.7), durvalumab + tremelimumab (HR = 0.7; 95% CI, 0.5-0.9) (both C30).Durvalumab ± tremelimumab versus chemotherapy reduced symptom burden and improved TTD of PROs, suggesting it had no detrimental effects on quality of life in metastatic NSCLC patients.', 2021, 'Clinical lung cancer', '', 3, NULL, NULL, 6),
(41, 'pubmed', '33737345', 'Severity of COVID-19 in patients with lung cancer: evidence and challenges.', 'Cancer patients are highly vulnerable to SARS-CoV-2 infections due to frequent contacts with the healthcare system, immunocompromised state from cancer or its therapies, supportive medications such as steroids and most importantly their advanced age and comorbidities. Patients with lung cancer have consistently been reported to suffer from an increased risk of death compared with other cancers. This is possibly due to the combination of specific pathophysiological aspects, including underlying pulmonary compromise due to smoking history and the increased specific pressures on respiratory healthcare services caused by the related pandemic. Rationally and safely treating patients with lung cancer during the pandemic has become a continuous challenge over the last year. Deciding whether to offer, modify, postpone or even cancel treatments for this particular patient\'s population has become the crucial recurrent dilemma for lung cancer professionals. Chemotherapy, immunotherapy and targeted agents represent distinct risks factors in the context of COVID-19 that should be balanced with the short-term and long-term consequences of delaying cancer care. Despite the rapid and persistent trend of the pandemic, declared by WHO on March 11, 2020, and still ongoing at the time of writing (January 2021), various efforts were made by oncologists worldwide to understand the impact of COVID-19 on patients with cancer. Adapted recommendations of our evidence-based practice guidelines have been developed for all stakeholders. Different small and large-scale registries, such as the COVID-19 and Cancer Consortium (CCC19) and Thoracic Cancers International COVID-19 Collaboration quickly collected data, supporting cancer care decisions under the challenging circumstance created by the COVID-19 pandemic. Several recommendations were developed as guidance for prioritizing the various aspects of lung cancer care in order to mitigate the adverse effects of the COVID-19 healthcare crisis, potentially reducing the morbidity and mortality of our patients from COVID-19 and from cancer. These recommendations helped inform decisions about treatment of established disease, continuation of clinical research and lung cancer screening. In this review, we summarize available evidence regarding the direct and indirect impact of the COVID-19 pandemic on lung cancer care and patients.', 2021, 'Journal for immunotherapy of cancer', 'PMC7978268', 2, NULL, NULL, 3),
(42, 'pubmed', '33730652', 'Safety of pemetrexed plus platinum in combination with pembrolizumab for metastatic nonsquamous non-small cell lung cancer: A post hoc analysis of KEYNOTE-189.', 'This post hoc analysis assessed the safety of pemetrexed and platinum in combination with pembrolizumab, including time-to-onset and time-to-resolution of all-cause any-grade and grade ≥3 adverse events (AEs) and renal AEs.Patient-level data from KEYNOTE-189 were analyzed in the all-subjects-as-treated population (pembrolizumab arm, n = 405; placebo arm, n = 202), and among patients who received ≥5 cycles of pemetrexed (pemetrexed/pembrolizumab/platinum arm, n = 310; pemetrexed/placebo/platinum arm, n = 135). All-cause AEs were selected based on ≥2 % incidence from previously reported KEYNOTE-189 data and included neutropenia, febrile neutropenia, anemia, thrombocytopenia, asthenia, fatigue, dyspnea, diarrhea, nausea, vomiting, pneumonitis, and renal events. Descriptive statistics summarized all-cause AEs. Medians and interquartile ranges were used to examine time-to-onset and time-to-resolution. The data cutoff was November 8, 2017.In both treatment arms, most non-hematologic (nausea, vomiting, diarrhea, and asthenia), and hematologic (febrile neutropenia, thrombocytopenia, and neutropenia) grade ≥3 AEs with ≥2 % incidence had a median time-to-onset within the first 4 cycles, and a median time-to-resolution of within 2 weeks from onset. A small number of AEs had longer median time-to-onset (pneumonitis and fatigue) and median time-to-resolution (pneumonitis, fatigue, acute kidney injury, and anemia). Among patients who received ≥5 cycles of pemetrexed, the incidence of any-grade renal toxicity in the pemetrexed/pembrolizumab/platinum arm was 2.3 % in Cycles 1-4, 4.8 % in Cycles 5-8, 2.6 % in Cycles 9-12, and 2.5 % in Cycles ≥13; and, in the pemetrexed/placebo/platinum arm, 0.7 % in Cycles 1-4, 1.5 % in Cycles 5-8, 1.3 % in Cycles 9-12, and 2.0 % in Cycles ≥13.Pemetrexed/pembrolizumab/platinum has manageable toxicity with longer duration of treatment. While the incidence of renal toxicity was slightly higher in the pembrolizumab combination as compared to pemetrexed, the incidence did not increase in later treatment cycles. These results support the safe use of the KEYNOTE-189 regimen in clinical practice.NCT02578680 (clinicaltrials.gov).', 2021, 'Lung cancer (Amsterdam, Netherlands)', '', 3, NULL, NULL, 6),
(43, 'pubmed', '33638808', 'Targeted Therapy Approaches for MET Abnormalities in Non-Small Cell Lung Cancer.', 'The tyrosine kinase receptor mesenchymal epithelial transition (MET) is a proto-oncogene that, through the activation of the MET-hepatocyte growth factor (HGF) pathway, encodes a variety of biological processes, including cell development, proliferation, invasion, and migration. Abnormal activation of the MET pathway, occurring through MET protein overexpression, and gene amplification or mutation, can contribute to oncogenesis and has been implicated in non-small cell lung cancer (NSCLC). Though it is associated with poor clinical outcome in NSCLCs, MET overexpression and its role as a therapeutic target remains somewhat elusive due to discrepancies in its occurrence. Unlike MET overexpression, MET amplification has demonstrated a stronger potential as a biomarker for therapeutic treatment, with clinical data indicating a compelling connection between a high MET gene copy number and a high response rate to targeted therapies. However, MET exon 14 skipping mutations, occurring in 3%-4 % of lung adenocarcinomas, are of particular interest, as tumors harboring these mutations have shown a significant response to MET inhibitors. Following the discovery of MET as a potential therapeutic target, extensive clinical studies have proposed three approaches to targeting MET: (1) MET tyrosine kinase inhibitors (TKIs), including crizotinib, capmatinib, tepotinib, savolinitib, and cabozantinib; (2) MET or HGF monoclonal antibodies, including emibetuzumab and ficlatuzumab; and (3) MET or HGF antibody drug conjugates, including telisotuzumab. Herein, we discuss the relevant clinical trials, particularly focusing on the efficacy as well as the safety and tolerability of the treatment options, in the promising field of targeting MET in NSCLC.', 2021, 'Drugs', '', 1, NULL, NULL, 6),
(44, 'pubmed', '33613137', 'A Conceptual Framework for Integrated Community Care.', 'The various health and social care services provided in a given local area (i.e., place-based) must not only deliver primary care in proximity to the population, but act upstream on the social determinants of health. This type of care, when provided in a holistic and integrated manner, aims to improve the physical and mental health-but also the well-being and social capital-of individuals, families, groups and communities. This type of approach is known as Integrated Community Care (ICC).This article was developed from a non-systematic review of scientific and grey literature followed by a qualitative analysis and researcher reflections on ICC.The article presents the core concepts of ICC, namely temporality, local area, health care, social care, proximity and integration. These concepts are unpacked and a conceptual diagram is set forth to put the dynamic links between the concepts into perspective.The purpose of the article is to provide a conceptual clarification of ICC. Three examples of practise (from Switzerland, Quebec [Canada] and Italy) are used as illustrations to provide a better understanding of ICC and to open up horizons.', 2021, 'International journal of integrated care', '', 4, NULL, NULL, 6),
(45, 'pubmed', '33430321', 'In Silico Identification of Potential Druggable Binding Sites on CIN85 SH3 Domain.', 'Protein-protein interactions (PPIs) play a pivotal role in the regulation of many physiological processes. The dysfunction of some PPIs interactions led to the alteration of different biological pathways causing various diseases including cancer. In this context, the inhibition of PPIs represents an attractive strategy for the design of new antitumoral agents. In recent years, computational approaches were successfully used to study the interactions between proteins, providing useful hints for the design of small molecules able to modulate PPIs. Targeting PPIs presents several challenges mainly due to the large and flat binding surface that lack the typical binding pockets of traditional drug targets. Despite these hurdles, substantial progress has been made in the last decade resulting in the identification of PPI modulators where some of them even found clinical use. This study focuses on MUC1-CIN85 PPI which is involved in the migration and invasion of cancer cells. Particularly, we investigated the presence of druggable binding sites on the CIN85 surface which provided new insights for the structure-based design of novel MUC1-CIN85 PPI inhibitors as anti-metastatic agents.', 2021, 'International journal of molecular sciences', 'PMC7825788', 4, NULL, NULL, 6),
(46, 'pubmed', '33368222', 'Assessment of Autism Symptoms From 6 to 18 Months of Age Using the Autism Observation Scale for Infants in a Prospective High-Risk Cohort.', 'The objectives were to characterize behavioral signs of autism spectrum disorder (ASD) in younger siblings of diagnosed children (high-risk; HR) and examine classification features of the Autism Observation Scale for Infants (AOSI). Participants (501 HR and 180 low-risk [LR]) were assessed between 6 and 18 months using the AOSI and at age 3 for ASD diagnoses. Total AOSI scores differentiated HR infants later diagnosed with ASD starting at 12 months. ROC analyses identified 12- and 18-month cutoff scores associated with 0.52 sensitivity and 0.74 specificity and 0.73 sensitivity and 0.65 specificity, respectively. Although classification accuracy does not support use as a standalone screen, the AOSI identifies features associated with ASD starting at 6 months and differentiates HR infants with ASD by 12 months.', 2020, 'Child development', '', 1, NULL, NULL, 1),
(47, 'pubmed', '33185664', 'Diversification of Fungal Chitinases and Their Functional Differentiation in Histoplasma capsulatum.', 'Chitinases enzymatically hydrolyze chitin, a highly abundant and utilized polymer of N-acetyl-glucosamine. Fungi are a rich source of chitinases; however, the phylogenetic and functional diversity of fungal chitinases are not well understood. We surveyed fungal chitinases from 373 publicly available genomes, characterized domain architecture, and conducted phylogenetic analyses of the glycoside hydrolase (GH18) domain. This large-scale analysis does not support the previous division of fungal chitinases into three major clades (A, B, C) as chitinases previously assigned to the \"C\" clade are not resolved as distinct from the \"A\" clade. Fungal chitinase diversity was partly shaped by horizontal gene transfer, and at least one clade of bacterial origin occurs among chitinases previously assigned to the \"B\" clade. Furthermore, chitin-binding domains (including the LysM domain) do not define specific clades, but instead are found more broadly across clades of chitinases. To gain insight into biological function diversity, we characterized all eight chitinases (Cts) from the thermally dimorphic fungus, Histoplasma capsulatum: six A clade, one B clade, and one formerly classified C clade chitinases. Expression analyses showed variable induction of chitinase genes in the presence of chitin but preferential expression of CTS3 in the mycelial stage. Activity assays demonstrated that Cts1 (B-I), Cts2 (A-V), Cts3 (A-V), Cts4 (A-V) have endochitinase activities with varying degrees of chitobiosidase function. Cts6 (C-I) has activity consistent with N-acetyl-glucosaminidase exochitinase function and Cts8 (A-II) has chitobiase activity. These results suggest chitinase activity is variable even within subclades and that predictions of functionality require more sophisticated models.', 2021, 'Molecular biology and evolution', 'PMC8042737', 1, NULL, NULL, 6),
(48, 'pubmed', '31451726', 'Proteomic enzyme analysis of the marine fungus Paradendryphiella salina reveals alginate lyase as a minimal adaptation strategy for brown algae degradation.', 'We set out to investigate the genetic adaptations of the marine fungus Paradendryphiella salina CBS112865 for degradation of brown macroalgae. We performed whole genome and transcriptome sequencing and shotgun proteomic analysis of the secretome of P. salina grown on three species of brown algae and under carbon limitation. Genome comparison with closely related terrestrial fungi revealed that P. salina had a similar but reduced CAZyme profile relative to the terrestrial fungi except for the presence of three putative alginate lyases from Polysaccharide Lyase (PL) family 7 and a putative PL8 with similarity to ascomycete chondroitin AC lyases. Phylogenetic and homology analyses place the PL7 sequences amongst mannuronic acid specific PL7 proteins from marine bacteria. Recombinant expression, purification and characterization of one of the PL7 genes confirmed the specificity. Proteomic analysis of the P. salina secretome when growing on brown algae, revealed the PL7 and PL8 enzymes abundantly secreted together with enzymes necessary for degradation of laminarin, cellulose, lipids and peptides. Our findings indicate that the basic CAZyme repertoire of saprobic and plant pathogenic ascomycetes, with the addition of PL7 alginate lyases, provide P. salina with sufficient enzymatic capabilities to degrade several types of brown algae polysaccharides.', 2019, 'Scientific reports', 'PMC6710412', 2, NULL, NULL, 6);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(49, 'pubmed', '33281771', 'α-Galactosidase and Sucrose-Kinase Relationships in a Bi-functional AgaSK Enzyme Produced by the Human Gut Symbiont  E1.', 'Plant α-galactosides belonging to the raffinose family oligosaccharides (RFOs) and considered as prebiotics, are commonly degraded by α-galactosidases produced by the human gut microbiome. In this environment, the  E1 symbiont-well-known for various benefit-is able to produce an original AgaSK bifunctional enzyme. This enzyme contains an hydrolytic α-galactosidase domain linked to an ATP dependent extra-domain, specifically involved in the α-galactoside hydrolysis and the phosphorylation of the glucose, respectively. However, the multi-modular relationships between both catalytic domains remained hitherto unexplored and has been, consequently, herein investigated. Biochemical characterization of heterologously expressed enzymes either in full-form or in separated domains revealed similar kinetic parameters. These results were supported by molecular modeling studies performed on the whole enzyme in complex with different RFOs. Further enzymatic analysis associated with kinetic degradation of various substrates followed by high pressure anionic exchange chromatography revealed that catalytic efficiency decreased as the number of D-galactosyl moieties branched onto the oligosaccharide increased, suggesting a preference of AgaSK for RFO\'s short chains. A wide prevalence and abundance study on a human metagenomic library showed a high prevalence of the AgaSK encoding gene whatever the health status of the individuals. Finally, phylogeny and synteny studies suggested a limited spread by horizontal transfer of the clusters\' containing AgaSK to only few species of Firmicutes, highlighting the importance of these undispersed tandem activities in the human gut microbiome.', 2020, 'Frontiers in microbiology', '', 1, NULL, NULL, 6),
(50, 'pubmed', '33004077', 'Investigating host-microbiome interactions by droplet based microfluidics.', 'Despite the importance of the mucosal interface between microbiota and the host in gut homeostasis, little is known about the mechanisms of bacterial gut colonization, involving foraging for glycans produced by epithelial cells. The slow pace of progress toward understanding the underlying molecular mechanisms is largely due to the lack of efficient discovery tools, especially those targeting the uncultured fraction of the microbiota.Here, we introduce an ultra-high-throughput metagenomic approach based on droplet microfluidics, to screen fosmid libraries. Thousands of bacterial genomes can be covered in 1 h of work, with less than ten micrograms of substrate. Applied to the screening of the mucosal microbiota for β-N-acetylgalactosaminidase activity, this approach allowed the identification of pathways involved in the degradation of human gangliosides and milk oligosaccharides, the structural homologs of intestinal mucin glycans. These pathways, whose prevalence is associated with inflammatory bowel diseases, could be the result of horizontal gene transfers with Bacteroides species. Such pathways represent novel targets to study the microbiota-host interactions in the context of inflammatory bowel diseases, in which the integrity of the mucosal barrier is impaired.By compartmentalizing experiments inside microfluidic droplets, this method speeds up and miniaturizes by several orders of magnitude the screening process compared to conventional approaches, to capture entire metabolic pathways from metagenomic libraries. The method is compatible with all types of (meta)genomic libraries, and employs a commercially available flow cytometer instead of a custom-made sorting system to detect intracellular or extracellular enzyme activities. This versatile and generic workflow will accelerate experimental exploration campaigns in functional metagenomics and holobiomics studies, to further decipher host-microbiota relationships. Video Abstract.', 2020, 'Microbiome', 'PMC7531118', 2, NULL, NULL, 6),
(52, 'pubmed', '33683866', 'Recapitulating the Binding Affinity of Nrf2 for KEAP1 in a Cyclic Heptapeptide, Guided by NMR, X-ray Crystallography, and Machine Learning.', 'Macrocycles, including macrocyclic peptides, have shown promise for targeting challenging protein-protein interactions (PPIs). One PPI of high interest is between Kelch-like ECH-Associated Protein-1 (KEAP1) and Nuclear Factor (Erythroid-derived 2)-like 2 (Nrf2). Guided by X-ray crystallography, NMR, modeling, and machine learning, we show that the full 20 nM binding affinity of Nrf2 for KEAP1 can be recapitulated in a cyclic 7-mer peptide, c[()-β-homoAla-DPETGE]. This compound was identified from the Nrf2-derived linear peptide GDEETGE ( = 4.3 μM) solely by optimizing the conformation of the cyclic compound, without changing any KEAP1 interacting residue. X-ray crystal structures were determined for each linear and cyclic peptide variant bound to KEAP1. Despite large variations in affinity, no obvious differences in the conformation of the peptide binding residues or in the interactions they made with KEAP1 were observed. However, analysis of the X-ray structures by machine learning showed that locations of strain in the bound ligand could be identified through patterns of subangstrom distortions from the geometry observed for unstrained linear peptides. We show that optimizing the cyclic peptide affinity was driven partly through conformational preorganization associated with a proline substitution at position 78 and with the geometry of the noninteracting residue Asp77 and partly by decreasing strain in the ETGE motif itself. This approach may have utility in dissecting the trade-off between conformational preorganization and strain in other ligand-receptor systems. We also identify a pair of conserved hydrophobic residues flanking the core DxETGE motif which play a conformational role in facilitating the high-affinity binding of Nrf2 to KEAP1.', 2021, 'Journal of the American Chemical Society', '', 1, NULL, NULL, 6),
(53, 'pubmed', '33617586', 'Preliminary validity and reliability evidence of the Brief Antisocial Behavior Scale (B-ABS) in young adults from four countries.', 'The present research built on the Self-Reported Delinquency interview and the Antisocial Behavior Scale to develop an updated brief instrument to measure antisocial behavior. College students (n = 3188, 67.75% women) from the USA, Argentina, the Netherlands and Spain completed an online survey. Analyses that combined approaches from the Classical Test Theory and Item Response Theory were conducted to select the items for the brief version. Findings suggested that a 13-item Brief Antisocial Behavior Scale (B-ABS) fulfilled the high-quality criteria: salient factor loadings, adequate discrimination, variability in response endorsement, adequate fit based on infit/outfit values, nondifferent item functioning across the four participating countries, and Cronbach\'s alpha and ordinal omega coefficients higher than .70. The B-ABS scores generally significantly correlated with personality scores, mental health and marijuana outcomes, showing criterion-related validity evidence. Our overall findings suggest that B-ABS adequately assesses antisocial behavior in young adults from different countries/cultures.', 2021, 'PloS one', 'PMC7899364', 1, NULL, NULL, 6),
(57, 'pubmed', '33252248', 'The Role of Violent Video Game Exposure, Personality, and Deviant Peers in Aggressive Behaviors Among Adolescents: A Two-Wave Longitudinal Study.', 'The impact of violent video game exposure (VVGE) on aggressive behaviors has been extensively explored, but still remains controversial. Although some studies have shown slight detrimental short-term effects of VVGE, other studies have failed to find any consequence. In addition, the existence of long-lasting effects on aggressiveness, or their impact on adolescents, are still not well established. One limitation of most of these studies is that they do not control for other important risk variables for aggressive behaviors, such as personality and deviant peers, nor have they investigated the possible moderation role of these risk factors in the link between VVGE and aggression. Therefore, the main aim was to examine the additive and interactive role of VVGE, personality, and deviant peers in adolescent aggressive behaviors cross-sectionally and longitudinally. Many regression analyses and a cross-lagged autoregressive model were carried out. At both waves, aggressive behavior was predicted by having deviant peers and specific personality traits, especially low agreeableness. VVGE also presented a slight but significant effect at both waves, but it became nonsignificant when controlling for other variables. No long-term effects on the relation between VVGE and aggressive behaviors were found. Some moderation effects were consistently found at both waves: when participants reported having more deviant peers, the effects of VVGE and low agreeableness on aggressive behaviors significantly increased. These findings suggest that multiple biopsychosocial variables and their complex interplay need to be examined to gain a better understanding of the origin and expression of aggressive behavior.', 2021, 'Cyberpsychology, behavior and social networking', '', 1, NULL, NULL, 6),
(58, 'pubmed', '33106654', 'FoxO maintains a genuine muscle stem-cell quiescent state until geriatric age.', 'Tissue regeneration declines with ageing but little is known about whether this arises from changes in stem-cell heterogeneity. Here, in homeostatic skeletal muscle, we identify two quiescent stem-cell states distinguished by relative CD34 expression: CD34, with stemness properties (genuine state), and CD34, committed to myogenic differentiation (primed state). The genuine-quiescent state is unexpectedly preserved into later life, succumbing only in extreme old age due to the acquisition of primed-state traits. Niche-derived IGF1-dependent Akt activation debilitates the genuine stem-cell state by imposing primed-state features via FoxO inhibition. Interventions to neutralize Akt and promote FoxO activity drive a primed-to-genuine state conversion, whereas FoxO inactivation deteriorates the genuine state at a young age, causing regenerative failure of muscle, as occurs in geriatric mice. These findings reveal transcriptional determinants of stem-cell heterogeneity that resist ageing more than previously anticipated and are only lost in extreme old age, with implications for the repair of geriatric muscle.', 2020, 'Nature cell biology', '', 1, NULL, NULL, 6);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(70, 'pubmed', '31539500', 'Interspecies Competition Impacts Targeted Manipulation of Human Gut Bacteria by Fiber-Derived Glycans.', 'Development of microbiota-directed foods (MDFs) that selectively increase the abundance of beneficial human gut microbes, and their expressed functions, requires knowledge of both the bioactive components of MDFs and the mechanisms underlying microbe-microbe interactions. Here, gnotobiotic mice were colonized with a defined consortium of human-gut-derived bacterial strains and fed different combinations of 34 food-grade fibers added to a representative low-fiber diet consumed in the United States. Bioactive carbohydrates in fiber preparations targeting particular Bacteroides species were identified using community-wide quantitative proteomic analyses of bacterial gene expression coupled with forward genetic screens. Deliberate manipulation of community membership combined with administration of retrievable artificial food particles, consisting of paramagnetic microscopic beads coated with dietary polysaccharides, disclosed the contributions of targeted species to fiber degradation. Our approach, including the use of bead-based biosensors, defines nutrient-harvesting strategies that underlie, as well as alleviate, competition between Bacteroides and control the selectivity of MDF components.', 2019, 'Cell', 'PMC6760872', 1, '<h1 class=\"content-title\">Interspecies competition impacts targeted manipulation of human gut bacteria by fiber-derived glycans</h1><br><span class=\"contrib-group fm-author\"><a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Patnode%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701600144368\" co-class=\"co-affbox\">Michael L. Patnode</a>,<sup>1,</sup><sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Beller%20ZW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701607682080\" co-class=\"co-affbox\">Zachary W. Beller</a>,<sup>1,</sup><sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Han%20ND%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701607680032\" co-class=\"co-affbox\">Nathan D. Han</a>,<sup>1,</sup><sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Cheng%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701607677424\" co-class=\"co-affbox\">Jiye Cheng</a>,<sup>1,</sup><sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Peters%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701599012096\" co-class=\"co-affbox\">Samantha L. Peters</a>,<sup>7</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Terrapon%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701599010112\" co-class=\"co-affbox\">Nicolas Terrapon</a>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Henrissat%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701599563312\" co-class=\"co-affbox\">Bernard Henrissat</a>,<sup>3,</sup><sup>4</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Gall%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701599560592\" co-class=\"co-affbox\">Sophie Le Gall</a>,<sup>5</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Saulnier%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701599558608\" co-class=\"co-affbox\">Luc Saulnier</a>,<sup>5</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Hayashi%20DK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701607253552\" co-class=\"co-affbox\">David K. Hayashi</a>,<sup>6</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Meynier%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701607251568\" co-class=\"co-affbox\">Alexandra Meynier</a>,<sup>6</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Vinoy%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701607249584\" co-class=\"co-affbox\">Sophie Vinoy</a>,<sup>6</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Giannone%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701603847776\" co-class=\"co-affbox\">Richard J. Giannone</a>,<sup>7</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Hettich%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701603845792\" co-class=\"co-affbox\">Robert L. Hettich</a>,<sup>7</sup> and  <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Gordon%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\" class=\"affpopup\" co-rid=\"_co_idm139701603843808\" co-class=\"co-affbox\">Jeffrey I. Gordon</a><sup>*,</sup><sup>1,</sup><sup>2</sup></span><span style=\"display:none\" class=\"contrib-group aff-tip\"><span id=\"_co_idm139701600144368\"><h3 class=\"no_margin\">Michael L. Patnode</h3><p><sup>1</sup>Edison Family Center for Genome Sciences and Systems Biology, Washington University School of Medicine St. Louis, MO 63110, USA</p><p><sup>2</sup>Center for Gut Microbiome and Nutrition Research, Washington University School of Medicine St. Louis, MO 63110, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Patnode%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Michael L. Patnode</a></span></span><span id=\"_co_idm139701607682080\"><h3 class=\"no_margin\">Zachary W. Beller</h3><p><sup>1</sup>Edison Family Center for Genome Sciences and Systems Biology, Washington University School of Medicine St. Louis, MO 63110, USA</p><p><sup>2</sup>Center for Gut Microbiome and Nutrition Research, Washington University School of Medicine St. Louis, MO 63110, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Beller%20ZW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Zachary W. Beller</a></span></span><span id=\"_co_idm139701607680032\"><h3 class=\"no_margin\">Nathan D. Han</h3><p><sup>1</sup>Edison Family Center for Genome Sciences and Systems Biology, Washington University School of Medicine St. Louis, MO 63110, USA</p><p><sup>2</sup>Center for Gut Microbiome and Nutrition Research, Washington University School of Medicine St. Louis, MO 63110, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Han%20ND%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Nathan D. Han</a></span></span><span id=\"_co_idm139701607677424\"><h3 class=\"no_margin\">Jiye Cheng</h3><p><sup>1</sup>Edison Family Center for Genome Sciences and Systems Biology, Washington University School of Medicine St. Louis, MO 63110, USA</p><p><sup>2</sup>Center for Gut Microbiome and Nutrition Research, Washington University School of Medicine St. Louis, MO 63110, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Cheng%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Jiye Cheng</a></span></span><span id=\"_co_idm139701599012096\"><h3 class=\"no_margin\">Samantha L. Peters</h3><p><sup>7</sup>Chemical Sciences Division, Oak Ridge National Laboratory, Oak Ridge, TN 37830, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Peters%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Samantha L. Peters</a></span></span><span id=\"_co_idm139701599010112\"><h3 class=\"no_margin\">Nicolas Terrapon</h3><p><sup>3</sup>Architecture et Fonction des Macromol&#x000e9;cules Biologiques, Centre National de la Recherche Scientifique and Aix-Marseille Universit&#x000e9; 13288 Marseille cedex 9, France</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Terrapon%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Nicolas Terrapon</a></span></span><span id=\"_co_idm139701599563312\"><h3 class=\"no_margin\">Bernard Henrissat</h3><p><sup>3</sup>Architecture et Fonction des Macromol&#x000e9;cules Biologiques, Centre National de la Recherche Scientifique and Aix-Marseille Universit&#x000e9; 13288 Marseille cedex 9, France</p><p><sup>4</sup>Department of Biological Sciences, King Abdulaziz University, Jeddah, Saudi Arabia</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Henrissat%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Bernard Henrissat</a></span></span><span id=\"_co_idm139701599560592\"><h3 class=\"no_margin\">Sophie Le Gall</h3><p><sup>5</sup>Institut National de la Recherche Agronomique, UR 1268 BIA (Biopolym&#x000e8;res, Interactions, Assemblages), 44316 Nantes cedex, France</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Gall%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Sophie Le Gall</a></span></span><span id=\"_co_idm139701599558608\"><h3 class=\"no_margin\">Luc Saulnier</h3><p><sup>5</sup>Institut National de la Recherche Agronomique, UR 1268 BIA (Biopolym&#x000e8;res, Interactions, Assemblages), 44316 Nantes cedex, France</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Saulnier%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Luc Saulnier</a></span></span><span id=\"_co_idm139701607253552\"><h3 class=\"no_margin\">David K. Hayashi</h3><p><sup>6</sup>Mondelez International, Deerfield, IL 60015, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Hayashi%20DK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">David K. Hayashi</a></span></span><span id=\"_co_idm139701607251568\"><h3 class=\"no_margin\">Alexandra Meynier</h3><p><sup>6</sup>Mondelez International, Deerfield, IL 60015, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Meynier%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Alexandra Meynier</a></span></span><span id=\"_co_idm139701607249584\"><h3 class=\"no_margin\">Sophie Vinoy</h3><p><sup>6</sup>Mondelez International, Deerfield, IL 60015, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Vinoy%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Sophie Vinoy</a></span></span><span id=\"_co_idm139701603847776\"><h3 class=\"no_margin\">Richard J. Giannone</h3><p><sup>7</sup>Chemical Sciences Division, Oak Ridge National Laboratory, Oak Ridge, TN 37830, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Giannone%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Richard J. Giannone</a></span></span><span id=\"_co_idm139701603845792\"><h3 class=\"no_margin\">Robert L. Hettich</h3><p><sup>7</sup>Chemical Sciences Division, Oak Ridge National Laboratory, Oak Ridge, TN 37830, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Hettich%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Robert L. Hettich</a></span></span><span id=\"_co_idm139701603843808\"><h3 class=\"no_margin\">Jeffrey I. Gordon</h3><p><sup>1</sup>Edison Family Center for Genome Sciences and Systems Biology, Washington University School of Medicine St. Louis, MO 63110, USA</p><p><sup>2</sup>Center for Gut Microbiome and Nutrition Research, Washington University School of Medicine St. Louis, MO 63110, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Gordon%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31539500\">Jeffrey I. Gordon</a></span></span></span></span><span class=\"fm-panel half_rhythm\"><span class=\"togglers<br><span id=\"idm139701596865568\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139701596865568title\">Footnotes</h2><!--back/fn-group--><span class=\"fm-sec half_rhythm small\"><p class=\"fn sec\" id=\"FN3\"><p id=\"P85\" class=\"p p-first\">Declaration of interests:</p><p id=\"P86\" class=\"p p-last\">J.I.G. is a co-founder of Matatu, Inc., a company characterizing the role of diet-by-microbiota interactions in animal health. Elements of this report are the subject of patent applications that are currently being submitted.</p></p><p class=\"fn sec\" id=\"FN4\"><p id=\"P87\" class=\"p p-first-last\"><strong>Publisher\'s Disclaimer: </strong>This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.</p></p></span></span><br><div class=\"ref-list-sec sec\" id=\"reference-list\"><ul class=\"back-ref-list\" style=\"list-style-type:decimal;\"><li id=\"R1\"><span class=\"mixed-citation\">Adey A, Morrison HG, Asan Xun, X., Kitzman JO, Turner EH, Stackhouse B, MacKenzie AP, Caruccio NC, Zhang X, et al. (2010). <span class=\"ref-title\">Rapid, low-input, low-bias construction of shotgun fragment libraries by high-density in vitro transposition</span>. <span class=\"ref-journal\">Genome Biol</span>\n<span class=\"ref-vol\">11</span>, R119. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3046479/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21143862\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Rapid,+low-input,+low-bias+construction+of+shotgun+fragment+libraries+by+high-density+in+vitro+transposition&amp;author=A+Adey&amp;author=HG+Morrison&amp;author=Xun,+X.+Asan&amp;author=JO+Kitzman&amp;author=EH+Turner&amp;volume=11&amp;publication_year=2010&amp;pages=R119&amp;pmid=21143862&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R2\"><span class=\"mixed-citation\">Atmodjo MA, Hao Z, and Mohnen D (2013). <span class=\"ref-title\">Evolving views of pectin biosynthesis</span>. <span class=\"ref-journal\">Annu Rev Plant Biol</span>\n<span class=\"ref-vol\">64</span>, 747&#x02013;779. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23451775\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Plant+Biol&amp;title=Evolving+views+of+pectin+biosynthesis&amp;author=MA+Atmodjo&amp;author=Z+Hao&amp;author=D+Mohnen&amp;volume=64&amp;publication_year=2013&amp;pages=747-779&amp;pmid=23451775&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R3\"><span class=\"mixed-citation\">Bokulich NA, Subramanian S, Faith JJ, Gevers D, Gordon JI, Knight R, Mills DA, and Caporaso JG (2013). <span class=\"ref-title\">Quality-filtering vastly improves diversity estimates from Illumina amplicon sequencing</span>. <span class=\"ref-journal\">Nature Methods</span>\n<span class=\"ref-vol\">10</span>, 57&#x02013;59. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3531572/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23202435\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature+Methods&amp;title=Quality-filtering+vastly+improves+diversity+estimates+from+Illumina+amplicon+sequencing&amp;author=NA+Bokulich&amp;author=S+Subramanian&amp;author=JJ+Faith&amp;author=D+Gevers&amp;author=JI+Gordon&amp;volume=10&amp;publication_year=2013&amp;pages=57-59&amp;pmid=23202435&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R4\"><span class=\"mixed-citation\">Buffetto F, Cornuault V, Rydahl MG, Ropartz D, Alvarado C, Echasserieau V, Le Gall S, Bouchet B, Tranquet O, Verhertbruggen Y, et al. (2015). <span class=\"ref-title\">The Deconstruction of Pectic Rhamnogalacturonan I Unmasks the Occurrence of a Novel Arabinogalactan Oligosaccharide Epitope</span>. <span class=\"ref-journal\">Plant Cell Physiol</span>\n<span class=\"ref-vol\">56</span>, 2181&#x02013;2196. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26384432\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Plant+Cell+Physiol&amp;title=The+Deconstruction+of+Pectic+Rhamnogalacturonan+I+Unmasks+the+Occurrence+of+a+Novel+Arabinogalactan+Oligosaccharide+Epitope&amp;author=F+Buffetto&amp;author=V+Cornuault&amp;author=MG+Rydahl&amp;author=D+Ropartz&amp;author=C+Alvarado&amp;volume=56&amp;publication_year=2015&amp;pages=2181-2196&amp;pmid=26384432&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R5\"><span class=\"mixed-citation\">Clarkson SM, Giannone RJ, Kridelbaugh DM, Elkins JG, Guss AM, and Michener JK (2017). <span class=\"ref-title\">Construction and Optimization of a Heterologous Pathway for Protocatechuate Catabolism in Escherichia coli Enables Bioconversion of Model Aromatic Compounds</span>. <span class=\"ref-journal\">Applied and Environmental Microbiol</span>\n<span class=\"ref-vol\">83</span>, e01313&#x02013;01317. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5583500/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28733280\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Applied+and+Environmental+Microbiol&amp;title=Construction+and+Optimization+of+a+Heterologous+Pathway+for+Protocatechuate+Catabolism+in+Escherichia+coli+Enables+Bioconversion+of+Model+Aromatic+Compounds&amp;author=SM+Clarkson&amp;author=RJ+Giannone&amp;author=DM+Kridelbaugh&amp;author=JG+Elkins&amp;author=AM+Guss&amp;volume=83&amp;publication_year=2017&amp;pages=e01313-01317&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R6\"><span class=\"mixed-citation\">Desai MS, Seekatz AM, Koropatkin NM, Kamada N, Hickey CA, Wolter M, Pudlo NA, Kitamoto S, Terrapon N, Muller A, et al. (2016). <span class=\"ref-title\">A Dietary Fiber-Deprived Gut Microbiota Degrades the Colonic Mucus Barrier and Enhances Pathogen Susceptibility</span>. <span class=\"ref-journal\">Cell</span>\n<span class=\"ref-vol\">167</span>, 1339&#x02013;1353 e1321. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5131798/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27863247\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A+Dietary+Fiber-Deprived+Gut+Microbiota+Degrades+the+Colonic+Mucus+Barrier+and+Enhances+Pathogen+Susceptibility&amp;author=MS+Desai&amp;author=AM+Seekatz&amp;author=NM+Koropatkin&amp;author=N+Kamada&amp;author=CA+Hickey&amp;volume=167&amp;publication_year=2016&amp;pages=1339-1353+e1321&amp;pmid=27863247&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R7\"><span class=\"mixed-citation\">Edgar RC (2010). <span class=\"ref-title\">Search and clustering orders of magnitude faster than BLAST</span>. <span class=\"ref-journal\">Bioinformatics</span>\n<span class=\"ref-vol\">26</span>, 2460&#x02013;2461. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20709691\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Search+and+clustering+orders+of+magnitude+faster+than+BLAST&amp;author=RC+Edgar&amp;volume=26&amp;publication_year=2010&amp;pages=2460-2461&amp;pmid=20709691&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R8\"><span class=\"mixed-citation\">Englyst HN, and Cummings JH (1988). <span class=\"ref-title\">Improved Method for Measurement of Dietary Fiber as Non-Starch Polysaccharides in Plant Foods</span>. <span class=\"ref-journal\">J Assoc Official Analytical Chemists</span>\n<span class=\"ref-vol\">71</span>, 808&#x02013;814. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n2458334\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Assoc+Official+Analytical+Chemists&amp;title=Improved+Method+for+Measurement+of+Dietary+Fiber+as+Non-Starch+Polysaccharides+in+Plant+Foods&amp;author=HN+Englyst&amp;author=JH+Cummings&amp;volume=71&amp;publication_year=1988&amp;pages=808-814&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R9\"><span class=\"mixed-citation\">Faith JJ, Ahern PP, Ridaura VK, Cheng J, and Gordon JI (2014). <span class=\"ref-title\">Identifying gut microbe-host phenotype relationships using combinatorial communities in gnotobiotic mice</span>. <span class=\"ref-journal\">Sci Transl Med</span>\n<span class=\"ref-vol\">6</span>, 220ra211. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3973144/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24452263\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Sci+Transl+Med&amp;title=Identifying+gut+microbe-host+phenotype+relationships+using+combinatorial+communities+in+gnotobiotic+mice&amp;author=JJ+Faith&amp;author=PP+Ahern&amp;author=VK+Ridaura&amp;author=J+Cheng&amp;author=JI+Gordon&amp;volume=6&amp;publication_year=2014&amp;pages=220ra211&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R10\"><span class=\"mixed-citation\">Faith JJ, Guruge JL, Charbonneau M, Subramanian S, Seedorf H, Goodman AL, Clemente JC, Knight R, Heath AC, Leibel RL, et al. (2013). <span class=\"ref-title\">The long-term stability of the human gut microbiota</span>. <span class=\"ref-journal\">Science</span>\n<span class=\"ref-vol\">341</span>, 1237439. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3791589/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23828941\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+long-term+stability+of+the+human+gut+microbiota&amp;author=JJ+Faith&amp;author=JL+Guruge&amp;author=M+Charbonneau&amp;author=S+Subramanian&amp;author=H+Seedorf&amp;volume=341&amp;publication_year=2013&amp;pages=1237439&amp;pmid=23828941&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R11\"><span class=\"mixed-citation\">Filisetti-Cozzi TM, and Carpita NC (1991). <span class=\"ref-title\">Measurement of uronic acids without interference from neutral sugars</span>. <span class=\"ref-journal\">Anal Biochem</span>\n<span class=\"ref-vol\">197</span>, 157&#x02013;162. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n1952059\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Anal+Biochem&amp;title=Measurement+of+uronic+acids+without+interference+from+neutral+sugars&amp;author=TM+Filisetti-Cozzi&amp;author=NC+Carpita&amp;volume=197&amp;publication_year=1991&amp;pages=157-162&amp;pmid=1952059&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R12\"><span class=\"mixed-citation\">Fragiadakis GK, Smits SA, Sonnenburg ED, Van Treuren W, Reid G, Knight R, Manjurano A, Changalucha J, Dominguez-Bello MG, Leach J, et al. (2018). <span class=\"ref-title\">Links between environment, diet, and the hunter-gatherer microbiome</span>. <span class=\"ref-journal\">Gut Microbes</span>, 1&#x02013;12. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC6546328/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n30118385\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Gut+Microbes&amp;title=Links+between+environment,+diet,+and+the+hunter-gatherer+microbiome&amp;author=GK+Fragiadakis&amp;author=SA+Smits&amp;author=ED+Sonnenburg&amp;author=W+Van+Treuren&amp;author=G+Reid&amp;publication_year=2018&amp;pages=1-12&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R13\"><span class=\"mixed-citation\">Glenwright AJ, Pothula KR, Bhamidimarri SP, Chorev DS, Basle A, Firbank SJ, Zheng H, Robinson CV, Winterhalter M, Kleinekathofer U, Bolam DN, et al. (2017). <span class=\"ref-title\">Structural basis for nutrient acquisition by dominant members of the human gut microbiota</span>. <span class=\"ref-journal\">Nature</span>\n<span class=\"ref-vol\">541</span>, 407&#x02013;411. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5497811/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28077872\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structural+basis+for+nutrient+acquisition+by+dominant+members+of+the+human+gut+microbiota&amp;author=AJ+Glenwright&amp;author=KR+Pothula&amp;author=SP+Bhamidimarri&amp;author=DS+Chorev&amp;author=A+Basle&amp;volume=541&amp;publication_year=2017&amp;pages=407-411&amp;pmid=28077872&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R14\"><span class=\"mixed-citation\">Goodman AL, McNulty NP, Zhao Y, Leip D, Mitra RD, Lozupone CA, Knight R, and Gordon JI (2009). <span class=\"ref-title\">Identifying genetic determinants needed to establish a human gut symbiont in its habitat</span>. <span class=\"ref-journal\">Cell Host &#x00026; Microbe</span>\n<span class=\"ref-vol\">6</span>, 279&#x02013;289. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2895552/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19748469\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cell+Host+&#x00026;+Microbe&amp;title=Identifying+genetic+determinants+needed+to+establish+a+human+gut+symbiont+in+its+habitat&amp;author=AL+Goodman&amp;author=NP+McNulty&amp;author=Y+Zhao&amp;author=D+Leip&amp;author=RD+Mitra&amp;volume=6&amp;publication_year=2009&amp;pages=279-289&amp;pmid=19748469&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R15\"><span class=\"mixed-citation\">Hibberd MC, Wu M, Rodionov DA, Li X, Cheng J, Griffin NW, Barratt MJ, Giannone RJ, Hettich RL, Osterman AL, et al. (2017). <span class=\"ref-title\">The effects of micronutrient deficiencies on bacterial species from the human gut microbiota</span>. <span class=\"ref-journal\">Sci Transl Med</span>\n<span class=\"ref-vol\">9</span>. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5524138/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28515336\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Sci+Transl+Med&amp;title=The+effects+of+micronutrient+deficiencies+on+bacterial+species+from+the+human+gut+microbiota&amp;author=MC+Hibberd&amp;author=M+Wu&amp;author=DA+Rodionov&amp;author=X+Li&amp;author=J+Cheng&amp;volume=9&amp;publication_year=2017&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R16\"><span class=\"mixed-citation\">Hibbing ME, Fuqua C, Parsek MR, and Peterson SB (2010). <span class=\"ref-title\">Bacterial competition: surviving and thriving in the microbial jungle</span>. <span class=\"ref-journal\">Nature Reviews Microbiology</span>\n<span class=\"ref-vol\">8</span>, 15&#x02013;25. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2879262/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19946288\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature+Reviews+Microbiology&amp;title=Bacterial+competition:+surviving+and+thriving+in+the+microbial+jungle&amp;author=ME+Hibbing&amp;author=C+Fuqua&amp;author=MR+Parsek&amp;author=SB+Peterson&amp;volume=8&amp;publication_year=2010&amp;pages=15-25&amp;pmid=19946288&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R17\"><span class=\"mixed-citation\">Heiss C, Klutts JS, Wang Z, Doering TL, and Azadi P (2009). <span class=\"ref-title\">The structure of Cryptococcus neoformans galactoxylomannan contains beta-D-glucuronic acid</span>. <span class=\"ref-journal\">Carbohydr Res</span>\n<span class=\"ref-vol\">344</span>, 915&#x02013;920. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2695399/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19345342\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Carbohydr+Res&amp;title=The+structure+of+Cryptococcus+neoformans+galactoxylomannan+contains+beta-D-glucuronic+acid&amp;author=C+Heiss&amp;author=JS+Klutts&amp;author=Z+Wang&amp;author=TL+Doering&amp;author=P+Azadi&amp;volume=344&amp;publication_year=2009&amp;pages=915-920&amp;pmid=19345342&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R18\"><span class=\"mixed-citation\">Kotarski SF, and Salyers AA (1984). <span class=\"ref-title\">Isolation and characterization of outer membranes of Bacteroides thetaiotaomicron grown on different carbohydrates</span>. <span class=\"ref-journal\">J Bacteriol</span>\n<span class=\"ref-vol\">158</span>, 102&#x02013;109. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC215385/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n6715279\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Isolation+and+characterization+of+outer+membranes+of+Bacteroides+thetaiotaomicron+grown+on+different+carbohydrates&amp;author=SF+Kotarski&amp;author=AA+Salyers&amp;volume=158&amp;publication_year=1984&amp;pages=102-109&amp;pmid=6715279&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R19\"><span class=\"mixed-citation\">Law CW, Chen Y, Shi W, and Smyth GK (2014). <span class=\"ref-title\">voom: Precision weights unlock linear model analysis tools for RNA-seq read counts</span>. <span class=\"ref-journal\">Genome Biol</span>\n<span class=\"ref-vol\">15</span>, R29. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4053721/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24485249\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=voom:+Precision+weights+unlock+linear+model+analysis+tools+for+RNA-seq+read+counts&amp;author=CW+Law&amp;author=Y+Chen&amp;author=W+Shi&amp;author=GK+Smyth&amp;volume=15&amp;publication_year=2014&amp;pages=R29&amp;pmid=24485249&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R20\"><span class=\"mixed-citation\">Levigne S, Thomas M, Ralet MC, Quemener BC, and Thibault JF (2002). <span class=\"ref-title\">Determination of the degrees of methylation and acetylation of pectins using a C18 column and internal standards</span>. <span class=\"ref-journal\">Food Hydrocolloids</span>\n<span class=\"ref-vol\">16</span>, 547&#x02013;550. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Food+Hydrocolloids&amp;title=Determination+of+the+degrees+of+methylation+and+acetylation+of+pectins+using+a+C18+column+and+internal+standards&amp;author=S+Levigne&amp;author=M+Thomas&amp;author=MC+Ralet&amp;author=BC+Quemener&amp;author=JF+Thibault&amp;volume=16&amp;publication_year=2002&amp;pages=547-550&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R21\"><span class=\"mixed-citation\">Luis AS, Briggs J, Zhang X, Farnell B, Ndeh D, Labourel A, Basle A, Cartmell A, Terrapon N, Stott K, et al. (2018). <span class=\"ref-title\">Dietary pectic glycans are degraded by coordinated enzyme pathways in human colonic Bacteroides</span>. <span class=\"ref-journal\">Nature Microbiol</span>\n<span class=\"ref-vol\">3</span>, 210&#x02013;219. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5784806/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n29255254\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature+Microbiol&amp;title=Dietary+pectic+glycans+are+degraded+by+coordinated+enzyme+pathways+in+human+colonic+Bacteroides&amp;author=AS+Luis&amp;author=J+Briggs&amp;author=X+Zhang&amp;author=B+Farnell&amp;author=D+Ndeh&amp;volume=3&amp;publication_year=2018&amp;pages=210-219&amp;pmid=29255254&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R22\"><span class=\"mixed-citation\">Lynch JB, and Sonnenburg JL (2012). <span class=\"ref-title\">Prioritization of a plant polysaccharide over a mucus carbohydrate is enforced by a Bacteroides hybrid two-component system</span>. <span class=\"ref-journal\">Mol Microbiol</span>\n<span class=\"ref-vol\">85</span>, 478&#x02013;491. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3404733/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22686399\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol+Microbiol&amp;title=Prioritization+of+a+plant+polysaccharide+over+a+mucus+carbohydrate+is+enforced+by+a+Bacteroides+hybrid+two-component+system&amp;author=JB+Lynch&amp;author=JL+Sonnenburg&amp;volume=85&amp;publication_year=2012&amp;pages=478-491&amp;pmid=22686399&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R23\"><span class=\"mixed-citation\">Ma ZQ, Dasari S, Chambers MC, Litton MD, Sobecki SM, Zimmerman LJ, Halvey PJ, Schilling B, Drake PM, Gibson BW, et al. (2009). <span class=\"ref-title\">IDPicker 2.0: Improved protein assembly with high discrimination peptide identification filtering</span>. <span class=\"ref-journal\">J Proteome Res</span>\n<span class=\"ref-vol\">8</span>, 3872&#x02013;3881. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2810655/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19522537\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Proteome+Res&amp;title=IDPicker+2.0:+Improved+protein+assembly+with+high+discrimination+peptide+identification+filtering&amp;author=ZQ+Ma&amp;author=S+Dasari&amp;author=MC+Chambers&amp;author=MD+Litton&amp;author=SM+Sobecki&amp;volume=8&amp;publication_year=2009&amp;pages=3872-3881&amp;pmid=19522537&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R24\"><span class=\"mixed-citation\">Martens EC, Lowe EC, Chiang H, Pudlo NA, Wu M, McNulty NP, Abbott DW, Henrissat B, Gilbert HJ, Bolam DN, et al. (2011). <span class=\"ref-title\">Recognition and degradation of plant cell wall polysaccharides by two human gut symbionts</span>. <span class=\"ref-journal\">PLoS Biology</span>\n<span class=\"ref-vol\">9</span>, e1001221. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3243724/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22205877\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+Biology&amp;title=Recognition+and+degradation+of+plant+cell+wall+polysaccharides+by+two+human+gut+symbionts&amp;author=EC+Martens&amp;author=EC+Lowe&amp;author=H+Chiang&amp;author=NA+Pudlo&amp;author=M+Wu&amp;volume=9&amp;publication_year=2011&amp;pages=e1001221&amp;pmid=22205877&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R25\"><span class=\"mixed-citation\">McDonald D, Price MN, Goodrich J, Nawrocki EP, DeSantis TZ, Probst A, Andersen GL, Knight R, and Hugenholtz P (2012). <span class=\"ref-title\">An improved Greengenes taxonomy with explicit ranks for ecological and evolutionary analyses of bacteria and archaea</span>. <span class=\"ref-journal\">ISME J</span>\n<span class=\"ref-vol\">6</span>, 610&#x02013;618. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3280142/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22134646\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=ISME+J&amp;title=An+improved+Greengenes+taxonomy+with+explicit+ranks+for+ecological+and+evolutionary+analyses+of+bacteria+and+archaea&amp;author=D+McDonald&amp;author=MN+Price&amp;author=J+Goodrich&amp;author=EP+Nawrocki&amp;author=TZ+DeSantis&amp;volume=6&amp;publication_year=2012&amp;pages=610-618&amp;pmid=22134646&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R26\"><span class=\"mixed-citation\">McNulty NP, Wu M, Erickson AR, Pan C, Erickson BK, Martens EC, Pudlo NA, Muegge BD, Henrissat B, Hettich RL, et al. (2013). <span class=\"ref-title\">Effects of diet on resource utilization by a model human gut microbiota containing Bacteroides cellulosilyticus WH2, a symbiont with an extensive glycobiome</span>. <span class=\"ref-journal\">PLoS Biology</span>\n<span class=\"ref-vol\">11</span>, e1001637. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3747994/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23976882\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+Biology&amp;title=Effects+of+diet+on+resource+utilization+by+a+model+human+gut+microbiota+containing+Bacteroides+cellulosilyticus+WH2,+a+symbiont+with+an+extensive+glycobiome&amp;author=NP+McNulty&amp;author=M+Wu&amp;author=AR+Erickson&amp;author=C+Pan&amp;author=BK+Erickson&amp;volume=11&amp;publication_year=2013&amp;pages=e1001637&amp;pmid=23976882&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R27\"><span class=\"mixed-citation\">Menni C, Jackson MA, Pallister T, Steves CJ, Spector TD, and Valdes AM (2017). <span class=\"ref-title\">Gut microbiome diversity and high-fibre intake are related to lower long-term weight gain</span>. <span class=\"ref-journal\">Int J Obes</span>\n<span class=\"ref-vol\">41</span>, 1099&#x02013;1105. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5500185/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28286339\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int+J+Obes&amp;title=Gut+microbiome+diversity+and+high-fibre+intake+are+related+to+lower+long-term+weight+gain&amp;author=C+Menni&amp;author=MA+Jackson&amp;author=T+Pallister&amp;author=CJ+Steves&amp;author=TD+Spector&amp;volume=41&amp;publication_year=2017&amp;pages=1099-1105&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R28\"><span class=\"mixed-citation\">Pattathil S, Avci U, Miller JS, and Hahn MG (2012). <span class=\"ref-title\">Immunological approaches to plant cell wall and biomass characterization: Glycome Profiling</span>. <span class=\"ref-journal\">Methods Mol Biol</span>\n<span class=\"ref-vol\">908</span>, 61&#x02013;72. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22843389\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Methods+Mol+Biol&amp;title=Immunological+approaches+to+plant+cell+wall+and+biomass+characterization:+Glycome+Profiling&amp;author=S+Pattathil&amp;author=U+Avci&amp;author=JS+Miller&amp;author=MG+Hahn&amp;volume=908&amp;publication_year=2012&amp;pages=61-72&amp;pmid=22843389&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R29\"><span class=\"mixed-citation\">Pettolino FA, Walsh C, Fincher GB, and Bacic A (2012). <span class=\"ref-title\">Determining the polysaccharide composition of plant cell walls</span>. <span class=\"ref-journal\">Nature Protocols</span>\n<span class=\"ref-vol\">7</span>, 1590&#x02013;1607. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22864200\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature+Protocols&amp;title=Determining+the+polysaccharide+composition+of+plant+cell+walls&amp;author=FA+Pettolino&amp;author=C+Walsh&amp;author=GB+Fincher&amp;author=A+Bacic&amp;volume=7&amp;publication_year=2012&amp;pages=1590-1607&amp;pmid=22864200&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R30\"><span class=\"mixed-citation\">Porter NT, and Martens EC (2017). <span class=\"ref-title\">The critical roles of polysaccharides in gut microbial ecology and physiology</span>. <span class=\"ref-journal\">Annu Rev Microbiol</span>\n<span class=\"ref-vol\">71</span>, 349&#x02013;369. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28657886\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Microbiol&amp;title=The+critical+roles+of+polysaccharides+in+gut+microbial+ecology+and+physiology&amp;author=NT+Porter&amp;author=EC+Martens&amp;volume=71&amp;publication_year=2017&amp;pages=349-369&amp;pmid=28657886&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R31\"><span class=\"mixed-citation\">Ridaura VK, Faith JJ, Rey FE, Cheng J, Duncan AE, Kau AL, Griffin NW, Lombard V, Henrissat B, Bain JR, et al. (2013). <span class=\"ref-title\">Gut microbiota from twins discordant for obesity modulate metabolism in mice</span>. <span class=\"ref-journal\">Science</span>\n<span class=\"ref-vol\">341</span>, 1241214. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3829625/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24009397\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Gut+microbiota+from+twins+discordant+for+obesity+modulate+metabolism+in+mice&amp;author=VK+Ridaura&amp;author=JJ+Faith&amp;author=FE+Rey&amp;author=J+Cheng&amp;author=AE+Duncan&amp;volume=341&amp;publication_year=2013&amp;pages=1241214&amp;pmid=24009397&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R32\"><span class=\"mixed-citation\">Rogowski A, Briggs JA, Mortimer JC, Tryfona T, Terrapon N, Lowe EC, Basle A, Morland C, Day AM, Zheng H, et al. (2015). <span class=\"ref-title\">Glycan complexity dictates microbial resource allocation in the large intestine</span>. <span class=\"ref-journal\">Nature Commun</span>\n<span class=\"ref-vol\">6</span>, 7481. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4491172/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26112186\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature+Commun&amp;title=Glycan+complexity+dictates+microbial+resource+allocation+in+the+large+intestine&amp;author=A+Rogowski&amp;author=JA+Briggs&amp;author=JC+Mortimer&amp;author=T+Tryfona&amp;author=N+Terrapon&amp;volume=6&amp;publication_year=2015&amp;pages=7481&amp;pmid=26112186&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R33\"><span class=\"mixed-citation\">Santander J, Martin T, Loh A, Pohlenz C, Gatlin DM 3rd, and Curtiss R 3rd (2013). <span class=\"ref-title\">Mechanisms of intrinsic resistance to antimicrobial peptides of Edwardsiella ictaluri and its influence on fish gut inflammation and virulence</span>. <span class=\"ref-journal\">Microbiology</span>\n<span class=\"ref-vol\">159</span>, 1471&#x02013;1486. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4085987/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23676433\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Mechanisms+of+intrinsic+resistance+to+antimicrobial+peptides+of+Edwardsiella+ictaluri+and+its+influence+on+fish+gut+inflammation+and+virulence&amp;author=J+Santander&amp;author=T+Martin&amp;author=A+Loh&amp;author=C+Pohlenz&amp;author=DM+Gatlin&amp;volume=159&amp;publication_year=2013&amp;pages=1471-1486&amp;pmid=23676433&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R34\"><span class=\"mixed-citation\">Schwalm ND 3rd, Townsend GE 2nd, and Groisman EA (2016). <span class=\"ref-title\">Multiple signals govern utilization of a polysaccharide in the gut bacterium Bacteroides thetaiotaomicron</span>. <span class=\"ref-journal\">MBio</span>\n<span class=\"ref-vol\">7</span>. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5061871/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27729509\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=MBio&amp;title=Multiple+signals+govern+utilization+of+a+polysaccharide+in+the+gut+bacterium+Bacteroides+thetaiotaomicron&amp;author=ND+Schwalm&amp;author=GE+Townsend&amp;author=EA+Groisman&amp;volume=7&amp;publication_year=2016&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R35\"><span class=\"mixed-citation\">Shepherd ES, DeLoache WC, Pruss KM, Whitaker WR, and Sonnenburg JL (2018). <span class=\"ref-title\">An exclusive metabolic niche enables strain engraftment in the gut microbiota</span>. <span class=\"ref-journal\">Nature</span>\n<span class=\"ref-vol\">557</span>, 434&#x02013;438. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC6126907/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n29743671\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+exclusive+metabolic+niche+enables+strain+engraftment+in+the+gut+microbiota&amp;author=ES+Shepherd&amp;author=WC+DeLoache&amp;author=KM+Pruss&amp;author=WR+Whitaker&amp;author=JL+Sonnenburg&amp;volume=557&amp;publication_year=2018&amp;pages=434-438&amp;pmid=29743671&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R36\"><span class=\"mixed-citation\">Smyth GK (2004). <span class=\"ref-title\">Linear models and empirical bayes methods for assessing differential expression in microarray experiments</span>. <span class=\"ref-journal\">Stat Appl Genet Mol Biol</span>\n<span class=\"ref-vol\">3</span>, Article3. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n16646809\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Stat+Appl+Genet+Mol+Biol&amp;title=Linear+models+and+empirical+bayes+methods+for+assessing+differential+expression+in+microarray+experiments&amp;author=GK+Smyth&amp;volume=3&amp;publication_year=2004&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R37\"><span class=\"mixed-citation\">Sonnenburg ED, Smits SA, Tikhonov M, Higginbottom SK, Wingreen NS, and Sonnenburg JL (2016). <span class=\"ref-title\">Diet-induced extinctions in the gut microbiota compound over generations</span>. <span class=\"ref-journal\">Nature</span>\n<span class=\"ref-vol\">529</span>, 212&#x02013;215. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4850918/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26762459\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Diet-induced+extinctions+in+the+gut+microbiota+compound+over+generations&amp;author=ED+Sonnenburg&amp;author=SA+Smits&amp;author=M+Tikhonov&amp;author=SK+Higginbottom&amp;author=NS+Wingreen&amp;volume=529&amp;publication_year=2016&amp;pages=212-215&amp;pmid=26762459&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R38\"><span class=\"mixed-citation\">Sonnenburg ED, Zheng H, Joglekar P, Higginbottom SK, Firbank SJ, Bolam DN, and Sonnenburg JL (2010). <span class=\"ref-title\">Specificity of polysaccharide use in intestinal bacteroides species determines diet-induced microbiota alterations</span>. <span class=\"ref-journal\">Cell</span>\n<span class=\"ref-vol\">141</span>, 1241&#x02013;1252. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2900928/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20603004\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Specificity+of+polysaccharide+use+in+intestinal+bacteroides+species+determines+diet-induced+microbiota+alterations&amp;author=ED+Sonnenburg&amp;author=H+Zheng&amp;author=P+Joglekar&amp;author=SK+Higginbottom&amp;author=SJ+Firbank&amp;volume=141&amp;publication_year=2010&amp;pages=1241-1252&amp;pmid=20603004&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R39\"><span class=\"mixed-citation\">Tabb DL, Fernando CG, and Chambers MC (2007). <span class=\"ref-title\">MyriMatch: highly accurate tandem mass spectral peptide identification by multivariate hypergeometric analysis</span>. <span class=\"ref-journal\">J Proteome Res</span>\n<span class=\"ref-vol\">6</span>, 654&#x02013;661. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2525619/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17269722\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Proteome+Res&amp;title=MyriMatch:+highly+accurate+tandem+mass+spectral+peptide+identification+by+multivariate+hypergeometric+analysis&amp;author=DL+Tabb&amp;author=CG+Fernando&amp;author=MC+Chambers&amp;volume=6&amp;publication_year=2007&amp;pages=654-661&amp;pmid=17269722&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R40\"><span class=\"mixed-citation\">Tauzin AS, Laville E, Xiao Y, Nouaille S, Le Bourgeois P, Heux S, Portais JC, Monsan P, Martens EC, Potocki-Veronese G, et al. (2016). <span class=\"ref-title\">Functional characterization of a gene locus from an uncultured gut Bacteroides conferring xylo-oligosaccharides utilization to Escherichia coli</span>. <span class=\"ref-journal\">Mol Microbiol</span>\n<span class=\"ref-vol\">102</span>, 579&#x02013;592. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27573446\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol+Microbiol&amp;title=Functional+characterization+of+a+gene+locus+from+an+uncultured+gut+Bacteroides+conferring+xylo-oligosaccharides+utilization+to+Escherichia+coli&amp;author=AS+Tauzin&amp;author=E+Laville&amp;author=Y+Xiao&amp;author=S+Nouaille&amp;author=P+Le+Bourgeois&amp;volume=102&amp;publication_year=2016&amp;pages=579-592&amp;pmid=27573446&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R41\"><span class=\"mixed-citation\">Terrapon N, Lombard V, Drula E, Lapebie P, Al-Masaudi S, Gilbert HJ, and Henrissat B (2018). <span class=\"ref-title\">PULDB: the expanded database of Polysaccharide Utilization Loci</span>. <span class=\"ref-journal\">Nucleic Acids Res</span>\n<span class=\"ref-vol\">46</span>, D677&#x02013;D683. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5753385/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n29088389\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=PULDB:+the+expanded+database+of+Polysaccharide+Utilization+Loci&amp;author=N+Terrapon&amp;author=V+Lombard&amp;author=E+Drula&amp;author=P+Lapebie&amp;author=S+Al-Masaudi&amp;volume=46&amp;publication_year=2018&amp;pages=D677-D683&amp;pmid=29088389&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R42\"><span class=\"mixed-citation\">Thibault JF (1979). <span class=\"ref-title\">Automatisation du dosage des substances pectiques par la m&#x000e9;thode au m&#x000e9;tahydroxydiph&#x000e9;nyle</span>. <span class=\"ref-journal\">Lebensml Wiss Technol</span>\n<span class=\"ref-vol\">12</span>, 247&#x02013;251. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Lebensml+Wiss+Technol&amp;title=Automatisation+du+dosage+des+substances+pectiques+par+la+m&#x000e9;thode+au+m&#x000e9;tahydroxydiph&#x000e9;nyle&amp;author=JF+Thibault&amp;volume=12&amp;publication_year=1979&amp;pages=247-251&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R43\"><span class=\"mixed-citation\">Ting L, Cowley MJ, Hoon SL, Guilhaus M, Raftery MJ, and Cavicchioli R (2009). <span class=\"ref-title\">Normalization and statistical analysis of quantitative proteomics data generated by metabolic labeling</span>. <span class=\"ref-journal\">Mol Cell Proteomics</span>\n<span class=\"ref-vol\">8</span>, 2227&#x02013;2242. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2758752/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19605365\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Proteomics&amp;title=Normalization+and+statistical+analysis+of+quantitative+proteomics+data+generated+by+metabolic+labeling&amp;author=L+Ting&amp;author=MJ+Cowley&amp;author=SL+Hoon&amp;author=M+Guilhaus&amp;author=MJ+Raftery&amp;volume=8&amp;publication_year=2009&amp;pages=2227-2242&amp;pmid=19605365&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R44\"><span class=\"mixed-citation\">Wu M, McNulty NP, Rodionov DA, Khoroshkin MS, Griffin NW, Cheng J, Latreille P, Kerstetter RA, Terrapon N, Henrissat B, et al. (2015). <span class=\"ref-title\">Genetic determinants of in vivo fitness and diet responsiveness in multiple human gut Bacteroides</span>. <span class=\"ref-journal\">Science</span>\n<span class=\"ref-vol\">350</span>, aac5992. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4608238/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26430127\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genetic+determinants+of+in+vivo+fitness+and+diet+responsiveness+in+multiple+human+gut+Bacteroides&amp;author=M+Wu&amp;author=NP+McNulty&amp;author=DA+Rodionov&amp;author=MS+Khoroshkin&amp;author=NW+Griffin&amp;volume=350&amp;publication_year=2015&amp;pages=aac5992&amp;pmid=26430127&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R45\"><span class=\"mixed-citation\">Zeevi D, Korem T, Zmora N, Israeli D, Rothschild D, Weinberger A, Ben-Yacov O, Lador D, Avnit-Sagi T, Lotan-Pompan M, et al. (2015). <span class=\"ref-title\">Personalized Nutrition by Prediction of Glycemic Responses</span>. <span class=\"ref-journal\">Cell</span>\n<span class=\"ref-vol\">163</span>, 1079&#x02013;1094. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26590418\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Personalized+Nutrition+by+Prediction+of+Glycemic+Responses&amp;author=D+Zeevi&amp;author=T+Korem&amp;author=N+Zmora&amp;author=D+Israeli&amp;author=D+Rothschild&amp;volume=163&amp;publication_year=2015&amp;pages=1079-1094&amp;pmid=26590418&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"R46\"><span class=\"mixed-citation\">Zhao L, Zhang F, Ding X, Wu G, Lam YY, Wang X, Fu H, Xue X, Lu C, Ma J, et al. (2018). <span class=\"ref-title\">Gut bacteria selectively promoted by dietary fibers alleviate type 2 diabetes</span>. <span class=\"ref-journal\">Science</span>\n<span class=\"ref-vol\">359</span>, 1151&#x02013;1156. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n29590046\" ref=\"reftype=pubmed&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Gut+bacteria+selectively+promoted+by+dietary+fibers+alleviate+type+2+diabetes&amp;author=L+Zhao&amp;author=F+Zhang&amp;author=X+Ding&amp;author=G+Wu&amp;author=YY+Lam&amp;volume=359&amp;publication_year=2018&amp;pages=1151-1156&amp;pmid=29590046&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=6760872&amp;issue-id=342668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li></ul></div></div><br>', NULL, 1);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(71, 'pubmed', '31539500', 'Interspecies Competition Impacts Targeted Manipulation of Human Gut Bacteria by Fiber-Derived Glycans.', 'Development of microbiota-directed foods (MDFs) that selectively increase the abundance of beneficial human gut microbes, and their expressed functions, requires knowledge of both the bioactive components of MDFs and the mechanisms underlying microbe-microbe interactions. Here, gnotobiotic mice were colonized with a defined consortium of human-gut-derived bacterial strains and fed different combinations of 34 food-grade fibers added to a representative low-fiber diet consumed in the United States. Bioactive carbohydrates in fiber preparations targeting particular Bacteroides species were identified using community-wide quantitative proteomic analyses of bacterial gene expression coupled with forward genetic screens. Deliberate manipulation of community membership combined with administration of retrievable artificial food particles, consisting of paramagnetic microscopic beads coated with dietary polysaccharides, disclosed the contributions of targeted species to fiber degradation. Our approach, including the use of bead-based biosensors, defines nutrient-harvesting strategies that underlie, as well as alleviate, competition between Bacteroides and control the selectivity of MDF components.', 2019, 'Cell', 'PMC6760872', 1, NULL, NULL, 2),
(73, 'pubmed', '28928724', 'The Maturing Development of Gut Microbiota in Commercial Piglets during the Weaning Transition.', 'Early weaned piglets are vulnerable to diarrhea because of weaning stress and immaturity of intestinal tract. Compelling evidence suggests that gut microbiota is vital to host health. However, it is not well understood on the composition and succession of piglet gut microbiota during the weaning transition. In our two trials, total 17 commercial piglets were studied in a pig farm in Jiangxi Province, China. Fresh feces were collected for four times (10 days before weaned, weaned day, 10 days after weaned, 21 days after weaned) by rectal massage. Fecal bacterial composition was assessed by 16S rRNA gene V3-V4 regions sequencing by Illumina Miseq platform. The results showed that the gut microbiota of piglets shifted quickly after weaned and reached relatively stable level in 10 days after weaned. The alpha diversity increased significantly with the age of piglets. The microbiota of suckling piglets was mainly represented by , and . This pattern contrasted with that of  XIVa, and , which were major representative genera after weaned. In summary, we delineated the development of piglet gut microbiota during the weaning transition. This study helps us understand the maturing development of gut microbiota in commercial piglets.', 2017, 'Frontiers in microbiology', '', 1, NULL, NULL, 2);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(74, 'pubmed', '28567446', 'Meta-analysis To Define a Core Microbiota in the Swine Gut.', 'The swine gut microbiota encompasses a large and diverse population of bacteria that play a significant role in pig health. As such, a number of recent studies have utilized high-throughput sequencing of the 16S rRNA gene to characterize the composition and structure of the swine gut microbiota, often in response to dietary feed additives. It is important to determine which factors shape the composition of the gut microbiota among multiple studies and if certain bacteria are always present in the gut microbiota of swine, independently of study variables such as country of origin and experimental design. Therefore, we performed a meta-analysis using 20 publically available data sets from high-throughput 16S rRNA gene sequence studies of the swine gut microbiota. Next to the \"study\" itself, the gastrointestinal (GI) tract section that was sampled had the greatest effect on the composition and structure of the swine gut microbiota ( = 0.0001). Technical variation among studies, particularly the 16S rRNA gene hypervariable region sequenced, also significantly affected the composition of the swine gut microbiota ( = 0.0001). Despite this, numerous commonalities were discovered. Among fecal samples, the genera , , , and  and the RC9 gut group were found in 99% of all fecal samples. Additionally, , , , , , , the RC9 gut group, and  were shared by >90% of all GI samples, suggesting a so-called \"core\" microbiota for commercial swine worldwide.  The results of this meta-analysis demonstrate that \"study\" and GI sample location are the most significant factors in shaping the swine gut microbiota. However, in comparisons of results from different studies, some biological factors may be obscured by technical variation among studies. Nonetheless, there are some bacterial taxa that appear to form a core microbiota within the swine GI tract regardless of country of origin, diet, age, or breed. Thus, these results provide the framework for future studies to manipulate the swine gut microbiota for potential health benefits.', 0000, 'mSystems', 'PMC5443231', 1, '<h1 class=\"content-title\">Meta-analysis To Define a Core Microbiota in the Swine Gut</h1><br><span class=\"contrib-group fm-author\"><a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Holman%20DB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28567446\" class=\"affpopup\" co-rid=\"_co_idm140162580283744\" co-class=\"co-affbox\">Devin B. Holman</a>,<sup>a</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Brunelle%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28567446\" class=\"affpopup\" co-rid=\"_co_idm140162579596336\" co-class=\"co-affbox\">Brian W. Brunelle</a>,<sup>a</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Trachsel%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28567446\" class=\"affpopup\" co-rid=\"_co_idm140162585884096\" co-class=\"co-affbox\">Julian Trachsel</a>,<sup>a,</sup><sup>b</sup> and  <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Allen%20HK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28567446\" class=\"affpopup\" co-rid=\"_co_idm140162569374576\" co-class=\"co-affbox\">Heather K. Allen</a><sup><img src=\"https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/corrauth.gif\"\r\n alt=\"corresponding author\" /></sup><sup>a</sup></span><span style=\"display:none\" class=\"contrib-group aff-tip\"><span id=\"_co_idm140162580283744\"><h3 class=\"no_margin\">Devin B. Holman</h3><p><sup>a</sup>Food Safety and Enteric Pathogens Research Unit, National Animal Disease Center, Agricultural Research Service, United States Department of Agriculture, Ames, Iowa, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Holman%20DB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28567446\">Devin B. Holman</a></span></span><span id=\"_co_idm140162579596336\"><h3 class=\"no_margin\">Brian W. Brunelle</h3><p><sup>a</sup>Food Safety and Enteric Pathogens Research Unit, National Animal Disease Center, Agricultural Research Service, United States Department of Agriculture, Ames, Iowa, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Brunelle%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28567446\">Brian W. Brunelle</a></span></span><span id=\"_co_idm140162585884096\"><h3 class=\"no_margin\">Julian Trachsel</h3><p><sup>a</sup>Food Safety and Enteric Pathogens Research Unit, National Animal Disease Center, Agricultural Research Service, United States Department of Agriculture, Ames, Iowa, USA</p><p><sup>b</sup>Interdepartmental Microbiology Graduate Program, Iowa State University, Ames, Iowa, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Trachsel%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28567446\">Julian Trachsel</a></span></span><span id=\"_co_idm140162569374576\"><h3 class=\"no_margin\">Heather K. Allen</h3><p><sup>a</sup>Food Safety and Enteric Pathogens Research Unit, National Animal Disease Center, Agricultural Research Service, United States Department of Agriculture, Ames, Iowa, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Allen%20HK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28567446\">Heather K. Allen</a></span></span></span></span><span class=\"contrib-group half_rhythm fm-editor\">Holly Bik, <span class=\"fm-role\">Editor</span><span><span class=\"fm-affl\">Holly Bik, </span><span class=\"fm-affl\" lang=\"en\" id=\"idm140162583072544\"> University of California, Riverside;</span></span></span><span class=\"fm-panel half_rhythm\"><span class=\"togglers<br><span id=\"idm140162571806064\"><strong>Citation</strong> Holman DB, Brunelle BW, Trachsel J, Allen HK. 2017. Meta-analysis to define a core microbiota in the swine gut. mSystems 2:e00004-17. <a href=\"https://doi.org/10.1128/mSystems.00004-17\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://doi.org/10.1128/mSystems.00004-17</a>.</span></span><span class=\"fm-article-notes fm-panel hide half_rhythm\" id=\"idm140162577190064_an\" style=\"display:none\"><span class=\"fm-pubdate half_rhythm\">Received 2017 Jan 4; Accepted 2017 Apr 21.</span></span><span class=\"permissions fm-panel half_rhythm hide\" id=\"idm140162577190064_cpl\" style=\"display:none\"><span class=\"fm-copyright half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al.</span><span class=\"license half_rhythm\">This is an open-access article distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span><span id=\"pmclinksbox\" class=\"links-box whole_rhythm\"><span class=\"fm-panel\"><span>This article has been <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/citedby/\">cited by</a> other articles in PMC.</span></span></span></span><span class=\"sec\"></span><span id=\"ass-data\" class=\"tsec fm-sec whole_rhythm\" data-section=\"Featured_PMC_Datacitation\"><h2 class=\"nomenu\">Associated Data</h2><dl data-count=\"5\" class=\"box-data-suppmats whole_rhythm no_bottom_margin\"><dt> \r\n</dt><dd id=\"data-suppmats\"  \r\n><span class=\"half_rhythm\"><span class=\"caption half_rhythm no_bottom_margin\"><strong>TABLE&#x000a0;S1&#x000a0;</strong></span><p>The percentage of samples from each specific gastrointestinal location that had at least one 16S rRNA gene sequence from each of the individual genera identified. Genera are listed in descending order of overall relative abundance. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103st4.pdf\">TABLE&#x000a0;S1, PDF file, 1.1 MB</a>.</p><span class=\"caption half_rhythm no_bottom_margin\"><span class=\"small permissions\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al. <span class=\"license\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span></span><span class=\"half_rhythm\"><span class=\"caption half_rhythm no_bottom_margin\"><strong>FIG&#x000a0;S1&#x000a0;</strong></span><p>Principal-coordinate analysis plots of Bray-Curtis dissimilarities by study. The numbers in parentheses in the figure legend refer to the number of samples included for each study. The percentages of variation explained by the principal coordinates are indicated on the axes. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103sf1.tif\">FIG&#x000a0;S1, TIF file, 2.1 MB</a>.</p><span class=\"caption half_rhythm no_bottom_margin\"><span class=\"small permissions\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al. <span class=\"license\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span></span><span class=\"half_rhythm\"><span class=\"caption half_rhythm no_bottom_margin\"><strong>FIG&#x000a0;S2&#x000a0;</strong></span><p>Principal-coordinate analysis plot of the weighted UniFrac distances for all samples with associated metadata for the age of the pigs used in the study. Samples are colored by pig age in weeks. The numbers in parentheses for the figure legend refer to the number of samples at each week. The percentages of variation explained by the principal coordinates are indicated on the axes. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103sf2.tif\">FIG&#x000a0;S2, TIF file, 2.5 MB</a>.</p><span class=\"caption half_rhythm no_bottom_margin\"><span class=\"small permissions\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al. <span class=\"license\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span></span><span class=\"half_rhythm\"><span class=\"caption half_rhythm no_bottom_margin\"><strong>TABLE&#x000a0;S2&#x000a0;</strong></span><p>The percentage of fecal samples grouped by specific 16S rRNA gene hypervariable region sequenced that had at least one 16S rRNA gene sequence from each of the individual genera identified. Genera are listed in descending order of overall relative abundance. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103st5.pdf\">TABLE&#x000a0;S2, PDF file, 0.5 MB</a>.</p><span class=\"caption half_rhythm no_bottom_margin\"><span class=\"small permissions\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al. <span class=\"license\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span></span><span class=\"half_rhythm\"><span class=\"caption half_rhythm no_bottom_margin\"><strong>FIG&#x000a0;S3&#x000a0;</strong></span><p>Principal-coordinate analysis plots of weighted UniFrac distances (A), unweighted UniFrac distances (B), and Bray-Curtis dissimilarities (C) for studies investigating the effect of an antimicrobial on the swine fecal microbiota. Samples are colored by study and antimicrobial treatment. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103sf3.tif\">FIG&#x000a0;S3, TIF file, 1.7 MB</a>.</p><span class=\"caption half_rhythm no_bottom_margin\"><span class=\"small permissions\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al. <span class=\"license\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span></span></dd></dl></span><span id=\"idm140162573535312\" lang=\"en\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140162573535312title\" style=\"text-transform:uppercase\">ABSTRACT</h2><!--article-meta--><span><p id=\"__p3\" class=\"p p-first\">The swine gut microbiota encompasses a large and diverse population of bacteria that play a significant role in pig health. As such, a number of recent studies have utilized high-throughput sequencing of the 16S rRNA gene to characterize the composition and structure of the swine gut microbiota, often in response to dietary feed additives. It is important to determine which factors shape the composition of the gut microbiota among multiple studies and if certain bacteria are always present in the gut microbiota of swine, independently of study variables such as country of origin and experimental design. Therefore, we performed a meta-analysis using 20 publically available data sets from high-throughput 16S rRNA gene sequence studies of the swine gut microbiota. Next to the &#x0201c;study&#x0201d; itself, the gastrointestinal (GI) tract section that was sampled had the greatest effect on the composition and structure of the swine gut microbiota (<em>P</em> = 0.0001). Technical variation among studies, particularly the 16S rRNA gene hypervariable region sequenced, also significantly affected the composition of the swine gut microbiota (<em>P</em> = 0.0001). Despite this, numerous commonalities were discovered. Among fecal samples, the genera <em>Prevotella</em>, <em>Clostridium</em>, <em>Alloprevotella</em>, and <em>Ruminococcus</em> and the RC9 gut group were found in 99% of all fecal samples. Additionally, <em>Clostridium</em>, <em>Blautia</em>, <em>Lactobacillus</em>, <em>Prevotella</em>, <em>Ruminococcus</em>, <em>Roseburia</em>, the RC9 gut group, and <em>Subdoligranulum</em> were shared by &#x0003e;90% of all GI samples, suggesting a so-called &#x0201c;core&#x0201d; microbiota for commercial swine worldwide.</p><p id=\"__p4\" class=\"p p-last\"><strong>IMPORTANCE</strong> The results of this meta-analysis demonstrate that &#x0201c;study&#x0201d; and GI sample location are the most significant factors in shaping the swine gut microbiota. However, in comparisons of results from different studies, some biological factors may be obscured by technical variation among studies. Nonetheless, there are some bacterial taxa that appear to form a core microbiota within the swine GI tract regardless of country of origin, diet, age, or breed. Thus, these results provide the framework for future studies to manipulate the swine gut microbiota for potential health benefits.</p></span><span class=\"sec\"><strong class=\"kwd-title\">KEYWORDS: </strong><span class=\"kwd-text\">16S rRNA gene, bacteria, gut microbiome, gut microbiota, livestock, meta-analysis, microbial ecology, swine</span></span></span><span id=\"s0\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s0title\" style=\"text-transform:uppercase\">INTRODUCTION</h2><p id=\"__p5\" class=\"p p-first\">The mammalian gut microbiota is populated by an estimated 100 trillion bacterial cells that provide health benefits to the host through the production of short-chain fatty acids and vitamins from otherwise nondigestible food components, inhibition and prevention of colonization by pathogens, and development and maintenance of the host immune system (<a href=\"#B1\" rid=\"B1\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776876\">1</a>,<a href=\"#B2\" rid=\"B2\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776895\">&#x02013;</a><a href=\"#B3\" rid=\"B3\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776882\">3</a>). These properties also make the gut microbiota an attractive target for dietary interventions to improve the health and production of food-producing animals, including swine. Therefore, the characterization of the gut microbiota of swine has become an active area of research in recent years as the adaptation and availability of high-throughput sequencing methods have continued to expand.</p><p id=\"__p6\">The majority of swine gut microbiota studies to date have focused on sequencing specific hypervariable regions within the 16S rRNA gene. The myriad of studies that have sequenced the 16S rRNA gene can be readily aggregated and compared in a meta-analysis that may reveal patterns among individual studies working on similar sample types. Recent meta-analyses of the avian gut microbiota (<a href=\"#B4\" rid=\"B4\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776884\">4</a>) and the indoor microbiota (<a href=\"#B5\" rid=\"B5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776874\">5</a>) have reported that study-level variables, compared to biological factors such as sample type, have the greatest effect on the microbiota. In swine, researchers have examined the effect of a number of different feed additives on the swine gut microbiota, including antimicrobial agents (<a href=\"#B6\" rid=\"B6\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776902\">6</a>,<a href=\"#B7\" rid=\"B7\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776869\">&#x02013;</a><a href=\"#B12\" rid=\"B12\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776894\">12</a>), amylose/amylopectin (<a href=\"#B13\" rid=\"B13\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776845\">13</a>), calcium-phosphorus (<a href=\"#B14\" rid=\"B14\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776844\">14</a>), distillers&#x02019; dried grains with solubles (<a href=\"#B15\" rid=\"B15\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776901\">15</a>), and resistant starch (<a href=\"#B16\" rid=\"B16\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776879\">16</a>). The effect of challenge with <em>Escherichia coli</em> (<a href=\"#B17\" rid=\"B17\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776846\">17</a>) and <em>Salmonella</em> (<a href=\"#B18\" rid=\"B18\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776873\">18</a>) on the swine gut microbiota has also been investigated. All of those studies found various degrees of change in the swine gut microbiota as a result of these treatments. Aggregating and reanalyzing the data could reveal whether alterations observed in individual studies are still detectable when combined with data from similar studies or whether they would be lost due to technical variation between studies.</p><p id=\"__p7\" class=\"p p-last\">Our objectives were to characterize the swine gut microbiota using 16S rRNA gene high-throughput sequencing data from gastrointestinal (GI) samples from 20 publically available studies and to determine if a core gut microbiota, defined here as being present in &#x02265;90% of samples, exists in swine. We evaluated how technical variables, such as study design, country of origin, and hypervariable region sequenced, affect the observed swine gut microbiota.</p></span><span id=\"s1\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s1title\" style=\"text-transform:uppercase\">RESULTS AND DISCUSSION</h2><span id=\"s1.1\" class=\"sec sec-first\"><p></p><h3 id=\"s1.1title\" class=\"inline\">Meta-analysis characteristics. </h3><p id=\"__p8\" class=\"p p-first\">A total of 25,157,796 quality-filtered 16S rRNA gene sequences from 939 swine gastrointestinal samples were available for meta-analysis (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab1/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"tab1\" rid-ob=\"ob-tab1\" co-legend-rid=\"\"><span>Table&#x000a0;1</span></a>). Overall, 18,743,484 (74.5%) of these sequences were at least 97% similar to a sequence in the SILVA database. These sequences represented 25,182 bacterial operational taxonomic units (OTUs), 35 phyla, and 887 genera prior to random subsampling. It was necessary to randomly subsample to a level of 1,000 sequences per sample to retain as many samples as possible; however, as few as 100 sequences per sample have been shown to be sufficient when large differences between sample groups are present (<a href=\"#B19\" rid=\"B19\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776872\">19</a>). Not surprisingly, the majority of sequences available for meta-analysis were derived from swine feces, as these samples are the easiest to obtain from the GI tract and allow repeated sampling over time. Other regions of the swine GI tract, such as the ileal and colonic mucosa and digesta, have also been well represented among publically available sequences (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig1\" rid-ob=\"ob-fig1\" co-legend-rid=\"lgnd_fig1\"><span>Fig.&#x000a0;1</span></a>). Conversely, the number of samples from the duodenum and jejunum (digesta and mucosa) was relatively small (<em>n</em> &#x02264; 5 samples) and belonged to a single study. Therefore, these samples were not included in the linear discriminant analysis (LDA) effect size (LEfSe) described below. Sequences were also available from three different continents and 10 countries, although approximately half of these were from studies conducted in the United States. The age of the pigs used in each study ranged from~3&#x000a0;weeks (preweaning piglets) to ~24&#x000a0;weeks (slaughter age).</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"tab1\"><h3>TABLE&#x000a0;1&#x000a0;</h3><!--caption a7--><span class=\"caption\"><p id=\"__p9\">Details of studies included in the meta-analysis</p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140162571680672\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><thead><tr><th rowspan=\"1\" colspan=\"1\">Reference</th><th rowspan=\"1\" colspan=\"1\">No. of samples</th><th rowspan=\"1\" colspan=\"1\">Hypervariable region(s)</th><th rowspan=\"1\" colspan=\"1\">Sequencing platform</th><th rowspan=\"1\" colspan=\"1\">Avg % assigned sequences per sample</th><th rowspan=\"1\" colspan=\"1\">DNA extraction method</th><th rowspan=\"1\" colspan=\"1\">Sample type(s)</th><th rowspan=\"1\" colspan=\"1\">Study observation(s)</th><th rowspan=\"1\" colspan=\"1\">Age of pigs at sampling</th><th rowspan=\"1\" colspan=\"1\">Country(ies) of origin</th><th rowspan=\"1\" colspan=\"1\">Data availability (accession no.)</th></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">J. Trachsel, B. E. Bass, and H. K. Allen, unpublished</td><td rowspan=\"1\" colspan=\"1\">52</td><td rowspan=\"1\" colspan=\"1\">V1 to V3</td><td rowspan=\"1\" colspan=\"1\">Illumina MiSeq</td><td rowspan=\"1\" colspan=\"1\">76.2%</td><td rowspan=\"1\" colspan=\"1\">PowerSoil-htp 96-well soil DNA isolation kit</td><td rowspan=\"1\" colspan=\"1\">Ileum digesta, fecal</td><td rowspan=\"1\" colspan=\"1\">Subset of samples taken before heat stress treatment</td><td rowspan=\"1\" colspan=\"1\">7&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">United States</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA335425\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA335425</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B17\" rid=\"B17\" class=\" bibr popnode\">17</a></td><td rowspan=\"1\" colspan=\"1\">24</td><td rowspan=\"1\" colspan=\"1\">V1 to V2</td><td rowspan=\"1\" colspan=\"1\">Illumina MiSeq</td><td rowspan=\"1\" colspan=\"1\">88.9%</td><td rowspan=\"1\" colspan=\"1\">NA<sup>b</sup></td><td rowspan=\"1\" colspan=\"1\">Ileum digesta</td><td rowspan=\"1\" colspan=\"1\">Pigs challenged with <em>E.&#x000a0;coli</em> F4+ and fed IgG had a lower relative abundance of <em>Enterococcaceae</em> and <em>Streptococcaceae</em> and increased ileal diversity</td><td rowspan=\"1\" colspan=\"1\">6&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">Denmark</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA302730\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA302730</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B15\" rid=\"B15\" class=\" bibr popnode\">15</a><sup>a</sup></td><td rowspan=\"1\" colspan=\"1\">18</td><td rowspan=\"1\" colspan=\"1\">V4</td><td rowspan=\"1\" colspan=\"1\">Illumina MiSeq</td><td rowspan=\"1\" colspan=\"1\">75.4%</td><td rowspan=\"1\" colspan=\"1\">QIAamp DNA stool minikit</td><td rowspan=\"1\" colspan=\"1\">Colon digesta</td><td rowspan=\"1\" colspan=\"1\">Pigs fed DDGS<sup>c</sup> had a different microbial community structure</td><td rowspan=\"1\" colspan=\"1\">9&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">United States</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJEB11368/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJEB11368</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B45\" rid=\"B45\" class=\" bibr popnode\">45</a><sup>a</sup></td><td rowspan=\"1\" colspan=\"1\">40</td><td rowspan=\"1\" colspan=\"1\">V3 to V4</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">74.1%</td><td rowspan=\"1\" colspan=\"1\">QIAamp DNA stool minikit</td><td rowspan=\"1\" colspan=\"1\">Colon digesta, fecal</td><td rowspan=\"1\" colspan=\"1\">No differences in the gut microbiota between conventionally and organically raised pigs</td><td rowspan=\"1\" colspan=\"1\">24&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">Denmark, France, Italy, Sweden</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA266576\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA266576</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B66\" rid=\"B66\" class=\" bibr popnode\">66</a></td><td rowspan=\"1\" colspan=\"1\">39</td><td rowspan=\"1\" colspan=\"1\">V4</td><td rowspan=\"1\" colspan=\"1\">Illumina MiSeq</td><td rowspan=\"1\" colspan=\"1\">85.2%</td><td rowspan=\"1\" colspan=\"1\">PowerFecal DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Fecal</td><td rowspan=\"1\" colspan=\"1\">Difference between the fecal microbiota of black and white breed pigs</td><td rowspan=\"1\" colspan=\"1\">3&#x02013;12&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">South Korea</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA252973\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA252973</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B67\" rid=\"B67\" class=\" bibr popnode\">67</a></td><td rowspan=\"1\" colspan=\"1\">53</td><td rowspan=\"1\" colspan=\"1\">V1 to V3</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">83.3%</td><td rowspan=\"1\" colspan=\"1\">QIAamp DNA stool minikit</td><td rowspan=\"1\" colspan=\"1\">Colon, ileal digesta</td><td rowspan=\"1\" colspan=\"1\">Sow-reared and formula-fed piglets had a different gut microbiota</td><td rowspan=\"1\" colspan=\"1\">3&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">United States</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA266806\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA266806</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B13\" rid=\"B13\" class=\" bibr popnode\">13</a></td><td rowspan=\"1\" colspan=\"1\">20</td><td rowspan=\"1\" colspan=\"1\">V4</td><td rowspan=\"1\" colspan=\"1\">Illumina MiSeq</td><td rowspan=\"1\" colspan=\"1\">75.8%</td><td rowspan=\"1\" colspan=\"1\">Custom</td><td rowspan=\"1\" colspan=\"1\">Ileum, cecum digesta</td><td rowspan=\"1\" colspan=\"1\">No effect of high or low amylose/amylopectin ratio diets on the gut microbiota</td><td rowspan=\"1\" colspan=\"1\">21&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">China</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA281781\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA281781</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B68\" rid=\"B68\" class=\" bibr popnode\">68</a></td><td rowspan=\"1\" colspan=\"1\">19</td><td rowspan=\"1\" colspan=\"1\">V4</td><td rowspan=\"1\" colspan=\"1\">Illumina MiSeq</td><td rowspan=\"1\" colspan=\"1\">81.3%</td><td rowspan=\"1\" colspan=\"1\">PowerFecal DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Fecal</td><td rowspan=\"1\" colspan=\"1\">Differences in fecal microbiota by production phase</td><td rowspan=\"1\" colspan=\"1\">Piglets, growers, finishers, sows</td><td rowspan=\"1\" colspan=\"1\">South Korea</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA267233\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA267233</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B16\" rid=\"B16\" class=\" bibr popnode\">16</a></td><td rowspan=\"1\" colspan=\"1\">14</td><td rowspan=\"1\" colspan=\"1\">V3 to V5</td><td rowspan=\"1\" colspan=\"1\">Illumina MiSeq</td><td rowspan=\"1\" colspan=\"1\">73.2%</td><td rowspan=\"1\" colspan=\"1\">PowerSoil DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Cecum digesta</td><td rowspan=\"1\" colspan=\"1\">Minor differences in the cecal microbiota of pigs fed 70% enzymatically modified cornstarch or control starch</td><td rowspan=\"1\" colspan=\"1\">14&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">Austria</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJEB8696/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJEB8696</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B11\" rid=\"B11\" class=\" bibr popnode\">11</a></td><td rowspan=\"1\" colspan=\"1\">62</td><td rowspan=\"1\" colspan=\"1\">V4</td><td rowspan=\"1\" colspan=\"1\">Illumina MiSeq</td><td rowspan=\"1\" colspan=\"1\">80.6%</td><td rowspan=\"1\" colspan=\"1\">PowerFecal DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Fecal</td><td rowspan=\"1\" colspan=\"1\">No effect of diet with 0.2% chlortetracycline, sulfathiazole, and penicillin on the fecal microbiota</td><td rowspan=\"1\" colspan=\"1\">3&#x02013;12&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">South Korea</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA257009\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA257009</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B10\" rid=\"B10\" class=\" bibr popnode\">10</a></td><td rowspan=\"1\" colspan=\"1\">94</td><td rowspan=\"1\" colspan=\"1\">V4</td><td rowspan=\"1\" colspan=\"1\">Illumina MiSeq</td><td rowspan=\"1\" colspan=\"1\">51.9%</td><td rowspan=\"1\" colspan=\"1\">ZR fecal DNA MiniPrep (Zymo Research)</td><td rowspan=\"1\" colspan=\"1\">Fecal</td><td rowspan=\"1\" colspan=\"1\">Subtherapeutic tylosin, but not chlortetracycline, altered the fecal microbiota</td><td rowspan=\"1\" colspan=\"1\">3&#x02013;19&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">Canada</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA244928\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA244928</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B12\" rid=\"B12\" class=\" bibr popnode\">12</a></td><td rowspan=\"1\" colspan=\"1\">44</td><td rowspan=\"1\" colspan=\"1\">V1 to V3</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">87.6%</td><td rowspan=\"1\" colspan=\"1\">PowerSoil DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Ileal mucosa</td><td rowspan=\"1\" colspan=\"1\">No effect of in-feed chlortetracycline or high-complexity diets on the ileal mucosa microbiota</td><td rowspan=\"1\" colspan=\"1\">6&#x02013;11&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">Canada</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA257705\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA257705</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B7\" rid=\"B7\" class=\" bibr popnode\">7</a></td><td rowspan=\"1\" colspan=\"1\">80</td><td rowspan=\"1\" colspan=\"1\">V1 to V3</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">62.8%</td><td rowspan=\"1\" colspan=\"1\">Ultra-Clean soil DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Gastric mucosa, duodenum, jejunum, ileum cecum, colon digesta and mucosa, fecal</td><td rowspan=\"1\" colspan=\"1\">Reduced diversity in the ileum versus colon, ASP250-altered gut microbiota</td><td rowspan=\"1\" colspan=\"1\">15&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">United States</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA72355\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA72355</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B8\" rid=\"B8\" class=\" bibr popnode\">8</a></td><td rowspan=\"1\" colspan=\"1\">101</td><td rowspan=\"1\" colspan=\"1\">V1 to V3</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">54.1%</td><td rowspan=\"1\" colspan=\"1\">PowerSoil DNA isolation kit</td><td rowspan=\"1\" colspan=\"1\">Fecal</td><td rowspan=\"1\" colspan=\"1\">Carbadox resulted in early but not late alterations of the fecal microbiota</td><td rowspan=\"1\" colspan=\"1\">6&#x02013;15&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">United States</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/237795\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA237795</a>; <a href=\"https://www.ncbi.nlm.nih.gov/biosample/SAMN02645017\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">SAMN02645017</a> to <a href=\"https://www.ncbi.nlm.nih.gov/biosample/SAMN02645066\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">SAMN02645066</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B69\" rid=\"B69\" class=\" bibr popnode\">69</a></td><td rowspan=\"1\" colspan=\"1\">23</td><td rowspan=\"1\" colspan=\"1\">V1 to V3</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">47.3%</td><td rowspan=\"1\" colspan=\"1\">Ultra-Clean Fecal DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Fecal</td><td rowspan=\"1\" colspan=\"1\">The fecal microbiota of Landrace pigs differed that of from pigs of the Duroc breed</td><td rowspan=\"1\" colspan=\"1\">15&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">South Korea</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA263439\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA263439</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B14\" rid=\"B14\" class=\" bibr popnode\">14</a></td><td rowspan=\"1\" colspan=\"1\">92</td><td rowspan=\"1\" colspan=\"1\">V1 to V2</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">83.2%</td><td rowspan=\"1\" colspan=\"1\">PowerSoil DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Gastric, ileum, colon mucosa</td><td rowspan=\"1\" colspan=\"1\">Gut microbiota affected by Ca-P content and diet type</td><td rowspan=\"1\" colspan=\"1\">7&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">Austria</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJEB1576/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJEB1576</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B70\" rid=\"B70\" class=\" bibr popnode\">70</a></td><td rowspan=\"1\" colspan=\"1\">10</td><td rowspan=\"1\" colspan=\"1\">V3</td><td rowspan=\"1\" colspan=\"1\">Illumina HiSeq 2000</td><td rowspan=\"1\" colspan=\"1\">88.8%</td><td rowspan=\"1\" colspan=\"1\">QIAamp DNA stool minikit (Qiagen)</td><td rowspan=\"1\" colspan=\"1\">Cecum digesta</td><td rowspan=\"1\" colspan=\"1\">Fostering of piglets changes cecal microbiota</td><td rowspan=\"1\" colspan=\"1\">3&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">China</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA214736\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA214736</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B18\" rid=\"B18\" class=\" bibr popnode\">18</a></td><td rowspan=\"1\" colspan=\"1\">59</td><td rowspan=\"1\" colspan=\"1\">V1 to V3</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">80.7%</td><td rowspan=\"1\" colspan=\"1\">PowerSoil DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Fecal</td><td rowspan=\"1\" colspan=\"1\">Fecal microbiota differed between high- and low-<em>Salmonella</em>-shedder pigs</td><td rowspan=\"1\" colspan=\"1\">8&#x02013;11&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">United States</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA169497\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA169497</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B9\" rid=\"B9\" class=\" bibr popnode\">9</a></td><td rowspan=\"1\" colspan=\"1\">12</td><td rowspan=\"1\" colspan=\"1\">V3</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">67.5%</td><td rowspan=\"1\" colspan=\"1\">Powermax soil DNA isolation kit (MoBio)</td><td rowspan=\"1\" colspan=\"1\">Fecal</td><td rowspan=\"1\" colspan=\"1\">In-feed ASP250 altered fecal microbiota</td><td rowspan=\"1\" colspan=\"1\">18&#x02013;20&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">United States</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA72355\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA72355</a>; <a href=\"https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP004660\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">SRP004660</a></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><a href=\"#B6\" rid=\"B6\" class=\" bibr popnode\">6</a></td><td rowspan=\"1\" colspan=\"1\">83</td><td rowspan=\"1\" colspan=\"1\">V1 to V3</td><td rowspan=\"1\" colspan=\"1\">454 FLX</td><td rowspan=\"1\" colspan=\"1\">44.4%</td><td rowspan=\"1\" colspan=\"1\">Custom</td><td rowspan=\"1\" colspan=\"1\">Fecal</td><td rowspan=\"1\" colspan=\"1\">In-feed ASP250 altered fecal microbiota</td><td rowspan=\"1\" colspan=\"1\">3&#x02013;13&#x000a0;wks</td><td rowspan=\"1\" colspan=\"1\">United States</td><td rowspan=\"1\" colspan=\"1\"><a href=\"https://www.ncbi.nlm.nih.gov/bioproject/PRJNA72355\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PRJNA72355</a>; <a href=\"https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP008126\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">SRP008126</a></td></tr></tbody></table></span><span id=\"largeobj_idm140162571680672\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab1/?report=objectonly\">Open in a separate window</a></span><span class=\"tblwrap-foot\"><span id=\"ngtab1.2\"><sup>a</sup>Data are from studies that describe using commercial pigs.</span><span id=\"ngtab1.1\"><sup>b</sup>NA, not available.</span><span id=\"ngtab1.3\"><sup>c</sup>DDGS, distillers&#x02019; dried grains with solubles.</span></span></span><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"fig1\" co-legend-rid=\"lgnd_fig1\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig1/\" target=\"figure\" rid-figpopup=\"fig1\" rid-ob=\"ob-fig1\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140162575750784\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5443231_sys0031721030001.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is sys0031721030001.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5443231/bin/sys0031721030001.jpg\" /></a></span><span id=\"largeobj_idm140162575750784\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig1/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_fig1\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig1/\" target=\"figure\" rid-figpopup=\"fig1\" rid-ob=\"ob-fig1\">FIG&#x000a0;1&#x000a0;</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p13\">Diagram of the swine gastrointestinal tract with major sections indicated as well as direction of movement of digesta in the colon. Original collection sites are labeled on the drawing.</p></span></span></span></span><span id=\"s1.2\" class=\"sec\"><p></p><h3 id=\"s1.2title\" class=\"inline\">Identifying the core microbiota of the swine gastrointestinal tract. </h3><p id=\"__p14\" class=\"p p-first\">The idea of defining a core microbiota in the pig gut is intriguing as it may identify potential targets for dietary or therapeutic interventions in the swine production environment. Although a core microbiota may not exist in swine according to a strict definition that requires a particular taxon to be present in each sample, several phyla and genera were found in more than 90% of all GI samples. In the current meta-analysis, the <em>Firmicutes</em> and <em>Bacteroidetes</em> phyla accounted for nearly 85% of the total 16S rRNA gene sequences among all gastrointestinal locations, with <em>Proteobacteria</em> being the only other phylum common to all GI samples (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\" co-legend-rid=\"lgnd_fig2\"><span>Fig.&#x000a0;2A</span></a>). No OTUs or genera were shared among all samples analyzed, but the genera <em>Clostridium</em>, <em>Blautia</em>, <em>Lactobacillus</em>, <em>Prevotella</em>, <em>Ruminococcus</em>, and <em>Roseburia</em>, the RC9 gut group, and <em>Subdoligranulum</em> were found in more than 90% of samples. <em>Prevotella</em> was also the most abundant genus among all genera that were identified (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\" co-legend-rid=\"lgnd_fig2\"><span>Fig.&#x000a0;2B</span></a>; see also <a href=\"#tabS1\" rid=\"tabS1\" class=\" supplementary-material\">Table&#x000a0;S1</a>&#x000a0;in the supplemental material). In addition, a single OTU classified as an uncultured member of the <em>Prevotella</em> was present in at least 75% of all GI samples.</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"fig2\" co-legend-rid=\"lgnd_fig2\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig2/\" target=\"figure\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140162577110112\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5443231_sys0031721030002.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is sys0031721030002.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5443231/bin/sys0031721030002.jpg\" /></a></span><span id=\"largeobj_idm140162577110112\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig2/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_fig2\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig2/\" target=\"figure\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\">FIG&#x000a0;2&#x000a0;</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p16\">Percent relative abundances of the three most abundant phyla (A) and 20 most abundant genera (B) by gastrointestinal tract sample type.</p></span></span></span><span class=\"sec suppmat\" id=\"tabS1\"><h4>TABLE&#x000a0;S1&#x000a0;</h4><p id=\"__p15\">The percentage of samples from each specific gastrointestinal location that had at least one 16S rRNA gene sequence from each of the individual genera identified. Genera are listed in descending order of overall relative abundance. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103st4.pdf\" id=\"tabs1\">TABLE&#x000a0;S1, PDF file, 1.1 MB</a>.</p><span class=\"small permissions half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al.<span class=\"license half_rhythm\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span><p id=\"__p17\">Gut locations and feces yielded discrete core microbiotas. In the fecal microbiota, more than 99% of fecal samples (12 studies) contained sequences from <em>Prevotella</em>, <em>Clostridium</em>, <em>Alloprevotella</em>, <em>Ruminococcus</em>, and the RC9 gut group. Certain genera were present in all samples from a particular GI tract location (<em>n</em> = &#x0003e;50 samples). These included <em>Prevotella</em> and <em>Blautia</em> in samples of colonic digesta (<em>n =</em> 91; 5 studies); <em>Anaerovibrio</em>, <em>Clostridium</em>, <em>Phascolarctobacterium</em>, <em>Ruminococcus</em>, <em>Sarcina</em>, and <em>Streptococcus</em> in cecal digesta (<em>n =</em> 52; 4 studies); and <em>Clostridium</em> in ileal mucosal samples (<em>n =</em> 91; 3 studies). <em>Clostridium</em> spp. were also present in all but one sample of ileal digesta (<em>n =</em> 81; 5 studies).</p><p id=\"__p18\">In commercial swine production, pigs are typically fed a cereal grain-based diet that is relatively high in carbohydrate content (<a href=\"#B20\" rid=\"B20\" class=\" bibr popnode\">20</a>). Meanwhile, <em>Prevotella</em> spp. have been reported to be associated with dietary carbohydrates in humans (<a href=\"#B21\" rid=\"B21\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776891\">21</a>, <a href=\"#B22\" rid=\"B22\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776885\">22</a>), producing polysaccharidases such as glucanase, mannanase, and xylanase (<a href=\"#B23\" rid=\"B23\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776862\">23</a>). Members of <em>Prevotella</em> are also relatively abundant in the rumen and/or gut microbiota of other animals such as cattle (<a href=\"#B24\" rid=\"B24\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776892\">24</a>), chimpanzees (<a href=\"#B25\" rid=\"B25\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776871\">25</a>), goats (<a href=\"#B26\" rid=\"B26\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776886\">26</a>), and sheep (<a href=\"#B27\" rid=\"B27\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776898\">27</a>), attesting to the ubiquity of this genus in the mammalian gut. <em>Clostridium</em>, <em>Blautia</em>, and <em>Ruminococcus</em> are all members of the <em>Clostridiales</em> order, and, similarly to <em>Prevotella</em>, are widely found in the mammalian gut (<a href=\"#B28\" rid=\"B28\" class=\" bibr popnode\">28</a>). These genera can produce butyrate, a short-chain fatty acid (SCFA), most often from acetate (also an SCFA) via the butyryl-coenzyme A (CoA):acetate CoA-transferase pathway (<a href=\"#B29\" rid=\"B29\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776878\">29</a>). <em>Prevotella</em> spp., which produce acetate in the gut, are thereby able to provide a source of energy for butyrate-producing bacteria (<a href=\"#B8\" rid=\"B8\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776880\">8</a>). Importantly, butyrate decreases inflammation in the gut of the host, and cells in the intestinal epithelium can use it as an energy source (<a href=\"#B30\" rid=\"B30\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776861\">30</a>).</p><p id=\"__p19\" class=\"p p-last\">The RC9 gut group, which belongs to the <em>Rikenellaceae</em> family, was identified in 90.5% of all GI samples and was among the most abundant genera in the lower GI tract (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig2\" rid-ob=\"ob-fig2\" co-legend-rid=\"lgnd_fig2\"><span>Fig.&#x000a0;2B</span></a>; <a href=\"#tabS1\" rid=\"tabS1\" class=\" supplementary-material\">Table&#x000a0;S1</a>). This finding is noteworthy as its presence in the swine gut microbiota was reported only recently (<a href=\"#B31\" rid=\"B31\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776866\">31</a>, <a href=\"#B32\" rid=\"B32\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776887\">32</a>). The <em>Rikenellaceae</em> family itself is a relatively new taxonomic classification within the <em>Bacteroidales</em> order, with only three genera currently described (<a href=\"#B33\" rid=\"B33\" class=\" bibr popnode\">33</a>). Similarly to <em>Prevotella</em>, also a member of the <em>Bacteroidales</em>, some bacteria in the <em>Rikenellaceae</em> family are acetate producers (<a href=\"#B34\" rid=\"B34\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776904\">34</a>), suggesting a high level of functional redundancy for this metabolite in the swine gut microbiome.</p></span><span id=\"s1.3\" class=\"sec\"><p></p><h3 id=\"s1.3title\" class=\"inline\">Factors affecting the swine gut microbiota. </h3><p id=\"__p20\" class=\"p p-first\">We determined which variables most strongly affected the structure of the swine gut microbiota using a permutational multivariate analysis of variance (PERMANOVA) test of the unweighted and weighted UniFrac distances and Bray-Curtis dissimilarities (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab2/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"tab2\" rid-ob=\"ob-tab2\" co-legend-rid=\"\"><span>Table&#x000a0;2</span></a>). Among all distance metrics, each metadata category tested was significantly associated with the microbial community structure. However, this finding is likely a result of the heterogeneity of dispersion within each group in each metadata category, as shown by the permutational analysis of multivariate dispersions (PERMDISP; <em>P</em> &#x0003c; 0.05). Overall, &#x0201c;study&#x0201d; was the most important factor affecting the swine gut microbiota, as it explained the greatest variation in the data set (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3</span></a>; also see <a href=\"#figS1\" rid=\"figS1\" class=\" supplementary-material\">Fig.&#x000a0;S1</a>&#x000a0;in the supplemental material). The GI location that was sampled was also strongly associated with the structure of the microbiota as assessed using weighted UniFrac distances (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3A</span></a>); however, this finding was not replicated in the results obtained with respect to the unweighted UniFrac distances and Bray-Curtis dissimilarities (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span>Fig.&#x000a0;3B</span></a> and <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\" co-legend-rid=\"lgnd_fig3\"><span style=\"position: relative;text-decoration:none;\">&#x200B;<span class=\"figpopup-sensitive-area\" style=\"left: -1.5em;\">andC).</span></span><span>C</span></a>). Therefore, researchers could reasonably expect that the taxa that are relatively abundant in their swine gut samples are similar to those in samples taken from the same GI location of different pigs compared with those found in samples obtained from different GI locations.</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"tab2\"><h3>TABLE&#x000a0;2&#x000a0;</h3><!--caption a7--><span class=\"caption\"><p id=\"__p22\">Factors associated with the community structure of the swine gut microbiota as measured using PERMANOVA with the adonis function (9,999 permutations) of the weighted and unweighted UniFrac distances and Bray-Curtis dissimilarities<sup>a</sup></p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140162584427712\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><thead><tr><th rowspan=\"3\" colspan=\"1\">Parameter</th><th colspan=\"9\" rowspan=\"1\">Value<hr /></th></tr><tr><th colspan=\"3\" rowspan=\"1\">Weighted UniFrac<hr /></th><th colspan=\"3\" rowspan=\"1\">Unweighted UniFrac<hr /></th><th colspan=\"3\" rowspan=\"1\">Bray-Curtis<hr /></th></tr><tr><th rowspan=\"1\" colspan=\"1\">Pseudo-F ratio</th><th rowspan=\"1\" colspan=\"1\"><em>R</em><sup>2</sup></th><th rowspan=\"1\" colspan=\"1\"><em>P</em> value</th><th rowspan=\"1\" colspan=\"1\">Pseudo-F ratio</th><th rowspan=\"1\" colspan=\"1\"><em>R</em><sup>2</sup></th><th rowspan=\"1\" colspan=\"1\"><em>P</em> value</th><th rowspan=\"1\" colspan=\"1\">Pseudo-F ratio</th><th rowspan=\"1\" colspan=\"1\"><em>R</em><sup>2</sup></th><th rowspan=\"1\" colspan=\"1\"><em>P</em> value</th></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">Study</td><td rowspan=\"1\" colspan=\"1\">28.2</td><td rowspan=\"1\" colspan=\"1\">0.37</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">18.6</td><td rowspan=\"1\" colspan=\"1\">0.28</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">24.4</td><td rowspan=\"1\" colspan=\"1\">0.34</td><td rowspan=\"1\" colspan=\"1\">0.0001</td></tr><tr><td rowspan=\"1\" colspan=\"1\">GI sampling location</td><td rowspan=\"1\" colspan=\"1\">35.4</td><td rowspan=\"1\" colspan=\"1\">0.29</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">12.9</td><td rowspan=\"1\" colspan=\"1\">0.13</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">12.1</td><td rowspan=\"1\" colspan=\"1\">0.13</td><td rowspan=\"1\" colspan=\"1\">0.0001</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Age</td><td rowspan=\"1\" colspan=\"1\">13.9</td><td rowspan=\"1\" colspan=\"1\">0.21</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">8.5</td><td rowspan=\"1\" colspan=\"1\">0.14</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">10.6</td><td rowspan=\"1\" colspan=\"1\">0.16</td><td rowspan=\"1\" colspan=\"1\">0.0001</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Country of origin</td><td rowspan=\"1\" colspan=\"1\">25.4</td><td rowspan=\"1\" colspan=\"1\">0.18</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">15.6</td><td rowspan=\"1\" colspan=\"1\">0.12</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">20.4</td><td rowspan=\"1\" colspan=\"1\">0.15</td><td rowspan=\"1\" colspan=\"1\">0.0001</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Hypervariable region sequenced</td><td rowspan=\"1\" colspan=\"1\">34.5</td><td rowspan=\"1\" colspan=\"1\">0.16</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">22.7</td><td rowspan=\"1\" colspan=\"1\">0.11</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">18.9</td><td rowspan=\"1\" colspan=\"1\">0.13</td><td rowspan=\"1\" colspan=\"1\">0.0001</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Sequencing platform</td><td rowspan=\"1\" colspan=\"1\">20.4</td><td rowspan=\"1\" colspan=\"1\">0.04</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">19.0</td><td rowspan=\"1\" colspan=\"1\">0.04</td><td rowspan=\"1\" colspan=\"1\">0.0001</td><td rowspan=\"1\" colspan=\"1\">25.6</td><td rowspan=\"1\" colspan=\"1\">0.05</td><td rowspan=\"1\" colspan=\"1\">0.0001</td></tr></tbody></table></span><span id=\"largeobj_idm140162584427712\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab2/?report=objectonly\">Open in a separate window</a></span><span class=\"tblwrap-foot\"><span id=\"ngtab2.1\"><sup>a</sup>For the age category, one study was excluded due to a lack of information about the age of the pigs used. PERMANOVA, permutational multivariate analysis of variance.</span></span></span><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"fig3\" co-legend-rid=\"lgnd_fig3\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig3/\" target=\"figure\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140162575154608\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5443231_sys0031721030003.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is sys0031721030003.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5443231/bin/sys0031721030003.jpg\" /></a></span><span id=\"largeobj_idm140162575154608\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig3/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_fig3\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig3/\" target=\"figure\" rid-figpopup=\"fig3\" rid-ob=\"ob-fig3\">FIG&#x000a0;3&#x000a0;</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p24\">Principal-coordinate analysis plots of weighted UniFrac distances (A), unweighted UniFrac distances (B), and Bray-Curtis dissimilarities (C) classified by gastrointestinal tract sample type. The percentages of variation explained by the principal coordinates are indicated on the axes.</p></span></span></span><span class=\"sec suppmat\" id=\"figS1\"><h4>FIG&#x000a0;S1&#x000a0;</h4><p id=\"__p21\">Principal-coordinate analysis plots of Bray-Curtis dissimilarities by study. The numbers in parentheses in the figure legend refer to the number of samples included for each study. The percentages of variation explained by the principal coordinates are indicated on the axes. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103sf1.tif\" id=\"figs1\">FIG&#x000a0;S1, TIF file, 2.1 MB</a>.</p><span class=\"small permissions half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al.<span class=\"license half_rhythm\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span><p id=\"__p25\">We evaluated the effect of age on the swine gut microbiota using samples from the 19 studies (of 20) that provided specific information for pig age (<em>n =</em> 920; <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab2/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"tab2\" rid-ob=\"ob-tab2\" co-legend-rid=\"\"><span>Table&#x000a0;2</span></a>; <a href=\"#figS2\" rid=\"figS2\" class=\" supplementary-material\">Fig.&#x000a0;S2</a>). Although the effect of the age of the pigs sampled was not as large as the effects of study and GI sampling location, it did explain more variation in the data set than the remaining three metadata categories for the weighted UniFrac distances. It should be noted that pig age, like several of the other metadata categories, is also associated with &#x0201c;study,&#x0201d; as many studies sampled from pigs at only one particular time point. In addition, the 3-week-old category included both preweaned and postweaned pigs, and weaning is known to result in significant changes to the swine gut microbiota (<a href=\"#B10\" rid=\"B10\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776850\">10</a>, <a href=\"#B35\" rid=\"B35\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776854\">35</a>).</p><span class=\"sec suppmat\" id=\"figS2\"><h4>FIG&#x000a0;S2&#x000a0;</h4><p id=\"__p26\">Principal-coordinate analysis plot of the weighted UniFrac distances for all samples with associated metadata for the age of the pigs used in the study. Samples are colored by pig age in weeks. The numbers in parentheses for the figure legend refer to the number of samples at each week. The percentages of variation explained by the principal coordinates are indicated on the axes. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103sf2.tif\" id=\"figs2\">FIG&#x000a0;S2, TIF file, 2.5 MB</a>.</p><span class=\"small permissions half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al.<span class=\"license half_rhythm\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span><p id=\"__p27\">Genera that were enriched in specific GI samples were identified using linear discriminant analysis (LDA) effect size (LEfSe) (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig4\" rid-ob=\"ob-fig4\" co-legend-rid=\"lgnd_fig4\"><span>Fig.&#x000a0;4</span></a>). Among the more notable genera, <em>Lactobacillus</em> was more abundant in the gastric mucosa, <em>Prevotella</em>, <em>Helicobacter</em>, and <em>Campylobacter</em> in the colonic mucosa, <em>Clostridium</em> in the ileal mucosa, <em>Alloprevotella</em> in the cecal mucosa, <em>Bacteroides</em> in the colon, and <em>Treponema</em> in fecal samples. These differences in the relative abundances of certain genera along the gastrointestinal tract likely reflect the physiological conditions at each section. For example, the pH is significantly lower in the stomach than at any other GI location and <em>Lactobacillus</em> spp. are relatively acid tolerant, possibly explaining their enrichment in this location (<a href=\"#B36\" rid=\"B36\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776896\">36</a>). The lactobacilli are also able to attach to the epithelial and mucosal layers, where they can form biofilm-like communities, thereby aiding their persistence (<a href=\"#B37\" rid=\"B37\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776889\">37</a>). <em>Lactobacillus</em> spp. were present among all types of GI samples; therefore, the population of lactobacilli in the stomach may serve as a source of these bacteria for other gut locations (<a href=\"#B38\" rid=\"B38\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776888\">38</a>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"fig4\" co-legend-rid=\"lgnd_fig4\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig4/\" target=\"figure\" rid-figpopup=\"fig4\" rid-ob=\"ob-fig4\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140162573451328\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5443231_sys0031721030004.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is sys0031721030004.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5443231/bin/sys0031721030004.jpg\" /></a></span><span id=\"largeobj_idm140162573451328\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig4/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_fig4\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig4/\" target=\"figure\" rid-figpopup=\"fig4\" rid-ob=\"ob-fig4\">FIG&#x000a0;4&#x000a0;</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p28\">Differentially abundant genera in each gastrointestinal tract sample type as assessed using linear discriminant analysis (LDA) with effect size (LEfSe) measurements. Only those genera with an LDA score (log<sub>10</sub>) of &#x0003e;4.0 are displayed. Samples of duodenum and jejunum mucosa and digesta were excluded from analysis as there were fewer than five samples for each.</p></span></span></span><p id=\"__p29\">Comparisons of alpha-diversity measures also revealed significant differences among samples from different GI locations, including feces (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab3/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"tab3\" rid-ob=\"ob-tab3\" co-legend-rid=\"\"><span>Table&#x000a0;3</span></a>). In particular, there was a clear demarcation between upper and lower GI samples, with significantly higher diversity, richness, and evenness in the lower GI samples (<em>P</em> &#x0003c; 0.05). Within the lower GI tract, fecal and cecal mucosa samples were the most rich in terms of the number of OTUs and phylogenetic diversity (PD). These observations are similar to those reported in the longitudinal swine gut microbiota study conducted by Looft et al. (<a href=\"#B7\" rid=\"B7\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776848\">7</a>). The only significant difference among upper GI tract sections was observed between the samples of ileal digesta and mucosa, where the mucosal samples had greater phylogenetic diversity as determined using PD whole tree. These results are expected, given that transit is faster in the upper GI tract than in the lower GI tract, thereby limiting adhesion and colonization of bacteria in the upper GI tract (<a href=\"#B39\" rid=\"B39\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776883\">39</a>). Also, competition for nutrients between the host and bacteria is greater in the upper GI tract (<a href=\"#B40\" rid=\"B40\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776847\">40</a>).</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"tab3\"><h3>TABLE&#x000a0;3&#x000a0;</h3><!--caption a7--><span class=\"caption\"><p id=\"__p30\">Alpha-diversity measures for each gastrointestinal location<sup>a</sup></p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140162571636048\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><thead><tr><th rowspan=\"2\" colspan=\"1\">GI sample type</th><th colspan=\"5\" rowspan=\"1\">Values<hr /></th></tr><tr><th rowspan=\"1\" colspan=\"1\">No. of OTUs</th><th rowspan=\"1\" colspan=\"1\">Phylogenetic diversity</th><th rowspan=\"1\" colspan=\"1\">Shannon index</th><th rowspan=\"1\" colspan=\"1\">Simpson&#x02019;s reciprocal index</th><th rowspan=\"1\" colspan=\"1\">Equitability</th></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">Gastric mucosa (<em>n</em> = 36)</td><td rowspan=\"1\" colspan=\"1\">135 &#x000b1; 66c</td><td rowspan=\"1\" colspan=\"1\">26.9 &#x000b1; 12.8c</td><td rowspan=\"1\" colspan=\"1\">3.04 &#x000b1; 1.02d\n</td><td rowspan=\"1\" colspan=\"1\">12 &#x000b1; 10.2b</td><td rowspan=\"1\" colspan=\"1\">0.62 &#x000b1; 0.15cd</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Duodenum (<em>n</em> = 3)</td><td rowspan=\"1\" colspan=\"1\">92 &#x000b1; 13c</td><td rowspan=\"1\" colspan=\"1\">14 &#x000b1; 1.7d</td><td rowspan=\"1\" colspan=\"1\">1.87 &#x000b1; 0.98d\n</td><td rowspan=\"1\" colspan=\"1\">3.7 &#x000b1; 3.7b</td><td rowspan=\"1\" colspan=\"1\">0.41 &#x000b1; 0.21d</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Duodenal mucosa (<em>n</em> = 2)</td><td rowspan=\"1\" colspan=\"1\">120 &#x000b1; 0c</td><td rowspan=\"1\" colspan=\"1\">23.2 &#x000b1; 2.7cd</td><td rowspan=\"1\" colspan=\"1\">2.85 &#x000b1; 0.02d\n</td><td rowspan=\"1\" colspan=\"1\">6.4 &#x000b1; 0.3b</td><td rowspan=\"1\" colspan=\"1\">0.59 &#x000b1; 0.00cd</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Jejunum (<em>n</em> = 4)</td><td rowspan=\"1\" colspan=\"1\">118 &#x000b1; 16c</td><td rowspan=\"1\" colspan=\"1\">18.2 &#x000b1; 4.2cd</td><td rowspan=\"1\" colspan=\"1\">3.06 &#x000b1; 0.40cd</td><td rowspan=\"1\" colspan=\"1\">9.4 &#x000b1; 4.3b</td><td rowspan=\"1\" colspan=\"1\">0.64 &#x000b1; 0.07bcd</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Jejunal mucosa (<em>n</em> = 3)</td><td rowspan=\"1\" colspan=\"1\">126 &#x000b1; 25c</td><td rowspan=\"1\" colspan=\"1\">21.4 &#x000b1; 7.2cd</td><td rowspan=\"1\" colspan=\"1\">3.22 &#x000b1; 0.27bcd</td><td rowspan=\"1\" colspan=\"1\">11.4 &#x000b1; 2.7b</td><td rowspan=\"1\" colspan=\"1\">0.67 &#x000b1; 0.03abc</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Ileum (<em>n</em> = 81)</td><td rowspan=\"1\" colspan=\"1\">94 &#x000b1; 42c</td><td rowspan=\"1\" colspan=\"1\">15.8 &#x000b1; 6.3d</td><td rowspan=\"1\" colspan=\"1\">2.72 &#x000b1; 0.89d</td><td rowspan=\"1\" colspan=\"1\">10.2 &#x000b1; 8.3b</td><td rowspan=\"1\" colspan=\"1\">0.60 &#x000b1; 0.14cd</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Ileal mucosa (<em>n</em> = 91)</td><td rowspan=\"1\" colspan=\"1\">128 &#x000b1; 54c</td><td rowspan=\"1\" colspan=\"1\">24.1 &#x000b1; 9.6c</td><td rowspan=\"1\" colspan=\"1\">3.02 &#x000b1; 0.86d</td><td rowspan=\"1\" colspan=\"1\">11.5 &#x000b1; 9.8b</td><td rowspan=\"1\" colspan=\"1\">0.62 &#x000b1; 0.14cd</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Cecum (<em>n</em> = 52)</td><td rowspan=\"1\" colspan=\"1\">213 &#x000b1; 81b</td><td rowspan=\"1\" colspan=\"1\">35.7 &#x000b1; 10.3b</td><td rowspan=\"1\" colspan=\"1\">4.02 &#x000b1; 0.83bc</td><td rowspan=\"1\" colspan=\"1\">30.3 &#x000b1; 24.5a</td><td rowspan=\"1\" colspan=\"1\">0.75 &#x000b1; 0.11b</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Cecal mucosa (<em>n</em> = 18)</td><td rowspan=\"1\" colspan=\"1\">282 &#x000b1; 51a</td><td rowspan=\"1\" colspan=\"1\">42.9 &#x000b1; 7.3ab</td><td rowspan=\"1\" colspan=\"1\">4.61 &#x000b1; 0.43a</td><td rowspan=\"1\" colspan=\"1\">42.8 &#x000b1; 18.5a</td><td rowspan=\"1\" colspan=\"1\">0.82 &#x000b1; 0.05a</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Colon (<em>n</em> = 91)</td><td rowspan=\"1\" colspan=\"1\">221 &#x000b1; 66b</td><td rowspan=\"1\" colspan=\"1\">37.7 &#x000b1; 9b</td><td rowspan=\"1\" colspan=\"1\">4.23 &#x000b1; 0.53ab</td><td rowspan=\"1\" colspan=\"1\">33.2 &#x000b1; 22.6a</td><td rowspan=\"1\" colspan=\"1\">0.79 &#x000b1; 0.06ab</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Colonic mucosa (<em>n</em> = 48)</td><td rowspan=\"1\" colspan=\"1\">210 &#x000b1; 90b</td><td rowspan=\"1\" colspan=\"1\">37.2 &#x000b1; 12b</td><td rowspan=\"1\" colspan=\"1\">4.05 &#x000b1; 0.74abc</td><td rowspan=\"1\" colspan=\"1\">30.9 &#x000b1; 26.9a</td><td rowspan=\"1\" colspan=\"1\">0.76 &#x000b1; 0.09ab</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Fecal (<em>n</em> = 510)</td><td rowspan=\"1\" colspan=\"1\">255 &#x000b1; 73a</td><td rowspan=\"1\" colspan=\"1\">44.1 &#x000b1; 11.7a</td><td rowspan=\"1\" colspan=\"1\">4.41 &#x000b1; 0.65a</td><td rowspan=\"1\" colspan=\"1\">40.6 &#x000b1; 29.2a</td><td rowspan=\"1\" colspan=\"1\">0.80 &#x000b1; 0.08a</td></tr></tbody></table></span><span id=\"largeobj_idm140162571636048\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab3/?report=objectonly\">Open in a separate window</a></span><span class=\"tblwrap-foot\"><span id=\"ngtab3.1\"><sup>a</sup>Different lowercase letters in each column indicate significant differences (<em>P</em> &#x0003c; 0.05). Data represent means &#x000b1; standard deviations.</span></span></span><p id=\"__p32\" class=\"p p-last\">Although it is reassuring that GI sampling location is a strong predictor of the swine gut microbial community structure, study-level effects are also significant, as evidenced above. The importance of technical variation among studies has been documented in recent meta-analyses of the avian gut microbiota (<a href=\"#B4\" rid=\"B4\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776860\">4</a>) and the indoor microbiota (<a href=\"#B5\" rid=\"B5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776857\">5</a>), where study was the most important driver of the microbiota. In the present meta-analysis, a wide range of ages were included for the pigs, as well as different diets, pig breeds, and administered treatments (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab1/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"tab1\" rid-ob=\"ob-tab1\" co-legend-rid=\"\"><span>Table&#x000a0;1</span></a>). In addition, at least seven different DNA extraction methods were used and six separate 16S rRNA gene hypervariable regions were sequenced on three different sequencing platforms. The use of standard or uniform DNA extraction, 16S rRNA gene library preparation, and sequencing protocols would likely help reduce some of this study-to-study variation. With that said, eliminating all sources of technical variation in a single study may prove difficult as even technical replicates from different sequencing centers can differ significantly (<a href=\"#B41\" rid=\"B41\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776881\">41</a>).</p></span><span id=\"s1.4\" class=\"sec\"><p></p><h3 id=\"s1.4title\" class=\"inline\">Effect of 16S rRNA gene hypervariable region used for sequencing. </h3><p id=\"__p33\" class=\"p p-first\">The choice of which 16S rRNA gene hypervariable region(s) to sequence is an important consideration for any 16S-based microbial community study as even so-called &#x0201c;universal primers&#x0201d; differ greatly in their amplification of certain bacterial groups (<a href=\"#B42\" rid=\"B42\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776900\">42</a>). To determine the effect of different 16S hypervariable region sequences on the characterization of the swine gut microbiota, we analyzed only fecal samples to eliminate one of the largest sources of variability among samples (i.e., the GI sample type). When unweighted and weighted UniFrac distances and Bray-Curtis dissimilarities were plotted using PCoA, samples clustered by which hypervariable region was sequenced, although the amount of variation explained was smaller for the weighted UniFrac distances and there was more overlap between samples sequenced with different hypervariable regions (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig5\" rid-ob=\"ob-fig5\" co-legend-rid=\"lgnd_fig5\"><span>Fig.&#x000a0;5</span></a>). In the fecal samples, the study effect was still stronger than the effect of the hypervariable region sequenced with PERMANOVA (adonis <em>R</em><sup>2</sup>&#x000a0;values of 0.31, 0.28, and 0.36 for weighted and unweighted UniFrac distances and Bray-Curtis dissimilarities, respectively). However, this is to be expected given that each study sequenced only one specific 16S rRNA gene hypervariable region. Fecal samples that were sequenced using primers that targeted the V4 region did have more phylogenetic diversity (PD whole tree) and a higher Simpson&#x02019;s reciprocal value than those that were sequenced using primers that targeted the other 16S hypervariable regions analyzed, although the richness and the Shannon index data were not significantly different (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab4/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"tab4\" rid-ob=\"ob-tab4\" co-legend-rid=\"\"><span>Table&#x000a0;4</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"fig5\" co-legend-rid=\"lgnd_fig5\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig5/\" target=\"figure\" rid-figpopup=\"fig5\" rid-ob=\"ob-fig5\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140162575363392\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5443231_sys0031721030005.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is sys0031721030005.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5443231/bin/sys0031721030005.jpg\" /></a></span><span id=\"largeobj_idm140162575363392\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig5/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_fig5\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig5/\" target=\"figure\" rid-figpopup=\"fig5\" rid-ob=\"ob-fig5\">FIG&#x000a0;5&#x000a0;</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p34\">Principal-coordinate analysis plots of weighted UniFrac distances (A), unweighted UniFrac distances (B), and Bray-Curtis dissimilarities (C) classified by hypervariable region sequenced for fecal samples only. Percentages of variation explained by the principal coordinates are indicated on the axes, and the <em>R</em>&#x000a0;values on each plot indicate the dissimilarities between the hypervariable region groups.</p></span></span></span><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"tab4\"><h3>TABLE&#x000a0;4&#x000a0;</h3><!--caption a7--><span class=\"caption\"><p id=\"__p35\">Alpha-diversity measures by hypervariable region sequenced for fecal samples only<sup>a</sup></p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140162573733824\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><thead><tr><th rowspan=\"2\" colspan=\"1\">Hypervariable region(s)</th><th colspan=\"5\" rowspan=\"1\">Value(s)<hr /></th></tr><tr><th rowspan=\"1\" colspan=\"1\">No. of OTUs</th><th rowspan=\"1\" colspan=\"1\">Phylogenetic diversity</th><th rowspan=\"1\" colspan=\"1\">Shannon index</th><th rowspan=\"1\" colspan=\"1\">Simpson&#x02019;s reciprocal index</th><th rowspan=\"1\" colspan=\"1\">Equitability</th></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">V1 to V3 (<em>n</em> = 270)</td><td rowspan=\"1\" colspan=\"1\">253 &#x000b1; 49</td><td rowspan=\"1\" colspan=\"1\">42.6 &#x000b1; 6.9a</td><td rowspan=\"1\" colspan=\"1\">4.46 &#x000b1; 0.44</td><td rowspan=\"1\" colspan=\"1\">37.6 &#x000b1; 23a</td><td rowspan=\"1\" colspan=\"1\">0.81 &#x000b1; 0.06a</td></tr><tr><td rowspan=\"1\" colspan=\"1\">V3 (<em>n</em> = 12)</td><td rowspan=\"1\" colspan=\"1\">232 &#x000b1; 35</td><td rowspan=\"1\" colspan=\"1\">42.1 &#x000b1; 5.7a</td><td rowspan=\"1\" colspan=\"1\">4.29 &#x000b1; 0.4</td><td rowspan=\"1\" colspan=\"1\">29 &#x000b1; 12.5a</td><td rowspan=\"1\" colspan=\"1\">0.79 &#x000b1; 0.06b</td></tr><tr><td rowspan=\"1\" colspan=\"1\">V3 and V4 (<em>n</em> = 14)</td><td rowspan=\"1\" colspan=\"1\">236 &#x000b1; 50</td><td rowspan=\"1\" colspan=\"1\">39.6 &#x000b1; 7.1a</td><td rowspan=\"1\" colspan=\"1\">4.26 &#x000b1; 0.69</td><td rowspan=\"1\" colspan=\"1\">30.3 &#x000b1; 17.5a</td><td rowspan=\"1\" colspan=\"1\">0.78 &#x000b1; 0.1b</td></tr><tr><td rowspan=\"1\" colspan=\"1\">V4 (<em>n</em> = 214)</td><td rowspan=\"1\" colspan=\"1\">259 &#x000b1; 97</td><td rowspan=\"1\" colspan=\"1\">46.3 &#x000b1; 15.9b</td><td rowspan=\"1\" colspan=\"1\">4.35 &#x000b1; 0.85</td><td rowspan=\"1\" colspan=\"1\">45.7 &#x000b1; 36.1b</td><td rowspan=\"1\" colspan=\"1\">0.79 &#x000b1; 0.1b</td></tr></tbody></table></span><span id=\"largeobj_idm140162573733824\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab4/?report=objectonly\">Open in a separate window</a></span><span class=\"tblwrap-foot\"><span id=\"ngtab4.1\"><sup>a</sup>Different lowercase letters in each column indicate significant differences. Data represent means &#x000b1; standard deviations.</span></span></span><p id=\"__p37\">Comparing the region V1 to V3 and region V4 sequences from fecal samples using LEfSe, the region V1 to V3 sequences contained a higher proportion of the <em>Firmicutes</em> phylum and <em>Blautia</em> genus (LDA score [log<sub>10</sub>] = &#x0003e;4.0). Sequences classified at the phylum level as <em>Spirochaetae</em> and at the genus level as <em>Bacteroides</em>, <em>Lactobacillus</em>, and <em>Treponema</em> were more abundant in fecal samples that were sequenced using the V4 region. Another notable difference at the genus level among the different hypervariable regions was the nearly complete absence of <em>Bifidobacterium</em> in fecal samples from studies using the region V1 to V3 and region V3 16S primer sets (1 sample of 288; <a href=\"#tabS2\" rid=\"tabS2\" class=\" supplementary-material\">Table&#x000a0;S2</a>). In contrast, 74.3% of fecal samples that were sequenced using the V4 hypervariable region and 93% of fecal samples using region V3 to V4 had at least one <em>Bifidobacterium</em> 16S rRNA gene sequence. The generic 27F primer sequence (5&#x02032;-AGAGTTTGATCCTGGCTCAG-3&#x02032;) which is often used to amplify regions V1 to V3 has been previously reported to underestimate the relative abundance of bifidobacteria unless modifications are made to the primer sequence (<a href=\"#B43\" rid=\"B43\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776863\">43</a>, <a href=\"#B44\" rid=\"B44\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776890\">44</a>). Although the relative abundance of <em>Bifidobacterium</em> in fecal samples remained low (&#x0003c;0.35%) in the present meta-analysis, this may be an important consideration for researchers interested in this particular genus.</p><span class=\"sec suppmat\" id=\"tabS2\"><h4>TABLE&#x000a0;S2&#x000a0;</h4><p id=\"__p38\">The percentage of fecal samples grouped by specific 16S rRNA gene hypervariable region sequenced that had at least one 16S rRNA gene sequence from each of the individual genera identified. Genera are listed in descending order of overall relative abundance. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103st5.pdf\" id=\"tabs2\">TABLE&#x000a0;S2, PDF file, 0.5 MB</a>.</p><span class=\"small permissions half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al.<span class=\"license half_rhythm\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span><p id=\"__p39\" class=\"p p-last\">Primers targeting the V4 region, however, poorly amplify sequences from <em>Spirochaeta</em> and <em>Desulfitibacter</em>, as these genera were identified in only 12.1% and 0.5% of fecal samples that were sequenced with V4 primers, compared to 91.5% and 77.8% of fecal samples analyzed with region V1 to V3 primers, respectively (<a href=\"#tabS2\" rid=\"tabS2\" class=\" supplementary-material\">Table&#x000a0;S2</a>). Although only 14 fecal samples from a single study were available for analysis of region V3 to V4 (<a href=\"#B45\" rid=\"B45\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776851\">45</a>), <em>Treponema</em> spp. were completely absent from these samples, an unexpected finding given the ubiquity of this genus among fecal samples sequenced using other 16S primers; therefore, those results may reflect the primer set used rather than the hypervariable region sequenced.</p></span><span id=\"s1.5\" class=\"sec\"><p></p><h3 id=\"s1.5title\" class=\"inline\">Studies including antimicrobial administration. </h3><p id=\"__p40\" class=\"p p-first\">Antimicrobials are frequently used in swine production for the purpose of growth promotion, disease prevention, or disease treatment (<a href=\"#B46\" rid=\"B46\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776877\">46</a>). However, their usage in pigs may help maintain a reservoir of antimicrobial-resistant bacteria and resistance determinants (<a href=\"#B47\" rid=\"B47\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776868\">47</a>, <a href=\"#B48\" rid=\"B48\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776853\">48</a>). The development of alternatives to antimicrobials in swine requires a better understanding of how the microbiota is affected by these agents (<a href=\"#B49\" rid=\"B49\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776852\">49</a>, <a href=\"#B50\" rid=\"B50\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776867\">50</a>). As 16S rRNA gene sequences were available from six separate studies that investigated the effect of antimicrobial use on the swine fecal microbiota, we combined these studies to determine if any pattern would emerge that was associated with the use of any type of antimicrobial agent in swine (<a href=\"#B6\" rid=\"B6\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776864\">6</a>,<a href=\"#B7\" rid=\"B7\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776897\">&#x02013;</a><a href=\"#B11\" rid=\"B11\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776859\">11</a>). Samples once again clustered by study rather than by antimicrobial use, as nonantimicrobial control samples were not separated from antimicrobial treatment samples within each study (<a href=\"#figS3\" rid=\"figS3\" class=\" supplementary-material\">Fig.&#x000a0;S3</a>). Also, no genera were significantly different between pigs that received antimicrobials and those that did not (LDA [log<sub>10</sub>] score, &#x0003c;4.0). This result is perhaps not surprising given that there is a strong study effect and that antimicrobials with different mechanisms of action and dosages were used in each study. Each study also reported findings that are not necessarily in agreement with each other (<a href=\"#B51\" rid=\"B51\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776865\">51</a>). For example, the <em>Bacteroidetes</em> phylum has been reported to be relatively less abundant at specific time points in pigs fed tylosin (<a href=\"#B10\" rid=\"B10\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776875\">10</a>) and ASP250 (<a href=\"#B49\" rid=\"B49\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776855\">49</a>). However, carbadox treatment has been shown to actually increase the proportion of <em>Bacteroidetes</em> during the early phase of administration (<a href=\"#B8\" rid=\"B8\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776893\">8</a>).</p><span class=\"sec suppmat\" id=\"figS3\"><h4>FIG&#x000a0;S3&#x000a0;</h4><p id=\"__p41\">Principal-coordinate analysis plots of weighted UniFrac distances (A), unweighted UniFrac distances (B), and Bray-Curtis dissimilarities (C) for studies investigating the effect of an antimicrobial on the swine fecal microbiota. Samples are colored by study and antimicrobial treatment. Download <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/bin/sys003172103sf3.tif\" id=\"figs3\">FIG&#x000a0;S3, TIF file, 1.7 MB</a>.</p><span class=\"small permissions half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright</a>  &#x000a9; 2017 Holman et al.<span class=\"license half_rhythm\">This content is distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution 4.0 International license</a>.</span></span></span><p id=\"__p42\" class=\"p p-last\">These findings emphasize the need for additional research into the effects of antimicrobials on the swine gut microbiota to better predict the response of the microbiota to these agents and to aid in the development of alternatives. The use of larger numbers of animals, longitudinal sampling, and the reporting of pig mass and other production attributes in each trial would all improve our understanding of the effect that antimicrobial agents have on gut microbiota of swine. It is also possible that any beneficial health or production effects observed in the pig are a result of changes in areas of the GI tract that preclude repeated sampling such as the ileum or cecum or that directly impact the host animal rather than the microbiota.</p></span><span id=\"s1.6\" class=\"sec sec-last\"><p></p><h3 id=\"s1.6title\" class=\"inline\">Conclusions. </h3><p id=\"__p43\" class=\"p p-first-last\">The results from our meta-analysis demonstrate that after &#x0201c;study&#x0201d; effects, GI tract location is the strongest predictor of the swine gut microbiota composition. In particular, the relatively abundant taxa are similar among samples from the same GI location, independently of other experimental variables. However, technical variation between individual studies may mask other biological differences among swine gut samples. For example, it was not possible to identify any consistent patterns related to antimicrobial use when six studies that included an antimicrobial treatment were analyzed together. The specificity of 16S rRNA gene primers and the hypervariable region that is sequenced also have a large effect on the analysis of the microbiota, as demonstrated in fecal samples from studies using different 16S regions. Nonetheless, a portion of the gut microbiota is shared among at least 90% of GI samples regardless of experimental variables, and this includes <em>Clostridium</em>, <em>Blautia</em>, <em>Lactobacillus</em>, <em>Prevotella</em>, <em>Ruminococcus</em>, <em>Roseburia</em>, the RC9 gut group, and <em>Subdoligranulum</em> at the genus level. These genera represent bacteria that are well adapted to the swine gut and may serve as potential markers of a typical swine gut microbiota. The inclusion of comprehensive metadata to accompany sequencing data sets would enhance future meta-analyses and allow researchers to compare their results directly with those from similar studies.</p></span></span><span id=\"s2\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s2title\" style=\"text-transform:uppercase\">MATERIALS AND METHODS</h2><span id=\"s2.1\" class=\"sec sec-first\"><p></p><h3 id=\"s2.1title\" class=\"inline\">Data acquisition and study inclusion criteria. </h3><p id=\"__p44\" class=\"p p-first-last\">A total of 20 studies were included in the meta-analysis and are described in <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/table/tab1/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"tab1\" rid-ob=\"ob-tab1\" co-legend-rid=\"\"><span>Table&#x000a0;1</span></a>. Studies were identified through a literature search of the Short Read Archive (SRA; <a href=\"http://www.ncbi.nlm.nih.gov/sra\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.ncbi.nlm.nih.gov/sra</a>), NCBI PubMed (<a href=\"http://www.ncbi.nlm.nih.gov/pubmed\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.ncbi.nlm.nih.gov/pubmed</a>), and Google Scholar (<a href=\"http://scholar.google.com\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://scholar.google.com</a>). To be included in the meta-analysis, all studies were required (i) to have been sampled from the swine GI tract, (ii) to have used high-throughput sequencing of the 16S rRNA gene, (iii) to have associated metadata and quality score files, and (iv) to have provided data that were publically available before 31&#x000a0;March 2016. Sequence files from each study were downloaded from the SRA or European Nucleotide Archive (ENA; <a href=\"http://www.ebi.ac.uk/ena\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.ebi.ac.uk/ena</a>).</p></span><span id=\"s2.2\" class=\"sec sec-last\"><p></p><h3 id=\"s2.2title\" class=\"inline\">Processing of 16S rRNA gene sequences. </h3><p id=\"__p45\" class=\"p p-first\">All downloaded 16S rRNA gene sequences were processed using the QIIME software package (v. 1.9.1) (<a href=\"#B52\" rid=\"B52\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776870\">52</a>). When paired-end sequence files were provided from Illumina MiSeq or HiSeq 2000-based studies, reads were joined using fastq-join (<a href=\"#B53\" rid=\"B53\" class=\" bibr popnode\">53</a>) with an allowed maximum difference of 15% and a minimum overlap of 35&#x000a0;bp. Sequences derived from the 454 FLX platform were converted from the fastq file format to fasta and qual files, and sequences were subsequently quality filtered using the <em>split_libraries.py</em> command with a Phred score cutoff of 25, a minimum read length of 200&#x000a0;bp, and a maximum homopolymer run of 6&#x000a0;bp. Illumina MiSeq and HiSeq 2000 reads were quality filtered using the <em>split_libraries_fastq.py</em> command with a Phred score cutoff of 25, reads were truncated following three consecutive base calls with a Phred score of &#x0003c;25, and reads with &#x0003c;75% of the original read length following truncation were discarded. The maximum length of each read was set based on the expected read length, as reads spanning regions V1 to V3, for example, are significantly longer than those corresponding to the V4 region.</p><p id=\"__p46\">The quality-filtered 16S rRNA gene sequences were clustered into operational taxonomic units (OTUs) at 97% similarity using a closed-reference approach, the SILVA release 119 database (<a href=\"#B54\" rid=\"B54\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776856\">54</a>), and USEARCH v. 6.1.544 (<a href=\"#B55\" rid=\"B55\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776849\">55</a>), with reverse strand matching enabled. In this method, reads that failed to match one of the 16S rRNA gene sequences in the SILVA database at 97% similarity were discarded. Depending upon the hypervariable region targeted, some 16S rRNA gene primers also amplify archaeal 16S rRNA gene sequences. Therefore, all sequences classified as <em>Archaea</em>, as well as chloroplast, mitochondrial, and eukaryotic sequences, were removed prior to downstream analysis. All samples were randomly subsampled to a level of 1,000 sequences per sample to account for uneven sequencing depth among samples and studies.</p><p id=\"__p47\">Samples were categorized based on sample type, treatment, diet, country of origin, age, sequencing platform, and 16S rRNA hypervariable region sequenced, where available. Only information about sample type, country of origin, sequencing platform, and hypervariable region sequenced was available for each sample. Linear discriminant analysis effect size (LEfSe) was used to determine which genera were significantly enriched in each sample type. Genera that were relatively more abundant in a particular sample group were identified by LEfSe using the Kruskal-Wallis test (<em>P</em> &#x0003c; 0.05), and the effect size of each of these genera was estimated using linear discriminant analysis (<a href=\"#B56\" rid=\"B56\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776899\">56</a>). An LDA score (log<sub>10</sub>) of 4.0 was used as the cutoff for identifying differentially abundant genera.</p><p id=\"__p48\" class=\"p p-last\">The between-sample (beta) diversity was assessed using the unweighted and weighted UniFrac (<a href=\"#B57\" rid=\"B57\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776858\">57</a>) distances and Bray-Curtis (<a href=\"#B58\" rid=\"B58\" class=\" bibr popnode\">58</a>) dissimilarities. Principal-coordinate analysis (PCoA) was used to visualize these distances using Emperor (<a href=\"#B59\" rid=\"B59\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_617776903\">59</a>). Permutational multivariate analysis of variance (PERMANOVA) using the adonis function (<a href=\"#B60\" rid=\"B60\" class=\" bibr popnode\">60</a>) with 9,999 permutations was implemented in QIIME to analyze the unweighted and weighted UniFrac distances and the Bray-Curtis dissimilarities for each gastrointestinal location, country of origin, hypervariable region, sequencing platform used, and study. Pig age was also assessed but with one study removed due to a lack of metadata. Permutational analysis of multivariate dispersions (PERMDISP) was used to test the homogeneity of dispersion for each metadata category as implemented in QIIME using the betadisper function of vegan (<a href=\"#B60\" rid=\"B60\" class=\" bibr popnode\">60</a>). The within-sample (alpha) diversity, richness, and evenness were calculated within QIIME using the Shannon index (<a href=\"#B61\" rid=\"B61\" class=\" bibr popnode\">61</a>), phylogenetic diversity (PD whole tree) (<a href=\"#B62\" rid=\"B62\" class=\" bibr popnode\">62</a>), Simpson reciprocal index (<a href=\"#B63\" rid=\"B63\" class=\" bibr popnode\">63</a>), and equitability (evenness) index. These metrics, as well as the metric of hypervariable region sequenced for fecal samples only, were compared among gastrointestinal sample types using a two-way ANOVA in R (v. 3.2.5) (<a href=\"#B64\" rid=\"B64\" class=\" bibr popnode\">64</a>) with hypervariable region and sample type as the independent factors, followed by Tukey&#x02019;s honestly significant difference (HSD) <em>post hoc</em> pairwise comparison test (agricolae package [<a href=\"#B65\" rid=\"B65\" class=\" bibr popnode\">65</a>]). All results were considered significant at <em>P</em> values of &#x0003c;0.05.</p></span></span><span id=\"idm140162575841136\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140162575841136title\" style=\"text-transform:uppercase\">ACKNOWLEDGMENTS</h2><span class=\"sec\"><p id=\"__p49\">We are grateful for Mike Marti&#x02019;s graphic design talents for the creation of <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5443231/figure/fig1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"fig1\" rid-ob=\"ob-fig1\" co-legend-rid=\"lgnd_fig1\"><span>Fig.&#x000a0;1</span></a>.</p><p id=\"__p50\">D.B.H. obtained the sequence data, performed the analyses, and wrote the manuscript. B.W.B., D.B.H., and H.K.A. conceived the meta-analysis. B.W.B., H.K.A., and J.T. helped in the design and draft of the manuscript and provided critical feedback.</p><p id=\"__p51\">Mention of trade names or commercial products in this publication is solely for the purpose of providing specific information and does not imply recommendation or endorsement by the US Department of the Agriculture. The USDA is an equal opportunity provider and employer.</p></span></span><br><div class=\"ref-list-sec sec\" id=\"reference-list\"><div class=\"ref-cit-blk half_rhythm\" id=\"B1\">1. <span class=\"mixed-citation\">Brestoff JR, Artis D. \n2013. \n<span class=\"ref-title\">Commensal bacteria at the interface of host metabolism and the immune system</span>. <span class=\"ref-journal\">Nat Immunol</span>\n<span class=\"ref-vol\">14</span>:676&#x02013;684. doi:10.1038/ni.2640. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4013146/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23778795\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fni.2640\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat+Immunol&amp;title=Commensal+bacteria+at+the+interface+of+host+metabolism+and+the+immune+system&amp;author=JR+Brestoff&amp;author=D+Artis&amp;volume=14&amp;publication_year=2013&amp;pages=676-684&amp;pmid=23778795&amp;doi=10.1038/ni.2640&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B2\">2. <span class=\"mixed-citation\">Tremaroli V, B&#x000e4;ckhed F. \n2012. \n<span class=\"ref-title\">Functional interactions between the gut microbiota and host metabolism</span>. <span class=\"ref-journal\">Nature</span>\n<span class=\"ref-vol\">489</span>:242&#x02013;249. doi:10.1038/nature11552. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22972297\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnature11552\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Functional+interactions+between+the+gut+microbiota+and+host+metabolism&amp;author=V+Tremaroli&amp;author=F+B&#x000e4;ckhed&amp;volume=489&amp;publication_year=2012&amp;pages=242-249&amp;pmid=22972297&amp;doi=10.1038/nature11552&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B3\">3. <span class=\"mixed-citation\">Lall&#x000e8;s JP, Bosi P, Smidt H, Stokes CR. \n2007. \n<span class=\"ref-title\">Nutritional management of gut health in pigs around weaning</span>. <span class=\"ref-journal\">Proc Nutr Soc</span>\n<span class=\"ref-vol\">66</span>:260&#x02013;268. doi:10.1017/S0029665107005484. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17466106\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1017%2FS0029665107005484\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc+Nutr+Soc&amp;title=Nutritional+management+of+gut+health+in+pigs+around+weaning&amp;author=JP+Lall&#x000e8;s&amp;author=P+Bosi&amp;author=H+Smidt&amp;author=CR+Stokes&amp;volume=66&amp;publication_year=2007&amp;pages=260-268&amp;pmid=17466106&amp;doi=10.1017/S0029665107005484&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B4\">4. <span class=\"mixed-citation\">Waite DW, Taylor MW. \n2014. \n<span class=\"ref-title\">Characterizing the avian gut microbiota: membership, driving influences, and potential function</span>. <span class=\"ref-journal\">Front Microbiol</span>\n<span class=\"ref-vol\">5</span>:223. doi:10.3389/fmicb.2014.00223. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4032936/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24904538\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3389%2Ffmicb.2014.00223\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front+Microbiol&amp;title=Characterizing+the+avian+gut+microbiota:+membership,+driving+influences,+and+potential+function&amp;author=DW+Waite&amp;author=MW+Taylor&amp;volume=5&amp;publication_year=2014&amp;pages=223&amp;pmid=24904538&amp;doi=10.3389/fmicb.2014.00223&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B5\">5. <span class=\"mixed-citation\">Adams RI, Bateman AC, Bik HM, Meadow JF. \n2015. \n<span class=\"ref-title\">Microbiota of the indoor environment: a meta-analysis</span>. <span class=\"ref-journal\">Microbiome</span>\n<span class=\"ref-vol\">3</span>:49. doi:10.1186/s40168-015-0108-3. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4604073/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26459172\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2Fs40168-015-0108-3\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Microbiota+of+the+indoor+environment:+a+meta-analysis&amp;author=RI+Adams&amp;author=AC+Bateman&amp;author=HM+Bik&amp;author=JF+Meadow&amp;volume=3&amp;publication_year=2015&amp;pages=49&amp;pmid=26459172&amp;doi=10.1186/s40168-015-0108-3&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B6\">6. <span class=\"mixed-citation\">Allen HK, Looft T, Bayles DO, Humphrey S, Levine UY, Alt D, Stanton TB. \n2011. \n<span class=\"ref-title\">Antibiotics in feed induce prophages in swine fecal microbiomes</span>. <span class=\"ref-journal\">mBio</span>\n<span class=\"ref-vol\">2</span>:e00260-11. doi:10.1128/mBio.00260-11. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3225969/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22128350\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FmBio.00260-11\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Antibiotics+in+feed+induce+prophages+in+swine+fecal+microbiomes&amp;author=HK+Allen&amp;author=T+Looft&amp;author=DO+Bayles&amp;author=S+Humphrey&amp;author=UY+Levine&amp;volume=2&amp;publication_year=2011&amp;pages=e00260-11&amp;pmid=22128350&amp;doi=10.1128/mBio.00260-11&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B7\">7. <span class=\"mixed-citation\">Looft T, Allen HK, Cantarel BL, Levine UY, Bayles DO, Alt DP, Henrissat B, Stanton TB. \n2014. \n<span class=\"ref-title\">Bacteria, phages and pigs: the effects of in-feed antibiotics on the microbiome at different gut locations</span>. <span class=\"ref-journal\">ISME J</span>\n<span class=\"ref-vol\">8</span>:1566&#x02013;1576. doi:10.1038/ismej.2014.12. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4817603/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24522263\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fismej.2014.12\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=ISME+J&amp;title=Bacteria,+phages+and+pigs:+the+effects+of+in-feed+antibiotics+on+the+microbiome+at+different+gut+locations&amp;author=T+Looft&amp;author=HK+Allen&amp;author=BL+Cantarel&amp;author=UY+Levine&amp;author=DO+Bayles&amp;volume=8&amp;publication_year=2014&amp;pages=1566-1576&amp;pmid=24522263&amp;doi=10.1038/ismej.2014.12&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B8\">8. <span class=\"mixed-citation\">Looft T, Allen HK, Casey TA, Alt DP, Stanton TB. \n2014. \n<span class=\"ref-title\">Carbadox has both temporary and lasting effects on the swine gut microbiota</span>. <span class=\"ref-journal\">Front Microbiol</span>\n<span class=\"ref-vol\">5</span>:276. doi:10.3389/fmicb.2014.00276. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4050737/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24959163\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3389%2Ffmicb.2014.00276\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front+Microbiol&amp;title=Carbadox+has+both+temporary+and+lasting+effects+on+the+swine+gut+microbiota&amp;author=T+Looft&amp;author=HK+Allen&amp;author=TA+Casey&amp;author=DP+Alt&amp;author=TB+Stanton&amp;volume=5&amp;publication_year=2014&amp;pages=276&amp;pmid=24959163&amp;doi=10.3389/fmicb.2014.00276&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B9\">9. <span class=\"mixed-citation\">Looft T, Johnson TA, Allen HK, Bayles DO, Alt DP, Stedtfeld RD, Sul WJ, Stedtfeld TM, Chai B, Cole JR, Hashsham SA, Tiedje JM, Stanton TB. \n2012. \n<span class=\"ref-title\">In-feed antibiotic effects on the swine intestinal microbiome</span>. <span class=\"ref-journal\">Proc Natl Acad Sci U S A</span>\n<span class=\"ref-vol\">109</span>:1691&#x02013;1696. doi:10.1073/pnas.1120238109. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3277147/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22307632\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1073%2Fpnas.1120238109\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=In-feed+antibiotic+effects+on+the+swine+intestinal+microbiome&amp;author=T+Looft&amp;author=TA+Johnson&amp;author=HK+Allen&amp;author=DO+Bayles&amp;author=DP+Alt&amp;volume=109&amp;publication_year=2012&amp;pages=1691-1696&amp;pmid=22307632&amp;doi=10.1073/pnas.1120238109&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B10\">10. <span class=\"mixed-citation\">Holman DB, Ch&#x000e9;nier MR. \n2014. \n<span class=\"ref-title\">Temporal changes and the effect of subtherapeutic concentrations of antibiotics in the gut microbiota of swine</span>. <span class=\"ref-journal\">FEMS Microbiol Ecol</span>\n<span class=\"ref-vol\">90</span>:599&#x02013;608. doi:10.1111/1574-6941.12419. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25187398\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2F1574-6941.12419\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=FEMS+Microbiol+Ecol&amp;title=Temporal+changes+and+the+effect+of+subtherapeutic+concentrations+of+antibiotics+in+the+gut+microbiota+of+swine&amp;author=DB+Holman&amp;author=MR+Ch&#x000e9;nier&amp;volume=90&amp;publication_year=2014&amp;pages=599-608&amp;pmid=25187398&amp;doi=10.1111/1574-6941.12419&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B11\">11. <span class=\"mixed-citation\">Unno T, Kim JM, Guevarra RB, Nguyen SG. \n2015. \n<span class=\"ref-title\">Effects of antibiotic growth promoter and characterization of ecological succession in swine gut microbiota</span>. <span class=\"ref-journal\">J Microbiol Biotechnol</span>\n<span class=\"ref-vol\">25</span>:431&#x02013;438. doi:10.4014/jmb.1408.08063. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25370726\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.4014%2Fjmb.1408.08063\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Microbiol+Biotechnol&amp;title=Effects+of+antibiotic+growth+promoter+and+characterization+of+ecological+succession+in+swine+gut+microbiota&amp;author=T+Unno&amp;author=JM+Kim&amp;author=RB+Guevarra&amp;author=SG+Nguyen&amp;volume=25&amp;publication_year=2015&amp;pages=431-438&amp;pmid=25370726&amp;doi=10.4014/jmb.1408.08063&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B12\">12. <span class=\"mixed-citation\">Levesque CL, Hooda S, Swanson KS, de Lange K. \n2014. \n<span class=\"ref-title\">Alterations in ileal mucosa bacteria related to diet complexity and growth performance in young pigs</span>. <span class=\"ref-journal\">PLoS One</span>\n<span class=\"ref-vol\">9</span>:e108472. doi:10.1371/journal.pone.0108472. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4172762/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25247930\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0108472\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Alterations+in+ileal+mucosa+bacteria+related+to+diet+complexity+and+growth+performance+in+young+pigs&amp;author=CL+Levesque&amp;author=S+Hooda&amp;author=KS+Swanson&amp;author=K+de+Lange&amp;volume=9&amp;publication_year=2014&amp;pages=e108472&amp;pmid=25247930&amp;doi=10.1371/journal.pone.0108472&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B13\">13. <span class=\"mixed-citation\">Luo YH, Yang C, Wright AD, He J, Chen DW. \n2015. \n<span class=\"ref-title\">Responses in ileal and cecal bacteria to low and high amylose/amylopectin ratio diets in growing pigs</span>. <span class=\"ref-journal\">Appl Microbiol Biotechnol</span>\n<span class=\"ref-vol\">99</span>:10627&#x02013;10638. doi:10.1007/s00253-015-6917-2. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26318448\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs00253-015-6917-2\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl+Microbiol+Biotechnol&amp;title=Responses+in+ileal+and+cecal+bacteria+to+low+and+high+amylose/amylopectin+ratio+diets+in+growing+pigs&amp;author=YH+Luo&amp;author=C+Yang&amp;author=AD+Wright&amp;author=J+He&amp;author=DW+Chen&amp;volume=99&amp;publication_year=2015&amp;pages=10627-10638&amp;pmid=26318448&amp;doi=10.1007/s00253-015-6917-2&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B14\">14. <span class=\"mixed-citation\">Mann E, Schmitz-Esser S, Zebeli Q, Wagner M, Ritzmann M, Metzler-Zebeli BU. \n2014. \n<span class=\"ref-title\">Mucosa-associated bacterial microbiome of the gastrointestinal tract of weaned pigs and dynamics linked to dietary calcium-phosphorus</span>. <span class=\"ref-journal\">PLoS One</span>\n<span class=\"ref-vol\">9</span>:e86950. doi:10.1371/journal.pone.0086950. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3900689/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24466298\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0086950\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Mucosa-associated+bacterial+microbiome+of+the+gastrointestinal+tract+of+weaned+pigs+and+dynamics+linked+to+dietary+calcium-phosphorus&amp;author=E+Mann&amp;author=S+Schmitz-Esser&amp;author=Q+Zebeli&amp;author=M+Wagner&amp;author=M+Ritzmann&amp;volume=9&amp;publication_year=2014&amp;pages=e86950&amp;pmid=24466298&amp;doi=10.1371/journal.pone.0086950&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B15\">15. <span class=\"mixed-citation\">Burrough ER, Arruda BL, Patience JF, Plummer PJ. \n2015. \n<span class=\"ref-title\">Alterations in the colonic microbiota of pigs associated with feeding distillers dried grains with solubles</span>. <span class=\"ref-journal\">PLoS One</span>\n<span class=\"ref-vol\">10</span>:e0141337. doi:10.1371/journal.pone.0141337. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4640664/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26555787\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0141337\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Alterations+in+the+colonic+microbiota+of+pigs+associated+with+feeding+distillers+dried+grains+with+solubles&amp;author=ER+Burrough&amp;author=BL+Arruda&amp;author=JF+Patience&amp;author=PJ+Plummer&amp;volume=10&amp;publication_year=2015&amp;pages=e0141337&amp;pmid=26555787&amp;doi=10.1371/journal.pone.0141337&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B16\">16. <span class=\"mixed-citation\">Metzler-Zebeli BU, Schmitz-Esser S, Mann E, Gr&#x000fc;ll D, Molnar T, Zebeli Q. \n2015. \n<span class=\"ref-title\">Adaptation of the cecal bacterial microbiome of growing pigs in response to resistant starch type 4</span>. <span class=\"ref-journal\">Appl Environ Microbiol</span>\n<span class=\"ref-vol\">81</span>:8489&#x02013;8499. doi:10.1128/AEM.02756-15. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4644661/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26431973\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.02756-15\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl+Environ+Microbiol&amp;title=Adaptation+of+the+cecal+bacterial+microbiome+of+growing+pigs+in+response+to+resistant+starch+type+4&amp;author=BU+Metzler-Zebeli&amp;author=S+Schmitz-Esser&amp;author=E+Mann&amp;author=D+Gr&#x000fc;ll&amp;author=T+Molnar&amp;volume=81&amp;publication_year=2015&amp;pages=8489-8499&amp;pmid=26431973&amp;doi=10.1128/AEM.02756-15&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B17\">17. <span class=\"mixed-citation\">Hedegaard CJ, Strube ML, Hansen MB, Lindved BK, Lihme A, Boye M, Heegaard PM. \n2016. \n<span class=\"ref-title\">Natural pig plasma immunoglobulins have antibacterial effects: potential for use as feed supplement for treatment of intestinal infections in pigs</span>. <span class=\"ref-journal\">PLoS One</span>\n<span class=\"ref-vol\">11</span>:e0147373. doi:10.1371/journal.pone.0147373. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4744083/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26824607\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0147373\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Natural+pig+plasma+immunoglobulins+have+antibacterial+effects:+potential+for+use+as+feed+supplement+for+treatment+of+intestinal+infections+in+pigs&amp;author=CJ+Hedegaard&amp;author=ML+Strube&amp;author=MB+Hansen&amp;author=BK+Lindved&amp;author=A+Lihme&amp;volume=11&amp;publication_year=2016&amp;pages=e0147373&amp;pmid=26824607&amp;doi=10.1371/journal.pone.0147373&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B18\">18. <span class=\"mixed-citation\">Bearson SM, Allen HK, Bearson BL, Looft T, Brunelle BW, Kich JD, Tuggle CK, Bayles DO, Alt D, Levine UY, Stanton TB. \n2013. \n<span class=\"ref-title\">Profiling the gastrointestinal microbiota in response to <em>Salmonella</em>: low versus high <em>Salmonella</em> shedding in the natural porcine host</span>. <span class=\"ref-journal\">Infect Genet Evol</span>\n<span class=\"ref-vol\">16</span>:330&#x02013;340. doi:10.1016/j.meegid.2013.03.022. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23535116\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.meegid.2013.03.022\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Infect+Genet+Evol&amp;title=Profiling+the+gastrointestinal+microbiota+in+response+to+Salmonella:+low+versus+high+Salmonella+shedding+in+the+natural+porcine+host&amp;author=SM+Bearson&amp;author=HK+Allen&amp;author=BL+Bearson&amp;author=T+Looft&amp;author=BW+Brunelle&amp;volume=16&amp;publication_year=2013&amp;pages=330-340&amp;pmid=23535116&amp;doi=10.1016/j.meegid.2013.03.022&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B19\">19. <span class=\"mixed-citation\">Kuczynski J, Liu Z, Lozupone C, McDonald D, Fierer N, Knight R. \n2010. \n<span class=\"ref-title\">Microbial community resemblance methods differ in their ability to detect biologically relevant patterns</span>. <span class=\"ref-journal\">Nat Methods</span>\n<span class=\"ref-vol\">7</span>:813&#x02013;819. doi:10.1038/nmeth.1499. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2948603/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20818378\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnmeth.1499\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=Microbial+community+resemblance+methods+differ+in+their+ability+to+detect+biologically+relevant+patterns&amp;author=J+Kuczynski&amp;author=Z+Liu&amp;author=C+Lozupone&amp;author=D+McDonald&amp;author=N+Fierer&amp;volume=7&amp;publication_year=2010&amp;pages=813-819&amp;pmid=20818378&amp;doi=10.1038/nmeth.1499&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B20\">20. <span class=\"mixed-citation\">NRC \n2012. \n<span class=\"ref-journal\">Nutrient requirements of swine</span>, 11th ed. <span class=\"ref-journal\">The National</span>\nAcademies Press, Washington, DC. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Nutrient+requirements+of+swine&amp;title=The+National&amp;publication_year=2012&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B21\">21. <span class=\"mixed-citation\">Wu GD, Chen J, Hoffmann C, Bittinger K, Chen YY, Keilbaugh SA, Bewtra M, Knights D, Walters WA, Knight R, Sinha R, Gilroy E, Gupta K, Baldassano R, Nessel L, Li H, Bushman FD, Lewis JD. \n2011. \n<span class=\"ref-title\">Linking long-term dietary patterns with gut microbial enterotypes</span>. <span class=\"ref-journal\">Science</span>\n<span class=\"ref-vol\">334</span>:105&#x02013;108. doi:10.1126/science.1208344. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3368382/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21885731\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1126%2Fscience.1208344\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Linking+long-term+dietary+patterns+with+gut+microbial+enterotypes&amp;author=GD+Wu&amp;author=J+Chen&amp;author=C+Hoffmann&amp;author=K+Bittinger&amp;author=YY+Chen&amp;volume=334&amp;publication_year=2011&amp;pages=105-108&amp;pmid=21885731&amp;doi=10.1126/science.1208344&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B22\">22. <span class=\"mixed-citation\">De Filippis F, Pellegrini N, Vannini L, Jeffery IB, La Storia A, Laghi L, Serrazanetti DI, Di Cagno R, Ferrocino I, Lazzi C, Turroni S, Cocolin L, Brigidi P, Neviani E, Gobbetti M, O&#x02019;Toole PW, Ercolini D. \n2016. \n<span class=\"ref-title\">High-level adherence to a Mediterranean diet beneficially impacts the gut microbiota and associated metabolome</span>. <span class=\"ref-journal\">Gut</span>\n<span class=\"ref-vol\">65</span>:1812&#x02013;1821. doi:10.1136/gutjnl-2015-309957. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26416813\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1136%2Fgutjnl-2015-309957\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=High-level+adherence+to+a+Mediterranean+diet+beneficially+impacts+the+gut+microbiota+and+associated+metabolome&amp;author=F+De+Filippis&amp;author=N+Pellegrini&amp;author=L+Vannini&amp;author=IB+Jeffery&amp;author=A+La+Storia&amp;volume=65&amp;publication_year=2016&amp;pages=1812-1821&amp;pmid=26416813&amp;doi=10.1136/gutjnl-2015-309957&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B23\">23. <span class=\"mixed-citation\">Flint HJ, Bayer EA. \n2008. \n<span class=\"ref-title\">Plant cell wall breakdown by anaerobic microorganisms from the mammalian digestive tract</span>. <span class=\"ref-journal\">Ann N Y Acad Sci</span>\n<span class=\"ref-vol\">1125</span>:280&#x02013;288. doi:10.1196/annals.1419.022. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n18378598\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1196%2Fannals.1419.022\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Ann+N+Y+Acad+Sci&amp;title=Plant+cell+wall+breakdown+by+anaerobic+microorganisms+from+the+mammalian+digestive+tract&amp;author=HJ+Flint&amp;author=EA+Bayer&amp;volume=1125&amp;publication_year=2008&amp;pages=280-288&amp;pmid=18378598&amp;doi=10.1196/annals.1419.022&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B24\">24. <span class=\"mixed-citation\">Kim M, Kim J, Kuehn LA, Bono JL, Berry ED, Kalchayanand N, Freetly HC, Benson AK, Wells JE. \n2014. \n<span class=\"ref-title\">Investigation of bacterial diversity in the feces of cattle fed different diets</span>. <span class=\"ref-journal\">J Anim Sci</span>\n<span class=\"ref-vol\">92</span>:683&#x02013;694. doi:10.2527/jas.2013-6841. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24352967\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.2527%2Fjas.2013-6841\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Anim+Sci&amp;title=Investigation+of+bacterial+diversity+in+the+feces+of+cattle+fed+different+diets&amp;author=M+Kim&amp;author=J+Kim&amp;author=LA+Kuehn&amp;author=JL+Bono&amp;author=ED+Berry&amp;volume=92&amp;publication_year=2014&amp;pages=683-694&amp;pmid=24352967&amp;doi=10.2527/jas.2013-6841&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B25\">25. <span class=\"mixed-citation\">Moeller AH, Degnan PH, Pusey AE, Wilson ML, Hahn BH, Ochman H. \n2012. \n<span class=\"ref-title\">Chimpanzees and humans harbour compositionally similar gut enterotypes</span>. <span class=\"ref-journal\">Nat Commun</span>\n<span class=\"ref-vol\">3</span>:1179. doi:10.1038/ncomms2159. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3520023/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23149725\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fncomms2159\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=Chimpanzees+and+humans+harbour+compositionally+similar+gut+enterotypes&amp;author=AH+Moeller&amp;author=PH+Degnan&amp;author=AE+Pusey&amp;author=ML+Wilson&amp;author=BH+Hahn&amp;volume=3&amp;publication_year=2012&amp;pages=1179&amp;pmid=23149725&amp;doi=10.1038/ncomms2159&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B26\">26. <span class=\"mixed-citation\">Huo W, Zhu W, Mao S. \n2014. \n<span class=\"ref-title\">Impact of subacute ruminal acidosis on the diversity of liquid and solid-associated bacteria in the rumen of goats</span>. <span class=\"ref-journal\">World J Microbiol Biotechnol</span>\n<span class=\"ref-vol\">30</span>:669&#x02013;680. doi:10.1007/s11274-013-1489-8. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24068532\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs11274-013-1489-8\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=World+J+Microbiol+Biotechnol&amp;title=Impact+of+subacute+ruminal+acidosis+on+the+diversity+of+liquid+and+solid-associated+bacteria+in+the+rumen+of+goats&amp;author=W+Huo&amp;author=W+Zhu&amp;author=S+Mao&amp;volume=30&amp;publication_year=2014&amp;pages=669-680&amp;pmid=24068532&amp;doi=10.1007/s11274-013-1489-8&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B27\">27. <span class=\"mixed-citation\">Castro-Carrera T, Toral PG, Frutos P, McEwan NR, Herv&#x000e1;s G, Abecia L, Pinloche E, Girdwood SE, Belenguer A. \n2014. \n<span class=\"ref-title\">Rumen bacterial community evaluated by 454 pyrosequencing and terminal restriction fragment length polymorphism analyses in dairy sheep fed marine algae</span>. <span class=\"ref-journal\">J Dairy Sci</span>\n<span class=\"ref-vol\">97</span>:1661&#x02013;1669. doi:10.3168/jds.2013-7243. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24440247\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3168%2Fjds.2013-7243\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Dairy+Sci&amp;title=Rumen+bacterial+community+evaluated+by+454+pyrosequencing+and+terminal+restriction+fragment+length+polymorphism+analyses+in+dairy+sheep+fed+marine+algae&amp;author=T+Castro-Carrera&amp;author=PG+Toral&amp;author=P+Frutos&amp;author=NR+McEwan&amp;author=G+Herv&#x000e1;s&amp;volume=97&amp;publication_year=2014&amp;pages=1661-1669&amp;pmid=24440247&amp;doi=10.3168/jds.2013-7243&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B28\">28. <span class=\"mixed-citation\">Biddle A, Stewart L, Blanchard J, Leschine S. \n2013. \n<span class=\"ref-title\">Untangling the genetic basis of fibrolytic specialization by <em>Lachnospiraceae</em> and <em>Ruminococcaceae</em> in diverse gut communities</span>. <span class=\"ref-journal\">Diversity</span>\n<span class=\"ref-vol\">5</span>:627&#x02013;640. doi:10.3390/d5030627. [<a href=\"//dx.doi.org/10.3390%2Fd5030627\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Diversity&amp;title=Untangling+the+genetic+basis+of+fibrolytic+specialization+by+Lachnospiraceae+and+Ruminococcaceae+in+diverse+gut+communities&amp;author=A+Biddle&amp;author=L+Stewart&amp;author=J+Blanchard&amp;author=S+Leschine&amp;volume=5&amp;publication_year=2013&amp;pages=627-640&amp;doi=10.3390/d5030627&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B29\">29. <span class=\"mixed-citation\">Vital M, Howe AC, Tiedje JM. \n2014. \n<span class=\"ref-title\">Revealing the bacterial butyrate synthesis pathways by analyzing (meta) genomic data</span>. <span class=\"ref-journal\">mBio</span>\n<span class=\"ref-vol\">5</span>:e00889-14. doi:10.1128/mBio.00889-14. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3994512/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24757212\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FmBio.00889-14\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Revealing+the+bacterial+butyrate+synthesis+pathways+by+analyzing+(meta)+genomic+data&amp;author=M+Vital&amp;author=AC+Howe&amp;author=JM+Tiedje&amp;volume=5&amp;publication_year=2014&amp;pages=e00889-14&amp;pmid=24757212&amp;doi=10.1128/mBio.00889-14&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B30\">30. <span class=\"mixed-citation\">Hamer HM, Jonkers D, Venema K, Vanhoutvin S, Troost FJ, Brummer RJ. \n2008. \n<span class=\"ref-title\">Review article: the role of butyrate on colonic function</span>. <span class=\"ref-journal\">Aliment Pharmacol Ther</span>\n<span class=\"ref-vol\">27</span>:104&#x02013;119. doi:10.1111/j.1365-2036.2007.03562.x. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17973645\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1365-2036.2007.03562.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Aliment+Pharmacol+Ther&amp;title=Review+article:+the+role+of+butyrate+on+colonic+function&amp;author=HM+Hamer&amp;author=D+Jonkers&amp;author=K+Venema&amp;author=S+Vanhoutvin&amp;author=FJ+Troost&amp;volume=27&amp;publication_year=2008&amp;pages=104-119&amp;pmid=17973645&amp;doi=10.1111/j.1365-2036.2007.03562.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B31\">31. <span class=\"mixed-citation\">Tran TH, Boudry C, Everaert N, Th&#x000e9;wis A, Portetelle D, Daube G, Nezer C, Taminiau B, Bindelle J. \n2016. \n<span class=\"ref-title\">Adding mucins to an <em>in vitro</em> batch fermentation model of the large intestine induces changes in microbial population isolated from porcine feces depending on the substrate</span>. <span class=\"ref-journal\">FEMS Microbiol Ecol</span>\n<span class=\"ref-vol\">92</span>:fiv165. doi:10.1093/femsec/fiv165. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26691596\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Ffemsec%2Ffiv165\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=FEMS+Microbiol+Ecol&amp;title=Adding+mucins+to+an+in+vitro+batch+fermentation+model+of+the+large+intestine+induces+changes+in+microbial+population+isolated+from+porcine+feces+depending+on+the+substrate&amp;author=TH+Tran&amp;author=C+Boudry&amp;author=N+Everaert&amp;author=A+Th&#x000e9;wis&amp;author=D+Portetelle&amp;volume=92&amp;publication_year=2016&amp;pages=fiv165&amp;pmid=26691596&amp;doi=10.1093/femsec/fiv165&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B32\">32. <span class=\"mixed-citation\">Sun Y, Zhou L, Fang L, Su Y, Zhu W. \n2015. \n<span class=\"ref-title\">Responses in colonic microbial community and gene expression of pigs to a long-term high resistant starch diet</span>. <span class=\"ref-journal\">Front Microbiol</span>\n<span class=\"ref-vol\">6</span>:877. doi:10.3389/fmicb.2015.00877. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4548152/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26379652\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3389%2Ffmicb.2015.00877\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front+Microbiol&amp;title=Responses+in+colonic+microbial+community+and+gene+expression+of+pigs+to+a+long-term+high+resistant+starch+diet&amp;author=Y+Sun&amp;author=L+Zhou&amp;author=L+Fang&amp;author=Y+Su&amp;author=W+Zhu&amp;volume=6&amp;publication_year=2015&amp;pages=877&amp;pmid=26379652&amp;doi=10.3389/fmicb.2015.00877&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B33\">33. <span class=\"mixed-citation\">Graf J.\n2014. \n<span class=\"ref-title\">The family <em>Rikenellaceae</em></span>, p 857&#x02013;859. <em>In</em>\n<span class=\"ref-journal\">The prokaryotes</span>. Springer, Berlin, Germany. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=The+prokaryotes&amp;author=J+Graf&amp;publication_year=2014&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B34\">34. <span class=\"mixed-citation\">Su XL, Tian Q, Zhang J, Yuan XZ, Shi XS, Guo RB, Qiu YL. \n2014. \n<span class=\"ref-title\"><em>Acetobacteroides hydrogenigenes</em> gen. nov., sp. nov., an anaerobic hydrogen-producing bacterium in the family <em>Rikenellaceae</em> isolated from a reed swamp</span>. <span class=\"ref-journal\">Int J Syst Evol Microbiol</span>\n<span class=\"ref-vol\">64</span>:2986&#x02013;2991. doi:10.1099/ijs.0.063917-0. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24899658\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1099%2Fijs.0.063917-0\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int+J+Syst+Evol+Microbiol&amp;title=Acetobacteroides+hydrogenigenes+gen.+nov.,+sp.+nov.,+an+anaerobic+hydrogen-producing+bacterium+in+the+family+Rikenellaceae+isolated+from+a+reed+swamp&amp;author=XL+Su&amp;author=Q+Tian&amp;author=J+Zhang&amp;author=XZ+Yuan&amp;author=XS+Shi&amp;volume=64&amp;publication_year=2014&amp;pages=2986-2991&amp;pmid=24899658&amp;doi=10.1099/ijs.0.063917-0&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B35\">35. <span class=\"mixed-citation\">Frese SA, Parker K, Calvert CC, Mills DA. \n2015. \n<span class=\"ref-title\">Diet shapes the gut microbiome of pigs during nursing and weaning</span>. <span class=\"ref-journal\">Microbiome</span>\n<span class=\"ref-vol\">3</span>:28. doi:10.1186/s40168-015-0091-8. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4499176/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26167280\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2Fs40168-015-0091-8\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Diet+shapes+the+gut+microbiome+of+pigs+during+nursing+and+weaning&amp;author=SA+Frese&amp;author=K+Parker&amp;author=CC+Calvert&amp;author=DA+Mills&amp;volume=3&amp;publication_year=2015&amp;pages=28&amp;pmid=26167280&amp;doi=10.1186/s40168-015-0091-8&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B36\">36. <span class=\"mixed-citation\">van Winsen RL, Urlings BA, Lipman LJ, Snijders JM, Keuzenkamp D, Verheijden JH, van Knapen F. \n2001. \n<span class=\"ref-title\">Effect of fermented feed on the microbial population of the gastrointestinal tracts of pigs</span>. <span class=\"ref-journal\">Appl Environ Microbiol</span>\n<span class=\"ref-vol\">67</span>:3071&#x02013;3076. doi:10.1128/AEM.67.7.3071-3076.2001. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC92983/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n11425724\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.67.7.3071-3076.2001\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl+Environ+Microbiol&amp;title=Effect+of+fermented+feed+on+the+microbial+population+of+the+gastrointestinal+tracts+of+pigs&amp;author=RL+van+Winsen&amp;author=BA+Urlings&amp;author=LJ+Lipman&amp;author=JM+Snijders&amp;author=D+Keuzenkamp&amp;volume=67&amp;publication_year=2001&amp;pages=3071-3076&amp;pmid=11425724&amp;doi=10.1128/AEM.67.7.3071-3076.2001&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B37\">37. <span class=\"mixed-citation\">Walter J.\n2008. \n<span class=\"ref-title\">Ecological role of lactobacilli in the gastrointestinal tract: implications for fundamental and biomedical research</span>. <span class=\"ref-journal\">Appl Environ Microbiol</span>\n<span class=\"ref-vol\">74</span>:4985&#x02013;4996. doi:10.1128/AEM.00753-08. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2519286/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n18539818\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.00753-08\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl+Environ+Microbiol&amp;title=Ecological+role+of+lactobacilli+in+the+gastrointestinal+tract:+implications+for+fundamental+and+biomedical+research&amp;author=J+Walter&amp;volume=74&amp;publication_year=2008&amp;pages=4985-4996&amp;pmid=18539818&amp;doi=10.1128/AEM.00753-08&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B38\">38. <span class=\"mixed-citation\">Tannock GW.\n2004. \n<span class=\"ref-title\">A special fondness for lactobacilli</span>. <span class=\"ref-journal\">Appl Environ Microbiol</span>\n<span class=\"ref-vol\">70</span>:3189&#x02013;3194. doi:10.1128/AEM.70.6.3189-3194.2004. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC427720/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n15184111\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.70.6.3189-3194.2004\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl+Environ+Microbiol&amp;title=A+special+fondness+for+lactobacilli&amp;author=GW+Tannock&amp;volume=70&amp;publication_year=2004&amp;pages=3189-3194&amp;pmid=15184111&amp;doi=10.1128/AEM.70.6.3189-3194.2004&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B39\">39. <span class=\"mixed-citation\">Heo JM, Opapeju FO, Pluske JR, Kim JC, Hampson DJ, Nyachoti CM. \n2013. \n<span class=\"ref-title\">Gastrointestinal health and function in weaned pigs: a review of feeding strategies to control post-weaning diarrhoea without using in-feed antimicrobial compounds</span>. <span class=\"ref-journal\">J Anim Physiol Anim Nutr</span>\n<span class=\"ref-vol\">97</span>:207&#x02013;237. doi:10.1111/j.1439-0396.2012.01284.x. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22416941\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1439-0396.2012.01284.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Anim+Physiol+Anim+Nutr&amp;title=Gastrointestinal+health+and+function+in+weaned+pigs:+a+review+of+feeding+strategies+to+control+post-weaning+diarrhoea+without+using+in-feed+antimicrobial+compounds&amp;author=JM+Heo&amp;author=FO+Opapeju&amp;author=JR+Pluske&amp;author=JC+Kim&amp;author=DJ+Hampson&amp;volume=97&amp;publication_year=2013&amp;pages=207-237&amp;doi=10.1111/j.1439-0396.2012.01284.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B40\">40. <span class=\"mixed-citation\">Leser TD, M&#x000f8;lbak L. \n2009. \n<span class=\"ref-title\">Better living through microbial action: the benefits of the mammalian gastrointestinal microbiota on the host</span>. <span class=\"ref-journal\">Environ Microbiol</span>\n<span class=\"ref-vol\">11</span>:2194&#x02013;2206. doi:10.1111/j.1462-2920.2009.01941.x. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19737302\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1462-2920.2009.01941.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Environ+Microbiol&amp;title=Better+living+through+microbial+action:+the+benefits+of+the+mammalian+gastrointestinal+microbiota+on+the+host&amp;author=TD+Leser&amp;author=L+M&#x000f8;lbak&amp;volume=11&amp;publication_year=2009&amp;pages=2194-2206&amp;pmid=19737302&amp;doi=10.1111/j.1462-2920.2009.01941.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B41\">41. <span class=\"mixed-citation\">Hiergeist A, Reischl U; Priority Program 1656 Intestinal Microbiota Consortium/ quality assessment participants, Gessner A. \n2016. \n<span class=\"ref-title\">Multicenter quality assessment of 16S ribosomal DNA-sequencing for microbiome analyses reveals high inter-center variability</span>. <span class=\"ref-journal\">Int J Med Microbiol</span>\n<span class=\"ref-vol\">306</span>:334&#x02013;342. doi:10.1016/j.ijmm.2016.03.005. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27052158\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.ijmm.2016.03.005\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int+J+Med+Microbiol&amp;title=Multicenter+quality+assessment+of+16S+ribosomal+DNA-sequencing+for+microbiome+analyses+reveals+high+inter-center+variability&amp;author=A+Hiergeist&amp;author=U+Reischl&amp;author=A+Gessner&amp;volume=306&amp;publication_year=2016&amp;pages=334-342&amp;pmid=27052158&amp;doi=10.1016/j.ijmm.2016.03.005&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B42\">42. <span class=\"mixed-citation\">Klindworth A, Pruesse E, Schweer T, Peplies J, Quast C, Horn M, Gl&#x000f6;ckner FO. \n2013. \n<span class=\"ref-title\">Evaluation of general 16S ribosomal RNA gene PCR primers for classical and next-generation sequencing-based diversity studies</span>. <span class=\"ref-journal\">Nucleic Acids Res</span>\n<span class=\"ref-vol\">41</span>:e1. doi:10.1093/nar/gks808. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3592464/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22933715\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgks808\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Evaluation+of+general+16S+ribosomal+RNA+gene+PCR+primers+for+classical+and+next-generation+sequencing-based+diversity+studies&amp;author=A+Klindworth&amp;author=E+Pruesse&amp;author=T+Schweer&amp;author=J+Peplies&amp;author=C+Quast&amp;volume=41&amp;publication_year=2013&amp;pages=e1&amp;pmid=22933715&amp;doi=10.1093/nar/gks808&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B43\">43. <span class=\"mixed-citation\">Hayashi H, Sakamoto M, Benno Y. \n2004. \n<span class=\"ref-title\">Evaluation of three different forward primers by terminal restriction fragment length polymorphism analysis for determination of fecal <em>Bifidobacterium</em> spp. in healthy subjects</span>. <span class=\"ref-journal\">Microbiol Immunol</span>\n<span class=\"ref-vol\">48</span>:1&#x02013;6. doi:10.1111/j.1348-0421.2004.tb03481.x. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n14734852\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1348-0421.2004.tb03481.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Microbiol+Immunol&amp;title=Evaluation+of+three+different+forward+primers+by+terminal+restriction+fragment+length+polymorphism+analysis+for+determination+of+fecal+Bifidobacterium+spp.+in+healthy+subjects&amp;author=H+Hayashi&amp;author=M+Sakamoto&amp;author=Y+Benno&amp;volume=48&amp;publication_year=2004&amp;pages=1-6&amp;pmid=14734852&amp;doi=10.1111/j.1348-0421.2004.tb03481.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B44\">44. <span class=\"mixed-citation\">Walker AW, Martin JC, Scott P, Parkhill J, Flint HJ, Scott KP. \n2015. \n<span class=\"ref-title\">16S rRNA gene-based profiling of the human infant gut microbiota is strongly influenced by sample processing and PCR primer choice</span>. <span class=\"ref-journal\">Microbiome</span>\n<span class=\"ref-vol\">3</span>:26. doi:10.1186/s40168-015-0087-4. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4482049/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26120470\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2Fs40168-015-0087-4\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=16S+rRNA+gene-based+profiling+of+the+human+infant+gut+microbiota+is+strongly+influenced+by+sample+processing+and+PCR+primer+choice&amp;author=AW+Walker&amp;author=JC+Martin&amp;author=P+Scott&amp;author=J+Parkhill&amp;author=HJ+Flint&amp;volume=3&amp;publication_year=2015&amp;pages=26&amp;pmid=26120470&amp;doi=10.1186/s40168-015-0087-4&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B45\">45. <span class=\"mixed-citation\">Gerzova L, Babak V, Sedlar K, Faldynova M, Videnska P, Cejkova D, Jensen AN, Denis M, Kerouanton A, Ricci A, Cibin V, &#x000d6;sterberg J, Rychlik I. \n2015. \n<span class=\"ref-title\">Characterization of antibiotic resistance gene abundance and microbiota composition in feces of organic and conventional pigs from four EU countries</span>. <span class=\"ref-journal\">PLoS One</span>\n<span class=\"ref-vol\">10</span>:e0132892. doi:10.1371/journal.pone.0132892. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4517930/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26218075\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0132892\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Characterization+of+antibiotic+resistance+gene+abundance+and+microbiota+composition+in+feces+of+organic+and+conventional+pigs+from+four+EU+countries&amp;author=L+Gerzova&amp;author=V+Babak&amp;author=K+Sedlar&amp;author=M+Faldynova&amp;author=P+Videnska&amp;volume=10&amp;publication_year=2015&amp;pages=e0132892&amp;pmid=26218075&amp;doi=10.1371/journal.pone.0132892&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B46\">46. <span class=\"mixed-citation\">Barton MD.\n2014. \n<span class=\"ref-title\">Impact of antibiotic use in the swine industry</span>. <span class=\"ref-journal\">Curr Opin Microbiol</span>\n<span class=\"ref-vol\">19</span>:9&#x02013;15. doi:10.1016/j.mib.2014.05.017. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24959754\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.mib.2014.05.017\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Microbiol&amp;title=Impact+of+antibiotic+use+in+the+swine+industry&amp;author=MD+Barton&amp;volume=19&amp;publication_year=2014&amp;pages=9-15&amp;pmid=24959754&amp;doi=10.1016/j.mib.2014.05.017&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B47\">47. <span class=\"mixed-citation\">Allen HK, Stanton TB. \n2014. \n<span class=\"ref-title\">Altered egos: antibiotic effects on food animal microbiomes</span>. <span class=\"ref-journal\">Annu Rev Microbiol</span>\n<span class=\"ref-vol\">68</span>:297&#x02013;315. doi:10.1146/annurev-micro-091213-113052. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25002091\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1146%2Fannurev-micro-091213-113052\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Microbiol&amp;title=Altered+egos:+antibiotic+effects+on+food+animal+microbiomes&amp;author=HK+Allen&amp;author=TB+Stanton&amp;volume=68&amp;publication_year=2014&amp;pages=297-315&amp;pmid=25002091&amp;doi=10.1146/annurev-micro-091213-113052&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B48\">48. <span class=\"mixed-citation\">Holman DB, Ch&#x000e9;nier MR. \n2013. \n<span class=\"ref-title\">Impact of subtherapeutic administration of tylosin and chlortetracycline on antimicrobial resistance in farrow-to-finish swine</span>. <span class=\"ref-journal\">FEMS Microbiol Ecol</span>\n<span class=\"ref-vol\">85</span>:1&#x02013;13. doi:10.1111/1574-6941.12093. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23397987\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2F1574-6941.12093\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=FEMS+Microbiol+Ecol&amp;title=Impact+of+subtherapeutic+administration+of+tylosin+and+chlortetracycline+on+antimicrobial+resistance+in+farrow-to-finish+swine&amp;author=DB+Holman&amp;author=MR+Ch&#x000e9;nier&amp;volume=85&amp;publication_year=2013&amp;pages=1-13&amp;pmid=23397987&amp;doi=10.1111/1574-6941.12093&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B49\">49. <span class=\"mixed-citation\">Allen HK, Levine UY, Looft T, Bandrick M, Casey TA. \n2013. \n<span class=\"ref-title\">Treatment, promotion, commotion: antibiotic alternatives in food-producing animals</span>. <span class=\"ref-journal\">Trends Microbiol</span>\n<span class=\"ref-vol\">21</span>:114&#x02013;119. doi:10.1016/j.tim.2012.11.001. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23473629\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.tim.2012.11.001\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Trends+Microbiol&amp;title=Treatment,+promotion,+commotion:+antibiotic+alternatives+in+food-producing+animals&amp;author=HK+Allen&amp;author=UY+Levine&amp;author=T+Looft&amp;author=M+Bandrick&amp;author=TA+Casey&amp;volume=21&amp;publication_year=2013&amp;pages=114-119&amp;pmid=23473629&amp;doi=10.1016/j.tim.2012.11.001&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B50\">50. <span class=\"mixed-citation\">Allen HK, Trachsel J, Looft T, Casey TA. \n2014. \n<span class=\"ref-title\">Finding alternatives to antibiotics</span>. <span class=\"ref-journal\">Ann N Y Acad Sci</span>\n<span class=\"ref-vol\">1323</span>:91&#x02013;100. doi:10.1111/nyas.12468. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24953233\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fnyas.12468\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Ann+N+Y+Acad+Sci&amp;title=Finding+alternatives+to+antibiotics&amp;author=HK+Allen&amp;author=J+Trachsel&amp;author=T+Looft&amp;author=TA+Casey&amp;volume=1323&amp;publication_year=2014&amp;pages=91-100&amp;pmid=24953233&amp;doi=10.1111/nyas.12468&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B51\">51. <span class=\"mixed-citation\">Holman DB, Ch&#x000e9;nier MR. \n2015. \n<span class=\"ref-title\">Antimicrobial use in swine production and its effect on the swine gut microbiota and antimicrobial resistance</span>. <span class=\"ref-journal\">Can J Microbiol</span>\n<span class=\"ref-vol\">61</span>:785&#x02013;798. doi:10.1139/cjm-2015-0239. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26414105\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1139%2Fcjm-2015-0239\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Can+J+Microbiol&amp;title=Antimicrobial+use+in+swine+production+and+its+effect+on+the+swine+gut+microbiota+and+antimicrobial+resistance&amp;author=DB+Holman&amp;author=MR+Ch&#x000e9;nier&amp;volume=61&amp;publication_year=2015&amp;pages=785-798&amp;pmid=26414105&amp;doi=10.1139/cjm-2015-0239&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B52\">52. <span class=\"mixed-citation\">Caporaso JG, Kuczynski J, Stombaugh J, Bittinger K, Bushman FD, Costello EK, Fierer N, Pe&#x000f1;a AG, Goodrich JK, Gordon JI, Huttley GA, Kelley ST, Knights D, Koenig JE, Ley RE, Lozupone CA, McDonald D, Muegge BD, Pirrung M, Reeder J, Sevinsky JR, Turnbaugh PJ, Walters WA, Widmann J, Yatsunenko T, Zaneveld J, Knight R. \n2010. \n<span class=\"ref-title\">QIIME allows analysis of high-throughput community sequencing data</span>. <span class=\"ref-journal\">Nat Methods</span>\n<span class=\"ref-vol\">7</span>:335&#x02013;336. doi:10.1038/nmeth.f.303. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3156573/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20383131\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1038%2Fnmeth.f.303\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=QIIME+allows+analysis+of+high-throughput+community+sequencing+data&amp;author=JG+Caporaso&amp;author=J+Kuczynski&amp;author=J+Stombaugh&amp;author=K+Bittinger&amp;author=FD+Bushman&amp;volume=7&amp;publication_year=2010&amp;pages=335-336&amp;pmid=20383131&amp;doi=10.1038/nmeth.f.303&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B53\">53. <span class=\"mixed-citation\">Aronesty E.\n2013. \n<span class=\"ref-title\">Comparison of sequencing utility programs</span>. <span class=\"ref-journal\">Open Bioinforma J</span>\n<span class=\"ref-vol\">7</span>:1&#x02013;8. doi:10.2174/1875036201307010001. [<a href=\"//dx.doi.org/10.2174%2F1875036201307010001\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Open+Bioinforma+J&amp;title=Comparison+of+sequencing+utility+programs&amp;author=E+Aronesty&amp;volume=7&amp;publication_year=2013&amp;pages=1-8&amp;doi=10.2174/1875036201307010001&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B54\">54. <span class=\"mixed-citation\">Quast C, Pruesse E, Yilmaz P, Gerken J, Schweer T, Yarza P, Peplies J, Gl&#x000f6;ckner FO. \n2013. \n<span class=\"ref-title\">The SILVA ribosomal RNA gene database project: improved data processing and Web-based tools</span>. <span class=\"ref-journal\">Nucleic Acids Res</span>\n<span class=\"ref-vol\">41</span>:D590&#x02013;D596. doi:10.1093/nar/gks1219. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3531112/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23193283\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgks1219\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+SILVA+ribosomal+RNA+gene+database+project:+improved+data+processing+and+Web-based+tools&amp;author=C+Quast&amp;author=E+Pruesse&amp;author=P+Yilmaz&amp;author=J+Gerken&amp;author=T+Schweer&amp;volume=41&amp;publication_year=2013&amp;pages=D590-D596&amp;pmid=23193283&amp;doi=10.1093/nar/gks1219&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B55\">55. <span class=\"mixed-citation\">Edgar RC.\n2010. \n<span class=\"ref-title\">Search and clustering orders of magnitude faster than BLAST</span>. <span class=\"ref-journal\">Bioinformatics</span>\n<span class=\"ref-vol\">26</span>:2460&#x02013;2461. doi:10.1093/bioinformatics/btq461. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20709691\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtq461\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Search+and+clustering+orders+of+magnitude+faster+than+BLAST&amp;author=RC+Edgar&amp;volume=26&amp;publication_year=2010&amp;pages=2460-2461&amp;pmid=20709691&amp;doi=10.1093/bioinformatics/btq461&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B56\">56. <span class=\"mixed-citation\">Segata N, Izard J, Waldron L, Gevers D, Miropolsky L, Garrett WS, Huttenhower C. \n2011. \n<span class=\"ref-title\">Metagenomic biomarker discovery and explanation</span>. <span class=\"ref-journal\">Genome Biol</span>\n<span class=\"ref-vol\">12</span>:R60. doi:10.1186/gb-2011-12-6-r60. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3218848/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21702898\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2Fgb-2011-12-6-r60\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Metagenomic+biomarker+discovery+and+explanation&amp;author=N+Segata&amp;author=J+Izard&amp;author=L+Waldron&amp;author=D+Gevers&amp;author=L+Miropolsky&amp;volume=12&amp;publication_year=2011&amp;pages=R60&amp;pmid=21702898&amp;doi=10.1186/gb-2011-12-6-r60&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B57\">57. <span class=\"mixed-citation\">Lozupone C, Knight R. \n2005. \n<span class=\"ref-title\">UniFrac: a new phylogenetic method for comparing microbial communities</span>. <span class=\"ref-journal\">Appl Environ Microbiol</span>\n<span class=\"ref-vol\">71</span>:8228&#x02013;8235. doi:10.1128/AEM.71.12.8228-8235.2005. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC1317376/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n16332807\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.71.12.8228-8235.2005\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl+Environ+Microbiol&amp;title=UniFrac:+a+new+phylogenetic+method+for+comparing+microbial+communities&amp;author=C+Lozupone&amp;author=R+Knight&amp;volume=71&amp;publication_year=2005&amp;pages=8228-8235&amp;pmid=16332807&amp;doi=10.1128/AEM.71.12.8228-8235.2005&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B58\">58. <span class=\"mixed-citation\">Bray JR, Curtis JT. \n1957. \n<span class=\"ref-title\">An ordination of the upland forest communities of southern Wisconsin</span>. <span class=\"ref-journal\">Ecol Monogr</span>\n<span class=\"ref-vol\">27</span>:325&#x02013;349. doi:10.2307/1942268. [<a href=\"//dx.doi.org/10.2307%2F1942268\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Ecol+Monogr&amp;title=An+ordination+of+the+upland+forest+communities+of+southern+Wisconsin&amp;author=JR+Bray&amp;author=JT+Curtis&amp;volume=27&amp;publication_year=1957&amp;pages=325-349&amp;doi=10.2307/1942268&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B59\">59. <span class=\"mixed-citation\">V&#x000e1;zquez-Baeza Y, Pirrung M, Gonzalez A, Knight R. \n2013. \n<span class=\"ref-title\">Emperor: a tool for visualizing high-throughput microbial community data</span>. <span class=\"ref-journal\">Gigascience</span>\n<span class=\"ref-vol\">2</span>:16. doi:10.1186/2047-217X-2-16. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4076506/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24280061\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2F2047-217X-2-16\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=Emperor:+a+tool+for+visualizing+high-throughput+microbial+community+data&amp;author=Y+V&#x000e1;zquez-Baeza&amp;author=M+Pirrung&amp;author=A+Gonzalez&amp;author=R+Knight&amp;volume=2&amp;publication_year=2013&amp;pages=16&amp;pmid=24280061&amp;doi=10.1186/2047-217X-2-16&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B60\">60. <span class=\"mixed-citation\">Oksanen J, Blanchet FG, Kindt R, Legendre P, Minchin PR, O&#x02019;hara R, Simpson GL, Solymos P, Stevens MHH, Wagner H. \n2013. \n<span class=\"ref-journal\">Package &#x0201c;vegan&#x0201d;</span>. Community ecology package, version 2\n<a href=\"https://cran.r-project.org/web/packages/vegan/vegan.pdf\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://cran.r-project.org/web/packages/vegan/vegan.pdf</a> <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Package+&#x0201c;vegan&#x0201d;&amp;author=J+Oksanen&amp;author=FG+Blanchet&amp;author=R+Kindt&amp;author=P+Legendre&amp;author=PR+Minchin&amp;publication_year=2013&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B61\">61. <span class=\"mixed-citation\">Shannon CE.\n1948. \n<span class=\"ref-title\">A mathematical theory of communication</span>. <span class=\"ref-journal\">Bell Syst Tech J</span>\n<span class=\"ref-vol\">27</span>:379&#x02013;423. doi:10.1002/j.1538-7305.1948.tb01338.x. [<a href=\"//dx.doi.org/10.1002%2Fj.1538-7305.1948.tb01338.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bell+Syst+Tech+J&amp;title=A+mathematical+theory+of+communication&amp;author=CE+Shannon&amp;volume=27&amp;publication_year=1948&amp;pages=379-423&amp;doi=10.1002/j.1538-7305.1948.tb01338.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B62\">62. <span class=\"mixed-citation\">Faith DP.\n1992. \n<span class=\"ref-title\">Conservation evaluation and phylogenetic diversity</span>. <span class=\"ref-journal\">Biol Conserv</span>\n<span class=\"ref-vol\">61</span>:1&#x02013;10. doi:10.1016/0006-3207(92)91201-3. [<a href=\"//dx.doi.org/10.1016%2F0006-3207(92)91201-3\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Biol+Conserv&amp;title=Conservation+evaluation+and+phylogenetic+diversity&amp;author=DP+Faith&amp;volume=61&amp;publication_year=1992&amp;pages=1-10&amp;doi=10.1016/0006-3207(92)91201-3&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B63\">63. <span class=\"mixed-citation\">Simpson EH.\n1949. \n<span class=\"ref-title\">Measurement of diversity</span>. <span class=\"ref-journal\">Nature</span>\n<span class=\"ref-vol\">163</span>:688&#x02013;688. doi:10.1038/163688a0. [<a href=\"//dx.doi.org/10.1038%2F163688a0\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Measurement+of+diversity&amp;author=EH+Simpson&amp;volume=163&amp;publication_year=1949&amp;pages=688-688&amp;doi=10.1038/163688a0&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B64\">64. <span class=\"mixed-citation\">R Development Core Team \n2013. \n<span class=\"ref-journal\">R: a language and environment for statistical computing</span>. The R Foundation for Statistical Computing, Vienna, Austria: \n<a href=\"http://www.R-project.org/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.R-project.org/</a>. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=R:+a+language+and+environment+for+statistical+computing&amp;publication_year=2013&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B65\">65. <span class=\"mixed-citation\">De Mendiburu F.\n2014. \n<span class=\"ref-journal\">Agricolae: statistical procedures for agricultural research</span>. R package version 1, p\n1&#x02013;6. <a href=\"https://CRAN.R-project.org/package=agricolae\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://CRAN.R-project.org/package=agricolae</a>. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Agricolae:+statistical+procedures+for+agricultural+research&amp;author=F+De+Mendiburu&amp;publication_year=2014&amp;pages=1-6&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B66\">66. <span class=\"mixed-citation\">Guevarra RB, Kim J, Nguyen SG, Unno T. \n2015. \n<span class=\"ref-title\">Comparison of fecal microbial communities between white and black pigs</span>. <span class=\"ref-journal\">J Appl Biol Chem</span>\n<span class=\"ref-vol\">58</span>:369&#x02013;375. doi:10.3839/jabc.2015.058. [<a href=\"//dx.doi.org/10.3839%2Fjabc.2015.058\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Appl+Biol+Chem&amp;title=Comparison+of+fecal+microbial+communities+between+white+and+black+pigs&amp;author=RB+Guevarra&amp;author=J+Kim&amp;author=SG+Nguyen&amp;author=T+Unno&amp;volume=58&amp;publication_year=2015&amp;pages=369-375&amp;doi=10.3839/jabc.2015.058&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B67\">67. <span class=\"mixed-citation\">Hoeflinger JL, Kashtanov DO, Cox SB, Dowd SE, Jouni ZE, Donovan SM, Miller MJ. \n2015. \n<span class=\"ref-title\">Characterization of the intestinal lactobacilli community following galactooligosaccharides and polydextrose supplementation in the neonatal piglet</span>. <span class=\"ref-journal\">PLoS One</span>\n<span class=\"ref-vol\">10</span>:e0135494. doi:10.1371/journal.pone.0135494. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4537252/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26275147\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0135494\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Characterization+of+the+intestinal+lactobacilli+community+following+galactooligosaccharides+and+polydextrose+supplementation+in+the+neonatal+piglet&amp;author=JL+Hoeflinger&amp;author=DO+Kashtanov&amp;author=SB+Cox&amp;author=SE+Dowd&amp;author=ZE+Jouni&amp;volume=10&amp;publication_year=2015&amp;pages=e0135494&amp;pmid=26275147&amp;doi=10.1371/journal.pone.0135494&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B68\">68. <span class=\"mixed-citation\">Kim J, Nguyen SG, Guevarra RB, Lee I, Unno T. \n2015. \n<span class=\"ref-title\">Analysis of swine fecal microbiota at various growth stages</span>. <span class=\"ref-journal\">Arch Microbiol</span>\n<span class=\"ref-vol\">197</span>:753&#x02013;759. doi:10.1007/s00203-015-1108-1. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25832348\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs00203-015-1108-1\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Arch+Microbiol&amp;title=Analysis+of+swine+fecal+microbiota+at+various+growth+stages&amp;author=J+Kim&amp;author=SG+Nguyen&amp;author=RB+Guevarra&amp;author=I+Lee&amp;author=T+Unno&amp;volume=197&amp;publication_year=2015&amp;pages=753-759&amp;pmid=25832348&amp;doi=10.1007/s00203-015-1108-1&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B69\">69. <span class=\"mixed-citation\">Pajarillo EA, Chae JP, Balolong MP, Kim HB, Seo KS, Kang DK. \n2014. \n<span class=\"ref-title\">Pyrosequencing-based analysis of fecal microbial communities in three purebred pig lines</span>. <span class=\"ref-journal\">J Microbiol</span>\n<span class=\"ref-vol\">52</span>:646&#x02013;651. doi:10.1007/s12275-014-4270-2. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25047525\" ref=\"reftype=pubmed&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs12275-014-4270-2\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Microbiol&amp;title=Pyrosequencing-based+analysis+of+fecal+microbial+communities+in+three+purebred+pig+lines&amp;author=EA+Pajarillo&amp;author=JP+Chae&amp;author=MP+Balolong&amp;author=HB+Kim&amp;author=KS+Seo&amp;volume=52&amp;publication_year=2014&amp;pages=646-651&amp;pmid=25047525&amp;doi=10.1007/s12275-014-4270-2&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B70\">70. <span class=\"mixed-citation\">Xian L, Li Y, Jiang Z, Ma J, Jin L, Chen L, Zhou C, Zhang J, Liu Y, Zhu L, Jiang A, Tang G, Xiao J, Wang J, Zeng B, Li M, Li X. \n2014. \n<span class=\"ref-title\">Alterations in cecal microbiota of Jinhua piglets fostered by a Yorkshire sow</span>. <span class=\"ref-journal\">Chin Sci Bull</span>\n<span class=\"ref-vol\">59</span>:4304&#x02013;4311. doi:10.1007/s11434-014-0532-y. [<a href=\"//dx.doi.org/10.1007%2Fs11434-014-0532-y\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Chin+Sci+Bull&amp;title=Alterations+in+cecal+microbiota+of+Jinhua+piglets+fostered+by+a+Yorkshire+sow&amp;author=L+Xian&amp;author=Y+Li&amp;author=Z+Jiang&amp;author=J+Ma&amp;author=L+Jin&amp;volume=59&amp;publication_year=2014&amp;pages=4304-4311&amp;doi=10.1007/s11434-014-0532-y&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5443231&amp;issue-id=292354&amp;journal-id=3105&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div></div><br>', NULL, 2);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(75, 'pubmed', '23203882', 'The PRoteomics IDEntifications (PRIDE) database and associated tools: status in 2013.', 'The PRoteomics IDEntifications (PRIDE, http://www.ebi.ac.uk/pride) database at the European Bioinformatics Institute is one of the most prominent data repositories of mass spectrometry (MS)-based proteomics data. Here, we summarize recent developments in the PRIDE database and related tools. First, we provide up-to-date statistics in data content, splitting the figures by groups of organisms and species, including peptide and protein identifications, and post-translational modifications. We then describe the tools that are part of the PRIDE submission pipeline, especially the recently developed PRIDE Converter 2 (new submission tool) and PRIDE Inspector (visualization and analysis tool). We also give an update about the integration of PRIDE with other MS proteomics resources in the context of the ProteomeXchange consortium. Finally, we briefly review the quality control efforts that are ongoing at present and outline our future plans.', 2013, 'Nucleic acids research', 'PMC3531176', 2, '<h1 class=\"content-title\">The Proteomics Identifications (PRIDE) database and associated tools: status in 2013</h1><br><span class=\"contrib-group fm-author\"><a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Vizca%26%23x000ed%3Bno%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\" class=\"affpopup\" co-rid=\"_co_idm140127380857184\" co-class=\"co-affbox\">Juan Antonio Vizca&#x000ed;no</a>,<sup>1,</sup><sup>*</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=C%26%23x000f4%3Bt%26%23x000e9%3B%20RG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\" class=\"affpopup\" co-rid=\"_co_idm140127292635072\" co-class=\"co-affbox\">Richard G. C&#x000f4;t&#x000e9;</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Csordas%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\" class=\"affpopup\" co-rid=\"_co_idm140127308254256\" co-class=\"co-affbox\">Attila Csordas</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Dianes%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\" class=\"affpopup\" co-rid=\"_co_idm140127309311200\" co-class=\"co-affbox\">Jos&#x000e9; A. Dianes</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Fabregat%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\" class=\"affpopup\" co-rid=\"_co_idm140127309309088\" co-class=\"co-affbox\">Antonio Fabregat</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Foster%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\" class=\"affpopup\" co-rid=\"_co_idm140127296305184\" co-class=\"co-affbox\">Joseph M. Foster</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Griss%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\" class=\"affpopup\" co-rid=\"_co_idm140127296303072\" co-class=\"co-affbox\">Johannes Griss</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Alpi%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\" class=\"affpopup\" co-rid=\"_co_idm140127344378592\" co-class=\"co-affbox\">Emanuele Alpi</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Birim%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\" class=\"affpopup\" co-rid=\"_co_idm140127344376480\" co-class=\"co-affbox\">Melih Birim</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Contell%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\" class=\"affpopup\" co-rid=\"_co_idm140127344964608\" co-class=\"co-affbox\">Javier Contell</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=O%26%23x02019%3BKelly%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\" class=\"affpopup\" co-rid=\"_co_idm140127344962496\" co-class=\"co-affbox\">Gavin O&#x02019;Kelly</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Schoenegger%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\" class=\"affpopup\" co-rid=\"_co_idm140127291690160\" co-class=\"co-affbox\">Andreas Schoenegger</a>,<sup>1,</sup><sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Ovelleiro%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\" class=\"affpopup\" co-rid=\"_co_idm140127291687184\" co-class=\"co-affbox\">David Ovelleiro</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=P%26%23x000e9%3Brez-Riverol%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\" class=\"affpopup\" co-rid=\"_co_idm140127294273872\" co-class=\"co-affbox\">Yasset P&#x000e9;rez-Riverol</a>,<sup>1,</sup><sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Reisinger%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\" class=\"affpopup\" co-rid=\"_co_idm140127304593072\" co-class=\"co-affbox\">Florian Reisinger</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=R%26%23x000ed%3Bos%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\" class=\"affpopup\" co-rid=\"_co_idm140127304590960\" co-class=\"co-affbox\">Daniel R&#x000ed;os</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Wang%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\" class=\"affpopup\" co-rid=\"_co_idm140127296316960\" co-class=\"co-affbox\">Rui Wang</a>,<sup>1</sup> and  <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Hermjakob%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\" class=\"affpopup\" co-rid=\"_co_idm140127296314848\" co-class=\"co-affbox\">Henning Hermjakob</a><sup>1</sup></span><span style=\"display:none\" class=\"contrib-group aff-tip\"><span id=\"_co_idm140127380857184\"><h3 class=\"no_margin\">Juan Antonio Vizca&#x000ed;no</h3><p><sup>1</sup>EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, <sup>2</sup>Department of Proteomics, Center for Genetic Engineering and Biotechnology, Havana, Cuba and <sup>3</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Lazarettgasse 14, AKH BT 25.3, 1090 Vienna, Austria</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Vizca%26%23x000ed%3Bno%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\">Juan Antonio Vizca&#x000ed;no</a></span></span><span id=\"_co_idm140127292635072\"><h3 class=\"no_margin\">Richard G. C&#x000f4;t&#x000e9;</h3><p><sup>1</sup>EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, <sup>2</sup>Department of Proteomics, Center for Genetic Engineering and Biotechnology, Havana, Cuba and <sup>3</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Lazarettgasse 14, AKH BT 25.3, 1090 Vienna, Austria</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=C%26%23x000f4%3Bt%26%23x000e9%3B%20RG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\">Richard G. C&#x000f4;t&#x000e9;</a></span></span><span id=\"_co_idm140127308254256\"><h3 class=\"no_margin\">Attila Csordas</h3><p><sup>1</sup>EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, <sup>2</sup>Department of Proteomics, Center for Genetic Engineering and Biotechnology, Havana, Cuba and <sup>3</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Lazarettgasse 14, AKH BT 25.3, 1090 Vienna, Austria</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Csordas%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\">Attila Csordas</a></span></span><span id=\"_co_idm140127309311200\"><h3 class=\"no_margin\">Jos&#x000e9; A. Dianes</h3><p><sup>1</sup>EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, <sup>2</sup>Department of Proteomics, Center for Genetic Engineering and Biotechnology, Havana, Cuba and <sup>3</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Lazarettgasse 14, AKH BT 25.3, 1090 Vienna, Austria</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Dianes%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\">Jos&#x000e9; A. Dianes</a></span></span><span id=\"_co_idm140127309309088\"><h3 class=\"no_margin\">Antonio Fabregat</h3><p><sup>1</sup>EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, <sup>2</sup>Department of Proteomics, Center for Genetic Engineering and Biotechnology, Havana, Cuba and <sup>3</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Lazarettgasse 14, AKH BT 25.3, 1090 Vienna, Austria</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Fabregat%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\">Antonio Fabregat</a></span></span><span id=\"_co_idm140127296305184\"><h3 class=\"no_margin\">Joseph M. Foster</h3><p><sup>1</sup>EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, <sup>2</sup>Department of Proteomics, Center for Genetic Engineering and Biotechnology, Havana, Cuba and <sup>3</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Lazarettgasse 14, AKH BT 25.3, 1090 Vienna, Austria</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Foster%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\">Joseph M. Foster</a></span></span><span id=\"_co_idm140127296303072\"><h3 class=\"no_margin\">Johannes Griss</h3><p><sup>1</sup>EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, <sup>2</sup>Department of Proteomics, Center for Genetic Engineering and Biotechnology, Havana, Cuba and <sup>3</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Lazarettgasse 14, AKH BT 25.3, 1090 Vienna, Austria</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Griss%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\">Johannes Griss</a></span></span><span id=\"_co_idm140127344378592\"><h3 class=\"no_margin\">Emanuele Alpi</h3><p><sup>1</sup>EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, <sup>2</sup>Department of Proteomics, Center for Genetic Engineering and Biotechnology, Havana, Cuba and <sup>3</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Lazarettgasse 14, AKH BT 25.3, 1090 Vienna, Austria</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Alpi%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\">Emanuele Alpi</a></span></span><span id=\"_co_idm140127344376480\"><h3 class=\"no_margin\">Melih Birim</h3><p><sup>1</sup>EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, <sup>2</sup>Department of Proteomics, Center for Genetic Engineering and Biotechnology, Havana, Cuba and <sup>3</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Lazarettgasse 14, AKH BT 25.3, 1090 Vienna, Austria</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Birim%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\">Melih Birim</a></span></span><span id=\"_co_idm140127344964608\"><h3 class=\"no_margin\">Javier Contell</h3><p><sup>1</sup>EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, <sup>2</sup>Department of Proteomics, Center for Genetic Engineering and Biotechnology, Havana, Cuba and <sup>3</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Lazarettgasse 14, AKH BT 25.3, 1090 Vienna, Austria</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Contell%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\">Javier Contell</a></span></span><span id=\"_co_idm140127344962496\"><h3 class=\"no_margin\">Gavin O&#x02019;Kelly</h3><p><sup>1</sup>EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, <sup>2</sup>Department of Proteomics, Center for Genetic Engineering and Biotechnology, Havana, Cuba and <sup>3</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Lazarettgasse 14, AKH BT 25.3, 1090 Vienna, Austria</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=O%26%23x02019%3BKelly%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\">Gavin O&#x02019;Kelly</a></span></span><span id=\"_co_idm140127291690160\"><h3 class=\"no_margin\">Andreas Schoenegger</h3><p><sup>1</sup>EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, <sup>2</sup>Department of Proteomics, Center for Genetic Engineering and Biotechnology, Havana, Cuba and <sup>3</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Lazarettgasse 14, AKH BT 25.3, 1090 Vienna, Austria</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Schoenegger%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\">Andreas Schoenegger</a></span></span><span id=\"_co_idm140127291687184\"><h3 class=\"no_margin\">David Ovelleiro</h3><p><sup>1</sup>EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, <sup>2</sup>Department of Proteomics, Center for Genetic Engineering and Biotechnology, Havana, Cuba and <sup>3</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Lazarettgasse 14, AKH BT 25.3, 1090 Vienna, Austria</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Ovelleiro%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\">David Ovelleiro</a></span></span><span id=\"_co_idm140127294273872\"><h3 class=\"no_margin\">Yasset P&#x000e9;rez-Riverol</h3><p><sup>1</sup>EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, <sup>2</sup>Department of Proteomics, Center for Genetic Engineering and Biotechnology, Havana, Cuba and <sup>3</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Lazarettgasse 14, AKH BT 25.3, 1090 Vienna, Austria</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=P%26%23x000e9%3Brez-Riverol%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\">Yasset P&#x000e9;rez-Riverol</a></span></span><span id=\"_co_idm140127304593072\"><h3 class=\"no_margin\">Florian Reisinger</h3><p><sup>1</sup>EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, <sup>2</sup>Department of Proteomics, Center for Genetic Engineering and Biotechnology, Havana, Cuba and <sup>3</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Lazarettgasse 14, AKH BT 25.3, 1090 Vienna, Austria</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Reisinger%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\">Florian Reisinger</a></span></span><span id=\"_co_idm140127304590960\"><h3 class=\"no_margin\">Daniel R&#x000ed;os</h3><p><sup>1</sup>EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, <sup>2</sup>Department of Proteomics, Center for Genetic Engineering and Biotechnology, Havana, Cuba and <sup>3</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Lazarettgasse 14, AKH BT 25.3, 1090 Vienna, Austria</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=R%26%23x000ed%3Bos%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\">Daniel R&#x000ed;os</a></span></span><span id=\"_co_idm140127296316960\"><h3 class=\"no_margin\">Rui Wang</h3><p><sup>1</sup>EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, <sup>2</sup>Department of Proteomics, Center for Genetic Engineering and Biotechnology, Havana, Cuba and <sup>3</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Lazarettgasse 14, AKH BT 25.3, 1090 Vienna, Austria</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Wang%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\">Rui Wang</a></span></span><span id=\"_co_idm140127296314848\"><h3 class=\"no_margin\">Henning Hermjakob</h3><p><sup>1</sup>EMBL Outstation, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK, <sup>2</sup>Department of Proteomics, Center for Genetic Engineering and Biotechnology, Havana, Cuba and <sup>3</sup>CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Lazarettgasse 14, AKH BT 25.3, 1090 Vienna, Austria</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Hermjakob%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23203882\">Henning Hermjakob</a></span></span></span></span><span class=\"fm-panel half_rhythm\"><span class=\"togglers<br><span id=\"idm140127295807696\" lang=\"en\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140127295807696title\" style=\"text-transform: uppercase;\">Abstract</h2><!--article-meta--><span><p id=\"__p1\" class=\"p p-first-last\">The PRoteomics IDEntifications (PRIDE, <a href=\"http://www.ebi.ac.uk/pride\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.ebi.ac.uk/pride</a>) database at the European Bioinformatics Institute is one of the most prominent data repositories of mass spectrometry (MS)-based proteomics data. Here, we summarize recent developments in the PRIDE database and related tools. First, we provide up-to-date statistics in data content, splitting the figures by groups of organisms and species, including peptide and protein identifications, and post-translational modifications. We then describe the tools that are part of the PRIDE submission pipeline, especially the recently developed PRIDE Converter 2 (new submission tool) and PRIDE Inspector (visualization and analysis tool). We also give an update about the integration of PRIDE with other MS proteomics resources in the context of the ProteomeXchange consortium. Finally, we briefly review the quality control efforts that are ongoing at present and outline our future plans.</p></span></span><span id=\"s\r\n1\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n1title\" style=\"text-transform: uppercase;\">INTRODUCTION</h2><p id=\"__p2\" class=\"p p-first\">Mass spectrometry (MS)-based proteomics approaches are widely used in the life sciences. There are three main workflows, with bottom-up proteomics the most extensively used technique (also known as shot-gun proteomics) (<a href=\"#gks1262-B1\" rid=\"gks1262-B1\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916102\">1</a>). In this experimental set up, the proteins to be analysed are enzymatically digested by a protease (most often trypsin) into potentially highly complex peptide mixtures, which are then subjected to fractionation by multidimensional liquid chromatography steps before they are measured in the mass spectrometer. Other main approaches are top-down, where intact proteins are measured (<a href=\"#gks1262-B2\" rid=\"gks1262-B2\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916069\">2</a>), and targeted proteomics (such as Selected Reaction Monitoring, SRM), where the researcher tries to detect specific proteins in a given sample (<a href=\"#gks1262-B3\" rid=\"gks1262-B3\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916118\">3</a>).</p><p id=\"__p3\">The PRoteomics IDEntifications (PRIDE, <a href=\"http://www.ebi.ac.uk/pride\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.ebi.ac.uk/pride</a>) database was originally set up in 2004 (<a href=\"#gks1262-B4\" rid=\"gks1262-B4\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916111\">4&#x02013;8</a>) to enable public data deposition in the MS proteomics field, and to support the experimental data described in publications during the manuscript review process. The main data types stored in PRIDE are protein and peptide identifications (IDs) and quantitative values (including post-translational modifications, PTMs), the analysed mass spectra and the related technical and biological metadata. PRIDE supports bottom-up proteomics approaches, mainly tandem MS (MS/MS) data, but also Peptide Mass Fingerprinting datasets, and presents the data as originally analysed by the researchers, with several popular search engines/analysis workflows fully supported. Unlike other MS proteomics resources, such as PeptideAtlas (<a href=\"#gks1262-B9\" rid=\"gks1262-B9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916119\">9</a>) and the Global Proteome Machine Database (GPMDB) (<a href=\"#gks1262-B10\" rid=\"gks1262-B10\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916112\">10</a>), no reprocessing of the data is performed because PRIDE aims to reflect the author&#x02019;s analysis view on the experimental data. In fact, PRIDE remains as the unique generic resource of this kind since National Center for Biotechnology Information (NCBI) the repository Peptidome (<a href=\"#gks1262-B11\" rid=\"gks1262-B11\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916086\">11</a>), its sibling resource in the USA, was discontinued in April 2011. Other MS data repositories, such as MaxQB (<a href=\"#gks1262-B12\" rid=\"gks1262-B12\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916097\">12</a>), are more specialized [for an extensive review, see (<a href=\"#gks1262-B13\" rid=\"gks1262-B13\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916117\">13</a>)] or are restricted to one particular analysis workflow.</p><p id=\"__p4\">For SRM data, the new PeptideAtlaS SRM Experiment Library (PASSEL) (<a href=\"#gks1262-B14\" rid=\"gks1262-B14\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916096\">14</a>) is the main available resource. At present, there is no widely used resource devoted to top-down proteomics approaches. In addition to the &#x02018;pure&#x02019; MS proteomics resources, there are other databases that can present an extra layer of information on top of the MS experiments without storing the underlying mass spectra. Some recently developed databases of this kind are the Model Organism Protein Expression Database (MOPED) (<a href=\"#gks1262-B15\" rid=\"gks1262-B15\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916080\">15</a>), PaxDB (<a href=\"#gks1262-B16\" rid=\"gks1262-B16\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916064\">16</a>) (both of them focused on protein expression information) and neXtProt (<a href=\"#gks1262-B17\" rid=\"gks1262-B17\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916079\">17</a>).</p><p id=\"__p5\">Several services have been developed by the PRIDE team, which are heavily used by external users but also by PRIDE itself, especially the &#x02018;Protein Identifier Cross-Reference&#x02019; (PICR) service (a protein identifier mapping resource) (<a href=\"#gks1262-B18\" rid=\"gks1262-B18\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916104\">18</a>) and the &#x02018;Ontology Lookup Service&#x02019; (OLS) (to query, browse and navigate biomedical ontologies) (<a href=\"#gks1262-B19\" rid=\"gks1262-B19\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916113\">19</a>). In addition, &#x02018;Database on Demand&#x02019; is a service to generate tailored databases for performing proteomics searches (<a href=\"#gks1262-B20\" rid=\"gks1262-B20\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916100\">20</a>). As a key point, to improve and make the data submission process easier, several tools have also been made available to the proteomics community, such as the popular PRIDE Converter (<a href=\"#gks1262-B21\" rid=\"gks1262-B21\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916077\">21</a>), PRIDE Inspector (<a href=\"#gks1262-B22\" rid=\"gks1262-B22\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916065\">22</a>) and the new PRIDE Converter 2 (<a href=\"#gks1262-B23\" rid=\"gks1262-B23\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916109\">23</a>). It is important to highlight that all the softwares, including the PRIDE core and web modules (<a href=\"http://code.google.com/p/ebi-pride/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://code.google.com/p/ebi-pride/</a>), are developed in Java and are open source. PRIDE is a recommended submission site of key journals such as <em>Proteomics</em>, <em>Molecular and Cellular Proteomics</em> and <em>Nature Biotechnology</em>. Currently, scientific journals and funding agencies alike are increasingly mandating public deposition of MS data to support the publication of related proteomics manuscripts.</p><p id=\"__p6\" class=\"p p-last\">In this manuscript, we summarize developments in the PRIDE database and associated tools since the previous <em>Nucleic Acids Research</em> (NAR) database update (<a href=\"#gks1262-B8\" rid=\"gks1262-B8\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916068\">8</a>). We will also outline the PRIDE data deposition process, introduce the ProteomeXchange (PX) consortium and quality control (QC) efforts and highlight future developments.</p></span><span id=\"s\r\n2\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n2title\" style=\"text-transform: uppercase;\">DATA CONTENT IN PRIDE AND HOW TO ACCESS IT</h2><p id=\"__p7\" class=\"p p-first\">There has been a substantial increase in the amount of stored data in PRIDE during the past years. By September 2012, PRIDE contains 25 853 MS-based proteomics experiments (compared with 9908 when the last NAR manuscript was submitted, in September 2009), around 11.1 million identified proteins (2.5 million in September 2009), 61.9 million identified peptides (11.5 million in September 2009) and 324 million spectra (50.3 million in September 2009). Note that these data holdings are absolute figures, not distinguishing public and pre-publication data. At the moment of writing, 66.7% (17 219) of the experiments were publicly available.</p><p id=\"__p8\">The complete set of data in PRIDE comprises 323 taxonomy identifiers (compared with 60, in September 2009), including human and many model organisms (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3531176/table/gks1262-T1/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"gks1262-T1\" rid-ob=\"ob-gks1262-T1\" co-legend-rid=\"\"><span>Table 1</span></a>). In comparison with figures from 3 years ago, animal species still provide the majority of the data. A total of 89 animal species are represented, contributing 62.2% and 61.9% of all protein and peptide IDs in PRIDE, respectively. Human continues to be the most represented species (28.1% and 37.2% of protein and peptide IDs, respectively). As a matter of fact, human and mouse alone account for almost as many IDs as the other species together: 44.2 % and 49.4 % of the protein and peptide IDs, respectively.\n</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"gks1262-T1\"><h3>Table 1.</h3><!--caption a7--><span class=\"caption\"><p id=\"__p9\">Data content in PRIDE split by taxonomic divisions</p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140127309245088\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><thead align=\"left\"><tr><th rowspan=\"1\" colspan=\"1\">Group of organisms (number of species)</th><th rowspan=\"1\" colspan=\"1\">% Protein IDs</th><th rowspan=\"1\" colspan=\"1\">% Peptide IDs</th></tr></thead><tbody align=\"left\"><tr><td rowspan=\"1\" colspan=\"1\">Animals (89)</td><td rowspan=\"1\" colspan=\"1\">62.2</td><td rowspan=\"1\" colspan=\"1\">61.9</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Plants (<a href=\"#gks1262-B46\" rid=\"gks1262-B46\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916075\">46</a>)</td><td rowspan=\"1\" colspan=\"1\">19.3</td><td rowspan=\"1\" colspan=\"1\">14.8</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Fungi (<a href=\"#gks1262-B22\" rid=\"gks1262-B22\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916114\">22</a>)</td><td rowspan=\"1\" colspan=\"1\">2.7</td><td rowspan=\"1\" colspan=\"1\">3.0</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Bacteria (122)</td><td rowspan=\"1\" colspan=\"1\">12.7</td><td rowspan=\"1\" colspan=\"1\">17.4</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Others (<a href=\"#gks1262-B44\" rid=\"gks1262-B44\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916073\">44</a>)</td><td rowspan=\"1\" colspan=\"1\">3.1</td><td rowspan=\"1\" colspan=\"1\">2.9</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Species</td><td rowspan=\"1\" colspan=\"1\"></td><td rowspan=\"1\" colspan=\"1\"></td></tr><tr><td rowspan=\"1\" colspan=\"1\">&#x02003;&#x02003;&#x02003;&#x02003;<em>Homo sapiens</em></td><td rowspan=\"1\" colspan=\"1\">28.1</td><td rowspan=\"1\" colspan=\"1\">37.2</td></tr><tr><td rowspan=\"1\" colspan=\"1\">&#x02003;&#x02003;&#x02003;&#x02003;<em>Mus musculus</em></td><td rowspan=\"1\" colspan=\"1\">16.1</td><td rowspan=\"1\" colspan=\"1\">12.2</td></tr><tr><td rowspan=\"1\" colspan=\"1\">&#x02003;&#x02003;&#x02003;&#x02003;<em>Zea mays</em></td><td rowspan=\"1\" colspan=\"1\">9.5</td><td rowspan=\"1\" colspan=\"1\">2.2</td></tr><tr><td rowspan=\"1\" colspan=\"1\">&#x02003;&#x02003;&#x02003;&#x02003;<em>Arabidopsis thaliana</em></td><td rowspan=\"1\" colspan=\"1\">6.8</td><td rowspan=\"1\" colspan=\"1\">10.1</td></tr><tr><td rowspan=\"1\" colspan=\"1\">&#x02003;&#x02003;&#x02003;&#x02003;<em>B. subtilis</em></td><td rowspan=\"1\" colspan=\"1\">5.5</td><td rowspan=\"1\" colspan=\"1\">4.1</td></tr><tr><td rowspan=\"1\" colspan=\"1\">&#x02003;&#x02003;&#x02003;&#x02003;<em>Sus scrofa</em></td><td rowspan=\"1\" colspan=\"1\">5.5</td><td rowspan=\"1\" colspan=\"1\">1.7</td></tr><tr><td rowspan=\"1\" colspan=\"1\">&#x02003;&#x02003;&#x02003;&#x02003;<em>Rattus norvegicus</em></td><td rowspan=\"1\" colspan=\"1\">3.7</td><td rowspan=\"1\" colspan=\"1\">2.8</td></tr><tr><td rowspan=\"1\" colspan=\"1\">&#x02003;&#x02003;&#x02003;&#x02003;<em>Drosophila melanogaster</em></td><td rowspan=\"1\" colspan=\"1\">3.3</td><td rowspan=\"1\" colspan=\"1\">2.3</td></tr><tr><td rowspan=\"1\" colspan=\"1\">&#x02003;&#x02003;&#x02003;&#x02003;<em>Danio rerio</em></td><td rowspan=\"1\" colspan=\"1\">1.7</td><td rowspan=\"1\" colspan=\"1\">0.9</td></tr><tr><td rowspan=\"1\" colspan=\"1\">&#x02003;&#x02003;&#x02003;&#x02003;<em>Puniceispirillum marinum</em></td><td rowspan=\"1\" colspan=\"1\">1.5</td><td rowspan=\"1\" colspan=\"1\">1.5</td></tr><tr><td rowspan=\"1\" colspan=\"1\">&#x02003;&#x02003;&#x02003;&#x02003;<em>E. coli</em></td><td rowspan=\"1\" colspan=\"1\">1.2</td><td rowspan=\"1\" colspan=\"1\">1.7</td></tr><tr><td rowspan=\"1\" colspan=\"1\">&#x02003;&#x02003;&#x02003;&#x02003;<em>S. cerevisiae</em></td><td rowspan=\"1\" colspan=\"1\">1.2</td><td rowspan=\"1\" colspan=\"1\">1.8</td></tr></tbody></table></span><span id=\"largeobj_idm140127309245088\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3531176/table/gks1262-T1/?report=objectonly\">Open in a separate window</a></span><span class=\"tblwrap-foot\"><span id=\"gks1262-TF1\"><p id=\"__p10\" class=\"p p-first-last\">Only the top 12 species in terms of protein and peptide identifications are shown.</p></span></span></span><p id=\"__p11\">However, the relative proportion of other groups of organisms has increased, especially in the case of plants (46 taxonomy identifiers, 19.3% and 14.8%, respectively) and bacteria (12.7% and 17.4%, respectively). Bacteria are again by far the group of organisms with the highest number of taxonomy identifiers (122). Fungi are also represented (22 taxonomy identifiers, 2.7 and 3.0%, respectively). Apart from human, the most represented organisms in PRIDE are (in this order) mouse, maize, Arabidopsis, <em>Bacillus subtilis</em>, pig, rat, Drosophila, zebrafish, <em>Puniceispirillum marinum</em>, <em>Escherichia coli</em> and <em>Saccharomyces cerevisiae</em> (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3531176/table/gks1262-T1/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"gks1262-T1\" rid-ob=\"ob-gks1262-T1\" co-legend-rid=\"\"><span>Table 1</span></a>).</p><p id=\"__p12\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3531176/table/gks1262-T2/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"gks1262-T2\" rid-ob=\"ob-gks1262-T2\" co-legend-rid=\"\"><span>Table 2</span></a> includes the most abundant PTMs present in the database. Not surprisingly, the most often found PTM is oxidation (5.7 million modified sites), mainly due to the high amount of methionine oxidation, a modification that can be biologically relevant (<a href=\"#gks1262-B24\" rid=\"gks1262-B24\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916101\">24</a>) but that, in most cases for MS proteomics experiments, is an artifact. Formylation is the second most abundant PTM (around 1.3 million sites), mainly owing to the data present from just one organism (maize). Phosphorylation comes in third place (around 1.1 million sites), with the highest proportion of data coming from human experiments. There is also a considerable amount of other PTMs such as dioxidation, deamidation, acetylation or dehydration, among others (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3531176/table/gks1262-T2/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"gks1262-T2\" rid-ob=\"ob-gks1262-T2\" co-legend-rid=\"\"><span>Table 2</span></a>).\n</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"gks1262-T2\"><h3>Table 2.</h3><!--caption a7--><span class=\"caption\"><p id=\"__p13\">Protein modification content in PRIDE as a whole, and split by species (human and main model organisms represented in PRIDE)</p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140127292674656\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><thead align=\"left\"><tr><th rowspan=\"1\" colspan=\"1\">Modification type</th><th rowspan=\"1\" colspan=\"1\">Total</th><th rowspan=\"1\" colspan=\"1\">Human</th><th rowspan=\"1\" colspan=\"1\">Mouse</th><th rowspan=\"1\" colspan=\"1\">Maize</th><th rowspan=\"1\" colspan=\"1\">Arabidopsis</th><th rowspan=\"1\" colspan=\"1\">Drosophila</th><th rowspan=\"1\" colspan=\"1\"><em>E. coli</em></th><th rowspan=\"1\" colspan=\"1\"><em>S. cerevisiae</em></th></tr></thead><tbody align=\"left\"><tr><td rowspan=\"1\" colspan=\"1\">Oxidation</td><td rowspan=\"1\" colspan=\"1\">5 707 426</td><td rowspan=\"1\" colspan=\"1\">2 291 925</td><td rowspan=\"1\" colspan=\"1\">883 599</td><td rowspan=\"1\" colspan=\"1\">25 907</td><td rowspan=\"1\" colspan=\"1\">449 530</td><td rowspan=\"1\" colspan=\"1\">2400</td><td rowspan=\"1\" colspan=\"1\">58 921</td><td rowspan=\"1\" colspan=\"1\">97 803</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Deamidation</td><td rowspan=\"1\" colspan=\"1\">663 884</td><td rowspan=\"1\" colspan=\"1\">333 091</td><td rowspan=\"1\" colspan=\"1\">48 120</td><td rowspan=\"1\" colspan=\"1\">30</td><td rowspan=\"1\" colspan=\"1\">6964</td><td rowspan=\"1\" colspan=\"1\">83</td><td rowspan=\"1\" colspan=\"1\">17 335</td><td rowspan=\"1\" colspan=\"1\">10 859</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Phosphorylation</td><td rowspan=\"1\" colspan=\"1\">1 143 766</td><td rowspan=\"1\" colspan=\"1\">741 619</td><td rowspan=\"1\" colspan=\"1\">112 910</td><td rowspan=\"1\" colspan=\"1\">171 539</td><td rowspan=\"1\" colspan=\"1\">35 521</td><td rowspan=\"1\" colspan=\"1\">239</td><td rowspan=\"1\" colspan=\"1\">0</td><td rowspan=\"1\" colspan=\"1\">33 670</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Acetylation</td><td rowspan=\"1\" colspan=\"1\">626 510</td><td rowspan=\"1\" colspan=\"1\">380 124</td><td rowspan=\"1\" colspan=\"1\">22 662</td><td rowspan=\"1\" colspan=\"1\">1419</td><td rowspan=\"1\" colspan=\"1\">10 614</td><td rowspan=\"1\" colspan=\"1\">290</td><td rowspan=\"1\" colspan=\"1\">1356</td><td rowspan=\"1\" colspan=\"1\">18 762</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Dioxidation</td><td rowspan=\"1\" colspan=\"1\">1 123 206</td><td rowspan=\"1\" colspan=\"1\">31 788</td><td rowspan=\"1\" colspan=\"1\">1625</td><td rowspan=\"1\" colspan=\"1\">928 614</td><td rowspan=\"1\" colspan=\"1\">0</td><td rowspan=\"1\" colspan=\"1\">0</td><td rowspan=\"1\" colspan=\"1\">0</td><td rowspan=\"1\" colspan=\"1\">0</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Deamination</td><td rowspan=\"1\" colspan=\"1\">132 145</td><td rowspan=\"1\" colspan=\"1\">87 016</td><td rowspan=\"1\" colspan=\"1\">29 374</td><td rowspan=\"1\" colspan=\"1\">462</td><td rowspan=\"1\" colspan=\"1\">0</td><td rowspan=\"1\" colspan=\"1\">262</td><td rowspan=\"1\" colspan=\"1\">272</td><td rowspan=\"1\" colspan=\"1\">0</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Dehydration</td><td rowspan=\"1\" colspan=\"1\">202 436</td><td rowspan=\"1\" colspan=\"1\">15 492</td><td rowspan=\"1\" colspan=\"1\">32 863</td><td rowspan=\"1\" colspan=\"1\">148 686</td><td rowspan=\"1\" colspan=\"1\">0</td><td rowspan=\"1\" colspan=\"1\">30</td><td rowspan=\"1\" colspan=\"1\">0</td><td rowspan=\"1\" colspan=\"1\">0</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Methylthio</td><td rowspan=\"1\" colspan=\"1\">110 727</td><td rowspan=\"1\" colspan=\"1\">74 177</td><td rowspan=\"1\" colspan=\"1\">0</td><td rowspan=\"1\" colspan=\"1\">0</td><td rowspan=\"1\" colspan=\"1\">2425</td><td rowspan=\"1\" colspan=\"1\">0</td><td rowspan=\"1\" colspan=\"1\">82</td><td rowspan=\"1\" colspan=\"1\">0</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Formylation</td><td rowspan=\"1\" colspan=\"1\">1 297 285</td><td rowspan=\"1\" colspan=\"1\">5286</td><td rowspan=\"1\" colspan=\"1\">629</td><td rowspan=\"1\" colspan=\"1\">1 289 660</td><td rowspan=\"1\" colspan=\"1\">0</td><td rowspan=\"1\" colspan=\"1\">0</td><td rowspan=\"1\" colspan=\"1\">0</td><td rowspan=\"1\" colspan=\"1\">0</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Monomethylation</td><td rowspan=\"1\" colspan=\"1\">299 246</td><td rowspan=\"1\" colspan=\"1\">261 941</td><td rowspan=\"1\" colspan=\"1\">48</td><td rowspan=\"1\" colspan=\"1\">0</td><td rowspan=\"1\" colspan=\"1\">277</td><td rowspan=\"1\" colspan=\"1\">0</td><td rowspan=\"1\" colspan=\"1\">0</td><td rowspan=\"1\" colspan=\"1\">0</td></tr></tbody></table></span><span id=\"largeobj_idm140127292674656\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3531176/table/gks1262-T2/?report=objectonly\">Open in a separate window</a></span></span><p id=\"__p14\">This wealth of data can be accessed in different ways (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3531176/figure/gks1262-F1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"gks1262-F1\" rid-ob=\"ob-gks1262-F1\" co-legend-rid=\"lgnd_gks1262-F1\"><span>Figure 1</span></a>):\n</p><ol class=\"enumerated\" style=\"list-style-type:lower-alpha\"><!--\nlist-behavior=enumerated\nprefix-word=\nmark-type=lower-alpha\nmax-label-size=0\n--><li><p id=\"__p15\">PRIDE web interface (<a href=\"http://www.ebi.ac.uk/pride\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.ebi.ac.uk/pride</a>). The home page was updated earlier in 2012, but no other major changes have been done to the current web interface. However, it is possible to access all experiments launching the PRIDE Inspector tool using Java Web Start (see below).</p></li><li><p id=\"__p16\">PRIDE BioMart (<a href=\"http://www.ebi.ac.uk/pride/prideMart.do\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.ebi.ac.uk/pride/prideMart.do</a>). The BioMart interface is useful for batch data retrieval (<a href=\"#gks1262-B25\" rid=\"gks1262-B25\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916082\">25</a>). In the current version of the PRIDE BioMart (running on BioMart version 0.7), data integration with Reactome (<a href=\"#gks1262-B26\" rid=\"gks1262-B26\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916103\">26</a>) has been extended (<a href=\"#gks1262-B27\" rid=\"gks1262-B27\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916106\">27</a>), by enabling the link between phosphorylated proteins present in Reactome pathways and phosphorylated proteins detected by MS approaches stored in PRIDE. The PRIDE BioMart data can also be accessed using a Representational State Transfer web service, which is heavily used. In addition, it is also possible to access the PRIDE BioMart at <a href=\"http://www.biomart.org\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">www.biomart.org</a>, together with many others. In that case, apart from Reactome, it is possible to perform common data searches involving PRIDE and other resources such as UniProt (<a href=\"#gks1262-B28\" rid=\"gks1262-B28\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916090\">28</a>), InterPro (<a href=\"#gks1262-B29\" rid=\"gks1262-B29\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916072\">29</a>), Ensembl (<a href=\"#gks1262-B30\" rid=\"gks1262-B30\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916095\">30</a>) and the Catalogue of Somatic Mutations in Cancer (COSMIC) (<a href=\"#gks1262-B31\" rid=\"gks1262-B31\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916107\">31</a>).</p></li><li><p id=\"__p17\">PRIDE FTP file server (<a href=\"http://ftp.ebi.ac.uk/pub/databases/pride/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">ftp.ebi.ac.uk/pub/databases/pride/</a>). At present, XML files corresponding to all public experiments in PRIDE can be downloaded in the mzData and PRIDE XML formats.</p></li><li><p id=\"__p18\">PRIDE Distributed Annotation System (DAS) server (<a href=\"http://www.ebi.ac.uk/pride-das/das/PrideDataSource/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.ebi.ac.uk/pride-das/das/PrideDataSource/</a>). A PRIDE DAS server (<a href=\"#gks1262-B32\" rid=\"gks1262-B32\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916094\">32</a>) was set up following the new specification 1.6. This service is publicly available and can be accessed through DAS clients such as Dasty (<a href=\"#gks1262-B33\" rid=\"gks1262-B33\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916081\">33</a>). The PRIDE DAS server has been designed for visualizing protein sequence and annotation, to display the identified peptides for the protein specified in the DAS request, together with the associated PTMs, and total peptide coverage.</p></li><li><p id=\"__p19\">A public PRIDE MySQL instance is now available and used, for instance, by the PRIDE Inspector, making this tool be the ideal way to access PRIDE data for most use cases (see next section).</p></li></ol><p>\n</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"gks1262-F1\" co-legend-rid=\"lgnd_gks1262-F1\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3531176/figure/gks1262-F1/\" target=\"figure\" rid-figpopup=\"gks1262-F1\" rid-ob=\"ob-gks1262-F1\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140127307264432\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3531176_gks1262f1p.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is gks1262f1p.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC3531176/bin/gks1262f1p.jpg\" /></a></span><span id=\"largeobj_idm140127307264432\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3531176/figure/gks1262-F1/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_gks1262-F1\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3531176/figure/gks1262-F1/\" target=\"figure\" rid-figpopup=\"gks1262-F1\" rid-ob=\"ob-gks1262-F1\">Figure 1.</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p21\">Summary of the ways the user can access and retrieve data from PRIDE. The web links to the existing PRIDE tools are also highlighted, including the PRIDE Converter 2 (needed for data submission).</p></span></span></span><p id=\"__p20\" class=\"p p-last\">Among the new datasets present in PRIDE, it is important to highlight that by July 2012, all data originally stored in the discontinued NCBI Peptidome (<a href=\"#gks1262-B11\" rid=\"gks1262-B11\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916063\">11</a>) had been reannotated, converted into a PRIDE compatible format and made publicly available under experiment accessions 17900-18271.\n</p></span><span id=\"s\r\n3\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n3title\" style=\"text-transform: uppercase;\">PRIDE SUBMISSION PROCESS AND RELATED TOOLS</h2><span id=\"s\r\n4\" class=\"sec sec-first\"><h3 id=\"s\r\n4title\">Submission tools</h3><p id=\"__p22\" class=\"p p-first\">At the moment of writing, submissions to PRIDE are performed using a publicly available XML data format called PRIDE XML, which is derived from mzData. The first step to perform a submission to PRIDE is then to generate PRIDE XML files. Several tools are available to make that process feasible and as straightforward as possible. The new PRIDE Converter 2 (<a href=\"#gks1262-B23\" rid=\"gks1262-B23\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916116\">23</a>) is now the recommended submission tool. It can convert a variety of popular proteomics data formats (e.g. Mascot.dat, X!Tandem.xml, OMSSA.csv, Proteome Discoverer.msf, plus all the mass spectral formats, among others) into well-annotated PRIDE XML files.</p><p id=\"__p23\">PRIDE Converter 2 can be used in two modes: (i) a Graphical User Interface mode, suited for most users; and (ii) a Command Line interface mode that makes possible the integration of the conversion process in external pipelines. Batch conversion of files is supported in both modes. Importantly, quantification results for the most popular techniques and 2D gel spot information can now be integrated in PRIDE XML files with PRIDE Converter 2, by providing that information in a new Proteomics Standards Initiative (PSI) tab-delimited standard format called mzTab (<a href=\"http://code.google.com/p/mztab/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://code.google.com/p/mztab/</a>). Detailed documentation for general users and developers is available at <a href=\"http://code.google.com/p/pride-converter-2/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://code.google.com/p/pride-converter-2/</a>. The PRIDE Converter 2 framework also includes the <em>PRIDE mzTab Generator, PRIDE XML Merger</em> and <em>PRIDE XML Filter</em> (<a href=\"#gks1262-B23\" rid=\"gks1262-B23\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916092\">23</a>). A mechanism to make a combined submission to PRIDE and IntAct (<a href=\"#gks1262-B34\" rid=\"gks1262-B34\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916087\">34</a>), the molecular interactions resource at the European Bioinformatics Institute, is also present in the PRIDE Converter 2.</p><p id=\"__p24\">The original PRIDE Converter tool (<a href=\"#gks1262-B21\" rid=\"gks1262-B21\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916070\">21</a>), one of the main reasons behind the large increase in data contents in PRIDE, is no longer recommended as the main submission tool and will not be maintained any more. Apart from the tools provided by the PRIDE team, there are several existing third-party pipelines/tools that produce PRIDE XML files, such as ProteinLynx Global Server (Waters), hEIDI (<a href=\"http://biodev.extra.cea.fr/docs/heidi\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://biodev.extra.cea.fr/docs/heidi</a>), OmicsHub Proteomics (Integromics), PeptideShaker (<a href=\"http://peptide-shaker.googlecode.com\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://peptide-shaker.googlecode.com</a>) and Proteios (<a href=\"#gks1262-B35\" rid=\"gks1262-B35\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916091\">35</a>).</p><p id=\"__p25\" class=\"p p-last\">At the moment, we are implementing full support in PRIDE for the new PSI standard formats mzML v1.1 (for MS data) (<a href=\"#gks1262-B36\" rid=\"gks1262-B36\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916099\">36</a>) and mzIdentML v1.1 (for protein and peptide IDs) (<a href=\"#gks1262-B37\" rid=\"gks1262-B37\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916071\">37</a>), based on the Java libraries jmzML (<a href=\"#gks1262-B38\" rid=\"gks1262-B38\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916074\">38</a>), jmzIdentML (<a href=\"#gks1262-B39\" rid=\"gks1262-B39\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916115\">39</a>) and jmzReader (<a href=\"#gks1262-B40\" rid=\"gks1262-B40\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916088\">40</a>). When this work is complete, data submissions in these formats will be natively supported, rather than having to be run through the PRIDE Converter 2. This will enable us to support in a much better way the reporting of protein inference (<a href=\"#gks1262-B41\" rid=\"gks1262-B41\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916067\">41</a>) and ambiguity in modification position.</p></span><span id=\"s\r\n5\" class=\"sec\"><h3 id=\"s\r\n5title\">PRIDE Inspector</h3><p id=\"__p26\" class=\"p p-first\">PRIDE Inspector is a popular tool introduced in 2011 (&#x0003e;4500 downloads by September 2012), which can be used to visualize and perform an initial quality assessment of the submitted data to PRIDE (<a href=\"#gks1262-B22\" rid=\"gks1262-B22\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916089\">22</a>) (<a href=\"http://code.google.com/p/pride-toolsuite/wiki/PRIDEInspector\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://code.google.com/p/pride-toolsuite/wiki/PRIDEInspector</a>). PRIDE Inspector provides different views on the available data, each focusing on a different aspect: experimental details (biological and technical metadata), protein, peptide, quantification values (if available) and summary charts. This last view is one of the major strengths of the tool because it is possible to perform an initial assessment of data quality, using a variety of simple charts that are generated automatically (<a href=\"#gks1262-B22\" rid=\"gks1262-B22\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916078\">22</a>). Using PRIDE Inspector, proteomics researchers can examine their own data sets before the actual submission to PRIDE is performed, and journal editors and reviewers can perform a thorough review of submitted and private data at the pre-publication stage.</p><p id=\"__p27\" class=\"p p-last\">In addition, through the &#x02018;Search PRIDE Database&#x02019; option, PRIDE Inspector can access data already in PRIDE for data mining purposes using the PRIDE public MySQL instance, which is updated regularly. Apart from being a stand-alone tool, as mentioned before, PRIDE Inspector can also be accessed using Java Web Start at the PRIDE web page.</p></span><span id=\"s\r\n6\" class=\"sec sec-last\"><h3 id=\"s\r\n6title\">PRIDE and the ProteomeXchange consortium</h3><p id=\"__p28\" class=\"p p-first-last\">PRIDE is a founding member of the PX consortium (<a href=\"http://www.proteomexchange.org\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.proteomexchange.org</a>) (<a href=\"#gks1262-B42\" rid=\"gks1262-B42\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916066\">42</a>). The members of the consortium, led by PRIDE and PeptideAtlas (<a href=\"#gks1262-B9\" rid=\"gks1262-B9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916084\">9</a>), are implementing a system to enable the automated and standardized sharing of MS-based proteomics data between the main existing MS proteomics repositories. In the first implementation of the data workflow, PRIDE acts as the initial submission point for MS/MS data. At the moment of writing, around 50 PX datasets have been submitted (see updated list of publicly available datasets at <a href=\"http://proteomecentral.proteomexchange.org\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://proteomecentral.proteomexchange.org</a>). As a result of the PX efforts, PRIDE has started to accept raw data (in either binary or an open XML format) because it is a mandatory component of a PX submission. The files are accessible through FTP and are stored at the EBI raw file repository (<a href=\"#gks1262-B43\" rid=\"gks1262-B43\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916085\">43</a>).</p></span></span><span id=\"s\r\n7\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n7title\" style=\"text-transform: uppercase;\">QUALITY CONTROL IN PRIDE</h2><p id=\"__p29\" class=\"p p-first\">A major focus of PRIDE development in the past 2 years was to ensure at the very least minimal annotation of experiments and to perform basic quality checks of the submitted data to PRIDE. The development of PRIDE Inspector was the key step forward in that direction, and many of its components have been used in the internal PRIDE submission pipeline. The automated pipeline allows the detection of clear errors in the submitted data that are notified to the submitter and can then be corrected (<a href=\"#gks1262-B44\" rid=\"gks1262-B44\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916110\">44</a>). Finally, the development of the PRIDE Converter 2 as the new submission tool has improved consistency at the level of experimental annotation.</p><p id=\"__p30\" class=\"p p-last\">In 2013, we will finalize and release a new resource called PRIDE-Q, as a quality-controlled subset of PRIDE, which can fulfil both a minimum level of annotation and Peptide Spectrum Match quality standards.</p></span><span id=\"s\r\n8\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n8title\" style=\"text-transform: uppercase;\">DISCUSSION</h2><p id=\"__p31\" class=\"p p-first\">In the past 3 years, PRIDE has worked on two main tasks: (i) development of robust data submission pipelines (such as the development of the PRIDE Converter 2), including the initial implementation of the PX consortium data workflow, and the possibility to capture quantification information in a standardized way; and (ii) establishment of QC checks, including the development of PRIDE Inspector and an internal data submission pipeline, able to flag obvious errors that can be then communicated to the submitters.</p><p id=\"__p32\">However, many more future efforts will be needed in both directions. We are working on the development of the new PRIDE system (including a new database schema and web interface) that will fully support the PSI standards mzML and mzIdentML. This will be a gradual process, and support for these formats will be added sequentially to the PRIDE system (also as submission formats) and tools, while we will also keep supporting PRIDE XML.</p><p id=\"__p33\">On the other hand, the release of PRIDE-Q, envisioned to help non-expert proteomics biologists to &#x02018;digest&#x02019; the potentially complex information coming from MS data, will happen in 2013. However, the quality requirements will need to be refined and will evolve dynamically over time.</p><p id=\"__p34\" class=\"p p-last\">It is worth highlighting that PRIDE has been used for research purposes in several recent studies involving the meta-analysis of combined data coming from very different proteomics experimental setups (<a href=\"#gks1262-B45\" rid=\"gks1262-B45\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916076\">45&#x02013;47</a>), the improvement and assessment of existing protein sequence databases (<a href=\"#gks1262-B48\" rid=\"gks1262-B48\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916093\">48</a>,<a href=\"#gks1262-B49\" rid=\"gks1262-B49\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916062\">49</a>) or to support genomics-related findings (<a href=\"#gks1262-B50\" rid=\"gks1262-B50\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916105\">50</a>,<a href=\"#gks1262-B51\" rid=\"gks1262-B51\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916083\">51</a>). Some research has also been performed to demonstrate the usefulness of data in PRIDE to perform <em>a posteriori</em> QC of the stored data (<a href=\"#gks1262-B52\" rid=\"gks1262-B52\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916098\">52</a>). We expect that this trend will continue to grow in the near future, and that PRIDE continues on a trajectory from a publication-centric repository to an integrative resource for MS-based protein expression data. PRIDE will keep playing an important role for the community, also in the context of the nascent Human Proteome Project (<a href=\"#gks1262-B53\" rid=\"gks1262-B53\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_404916108\">53</a>). We invite interested parties in PRIDE developments (including the associated software and tools) to follow the PRIDE Twitter account (@pride_ebi).</p></span><span id=\"s\r\n9\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n9title\" style=\"text-transform: uppercase;\">FUNDING</h2><p id=\"__p35\" class=\"p p-first\">The PRIDE team is funded by the Wellcome Trust [<span>WT085949MA</span>]; EU FP7 grants &#x02018;Sling&#x02019; [<span>226073</span>]; &#x02018;ProteomeXchange&#x02019; [<span>260558</span>]; &#x02018;PRIME-XS&#x02019; [<span>262067</span>]; &#x02018;LipidomicNet&#x02019; [<span>202272</span>]; BBSRC grant &#x02018;PRIDE Converter&#x02019; [reference <span>BB/I024204/1</span>] and EMBL core funding. Funding for open access charge: Wellcome Trust [WT085949MA].</p><p id=\"__p36\" class=\"p p-last\"><em>Conflict of interest statement.</em> None declared.</p></span><span id=\"idm140127297219632\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140127297219632title\" style=\"text-transform: uppercase;\">ACKNOWLEDGEMENTS</h2><span class=\"sec\"><p id=\"__p37\">The PRIDE team would like to thank all data submitters for their contributions.</p></span></span><br><div class=\"ref-list-sec sec\" id=\"reference-list\"><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B1\">1. <span class=\"element-citation\">Mallick P, Kuster B. Proteomics: a pragmatic perspective. <span><span class=\"ref-journal\">Nat. Biotechnol. </span>2010;<span class=\"ref-vol\">28</span>:695–709.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20622844\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=Proteomics:+a+pragmatic+perspective&amp;author=P+Mallick&amp;author=B+Kuster&amp;volume=28&amp;publication_year=2010&amp;pages=695-709&amp;pmid=20622844&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B2\">2. <span class=\"element-citation\">Cui W, Rohrs HW, Gross ML. Top-down mass spectrometry: recent developments, applications and perspectives. <span><span class=\"ref-journal\">Analyst. </span>2011;<span class=\"ref-vol\">136</span>:3854–3864.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3505190/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21826297\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Analyst&amp;title=Top-down+mass+spectrometry:+recent+developments,+applications+and+perspectives&amp;author=W+Cui&amp;author=HW+Rohrs&amp;author=ML+Gross&amp;volume=136&amp;publication_year=2011&amp;pages=3854-3864&amp;pmid=21826297&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B3\">3. <span class=\"element-citation\">Picotti P, Aebersold R. Selected reaction monitoring-based proteomics: workflows, potential, pitfalls and future directions. <span><span class=\"ref-journal\">Nat. Methods. </span>2012;<span class=\"ref-vol\">9</span>:555–566.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22669653\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Selected+reaction+monitoring-based+proteomics:+workflows,+potential,+pitfalls+and+future+directions&amp;author=P+Picotti&amp;author=R+Aebersold&amp;volume=9&amp;publication_year=2012&amp;pages=555-566&amp;pmid=22669653&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B4\">4. <span class=\"element-citation\">Martens L, Hermjakob H, Jones P, Adamski M, Taylor C, States D, Gevaert K, Vandekerckhove J, Apweiler R. PRIDE: the proteomics identifications database. <span><span class=\"ref-journal\">Proteomics. </span>2005;<span class=\"ref-vol\">5</span>:3537–3545.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n16041671\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=PRIDE:+the+proteomics+identifications+database&amp;author=L+Martens&amp;author=H+Hermjakob&amp;author=P+Jones&amp;author=M+Adamski&amp;author=C+Taylor&amp;volume=5&amp;publication_year=2005&amp;pages=3537-3545&amp;pmid=16041671&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B5\">5. <span class=\"element-citation\">Jones P, Cote RG, Martens L, Quinn AF, Taylor CF, Derache W, Hermjakob H, Apweiler R. PRIDE: a public repository of protein and peptide identifications for the proteomics community. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2006;<span class=\"ref-vol\">34</span>:D659–D663.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC1347500/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n16381953\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=PRIDE:+a+public+repository+of+protein+and+peptide+identifications+for+the+proteomics+community&amp;author=P+Jones&amp;author=RG+Cote&amp;author=L+Martens&amp;author=AF+Quinn&amp;author=CF+Taylor&amp;volume=34&amp;publication_year=2006&amp;pages=D659-D663&amp;pmid=16381953&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B6\">6. <span class=\"element-citation\">Jones P, Cote RG, Cho SY, Klie S, Martens L, Quinn AF, Thorneycroft D, Hermjakob H. PRIDE: new developments and new datasets. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2008;<span class=\"ref-vol\">36</span>:D878–D883.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2238846/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n18033805\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=PRIDE:+new+developments+and+new+datasets&amp;author=P+Jones&amp;author=RG+Cote&amp;author=SY+Cho&amp;author=S+Klie&amp;author=L+Martens&amp;volume=36&amp;publication_year=2008&amp;pages=D878-D883&amp;pmid=18033805&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B7\">7. <span class=\"element-citation\">Vizcaino JA, C&#x000f4;t&#x000e9; R, Reisinger F, Foster J, Mueller M, Rameseder J, Hermjakob H, Martens L. A guide to the Proteomics Identifications Database proteomics data repository. <span><span class=\"ref-journal\">Proteomics. </span>2009;<span class=\"ref-vol\">9</span>:4276–4283.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2970915/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19662629\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=A+guide+to+the+Proteomics+Identifications+Database+proteomics+data+repository&amp;author=JA+Vizcaino&amp;author=R+C&#x000f4;t&#x000e9;&amp;author=F+Reisinger&amp;author=J+Foster&amp;author=M+Mueller&amp;volume=9&amp;publication_year=2009&amp;pages=4276-4283&amp;pmid=19662629&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B8\">8. <span class=\"element-citation\">Vizcaino JA, Foster JM, Martens L. Proteomics data repositories: providing a safe haven for your data and acting as a springboard for further research. <span><span class=\"ref-journal\">J Proteomics. </span>2010;<span class=\"ref-vol\">73</span>:2136–2146.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2958306/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20615486\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Proteomics&amp;title=Proteomics+data+repositories:+providing+a+safe+haven+for+your+data+and+acting+as+a+springboard+for+further+research&amp;author=JA+Vizcaino&amp;author=JM+Foster&amp;author=L+Martens&amp;volume=73&amp;publication_year=2010&amp;pages=2136-2146&amp;pmid=20615486&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B9\">9. <span class=\"element-citation\">Deutsch EW, Lam H, Aebersold R. PeptideAtlas: a resource for target selection for emerging targeted proteomics workflows. <span><span class=\"ref-journal\">EMBO Rep. </span>2008;<span class=\"ref-vol\">9</span>:429–434.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2373374/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n18451766\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=EMBO+Rep.&amp;title=PeptideAtlas:+a+resource+for+target+selection+for+emerging+targeted+proteomics+workflows&amp;author=EW+Deutsch&amp;author=H+Lam&amp;author=R+Aebersold&amp;volume=9&amp;publication_year=2008&amp;pages=429-434&amp;pmid=18451766&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B10\">10. <span class=\"element-citation\">Craig R, Cortens JP, Beavis RC. Open source system for analyzing, validating, and storing protein identification data. <span><span class=\"ref-journal\">J. Proteome Res. </span>2004;<span class=\"ref-vol\">3</span>:1234–1242.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n15595733\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Proteome+Res.&amp;title=Open+source+system+for+analyzing,+validating,+and+storing+protein+identification+data&amp;author=R+Craig&amp;author=JP+Cortens&amp;author=RC+Beavis&amp;volume=3&amp;publication_year=2004&amp;pages=1234-1242&amp;pmid=15595733&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B11\">11. <span class=\"element-citation\">Slotta DJ, Barrett T, Edgar R. NCBI Peptidome: a new public repository for mass spectrometry peptide identifications. <span><span class=\"ref-journal\">Nat. Biotechnol. </span>2009;<span class=\"ref-vol\">27</span>:600–601.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2745988/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19587658\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=NCBI+Peptidome:+a+new+public+repository+for+mass+spectrometry+peptide+identifications&amp;author=DJ+Slotta&amp;author=T+Barrett&amp;author=R+Edgar&amp;volume=27&amp;publication_year=2009&amp;pages=600-601&amp;pmid=19587658&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B12\">12. <span class=\"element-citation\">Schaab C, Geiger T, Stoehr G, Cox J, Mann M. Analysis of high accuracy, quantitative proteomics data in the MaxQB database. <span><span class=\"ref-journal\">Mol. Cell Proteomics. </span>2012;<span class=\"ref-vol\">11</span>:M111.014068.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3316731/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22301388\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Cell+Proteomics&amp;title=Analysis+of+high+accuracy,+quantitative+proteomics+data+in+the+MaxQB+database&amp;author=C+Schaab&amp;author=T+Geiger&amp;author=G+Stoehr&amp;author=J+Cox&amp;author=M+Mann&amp;volume=11&amp;publication_year=2012&amp;pages=M111.014068&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B13\">13. <span class=\"element-citation\">Mead JA, Bianco L, Bessant C. Recent developments in public proteomic MS repositories and pipelines. <span><span class=\"ref-journal\">Proteomics. </span>2009;<span class=\"ref-vol\">9</span>:861–881.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19212957\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Recent+developments+in+public+proteomic+MS+repositories+and+pipelines&amp;author=JA+Mead&amp;author=L+Bianco&amp;author=C+Bessant&amp;volume=9&amp;publication_year=2009&amp;pages=861-881&amp;pmid=19212957&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B14\">14. <span class=\"element-citation\">Farrah T, Deutsch EW, Kreisberg R, Sun Z, Campbell DS, Mendoza L, Kusebauch U, Brusniak MY, Huttenhain R, Schiess R, et al.  PASSEL: the PeptideAtlas SRMexperiment library. <span><span class=\"ref-journal\">Proteomics. </span>2012;<span class=\"ref-vol\">12</span>:1170–1175.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3832291/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22318887\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=PASSEL:+the+PeptideAtlas+SRMexperiment+library&amp;author=T+Farrah&amp;author=EW+Deutsch&amp;author=R+Kreisberg&amp;author=Z+Sun&amp;author=DS+Campbell&amp;volume=12&amp;publication_year=2012&amp;pages=1170-1175&amp;pmid=22318887&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B15\">15. <span class=\"element-citation\">Kolker E, Higdon R, Haynes W, Welch D, Broomall W, Lancet D, Stanberry L, Kolker N. MOPED: model organism protein expression database. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2012;<span class=\"ref-vol\">40</span>:D1093–D1099.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3245040/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22139914\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=MOPED:+model+organism+protein+expression+database&amp;author=E+Kolker&amp;author=R+Higdon&amp;author=W+Haynes&amp;author=D+Welch&amp;author=W+Broomall&amp;volume=40&amp;publication_year=2012&amp;pages=D1093-D1099&amp;pmid=22139914&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B16\">16. <span class=\"element-citation\">Wang M, Weiss M, Simonovic M, Haertinger G, Schrimpf SP, Hengartner MO, von Mering C. PaxDb, a database of protein abundance averages across all three domains of life. <span><span class=\"ref-journal\">Mol. Cell Proteomics. </span>2012;<span class=\"ref-vol\">11</span>:492–500.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3412977/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22535208\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Cell+Proteomics&amp;title=PaxDb,+a+database+of+protein+abundance+averages+across+all+three+domains+of+life&amp;author=M+Wang&amp;author=M+Weiss&amp;author=M+Simonovic&amp;author=G+Haertinger&amp;author=SP+Schrimpf&amp;volume=11&amp;publication_year=2012&amp;pages=492-500&amp;pmid=22535208&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B17\">17. <span class=\"element-citation\">Lane L, Argoud-Puy G, Britan A, Cusin I, Duek PD, Evalet O, Gateau A, Gaudet P, Gleizes A, Masselot A, et al.  neXtProt: a knowledge platform for human proteins. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2012;<span class=\"ref-vol\">40</span>:D76–D83.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3245017/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22139911\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=neXtProt:+a+knowledge+platform+for+human+proteins&amp;author=L+Lane&amp;author=G+Argoud-Puy&amp;author=A+Britan&amp;author=I+Cusin&amp;author=PD+Duek&amp;volume=40&amp;publication_year=2012&amp;pages=D76-D83&amp;pmid=22139911&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B18\">18. <span class=\"element-citation\">Wein SP, Cote RG, Dumousseau M, Reisinger F, Hermjakob H, Vizcaino JA. Improvements in the protein identifier cross-reference service. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2012;<span class=\"ref-vol\">40</span>:W276–W280.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3394263/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22544604\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Improvements+in+the+protein+identifier+cross-reference+service&amp;author=SP+Wein&amp;author=RG+Cote&amp;author=M+Dumousseau&amp;author=F+Reisinger&amp;author=H+Hermjakob&amp;volume=40&amp;publication_year=2012&amp;pages=W276-W280&amp;pmid=22544604&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B19\">19. <span class=\"element-citation\">Cote R, Reisinger F, Martens L, Barsnes H, Vizcaino JA, Hermjakob H. The Ontology Lookup Service: bigger and better. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2010;<span class=\"ref-vol\">38</span>:W155–W160.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2896109/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20460452\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+Ontology+Lookup+Service:+bigger+and+better&amp;author=R+Cote&amp;author=F+Reisinger&amp;author=L+Martens&amp;author=H+Barsnes&amp;author=JA+Vizcaino&amp;volume=38&amp;publication_year=2010&amp;pages=W155-W160&amp;pmid=20460452&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B20\">20. <span class=\"element-citation\">Reisinger F, Martens L. Database on demand&#x02014;an online tool for the custom generation of FASTA formatted sequence databases. <span><span class=\"ref-journal\">Proteomics. </span>2009;<span class=\"ref-vol\">9</span>:4421–4424.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19725076\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Database+on+demand&#x02014;an+online+tool+for+the+custom+generation+of+FASTA+formatted+sequence+databases&amp;author=F+Reisinger&amp;author=L+Martens&amp;volume=9&amp;publication_year=2009&amp;pages=4421-4424&amp;pmid=19725076&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B21\">21. <span class=\"element-citation\">Barsnes H, Vizcaino JA, Eidhammer I, Martens L. PRIDE Converter: making proteomics data-sharing easy. <span><span class=\"ref-journal\">Nat. Biotechnol. </span>2009;<span class=\"ref-vol\">27</span>:598–599.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19587657\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=PRIDE+Converter:+making+proteomics+data-sharing+easy&amp;author=H+Barsnes&amp;author=JA+Vizcaino&amp;author=I+Eidhammer&amp;author=L+Martens&amp;volume=27&amp;publication_year=2009&amp;pages=598-599&amp;pmid=19587657&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B22\">22. <span class=\"element-citation\">Wang R, Fabregat A, Rios D, Ovelleiro D, Foster JM, Cote RG, Griss J, Csordas A, Perez-Riverol Y, Reisinger F, et al.  PRIDE Inspector: a tool to visualize and validate MS proteomics data. <span><span class=\"ref-journal\">Nat. Biotechnol. </span>2012;<span class=\"ref-vol\">30</span>:135–137.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3277942/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22318026\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=PRIDE+Inspector:+a+tool+to+visualize+and+validate+MS+proteomics+data&amp;author=R+Wang&amp;author=A+Fabregat&amp;author=D+Rios&amp;author=D+Ovelleiro&amp;author=JM+Foster&amp;volume=30&amp;publication_year=2012&amp;pages=135-137&amp;pmid=22318026&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B23\">23. <span class=\"element-citation\">Cote RG, Griss J, Dianes JA, Wang R, Wright JC, van den Toorn HW, van Breukelen B, Heck AJ, Hulstaert N, Martens L, et al.  The PRIDE Converter 2 framework: an improved suite of tools to facilitate data submission to the PRIDE database and the ProteomeXchange consortium. <span><span class=\"ref-journal\">Mol. Cell Proteomics. </span>2012;<span class=\"ref-vol\">11</span>:1682–1689.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3518121/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22949509\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Cell+Proteomics&amp;title=The+PRIDE+Converter+2+framework:+an+improved+suite+of+tools+to+facilitate+data+submission+to+the+PRIDE+database+and+the+ProteomeXchange+consortium&amp;author=RG+Cote&amp;author=J+Griss&amp;author=JA+Dianes&amp;author=R+Wang&amp;author=JC+Wright&amp;volume=11&amp;publication_year=2012&amp;pages=1682-1689&amp;pmid=22949509&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B24\">24. <span class=\"element-citation\">Stadtman ER, Van Remmen H, Richardson A, Wehr NB, Levine RL. Methionine oxidation and aging. <span><span class=\"ref-journal\">Biochim. Biophys. Acta. </span>2005;<span class=\"ref-vol\">1703</span>:135–140.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n15680221\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Biochim.+Biophys.+Acta&amp;title=Methionine+oxidation+and+aging&amp;author=ER+Stadtman&amp;author=H+Van+Remmen&amp;author=A+Richardson&amp;author=NB+Wehr&amp;author=RL+Levine&amp;volume=1703&amp;publication_year=2005&amp;pages=135-140&amp;pmid=15680221&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B25\">25. <span class=\"element-citation\">Zhang J, Haider S, Baran J, Cros A, Guberman JM, Hsu J, Liang Y, Yao L, Kasprzyk A. BioMart: a data federation framework for large collaborative projects. <span><span class=\"ref-journal\">Database. </span>2011;<span class=\"ref-vol\">2011</span>:bar038.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3175789/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21930506\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Database&amp;title=BioMart:+a+data+federation+framework+for+large+collaborative+projects&amp;author=J+Zhang&amp;author=S+Haider&amp;author=J+Baran&amp;author=A+Cros&amp;author=JM+Guberman&amp;volume=2011&amp;publication_year=2011&amp;pages=bar038&amp;pmid=21930506&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B26\">26. <span class=\"element-citation\">Croft D, O\'Kelly G, Wu G, Haw R, Gillespie M, Matthews L, Caudy M, Garapati P, Gopinath G, Jassal B, et al.  Reactome: a database of reactions, pathways and biological processes. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2011;<span class=\"ref-vol\">39</span>:D691–D697.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3013646/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21067998\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Reactome:+a+database+of+reactions,+pathways+and+biological+processes&amp;author=D+Croft&amp;author=G+O\'Kelly&amp;author=G+Wu&amp;author=R+Haw&amp;author=M+Gillespie&amp;volume=39&amp;publication_year=2011&amp;pages=D691-D697&amp;pmid=21067998&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B27\">27. <span class=\"element-citation\">Ndegwa N, Cote RG, Ovelleiro D, D\'Eustachio P, Hermjakob H, Vizcaino JA, Croft D. Critical amino acid residues in proteins: a BioMart integration of Reactome protein annotations with PRIDE mass spectrometry data and COSMIC somatic mutations. <span><span class=\"ref-journal\">Database (Oxford) </span>2011;<span class=\"ref-vol\">2011</span>:bar047.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3199918/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22025670\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Database+(Oxford)&amp;title=Critical+amino+acid+residues+in+proteins:+a+BioMart+integration+of+Reactome+protein+annotations+with+PRIDE+mass+spectrometry+data+and+COSMIC+somatic+mutations&amp;author=N+Ndegwa&amp;author=RG+Cote&amp;author=D+Ovelleiro&amp;author=P+D\'Eustachio&amp;author=H+Hermjakob&amp;volume=2011&amp;publication_year=2011&amp;pages=bar047&amp;pmid=22025670&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B28\">28. <span class=\"element-citation\">The UniProt Consortium. Reorganizing the protein space at the Universal Protein Resource (UniProt) <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2012;<span class=\"ref-vol\">40</span>:D71–D75.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3245120/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22102590\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Reorganizing+the+protein+space+at+the+Universal+Protein+Resource+(UniProt)&amp;volume=40&amp;publication_year=2012&amp;pages=D71-D75&amp;pmid=22102590&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B29\">29. <span class=\"element-citation\">Hunter S, Jones P, Mitchell A, Apweiler R, Attwood TK, Bateman A, Bernard T, Binns D, Bork P, Burge S, et al.  InterPro in 2011: new developments in the family and domain prediction database. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2012;<span class=\"ref-vol\">40</span>:D306–D312.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3245097/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22096229\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=InterPro+in+2011:+new+developments+in+the+family+and+domain+prediction+database&amp;author=S+Hunter&amp;author=P+Jones&amp;author=A+Mitchell&amp;author=R+Apweiler&amp;author=TK+Attwood&amp;volume=40&amp;publication_year=2012&amp;pages=D306-D312&amp;pmid=22096229&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B30\">30. <span class=\"element-citation\">Flicek P, Amode MR, Barrell D, Beal K, Brent S, Carvalho-Silva D, Clapham P, Coates G, Fairley S, Fitzgerald S, et al.  Ensembl 2012. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2012;<span class=\"ref-vol\">40</span>:D84–D90.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3245178/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22086963\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Ensembl+2012&amp;author=P+Flicek&amp;author=MR+Amode&amp;author=D+Barrell&amp;author=K+Beal&amp;author=S+Brent&amp;volume=40&amp;publication_year=2012&amp;pages=D84-D90&amp;pmid=22086963&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B31\">31. <span class=\"element-citation\">Forbes SA, Bindal N, Bamford S, Cole C, Kok CY, Beare D, Jia M, Shepherd R, Leung K, Menzies A, et al.  COSMIC: mining complete cancer genomes in the catalogue of somatic mutations in cancer. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2011;<span class=\"ref-vol\">39</span>:D945–D950.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3013785/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20952405\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=COSMIC:+mining+complete+cancer+genomes+in+the+catalogue+of+somatic+mutations+in+cancer&amp;author=SA+Forbes&amp;author=N+Bindal&amp;author=S+Bamford&amp;author=C+Cole&amp;author=CY+Kok&amp;volume=39&amp;publication_year=2011&amp;pages=D945-D950&amp;pmid=20952405&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B32\">32. <span class=\"element-citation\">Dowell RD, Jokerst RM, Day A, Eddy SR, Stein L. The distributed annotation system. <span><span class=\"ref-journal\">BMC Bioinformatics. </span>2001;<span class=\"ref-vol\"> 2</span>:7.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC58584/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n11667947\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=The+distributed+annotation+system&amp;author=RD+Dowell&amp;author=RM+Jokerst&amp;author=A+Day&amp;author=SR+Eddy&amp;author=L+Stein&amp;volume=+2&amp;publication_year=2001&amp;pages=7&amp;pmid=11667947&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B33\">33. <span class=\"element-citation\">Villaveces JM, Jimenez RC, Garcia LJ, Salazar GA, Gel B, Mulder N, Martin M, Garcia A, Hermjakob H. Dasty3, a WEB framework for DAS. <span><span class=\"ref-journal\">Bioinformatics. </span>2011;<span class=\"ref-vol\">27</span>:2616–2617.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3167052/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21798964\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Dasty3,+a+WEB+framework+for+DAS&amp;author=JM+Villaveces&amp;author=RC+Jimenez&amp;author=LJ+Garcia&amp;author=GA+Salazar&amp;author=B+Gel&amp;volume=27&amp;publication_year=2011&amp;pages=2616-2617&amp;pmid=21798964&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B34\">34. <span class=\"element-citation\">Kerrien S, Aranda B, Breuza L, Bridge A, Broackes-Carter F, Chen C, Duesbury M, Dumousseau M, Feuermann M, Hinz U, et al.  The IntAct molecular interaction database in 2012. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2012;<span class=\"ref-vol\">40</span>:D841–D846.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3245075/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22121220\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+IntAct+molecular+interaction+database+in+2012&amp;author=S+Kerrien&amp;author=B+Aranda&amp;author=L+Breuza&amp;author=A+Bridge&amp;author=F+Broackes-Carter&amp;volume=40&amp;publication_year=2012&amp;pages=D841-D846&amp;pmid=22121220&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B35\">35. <span class=\"element-citation\">Hakkinen J, Vincic G, Mansson O, Warell K, Levander F. The proteios software environment: an extensible multiuser platform for management and analysis of proteomics data. <span><span class=\"ref-journal\">J. Proteome Res. </span>2009;<span class=\"ref-vol\">8</span>:3037–3043.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19354269\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Proteome+Res.&amp;title=The+proteios+software+environment:+an+extensible+multiuser+platform+for+management+and+analysis+of+proteomics+data&amp;author=J+Hakkinen&amp;author=G+Vincic&amp;author=O+Mansson&amp;author=K+Warell&amp;author=F+Levander&amp;volume=8&amp;publication_year=2009&amp;pages=3037-3043&amp;pmid=19354269&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B36\">36. <span class=\"element-citation\">Martens L, Chambers M, Sturm M, Kessner D, Levander F, Shofstahl J, Tang WH, Rompp A, Neumann S, Pizarro AD, et al.  mzML&#x02014;a community standard for mass spectrometry data. <span><span class=\"ref-journal\">Mol. Cell Proteomics. </span>2011;<span class=\"ref-vol\">10</span>:R110 000133.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3013463/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20716697\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Cell+Proteomics&amp;title=mzML&#x02014;a+community+standard+for+mass+spectrometry+data&amp;author=L+Martens&amp;author=M+Chambers&amp;author=M+Sturm&amp;author=D+Kessner&amp;author=F+Levander&amp;volume=10&amp;publication_year=2011&amp;pages=R110+000133&amp;pmid=20716697&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B37\">37. <span class=\"element-citation\">Jones AR, Eisenacher M, Mayer G, Kohlbacher O, Siepen J, Hubbard SJ, Selley JN, Searle BC, Shofstahl J, Seymour SL, et al.  The mzIdentML data standard for mass spectrometry-based proteomics results. <span><span class=\"ref-journal\">Mol. Cell Proteomics. </span>2012;<span class=\"ref-vol\">11</span>:M111.014381.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3394945/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22375074\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Cell+Proteomics&amp;title=The+mzIdentML+data+standard+for+mass+spectrometry-based+proteomics+results&amp;author=AR+Jones&amp;author=M+Eisenacher&amp;author=G+Mayer&amp;author=O+Kohlbacher&amp;author=J+Siepen&amp;volume=11&amp;publication_year=2012&amp;pages=M111.014381&amp;pmid=22375074&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B38\">38. <span class=\"element-citation\">Cote RG, Reisinger F, Martens L. jmzML, an open-source Java API for mzML, the PSI standard for MS data. <span><span class=\"ref-journal\">Proteomics. </span>2010;<span class=\"ref-vol\">10</span>:1332–1335.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20127693\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=jmzML,+an+open-source+Java+API+for+mzML,+the+PSI+standard+for+MS+data&amp;author=RG+Cote&amp;author=F+Reisinger&amp;author=L+Martens&amp;volume=10&amp;publication_year=2010&amp;pages=1332-1335&amp;pmid=20127693&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B39\">39. <span class=\"element-citation\">Reisinger F, Krishna R, Ghali F, Rios D, Hermjakob H, Vizcaino JA, Jones AR. jmzIdentML API: a Java interface to the mzIdentML standard for peptide and protein identification data. <span><span class=\"ref-journal\">Proteomics. </span>2012;<span class=\"ref-vol\">12</span>:790–794.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3933944/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22539429\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=jmzIdentML+API:+a+Java+interface+to+the+mzIdentML+standard+for+peptide+and+protein+identification+data&amp;author=F+Reisinger&amp;author=R+Krishna&amp;author=F+Ghali&amp;author=D+Rios&amp;author=H+Hermjakob&amp;volume=12&amp;publication_year=2012&amp;pages=790-794&amp;pmid=22539429&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B40\">40. <span class=\"element-citation\">Griss J, Reisinger F, Hermjakob H, Vizcaino JA. jmzReader: a Java parser library to process and visualize multiple text and XML-based mass spectrometry data formats. <span><span class=\"ref-journal\">Proteomics. </span>2012;<span class=\"ref-vol\">12</span>:795–798.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3472022/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22539430\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=jmzReader:+a+Java+parser+library+to+process+and+visualize+multiple+text+and+XML-based+mass+spectrometry+data+formats&amp;author=J+Griss&amp;author=F+Reisinger&amp;author=H+Hermjakob&amp;author=JA+Vizcaino&amp;volume=12&amp;publication_year=2012&amp;pages=795-798&amp;pmid=22539430&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B41\">41. <span class=\"element-citation\">Nesvizhskii AI, Aebersold R. Interpretation of shotgun proteomic data: the protein inference problem. <span><span class=\"ref-journal\">Mol. Cell Proteomics. </span>2005;<span class=\"ref-vol\">4</span>:1419–1440.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n16009968\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Cell+Proteomics&amp;title=Interpretation+of+shotgun+proteomic+data:+the+protein+inference+problem&amp;author=AI+Nesvizhskii&amp;author=R+Aebersold&amp;volume=4&amp;publication_year=2005&amp;pages=1419-1440&amp;pmid=16009968&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B42\">42. <span class=\"element-citation\">Hermjakob H, Apweiler R. The Proteomics Identifications Database (PRIDE) and the ProteomExchange Consortium: making proteomics data accessible. <span><span class=\"ref-journal\">Expert Rev. Proteomics. </span>2006;<span class=\"ref-vol\">3</span>:1–3.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n16445344\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Expert+Rev.+Proteomics&amp;title=The+Proteomics+Identifications+Database+(PRIDE)+and+the+ProteomExchange+Consortium:+making+proteomics+data+accessible&amp;author=H+Hermjakob&amp;author=R+Apweiler&amp;volume=3&amp;publication_year=2006&amp;pages=1-3&amp;pmid=16445344&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B43\">43. <span class=\"element-citation\">Editorial. A home for raw proteomics data. <span><span class=\"ref-journal\">Nat. Methods. </span>2012;<span class=\"ref-vol\">9</span>:419.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22803195\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=A+home+for+raw+proteomics+data&amp;volume=9&amp;publication_year=2012&amp;pages=419&amp;pmid=22803195&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B44\">44. <span class=\"element-citation\">Csordas A, Ovelleiro D, Wang R, Foster JM, Rios D, Vizcaino JA, Hermjakob H. PRIDE: quality control in a proteomics data repository. <span><span class=\"ref-journal\">Database (Oxford) </span>2012;<span class=\"ref-vol\">2012</span>:bas004.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3308160/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22434838\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Database+(Oxford)&amp;title=PRIDE:+quality+control+in+a+proteomics+data+repository&amp;author=A+Csordas&amp;author=D+Ovelleiro&amp;author=R+Wang&amp;author=JM+Foster&amp;author=D+Rios&amp;volume=2012&amp;publication_year=2012&amp;pages=bas004&amp;pmid=22434838&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B45\">45. <span class=\"element-citation\">Mueller M, Vizcaino JA, Jones P, Cote R, Thorneycroft D, Apweiler R, Hermjakob H, Martens L. Analysis of the experimental detection of central nervous system-related genes in human brain and cerebrospinal fluid datasets. <span><span class=\"ref-journal\">Proteomics. </span>2008;<span class=\"ref-vol\">8</span>:1138–1148.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n18283668\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Analysis+of+the+experimental+detection+of+central+nervous+system-related+genes+in+human+brain+and+cerebrospinal+fluid+datasets&amp;author=M+Mueller&amp;author=JA+Vizcaino&amp;author=P+Jones&amp;author=R+Cote&amp;author=D+Thorneycroft&amp;volume=8&amp;publication_year=2008&amp;pages=1138-1148&amp;pmid=18283668&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B46\">46. <span class=\"element-citation\">Klie S, Martens L, Vizcaino JA, Cote R, Jones P, Apweiler R, Hinneburg A, Hermjakob H. Analyzing large-scale proteomics projects with latent semantic indexing. <span><span class=\"ref-journal\">J. Proteome Res. </span>2008;<span class=\"ref-vol\">7</span>:182–191.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n18047271\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Proteome+Res.&amp;title=Analyzing+large-scale+proteomics+projects+with+latent+semantic+indexing&amp;author=S+Klie&amp;author=L+Martens&amp;author=JA+Vizcaino&amp;author=R+Cote&amp;author=P+Jones&amp;volume=7&amp;publication_year=2008&amp;pages=182-191&amp;pmid=18047271&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B47\">47. <span class=\"element-citation\">Gonnelli G, Hulstaert N, Degroeve S, Martens L. Towards a human proteomics atlas. <span><span class=\"ref-journal\">Anal Bioanal Chem. </span>2012;<span class=\"ref-vol\">404</span>:1069–1077.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22447219\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Anal+Bioanal+Chem&amp;title=Towards+a+human+proteomics+atlas&amp;author=G+Gonnelli&amp;author=N+Hulstaert&amp;author=S+Degroeve&amp;author=L+Martens&amp;volume=404&amp;publication_year=2012&amp;pages=1069-1077&amp;pmid=22447219&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B48\">48. <span class=\"element-citation\">Griss J, Cote RG, Gerner C, Hermjakob H, Vizcaino JA. Published and perished? The influence of the searched protein database on the long-term storage of proteomics data. <span><span class=\"ref-journal\">Mol. Cell Proteomics. </span>2011;<span class=\"ref-vol\">10</span>:M111.008490.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3186200/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21700957\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Cell+Proteomics&amp;title=Published+and+perished?+The+influence+of+the+searched+protein+database+on+the+long-term+storage+of+proteomics+data&amp;author=J+Griss&amp;author=RG+Cote&amp;author=C+Gerner&amp;author=H+Hermjakob&amp;author=JA+Vizcaino&amp;volume=10&amp;publication_year=2011&amp;pages=M111.008490&amp;pmid=21700957&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B49\">49. <span class=\"element-citation\">Griss J, Martin M, O\'Donovan C, Apweiler R, Hermjakob H, Vizcaino JA. Consequences of the discontinuation of the International Protein Index (IPI) database and its substitution by the UniProtKB &#x0201c;complete proteome&#x0201d; sets. <span><span class=\"ref-journal\">Proteomics. </span>2011;<span class=\"ref-vol\">11</span>:4434–4438.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3556690/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21932440\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Consequences+of+the+discontinuation+of+the+International+Protein+Index+(IPI)+database+and+its+substitution+by+the+UniProtKB+&#x0201c;complete+proteome&#x0201d;+sets&amp;author=J+Griss&amp;author=M+Martin&amp;author=C+O\'Donovan&amp;author=R+Apweiler&amp;author=H+Hermjakob&amp;volume=11&amp;publication_year=2011&amp;pages=4434-4438&amp;pmid=21932440&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B50\">50. <span class=\"element-citation\">Knowles DG, McLysaght A. Recent de novo origin of human protein-coding genes. <span><span class=\"ref-journal\">Genome Res. </span>2009;<span class=\"ref-vol\">19</span>:1752–1759.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2765279/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19726446\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=Recent+de+novo+origin+of+human+protein-coding+genes&amp;author=DG+Knowles&amp;author=A+McLysaght&amp;volume=19&amp;publication_year=2009&amp;pages=1752-1759&amp;pmid=19726446&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B51\">51. <span class=\"element-citation\">Panchin AY, Gelfand MS, Ramensky VE, Artamonova II. Asymmetric and non-uniform evolution of recently duplicated human genes. <span><span class=\"ref-journal\">Biol. Direct. </span>2010;<span class=\"ref-vol\">5</span>:54.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2942815/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20825637\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Biol.+Direct.&amp;title=Asymmetric+and+non-uniform+evolution+of+recently+duplicated+human+genes&amp;author=AY+Panchin&amp;author=MS+Gelfand&amp;author=VE+Ramensky&amp;author=II+Artamonova&amp;volume=5&amp;publication_year=2010&amp;pages=54&amp;pmid=20825637&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B52\">52. <span class=\"element-citation\">Foster JM, Degroeve S, Gatto L, Visser M, Wang R, Griss J, Apweiler R, Martens L. A posteriori quality control for the curation and reuse of public proteomics data. <span><span class=\"ref-journal\">Proteomics. </span>2011;<span class=\"ref-vol\">11</span>:2182–2194.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21538885\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=A+posteriori+quality+control+for+the+curation+and+reuse+of+public+proteomics+data&amp;author=JM+Foster&amp;author=S+Degroeve&amp;author=L+Gatto&amp;author=M+Visser&amp;author=R+Wang&amp;volume=11&amp;publication_year=2011&amp;pages=2182-2194&amp;pmid=21538885&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"gks1262-B53\">53. <span class=\"element-citation\">Paik YK, Omenn GS, Uhlen M, Hanash S, Marko-Varga G, Aebersold R, Bairoch A, Yamamoto T, Legrain P, Lee HJ, et al.  Standard guidelines for the chromosome-centric human proteome project. <span><span class=\"ref-journal\">J. Proteome Res. </span>2012;<span class=\"ref-vol\">11</span>:2005–2013.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22443261\" ref=\"reftype=pubmed&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Proteome+Res.&amp;title=Standard+guidelines+for+the+chromosome-centric+human+proteome+project&amp;author=YK+Paik&amp;author=GS+Omenn&amp;author=M+Uhlen&amp;author=S+Hanash&amp;author=G+Marko-Varga&amp;volume=11&amp;publication_year=2012&amp;pages=2005-2013&amp;pmid=22443261&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=3531176&amp;issue-id=217633&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div></div><br>', NULL, 2);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(79, 'pubmed', '33879305', 'Self-reported DSM-5 Anxiety Severity Measures: Evidence of Validity and Reliability in Spanish youths.', 'Very few studies about the psychometric properties of the Anxiety Severity Measures (ASM) proposed in the DSM-5 exist, and none in Spanish-speaking populations. Thus, the aim of the present study was to provide validity and reliability evidence for the Spanish versions of the Agoraphobia, Social Anxiety, Separation Anxiety, Panic, Generalized Anxiety and Specific Phobia Severity measures.Participants included 567 Spaniards (mean age=21.26, SD=3.61; 68.3% women). We performed Exploratory and Confirmatory Factor Analyses to test the structure of the scales, Differential Item Functioning (DIF) by sex, Cronbach\'s alpha and ordinal omega to test reliability, and the Pearson correlations between the ASM and different outcomes to provide evidence for convergent/discriminant (internalizing/externalizing symptoms) and criterion validity (satisfaction, quality of life and personality).Structural analyses supported a one-factor solution for all of the ASM except for the Specific Phobia scale, which was also the only scale that exhibited DIF. Reliability indices ranked from .82 to .93. All six scales showed stronger associations with internalizing than externalizing measures and were also negatively related to criterion measures.The Spanish version of ASM is suitable for assessing anxiety-related symptoms, except the Specific Phobia Scale, which requires further examination.', 2021, 'Psicothema', '', 2, NULL, NULL, 2);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(80, 'pubmed', '33530561', 'Amplifying and Fine-Tuning Rsm sRNAs Expression and Stability to Optimize the Survival of  in Nutrient-Poor Environments.', 'In the beneficial plant root-associated  strain NFM421, the GacS/GacA two-component system positively controls biofilm formation and the production of secondary metabolites through the synthesis of ,  and . Here, we evidenced the genetic amplification of Rsm sRNAs by the discovery of a novel 110-nt long sRNA encoding gene, , generated by the duplication of  (formerly ). Like the others  genes, its overexpression overrides the  mutation. We explored the expression and the stability of , ,  and  encoding genes under rich or nutrient-poor conditions, and showed that their amount is fine-tuned at the transcriptional and more interestingly at the post-transcriptional level. Unlike  and , we noticed that the expression of  and  genes was exclusively GacA-dependent. The highest expression level and longest half-life for each sRNA were correlated with the highest ppGpp and cyclic-di-GMP levels and were recorded under nutrient-poor conditions. Together, these data support the view that the Rsm system in  is likely linked to the stringent response, and seems to be required for bacterial adaptation to nutritional stress.', 2021, 'Microorganisms', 'PMC7911923', 2, '<h1 class=\"content-title\">Amplifying and Fine-Tuning Rsm sRNAs Expression and Stability to Optimize the Survival of <em>Pseudomonas brassicacerum</em> in Nutrient-Poor Environments</h1><br><span class=\"contrib-group fm-author\"><a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Lalaouna%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33530561\" class=\"affpopup\" co-rid=\"_co_idm139634170951232\" co-class=\"co-affbox\">David Lalaouna</a>,<sup>1,</sup><sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Fochesato%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33530561\" class=\"affpopup\" co-rid=\"_co_idm139634171135296\" co-class=\"co-affbox\">Sylvain Fochesato</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Harir%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33530561\" class=\"affpopup\" co-rid=\"_co_idm139634171133760\" co-class=\"co-affbox\">Mourad Harir</a>,<sup>3,</sup><sup>4</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Ortet%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33530561\" class=\"affpopup\" co-rid=\"_co_idm139634171131040\" co-class=\"co-affbox\">Philippe Ortet</a>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Schmitt-Kopplin%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33530561\" class=\"affpopup\" co-rid=\"_co_idm139634168227088\" co-class=\"co-affbox\">Philippe Schmitt-Kopplin</a>,<sup>3,</sup><sup>4</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Heulin%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33530561\" class=\"affpopup\" co-rid=\"_co_idm139634168224368\" co-class=\"co-affbox\">Thierry Heulin</a>,<sup>1</sup> and  <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Achouak%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33530561\" class=\"affpopup\" co-rid=\"_co_idm139634160116544\" co-class=\"co-affbox\">Wafa Achouak</a><sup>1,</sup><sup>*</sup></span><span style=\"display:none\" class=\"contrib-group aff-tip\"><span id=\"_co_idm139634170951232\"><h3 class=\"no_margin\">David Lalaouna</h3><p><sup>1</sup>Aix Marseille Univ, CEA, CNRS, BIAM, Lab Microbial Ecology of the Rhizosphere (LEMiRE), F-13108 Saint-Paul-Lez-Durance, France; <a href=\"mailto:dev@null\" data-email=\"rf.artsinu.srnc-cmbi@anuoalal.d\" class=\"oemail\">rf.artsinu.srnc-cmbi@anuoalal.d</a> (D.L.); <a href=\"mailto:dev@null\" data-email=\"rf.aec@otasehcof.niavlys\" class=\"oemail\">rf.aec@otasehcof.niavlys</a> (S.F.); <a href=\"mailto:dev@null\" data-email=\"rf.aec@tetro.eppilihP\" class=\"oemail\">rf.aec@tetro.eppilihP</a> (P.O.); <a href=\"mailto:dev@null\" data-email=\"rf.aec@nilueh.yrreiht\" class=\"oemail\">rf.aec@nilueh.yrreiht</a> (T.H.)</p><p><sup>2</sup>Universit&#x000e9; de Strasbourg, CNRS, ARN UPR 9002, F-67000 Strasbourg, France</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Lalaouna%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33530561\">David Lalaouna</a></span></span><span id=\"_co_idm139634171135296\"><h3 class=\"no_margin\">Sylvain Fochesato</h3><p><sup>1</sup>Aix Marseille Univ, CEA, CNRS, BIAM, Lab Microbial Ecology of the Rhizosphere (LEMiRE), F-13108 Saint-Paul-Lez-Durance, France; <a href=\"mailto:dev@null\" data-email=\"rf.artsinu.srnc-cmbi@anuoalal.d\" class=\"oemail\">rf.artsinu.srnc-cmbi@anuoalal.d</a> (D.L.); <a href=\"mailto:dev@null\" data-email=\"rf.aec@otasehcof.niavlys\" class=\"oemail\">rf.aec@otasehcof.niavlys</a> (S.F.); <a href=\"mailto:dev@null\" data-email=\"rf.aec@tetro.eppilihP\" class=\"oemail\">rf.aec@tetro.eppilihP</a> (P.O.); <a href=\"mailto:dev@null\" data-email=\"rf.aec@nilueh.yrreiht\" class=\"oemail\">rf.aec@nilueh.yrreiht</a> (T.H.)</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Fochesato%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33530561\">Sylvain Fochesato</a></span></span><span id=\"_co_idm139634171133760\"><h3 class=\"no_margin\">Mourad Harir</h3><p><sup>3</sup>Research Unit Analytical BioGeoChemistry, Helmholtz Zentrum M&#x000fc;nchen, 85764 Neuherberg, Germany; <a href=\"mailto:dev@null\" data-email=\"ed.nehcneum-ztlohmleh@rirah.daruom\" class=\"oemail\">ed.nehcneum-ztlohmleh@rirah.daruom</a> (M.H.); <a href=\"mailto:dev@null\" data-email=\"ed.nehcneum-ztlohmleh@nilppok-ttimhcs\" class=\"oemail\">ed.nehcneum-ztlohmleh@nilppok-ttimhcs</a> (P.S.-K.)</p><p><sup>4</sup>Technical University of Munich, Maximus-von-Imhof-Forum 2, 85354 Freising, Germany</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Harir%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33530561\">Mourad Harir</a></span></span><span id=\"_co_idm139634171131040\"><h3 class=\"no_margin\">Philippe Ortet</h3><p><sup>1</sup>Aix Marseille Univ, CEA, CNRS, BIAM, Lab Microbial Ecology of the Rhizosphere (LEMiRE), F-13108 Saint-Paul-Lez-Durance, France; <a href=\"mailto:dev@null\" data-email=\"rf.artsinu.srnc-cmbi@anuoalal.d\" class=\"oemail\">rf.artsinu.srnc-cmbi@anuoalal.d</a> (D.L.); <a href=\"mailto:dev@null\" data-email=\"rf.aec@otasehcof.niavlys\" class=\"oemail\">rf.aec@otasehcof.niavlys</a> (S.F.); <a href=\"mailto:dev@null\" data-email=\"rf.aec@tetro.eppilihP\" class=\"oemail\">rf.aec@tetro.eppilihP</a> (P.O.); <a href=\"mailto:dev@null\" data-email=\"rf.aec@nilueh.yrreiht\" class=\"oemail\">rf.aec@nilueh.yrreiht</a> (T.H.)</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Ortet%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33530561\">Philippe Ortet</a></span></span><span id=\"_co_idm139634168227088\"><h3 class=\"no_margin\">Philippe Schmitt-Kopplin</h3><p><sup>3</sup>Research Unit Analytical BioGeoChemistry, Helmholtz Zentrum M&#x000fc;nchen, 85764 Neuherberg, Germany; <a href=\"mailto:dev@null\" data-email=\"ed.nehcneum-ztlohmleh@rirah.daruom\" class=\"oemail\">ed.nehcneum-ztlohmleh@rirah.daruom</a> (M.H.); <a href=\"mailto:dev@null\" data-email=\"ed.nehcneum-ztlohmleh@nilppok-ttimhcs\" class=\"oemail\">ed.nehcneum-ztlohmleh@nilppok-ttimhcs</a> (P.S.-K.)</p><p><sup>4</sup>Technical University of Munich, Maximus-von-Imhof-Forum 2, 85354 Freising, Germany</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Schmitt-Kopplin%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33530561\">Philippe Schmitt-Kopplin</a></span></span><span id=\"_co_idm139634168224368\"><h3 class=\"no_margin\">Thierry Heulin</h3><p><sup>1</sup>Aix Marseille Univ, CEA, CNRS, BIAM, Lab Microbial Ecology of the Rhizosphere (LEMiRE), F-13108 Saint-Paul-Lez-Durance, France; <a href=\"mailto:dev@null\" data-email=\"rf.artsinu.srnc-cmbi@anuoalal.d\" class=\"oemail\">rf.artsinu.srnc-cmbi@anuoalal.d</a> (D.L.); <a href=\"mailto:dev@null\" data-email=\"rf.aec@otasehcof.niavlys\" class=\"oemail\">rf.aec@otasehcof.niavlys</a> (S.F.); <a href=\"mailto:dev@null\" data-email=\"rf.aec@tetro.eppilihP\" class=\"oemail\">rf.aec@tetro.eppilihP</a> (P.O.); <a href=\"mailto:dev@null\" data-email=\"rf.aec@nilueh.yrreiht\" class=\"oemail\">rf.aec@nilueh.yrreiht</a> (T.H.)</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Heulin%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33530561\">Thierry Heulin</a></span></span><span id=\"_co_idm139634160116544\"><h3 class=\"no_margin\">Wafa Achouak</h3><p><sup>1</sup>Aix Marseille Univ, CEA, CNRS, BIAM, Lab Microbial Ecology of the Rhizosphere (LEMiRE), F-13108 Saint-Paul-Lez-Durance, France; <a href=\"mailto:dev@null\" data-email=\"rf.artsinu.srnc-cmbi@anuoalal.d\" class=\"oemail\">rf.artsinu.srnc-cmbi@anuoalal.d</a> (D.L.); <a href=\"mailto:dev@null\" data-email=\"rf.aec@otasehcof.niavlys\" class=\"oemail\">rf.aec@otasehcof.niavlys</a> (S.F.); <a href=\"mailto:dev@null\" data-email=\"rf.aec@tetro.eppilihP\" class=\"oemail\">rf.aec@tetro.eppilihP</a> (P.O.); <a href=\"mailto:dev@null\" data-email=\"rf.aec@nilueh.yrreiht\" class=\"oemail\">rf.aec@nilueh.yrreiht</a> (T.H.)</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Achouak%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33530561\">Wafa Achouak</a></span></span></span></span><span class=\"contrib-group half_rhythm fm-editor\">Jacob G. Malone, <span class=\"fm-role\">Academic Editor</span></span><span class=\"fm-panel half_rhythm\"><span class=\"togglers<br><span id=\"idm139634171392832\" lang=\"en\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139634171392832title\">Abstract</h2><!--article-meta--><span><p id=\"__p1\" class=\"p p-first-last\">In the beneficial plant root-associated <em>Pseudomonas brassicacearum</em> strain NFM421, the GacS/GacA two-component system positively controls biofilm formation and the production of secondary metabolites through the synthesis of <em>rsmX</em>, <em>rsmY</em> and <em>rsmZ</em>. Here, we evidenced the genetic amplification of Rsm sRNAs by the discovery of a novel 110-nt long sRNA encoding gene, <em>rsmX-2</em>, generated by the duplication of <em>rsmX-1</em> (formerly <em>rsmX</em>). Like the others <em>rsm</em> genes, its overexpression overrides the <em>gacA</em> mutation. We explored the expression and the stability of <em>rsmX-1</em>, <em>rsmX-2</em>, <em>rsmY</em> and <em>rsmZ</em> encoding genes under rich or nutrient-poor conditions, and showed that their amount is fine-tuned at the transcriptional and more interestingly at the post-transcriptional level. Unlike <em>rsmY</em> and <em>rsmZ</em>, we noticed that the expression of <em>rsmX-1</em> and <em>rsmX-2</em> genes was exclusively GacA-dependent. The highest expression level and longest half-life for each sRNA were correlated with the highest ppGpp and cyclic-di-GMP levels and were recorded under nutrient-poor conditions. Together, these data support the view that the Rsm system in <em>P. brassicacearum</em> is likely linked to the stringent response, and seems to be required for bacterial adaptation to nutritional stress.</p></span><span class=\"sec\"><strong class=\"kwd-title\">Keywords: </strong><span class=\"kwd-text\">Rsm sRNAs, <em>Pseudomonas brassicacearum</em>, sRNAs stability, nutritional stress, GacA-dependent</span></span></span><span id=\"s\r\n1-microorganisms-09-00250\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n1-microorganisms-09-00250title\">1. Introduction</h2><p id=\"__p2\" class=\"p p-first\">The RNA-binding protein CsrA (carbon storage regulator) and its homolog RsmA (regulator of secondary metabolites) regulate a variety of genes at the post-transcriptional level. Their regulatory activity is modulated by regulatory small RNA (sRNA) antagonists of CsrA/RsmA. Functional homologs of the Csr/Rsm system have been discovered in many Gram-negative genera, including <em>Escherichia</em> [<a href=\"#B1-microorganisms-09-00250\" rid=\"B1-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605042\">1</a>], <em>Salmonella</em> [<a href=\"#B2-microorganisms-09-00250\" rid=\"B2-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605023\">2</a>], <em>Vibrio</em> [<a href=\"#B3-microorganisms-09-00250\" rid=\"B3-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605047\">3</a>], <em>Legionella</em> [<a href=\"#B4-microorganisms-09-00250\" rid=\"B4-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925604994\">4</a>], <em>Yersinia</em> [<a href=\"#B5-microorganisms-09-00250\" rid=\"B5-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925604990\">5</a>] and <em>Pseudomonas</em> [<a href=\"#B6-microorganisms-09-00250\" rid=\"B6-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605034\">6</a>,<a href=\"#B7-microorganisms-09-00250\" rid=\"B7-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605021\">7</a>,<a href=\"#B8-microorganisms-09-00250\" rid=\"B8-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_932605642\">8</a>], as well as in some Gram-positive bacteria such as the <em>Bacillus</em> genus [<a href=\"#B9-microorganisms-09-00250\" rid=\"B9-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605044\">9</a>].</p><p id=\"__p3\">The wide distribution of the Csr/Rsm regulatory system in Eubacteria, including animal/plant pathogenic species as well as plant growth-promoting rhizobacteria (PGPR), illustrates that this well-conserved system is involved in various essential functions [<a href=\"#B10-microorganisms-09-00250\" rid=\"B10-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925604985\">10</a>,<a href=\"#B11-microorganisms-09-00250\" rid=\"B11-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605007\">11</a>]. Indeed, small RNAs positively controlled by either the BarA/UvrY or GacS/GacA two-component systems have been shown to regulate various functions by coordinating the expression of a set of target genes post-transcriptionally [<a href=\"#B12-microorganisms-09-00250\" rid=\"B12-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925604997\">12</a>,<a href=\"#B13-microorganisms-09-00250\" rid=\"B13-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605041\">13</a>]</p><p id=\"__p4\">The number and size of sRNAs involved in this system are variable. In <em>Escherichia coli</em>, the protein repressor CsrA is regulated by two sRNAs, CsrB and CsrC. These sRNAs sequester CsrA by mimicking its recognition sequence [<a href=\"#B14-microorganisms-09-00250\" rid=\"B14-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605005\">14</a>,<a href=\"#B15-microorganisms-09-00250\" rid=\"B15-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605040\">15</a>]. Here, CsrB sRNA, a 360-nucleotide long RNA, works in concert with CsrC sRNA (250 nts). Two homologs were found in <em>Pseudomonas aeruginosa</em> and named RsmY (124 nt) and RsmZ (115 nt) [<a href=\"#B7-microorganisms-09-00250\" rid=\"B7-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605027\">7</a>]. A third Rsm sRNA, RsmX (119 nt), was discovered in <em>Pseudomonas protegens</em> (formerly <em>Pseudomonas fluorescens</em>) CHA0 [<a href=\"#B6-microorganisms-09-00250\" rid=\"B6-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925604993\">6</a>] and predicted to be found in other species including <em>P. fluorescens</em> Pf-5, <em>P. fluorescens</em> Pf0-1, <em>Pseudomonas putida</em> KT2440 and <em>Pseudomonas syringae</em> pv. tomato DC3000. Surprisingly, up to five RsmX encoding genes have been described in <em>P. syringae</em> [<a href=\"#B8-microorganisms-09-00250\" rid=\"B8-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_932605646\">8</a>]. Moreover, experiments with artificial RsmY-like sRNAs have demonstrated that conservation of size and/or primary sequence is not essential for Rsm functions [<a href=\"#B16-microorganisms-09-00250\" rid=\"B16-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605046\">16</a>]. In fact, the regulatory mechanism is linked to appropriate exposure of mimicked CsrA/RsmA recognition motif.</p><p id=\"__p5\">In general, Csr and Rsm sRNAs seem to be redundant [<a href=\"#B7-microorganisms-09-00250\" rid=\"B7-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925604998\">7</a>,<a href=\"#B17-microorganisms-09-00250\" rid=\"B17-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605032\">17</a>,<a href=\"#B18-microorganisms-09-00250\" rid=\"B18-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925604992\">18</a>]. However, although all Rsm sRNAs are positively controlled by GacA, additional regulators have been identified, such as PsrA, which activates <em>rsmZ</em> in <em>P. protegens</em> CHA0 [<a href=\"#B19-microorganisms-09-00250\" rid=\"B19-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605016\">19</a>] and HptB, which exclusively controls <em>rsmY</em> in <em>P. aeruginosa</em> [<a href=\"#B20-microorganisms-09-00250\" rid=\"B20-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605009\">20</a>]. These additional regulators certainly enable a fine-tuning control by integrating various signals.</p><p id=\"__p6\">We have previously, shown that spontaneous mutations in either <em>gacS</em> or <em>gacA</em> in <em>P. brassicacearum</em> can alter many phenotypes typically associated with phytobeneficial traits, as well as root colonization and modification of the root architecture [<a href=\"#B18-microorganisms-09-00250\" rid=\"B18-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605051\">18</a>,<a href=\"#B21-microorganisms-09-00250\" rid=\"B21-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605017\">21</a>,<a href=\"#B22-microorganisms-09-00250\" rid=\"B22-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605054\">22</a>,<a href=\"#B23-microorganisms-09-00250\" rid=\"B23-microorganisms-09-00250\" class=\" bibr popnode\">23</a>]. These traits include secretion of exoenzymes such as protease and lipase [<a href=\"#B24-microorganisms-09-00250\" rid=\"B24-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605006\">24</a>], production of antifungal metabolites (DAPG and cyanide) and <em>N</em>-acyl-homoserine lactones, the phytohormone auxin, biofilm formation, alginate production, chemotaxis, and type VI secretion [<a href=\"#B18-microorganisms-09-00250\" rid=\"B18-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605003\">18</a>]. In addition, overexpression of either <em>rsmX</em>, <em>rsmY</em>, or <em>rsmZ</em> fully suppressed the pleiotropic <em>gacA</em> or <em>gacS</em> mutations and restored the WT phenotypes in <em>P. brassicacearum</em> variants [<a href=\"#B18-microorganisms-09-00250\" rid=\"B18-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605033\">18</a>].</p><p id=\"__p7\" class=\"p p-last\">The primary goal of this study was to determine whether <em>P. brassicacearum</em> genome encodes other Rsm sRNAs and whether their overexpression could override the <em>gacA</em> or <em>gacS</em> mutation. The second part of our work aimed at evaluating the expression and stability of Rsm sRNAs under conditions of different nutrient levels. Finally, to establish the reliability of the results, the analysis of metabolites involved in stringency and biofilm formation was carried out.</p></span><span id=\"s\r\n2-microorganisms-09-00250\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n2-microorganisms-09-00250title\">2. Experimental Procedures</h2><span id=\"s\r\n2dot1-microorganisms-09-00250\" class=\"sec sec-first\"><h3 id=\"s\r\n2dot1-microorganisms-09-00250title\">2.1. Bacterial Strains, Plasmids and Growth Conditions</h3><p id=\"__p8\" class=\"p p-first\">The bacterial strains and plasmids used in this study are listed in <a href=\"#app1-microorganisms-09-00250\" rid=\"app1-microorganisms-09-00250\" class=\" app\">Table S1</a>. <em>P. brassicacearum</em> NFM 421 and <em>gacA</em> mutant were grown as described in Lalaouna et al. [<a href=\"#B18-microorganisms-09-00250\" rid=\"B18-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605031\">18</a>]. <em>Escherichia coli</em> GM2163, TOP10 and S17-1 strains were grown in LB medium at 37 &#x000b0;C.</p><p id=\"__p9\" class=\"p p-last\">In order to test influences of carbon availability, bacterial strains were grown in TSB (BD), TSB/10 and TSB/20 media. For growth on plates, media were solidified with 15 g/L agar (Sigma, St. Louis, USA). Pseudomonas agar F (PAF) (Difco) was used to compare bacterial colony morphology of NFM421 WT, <em>gacS</em> and <em>gacA</em> mutants.</p></span><span id=\"s\r\n2dot2-microorganisms-09-00250\" class=\"sec\"><h3 id=\"s\r\n2dot2-microorganisms-09-00250title\">2.2. DNA Manipulation</h3><p id=\"__p10\" class=\"p p-first-last\">Chromosomal DNA from <em>P. brassicacearum</em> NFM421 was prepared by phenol/chloroform extraction. Plasmid extraction using the &#x0201c;QIAprep Spin Miniprep Kit&#x0201d; (Qiagen, Hilden, Germany), and purification of DNA fragments from agarose gels with the &#x0201c;QIAquick Gel extraction Kit&#x0201d; (Qiagen, Hilden, Germany) were performed according to the manufacturer&#x02019;s instructions.</p></span><span id=\"s\r\n2dot3-microorganisms-09-00250\" class=\"sec\"><h3 id=\"s\r\n2dot3-microorganisms-09-00250title\">2.3. RNA Manipulation</h3><p id=\"__p11\" class=\"p p-first-last\">For RNA extractions, we used &#x0201c;RNAprotect Bacteria Reagent&#x0201d; and &#x0201c;RNeasy Mini Kit&#x0201d; (Qiagen, Hilden, Germany). RT-PCR assays were performed with the &#x0201c;Transcriptor First Strand cDNA Synthesis Kit&#x0201d; (Roche, Manheim, Germany). Gene-specific primers for real-time PCR were designed based on <em>P. brassicacearum</em> NFM421 <em>rsmX-1</em>, <em>rsmX-2</em>, <em>rsmY</em> and <em>rsmZ</em> sequences to obtain predicted PCR products of 200&#x02013;250 bases (<a href=\"#app1-microorganisms-09-00250\" rid=\"app1-microorganisms-09-00250\" class=\" app\">Table S1</a>). Amplifications were performed according to the real Q-PCR Light Cycler 480 SYBR Green I Master kit instructions for the Light Cycler 480 Real Time PCR System (Roche). Real-time PCR was performed in triplicate, and mRNA relative expression was normalized to the 16S reference gene.</p></span><span id=\"s\r\n2dot4-microorganisms-09-00250\" class=\"sec\"><h3 id=\"s\r\n2dot4-microorganisms-09-00250title\">2.4. 5&#x02032; RACE</h3><p id=\"__p12\" class=\"p p-first-last\">The 5&#x02032;-end of the <em>rsmX-2</em> transcript was mapped by RACE. Total RNA from <em>P. brassicacearum</em> strain NFM421 isolated in late stationary phase was used for a reverse transcription reaction with the Transcriptor First Strand cDNA Synthesis Kit (Roche, Manheim, Germany). The 5&#x02032;-phosphorylated, 3&#x02032;-end cordecypin-blocked oligonucleotide DT88 [<a href=\"#B25-microorganisms-09-00250\" rid=\"B25-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605043\">25</a>] was ligated to the single-strand cDNA with T4 RNA ligase (New England Biolabs, Ipswich, USA). The anchor-ligated cDNA was first amplified with primers DT89 (anchor-specific primer) and a <em>rsmX-2</em>-specific primer (<em>rsmX-2</em>-R1; <a href=\"#app1-microorganisms-09-00250\" rid=\"app1-microorganisms-09-00250\" class=\" app\">Table S1</a>). Next, a nested PCR was performed using DT89 and a second <em>rsm</em>-specific internal primer (<em>rsmX-2</em>-R2). Finally, the PCR product was cloned into the cloning vector pCR<sup>&#x000ae;</sup>4Blunt-TOPO<sup>&#x000ae;</sup> (Thermo Fisher Scientific, Waltham, USA). Three independent clones were sequenced using T7 primer (GATC Biotech, Konstanz, Germany).</p></span><span id=\"s\r\n2dot5-microorganisms-09-00250\" class=\"sec\"><h3 id=\"s\r\n2dot5-microorganisms-09-00250title\">2.5. Overexpression of sRNAs</h3><p id=\"__p13\" class=\"p p-first-last\">To overexpress <em>rsmX-2</em>, we used the pME6032 plasmid [<a href=\"#B26-microorganisms-09-00250\" rid=\"B26-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605056\">26</a>]. The Shine-Dalgarno sequence was removed, as it is unsuitable for the expression of small RNAs. The <em>rsmX-2</em> gene was amplified from chromosomal DNA by PCR using primers <em>rsmX-2</em>-PstI and <em>rsmX-2</em>-KpnI (<a href=\"#app1-microorganisms-09-00250\" rid=\"app1-microorganisms-09-00250\" class=\" app\">Table S1</a>), digested by PstI and KpnI, and inserted into PstI/KpnI-cut pME6032. The introduction of the PstI restriction site adds 5 nt and modifies 1 nt of the RsmX-2 sRNA at the 5&#x02032; end (5&#x02032;-<span class=\"underline\">CTGCAG</span>CCACTG&#x02026; in place of 5&#x02032;-TCCACTG). All mutations were verified by sequencing the inserts.</p></span><span id=\"s\r\n2dot6-microorganisms-09-00250\" class=\"sec\"><h3 id=\"s\r\n2dot6-microorganisms-09-00250title\">2.6. Construction of Transcriptional lacZ Fusions</h3><p id=\"__p14\" class=\"p p-first-last\">To construct the <em>rsmX-2</em> + 16 transcriptional fusion, we amplified the promoter region and the first 16 nts of rsmX-2 sequence by PCR with primers listed in <a href=\"#app1-microorganisms-09-00250\" rid=\"app1-microorganisms-09-00250\" class=\" app\">Table S1</a>. The PCR products were subsequently digested with specific restriction enzymes and cloned into pME6016 plasmids [<a href=\"#B27-microorganisms-09-00250\" rid=\"B27-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605026\">27</a>] and then sequenced. The pME6016-<em>rsmX-2</em>-<em>lacZ</em> plasmid was introduced into competent cells of NFM421 strain by electroporation ((2500 V for 5 ms using an Eppendorf Multiporator<sup>&#x000ae;</sup> electroporator, Hambourg, Germany).</p></span><span id=\"s\r\n2dot7-microorganisms-09-00250\" class=\"sec\"><h3 id=\"s\r\n2dot7-microorganisms-09-00250title\">2.7. Northern Blot Analysis</h3><p id=\"__p15\" class=\"p p-first-last\">RNA was extracted from overnight cultures of NFM421 WT strain and from &#x02206;<em>gacA</em> mutant using &#x0201c;RNeasy Mini Kit&#x0201d; (Qiagen, Hambourg, Germany). The electrophoresis of total RNA was performed on a polyacrylamide gel (5% acrylamide 29:1, 8 M urea). RNAs were then transferred on a Hybond N+ nitrocellulose membrane (GE Healthcare Life Sciences, Uppsala, Swenden). RsmX-2 or 5S-specific digoxygenin-labelled probes were used (<a href=\"#app1-microorganisms-09-00250\" rid=\"app1-microorganisms-09-00250\" class=\" app\">Table S1</a>). 5S rRNA was used as loading control. Luminescent detection was carried out as previously described [<a href=\"#B28-microorganisms-09-00250\" rid=\"B28-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605013\">28</a>].</p></span><span id=\"s\r\n2dot8-microorganisms-09-00250\" class=\"sec\"><h3 id=\"s\r\n2dot8-microorganisms-09-00250title\">2.8. Protease Activity Assays</h3><p id=\"__p16\" class=\"p p-first-last\">In order to detect the extracellular protease activity, the bacteria are inoculated on TSB/10 solid medium containing 1% skim milk powder (TSA lec). The presence of a halo around the colony attests to protease activity.</p></span><span id=\"s\r\n2dot9-microorganisms-09-00250\" class=\"sec\"><h3 id=\"s\r\n2dot9-microorganisms-09-00250title\">2.9. &#x003b2;-Galactosidase Assays</h3><p id=\"__p17\" class=\"p p-first-last\">Cultures containing a <em>rsmX-2</em> + 16-<em>lacZ</em> construct were grown overnight, diluted 1:200 into 8 mL of TSB with tetracycline (20 &#x000b5;g/mL), and grown over a period of 24 h. For the time-course of <em>rsmX-2</em> expression, 50 mL cultures were sampled at different times and assayed immediately. &#x003b2;-galactosidase activities were quantified as previously described [<a href=\"#B18-microorganisms-09-00250\" rid=\"B18-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605049\">18</a>]. Experiments were conducted in triplicates.</p></span><span id=\"s\r\n2dot10-microorganisms-09-00250\" class=\"sec\"><h3 id=\"s\r\n2dot10-microorganisms-09-00250title\">2.10. Biofilm Assays</h3><p id=\"__p18\" class=\"p p-first-last\">As previously described [<a href=\"#B18-microorganisms-09-00250\" rid=\"B18-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605015\">18</a>], overnight cultures were diluted to OD<sub>600nm</sub> = 0.05 in K10T-1 medium [<a href=\"#B29-microorganisms-09-00250\" rid=\"B29-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605012\">29</a>] and 1 mL aliquots were dispensed into glass tubes in triplicate. Following static incubation at 30 &#x000b0;C, the medium was removed, and tubes were washed gently with distilled water. Biofilm formation was visualized by crystal violet staining.</p></span><span id=\"s\r\n2dot11-microorganisms-09-00250\" class=\"sec\"><h3 id=\"s\r\n2dot11-microorganisms-09-00250title\">2.11. Stability of Rsm sRNAs</h3><p id=\"__p19\" class=\"p p-first\">Wild-type cells were grown in TSB/10 or TSB media. To arrest RNA transcription, rifampicin was added at a final concentration of 200 &#x000b5;g/mL, either in early stationary phase (OD<sub>600nm</sub> 1.5 in TSB and 0.5 in TSB/10) or in late stationary phase (OD<sub>600nm</sub> 4.5 in TSB and 0.9 in TSB/10). At different time points (1, 2, 5, 10, 15, 20, 30, 45, 60, 90 and 120 min), 0.5 mL from each sample was mixed with 1 mL of RNA protect (Qiagen, Hilden, Germany), and total RNAs were extracted as described above. TURBO DNase treatment following reverse transcription, TaqMan real-time PCR analysis was used to determine the amount of each Rsm.</p><p id=\"__p20\" class=\"p p-last\">TaqMan PCR reactions were performed as described in the &#x0201c;LightCycler<sup>&#x000ae;</sup> 480 Probes Master&#x0201d; (Roche) protocol, with double-labelled oligonucleotide probe (FAM and TAMRA dyes in 5&#x02032; and 3&#x02032; ends, respectively; Eurogentec). Reactions were run in a LightCycler<sup>&#x000ae;</sup> 480 System. Threshold detection parameters were determined using the second derivative method. PCR efficiencies were determined by amplification of four 10-fold serial dilutions of all target sequences. Triplicated biological samples were quantified, and mean values were used to express Rsm sRNAs abundance relative to 16S RNA.</p></span><span id=\"s\r\n2dot12-microorganisms-09-00250\" class=\"sec\"><h3 id=\"s\r\n2dot12-microorganisms-09-00250title\">2.12. High Resolution Mass Spectrometry Analysis</h3><p id=\"__p21\" class=\"p p-first-last\">WT cells were grown in TSB/20, TSB/10 or TSB media. Cells were harvested in late stationary phase and washed with ultrapure water. The lyophilized samples were extracted in methanol water (<em>v</em>:<em>v</em>) in an ultrasonic bath for 15 min. The pellets were centrifuged at 14,000 rpm for 5 min and the supernatants were analyzed in negative mode using 12 Tesla SOLARIX Fourier transform ion cyclotrom mass spectrometer (FT-ICR/MS) from Bruker Daltonics, Bremen, Germany. The injections were performed using a micro-liter pump at a liquid flow rate of 120 &#x003bc;L h<sup>&#x02212;1</sup>. Nitrogen was used for both sheath gas as well as curtain gas. A source heater temperature of 200 &#x000b0;C was maintained to ensure rapid solvent evaporation in the ionized droplets. Data acquisition and handling were performed by using Data Analysis Software from Bruker (Bruker Daltonics, Bremen, Germany).</p></span><span id=\"s\r\n2dot13-microorganisms-09-00250\" class=\"sec sec-last\"><h3 id=\"s\r\n2dot13-microorganisms-09-00250title\">2.13. MassTRIX Metabolite Annotation</h3><p id=\"__p22\" class=\"p p-first-last\">Samples were annotated to possible metabolites of <em>P. brassicacearum</em> using the KEGG metabolome database. The exact mass lists (asc files) were uploaded and compared with a 1.0 ppm accuracy window to the metabolome mass translator into pathways (MassTRIX) [<a href=\"#B30-microorganisms-09-00250\" rid=\"B30-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925604983\">30</a>].</p></span></span><span id=\"s\r\n3-microorganisms-09-00250\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n3-microorganisms-09-00250title\">3. Results</h2><span id=\"s\r\n3dot1-microorganisms-09-00250\" class=\"sec sec-first\"><h3 id=\"s\r\n3dot1-microorganisms-09-00250title\">3.1. Identification of a Fourth Rsm sRNA in P. brassicacearum</h3><p id=\"__p23\" class=\"p p-first\">We identified a fourth Rsm sRNA by performing a BLAST search against the <em>P. brassicacearum</em> NFM421 genome and using the upstream activating sequence (UAS; TATAGCGAAACGCTTACA) recognized by GacA as query sequence. This revealed an additional UAS located upstream the gene encoding a putative sRNA (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f001/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f001\" rid-ob=\"ob-microorganisms-09-00250-f001\" co-legend-rid=\"lgnd_microorganisms-09-00250-f001\"><span>Figure 1</span></a>A), which shares 75% similarity with <em>rsmX</em> gene, while the UAS sequence is almost identical (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f001/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f001\" rid-ob=\"ob-microorganisms-09-00250-f001\" co-legend-rid=\"lgnd_microorganisms-09-00250-f001\"><span>Figure 1</span></a>B). We renamed RsmX to RsmX-1, and the novel Rsm sRNA was named RsmX-2. The secondary structure of RsmX-2 was predicted with RNAfold and visualized using vaRNA software [<a href=\"#B31-microorganisms-09-00250\" rid=\"B31-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605001\">31</a>]. At least three potentially mimicked recognition motifs of RsmA/E (i.e., ANGGA or AGGA motifs) [<a href=\"#B32-microorganisms-09-00250\" rid=\"B32-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605019\">32</a>], are exposed within a hairpin loop, suggesting that RsmX-2 could sequester these proteins (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f001/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f001\" rid-ob=\"ob-microorganisms-09-00250-f001\" co-legend-rid=\"lgnd_microorganisms-09-00250-f001\"><span>Figure 1</span></a>C).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"microorganisms-09-00250-f001\" co-legend-rid=\"lgnd_microorganisms-09-00250-f001\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f001/\" target=\"figure\" rid-figpopup=\"microorganisms-09-00250-f001\" rid-ob=\"ob-microorganisms-09-00250-f001\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139634207113312\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7911923_microorganisms-09-00250-g001.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is microorganisms-09-00250-g001.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC7911923/bin/microorganisms-09-00250-g001.jpg\" /></a></span><span id=\"largeobj_idm139634207113312\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f001/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_microorganisms-09-00250-f001\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f001/\" target=\"figure\" rid-figpopup=\"microorganisms-09-00250-f001\" rid-ob=\"ob-microorganisms-09-00250-f001\">Figure 1</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p64\"><em>rsmX</em>-2 identification and predicted structure. (<strong>A</strong>) Organization of the <em>rsmX-1</em> and <em>rsmX-2</em> genomic region of <em>P. brassicacearum</em>. The upstream activating sequence (UAS) of the GacA protein is represented within the box. GGA motifs involved in the sequestration of RsmA/E proteins are underlined. The transcriptional start has been determined by 5&#x02032;RACE method. <em>rsmX-2</em> gene sequence is indicated in bold; (<strong>B</strong>) Alignment of <em>rsmX-1</em> and <em>rsmX-2</em> UAS sequence; (<strong>C</strong>) <em>rsmX-2</em> secondary structure predicted using RNAfold and visualised using vaRNA.</p></span></span></span><p id=\"__p24\">To determine, the relationship and evolutionary history, we performed a phylogenetic analysis of Rsm sRNAs genes. Phylogenetic tree Unweighted Pair Group Method with Arithmetic Mean (UPGMA) tree was constructed using MAFFT toolkit with a modified version of UPGMA. Rsm sRNAs exhibit a remarkably uniform distribution among <em>Pseudomonas</em> species. The UPGMA phylogenetic tree shows that <em>rsmX</em>, <em>rsmY</em> and <em>rsmZ</em> are sorted into three clades (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f002/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f002\" rid-ob=\"ob-microorganisms-09-00250-f002\" co-legend-rid=\"lgnd_microorganisms-09-00250-f002\"><span>Figure 2</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"microorganisms-09-00250-f002\" co-legend-rid=\"lgnd_microorganisms-09-00250-f002\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f002/\" target=\"figure\" rid-figpopup=\"microorganisms-09-00250-f002\" rid-ob=\"ob-microorganisms-09-00250-f002\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139634207108208\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7911923_microorganisms-09-00250-g002.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is microorganisms-09-00250-g002.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC7911923/bin/microorganisms-09-00250-g002.jpg\" /></a></span><span id=\"largeobj_idm139634207108208\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f002/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_microorganisms-09-00250-f002\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f002/\" target=\"figure\" rid-figpopup=\"microorganisms-09-00250-f002\" rid-ob=\"ob-microorganisms-09-00250-f002\">Figure 2</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p65\">Phylogenetic analysis of <em>rsm</em> RNA sequences. <em>rsm</em> genes from <em>P. brassicacearum</em> NFM 421 are shown in bold.</p></span></span></span><p id=\"__p25\" class=\"p p-last\">Typically, Pseudomonads contain single copies of RsmY and RsmZ, however, the copy number of RsmX is variable. <em>P. protegens</em> CHA0 possesses one copy of RsmX [<a href=\"#B6-microorganisms-09-00250\" rid=\"B6-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605038\">6</a>], while <em>P. syringae</em> contains up to five copies [<a href=\"#B8-microorganisms-09-00250\" rid=\"B8-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_932605644\">8</a>]. In this study we identified a fourth Rsm RNA in <em>P. brassicacearum</em> NFM421 found in tandem with <em>rsmX</em> with which it shares 75% of sequence identity. We also show a potential fourth Rsm sRNA in <em>P. fluorescens</em> F113 and <em>Pseudomonas stutzeri</em>, which are in tandem with <em>rsmX</em> and share 75% and 81% sequence identity respectively (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/table/microorganisms-09-00250-t001/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-t001\" rid-ob=\"ob-microorganisms-09-00250-t001\" co-legend-rid=\"\"><span>Table 1</span></a>).</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"microorganisms-09-00250-t001\"><h3>Table 1</h3><!--caption a7--><span class=\"caption\"><p id=\"__p71\">Tandem copy of rsmX gene in <em>Pseudomonas.</em></p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139634170373056\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><thead><tr><th align=\"center\" valign=\"middle\" style=\"border-top:solid thin;border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">Strains</th><th align=\"center\" valign=\"middle\" style=\"border-top:solid thin;border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">Gene</th><th align=\"center\" valign=\"middle\" style=\"border-top:solid thin;border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">Begin</th><th align=\"center\" valign=\"middle\" style=\"border-top:solid thin;border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">End</th><th align=\"center\" valign=\"middle\" style=\"border-top:solid thin;border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">Size (nt)</th><th align=\"center\" valign=\"middle\" style=\"border-top:solid thin;border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">Sequence Identity (%)</th><th align=\"center\" valign=\"middle\" style=\"border-top:solid thin;border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">Number of <em>rsmX-Like</em> Genes</th></tr></thead><tbody><tr><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" colspan=\"1\"><em>Pseudomonas brassicacearum</em> NFM421</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">\n<em>rsmX-1</em>\n</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">4839892</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">4840003</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">111</td><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" colspan=\"1\">75</td><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" colspan=\"1\">2</td></tr><tr><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">\n<em>rsmX-2</em>\n</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">4839681</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">4839791</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">110</td></tr><tr><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" colspan=\"1\"><em>Pseudomonas fluorescens</em> F113</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">\n<em>rsmX-1 *</em>\n</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2113839</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2113950</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">111</td><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" colspan=\"1\">75</td><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" colspan=\"1\">2</td></tr><tr><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">\n<em>rsmX-2 *</em>\n</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">2114052</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">2114160</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">108</td></tr><tr><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" colspan=\"1\"><em>Pseudomonas stutzeri</em> A150</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">\n<em>rsmX-1 *</em>\n</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">356904</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">357013</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">109</td><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" colspan=\"1\">81.5</td><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" colspan=\"1\">2</td></tr><tr><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">\n<em>rsmX-2 *</em>\n</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">357143</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">357250</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">107</td></tr><tr><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" colspan=\"1\"><em>Pseudomonas syringae</em> pv. <em>phaseolicola</em> 1448A</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">\n<em>rsmX-4 **</em>\n</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">160448</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">160336</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">112</td><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" colspan=\"1\">85</td><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" colspan=\"1\">5</td></tr><tr><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">\n<em>rsmX-3 **</em>\n</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">160740</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">160627</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">113</td></tr><tr><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" colspan=\"1\"><em>Pseudomonas syringae</em> pv. <em>syringae</em> B728a</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">\n<em>rsmX-3 **</em>\n</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">5867169</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">5867282</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">113</td><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" colspan=\"1\">84</td><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" colspan=\"1\">5</td></tr><tr><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">\n<em>rsmX-4 **</em>\n</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">5867461</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">5867572</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">111</td></tr><tr><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" colspan=\"1\"><em>Pseudomonas syringae</em> pv. <em>tomato</em> str. DC3000</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">\n<em>rsmX-3</em>\n</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">6144830</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">6144943</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">113</td><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" colspan=\"1\">87</td><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" colspan=\"1\">5</td></tr><tr><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">\n<em>rsmX-4</em>\n</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">6145122</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">6145235</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">113</td></tr></tbody></table></span><span id=\"largeobj_idm139634170373056\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/table/microorganisms-09-00250-t001/?report=objectonly\">Open in a separate window</a></span><span class=\"tblwrap-foot\"><span id=\"idm139634162922736\"><p id=\"__p72\" class=\"p p-first-last\">* in comparison to <em>P. brassicacearum</em> NFM421. ** in comparison to <em>P. syringae pv.</em> tomato str. DC3000.</p></span></span></span></span><span id=\"s\r\n3dot2-microorganisms-09-00250\" class=\"sec\"><h3 id=\"s\r\n3dot2-microorganisms-09-00250title\">3.2. rsmX-2 Expression Is Exclusively GacA-Dependent</h3><p id=\"__p26\" class=\"p p-first\">The Northern blot analysis (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f003/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f003\" rid-ob=\"ob-microorganisms-09-00250-f003\" co-legend-rid=\"lgnd_microorganisms-09-00250-f003\"><span>Figure 3</span></a>A) and 5&#x02032; end determination of the transcript by 5&#x02032;RACE method confirmed that the novel 110-nt long Rsm sRNA is transcribed only in WT and not in the &#x02206;<em>gacA</em> mutant. This was confirmed by the analysis of <em>rsmX-2</em> expression in wild-type and &#x02206;<em>gacA</em> mutant cells during growth in 10-fold diluted TSB medium, by measuring &#x003b2;-galactosidase activity of the transcriptional <em>rsmX-2</em> + 16-<em>lacZ</em> fusion (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f003/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f003\" rid-ob=\"ob-microorganisms-09-00250-f003\" co-legend-rid=\"lgnd_microorganisms-09-00250-f003\"><span>Figure 3</span></a>B).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"microorganisms-09-00250-f003\" co-legend-rid=\"lgnd_microorganisms-09-00250-f003\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f003/\" target=\"figure\" rid-figpopup=\"microorganisms-09-00250-f003\" rid-ob=\"ob-microorganisms-09-00250-f003\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139634165434800\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7911923_microorganisms-09-00250-g003.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is microorganisms-09-00250-g003.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC7911923/bin/microorganisms-09-00250-g003.jpg\" /></a></span><span id=\"largeobj_idm139634165434800\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f003/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_microorganisms-09-00250-f003\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f003/\" target=\"figure\" rid-figpopup=\"microorganisms-09-00250-f003\" rid-ob=\"ob-microorganisms-09-00250-f003\">Figure 3</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p66\"><em>rsmX-2</em> expression is GacA dependent. (<strong>A</strong>) Northern blot analysis of <em>rsmX-2</em> expression in WT and <em>gacA</em> backgrounds, 5S rRNA is used as loading control; (<strong>B</strong>) &#x003b2;-galactosidase measurements of a transcriptional <em>rsmX-2</em> + 16-<em>lacZ</em> fusion in the WT and &#x02206;<em>gacA</em> strains.</p></span></span></span><p id=\"__p27\" class=\"p p-last\">Moreover, the overexpression of <em>rsmX-2</em> from a vector under the control of P<em>tac</em> promoter restored the wild-type phenotype in <em>gacA</em> mutant cells (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f004/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f004\" rid-ob=\"ob-microorganisms-09-00250-f004\" co-legend-rid=\"lgnd_microorganisms-09-00250-f004\"><span>Figure 4</span></a>), indicating that the overall function of RsmX-2 is likely similar to that of RsmX-1, RsmY and RsmZ [<a href=\"#B18-microorganisms-09-00250\" rid=\"B18-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605004\">18</a>]. This is illustrated by the restoration of protease activity (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f004/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f004\" rid-ob=\"ob-microorganisms-09-00250-f004\" co-legend-rid=\"lgnd_microorganisms-09-00250-f004\"><span>Figure 4</span></a>A), colony morphology (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f004/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f004\" rid-ob=\"ob-microorganisms-09-00250-f004\" co-legend-rid=\"lgnd_microorganisms-09-00250-f004\"><span>Figure 4</span></a>B) and biofilm formation in a <em>gacA</em> mutant (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f004/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f004\" rid-ob=\"ob-microorganisms-09-00250-f004\" co-legend-rid=\"lgnd_microorganisms-09-00250-f004\"><span>Figure 4</span></a>C). These findings indicate that RsmX-2 is part of the Gac-Rsm regulatory system.</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"microorganisms-09-00250-f004\" co-legend-rid=\"lgnd_microorganisms-09-00250-f004\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f004/\" target=\"figure\" rid-figpopup=\"microorganisms-09-00250-f004\" rid-ob=\"ob-microorganisms-09-00250-f004\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139634164733376\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7911923_microorganisms-09-00250-g004.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is microorganisms-09-00250-g004.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC7911923/bin/microorganisms-09-00250-g004.jpg\" /></a></span><span id=\"largeobj_idm139634164733376\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f004/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_microorganisms-09-00250-f004\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f004/\" target=\"figure\" rid-figpopup=\"microorganisms-09-00250-f004\" rid-ob=\"ob-microorganisms-09-00250-f004\">Figure 4</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p67\"><em>rsmX</em>-2 restores GacA-dependent phenotypes. Restoration of wild-type phenotype by overexpression of <em>rsmX-2</em> from a plasmid under the control of P<em>tac</em> promoter in <em>gacA</em> cells as determined by (<strong>A</strong>) Protease activity on skimmed milk-containing medium, (<strong>B</strong>) Colony morphology on PAF medium and (<strong>C</strong>) Biofilm formation.</p></span></span></span></span><span id=\"s\r\n3dot3-microorganisms-09-00250\" class=\"sec\"><h3 id=\"s\r\n3dot3-microorganisms-09-00250title\">3.3. Stringent Conditions Activate rsm sRNAs Genes Expression</h3><p id=\"__p28\" class=\"p p-first-last\">The expression of the four <em>rsm</em> genes <em>rsmX-1</em>, <em>rsmX-2</em>, <em>rsmY</em> and <em>rsmZ</em> increases over time until a maximum is reached in stationary phase (24 h of growth), when nutrients become limited. We therefore wondered whether carbon deprivation could activate the <em>rsm</em> expression, as previously reported for the Csr system [<a href=\"#B33-microorganisms-09-00250\" rid=\"B33-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605028\">33</a>]. To assess the biological relevance of these observations, we monitored the expression level of the four <em>rsm</em> genes under different nutrient level conditions. Here, bacteria were grown in either undiluted TSB medium (considered as a rich medium), 10-fold diluted TSB (TSB/10), or 20-fold diluted TSB (TSB/20) (which was considered as a nutrient-poor medium in which <em>P. brassicacearum</em> growth may occur). Using transcriptional fusions for the promoter from each <em>rsm</em> gene, we found a greater increase in <em>rsm</em> expression when nutrient availability declines (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f005/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f005\" rid-ob=\"ob-microorganisms-09-00250-f005\" co-legend-rid=\"lgnd_microorganisms-09-00250-f005\"><span>Figure 5</span></a>). Thus, even if growth decreases when nutrients are scarce, as indicated by OD<sub>600 nm</sub> monitoring in tables beneath each graph of <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f005/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f005\" rid-ob=\"ob-microorganisms-09-00250-f005\" co-legend-rid=\"lgnd_microorganisms-09-00250-f005\"><span>Figure 5</span></a>, the expression of the four <em>rsm</em> genes highly increases. The greatest effect was observed for <em>rsmZ</em> (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f005/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f005\" rid-ob=\"ob-microorganisms-09-00250-f005\" co-legend-rid=\"lgnd_microorganisms-09-00250-f005\"><span>Figure 5</span></a>). Nutrient starvation conditions influenced the expression levels of <em>rsm</em> genes and also certain targets of RsmA/E synthesis, as indicated by an enhancement of protease activity (<a href=\"#app1-microorganisms-09-00250\" rid=\"app1-microorganisms-09-00250\" class=\" app\">Figure S1</a>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"microorganisms-09-00250-f005\" co-legend-rid=\"lgnd_microorganisms-09-00250-f005\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f005/\" target=\"figure\" rid-figpopup=\"microorganisms-09-00250-f005\" rid-ob=\"ob-microorganisms-09-00250-f005\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139634171432176\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7911923_microorganisms-09-00250-g005.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is microorganisms-09-00250-g005.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC7911923/bin/microorganisms-09-00250-g005.jpg\" /></a></span><span id=\"largeobj_idm139634171432176\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f005/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_microorganisms-09-00250-f005\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f005/\" target=\"figure\" rid-figpopup=\"microorganisms-09-00250-f005\" rid-ob=\"ob-microorganisms-09-00250-f005\">Figure 5</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p68\">The expression level of <em>rsmX-1</em>, <em>rsmX-2</em>, <em>rsmY</em> and <em>rsmZ</em> depends on nutrient availability. Activity of a <em>lacZ</em> reporter fused to the promoter regions of <em>rsmX-1</em>, <em>rsmX-2</em>, <em>rsmY</em> and <em>rsmZ</em> were measured in the <em>P. brassicacearum</em> wild-type strain and are reported as Miller units of &#x003b2;-galactosidase activity (&#x000b1; standard deviation). The cultures were grown for 24 h at 30 &#x000b0;C before being tested and the tables below each graph indicate the OD<sub>600</sub> of the cultures. Experiments were conducted in triplicates.</p></span></span></span></span><span id=\"s\r\n3dot4-microorganisms-09-00250\" class=\"sec\"><h3 id=\"s\r\n3dot4-microorganisms-09-00250title\">3.4. Stability of Rsm sRNA</h3><p id=\"__p29\" class=\"p p-first\">If most sRNAs indeed regulate translational efficiency, their turnover in varying environments deserves a closer examination. For this, we examined the stability and steady state level of Rsm sRNAs by quantitative real-time reverse transcription polymerase chain reactions (qRT-PCR). We used two different media: TSB/10 and undiluted TSB medium. To determine the in vivo stability of Rsm sRNAs, exponentially phase and late stationary phase growing bacteria were treated with rifampicin to block any further initiation of transcription. Samples were taken at 1, 2, 5, 10, 15, 20, 30, 45, 60, 90 and 120 min and DNase-treated total RNA was used for qRT-PCR (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/table/microorganisms-09-00250-t002/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-t002\" rid-ob=\"ob-microorganisms-09-00250-t002\" co-legend-rid=\"\"><span>Table 2</span></a>, <a href=\"#app1-microorganisms-09-00250\" rid=\"app1-microorganisms-09-00250\" class=\" app\">Figure S2</a>). The half-life of each Rsm sRNA was calculated, revealing that these sRNAs are not equally stable. Overall, the half-lives of RsmY and RsmZ are much longer than those of RsmX-1 and RsmX-2 (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/table/microorganisms-09-00250-t002/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-t002\" rid-ob=\"ob-microorganisms-09-00250-t002\" co-legend-rid=\"\"><span>Table 2</span></a>, <a href=\"#app1-microorganisms-09-00250\" rid=\"app1-microorganisms-09-00250\" class=\" app\">Figure S2</a>).</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"microorganisms-09-00250-t002\"><h3>Table 2</h3><!--caption a7--><span class=\"caption\"><p id=\"__p73\">Stability of Rsm sRNAs. Half-lives of <em>P. brassicacearum</em> NFM421 RsmX-1, RsmX-2, RsmY and RsmZ. Experiments were conducted in triplicates.</p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139634168472336\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><thead><tr><th rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-top:solid thin;border-bottom:solid thin\" colspan=\"1\">Medium</th><th rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-top:solid thin;border-bottom:solid thin\" colspan=\"1\">Phase</th><th align=\"center\" valign=\"middle\" style=\"border-top:solid thin;border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">\n</th><th colspan=\"4\" align=\"center\" valign=\"middle\" style=\"border-top:solid thin;border-bottom:solid thin\" rowspan=\"1\">Half-Lives in WT (in Min)</th></tr><tr><th align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">OD<sub>600nm</sub></th><th align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">RsmX-1</th><th align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">RsmX-2</th><th align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">RsmY</th><th align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">RsmZ</th></tr></thead><tbody><tr><td rowspan=\"2\" align=\"center\" valign=\"middle\" colspan=\"1\">TSB</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Expo</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">1.5</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">12 &#x000b1; 1.1</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">5 &#x000b1; 0.6</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">8 &#x000b1; 0.2</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">37 &#x000b1; 2.6</td></tr><tr><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">Stat</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">4.5</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">23 &#x000b1; 5.4</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">16 &#x000b1; 1.5</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">45 &#x000b1; 5.2</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">53 &#x000b1; 0.9</td></tr><tr><td rowspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-top:solid thin;border-bottom:solid thin\" colspan=\"1\">TSB/10</td><td align=\"center\" valign=\"middle\" style=\"border-top:solid thin\" rowspan=\"1\" colspan=\"1\">Expo</td><td align=\"center\" valign=\"middle\" style=\"border-top:solid thin\" rowspan=\"1\" colspan=\"1\">0.5</td><td align=\"center\" valign=\"middle\" style=\"border-top:solid thin\" rowspan=\"1\" colspan=\"1\">7 &#x000b1; 1.5</td><td align=\"center\" valign=\"middle\" style=\"border-top:solid thin\" rowspan=\"1\" colspan=\"1\">5 &#x000b1; 1.6</td><td align=\"center\" valign=\"middle\" style=\"border-top:solid thin\" rowspan=\"1\" colspan=\"1\">8 &#x000b1; 1.7</td><td align=\"center\" valign=\"middle\" style=\"border-top:solid thin\" rowspan=\"1\" colspan=\"1\">26 &#x000b1; 3.6</td></tr><tr><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">Stat</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">0.9</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">37 &#x000b1; 4.4</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">40 &#x000b1; 8.6</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">&#x02265;60</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">&#x02265;90</td></tr></tbody></table></span><span id=\"largeobj_idm139634168472336\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/table/microorganisms-09-00250-t002/?report=objectonly\">Open in a separate window</a></span></span><p id=\"__p30\">The half-life of the four Rsm sRNAs increased during growth to reach their maxima during the stationary phase. For example, RsmX-1 and RsmX-2 stability increased ~8-fold in TSB/10 in the stationary phase compared to the exponential phase. RsmY and RsmZ were observed to be highly stable with a half-life of at least 60 min (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/table/microorganisms-09-00250-t002/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-t002\" rid-ob=\"ob-microorganisms-09-00250-t002\" co-legend-rid=\"\"><span>Table 2</span></a>, <a href=\"#app1-microorganisms-09-00250\" rid=\"app1-microorganisms-09-00250\" class=\" app\">Figure S2</a>).</p><p id=\"__p31\">Moreover, the choice of nutrients condition alters their stability; the four Rsm sRNAs were observed to be more stable in TSB/10 medium at stationary phase (e.g., half-life of RsmY and RsmZ is &#x0003e;60 min and ~50 min in TSB/10 and TSB, respectively). A similar effect was observed for RsmX-1 and RsmX-2, whereas, they both exhibited half-lives of 30&#x02013;34 min in TSB/10 medium, which were reduced in TSB medium to 23 and 16 min, respectively.</p><p id=\"__p32\" class=\"p p-last\">In <em>P. brassicacearum</em>, we hypothesize a correlation between the number of GGA motifs and the stability of the four Rsm sRNAs transcripts conferred by RsmA/E. RsmZ, which showed the highest stability, possesses the highest number of GGA motifs (10 with 4&#x02013;8 exposed); in contrast, RsmX-1 and RsmX-2 have the lowest number of GGA motifs (6 with 3&#x02013;4 exposed) and were less stable. It should be noted that RsmY is moderately stable, although it only contains 7 GGA motifs (4&#x02013;5 exposed) (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/table/microorganisms-09-00250-t003/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-t003\" rid-ob=\"ob-microorganisms-09-00250-t003\" co-legend-rid=\"\"><span>Table 3</span></a>).</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"microorganisms-09-00250-t003\"><h3>Table 3</h3><!--caption a7--><span class=\"caption\"><p id=\"__p74\">Rsm sRNAs stability is correlated to the number of GGA motifs. Relationships between the number of GGA motifs and Rsm sRNAs half-lives of <em>P. brassicacearum</em> NFM421.</p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139634169089904\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><thead><tr><th align=\"center\" valign=\"middle\" style=\"border-top:solid thin;border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">\n</th><th align=\"center\" valign=\"middle\" style=\"border-top:solid thin;border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">\n</th><th align=\"center\" valign=\"middle\" style=\"border-top:solid thin;border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">\n</th><th colspan=\"2\" align=\"center\" valign=\"middle\" style=\"border-top:solid thin;border-bottom:solid thin\" rowspan=\"1\">Half-Lives (in Min)</th></tr><tr><th align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">Genes</th><th align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">GGA Motifs</th><th align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">GGA Motifs Exposed *</th><th align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">TSB/10 24 h</th><th align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">TSB 24 h</th></tr></thead><tbody><tr><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">RsmX-1</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">6</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">2&#x02013;4</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">37 &#x000b1; 4.4</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">23 &#x000b1; 5.4</td></tr><tr><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">RsmX-2</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">6</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">3</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">40 &#x000b1; 8.6</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">16 &#x000b1; 1.5</td></tr><tr><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">RsmY</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">5&#x02013;4</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">&#x02265;60</td><td align=\"center\" valign=\"middle\" rowspan=\"1\" colspan=\"1\">45 &#x000b1; 5.2</td></tr><tr><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">RsmZ</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">8&#x02013;7</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">&#x02265;90</td><td align=\"center\" valign=\"middle\" style=\"border-bottom:solid thin\" rowspan=\"1\" colspan=\"1\">53 &#x000b1; 0.9</td></tr></tbody></table></span><span id=\"largeobj_idm139634169089904\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/table/microorganisms-09-00250-t003/?report=objectonly\">Open in a separate window</a></span><span class=\"tblwrap-foot\"><span id=\"idm139634159990864\"><p id=\"__p75\" class=\"p p-first-last\">* predicted with Mfold and RNAfold softwares.</p></span></span></span></span><span id=\"s\r\n3dot5-microorganisms-09-00250\" class=\"sec sec-last\"><h3 id=\"s\r\n3dot5-microorganisms-09-00250title\">3.5. Stringent Conditions and Sedentary Lifestyle Signalling Molecules</h3><p id=\"__p33\" class=\"p p-first\">Given that the Rsm sRNAs expression and stability were increased under starvation conditions and that the bacterial stringent response, is mediated by the ppGpp, we evaluated the production of ppGpp under these culture conditions. Spectra were acquired with a time domain of 4 MW and a total number of 750 scans were accumulated (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f006/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f006\" rid-ob=\"ob-microorganisms-09-00250-f006\" co-legend-rid=\"lgnd_microorganisms-09-00250-f006\"><span>Figure 6</span></a>A). As expected, the level of ppGpp is inversely proportional to the availability of nutrients (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f006/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f006\" rid-ob=\"ob-microorganisms-09-00250-f006\" co-legend-rid=\"lgnd_microorganisms-09-00250-f006\"><span>Figure 6</span></a>B), while it evolves in the same way as the expression of Rsm sRNAs and in particular <em>rsmZ</em> [<a href=\"#B34-microorganisms-09-00250\" rid=\"B34-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605057\">34</a>].</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"microorganisms-09-00250-f006\" co-legend-rid=\"lgnd_microorganisms-09-00250-f006\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f006/\" target=\"figure\" rid-figpopup=\"microorganisms-09-00250-f006\" rid-ob=\"ob-microorganisms-09-00250-f006\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139634168188048\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7911923_microorganisms-09-00250-g006.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is microorganisms-09-00250-g006.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC7911923/bin/microorganisms-09-00250-g006.jpg\" /></a></span><span id=\"largeobj_idm139634168188048\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f006/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_microorganisms-09-00250-f006\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f006/\" target=\"figure\" rid-figpopup=\"microorganisms-09-00250-f006\" rid-ob=\"ob-microorganisms-09-00250-f006\">Figure 6</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p69\">Signal molecules analysis. (<strong>A</strong>) Full ICR-FT/mass spectra of the wild type (WT) grown in TSB, tenfold dilutes TSB (TSB/10) and twenty-fold diluted TSB (TSB/20). (<strong>B</strong>,<strong>C</strong>) detection of (p)ppGpp and c-di-GMP compounds as annotated using metabolome database (MassTRIX: Suhre and Schmitt-Kopplin, 2008).</p></span></span></span><p id=\"__p34\" class=\"p p-last\">Moreover, since Rsm sRNAs promote sedentary state and multicellularity through biofilm formation, a process also coordinated by the bacterial second messenger, cyclic diguanylate monophosphate (c-di-GMP), we also measured its cellular level. The same trend as for ppGpp and Rsm sRNAs was observed, suggesting a link between Gac-Rsm system and these two signal molecules (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f006/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f006\" rid-ob=\"ob-microorganisms-09-00250-f006\" co-legend-rid=\"lgnd_microorganisms-09-00250-f006\"><span>Figure 6</span></a>C) as previously shown for other <em>Pseudomonas</em> species [<a href=\"#B35-microorganisms-09-00250\" rid=\"B35-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925604989\">35</a>,<a href=\"#B36-microorganisms-09-00250\" rid=\"B36-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_932605639\">36</a>,<a href=\"#B37-microorganisms-09-00250\" rid=\"B37-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605053\">37</a>].</p></span></span><span id=\"s\r\n4-microorganisms-09-00250\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n4-microorganisms-09-00250title\">4. Discussion</h2><span id=\"s\r\n4dot1-microorganisms-09-00250\" class=\"sec sec-first\"><h3 id=\"s\r\n4dot1-microorganisms-09-00250title\">4.1. P. brassicacearum Rsm sRNA Amplification by a Duplication of rsmX</h3><p id=\"__p35\" class=\"p p-first\">The evolutionary relationships between the <em>rsmX</em> genes indicate the duplication events that occurred in different <em>Pseudomonas</em> species (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f002/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f002\" rid-ob=\"ob-microorganisms-09-00250-f002\" co-legend-rid=\"lgnd_microorganisms-09-00250-f002\"><span>Figure 2</span></a>) with up to 5 copies in <em>P. syringae</em> was reported by Moll et al. [<a href=\"#B8-microorganisms-09-00250\" rid=\"B8-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_932605640\">8</a>], who correlated the number of Csr/Rsm sRNAs with the number of Csr/Rsm proteins. In <em>E. coli</em> and in most <em>Pseudomonas</em> species, there are two sRNAs for one or two regulatory proteins. However, three to five RsmA homologs are predicted in <em>P. syringae</em>, where up to seven Rsm sRNAs are found [<a href=\"#B8-microorganisms-09-00250\" rid=\"B8-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_932605643\">8</a>]. In <em>P. brassicacearum</em>, three RsmA homologs are found: RsmA, RsmE and a third putative RsmA-like protein. Recently, Sobrero and Valverde [<a href=\"#B11-microorganisms-09-00250\" rid=\"B11-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605055\">11</a>] performed a comparative genomics and evolutionary analysis of Rsm RNAs-binding proteins of the CsrA family in the genus <em>Pseudomonas</em> and suggested that the presence of redundant Rsm proteins that can replace or by-pass each other&#x02019;s activities could help bacteria achieve greater plasticity via post-transcriptional regulation and better noise control in gene expression.</p><p id=\"__p36\">The amplification of regulatory RNAs is not exclusive to Rsm sRNAs. In <em>Vibrio cholerae</em>, four Qrr (quorum regulatory RNA), which are almost 80% identical in sequence and predicted to have similar secondary structures [<a href=\"#B3-microorganisms-09-00250\" rid=\"B3-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605008\">3</a>], are involved in virulence and biofilm formation [<a href=\"#B38-microorganisms-09-00250\" rid=\"B38-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605010\">38</a>]. Furthermore, five Qrr sRNAs are involved in the <em>Vibrio harveyi</em> quorum-sensing cascade [<a href=\"#B39-microorganisms-09-00250\" rid=\"B39-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605022\">39</a>]. In different bacterial species, two quite identical sRNAs that control iron homeostasis, as PrrF1 and PrrF2 in <em>Pseudomonas</em> [<a href=\"#B40-microorganisms-09-00250\" rid=\"B40-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925604986\">40</a>] and, RyhB1 and RyB2 in <em>Salmonella</em> [<a href=\"#B41-microorganisms-09-00250\" rid=\"B41-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605029\">41</a>].</p><p id=\"__p37\">Gene duplication is an important feature in evolution because it provides raw material for adaptation to environmental challenges [<a href=\"#B42-microorganisms-09-00250\" rid=\"B42-microorganisms-09-00250\" class=\" bibr popnode\">42</a>]. This phenomenon is also implicated in enabling gene amplification, and allowing cells to proliferate under growth-limiting conditions [<a href=\"#B43-microorganisms-09-00250\" rid=\"B43-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925604988\">43</a>]. Any environmental condition favouring cells with more copies of a gene would permit them to outgrow the rest of the population in a short time; this includes duplications of rRNA genes that might confer a selective advantage under fast growth rate conditions [<a href=\"#B44-microorganisms-09-00250\" rid=\"B44-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605020\">44</a>]. Other cases of gene amplification are found in response to antibiotics [<a href=\"#B45-microorganisms-09-00250\" rid=\"B45-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925604984\">45</a>].</p><p id=\"__p38\" class=\"p p-last\">Gene duplication-amplification is a frequent process in bacterial genomes, that often disappears after only a few generations of growth in the absence of selection pressure. However, gene copy (ies) may be preserved if they confer an adaptive evolution by increasing gene dosage in response to certain environmental constraints. We previously showed the key role of Rsm sRNAs in activating beneficial traits in plant-associated bacteria as well as virulence genes in phytopathogens [<a href=\"#B10-microorganisms-09-00250\" rid=\"B10-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925604996\">10</a>,<a href=\"#B18-microorganisms-09-00250\" rid=\"B18-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605052\">18</a>].</p></span><span id=\"s\r\n4dot2-microorganisms-09-00250\" class=\"sec\"><h3 id=\"s\r\n4dot2-microorganisms-09-00250title\">4.2. RsmX-2 Is Part of Gac-Rsm System under Exclusive Control of GacA</h3><p id=\"__p39\" class=\"p p-first\">Despite the similarity between the <em>rsmX-1</em> and <em>rsmX-2</em> sequences, the expression of <em>rsmX-2</em> differs from that of <em>rsmX-1</em>. The few mutations in the UAS (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f001/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f001\" rid-ob=\"ob-microorganisms-09-00250-f001\" co-legend-rid=\"lgnd_microorganisms-09-00250-f001\"><span>Figure 1</span></a>B) of <em>rsmX-2</em> may explain the 8-fold decrease in the expression level. Our data confirm that RsmX-2 sRNA is a member of the Rsm sRNAs family. Indeed, the exclusive overexpression of <em>rsmX-2</em> in the <em>gacA</em> mutant suppresses the effect of <em>gacA</em> deletion, suggesting that the four Rsm sRNAs share a redundant function, which appears to be the sequestration of RsmA/E proteins.</p><p id=\"__p40\">In the absence of GacA in <em>P. aeruginosa</em> and <em>P. brassicacearum</em>, transcription of <em>rsmY</em> and <em>rsmZ</em> is still achieved but to a lesser degree, suggesting the involvement of additional regulatory pathways [<a href=\"#B7-microorganisms-09-00250\" rid=\"B7-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605045\">7</a>,<a href=\"#B18-microorganisms-09-00250\" rid=\"B18-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925604991\">18</a>]. We previously demonstrated the activation of <em>rsmZ</em> expression, which depends on an enhancer sequence located in the <em>rpoS</em> coding region. This suggested the role of additional transcriptional factors as part of the complex network controlling the expression of <em>rsmZ</em>. We showed that the conserved palindromic UAS required for GacA-controlled sRNA genes in Gammaproteobacteria is essential but not sufficient for the full expression of the <em>rsmZ</em> gene in <em>P. brassicacearum</em> NFM 421 [<a href=\"#B46-microorganisms-09-00250\" rid=\"B46-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925604995\">46</a>].</p><p id=\"__p41\" class=\"p p-last\">Finally, unlike the two novels recently discovered sRNAs, RsmV [<a href=\"#B47-microorganisms-09-00250\" rid=\"B47-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925604987\">47</a>] and RsmW [<a href=\"#B48-microorganisms-09-00250\" rid=\"B48-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605002\">48</a>], in <em>P. aeruginosa</em>, which transcription is independent of the GacS-GacA, <em>rsmX-1</em> and <em>rsmX-2</em> expression is exclusively GacS-GacA-dependent.</p></span><span id=\"s\r\n4dot3-microorganisms-09-00250\" class=\"sec\"><h3 id=\"s\r\n4dot3-microorganisms-09-00250title\">4.3. Regulation of the Amount of RNA by Degradation</h3><p id=\"__p42\" class=\"p p-first\">sRNAs often modulate mRNA targets stability, notably by recruitment of RNases [<a href=\"#B49-microorganisms-09-00250\" rid=\"B49-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_932605645\">49</a>]. This indicates that degradation is a regulatory process, by which bacteria adjust the translation of certain transcripts in response to changing environments. However, turnover of Csr/Rsm sRNAs has not been well-reported in the literature, with the exception of a study reported by Suzuki et al. [<a href=\"#B50-microorganisms-09-00250\" rid=\"B50-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605037\">50</a>]. These authors identified a regulatory protein (CsrD) that targets the global regulatory RNAs CsrB and CsrC for degradation by RNase E. Investigation of Rsm sRNAs degradation in <em>P. brassicacearum</em> indicates that the half-lives of the four Rsm sRNAs increase in the stationary phase and when nutrients are limited. The rate of turnover for Rsm sRNAs thus seems to be related to growth conditions and cell physiology. Furthermore, the four Rsm sRNAs are not equally stable, with RsmZ appearing to be extremely stable even during the exponential phase (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/table/microorganisms-09-00250-t002/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-t002\" rid-ob=\"ob-microorganisms-09-00250-t002\" co-legend-rid=\"\"><span>Table 2</span></a>).</p><p id=\"__p43\">Regulation of RNA amount by degradation is achieved by ribonucleases (RNases), whose activity is dependent on the sequence and/or the structural elements of the RNA molecule. In <em>Salmonella enterica</em> serovar Typhimurium, the endoribonuclease RNase III has been shown to regulate MicA (an sRNA involved in porin regulation) in a target-coupled way, whereas RNase E is responsible for the control of free MicA levels in the cell [<a href=\"#B51-microorganisms-09-00250\" rid=\"B51-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605048\">51</a>].</p><p id=\"__p44\">We hypothesize that the observed differences in Rsm sRNAs are mainly due to the sequence and structural elements of each Rsm sRNA. In <em>P. protegens</em> CHA0, the estimated half-lives of RsmY and RsmZ are &#x0003e;20 min in the wild-type and &#x0003c;10 min in the <em>rsmA rsmE</em> double mutant [<a href=\"#B52-microorganisms-09-00250\" rid=\"B52-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605018\">52</a>]. RsmA/E appears to stabilize Rsm sRNAs in vivo probably by protecting them from degradation by RNases. In <em>Erwinia carotovora</em> subsp. <em>carotovora</em>, RsmA also increases the half-life of the RsmB riboregulator [<a href=\"#B53-microorganisms-09-00250\" rid=\"B53-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605014\">53</a>]. Higher levels of RsmB RNA in the <em>rsmA<sup>+</sup></em> strain than in the <em>rsmA<sup>&#x02212;</sup></em> strain were shown to be due to the increase in RsmB stability and not to an increase in transcription.</p><p id=\"__p45\" class=\"p p-last\">It has been suggested that RNase E and Hfq have similar (AU-rich) target sequences that should permit Hfq to protect sRNA from RNase E attack [<a href=\"#B54-microorganisms-09-00250\" rid=\"B54-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605000\">54</a>]. In <em>P. aeruginosa</em>, RsmY has been described as stabilized by Hfq, which sequesters RNase E cleavage sites [<a href=\"#B55-microorganisms-09-00250\" rid=\"B55-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605036\">55</a>], and thus enhances RsmA sequestration by RsmY [<a href=\"#B56-microorganisms-09-00250\" rid=\"B56-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605025\">56</a>]. In <em>P. brassicacearum</em>, an interaction of Hfq with RsmY and RsmZ may explain the extreme stability of these two sRNAs. Nevertheless, in <em>P. aeruginosa</em>, Hfq doesn&#x02019;t affect RsmZ stability, whereas in <em>P. brassicacearum</em>, RsmZ exhibits an A-U rich loop, which could interact with Hfq.</p></span><span id=\"s\r\n4dot4-microorganisms-09-00250\" class=\"sec sec-last\"><h3 id=\"s\r\n4dot4-microorganisms-09-00250title\">4.4. Modulation of Rsm sRNAs Expression in Response to Nutrient-Poor Conditions</h3><p id=\"__p46\" class=\"p p-first\">The stimuli that activate the Gac-Rsm system are still unknown [<a href=\"#B57-microorganisms-09-00250\" rid=\"B57-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605039\">57</a>]. In this study, we demonstrated that <em>rsm</em> genes are highly up-regulated under nutrient-poor conditions. Even if Rsm sRNAs are principally synthesized in the stationary phase, our results indicate that their expression is not only related to cellular density, but also and essentially to the physiological and metabolic state of cells. According to carbon source nature and availability, certain metabolites may accumulate within cells and induce specific responses. In <em>P. protegens</em> CHA0, pools of 2-oxoglutarate, succinate, and fumarate were shown to be positively correlated with the expression level of <em>rsm</em> genes [<a href=\"#B58-microorganisms-09-00250\" rid=\"B58-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605050\">58</a>]. Moreover, the expression of Rsm sRNAs is attenuated in the double mutant <em>relA spoT</em>, for which ppGpp synthesis has been completely abolished [<a href=\"#B34-microorganisms-09-00250\" rid=\"B34-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605035\">34</a>]. Our data show an activation of (p)ppGpp synthesis in response to nutrient starvation, correlating well with an increase of the four Rsm sRNAs in <em>P. brassicacearum</em> NFM421.</p><p id=\"__p47\">Another signaling molecule, the cyclic-di-GMP second messenger, which modulates the transition from the planktonic to the biofilm state [<a href=\"#B59-microorganisms-09-00250\" rid=\"B59-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925604999\">59</a>], was shown to be positively regulated by GacS-GacA system in <em>P. aeruginosa</em> [<a href=\"#B35-microorganisms-09-00250\" rid=\"B35-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605030\">35</a>,<a href=\"#B37-microorganisms-09-00250\" rid=\"B37-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605024\">37</a>]. More recently, Liang et al. [<a href=\"#B36-microorganisms-09-00250\" rid=\"B36-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_932605641\">36</a>] showed that the QS system and polyketide antibiotic 2,4-DAPG production are regulated by c-di-GMP through RsmA and RsmE proteins in <em>P. fluorescens</em> 2P24. Our work also suggests a connection between the Gac-Rsm cascade and the c-di-GMP signaling pathway in phytobeneficial <em>Pseudomonas</em>.</p><p id=\"__p48\">Surprisingly, under circumstances that are not advantageous for <em>P. brassicacearum</em> proliferation, such as when nutrients are depleted, bacteria favor the activation of <em>rsm</em> expression and enhance their stability and increase (p)ppGpp and c-di-GMP intracellular levels. Consequently, production of certain secondary metabolites and biofilm formation might be enhanced, suggesting their importance in the adaptive response to environmental constraints.</p><p id=\"__p49\" class=\"p p-last\">Based on the present results and those of previous studies [<a href=\"#B34-microorganisms-09-00250\" rid=\"B34-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_925605011\">34</a>,<a href=\"#B36-microorganisms-09-00250\" rid=\"B36-microorganisms-09-00250\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_932605647\">36</a>], we propose a model according to which ppGpp as well as c-di-GMP are directly or indirectly positively regulated by GacS/GacA system under nutrient limitations conditions (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f007/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"microorganisms-09-00250-f007\" rid-ob=\"ob-microorganisms-09-00250-f007\" co-legend-rid=\"lgnd_microorganisms-09-00250-f007\"><span>Figure 7</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"microorganisms-09-00250-f007\" co-legend-rid=\"lgnd_microorganisms-09-00250-f007\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f007/\" target=\"figure\" rid-figpopup=\"microorganisms-09-00250-f007\" rid-ob=\"ob-microorganisms-09-00250-f007\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139634168183984\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7911923_microorganisms-09-00250-g007.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is microorganisms-09-00250-g007.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC7911923/bin/microorganisms-09-00250-g007.jpg\" /></a></span><span id=\"largeobj_idm139634168183984\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f007/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_microorganisms-09-00250-f007\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/figure/microorganisms-09-00250-f007/\" target=\"figure\" rid-figpopup=\"microorganisms-09-00250-f007\" rid-ob=\"ob-microorganisms-09-00250-f007\">Figure 7</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p70\">Model for gene regulation by ppGpp and c-di-GMP in the Gac-Rsm system of <em>P. brassicacearum</em> NFM421. This pathway is supported by evidence from this work and previous studies (Lalaouna et al., 2012, Takeuchi et al., 2012, Liang et al., 2020). &#x02193;, positive effect; &#x02534;, negative effect, dotted lines, indirect effects. The new conclusion of this study is that starvation conditions seem to activate the Gac-Rsm system concomitantly with the production of ppGpp and c-di-GMP.</p></span></span></span></span></span><span id=\"s\r\n5-microorganisms-09-00250\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n5-microorganisms-09-00250title\">5. Conclusions</h2><p id=\"__p50\" class=\"p p-first\">The amplification of Rsm sRNAs and the preservation of several active copies indicate the importance of this post-transcriptional regulation, which allows the bacterium to increase its plasticity by rapidly adjusting its physiology in response to environmental constraints.</p><p id=\"__p51\">The existence of multiple factors devoted to <em>rsm</em> expression and degradation, illustrates that bacteria have evolved versatile mechanisms to control Rsm sRNAs levels. The higher expression level and the greater stability of the four Rsm sRNAs when nutrient conditions are limited underscore their relevance in the ecological niche of <em>P. brassicacearum,</em> a soil in which nutrients are poorly available. It is particularly interesting to note that this transcriptional activation of the four Rsm sRNAS occurs in concert with the increase in the intracellular level of the two signal molecules, (p)ppGpp and c-di-GMP under stringency conditions.</p><p id=\"__p52\">The differences observed in expression and stability between the four Rsm sRNAs illustrate that the subtleties of their respective roles in bacterial adaptation to challenging conditions remain far from being completely understood, as well as the regulation network involving (p)ppGpp and c-diGMP. This definitively deserves to be investigated in further detail.</p><p id=\"__p53\" class=\"p p-last\">Finally, although Rsm sRNA concentrations in the cell vary due to noise in gene expression and transcript stability, the overall combined concentrations of each Rsm sRNA are probably maintained in homeostatic balance.</p></span><span id=\"idm139634171335472\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139634171335472title\">Acknowledgments</h2><span class=\"sec\"><p id=\"__p54\">This work was supported by IRTELIS PhD program Grant from CEA.</p></span></span><span id=\"app1-microorganisms-09-00250\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"app1-microorganisms-09-00250title\">Supplementary Materials</h2><!--/article/back/app-group/app/--><p id=\"__p56\">The following are available online at <a href=\"https://www.mdpi.com/2076-2607/9/2/250/s1\">https://www.mdpi.com/2076-2607/9/2/250/s1</a>. Table S1. Primers: plasmids and strains used in this study. Figure S1. Detection of protease activity under nutrient starvation conditions. (A) Wild-type and <em>gacA</em> strains on TSA/20 lec, on TSA 1/10 lec and on undiluted TSA lec media. (B) Halo thickness (distance from bacteria to halo edge) is expressed as the mean of three measurements, in cm. lec means Skimmed milk (10/%) was added to growth medium. Figure S2. Rsm RNA stability. In vivo assay of Rsm RNA degradation of <em>P. brassicacearum</em> NFM421 culture at exponential and stationary growth phases under nutrient replete and deplete conditions and rifampicin treatment. Samples were taken after 5, 15, 30, 45, 60, 90, 120, 150 and 180 min for RNA extraction and q-RT-PCR analysis. Data were normalized to the values obtained from the analysis of 16S rRNA as an internal control.</p><span class=\"sec suppmat\" id=\"microorganisms-09-00250-s001\"><span class=\"sup-box half_rhythm\" id=\"idm139634167324912\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7911923/bin/microorganisms-09-00250-s001.pdf\" data-ga-action=\"click_feat_suppl\">Click here for additional data file.</a><sup>(814K, pdf)</sup></span></span></span><span id=\"idm139634167323440\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139634167323440title\">Author Contributions</h2><p id=\"__p58\">Conceptualization, D.L. and W.A.; D.L. performed DNA and RNA experiments; S.F. contributed to RNA decay experiments; M.H. performed metabolites analysis; P.O. performed phylogeny analysis; writing-original draft preparation, D.L.; writing-review and editing, W.A., T.H., M.H., P.O., P.S.-K. and D.L. All authors have read and agreed to the published version of the manuscript.</p></span><span id=\"idm139634167322160\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139634167322160title\">Funding</h2><p id=\"__p59\">This research received no external funding.</p></span><span id=\"idm139634167321280\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139634167321280title\">Institutional Review Board Statement</h2><p id=\"__p60\">Not Applicable.</p></span><span id=\"idm139634167320400\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139634167320400title\">Informed Consent Statement</h2><p id=\"__p61\">Not Applicable.</p></span><span id=\"idm139634167319520\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139634167319520title\">Data Availability Statement</h2><p id=\"__p62\">All data in this article is openly available without any restrictions.</p></span><span id=\"idm139634171680352\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139634171680352title\">Conflicts of Interest</h2><p id=\"__p63\">The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p></span><span id=\"idm139634171334512\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139634171334512title\">Footnotes</h2><!--back/fn-group--><span class=\"fm-sec half_rhythm small\"><p class=\"fn sec\" id=\"idm139634171334384\"><p id=\"__p55\" class=\"p p-first-last\"><strong>Publisher&#x02019;s Note:</strong> MDPI stays neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></p></span></span><br><div class=\"ref-list-sec sec\" id=\"reference-list\"><div class=\"ref-cit-blk half_rhythm\" id=\"B1-microorganisms-09-00250\">1. <span class=\"element-citation\">Romeo T. Global regulation by the small RNA-binding protein CsrA and the non-coding RNA molecule CsrB. <span><span class=\"ref-journal\">Mol. Microbiol. </span>1998;<span class=\"ref-vol\">29</span>:1321–1330. doi: 10.1046/j.1365-2958.1998.01021.x.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n9781871\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1046%2Fj.1365-2958.1998.01021.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol.&amp;title=Global+regulation+by+the+small+RNA-binding+protein+CsrA+and+the+non-coding+RNA+molecule+CsrB&amp;author=T.+Romeo&amp;volume=29&amp;publication_year=1998&amp;pages=1321-1330&amp;pmid=9781871&amp;doi=10.1046/j.1365-2958.1998.01021.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B2-microorganisms-09-00250\">2. <span class=\"element-citation\">Fortune D.R., Suyemoto M., Altier C. Identification of CsrC and characterization of its role in epithelial cell invasion in <em>Salmonella enterica</em> serovar Typhimurium. <span><span class=\"ref-journal\">Infect. Immun. </span>2006;<span class=\"ref-vol\">74</span>:331–339. doi: 10.1128/IAI.74.1.331-339.2006.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC1346597/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n16368988\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FIAI.74.1.331-339.2006\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Infect.+Immun.&amp;title=Identification+of+CsrC+and+characterization+of+its+role+in+epithelial+cell+invasion+in+Salmonella+enterica+serovar+Typhimurium&amp;author=D.R.+Fortune&amp;author=M.+Suyemoto&amp;author=C.+Altier&amp;volume=74&amp;publication_year=2006&amp;pages=331-339&amp;pmid=16368988&amp;doi=10.1128/IAI.74.1.331-339.2006&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B3-microorganisms-09-00250\">3. <span class=\"element-citation\">Lenz D.H., Miller M.B., Zhu J., Kulkarni R.V., Bassler B.L. CsrA and three redundant small RNAs regulate quorum sensing in <em>Vibrio cholerae</em>. <span><span class=\"ref-journal\">Mol. Microbiol. </span>2005;<span class=\"ref-vol\">58</span>:1186–1202. doi: 10.1111/j.1365-2958.2005.04902.x.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n16262799\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1365-2958.2005.04902.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol.&amp;title=CsrA+and+three+redundant+small+RNAs+regulate+quorum+sensing+in+Vibrio+cholerae&amp;author=D.H.+Lenz&amp;author=M.B.+Miller&amp;author=J.+Zhu&amp;author=R.V.+Kulkarni&amp;author=B.L.+Bassler&amp;volume=58&amp;publication_year=2005&amp;pages=1186-1202&amp;pmid=16262799&amp;doi=10.1111/j.1365-2958.2005.04902.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B4-microorganisms-09-00250\">4. <span class=\"element-citation\">Sahr T., Bruggemann H., Jules M., Lomma M., Albert-Weissenberger C., Cazalet C., Buchrieser C. Two small ncRNAs jointly govern virulence and transmission in <em>Legionella pneumophila</em>. <span><span class=\"ref-journal\">Mol. Microbiol. </span>2009;<span class=\"ref-vol\">72</span>:741–762. doi: 10.1111/j.1365-2958.2009.06677.x.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2888818/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19400772\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1365-2958.2009.06677.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol.&amp;title=Two+small+ncRNAs+jointly+govern+virulence+and+transmission+in+Legionella+pneumophila&amp;author=T.+Sahr&amp;author=H.+Bruggemann&amp;author=M.+Jules&amp;author=M.+Lomma&amp;author=C.+Albert-Weissenberger&amp;volume=72&amp;publication_year=2009&amp;pages=741-762&amp;pmid=19400772&amp;doi=10.1111/j.1365-2958.2009.06677.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B5-microorganisms-09-00250\">5. <span class=\"element-citation\">Heroven A.K., Bohme K., Rohde M., Dersch P. A Csr-type regulatory system, including small non-coding RNAs, regulates the global virulence regulator RovA of <em>Yersinia pseudotuberculosis</em> through RovM. <span><span class=\"ref-journal\">Mol. Microbiol. </span>2008;<span class=\"ref-vol\">68</span>:1179–1195. doi: 10.1111/j.1365-2958.2008.06218.x.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n18430141\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1365-2958.2008.06218.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol.&amp;title=A+Csr-type+regulatory+system,+including+small+non-coding+RNAs,+regulates+the+global+virulence+regulator+RovA+of+Yersinia+pseudotuberculosis+through+RovM&amp;author=A.K.+Heroven&amp;author=K.+Bohme&amp;author=M.+Rohde&amp;author=P.+Dersch&amp;volume=68&amp;publication_year=2008&amp;pages=1179-1195&amp;pmid=18430141&amp;doi=10.1111/j.1365-2958.2008.06218.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B6-microorganisms-09-00250\">6. <span class=\"element-citation\">Kay E., Dubuis C., Haas D. Three small RNAs jointly ensure secondary metabolism and biocontrol in <em>Pseudomonas fluorescens</em> CHA0. <span><span class=\"ref-journal\">Proc. Natl. Acad. Sci. USA. </span>2005;<span class=\"ref-vol\">102</span>:17136–17141. doi: 10.1073/pnas.0505673102.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC1287983/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n16286659\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1073%2Fpnas.0505673102\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Three+small+RNAs+jointly+ensure+secondary+metabolism+and+biocontrol+in+Pseudomonas+fluorescens+CHA0&amp;author=E.+Kay&amp;author=C.+Dubuis&amp;author=D.+Haas&amp;volume=102&amp;publication_year=2005&amp;pages=17136-17141&amp;pmid=16286659&amp;doi=10.1073/pnas.0505673102&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B7-microorganisms-09-00250\">7. <span class=\"element-citation\">Kay E., Humair B., Denervaud V., Riedel K., Spahr S., Eberl L., Valverde C., Haas D. Two GacA-dependent small RNAs modulate the quorum-sensing response in <em>Pseudomonas aeruginosa</em>. <span><span class=\"ref-journal\">J. Bacteriol. </span>2006;<span class=\"ref-vol\">188</span>:6026–6033. doi: 10.1128/JB.00409-06.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC1540078/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n16885472\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FJB.00409-06\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Bacteriol.&amp;title=Two+GacA-dependent+small+RNAs+modulate+the+quorum-sensing+response+in+Pseudomonas+aeruginosa&amp;author=E.+Kay&amp;author=B.+Humair&amp;author=V.+Denervaud&amp;author=K.+Riedel&amp;author=S.+Spahr&amp;volume=188&amp;publication_year=2006&amp;pages=6026-6033&amp;pmid=16885472&amp;doi=10.1128/JB.00409-06&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B8-microorganisms-09-00250\">8. <span class=\"element-citation\">Moll S., Schneider D.J., Stodghill P., Myers C.R., Cartinhour S.W., Filiatrault M.J. Construction of an <em>rsmX</em> co-variance model and identification of five <em>rsmX</em> non-coding RNAs in <em>Pseudomonas syringae</em> pv. tomato DC3000. <span><span class=\"ref-journal\">RNA Biol. </span>2010;<span class=\"ref-vol\">7</span>:3–11. doi: 10.4161/rna.7.5.12687.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3073247/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21060253\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.4161%2Frna.7.5.12687\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=RNA+Biol.&amp;title=Construction+of+an+rsmX+co-variance+model+and+identification+of+five+rsmX+non-coding+RNAs+in+Pseudomonas+syringae+pv.+tomato+DC3000&amp;author=S.+Moll&amp;author=D.J.+Schneider&amp;author=P.+Stodghill&amp;author=C.R.+Myers&amp;author=S.W.+Cartinhour&amp;volume=7&amp;publication_year=2010&amp;pages=3-11&amp;doi=10.4161/rna.7.5.12687&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B9-microorganisms-09-00250\">9. <span class=\"element-citation\">White D., Hart M.E., Romeo T. Phylogenetic distribution of the global regulatory gene <em>csrA</em> among eubacteria. <span><span class=\"ref-journal\">Gene. </span>1996;<span class=\"ref-vol\">182</span>:221–223. doi: 10.1016/S0378-1119(96)00547-1.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n8982093\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2FS0378-1119(96)00547-1\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Phylogenetic+distribution+of+the+global+regulatory+gene+csrA+among+eubacteria&amp;author=D.+White&amp;author=M.E.+Hart&amp;author=T.+Romeo&amp;volume=182&amp;publication_year=1996&amp;pages=221-223&amp;pmid=8982093&amp;doi=10.1016/S0378-1119(96)00547-1&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B10-microorganisms-09-00250\">10. <span class=\"element-citation\">Harfouche L., Haichar F.Z., Achouak W. Small regulatory RNAs and the fine-tuning of plant-bacteria interactions. <span><span class=\"ref-journal\">New Phytol. </span>2015;<span class=\"ref-vol\">206</span>:98–106. doi: 10.1111/nph.13195.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25866855\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fnph.13195\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=New+Phytol.&amp;title=Small+regulatory+RNAs+and+the+fine-tuning+of+plant-bacteria+interactions&amp;author=L.+Harfouche&amp;author=F.Z.+Haichar&amp;author=W.+Achouak&amp;volume=206&amp;publication_year=2015&amp;pages=98-106&amp;pmid=25866855&amp;doi=10.1111/nph.13195&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B11-microorganisms-09-00250\">11. <span class=\"element-citation\">Sobrero P.M., Valverde C. Comparative genomics and evolutionary analysis of Rna-binding proteins of the CsrA family in the genus <em>pseudomonas</em>. <span><span class=\"ref-journal\">Front. Mol. Biosci. </span>2020;<span class=\"ref-vol\">7</span>:127. doi: 10.3389/fmolb.2020.00127.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7366521/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n32754614\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3389%2Ffmolb.2020.00127\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front.+Mol.+Biosci.&amp;title=Comparative+genomics+and+evolutionary+analysis+of+Rna-binding+proteins+of+the+CsrA+family+in+the+genus+pseudomonas&amp;author=P.M.+Sobrero&amp;author=C.+Valverde&amp;volume=7&amp;publication_year=2020&amp;pages=127&amp;pmid=32754614&amp;doi=10.3389/fmolb.2020.00127&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B12-microorganisms-09-00250\">12. <span class=\"element-citation\">Babitzke P., Romeo T. CsrB sRNA family, sequestration of RNA-binding regulatory proteins. <span><span class=\"ref-journal\">Curr. Opin. Microbiol. </span>2007;<span class=\"ref-vol\">10</span>:156–163. doi: 10.1016/j.mib.2007.03.007.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17383221\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.mib.2007.03.007\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Microbiol.&amp;title=CsrB+sRNA+family,+sequestration+of+RNA-binding+regulatory+proteins&amp;author=P.+Babitzke&amp;author=T.+Romeo&amp;volume=10&amp;publication_year=2007&amp;pages=156-163&amp;pmid=17383221&amp;doi=10.1016/j.mib.2007.03.007&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B13-microorganisms-09-00250\">13. <span class=\"element-citation\">Gebhardt M.J., Kambara T.K., Ramsey K.M., Dove S.L. Widespread targeting of nascent transcripts by RsmA in <em>Pseudomonas aeruginosa</em>. <span><span class=\"ref-journal\">Proc. Natl. Acad. Sci. USA. </span>2020;<span class=\"ref-vol\">117</span>:10520–10529. doi: 10.1073/pnas.1917587117.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7229658/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n32332166\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1073%2Fpnas.1917587117\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Widespread+targeting+of+nascent+transcripts+by+RsmA+in+Pseudomonas+aeruginosa&amp;author=M.J.+Gebhardt&amp;author=T.K.+Kambara&amp;author=K.M.+Ramsey&amp;author=S.L.+Dove&amp;volume=117&amp;publication_year=2020&amp;pages=10520-10529&amp;pmid=32332166&amp;doi=10.1073/pnas.1917587117&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B14-microorganisms-09-00250\">14. <span class=\"element-citation\">Liu M.Y., Gui G., Wei B., Preston J.F., Oakford L., Yuksel U., Giedroc D.P., Romero T. The RNA molecule CsrB binds to the global regulatory protein CsrA and antagonizes its activity in <em>Escherichia coli</em>. <span><span class=\"ref-journal\">J. Biol. Chem. </span>1997;<span class=\"ref-vol\">272</span>:17502–17510. doi: 10.1074/jbc.272.28.17502.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n9211896\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1074%2Fjbc.272.28.17502\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=The+RNA+molecule+CsrB+binds+to+the+global+regulatory+protein+CsrA+and+antagonizes+its+activity+in+Escherichia+coli&amp;author=M.Y.+Liu&amp;author=G.+Gui&amp;author=B.+Wei&amp;author=J.F.+Preston&amp;author=L.+Oakford&amp;volume=272&amp;publication_year=1997&amp;pages=17502-17510&amp;pmid=9211896&amp;doi=10.1074/jbc.272.28.17502&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B15-microorganisms-09-00250\">15. <span class=\"element-citation\">Weilbacher T., Suzuki K., Dubey A.K., Wang X., Gudapaty S., Morozov I., Baker C.S., Georgellis D., Babitzke P., Romeo T. A novel sRNA component of the carbon storage regulatory system <em>of Escherichia coli</em>. <span><span class=\"ref-journal\">Mol. Microbiol. </span>2003;<span class=\"ref-vol\">48</span>:657–670. doi: 10.1046/j.1365-2958.2003.03459.x.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n12694612\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1046%2Fj.1365-2958.2003.03459.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol.&amp;title=A+novel+sRNA+component+of+the+carbon+storage+regulatory+system+of+Escherichia+coli&amp;author=T.+Weilbacher&amp;author=K.+Suzuki&amp;author=A.K.+Dubey&amp;author=X.+Wang&amp;author=S.+Gudapaty&amp;volume=48&amp;publication_year=2003&amp;pages=657-670&amp;pmid=12694612&amp;doi=10.1046/j.1365-2958.2003.03459.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B16-microorganisms-09-00250\">16. <span class=\"element-citation\">Valverde C. Artificial sRNAs activating the Gac/Rsm signal transduction pathway in <em>Pseudomonas fluorescens</em>. <span><span class=\"ref-journal\">Arch. Microbiol. </span>2009;<span class=\"ref-vol\">191</span>:349–359. doi: 10.1007/s00203-009-0459-x.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19214475\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs00203-009-0459-x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Arch.+Microbiol.&amp;title=Artificial+sRNAs+activating+the+Gac/Rsm+signal+transduction+pathway+in+Pseudomonas+fluorescens&amp;author=C.+Valverde&amp;volume=191&amp;publication_year=2009&amp;pages=349-359&amp;pmid=19214475&amp;doi=10.1007/s00203-009-0459-x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B17-microorganisms-09-00250\">17. <span class=\"element-citation\">Brencic A., McFarland K.A., McManus H.R., Castang S., Mogno I., Dove S.L., Lory S. The GacS/GacA signal transduction system of <em>Pseudomonas aeruginosa</em> acts exclusively through its control over the transcription of the RsmY and RsmZ regulatory small RNAs. <span><span class=\"ref-journal\">Mol. Microbiol. </span>2009;<span class=\"ref-vol\">73</span>:434–445. doi: 10.1111/j.1365-2958.2009.06782.x.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2761719/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19602144\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1365-2958.2009.06782.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol.&amp;title=The+GacS/GacA+signal+transduction+system+of+Pseudomonas+aeruginosa+acts+exclusively+through+its+control+over+the+transcription+of+the+RsmY+and+RsmZ+regulatory+small+RNAs&amp;author=A.+Brencic&amp;author=K.A.+McFarland&amp;author=H.R.+McManus&amp;author=S.+Castang&amp;author=I.+Mogno&amp;volume=73&amp;publication_year=2009&amp;pages=434-445&amp;pmid=19602144&amp;doi=10.1111/j.1365-2958.2009.06782.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B18-microorganisms-09-00250\">18. <span class=\"element-citation\">Lalaouna D., Fochesato S., Sanchez L., Schmitt-Kopplin P., Haas D., Heulin T., Achouak W. Phenotypic switching involves GacS- and GacA-dependent Rsm small RNAs in <em>Pseudomonas brassicacearum</em>. <span><span class=\"ref-journal\">Appl. Environ. Microbiol. </span>2012;<span class=\"ref-vol\">78</span>:1658–1665. doi: 10.1128/AEM.06769-11.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3298130/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22247157\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.06769-11\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microbiol.&amp;title=Phenotypic+switching+involves+GacS-+and+GacA-dependent+Rsm+small+RNAs+in+Pseudomonas+brassicacearum&amp;author=D.+Lalaouna&amp;author=S.+Fochesato&amp;author=L.+Sanchez&amp;author=P.+Schmitt-Kopplin&amp;author=D.+Haas&amp;volume=78&amp;publication_year=2012&amp;pages=1658-1665&amp;pmid=22247157&amp;doi=10.1128/AEM.06769-11&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B19-microorganisms-09-00250\">19. <span class=\"element-citation\">Humair B., Wackwitz B., Haas D. GacA-controlled activation of promoters for small RNA genes in <em>Pseudomonas fluorescens</em>. <span><span class=\"ref-journal\">Appl. Environ. Microbiol. </span>2010;<span class=\"ref-vol\">76</span>:1497–1506. doi: 10.1128/AEM.02014-09.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2832403/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20048056\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.02014-09\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microbiol.&amp;title=GacA-controlled+activation+of+promoters+for+small+RNA+genes+in+Pseudomonas+fluorescens&amp;author=B.+Humair&amp;author=B.+Wackwitz&amp;author=D.+Haas&amp;volume=76&amp;publication_year=2010&amp;pages=1497-1506&amp;pmid=20048056&amp;doi=10.1128/AEM.02014-09&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B20-microorganisms-09-00250\">20. <span class=\"element-citation\">Bordi C., Lamy M.C., Ventre I., Termine E., Hachani A., Fillet S., Roche B., Bleves C., M&#x000e9;jean V., Lazdunski A., et al.  Regulatory RNAs and the HptB/RetS signalling pathways fine-tune <em>Pseudomonas aeruginosa</em> pathogenesis. <span><span class=\"ref-journal\">Mol. Microbiol. </span>2010;<span class=\"ref-vol\">76</span>:1427–1443. doi: 10.1111/j.1365-2958.2010.07146.x.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2904497/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20398205\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1365-2958.2010.07146.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol.&amp;title=Regulatory+RNAs+and+the+HptB/RetS+signalling+pathways+fine-tune+Pseudomonas+aeruginosa+pathogenesis&amp;author=C.+Bordi&amp;author=M.C.+Lamy&amp;author=I.+Ventre&amp;author=E.+Termine&amp;author=A.+Hachani&amp;volume=76&amp;publication_year=2010&amp;pages=1427-1443&amp;pmid=20398205&amp;doi=10.1111/j.1365-2958.2010.07146.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B21-microorganisms-09-00250\">21. <span class=\"element-citation\">Achouak W., Sutra L., Heulin T., Meyer J.M., Fromin N., Degraeve S., Christen R., Gardan L. <em>Pseudomonas brassicacearum</em> sp nov and <em>Pseudomonas thivervalensis</em> sp nov.; two root-associated bacteria isolated from Brassica napus and Arabidopsis thaliana. <span><span class=\"ref-journal\">Int. J. Syst. Evol. Microbiol. </span>2000;<span class=\"ref-vol\">50</span>:9–18. doi: 10.1099/00207713-50-1-9.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n10826782\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1099%2F00207713-50-1-9\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int.+J.+Syst.+Evol.+Microbiol.&amp;title=Pseudomonas+brassicacearum+sp+nov+and+Pseudomonas+thivervalensis+sp+nov.;+two+root-associated+bacteria+isolated+from+Brassica+napus+and+Arabidopsis+thaliana&amp;author=W.+Achouak&amp;author=L.+Sutra&amp;author=T.+Heulin&amp;author=J.M.+Meyer&amp;author=N.+Fromin&amp;volume=50&amp;publication_year=2000&amp;pages=9-18&amp;pmid=10826782&amp;doi=10.1099/00207713-50-1-9&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B22-microorganisms-09-00250\">22. <span class=\"element-citation\">Achouak W., Conrod S., Cohen V., Heulin T. Phenotypic variation of <em>Pseudomonas brassicacearum</em> as a plant root-colonization strategy. <span><span class=\"ref-journal\">Mol. Plant-Microbe Interact. </span>2004;<span class=\"ref-vol\">17</span>:872–879. doi: 10.1094/MPMI.2004.17.8.872.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n15305608\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1094%2FMPMI.2004.17.8.872\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Plant-Microbe+Interact.&amp;title=Phenotypic+variation+of+Pseudomonas+brassicacearum+as+a+plant+root-colonization+strategy&amp;author=W.+Achouak&amp;author=S.+Conrod&amp;author=V.+Cohen&amp;author=T.+Heulin&amp;volume=17&amp;publication_year=2004&amp;pages=872-879&amp;pmid=15305608&amp;doi=10.1094/MPMI.2004.17.8.872&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B23-microorganisms-09-00250\">23. <span class=\"element-citation\">Achouak W., Conrod S., Robert S., Heulin T.  Phase variable effects of <em>Pseudomonas brassicacearum</em> on <em>Arabidopsis thaliana</em> root architecture. In: Tikhonovich I., Lugtenberg B., Provorov N., editors. <span class=\"ref-journal\">Biology of Molecular Plant-Microbe Interactions.</span> Volume 4. International Society for Molecular Plant-Microbe Interactions; St. Paul, MN, USA: 2004. pp. 359–363. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Biology+of+Molecular+Plant-Microbe+Interactions&amp;author=W.+Achouak&amp;author=S.+Conrod&amp;author=S.+Robert&amp;author=T.+Heulin&amp;publication_year=2004&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B24-microorganisms-09-00250\">24. <span class=\"element-citation\">Chabeaud P., de Groot A., Bitter W., Tommassen J., Heulin T., Achouak W. Phase-variable expression of an operon encoding extracellular alkaline protease, a serine protease homolog, and lipase in <em>Pseudomonas brassicacearum</em>. <span><span class=\"ref-journal\">J. Bacteriol. </span>2001;<span class=\"ref-vol\">183</span>:2117–2120. doi: 10.1128/JB.183.6.2117-2120.2001.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC95110/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n11222613\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FJB.183.6.2117-2120.2001\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Bacteriol.&amp;title=Phase-variable+expression+of+an+operon+encoding+extracellular+alkaline+protease,+a+serine+protease+homolog,+and+lipase+in+Pseudomonas+brassicacearum&amp;author=P.+Chabeaud&amp;author=A.+de+Groot&amp;author=W.+Bitter&amp;author=J.+Tommassen&amp;author=T.+Heulin&amp;volume=183&amp;publication_year=2001&amp;pages=2117-2120&amp;pmid=11222613&amp;doi=10.1128/JB.183.6.2117-2120.2001&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B25-microorganisms-09-00250\">25. <span class=\"element-citation\">Tillett D., Burns B.P., Neilan B.A. Optimized rapid amplification of cDNA ends (RACE) for mapping bacterial mRNA transcripts. <span><span class=\"ref-journal\">Biotechniques. </span>2000;<span class=\"ref-vol\">28</span>:448–456. doi: 10.2144/00283st01.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n10723556\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.2144%2F00283st01\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Biotechniques&amp;title=Optimized+rapid+amplification+of+cDNA+ends+(RACE)+for+mapping+bacterial+mRNA+transcripts&amp;author=D.+Tillett&amp;author=B.P.+Burns&amp;author=B.A.+Neilan&amp;volume=28&amp;publication_year=2000&amp;pages=448-456&amp;pmid=10723556&amp;doi=10.2144/00283st01&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B26-microorganisms-09-00250\">26. <span class=\"element-citation\">Heeb S., Blumer C., Haas D. Regulatory RNA as mediator in GacA/RsmA-dependent global control of exoproduct formation in <em>Pseudomonas fluorescens</em> CHA0. <span><span class=\"ref-journal\">J. Bacteriol. </span>2002;<span class=\"ref-vol\">184</span>:1046–1056. doi: 10.1128/jb.184.4.1046-1056.2002.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC134805/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n11807065\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2Fjb.184.4.1046-1056.2002\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Bacteriol.&amp;title=Regulatory+RNA+as+mediator+in+GacA/RsmA-dependent+global+control+of+exoproduct+formation+in+Pseudomonas+fluorescens+CHA0&amp;author=S.+Heeb&amp;author=C.+Blumer&amp;author=D.+Haas&amp;volume=184&amp;publication_year=2002&amp;pages=1046-1056&amp;pmid=11807065&amp;doi=10.1128/jb.184.4.1046-1056.2002&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B27-microorganisms-09-00250\">27. <span class=\"element-citation\">Schnider-Keel U., Seematter A., Maurhofer M., Blumer C., Duffy B., Gigot-Bonnefoy C., Reimmann C., Notz R., Defago G., Haas D., et al.  Autoinduction of 2,4-diacetylphloroglucinol biosynthesis in the biocontrol agent <em>Pseudomonas fluorescens</em> CHA0 and repression by the bacterial metabolites salicylate and pyoluteorin. <span><span class=\"ref-journal\">J. Bacteriol. </span>2000;<span class=\"ref-vol\">182</span>:1215–1225. doi: 10.1128/JB.182.5.1215-1225.2000.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC94405/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n10671440\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FJB.182.5.1215-1225.2000\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Bacteriol.&amp;title=Autoinduction+of+2,4-diacetylphloroglucinol+biosynthesis+in+the+biocontrol+agent+Pseudomonas+fluorescens+CHA0+and+repression+by+the+bacterial+metabolites+salicylate+and+pyoluteorin&amp;author=U.+Schnider-Keel&amp;author=A.+Seematter&amp;author=M.+Maurhofer&amp;author=C.+Blumer&amp;author=B.+Duffy&amp;volume=182&amp;publication_year=2000&amp;pages=1215-1225&amp;pmid=10671440&amp;doi=10.1128/JB.182.5.1215-1225.2000&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B28-microorganisms-09-00250\">28. <span class=\"element-citation\">Boisset S., Geissmann T., Huntzinger E., Fechter P., Bendridi N., Possedko M., Chevalier C., Helfer A.C., Benito Y., Jacquier A., et al.  <em>Staphylococcus aureus</em> RNAIII coordinately represses the synthesis of virulence factors and the transcription regulator Rot by an antisense mechanism. <span><span class=\"ref-journal\">Genes Dev. </span>2007;<span class=\"ref-vol\">21</span>:1353–1366. doi: 10.1101/gad.423507.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC1877748/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17545468\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1101%2Fgad.423507\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genes+Dev.&amp;title=Staphylococcus+aureus+RNAIII+coordinately+represses+the+synthesis+of+virulence+factors+and+the+transcription+regulator+Rot+by+an+antisense+mechanism&amp;author=S.+Boisset&amp;author=T.+Geissmann&amp;author=E.+Huntzinger&amp;author=P.+Fechter&amp;author=N.+Bendridi&amp;volume=21&amp;publication_year=2007&amp;pages=1353-1366&amp;pmid=17545468&amp;doi=10.1101/gad.423507&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B29-microorganisms-09-00250\">29. <span class=\"element-citation\">Newell P.D., Monds R.D., O&#x02019;Toole G.A. LapD is a bis-(3&#x02032;,5&#x02032;)-cyclic dimeric GMP-binding protein that regulates surface attachment by <em>Pseudomonas fluorescens</em> Pf0-1. <span><span class=\"ref-journal\">Proc. Natl. Acad. Sci. USA. </span>2009;<span class=\"ref-vol\">106</span>:3461–3466. doi: 10.1073/pnas.0808933106.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2651287/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19218451\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1073%2Fpnas.0808933106\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=LapD+is+a+bis-(3&#x02032;,5&#x02032;)-cyclic+dimeric+GMP-binding+protein+that+regulates+surface+attachment+by+Pseudomonas+fluorescens+Pf0-1&amp;author=P.D.+Newell&amp;author=R.D.+Monds&amp;author=G.A.+O&#x02019;Toole&amp;volume=106&amp;publication_year=2009&amp;pages=3461-3466&amp;pmid=19218451&amp;doi=10.1073/pnas.0808933106&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B30-microorganisms-09-00250\">30. <span class=\"element-citation\">Suhre K., Schmitt-Kopplin P. MassTRIX: Mass translator into pathways. <span><span class=\"ref-journal\">Nucl. Acids Res. </span>2008;<span class=\"ref-vol\">36</span>:W481–W484. doi: 10.1093/nar/gkn194.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2447776/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n18442993\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgkn194\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucl.+Acids+Res.&amp;title=MassTRIX:+Mass+translator+into+pathways&amp;author=K.+Suhre&amp;author=P.+Schmitt-Kopplin&amp;volume=36&amp;publication_year=2008&amp;pages=W481-W484&amp;pmid=18442993&amp;doi=10.1093/nar/gkn194&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B31-microorganisms-09-00250\">31. <span class=\"element-citation\">Darty K., Denise A., Ponty Y. VARNA, Interactive drawing and editing of the RNA secondary structure. <span><span class=\"ref-journal\">Bioinformatics. </span>2009;<span class=\"ref-vol\">25</span>:1974–1975. doi: 10.1093/bioinformatics/btp250.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2712331/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19398448\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fbioinformatics%2Fbtp250\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=VARNA,+Interactive+drawing+and+editing+of+the+RNA+secondary+structure&amp;author=K.+Darty&amp;author=A.+Denise&amp;author=Y.+Ponty&amp;volume=25&amp;publication_year=2009&amp;pages=1974-1975&amp;pmid=19398448&amp;doi=10.1093/bioinformatics/btp250&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B32-microorganisms-09-00250\">32. <span class=\"element-citation\">Lapouge K., Schubert M., Allain F.H.T., Haas D. Gac/Rsm signal transduction pathway of gamma-proteobacteria, from RNA recognition to regulation of social behaviour. <span><span class=\"ref-journal\">Mol. Microbiol. </span>2008;<span class=\"ref-vol\">67</span>:241–253. doi: 10.1111/j.1365-2958.2007.06042.x.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n18047567\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1365-2958.2007.06042.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol.&amp;title=Gac/Rsm+signal+transduction+pathway+of+gamma-proteobacteria,+from+RNA+recognition+to+regulation+of+social+behaviour&amp;author=K.+Lapouge&amp;author=M.+Schubert&amp;author=F.H.T.+Allain&amp;author=D.+Haas&amp;volume=67&amp;publication_year=2008&amp;pages=241-253&amp;pmid=18047567&amp;doi=10.1111/j.1365-2958.2007.06042.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B33-microorganisms-09-00250\">33. <span class=\"element-citation\">Edwards A.N., Patterson-Fortin L.M., Vakulskas C.A., Mercante J.W., Potrykus K., Vinella D., Camacho M.I., Fields J.A., Thopson S.A., Georgellis D., et al.  Circuitry linking the Csr and stringent response global regulatory systems. <span><span class=\"ref-journal\">Mol. Microbiol. </span>2011;<span class=\"ref-vol\">80</span>:1561–1580. doi: 10.1111/j.1365-2958.2011.07663.x.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3115499/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21488981\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1365-2958.2011.07663.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol.&amp;title=Circuitry+linking+the+Csr+and+stringent+response+global+regulatory+systems&amp;author=A.N.+Edwards&amp;author=L.M.+Patterson-Fortin&amp;author=C.A.+Vakulskas&amp;author=J.W.+Mercante&amp;author=K.+Potrykus&amp;volume=80&amp;publication_year=2011&amp;pages=1561-1580&amp;pmid=21488981&amp;doi=10.1111/j.1365-2958.2011.07663.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B34-microorganisms-09-00250\">34. <span class=\"element-citation\">Takeuchi K., Yamada K., Haas D. ppGpp controlled by the Gac/Rsm regulatory pathway sustains biocontrol activity in <em>Pseudomonas fluorescens</em> CHA0. <span><span class=\"ref-journal\">Mol. Plant Microbe Interact. </span>2012;<span class=\"ref-vol\">25</span>:1440–1449. doi: 10.1094/MPMI-02-12-0034-R.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23035953\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1094%2FMPMI-02-12-0034-R\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Plant+Microbe+Interact.&amp;title=ppGpp+controlled+by+the+Gac/Rsm+regulatory+pathway+sustains+biocontrol+activity+in+Pseudomonas+fluorescens+CHA0&amp;author=K.+Takeuchi&amp;author=K.+Yamada&amp;author=D.+Haas&amp;volume=25&amp;publication_year=2012&amp;pages=1440-1449&amp;pmid=23035953&amp;doi=10.1094/MPMI-02-12-0034-R&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B35-microorganisms-09-00250\">35. <span class=\"element-citation\">Moscoso J.A., Mikkelsen H., Heeb S., Williams P., Filloux A. The <em>Pseudomonas aeruginosa</em> sensor RetS switches type III and type VI secretion via c-di-GMP signalling. <span><span class=\"ref-journal\">Environ. Microbiol. </span>2011;<span class=\"ref-vol\">13</span>:3128–3138. doi: 10.1111/j.1462-2920.2011.02595.x.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21955777\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1462-2920.2011.02595.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Environ.+Microbiol.&amp;title=The+Pseudomonas+aeruginosa+sensor+RetS+switches+type+III+and+type+VI+secretion+via+c-di-GMP+signalling&amp;author=J.A.+Moscoso&amp;author=H.+Mikkelsen&amp;author=S.+Heeb&amp;author=P.+Williams&amp;author=A.+Filloux&amp;volume=13&amp;publication_year=2011&amp;pages=3128-3138&amp;pmid=21955777&amp;doi=10.1111/j.1462-2920.2011.02595.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B36-microorganisms-09-00250\">36. <span class=\"element-citation\">Liang F., Zhang B., Yang Q., Zhang Z., Zheng D., Zhang L.Q., Yan Q., Wu X. Cyclic-di-GMP regulates the quorum-sensing system and the biocontrol activity of <em>Pseudomonas fluorescens</em> 2P24 through RsmA and RsmE proteins. <span><span class=\"ref-journal\">Appl. Environ. Microbiol. </span>2020 doi: 10.1128/AEM.02016-20.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7688223/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n33036989\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FAEM.02016-20\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microbiol.&amp;title=Cyclic-di-GMP+regulates+the+quorum-sensing+system+and+the+biocontrol+activity+of+Pseudomonas+fluorescens+2P24+through+RsmA+and+RsmE+proteins&amp;author=F.+Liang&amp;author=B.+Zhang&amp;author=Q.+Yang&amp;author=Z.+Zhang&amp;author=D.+Zheng&amp;publication_year=2020&amp;doi=10.1128/AEM.02016-20&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B37-microorganisms-09-00250\">37. <span class=\"element-citation\">Frangipani E., Visaggio D., Heeb S., Kaever V., Camara M., Visca P., Imperi F. The Gac/-Rsm and cyclic-di-GMP signalling networks coordinately regulate iron uptake in <em>Pseudomonas aeruginosa</em>. <span><span class=\"ref-journal\">Environ. Microbiol. </span>2014;<span class=\"ref-vol\">16</span>:676–688. doi: 10.1111/1462-2920.12164.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23796404\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2F1462-2920.12164\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Environ.+Microbiol.&amp;title=The+Gac/-Rsm+and+cyclic-di-GMP+signalling+networks+coordinately+regulate+iron+uptake+in+Pseudomonas+aeruginosa&amp;author=E.+Frangipani&amp;author=D.+Visaggio&amp;author=S.+Heeb&amp;author=V.+Kaever&amp;author=M.+Camara&amp;volume=16&amp;publication_year=2014&amp;pages=676-688&amp;pmid=23796404&amp;doi=10.1111/1462-2920.12164&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B38-microorganisms-09-00250\">38. <span class=\"element-citation\">Hammer B.K., Bassler B.L. Regulatory small RNAs circumvent the conventional quorum sensing pathway in pandemic <em>Vibrio cholerae</em>. <span><span class=\"ref-journal\">Proc. Natl. Acad. Sci. USA. </span>2007;<span class=\"ref-vol\">104</span>:11145–11149. doi: 10.1073/pnas.0703860104.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC1888797/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17556542\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1073%2Fpnas.0703860104\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Regulatory+small+RNAs+circumvent+the+conventional+quorum+sensing+pathway+in+pandemic+Vibrio+cholerae&amp;author=B.K.+Hammer&amp;author=B.L.+Bassler&amp;volume=104&amp;publication_year=2007&amp;pages=11145-11149&amp;pmid=17556542&amp;doi=10.1073/pnas.0703860104&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B39-microorganisms-09-00250\">39. <span class=\"element-citation\">Rutherford S.T., Valastyan J.S., Taillefumier T., Wingreen N.S., Bassler B.L. Comprehensive analysis reveals how single nucleotides contribute to noncoding RNA function in bacterial quorum sensing. <span><span class=\"ref-journal\">Proc. Natl. Acad. Sci. USA. </span>2015;<span class=\"ref-vol\">112</span>:E6038–E6047. doi: 10.1073/pnas.1518958112.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4640798/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26483489\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1073%2Fpnas.1518958112\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Comprehensive+analysis+reveals+how+single+nucleotides+contribute+to+noncoding+RNA+function+in+bacterial+quorum+sensing&amp;author=S.T.+Rutherford&amp;author=J.S.+Valastyan&amp;author=T.+Taillefumier&amp;author=N.S.+Wingreen&amp;author=B.L.+Bassler&amp;volume=112&amp;publication_year=2015&amp;pages=E6038-E6047&amp;pmid=26483489&amp;doi=10.1073/pnas.1518958112&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B40-microorganisms-09-00250\">40. <span class=\"element-citation\">Wilderman P.J., Sowa N.A., FitzGerald D.J., FitzGerald P.C., Gottesman S., Ochsner U.A., Vasil M.L. Identification of tandem duplicate regulatory small RNAs in <em>Pseudomonas aeruginosa</em> involved in iron homeostasis. <span><span class=\"ref-journal\">Proc. Natl. Acad. Sci. USA. </span>2004;<span class=\"ref-vol\">101</span>:9792–9797. doi: 10.1073/pnas.0403423101.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC470753/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n15210934\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1073%2Fpnas.0403423101\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Identification+of+tandem+duplicate+regulatory+small+RNAs+in+Pseudomonas+aeruginosa+involved+in+iron+homeostasis&amp;author=P.J.+Wilderman&amp;author=N.A.+Sowa&amp;author=D.J.+FitzGerald&amp;author=P.C.+FitzGerald&amp;author=S.+Gottesman&amp;volume=101&amp;publication_year=2004&amp;pages=9792-9797&amp;pmid=15210934&amp;doi=10.1073/pnas.0403423101&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B41-microorganisms-09-00250\">41. <span class=\"element-citation\">Padalon-Brauch G., Hershberg R., Elgrably-Weiss M., Baruch K., Rosenshine I., Mar-galit H., Altuvia S. Small RNAs encoded within genetic islands of <em>Salmonella typhimurium</em> show host-induced expression and role in virulence. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2008;<span class=\"ref-vol\">36</span>:1913–1927. doi: 10.1093/nar/gkn050.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2330248/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n18267966\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgkn050\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Small+RNAs+encoded+within+genetic+islands+of+Salmonella+typhimurium+show+host-induced+expression+and+role+in+virulence&amp;author=G.+Padalon-Brauch&amp;author=R.+Hershberg&amp;author=M.+Elgrably-Weiss&amp;author=K.+Baruch&amp;author=I.+Rosenshine&amp;volume=36&amp;publication_year=2008&amp;pages=1913-1927&amp;pmid=18267966&amp;doi=10.1093/nar/gkn050&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B42-microorganisms-09-00250\">42. <span class=\"element-citation\">Macintyre R.J. Evolution and ecological value of duplicate genes. <span><span class=\"ref-journal\">Annu. Rev. Ecol. Syst. </span>1976;<span class=\"ref-vol\">7</span>:421–468. doi: 10.1146/annurev.es.07.110176.002225.</span> [<a href=\"//dx.doi.org/10.1146%2Fannurev.es.07.110176.002225\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Ecol.+Syst.&amp;title=Evolution+and+ecological+value+of+duplicate+genes&amp;author=R.J.+Macintyre&amp;volume=7&amp;publication_year=1976&amp;pages=421-468&amp;doi=10.1146/annurev.es.07.110176.002225&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B43-microorganisms-09-00250\">43. <span class=\"element-citation\">Reams A.B., Neidle E.L. Selection for gene clustering by tandem duplication. <span><span class=\"ref-journal\">Annu. Rev. Microbiol. </span>2004;<span class=\"ref-vol\">58</span>:119–142. doi: 10.1146/annurev.micro.58.030603.123806.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n15487932\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1146%2Fannurev.micro.58.030603.123806\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Microbiol.&amp;title=Selection+for+gene+clustering+by+tandem+duplication&amp;author=A.B.+Reams&amp;author=E.L.+Neidle&amp;volume=58&amp;publication_year=2004&amp;pages=119-142&amp;pmid=15487932&amp;doi=10.1146/annurev.micro.58.030603.123806&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B44-microorganisms-09-00250\">44. <span class=\"element-citation\">Anderson R.P., Roth J.R. Tandem genetic duplications in phage and bacteria. <span><span class=\"ref-journal\">Annu. Rev. Microbiol. </span>1977;<span class=\"ref-vol\">31</span>:473–505. doi: 10.1146/annurev.mi.31.100177.002353.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n334045\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1146%2Fannurev.mi.31.100177.002353\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Microbiol.&amp;title=Tandem+genetic+duplications+in+phage+and+bacteria&amp;author=R.P.+Anderson&amp;author=J.R.+Roth&amp;volume=31&amp;publication_year=1977&amp;pages=473-505&amp;pmid=334045&amp;doi=10.1146/annurev.mi.31.100177.002353&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B45-microorganisms-09-00250\">45. <span class=\"element-citation\">Edlund T., Grundstrom T., Normark S. Isolation and characterization of DNA repetitions carrying the chromosomal beta-lactamase gene of <em>Escherichia coli</em> K-12. <span><span class=\"ref-journal\">Mol. Gen. Genet. </span>1979;<span class=\"ref-vol\">173</span>:115–125. doi: 10.1007/BF00330301.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n386030\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2FBF00330301\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Mol.+Gen.+Genet.&amp;title=Isolation+and+characterization+of+DNA+repetitions+carrying+the+chromosomal+beta-lactamase+gene+of+Escherichia+coli+K-12&amp;author=T.+Edlund&amp;author=T.+Grundstrom&amp;author=S.+Normark&amp;volume=173&amp;publication_year=1979&amp;pages=115-125&amp;pmid=386030&amp;doi=10.1007/BF00330301&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B46-microorganisms-09-00250\">46. <span class=\"element-citation\">Lalaouna D., Fochesato S., Barakat M., Ortet P., Achouak W. Multiple transcription-activating sequences regulate the RsmZ regulatory small RNA of <em>Pseudomonas brassicacearum</em>. <span><span class=\"ref-journal\">J. Bacteriol. </span>2012;<span class=\"ref-vol\">194</span>:4888–4893. doi: 10.1128/JB.00408-12.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3430305/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22773788\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FJB.00408-12\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Bacteriol.&amp;title=Multiple+transcription-activating+sequences+regulate+the+RsmZ+regulatory+small+RNA+of+Pseudomonas+brassicacearum&amp;author=D.+Lalaouna&amp;author=S.+Fochesato&amp;author=M.+Barakat&amp;author=P.+Ortet&amp;author=W.+Achouak&amp;volume=194&amp;publication_year=2012&amp;pages=4888-4893&amp;pmid=22773788&amp;doi=10.1128/JB.00408-12&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B47-microorganisms-09-00250\">47. <span class=\"element-citation\">Janssen K.H., Diaz M.R., Gode C.J., Wolfgang M.C., Yahr T.L. RsmV, a small noncoding regulatory RNA in <em>Pseudomonas aeruginosa</em> that sequesters RsmA and RsmF from target mRNAs. <span><span class=\"ref-journal\">J. Bacteriol. </span>2018;<span class=\"ref-vol\">200</span>:e00277-18. doi: 10.1128/JB.00277-18.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC6060366/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n29866805\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FJB.00277-18\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Bacteriol.&amp;title=RsmV,+a+small+noncoding+regulatory+RNA+in+Pseudomonas+aeruginosa+that+sequesters+RsmA+and+RsmF+from+target+mRNAs&amp;author=K.H.+Janssen&amp;author=M.R.+Diaz&amp;author=C.J.+Gode&amp;author=M.C.+Wolfgang&amp;author=T.L.+Yahr&amp;volume=200&amp;publication_year=2018&amp;pages=e00277-18&amp;pmid=29866805&amp;doi=10.1128/JB.00277-18&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B48-microorganisms-09-00250\">48. <span class=\"element-citation\">Miller C.L., Romero M., Karna S.L., Chen T., Heeb S., Leung K.P. RsmW, <em>Pseudomonas aeruginosa</em> small non-coding RsmA-binding RNA upregulated in biofilm versus planktonic growth conditions. <span><span class=\"ref-journal\">BMC Microbiol. </span>2016;<span class=\"ref-vol\">16</span>:155.  doi: 10.1186/s12866-016-0771-y.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4950607/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27430253\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1186%2Fs12866-016-0771-y\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Microbiol.&amp;title=RsmW,+Pseudomonas+aeruginosa+small+non-coding+RsmA-binding+RNA+upregulated+in+biofilm+versus+planktonic+growth+conditions&amp;author=C.L.+Miller&amp;author=M.+Romero&amp;author=S.L.+Karna&amp;author=T.+Chen&amp;author=S.+Heeb&amp;volume=16&amp;publication_year=2016&amp;pages=155&amp;pmid=27430253&amp;doi=10.1186/s12866-016-0771-y&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B49-microorganisms-09-00250\">49. <span class=\"element-citation\">Storz G., Opdyke J.A., Zhang A. Controlling mRNA stability and translation with small, noncoding RNAs. <span><span class=\"ref-journal\">Curr. Opin. Microbio. </span>2004;<span class=\"ref-vol\">7</span>:140–144. doi: 10.1016/j.mib.2004.02.015.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n15063850\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.mib.2004.02.015\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Microbio.&amp;title=Controlling+mRNA+stability+and+translation+with+small,+noncoding+RNAs&amp;author=G.+Storz&amp;author=J.A.+Opdyke&amp;author=A.+Zhang&amp;volume=7&amp;publication_year=2004&amp;pages=140-144&amp;doi=10.1016/j.mib.2004.02.015&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B50-microorganisms-09-00250\">50. <span class=\"element-citation\">Suzuki K., Babitzke P., Kushner S.R., Romeo T. Identification of a novel regulatory protein (CsrD) that targets the global regulatory RNAs CsrB and CsrC for degradation by RNase E. <span><span class=\"ref-journal\">Genes Dev. </span>2006;<span class=\"ref-vol\">20</span>:2605–2617. doi: 10.1101/gad.1461606.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC1578682/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n16980588\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1101%2Fgad.1461606\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genes+Dev.&amp;title=Identification+of+a+novel+regulatory+protein+(CsrD)+that+targets+the+global+regulatory+RNAs+CsrB+and+CsrC+for+degradation+by+RNase+E&amp;author=K.+Suzuki&amp;author=P.+Babitzke&amp;author=S.R.+Kushner&amp;author=T.+Romeo&amp;volume=20&amp;publication_year=2006&amp;pages=2605-2617&amp;pmid=16980588&amp;doi=10.1101/gad.1461606&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B51-microorganisms-09-00250\">51. <span class=\"element-citation\">Viegas S.C., Silva I.J., Saramago M., Domingues S., Arraiano C.M. Regulation of the small regulatory RNA MicA by ribonuclease III, a target-dependent pathway. <span><span class=\"ref-journal\">Nucleic Acids Res. </span>2011;<span class=\"ref-vol\">39</span>:2918–2930. doi: 10.1093/nar/gkq1239.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3074148/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21138960\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1093%2Fnar%2Fgkq1239\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Regulation+of+the+small+regulatory+RNA+MicA+by+ribonuclease+III,+a+target-dependent+pathway&amp;author=S.C.+Viegas&amp;author=I.J.+Silva&amp;author=M.+Saramago&amp;author=S.+Domingues&amp;author=C.M.+Arraiano&amp;volume=39&amp;publication_year=2011&amp;pages=2918-2930&amp;pmid=21138960&amp;doi=10.1093/nar/gkq1239&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B52-microorganisms-09-00250\">52. <span class=\"element-citation\">Reimmann C., Valverde C., Kay E., Haas D. Posttranscriptional repression of GacS/GacA-controlled genes by the RNA-binding protein RsmE acting together with RsmA in Biocontrol strain <em>Pseudomonas fluorescens</em> CHA0. <span><span class=\"ref-journal\">J. Bacteriol. </span>2005;<span class=\"ref-vol\">187</span>:276–285. doi: 10.1128/JB.187.1.276-285.2005.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC538806/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n15601712\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FJB.187.1.276-285.2005\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Bacteriol.&amp;title=Posttranscriptional+repression+of+GacS/GacA-controlled+genes+by+the+RNA-binding+protein+RsmE+acting+together+with+RsmA+in+Biocontrol+strain+Pseudomonas+fluorescens+CHA0&amp;author=C.+Reimmann&amp;author=C.+Valverde&amp;author=E.+Kay&amp;author=D.+Haas&amp;volume=187&amp;publication_year=2005&amp;pages=276-285&amp;pmid=15601712&amp;doi=10.1128/JB.187.1.276-285.2005&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B53-microorganisms-09-00250\">53. <span class=\"element-citation\">Chatterjee A., Cui Y., Chatterjee A.K. RsmA and the quorum-sensing signal, N-[3-oxohexanoyl]-L-homoserine lactone, control the levels of rsmB RNA in <em>Erwinia carotovora</em> subsp. <em>carotovora</em> by affecting its stability. <span><span class=\"ref-journal\">J. Bacteriol. </span>2002;<span class=\"ref-vol\">184</span>:4089–4095. doi: 10.1128/JB.184.15.4089-4095.2002.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC135201/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n12107125\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FJB.184.15.4089-4095.2002\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Bacteriol.&amp;title=RsmA+and+the+quorum-sensing+signal,+N-[3-oxohexanoyl]-L-homoserine+lactone,+control+the+levels+of+rsmB+RNA+in+Erwinia+carotovora+subsp.+carotovora+by+affecting+its+stability&amp;author=A.+Chatterjee&amp;author=Y.+Cui&amp;author=A.K.+Chatterjee&amp;volume=184&amp;publication_year=2002&amp;pages=4089-4095&amp;pmid=12107125&amp;doi=10.1128/JB.184.15.4089-4095.2002&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B54-microorganisms-09-00250\">54. <span class=\"element-citation\">Masse E., Escorcia F.E., Gottesman S. Coupled degradation of a small regulatory RNA and its mRNA targets in <em>Escherichia coli</em>. <span><span class=\"ref-journal\">Genes Dev. </span>2003;<span class=\"ref-vol\">17</span>:2374–2383. doi: 10.1101/gad.1127103.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC218075/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n12975324\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1101%2Fgad.1127103\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genes+Dev.&amp;title=Coupled+degradation+of+a+small+regulatory+RNA+and+its+mRNA+targets+in+Escherichia+coli&amp;author=E.+Masse&amp;author=F.E.+Escorcia&amp;author=S.+Gottesman&amp;volume=17&amp;publication_year=2003&amp;pages=2374-2383&amp;pmid=12975324&amp;doi=10.1101/gad.1127103&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B55-microorganisms-09-00250\">55. <span class=\"element-citation\">Sorger-Domenigg T., Sonnleitner E., Kaberdin V.R., Blasi U. Distinct and overlapping binding sites of <em>Pseudomonas aeruginosa</em> Hfq and RsmA proteins on the non-coding RNA RsmY. <span><span class=\"ref-journal\">Biochem. Biophys. Res. Commun. </span>2007;<span class=\"ref-vol\">352</span>:769–773. doi: 10.1016/j.bbrc.2006.11.084.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17141182\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.bbrc.2006.11.084\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Biochem.+Biophys.+Res.+Commun.&amp;title=Distinct+and+overlapping+binding+sites+of+Pseudomonas+aeruginosa+Hfq+and+RsmA+proteins+on+the+non-coding+RNA+RsmY&amp;author=T.+Sorger-Domenigg&amp;author=E.+Sonnleitner&amp;author=V.R.+Kaberdin&amp;author=U.+Blasi&amp;volume=352&amp;publication_year=2007&amp;pages=769-773&amp;pmid=17141182&amp;doi=10.1016/j.bbrc.2006.11.084&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B56-microorganisms-09-00250\">56. <span class=\"element-citation\">Janssen K.H., Corley J.M., Djapgne L., Cribbs J.T., Voelker D., Slusher Z., Nordell R., Regulski E.E., Kazmierczak B.I., McMackin E.W., et al.  Hfq and sRNA 179 inhibit expression of the <em>Pseudomonas aeruginosa</em> cAMP-Vfr and type III secretion regulons. <span><span class=\"ref-journal\">mBio. </span>2020;<span class=\"ref-vol\">11</span>:e00363-20. doi: 10.1128/mBio.00363-20.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7298702/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n32546612\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1128%2FmBio.00363-20\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Hfq+and+sRNA+179+inhibit+expression+of+the+Pseudomonas+aeruginosa+cAMP-Vfr+and+type+III+secretion+regulons&amp;author=K.H.+Janssen&amp;author=J.M.+Corley&amp;author=L.+Djapgne&amp;author=J.T.+Cribbs&amp;author=D.+Voelker&amp;volume=11&amp;publication_year=2020&amp;pages=e00363-20&amp;pmid=32546612&amp;doi=10.1128/mBio.00363-20&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B57-microorganisms-09-00250\">57. <span class=\"element-citation\">Haas D., Defago G. Biological control of soil-borne pathogens by fluorescent pseudomonads. <span><span class=\"ref-journal\">Nat. Rev. Microbiol. </span>2005;<span class=\"ref-vol\">3</span>:307–319.</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n15759041\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Microbiol.&amp;title=Biological+control+of+soil-borne+pathogens+by+fluorescent+pseudomonads&amp;author=D.+Haas&amp;author=G.+Defago&amp;volume=3&amp;publication_year=2005&amp;pages=307-319&amp;pmid=15759041&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B58-microorganisms-09-00250\">58. <span class=\"element-citation\">Takeuchi K., Kiefer P., Reimmann C., Keel C., Dubuis C., Rolli J., Vorgolt J.A., Haas D. Small RNA-dependent Expression of Secondary Metabolism Is Controlled by Krebs Cycle Function in <em>Pseudomonas fluorescens</em>. <span><span class=\"ref-journal\">J. Biol. Chem. </span>2009;<span class=\"ref-vol\">284</span>:34976–34985. doi: 10.1074/jbc.M109.052571.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2787360/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19840935\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1074%2Fjbc.M109.052571\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Small+RNA-dependent+Expression+of+Secondary+Metabolism+Is+Controlled+by+Krebs+Cycle+Function+in+Pseudomonas+fluorescens&amp;author=K.+Takeuchi&amp;author=P.+Kiefer&amp;author=C.+Reimmann&amp;author=C.+Keel&amp;author=C.+Dubuis&amp;volume=284&amp;publication_year=2009&amp;pages=34976-34985&amp;pmid=19840935&amp;doi=10.1074/jbc.M109.052571&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"B59-microorganisms-09-00250\">59. <span class=\"element-citation\">Romling U., Galperin M.Y., Gomelsky M. Cyclic di-GMP: The first 25 years of a universal bacterial second messenger. <span><span class=\"ref-journal\">Microbiol. Mol. Biol. Rev. </span>2013;<span class=\"ref-vol\">77</span>:1–52.</span> <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3591986/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23471616\" ref=\"reftype=pubmed&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Microbiol.+Mol.+Biol.+Rev.&amp;title=Cyclic+di-GMP:+The+first+25+years+of+a+universal+bacterial+second+messenger&amp;author=U.+Romling&amp;author=M.Y.+Galperin&amp;author=M.+Gomelsky&amp;volume=77&amp;publication_year=2013&amp;pages=1-52&amp;pmid=23471616&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7911923&amp;issue-id=376201&amp;journal-id=3054&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div></div><br>', NULL, 2);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(81, 'pubmed', '33095762', 'The development and validation of the Videogaming Motives Questionnaire (VMQ).', 'Gaming motives are important factors for explaining individual differences in videogame-related behaviors. The aim of the present study was to develop a new comprehensive but brief instrument-the Videogaming Motives Questionnaire (VMQ)-which embraces some of the most relevant gaming motives. In a first study, a pilot exploratory factor analysis (EFA) with data from 140 undergraduates was performed on items from twelve potential motives. This identified eight main factors: recreation, social interaction, coping, violent reward, fantasy, cognitive development, customization, and competition. In Studies 2 and 3, an EFA and a confirmatory factor analysis were performed on two independent samples of 407 adolescents and 260 young adults, respectively. The VMQ presented a robust eight-factor structure, with all scales showing adequate reliability indices. In reference to criterion validity, all motives presented specific associations with hours spent playing videogames, disordered gaming, and game genre preferences. More specifically, and in both adolescents and young adults, social interaction was the main motive related to time spent gaming, whereas disordered gaming was related to both coping and social interaction motives. Based on these findings, it is concluded that the VMQ is a brief and psychometrically appropriate tool for assessing the most relevant videogaming motives.', 2020, 'PloS one', 'PMC7584249', 2, '<h1 class=\"content-title\">The development and validation of the Videogaming Motives Questionnaire (VMQ)</h1><br><span class=\"contrib-group fm-author\"><a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=L%26%23x000f3%3Bpez-Fern%26%23x000e1%3Bndez%20FJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33095762\" class=\"affpopup\" co-rid=\"_co_idm140210193605472\" co-class=\"co-affbox\">Francisco J. L&#x000f3;pez-Fern&#x000e1;ndez</a>, <span class=\"fm-role\">Conceptualization</span>, <span class=\"fm-role\">Data curation</span>, <span class=\"fm-role\">Formal analysis</span>, <span class=\"fm-role\">Investigation</span>, <span class=\"fm-role\">Methodology</span>, <span class=\"fm-role\">Writing &#x02013; original draft</span>,<sup>1</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Mezquita%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33095762\" class=\"affpopup\" co-rid=\"_co_idm140210204775488\" co-class=\"co-affbox\">Laura Mezquita</a>, <span class=\"fm-role\">Formal analysis</span>, <span class=\"fm-role\">Methodology</span>, <span class=\"fm-role\">Writing &#x02013; review &#x00026; editing</span>,<sup>1,</sup><sup>2</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Griffiths%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33095762\" class=\"affpopup\" co-rid=\"_co_idm140210196722512\" co-class=\"co-affbox\">Mark D. Griffiths</a>, <span class=\"fm-role\">Supervision</span>, <span class=\"fm-role\">Writing &#x02013; review &#x00026; editing</span>,<sup>3</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Ortet%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33095762\" class=\"affpopup\" co-rid=\"_co_idm140210196718400\" co-class=\"co-affbox\">Gener&#x000f3;s Ortet</a>, <span class=\"fm-role\">Funding acquisition</span>, <span class=\"fm-role\">Project administration</span>, <span class=\"fm-role\">Resources</span>, <span class=\"fm-role\">Writing &#x02013; review &#x00026; editing</span>,<sup>1,</sup><sup>2</sup> and  <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Ib%26%23x000e1%3B%26%23x000f1%3Bez%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33095762\" class=\"affpopup\" co-rid=\"_co_idm140210205791712\" co-class=\"co-affbox\">Manuel I. Ib&#x000e1;&#x000f1;ez</a>, <span class=\"fm-role\">Conceptualization</span>, <span class=\"fm-role\">Funding acquisition</span>, <span class=\"fm-role\">Methodology</span>, <span class=\"fm-role\">Project administration</span>, <span class=\"fm-role\">Supervision</span>, <span class=\"fm-role\">Writing &#x02013; review &#x00026; editing</span><sup>1,</sup><sup>2,</sup><sup>*</sup></span><span style=\"display:none\" class=\"contrib-group aff-tip\"><span id=\"_co_idm140210193605472\"><h3 class=\"no_margin\">Francisco J. L&#x000f3;pez-Fern&#x000e1;ndez</h3><p><sup>1</sup>\nDepartment of Basic and Clinical Psychology and Psychobiology, Universitat Jaume I, Castell&#x000f3;, Spain</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=L%26%23x000f3%3Bpez-Fern%26%23x000e1%3Bndez%20FJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33095762\">Francisco J. L&#x000f3;pez-Fern&#x000e1;ndez</a></span></span><span id=\"_co_idm140210204775488\"><h3 class=\"no_margin\">Laura Mezquita</h3><p><sup>1</sup>\nDepartment of Basic and Clinical Psychology and Psychobiology, Universitat Jaume I, Castell&#x000f3;, Spain</p><p><sup>2</sup>\nInstituto de Salud Carlos III, Centro de Investigaci&#x000f3;n Biom&#x000e9;dica en Red de Salud Mental (CIBERSAM), Castell&#x000f3;, Spain</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Mezquita%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33095762\">Laura Mezquita</a></span></span><span id=\"_co_idm140210196722512\"><h3 class=\"no_margin\">Mark D. Griffiths</h3><p><sup>3</sup>\nInternational Gaming Research Unit, Psychology Department, Nottingham Trent University, Nottingham, United Kingdom</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Griffiths%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33095762\">Mark D. Griffiths</a></span></span><span id=\"_co_idm140210196718400\"><h3 class=\"no_margin\">Gener&#x000f3;s Ortet</h3><p><sup>1</sup>\nDepartment of Basic and Clinical Psychology and Psychobiology, Universitat Jaume I, Castell&#x000f3;, Spain</p><p><sup>2</sup>\nInstituto de Salud Carlos III, Centro de Investigaci&#x000f3;n Biom&#x000e9;dica en Red de Salud Mental (CIBERSAM), Castell&#x000f3;, Spain</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Ortet%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33095762\">Gener&#x000f3;s Ortet</a></span></span><span id=\"_co_idm140210205791712\"><h3 class=\"no_margin\">Manuel I. Ib&#x000e1;&#x000f1;ez</h3><p><sup>1</sup>\nDepartment of Basic and Clinical Psychology and Psychobiology, Universitat Jaume I, Castell&#x000f3;, Spain</p><p><sup>2</sup>\nInstituto de Salud Carlos III, Centro de Investigaci&#x000f3;n Biom&#x000e9;dica en Red de Salud Mental (CIBERSAM), Castell&#x000f3;, Spain</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Ib%26%23x000e1%3B%26%23x000f1%3Bez%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33095762\">Manuel I. Ib&#x000e1;&#x000f1;ez</a></span></span></span></span><span class=\"contrib-group half_rhythm fm-editor\">Francesca Chiesi, <span class=\"fm-role\">Editor</span><sup></sup></span><span class=\"fm-panel half_rhythm\"><span class=\"togglers<br><span id=\"idm140210205615328\" lang=\"en\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140210205615328title\">Abstract</h2><!--article-meta--><span><p id=\"__p2\" class=\"p p-first-last\">Gaming motives are important factors for explaining individual differences in videogame-related behaviors. The aim of the present study was to develop a new comprehensive but brief instrument&#x02013;the Videogaming Motives Questionnaire (VMQ)&#x02013;which embraces some of the most relevant gaming motives. In a first study, a pilot exploratory factor analysis (EFA) with data from 140 undergraduates was performed on items from twelve potential motives. This identified eight main factors: recreation, social interaction, coping, violent reward, fantasy, cognitive development, customization, and competition. In Studies 2 and 3, an EFA and a confirmatory factor analysis were performed on two independent samples of 407 adolescents and 260 young adults, respectively. The VMQ presented a robust eight-factor structure, with all scales showing adequate reliability indices. In reference to criterion validity, all motives presented specific associations with hours spent playing videogames, disordered gaming, and game genre preferences. More specifically, and in both adolescents and young adults, social interaction was the main motive related to time spent gaming, whereas disordered gaming was related to both coping and social interaction motives. Based on these findings, it is concluded that the VMQ is a brief and psychometrically appropriate tool for assessing the most relevant videogaming motives.</p></span></span><span id=\"s\r\n001\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n001title\">Introduction</h2><p id=\"__p4\" class=\"p p-first\">In the past three decades, videogame playing has become one of the most popular leisure activities worldwide [<a href=\"#pone.0240726.ref001\" rid=\"pone.0240726.ref001\" class=\" bibr popnode\">1</a>]. In Spain (where the present study was carried out), 13 million people play videogames weekly and the highest rates of use are among adolescents (12&#x02013;17 years) and young adults (18&#x02013;24 year) [<a href=\"#pone.0240726.ref002\" rid=\"pone.0240726.ref002\" class=\" bibr popnode\">2</a>]. Although most gamers play without any problems, there has been concern about the consequences of excessive use among a minority of individuals. This led to the introduction of Internet Gaming Disorder (IGD) in Section 3 (&#x0201c;Conditions for Further Study&#x0201d;) of the latest (fifth) edition of the <em>Diagnostic and Statistical Manual of Mental Disorders</em>, DSM-5, defined as a &#x0201c;<em>persistent and recurrent use of the Internet to engage in games</em>, <em>often with other players</em>, <em>leading to clinically significant impairment or distress</em>. . .&#x0201d; [<a href=\"#pone.0240726.ref003\" rid=\"pone.0240726.ref003\" class=\" bibr popnode\">3</a>]. In addition, the World Health Organization [<a href=\"#pone.0240726.ref004\" rid=\"pone.0240726.ref004\" class=\" bibr popnode\">4</a>] included Gaming Disorder in the latest (eleventh) edition of the International Classification of Diseases (ICD-11) in the section &#x0201c;Disorders due to substance use or addictive behaviors&#x0201d;.</p><p id=\"__p5\">These inclusions in the DSM-5 and ICD-11 assume that disordered gaming (for simplicity, this term will be used throughout the manuscript to describe the syndrome) shares a common symptomatology with substance use and gambling disorders [<a href=\"#pone.0240726.ref005\" rid=\"pone.0240726.ref005\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889801\">5</a>]. The worldwide prevalence rates among representative samples range between 1 and 9% depending on age groups and sociocultural differences [<a href=\"#pone.0240726.ref006\" rid=\"pone.0240726.ref006\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889797\">6</a>], with the highest prevalence rates observed at younger ages and among males [<a href=\"#pone.0240726.ref007\" rid=\"pone.0240726.ref007\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889761\">7</a>]. In Spain, the prevalence of adolescent disordered gaming is around 7&#x02013;8% [<a href=\"#pone.0240726.ref008\" rid=\"pone.0240726.ref008\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889727\">8</a>&#x02013;<a href=\"#pone.0240726.ref010\" rid=\"pone.0240726.ref010\" class=\" bibr popnode\">10</a>]. Disordered gaming behavior can result in significant clinical impairment in the functioning of life areas (e.g., relationships, education and/or occupation, and psychological wellbeing) [<a href=\"#pone.0240726.ref007\" rid=\"pone.0240726.ref007\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889788\">7</a>,<a href=\"#pone.0240726.ref011\" rid=\"pone.0240726.ref011\" class=\" bibr popnode\">11</a>,<a href=\"#pone.0240726.ref012\" rid=\"pone.0240726.ref012\" class=\" bibr popnode\">12</a>]. Furthermore, some studies have demonstrated that disordered gaming predicts internalizing and externalizing problems as well as poor academic performance over time [<a href=\"#pone.0240726.ref013\" rid=\"pone.0240726.ref013\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889795\">13</a>&#x02013;<a href=\"#pone.0240726.ref017\" rid=\"pone.0240726.ref017\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889772\">17</a>].</p><p id=\"__p6\">Although structural and situational characteristics are involved in the acquisition, development, and maintenance of disordered gaming [<a href=\"#pone.0240726.ref018\" rid=\"pone.0240726.ref018\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889789\">18</a>,<a href=\"#pone.0240726.ref019\" rid=\"pone.0240726.ref019\" class=\" bibr popnode\">19</a>], personal variables of the gamers are also important, including motives for playing. Some models that contemplate the interrelated role of biological, psychological and social variables in the development of regular and disordered gaming consider motives as important predisposition components in gaming behavior [<a href=\"#pone.0240726.ref020\" rid=\"pone.0240726.ref020\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889783\">20</a>,<a href=\"#pone.0240726.ref021\" rid=\"pone.0240726.ref021\" class=\" bibr popnode\">21</a>]. Motives have shown to be especially relevant in other addictive-related behaviors, such as in the use and abuse of alcohol [<a href=\"#pone.0240726.ref022\" rid=\"pone.0240726.ref022\" class=\" bibr popnode\">22</a>,<a href=\"#pone.0240726.ref023\" rid=\"pone.0240726.ref023\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889742\">23</a>], marijuana [<a href=\"#pone.0240726.ref024\" rid=\"pone.0240726.ref024\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889722\">24</a>,<a href=\"#pone.0240726.ref025\" rid=\"pone.0240726.ref025\" class=\" bibr popnode\">25</a>] and gambling [<a href=\"#pone.0240726.ref026\" rid=\"pone.0240726.ref026\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889766\">26</a>,<a href=\"#pone.0240726.ref027\" rid=\"pone.0240726.ref027\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889760\">27</a>].</p><p id=\"__p7\">In the past few decades, different gaming motives questionnaires have been developed following two main approaches. From a more theoretical approach, research has attempted to identify relevant gaming motives based on well-established theories, such as <em>self-determination theory</em> [<a href=\"#pone.0240726.ref028\" rid=\"pone.0240726.ref028\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889770\">28</a>], <em>uses and gratification theory</em> [<a href=\"#pone.0240726.ref029\" rid=\"pone.0240726.ref029\" class=\" bibr popnode\">29</a>] and <em>social cognitive theory</em> [<a href=\"#pone.0240726.ref030\" rid=\"pone.0240726.ref030\" class=\" bibr popnode\">30</a>]. For instance, based on self-determination theory, Ryan, Rigby and Przybylski [<a href=\"#pone.0240726.ref031\" rid=\"pone.0240726.ref031\" class=\" bibr popnode\">31</a>] postulated that players would seek to satisfy universal psychological needs, such as autonomy, competence, relatedness, and presence. On the other hand, Sherry et al. [<a href=\"#pone.0240726.ref032\" rid=\"pone.0240726.ref032\" class=\" bibr popnode\">32</a>] focused on why people use videogames to satisfy their psychological and social needs based on uses and gratification theory. Consequently, the scale developed by the authors comprised six motivational dimensions: arousal, competition, challenge, social interaction, recreation, and fantasy. More recently, De Grove, Cauberghe and Van Looy [<a href=\"#pone.0240726.ref033\" rid=\"pone.0240726.ref033\" class=\" bibr popnode\">33</a>] developed a motives scale for digital gaming based on the social cognitive theory. The main tenet of this theory is that individuals are motivated to act as a function of outcome expectancies, and according to their study, they identified eight main motives for playing: social, narrative, escapism, agency, performance, pastime, moral self-reaction, and habit.</p><p id=\"__p8\">Following an empirical approach, a number of scales to assess videogame motives have been developed using exploratory procedures, such as factor analysis. For example, Yee [<a href=\"#pone.0240726.ref034\" rid=\"pone.0240726.ref034\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889763\">34</a>,<a href=\"#pone.0240726.ref035\" rid=\"pone.0240726.ref035\" class=\" bibr popnode\">35</a>] conducted two large studies among players of Massively Multiplayer Online Role-Playing Game (MMORPG) in order to identify their motivations. In the first study, Yee [<a href=\"#pone.0240726.ref034\" rid=\"pone.0240726.ref034\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889756\">34</a>] described a hierarchical structure of motives, with three supra-factors and ten sub-factors: achievement (advancement, mechanics, and competition), social (socializing, relationship, and teamwork), and immersion (discovery, role-playing, customization, and escapism). In the second study, Yee [<a href=\"#pone.0240726.ref035\" rid=\"pone.0240726.ref035\" class=\" bibr popnode\">35</a>] identified five main motives: relationship, immersion, escapism, achievement, and manipulation. Other researchers have extended the examination of motives in general. For instance, one of the motives scales most used for general videogame playing is the Motives for Online Gaming Questionnaire from Demetrovics et al. [<a href=\"#pone.0240726.ref036\" rid=\"pone.0240726.ref036\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889725\">36</a>]. This scale, based on a literature review and players&#x02019; interviews, posited seven global motives to play videogames: escape, coping, fantasy, skill development, recreation, competition, and social. <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/table/pone.0240726.t001/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"pone.0240726.t001\" rid-ob=\"ob-pone.0240726.t001\" co-legend-rid=\"\"><span>Table 1</span></a> summarizes most of scales developed during the past decades as well as the most common motives identified in them.</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"pone.0240726.t001\"><h3>Table 1</h3><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Comparison of the scales employed to develop the RPG scale according to the most relevant motives in video gaming research.</strong></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140210196576800\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><thead><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\">Motives</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">De Grove et al. [<a href=\"#pone.0240726.ref033\" rid=\"pone.0240726.ref033\" class=\" bibr popnode\">33</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Demetrovics et al. [<a href=\"#pone.0240726.ref036\" rid=\"pone.0240726.ref036\" class=\" bibr popnode\">36</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Ferguson &#x00026; Olson, [<a href=\"#pone.0240726.ref078\" rid=\"pone.0240726.ref078\" class=\" bibr popnode\">78</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Floros &#x00026; Siomos [<a href=\"#pone.0240726.ref085\" rid=\"pone.0240726.ref085\" class=\" bibr popnode\">85</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Fuster et al. [<a href=\"#pone.0240726.ref061\" rid=\"pone.0240726.ref061\" class=\" bibr popnode\">61</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Hilgard et al. [<a href=\"#pone.0240726.ref059\" rid=\"pone.0240726.ref059\" class=\" bibr popnode\">59</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Kahn, et al. [<a href=\"#pone.0240726.ref044\" rid=\"pone.0240726.ref044\" class=\" bibr popnode\">44</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Kim &#x00026; Ross [<a href=\"#pone.0240726.ref086\" rid=\"pone.0240726.ref086\" class=\" bibr popnode\">86</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Lafreni&#x000e8;re, Verner-Filion &#x00026; Vallerand [<a href=\"#pone.0240726.ref087\" rid=\"pone.0240726.ref087\" class=\" bibr popnode\">87</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Lee &#x00026; LaRose [<a href=\"#pone.0240726.ref088\" rid=\"pone.0240726.ref088\" class=\" bibr popnode\">88</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Li, Liau, Gentile, Khoo &#x00026; Cheong [<a href=\"#pone.0240726.ref089\" rid=\"pone.0240726.ref089\" class=\" bibr popnode\">89</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Nackle, Bateman, Mandryk [<a href=\"#pone.0240726.ref090\" rid=\"pone.0240726.ref090\" class=\" bibr popnode\">90</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Nije et al. [<a href=\"#pone.0240726.ref060\" rid=\"pone.0240726.ref060\" class=\" bibr popnode\">60</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Olson [<a href=\"#pone.0240726.ref091\" rid=\"pone.0240726.ref091\" class=\" bibr popnode\">91</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Rodr&#x000ed;guez de Sep&#x000fa;lveda &#x00026; Igartua [<a href=\"#pone.0240726.ref092\" rid=\"pone.0240726.ref092\" class=\" bibr popnode\">92</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Ryan et al. [<a href=\"#pone.0240726.ref031\" rid=\"pone.0240726.ref031\" class=\" bibr popnode\">31</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Scharkow et al. [<a href=\"#pone.0240726.ref058\" rid=\"pone.0240726.ref058\" class=\" bibr popnode\">58</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Sherry et al. [<a href=\"#pone.0240726.ref032\" rid=\"pone.0240726.ref032\" class=\" bibr popnode\">32</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Tondello et al. [<a href=\"#pone.0240726.ref093\" rid=\"pone.0240726.ref093\" class=\" bibr popnode\">93</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Wallenius, et al. [<a href=\"#pone.0240726.ref094\" rid=\"pone.0240726.ref094\" class=\" bibr popnode\">94</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Westwood &#x00026; Griffiths [<a href=\"#pone.0240726.ref095\" rid=\"pone.0240726.ref095\" class=\" bibr popnode\">95</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Wu, Wang &#x00026; Tsai, [<a href=\"#pone.0240726.ref096\" rid=\"pone.0240726.ref096\" class=\" bibr popnode\">96</a>]</th><th align=\"center\" rowspan=\"1\" colspan=\"1\">Yee [<a href=\"#pone.0240726.ref034\" rid=\"pone.0240726.ref034\" class=\" bibr popnode\">34</a>,<a href=\"#pone.0240726.ref035\" rid=\"pone.0240726.ref035\" class=\" bibr popnode\">35</a>] &#x00026; Yee et al. [<a href=\"#pone.0240726.ref045\" rid=\"pone.0240726.ref045\" class=\" bibr popnode\">45</a>]</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Social interaction</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Fantasy</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Achievement-challenge</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Competition</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Recreation</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Escape-coping</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Arousal</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Customization</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Skill development</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">X</td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td></tr></tbody></table></span><span id=\"largeobj_idm140210196576800\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/table/pone.0240726.t001/?report=objectonly\">Open in a separate window</a></span></span><p id=\"__p9\">One of the videogame genres in which motives have been more extensively studied is MMORPGs, mainly because of the high IGD prevalence reported in this genre [<a href=\"#pone.0240726.ref037\" rid=\"pone.0240726.ref037\" class=\" bibr popnode\">37</a>&#x02013;<a href=\"#pone.0240726.ref039\" rid=\"pone.0240726.ref039\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889798\">39</a>]. Research examining MMORPGs has reported associations between gaming motives and disordered gaming (e.g., [<a href=\"#pone.0240726.ref034\" rid=\"pone.0240726.ref034\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889780\">34</a>,<a href=\"#pone.0240726.ref040\" rid=\"pone.0240726.ref040\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889732\">40</a>,<a href=\"#pone.0240726.ref041\" rid=\"pone.0240726.ref041\" class=\" bibr popnode\">41</a>]), time spent gaming [<a href=\"#pone.0240726.ref035\" rid=\"pone.0240726.ref035\" class=\" bibr popnode\">35</a>,<a href=\"#pone.0240726.ref042\" rid=\"pone.0240726.ref042\" class=\" bibr popnode\">42</a>], types of positive and negative passion (or engagement) in gaming [<a href=\"#pone.0240726.ref043\" rid=\"pone.0240726.ref043\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889746\">43</a>], and in-game behaviors [<a href=\"#pone.0240726.ref041\" rid=\"pone.0240726.ref041\" class=\" bibr popnode\">41</a>,<a href=\"#pone.0240726.ref044\" rid=\"pone.0240726.ref044\" class=\" bibr popnode\">44</a>,<a href=\"#pone.0240726.ref045\" rid=\"pone.0240726.ref045\" class=\" bibr popnode\">45</a>]. Motives have also been related to different uses of other Massively Multiplayer Online games such as Massively Online Battle Arena (MOBA) games, and First-Person Shooter games [<a href=\"#pone.0240726.ref046\" rid=\"pone.0240726.ref046\" class=\" bibr popnode\">46</a>,<a href=\"#pone.0240726.ref047\" rid=\"pone.0240726.ref047\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889793\">47</a>]. Similarly, gaming motivations have been associated with disordered use in general gaming [<a href=\"#pone.0240726.ref047\" rid=\"pone.0240726.ref047\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889762\">47</a>&#x02013;<a href=\"#pone.0240726.ref053\" rid=\"pone.0240726.ref053\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889781\">53</a>].</p><p id=\"__p10\">Other studies have demonstrated that some gaming motives mediate the relationships between psychiatric problems and disordered gaming (e.g., [<a href=\"#pone.0240726.ref052\" rid=\"pone.0240726.ref052\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889757\">52</a>,<a href=\"#pone.0240726.ref054\" rid=\"pone.0240726.ref054\" class=\" bibr popnode\">54</a>&#x02013;<a href=\"#pone.0240726.ref056\" rid=\"pone.0240726.ref056\" class=\" bibr popnode\">56</a>]). Additionally, motives have been associated with higher video gaming frequency [<a href=\"#pone.0240726.ref032\" rid=\"pone.0240726.ref032\" class=\" bibr popnode\">32</a>,<a href=\"#pone.0240726.ref049\" rid=\"pone.0240726.ref049\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889735\">49</a>,<a href=\"#pone.0240726.ref053\" rid=\"pone.0240726.ref053\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889767\">53</a>,<a href=\"#pone.0240726.ref057\" rid=\"pone.0240726.ref057\" class=\" bibr popnode\">57</a>], genre preferences [<a href=\"#pone.0240726.ref047\" rid=\"pone.0240726.ref047\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889768\">47</a>,<a href=\"#pone.0240726.ref057\" rid=\"pone.0240726.ref057\" class=\" bibr popnode\">57</a>,<a href=\"#pone.0240726.ref058\" rid=\"pone.0240726.ref058\" class=\" bibr popnode\">58</a>] and favorite game franchises [<a href=\"#pone.0240726.ref059\" rid=\"pone.0240726.ref059\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889784\">59</a>].</p><p id=\"__p11\">Taken together, these studies demonstrate the importance of motives in the psychological study of gaming behaviors, and the need to properly identify and assess them. However, the research to date presents some limitations. As shown, utilizing different theoretical background results in a different number and nature of motives. Furthermore, some of the described motives suffer from a form of &#x02018;jingle-jangle&#x02019; fallacies, that is, that apparently similar constructs are sometimes given different labels (jangle fallacy) whereas the same labels are sometimes applied to conceptually different constructs (jingle fallacy). For instance, in relation to the fantasy component of the game, some scales highlight different aspects within the same motive, such as discovery and role-playing [<a href=\"#pone.0240726.ref034\" rid=\"pone.0240726.ref034\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889728\">34</a>], fantasy, exploration and narration [<a href=\"#pone.0240726.ref058\" rid=\"pone.0240726.ref058\" class=\" bibr popnode\">58</a>], and fantasy-escape and fantasy-arousal [<a href=\"#pone.0240726.ref060\" rid=\"pone.0240726.ref060\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889758\">60</a>]. Conversely, different studies describe this component using distinct labels, such as immersion [<a href=\"#pone.0240726.ref034\" rid=\"pone.0240726.ref034\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889738\">34</a>,<a href=\"#pone.0240726.ref045\" rid=\"pone.0240726.ref045\" class=\" bibr popnode\">45</a>], presence [<a href=\"#pone.0240726.ref031\" rid=\"pone.0240726.ref031\" class=\" bibr popnode\">31</a>], fantasy [<a href=\"#pone.0240726.ref032\" rid=\"pone.0240726.ref032\" class=\" bibr popnode\">32</a>,<a href=\"#pone.0240726.ref036\" rid=\"pone.0240726.ref036\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889743\">36</a>], narrative [<a href=\"#pone.0240726.ref033\" rid=\"pone.0240726.ref033\" class=\" bibr popnode\">33</a>], story-driven [<a href=\"#pone.0240726.ref045\" rid=\"pone.0240726.ref045\" class=\" bibr popnode\">45</a>], story [<a href=\"#pone.0240726.ref059\" rid=\"pone.0240726.ref059\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889748\">59</a>], and exploration [<a href=\"#pone.0240726.ref061\" rid=\"pone.0240726.ref061\" class=\" bibr popnode\">61</a>]. Finally, many studies have focused on specific games or genres, especially MMORPGs; e.g., [<a href=\"#pone.0240726.ref034\" rid=\"pone.0240726.ref034\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889790\">34</a>,<a href=\"#pone.0240726.ref045\" rid=\"pone.0240726.ref045\" class=\" bibr popnode\">45</a>,<a href=\"#pone.0240726.ref061\" rid=\"pone.0240726.ref061\" class=\" bibr popnode\">61</a>], so it is unclear to what extent the proposed motives are specific to this genre, or common to all genres. This variability across the different gaming scales impedes the development of generalizable taxonomies of video game motives and hinder accurate communication in this field.</p><p id=\"__p12\" class=\"p p-last\">Therefore, and following an empirical approach, the main aim of the present study was to develop a new scale of gaming motives in which the most relevant and recurrent motivational components found in previous studies were reflected under unified labels that can be used for any gaming genre. To do so, three studies were carried out. The first study identified some of the most relevant motives through a literature-based scoping study and comprised an initial exploratory factor analysis (EFA) on items reflecting these motives in data collected from a sample of young adults. In Study 2, an EFA of selected items encompassing the videogame motives more clearly identified in Study 1 was conducted using data from an adolescent sample. Finally, in Study 3, a confirmatory factor analysis (CFA) was performed using data from a sample of young adults. The role of the identified motives was explored in relation to the number of hours spent gaming, disordered gaming, and game genre preferences in Studies 2 and 3.</p></span><span id=\"s\r\n002\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n002title\">Study 1. Development of pilot scale and initial testing</h2><span id=\"s\r\n003\" class=\"sec sec-first\"><h3 id=\"s\r\n003title\">Method</h3><span id=\"s\r\n004\" class=\"sec sec-first\"><p></p><h4 id=\"s\r\n004title\" class=\"inline\">Participants and procedure </h4><p id=\"__p13\" class=\"p p-first-last\">For the development of a pilot scale and the initial selection of items, a convenience sample of 140 young adult players was used (76 undergraduates and 64 participants recruited from the LAN party, &#x0201c;UJI Game Experience&#x0201d;, an event at the Jaume I University in which gamers gathered to play multiplayer video games; 79 males; mean age = 20.71 years [<em>SD</em> = 3.55]). Participants voluntarily completed an online survey without any compensation via <em>Google Forms</em> comprising the pilot motive scale together with sociodemographic data.</p></span><span id=\"s\r\n005\" class=\"sec\"><p></p><h4 id=\"s\r\n005title\" class=\"inline\">Measures </h4><p id=\"__p14\" class=\"p p-first-last\"><em>Pilot Videogaming Motives Questionnaire (VMQ)</em>. Through an extensive review of the literature concerning gaming motivation scales, 25 relevant gaming motive scales were selected. For the review, a search was conducted in the PsycINFO and Google Scholar databases from 2000 to present, combining the key words motives or motivation, video game or gaming, and scale or questionnaire. Furthermore, citations from selected studies of our database were used to continue the review process until no other relevant study was found. The nine most frequently found motives in the literature were: arousal, recreation, social interaction, escape-coping, fantasy, skill development, customization, achievement-challenge, and competition. Two other motives&#x02013;violence catharsis and violent reward&#x02013;from Hilgard et al.&#x02019;s scale [<a href=\"#pone.0240726.ref059\" rid=\"pone.0240726.ref059\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889747\">59</a>] were also initially chosen given the potential relevance of violence and aggression in gaming [<a href=\"#pone.0240726.ref062\" rid=\"pone.0240726.ref062\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889794\">62</a>]. Last, we also took into account the common motives for the consumption of different addictive substances, such as alcohol, tobacco and marijuana, i.e. enhancement/recreation, social, coping and conformity, so this last motive was also included [<a href=\"#pone.0240726.ref063\" rid=\"pone.0240726.ref063\" class=\" bibr popnode\">63</a>]. A total of 62 items based on these motives were selected, with five items per motive except violent catharsis and social interaction (six items each) and adapted to Spanish from the original scales utilizing an internationally standardized back-translation process [<a href=\"#pone.0240726.ref064\" rid=\"pone.0240726.ref064\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889802\">64</a>,<a href=\"#pone.0240726.ref065\" rid=\"pone.0240726.ref065\" class=\" bibr popnode\">65</a>]. Participants had to answer each item using a five-point Likert scale, from 1 (<em>strongly disagree</em>) to 5 (<em>strongly agree</em>).</p></span><span id=\"s\r\n006\" class=\"sec\"><p></p><h4 id=\"s\r\n006title\" class=\"inline\">Statistical analysis </h4><p id=\"__p15\" class=\"p p-first-last\">The IBM SPSS Statistics V21.0 software was employed for the EFA, with principal axis factoring and oblimin rotation on the 62 items selected. For the parallel analyses, Monte Carlo PCA [<a href=\"#pone.0240726.ref066\" rid=\"pone.0240726.ref066\" class=\" bibr popnode\">66</a>] was utilized. Although the sample size was below the expected norms in relation to the number of items factorized [<a href=\"#pone.0240726.ref067\" rid=\"pone.0240726.ref067\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889771\">67</a>], the pilot study was only exploratory, and its main aim was to identify the clearer and more robust factors underlying the analyzed items.</p></span><span id=\"s\r\n007\" class=\"sec sec-last\"><p></p><h4 id=\"s\r\n007title\" class=\"inline\">Ethics </h4><p id=\"__p16\" class=\"p p-first-last\">Participants were adequately informed about the study, confidentiality and treatment of data, as well as the data protection procedure; and gave written informed consents. This study was approved by the ethical committee from the Universitat Jaume I and was carried out in accordance with the Declaration of Helsinki.</p></span></span><span id=\"s\r\n008\" class=\"sec sec-last\"><h3 id=\"s\r\n008title\">Results</h3><p id=\"__p17\" class=\"p p-first-last\">The eigenvalue criteria [<a href=\"#pone.0240726.ref068\" rid=\"pone.0240726.ref068\" class=\" bibr popnode\">68</a>] suggested 12 factors whereas parallel analyses [<a href=\"#pone.0240726.ref067\" rid=\"pone.0240726.ref067\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889791\">67</a>] suggested seven factors. Consequently, six EFAs were performed extracting from 7 to 12 factors. Across different EFAs, eight intercorrelated factors became more robust and well-defined (see <a href=\"#pone.0240726.s001\" rid=\"pone.0240726.s001\" class=\" supplementary-material\">S1 Table</a>) explaining 78.7% of the variance: recreation, social interaction, coping-escape, violent reward, fantasy, customization, competition, and skill development. The other four motives were not well identified in the 12-factor solution: achievement-challenge, conformity, and violent catharsis were formed by only two items with high loadings; whereas the arousal items were distributed along different factors. In the eight-factor solution, the arousal items loaded principally on the recreation and fantasy motivations; violent catharsis items loaded on the escape-coping and violent reward dimension; conformity items loaded mainly on the social interaction dimension; and achievement-challenge items were distributed between different motives.</p></span></span><span id=\"s\r\n009\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n009title\">Study 2. Adolescent sample study</h2><span id=\"s\r\n010\" class=\"sec sec-first\"><h3 id=\"s\r\n010title\">Method</h3><span id=\"s\r\n011\" class=\"sec sec-first\"><p></p><h4 id=\"s\r\n011title\" class=\"inline\">Participants and procedures </h4><p id=\"__p18\" class=\"p p-first\">Adolescent participants were recruited from two Spanish high schools utilizing convenience sampling. Of the 1,106 students invited to participate, 835 adolescents returned signed written parental consent and received school supplies as an incentive. From these, 407 reported a frequency of gaming of at least one hour weekly and completed the VMQ. Approximately two-thirds of the sample participants were male (68.2%) with a mean age of 14.99 years (<em>SD</em> = 1.13).</p><p id=\"__p19\" class=\"p p-last\">This study was part of broader research into psychosocial risk and protective factors affecting mental health (see Moya-Higueras et al. [<a href=\"#pone.0240726.ref069\" rid=\"pone.0240726.ref069\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889733\">69</a>] for more details). Trained research assistants administered the battery of questionnaires in two sessions, separated by approximately one week. A convenience sub-sample of 153 participants (75.2% boys; mean age = 15.08, <em>SD</em> = 1.09) completed a third session one month later, in order to explore test-retest reliability.</p></span><span id=\"s\r\n012\" class=\"sec\"><p></p><h4 id=\"s\r\n012title\" class=\"inline\">Measures </h4><p id=\"__p20\" class=\"p p-first\"><em>Videogaming Motives Questionnaire (VMQ)</em>. In order to develop a short instrument that includes scales with good reliability and adequate content validity, for each of the eight motives, three non-redundant items with high loadings and intercorrelations were selected and that adequately represented the content of the motive. Therefore, the 24-item definitive version of VMQ was used to assess eight gaming motivations: recreation, social interaction, coping, violent reward, fantasy, cognitive challenge, customization and competition using a five-point Likert scale (0 = strongly disagree; 4 = strongly agree). In relation to the coping-escape motive, the dimension became coping only because the three most representative items were exclusively related to coping aspects. Finally, the skill development motive became cognitive development because its most relevant items concerned mental challenge and development rather than improvement of abilities in general.</p><p id=\"__p21\"><em>Gaming behaviors</em>. Data were collected on the number of hours spent gaming daily online and offline per week (from Monday to Friday) and on weekends (Saturday and Sunday). Weekly number of gaming hours was obtained by adding online and offline daily hours and multiplying the days of these periods of the week by the total number of hours. Participants were also asked to provide up to five of their most played videogames. These games were then classified into the most popular videogame genres, based on previous studies [e.g., <a href=\"#pone.0240726.ref009\" rid=\"pone.0240726.ref009\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889776\">9</a>]. These gaming genres comprised: shooter games (e.g., <em>Call of Duty</em>), MOBA (e.g., <em>League of Legends</em>), strategy games (e.g., <em>Clash of Clans</em>), MMORPG (e.g., <em>World of Warcraft</em>), role-playing games (e.g., <em>Dark Souls</em>), action-adventure games (e.g., <em>The Legend of Zelda</em>), sport games (e.g., <em>FIFA</em>), casual games (e.g., <em>Candy Crush</em>), social simulation games (e.g., <em>The Sims</em>), construction games (e.g., <em>Minecraft</em>), platform games (e.g., <em>Super Mario Bros</em>), fighting games (e.g., <em>Street Fighter</em>) and other miscellaneous games.</p><p id=\"__p22\" class=\"p p-last\"><em>Disordered gaming</em>. A Spanish adaptation [<a href=\"#pone.0240726.ref009\" rid=\"pone.0240726.ref009\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889754\">9</a>] of a measure especially developed to assess disordered gaming in adolescents [<a href=\"#pone.0240726.ref070\" rid=\"pone.0240726.ref070\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889803\">70</a>] was employed. The 11-item scale is based on symptoms of addiction, such as withdrawal, conflict, tolerance, salience, mood modification and relapse. Participants indicate their frequency of videogame-related problems on a four-point Likert scale (from 0 = <em>&#x0201c;never or almost never&#x0201d;</em> to 3 <em>&#x0201c;almost always or always&#x0201d;</em>) during the past year. The Cronbach&#x02019;s alpha in the present study was very good (.85).</p></span><span id=\"s\r\n013\" class=\"sec\"><p></p><h4 id=\"s\r\n013title\" class=\"inline\">Statistical analysis </h4><p id=\"__p23\" class=\"p p-first-last\">As the pilot study showed that motives were intercorrelated, an EFA with principal axis factoring and oblique rotation (oblimin) was performed for testing the VMQ with original eight-factor model using IBM SPSS Statistics V21.0. The Cronbach&#x02019;s alpha of each VMQ factor was calculated. Temporal stability reliability was computed utilizing a two-week test-retest correlation. In order to determine the criterion validity of the VMQ, the scores of the motives were correlated with number of weekly hours spent gaming, disordered gaming, gaming genre, age, and gender. Finally, hierarchical multiple regression analyses were carried out to examine the role of motives in hours spent gaming per week and disordered use. Previous research has shown that gender and age were relevant variables in video game&#x02013;related behaviors, such as disordered use or video game genre preferences (e.g., [<a href=\"#pone.0240726.ref009\" rid=\"pone.0240726.ref009\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889750\">9</a>,<a href=\"#pone.0240726.ref032\" rid=\"pone.0240726.ref032\" class=\" bibr popnode\">32</a>,<a href=\"#pone.0240726.ref057\" rid=\"pone.0240726.ref057\" class=\" bibr popnode\">57</a>]). Therefore, hierarchical multiple regressions were controlled for gender and age by introducing them in step 1, whereas motives were entered in step 2&#x0201d;.</p></span><span id=\"s\r\n014\" class=\"sec sec-last\"><p></p><h4 id=\"s\r\n014title\" class=\"inline\">Ethics </h4><p id=\"__p24\" class=\"p p-first-last\">Parents or legal guardians of the adolescents were informed about the study, confidentiality, treatment of data, as well as the data protection procedure; and given written informed consents. The study was approved by the ethical committee from the Universitat Jaume I, and authorized by the school board of the participating high schools as well as by the Valencian regional education authorities, and has been carried out in accordance with the Declaration of Helsinki.</p></span></span><span id=\"s\r\n015\" class=\"sec sec-last\"><h3 id=\"s\r\n015title\">Results</h3><span id=\"s\r\n016\" class=\"sec sec-first\"><p></p><h4 id=\"s\r\n016title\" class=\"inline\">Gaming behavior </h4><p id=\"__p25\" class=\"p p-first-last\">Participants reported playing an average of 1.78 hours a day during the week (<em>SD</em> = 1.86), and four hours a day at the weekends (<em>SD</em> = 3.01). Therefore, the mean weekly time spent gaming was 16.89 hours (SD = 13.45). The most played videogames reported by participants (n = 359) were classified according to gaming genre: shooter (n = 197), sports (n = 150), MOBA (n = 71), strategy (n = 64), action-adventure (n = 45), social simulation (n = 35), casual (n = 33), role-playing (n = 23), platform (n = 23), construction (n = 21), fighting (n = 20), MMORPGs (4) and other miscellaneous games (n = 58).</p></span><span id=\"s\r\n017\" class=\"sec\"><p></p><h4 id=\"s\r\n017title\" class=\"inline\">Factor structure </h4><p id=\"__p26\" class=\"p p-first\">The EFA in the adolescent sample replicated the eight-factor solution found in Study 1 and explained 78.5% of the variance (see <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/table/pone.0240726.t002/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"pone.0240726.t002\" rid-ob=\"ob-pone.0240726.t002\" co-legend-rid=\"\"><span>Table 2</span></a>). The factor loadings for every item were greater than .50 except for Item 19 that had a factor loading of .36. Furthermore, the internal consistency of each motive was good to excellent with Cronbach alphas ranging from .76 to .93. The test-retest scores were acceptable regarding the low number of items per dimension, with all the values higher than .60 [<a href=\"#pone.0240726.ref071\" rid=\"pone.0240726.ref071\" class=\" bibr popnode\">71</a>].</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"pone.0240726.t002\"><h3>Table 2</h3><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Exploratory factor analysis and reliability of the VMQ.</strong></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140210203369200\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><thead><tr><th align=\"left\" rowspan=\"1\" colspan=\"1\">Item</th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Factor Loading</th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Item</th><th align=\"left\" rowspan=\"1\" colspan=\"1\">Factor Loading</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"2\" colspan=\"1\"><strong>Recreation</strong></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Alpha = .84</td><td align=\"left\" rowspan=\"2\" colspan=\"1\"><strong>Social interaction</strong></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Alpha = .79</td></tr><tr><td align=\"center\" rowspan=\"1\" colspan=\"1\">Retest = .60**</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Retest = .78**</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">1. Disfruto jugando. (<em>I enjoy gaming</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.82</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">5. Hago nuevos amigos. (<em>I make new friends</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.75</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">9.Me lo paso bien. (<em>I have fun</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.78</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">13. Mediante el juego estoy en contacto con mis amigos. (<em>I keep in touch with my friends by gaming</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.66</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">17. Es divertido. (<em>It is entertaining</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.73</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">21. As&#x000ed; encajo en un grupo de gente que me gusta. (<em>Thus</em>, <em>I fit in with a group I like</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.53</td></tr><tr><td align=\"left\" rowspan=\"2\" colspan=\"1\"><strong>Competition</strong></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Alpha = .76</td><td align=\"left\" rowspan=\"2\" colspan=\"1\"><strong>Violent reward</strong></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Alpha = .93</td></tr><tr><td align=\"center\" rowspan=\"1\" colspan=\"1\">Retest = .67**</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Retest = .82**</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">2. Me gusta ganar. (<em>I like to win</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.55</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">6. Me gusta la violencia en el juego, cuanto m&#x000e1;s mejor. (<em>I like violence in video games</em>, <em>the more violent the better</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.93</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">10. Me gusta demostrar que soy mejor que otros jugadores. (<em>I like to prove that I am better than other players</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.55</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">14. En el juego es divertido disparar a alguien en la cabeza. (<em>Shooting someone in the head in a game is deeply satisfying</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.91</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">18. Disfruto compitiendo con otros. (<em>I enjoy competing with others</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.50</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">22. Disfruto de las peleas y luchas violentas en el juego. (<em>I enjoy the violent fights in video games</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.85</td></tr><tr><td align=\"left\" rowspan=\"2\" colspan=\"1\"><strong>Cognitive development</strong></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Alpha = .81</td><td align=\"left\" rowspan=\"2\" colspan=\"1\"><strong>Customization</strong></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Alpha = .88</td></tr><tr><td align=\"center\" rowspan=\"1\" colspan=\"1\">Retest = .69**</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Retest = .68**</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">3. Me hacen pensar/calentarme la cabeza. (<em>Games make me think</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.88</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">7. Disfruto dise&#x000f1;ando cosas en el juego. (<em>I enjoy customizing things in games</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.95</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">11. Me suponen un reto mental. (<em>Games imply a mental challenge</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.66</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">15. Me gusta crear cosas en el juego, como casas u otras construcciones. (<em>I like making things in video games</em>, <em>like houses or other constructions</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.85</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">19. Me hacen m&#x000e1;s inteligente (<em>Games make me smarter</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.36</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">23. Me gusta crear mi propio mundo en el juego. (<em>I like to create my own world in games</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.64</td></tr><tr><td align=\"left\" rowspan=\"2\" colspan=\"1\"><strong>Coping</strong></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Alpha = .87</td><td align=\"left\" rowspan=\"2\" colspan=\"1\"><strong>Fantasy</strong></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Alpha = .82</td></tr><tr><td align=\"center\" rowspan=\"1\" colspan=\"1\">Retest = .69**</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Retest = .68**</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">4. Alivia mi estr&#x000e9;s. (<em>It helps me get rid of stress</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.81</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">8. Disfruto meti&#x000e9;ndome en la piel de un nuevo personaje en cada juego. (<em>I enjoy putting myself into a new character&#x02019;s shoes in each game</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.66</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">12. Me ayuda a mejorar mi estado de &#x000e1;nimo. (<em>Gaming helps me improve my mood</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.76</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">16. Me gusta sentirme parte de una historia. (<em>I like feeling that I&#x02019;m part of a story</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.62</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">20. Me permite sentirme mejor cuando estoy frustrado. (<em>Gaming allows me to feel better when I am frustrated</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.71</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">24. Me siento inmerso en un mundo fant&#x000e1;stico/ficticio. (<em>I feel immersed in a fantastic/fictitious world</em>).</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.59</td></tr></tbody></table></span><span id=\"largeobj_idm140210203369200\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/table/pone.0240726.t002/?report=objectonly\">Open in a separate window</a></span></span></span><span id=\"s\r\n018\" class=\"sec sec-last\"><p></p><h4 id=\"s\r\n018title\" class=\"inline\">Criterion validity </h4><p id=\"__p27\" class=\"p p-first\">Results showed that VMQ motives had significant correlations with various gaming-related indicators (see <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/table/pone.0240726.t003/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"pone.0240726.t003\" rid-ob=\"ob-pone.0240726.t003\" co-legend-rid=\"\"><span>Table 3</span></a>). All motives were associated with the number of hours spent gaming weekly and disordered use, although the motives that presented higher associations were social interaction, coping, and competition. Significant relationships were also found among gaming genres. For instance, according to those genres most consumed, shooters were mainly associated with violent reward, social interaction and competition, whereas playing MOBA games and sport games were strongly related to competition. All the motives were associated with male gender except customization. The largest associations among males were observed with competition, violent reward, and social interaction. Descriptive data by gender and correlations between motives are displayed in <a href=\"#pone.0240726.s002\" rid=\"pone.0240726.s002\" class=\" supplementary-material\">S2</a> and <a href=\"#pone.0240726.s003\" rid=\"pone.0240726.s003\" class=\" supplementary-material\">S3</a> Tables.</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"pone.0240726.t003\"><h3>Table 3</h3><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Correlation analysis between motivations of the VMQ and other relevant variables for Studies 2 and 3.</strong></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140210192796320\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"><strong>Study 2 Adolescent sample</strong></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Age</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Gender</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Gaming hours</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Disordered gaming</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Shooter (197)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">MOBA (71)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Strategy (64)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">MMORPG (3)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Role-playing (23)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Action-adventure (45)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Sport (150)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Casual (33)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Social simulation (35)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Construction (21)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Platform (23)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Fighting (20)</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Recreation</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.00</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.17<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.23<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.18<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.23<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.00</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.08</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.05</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.15<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.07</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.04</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.06</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.02</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.10</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.10</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.06</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Competition</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.46<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.32<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.39<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.32<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.20<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.02</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.06</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.02</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.25<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.12<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.26<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.04</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.18<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.07</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Cognitive development</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.05</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.20<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.24<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.37<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.21<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.12<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.06</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.06</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.07</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.19<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.15<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.08</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.03</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Coping</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.00</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.18<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.34<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.48<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.20<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.05</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.04</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.10</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.14<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.10</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.05</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.15<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.04</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.11<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.08</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Social interaction</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.02</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.34<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.37<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.51<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.40<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.15<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.13<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.09</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.02</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.06</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.20<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.17<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.02</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Violent reward</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.01</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.42<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.28<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.36<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.47<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.09</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.15<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.00</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.09</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.02</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.15<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.19<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.25<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.00</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.22<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.10</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Customization</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.01</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.06</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.19<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.18<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.08</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.04</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.04</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.08</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.09</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.12<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.12<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.07</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.27<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.18<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.04</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.01</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Fantasy</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.05</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.15<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.26<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.31<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">20<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.06</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.05</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.14<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.18<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.22<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.01</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.04</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.12<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.07</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.01</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"><strong>Study 3 Young adult sample</strong></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Age</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Gender</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Gaming hours</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Disordered gaming</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Shooter (77)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">MOBA (74)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Strategy (46)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">MMORPG (11)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Role-playing (61)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Action-adventure (25)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Sport (33)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Casual (51)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Social simulation (23)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Construction (8)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Platform (13)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Fighting (10)</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Recreation</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.14<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.30<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.29<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.29<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.26<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.19<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.05</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.15<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.23<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.20<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.12</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.18<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.13<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.02</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.04</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.07</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Competition</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.00</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.36<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.15<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.28<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.12</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.18<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.13<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.04</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.19<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.26<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.12</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.02</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.08</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.10</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Cognitive development</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.19<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.18<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.12<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.21<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.13<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.06</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.04</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.06</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.17<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.01</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.08</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.02</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.08</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Coping</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.15<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.16<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.24<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.39<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.18<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.06</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.10</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.18<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.11</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.22<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.11</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.11</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.02</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.00</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.07</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.04</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Social interaction</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.02</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.40<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.42<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.41<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.30<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.43<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.05</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.26<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.17<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.04</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.02</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.31<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.16<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.05</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.09</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.12</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Violent reward</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.06</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.46<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.21<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.43<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.37<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.11</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.14<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.08</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.14<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.24<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.18<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.34<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.11</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.09</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.13<sup>*</sup></td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Customization</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.06</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.09</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.15<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.10</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.13<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.04</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.04</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.08</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.20<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.07</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.04</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.16<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.35<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.12</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.05</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.02</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Fantasy</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.04</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.17<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.24<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.34<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.25<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.07</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.09</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.18<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.30<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.32<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.06</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.29<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.13<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.01</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.11</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.09</td></tr></tbody></table></span><span id=\"largeobj_idm140210192796320\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/table/pone.0240726.t003/?report=objectonly\">Open in a separate window</a></span><span class=\"tblwrap-foot\"><span id=\"t003fn001\"><p id=\"__p28\" class=\"p p-first-last\"><em>Note</em>.</p></span><span id=\"t003fn002\"><p id=\"__p29\" class=\"p p-first-last\"><em>*p &#x0003c;</em> .<em>05</em></p></span><span id=\"t003fn003\"><p id=\"__p30\" class=\"p p-first-last\"><em>**p &#x0003c;</em> .<em>01</em>.</p></span><span id=\"t003fn004\"><p id=\"__p31\" class=\"p p-first-last\">Gender: 1 = males, 2 = females. The number of players by genre are indicated between parentheses.</p></span></span></span><p id=\"__p32\" class=\"p\">Multiple linear regression analyses were conducted to identify the role of motives in number of hours spent gaming weekly and disordered use, controlling for gender and age (see <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/table/pone.0240726.t004/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"pone.0240726.t004\" rid-ob=\"ob-pone.0240726.t004\" co-legend-rid=\"\"><span>Table 4</span></a>). Higher social interaction and coping predicted greater gaming use, whereas coping, social interaction and, to a lesser extent, violent reward predicted disordered videogame use.</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"pone.0240726.t004\"><h3>Table 4</h3><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Multiple linear regressions of hours spent gaming and disordered gaming for Studies 2 and 3.</strong></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140210198760624\" class=\"xtable\"><table frame=\"hsides\" rules=\"groups\" class=\"rendered small default_table\"><thead><tr><th align=\"left\" rowspan=\"2\" colspan=\"1\"></th><th align=\"center\" rowspan=\"2\" colspan=\"1\"></th><th align=\"center\" colspan=\"2\" rowspan=\"1\">Study 2</th><th align=\"center\" colspan=\"2\" rowspan=\"1\">Study 3</th></tr><tr><th align=\"center\" colspan=\"2\" rowspan=\"1\">Adolescent sample</th><th align=\"center\" colspan=\"2\" rowspan=\"1\">Young adult sample</th></tr></thead><tbody><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\"></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Gaming hours</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Disordered gaming</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Gaming hours</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">Disordered gaming</td></tr><tr><td align=\"center\" rowspan=\"2\" colspan=\"1\">1</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Gender</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.28<sup>***</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.26<sup>***</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.32<sup>***</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.39<sup>**</sup></td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Age</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.02</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.04</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.07</td></tr><tr><td align=\"right\" rowspan=\"1\" colspan=\"1\"></td><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x00394;<em>R</em><sup>2</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.08<sup>***</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.07<sup>***</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.10<sup>***</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.15<sup>***</sup></td></tr><tr><td align=\"left\" rowspan=\"8\" colspan=\"1\">2</td><td align=\"left\" rowspan=\"1\" colspan=\"1\">Recreation</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.05</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.08</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.13</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.00</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Competition</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.02</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.02</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.11</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.01</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Cognitive development</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.06</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.04</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.08</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.03</td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Coping</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.18<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.30<sup>***</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.05</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.21<sup>**</sup></td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Social interaction</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.19<sup>**</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.29<sup>***</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.33<sup>***</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.19<sup>**</sup></td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Violent reward</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.06</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.12<sup>*</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.01</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.18<sup>*</sup></td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Customization</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.09</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.00</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.15<sup>*</sup></td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\">Fantasy</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.02</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">-.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.03</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.17<sup>*</sup></td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"></td><td align=\"left\" rowspan=\"1\" colspan=\"1\">&#x00394;<em>R</em><sup>2</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.13<sup>***</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.27<sup>***</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.13<sup>***</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.18<sup>***</sup></td></tr><tr><td align=\"left\" rowspan=\"1\" colspan=\"1\"></td><td align=\"left\" rowspan=\"1\" colspan=\"1\"><em>R</em><sup>2</sup></td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.21</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.34</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.23</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">.33</td></tr></tbody></table></span><span id=\"largeobj_idm140210198760624\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/table/pone.0240726.t004/?report=objectonly\">Open in a separate window</a></span><span class=\"tblwrap-foot\"><span id=\"t004fn001\"><p id=\"__p33\" class=\"p p-first-last\"><em>Note</em>. 1 = males, 2 = females. &#x003b2; = standardized beta; &#x00394;R<sup>2</sup> = change in variance; R<sup>2</sup> = total R<sup>2</sup>.</p></span><span id=\"t004fn002\"><p id=\"__p34\" class=\"p p-first-last\">*<em>p &#x0003c;</em> .05.</p></span><span id=\"t004fn003\"><p id=\"__p35\" class=\"p p-first-last\">**<em>p &#x0003c;</em> .01.</p></span><span id=\"t004fn004\"><p id=\"__p36\" class=\"p p-first-last\">***<em>p &#x0003c;</em> .001.</p></span></span></span></span></span></span><span id=\"s\r\n019\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n019title\">Study 3. Young adult sample study</h2><span id=\"s\r\n020\" class=\"sec sec-first\"><h3 id=\"s\r\n020title\">Method</h3><span id=\"s\r\n021\" class=\"sec sec-first\"><p></p><h4 id=\"s\r\n021title\" class=\"inline\">Participants and procedure </h4><p id=\"__p37\" class=\"p p-first-last\">The convenience young adult sample that participated in the survey was comprised of 260 undergraduate participants who played at least one hour a week (41.9% males) with a mean age of 20.53 years (<em>SD</em> = 3.63). The survey was selflessly completed online via <em>Google Forms</em>.</p></span><span id=\"s\r\n022\" class=\"sec\"><p></p><h4 id=\"s\r\n022title\" class=\"inline\">Measures </h4><p id=\"__p38\" class=\"p p-first\"><em>Videogaming Motives Questionnaire (VMQ)</em>. The VMQ administered to adolescents was used in order to examine its psychometric properties as well as to confirm the eight-factor solution in the young adult sample. Please see Study 2 Methods section for details about the VMQ item structure.</p><p id=\"__p39\"><em>Gaming behaviors</em>. As in the adolescent sample (Study 2), participants reported the number of weekly hours spent gaming, online and offline, and provided up to five of their most played videogames. Videogame genres were classified in the same way as in Study 2.</p><p id=\"__p40\" class=\"p p-last\"><em>Internet Gaming Disorder Test (IGD-20 Test)-Spanish versi&#x000f3;n</em>. The Spanish version of the IGD-20 test [<a href=\"#pone.0240726.ref072\" rid=\"pone.0240726.ref072\" class=\" bibr popnode\">72</a>] was used to assess disordered gaming in young adults. The scale includes 20 items answered using a 5-point Likert scale from 1 (<em>strongly disagree</em>) to 5 (<em>strongly agree</em>). The IGD-20 Test assesses disordered gaming activity during the past 12 months using the nine criteria for IGD. The Cronbach&#x02019;s alpha was very good in the present study (.86).</p></span><span id=\"s\r\n023\" class=\"sec\"><p></p><h4 id=\"s\r\n023title\" class=\"inline\">Statistical analysis </h4><p id=\"__p41\" class=\"p p-first\">A CFA was performed on the VMQ testing the eight-factor model with robust methods using the EQS software, version 6.1 [<a href=\"#pone.0240726.ref073\" rid=\"pone.0240726.ref073\" class=\" bibr popnode\">73</a>]. The model&#x02019;s goodness-of-fit was assessed by employing the following fit indices: Satorra-Bentler chi-squared (S&#x02212;B&#x003c7;<sup>2</sup>), Satorra-Bentler normed chi-squared (S&#x02212;B&#x003c7;<sup>2</sup>/d.f.), the comparative fit index (CFI), the incremental fit index (IFI), the non-normed fit index (NNFI), and the root mean square error of approximation (RMSEA). For a good model fit, the fit indices must meet the following values: S&#x02212;B&#x003c7;<sup>2</sup> (non-significant), S&#x02212;B&#x003c7;<sup>2</sup>/df (between 1 and 2), CFI, IFI, and NNFI (&#x02265; .95), and RMSEA &#x02264; .05 [<a href=\"#pone.0240726.ref074\" rid=\"pone.0240726.ref074\" class=\" bibr popnode\">74</a>]. Last, as the &#x003c7;<sup>2</sup> is very sensitive to the model complexity or sample size, we used the S&#x02212;B&#x003c7;<sup>2</sup>/df to overcome this problem [<a href=\"#pone.0240726.ref074\" rid=\"pone.0240726.ref074\" class=\" bibr popnode\">74</a>].</p><p id=\"__p42\" class=\"p p-last\">As with the adolescent sample (Study 2), the correlations between the VMQ subscales and the number of weekly hours spent gaming, disordered gaming, and gaming genre were calculated. Finally, in order to test the role of gaming motives in the number of hours spent gaming weekly and disordered gaming, hierarchical multiple regression analyses were conducted, controlling for age and gender.</p></span><span id=\"s\r\n024\" class=\"sec sec-last\"><p></p><h4 id=\"s\r\n024title\" class=\"inline\">Ethics </h4><p id=\"__p43\" class=\"p p-first-last\">As in Study 1, participants provided written informed consent and were informed about the study, confidentiality and treatment of data, as well as the data protection procedure. The study was approved by the ethical committee from the Universitat Jaume I and was conducted in accordance with the Declaration of Helsinki.</p></span></span><span id=\"s\r\n025\" class=\"sec sec-last\"><h3 id=\"s\r\n025title\">Results</h3><span id=\"s\r\n026\" class=\"sec sec-first\"><p></p><h4 id=\"s\r\n026title\" class=\"inline\">Gaming behavior </h4><p id=\"__p44\" class=\"p p-first-last\">The young adult sample played an average of 9.26 hours a week (<em>SD</em> = 11.97). The videogames played reported by participants (n = 250) were classified according to gaming genre: shooter (n = 89), MOBA (80), casual (n = 71), role-playing (n = 70), strategy (n = 57), sports (n = 40), social simulation (n = 33), action-adventure (n = 28), platform (n = 17), MMORPGs (n = 11), fighting (n = 11), construction (n = 9) and other miscellaneous games (n = 33).</p></span><span id=\"s\r\n027\" class=\"sec\"><p></p><h4 id=\"s\r\n027title\" class=\"inline\">Factor structure </h4><p id=\"__p45\" class=\"p p-first\">Results from the CFA demonstrated a good model fit (S&#x02212;B&#x003c7;<sup>2</sup> = 330.13, <em>df =</em> 224, <em>p &#x0003c;</em> .001; S&#x02212;B&#x003c7;<sup>2</sup>/df = 1.47; CFI = .97; IFI = .97; NNFI = .96; RMSEA = .04), although the chi-square test did not acquire non-significance. This result from the chi-square test was expected due to the index&#x02019;s sensitivity to sample size. The loadings ranged from .62 to .95 (see <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/figure/pone.0240726.g001/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"pone.0240726.g001\" rid-ob=\"ob-pone.0240726.g001\" co-legend-rid=\"lgnd_pone.0240726.g001\"><span>Fig 1</span></a>); and 81.9% of the variance was explained by the 8-factor model.</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"pone.0240726.g001\" co-legend-rid=\"lgnd_pone.0240726.g001\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/figure/pone.0240726.g001/\" target=\"figure\" rid-figpopup=\"pone.0240726.g001\" rid-ob=\"ob-pone.0240726.g001\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm140210183592240\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7584249_pone.0240726.g001.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is pone.0240726.g001.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC7584249/bin/pone.0240726.g001.jpg\" /></a></span><span id=\"largeobj_idm140210183592240\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/figure/pone.0240726.g001/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_pone.0240726.g001\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/figure/pone.0240726.g001/\" target=\"figure\" rid-figpopup=\"pone.0240726.g001\" rid-ob=\"ob-pone.0240726.g001\">Fig 1</a></span><!--caption a7--><span class=\"caption\"><!--caption a8--><strong>Confirmatory factor analysis of the VMQ among the young adult sample.</strong><p id=\"__p46\">For clarity, covariances between motives and errors were not depicted in the figure. All factor loadings were significant at <em>p</em> &#x0003c; .001.</p></span></span></span></span><span id=\"s\r\n028\" class=\"sec sec-last\"><p></p><h4 id=\"s\r\n028title\" class=\"inline\">Criterion validity </h4><p id=\"__p47\" class=\"p p-first\">As with the adolescent sample (Study 2), VMQ motives demonstrated significant correlations with other relevant variables in gaming (see <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/table/pone.0240726.t003/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"pone.0240726.t003\" rid-ob=\"ob-pone.0240726.t003\" co-legend-rid=\"\"><span>Table 3</span></a>). The highest correlations between motivation scores and number of hours spent gaming weekly were for social interaction and recreation. In relation to disordered gaming, the highest correlations were with violent reward, social interaction, and coping. Regarding gaming genre, compared to the adolescent sample, similar findings were found among the most popular genres. Shooter games were strongly associated with violent reward and social interaction, and MOBA games were highly related to social interaction. As in the adolescent sample, male gender showed relationships with every motive except customization. The largest associations among males were violent reward, social interaction, and competition. In addition, customization was broadly associated with higher age. Descriptive data by gender and correlations between motives are displayed in <a href=\"#pone.0240726.s001\" rid=\"pone.0240726.s001\" class=\" supplementary-material\">S1</a>&#x02013;<a href=\"#pone.0240726.s003\" rid=\"pone.0240726.s003\" class=\" supplementary-material\">S3</a> Tables.</p><p id=\"__p48\" class=\"p p-last\">Multiple linear regression analyses were carried out to identify the role of motives in number of hours spent gaming weekly and disordered gaming, controlling for age and gender (see <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/table/pone.0240726.t004/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"pone.0240726.t004\" rid-ob=\"ob-pone.0240726.t004\" co-legend-rid=\"\"><span>Table 4</span></a>). Results showed that social interaction was associated with higher gaming use whereas coping and, to a lesser extent, social interaction and violent reward mainly predicted disordered gaming.</p></span></span></span><span id=\"s\r\n029\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n029title\">Discussion</h2><p id=\"__p49\" class=\"p p-first\">The aim of the present study was to develop a new psychometrically robust scale for assessing motives for videogame playing, the Videogaming Motives Questionnaire (VMQ). As a result, the VMQ consisted of an 8-intercorrelated-factor model that appeared to best explain the data. It showed good internal consistency and temporal stability reliability indices, and also presented evidence of validity when predicting video game frequency of use and disordered gaming.</p><p id=\"__p50\">Motives are important antecedents of other addictive-related behavior such as using alcohol, tobacco, marijuana and gambling (e.g., [<a href=\"#pone.0240726.ref027\" rid=\"pone.0240726.ref027\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889792\">27</a>,<a href=\"#pone.0240726.ref063\" rid=\"pone.0240726.ref063\" class=\" bibr popnode\">63</a>]). Based on the motivational model of Cox and Klinger [<a href=\"#pone.0240726.ref075\" rid=\"pone.0240726.ref075\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889741\">75</a>], three main and common motives to all these addictive behaviors have been identified: social, enhancement, and coping [<a href=\"#pone.0240726.ref022\" rid=\"pone.0240726.ref022\" class=\" bibr popnode\">22</a>,<a href=\"#pone.0240726.ref025\" rid=\"pone.0240726.ref025\" class=\" bibr popnode\">25</a>,<a href=\"#pone.0240726.ref027\" rid=\"pone.0240726.ref027\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889730\">27</a>]. A fourth motive of conformity (drinking alcohol or smoking cigarettes in response to social pressures) has also been proposed [<a href=\"#pone.0240726.ref022\" rid=\"pone.0240726.ref022\" class=\" bibr popnode\">22</a>,<a href=\"#pone.0240726.ref025\" rid=\"pone.0240726.ref025\" class=\" bibr popnode\">25</a>], although its role in substance use has usually been small or even negligible [<a href=\"#pone.0240726.ref063\" rid=\"pone.0240726.ref063\" class=\" bibr popnode\">63</a>,<a href=\"#pone.0240726.ref076\" rid=\"pone.0240726.ref076\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889755\">76</a>]. Accordingly, the main three common motives, coping, social, and enhancement (also labeled as <em>diversion</em> or <em>recreation</em>), have been identified in previous studies on videogame playing (see <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/table/pone.0240726.t001/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"pone.0240726.t001\" rid-ob=\"ob-pone.0240726.t001\" co-legend-rid=\"\"><span>Table 1</span></a>), and in the present study.</p><p id=\"__p51\">The coping motivation refers to playing for stress-reduction and mood enhancement. This motivation is widely represented in most gaming motive scales that include an escape/coping motivational component. Demetrovics et al. [<a href=\"#pone.0240726.ref036\" rid=\"pone.0240726.ref036\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889752\">36</a>] through a literature review, distinguished between escapism (playing to forget daily problems) and coping motives. However, these motives were highly correlated. <em>r</em> &#x0003e; .6; [<a href=\"#pone.0240726.ref036\" rid=\"pone.0240726.ref036\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889769\">36</a>,<a href=\"#pone.0240726.ref049\" rid=\"pone.0240726.ref049\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889777\">49</a>] and, in the present study, both motives converged as a unique factor with coping items loading more strongly. In the current study, the coping motive was associated with shooter and action-adventure games both in adolescents and young adults, and with MMORPGs in young adults. Likewise, previous research has shown the escape/coping motivational component as being highly related to MMORPGs among adults [<a href=\"#pone.0240726.ref047\" rid=\"pone.0240726.ref047\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889765\">47</a>,<a href=\"#pone.0240726.ref059\" rid=\"pone.0240726.ref059\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889726\">59</a>].</p><p id=\"__p52\">In the present study, coping was the strongest and somewhat selective predictor of disordered gaming scores among both adults and adolescents. This result is in accordance with previous studies in which escape/coping motives presented the highest association with disordered gaming [<a href=\"#pone.0240726.ref048\" rid=\"pone.0240726.ref048\" class=\" bibr popnode\">48</a>&#x02013;<a href=\"#pone.0240726.ref050\" rid=\"pone.0240726.ref050\" class=\" bibr popnode\">50</a>,<a href=\"#pone.0240726.ref052\" rid=\"pone.0240726.ref052\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889799\">52</a>,<a href=\"#pone.0240726.ref053\" rid=\"pone.0240726.ref053\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889785\">53</a>,<a href=\"#pone.0240726.ref059\" rid=\"pone.0240726.ref059\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889740\">59</a>,<a href=\"#pone.0240726.ref077\" rid=\"pone.0240726.ref077\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889737\">77</a>] and with problematic substance use and gambling [<a href=\"#pone.0240726.ref022\" rid=\"pone.0240726.ref022\" class=\" bibr popnode\">22</a>&#x02013;<a href=\"#pone.0240726.ref027\" rid=\"pone.0240726.ref027\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889782\">27</a>]. This motive appears to be especially relevant in the presence of psychological problems and environmental stressors, such as depression and anxiety symptoms [<a href=\"#pone.0240726.ref050\" rid=\"pone.0240726.ref050\" class=\" bibr popnode\">50</a>,<a href=\"#pone.0240726.ref054\" rid=\"pone.0240726.ref054\" class=\" bibr popnode\">54</a>&#x02013;<a href=\"#pone.0240726.ref056\" rid=\"pone.0240726.ref056\" class=\" bibr popnode\">56</a>] or stressful life events [<a href=\"#pone.0240726.ref078\" rid=\"pone.0240726.ref078\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889779\">78</a>]. Accordingly, motivation for escape or relief from negative mood constitutes an important criterion for the diagnoses of IGD [<a href=\"#pone.0240726.ref005\" rid=\"pone.0240726.ref005\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889739\">5</a>] and other addictions in the DSM-5 [<a href=\"#pone.0240726.ref003\" rid=\"pone.0240726.ref003\" class=\" bibr popnode\">3</a>].</p><p id=\"__p53\">The social interaction motive is based on bonding with friends and making new ones. This motive is also widely represented in most of the scales. In the present study, social interaction was broadly associated with shooters and MOBAs both in adolescents and young adults, and MMORPGs among young adults, whereas the motive was negatively associated with games that are usually played alone, such as social simulation games (for both adolescents and young adults). Similarly, social interaction has been associated with higher preferences for online role-playing, shooters, and real time strategy games [<a href=\"#pone.0240726.ref047\" rid=\"pone.0240726.ref047\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889734\">47</a>] and lower engagement in casual games [<a href=\"#pone.0240726.ref057\" rid=\"pone.0240726.ref057\" class=\" bibr popnode\">57</a>]. In the present study, social interaction was the highest predictor of number of hours spent gaming weekly among both samples, in line with other studies (e.g., [<a href=\"#pone.0240726.ref032\" rid=\"pone.0240726.ref032\" class=\" bibr popnode\">32</a>,<a href=\"#pone.0240726.ref049\" rid=\"pone.0240726.ref049\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889764\">49</a>,<a href=\"#pone.0240726.ref053\" rid=\"pone.0240726.ref053\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889800\">53</a>,<a href=\"#pone.0240726.ref057\" rid=\"pone.0240726.ref057\" class=\" bibr popnode\">57</a>,<a href=\"#pone.0240726.ref079\" rid=\"pone.0240726.ref079\" class=\" bibr popnode\">79</a>]). Therefore, more socially motivated players like to spend their time playing videogames that offer social interactions such as MOBAs or/and shooters.</p><p id=\"__p54\">The present study also found that social interaction was highly associated with disordered gaming (after controlling for age, gender, and the remaining motives in both samples). Similarly, previous studies have found that after escape/coping, the gaming motive with the next highest relationship with disordered gaming is social interaction [<a href=\"#pone.0240726.ref051\" rid=\"pone.0240726.ref051\" class=\" bibr popnode\">51</a>,<a href=\"#pone.0240726.ref059\" rid=\"pone.0240726.ref059\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889751\">59</a>]. It has been suggested that high scores in social interaction within gaming might reflect low social competence in real life, and that gaming compensates for this characteristic, resulting in disordered gaming [<a href=\"#pone.0240726.ref014\" rid=\"pone.0240726.ref014\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889744\">14</a>,<a href=\"#pone.0240726.ref015\" rid=\"pone.0240726.ref015\" class=\" bibr popnode\">15</a>].</p><p id=\"__p55\">Finally, recreation (enhancement or recreational) motives&#x02013;focusing on the enjoyable and recreational gaming component&#x02013;was mainly related to higher use of shooter and role-playing games in both samples. In addition, some studies have found that recreation is the major motivation related to greater gaming duration [<a href=\"#pone.0240726.ref032\" rid=\"pone.0240726.ref032\" class=\" bibr popnode\">32</a>,<a href=\"#pone.0240726.ref057\" rid=\"pone.0240726.ref057\" class=\" bibr popnode\">57</a>]. In the present study, recreation motives were correlated with frequency of gaming and disordered gaming, but when age and gender and other gaming motives were controlled for, these associations became nonsignificant, probably due to the interrelation with other motives. Therefore, in our samples, other motives such as social interaction may explain better the variance in gaming frequency rather than playing for mere enjoyment.</p><p id=\"__p56\">In addition to these three common motives, specific motivational factors have also been identified for other addictive behaviors, such as expanse motives for marijuana use and abuse [<a href=\"#pone.0240726.ref024\" rid=\"pone.0240726.ref024\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889774\">24</a>,<a href=\"#pone.0240726.ref025\" rid=\"pone.0240726.ref025\" class=\" bibr popnode\">25</a>] or financial motives for gambling behavior [<a href=\"#pone.0240726.ref026\" rid=\"pone.0240726.ref026\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889773\">26</a>]. In the field of gaming, previous studies have also identified specific motives for playing videogames, such as arousal (playing to excite emotions), achievement-challenge (playing to obtain in-game rewards or prestige), skill development, fantasy, competition, and customization. In addition, Hilgard et al. [<a href=\"#pone.0240726.ref059\" rid=\"pone.0240726.ref059\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889719\">59</a>] identified two additional relevant motives for playing&#x02013;violence catharsis (in-game violence helping to release negative moods or aggression) and violent reward&#x02013;which were included in the present study due to the potential relevance of aggressive gaming in violent behaviors [<a href=\"#pone.0240726.ref062\" rid=\"pone.0240726.ref062\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889729\">62</a>]. Of these specific motives, the study clearly identified skill development (cognitive development in the present study), fantasy, competition, customization, and violent reward. In Study 1, items from arousal motives mainly loaded on the recreation factor, violent catharsis items loaded on the escape-coping factor, and achievement-challenge items were distributed between different motives.</p><p id=\"__p57\">The <em>cognitive development</em> motive refers to the intellectual activity stimulation during videogame play. In the first study, this motive encompassed two previous motives: skill development [<a href=\"#pone.0240726.ref036\" rid=\"pone.0240726.ref036\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889796\">36</a>] and &#x02018;smarty-pants&#x02019; [<a href=\"#pone.0240726.ref044\" rid=\"pone.0240726.ref044\" class=\" bibr popnode\">44</a>], although when selecting the items with higher loadings for the final VMQ, it became a more cognitive development and mental challenge factor. In the present study, this motive was associated with action-adventure and shooter games, in both samples, and with MOBA games among adolescents. Previous research has found that shooters and real time strategy genres have been highly associated with skill development [<a href=\"#pone.0240726.ref047\" rid=\"pone.0240726.ref047\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889759\">47</a>]. Consequently, individuals with higher levels in this motive may be more attracted to those games where improving skill and abilities are important elements of the in-game experience.</p><p id=\"__p58\">The <em>fantasy</em> motive is defined by playing for the immersion in the gaming world and the story&#x02019;s in-game characters. This motive has received different labels in gaming motive research such as immersion [<a href=\"#pone.0240726.ref034\" rid=\"pone.0240726.ref034\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889787\">34</a>], presence [<a href=\"#pone.0240726.ref031\" rid=\"pone.0240726.ref031\" class=\" bibr popnode\">31</a>], narrative [<a href=\"#pone.0240726.ref033\" rid=\"pone.0240726.ref033\" class=\" bibr popnode\">33</a>], story [<a href=\"#pone.0240726.ref059\" rid=\"pone.0240726.ref059\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889724\">59</a>], or exploration [<a href=\"#pone.0240726.ref061\" rid=\"pone.0240726.ref061\" class=\" bibr popnode\">61</a>]. In the present study, fantasy was significantly associated with game genre use in which a storyline is developed, usually through a campaign, such as action-adventure games, shooter game, role-playing games, and MMORPGs.</p><p id=\"__p59\">Previous research has reported similar findings in which fantasy is strongly associated with the use and preference for shooters, role-playing, and action-adventure games [<a href=\"#pone.0240726.ref046\" rid=\"pone.0240726.ref046\" class=\" bibr popnode\">46</a>,<a href=\"#pone.0240726.ref047\" rid=\"pone.0240726.ref047\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889753\">47</a>,<a href=\"#pone.0240726.ref057\" rid=\"pone.0240726.ref057\" class=\" bibr popnode\">57</a>,<a href=\"#pone.0240726.ref058\" rid=\"pone.0240726.ref058\" class=\" bibr popnode\">58</a>] as well as for franchises of such genres [<a href=\"#pone.0240726.ref059\" rid=\"pone.0240726.ref059\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889736\">59</a>]. In addition, the present study showed that fantasy significantly predicted higher disordered gaming among young adults, in line with some studies [<a href=\"#pone.0240726.ref040\" rid=\"pone.0240726.ref040\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889721\">40</a>,<a href=\"#pone.0240726.ref050\" rid=\"pone.0240726.ref050\" class=\" bibr popnode\">50</a>,<a href=\"#pone.0240726.ref054\" rid=\"pone.0240726.ref054\" class=\" bibr popnode\">54</a>]. Therefore, in-game immersion, at least among adults, could impair the player&#x02019;s life by neglecting important life domains. In other words, this higher immersion feeling in videogame playing may disrupt professional careers and social relationships affecting players&#x02019; psychological wellbeing.</p><p id=\"__p60\">The <em>competition</em> motive&#x02013;based on the pleasure of competing and winning against others&#x02013;was strongly associated with competitive gaming genre use such as shooter games (in the adolescent sample), MOBA games and sports games (for both samples). In addition, competition was negatively related to the use of non-competitive genres such as casual games (in both samples). Similarly, the competition motive has been found to be strongly associated with competitive gaming genres, such as sports games, action games, or real time strategy games, and to a lesser extent noncompetitive games and genres such as casual games [<a href=\"#pone.0240726.ref047\" rid=\"pone.0240726.ref047\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889745\">47</a>,<a href=\"#pone.0240726.ref057\" rid=\"pone.0240726.ref057\" class=\" bibr popnode\">57</a>,<a href=\"#pone.0240726.ref058\" rid=\"pone.0240726.ref058\" class=\" bibr popnode\">58</a>]. Taken together, these studies highlight that the competition motive appears to play an important role in the player&#x02019;s game genre preferences.</p><p id=\"__p61\"><em>Customization</em> motives refers to the creation and design of things in-game. Consequently, this motive was highly associated with use of social simulation games in both samples. Customization was also extensively associated with construction (among adolescents) and role-playing games (among young adults). In previous research, customization has been positively associated with higher preferences for role-playing games such as <em>Final Fantasy</em> and <em>Skyrim</em> [<a href=\"#pone.0240726.ref059\" rid=\"pone.0240726.ref059\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889775\">59</a>]. Among the adult sample in the present study, customization may have acted as a protective factor in preventing disordered gaming. The gratification of creating and designing in-game things might underlie facets of conscientiousness that are negatively associated with disordered gaming [<a href=\"#pone.0240726.ref009\" rid=\"pone.0240726.ref009\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889749\">9</a>].</p><p id=\"__p62\">Finally, <em>violent reward</em> refers to the gratification obtained via in-game violence. Surprisingly, this motive has been very scarcely studied, despite its potential role in the development of aggressive preferences and violent behavior. Accordingly, this motive was strongly associated with use of shooters and negatively related to non-violent and more peaceful videogames such as casual games, both in adolescents and in young adults. Similarly, Hilgard et al. [<a href=\"#pone.0240726.ref059\" rid=\"pone.0240726.ref059\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889786\">59</a>] reported that this motive was strongly associated with preferences for the action shooter franchise of <em>Grand Theft Auto</em>. In addition, in the present study, violent reward also had a positive association with disordered gaming among both adolescents and young adults. As far as the present authors are aware, this is the first time that this association has been reported, although other studies have shown that the acceptance of violence is associated with higher disordered gaming [<a href=\"#pone.0240726.ref080\" rid=\"pone.0240726.ref080\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889778\">80</a>], and that MMORPG players attracted to the release of aggressive and antisocial feelings through gaming have a higher risk of addiction [<a href=\"#pone.0240726.ref081\" rid=\"pone.0240726.ref081\" class=\" bibr popnode\">81</a>]. Overall, these relationships highlight the importance of this often-neglected motive in particularly harmful videogame behaviors, such as disordered gaming and the preference for violent games.</p><p id=\"__p63\">To sum up, adolescents and young adults displayed a similar pattern in the relationship between motives and gaming behaviors. For both samples, disordered gaming was strongly related to coping, followed by social interaction and violent reward, this last motive with a lower effect. In addition, a larger number of gaming hours was associated with higher scores in social interaction. However, slight differences existed between young adults and adolescents. For instance, disordered gaming was slightly related to fantasy and negatively to customization in young adults, whereas number of hours spent gaming weekly were also associated with coping in adolescents. With regard to videogame genres, a similar pattern was also found, replicating most of the highest associations between motives and games played in both samples.</p><p id=\"__p64\">Regarding gender, adolescent and young adult males presented higher scores than females in nearly all scales apart from customization, especially in competitive, violent reward and social interaction motives. These higher motivational levels in male gender for almost all motives has been systematically found in previous studies (e.g., [<a href=\"#pone.0240726.ref032\" rid=\"pone.0240726.ref032\" class=\" bibr popnode\">32</a>&#x02013;<a href=\"#pone.0240726.ref036\" rid=\"pone.0240726.ref036\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889720\">36</a>,<a href=\"#pone.0240726.ref057\" rid=\"pone.0240726.ref057\" class=\" bibr popnode\">57</a>]). Along these lines, it has been reported that adolescent males prefer competitive and violent game genres, such as action-<em>shooters</em>, sport, fight or strategy games; whereas girls prefer game genres such as brain and skill games, and social simulation games, more characterized by customization aspects [<a href=\"#pone.0240726.ref009\" rid=\"pone.0240726.ref009\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889723\">9</a>]. These gender-based differences on motives for gaming may help to explain why males, who prefer games featuring more time-consuming and engagement characteristics, presented higher prevalence of regular and disordered use of video games than females [<a href=\"#pone.0240726.ref007\" rid=\"pone.0240726.ref007\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_888889731\">7</a>].</p><p id=\"__p65\" class=\"p p-last\">The present study is not without its limitations. First, the studies only comprised Spanish adolescents and young adults. Therefore, validation of the VMQ in other socio-cultural contexts are necessary to generalize the findings. Second, the present study was cross-sectional, consequently longitudinal studies are needed to offer stronger evidence of motives as determinants of gaming behaviors. Third, despite trying to be exhaustive, there could be other relevant motives not identified in the present study. Fourth, the time engaged in gaming was obtained by self-reported retrospective estimates, which have been found to be less accurate than other procedures such as Timeline Followback [<a href=\"#pone.0240726.ref082\" rid=\"pone.0240726.ref082\" class=\" bibr popnode\">82</a>]. Finally, in order to systematically explain individual differences in gaming behaviors, future studies should include other relevant variables such as personality, which may allow exploration of the complex interplay between these variables (e.g. the mediational role of motives in the personality-gaming associations) [<a href=\"#pone.0240726.ref083\" rid=\"pone.0240726.ref083\" class=\" bibr popnode\">83</a>].</p></span><span id=\"s\r\n030\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s\r\n030title\">Conclusion</h2><p id=\"__p66\" class=\"p p-first\">The Videogaming Motives Questionnaire showed an underlying robust structure comprising eight motives: recreation, social interaction, coping, violent reward, competition, fantasy, cognitive development and customization. In addition, it showed meaningful associations with different game-related behaviors in two independent samples of adolescents and young adults, such as weekly time spent gaming, disordered gaming, and gaming genres most played.</p><p id=\"__p67\" class=\"p p-last\">Thus, VMQ constitutes a new psychometric instrument that produces valid and reliable scores for the assessment of some of the more representative gaming motives, and in a relatively brief way. The possibility to assess gaming motives briefly and comprehensively may help to incorporate the study of motives in the research of individual differences on gaming-related behaviors, and may be useful in the development of personalized programs to prevent and treat disordered gaming [<a href=\"#pone.0240726.ref084\" rid=\"pone.0240726.ref084\" class=\" bibr popnode\">84</a>].</p></span><span id=\"s\r\n031\" class=\"tsec sec\"><a id=\"supplementary-material-sec\"></a><h2 class=\"head no_bottom_margin\" id=\"s\r\n031title\">Supporting information</h2><!--/article/body/sec/--><span class=\"sec suppmat\" id=\"pone.0240726.s001\"><h4>S1 Table</h4><!--caption a9--><strong>Study 1: Pattern matrix for oblimin eight-factor solution extracted from pilot Videogaming Motives Questionnaire (VMQ). </strong><p id=\"__p68\">(DOCX)</p><span class=\"sup-box half_rhythm\" id=\"idm140210184228192\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/bin/pone.0240726.s001.docx\" data-ga-action=\"click_feat_suppl\">Click here for additional data file.</a><sup>(30K, docx)</sup></span></span><span class=\"sec suppmat\" id=\"pone.0240726.s002\"><h4>S2 Table</h4><!--caption a9--><strong>Study 2: Descriptive data by gender, gender differences and correlation matrix. </strong><p id=\"__p70\">(DOCX)</p><span class=\"sup-box half_rhythm\" id=\"idm140210184224048\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/bin/pone.0240726.s002.docx\" data-ga-action=\"click_feat_suppl\">Click here for additional data file.</a><sup>(16K, docx)</sup></span></span><span class=\"sec suppmat\" id=\"pone.0240726.s003\"><h4>S3 Table</h4><!--caption a9--><strong>Study 3: Descriptive data by gender, gender differences and correlation matrix. </strong><p id=\"__p72\">(DOCX)</p><span class=\"sup-box half_rhythm\" id=\"idm140210205808336\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/bin/pone.0240726.s003.docx\" data-ga-action=\"click_feat_suppl\">Click here for additional data file.</a><sup>(16K, docx)</sup></span></span></span><span id=\"idm140210205806544\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140210205806544title\">Acknowledgments</h2><span class=\"sec\"><p id=\"__p74\">The authors wish to thank all the undergraduate participants, and all the students, parents, and teachers of the high schools <em>Bovalar</em> and <em>El Camin&#x000e0;s</em> for making this study possible.</p></span></span><span id=\"idm140210184081424\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140210184081424title\">Funding Statement</h2><p>The author(s) disclose receiving the following financial support for the research, authorship, and/or for the publication of this article: RTI2018-099800-B-I00 from the Spanish Ministry of Science, Innovation and Universities (MICIU/FEDER); GV/2016/158 and AICO/2019/197 from the Valencian Regional Government; UJI-A2017-18, UJI-B2017-74, UJI-A2019-08 and E-2018-16 from the Universitat Jaume I.</p></span><span id=\"idm140210183609424\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140210183609424title\">Data Availability</h2><p id=\"__p3\">All relevant data are within the manuscript and its Supporting Information files.</p></span><span id=\"idm140210205804336\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm140210205804336title\">References</h2><span class=\"ref-list-sec sec\" id=\"reference-list\"><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref001\">1. <span class=\"mixed-citation\">Egenfeldt-Nielsen S, Smith JH, Tosca SP. <span class=\"ref-journal\">Understanding video games: The essential introduction</span>. 4th ed. New York: Routledge; 2019. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Understanding+video+games:+The+essential+introduction&amp;author=S+Egenfeldt-Nielsen&amp;author=JH+Smith&amp;author=SP+Tosca&amp;publication_year=2019&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref002\">2. <span class=\"mixed-citation\">Asociaci&#x000f3;n Espa&#x000f1;ola de Videojuegos (AEVI). El anuario del videojuego [The yearbook of videogames]. 2019 June 1 [cited 2 April 2020]. Available from: <a href=\"http://www.aevi.org.es/web/wp-content/uploads/2019/05/AEVI_Anuario_2018.pdf\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.aevi.org.es/web/wp-content/uploads/2019/05/AEVI_Anuario_2018.pdf</a>.</span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref003\">3. <span class=\"mixed-citation\">American Psychiatric Association. <span class=\"ref-journal\">Diagnostic and statistical manual of mental disorders (DSM-5&#x000ae;)</span>. 5th ed. Arlington VA: American Psychiatric Publishing; 2013. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Diagnostic+and+statistical+manual+of+mental+disorders+(DSM-5&#x000ae;)&amp;publication_year=2013&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref004\">4. <span class=\"mixed-citation\">World Health Organization. ICD-11 beta draft. Mental, behavioural or neurodevelopmental disorders. 2017 [cited 2 April 2020]. Available from <a href=\"https://icd.who.int/dev11/lm/en#/http%3a%2f%2fid.who.int%2ficd%2fentity%2f338347362\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://icd.who.int/dev11/lm/en#/http%3a%2f%2fid.who.int%2ficd%2fentity%2f338347362</a>.</span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref005\">5. <span class=\"mixed-citation\">Petry NM, Rehbein F, Ko CH, O&#x02019;Brien CP. <span class=\"ref-title\">Internet gaming disorder in the DSM-5.</span>\n<span class=\"ref-journal\">Curr Psychiatry Rep.</span>\n2015;<span class=\"ref-vol\">17</span>: 72&#x02013;80. 10.1007/s11920-015-0610-0\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26216590\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs11920-015-0610-0\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Curr+Psychiatry+Rep.&amp;title=Internet+gaming+disorder+in+the+DSM-5.&amp;author=NM+Petry&amp;author=F+Rehbein&amp;author=CH+Ko&amp;author=CP+O&#x02019;Brien&amp;volume=17&amp;publication_year=2015&amp;pages=72-80&amp;pmid=26216590&amp;doi=10.1007/s11920-015-0610-0&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref006\">6. <span class=\"mixed-citation\">Gentile DA, Bailey K, Bavelier D, Brockmyer JF, Cash H, Coyne S M, et al.\n<span class=\"ref-title\">Internet gaming disorder in children and adolescents</span>. <span class=\"ref-journal\">Pediatrics</span>. 2017;<span class=\"ref-vol\">140</span>: S81&#x02013;S85. 10.1542/peds.2016-1758H\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n29093038\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1542%2Fpeds.2016-1758H\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Pediatrics&amp;title=Internet+gaming+disorder+in+children+and+adolescents&amp;author=DA+Gentile&amp;author=K+Bailey&amp;author=D+Bavelier&amp;author=JF+Brockmyer&amp;author=H+Cash&amp;volume=140&amp;publication_year=2017&amp;pages=S81-S85&amp;pmid=29093038&amp;doi=10.1542/peds.2016-1758H&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref007\">7. <span class=\"mixed-citation\">Mihara S, Higuchi S. <span class=\"ref-title\">Cross&#x02010;sectional and longitudinal epidemiological studies of internet gaming disorder: A systematic review of the literature</span>. <span class=\"ref-journal\">Psychiatry Clin Neurosci</span>. 2017;<span class=\"ref-vol\">71</span>: 425&#x02013;444. 10.1111/pcn.12532\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28436212\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fpcn.12532\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychiatry+Clin+Neurosci&amp;title=Cross&#x02010;sectional+and+longitudinal+epidemiological+studies+of+internet+gaming+disorder:+A+systematic+review+of+the+literature&amp;author=S+Mihara&amp;author=S+Higuchi&amp;volume=71&amp;publication_year=2017&amp;pages=425-444&amp;pmid=28436212&amp;doi=10.1111/pcn.12532&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref008\">8. <span class=\"mixed-citation\">Buiza-Aguado C, Alonso-Canovas A, Conde-Mateos C, Buiza-Navarrete JJ, Gentile D. <span class=\"ref-title\">Problematic video gaming in a young Spanish population: Association with psychosocial health.</span>\n<span class=\"ref-journal\">Cyberpsychol Behav Soc Netw</span>. 2018;<span class=\"ref-vol\">21</span>: 388&#x02013;394. 10.1089/cyber.2017.0599\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n29792521\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1089%2Fcyber.2017.0599\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cyberpsychol+Behav+Soc+Netw&amp;title=Problematic+video+gaming+in+a+young+Spanish+population:+Association+with+psychosocial+health.&amp;author=C+Buiza-Aguado&amp;author=A+Alonso-Canovas&amp;author=C+Conde-Mateos&amp;author=JJ+Buiza-Navarrete&amp;author=D+Gentile&amp;volume=21&amp;publication_year=2018&amp;pages=388-394&amp;pmid=29792521&amp;doi=10.1089/cyber.2017.0599&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref009\">9. <span class=\"mixed-citation\">L&#x000f3;pez-Fern&#x000e1;ndez FJ, Mezquita L, Griffiths MD, Ortet F, Ib&#x000e1;&#x000f1;ez MI. <span class=\"ref-title\">The role of personality on disordered gaming and game genre preferences in adolescence: gender differences and person-environment transactions.</span>\n<span class=\"ref-journal\">Adicciones.</span>\n2020. Advance publication online: doi: 10.20882/adicciones.0.0 [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n32100046\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.20882%2Fadicciones.0.0\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Adicciones.&amp;title=The+role+of+personality+on+disordered+gaming+and+game+genre+preferences+in+adolescence:+gender+differences+and+person-environment+transactions.&amp;author=FJ+L&#x000f3;pez-Fern&#x000e1;ndez&amp;author=L+Mezquita&amp;author=MD+Griffiths&amp;author=F+Ortet&amp;author=MI+Ib&#x000e1;&#x000f1;ez&amp;publication_year=2020&amp;doi=10.20882/adicciones.0.0&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref010\">10. <span class=\"mixed-citation\">Lopez-Fernandez O, Honrubia-Serrano ML, Baguley T, Griffiths MD. <span class=\"ref-title\">Pathological video game playing in Spanish and British adolescents: Towards the exploration of Internet Gaming Disorder symptomatology</span>. <span class=\"ref-journal\">Comput Human Behav</span>. 2014;<span class=\"ref-vol\">41</span>: 304&#x02013;312. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Human+Behav&amp;title=Pathological+video+game+playing+in+Spanish+and+British+adolescents:+Towards+the+exploration+of+Internet+Gaming+Disorder+symptomatology&amp;author=O+Lopez-Fernandez&amp;author=ML+Honrubia-Serrano&amp;author=T+Baguley&amp;author=MD+Griffiths&amp;volume=41&amp;publication_year=2014&amp;pages=304-312&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref011\">11. <span class=\"mixed-citation\">Gervasi AM, La Marca L, Costanzo A, Pace U, Guglielmucci F, Schimmenti A. <span class=\"ref-title\">Personality and Internet gaming disorder: A systematic review of recent literature</span>. <span class=\"ref-journal\">Curr Addict Rep</span>. 2017;<span class=\"ref-vol\">4</span>: 293&#x02013;307. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Curr+Addict+Rep&amp;title=Personality+and+Internet+gaming+disorder:+A+systematic+review+of+recent+literature&amp;author=AM+Gervasi&amp;author=L+La+Marca&amp;author=A+Costanzo&amp;author=U+Pace&amp;author=F+Guglielmucci&amp;volume=4&amp;publication_year=2017&amp;pages=293-307&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref012\">12. <span class=\"mixed-citation\">Griffiths MD, Kuss DJ, King DL. <span class=\"ref-title\">Video game addiction: Past, present and future.</span>\n<span class=\"ref-journal\">Curr Psychiatry Rev.</span>\n2012;<span class=\"ref-vol\">8</span>: 308&#x02013;318. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Curr+Psychiatry+Rev.&amp;title=Video+game+addiction:+Past,+present+and+future.&amp;author=MD+Griffiths&amp;author=DJ+Kuss&amp;author=DL+King&amp;volume=8&amp;publication_year=2012&amp;pages=308-318&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref013\">13. <span class=\"mixed-citation\">Brunborg GS, Mentzoni RA, Fr&#x000f8;yland LR. <span class=\"ref-title\">Is video gaming, or video game addiction, associated with depression, academic achievement, heavy episodic drinking, or conduct problems?</span>\n<span class=\"ref-journal\">J Behav Addict</span>. 2014;<span class=\"ref-vol\">3</span>: 27&#x02013;32. 10.1556/JBA.3.2014.002\n <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4117274/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25215212\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1556%2FJBA.3.2014.002\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Behav+Addict&amp;title=Is+video+gaming,+or+video+game+addiction,+associated+with+depression,+academic+achievement,+heavy+episodic+drinking,+or+conduct+problems?&amp;author=GS+Brunborg&amp;author=RA+Mentzoni&amp;author=LR+Fr&#x000f8;yland&amp;volume=3&amp;publication_year=2014&amp;pages=27-32&amp;pmid=25215212&amp;doi=10.1556/JBA.3.2014.002&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref014\">14. <span class=\"mixed-citation\">Gentile DA, Choo H, Liau A, Sim T, Li D, Fung D, et al.\n<span class=\"ref-title\">Pathological video game use among youths: A two-year longitudinal study</span>. <span class=\"ref-journal\">Pediatrics</span>. 2011;<span class=\"ref-vol\">127</span>: 319&#x02013;329. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21242221\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Pediatrics&amp;title=Pathological+video+game+use+among+youths:+A+two-year+longitudinal+study&amp;author=DA+Gentile&amp;author=H+Choo&amp;author=A+Liau&amp;author=T+Sim&amp;author=D+Li&amp;volume=127&amp;publication_year=2011&amp;pages=319-329&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref015\">15. <span class=\"mixed-citation\">Lemmens JS, Valkenburg PM., Peter J. <span class=\"ref-title\">Psychosocial causes and consequences of pathological gaming.</span>\n<span class=\"ref-journal\">Comput Human Behav.</span>\n2011;<span class=\"ref-vol\">27</span>: 144&#x02013;152. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Human+Behav.&amp;title=Psychosocial+causes+and+consequences+of+pathological+gaming.&amp;author=JS+Lemmens&amp;author=PM+Valkenburg&amp;author=J+Peter&amp;volume=27&amp;publication_year=2011&amp;pages=144-152&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref016\">16. <span class=\"mixed-citation\">Lemmens JS, Valkenburg PM., Peter J. <span class=\"ref-title\">The effects of pathological gaming on aggressive behavior.</span>\n<span class=\"ref-journal\">J. Youth Adolesc</span>. 2011;<span class=\"ref-vol\">40</span>: 38&#x02013;47. 10.1007/s10964-010-9558-x\n <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3003785/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20549320\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs10964-010-9558-x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Youth+Adolesc&amp;title=The+effects+of+pathological+gaming+on+aggressive+behavior.&amp;author=JS+Lemmens&amp;author=PM+Valkenburg&amp;author=J+Peter&amp;volume=40&amp;publication_year=2011&amp;pages=38-47&amp;pmid=20549320&amp;doi=10.1007/s10964-010-9558-x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref017\">17. <span class=\"mixed-citation\">van Rooij AJ, Schoenmakers TM, Vermulst AA, van den Eijnden R, van de Mheen D. <span class=\"ref-title\">Online video game addiction: Identification of addicted adolescent gamers.</span>\n<span class=\"ref-journal\">Addiction</span>. 2011;<span class=\"ref-vol\">106</span>: 205&#x02013;212. 10.1111/j.1360-0443.2010.03104.x\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20840209\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1360-0443.2010.03104.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Addiction&amp;title=Online+video+game+addiction:+Identification+of+addicted+adolescent+gamers.&amp;author=AJ+van+Rooij&amp;author=TM+Schoenmakers&amp;author=AA+Vermulst&amp;author=R+van+den+Eijnden&amp;author=D+van+de+Mheen&amp;volume=106&amp;publication_year=2011&amp;pages=205-212&amp;pmid=20840209&amp;doi=10.1111/j.1360-0443.2010.03104.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref018\">18. <span class=\"mixed-citation\">Griffiths MD, Nuyens F. <span class=\"ref-title\">An overview of structural characteristics in problematic videogame playing.</span>\n<span class=\"ref-journal\">Curr Addict Rep</span>. 2017;<span class=\"ref-vol\">4</span>: 272&#x02013;283. 10.1007/s40429-017-0162-y\n <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5554268/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28845381\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs40429-017-0162-y\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Curr+Addict+Rep&amp;title=An+overview+of+structural+characteristics+in+problematic+videogame+playing.&amp;author=MD+Griffiths&amp;author=F+Nuyens&amp;volume=4&amp;publication_year=2017&amp;pages=272-283&amp;pmid=28845381&amp;doi=10.1007/s40429-017-0162-y&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref019\">19. <span class=\"mixed-citation\">King DL, Delfabbro PH, Griffiths MD. <span class=\"ref-title\">Video game structural characteristics: A new psychological taxonomy.</span>\n<span class=\"ref-journal\">Int J Ment Health Addict</span>. 2010;<span class=\"ref-vol\">8</span>: 90&#x02013;106. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int+J+Ment+Health+Addict&amp;title=Video+game+structural+characteristics:+A+new+psychological+taxonomy.&amp;author=DL+King&amp;author=PH+Delfabbro&amp;author=MD+Griffiths&amp;volume=8&amp;publication_year=2010&amp;pages=90-106&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref020\">20. <span class=\"mixed-citation\">Brand M, Young KS, Laier C, W&#x000f6;lfling K, Potenza MN. <span class=\"ref-title\">Integrating psychological and neurobiological considerations regarding the development and maintenance of specific Internet-use disorders: An Interaction of Person-Affect-Cognition-Execution (I-PACE) model.</span>\n<span class=\"ref-journal\">Neurosci Biobehav Rev.</span>\n2016;<span class=\"ref-vol\">71</span>: 252&#x02013;266. 10.1016/j.neubiorev.2016.08.033\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27590829\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.neubiorev.2016.08.033\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Neurosci+Biobehav+Rev.&amp;title=Integrating+psychological+and+neurobiological+considerations+regarding+the+development+and+maintenance+of+specific+Internet-use+disorders:+An+Interaction+of+Person-Affect-Cognition-Execution+(I-PACE)+model.&amp;author=M+Brand&amp;author=KS+Young&amp;author=C+Laier&amp;author=K+W&#x000f6;lfling&amp;author=MN+Potenza&amp;volume=71&amp;publication_year=2016&amp;pages=252-266&amp;pmid=27590829&amp;doi=10.1016/j.neubiorev.2016.08.033&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref021\">21. <span class=\"mixed-citation\">Dreier M, W&#x000f6;lfling K, M&#x000fc;ller K. <span class=\"ref-title\">Psychological research and a sociological perspective on problematic and addictive computer game use in adolescence</span>. <span class=\"ref-journal\">Int J Child Adolesc health</span>. 2013;<span class=\"ref-vol\">6</span>: 422&#x02013;435. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int+J+Child+Adolesc+health&amp;title=Psychological+research+and+a+sociological+perspective+on+problematic+and+addictive+computer+game+use+in+adolescence&amp;author=M+Dreier&amp;author=K+W&#x000f6;lfling&amp;author=K+M&#x000fc;ller&amp;volume=6&amp;publication_year=2013&amp;pages=422-435&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref022\">22. <span class=\"mixed-citation\">Cooper ML. <span class=\"ref-title\">Motivations for alcohol use among adolescents: Development and validation of a four-factor-model</span>. <span class=\"ref-journal\">Psychol Assess</span>. 1994;<span class=\"ref-vol\">6</span>: 117&#x02013;128. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychol+Assess&amp;title=Motivations+for+alcohol+use+among+adolescents:+Development+and+validation+of+a+four-factor-model&amp;author=ML+Cooper&amp;volume=6&amp;publication_year=1994&amp;pages=117-128&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref023\">23. <span class=\"mixed-citation\">Bravo AJ, Pilatti A, Pearson MR, Mezquita L, Ib&#x000e1;&#x000f1;ez MI, Ortet G. <span class=\"ref-title\">Depressive symptoms, ruminative thinking, drinking motives, and alcohol outcomes: A multiple mediation model among college students in three countries</span>. <span class=\"ref-journal\">Addict Behav</span>. 2018;<span class=\"ref-vol\">76</span>: 319&#x02013;327. 10.1016/j.addbeh.2017.08.028\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28889061\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.addbeh.2017.08.028\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Addict+Behav&amp;title=Depressive+symptoms,+ruminative+thinking,+drinking+motives,+and+alcohol+outcomes:+A+multiple+mediation+model+among+college+students+in+three+countries&amp;author=AJ+Bravo&amp;author=A+Pilatti&amp;author=MR+Pearson&amp;author=L+Mezquita&amp;author=MI+Ib&#x000e1;&#x000f1;ez&amp;volume=76&amp;publication_year=2018&amp;pages=319-327&amp;pmid=28889061&amp;doi=10.1016/j.addbeh.2017.08.028&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref024\">24. <span class=\"mixed-citation\">Mezquita L, Ruiz-Valero L, Mart&#x000ed;nez-G&#x000f3;mez N, Ib&#x000e1;&#x000f1;ez MI, Ortet G. <span class=\"ref-title\">Desarrollo y validaci&#x000f3;n de la versi&#x000f3;n breve del cuestionario de motivos de consumo de marihuana (MMM SF).</span>\n<span class=\"ref-journal\">Adicciones</span>. 2019;<span class=\"ref-vol\">31</span>: 106&#x02013;116. 10.20882/adicciones.979\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n29353298\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.20882%2Fadicciones.979\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Adicciones&amp;title=Desarrollo+y+validaci&#x000f3;n+de+la+versi&#x000f3;n+breve+del+cuestionario+de+motivos+de+consumo+de+marihuana+(MMM+SF).&amp;author=L+Mezquita&amp;author=L+Ruiz-Valero&amp;author=N+Mart&#x000ed;nez-G&#x000f3;mez&amp;author=MI+Ib&#x000e1;&#x000f1;ez&amp;author=G+Ortet&amp;volume=31&amp;publication_year=2019&amp;pages=106-116&amp;pmid=29353298&amp;doi=10.20882/adicciones.979&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref025\">25. <span class=\"mixed-citation\">Simons J, Correia CJ, Carey KB, Borsari BE. <span class=\"ref-title\">Validating a five-factor marijuana motives measure: Relations with use, problems, and alcohol motives</span>. <span class=\"ref-journal\">J Couns Psychol</span>. 1998;<span class=\"ref-vol\">45</span>: 265&#x02013;273. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Couns+Psychol&amp;title=Validating+a+five-factor+marijuana+motives+measure:+Relations+with+use,+problems,+and+alcohol+motives&amp;author=J+Simons&amp;author=CJ+Correia&amp;author=KB+Carey&amp;author=BE+Borsari&amp;volume=45&amp;publication_year=1998&amp;pages=265-273&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref026\">26. <span class=\"mixed-citation\">Lee HP, Chae PK, Lee HS, Kim YK. <span class=\"ref-title\">The five-factor gambling motivation model.</span>\n<span class=\"ref-journal\">Psychiatry Res.</span>\n2007;<span class=\"ref-vol\">150</span>: 21&#x02013;32. 10.1016/j.psychres.2006.04.005\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17270278\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.psychres.2006.04.005\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychiatry+Res.&amp;title=The+five-factor+gambling+motivation+model.&amp;author=HP+Lee&amp;author=PK+Chae&amp;author=HS+Lee&amp;author=YK+Kim&amp;volume=150&amp;publication_year=2007&amp;pages=21-32&amp;pmid=17270278&amp;doi=10.1016/j.psychres.2006.04.005&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref027\">27. <span class=\"mixed-citation\">Stewart SH, Zack M. <span class=\"ref-title\">Development and psychometric evaluation of a three&#x02010;dimensional Gambling Motives Questionnaire</span>. <span class=\"ref-journal\">Addiction</span>. 2008;<span class=\"ref-vol\">103</span>: 1110&#x02013;1117. 10.1111/j.1360-0443.2008.02235.x\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n18554344\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1360-0443.2008.02235.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Addiction&amp;title=Development+and+psychometric+evaluation+of+a+three&#x02010;dimensional+Gambling+Motives+Questionnaire&amp;author=SH+Stewart&amp;author=M+Zack&amp;volume=103&amp;publication_year=2008&amp;pages=1110-1117&amp;pmid=18554344&amp;doi=10.1111/j.1360-0443.2008.02235.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref028\">28. <span class=\"mixed-citation\">Ryan RM, Deci EL. <span class=\"ref-title\">Intrinsic and extrinsic motivations: Classic definitions and new directions.</span>\n<span class=\"ref-journal\">Contemp Educ Psychol</span>. 2000;<span class=\"ref-vol\">25</span>: 54&#x02013;67. 10.1006/ceps.1999.1020\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n10620381\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1006%2Fceps.1999.1020\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Contemp+Educ+Psychol&amp;title=Intrinsic+and+extrinsic+motivations:+Classic+definitions+and+new+directions.&amp;author=RM+Ryan&amp;author=EL+Deci&amp;volume=25&amp;publication_year=2000&amp;pages=54-67&amp;pmid=10620381&amp;doi=10.1006/ceps.1999.1020&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref029\">29. <span class=\"mixed-citation\">Blumler J, Katz E. <span class=\"ref-journal\">The uses of mass communications: Current perspectives on gratifications</span>. 1st ed. Beverly Hills: Sage; 1974. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=The+uses+of+mass+communications:+Current+perspectives+on+gratifications&amp;author=J+Blumler&amp;author=E+Katz&amp;publication_year=1974&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref030\">30. <span class=\"mixed-citation\">Bandura A. <span class=\"ref-journal\">Social foundations of thought and action: A social cognitive theory</span>. 1st ed. Englewood Cliffs: Prentice-Hall; 1986. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Social+foundations+of+thought+and+action:+A+social+cognitive+theory&amp;author=A+Bandura&amp;publication_year=1986&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref031\">31. <span class=\"mixed-citation\">Ryan RM, Rigby CS, Przybylski A. <span class=\"ref-title\">The motivational pull of video games: A self-determination theory approach.</span>\n<span class=\"ref-journal\">Motiv Emot.</span>\n2006;<span class=\"ref-vol\">30</span>: 344&#x02013;360. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Motiv+Emot.&amp;title=The+motivational+pull+of+video+games:+A+self-determination+theory+approach.&amp;author=RM+Ryan&amp;author=CS+Rigby&amp;author=A+Przybylski&amp;volume=30&amp;publication_year=2006&amp;pages=344-360&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref032\">32. <span class=\"mixed-citation\">Sherry JL, Lucas K, Greenberg BS, Lachlan K. <span class=\"ref-journal\">Video game uses and gratifications as predictors of use and game preference.</span> In: Vorderer P, Bryant J, editors. Playing computer games: Motives, responses, and consequences.\nMahwah: Lawrence Erlbaum; 2006.  pp. 213&#x02013;224. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Video+game+uses+and+gratifications+as+predictors+of+use+and+game+preference.&amp;author=JL+Sherry&amp;author=K+Lucas&amp;author=BS+Greenberg&amp;author=K+Lachlan&amp;author=P+Vorderer&amp;publication_year=2006&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref033\">33. <span class=\"mixed-citation\">De Grove F, Cauberghe V, Van Looy J. <span class=\"ref-title\">Development and validation of an instrument for measuring individual motives for playing digital games</span>. <span class=\"ref-journal\">Media Psychol</span>. 2016;<span class=\"ref-vol\">19</span>: 101&#x02013;125. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Media+Psychol&amp;title=Development+and+validation+of+an+instrument+for+measuring+individual+motives+for+playing+digital+games&amp;author=F+De+Grove&amp;author=V+Cauberghe&amp;author=J+Van+Looy&amp;volume=19&amp;publication_year=2016&amp;pages=101-125&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref034\">34. <span class=\"mixed-citation\">Yee N. <span class=\"ref-title\">Motivations for play in online games.</span>\n<span class=\"ref-journal\">Cyberpsychol Behav.</span>\n2006;<span class=\"ref-vol\">9</span>: 772&#x02013;775. 10.1089/cpb.2006.9.772\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17201605\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1089%2Fcpb.2006.9.772\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cyberpsychol+Behav.&amp;title=Motivations+for+play+in+online+games.&amp;author=N+Yee&amp;volume=9&amp;publication_year=2006&amp;pages=772-775&amp;pmid=17201605&amp;doi=10.1089/cpb.2006.9.772&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref035\">35. <span class=\"mixed-citation\">Yee N. <span class=\"ref-title\">The demographics, motivations, and derived experiences of users of massively multi-user online graphical environments.</span>\n<span class=\"ref-journal\">Presence (Camb).</span>\n2006;<span class=\"ref-vol\">15</span>: 309&#x02013;329. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Presence+(Camb).&amp;title=The+demographics,+motivations,+and+derived+experiences+of+users+of+massively+multi-user+online+graphical+environments.&amp;author=N+Yee&amp;volume=15&amp;publication_year=2006&amp;pages=309-329&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref036\">36. <span class=\"mixed-citation\">Demetrovics Z, Urb&#x000e1;n R, Nagygy&#x000f6;rgy K, Farkas J, Zilahy D, Merv&#x000f3; B, et al.\n<span class=\"ref-title\">Why do you play? The development of the motives for online gaming questionnaire (MOGQ).</span>\n<span class=\"ref-journal\">Behav Res Methods.</span>\n2011;<span class=\"ref-vol\">43</span>, 814&#x02013;825. 10.3758/s13428-011-0091-y\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21487899\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3758%2Fs13428-011-0091-y\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Behav+Res+Methods.&amp;title=Why+do+you+play?+The+development+of+the+motives+for+online+gaming+questionnaire+(MOGQ).&amp;author=Z+Demetrovics&amp;author=R+Urb&#x000e1;n&amp;author=K+Nagygy&#x000f6;rgy&amp;author=J+Farkas&amp;author=D+Zilahy&amp;volume=43&amp;publication_year=2011&amp;pages=814-825&amp;pmid=21487899&amp;doi=10.3758/s13428-011-0091-y&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref037\">37. <span class=\"mixed-citation\">Billieux J, Deleuze J, Griffiths MD, Kuss DJ. <span class=\"ref-journal\">Internet gaming addiction: The case of massively multiplayer online role-playing games. In: Textbook of addiction treatment: International perspectives</span>. Milan: Springer: 2015.  pp. 1515&#x02013;1525. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Internet+gaming+addiction:+The+case+of+massively+multiplayer+online+role-playing+games.+In:+Textbook+of+addiction+treatment:+International+perspectives&amp;author=J+Billieux&amp;author=J+Deleuze&amp;author=MD+Griffiths&amp;author=DJ+Kuss&amp;publication_year=2015&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref038\">38. <span class=\"mixed-citation\">Subramaniam M, Chua BY, Abdin E, Pang S, Satghare P, Vaingankar JA, et al.\n<span class=\"ref-title\">Prevalence and correlates of internet gaming problem among internet users: Results from an internet survey</span>. <span class=\"ref-journal\">Ann Acad Med Singapore</span>. 2016;<span class=\"ref-vol\">45</span>: 174&#x02013;83.  [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27383716\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Ann+Acad+Med+Singapore&amp;title=Prevalence+and+correlates+of+internet+gaming+problem+among+internet+users:+Results+from+an+internet+survey&amp;author=M+Subramaniam&amp;author=BY+Chua&amp;author=E+Abdin&amp;author=S+Pang&amp;author=P+Satghare&amp;volume=45&amp;publication_year=2016&amp;pages=174-83&amp;pmid=27383716&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref039\">39. <span class=\"mixed-citation\">Thorens G, Achab S, Billieux J, Khazaal Y, Khan R, Pivin E, et al.\n<span class=\"ref-title\">Characteristics and treatment response of self-identified problematic Internet users in a behavioral addiction outpatient clinic</span>. <span class=\"ref-journal\">J Behav Addict</span>. 2013;<span class=\"ref-vol\">3</span>: 78&#x02013;81. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4117276/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25215217\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Behav+Addict&amp;title=Characteristics+and+treatment+response+of+self-identified+problematic+Internet+users+in+a+behavioral+addiction+outpatient+clinic&amp;author=G+Thorens&amp;author=S+Achab&amp;author=J+Billieux&amp;author=Y+Khazaal&amp;author=R+Khan&amp;volume=3&amp;publication_year=2013&amp;pages=78-81&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref040\">40. <span class=\"mixed-citation\">Billieux J, Chanal J, Khazaal Y, Rochat L, Gay P, Zullino D. <span class=\"ref-title\">Psychological predictors of problematic involvement in Massively Multiplayer Online Role Playing Games (MMORPG): Illustration in a sample of male cybercaf&#x000e9;s players.</span>\n<span class=\"ref-journal\">Psychopathology</span>. 2011;<span class=\"ref-vol\">44</span>: 165&#x02013;171. 10.1159/000322525\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21372629\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1159%2F000322525\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychopathology&amp;title=Psychological+predictors+of+problematic+involvement+in+Massively+Multiplayer+Online+Role+Playing+Games+(MMORPG):+Illustration+in+a+sample+of+male+cybercaf&#x000e9;s+players.&amp;author=J+Billieux&amp;author=J+Chanal&amp;author=Y+Khazaal&amp;author=L+Rochat&amp;author=P+Gay&amp;volume=44&amp;publication_year=2011&amp;pages=165-171&amp;pmid=21372629&amp;doi=10.1159/000322525&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref041\">41. <span class=\"mixed-citation\">Billieux J, van der Linden M, Achab S, Khazaal Y, Paraskevopoulos L, Zullino D, et al.\n<span class=\"ref-title\">Why do you play World of Warcraft? An in-depth exploration of self-reported motivations to play online and in-game behaviours in the virtual world of Azeroth.</span>\n<span class=\"ref-journal\">Comput Human Behav.</span>\n2013;<span class=\"ref-vol\">29</span>: 103&#x02013;109. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Human+Behav.&amp;title=Why+do+you+play+World+of+Warcraft?+An+in-depth+exploration+of+self-reported+motivations+to+play+online+and+in-game+behaviours+in+the+virtual+world+of+Azeroth.&amp;author=J+Billieux&amp;author=M+van+der+Linden&amp;author=S+Achab&amp;author=Y+Khazaal&amp;author=L+Paraskevopoulos&amp;volume=29&amp;publication_year=2013&amp;pages=103-109&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref042\">42. <span class=\"mixed-citation\">Williams D, Yee N, Caplan SE. <span class=\"ref-title\">Who plays, how much, and why? Debunking the stereotypical gamer profile</span>. <span class=\"ref-journal\">J Comput Mediat Commun</span>. 2008;<span class=\"ref-vol\">13</span>: 993&#x02013;1018. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Comput+Mediat+Commun&amp;title=Who+plays,+how+much,+and+why?+Debunking+the+stereotypical+gamer+profile&amp;author=D+Williams&amp;author=N+Yee&amp;author=SE+Caplan&amp;volume=13&amp;publication_year=2008&amp;pages=993-1018&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref043\">43. <span class=\"mixed-citation\">Fuster H, Chamarro A, Carbonell X, Vallerand RJ. <span class=\"ref-title\">Relationship between passion and motivation for gaming in players of massively multiplayer online role-playing games.</span>\n<span class=\"ref-journal\">Cyberpsychol Behav Soc Netw</span>. 2014;<span class=\"ref-vol\">17</span>: 292&#x02013;297. 10.1089/cyber.2013.0349\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24611801\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1089%2Fcyber.2013.0349\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cyberpsychol+Behav+Soc+Netw&amp;title=Relationship+between+passion+and+motivation+for+gaming+in+players+of+massively+multiplayer+online+role-playing+games.&amp;author=H+Fuster&amp;author=A+Chamarro&amp;author=X+Carbonell&amp;author=RJ+Vallerand&amp;volume=17&amp;publication_year=2014&amp;pages=292-297&amp;pmid=24611801&amp;doi=10.1089/cyber.2013.0349&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref044\">44. <span class=\"mixed-citation\">Kahn AS, Shen C, Lu L, Ratan RA, Coary S, Hou J, et al.\n<span class=\"ref-title\">The Trojan Player Typology: A cross-genre, cross-cultural, behaviorally validated scale of video game play motivations.</span>\n<span class=\"ref-journal\">Comput Human Behav.</span>\n2015;<span class=\"ref-vol\">49</span>: 354&#x02013;361. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Human+Behav.&amp;title=The+Trojan+Player+Typology:+A+cross-genre,+cross-cultural,+behaviorally+validated+scale+of+video+game+play+motivations.&amp;author=AS+Kahn&amp;author=C+Shen&amp;author=L+Lu&amp;author=RA+Ratan&amp;author=S+Coary&amp;volume=49&amp;publication_year=2015&amp;pages=354-361&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref045\">45. <span class=\"mixed-citation\">Yee N, Ducheneaut N, Nelson L. Online gaming motivations scale: Development and validation. Proceedings of the ACM International Conference on Human Factors in Computing Systems (CHI 2012). New York: ACM; 2012; pp. 2803&#x02013;2806.</span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref046\">46. <span class=\"mixed-citation\">Ghuman D, Griffiths M. <span class=\"ref-title\">A cross-genre study of online Gaming.</span>\n<span class=\"ref-journal\">Int. J. Cyber Behav. Psychol</span>. 2012;<span class=\"ref-vol\">2</span>: 13&#x02013;29. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int.+J.+Cyber+Behav.+Psychol&amp;title=A+cross-genre+study+of+online+Gaming.&amp;author=D+Ghuman&amp;author=M+Griffiths&amp;volume=2&amp;publication_year=2012&amp;pages=13-29&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref047\">47. <span class=\"mixed-citation\">Kim NR, Hwang SSH, Choi JS, Kim DJ, Demetrovics Z, Kir&#x000e1;ly O, et al.\n<span class=\"ref-title\">Characteristics and psychiatric symptoms of internet gaming disorder among adults using self-reported DSM-5 criteria.</span>\n<span class=\"ref-journal\">Psychiatry Investig.</span>\n2016;<span class=\"ref-vol\">13</span>: 58&#x02013;66. 10.4306/pi.2016.13.1.58\n <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4701686/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26766947\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.4306%2Fpi.2016.13.1.58\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychiatry+Investig.&amp;title=Characteristics+and+psychiatric+symptoms+of+internet+gaming+disorder+among+adults+using+self-reported+DSM-5+criteria.&amp;author=NR+Kim&amp;author=SSH+Hwang&amp;author=JS+Choi&amp;author=DJ+Kim&amp;author=Z+Demetrovics&amp;volume=13&amp;publication_year=2016&amp;pages=58-66&amp;pmid=26766947&amp;doi=10.4306/pi.2016.13.1.58&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref048\">48. <span class=\"mixed-citation\">Chang SM, Lin SS. <span class=\"ref-title\">Online gaming motive profiles in late adolescence and the related longitudinal development of stress, depression, and problematic internet use.</span>\n<span class=\"ref-journal\">Comput Educ.</span>\n2019;<span class=\"ref-vol\">135</span>: 123&#x02013;137. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Educ.&amp;title=Online+gaming+motive+profiles+in+late+adolescence+and+the+related+longitudinal+development+of+stress,+depression,+and+problematic+internet+use.&amp;author=SM+Chang&amp;author=SS+Lin&amp;volume=135&amp;publication_year=2019&amp;pages=123-137&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref049\">49. <span class=\"mixed-citation\">Kir&#x000e1;ly O, T&#x000f3;th D, Urb&#x000e1;n R, Demetrovics Z, Maraz A. <span class=\"ref-title\">Intense video gaming is not essentially problematic.</span>\n<span class=\"ref-journal\">Psychol Addict Behav</span>. 2017;<span class=\"ref-vol\">31</span>: 807&#x02013;817. 10.1037/adb0000316\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28956935\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1037%2Fadb0000316\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychol+Addict+Behav&amp;title=Intense+video+gaming+is+not+essentially+problematic.&amp;author=O+Kir&#x000e1;ly&amp;author=D+T&#x000f3;th&amp;author=R+Urb&#x000e1;n&amp;author=Z+Demetrovics&amp;author=A+Maraz&amp;volume=31&amp;publication_year=2017&amp;pages=807-817&amp;pmid=28956935&amp;doi=10.1037/adb0000316&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref050\">50. <span class=\"mixed-citation\">Laconi S, Pir&#x000e8;s S, Chabrol H. <span class=\"ref-title\">Internet gaming disorder, motives, game genres and psychopathology.</span>\n<span class=\"ref-journal\">Comput Human Behav.</span>\n2017;<span class=\"ref-vol\">75</span>: 652&#x02013;659. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Human+Behav.&amp;title=Internet+gaming+disorder,+motives,+game+genres+and+psychopathology.&amp;author=S+Laconi&amp;author=S+Pir&#x000e8;s&amp;author=H+Chabrol&amp;volume=75&amp;publication_year=2017&amp;pages=652-659&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref051\">51. <span class=\"mixed-citation\">M&#x000e4;nnikk&#x000f6; N, Billieux J, Nordstr&#x000f6;m T, Koivisto K, K&#x000e4;&#x000e4;ri&#x000e4;inen M. <span class=\"ref-title\">Problematic gaming behaviour in Finnish adolescents and young adults: Relation to game genres, gaming motives and self-awareness of problematic use.</span>\n<span class=\"ref-journal\">Int J Ment Health Addict.</span>\n2017;<span class=\"ref-vol\">15</span>: 324&#x02013;338. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int+J+Ment+Health+Addict.&amp;title=Problematic+gaming+behaviour+in+Finnish+adolescents+and+young+adults:+Relation+to+game+genres,+gaming+motives+and+self-awareness+of+problematic+use.&amp;author=N+M&#x000e4;nnikk&#x000f6;&amp;author=J+Billieux&amp;author=T+Nordstr&#x000f6;m&amp;author=K+Koivisto&amp;author=M+K&#x000e4;&#x000e4;ri&#x000e4;inen&amp;volume=15&amp;publication_year=2017&amp;pages=324-338&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref052\">52. <span class=\"mixed-citation\">Montag C, Schivinski B, Sariyska R, Kannen C, Demetrovics Z, Pontes HM. <span class=\"ref-title\">Psychopathological symptoms and gaming motives in disordered gaming&#x02014;A psychometric comparison between the WHO and APA diagnostic frameworks.</span>\n<span class=\"ref-journal\">J Clin Med</span>. 2019;<span class=\"ref-vol\">8</span>: 1691&#x02013;1709. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC6832511/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n31618950\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Clin+Med&amp;title=Psychopathological+symptoms+and+gaming+motives+in+disordered+gaming&#x02014;A+psychometric+comparison+between+the+WHO+and+APA+diagnostic+frameworks.&amp;author=C+Montag&amp;author=B+Schivinski&amp;author=R+Sariyska&amp;author=C+Kannen&amp;author=Z+Demetrovics&amp;volume=8&amp;publication_year=2019&amp;pages=1691-1709&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref053\">53. <span class=\"mixed-citation\">Wu AM, Lai MH, Yu S, Lau JT, Lei MW. <span class=\"ref-title\">Motives for online gaming questionnaire: Its psychometric properties and correlation with Internet gaming disorder symptoms among Chinese people</span>. <span class=\"ref-journal\">J Behav Addict</span>. 2016;<span class=\"ref-vol\">6</span>: 11&#x02013;20. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5572999/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28264590\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Behav+Addict&amp;title=Motives+for+online+gaming+questionnaire:+Its+psychometric+properties+and+correlation+with+Internet+gaming+disorder+symptoms+among+Chinese+people&amp;author=AM+Wu&amp;author=MH+Lai&amp;author=S+Yu&amp;author=JT+Lau&amp;author=MW+Lei&amp;volume=6&amp;publication_year=2016&amp;pages=11-20&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref054\">54. <span class=\"mixed-citation\">Ballabio M, Griffiths MD, Urb&#x000e1;n R, Quartiroli A, Demetrovics Z, Kir&#x000e1;ly O. <span class=\"ref-title\">Do gaming motives mediate between psychiatric symptoms and problematic gaming? An empirical survey study.</span>\n<span class=\"ref-journal\"><em>Addict</em> Res <em>Theory</em></span>. 2017;<span class=\"ref-vol\">25</span>: 397&#x02013;408. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Addict+Res+Theory&amp;title=Do+gaming+motives+mediate+between+psychiatric+symptoms+and+problematic+gaming?+An+empirical+survey+study.&amp;author=M+Ballabio&amp;author=MD+Griffiths&amp;author=R+Urb&#x000e1;n&amp;author=A+Quartiroli&amp;author=Z+Demetrovics&amp;volume=25&amp;publication_year=2017&amp;pages=397-408&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref055\">55. <span class=\"mixed-citation\">Kir&#x000e1;ly O, Urb&#x000e1;n R, Griffiths MD, &#x000c1;goston C, Nagygy&#x000f6;rgy K, K&#x000f6;k&#x000f6;nyei G, et al.\n<span class=\"ref-title\">The mediating effect of gaming motivation between psychiatric symptoms and problematic online gaming: An online survey</span>. <span class=\"ref-journal\">J Med Internet Res</span>. 2015;<span class=\"ref-vol\">17</span>: 188&#x02013;202. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4405620/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25855558\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Med+Internet+Res&amp;title=The+mediating+effect+of+gaming+motivation+between+psychiatric+symptoms+and+problematic+online+gaming:+An+online+survey&amp;author=O+Kir&#x000e1;ly&amp;author=R+Urb&#x000e1;n&amp;author=MD+Griffiths&amp;author=C+&#x000c1;goston&amp;author=K+Nagygy&#x000f6;rgy&amp;volume=17&amp;publication_year=2015&amp;pages=188-202&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref056\">56. <span class=\"mixed-citation\">Plante CN, Gentile DA, Groves CL, Modlin A, Blanco-Herrera J. <span class=\"ref-title\">Video games as coping mechanisms in the etiology of video game addiction.</span>\n<span class=\"ref-journal\">Psychol. Pop. Media Cult</span>. 2019;<span class=\"ref-vol\">8</span>: 385&#x02013;394. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychol.+Pop.+Media+Cult&amp;title=Video+games+as+coping+mechanisms+in+the+etiology+of+video+game+addiction.&amp;author=CN+Plante&amp;author=DA+Gentile&amp;author=CL+Groves&amp;author=A+Modlin&amp;author=J+Blanco-Herrera&amp;volume=8&amp;publication_year=2019&amp;pages=385-394&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref057\">57. <span class=\"mixed-citation\">Greenberg BS, Sherry J, Lachlan K, Lucas K, Holmstrom A. <span class=\"ref-title\">Orientations to video games among gender and age groups.</span>\n<span class=\"ref-journal\">Simul Gaming.</span>\n2010;<span class=\"ref-vol\">41</span>: 238&#x02013;259. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Simul+Gaming.&amp;title=Orientations+to+video+games+among+gender+and+age+groups.&amp;author=BS+Greenberg&amp;author=J+Sherry&amp;author=K+Lachlan&amp;author=K+Lucas&amp;author=A+Holmstrom&amp;volume=41&amp;publication_year=2010&amp;pages=238-259&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref058\">58. <span class=\"mixed-citation\">Scharkow M, Festl R, Vogelgesang J, Quandt T. <span class=\"ref-title\">Beyond the &#x0201c;core-gamer&#x0201d;: Genre preferences and gratifications in computer games.</span>\n<span class=\"ref-journal\">Comput Human Behav</span>. 2015;<span class=\"ref-vol\">44</span>: 293&#x02013;298. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Human+Behav&amp;title=Beyond+the+&#x0201c;core-gamer&#x0201d;:+Genre+preferences+and+gratifications+in+computer+games.&amp;author=M+Scharkow&amp;author=R+Festl&amp;author=J+Vogelgesang&amp;author=T+Quandt&amp;volume=44&amp;publication_year=2015&amp;pages=293-298&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref059\">59. <span class=\"mixed-citation\">Hilgard J, Engelhardt CR, Bartholow BD. <span class=\"ref-title\">Individual differences in motives, preferences, and pathology in video games: The gaming attitudes, motives, and experiences scales (GAMES).</span>\n<span class=\"ref-journal\">Front Psychol</span>. 2013;<span class=\"ref-vol\">4</span>: 608\n10.3389/fpsyg.2013.00608\n <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3766857/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24058355\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3389%2Ffpsyg.2013.00608\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front+Psychol&amp;title=Individual+differences+in+motives,+preferences,+and+pathology+in+video+games:+The+gaming+attitudes,+motives,+and+experiences+scales+(GAMES).&amp;author=J+Hilgard&amp;author=CR+Engelhardt&amp;author=BD+Bartholow&amp;volume=4&amp;publication_year=2013&amp;pages=608&amp;pmid=24058355&amp;doi=10.3389/fpsyg.2013.00608&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref060\">60. <span class=\"mixed-citation\">Nije Bijvank M, Konijn EA, Bushman BJ. <span class=\"ref-title\">&#x0201c;We don&#x02019;t need no education&#x0201d;: Video game preferences, video game motivations, and aggressiveness among adolescent boys of different educational ability levels.</span>\n<span class=\"ref-journal\">J Adolesc.</span>\n2012;<span class=\"ref-vol\">35</span>: 153&#x02013;162. 10.1016/j.adolescence.2011.04.001\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21529925\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.adolescence.2011.04.001\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Adolesc.&amp;title=&#x0201c;We+don&#x02019;t+need+no+education&#x0201d;:+Video+game+preferences,+video+game+motivations,+and+aggressiveness+among+adolescent+boys+of+different+educational+ability+levels.&amp;author=M+Nije+Bijvank&amp;author=EA+Konijn&amp;author=BJ+Bushman&amp;volume=35&amp;publication_year=2012&amp;pages=153-162&amp;pmid=21529925&amp;doi=10.1016/j.adolescence.2011.04.001&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref061\">61. <span class=\"mixed-citation\">Fuster H, Oberst U, Griffiths M, Carbonell X, Chamarro A, Talarn A. <span class=\"ref-title\">Psychological motivation in online role-playing games: A study of Spanish World of Warcraft players.</span>\n<span class=\"ref-journal\">An Psicol-Spain</span>. 2012;<span class=\"ref-vol\">28</span>: 274&#x02013;280. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=An+Psicol-Spain&amp;title=Psychological+motivation+in+online+role-playing+games:+A+study+of+Spanish+World+of+Warcraft+players.&amp;author=H+Fuster&amp;author=U+Oberst&amp;author=M+Griffiths&amp;author=X+Carbonell&amp;author=A+Chamarro&amp;volume=28&amp;publication_year=2012&amp;pages=274-280&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref062\">62. <span class=\"mixed-citation\">Calvert SL, Appelbaum M, Dodge KA, Graham S, Nagayama Hall GC, Hamby S. <span class=\"ref-title\">The American Psychological Association Task Force assessment of violent: Science in the service of public interest</span>. <span class=\"ref-journal\">Am. Psychol</span>. 2017;<span class=\"ref-vol\">72</span>: 126&#x02013;143. 10.1037/a0040413\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28221065\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1037%2Fa0040413\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Am.+Psychol&amp;title=The+American+Psychological+Association+Task+Force+assessment+of+violent:+Science+in+the+service+of+public+interest&amp;author=SL+Calvert&amp;author=M+Appelbaum&amp;author=KA+Dodge&amp;author=S+Graham&amp;author=GC+Nagayama+Hall&amp;volume=72&amp;publication_year=2017&amp;pages=126-143&amp;pmid=28221065&amp;doi=10.1037/a0040413&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref063\">63. <span class=\"mixed-citation\">Cooper ML, Kuntsche E, Levitt A, Barber LL, Wolf S. <span class=\"ref-journal\">Motivational models of substance use: A review of theory and research on motives for using alcohol, marijuana, and tobacco.</span> In: Sher KJ, editor. <em>Oxford library of psychology</em>. <em>The Oxford handbook of substance use and substance use disorders</em>\nOxford: Oxford University Press; 2016.  pp. 375&#x02013;421. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Motivational+models+of+substance+use:+A+review+of+theory+and+research+on+motives+for+using+alcohol,+marijuana,+and+tobacco.&amp;author=ML+Cooper&amp;author=E+Kuntsche&amp;author=A+Levitt&amp;author=LL+Barber&amp;author=S+Wolf&amp;publication_year=2016&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref064\">64. <span class=\"mixed-citation\">Blanch A, Aluja A. <span class=\"ref-title\">Text mining a self-report back-translation.</span>\n<span class=\"ref-journal\">Psychol Assess.</span>\n2016;<span class=\"ref-vol\">28</span>: 750&#x02013;764. 10.1037/pas0000213\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26302100\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1037%2Fpas0000213\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychol+Assess.&amp;title=Text+mining+a+self-report+back-translation.&amp;author=A+Blanch&amp;author=A+Aluja&amp;volume=28&amp;publication_year=2016&amp;pages=750-764&amp;pmid=26302100&amp;doi=10.1037/pas0000213&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref065\">65. <span class=\"mixed-citation\">Geisinger KF. <span class=\"ref-title\">Cross-cultural normative assessment: Translation and adaptation issues influencing the normative interpretation of assessment instruments.</span>\n<span class=\"ref-journal\">Psychol Assess.</span>\n1994;<span class=\"ref-vol\">6</span>: 304&#x02013;312. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychol+Assess.&amp;title=Cross-cultural+normative+assessment:+Translation+and+adaptation+issues+influencing+the+normative+interpretation+of+assessment+instruments.&amp;author=KF+Geisinger&amp;volume=6&amp;publication_year=1994&amp;pages=304-312&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref066\">66. <span class=\"mixed-citation\">Watkins MW. <span class=\"ref-title\">Determining parallel analysis criteria.</span>\n<span class=\"ref-journal\">J Mod Appl Stat Methods</span>. 2006;<span class=\"ref-vol\">5</span>: 344&#x02013;346. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Mod+Appl+Stat+Methods&amp;title=Determining+parallel+analysis+criteria.&amp;author=MW+Watkins&amp;volume=5&amp;publication_year=2006&amp;pages=344-346&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref067\">67. <span class=\"mixed-citation\">Lautenschlager GJ. <span class=\"ref-title\">A comparison of alternatives to conducting Monte Carlo analyses for determining parallel analysis criteria.</span>\n<span class=\"ref-journal\">Multivariate Behav Res</span>.1989;<span class=\"ref-vol\">24</span>: 365&#x02013;395. 10.1207/s15327906mbr2403_6\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26750503\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1207%2Fs15327906mbr2403_6\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Multivariate+Behav+Res&amp;title=A+comparison+of+alternatives+to+conducting+Monte+Carlo+analyses+for+determining+parallel+analysis+criteria.&amp;author=GJ+Lautenschlager&amp;volume=24&amp;publication_year=1989&amp;pages=365-395&amp;pmid=26750503&amp;doi=10.1207/s15327906mbr2403_6&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref068\">68. <span class=\"mixed-citation\">Cliff N. <span class=\"ref-title\">The eigenvalues-greater-than-one rule and the reliability of components.</span>\n<span class=\"ref-journal\">Psychol Bull</span>. 1988;<span class=\"ref-vol\">103</span>: 276&#x02013;279. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychol+Bull&amp;title=The+eigenvalues-greater-than-one+rule+and+the+reliability+of+components.&amp;author=N+Cliff&amp;volume=103&amp;publication_year=1988&amp;pages=276-279&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref069\">69. <span class=\"mixed-citation\">Moya-Higueras J, Cuevas A, Marques-Feixa L, Mezquita L, Mayoral M, Fa&#x000f1;an&#x000e1;s L. <span class=\"ref-title\">Recent stressful life events (SLE) and adolescent mental health: initial validation of the LEIA, a new Checklist for SLE assessment according to their severity, interpersonal, and dependent nature.</span>\n<span class=\"ref-journal\">Assessment</span>. 2018 Epub ahead of print. 10.1177/1073191118817648. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n30539643\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1177%2F1073191118817648\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Assessment&amp;title=Recent+stressful+life+events+(SLE)+and+adolescent+mental+health:+initial+validation+of+the+LEIA,+a+new+Checklist+for+SLE+assessment+according+to+their+severity,+interpersonal,+and+dependent+nature.&amp;author=J+Moya-Higueras&amp;author=A+Cuevas&amp;author=L+Marques-Feixa&amp;author=L+Mezquita&amp;author=M+Mayoral&amp;doi=10.1177/1073191118817648&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref070\">70. <span class=\"mixed-citation\">Gentile D. <span class=\"ref-title\">Pathological video-game use among youth ages 8 to 18: A national study.</span>\n<span class=\"ref-journal\">Psychol Sci</span>. 2009;<span class=\"ref-vol\">20</span>: 594&#x02013;602. 10.1111/j.1467-9280.2009.02340.x\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19476590\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1467-9280.2009.02340.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychol+Sci&amp;title=Pathological+video-game+use+among+youth+ages+8+to+18:+A+national+study.&amp;author=D+Gentile&amp;volume=20&amp;publication_year=2009&amp;pages=594-602&amp;pmid=19476590&amp;doi=10.1111/j.1467-9280.2009.02340.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref071\">71. <span class=\"mixed-citation\">DeVellis RF. <span class=\"ref-journal\">Scale development: Theory and applications</span>. 1st ed. London: Sage; 2016. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Scale+development:+Theory+and+applications&amp;author=RF+DeVellis&amp;publication_year=2016&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref072\">72. <span class=\"mixed-citation\">Fuster H, Carbonell X, Pontes HM, Griffiths MD. <span class=\"ref-title\">Spanish validation of the internet gaming disorder-20 (IGD-20) test.</span>\n<span class=\"ref-journal\">Comput Human Behav.</span>\n2016;<span class=\"ref-vol\">56</span>: 215&#x02013;224. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Human+Behav.&amp;title=Spanish+validation+of+the+internet+gaming+disorder-20+(IGD-20)+test.&amp;author=H+Fuster&amp;author=X+Carbonell&amp;author=HM+Pontes&amp;author=MD+Griffiths&amp;volume=56&amp;publication_year=2016&amp;pages=215-224&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref073\">73. <span class=\"mixed-citation\">Bentler PM. <span class=\"ref-journal\">EQS structural equations program manual</span>. 6th ed. Los Angeles: BDMP Statistical Software; 1995. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=EQS+structural+equations+program+manual&amp;author=PM+Bentler&amp;publication_year=1995&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref074\">74. <span class=\"mixed-citation\">Byrne BM. <span class=\"ref-journal\">Structural equation modeling with EQS: Basic concepts, applications, and programming.</span>\n1st ed. London: Routledge; 2013. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Structural+equation+modeling+with+EQS:+Basic+concepts,+applications,+and+programming.&amp;author=BM+Byrne&amp;publication_year=2013&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref075\">75. <span class=\"mixed-citation\">Cox WM, Klinger E. <span class=\"ref-title\">A motivational model of alcohol use</span>. <span class=\"ref-journal\">J Abnorm Psychol</span>. 1988;<span class=\"ref-vol\">97</span>: 168&#x02013;180. 10.1037//0021-843x.97.2.168\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n3290306\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1037%2F%2F0021-843x.97.2.168\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Abnorm+Psychol&amp;title=A+motivational+model+of+alcohol+use&amp;author=WM+Cox&amp;author=E+Klinger&amp;volume=97&amp;publication_year=1988&amp;pages=168-180&amp;pmid=3290306&amp;doi=10.1037//0021-843x.97.2.168&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref076\">76. <span class=\"mixed-citation\">Mezquita L, Stewart SH, Ib&#x000e1;&#x000f1;ez MI, Ruip&#x000e9;rez MA, Villa H, Moya J, et al.\n<span class=\"ref-title\">Drinking motives in clinical and general populations.</span>\n<span class=\"ref-journal\">Eur Addict Res</span>. 2011;<span class=\"ref-vol\">17</span>: 250&#x02013;261. 10.1159/000328510\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21691118\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1159%2F000328510\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Eur+Addict+Res&amp;title=Drinking+motives+in+clinical+and+general+populations.&amp;author=L+Mezquita&amp;author=SH+Stewart&amp;author=MI+Ib&#x000e1;&#x000f1;ez&amp;author=MA+Ruip&#x000e9;rez&amp;author=H+Villa&amp;volume=17&amp;publication_year=2011&amp;pages=250-261&amp;pmid=21691118&amp;doi=10.1159/000328510&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref077\">77. <span class=\"mixed-citation\">Moudiab S, Spada MM. <span class=\"ref-title\">The relative contribution of motives and maladaptive cognitions to levels of Internet Gaming Disorder</span>. <span class=\"ref-journal\">Addict Behav Rep. 219</span>;<span class=\"ref-vol\">9</span>: 100&#x02013;160. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC6348280/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n30705935\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Addict+Behav+Rep.+219&amp;title=The+relative+contribution+of+motives+and+maladaptive+cognitions+to+levels+of+Internet+Gaming+Disorder&amp;author=S+Moudiab&amp;author=MM+Spada&amp;volume=9&amp;pages=100-160&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref078\">78. <span class=\"mixed-citation\">Li H, Zou Y, Wang J, Yang X. <span class=\"ref-title\">Role of stressful life events, avoidant coping styles, and neuroticism in online game addiction among college students: a moderated mediation model.</span>\n<span class=\"ref-journal\">Front. Psychol.</span>\n2016;<span class=\"ref-vol\">7</span>: 1&#x02013;11\n10.3389/fpsyg.2016.00001\n <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5118950/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27920734\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3389%2Ffpsyg.2016.00001\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front.+Psychol.&amp;title=Role+of+stressful+life+events,+avoidant+coping+styles,+and+neuroticism+in+online+game+addiction+among+college+students:+a+moderated+mediation+model.&amp;author=H+Li&amp;author=Y+Zou&amp;author=J+Wang&amp;author=X+Yang&amp;volume=7&amp;publication_year=2016&amp;pages=1-11&amp;pmid=26858668&amp;doi=10.3389/fpsyg.2016.00001&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref079\">79. <span class=\"mixed-citation\">Ferguson CJ, Olson CK. <span class=\"ref-title\">Friends, fun, frustration and fantasy: Child motivations for video game play.</span>\n<span class=\"ref-journal\">Motiv Emot</span>. 2013;<span class=\"ref-vol\">37</span>: 154&#x02013;164. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Motiv+Emot&amp;title=Friends,+fun,+frustration+and+fantasy:+Child+motivations+for+video+game+play.&amp;author=CJ+Ferguson&amp;author=CK+Olson&amp;volume=37&amp;publication_year=2013&amp;pages=154-164&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref080\">80. <span class=\"mixed-citation\">Rehbein F, Kleimann M, M&#x000f6;&#x000df;le T. <span class=\"ref-title\">Prevalence and risk factors of video game dependency in adolescence: Results of a German nationwide survey.</span>\n<span class=\"ref-journal\">Cyberpsychol Behav Soc Netw.</span>\n2010;<span class=\"ref-vol\">13</span>: 269&#x02013;277. 10.1089/cyber.2009.0227\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20557246\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1089%2Fcyber.2009.0227\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cyberpsychol+Behav+Soc+Netw.&amp;title=Prevalence+and+risk+factors+of+video+game+dependency+in+adolescence:+Results+of+a+German+nationwide+survey.&amp;author=F+Rehbein&amp;author=M+Kleimann&amp;author=T+M&#x000f6;&#x000df;le&amp;volume=13&amp;publication_year=2010&amp;pages=269-277&amp;pmid=20557246&amp;doi=10.1089/cyber.2009.0227&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref081\">81. <span class=\"mixed-citation\">Hussain Z, Williams GA, Griffiths MD. <span class=\"ref-title\">An exploratory study of the association between online gaming addiction and enjoyment motivations for playing massively multiplayer online role-playing games.</span>\n<span class=\"ref-journal\">Comput Human Behav.</span>\n2015;<span class=\"ref-vol\">50</span>: 221&#x02013;230. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Human+Behav.&amp;title=An+exploratory+study+of+the+association+between+online+gaming+addiction+and+enjoyment+motivations+for+playing+massively+multiplayer+online+role-playing+games.&amp;author=Z+Hussain&amp;author=GA+Williams&amp;author=MD+Griffiths&amp;volume=50&amp;publication_year=2015&amp;pages=221-230&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref082\">82. <span class=\"mixed-citation\">Sobell LC, Sobell MB. Timeline followback. In: Litten R, Allen J, editors. Measuring alcohol consumption.\nTotowa: Humana Press; 1992.  pp. 41&#x02013;72. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar?q=Sobell+LC+,++Sobell+MB+.++Timeline+followback.++In:++Litten+R+,++Allen+J+,+editors.++Measuring+alcohol+consumption.+&#10;+Totowa+:++Humana+Press+;++1992++pp.++41+&#x02013;+72+.+\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref083\">83. <span class=\"mixed-citation\">Kircaburun K, Jonason PK, Griffiths MD. <span class=\"ref-title\">The Dark Tetrad traits and problematic online gaming: The mediating role of online gaming motives and moderating role of game types.</span>\n<span class=\"ref-journal\">Pers Individ Dif.</span>\n2018;<span class=\"ref-vol\">135</span>: 298&#x02013;303. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Pers+Individ+Dif.&amp;title=The+Dark+Tetrad+traits+and+problematic+online+gaming:+The+mediating+role+of+online+gaming+motives+and+moderating+role+of+game+types.&amp;author=K+Kircaburun&amp;author=PK+Jonason&amp;author=MD+Griffiths&amp;volume=135&amp;publication_year=2018&amp;pages=298-303&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref084\">84. <span class=\"mixed-citation\">Steadman JL. <span class=\"ref-title\">Gaming motivations as a guide to treating problematic gaming behaviors.</span>\n<span class=\"ref-journal\">J Technol Behav Sci</span>. 2019;<span class=\"ref-vol\">4</span>: 332&#x02013;339. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Technol+Behav+Sci&amp;title=Gaming+motivations+as+a+guide+to+treating+problematic+gaming+behaviors.&amp;author=JL+Steadman&amp;volume=4&amp;publication_year=2019&amp;pages=332-339&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref085\">85. <span class=\"mixed-citation\">Floros G, Siomos K. <span class=\"ref-title\">Patterns of choices on video game genres and internet addiction</span>. <span class=\"ref-journal\">Cyberpsychol Behav Soc Netw</span>. 2012;<span class=\"ref-vol\">15</span>: 417&#x02013;424. 10.1089/cyber.2012.0064\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22823448\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1089%2Fcyber.2012.0064\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cyberpsychol+Behav+Soc+Netw&amp;title=Patterns+of+choices+on+video+game+genres+and+internet+addiction&amp;author=G+Floros&amp;author=K+Siomos&amp;volume=15&amp;publication_year=2012&amp;pages=417-424&amp;pmid=22823448&amp;doi=10.1089/cyber.2012.0064&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref086\">86. <span class=\"mixed-citation\">Kim Y, Ross SD. <span class=\"ref-title\">An exploration of motives in sport video gaming.</span>\n<span class=\"ref-journal\">IJSMS</span>. 2006;<span class=\"ref-vol\">8</span>: 28&#x02013;40. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=IJSMS&amp;title=An+exploration+of+motives+in+sport+video+gaming.&amp;author=Y+Kim&amp;author=SD+Ross&amp;volume=8&amp;publication_year=2006&amp;pages=28-40&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref087\">87. <span class=\"mixed-citation\">Lafreni&#x000e8;re MAK, Verner-Filion J, Vallerand RJ. <span class=\"ref-title\">Development and validation of the Gaming Motivation Scale (GAMS).</span>\n<span class=\"ref-journal\">Pers Individ Dif.</span>\n2012;<span class=\"ref-vol\">53</span>: 827&#x02013;831. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Pers+Individ+Dif.&amp;title=Development+and+validation+of+the+Gaming+Motivation+Scale+(GAMS).&amp;author=MAK+Lafreni&#x000e8;re&amp;author=J+Verner-Filion&amp;author=RJ+Vallerand&amp;volume=53&amp;publication_year=2012&amp;pages=827-831&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref088\">88. <span class=\"mixed-citation\">Lee D, LaRose R. <span class=\"ref-title\">A socio-cognitive model of video game usage.</span>\n<span class=\"ref-journal\">J Broadcast Electron Media</span>. 2007;<span class=\"ref-vol\">51</span>: 632&#x02013;650. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Broadcast+Electron+Media&amp;title=A+socio-cognitive+model+of+video+game+usage.&amp;author=D+Lee&amp;author=R+LaRose&amp;volume=51&amp;publication_year=2007&amp;pages=632-650&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref089\">89. <span class=\"mixed-citation\">Li DD, Liau AK, Gentile DA, Khoo A, Cheong WD. <span class=\"ref-title\">Construct and predictive validity of a brief MMO player motivation scale: Cross-sectional and longitudinal evidence based on Singaporean young gamers.</span>\n<span class=\"ref-journal\">J Child Media</span>. 2013;<span class=\"ref-vol\">7</span>, 287&#x02013;306. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Child+Media&amp;title=Construct+and+predictive+validity+of+a+brief+MMO+player+motivation+scale:+Cross-sectional+and+longitudinal+evidence+based+on+Singaporean+young+gamers.&amp;author=DD+Li&amp;author=AK+Liau&amp;author=DA+Gentile&amp;author=A+Khoo&amp;author=WD+Cheong&amp;volume=7&amp;publication_year=2013&amp;pages=287-306&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref090\">90. <span class=\"mixed-citation\">Nacke LE, Bateman C, Mandryk RL. <span class=\"ref-title\">BrainHex: A neurobiological gamer typology survey.</span>\n<span class=\"ref-journal\">Entertain Comput</span>. 2014;<span class=\"ref-vol\">5</span>: 55&#x02013;62. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Entertain+Comput&amp;title=BrainHex:+A+neurobiological+gamer+typology+survey.&amp;author=LE+Nacke&amp;author=C+Bateman&amp;author=RL+Mandryk&amp;volume=5&amp;publication_year=2014&amp;pages=55-62&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref091\">91. <span class=\"mixed-citation\">Olson CK. <span class=\"ref-title\">Children&#x02019;s motivations for video game play in the context of normal development</span>. <span class=\"ref-journal\">Rev Gen Psychol</span>. 2010;<span class=\"ref-vol\">14</span>: 180&#x02013;187. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Rev+Gen+Psychol&amp;title=Children&#x02019;s+motivations+for+video+game+play+in+the+context+of+normal+development&amp;author=CK+Olson&amp;volume=14&amp;publication_year=2010&amp;pages=180-187&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref092\">92. <span class=\"mixed-citation\">Rodr&#x000ed;guez de Sep&#x000fa;lveda D, Igartua JJ. Creaci&#x000f3;n y validaci&#x000f3;n de una escala de motivos para videojugar. In: Ortega F, Carde&#x000f1;osa L, editors. <em>Nuevos medios, nueva comunicaci&#x000f3;n</em>. Salamanca: II Congreso Internacional Comunicaci&#x000f3;n 3.0. 2011. pp. 217&#x02013;234.</span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref093\">93. <span class=\"mixed-citation\">Tondello GF, Wehbe RR, Diamond L, Busch M, Marczewski A, Nacke LE. The gamification user types hexad scale. In: Proceedings of the 2016 annual symposium on computer-human interaction in play. 2016. pp. 229&#x02013;243.</span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref094\">94. <span class=\"mixed-citation\">Wallenius M, Rimpel&#x000e4; A, Punam&#x000e4;ki RL, Lintonen T. <span class=\"ref-title\">Digital game playing motives among adolescents: Relations to parent&#x02013;child communication, school performance, sleeping habits, and perceived health.</span>\n<span class=\"ref-journal\">J Appl Dev Psychol</span>. 2009;<span class=\"ref-vol\">30</span>: 463&#x02013;474. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Appl+Dev+Psychol&amp;title=Digital+game+playing+motives+among+adolescents:+Relations+to+parent&#x02013;child+communication,+school+performance,+sleeping+habits,+and+perceived+health.&amp;author=M+Wallenius&amp;author=A+Rimpel&#x000e4;&amp;author=RL+Punam&#x000e4;ki&amp;author=T+Lintonen&amp;volume=30&amp;publication_year=2009&amp;pages=463-474&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref095\">95. <span class=\"mixed-citation\">Westwood D, Griffiths MD. <span class=\"ref-title\">The role of structural characteristics in video-game play motivation: A Q-methodology study.</span>\n<span class=\"ref-journal\">Cyberpsychol Behav Soc Netw</span>. 2010;<span class=\"ref-vol\">13</span>: 581&#x02013;585. 10.1089/cyber.2009.0361\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20950185\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1089%2Fcyber.2009.0361\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cyberpsychol+Behav+Soc+Netw&amp;title=The+role+of+structural+characteristics+in+video-game+play+motivation:+A+Q-methodology+study.&amp;author=D+Westwood&amp;author=MD+Griffiths&amp;volume=13&amp;publication_year=2010&amp;pages=581-585&amp;pmid=20950185&amp;doi=10.1089/cyber.2009.0361&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span><span class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref096\">96. <span class=\"mixed-citation\">Wu JH, Wang SC, Tsai HH. <span class=\"ref-title\">Falling in love with online games: The uses and gratifications perspective.</span>\n<span class=\"ref-journal\">Comput Human Behav.</span>\n2010;<span class=\"ref-vol\">26</span>: 1862&#x02013;1871. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Human+Behav.&amp;title=Falling+in+love+with+online+games:+The+uses+and+gratifications+perspective.&amp;author=JH+Wu&amp;author=SC+Wang&amp;author=HH+Tsai&amp;volume=26&amp;publication_year=2010&amp;pages=1862-1871&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></span></span></span></span><span class=\"sub-navigation nav-top space small\"><ul class=\"page-breadcrumbs inline_list small\"><li><a href=\"#top\" class=\"navlink\">PLoS One. 2020; 15(10): e0240726. </a></li><span> » </span><li>Decision Letter 0</li></ul></span><span class=\"sub-content half_rhythm\" id=\"pone.0240726.r001\"><span class=\"jig-ncbiinpagenav\" data-jigconfig=\"smoothScroll: false, allHeadingLevels: [\'h2\'], headingExclude: \':hidden,.nomenu\'\"><span class=\"fm-sec half_rhythm no_top_margin\"><span class=\"fm-flexbox\"><span class=\"fm-citation\"><span class=\"part1\"><span class=\"cit-abbr\"> </span><span class=\"cit-pub-date\">2020; </span><span class=\"cit-vol\">15</span><span class=\"cit-iss\">(10)</span><span class=\"cit-flpgs\">: e0240726. </span></span><span class=\"part2\"><span class=\"fm-vol-iss-date\">Published online 2020 Oct 23. </span><span class=\"doi\"><span>doi: </span><a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0240726.r001\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">10.1371/journal.pone.0240726.r001</a></span></span></span></span><h1 class=\"sub-content-title\">Decision Letter 0</h1><span class=\"half_rhythm\"><span class=\"contrib-group fm-author\"><a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Chiesi%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33095762\">Francesca Chiesi</a>, <span class=\"fm-role\">Academic Editor</span></span><span style=\"display:none\" class=\"contrib-group aff-tip\"></span></span><span class=\"fm-panel half_rhythm\"><span class=\"togglers fm-copyright-license\"> <a href=\"#\" class=\"pmctoggle\" rid=\"idm140210194435328_cpl\">Copyright and License information</a> <a href=\"/pmc/about/disclaimer/\" style=\"margin-left: 1em\">Disclaimer</a></span><span class=\"fm-authors-info fm-panel half_rhythm\"><span></span></span><span class=\"fm-article-notes fm-panel half_rhythm\"></span><span class=\"permissions fm-panel half_rhythm hide\" id=\"idm140210194435328_cpl\" style=\"display:none\"><span class=\"fm-copyright half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright</a> &#x000a9; 2020 Francesca Chiesi</span><span class=\"fm-copyright half_rhythm\">This is an open access article distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></span></span><span id=\"idm140210198969856\" class=\"tsec sec\"><h2 class=\"headless nomenu\"></h2><p id=\"__p75\" class=\"p p-first\">\n<em class=\"letter-date\">30 Jul 2020</em>\n</p><p id=\"__p76\">PONE-D-20-12358</p><p id=\"__p77\">The development and validation of the Videogaming Motives Questionnaire (VMQ)</p><p id=\"__p78\">PLOS ONE</p><p id=\"__p79\">Dear Dr. Ib&#x000e1;&#x000f1;ez,</p><p id=\"__p80\">Thank you for submitting your manuscript to PLOS ONE. After careful consideration, we feel that it has merit but does not fully meet PLOS ONE&#x02019;s publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the points raised during the review process.</p><p id=\"__p81\">Please submit your revised manuscript by Sep 13 2020 11:59PM. If you will need more time than this to complete your revisions, please reply to this message or contact the journal office at <a href=\"mailto:dev@null\" data-email=\"gro.solp@enosolp\" class=\"oemail\">gro.solp@enosolp</a>. When you\'re ready to submit your revision, log on to <a href=\"https://www.editorialmanager.com/pone/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://www.editorialmanager.com/pone/</a> and select the \'Submissions Needing Revision\' folder to locate your manuscript file.</p><p id=\"__p82\">Please include the following items when submitting your revised manuscript:</p><ul class=\"unordered\" style=\"list-style-type:disc\"><!--\nlist-behavior=unordered\nprefix-word=\nmark-type=disc\nmax-label-size=0\n--><li><p id=\"__p84\">A rebuttal letter that responds to each point raised by the academic editor and reviewer(s). You should upload this letter as a separate file labeled \'Response to Reviewers\'.</p></li><li><p id=\"__p85\">A marked-up copy of your manuscript that highlights changes made to the original version. You should upload this as a separate file labeled \'Revised Manuscript with Track Changes\'.</p></li><li><p id=\"__p86\">An unmarked version of your revised paper without tracked changes. You should upload this as a separate file labeled \'Manuscript\'.</p></li></ul><p></p><p id=\"__p87\">If you would like to make changes to your financial disclosure, please include your updated statement in your cover letter. Guidelines for resubmitting your figure files are available below the reviewer comments at the end of this letter.</p><p id=\"__p88\">If applicable, we recommend that you deposit your laboratory protocols in protocols.io to enhance the reproducibility of your results. Protocols.io assigns your protocol its own identifier (DOI) so that it can be cited independently in the future. For instructions see:&#x000a0;<a href=\"http://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols</a></p><p id=\"__p89\">We look forward to receiving your revised manuscript.</p><p id=\"__p90\">Kind regards,</p><p id=\"__p91\">Francesca Chiesi</p><p id=\"__p92\">Academic Editor</p><p id=\"__p93\">PLOS ONE</p><p id=\"__p94\">Journal Requirements:</p><p id=\"__p95\">When submitting your revision, we need you to address these additional requirements.</p><p id=\"__p96\">1. Please ensure that your manuscript meets PLOS ONE\'s style requirements, including those for file naming. The PLOS ONE style templates can be found at</p><p id=\"__p97\"><a href=\"https://journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf</a> and</p><p id=\"__p98\"><a href=\"https://journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf</a></p><p id=\"__p99\">2. Please include additional information about your scoping review in your methods section, including search strategy, inclusion and exclusion criteria, to enable reproducibility and replicability.</p><p id=\"__p100\">3. Please proofread your manuscript for typos (for instance item 3 of table 2).</p><p id=\"__p101\">4. Your ethics statement must appear in the Methods section of your manuscript. If your ethics statement is written in any section besides the Methods, please move it to the Methods section and delete it from any other section. Please also ensure that your ethics statement is included in your manuscript, as the ethics section of your online submission will not be published alongside your manuscript.</p><p id=\"__p102\">[Note: HTML markup is below. Please do not edit.]</p><p id=\"__p103\">Reviewers\' comments:</p><p id=\"__p104\">Reviewer\'s Responses to Questions</p><p id=\"__p105\"><strong>Comments to the Author</strong></p><p id=\"__p106\">1. Is the manuscript technically sound, and do the data support the conclusions?</p><p id=\"__p107\">The manuscript must describe a technically sound piece of scientific research with data that supports the conclusions. Experiments must have been conducted rigorously, with appropriate controls, replication, and sample sizes. The conclusions must be drawn appropriately based on the data presented. </p><p id=\"__p108\">Reviewer #1:&#x000a0;Yes</p><p id=\"__p109\">**********</p><p id=\"__p110\">2. Has the statistical analysis been performed appropriately and rigorously? </p><p id=\"__p111\">Reviewer #1:&#x000a0;Yes</p><p id=\"__p112\">**********</p><p id=\"__p113\">3. Have the authors made all data underlying the findings in their manuscript fully available?</p><p id=\"__p114\">The <a href=\"http://www.plosone.org/static/policies.action#sharing\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">PLOS Data policy</a> requires authors to make all data underlying the findings described in their manuscript fully available without restriction, with rare exception (please refer to the Data Availability Statement in the manuscript PDF file). The data should be provided as part of the manuscript or its supporting information, or deposited to a public repository. For example, in addition to summary statistics, the data points behind means, medians and variance measures should be available. If there are restrictions on publicly sharing data&#x02014;e.g. participant privacy or use of data from a third party&#x02014;those must be specified.</p><p id=\"__p115\">Reviewer #1:&#x000a0;Yes</p><p id=\"__p116\">**********</p><p id=\"__p117\">4. Is the manuscript presented in an intelligible fashion and written in standard English?</p><p id=\"__p118\">PLOS ONE does not copyedit accepted manuscripts, so the language in submitted articles must be clear, correct, and unambiguous. Any typographical or grammatical errors should be corrected at revision, so please note any specific errors here.</p><p id=\"__p119\">Reviewer #1:&#x000a0;Yes</p><p id=\"__p120\">**********</p><p id=\"__p121\">5. Review Comments to the Author</p><p id=\"__p122\">Please use the space provided to explain your answers to the questions above. You may also include additional comments for the author, including concerns about dual publication, research ethics, or publication ethics. (Please upload your review as an attachment if it exceeds 20,000 characters)</p><p id=\"__p123\">Reviewer #1:&#x000a0;Ms #: PONE-D-20-12358</p><p id=\"__p124\">Title: The development and validation of the Videogaming Motives Questionnaire (VMQ)</p><p id=\"__p125\">Authors: L&#x000f3;pez F, et al.</p><p id=\"__p126\">Summary:</p><p id=\"__p127\">This study examined the factor structure of a new instrument, the Videogaming Motives Questionnaire, in three samples of Spanish adolescents and young adults. Study 1 used exploratory factor analysis (EFA) to define empirical factors in young adults. The EFA revealed that an 8-factor model best explained the data. This pattern was replicated in Study 2 in a sample of adolescents. Finally, Study 3 used confirmatory factor analysis to demonstrate a selective mapping/loading of items to the 8 respective motive factors. Overall psychometric properties indicated clear simple structure and internal reliability of all factors. The sample size was sufficient to conclude the models are stable. Coping was identified as a key motive linked with gaming duration and with disordered gaming (DG), based on an author-compiled index of DG. Social interaction was also strongly linked with heavier gaming and DG. The latter result may reflect a preference for on-line socializing by people who are not entirely comfortable with direct interpersonal encounters. Violent motives also coincided with DG, while male gender coincided with stronger endorsement of all motives, aside from Customization, and DG, consistent with other addictive behaviors. The Discussion interpreted the findings for each of the key motives and proposed possible reasons for the pattern of effects.</p><p id=\"__p128\">Comments/Questions:</p><p id=\"__p129\">1. Ms p. 3, line 51</p><p id=\"__p130\">The term \"gaming addiction\" is used along with a number of others (disordered gaming, pathological video gaming, disordered gaming behavior) to describe the syndrome under investigation. For simplicity, it would be helpful if the authors select a single term to describe the syndrome and use that consistently throughout the manuscript. They might even mention, parenthetically, that they will be using this term to describe the target syndrome, the first time it is introduced.</p><p id=\"__p131\">2. Ms p. 7 - line 103</p><p id=\"__p132\">The term \"IGD\" is introduced but not defined. Perhaps Internet Gaming Disorder (IGD) is the term they wish to use as their standard description of the syndrome?</p><p id=\"__p133\">3. Table 1 provides a very clear summary of the findings to date.</p><p id=\"__p134\">4. Ms p. 8, lines 129-130:</p><p id=\"__p135\">\"All these deficiencies impede...\" It is not clear that these are \"deficiencies\" as opposed to different conceptual or explanatory frameworks or preferred terminology. Perhaps a more neutral description (e.g., \"This variability across the different gaming scales impedes...\") would work equally well?</p><p id=\"__p136\">5. In the Introduction, it would be helpful to specify the age of &#x0201c;adolescents&#x0201d; (12-17 years)? \"young adults\" (18-25 years)?</p><p id=\"__p137\">6. Page 8 - When describing the Participants, it would be helpful to report any incentive offered or alternative reason to participate in the study, or if none was offered, this could be stated as well. Same for studies 2 and 3.</p><p id=\"__p138\">6. Ms p. 9, line 151 \"recruited from the LAN party\" The term LAN should be defined for general readership</p><p id=\"__p139\">7. Ms p. 9, line 156 \"Through an exhaustive scoping review of the literature\"</p><p id=\"__p140\">It would be helpful to specify which databases were examined, the years of publication deemed eligible for inclusion, and a sample of the search terms used.</p><p id=\"__p141\">8. Ms p 9, line 172</p><p id=\"__p142\">What was the rationale for using oblique as opposed to orthogonal rotation to achieve simple structure in the two EFAs?</p><p id=\"__p143\">9. Ms p 11, lines 220-2</p><p id=\"__p144\">Weekly number of gaming hours was obtained by adding online and offline daily hours and multiplying the days of these periods of the week by the total number of hours. Global retrospective estimates of time engaged in activities like drinking or gambling have been found to be less accurate than values based on a Timeline Followback (see Sobell and Sobell 1992) procedure. Since Timeline Followback was not mentioned here, the limitations of the current reporting procedure should be acknowledged in the Discussion</p><p id=\"__p145\">10. Ms p 12, line 246</p><p id=\"__p146\">Hierarchical multiple regressions were used to examine the relationship between motives, gaming intensity and problems, while controlling for age and gender. What was the method of entry of predictors in each stage, and briefly what was the rationale for this?</p><p id=\"__p147\">11. Ms p. 18</p><p id=\"__p148\">Study 3 refers to the Emerging Adulthood sample. This is the only time this terminology is used. Were the age inclusion criteria different than for study 2? If not, perhaps it is best to stick with \'Young Adult\' as the description?</p><p id=\"__p149\">12. Ms p. 21</p><p id=\"__p150\">In the Discussion, before launching into the specific motives that emerged, it would be good to state that an 8-factor model appeared to best explain the data in all 3 samples, and to report the proportion of variance explained by these factors in each case. The first paragraph under Conclusion, p 26 addresses many of these points and could be moved up to the beginning of this section.</p><p id=\"__p151\">Some comment on the inter-correlation vs. orthogonality of factors might also be made and what this overall pattern of scores means for VMQ as assessment tool, generally. The consistent elevation in motive endorsement by males, aside from customization, may also merit some comment.</p><p id=\"__p152\">13. As the authors note, Coping emerged as a reliable and somewhat selective predictor of disordered gaming (as compared with Social Interaction, for example). Although this makes intuitive sense, I think this linkage, which, again as noted, is seen with other addictive behaviors, might be considered a bit more with respect to addiction pathology.</p><p id=\"__p153\">For example, what is the difference between wanting to engage in a behavior and needing to do so?</p><p id=\"__p154\">How do these different global motivations correspond to the concept of \"dependence\" within the addiction framework?</p><p id=\"__p155\">And what would be the expected role of environmental factors like \"stress\" as a conditioned (i.e., involuntary) cue for gaming in people with these different motivational profiles?</p><p id=\"__p156\">Minor Points:</p><p id=\"__p157\">1. Ms p. 4, line 71 \"based on well-stablished theories\" Typo: well-[e]stablished</p><p id=\"__p158\">2. Ms p. 7, line 119 \"the described motives suffer from [?] of &#x02018;jingle-jangle&#x02019; fallacies,\" Missing words [a form]</p><p id=\"__p159\">3. Table 2, Cognitive Development factor: Item 3 (Games make me thing). Typo - Games make me thin[k]</p><p id=\"__p160\">4. Ms p. 19, line 320 \"we used the S&#x02212;B&#x003c7;2/df to overcome such problem [74].\" Syntax \"to overcome [this] problem\"</p><p id=\"__p161\">**********</p><p id=\"__p162\">6. PLOS authors have the option to publish the peer review history of their article (<a href=\"https://journals.plos.org/plosone/s/editorial-and-peer-review-process#loc-peer-review-history\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">what does this mean?</a>). If published, this will include your full peer review and any attached files.</p><p id=\"__p163\">If you choose &#x0201c;no&#x0201d;, your identity will remain anonymous but your review may still be made public.</p><p id=\"__p164\"><strong>Do you want your identity to be public for this peer review?</strong> For information about this choice, including consent withdrawal, please see our <a href=\"https://www.plos.org/privacy-policy\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Privacy Policy</a>.</p><p id=\"__p165\">Reviewer #1:&#x000a0;No</p><p id=\"__p166\">[NOTE: If reviewer comments were submitted as an attachment file, they will be attached to this email and accessible via the submission site. Please log into your account, locate the manuscript record, and check for the action link \"View Attachments\". If this link does not appear, there are no attachment files.]</p><p id=\"__p167\" class=\"p\">While revising your submission, please upload your figure files to the Preflight Analysis and Conversion Engine (PACE) digital diagnostic tool,&#x000a0;<a href=\"https://pacev2.apexcovantage.com/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://pacev2.apexcovantage.com/</a>. PACE helps ensure that figures meet PLOS requirements. To use PACE, you must first register as a user. Registration is free. Then, login and navigate to the UPLOAD tab, where you will find detailed instructions on how to use the tool. If you encounter any issues or have any questions when using PACE, please email PLOS at&#x000a0;<a href=\"mailto:dev@null\" data-email=\"gro.solp@serugif\" class=\"oemail\">gro.solp@serugif</a>. Please note that Supporting Information files do not need this step.</p><span class=\"sec suppmat\" id=\"pone.0240726.s004\"><h4>Attachment</h4><p id=\"__p168\">Submitted filename: <em class=\"submitted-filename\">PONE-D-20-12358-rev.pdf</em></p><span class=\"sup-box half_rhythm\" id=\"idm140210184658064\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/bin/pone.0240726.s004.pdf\" data-ga-action=\"click_feat_suppl\">Click here for additional data file.</a><sup>(73K, pdf)</sup></span></span></span></span></span><span class=\"sub-navigation nav-top space small\"><ul class=\"page-breadcrumbs inline_list small\"><li><a href=\"#top\" class=\"navlink\">PLoS One. 2020; 15(10): e0240726. </a></li><span> » </span><li>Author response to Decision Letter 0</li></ul></span><span class=\"sub-content half_rhythm\" id=\"pone.0240726.r002\"><span class=\"jig-ncbiinpagenav\" data-jigconfig=\"smoothScroll: false, allHeadingLevels: [\'h2\'], headingExclude: \':hidden,.nomenu\'\"><span class=\"fm-sec half_rhythm no_top_margin\"><span class=\"fm-flexbox\"><span class=\"fm-citation\"><span class=\"part1\"><span class=\"cit-abbr\"> </span><span class=\"cit-pub-date\">2020; </span><span class=\"cit-vol\">15</span><span class=\"cit-iss\">(10)</span><span class=\"cit-flpgs\">: e0240726. </span></span><span class=\"part2\"><span class=\"fm-vol-iss-date\">Published online 2020 Oct 23. </span><span class=\"doi\"><span>doi: </span><a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0240726.r002\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">10.1371/journal.pone.0240726.r002</a></span></span></span></span><h1 class=\"sub-content-title\">Author response to Decision Letter 0</h1><span class=\"fm-panel half_rhythm\"><span class=\"togglers fm-copyright-license\"> <a href=\"#\" class=\"pmctoggle\" rid=\"idm140210184656400_cpl\">Copyright and License information</a> <a href=\"/pmc/about/disclaimer/\" style=\"margin-left: 1em\">Disclaimer</a></span><span class=\"fm-authors-info fm-panel half_rhythm\"><span></span></span><span class=\"fm-article-notes fm-panel half_rhythm\"></span><span class=\"permissions fm-panel half_rhythm hide\" id=\"idm140210184656400_cpl\" style=\"display:none\"><span class=\"fm-copyright half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright notice</a> </span></span></span></span><span id=\"idm140210184652736\" class=\"tsec sec\"><h2 class=\"headless nomenu\"></h2><p id=\"__p170\" class=\"p p-first\">\n<em class=\"author-response-date\">11 Sep 2020</em>\n</p><p id=\"__p171\">Journal Requirements:</p><p id=\"__p172\">When submitting your revision, we need you to address these additional requirements.</p><p id=\"__p173\">1. Please ensure that your manuscript meets PLOS ONE\'s style requirements, including those for file naming. The PLOS ONE style templates can be found at</p><p id=\"__p174\"><a href=\"https://journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf</a> and</p><p id=\"__p175\"><a href=\"https://journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf</a></p><p id=\"__p176\">Response: Done</p><p id=\"__p177\">2. Please include additional information about your scoping review in your methods section, including search strategy, inclusion and exclusion criteria, to enable reproducibility and replicability.</p><p id=\"__p178\">Response: Dear editor, </p><p id=\"__p179\">We regret to inform you that there was a translation misunderstanding (an example of &#x0201c;lost in translation&#x0201d;) with regard to the term &#x0201c;scoping review&#x0201d;. We conducted an extensive review of motivational scale studies, selecting most of the relevant questionnaires used in research, but it was not a scoping review. Later, when the English language revision of the paper was carried out, &#x0201c;extensive review&#x0201d; was changed to &#x0201c;scoping review&#x0201d;. This was unfortunate as these concepts are different. We would like to apologize again. Accordingly, we have changed the term &#x0201c;scoping review&#x0201d; for the original &#x0201c;extensive review&#x0201d;. In any case, we have described in more detail which databases were examined, the years of publication eligible for inclusion, and a sample of the search terms used, as suggested (see p. 9, lines 160-165)</p><p id=\"__p180\">3. Please proofread your manuscript for typos (for instance item 3 of table 2).</p><p id=\"__p181\">Response: Done</p><p id=\"__p182\">4. Your ethics statement must appear in the Methods section of your manuscript. If your ethics statement is written in any section besides the Methods, please move it to the Methods section and delete it from any other section. Please also ensure that your ethics statement is included in your manuscript, as the ethics section of your online submission will not be published alongside your manuscript.</p><p id=\"__p183\">Response: Done</p><p id=\"__p184\">Response to reviewer:</p><p id=\"__p185\">We really appreciate the reviewer&#x02019;s contribution. </p><p id=\"__p186\">1. Ms p. 3, line 51</p><p id=\"__p187\">The term \"gaming addiction\" is used along with a number of others (disordered gaming, pathological video gaming, disordered gaming behavior) to describe the syndrome under investigation. For simplicity, it would be helpful if the authors select a single term to describe the syndrome and use that consistently throughout the manuscript. They might even mention, parenthetically, that they will be using this term to describe the target syndrome, the first time it is introduced.</p><p id=\"__p188\">Response: p. 3, line 53-54 </p><p id=\"__p189\">&#x0201c;These inclusions in the DSM-5 and ICD-11 assume that disordered gaming, (for simplicity, this term will be used throughout the manuscript to describe the syndrome)&#x02026; &#x0201c;</p><p id=\"__p190\">2. Ms p. 7 - line 103</p><p id=\"__p191\">The term \"IGD\" is introduced but not defined. Perhaps Internet Gaming Disorder (IGD) is the term they wish to use as their standard description of the syndrome?</p><p id=\"__p192\">Response: p. 3, line 47-49</p><p id=\"__p193\">&#x02026; defined as a &#x0201c;persistent and recurrent use of the Internet to engage in games, often with other players, leading to clinically significant impairment or distress ...&#x0201d;</p><p id=\"__p194\">3. Table 1 provides a very clear summary of the findings to date.</p><p id=\"__p195\">4. Ms p. 8, lines 129-130:</p><p id=\"__p196\">\"All these deficiencies impede...\" It is not clear that these are \"deficiencies\" as opposed to different conceptual or explanatory frameworks or preferred terminology. Perhaps a more neutral description (e.g., \"This variability across the different gaming scales impedes...\") would work equally well?</p><p id=\"__p197\">Response: p. 8, lines 132-133</p><p id=\"__p198\">&#x0201c;This variability across the different gaming scales impedes&#x02026;&#x0201d;</p><p id=\"__p199\">5. In the Introduction, it would be helpful to specify the age of &#x0201c;adolescents&#x0201d; (12-17 years)? \"young adults\" (18-25 years)?</p><p id=\"__p200\">Response: p. 3, line 43</p><p id=\"__p201\">&#x0201c;&#x02026; adolescents (12-17 years) and young adults (18-24 year)&#x0201d;</p><p id=\"__p202\">6. Page 8 - When describing the Participants, it would be helpful to report any incentive offered or alternative reason to participate in the study, or if none was offered, this could be stated as well. Same for studies 2 and 3.</p><p id=\"__p203\">Study 1</p><p id=\"__p204\">Response: p. 9, line 156</p><p id=\"__p205\">&#x0201c;Participants voluntarily completed an online survey without any compensation&#x0201d;</p><p id=\"__p206\">Study 2</p><p id=\"__p207\">Response: p. 11, line 210</p><p id=\"__p208\">&#x0201c;&#x02026; and received school supplies as an incentive&#x0201d;</p><p id=\"__p209\">Study 3</p><p id=\"__p210\">Response: p. 19, line 323</p><p id=\"__p211\">&#x0201c;The survey was selflessly completed online&#x0201d;</p><p id=\"__p212\">6. Ms p. 9, line 151 \"recruited from the LAN party\" The term LAN should be defined for general readership</p><p id=\"__p213\">Response: pp. 8-9, lines 153-154</p><p id=\"__p214\">&#x0201c;.. an event at the Jaume I University in which gamers gathered to play multiplayer video games, ..&#x0201d;</p><p id=\"__p215\">7. Ms p. 9, line 156 \"Through an exhaustive scoping review of the literature\"</p><p id=\"__p216\">It would be helpful to specify which databases were examined, the years of publication deemed eligible for inclusion, and a sample of the search terms used.</p><p id=\"__p217\">Response: p. 9, lines 160-165</p><p id=\"__p218\">Through an extensive review of the literature concerning gaming motivation scales, 25 relevant gaming motive scales were selected. For the review, it was conducted a search in the databases PsycINFO and Google Scholar from 2000 to present, combining the key words motives or motivation, video game or gaming, scale or questionnaire. Furthermore, citations from selected studies of our database were used to continue the review process until no other relevant study was found.</p><p id=\"__p219\">8. Ms p 9, line 172</p><p id=\"__p220\">What was the rationale for using oblique as opposed to orthogonal rotation to achieve simple structure in the two EFAs?</p><p id=\"__p221\">Response: </p><p id=\"__p222\">Oblimin rotation (which does not assume prior relations between factors and allows axes to freely rotate) is somewhat more exploratory than orthogonal rotation (which assumes independence between factors and forces axes to be orthogonal), and also allows to estimate the intercorrelations between factors. Based on previous psychometric studies on motives for gaming and other addictive behaviors, we expected high intercorrelations between factors. Accordingly, oblimin rotation was used in the pilot study. The pilot study confirmed that factors were highly intercorrelated, so the oblimin rotation was also used in the EFA of the second study, and the correlations between factors were specified in the CFA of the third study.</p><p id=\"__p223\">Thus, we have added in Statistical Analysis section of study 2:</p><p id=\"__p224\">p. 13, lines 255-257 &#x0201c;As the pilot study showed that motives were intercorrelated, an EFA with principal axis factoring and oblique rotation (oblimin) was performed for testing the VMQ with original eight-factor model using IBM SPSS Statistics V21.0.&#x0201d;</p><p id=\"__p225\">9. Ms p 11, lines 220-2</p><p id=\"__p226\">Weekly number of gaming hours was obtained by adding online and offline daily hours and multiplying the days of these periods of the week by the total number of hours. Global retrospective estimates of time engaged in activities like drinking or gambling have been found to be less accurate than values based on a Timeline Followback (see Sobell and Sobell 1992) procedure. Since Timeline Followback was not mentioned here, the limitations of the current reporting procedure should be acknowledged in the Response: Discussion</p><p id=\"__p227\">p. 29, lines 562-564</p><p id=\"__p228\">&#x0201c;Fourth, the time engaged in gaming was obtained by self-report retrospective estimates, which have been found to be less accurate than other procedures such as Timeline Followback [82]&#x0201d;.</p><p id=\"__p229\">10. Ms p 12, line 246</p><p id=\"__p230\">Hierarchical multiple regressions were used to examine the relationship between motives, gaming intensity and problems, while controlling for age and gender. What was the method of entry of predictors in each stage, and briefly what was the</p><p id=\"__p231\">rationale for this?</p><p id=\"__p232\">Response: p. 13, lines 263-267</p><p id=\"__p233\">&#x0201c;Previous research has shown that gender and age were relevant variables in video game&#x02013;related behaviors, such as disordered use or video game genre preferences (e.g., [9,32,57]). Therefore, hierarchical multiple regressions were controlled for gender and age by introducing them in step 1, whereas motives were entered in step 2&#x0201d;.</p><p id=\"__p234\">11. Ms p. 18</p><p id=\"__p235\">Study 3 refers to the Emerging Adulthood sample. This is the only time this terminology is used. Were the age inclusion criteria different than for study 2? If not, perhaps it is best to stick with \'Young Adult\' as the description?</p><p id=\"__p236\">Response: p. 19, line 318</p><p id=\"__p237\">&#x0201c;Study 3. Young adult sample study&#x0201d;</p><p id=\"__p238\">12. Ms p. 21</p><p id=\"__p239\">In the Discussion, before launching into the specific motives that emerged, it would be good to state that an 8-factor model appeared to best explain the data in all 3 samples, and to report the proportion of variance explained by these factors in each case. </p><p id=\"__p240\">AND</p><p id=\"__p241\">The first paragraph under Conclusion, p 26 addresses many of these points and could be moved up to the beginning of this section. </p><p id=\"__p242\">AND</p><p id=\"__p243\">Some comment on the inter-correlation vs. orthogonality of factors might also be made and what this overall pattern of scores means for VMQ as assessment tool, generally. </p><p id=\"__p244\">AND </p><p id=\"__p245\">The consistent elevation in motive endorsement by males, aside from customization, may also merit some comment.</p><p id=\"__p246\">Response: p. 22., lines 402-405</p><p id=\"__p247\">&#x0201c;As a result, the VMQ consisted of an 8-intercorrelated-factor model that appeared to best explain the data. It showed good internal consistency and temporal stability reliability indices, and also presented evidence of validity when predicting video game frequency of use and disordered gaming.&#x0201d; </p><p id=\"__p248\">And the information of % of variance is now presented at the Result sections of each study (Factor Structure) </p><p id=\"__p249\">Initial testing: 78.7%; Adolescent sample study: 78.5% &#x00026; Young adulthood sample: 81.9%</p><p id=\"__p250\">p. 28, lines 546-556</p><p id=\"__p251\">&#x0201c;Regarding gender, adolescent and young adult males presented higher scores than females in nearly all scales apart from customization, especially in competitive, violent reward and social interaction motives. These higher motivational levels in male gender for almost all motives has been systematically found in previous studies (e.g., [32-36, 57]). Along these lines, it has been reported that adolescent males prefer competitive and violent game genres, such as action-shooters, sport, fight or strategy games; whereas girls prefer game genres such as brain and skill games, and social simulation games, more characterized by customization aspects [9]. These gender-based differences on motives for gaming may help to explain why males, who prefer games featuring more time-consuming and engagement characteristics, presented higher prevalence of regular and disordered use of video games than females [7].&#x0201d;</p><p id=\"__p252\">13. As the authors note, Coping emerged as a reliable and somewhat selective predictor of disordered gaming (as compared with Social Interaction, for example). Although this makes intuitive sense, I think this linkage, which, again as noted, is seen with other addictive behaviors, might be considered a bit more with respect to addiction pathology.</p><p id=\"__p253\">For example, what is the difference between wanting to engage in a behavior and needing to do so?</p><p id=\"__p254\">How do these different global motivations correspond to the concept of \"dependence\" within the addiction framework? And what would be the expected role of environmental factors like \"stress\" as a conditioned (i.e., involuntary) cue for gaming in people with these different motivational profiles?</p><p id=\"__p255\">Response: We have tried to reflect all these very insightful considerations in the following paragraph: </p><p id=\"__p256\">p. 23, lines 426-434</p><p id=\"__p257\">&#x0201c;In the present study, coping was the strongest and somewhat selective predictor of disordered gaming scores among both adults and adolescents. This result is in accordance with previous studies in which escape/coping motives presented the highest association with disordered gaming [48-50,52,53,59,77] and with problematic substance use and gambling [22-27]. This motive appears to be especially relevant in the presence of psychological problems and environmental stressors, such as depression and anxiety symptoms [50, 54-56] or stressful life events [78]. Accordingly, motivation for escape or relief from negative mood constitutes an important criterion for the diagnoses of IGD [5] and other addictions in the DSM-5 [3] </p><p id=\"__p258\">Minor Points:</p><p id=\"__p259\" class=\"p\">Done</p><span class=\"sec suppmat\" id=\"pone.0240726.s005\"><h4>Attachment</h4><p id=\"__p260\">Submitted filename: <em class=\"submitted-filename\">Response To Reviewers.docx</em></p><span class=\"sup-box half_rhythm\" id=\"idm140210184162624\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/bin/pone.0240726.s005.docx\" data-ga-action=\"click_feat_suppl\">Click here for additional data file.</a><sup>(29K, docx)</sup></span></span></span></span></span><span class=\"sub-navigation nav-top space small\"><ul class=\"page-breadcrumbs inline_list small\"><li><a href=\"#top\" class=\"navlink\">PLoS One. 2020; 15(10): e0240726. </a></li><span> » </span><li>Decision Letter 1</li></ul></span><span class=\"sub-content half_rhythm\" id=\"pone.0240726.r003\"><span class=\"jig-ncbiinpagenav\" data-jigconfig=\"smoothScroll: false, allHeadingLevels: [\'h2\'], headingExclude: \':hidden,.nomenu\'\"><span class=\"fm-sec half_rhythm no_top_margin\"><span class=\"fm-flexbox\"><span class=\"fm-citation\"><span class=\"part1\"><span class=\"cit-abbr\"> </span><span class=\"cit-pub-date\">2020; </span><span class=\"cit-vol\">15</span><span class=\"cit-iss\">(10)</span><span class=\"cit-flpgs\">: e0240726. </span></span><span class=\"part2\"><span class=\"fm-vol-iss-date\">Published online 2020 Oct 23. </span><span class=\"doi\"><span>doi: </span><a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0240726.r003\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">10.1371/journal.pone.0240726.r003</a></span></span></span></span><h1 class=\"sub-content-title\">Decision Letter 1</h1><span class=\"half_rhythm\"><span class=\"contrib-group fm-author\"><a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Chiesi%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33095762\">Francesca Chiesi</a>, <span class=\"fm-role\">Academic Editor</span></span><span style=\"display:none\" class=\"contrib-group aff-tip\"></span></span><span class=\"fm-panel half_rhythm\"><span class=\"togglers fm-copyright-license\"> <a href=\"#\" class=\"pmctoggle\" rid=\"idm140210184160912_cpl\">Copyright and License information</a> <a href=\"/pmc/about/disclaimer/\" style=\"margin-left: 1em\">Disclaimer</a></span><span class=\"fm-authors-info fm-panel half_rhythm\"><span></span></span><span class=\"fm-article-notes fm-panel half_rhythm\"></span><span class=\"permissions fm-panel half_rhythm hide\" id=\"idm140210184160912_cpl\" style=\"display:none\"><span class=\"fm-copyright half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright</a> &#x000a9; 2020 Francesca Chiesi</span><span class=\"fm-copyright half_rhythm\">This is an open access article distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></span></span><span id=\"idm140210194197648\" class=\"tsec sec\"><h2 class=\"headless nomenu\"></h2><p id=\"__p262\" class=\"p p-first\">\n<em class=\"letter-date\">23 Sep 2020</em>\n</p><p id=\"__p263\">PONE-D-20-12358R1</p><p id=\"__p264\">The development and validation of the Videogaming Motives Questionnaire (VMQ)</p><p id=\"__p265\">PLOS ONE</p><p id=\"__p266\">Dear Dr. Ib&#x000e1;&#x000f1;ez,</p><p id=\"__p267\">Thank you for submitting your manuscript to PLOS ONE. After careful consideration, we feel that it has merit but does not fully meet PLOS ONE&#x02019;s publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the issues detailed below (see Additional Editor Comments).</p><p id=\"__p268\">Please submit your revised manuscript by Nov 07 2020 11:59PM. If you will need more time than this to complete your revisions, please reply to this message or contact the journal office at <a href=\"mailto:dev@null\" data-email=\"gro.solp@enosolp\" class=\"oemail\">gro.solp@enosolp</a>. When you\'re ready to submit your revision, log on to <a href=\"https://www.editorialmanager.com/pone/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://www.editorialmanager.com/pone/</a> and select the \'Submissions Needing Revision\' folder to locate your manuscript file.</p><p id=\"__p269\">Please include the following items when submitting your revised manuscript:</p><ul class=\"unordered\" style=\"list-style-type:disc\"><!--\nlist-behavior=unordered\nprefix-word=\nmark-type=disc\nmax-label-size=0\n--><li><p id=\"__p271\">A rebuttal letter that responds to each point raised by the academic editor and reviewer(s). You should upload this letter as a separate file labeled \'Response to Reviewers\'.</p></li><li><p id=\"__p272\">A marked-up copy of your manuscript that highlights changes made to the original version. You should upload this as a separate file labeled \'Revised Manuscript with Track Changes\'.</p></li><li><p id=\"__p273\">An unmarked version of your revised paper without tracked changes. You should upload this as a separate file labeled \'Manuscript\'.</p></li></ul><p></p><p id=\"__p274\">If you would like to make changes to your financial disclosure, please include your updated statement in your cover letter. Guidelines for resubmitting your figure files are available below the reviewer comments at the end of this letter.</p><p id=\"__p275\">If applicable, we recommend that you deposit your laboratory protocols in protocols.io to enhance the reproducibility of your results. Protocols.io assigns your protocol its own identifier (DOI) so that it can be cited independently in the future. For instructions see:&#x000a0;<a href=\"http://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols</a></p><p id=\"__p276\">We look forward to receiving your revised manuscript.</p><p id=\"__p277\">Kind regards,</p><p id=\"__p278\">Francesca Chiesi</p><p id=\"__p279\">Academic Editor</p><p id=\"__p280\">PLOS ONE</p><p id=\"__p281\">Additional Editor Comments:</p><p id=\"__p282\">Dear Authors:</p><p id=\"__p283\">I&#x000a0;appreciate the strength of the paper and I believe that the changes made in response to the reviewer&#x02019;s comments contributed to improve it. However, I have a minor comment about the scales you used to measure disordered gaming. You employed two different scales in Study 2 and Study 3, so I believe that this point deserves some explanation. Could you motivate your choice? Did this choice have an impact on your results? Additionally, was the Gentile&#x02019;s scale&#x000a0;validated in Spanish (as the IGD-20 )? If not, how do you translate it?&#x000a0;&#x000a0;</p><p id=\"__p284\">[Note: HTML markup is below. Please do not edit.]</p><p id=\"__p285\">[NOTE: If reviewer comments were submitted as an attachment file, they will be attached to this email and accessible via the submission site. Please log into your account, locate the manuscript record, and check for the action link \"View Attachments\". If this link does not appear, there are no attachment files.]</p><p id=\"__p286\" class=\"p p-last\">While revising your submission, please upload your figure files to the Preflight Analysis and Conversion Engine (PACE) digital diagnostic tool,&#x000a0;<a href=\"https://pacev2.apexcovantage.com/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://pacev2.apexcovantage.com/</a>. PACE helps ensure that figures meet PLOS requirements. To use PACE, you must first register as a user. Registration is free. Then, login and navigate to the UPLOAD tab, where you will find detailed instructions on how to use the tool. If you encounter any issues or have any questions when using PACE, please email PLOS at&#x000a0;<a href=\"mailto:dev@null\" data-email=\"gro.solp@serugif\" class=\"oemail\">gro.solp@serugif</a>. Please note that Supporting Information files do not need this step.</p></span></span></span><span class=\"sub-navigation nav-top space small\"><ul class=\"page-breadcrumbs inline_list small\"><li><a href=\"#top\" class=\"navlink\">PLoS One. 2020; 15(10): e0240726. </a></li><span> » </span><li>Author response to Decision Letter 1</li></ul></span><span class=\"sub-content half_rhythm\" id=\"pone.0240726.r004\"><span class=\"jig-ncbiinpagenav\" data-jigconfig=\"smoothScroll: false, allHeadingLevels: [\'h2\'], headingExclude: \':hidden,.nomenu\'\"><span class=\"fm-sec half_rhythm no_top_margin\"><span class=\"fm-flexbox\"><span class=\"fm-citation\"><span class=\"part1\"><span class=\"cit-abbr\"> </span><span class=\"cit-pub-date\">2020; </span><span class=\"cit-vol\">15</span><span class=\"cit-iss\">(10)</span><span class=\"cit-flpgs\">: e0240726. </span></span><span class=\"part2\"><span class=\"fm-vol-iss-date\">Published online 2020 Oct 23. </span><span class=\"doi\"><span>doi: </span><a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0240726.r004\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">10.1371/journal.pone.0240726.r004</a></span></span></span></span><h1 class=\"sub-content-title\">Author response to Decision Letter 1</h1><span class=\"fm-panel half_rhythm\"><span class=\"togglers fm-copyright-license\"> <a href=\"#\" class=\"pmctoggle\" rid=\"idm140210184062064_cpl\">Copyright and License information</a> <a href=\"/pmc/about/disclaimer/\" style=\"margin-left: 1em\">Disclaimer</a></span><span class=\"fm-authors-info fm-panel half_rhythm\"><span></span></span><span class=\"fm-article-notes fm-panel half_rhythm\"></span><span class=\"permissions fm-panel half_rhythm hide\" id=\"idm140210184062064_cpl\" style=\"display:none\"><span class=\"fm-copyright half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright notice</a> </span></span></span></span><span id=\"idm140210184058256\" class=\"tsec sec\"><h2 class=\"headless nomenu\"></h2><p id=\"__p287\" class=\"p p-first\">\n<em class=\"author-response-date\">29 Sep 2020</em>\n</p><p id=\"__p288\">Additional Editor Comments:</p><p id=\"__p289\">Dear Authors:</p><p id=\"__p290\">I appreciate the strength of the paper and I believe that the changes made in response to the reviewer&#x02019;s comments contributed to improve it. However, I have a minor comment about the scales you used to measure disordered gaming. You employed two different scales in Study 2 and Study 3, so I believe that this point deserves some explanation. Could you motivate your choice? Did this choice have an impact on your results? Additionally, was the Gentile&#x02019;s scale validated in Spanish (as the IGD-20 )? If not, how do you translate it? </p><p id=\"__p291\">Response: </p><p id=\"__p292\">Dear Editor:</p><p id=\"__p293\">Thanks for your comments.</p><p id=\"__p294\">We employed different scales to measure disordered gaming in each study because of the sample&#x02019;s age. The scale used in the adolescent sample of Study 2 was originally developed for the assessment of gaming disorders in youths aged from 8 to 18 years (see Gentile, 2009), and validated in Spain with an adolescent sample (the mean age of the validation sample in Spain was 14,29 years old; see L&#x000f3;pez-Fern&#x000e1;ndez et al., 2020). In the sample of young adults of Study 3, we considered a better option to use the IGD-20, originally developed in a sample of young adults (Pontes et al., 2014) and validated two years later in Spain (Fuster et al., 2016) (mean age of the original sample of 26.5 years old, and 26.14 years old in in the Spanish sample).</p><p id=\"__p295\">Although these scales somewhat different, both were based on similar DSM criteria, so we consider that this difference in the assessment of gaming disorder would have a limited impact on the results. This idea has been reinforced by the fact that disordered gaming assessed with the different scales have converged in finding that the most important motives for disordered gaming have been coping, social interaction and violent reward (in the same order). In any case, we have added a new limitation:</p><p id=\"__p296\">&#x0201c;Five, two different scales was employed for assessing disordered gaming in adolescents and young adults. Although we have found that main motives related to disordered gaming were the same in both samples, some of the differences found between adolescents and young adults could be attributed to the different scales used.&#x0201d;</p><p id=\"__p297\">Last, the Gentile&#x02019;s scale usen in study 2 was validated in Spanish adolescents in a previous study (L&#x000f3;pez-Fern&#x000e1;ndez et al., 2020). Extracted from L&#x000f3;pez-Fern&#x000e1;ndez et al., 2020: </p><p id=\"__p298\">&#x0201c;The original 11 items was back-translated&#x02026; For the study, the participants indicated their frequency of video game-related problems on a 4-point Likert scale (from 0 = &#x0201c;never or almost never&#x0201d; to 3 &#x0201c;almost always or always&#x0201d;) during the last 12 months. According to the parallel analysis run using Monte Carlo PCA (Watkins, 2006), a one-factor structure was obtained with the EFA, where all items presented adequate factor loadings ranging from .49 to .81. Cronbach&#x02019;s alpha in this sample was .88&#x0201d;.</p><p id=\"__p299\">Thus, now we have included this fact in the text as follows:</p><p id=\"__p300\">p. 13, lines 248-249</p><p id=\"__p301\" class=\"p\">&#x0201c;A Spanish adaptation [9] of a measure especially developed to assess disordered gaming in adolescents [70] was employed&#x0201d;.</p><span class=\"sec suppmat\" id=\"pone.0240726.s006\"><h4>Attachment</h4><p id=\"__p302\">Submitted filename: <em class=\"submitted-filename\">Response To Reviewers.docx</em></p><span class=\"sup-box half_rhythm\" id=\"idm140210184042576\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC7584249/bin/pone.0240726.s006.docx\" data-ga-action=\"click_feat_suppl\">Click here for additional data file.</a><sup>(19K, docx)</sup></span></span></span></span></span><span class=\"sub-navigation nav-top space small\"><ul class=\"page-breadcrumbs inline_list small\"><li><a href=\"#top\" class=\"navlink\">PLoS One. 2020; 15(10): e0240726. </a></li><span> » </span><li>Decision Letter 2</li></ul></span><span class=\"sub-content half_rhythm\" id=\"pone.0240726.r005\"><span class=\"jig-ncbiinpagenav\" data-jigconfig=\"smoothScroll: false, allHeadingLevels: [\'h2\'], headingExclude: \':hidden,.nomenu\'\"><span class=\"fm-sec half_rhythm no_top_margin\"><span class=\"fm-flexbox\"><span class=\"fm-citation\"><span class=\"part1\"><span class=\"cit-abbr\"> </span><span class=\"cit-pub-date\">2020; </span><span class=\"cit-vol\">15</span><span class=\"cit-iss\">(10)</span><span class=\"cit-flpgs\">: e0240726. </span></span><span class=\"part2\"><span class=\"fm-vol-iss-date\">Published online 2020 Oct 23. </span><span class=\"doi\"><span>doi: </span><a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0240726.r005\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">10.1371/journal.pone.0240726.r005</a></span></span></span></span><h1 class=\"sub-content-title\">Decision Letter 2</h1><span class=\"half_rhythm\"><span class=\"contrib-group fm-author\"><a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Chiesi%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33095762\">Francesca Chiesi</a>, <span class=\"fm-role\">Academic Editor</span></span><span style=\"display:none\" class=\"contrib-group aff-tip\"></span></span><span class=\"fm-panel half_rhythm\"><span class=\"togglers fm-copyright-license\"> <a href=\"#\" class=\"pmctoggle\" rid=\"idm140210184010208_cpl\">Copyright and License information</a> <a href=\"/pmc/about/disclaimer/\" style=\"margin-left: 1em\">Disclaimer</a></span><span class=\"fm-authors-info fm-panel half_rhythm\"><span></span></span><span class=\"fm-article-notes fm-panel half_rhythm\"></span><span class=\"permissions fm-panel half_rhythm hide\" id=\"idm140210184010208_cpl\" style=\"display:none\"><span class=\"fm-copyright half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright</a> &#x000a9; 2020 Francesca Chiesi</span><span class=\"fm-copyright half_rhythm\">This is an open access article distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></span></span><span id=\"idm140210184001856\" class=\"tsec sec\"><h2 class=\"headless nomenu\"></h2><p id=\"__p304\" class=\"p p-first\">\n<em class=\"letter-date\">2 Oct 2020</em>\n</p><p id=\"__p305\">The development and validation of the Videogaming Motives Questionnaire (VMQ)</p><p id=\"__p306\">PONE-D-20-12358R2</p><p id=\"__p307\">Dear Dr. Ib&#x000e1;&#x000f1;ez,</p><p id=\"__p308\">We&#x02019;re pleased to inform you that your manuscript has been judged scientifically suitable for publication and will be formally accepted for publication once it meets all outstanding technical requirements.</p><p id=\"__p309\">Within one week, you&#x02019;ll receive an e-mail detailing the required amendments. When these have been addressed, you&#x02019;ll receive a formal acceptance letter and your manuscript will be scheduled for publication.</p><p id=\"__p310\">An invoice for payment will follow shortly after the formal acceptance. To ensure an efficient process, please log into Editorial Manager at <a href=\"http://www.editorialmanager.com/pone/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.editorialmanager.com/pone/</a>, click the \'Update My Information\' link at the top of the page, and double check that your user information is up-to-date. If you have any billing related questions, please contact our Author Billing department directly at <a href=\"mailto:dev@null\" data-email=\"gro.solp@gnillibrohtua\" class=\"oemail\">gro.solp@gnillibrohtua</a>.</p><p id=\"__p311\">If your institution or institutions have a press office, please notify them about your upcoming paper to help maximize its impact. If they&#x02019;ll be preparing press materials, please inform our press team as soon as possible -- no later than 48 hours after receiving the formal acceptance. Your manuscript will remain under strict press embargo until 2 pm Eastern Time on the date of publication. For more information, please contact <a href=\"mailto:dev@null\" data-email=\"gro.solp@sserpeno\" class=\"oemail\">gro.solp@sserpeno</a>.</p><p id=\"__p312\">Kind regards,</p><p id=\"__p313\">Francesca Chiesi</p><p id=\"__p314\">Academic Editor</p><p id=\"__p315\" class=\"p p-last\">PLOS ONE</p></span></span></span><span class=\"sub-navigation nav-top space small\"><ul class=\"page-breadcrumbs inline_list small\"><li><a href=\"#top\" class=\"navlink\">PLoS One. 2020; 15(10): e0240726. </a></li><span> » </span><li>Acceptance letter</li></ul></span><span class=\"sub-content half_rhythm\" id=\"pone.0240726.r006\"><span class=\"jig-ncbiinpagenav\" data-jigconfig=\"smoothScroll: false, allHeadingLevels: [\'h2\'], headingExclude: \':hidden,.nomenu\'\"><span class=\"fm-sec half_rhythm no_top_margin\"><span class=\"fm-flexbox\"><span class=\"fm-citation\"><span class=\"part1\"><span class=\"cit-abbr\"> </span><span class=\"cit-pub-date\">2020; </span><span class=\"cit-vol\">15</span><span class=\"cit-iss\">(10)</span><span class=\"cit-flpgs\">: e0240726. </span></span><span class=\"part2\"><span class=\"fm-vol-iss-date\">Published online 2020 Oct 23. </span><span class=\"doi\"><span>doi: </span><a href=\"//dx.doi.org/10.1371%2Fjournal.pone.0240726.r006\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">10.1371/journal.pone.0240726.r006</a></span></span></span></span><h1 class=\"sub-content-title\">Acceptance letter</h1><span class=\"half_rhythm\"><span class=\"contrib-group fm-author\"><a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Chiesi%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=33095762\">Francesca Chiesi</a>, <span class=\"fm-role\">Academic Editor</span></span><span style=\"display:none\" class=\"contrib-group aff-tip\"></span></span><span class=\"fm-panel half_rhythm\"><span class=\"togglers fm-copyright-license\"> <a href=\"#\" class=\"pmctoggle\" rid=\"idm140210183990672_cpl\">Copyright and License information</a> <a href=\"/pmc/about/disclaimer/\" style=\"margin-left: 1em\">Disclaimer</a></span><span class=\"fm-authors-info fm-panel half_rhythm\"><span></span></span><span class=\"fm-article-notes fm-panel half_rhythm\"></span><span class=\"permissions fm-panel half_rhythm hide\" id=\"idm140210183990672_cpl\" style=\"display:none\"><span class=\"fm-copyright half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright</a> &#x000a9; 2020 Francesca Chiesi</span><span class=\"fm-copyright half_rhythm\">This is an open access article distributed under the terms of the <a href=\"http://creativecommons.org/licenses/by/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></span></span><br><div class=\"ref-list-sec sec\" id=\"reference-list\"><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref001\">1. <span class=\"mixed-citation\">Egenfeldt-Nielsen S, Smith JH, Tosca SP. <span class=\"ref-journal\">Understanding video games: The essential introduction</span>. 4th ed. New York: Routledge; 2019. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Understanding+video+games:+The+essential+introduction&amp;author=S+Egenfeldt-Nielsen&amp;author=JH+Smith&amp;author=SP+Tosca&amp;publication_year=2019&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref002\">2. <span class=\"mixed-citation\">Asociaci&#x000f3;n Espa&#x000f1;ola de Videojuegos (AEVI). El anuario del videojuego [The yearbook of videogames]. 2019 June 1 [cited 2 April 2020]. Available from: <a href=\"http://www.aevi.org.es/web/wp-content/uploads/2019/05/AEVI_Anuario_2018.pdf\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.aevi.org.es/web/wp-content/uploads/2019/05/AEVI_Anuario_2018.pdf</a>.</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref003\">3. <span class=\"mixed-citation\">American Psychiatric Association. <span class=\"ref-journal\">Diagnostic and statistical manual of mental disorders (DSM-5&#x000ae;)</span>. 5th ed. Arlington VA: American Psychiatric Publishing; 2013. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Diagnostic+and+statistical+manual+of+mental+disorders+(DSM-5&#x000ae;)&amp;publication_year=2013&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref004\">4. <span class=\"mixed-citation\">World Health Organization. ICD-11 beta draft. Mental, behavioural or neurodevelopmental disorders. 2017 [cited 2 April 2020]. Available from <a href=\"https://icd.who.int/dev11/lm/en#/http%3a%2f%2fid.who.int%2ficd%2fentity%2f338347362\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI\" target=\"_blank\">https://icd.who.int/dev11/lm/en#/http%3a%2f%2fid.who.int%2ficd%2fentity%2f338347362</a>.</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref005\">5. <span class=\"mixed-citation\">Petry NM, Rehbein F, Ko CH, O&#x02019;Brien CP. <span class=\"ref-title\">Internet gaming disorder in the DSM-5.</span>\n<span class=\"ref-journal\">Curr Psychiatry Rep.</span>\n2015;<span class=\"ref-vol\">17</span>: 72&#x02013;80. 10.1007/s11920-015-0610-0\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26216590\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs11920-015-0610-0\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Curr+Psychiatry+Rep.&amp;title=Internet+gaming+disorder+in+the+DSM-5.&amp;author=NM+Petry&amp;author=F+Rehbein&amp;author=CH+Ko&amp;author=CP+O&#x02019;Brien&amp;volume=17&amp;publication_year=2015&amp;pages=72-80&amp;pmid=26216590&amp;doi=10.1007/s11920-015-0610-0&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref006\">6. <span class=\"mixed-citation\">Gentile DA, Bailey K, Bavelier D, Brockmyer JF, Cash H, Coyne S M, et al.\n<span class=\"ref-title\">Internet gaming disorder in children and adolescents</span>. <span class=\"ref-journal\">Pediatrics</span>. 2017;<span class=\"ref-vol\">140</span>: S81&#x02013;S85. 10.1542/peds.2016-1758H\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n29093038\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1542%2Fpeds.2016-1758H\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Pediatrics&amp;title=Internet+gaming+disorder+in+children+and+adolescents&amp;author=DA+Gentile&amp;author=K+Bailey&amp;author=D+Bavelier&amp;author=JF+Brockmyer&amp;author=H+Cash&amp;volume=140&amp;publication_year=2017&amp;pages=S81-S85&amp;pmid=29093038&amp;doi=10.1542/peds.2016-1758H&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref007\">7. <span class=\"mixed-citation\">Mihara S, Higuchi S. <span class=\"ref-title\">Cross&#x02010;sectional and longitudinal epidemiological studies of internet gaming disorder: A systematic review of the literature</span>. <span class=\"ref-journal\">Psychiatry Clin Neurosci</span>. 2017;<span class=\"ref-vol\">71</span>: 425&#x02013;444. 10.1111/pcn.12532\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28436212\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fpcn.12532\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychiatry+Clin+Neurosci&amp;title=Cross&#x02010;sectional+and+longitudinal+epidemiological+studies+of+internet+gaming+disorder:+A+systematic+review+of+the+literature&amp;author=S+Mihara&amp;author=S+Higuchi&amp;volume=71&amp;publication_year=2017&amp;pages=425-444&amp;pmid=28436212&amp;doi=10.1111/pcn.12532&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref008\">8. <span class=\"mixed-citation\">Buiza-Aguado C, Alonso-Canovas A, Conde-Mateos C, Buiza-Navarrete JJ, Gentile D. <span class=\"ref-title\">Problematic video gaming in a young Spanish population: Association with psychosocial health.</span>\n<span class=\"ref-journal\">Cyberpsychol Behav Soc Netw</span>. 2018;<span class=\"ref-vol\">21</span>: 388&#x02013;394. 10.1089/cyber.2017.0599\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n29792521\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1089%2Fcyber.2017.0599\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cyberpsychol+Behav+Soc+Netw&amp;title=Problematic+video+gaming+in+a+young+Spanish+population:+Association+with+psychosocial+health.&amp;author=C+Buiza-Aguado&amp;author=A+Alonso-Canovas&amp;author=C+Conde-Mateos&amp;author=JJ+Buiza-Navarrete&amp;author=D+Gentile&amp;volume=21&amp;publication_year=2018&amp;pages=388-394&amp;pmid=29792521&amp;doi=10.1089/cyber.2017.0599&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref009\">9. <span class=\"mixed-citation\">L&#x000f3;pez-Fern&#x000e1;ndez FJ, Mezquita L, Griffiths MD, Ortet F, Ib&#x000e1;&#x000f1;ez MI. <span class=\"ref-title\">The role of personality on disordered gaming and game genre preferences in adolescence: gender differences and person-environment transactions.</span>\n<span class=\"ref-journal\">Adicciones.</span>\n2020. Advance publication online: doi: 10.20882/adicciones.0.0 [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n32100046\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.20882%2Fadicciones.0.0\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Adicciones.&amp;title=The+role+of+personality+on+disordered+gaming+and+game+genre+preferences+in+adolescence:+gender+differences+and+person-environment+transactions.&amp;author=FJ+L&#x000f3;pez-Fern&#x000e1;ndez&amp;author=L+Mezquita&amp;author=MD+Griffiths&amp;author=F+Ortet&amp;author=MI+Ib&#x000e1;&#x000f1;ez&amp;publication_year=2020&amp;doi=10.20882/adicciones.0.0&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref010\">10. <span class=\"mixed-citation\">Lopez-Fernandez O, Honrubia-Serrano ML, Baguley T, Griffiths MD. <span class=\"ref-title\">Pathological video game playing in Spanish and British adolescents: Towards the exploration of Internet Gaming Disorder symptomatology</span>. <span class=\"ref-journal\">Comput Human Behav</span>. 2014;<span class=\"ref-vol\">41</span>: 304&#x02013;312. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Human+Behav&amp;title=Pathological+video+game+playing+in+Spanish+and+British+adolescents:+Towards+the+exploration+of+Internet+Gaming+Disorder+symptomatology&amp;author=O+Lopez-Fernandez&amp;author=ML+Honrubia-Serrano&amp;author=T+Baguley&amp;author=MD+Griffiths&amp;volume=41&amp;publication_year=2014&amp;pages=304-312&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref011\">11. <span class=\"mixed-citation\">Gervasi AM, La Marca L, Costanzo A, Pace U, Guglielmucci F, Schimmenti A. <span class=\"ref-title\">Personality and Internet gaming disorder: A systematic review of recent literature</span>. <span class=\"ref-journal\">Curr Addict Rep</span>. 2017;<span class=\"ref-vol\">4</span>: 293&#x02013;307. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Curr+Addict+Rep&amp;title=Personality+and+Internet+gaming+disorder:+A+systematic+review+of+recent+literature&amp;author=AM+Gervasi&amp;author=L+La+Marca&amp;author=A+Costanzo&amp;author=U+Pace&amp;author=F+Guglielmucci&amp;volume=4&amp;publication_year=2017&amp;pages=293-307&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref012\">12. <span class=\"mixed-citation\">Griffiths MD, Kuss DJ, King DL. <span class=\"ref-title\">Video game addiction: Past, present and future.</span>\n<span class=\"ref-journal\">Curr Psychiatry Rev.</span>\n2012;<span class=\"ref-vol\">8</span>: 308&#x02013;318. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Curr+Psychiatry+Rev.&amp;title=Video+game+addiction:+Past,+present+and+future.&amp;author=MD+Griffiths&amp;author=DJ+Kuss&amp;author=DL+King&amp;volume=8&amp;publication_year=2012&amp;pages=308-318&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref013\">13. <span class=\"mixed-citation\">Brunborg GS, Mentzoni RA, Fr&#x000f8;yland LR. <span class=\"ref-title\">Is video gaming, or video game addiction, associated with depression, academic achievement, heavy episodic drinking, or conduct problems?</span>\n<span class=\"ref-journal\">J Behav Addict</span>. 2014;<span class=\"ref-vol\">3</span>: 27&#x02013;32. 10.1556/JBA.3.2014.002\n <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4117274/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25215212\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1556%2FJBA.3.2014.002\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Behav+Addict&amp;title=Is+video+gaming,+or+video+game+addiction,+associated+with+depression,+academic+achievement,+heavy+episodic+drinking,+or+conduct+problems?&amp;author=GS+Brunborg&amp;author=RA+Mentzoni&amp;author=LR+Fr&#x000f8;yland&amp;volume=3&amp;publication_year=2014&amp;pages=27-32&amp;pmid=25215212&amp;doi=10.1556/JBA.3.2014.002&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref014\">14. <span class=\"mixed-citation\">Gentile DA, Choo H, Liau A, Sim T, Li D, Fung D, et al.\n<span class=\"ref-title\">Pathological video game use among youths: A two-year longitudinal study</span>. <span class=\"ref-journal\">Pediatrics</span>. 2011;<span class=\"ref-vol\">127</span>: 319&#x02013;329. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21242221\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Pediatrics&amp;title=Pathological+video+game+use+among+youths:+A+two-year+longitudinal+study&amp;author=DA+Gentile&amp;author=H+Choo&amp;author=A+Liau&amp;author=T+Sim&amp;author=D+Li&amp;volume=127&amp;publication_year=2011&amp;pages=319-329&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref015\">15. <span class=\"mixed-citation\">Lemmens JS, Valkenburg PM., Peter J. <span class=\"ref-title\">Psychosocial causes and consequences of pathological gaming.</span>\n<span class=\"ref-journal\">Comput Human Behav.</span>\n2011;<span class=\"ref-vol\">27</span>: 144&#x02013;152. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Human+Behav.&amp;title=Psychosocial+causes+and+consequences+of+pathological+gaming.&amp;author=JS+Lemmens&amp;author=PM+Valkenburg&amp;author=J+Peter&amp;volume=27&amp;publication_year=2011&amp;pages=144-152&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref016\">16. <span class=\"mixed-citation\">Lemmens JS, Valkenburg PM., Peter J. <span class=\"ref-title\">The effects of pathological gaming on aggressive behavior.</span>\n<span class=\"ref-journal\">J. Youth Adolesc</span>. 2011;<span class=\"ref-vol\">40</span>: 38&#x02013;47. 10.1007/s10964-010-9558-x\n <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3003785/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20549320\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs10964-010-9558-x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J.+Youth+Adolesc&amp;title=The+effects+of+pathological+gaming+on+aggressive+behavior.&amp;author=JS+Lemmens&amp;author=PM+Valkenburg&amp;author=J+Peter&amp;volume=40&amp;publication_year=2011&amp;pages=38-47&amp;pmid=20549320&amp;doi=10.1007/s10964-010-9558-x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref017\">17. <span class=\"mixed-citation\">van Rooij AJ, Schoenmakers TM, Vermulst AA, van den Eijnden R, van de Mheen D. <span class=\"ref-title\">Online video game addiction: Identification of addicted adolescent gamers.</span>\n<span class=\"ref-journal\">Addiction</span>. 2011;<span class=\"ref-vol\">106</span>: 205&#x02013;212. 10.1111/j.1360-0443.2010.03104.x\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20840209\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1360-0443.2010.03104.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Addiction&amp;title=Online+video+game+addiction:+Identification+of+addicted+adolescent+gamers.&amp;author=AJ+van+Rooij&amp;author=TM+Schoenmakers&amp;author=AA+Vermulst&amp;author=R+van+den+Eijnden&amp;author=D+van+de+Mheen&amp;volume=106&amp;publication_year=2011&amp;pages=205-212&amp;pmid=20840209&amp;doi=10.1111/j.1360-0443.2010.03104.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref018\">18. <span class=\"mixed-citation\">Griffiths MD, Nuyens F. <span class=\"ref-title\">An overview of structural characteristics in problematic videogame playing.</span>\n<span class=\"ref-journal\">Curr Addict Rep</span>. 2017;<span class=\"ref-vol\">4</span>: 272&#x02013;283. 10.1007/s40429-017-0162-y\n <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5554268/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28845381\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1007%2Fs40429-017-0162-y\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Curr+Addict+Rep&amp;title=An+overview+of+structural+characteristics+in+problematic+videogame+playing.&amp;author=MD+Griffiths&amp;author=F+Nuyens&amp;volume=4&amp;publication_year=2017&amp;pages=272-283&amp;pmid=28845381&amp;doi=10.1007/s40429-017-0162-y&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref019\">19. <span class=\"mixed-citation\">King DL, Delfabbro PH, Griffiths MD. <span class=\"ref-title\">Video game structural characteristics: A new psychological taxonomy.</span>\n<span class=\"ref-journal\">Int J Ment Health Addict</span>. 2010;<span class=\"ref-vol\">8</span>: 90&#x02013;106. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int+J+Ment+Health+Addict&amp;title=Video+game+structural+characteristics:+A+new+psychological+taxonomy.&amp;author=DL+King&amp;author=PH+Delfabbro&amp;author=MD+Griffiths&amp;volume=8&amp;publication_year=2010&amp;pages=90-106&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref020\">20. <span class=\"mixed-citation\">Brand M, Young KS, Laier C, W&#x000f6;lfling K, Potenza MN. <span class=\"ref-title\">Integrating psychological and neurobiological considerations regarding the development and maintenance of specific Internet-use disorders: An Interaction of Person-Affect-Cognition-Execution (I-PACE) model.</span>\n<span class=\"ref-journal\">Neurosci Biobehav Rev.</span>\n2016;<span class=\"ref-vol\">71</span>: 252&#x02013;266. 10.1016/j.neubiorev.2016.08.033\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27590829\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.neubiorev.2016.08.033\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Neurosci+Biobehav+Rev.&amp;title=Integrating+psychological+and+neurobiological+considerations+regarding+the+development+and+maintenance+of+specific+Internet-use+disorders:+An+Interaction+of+Person-Affect-Cognition-Execution+(I-PACE)+model.&amp;author=M+Brand&amp;author=KS+Young&amp;author=C+Laier&amp;author=K+W&#x000f6;lfling&amp;author=MN+Potenza&amp;volume=71&amp;publication_year=2016&amp;pages=252-266&amp;pmid=27590829&amp;doi=10.1016/j.neubiorev.2016.08.033&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref021\">21. <span class=\"mixed-citation\">Dreier M, W&#x000f6;lfling K, M&#x000fc;ller K. <span class=\"ref-title\">Psychological research and a sociological perspective on problematic and addictive computer game use in adolescence</span>. <span class=\"ref-journal\">Int J Child Adolesc health</span>. 2013;<span class=\"ref-vol\">6</span>: 422&#x02013;435. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int+J+Child+Adolesc+health&amp;title=Psychological+research+and+a+sociological+perspective+on+problematic+and+addictive+computer+game+use+in+adolescence&amp;author=M+Dreier&amp;author=K+W&#x000f6;lfling&amp;author=K+M&#x000fc;ller&amp;volume=6&amp;publication_year=2013&amp;pages=422-435&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref022\">22. <span class=\"mixed-citation\">Cooper ML. <span class=\"ref-title\">Motivations for alcohol use among adolescents: Development and validation of a four-factor-model</span>. <span class=\"ref-journal\">Psychol Assess</span>. 1994;<span class=\"ref-vol\">6</span>: 117&#x02013;128. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychol+Assess&amp;title=Motivations+for+alcohol+use+among+adolescents:+Development+and+validation+of+a+four-factor-model&amp;author=ML+Cooper&amp;volume=6&amp;publication_year=1994&amp;pages=117-128&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref023\">23. <span class=\"mixed-citation\">Bravo AJ, Pilatti A, Pearson MR, Mezquita L, Ib&#x000e1;&#x000f1;ez MI, Ortet G. <span class=\"ref-title\">Depressive symptoms, ruminative thinking, drinking motives, and alcohol outcomes: A multiple mediation model among college students in three countries</span>. <span class=\"ref-journal\">Addict Behav</span>. 2018;<span class=\"ref-vol\">76</span>: 319&#x02013;327. 10.1016/j.addbeh.2017.08.028\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28889061\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.addbeh.2017.08.028\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Addict+Behav&amp;title=Depressive+symptoms,+ruminative+thinking,+drinking+motives,+and+alcohol+outcomes:+A+multiple+mediation+model+among+college+students+in+three+countries&amp;author=AJ+Bravo&amp;author=A+Pilatti&amp;author=MR+Pearson&amp;author=L+Mezquita&amp;author=MI+Ib&#x000e1;&#x000f1;ez&amp;volume=76&amp;publication_year=2018&amp;pages=319-327&amp;pmid=28889061&amp;doi=10.1016/j.addbeh.2017.08.028&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref024\">24. <span class=\"mixed-citation\">Mezquita L, Ruiz-Valero L, Mart&#x000ed;nez-G&#x000f3;mez N, Ib&#x000e1;&#x000f1;ez MI, Ortet G. <span class=\"ref-title\">Desarrollo y validaci&#x000f3;n de la versi&#x000f3;n breve del cuestionario de motivos de consumo de marihuana (MMM SF).</span>\n<span class=\"ref-journal\">Adicciones</span>. 2019;<span class=\"ref-vol\">31</span>: 106&#x02013;116. 10.20882/adicciones.979\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n29353298\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.20882%2Fadicciones.979\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Adicciones&amp;title=Desarrollo+y+validaci&#x000f3;n+de+la+versi&#x000f3;n+breve+del+cuestionario+de+motivos+de+consumo+de+marihuana+(MMM+SF).&amp;author=L+Mezquita&amp;author=L+Ruiz-Valero&amp;author=N+Mart&#x000ed;nez-G&#x000f3;mez&amp;author=MI+Ib&#x000e1;&#x000f1;ez&amp;author=G+Ortet&amp;volume=31&amp;publication_year=2019&amp;pages=106-116&amp;pmid=29353298&amp;doi=10.20882/adicciones.979&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref025\">25. <span class=\"mixed-citation\">Simons J, Correia CJ, Carey KB, Borsari BE. <span class=\"ref-title\">Validating a five-factor marijuana motives measure: Relations with use, problems, and alcohol motives</span>. <span class=\"ref-journal\">J Couns Psychol</span>. 1998;<span class=\"ref-vol\">45</span>: 265&#x02013;273. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Couns+Psychol&amp;title=Validating+a+five-factor+marijuana+motives+measure:+Relations+with+use,+problems,+and+alcohol+motives&amp;author=J+Simons&amp;author=CJ+Correia&amp;author=KB+Carey&amp;author=BE+Borsari&amp;volume=45&amp;publication_year=1998&amp;pages=265-273&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref026\">26. <span class=\"mixed-citation\">Lee HP, Chae PK, Lee HS, Kim YK. <span class=\"ref-title\">The five-factor gambling motivation model.</span>\n<span class=\"ref-journal\">Psychiatry Res.</span>\n2007;<span class=\"ref-vol\">150</span>: 21&#x02013;32. 10.1016/j.psychres.2006.04.005\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17270278\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.psychres.2006.04.005\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychiatry+Res.&amp;title=The+five-factor+gambling+motivation+model.&amp;author=HP+Lee&amp;author=PK+Chae&amp;author=HS+Lee&amp;author=YK+Kim&amp;volume=150&amp;publication_year=2007&amp;pages=21-32&amp;pmid=17270278&amp;doi=10.1016/j.psychres.2006.04.005&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref027\">27. <span class=\"mixed-citation\">Stewart SH, Zack M. <span class=\"ref-title\">Development and psychometric evaluation of a three&#x02010;dimensional Gambling Motives Questionnaire</span>. <span class=\"ref-journal\">Addiction</span>. 2008;<span class=\"ref-vol\">103</span>: 1110&#x02013;1117. 10.1111/j.1360-0443.2008.02235.x\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n18554344\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1360-0443.2008.02235.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Addiction&amp;title=Development+and+psychometric+evaluation+of+a+three&#x02010;dimensional+Gambling+Motives+Questionnaire&amp;author=SH+Stewart&amp;author=M+Zack&amp;volume=103&amp;publication_year=2008&amp;pages=1110-1117&amp;pmid=18554344&amp;doi=10.1111/j.1360-0443.2008.02235.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref028\">28. <span class=\"mixed-citation\">Ryan RM, Deci EL. <span class=\"ref-title\">Intrinsic and extrinsic motivations: Classic definitions and new directions.</span>\n<span class=\"ref-journal\">Contemp Educ Psychol</span>. 2000;<span class=\"ref-vol\">25</span>: 54&#x02013;67. 10.1006/ceps.1999.1020\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n10620381\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1006%2Fceps.1999.1020\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Contemp+Educ+Psychol&amp;title=Intrinsic+and+extrinsic+motivations:+Classic+definitions+and+new+directions.&amp;author=RM+Ryan&amp;author=EL+Deci&amp;volume=25&amp;publication_year=2000&amp;pages=54-67&amp;pmid=10620381&amp;doi=10.1006/ceps.1999.1020&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref029\">29. <span class=\"mixed-citation\">Blumler J, Katz E. <span class=\"ref-journal\">The uses of mass communications: Current perspectives on gratifications</span>. 1st ed. Beverly Hills: Sage; 1974. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=The+uses+of+mass+communications:+Current+perspectives+on+gratifications&amp;author=J+Blumler&amp;author=E+Katz&amp;publication_year=1974&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref030\">30. <span class=\"mixed-citation\">Bandura A. <span class=\"ref-journal\">Social foundations of thought and action: A social cognitive theory</span>. 1st ed. Englewood Cliffs: Prentice-Hall; 1986. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Social+foundations+of+thought+and+action:+A+social+cognitive+theory&amp;author=A+Bandura&amp;publication_year=1986&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref031\">31. <span class=\"mixed-citation\">Ryan RM, Rigby CS, Przybylski A. <span class=\"ref-title\">The motivational pull of video games: A self-determination theory approach.</span>\n<span class=\"ref-journal\">Motiv Emot.</span>\n2006;<span class=\"ref-vol\">30</span>: 344&#x02013;360. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Motiv+Emot.&amp;title=The+motivational+pull+of+video+games:+A+self-determination+theory+approach.&amp;author=RM+Ryan&amp;author=CS+Rigby&amp;author=A+Przybylski&amp;volume=30&amp;publication_year=2006&amp;pages=344-360&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref032\">32. <span class=\"mixed-citation\">Sherry JL, Lucas K, Greenberg BS, Lachlan K. <span class=\"ref-journal\">Video game uses and gratifications as predictors of use and game preference.</span> In: Vorderer P, Bryant J, editors. Playing computer games: Motives, responses, and consequences.\nMahwah: Lawrence Erlbaum; 2006.  pp. 213&#x02013;224. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Video+game+uses+and+gratifications+as+predictors+of+use+and+game+preference.&amp;author=JL+Sherry&amp;author=K+Lucas&amp;author=BS+Greenberg&amp;author=K+Lachlan&amp;author=P+Vorderer&amp;publication_year=2006&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref033\">33. <span class=\"mixed-citation\">De Grove F, Cauberghe V, Van Looy J. <span class=\"ref-title\">Development and validation of an instrument for measuring individual motives for playing digital games</span>. <span class=\"ref-journal\">Media Psychol</span>. 2016;<span class=\"ref-vol\">19</span>: 101&#x02013;125. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Media+Psychol&amp;title=Development+and+validation+of+an+instrument+for+measuring+individual+motives+for+playing+digital+games&amp;author=F+De+Grove&amp;author=V+Cauberghe&amp;author=J+Van+Looy&amp;volume=19&amp;publication_year=2016&amp;pages=101-125&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref034\">34. <span class=\"mixed-citation\">Yee N. <span class=\"ref-title\">Motivations for play in online games.</span>\n<span class=\"ref-journal\">Cyberpsychol Behav.</span>\n2006;<span class=\"ref-vol\">9</span>: 772&#x02013;775. 10.1089/cpb.2006.9.772\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17201605\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1089%2Fcpb.2006.9.772\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cyberpsychol+Behav.&amp;title=Motivations+for+play+in+online+games.&amp;author=N+Yee&amp;volume=9&amp;publication_year=2006&amp;pages=772-775&amp;pmid=17201605&amp;doi=10.1089/cpb.2006.9.772&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref035\">35. <span class=\"mixed-citation\">Yee N. <span class=\"ref-title\">The demographics, motivations, and derived experiences of users of massively multi-user online graphical environments.</span>\n<span class=\"ref-journal\">Presence (Camb).</span>\n2006;<span class=\"ref-vol\">15</span>: 309&#x02013;329. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Presence+(Camb).&amp;title=The+demographics,+motivations,+and+derived+experiences+of+users+of+massively+multi-user+online+graphical+environments.&amp;author=N+Yee&amp;volume=15&amp;publication_year=2006&amp;pages=309-329&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref036\">36. <span class=\"mixed-citation\">Demetrovics Z, Urb&#x000e1;n R, Nagygy&#x000f6;rgy K, Farkas J, Zilahy D, Merv&#x000f3; B, et al.\n<span class=\"ref-title\">Why do you play? The development of the motives for online gaming questionnaire (MOGQ).</span>\n<span class=\"ref-journal\">Behav Res Methods.</span>\n2011;<span class=\"ref-vol\">43</span>, 814&#x02013;825. 10.3758/s13428-011-0091-y\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21487899\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3758%2Fs13428-011-0091-y\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Behav+Res+Methods.&amp;title=Why+do+you+play?+The+development+of+the+motives+for+online+gaming+questionnaire+(MOGQ).&amp;author=Z+Demetrovics&amp;author=R+Urb&#x000e1;n&amp;author=K+Nagygy&#x000f6;rgy&amp;author=J+Farkas&amp;author=D+Zilahy&amp;volume=43&amp;publication_year=2011&amp;pages=814-825&amp;pmid=21487899&amp;doi=10.3758/s13428-011-0091-y&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref037\">37. <span class=\"mixed-citation\">Billieux J, Deleuze J, Griffiths MD, Kuss DJ. <span class=\"ref-journal\">Internet gaming addiction: The case of massively multiplayer online role-playing games. In: Textbook of addiction treatment: International perspectives</span>. Milan: Springer: 2015.  pp. 1515&#x02013;1525. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Internet+gaming+addiction:+The+case+of+massively+multiplayer+online+role-playing+games.+In:+Textbook+of+addiction+treatment:+International+perspectives&amp;author=J+Billieux&amp;author=J+Deleuze&amp;author=MD+Griffiths&amp;author=DJ+Kuss&amp;publication_year=2015&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref038\">38. <span class=\"mixed-citation\">Subramaniam M, Chua BY, Abdin E, Pang S, Satghare P, Vaingankar JA, et al.\n<span class=\"ref-title\">Prevalence and correlates of internet gaming problem among internet users: Results from an internet survey</span>. <span class=\"ref-journal\">Ann Acad Med Singapore</span>. 2016;<span class=\"ref-vol\">45</span>: 174&#x02013;83.  [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27383716\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Ann+Acad+Med+Singapore&amp;title=Prevalence+and+correlates+of+internet+gaming+problem+among+internet+users:+Results+from+an+internet+survey&amp;author=M+Subramaniam&amp;author=BY+Chua&amp;author=E+Abdin&amp;author=S+Pang&amp;author=P+Satghare&amp;volume=45&amp;publication_year=2016&amp;pages=174-83&amp;pmid=27383716&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref039\">39. <span class=\"mixed-citation\">Thorens G, Achab S, Billieux J, Khazaal Y, Khan R, Pivin E, et al.\n<span class=\"ref-title\">Characteristics and treatment response of self-identified problematic Internet users in a behavioral addiction outpatient clinic</span>. <span class=\"ref-journal\">J Behav Addict</span>. 2013;<span class=\"ref-vol\">3</span>: 78&#x02013;81. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4117276/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25215217\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Behav+Addict&amp;title=Characteristics+and+treatment+response+of+self-identified+problematic+Internet+users+in+a+behavioral+addiction+outpatient+clinic&amp;author=G+Thorens&amp;author=S+Achab&amp;author=J+Billieux&amp;author=Y+Khazaal&amp;author=R+Khan&amp;volume=3&amp;publication_year=2013&amp;pages=78-81&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref040\">40. <span class=\"mixed-citation\">Billieux J, Chanal J, Khazaal Y, Rochat L, Gay P, Zullino D. <span class=\"ref-title\">Psychological predictors of problematic involvement in Massively Multiplayer Online Role Playing Games (MMORPG): Illustration in a sample of male cybercaf&#x000e9;s players.</span>\n<span class=\"ref-journal\">Psychopathology</span>. 2011;<span class=\"ref-vol\">44</span>: 165&#x02013;171. 10.1159/000322525\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21372629\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1159%2F000322525\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychopathology&amp;title=Psychological+predictors+of+problematic+involvement+in+Massively+Multiplayer+Online+Role+Playing+Games+(MMORPG):+Illustration+in+a+sample+of+male+cybercaf&#x000e9;s+players.&amp;author=J+Billieux&amp;author=J+Chanal&amp;author=Y+Khazaal&amp;author=L+Rochat&amp;author=P+Gay&amp;volume=44&amp;publication_year=2011&amp;pages=165-171&amp;pmid=21372629&amp;doi=10.1159/000322525&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref041\">41. <span class=\"mixed-citation\">Billieux J, van der Linden M, Achab S, Khazaal Y, Paraskevopoulos L, Zullino D, et al.\n<span class=\"ref-title\">Why do you play World of Warcraft? An in-depth exploration of self-reported motivations to play online and in-game behaviours in the virtual world of Azeroth.</span>\n<span class=\"ref-journal\">Comput Human Behav.</span>\n2013;<span class=\"ref-vol\">29</span>: 103&#x02013;109. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Human+Behav.&amp;title=Why+do+you+play+World+of+Warcraft?+An+in-depth+exploration+of+self-reported+motivations+to+play+online+and+in-game+behaviours+in+the+virtual+world+of+Azeroth.&amp;author=J+Billieux&amp;author=M+van+der+Linden&amp;author=S+Achab&amp;author=Y+Khazaal&amp;author=L+Paraskevopoulos&amp;volume=29&amp;publication_year=2013&amp;pages=103-109&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref042\">42. <span class=\"mixed-citation\">Williams D, Yee N, Caplan SE. <span class=\"ref-title\">Who plays, how much, and why? Debunking the stereotypical gamer profile</span>. <span class=\"ref-journal\">J Comput Mediat Commun</span>. 2008;<span class=\"ref-vol\">13</span>: 993&#x02013;1018. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Comput+Mediat+Commun&amp;title=Who+plays,+how+much,+and+why?+Debunking+the+stereotypical+gamer+profile&amp;author=D+Williams&amp;author=N+Yee&amp;author=SE+Caplan&amp;volume=13&amp;publication_year=2008&amp;pages=993-1018&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref043\">43. <span class=\"mixed-citation\">Fuster H, Chamarro A, Carbonell X, Vallerand RJ. <span class=\"ref-title\">Relationship between passion and motivation for gaming in players of massively multiplayer online role-playing games.</span>\n<span class=\"ref-journal\">Cyberpsychol Behav Soc Netw</span>. 2014;<span class=\"ref-vol\">17</span>: 292&#x02013;297. 10.1089/cyber.2013.0349\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24611801\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1089%2Fcyber.2013.0349\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cyberpsychol+Behav+Soc+Netw&amp;title=Relationship+between+passion+and+motivation+for+gaming+in+players+of+massively+multiplayer+online+role-playing+games.&amp;author=H+Fuster&amp;author=A+Chamarro&amp;author=X+Carbonell&amp;author=RJ+Vallerand&amp;volume=17&amp;publication_year=2014&amp;pages=292-297&amp;pmid=24611801&amp;doi=10.1089/cyber.2013.0349&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref044\">44. <span class=\"mixed-citation\">Kahn AS, Shen C, Lu L, Ratan RA, Coary S, Hou J, et al.\n<span class=\"ref-title\">The Trojan Player Typology: A cross-genre, cross-cultural, behaviorally validated scale of video game play motivations.</span>\n<span class=\"ref-journal\">Comput Human Behav.</span>\n2015;<span class=\"ref-vol\">49</span>: 354&#x02013;361. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Human+Behav.&amp;title=The+Trojan+Player+Typology:+A+cross-genre,+cross-cultural,+behaviorally+validated+scale+of+video+game+play+motivations.&amp;author=AS+Kahn&amp;author=C+Shen&amp;author=L+Lu&amp;author=RA+Ratan&amp;author=S+Coary&amp;volume=49&amp;publication_year=2015&amp;pages=354-361&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref045\">45. <span class=\"mixed-citation\">Yee N, Ducheneaut N, Nelson L. Online gaming motivations scale: Development and validation. Proceedings of the ACM International Conference on Human Factors in Computing Systems (CHI 2012). New York: ACM; 2012; pp. 2803&#x02013;2806.</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref046\">46. <span class=\"mixed-citation\">Ghuman D, Griffiths M. <span class=\"ref-title\">A cross-genre study of online Gaming.</span>\n<span class=\"ref-journal\">Int. J. Cyber Behav. Psychol</span>. 2012;<span class=\"ref-vol\">2</span>: 13&#x02013;29. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int.+J.+Cyber+Behav.+Psychol&amp;title=A+cross-genre+study+of+online+Gaming.&amp;author=D+Ghuman&amp;author=M+Griffiths&amp;volume=2&amp;publication_year=2012&amp;pages=13-29&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref047\">47. <span class=\"mixed-citation\">Kim NR, Hwang SSH, Choi JS, Kim DJ, Demetrovics Z, Kir&#x000e1;ly O, et al.\n<span class=\"ref-title\">Characteristics and psychiatric symptoms of internet gaming disorder among adults using self-reported DSM-5 criteria.</span>\n<span class=\"ref-journal\">Psychiatry Investig.</span>\n2016;<span class=\"ref-vol\">13</span>: 58&#x02013;66. 10.4306/pi.2016.13.1.58\n <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4701686/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26766947\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.4306%2Fpi.2016.13.1.58\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychiatry+Investig.&amp;title=Characteristics+and+psychiatric+symptoms+of+internet+gaming+disorder+among+adults+using+self-reported+DSM-5+criteria.&amp;author=NR+Kim&amp;author=SSH+Hwang&amp;author=JS+Choi&amp;author=DJ+Kim&amp;author=Z+Demetrovics&amp;volume=13&amp;publication_year=2016&amp;pages=58-66&amp;pmid=26766947&amp;doi=10.4306/pi.2016.13.1.58&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref048\">48. <span class=\"mixed-citation\">Chang SM, Lin SS. <span class=\"ref-title\">Online gaming motive profiles in late adolescence and the related longitudinal development of stress, depression, and problematic internet use.</span>\n<span class=\"ref-journal\">Comput Educ.</span>\n2019;<span class=\"ref-vol\">135</span>: 123&#x02013;137. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Educ.&amp;title=Online+gaming+motive+profiles+in+late+adolescence+and+the+related+longitudinal+development+of+stress,+depression,+and+problematic+internet+use.&amp;author=SM+Chang&amp;author=SS+Lin&amp;volume=135&amp;publication_year=2019&amp;pages=123-137&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref049\">49. <span class=\"mixed-citation\">Kir&#x000e1;ly O, T&#x000f3;th D, Urb&#x000e1;n R, Demetrovics Z, Maraz A. <span class=\"ref-title\">Intense video gaming is not essentially problematic.</span>\n<span class=\"ref-journal\">Psychol Addict Behav</span>. 2017;<span class=\"ref-vol\">31</span>: 807&#x02013;817. 10.1037/adb0000316\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28956935\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1037%2Fadb0000316\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychol+Addict+Behav&amp;title=Intense+video+gaming+is+not+essentially+problematic.&amp;author=O+Kir&#x000e1;ly&amp;author=D+T&#x000f3;th&amp;author=R+Urb&#x000e1;n&amp;author=Z+Demetrovics&amp;author=A+Maraz&amp;volume=31&amp;publication_year=2017&amp;pages=807-817&amp;pmid=28956935&amp;doi=10.1037/adb0000316&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref050\">50. <span class=\"mixed-citation\">Laconi S, Pir&#x000e8;s S, Chabrol H. <span class=\"ref-title\">Internet gaming disorder, motives, game genres and psychopathology.</span>\n<span class=\"ref-journal\">Comput Human Behav.</span>\n2017;<span class=\"ref-vol\">75</span>: 652&#x02013;659. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Human+Behav.&amp;title=Internet+gaming+disorder,+motives,+game+genres+and+psychopathology.&amp;author=S+Laconi&amp;author=S+Pir&#x000e8;s&amp;author=H+Chabrol&amp;volume=75&amp;publication_year=2017&amp;pages=652-659&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref051\">51. <span class=\"mixed-citation\">M&#x000e4;nnikk&#x000f6; N, Billieux J, Nordstr&#x000f6;m T, Koivisto K, K&#x000e4;&#x000e4;ri&#x000e4;inen M. <span class=\"ref-title\">Problematic gaming behaviour in Finnish adolescents and young adults: Relation to game genres, gaming motives and self-awareness of problematic use.</span>\n<span class=\"ref-journal\">Int J Ment Health Addict.</span>\n2017;<span class=\"ref-vol\">15</span>: 324&#x02013;338. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Int+J+Ment+Health+Addict.&amp;title=Problematic+gaming+behaviour+in+Finnish+adolescents+and+young+adults:+Relation+to+game+genres,+gaming+motives+and+self-awareness+of+problematic+use.&amp;author=N+M&#x000e4;nnikk&#x000f6;&amp;author=J+Billieux&amp;author=T+Nordstr&#x000f6;m&amp;author=K+Koivisto&amp;author=M+K&#x000e4;&#x000e4;ri&#x000e4;inen&amp;volume=15&amp;publication_year=2017&amp;pages=324-338&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref052\">52. <span class=\"mixed-citation\">Montag C, Schivinski B, Sariyska R, Kannen C, Demetrovics Z, Pontes HM. <span class=\"ref-title\">Psychopathological symptoms and gaming motives in disordered gaming&#x02014;A psychometric comparison between the WHO and APA diagnostic frameworks.</span>\n<span class=\"ref-journal\">J Clin Med</span>. 2019;<span class=\"ref-vol\">8</span>: 1691&#x02013;1709. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC6832511/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n31618950\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Clin+Med&amp;title=Psychopathological+symptoms+and+gaming+motives+in+disordered+gaming&#x02014;A+psychometric+comparison+between+the+WHO+and+APA+diagnostic+frameworks.&amp;author=C+Montag&amp;author=B+Schivinski&amp;author=R+Sariyska&amp;author=C+Kannen&amp;author=Z+Demetrovics&amp;volume=8&amp;publication_year=2019&amp;pages=1691-1709&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref053\">53. <span class=\"mixed-citation\">Wu AM, Lai MH, Yu S, Lau JT, Lei MW. <span class=\"ref-title\">Motives for online gaming questionnaire: Its psychometric properties and correlation with Internet gaming disorder symptoms among Chinese people</span>. <span class=\"ref-journal\">J Behav Addict</span>. 2016;<span class=\"ref-vol\">6</span>: 11&#x02013;20. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5572999/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28264590\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Behav+Addict&amp;title=Motives+for+online+gaming+questionnaire:+Its+psychometric+properties+and+correlation+with+Internet+gaming+disorder+symptoms+among+Chinese+people&amp;author=AM+Wu&amp;author=MH+Lai&amp;author=S+Yu&amp;author=JT+Lau&amp;author=MW+Lei&amp;volume=6&amp;publication_year=2016&amp;pages=11-20&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref054\">54. <span class=\"mixed-citation\">Ballabio M, Griffiths MD, Urb&#x000e1;n R, Quartiroli A, Demetrovics Z, Kir&#x000e1;ly O. <span class=\"ref-title\">Do gaming motives mediate between psychiatric symptoms and problematic gaming? An empirical survey study.</span>\n<span class=\"ref-journal\"><em>Addict</em> Res <em>Theory</em></span>. 2017;<span class=\"ref-vol\">25</span>: 397&#x02013;408. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Addict+Res+Theory&amp;title=Do+gaming+motives+mediate+between+psychiatric+symptoms+and+problematic+gaming?+An+empirical+survey+study.&amp;author=M+Ballabio&amp;author=MD+Griffiths&amp;author=R+Urb&#x000e1;n&amp;author=A+Quartiroli&amp;author=Z+Demetrovics&amp;volume=25&amp;publication_year=2017&amp;pages=397-408&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref055\">55. <span class=\"mixed-citation\">Kir&#x000e1;ly O, Urb&#x000e1;n R, Griffiths MD, &#x000c1;goston C, Nagygy&#x000f6;rgy K, K&#x000f6;k&#x000f6;nyei G, et al.\n<span class=\"ref-title\">The mediating effect of gaming motivation between psychiatric symptoms and problematic online gaming: An online survey</span>. <span class=\"ref-journal\">J Med Internet Res</span>. 2015;<span class=\"ref-vol\">17</span>: 188&#x02013;202. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4405620/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25855558\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Med+Internet+Res&amp;title=The+mediating+effect+of+gaming+motivation+between+psychiatric+symptoms+and+problematic+online+gaming:+An+online+survey&amp;author=O+Kir&#x000e1;ly&amp;author=R+Urb&#x000e1;n&amp;author=MD+Griffiths&amp;author=C+&#x000c1;goston&amp;author=K+Nagygy&#x000f6;rgy&amp;volume=17&amp;publication_year=2015&amp;pages=188-202&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref056\">56. <span class=\"mixed-citation\">Plante CN, Gentile DA, Groves CL, Modlin A, Blanco-Herrera J. <span class=\"ref-title\">Video games as coping mechanisms in the etiology of video game addiction.</span>\n<span class=\"ref-journal\">Psychol. Pop. Media Cult</span>. 2019;<span class=\"ref-vol\">8</span>: 385&#x02013;394. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychol.+Pop.+Media+Cult&amp;title=Video+games+as+coping+mechanisms+in+the+etiology+of+video+game+addiction.&amp;author=CN+Plante&amp;author=DA+Gentile&amp;author=CL+Groves&amp;author=A+Modlin&amp;author=J+Blanco-Herrera&amp;volume=8&amp;publication_year=2019&amp;pages=385-394&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref057\">57. <span class=\"mixed-citation\">Greenberg BS, Sherry J, Lachlan K, Lucas K, Holmstrom A. <span class=\"ref-title\">Orientations to video games among gender and age groups.</span>\n<span class=\"ref-journal\">Simul Gaming.</span>\n2010;<span class=\"ref-vol\">41</span>: 238&#x02013;259. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Simul+Gaming.&amp;title=Orientations+to+video+games+among+gender+and+age+groups.&amp;author=BS+Greenberg&amp;author=J+Sherry&amp;author=K+Lachlan&amp;author=K+Lucas&amp;author=A+Holmstrom&amp;volume=41&amp;publication_year=2010&amp;pages=238-259&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref058\">58. <span class=\"mixed-citation\">Scharkow M, Festl R, Vogelgesang J, Quandt T. <span class=\"ref-title\">Beyond the &#x0201c;core-gamer&#x0201d;: Genre preferences and gratifications in computer games.</span>\n<span class=\"ref-journal\">Comput Human Behav</span>. 2015;<span class=\"ref-vol\">44</span>: 293&#x02013;298. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Human+Behav&amp;title=Beyond+the+&#x0201c;core-gamer&#x0201d;:+Genre+preferences+and+gratifications+in+computer+games.&amp;author=M+Scharkow&amp;author=R+Festl&amp;author=J+Vogelgesang&amp;author=T+Quandt&amp;volume=44&amp;publication_year=2015&amp;pages=293-298&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref059\">59. <span class=\"mixed-citation\">Hilgard J, Engelhardt CR, Bartholow BD. <span class=\"ref-title\">Individual differences in motives, preferences, and pathology in video games: The gaming attitudes, motives, and experiences scales (GAMES).</span>\n<span class=\"ref-journal\">Front Psychol</span>. 2013;<span class=\"ref-vol\">4</span>: 608\n10.3389/fpsyg.2013.00608\n <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3766857/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24058355\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3389%2Ffpsyg.2013.00608\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front+Psychol&amp;title=Individual+differences+in+motives,+preferences,+and+pathology+in+video+games:+The+gaming+attitudes,+motives,+and+experiences+scales+(GAMES).&amp;author=J+Hilgard&amp;author=CR+Engelhardt&amp;author=BD+Bartholow&amp;volume=4&amp;publication_year=2013&amp;pages=608&amp;pmid=24058355&amp;doi=10.3389/fpsyg.2013.00608&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref060\">60. <span class=\"mixed-citation\">Nije Bijvank M, Konijn EA, Bushman BJ. <span class=\"ref-title\">&#x0201c;We don&#x02019;t need no education&#x0201d;: Video game preferences, video game motivations, and aggressiveness among adolescent boys of different educational ability levels.</span>\n<span class=\"ref-journal\">J Adolesc.</span>\n2012;<span class=\"ref-vol\">35</span>: 153&#x02013;162. 10.1016/j.adolescence.2011.04.001\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21529925\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1016%2Fj.adolescence.2011.04.001\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Adolesc.&amp;title=&#x0201c;We+don&#x02019;t+need+no+education&#x0201d;:+Video+game+preferences,+video+game+motivations,+and+aggressiveness+among+adolescent+boys+of+different+educational+ability+levels.&amp;author=M+Nije+Bijvank&amp;author=EA+Konijn&amp;author=BJ+Bushman&amp;volume=35&amp;publication_year=2012&amp;pages=153-162&amp;pmid=21529925&amp;doi=10.1016/j.adolescence.2011.04.001&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref061\">61. <span class=\"mixed-citation\">Fuster H, Oberst U, Griffiths M, Carbonell X, Chamarro A, Talarn A. <span class=\"ref-title\">Psychological motivation in online role-playing games: A study of Spanish World of Warcraft players.</span>\n<span class=\"ref-journal\">An Psicol-Spain</span>. 2012;<span class=\"ref-vol\">28</span>: 274&#x02013;280. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=An+Psicol-Spain&amp;title=Psychological+motivation+in+online+role-playing+games:+A+study+of+Spanish+World+of+Warcraft+players.&amp;author=H+Fuster&amp;author=U+Oberst&amp;author=M+Griffiths&amp;author=X+Carbonell&amp;author=A+Chamarro&amp;volume=28&amp;publication_year=2012&amp;pages=274-280&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref062\">62. <span class=\"mixed-citation\">Calvert SL, Appelbaum M, Dodge KA, Graham S, Nagayama Hall GC, Hamby S. <span class=\"ref-title\">The American Psychological Association Task Force assessment of violent: Science in the service of public interest</span>. <span class=\"ref-journal\">Am. Psychol</span>. 2017;<span class=\"ref-vol\">72</span>: 126&#x02013;143. 10.1037/a0040413\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n28221065\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1037%2Fa0040413\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Am.+Psychol&amp;title=The+American+Psychological+Association+Task+Force+assessment+of+violent:+Science+in+the+service+of+public+interest&amp;author=SL+Calvert&amp;author=M+Appelbaum&amp;author=KA+Dodge&amp;author=S+Graham&amp;author=GC+Nagayama+Hall&amp;volume=72&amp;publication_year=2017&amp;pages=126-143&amp;pmid=28221065&amp;doi=10.1037/a0040413&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref063\">63. <span class=\"mixed-citation\">Cooper ML, Kuntsche E, Levitt A, Barber LL, Wolf S. <span class=\"ref-journal\">Motivational models of substance use: A review of theory and research on motives for using alcohol, marijuana, and tobacco.</span> In: Sher KJ, editor. <em>Oxford library of psychology</em>. <em>The Oxford handbook of substance use and substance use disorders</em>\nOxford: Oxford University Press; 2016.  pp. 375&#x02013;421. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Motivational+models+of+substance+use:+A+review+of+theory+and+research+on+motives+for+using+alcohol,+marijuana,+and+tobacco.&amp;author=ML+Cooper&amp;author=E+Kuntsche&amp;author=A+Levitt&amp;author=LL+Barber&amp;author=S+Wolf&amp;publication_year=2016&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref064\">64. <span class=\"mixed-citation\">Blanch A, Aluja A. <span class=\"ref-title\">Text mining a self-report back-translation.</span>\n<span class=\"ref-journal\">Psychol Assess.</span>\n2016;<span class=\"ref-vol\">28</span>: 750&#x02013;764. 10.1037/pas0000213\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26302100\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1037%2Fpas0000213\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychol+Assess.&amp;title=Text+mining+a+self-report+back-translation.&amp;author=A+Blanch&amp;author=A+Aluja&amp;volume=28&amp;publication_year=2016&amp;pages=750-764&amp;pmid=26302100&amp;doi=10.1037/pas0000213&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref065\">65. <span class=\"mixed-citation\">Geisinger KF. <span class=\"ref-title\">Cross-cultural normative assessment: Translation and adaptation issues influencing the normative interpretation of assessment instruments.</span>\n<span class=\"ref-journal\">Psychol Assess.</span>\n1994;<span class=\"ref-vol\">6</span>: 304&#x02013;312. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychol+Assess.&amp;title=Cross-cultural+normative+assessment:+Translation+and+adaptation+issues+influencing+the+normative+interpretation+of+assessment+instruments.&amp;author=KF+Geisinger&amp;volume=6&amp;publication_year=1994&amp;pages=304-312&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref066\">66. <span class=\"mixed-citation\">Watkins MW. <span class=\"ref-title\">Determining parallel analysis criteria.</span>\n<span class=\"ref-journal\">J Mod Appl Stat Methods</span>. 2006;<span class=\"ref-vol\">5</span>: 344&#x02013;346. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Mod+Appl+Stat+Methods&amp;title=Determining+parallel+analysis+criteria.&amp;author=MW+Watkins&amp;volume=5&amp;publication_year=2006&amp;pages=344-346&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref067\">67. <span class=\"mixed-citation\">Lautenschlager GJ. <span class=\"ref-title\">A comparison of alternatives to conducting Monte Carlo analyses for determining parallel analysis criteria.</span>\n<span class=\"ref-journal\">Multivariate Behav Res</span>.1989;<span class=\"ref-vol\">24</span>: 365&#x02013;395. 10.1207/s15327906mbr2403_6\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26750503\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1207%2Fs15327906mbr2403_6\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Multivariate+Behav+Res&amp;title=A+comparison+of+alternatives+to+conducting+Monte+Carlo+analyses+for+determining+parallel+analysis+criteria.&amp;author=GJ+Lautenschlager&amp;volume=24&amp;publication_year=1989&amp;pages=365-395&amp;pmid=26750503&amp;doi=10.1207/s15327906mbr2403_6&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref068\">68. <span class=\"mixed-citation\">Cliff N. <span class=\"ref-title\">The eigenvalues-greater-than-one rule and the reliability of components.</span>\n<span class=\"ref-journal\">Psychol Bull</span>. 1988;<span class=\"ref-vol\">103</span>: 276&#x02013;279. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychol+Bull&amp;title=The+eigenvalues-greater-than-one+rule+and+the+reliability+of+components.&amp;author=N+Cliff&amp;volume=103&amp;publication_year=1988&amp;pages=276-279&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref069\">69. <span class=\"mixed-citation\">Moya-Higueras J, Cuevas A, Marques-Feixa L, Mezquita L, Mayoral M, Fa&#x000f1;an&#x000e1;s L. <span class=\"ref-title\">Recent stressful life events (SLE) and adolescent mental health: initial validation of the LEIA, a new Checklist for SLE assessment according to their severity, interpersonal, and dependent nature.</span>\n<span class=\"ref-journal\">Assessment</span>. 2018 Epub ahead of print. 10.1177/1073191118817648. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n30539643\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1177%2F1073191118817648\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Assessment&amp;title=Recent+stressful+life+events+(SLE)+and+adolescent+mental+health:+initial+validation+of+the+LEIA,+a+new+Checklist+for+SLE+assessment+according+to+their+severity,+interpersonal,+and+dependent+nature.&amp;author=J+Moya-Higueras&amp;author=A+Cuevas&amp;author=L+Marques-Feixa&amp;author=L+Mezquita&amp;author=M+Mayoral&amp;doi=10.1177/1073191118817648&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref070\">70. <span class=\"mixed-citation\">Gentile D. <span class=\"ref-title\">Pathological video-game use among youth ages 8 to 18: A national study.</span>\n<span class=\"ref-journal\">Psychol Sci</span>. 2009;<span class=\"ref-vol\">20</span>: 594&#x02013;602. 10.1111/j.1467-9280.2009.02340.x\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19476590\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1111%2Fj.1467-9280.2009.02340.x\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Psychol+Sci&amp;title=Pathological+video-game+use+among+youth+ages+8+to+18:+A+national+study.&amp;author=D+Gentile&amp;volume=20&amp;publication_year=2009&amp;pages=594-602&amp;pmid=19476590&amp;doi=10.1111/j.1467-9280.2009.02340.x&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref071\">71. <span class=\"mixed-citation\">DeVellis RF. <span class=\"ref-journal\">Scale development: Theory and applications</span>. 1st ed. London: Sage; 2016. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Scale+development:+Theory+and+applications&amp;author=RF+DeVellis&amp;publication_year=2016&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref072\">72. <span class=\"mixed-citation\">Fuster H, Carbonell X, Pontes HM, Griffiths MD. <span class=\"ref-title\">Spanish validation of the internet gaming disorder-20 (IGD-20) test.</span>\n<span class=\"ref-journal\">Comput Human Behav.</span>\n2016;<span class=\"ref-vol\">56</span>: 215&#x02013;224. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Human+Behav.&amp;title=Spanish+validation+of+the+internet+gaming+disorder-20+(IGD-20)+test.&amp;author=H+Fuster&amp;author=X+Carbonell&amp;author=HM+Pontes&amp;author=MD+Griffiths&amp;volume=56&amp;publication_year=2016&amp;pages=215-224&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref073\">73. <span class=\"mixed-citation\">Bentler PM. <span class=\"ref-journal\">EQS structural equations program manual</span>. 6th ed. Los Angeles: BDMP Statistical Software; 1995. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=EQS+structural+equations+program+manual&amp;author=PM+Bentler&amp;publication_year=1995&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref074\">74. <span class=\"mixed-citation\">Byrne BM. <span class=\"ref-journal\">Structural equation modeling with EQS: Basic concepts, applications, and programming.</span>\n1st ed. London: Routledge; 2013. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Structural+equation+modeling+with+EQS:+Basic+concepts,+applications,+and+programming.&amp;author=BM+Byrne&amp;publication_year=2013&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref075\">75. <span class=\"mixed-citation\">Cox WM, Klinger E. <span class=\"ref-title\">A motivational model of alcohol use</span>. <span class=\"ref-journal\">J Abnorm Psychol</span>. 1988;<span class=\"ref-vol\">97</span>: 168&#x02013;180. 10.1037//0021-843x.97.2.168\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n3290306\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1037%2F%2F0021-843x.97.2.168\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Abnorm+Psychol&amp;title=A+motivational+model+of+alcohol+use&amp;author=WM+Cox&amp;author=E+Klinger&amp;volume=97&amp;publication_year=1988&amp;pages=168-180&amp;pmid=3290306&amp;doi=10.1037//0021-843x.97.2.168&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref076\">76. <span class=\"mixed-citation\">Mezquita L, Stewart SH, Ib&#x000e1;&#x000f1;ez MI, Ruip&#x000e9;rez MA, Villa H, Moya J, et al.\n<span class=\"ref-title\">Drinking motives in clinical and general populations.</span>\n<span class=\"ref-journal\">Eur Addict Res</span>. 2011;<span class=\"ref-vol\">17</span>: 250&#x02013;261. 10.1159/000328510\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21691118\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1159%2F000328510\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Eur+Addict+Res&amp;title=Drinking+motives+in+clinical+and+general+populations.&amp;author=L+Mezquita&amp;author=SH+Stewart&amp;author=MI+Ib&#x000e1;&#x000f1;ez&amp;author=MA+Ruip&#x000e9;rez&amp;author=H+Villa&amp;volume=17&amp;publication_year=2011&amp;pages=250-261&amp;pmid=21691118&amp;doi=10.1159/000328510&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref077\">77. <span class=\"mixed-citation\">Moudiab S, Spada MM. <span class=\"ref-title\">The relative contribution of motives and maladaptive cognitions to levels of Internet Gaming Disorder</span>. <span class=\"ref-journal\">Addict Behav Rep. 219</span>;<span class=\"ref-vol\">9</span>: 100&#x02013;160. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC6348280/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n30705935\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Addict+Behav+Rep.+219&amp;title=The+relative+contribution+of+motives+and+maladaptive+cognitions+to+levels+of+Internet+Gaming+Disorder&amp;author=S+Moudiab&amp;author=MM+Spada&amp;volume=9&amp;pages=100-160&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref078\">78. <span class=\"mixed-citation\">Li H, Zou Y, Wang J, Yang X. <span class=\"ref-title\">Role of stressful life events, avoidant coping styles, and neuroticism in online game addiction among college students: a moderated mediation model.</span>\n<span class=\"ref-journal\">Front. Psychol.</span>\n2016;<span class=\"ref-vol\">7</span>: 1&#x02013;11\n10.3389/fpsyg.2016.00001\n <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5118950/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27920734\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.3389%2Ffpsyg.2016.00001\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front.+Psychol.&amp;title=Role+of+stressful+life+events,+avoidant+coping+styles,+and+neuroticism+in+online+game+addiction+among+college+students:+a+moderated+mediation+model.&amp;author=H+Li&amp;author=Y+Zou&amp;author=J+Wang&amp;author=X+Yang&amp;volume=7&amp;publication_year=2016&amp;pages=1-11&amp;pmid=26858668&amp;doi=10.3389/fpsyg.2016.00001&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref079\">79. <span class=\"mixed-citation\">Ferguson CJ, Olson CK. <span class=\"ref-title\">Friends, fun, frustration and fantasy: Child motivations for video game play.</span>\n<span class=\"ref-journal\">Motiv Emot</span>. 2013;<span class=\"ref-vol\">37</span>: 154&#x02013;164. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Motiv+Emot&amp;title=Friends,+fun,+frustration+and+fantasy:+Child+motivations+for+video+game+play.&amp;author=CJ+Ferguson&amp;author=CK+Olson&amp;volume=37&amp;publication_year=2013&amp;pages=154-164&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref080\">80. <span class=\"mixed-citation\">Rehbein F, Kleimann M, M&#x000f6;&#x000df;le T. <span class=\"ref-title\">Prevalence and risk factors of video game dependency in adolescence: Results of a German nationwide survey.</span>\n<span class=\"ref-journal\">Cyberpsychol Behav Soc Netw.</span>\n2010;<span class=\"ref-vol\">13</span>: 269&#x02013;277. 10.1089/cyber.2009.0227\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20557246\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1089%2Fcyber.2009.0227\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cyberpsychol+Behav+Soc+Netw.&amp;title=Prevalence+and+risk+factors+of+video+game+dependency+in+adolescence:+Results+of+a+German+nationwide+survey.&amp;author=F+Rehbein&amp;author=M+Kleimann&amp;author=T+M&#x000f6;&#x000df;le&amp;volume=13&amp;publication_year=2010&amp;pages=269-277&amp;pmid=20557246&amp;doi=10.1089/cyber.2009.0227&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref081\">81. <span class=\"mixed-citation\">Hussain Z, Williams GA, Griffiths MD. <span class=\"ref-title\">An exploratory study of the association between online gaming addiction and enjoyment motivations for playing massively multiplayer online role-playing games.</span>\n<span class=\"ref-journal\">Comput Human Behav.</span>\n2015;<span class=\"ref-vol\">50</span>: 221&#x02013;230. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Human+Behav.&amp;title=An+exploratory+study+of+the+association+between+online+gaming+addiction+and+enjoyment+motivations+for+playing+massively+multiplayer+online+role-playing+games.&amp;author=Z+Hussain&amp;author=GA+Williams&amp;author=MD+Griffiths&amp;volume=50&amp;publication_year=2015&amp;pages=221-230&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref082\">82. <span class=\"mixed-citation\">Sobell LC, Sobell MB. Timeline followback. In: Litten R, Allen J, editors. Measuring alcohol consumption.\nTotowa: Humana Press; 1992.  pp. 41&#x02013;72. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar?q=Sobell+LC+,++Sobell+MB+.++Timeline+followback.++In:++Litten+R+,++Allen+J+,+editors.++Measuring+alcohol+consumption.+&#10;+Totowa+:++Humana+Press+;++1992++pp.++41+&#x02013;+72+.+\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref083\">83. <span class=\"mixed-citation\">Kircaburun K, Jonason PK, Griffiths MD. <span class=\"ref-title\">The Dark Tetrad traits and problematic online gaming: The mediating role of online gaming motives and moderating role of game types.</span>\n<span class=\"ref-journal\">Pers Individ Dif.</span>\n2018;<span class=\"ref-vol\">135</span>: 298&#x02013;303. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Pers+Individ+Dif.&amp;title=The+Dark+Tetrad+traits+and+problematic+online+gaming:+The+mediating+role+of+online+gaming+motives+and+moderating+role+of+game+types.&amp;author=K+Kircaburun&amp;author=PK+Jonason&amp;author=MD+Griffiths&amp;volume=135&amp;publication_year=2018&amp;pages=298-303&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref084\">84. <span class=\"mixed-citation\">Steadman JL. <span class=\"ref-title\">Gaming motivations as a guide to treating problematic gaming behaviors.</span>\n<span class=\"ref-journal\">J Technol Behav Sci</span>. 2019;<span class=\"ref-vol\">4</span>: 332&#x02013;339. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Technol+Behav+Sci&amp;title=Gaming+motivations+as+a+guide+to+treating+problematic+gaming+behaviors.&amp;author=JL+Steadman&amp;volume=4&amp;publication_year=2019&amp;pages=332-339&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref085\">85. <span class=\"mixed-citation\">Floros G, Siomos K. <span class=\"ref-title\">Patterns of choices on video game genres and internet addiction</span>. <span class=\"ref-journal\">Cyberpsychol Behav Soc Netw</span>. 2012;<span class=\"ref-vol\">15</span>: 417&#x02013;424. 10.1089/cyber.2012.0064\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22823448\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1089%2Fcyber.2012.0064\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cyberpsychol+Behav+Soc+Netw&amp;title=Patterns+of+choices+on+video+game+genres+and+internet+addiction&amp;author=G+Floros&amp;author=K+Siomos&amp;volume=15&amp;publication_year=2012&amp;pages=417-424&amp;pmid=22823448&amp;doi=10.1089/cyber.2012.0064&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref086\">86. <span class=\"mixed-citation\">Kim Y, Ross SD. <span class=\"ref-title\">An exploration of motives in sport video gaming.</span>\n<span class=\"ref-journal\">IJSMS</span>. 2006;<span class=\"ref-vol\">8</span>: 28&#x02013;40. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=IJSMS&amp;title=An+exploration+of+motives+in+sport+video+gaming.&amp;author=Y+Kim&amp;author=SD+Ross&amp;volume=8&amp;publication_year=2006&amp;pages=28-40&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref087\">87. <span class=\"mixed-citation\">Lafreni&#x000e8;re MAK, Verner-Filion J, Vallerand RJ. <span class=\"ref-title\">Development and validation of the Gaming Motivation Scale (GAMS).</span>\n<span class=\"ref-journal\">Pers Individ Dif.</span>\n2012;<span class=\"ref-vol\">53</span>: 827&#x02013;831. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Pers+Individ+Dif.&amp;title=Development+and+validation+of+the+Gaming+Motivation+Scale+(GAMS).&amp;author=MAK+Lafreni&#x000e8;re&amp;author=J+Verner-Filion&amp;author=RJ+Vallerand&amp;volume=53&amp;publication_year=2012&amp;pages=827-831&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref088\">88. <span class=\"mixed-citation\">Lee D, LaRose R. <span class=\"ref-title\">A socio-cognitive model of video game usage.</span>\n<span class=\"ref-journal\">J Broadcast Electron Media</span>. 2007;<span class=\"ref-vol\">51</span>: 632&#x02013;650. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Broadcast+Electron+Media&amp;title=A+socio-cognitive+model+of+video+game+usage.&amp;author=D+Lee&amp;author=R+LaRose&amp;volume=51&amp;publication_year=2007&amp;pages=632-650&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref089\">89. <span class=\"mixed-citation\">Li DD, Liau AK, Gentile DA, Khoo A, Cheong WD. <span class=\"ref-title\">Construct and predictive validity of a brief MMO player motivation scale: Cross-sectional and longitudinal evidence based on Singaporean young gamers.</span>\n<span class=\"ref-journal\">J Child Media</span>. 2013;<span class=\"ref-vol\">7</span>, 287&#x02013;306. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Child+Media&amp;title=Construct+and+predictive+validity+of+a+brief+MMO+player+motivation+scale:+Cross-sectional+and+longitudinal+evidence+based+on+Singaporean+young+gamers.&amp;author=DD+Li&amp;author=AK+Liau&amp;author=DA+Gentile&amp;author=A+Khoo&amp;author=WD+Cheong&amp;volume=7&amp;publication_year=2013&amp;pages=287-306&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref090\">90. <span class=\"mixed-citation\">Nacke LE, Bateman C, Mandryk RL. <span class=\"ref-title\">BrainHex: A neurobiological gamer typology survey.</span>\n<span class=\"ref-journal\">Entertain Comput</span>. 2014;<span class=\"ref-vol\">5</span>: 55&#x02013;62. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Entertain+Comput&amp;title=BrainHex:+A+neurobiological+gamer+typology+survey.&amp;author=LE+Nacke&amp;author=C+Bateman&amp;author=RL+Mandryk&amp;volume=5&amp;publication_year=2014&amp;pages=55-62&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref091\">91. <span class=\"mixed-citation\">Olson CK. <span class=\"ref-title\">Children&#x02019;s motivations for video game play in the context of normal development</span>. <span class=\"ref-journal\">Rev Gen Psychol</span>. 2010;<span class=\"ref-vol\">14</span>: 180&#x02013;187. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Rev+Gen+Psychol&amp;title=Children&#x02019;s+motivations+for+video+game+play+in+the+context+of+normal+development&amp;author=CK+Olson&amp;volume=14&amp;publication_year=2010&amp;pages=180-187&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref092\">92. <span class=\"mixed-citation\">Rodr&#x000ed;guez de Sep&#x000fa;lveda D, Igartua JJ. Creaci&#x000f3;n y validaci&#x000f3;n de una escala de motivos para videojugar. In: Ortega F, Carde&#x000f1;osa L, editors. <em>Nuevos medios, nueva comunicaci&#x000f3;n</em>. Salamanca: II Congreso Internacional Comunicaci&#x000f3;n 3.0. 2011. pp. 217&#x02013;234.</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref093\">93. <span class=\"mixed-citation\">Tondello GF, Wehbe RR, Diamond L, Busch M, Marczewski A, Nacke LE. The gamification user types hexad scale. In: Proceedings of the 2016 annual symposium on computer-human interaction in play. 2016. pp. 229&#x02013;243.</span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref094\">94. <span class=\"mixed-citation\">Wallenius M, Rimpel&#x000e4; A, Punam&#x000e4;ki RL, Lintonen T. <span class=\"ref-title\">Digital game playing motives among adolescents: Relations to parent&#x02013;child communication, school performance, sleeping habits, and perceived health.</span>\n<span class=\"ref-journal\">J Appl Dev Psychol</span>. 2009;<span class=\"ref-vol\">30</span>: 463&#x02013;474. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Appl+Dev+Psychol&amp;title=Digital+game+playing+motives+among+adolescents:+Relations+to+parent&#x02013;child+communication,+school+performance,+sleeping+habits,+and+perceived+health.&amp;author=M+Wallenius&amp;author=A+Rimpel&#x000e4;&amp;author=RL+Punam&#x000e4;ki&amp;author=T+Lintonen&amp;volume=30&amp;publication_year=2009&amp;pages=463-474&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref095\">95. <span class=\"mixed-citation\">Westwood D, Griffiths MD. <span class=\"ref-title\">The role of structural characteristics in video-game play motivation: A Q-methodology study.</span>\n<span class=\"ref-journal\">Cyberpsychol Behav Soc Netw</span>. 2010;<span class=\"ref-vol\">13</span>: 581&#x02013;585. 10.1089/cyber.2009.0361\n [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n20950185\" ref=\"reftype=pubmed&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] [<a href=\"//dx.doi.org/10.1089%2Fcyber.2009.0361\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI\">CrossRef</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Cyberpsychol+Behav+Soc+Netw&amp;title=The+role+of+structural+characteristics+in+video-game+play+motivation:+A+Q-methodology+study.&amp;author=D+Westwood&amp;author=MD+Griffiths&amp;volume=13&amp;publication_year=2010&amp;pages=581-585&amp;pmid=20950185&amp;doi=10.1089/cyber.2009.0361&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div><div class=\"ref-cit-blk half_rhythm\" id=\"pone.0240726.ref096\">96. <span class=\"mixed-citation\">Wu JH, Wang SC, Tsai HH. <span class=\"ref-title\">Falling in love with online games: The uses and gratifications perspective.</span>\n<span class=\"ref-journal\">Comput Human Behav.</span>\n2010;<span class=\"ref-vol\">26</span>: 1862&#x02013;1871. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Comput+Human+Behav.&amp;title=Falling+in+love+with+online+games:+The+uses+and+gratifications+perspective.&amp;author=JH+Wu&amp;author=SC+Wang&amp;author=HH+Tsai&amp;volume=26&amp;publication_year=2010&amp;pages=1862-1871&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=7584249&amp;issue-id=366205&amp;journal-id=440&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></div></div><br>', NULL, 2);
INSERT INTO `article` (`id_article`, `origin`, `num_access`, `title`, `abstract`, `year`, `journal`, `pmcid`, `status`, `html_xml`, `general_note`, `user`) VALUES
(82, 'pubmed', '28298430', 'metaSPAdes: a new versatile metagenomic assembler.', 'While metagenomics has emerged as a technology of choice for analyzing bacterial populations, the assembly of metagenomic data remains challenging, thus stifling biological discoveries. Moreover, recent studies revealed that complex bacterial populations may be composed from dozens of related strains, thus further amplifying the challenge of metagenomic assembly. metaSPAdes addresses various challenges of metagenomic assembly by capitalizing on computational ideas that proved to be useful in assemblies of single cells and highly polymorphic diploid genomes. We benchmark metaSPAdes against other state-of-the-art metagenome assemblers and demonstrate that it results in high-quality assemblies across diverse data sets.', 2017, 'Genome research', 'PMC5411777', 2, '<h1 class=\"content-title\">metaSPAdes: a new versatile metagenomic assembler</h1><br><span class=\"contrib-group fm-author\"><a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Nurk%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28298430\" class=\"affpopup\" co-rid=\"_co_idm139774693832512\" co-class=\"co-affbox\">Sergey Nurk</a>,<sup>1,</sup><sup>4</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Meleshko%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28298430\" class=\"affpopup\" co-rid=\"_co_idm139774693829904\" co-class=\"co-affbox\">Dmitry Meleshko</a>,<sup>1,</sup><sup>4</sup> <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Korobeynikov%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28298430\" class=\"affpopup\" co-rid=\"_co_idm139774691077584\" co-class=\"co-affbox\">Anton Korobeynikov</a>,<sup>1,</sup><sup>2</sup> and  <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Pevzner%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28298430\" class=\"affpopup\" co-rid=\"_co_idm139774691074128\" co-class=\"co-affbox\">Pavel A. Pevzner</a><sup>1,</sup><sup>3</sup></span><span style=\"display:none\" class=\"contrib-group aff-tip\"><span id=\"_co_idm139774693832512\"><h3 class=\"no_margin\">Sergey Nurk</h3><p><sup>1</sup>Center for Algorithmic Biotechnology, Institute for Translational Biomedicine, St. Petersburg State University, St. Petersburg, Russia 199004;</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Nurk%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28298430\">Sergey Nurk</a></span></span><span id=\"_co_idm139774693829904\"><h3 class=\"no_margin\">Dmitry Meleshko</h3><p><sup>1</sup>Center for Algorithmic Biotechnology, Institute for Translational Biomedicine, St. Petersburg State University, St. Petersburg, Russia 199004;</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Meleshko%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28298430\">Dmitry Meleshko</a></span></span><span id=\"_co_idm139774691077584\"><h3 class=\"no_margin\">Anton Korobeynikov</h3><p><sup>1</sup>Center for Algorithmic Biotechnology, Institute for Translational Biomedicine, St. Petersburg State University, St. Petersburg, Russia 199004;</p><p><sup>2</sup>Department of Statistical Modelling, St. Petersburg State University, St. Petersburg, Russia 198515;</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Korobeynikov%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28298430\">Anton Korobeynikov</a></span></span><span id=\"_co_idm139774691074128\"><h3 class=\"no_margin\">Pavel A. Pevzner</h3><p><sup>1</sup>Center for Algorithmic Biotechnology, Institute for Translational Biomedicine, St. Petersburg State University, St. Petersburg, Russia 199004;</p><p><sup>3</sup>Department of Computer Science and Engineering, University of California, San Diego, California 92093-0404, USA</p><span>Find articles by <a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n?term=Pevzner%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28298430\">Pavel A. Pevzner</a></span></span></span></span><span class=\"fm-panel half_rhythm\"><span class=\"togglers<br><span id=\"idm139774691551424\">Corresponding author: <a href=\"mailto:dev@null\" data-email=\"moc.liamg@krunyegres\" class=\"oemail\">moc.liamg@krunyegres</a></span></span><span class=\"fm-article-notes fm-panel hide half_rhythm\" id=\"idm139774688305136_an\" style=\"display:none\"><span class=\"fm-pubdate half_rhythm\">Received 2016 Aug 1; Accepted 2017 Mar 13.</span></span><span class=\"permissions fm-panel half_rhythm hide\" id=\"idm139774688305136_cpl\" style=\"display:none\"><span class=\"fm-copyright half_rhythm\"><a href=\"/pmc/about/copyright/\">Copyright</a> <a href=\"http://genome.cshlp.org/site/misc/terms.xhtml\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI\" target=\"_blank\">&#x000a9; 2017 Nurk et al.; Published by Cold Spring Harbor Laboratory Press</a></span><span class=\"license half_rhythm\">This article is distributed exclusively by Cold Spring Harbor Laboratory Press for the first six months after the full-issue publication date (see <a href=\"http://genome.cshlp.org/site/misc/terms.xhtml\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://genome.cshlp.org/site/misc/terms.xhtml</a>). After six months, it is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), as described at <a href=\"http://creativecommons.org/licenses/by-nc/4.0/\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://creativecommons.org/licenses/by-nc/4.0/</a>.</span></span></span><span id=\"pmclinksbox\" class=\"links-box whole_rhythm\"><span class=\"fm-panel\"><span>This article has been <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/citedby/\">cited by</a> other articles in PMC.</span></span></span></span><span class=\"sec\"></span><span id=\"ass-data\" class=\"tsec fm-sec whole_rhythm\" data-section=\"Featured_PMC_Datacitation\"><h2 class=\"nomenu\">Associated Data</h2><dl data-count=\"1\" class=\"box-data-suppmats whole_rhythm no_bottom_margin\"><dt> \r\n</dt><dd id=\"data-suppmats\"  \r\n><span class=\"half_rhythm\"><span class=\"caption half_rhythm no_bottom_margin\">\nSupplemental Material\n</span>\n\n<span><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/bin/supp_27_5_824__index.html\" data-ga-action=\"click_feat_suppl\">supp_27_5_824__index.html</a><span style=\"color:gray\"> (1.1K)</span></span><span class=\"small guid\">GUID: 3D2ADE42-802E-4189-86E1-3FBA83DC2409</span>\n<span><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/bin/supp_gr.213959.116_Supplemental_Table_S1.xlsx\" data-ga-action=\"click_feat_suppl\">supp_gr.213959.116_Supplemental_Table_S1.xlsx</a><span style=\"color:gray\"> (9.1K)</span></span><span class=\"small guid\">GUID: EBCCB73A-E6F4-4EF2-A978-93E016658108</span>\n<span><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/bin/supp_gr.213959.116_Supplemental_Table_S2.xlsx\" data-ga-action=\"click_feat_suppl\">supp_gr.213959.116_Supplemental_Table_S2.xlsx</a><span style=\"color:gray\"> (13K)</span></span><span class=\"small guid\">GUID: F9FA3B36-7028-49B0-BBDE-DEF95AF59BE4</span>\n<span><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/bin/supp_gr.213959.116_Supplemental_Table_S3.xlsx\" data-ga-action=\"click_feat_suppl\">supp_gr.213959.116_Supplemental_Table_S3.xlsx</a><span style=\"color:gray\"> (44K)</span></span><span class=\"small guid\">GUID: 99F2C03E-B7E1-4BAB-800A-55E3EC85BE97</span>\n<span><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/bin/supp_gr.213959.116_Supplemental_Information.pdf\" data-ga-action=\"click_feat_suppl\">supp_gr.213959.116_Supplemental_Information.pdf</a><span style=\"color:gray\"> (3.0M)</span></span><span class=\"small guid\">GUID: 9A7421B1-980F-46F7-BF97-11449F314EB9</span>\n<span><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/bin/supp_gr.213959.116_Supplemental.zip\" data-ga-action=\"click_feat_suppl\">supp_gr.213959.116_Supplemental.zip</a><span style=\"color:gray\"> (12M)</span></span><span class=\"small guid\">GUID: DDE5B1A2-D193-44AC-8AFB-191CEA65D253</span>\n<span><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/bin/supp_gr.213959.116_Supplemental_Material.pdf\" data-ga-action=\"click_feat_suppl\">supp_gr.213959.116_Supplemental_Material.pdf</a><span style=\"color:gray\"> (3.0M)</span></span><span class=\"small guid\">GUID: 11697CA8-0995-4AA0-AFA3-2BE8A66F9DBC</span>\n</span></dd></dl></span><span id=\"idm139774689461152\" lang=\"en\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139774689461152title\">Abstract</h2><!--article-meta--><span><p id=\"__p2\" class=\"p p-first-last\">While metagenomics has emerged as a technology of choice for analyzing bacterial populations, the assembly of metagenomic data remains challenging, thus stifling biological discoveries. Moreover, recent studies revealed that complex bacterial populations may be composed from dozens of related strains, thus further amplifying the challenge of metagenomic assembly. metaSPAdes addresses various challenges of metagenomic assembly by capitalizing on computational ideas that proved to be useful in assemblies of single cells and highly polymorphic diploid genomes. We benchmark metaSPAdes against other state-of-the-art metagenome assemblers and demonstrate that it results in high-quality assemblies across diverse data sets.</p></span></span><span id=\"idm139774689458288\" class=\"tsec sec\"><h2 class=\"headless nomenu\"></h2><p id=\"__p3\" class=\"p p-first\">Metagenome sequencing has emerged as a technology of choice for analyzing bacterial populations and the discovery of novel organisms and genes (<a href=\"#NURKGR213959C64\" rid=\"NURKGR213959C64\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388614\">Tyson et al. 2004</a>; <a href=\"#NURKGR213959C66\" rid=\"NURKGR213959C66\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388667\">Venter et al. 2004</a>; <a href=\"#NURKGR213959C71\" rid=\"NURKGR213959C71\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388568\">Yooseph et al. 2007</a>; <a href=\"#NURKGR213959C3\" rid=\"NURKGR213959C3\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388613\">Arumugam et al. 2011</a>). In one of the early metagenomics studies, <a href=\"#NURKGR213959C66\" rid=\"NURKGR213959C66\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388678\">Venter et al. (2004)</a> attempted to assemble the complex Sargasso Sea microbial community but, as the study stated, failed. On the other side of the spectrum of metagenomics studies, <a href=\"#NURKGR213959C64\" rid=\"NURKGR213959C64\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388681\">Tyson et al. (2004)</a> succeeded in assembling a simple microbial community consisting of a few species.</p><p id=\"__p4\">These landmark studies (<a href=\"#NURKGR213959C64\" rid=\"NURKGR213959C64\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388612\">Tyson et al. 2004</a>; <a href=\"#NURKGR213959C66\" rid=\"NURKGR213959C66\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388664\">Venter et al. 2004</a>) used conventional assembly tools&#x02014;namely, Celera (<a href=\"#NURKGR213959C46\" rid=\"NURKGR213959C46\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388601\">Myers et al. 2000</a>) and JAZZ (<a href=\"#NURKGR213959C2\" rid=\"NURKGR213959C2\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388621\">Aparicio et al. 2002</a>)&#x02014;with minor modifications. Since they were published, many specialized metagenomic assemblers have been developed (<a href=\"#NURKGR213959C31\" rid=\"NURKGR213959C31\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388572\">Koren et al. 2011</a>; <a href=\"#NURKGR213959C36\" rid=\"NURKGR213959C36\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388620\">Laserson et al. 2011</a>; <a href=\"#NURKGR213959C51\" rid=\"NURKGR213959C51\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388626\">Peng et al. 2011</a>, <a href=\"#NURKGR213959C52\" rid=\"NURKGR213959C52\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388682\">2012</a>; <a href=\"#NURKGR213959C9\" rid=\"NURKGR213959C9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388597\">Boisvert et al. 2012</a>; <a href=\"#NURKGR213959C48\" rid=\"NURKGR213959C48\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388575\">Namiki et al. 2012</a>; <a href=\"#NURKGR213959C21\" rid=\"NURKGR213959C21\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388648\">Haider et al. 2014</a>; <a href=\"#NURKGR213959C39\" rid=\"NURKGR213959C39\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388611\">Li et al. 2016</a>). However, bioinformaticians are still struggling to bridge the gap between assembling simple and complex microbial communities (for a review, see <a href=\"#NURKGR213959C19\" rid=\"NURKGR213959C19\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388571\">Gevers et al. 2012</a>). Meanwhile, many researchers succeeded in isolating abundant population genomes out of complex metagenomes (<a href=\"#NURKGR213959C22\" rid=\"NURKGR213959C22\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388595\">Hess et al. 2011</a>; <a href=\"#NURKGR213959C16\" rid=\"NURKGR213959C16\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388624\">Dupont et al. 2012</a>; <a href=\"#NURKGR213959C26\" rid=\"NURKGR213959C26\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388684\">Iverson et al. 2012</a>; CL Dupont, D Kaul, A Bankevich, DB Rusch, RA Richter, J Zhang, J Stuzka, V Montel, A Young, AE Allen, in prep.) by complementing de novo assembly with a partition of contigs into bins based on coverage depth, sequence composition, mate-pair information, and other criteria (<a href=\"#NURKGR213959C14\" rid=\"NURKGR213959C14\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388596\">Dick et al. 2009</a>; <a href=\"#NURKGR213959C68\" rid=\"NURKGR213959C68\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388602\">Wu and Ye 2011</a>; <a href=\"#NURKGR213959C69\" rid=\"NURKGR213959C69\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388565\">Wu et al. 2014</a>). However, this approach often faces difficulties because high fragmentation of metagenomic assemblies negatively affects both the accuracy of binning and the contiguity of genomes attributed to specific bins. Thus, development of better assemblers remains an important goal in metagenomics.</p><p id=\"__p5\">Recent applications of single-cell (<a href=\"#NURKGR213959C28\" rid=\"NURKGR213959C28\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388658\">Kashtan et al. 2014</a>) and TruSeq Synthetic Long Reads (TSLRs) (<a href=\"#NURKGR213959C60\" rid=\"NURKGR213959C60\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388615\">Sharon et al. 2015</a>) technologies revealed an enormous microdiversity of related strains within various microbial communities. While strains share most of the genomic sequence, they often have significant variation arising from mutations, insertions of mobile elements, genome rearrangements, or horizontal gene transfer. For example, single-cell sequencing revealed that the wild <em>Prochlorococcus</em> (the most abundant photosynthetic bacteria on earth) population can be viewed as a &#x0201c;federation&#x0201d; of hundreds of distinct subpopulations (some differing in &#x0003c;5% of positions) (<a href=\"#NURKGR213959C28\" rid=\"NURKGR213959C28\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388676\">Kashtan et al. 2014</a>; <a href=\"#NURKGR213959C8\" rid=\"NURKGR213959C8\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388591\">Biller et al. 2015</a>). Moreover, nearly all analyzed single cells carried at least one gene cassette not found in other cells from the same subpopulation. By using TSLRs, <a href=\"#NURKGR213959C60\" rid=\"NURKGR213959C60\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388607\">Sharon et al. (2015)</a> showed that the most abundant species in their sediment samples are represented by dozens of related strains. Moreover, investigators argued that this microdiversity was responsible for the poor reconstruction of the corresponding genomes from short-read libraries. But microdiversity is just one of many metagenomic assembly challenges that we discuss below.</p><p id=\"__p6\">First, widely different abundance levels of various species in a microbial sample result in a highly nonuniform read coverage across different genomes. Moreover, coverage of most species in a typical metagenomic data set is much lower than in a typical sequencing project of a cultivated sample. As a result, standard assembly techniques aimed at isolate genomes with high and rather uniform coverage generate fragmented and error-prone metagenomic assemblies.</p><p id=\"__p7\">Second, various species within a microbial community often share highly conserved genomic regions. Besides complicating the assembly and fragmenting contigs, such &#x0201c;interspecies repeats,&#x0201d; together with low coverage of most species, may trigger intergenomic assembly errors.</p><p id=\"__p8\">Third, many bacterial species in a microbial sample are represented by <em>strain mixtures</em>, that is, multiple related strains with varying abundances (Biller et al. 2014; <a href=\"#NURKGR213959C28\" rid=\"NURKGR213959C28\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388619\">Kashtan et al. 2014</a>; <a href=\"#NURKGR213959C56\" rid=\"NURKGR213959C56\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388583\">Rosen et al. 2015</a>; <a href=\"#NURKGR213959C60\" rid=\"NURKGR213959C60\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388631\">Sharon et al. 2015</a>). Although various studies outside the field of metagenomics extensively addressed a similar challenge of assembling two haplomes within a highly polymorphic eukaryotic genome (<a href=\"#NURKGR213959C13\" rid=\"NURKGR213959C13\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388656\">Dehal et al. 2002</a>; <a href=\"#NURKGR213959C67\" rid=\"NURKGR213959C67\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388629\">Vinson et al. 2005</a>; <a href=\"#NURKGR213959C15\" rid=\"NURKGR213959C15\" class=\" bibr popnode\">Donmez and Brudno 2011</a>; <a href=\"#NURKGR213959C27\" rid=\"NURKGR213959C27\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388599\">Kajitani et al. 2014</a>; <a href=\"#NURKGR213959C57\" rid=\"NURKGR213959C57\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388598\">Safonova et al. 2015</a>), assembly of many closely related bacterial strains is a somewhat different problem with unique computational challenges. While some studies described the initial steps toward identification of complex strain variants (<a href=\"#NURKGR213959C31\" rid=\"NURKGR213959C31\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388610\">Koren et al. 2011</a>; <a href=\"#NURKGR213959C51\" rid=\"NURKGR213959C51\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388627\">Peng et al. 2011</a>; <a href=\"#NURKGR213959C49\" rid=\"NURKGR213959C49\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388625\">Nijkamp et al. 2013</a>), popular metagenomic assembly tools (<a href=\"#NURKGR213959C9\" rid=\"NURKGR213959C9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388606\">Boisvert et al. 2012</a>; <a href=\"#NURKGR213959C52\" rid=\"NURKGR213959C52\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388635\">Peng et al. 2012</a>; <a href=\"#NURKGR213959C39\" rid=\"NURKGR213959C39\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388628\">Li et al. 2016</a>) still include only rudimentary procedures for assembling strain mixtures with high level of microdiversity.</p><p id=\"__p9\">We note that each of the challenges described above has already been addressed in the course of development of the SPAdes assembly toolkit, albeit in applications outside the field of metagenomics. SPAdes was initially developed to assemble data sets with nonuniform coverage, one of the key challenges of single-cell assembly (<a href=\"#NURKGR213959C6\" rid=\"NURKGR213959C6\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388637\">Bankevich et al. 2012</a>; <a href=\"#NURKGR213959C50\" rid=\"NURKGR213959C50\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388661\">Nurk et al. 2013</a>). exSPAnder repeat resolution module of SPAdes (<a href=\"#NURKGR213959C54\" rid=\"NURKGR213959C54\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388674\">Prjibelski et al. 2014</a>; <a href=\"#NURKGR213959C65\" rid=\"NURKGR213959C65\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388580\">Vasilinetc et al. 2015</a>; <a href=\"#NURKGR213959C1\" rid=\"NURKGR213959C1\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388630\">Antipov et al. 2016</a>) was developed to accurately resolve genomic repeats by combining multiple libraries sequenced with various technologies. Lastly, dipSPAdes (<a href=\"#NURKGR213959C57\" rid=\"NURKGR213959C57\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388652\">Safonova et al. 2015</a>) was developed to address the challenge of assembling a two-haplome mixture within a highly polymorphic diploid genome.</p><p id=\"__p10\">While these recently developed SPAdes tools address challenging assembly problems, metagenomic assembly is arguably an even more difficult problem with data set sizes that dwarf most other DNA sequencing projects. Nevertheless, despite the fact that SPAdes was not designed for metagenomics applications, various groups successfully applied it to their metagenomics studies (<a href=\"#NURKGR213959C42\" rid=\"NURKGR213959C42\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388617\">McLean et al. 2013</a>; <a href=\"#NURKGR213959C50\" rid=\"NURKGR213959C50\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388561\">Nurk et al. 2013</a>; <a href=\"#NURKGR213959C11\" rid=\"NURKGR213959C11\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388585\">Coates et al. 2014</a>; <a href=\"#NURKGR213959C12\" rid=\"NURKGR213959C12\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388579\">Cotten et al. 2014</a>; <a href=\"#NURKGR213959C7\" rid=\"NURKGR213959C7\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388594\">Bertin et al. 2015</a>; <a href=\"#NURKGR213959C18\" rid=\"NURKGR213959C18\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388662\">Garc&#x000ed;a-L&#x000f3;pez et al. 2015</a>; <a href=\"#NURKGR213959C29\" rid=\"NURKGR213959C29\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388647\">Kleigrewe et al. 2015</a>; <a href=\"#NURKGR213959C30\" rid=\"NURKGR213959C30\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388650\">Kleiner et al. 2015</a>; <a href=\"#NURKGR213959C45\" rid=\"NURKGR213959C45\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388582\">Miller et al. 2016</a>; <a href=\"#NURKGR213959C62\" rid=\"NURKGR213959C62\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388638\">Tsai et al. 2016</a>; <a href=\"#NURKGR213959C70\" rid=\"NURKGR213959C70\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388588\">Xie et al. 2016</a>). However, while SPAdes indeed works well for assembling low-complexity metagenomes like cyanobacterial filaments (<a href=\"#NURKGR213959C11\" rid=\"NURKGR213959C11\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388587\">Coates et al. 2014</a>), or MDA-amplified mixtures of a small number of randomly selected bacterial cells (<a href=\"#NURKGR213959C50\" rid=\"NURKGR213959C50\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388586\">Nurk et al. 2013</a>), its performance deteriorates in the case of complex bacterial communities.</p><p id=\"__p11\" class=\"p\">Our novel metaSPAdes software combines new algorithmic ideas with proven solutions from the SPAdes toolkit to address various challenges of metagenomic assembly. Below we describe algorithmic approaches used in metaSPAdes and benchmark it against the state-of-the-art metagenomic assemblers IDBA-UD (<a href=\"#NURKGR213959C52\" rid=\"NURKGR213959C52\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388589\">Peng et al. 2012</a>), Ray-Meta (<a href=\"#NURKGR213959C9\" rid=\"NURKGR213959C9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388675\">Boisvert et al. 2012</a>), and MEGAHIT (<a href=\"#NURKGR213959C38\" rid=\"NURKGR213959C38\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388646\">Li et al. 2015</a>).</p></span><span id=\"s1\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s1title\">Results</h2><span id=\"s1a\" class=\"sec sec-first\"><h3 id=\"s1atitle\">Outline of metaSPAdes pipeline</h3><p id=\"__p12\" class=\"p p-first-last\">metaSPAdes first constructs the de Bruijn graph of all reads using SPAdes, transforms it into the assembly graph using various graph simplification procedures, and reconstructs paths in the assembly graph that correspond to long genomic fragments within a metagenome (<a href=\"#NURKGR213959C6\" rid=\"NURKGR213959C6\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388666\">Bankevich et al. 2012</a>; <a href=\"#NURKGR213959C50\" rid=\"NURKGR213959C50\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388577\">Nurk et al. 2013</a>). metaSPAdes works across a wide range of coverage depths and attempts to maintain a trade-off between the accuracy and the contiguity of assemblies. Responding to the microdiversity challenge, metaSPAdes focuses on reconstructing a consensus backbone of a strain mixture, thus ignoring some strain-specific features corresponding to rare strains.</p></span><span id=\"s1b\" class=\"sec\"><h3 id=\"s1btitle\">Benchmarking challenge</h3><p id=\"__p13\" class=\"p p-first\">Genome assemblers are usually benchmarked on isolates with known reference genomes using various metrics (<a href=\"#NURKGR213959C58\" rid=\"NURKGR213959C58\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388564\">Salzberg et al. 2012</a>; <a href=\"#NURKGR213959C20\" rid=\"NURKGR213959C20\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388645\">Gurevich et al. 2013</a>). Benchmarking of metagenomic assemblers is a more difficult task because no reference metagenomes are available for microbial communities of even moderate complexity.</p><p id=\"__p14\">One approach to address this issue relies on identifying reference genomes closely related to some genomes in a metagenome (<a href=\"#NURKGR213959C31\" rid=\"NURKGR213959C31\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388634\">Koren et al. 2011</a>; <a href=\"#NURKGR213959C61\" rid=\"NURKGR213959C61\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388569\">Treangen et al. 2013</a>). However, this approach is limited since (1) closely related reference genomes are available only for a fraction of species in a metagenome, and (2) differences between identified references and their counterparts in a metagenome are often misinterpreted as assembly errors (see &#x0201c;Analysis of the HMP Dataset&#x0201d; in the <a href=\"http://genome.cshlp.org/lookup/suppl/doi:10.1101/gr.213959.116/-/DC1\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Supplemental Material</a>). Another approach to benchmarking metagenomic assemblers uses synthetic data sets with known community members. Such data sets can be obtained by sequencing the mixtures of bacteria with known genomes (<a href=\"#NURKGR213959C63\" rid=\"NURKGR213959C63\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388563\">Turnbaugh et al. 2007</a>; <a href=\"#NURKGR213959C59\" rid=\"NURKGR213959C59\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388640\">Shakya et al. 2013</a>), mixed from isolate sequencing data (<a href=\"#NURKGR213959C40\" rid=\"NURKGR213959C40\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388663\">Mavromatis et al. 2007</a>), or simulated from reference sequences (<a href=\"#NURKGR213959C55\" rid=\"NURKGR213959C55\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388660\">Richter et al. 2008</a>; <a href=\"#NURKGR213959C43\" rid=\"NURKGR213959C43\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388639\">Mende et al. 2012</a>). However, while synthetic data sets proved to be useful in various benchmarking efforts, they are typically less complex than real metagenomes (<a href=\"#NURKGR213959C31\" rid=\"NURKGR213959C31\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388567\">Koren et al. 2011</a>; <a href=\"#NURKGR213959C52\" rid=\"NURKGR213959C52\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388669\">Peng et al. 2012</a>).</p><p id=\"__p15\" class=\"p p-last\">We benchmarked metaSPAdes against three popular metagenomic assemblers&#x02014;IDBA-UD (<a href=\"#NURKGR213959C52\" rid=\"NURKGR213959C52\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388677\">Peng et al. 2012</a>), Ray-Meta (<a href=\"#NURKGR213959C9\" rid=\"NURKGR213959C9\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388578\">Boisvert et al. 2012</a>), and MEGAHIT (<a href=\"#NURKGR213959C38\" rid=\"NURKGR213959C38\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388622\">Li et al. 2015</a>)&#x02014;across diverse synthetic and real data sets using metaQUAST (<a href=\"#NURKGR213959C44\" rid=\"NURKGR213959C44\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388560\">Mikheenko et al. 2016</a>). The data sets were preprocessed as described in &#x0201c;Data Preprocessing&#x0201d; in the <a href=\"http://genome.cshlp.org/lookup/suppl/doi:10.1101/gr.213959.116/-/DC1\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Supplemental Material</a>.</p></span><span id=\"s1c\" class=\"sec\"><h3 id=\"s1ctitle\">Data sets</h3><p id=\"__p16\" class=\"p p-first\">We analyzed the following data sets.</p><span id=\"s1c1\" class=\"sec\"><h4 id=\"s1c1title\">Synthetic community data set</h4><p id=\"__p17\" class=\"p p-first-last\">Synthetic community data set (SYNTH) is a set of reads from the genomic DNA mixture of 64 diverse bacterial and archaeal species (SRA acc. no. SRX200676) (<a href=\"#NURKGR213959C59\" rid=\"NURKGR213959C59\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388673\">Shakya et al. 2013</a>) that was used for benchmarking the Omega assembler (<a href=\"#NURKGR213959C21\" rid=\"NURKGR213959C21\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388600\">Haider et al. 2014</a>). It contains 109 million Illumina HiSeq 100-bp paired-end reads with mean insert size of 206 bp. Since the reference genomes for all 64 species forming the SYNTH data set are known, we used them to assess the quality of various SYNTH assemblies.</p></span><span id=\"s1c2\" class=\"sec\"><h4 id=\"s1c2title\">Human Microbiome Project data set</h4><p id=\"__p18\" class=\"p p-first-last\">Human Microbiome Project data set (HMP) is a female tongue dorsum data set (SRA acc. no. SRX024329) generated by the Human Microbiome Project (<a href=\"#NURKGR213959C24\" rid=\"NURKGR213959C24\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388633\">The Human Microbiome Project Consortium 2012</a>) that was used for benchmarking by <a href=\"#NURKGR213959C51\" rid=\"NURKGR213959C51\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388623\">Peng et al. (2011)</a>, <a href=\"#NURKGR213959C61\" rid=\"NURKGR213959C61\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388649\">Treangen et al. (2013)</a>, and <a href=\"#NURKGR213959C44\" rid=\"NURKGR213959C44\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388644\">Mikheenko et al. (2016)</a>. It contains 75 million Illumina HiSeq 95-bp paired-end reads with mean insert size of 213 bp. Although the genomes comprising the HMP sample are unknown, we cautiously selected three reference genomes that are similar to the genomes within the sample for benchmarking.</p></span><span id=\"s1c3\" class=\"sec\"><h4 id=\"s1c3title\">Marine metagenome data set</h4><p id=\"__p19\" class=\"p p-first-last\">Marine metagenome data set (MARINE) is a 300-m-depth marine metagenome data set (SRA acc. no. SRX1991080) originating from the functional genomics study of an oxygen minimum zone in the equatorial Pacific (<a href=\"http://genome.jgi.doe.gov/FungenequPacific/FungenequPacific.info.html\">http://genome.jgi.doe.gov/FungenequPacific/FungenequPacific.info.html</a>). It contains 48 million Illumina HiSeq 150-bp paired-end reads with mean insert size of 245 bp.</p></span><span id=\"s1c4\" class=\"sec sec-last\"><h4 id=\"s1c4title\">Aquifer sediment data set</h4><p id=\"__p20\" class=\"p p-first\">Aquifer sediment data set (SOIL; SRA acc. no. SRX2021633) is one of the Illumina data sets from the study of the sediment samples collected in an aquifer adjacent to the Colorado River (<a href=\"#NURKGR213959C10\" rid=\"NURKGR213959C10\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388651\">Castelle et al. 2013</a>; <a href=\"#NURKGR213959C23\" rid=\"NURKGR213959C23\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388683\">Hug et al. 2013</a>), containing 32 million Illumina HiSeq 150-bp paired-end reads with mean insert size of 460 bp (even though aquifer sediments represent a different environment than soil, we took a liberty to refer to this data set as SOIL). <a href=\"#NURKGR213959C60\" rid=\"NURKGR213959C60\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388581\">Sharon et al. (2015)</a> improved on the original analysis using TSLR sequencing (<a href=\"#NURKGR213959C32\" rid=\"NURKGR213959C32\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388559\">Kuleshov et al. 2014</a>; <a href=\"#NURKGR213959C41\" rid=\"NURKGR213959C41\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388680\">McCoy et al. 2014</a>). Since the TSLR technology results in greatly improved metagenomics assemblies (<a href=\"#NURKGR213959C33\" rid=\"NURKGR213959C33\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388659\">Kuleshov et al. 2015</a>; <a href=\"#NURKGR213959C5\" rid=\"NURKGR213959C5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388665\">Bankevich and Pevzner 2016</a>; CL Dupont, D Kaul, A Bankevich, DB Rusch, RA Richter, J Zhang, J Stuzka, V Montel, A Young, AE Allen, in prep.), this data set provides a unique opportunity to benchmark various metagenomic assemblers based on how well they reconstruct genomic regions captured by TSLRs.</p><p id=\"__p21\" class=\"p p-last\"><a href=\"http://genome.cshlp.org/lookup/suppl/doi:10.1101/gr.213959.116/-/DC1\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Supplemental Materials</a> &#x0201c;CAMI Datasets&#x0201d; and &#x0201c;Analysis of CAMI Datasets&#x0201d; also present results of benchmarking on two synthetic data sets, simulated from reference genomes within the &#x0201c;Critical Assessment of Metagenome Interpretation&#x0201d; (CAMI) initiative (<a href=\"http://www.cami-challenge.org/\">http://www.cami-challenge.org/</a>).</p></span></span><span id=\"s1d\" class=\"sec\"><h3 id=\"s1dtitle\">Assembly parameters</h3><p id=\"__p22\" class=\"p p-first-last\">metaSPAdes from SPAdes v3.10 prerelease package (see &#x0201c;Software Availability&#x0201d; section), MEGAHIT v1.0.6.1, IDBA-UD v1.1.1, and Ray-Meta v2.3.1 have been launched in 16 threads with (mostly) default parameters. IDBA-UD was launched with read error-correction enabled as recommended in the manual for metagenomic data. Ray-Meta was launched with a <em>k</em>-mer size equal to 31. <a href=\"http://genome.cshlp.org/lookup/suppl/doi:10.1101/gr.213959.116/-/DC1\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Supplemental Table S1</a> provides information about the running time and memory footprints of various assemblers. All assemblers were benchmarked using metaQUAST from QUAST v4.5 package (<a href=\"#NURKGR213959C20\" rid=\"NURKGR213959C20\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388618\">Gurevich et al. 2013</a>) (with &#x0201c;-m 1000 --scaffolds&#x0201d; options). metaQUAST classifies a position in a scaffold as an <em>intragenomic misassembly</em> if its flanking regions align to nonconsecutive regions of the same reference genome and as an <em>intergenomic misassembly</em> if they align to different reference genomes.</p></span><span id=\"s1e\" class=\"sec sec-last\"><h3 id=\"s1etitle\">Benchmarking</h3><p id=\"__p23\" class=\"p p-first\">We benchmarked four assemblers (metaSPAdes, MEGAHIT, UDBA-UD, and Ray-Meta) on four data sets (SYNTH, HMP, MARINE, and SOIL) across a wide array of metrics described below.</p><span id=\"s1e1\" class=\"sec\"><h4 id=\"s1e1title\">Scaffold length statistics</h4><p id=\"__p24\" class=\"p p-first\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/table/NURKGR213959TB1/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959TB1\" rid-ob=\"ob-NURKGR213959TB1\" co-legend-rid=\"\"><span>Table 1</span></a> provides the scaffold length statistics and demonstrates that, with respect to the total length of scaffolds (&#x0003e;1 kb), Ray-Meta generated inferior results compared with other assemblers on all data sets. metaSPAdes significantly improved the total scaffold length in the case of the most diverse SOIL data set (21% and 40% increase compared with IDBA-UD and MEGAHIT, respectively). Although all tools generated assemblies with similar total scaffold length for the HMP data set, metaSPAdes significantly improved on the length of the 1000 longest scaffolds (36.5 Mb) compared with MEGAHIT (26.6 Mb) and IDBA-UD (29.4 Mb). Similarly, while metaSPAdes and IDBA-UD resulted in assemblies of similar total length on MARINE data set (269.5 and 273.7 Mb, respectively), metaSPAdes significantly improved on the length of the 1000 longest scaffolds compared with IDBA-UD (31.6 Mb vs. 19.3 Mb). MEGAHIT resulted in an inferior assembly compared with metaSPAdes and IDBA-UD on this data set.</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"NURKGR213959TB1\"><h3>Table 1.</h3><!--caption a7--><span class=\"caption\"><p id=\"__p25\">The total length of scaffolds (in megabases) for all data sets and all assemblers</p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139774689834768\" class=\"figure\"><img loading=\"lazy\" class=\"fig-image\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 824tb01.jpg\" title=\"An external file that holds a picture, illustration, etc.&#10;Object name is 824tb01.jpg\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5411777/bin/824tb01.jpg\" /></span><span id=\"largeobj_idm139774689834768\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/table/NURKGR213959TB1/?report=objectonly\">Open in a separate window</a></span></span><p id=\"__p26\" class=\"p\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F1/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959F1\" rid-ob=\"ob-NURKGR213959F1\" co-legend-rid=\"lgnd_NURKGR213959F1\"><span>Figure 1</span></a> provides cumulative scaffold length plots illustrating that metaSPAdes improves the contiguity of assemblies over all other assemblers for the HMP, MARINE, and SOIL data sets. A surprising conclusion of our benchmarking is that IDBA-UD (often viewed as a slower predecessor of MEGAHIT) improved on the contiguity of MEGAHIT assemblies on all data sets. &#x0201c;The summary of Nx statistics&#x0201d; in the <a href=\"http://genome.cshlp.org/lookup/suppl/doi:10.1101/gr.213959.116/-/DC1\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Supplemental Material</a> presents Nx plots across all data sets (for details on the Nx statistics, see metaQUAST manual).</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"NURKGR213959F1\" co-legend-rid=\"lgnd_NURKGR213959F1\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F1/\" target=\"figure\" rid-figpopup=\"NURKGR213959F1\" rid-ob=\"ob-NURKGR213959F1\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139774692309712\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5411777_824f01.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 824f01.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5411777/bin/824f01.jpg\" /></a></span><span id=\"largeobj_idm139774692309712\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F1/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_NURKGR213959F1\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F1/\" target=\"figure\" rid-figpopup=\"NURKGR213959F1\" rid-ob=\"ob-NURKGR213959F1\">Figure 1.</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p27\">The cumulative scaffold lengths plots. On the <em>x</em>-axis, scaffolds are ordered from the longest to the shortest. The <em>y</em>-axis shows the total length of x longest scaffolds in the assembly.</p></span></span></span></span><span id=\"s1e2\" class=\"sec\"><h4 id=\"s1e2title\">Gene prediction statistics</h4><p id=\"__p28\" class=\"p p-first\">To further evaluate how fragmented the resulting assemblies are, we used MetaProdigal v2.6.2 (<a href=\"#NURKGR213959C25\" rid=\"NURKGR213959C25\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388654\">Hyatt et al. 2012</a>) to predict the complete genes (option -c) in each assembly. Predicted genes were then passed through CD-HIT v4.6 (<a href=\"#NURKGR213959C37\" rid=\"NURKGR213959C37\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388632\">Li and Godzik 2006</a>; <a href=\"#NURKGR213959C17\" rid=\"NURKGR213959C17\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388570\">Fu et al. 2012</a>) clustering software (with 99% similarity) to correct for potential advantage of more redundant assemblies and to retain only the longest predicted gene in a cluster. <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/table/NURKGR213959TB2/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959TB2\" rid-ob=\"ob-NURKGR213959TB2\" co-legend-rid=\"\"><span>Table 2</span></a> reports the number and the total length of predicted genes &#x0003e;800 bp (length threshold was set to filter less reliable short gene predictions). In the case of the most complex MARINE and SOIL data sets, the number of predicted long genes in metaSPAdes assemblies is significantly larger compared with other assemblers (14% and 66% increase for the MARINE data set and 17% and 49% increase for the SOIL data set compared with IDBA-UD and MEGAHIT, respectively). Although we cannot rule out many false-positive gene predictions, there is no reason to believe that their rate significantly varies across various assemblers.</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"NURKGR213959TB2\"><h3>Table 2.</h3><!--caption a7--><span class=\"caption\"><p id=\"__p29\">Number (in thousands) and total length (in Mb) of predicted genes &#x0003e;800 bp for all data sets and all assemblers</p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139774693818736\" class=\"figure\"><img loading=\"lazy\" class=\"fig-image\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 824tb02.jpg\" title=\"An external file that holds a picture, illustration, etc.&#10;Object name is 824tb02.jpg\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5411777/bin/824tb02.jpg\" /></span><span id=\"largeobj_idm139774693818736\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/table/NURKGR213959TB2/?report=objectonly\">Open in a separate window</a></span></span></span><span id=\"s1e3\" class=\"sec\"><h4 id=\"s1e3title\">Read alignment statistics</h4><p id=\"__p30\" class=\"p p-first\">For each data set and assembler, we further aligned read-pairs to scaffolds (&#x0003e;1 kb) with Bowtie 2 v2.2.4 (<a href=\"#NURKGR213959C35\" rid=\"NURKGR213959C35\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388593\">Langmead and Salzberg 2012</a>). A read-pair is classified as aligned if both reads align to the same scaffold within 1 kb from each other with proper orientation. We further distinguished between uniquely and nonuniquely aligned read-pairs and reported the fractions of aligned single reads and read-pairs for all data sets and assemblers. Better assemblies are characterized by higher fractions of uniquely aligned read-pairs and smaller fractions of nonuniquely aligned read-pairs.</p><p id=\"__p31\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/table/NURKGR213959TB3/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959TB3\" rid-ob=\"ob-NURKGR213959TB3\" co-legend-rid=\"\"><span>Table 3</span></a> illustrates that metaSPAdes resulted in a substantially higher fraction of uniquely aligned read-pairs for the HMP and MARINE data sets compared with all other assemblers. It also shows that only 15%, 11%, 13%, and 2% of read-pairs in the SOIL data set align to assemblies generated by metaSPAdes, MEGAHIT, IDBA-UD, and Ray-Meta, respectively, confirming high diversity of this community. The small fraction of the uniquely aligned read-pairs for this complex data set suggests that the vast majority of genomes in this metagenome have low depth of read coverage, thus preventing their assemblies. <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/table/NURKGR213959TB3/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959TB3\" rid-ob=\"ob-NURKGR213959TB3\" co-legend-rid=\"\"><span>Table 3</span></a> also provides the fraction of nonuniquely aligned read-pairs (that align to multiple regions in an assembly), thus revealing the potential redundancies (duplicated fragments) in the assemblies. metaSPAdes assemblies have lower rates of nonunique paired-read alignments, indicating that they are less redundant. For example, in the case of the HMP data set, metaSPAdes had &#x0003c;9% of nonuniquely aligned read-pairs compared with 14%, 19%, and 38% for IDBA-UD, MEGAHIT, and Ray-Meta, respectively.</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"NURKGR213959TB3\"><h3>Table 3.</h3><!--caption a7--><span class=\"caption\"><p id=\"__p32\">Fraction of aligned single and paired reads (both unique and nonunique) for all data sets and all assemblers (in percentages)</p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139774686252032\" class=\"figure\"><img loading=\"lazy\" class=\"fig-image\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 824tb03.jpg\" title=\"An external file that holds a picture, illustration, etc.&#10;Object name is 824tb03.jpg\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5411777/bin/824tb03.jpg\" /></span><span id=\"largeobj_idm139774686252032\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/table/NURKGR213959TB3/?report=objectonly\">Open in a separate window</a></span></span><p id=\"__p33\" class=\"p p-last\">Below we discuss benchmarking results for each data set in more details. metaQUAST reports the NGA50 statistics (NG50 statistics corrected for assembly errors) to evaluate the quality of assembly of individual genomes within a metagenome. To compute NGA50, the contigs are first broken into smaller segments at the identified misassembly breakpoints. NGA50 for a given reference genome is the maximal value such that the broken segments (that align to this reference) of at least that length cover at least half of the bases of the reference.</p></span><span id=\"s1e4\" class=\"sec\"><h4 id=\"s1e4title\">SYNTH data set</h4><p id=\"__p34\" class=\"p p-first\">Details about the references comprising the SYNTH data set are given in <a href=\"http://genome.cshlp.org/lookup/suppl/doi:10.1101/gr.213959.116/-/DC1\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Supplemental Table S2</a>. Assembly statistics for all assemblers and references is summarized in <a href=\"http://genome.cshlp.org/lookup/suppl/doi:10.1101/gr.213959.116/-/DC1\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Supplemental Table S3</a>. &#x0201c;Analysis of the SYNTH Dataset&#x0201d; in the <a href=\"http://genome.cshlp.org/lookup/suppl/doi:10.1101/gr.213959.116/-/DC1\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Supplemental Material</a> discusses significant differences in the performance of various assemblers even for this rather simple data set. Assembly statistics for 20 most abundant references is summarized on <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F2/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959F2\" rid-ob=\"ob-NURKGR213959F2\" co-legend-rid=\"lgnd_NURKGR213959F2\"><span>Figure 2</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"NURKGR213959F2\" co-legend-rid=\"lgnd_NURKGR213959F2\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F2/\" target=\"figure\" rid-figpopup=\"NURKGR213959F2\" rid-ob=\"ob-NURKGR213959F2\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139774693436576\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5411777_824f02.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 824f02.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5411777/bin/824f02.jpg\" /></a></span><span id=\"largeobj_idm139774693436576\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F2/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_NURKGR213959F2\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F2/\" target=\"figure\" rid-figpopup=\"NURKGR213959F2\" rid-ob=\"ob-NURKGR213959F2\">Figure 2.</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p35\">metaQUAST statistics for 20 most abundant species comprising the SYNTH data set. The NGA50 statistics (<em>top left</em>), the fraction of the reconstructed genome compared with the total genome length (<em>top right</em>), the number of intragenomic misassemblies (<em>bottom left</em>), and the number of intergenomic misassemblies (<em>bottom right</em>) for 20 most abundant species comprising the SYNTH data set. References are denoted by their RefSeq IDs (see <a href=\"http://genome.cshlp.org/lookup/suppl/doi:10.1101/gr.213959.116/-/DC1\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Supplemental Table S2</a>) and arranged in the decreasing order of the coverage depths.</p></span></span></span></span><span id=\"s1e5\" class=\"sec\"><h4 id=\"s1e5title\">HMP data set</h4><p id=\"__p36\" class=\"p p-first-last\">Since the genomes comprising bacterial communities are typically unknown, the HMP consortium identified a number of reference genomes (listed at HMP Shotgun Community profiling SRS077736) similar to the genomes present in the HMP data set (<a href=\"#NURKGR213959C61\" rid=\"NURKGR213959C61\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388657\">Treangen et al. 2013</a>). However, our attempt to use this resource for reliable quality assessment faced difficulties: Only three genomes in this list (<em>Streptococcus salivarius <em>SK126</em>, Neisseria subflava</em> NJ9703, and <em>Prevotella melaninogenica</em> ATCC 25845) were at least 70% covered by contigs generated by assemblers included in this study. Moreover, we revealed substantial differences between these references and the genomes in the sample, making metaQUAST analysis unreliable (see &#x0201c;Analysis of the HMP Dataset&#x0201d; in the <a href=\"http://genome.cshlp.org/lookup/suppl/doi:10.1101/gr.213959.116/-/DC1\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Supplemental Material</a>).</p></span><span id=\"s1e6\" class=\"sec\"><h4 id=\"s1e6title\">MARINE data set</h4><p id=\"__p37\" class=\"p p-first-last\">Based on the fraction of aligned reads (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/table/NURKGR213959TB3/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959TB3\" rid-ob=\"ob-NURKGR213959TB3\" co-legend-rid=\"\"><span>Table 3</span></a>), we conclude that MARINE data set represents a more diverse community than the HMP data set but is less diverse that the SOIL data set. <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/table/NURKGR213959TB1/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959TB1\" rid-ob=\"ob-NURKGR213959TB1\" co-legend-rid=\"\"><span>Tables 1</span></a> through <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/table/NURKGR213959TB3/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959TB3\" rid-ob=\"ob-NURKGR213959TB3\" co-legend-rid=\"\"><span style=\"position: relative;text-decoration:none;\">&#x200B;<span class=\"figpopup-sensitive-area\" style=\"left: -3.5em;\">through3</span></span><span>3</span></a> illustrate that metaSPAdes results in more contiguous and complete assembly of the MARINE data set than all other assemblers.</p></span><span id=\"s1e7\" class=\"sec\"><h4 id=\"s1e7title\">SOIL data set</h4><p id=\"__p38\" class=\"p p-first\">We compared assemblies of the SOIL data set against the set of contigs obtained by <a href=\"#NURKGR213959C5\" rid=\"NURKGR213959C5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388616\">Bankevich and Pevzner (2016)</a> from TSLRs for all three samples described by <a href=\"#NURKGR213959C60\" rid=\"NURKGR213959C60\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388642\">Sharon et al. (2015)</a> (which improves on the TSLR assemblies from the original study). Contigs &#x0003e;20 kb (total length 103 Mb) were selected as a &#x0201c;reference genome&#x0201d; for computing metaQUAST statistics (with additional &#x0201c;--fragmented&#x0201d; option). Results are summarized in <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/table/NURKGR213959TB4/\" target=\"table\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959TB4\" rid-ob=\"ob-NURKGR213959TB4\" co-legend-rid=\"\"><span>Table 4</span></a>. Only 27.6 Mb (&#x02248;13.6%) of the total length of the metaSPAdes scaffolds &#x0003e;1 kb (196 Mb) overlapped with TSLR contigs, covering just &#x02248;26% of the total length of the TSLR assembly (best result across all assemblers).</p><!--table ft1--><!--table-wrap mode=\"anchored\" t5--><span class=\"table-wrap anchored whole_rhythm\" id=\"NURKGR213959TB4\"><h3>Table 4.</h3><!--caption a7--><span class=\"caption\"><p id=\"__p39\">Comparison of long scaffolds (&#x0003e;1 kb) generated by various metagenomic assemblers for the SOIL data set against TSLR contigs generated by <a href=\"#NURKGR213959C5\" rid=\"NURKGR213959C5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388605\">Bankevich and Pevzner (2016)</a></p></span><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139774687718992\" class=\"figure\"><img loading=\"lazy\" class=\"fig-image\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 824tb04.jpg\" title=\"An external file that holds a picture, illustration, etc.&#10;Object name is 824tb04.jpg\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5411777/bin/824tb04.jpg\" /></span><span id=\"largeobj_idm139774687718992\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/table/NURKGR213959TB4/?report=objectonly\">Open in a separate window</a></span></span></span><span id=\"s1e8\" class=\"sec sec-last\"><h4 id=\"s1e8title\">Additional benchmarks</h4><p id=\"__p40\" class=\"p p-first-last\">In addition to the benchmarking results presented above, the <a href=\"http://genome.cshlp.org/lookup/suppl/doi:10.1101/gr.213959.116/-/DC1\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Supplemental Materials</a> also include benchmarking of all assemblers on two CAMI data sets (&#x0201c;CAMI Datasets&#x0201d; and &#x0201c;Analysis of the CAMI Datasets&#x0201d;), comparison of metaSPAdes and SPAdes assemblies on the SYNTH data set (&#x0201c;Benchmarking SPAdes against metaSPAdes&#x0201d;), and discussion of how the novel algorithms proposed in this work affect the quality of assemblies (&#x0201c;Effect of Novel Algorithmic Approaches in metaSPAdes on Assembly Quality&#x0201d;).</p></span></span></span><span id=\"s2\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s2title\">Discussion</h2><p id=\"__p41\" class=\"p p-first\">metaSPAdes addresses a number of challenges in metagenomic assembly and implemented several novel features, such as efficient assembly graph processing to address the microdiversity challenge, a new repeat resolution approach that utilizes rare strain variants to improve the consensus assembly of strain mixtures, and fast algorithms for constructing assembly graphs and error-correcting reads.</p><p id=\"__p42\">These features contributed to improvements in metaSPAdes assemblies (compared with the state-of-the-art assemblers MEGAHIT, IDBA-UD, and Ray-Meta) and enabled us to scale metaSPAdes for analyzing large metagenomes.</p><p id=\"__p43\" class=\"p p-last\">In addition to the intrinsic biological challenges discussed in this article, the field of metagenomic assembly also faces technological challenges caused by innovations in sequencing and library preparation techniques. For example, recently introduced high-quality jumping (mate-pair) libraries (such as Nextera Mate Pair Libraries) have a potential to significantly improve assembly quality (<a href=\"#NURKGR213959C65\" rid=\"NURKGR213959C65\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388590\">Vasilinetc et al. 2015</a>). However, metagenomic assemblers have not caught up with this technological innovation yet. Another example is the TSLR technology (<a href=\"#NURKGR213959C32\" rid=\"NURKGR213959C32\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388562\">Kuleshov et al. 2014</a>; <a href=\"#NURKGR213959C41\" rid=\"NURKGR213959C41\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388604\">McCoy et al. 2014</a>), whose first metagenomic applications highlighted the need for developing methods to reliably combine it with the paired-end libraries (<a href=\"#NURKGR213959C33\" rid=\"NURKGR213959C33\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388655\">Kuleshov et al. 2015</a>; <a href=\"#NURKGR213959C60\" rid=\"NURKGR213959C60\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388574\">Sharon et al. 2015</a>; <a href=\"#NURKGR213959C5\" rid=\"NURKGR213959C5\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388558\">Bankevich and Pevzner 2016</a>). metaSPAdes now faces the challenge of incorporating these emerging technologies into its assembly pipeline.</p></span><span id=\"s3\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"s3title\">Methods</h2><span id=\"s3a\" class=\"sec sec-first\"><h3 id=\"s3atitle\">Detecting and masking strain variation</h3><p id=\"__p44\" class=\"p p-first\">Genomic differences between related strains often result in &#x0201c;bulges&#x0201d; and &#x0201c;tips&#x0201d; in the de Bruijn graphs that are not unlike artifacts caused by sequencing errors in genome assembly (<a href=\"#NURKGR213959C53\" rid=\"NURKGR213959C53\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388636\">Pevzner et al. 2004</a>; <a href=\"#NURKGR213959C72\" rid=\"NURKGR213959C72\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388592\">Zerbino and Birney 2008</a>). For example, a sequencing error often results in a bulge formed by two short alternative paths between the same vertices in the de Bruijn graph, a &#x0201c;correct&#x0201d; path with high coverage and an &#x0201c;erroneous&#x0201d; path with low coverage. Similarly, a substitution or a small indel in a rare strain (compared with an abundant strain) often results in a bulge formed by a high-coverage path corresponding to the abundant strain and an alternative low-coverage path corresponding to the rare strain.</p><p id=\"__p45\" class=\"p p-last\">Aiming at the consensus assembly of a strain mixture, metaSPAdes masks the majority of strain differences using a modification of the SPAdes procedures for masking sequencing errors (the algorithms for removal of tips, &#x0201c;simple&#x0201d; bulges [<a href=\"#NURKGR213959C6\" rid=\"NURKGR213959C6\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388670\">Bankevich et al. 2012</a>], and &#x0201c;complex&#x0201d; bulges [<a href=\"#NURKGR213959C50\" rid=\"NURKGR213959C50\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388609\">Nurk et al. 2013</a>]). metaSPAdes uses more aggressive settings than the ones used in assemblies of isolates; for example, it collapses larger bulges and removes longer tips than SPAdes. We note that the bulge projection approach in SPAdes improves on the originally proposed bulge removal approach (<a href=\"#NURKGR213959C53\" rid=\"NURKGR213959C53\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388584\">Pevzner et al. 2004</a>; <a href=\"#NURKGR213959C72\" rid=\"NURKGR213959C72\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388566\">Zerbino and Birney 2008</a>) used in most existing assemblers since it stores valuable information about the processed bulges (see &#x0201c;Bulge Projection Approach&#x0201d; in the <a href=\"http://genome.cshlp.org/lookup/suppl/doi:10.1101/gr.213959.116/-/DC1\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Supplemental Material</a>). This feature is important for the repeat resolution approach in metaSPAdes described below.</p></span><span id=\"s3b\" class=\"sec\"><h3 id=\"s3btitle\">Analyzing filigree edges in the assembly graph</h3><p id=\"__p46\" class=\"p p-first\">In addition to single-nucleotide variants and small indels, strain variation is often manifested as highly diverged regions, insertions of mobile elements, rearrangements, large deletions, parallel gene transfer, etc. The green edges in the assembly graph shown in <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959F3\" rid-ob=\"ob-NURKGR213959F3\" co-legend-rid=\"lgnd_NURKGR213959F3\"><span>Figure 3</span></a> result from an additional copy of a mobile element in a rare <em>strain</em><sub><em>2</em></sub> (compared with the abundant <em>strain</em><sub><em>1</em></sub>), while the blue edge corresponds to a horizontally transferred gene (or a highly diverged genomic region) in a rare <em>strain</em><sub><em>3</em></sub> (compared to the abundant <em>strain</em><sub><em>1</em></sub>). Such edges fragment contigs corresponding to the abundant <em>strain</em><sub><em>1</em></sub>; for example, the green edges in <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F3/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959F3\" rid-ob=\"ob-NURKGR213959F3\" co-legend-rid=\"lgnd_NURKGR213959F3\"><span>Figure 3</span></a> (bottom right) break the edge <em>c</em> into three shorter edges. We note that the edges in the assembly graph are <em>condensed</em>; that is, they represent nonbranching paths formed by <em>k</em>-mers.</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"NURKGR213959F3\" co-legend-rid=\"lgnd_NURKGR213959F3\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F3/\" target=\"figure\" rid-figpopup=\"NURKGR213959F3\" rid-ob=\"ob-NURKGR213959F3\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139774766164624\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5411777_824f03.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 824f03.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5411777/bin/824f03.jpg\" /></a></span><span id=\"largeobj_idm139774766164624\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F3/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_NURKGR213959F3\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F3/\" target=\"figure\" rid-figpopup=\"NURKGR213959F3\" rid-ob=\"ob-NURKGR213959F3\">Figure 3.</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p47\">The de Bruijn graphs of three strains and their strain mixture. The figure shows only a small subgraph of the de Bruijn graph. The abundant strain (<em>strain</em><sub><em>1</em></sub>) is shown by thick lines, and the rare strains (<em>strain</em><sub><em>2</em></sub> and <em>strain</em><sub><em>3</em></sub>) are shown by thin lines. The genomic repeat R is shown in red. (<em>Top left</em>) The de Bruijn graph of the abundant <em>strain</em><sub><em>1</em></sub>. (<em>Top right</em>) The rare <em>strain</em><sub><em>2</em></sub> differs from the abundant <em>strain</em><sub><em>1</em></sub> by an insertion of an additional copy or repeat R. The two breakpoint edges resulting from this insertion are shown in green. These filigree edges are not removed by the graph simplification procedures in the standard assembly tools aimed at isolates. (<em>Bottom left</em>) The rare <em>strain</em><sub><em>3</em></sub> differs from the abundant <em>strain</em><sub><em>1</em></sub> by an insertion of a horizontally transferred gene (or a highly diverged genomic region). (<em>Bottom right</em>) The de Bruijn graph of the mixture of three strains.</p></span></span></span><p id=\"__p48\">We refer to edges originating from rare strain variants within the assembly graph of a strain mixture as <em>filigree edges</em>. Traditional genome assemblers use a global threshold on read coverage to remove the low-coverage edges (that typically result from sequencing errors) from the assembly graph during the graph simplification step. However, this approach does not work well for metagenomic assemblies, since there is no global threshold that (1) removes edges corresponding to rare strains and (2) preserves edges corresponding to rare species. Similarly to IDBA-UD and MEGAHIT, metaSPAdes analyzes the coverage ratios between adjacent edges in the assembly graph, classifying edges with low-coverage ratios as potential filigree edges.</p><p id=\"__p49\">We denote the coverage of an edge <em>e</em> in the assembly graph as <em>cov(e)</em> and define the coverage <em>cov(v)</em> of a vertex <em>v</em> as the maximum of <em>cov(e)</em> over all edges <em>e</em> incident to <em>v.</em> Given an edge <em>e</em> incident to a vertex <em>v</em> and a threshold <em>ratio</em> (the default value is 10), a vertex <em>v predominates</em> an edge <em>e</em> if its coverage is significantly higher than the coverage of the edge <em>e</em>; that is, if <em>ratio</em> &#x000b7; <em>cov(e)</em> &#x0003c; <em>cov(v).</em> An edge (<em>v,w</em>) is <em>weak</em> if it is predominated by either <em>v</em> or <em>w</em>. Note that filigree edges are often classified as weak since their coverage is much lower than the coverage of adjacent edges resulting from abundant strains.</p><p id=\"__p50\" class=\"p p-last\">metaSPAdes <em>disconnects</em> all weak edges from their predominating vertices in the assembly graph. Disconnection of a weak edge (<em>v,w</em>) in the assembly graph from its starting vertex <em>v</em> (ending vertex <em>w</em>) is simply a removal of its first (last) <em>k</em>-mer rather than removal of the entire condensed edge. We emphasize that, in contrast to IDBA-UD and MEGAHIT, we disconnect rather than remove weak edges in the assembly graph since our goal is to preserve the information about rare strains whenever possible, that is, when it does not lead to a deterioration of the consensus backbone.</p></span><span id=\"s3c\" class=\"sec\"><h3 id=\"s3ctitle\">Repeat resolution with exSPAnder</h3><p id=\"__p51\" class=\"p p-first\">exSPAnder (<a href=\"#NURKGR213959C54\" rid=\"NURKGR213959C54\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388608\">Prjibelski et al. 2014</a>; <a href=\"#NURKGR213959C65\" rid=\"NURKGR213959C65\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388643\">Vasilinetc et al. 2015</a>; <a href=\"#NURKGR213959C1\" rid=\"NURKGR213959C1\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388603\">Antipov et al. 2016</a>) is a module of SPAdes that combines various sources of information (e.g., paired reads or long error-prone reads) for resolving repeats and scaffolding in the assembly graph. Starting from a path consisting of a single condensed edge in the assembly graph, exSPAnder iteratively attempts to extend it to a longer path that represents a contiguous segment of the genome (<em>genomic path</em>). To extend a path, exSPAnder selects one of its <em>extension edges</em> (edges that start at the terminal vertex of this path). Choice of the extension edge is controlled by the <em>decision rule</em> that evaluates whether a particular extension edge is sufficiently supported by the data, while other extension edges are not (given the existing path). exSPAnder further removes overlaps between generated genomic paths (<em>overlap reduction</em> step) and outputs the strings spelled by the resulting paths as a set of contigs.</p><p id=\"__p52\" class=\"p p-last\">metaSPAdes modifies the decision rule of exSPAnder to account for the local read coverage, denoted <em>localCov</em>, of the specific genomic region that is being reconstructed during the path extension process. For details, see &#x0201c;Modifying the Decision Rule in exSPAnder for Metagenomic Data&#x0201d; in the <a href=\"http://genome.cshlp.org/lookup/suppl/doi:10.1101/gr.213959.116/-/DC1\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Supplemental Material</a>. The value <em>localCov</em> is estimated as the minimum across the average coverages of the edges in the path that is being extended. Taking minimum (rather than the average) coverage excludes the repetitive edges in the path from consideration and typically underestimates the real coverage of the region, making the decision rule more conservative.</p></span><span id=\"s3d\" class=\"sec\"><h3 id=\"s3dtitle\">A new metagenomic decision rule in metaSPAdes</h3><p id=\"__p53\" class=\"p p-first\">Some intergenomic repeats between species of different abundances can be resolved based on the differences in the depth of read coverage (<a href=\"#NURKGR213959C21\" rid=\"NURKGR213959C21\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388672\">Haider et al. 2014</a>; <a href=\"#NURKGR213959C48\" rid=\"NURKGR213959C48\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388653\">Namiki et al. 2012</a>). metaSPAdes introduces an additional metagenomics-specific decision rule that filters out unlikely path extensions using the coverage estimate of the region that is being reconstructed (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959F4\" rid-ob=\"ob-NURKGR213959F4\" co-legend-rid=\"lgnd_NURKGR213959F4\"><span>Fig. 4</span></a>). It often allows metaSPAdes to pass through long inter-species repeats during reconstruction of abundant species. metaSPAdes applies a new decision rule described below only if the paired reads failed to provide sufficient evidence to discriminate between extension edges.</p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"NURKGR213959F4\" co-legend-rid=\"lgnd_NURKGR213959F4\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F4/\" target=\"figure\" rid-figpopup=\"NURKGR213959F4\" rid-ob=\"ob-NURKGR213959F4\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139774686324992\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5411777_824f04.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 824f04.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5411777/bin/824f04.jpg\" /></a></span><span id=\"largeobj_idm139774686324992\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F4/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_NURKGR213959F4\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F4/\" target=\"figure\" rid-figpopup=\"NURKGR213959F4\" rid-ob=\"ob-NURKGR213959F4\">Figure 4.</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p54\">Applying the metagenomics-specific decision rule for repeat resolution. The figure shows only a small subgraph of the assembly graph. (<em>A</em>) The path that is currently being extended (formed by green edges) along with its blue extension edges <em>e and e</em>&#x02032;. (<em>B</em>) The short-edge traversal from the end of the extension edge <em>e</em>. The dotted curve shows the boundary <em>frontier(e)</em> of the traversal. The edges in the set <em>next(e)</em> are shown in red with low-coverage edges represented as dashed arrows (other edges in <em>next(e)</em> are represented as solid arrows). Since all edges in <em>next(e)</em> have low coverage, the edge <em>e</em> is ruled out as an unlikely extension candidate. (<em>C</em>) The short-edge traversal from the end of the extension edge <em>e</em>&#x02032;. (<em>D</em>) Since <em>e</em>&#x02032; is a single extension edge that was not ruled out (there is a solid edge in <em>next(e&#x02032;)</em>), it is added to the growing path and the extension process continues.</p></span></span></span><p id=\"__p55\">An edge in the assembly graph is called <em>long</em> if its length exceeds a certain threshold (1500 bp by default) and <em>short</em> otherwise. We say that a long edge <em>e<sub>2</sub> follows</em> a long edge <em>e</em><sub><em>1</em></sub> in a genomic path if all edges between the end of <em>e</em><sub><em>1</em></sub> and the start of <em>e</em><sub><em>2</em></sub> in this path are short.</p><p id=\"__p56\" class=\"p p-last\">While considering an extension edge <em>e</em>, metaSPAdes performs a directed traversal of the graph (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959F4\" rid-ob=\"ob-NURKGR213959F4\" co-legend-rid=\"lgnd_NURKGR213959F4\"><span>Fig. 4</span></a>B), starting from the end of <em>e</em> and walking along the short edges. We define the set of all vertices that are reached by this traversal as <em>frontier(e)</em> and consider the set <em>next(e)</em> of all long edges starting in <em>frontier(e)</em>. This procedure is aimed at finding nonrepetitive long edges that can follow <em>e</em> in the (unknown) genomic path. We classify an edge in the set <em>next(e)</em> as a <em>low-coverage edge</em> if the coverage estimate of the region that is being reconstructed, <em>localCov,</em> exceeds its coverage at least by a factor &#x003b2; (the default value &#x003b2; = 2). If all edges in <em>next(e)</em> are low-coverage edges, then <em>e</em> is considered an unlikely candidate for an extension of the current path. If all but a single edge <em>e</em>&#x02032; represent unlikely extensions, the path is extended by <em>e</em>&#x02032; (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F4/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959F4\" rid-ob=\"ob-NURKGR213959F4\" co-legend-rid=\"lgnd_NURKGR213959F4\"><span>Fig. 4</span></a>C).</p></span><span id=\"s3e\" class=\"sec\"><h3 id=\"s3etitle\">Utilizing strain differences for repeat resolution in metaSPAdes</h3><p id=\"__p57\" class=\"p p-first\"><a href=\"#NURKGR213959C57\" rid=\"NURKGR213959C57\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388641\">Safonova et al. (2015)</a> showed that differences between haplomes can be used to improve the quality of consensus assembly of a highly polymorphic diploid genome. metaSPAdes capitalizes on the similar observation that the differences between strains can be, somewhat counter-intuitively, used to improve the quality of consensus assembly of a strain mixture. In particular, contigs generated prior to masking strain differences in assembly graph and thus representing genomic fragments of individual strains (<em>strain-contigs</em>) often provide additional long-range information for reconstruction of a strain-mixture backbone.</p><p id=\"__p58\">Inspired by dipSPAdes (<a href=\"#NURKGR213959C57\" rid=\"NURKGR213959C57\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388671\">Safonova et al. 2015</a>), metaSPAdes uses the following pipeline that includes two launches of exSPAnder (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959F5\" rid-ob=\"ob-NURKGR213959F5\" co-legend-rid=\"lgnd_NURKGR213959F5\"><span>Fig. 5</span></a>).\n</p><ul class=\"unordered\" style=\"list-style-type:disc\"><!--\nlist-behavior=unordered\nprefix-word=\nmark-type=disc\nmax-label-size=0\n--><li><p id=\"__p59\"><em>Generating strain-contigs.</em> After constructing the assembly graph (that encodes both abundant and rare strains), we launch exSPAnder to generate a set of strain-contigs representing both rare and abundant strains (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959F5\" rid-ob=\"ob-NURKGR213959F5\" co-legend-rid=\"lgnd_NURKGR213959F5\"><span>Fig. 5</span></a>C). Strain-contigs are not subjected to the default overlap reduction step in exSPAnder.</p></li><li><p id=\"__p60\"><em>Transforming assembly graph into consensus assembly graph.</em> metaSPAdes identifies and masks rare strain variants, resulting in the <em>consensus assembly graph</em> (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959F5\" rid-ob=\"ob-NURKGR213959F5\" co-legend-rid=\"lgnd_NURKGR213959F5\"><span>Fig. 5</span></a>D).</p></li><li><p id=\"__p61\"><em>Generating strain-paths in the consensus assembly graph.</em> Capitalizing on the bulge projection approach (see &#x0201c;Bulge Projection Approach&#x0201d; in the <a href=\"http://genome.cshlp.org/lookup/suppl/doi:10.1101/gr.213959.116/-/DC1\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Supplemental Material</a>), metaSPAdes reconstructs paths in the consensus assembly graph corresponding to strain-contigs, referred to as <em>strain-paths</em> (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959F5\" rid-ob=\"ob-NURKGR213959F5\" co-legend-rid=\"lgnd_NURKGR213959F5\"><span>Fig. 5</span></a>E).</p></li><li><p id=\"__p62\"><em>Repeat resolution using strain-paths.</em> This step utilizes the hybrid mode of exSPAnder originally developed to incorporate long error-prone Pacific Biosciences and Oxford Nanopore reads in the repeat resolution process (<a href=\"#NURKGR213959C4\" rid=\"NURKGR213959C4\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388576\">Ashton et al. 2014</a>; <a href=\"#NURKGR213959C34\" rid=\"NURKGR213959C34\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388679\">Labont&#x000e9; et al. 2015</a>; <a href=\"#NURKGR213959C1\" rid=\"NURKGR213959C1\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388573\">Antipov et al. 2016</a>). Instead of working with long error-prone reads, we modified exSPAnder to work with virtual reads spelled by the strain-paths to facilitate resolution of repeats in the consensus assembly graph (<a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959F5\" rid-ob=\"ob-NURKGR213959F5\" co-legend-rid=\"lgnd_NURKGR213959F5\"><span>Fig. 5</span></a>F).</p></li></ul><p></p><!--fig ft0--><!--fig mode=article f1--><span class=\"fig iconblock whole_rhythm\" id=\"NURKGR213959F5\" co-legend-rid=\"lgnd_NURKGR213959F5\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F5/\" target=\"figure\" rid-figpopup=\"NURKGR213959F5\" rid-ob=\"ob-NURKGR213959F5\"><!--fig/graphic|fig/alternatives/graphic mode=\"anchored\" m1--><span data-largeobj=\"\" data-largeobj-link-rid=\"largeobj_idm139774689539536\" class=\"figure\"><a class=\"inline_block ts_canvas\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/core/\r\nlw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5411777_824f05.jpg\" target=\"tileshopwindow\" rel=\"noopener\"><span class=\"ts_bar small\" title=\"Click on image to zoom\"></span><img loading=\"lazy\" alt=\"An external file that holds a picture, illustration, etc.&#10;Object name is 824f05.jpg\" title=\"Click on image to zoom\" class=\"tileshop\" (target=\"_BLANK\" src=\"https://www.ncbi.nlm.nih.gov/pmc/articles)\r\n/PMC5411777/bin/824f05.jpg\" /></a></span><span id=\"largeobj_idm139774689539536\" class=\"largeobj-link align_right\" style=\"display: none\"><a target=\"object\" rel=\"noopener\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F5/?report=objectonly\">Open in a separate window</a></span></a><span class=\"icnblk_cntnt\" id=\"lgnd_NURKGR213959F5\"><span><a class=\"figpopup\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F5/\" target=\"figure\" rid-figpopup=\"NURKGR213959F5\" rid-ob=\"ob-NURKGR213959F5\">Figure 5.</a></span><!--caption a7--><span class=\"caption\"><p id=\"__p63\">Repeat resolution in metagenomic assembly. (<em>A</em>) One of two identical copies of a long (longer than the insert size) repeat R (red) in the abundant strain has mutated into a unique genomic &#x0201c;green&#x0201d; region R&#x02032; in the rare strain. (<em>B</em>) The assembly graph resulting from a mixture of reads from the abundant and rare strains. Two alternative paths between the start and the end of the green edge (one formed by a single green edge and another formed by two black and one red edge) form a bulge. (<em>C</em>) The strain-contig spanning R&#x02032; (shown by green dashed line) constructed by exSPAnder at the &#x0201c;generating strain-contigs&#x0201d; step. (<em>D</em>) Masking of the strain variation at the &#x0201c;transforming assembly graph into consensus assembly graph&#x0201d; step leads to a projection of a bulge (formed by red and green edges) and results in the consensus assembly graph shown in <em>E</em>. The blue arrows emphasize that SPAdes <em>projects</em> rather than <em>deletes</em> bulges, facilitating the subsequent reconstruction of strain-paths in the consensus assembly graph. (<em>E</em>) Reconstruction of the strain-path (green dotted line), corresponding to a strain-contig (green dashed line) at the &#x0201c;generating strain-paths in the consensus assembly graph&#x0201d; step. (<em>F</em>) At the &#x0201c;repeat resolution using strain-paths&#x0201d; step, metaSPAdes utilizes both strain-paths and paired reads to resolve repeats in the consensus graph. The green dotted strain-path from <em>E</em> is used as additional information to reconstruct the consensus contig <em>cRd</em> spanning the long repeat.</p></span></span></span><p id=\"__p64\" class=\"p p-last\">Note that in the example in <a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/figure/NURKGR213959F5/\" target=\"figure\" class=\"fig-table-link figpopup\" rid-figpopup=\"NURKGR213959F5\" rid-ob=\"ob-NURKGR213959F5\" co-legend-rid=\"lgnd_NURKGR213959F5\"><span>Figure 5</span></a>, the long red repeat with multiplicity 2 in the abundant strain is resolved because of the variants (diverged green copy of the repeat) in the rare strain.</p></span><span id=\"s3f\" class=\"sec\"><h3 id=\"s3ftitle\">Scaling metaSPAdes</h3><p id=\"__p65\" class=\"p p-first-last\">Since some metagenomic data sets contain billions of reads, metagenomic assemblers have to be optimized with respect to both speed and memory footprint (<a href=\"#NURKGR213959C47\" rid=\"NURKGR213959C47\" class=\" bibr popnode tag_hotlink tag_tooltip\" id=\"__tag_606388668\">Nagarajan and Pop 2013</a>). &#x0201c;Reducing Running Time and Memory Footprint of metaSPAdes&#x0201d; in the <a href=\"http://genome.cshlp.org/lookup/suppl/doi:10.1101/gr.213959.116/-/DC1\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Supplemental Material</a> describes efforts to scale metaSPAdes for assembling large metagenomic data sets.</p></span><span id=\"s3g\" class=\"sec sec-last\"><h3 id=\"s3gtitle\">Software availability</h3><p id=\"__p66\" class=\"p p-first-last\">The latest version of the SPAdes toolkit that includes metaSPAdes is available from <a href=\"http://cab.spbu.ru/software/spades\">http://cab.spbu.ru/software/spades</a>. The source code for the v3.10 prerelease package used for benchmarking in this article is available as a <a href=\"http://genome.cshlp.org/lookup/suppl/doi:10.1101/gr.213959.116/-/DC1\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">Supplemental Material</a> or alternatively can be accessed as git revision 3d6df0c62ca31a187cb7c2209c892e6e5711229e at <a href=\"https://github.com/ablab/spades\">https://github.com/ablab/spades</a>. The QUAST v4.5 package is available from <a href=\"https://github.com/ablab/quast/tree/release_4.5\">https://github.com/ablab/quast/tree/release_4.5</a>.</p></span></span><span id=\"s\r\n1\" class=\"tsec sec\"><a id=\"supplementary-material-sec\"></a><h2 class=\"head no_bottom_margin\" id=\"s\r\n1title\">Supplementary Material</h2><!--/article/body/sec/--><span class=\"sec suppmat\" id=\"PMC_1\">\n\n<!--caption a9--><strong>Supplemental Material:</strong>\n\n<span class=\"sup-box half_rhythm\" id=\"idm139774691562064\"><a target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5411777/bin/supp_27_5_824__index.html\" data-ga-action=\"click_feat_suppl\">Click here to view.</a></span>\n\n\n\n\n\n\n</span></span><span id=\"idm139774694343424\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139774694343424title\">Acknowledgments</h2><span class=\"sec\"><p id=\"__p67\">We thank Chris Dupont, Rob Knight, Mihai Pop, and Bahar Behsaz for useful comments. We also thank Alla Lapidus, who brought our attention to the field of metagenomics. This work was supported by the Russian Science Foundation (grant 14-50-00069).</p></span></span><span id=\"idm139774687873024\" class=\"tsec sec\"><h2 class=\"head no_bottom_margin\" id=\"idm139774687873024title\">Footnotes</h2><!--back/fn-group--><span class=\"fm-sec half_rhythm small\"><p class=\"fn sec\" id=\"idm139774687872896\"><p id=\"__p68\" class=\"p p-first-last\">[Supplemental material is available for this article.]</p></p><p class=\"fn sec\" id=\"idm139774687871936\"><p id=\"__p69\" class=\"p p-first-last\">Article published online before print. Article, supplemental material, and publication date are at <a href=\"http://www.genome.org/cgi/doi/10.1101/gr.213959.116\" data-ga-action=\"click_feat_suppl\" ref=\"reftype=extlink&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI\" target=\"_blank\">http://www.genome.org/cgi/doi/10.1101/gr.213959.116</a>.</p></p></span></span><br><div class=\"ref-list-sec sec\" id=\"reference-list\"><ul class=\"first-line-outdent\"><li id=\"NURKGR213959C1\"><span class=\"mixed-citation\">Antipov D, Korobeynikov A, McLean JS, Pevzner PA. 2016. \n<span class=\"ref-title\">hybridSPAdes: an algorithm for hybrid assembly of short and long reads</span>. <span class=\"ref-journal\">Bioinformatics</span>\n<span class=\"ref-vol\">32</span>: 1009&#x02013;1015. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4907386/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26589280\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=hybridSPAdes:+an+algorithm+for+hybrid+assembly+of+short+and+long+reads&amp;volume=32&amp;publication_year=2016&amp;pages=1009-1015&amp;pmid=26589280&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C2\"><span class=\"mixed-citation\">Aparicio S, Chapman J, Stupka E, Putnam N, Chia J-M, Dehal P, Christoffels A, Rash S, Hoon S, Smit A, et al.\n2002. \n<span class=\"ref-title\">Whole-genome shotgun assembly and analysis of the genome of <em>Fugu rubripes</em></span>. <span class=\"ref-journal\">Science</span>\n<span class=\"ref-vol\">297</span>: 1301&#x02013;1310. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n12142439\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Whole-genome+shotgun+assembly+and+analysis+of+the+genome+of+Fugu+rubripes&amp;volume=297&amp;publication_year=2002&amp;pages=1301-1310&amp;pmid=12142439&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C3\"><span class=\"mixed-citation\">Arumugam M, Raes J, Pelletier E, Le Paslier D, Yamada T, Mende DR, Fernandes GR, Tap J, Bruls T, Batto J, et al.\n2011. \n<span class=\"ref-title\">Enterotypes of the human gut microbiome</span>. <span class=\"ref-journal\">Nature</span>\n<span class=\"ref-vol\">473</span>: 1&#x02013;7. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3728647/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21508958\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Enterotypes+of+the+human+gut+microbiome&amp;volume=473&amp;publication_year=2011&amp;pages=1-7&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C4\"><span class=\"mixed-citation\">Ashton PM, Nair S, Dallman T, Rubino S, Rabsch W, Mwaigwisya S, Wain J, O\'Grady J. 2014. \n<span class=\"ref-title\">MinION nanopore sequencing identifies the position and structure of a bacterial antibiotic resistance island</span>. <span class=\"ref-journal\">Nat Biotechnol</span>\n<span class=\"ref-vol\">33</span>: 296&#x02013;300. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25485618\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat+Biotechnol&amp;title=MinION+nanopore+sequencing+identifies+the+position+and+structure+of+a+bacterial+antibiotic+resistance+island&amp;volume=33&amp;publication_year=2014&amp;pages=296-300&amp;pmid=25485618&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C5\"><span class=\"mixed-citation\">Bankevich A, Pevzner PA. 2016. \n<span class=\"ref-title\">TruSPAdes: barcode assembly of TruSeq synthetic long reads</span>. <span class=\"ref-journal\">Nat Methods</span>\n<span class=\"ref-vol\">13</span>: 248&#x02013;250. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26828418\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=TruSPAdes:+barcode+assembly+of+TruSeq+synthetic+long+reads&amp;volume=13&amp;publication_year=2016&amp;pages=248-250&amp;pmid=26828418&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C6\"><span class=\"mixed-citation\">Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, et al.\n2012. \n<span class=\"ref-title\">SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing</span>. <span class=\"ref-journal\">J Comput Biol</span>\n<span class=\"ref-vol\">19</span>: 455&#x02013;477. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3342519/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22506599\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Comput+Biol&amp;title=SPAdes:+a+new+genome+assembly+algorithm+and+its+applications+to+single-cell+sequencing&amp;volume=19&amp;publication_year=2012&amp;pages=455-477&amp;pmid=22506599&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C7\"><span class=\"mixed-citation\">Bertin MJ, Schwartz SL, Lee J, Korobeynikov A, Dorrestein PC, Gerwick L, Gerwick WH. 2015. \n<span class=\"ref-title\">Spongosine production by a <em>Vibrio harveyi</em> strain associated with the sponge Tectitethya crypta</span>. <span class=\"ref-journal\">J Nat Prod</span>\n<span class=\"ref-vol\">78</span>: 493&#x02013;499. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5065018/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25668560\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Nat+Prod&amp;title=Spongosine+production+by+a+Vibrio+harveyi+strain+associated+with+the+sponge+Tectitethya+crypta&amp;volume=78&amp;publication_year=2015&amp;pages=493-499&amp;pmid=25668560&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C8\"><span class=\"mixed-citation\">Biller SJ, Berube PM, Lindell D, Chisholm SW. 2015. \n<span class=\"ref-title\"><em>Prochlorococcus</em>: the structure and function of collective diversity</span>. <span class=\"ref-journal\">Nat Rev Microbiol</span>\n<span class=\"ref-vol\">13</span>: 13&#x02013;27. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25435307\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Microbiol&amp;title=Prochlorococcus:+the+structure+and+function+of+collective+diversity&amp;volume=13&amp;publication_year=2015&amp;pages=13-27&amp;pmid=25435307&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C9\"><span class=\"mixed-citation\">Boisvert S, Raymond F, Godzaridis &#x000c9;, Laviolette F, Corbeil J. 2012. \n<span class=\"ref-title\">Ray Meta: scalable de novo metagenome assembly and profiling</span>. <span class=\"ref-journal\">Genome Biol</span>\n<span class=\"ref-vol\">13</span>: R122. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4056372/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23259615\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Ray+Meta:+scalable+de+novo+metagenome+assembly+and+profiling&amp;volume=13&amp;publication_year=2012&amp;pages=R122&amp;pmid=23259615&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C10\"><span class=\"mixed-citation\">Castelle CJ, Hug LA, Wrighton KC, Thomas BC, Williams KH, Wu D, Tringe SG, Singer SW, Eisen JA, Banfield JF. 2013. \n<span class=\"ref-title\">Extraordinary phylogenetic diversity and metabolic versatility in aquifer sediment</span>. <span class=\"ref-journal\">Nat Commun</span>\n<span class=\"ref-vol\">4</span>: 2120. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3903129/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23979677\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=Extraordinary+phylogenetic+diversity+and+metabolic+versatility+in+aquifer+sediment&amp;volume=4&amp;publication_year=2013&amp;pages=2120&amp;pmid=23979677&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C11\"><span class=\"mixed-citation\">Coates RC, Podell S, Korobeynikov A, Lapidus A, Pevzner P, Sherman DH, Allen EE, Gerwick L, Gerwick WH. 2014. \n<span class=\"ref-title\">Characterization of cyanobacterial hydrocarbon composition and distribution of biosynthetic pathways</span>. <span class=\"ref-journal\">PLoS One</span>\n<span class=\"ref-vol\">9</span>: e851. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3903477/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24475038\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Characterization+of+cyanobacterial+hydrocarbon+composition+and+distribution+of+biosynthetic+pathways&amp;volume=9&amp;publication_year=2014&amp;pages=e851&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C12\"><span class=\"mixed-citation\">Cotten M, Oude Munnink B, Canuti M, Deijs M, Watson SJ, Kellam P, van der Hoek L. 2014. \n<span class=\"ref-title\">Full genome virus detection in fecal samples using sensitive nucleic acid preparation, deep sequencing, and a novel iterative sequence classification algorithm</span>. <span class=\"ref-journal\">PLoS One</span>\n<span class=\"ref-vol\">9</span>: e93269. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3973683/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24695106\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Full+genome+virus+detection+in+fecal+samples+using+sensitive+nucleic+acid+preparation,+deep+sequencing,+and+a+novel+iterative+sequence+classification+algorithm&amp;volume=9&amp;publication_year=2014&amp;pages=e93269&amp;pmid=24695106&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C13\"><span class=\"mixed-citation\">Dehal P, Satou Y, Campbell RK, Chapman J, Degnan B, De Tomaso A, Davidson B, Di Gregorio A, Gelpke M, Goodstein DM, et al.\n2002. \n<span class=\"ref-title\">The draft genome of <em>Ciona intestinalis</em>: insights into chordate and vertebrate origins</span>. <span class=\"ref-journal\">Science</span>\n<span class=\"ref-vol\">298</span>: 2157&#x02013;2167. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n12481130\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+draft+genome+of+Ciona+intestinalis:+insights+into+chordate+and+vertebrate+origins&amp;volume=298&amp;publication_year=2002&amp;pages=2157-2167&amp;pmid=12481130&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C14\"><span class=\"mixed-citation\">Dick GJ, Andersson AF, Baker BJ, Simmons SL, Thomas BC, Yelton AP, Banfield JF. 2009. \n<span class=\"ref-title\">Community-wide analysis of microbial genome sequence signatures</span>. <span class=\"ref-journal\">Genome Biol</span>\n<span class=\"ref-vol\">10</span>: R85. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2745766/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n19698104\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Community-wide+analysis+of+microbial+genome+sequence+signatures&amp;volume=10&amp;publication_year=2009&amp;pages=R85&amp;pmid=19698104&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C15\"><span class=\"mixed-citation\">Donmez N, Brudno M. 2011. \n<span class=\"ref-title\">Hapsembler: an assembler for highly polymorphic genomes</span>. In <span class=\"ref-journal\">Research in computational molecular biology</span> (ed. Bafna V, Sahinalp SC), Vol. <span class=\"ref-vol\">6577</span>, pp. 38&#x02013;52. Springer, Berlin, Heidelberg. <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?title=Research+in+computational+molecular+biology&amp;publication_year=2011&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C16\"><span class=\"mixed-citation\">Dupont CL, Rusch DB, Yooseph S, Lombardo M-J, Alexander Richter R, Valas R, Novotny M, Yee-Greenbaum J, Selengut JD, Haft DH, et al.\n2012. \n<span class=\"ref-title\">Genomic insights to SAR86, an abundant and uncultivated marine bacterial lineage</span>. <span class=\"ref-journal\">ISME J</span>\n<span class=\"ref-vol\">6</span>: 1186&#x02013;1199. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3358033/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22170421\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=ISME+J&amp;title=Genomic+insights+to+SAR86,+an+abundant+and+uncultivated+marine+bacterial+lineage&amp;volume=6&amp;publication_year=2012&amp;pages=1186-1199&amp;pmid=22170421&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C17\"><span class=\"mixed-citation\">Fu L, Niu B, Zhu Z, Wu S, Li W. 2012. \n<span class=\"ref-title\">CD-HIT: accelerated for clustering the next-generation sequencing data</span>. <span class=\"ref-journal\">Bioinformatics</span>\n<span class=\"ref-vol\">28</span>: 3150&#x02013;3152. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3516142/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23060610\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=CD-HIT:+accelerated+for+clustering+the+next-generation+sequencing+data&amp;volume=28&amp;publication_year=2012&amp;pages=3150-3152&amp;pmid=23060610&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C18\"><span class=\"mixed-citation\">Garc&#x000ed;a-L&#x000f3;pez R, V&#x000e1;zquez-Castellanos JF, Moya A. 2015. \n<span class=\"ref-title\">Fragmentation and coverage variation in viral metagenome assemblies, and their effect in diversity calculations</span>. <span class=\"ref-journal\">Front Bioeng Biotechnol</span>\n<span class=\"ref-vol\">3</span>: 141. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4585024/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26442255\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front+Bioeng+Biotechnol&amp;title=Fragmentation+and+coverage+variation+in+viral+metagenome+assemblies,+and+their+effect+in+diversity+calculations&amp;volume=3&amp;publication_year=2015&amp;pages=141&amp;pmid=26442255&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C19\"><span class=\"mixed-citation\">Gevers D, Pop M, Schloss PD, Huttenhower C. 2012. \n<span class=\"ref-title\">Bioinformatics for the human microbiome project</span>. <span class=\"ref-journal\">PLoS Comput Biol</span>\n<span class=\"ref-vol\">8</span>: e1002779. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3510052/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23209389\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+Comput+Biol&amp;title=Bioinformatics+for+the+human+microbiome+project&amp;volume=8&amp;publication_year=2012&amp;pages=e1002779&amp;pmid=23209389&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C20\"><span class=\"mixed-citation\">Gurevich A, Saveliev V, Vyahhi N, Tesler G. 2013. \n<span class=\"ref-title\">QUAST: quality assessment tool for genome assemblies</span>. <span class=\"ref-journal\">Bioinformatics</span>\n<span class=\"ref-vol\">29</span>: 1072&#x02013;1075. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3624806/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23422339\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=QUAST:+quality+assessment+tool+for+genome+assemblies&amp;volume=29&amp;publication_year=2013&amp;pages=1072-1075&amp;pmid=23422339&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C21\"><span class=\"mixed-citation\">Haider B, Ahn T-H, Bushnell B, Chai J, Copeland A, Pan C. 2014. \n<span class=\"ref-title\">Omega: an overlap-graph de novo assembler for metagenomics</span>. <span class=\"ref-journal\">Bioinformatics</span>\n<span class=\"ref-vol\">30</span>: 2717&#x02013;2722. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24947750\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Omega:+an+overlap-graph+de+novo+assembler+for+metagenomics&amp;volume=30&amp;publication_year=2014&amp;pages=2717-2722&amp;pmid=24947750&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C22\"><span class=\"mixed-citation\">Hess M, Sczyrba A, Egan R, Kim T-W, Chokhawala H, Schroth G, Luo S, Clark DS, Chen F, Zhang T, et al.\n2011. \n<span class=\"ref-title\">Metagenomic discovery of biomass-degrading genes and genomes from cow rumen</span>. <span class=\"ref-journal\">Science</span>\n<span class=\"ref-vol\">331</span>: 463&#x02013;467. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21273488\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Metagenomic+discovery+of+biomass-degrading+genes+and+genomes+from+cow+rumen&amp;volume=331&amp;publication_year=2011&amp;pages=463-467&amp;pmid=21273488&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C23\"><span class=\"mixed-citation\">Hug LA, Castelle CJ, Wrighton KC, Thomas BC, Sharon I, Frischkorn KR, Williams KH, Tringe SG, Banfield JF. 2013. \n<span class=\"ref-title\">Community genomic analyses constrain the distribution of metabolic traits across the Chloroflexi phylum and indicate roles in sediment carbon cycling</span>. <span class=\"ref-journal\">Microbiome</span>\n<span class=\"ref-vol\">1</span>: 22. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3971608/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24450983\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Community+genomic+analyses+constrain+the+distribution+of+metabolic+traits+across+the+Chloroflexi+phylum+and+indicate+roles+in+sediment+carbon+cycling&amp;volume=1&amp;publication_year=2013&amp;pages=22&amp;pmid=24450983&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C24\"><span class=\"mixed-citation\">The Human Microbiome Project Consortium. 2012. \n<span class=\"ref-title\">Structure, function and diversity of the healthy human microbiome</span>. <span class=\"ref-journal\">Nature</span>\n<span class=\"ref-vol\">486</span>: 207&#x02013;214. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3564958/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22699609\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structure,+function+and+diversity+of+the+healthy+human+microbiome&amp;volume=486&amp;publication_year=2012&amp;pages=207-214&amp;pmid=22699609&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C25\"><span class=\"mixed-citation\">Hyatt D, Locascio PF, Hauser LJ, Uberbacher EC. 2012. \n<span class=\"ref-title\">Gene and translation initiation site prediction in metagenomic sequences</span>. <span class=\"ref-journal\">Bioinformatics</span>\n<span class=\"ref-vol\">28</span>: 2223&#x02013;2230. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22796954\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Gene+and+translation+initiation+site+prediction+in+metagenomic+sequences&amp;volume=28&amp;publication_year=2012&amp;pages=2223-2230&amp;pmid=22796954&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C26\"><span class=\"mixed-citation\">Iverson V, Morris RM, Frazar CD, Berthiaume CT, Morales RL, Armbrust EV. 2012. \n<span class=\"ref-title\">Untangling genomes from metagenomes: revealing an uncultured class of marine euryarchaeota</span>. <span class=\"ref-journal\">Science</span>\n<span class=\"ref-vol\">335</span>: 587&#x02013;590. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22301318\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Untangling+genomes+from+metagenomes:+revealing+an+uncultured+class+of+marine+euryarchaeota&amp;volume=335&amp;publication_year=2012&amp;pages=587-590&amp;pmid=22301318&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C27\"><span class=\"mixed-citation\">Kajitani R, Toshimoto K, Noguchi H, Toyoda A, Ogura Y, Okuno M, Yabana M, Harada M, Nagayasu E, Maruyama H, et al.\n2014. \n<span class=\"ref-title\">Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads</span>. <span class=\"ref-journal\">Genome Res</span>\n<span class=\"ref-vol\">24</span>: 1384&#x02013;1395. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4120091/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24755901\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Efficient+de+novo+assembly+of+highly+heterozygous+genomes+from+whole-genome+shotgun+short+reads&amp;volume=24&amp;publication_year=2014&amp;pages=1384-1395&amp;pmid=24755901&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C28\"><span class=\"mixed-citation\">Kashtan N, Roggensack SE, Rodrigue S, Thompson JW, Biller SJ, Coe A, Ding H, Marttinen P, Malmstrom RR, Stocker R, et al.\n2014. \n<span class=\"ref-title\">Single-cell genomics reveals hundreds of coexisting subpopulations in wild prochlorococcus</span>. <span class=\"ref-journal\">Science</span>\n<span class=\"ref-vol\">344</span>: 416&#x02013;420. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24763590\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Single-cell+genomics+reveals+hundreds+of+coexisting+subpopulations+in+wild+prochlorococcus&amp;volume=344&amp;publication_year=2014&amp;pages=416-420&amp;pmid=24763590&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C29\"><span class=\"mixed-citation\">Kleigrewe K, Almaliti J, Tian IY, Kinnel RB, Korobeynikov A, Monroe EA, Duggan BM, Di Marzo V, Sherman DH, Dorrestein PC, et al.\n2015. \n<span class=\"ref-title\">Combining mass spectrometric metabolic profiling with genomic analysis: a powerful approach for discovering natural products from cyanobacteria</span>. <span class=\"ref-journal\">J Nat Prod</span>\n<span class=\"ref-vol\">78</span>: 1671&#x02013;1682. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4681511/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26149623\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Nat+Prod&amp;title=Combining+mass+spectrometric+metabolic+profiling+with+genomic+analysis:+a+powerful+approach+for+discovering+natural+products+from+cyanobacteria&amp;volume=78&amp;publication_year=2015&amp;pages=1671-1682&amp;pmid=26149623&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C30\"><span class=\"mixed-citation\">Kleiner M, Hooper LV, Duerkop BA. 2015. \n<span class=\"ref-title\">Evaluation of methods to purify virus-like particles for metagenomic sequencing of intestinal viromes</span>. <span class=\"ref-journal\">BMC Genomics</span>\n<span class=\"ref-vol\">16</span>: 7. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4308010/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25608871\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=BMC+Genomics&amp;title=Evaluation+of+methods+to+purify+virus-like+particles+for+metagenomic+sequencing+of+intestinal+viromes&amp;volume=16&amp;publication_year=2015&amp;pages=7&amp;pmid=25608871&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C31\"><span class=\"mixed-citation\">Koren S, Treangen TJ, Pop M. 2011. \n<span class=\"ref-title\">Bambus 2: scaffolding metagenomes</span>. <span class=\"ref-journal\">Bioinformatics</span>\n<span class=\"ref-vol\">27</span>: 2964&#x02013;2971. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3198580/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21926123\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Bambus+2:+scaffolding+metagenomes&amp;volume=27&amp;publication_year=2011&amp;pages=2964-2971&amp;pmid=21926123&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C32\"><span class=\"mixed-citation\">Kuleshov V, Xie D, Chen R, Pushkarev D, Ma Z, Blauwkamp T, Kertesz M, Snyder M. 2014. \n<span class=\"ref-title\">Whole-genome haplotyping using long reads and statistical methods</span>. <span class=\"ref-journal\">Nat Biotechnol</span>\n<span class=\"ref-vol\">32</span>: 261&#x02013;266. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4073643/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24561555\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat+Biotechnol&amp;title=Whole-genome+haplotyping+using+long+reads+and+statistical+methods&amp;volume=32&amp;publication_year=2014&amp;pages=261-266&amp;pmid=24561555&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C33\"><span class=\"mixed-citation\">Kuleshov V, Jiang C, Zhou W, Jahanbani F, Batzoglou S, Snyder M. 2015. \n<span class=\"ref-title\">Synthetic long-read sequencing reveals intraspecies diversity in the human microbiome</span>. <span class=\"ref-journal\">Nat Biotechnol</span>\n<span class=\"ref-vol\">34</span>: 64&#x02013;69. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4884093/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26655498\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat+Biotechnol&amp;title=Synthetic+long-read+sequencing+reveals+intraspecies+diversity+in+the+human+microbiome&amp;volume=34&amp;publication_year=2015&amp;pages=64-69&amp;pmid=26655498&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C34\"><span class=\"mixed-citation\">Labont&#x000e9; JM, Swan BK, Poulos B, Luo H, Koren S, Hallam SJ, Sullivan MB, Woyke T, Wommack KE, Stepanauskas R. 2015. \n<span class=\"ref-title\">Single-cell genomics-based analysis of virus-host interactions in marine surface bacterioplankton</span>. <span class=\"ref-journal\">ISME J</span>\n<span class=\"ref-vol\">9</span>: 2386&#x02013;99. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4611503/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25848873\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=ISME+J&amp;title=Single-cell+genomics-based+analysis+of+virus-host+interactions+in+marine+surface+bacterioplankton&amp;volume=9&amp;publication_year=2015&amp;pages=2386-99&amp;pmid=25848873&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C35\"><span class=\"mixed-citation\">Langmead B, Salzberg SL. 2012. \n<span class=\"ref-title\">Fast gapped-read alignment with Bowtie 2</span>. <span class=\"ref-journal\">Nat Methods</span>\n<span class=\"ref-vol\">9</span>: 357&#x02013;359. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3322381/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22388286\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=Fast+gapped-read+alignment+with+Bowtie+2&amp;volume=9&amp;publication_year=2012&amp;pages=357-359&amp;pmid=22388286&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C36\"><span class=\"mixed-citation\">Laserson J, Jojic V, Koller D. 2011. \n<span class=\"ref-title\">Genovo: de novo assembly for metagenomes</span>. <span class=\"ref-journal\">J Comput Biol</span>\n<span class=\"ref-vol\">18</span>: 429&#x02013;443. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21385045\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Comput+Biol&amp;title=Genovo:+de+novo+assembly+for+metagenomes&amp;volume=18&amp;publication_year=2011&amp;pages=429-443&amp;pmid=21385045&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C37\"><span class=\"mixed-citation\">Li W, Godzik A. 2006. \n<span class=\"ref-title\">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences</span>. <span class=\"ref-journal\">Bioinformatics</span>\n<span class=\"ref-vol\">22</span>: 1658&#x02013;1659. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n16731699\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Cd-hit:+a+fast+program+for+clustering+and+comparing+large+sets+of+protein+or+nucleotide+sequences&amp;volume=22&amp;publication_year=2006&amp;pages=1658-1659&amp;pmid=16731699&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C38\"><span class=\"mixed-citation\">Li D, Liu C-M, Luo R, Sadakane K, Lam T-W. 2015. \n<span class=\"ref-title\">MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph</span>. <span class=\"ref-journal\">Bioinformatics</span>\n<span class=\"ref-vol\">31</span>: 1674&#x02013;1676. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25609793\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=MEGAHIT:+an+ultra-fast+single-node+solution+for+large+and+complex+metagenomics+assembly+via+succinct+de+Bruijn+graph&amp;volume=31&amp;publication_year=2015&amp;pages=1674-1676&amp;pmid=25609793&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C39\"><span class=\"mixed-citation\">Li D, Luo R, Liu C-M, Leung C-M, Ting H-F, Sadakane K, Yamashita H, Lam T-W. 2016. \n<span class=\"ref-title\">MEGAHIT v1.0: a fast and scalable metagenome assembler driven by advanced methodologies and community practices</span>. <span class=\"ref-journal\">Methods</span>\n<span class=\"ref-vol\">102</span>: 3&#x02013;11. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27012178\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=MEGAHIT+v1.0:+a+fast+and+scalable+metagenome+assembler+driven+by+advanced+methodologies+and+community+practices&amp;volume=102&amp;publication_year=2016&amp;pages=3-11&amp;pmid=27012178&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C40\"><span class=\"mixed-citation\">Mavromatis K, Ivanova N, Barry KW, Shapiro HJ, Goltsman E, McHardy AC, Rigoutsos I, Salamov A, Korzeniewski F, Land M, et al.\n2007. \n<span class=\"ref-title\">Use of simulated data sets to evaluate the fidelity of metagenomic processing methods</span>. <span class=\"ref-journal\">Nat Methods</span>\n<span class=\"ref-vol\">4</span>: 495&#x02013;500. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17468765\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=Use+of+simulated+data+sets+to+evaluate+the+fidelity+of+metagenomic+processing+methods&amp;volume=4&amp;publication_year=2007&amp;pages=495-500&amp;pmid=17468765&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C41\"><span class=\"mixed-citation\">McCoy RC, Taylor RW, Blauwkamp TA, Kelley JL, Kertesz M, Pushkarev D, Petrov DA, Fiston-Lavier A-S. 2014. \n<span class=\"ref-title\">Illumina TruSeq synthetic long-reads empower de novo assembly and resolve complex, highly-repetitive transposable elements</span>. <span class=\"ref-journal\">PLoS One</span>\n<span class=\"ref-vol\">9</span>: e106689. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4154752/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25188499\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Illumina+TruSeq+synthetic+long-reads+empower+de+novo+assembly+and+resolve+complex,+highly-repetitive+transposable+elements&amp;volume=9&amp;publication_year=2014&amp;pages=e106689&amp;pmid=25188499&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C42\"><span class=\"mixed-citation\">McLean JS, Lombardo M-J, Badger JH, Edlund A, Novotny M, Yee-Greenbaum J, Vyahhi N, Hall AP, Yang Y, Dupont CL, et al.\n2013. \n<span class=\"ref-title\">Candidate phylum TM6 genome recovered from a hospital sink biofilm provides genomic insights into this uncultivated phylum</span>. <span class=\"ref-journal\">Proc Natl Acad Sci</span>\n<span class=\"ref-vol\">110</span>: E2390&#x02013;E2399. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3696752/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23754396\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci&amp;title=Candidate+phylum+TM6+genome+recovered+from+a+hospital+sink+biofilm+provides+genomic+insights+into+this+uncultivated+phylum&amp;volume=110&amp;publication_year=2013&amp;pages=E2390-E2399&amp;pmid=23754396&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C43\"><span class=\"mixed-citation\">Mende DR, Waller AS, Sunagawa S, J&#x000e4;rvelin AI, Chan MM, Arumugam M, Raes J, Bork P. 2012. \n<span class=\"ref-title\">Assessment of metagenomic assembly using simulated next generation sequencing data</span>. <span class=\"ref-journal\">PLoS One</span>\n<span class=\"ref-vol\">7</span>: e31386. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3285633/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22384016\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Assessment+of+metagenomic+assembly+using+simulated+next+generation+sequencing+data&amp;volume=7&amp;publication_year=2012&amp;pages=e31386&amp;pmid=22384016&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C44\"><span class=\"mixed-citation\">Mikheenko A, Saveliev V, Gurevich A. 2016. \n<span class=\"ref-title\">MetaQUAST: evaluation of metagenome assemblies</span>. <span class=\"ref-journal\">Bioinformatics</span>\n<span class=\"ref-vol\">32</span>: 1088&#x02013;1090. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26614127\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=MetaQUAST:+evaluation+of+metagenome+assemblies&amp;volume=32&amp;publication_year=2016&amp;pages=1088-1090&amp;pmid=26614127&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C45\"><span class=\"mixed-citation\">Miller IJ, Weyna TR, Fong SS, Lim-Fong GE, Kwan JC. 2016. \n<span class=\"ref-title\">Single sample resolution of rare microbial dark matter in a marine invertebrate metagenome</span>. <span class=\"ref-journal\">Sci Rep</span>\n<span class=\"ref-vol\">6</span>: 34362. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC5041132/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n27681823\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Sci+Rep&amp;title=Single+sample+resolution+of+rare+microbial+dark+matter+in+a+marine+invertebrate+metagenome&amp;volume=6&amp;publication_year=2016&amp;pages=34362&amp;pmid=27681823&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C46\"><span class=\"mixed-citation\">Myers EW, Sutton GG, Delcher AL, Dew IM, Fasulo DP, Flanigan MJ, Kravitz SA, Mobarry CM, Reinert KH, Remington KA, et al.\n2000. \n<span class=\"ref-title\">A whole-genome assembly of <em>Drosophila</em></span>. <span class=\"ref-journal\">Science</span>\n<span class=\"ref-vol\">287</span>: 2196&#x02013;2204. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n10731133\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+whole-genome+assembly+of+Drosophila&amp;volume=287&amp;publication_year=2000&amp;pages=2196-2204&amp;pmid=10731133&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C47\"><span class=\"mixed-citation\">Nagarajan N, Pop M. 2013. \n<span class=\"ref-title\">Sequence assembly demystified</span>. <span class=\"ref-journal\">Nat Rev Genet</span>\n<span class=\"ref-vol\">14</span>: 157&#x02013;67. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23358380\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Sequence+assembly+demystified&amp;volume=14&amp;publication_year=2013&amp;pages=157-67&amp;pmid=23358380&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C48\"><span class=\"mixed-citation\">Namiki T, Hachiya T, Tanaka H, Sakakibara Y. 2012. \n<span class=\"ref-title\">MetaVelvet: an extension of Velvet assembler to de novo metagenome assembly from short sequence reads</span>. <span class=\"ref-journal\">Nucleic Acids Res</span>\n<span class=\"ref-vol\">40</span>: e155&#x02013;e155. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3488206/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22821567\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=MetaVelvet:+an+extension+of+Velvet+assembler+to+de+novo+metagenome+assembly+from+short+sequence+reads&amp;volume=40&amp;publication_year=2012&amp;pages=e155-e155&amp;pmid=22821567&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C49\"><span class=\"mixed-citation\">Nijkamp JF, Pop M, Reinders MJT, de Ridder D. 2013. \n<span class=\"ref-title\">Exploring variation-aware contig graphs for (comparative) metagenomics using MaryGold</span>. <span class=\"ref-journal\">Bioinformatics</span>\n<span class=\"ref-vol\">29</span>: 2826&#x02013;2834. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3916741/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24058058\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Exploring+variation-aware+contig+graphs+for+(comparative)+metagenomics+using+MaryGold&amp;volume=29&amp;publication_year=2013&amp;pages=2826-2834&amp;pmid=24058058&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C50\"><span class=\"mixed-citation\">Nurk S, Bankevich A, Antipov D, Gurevich AA, Korobeynikov A, Lapidus A, Prjibelski AD, Pyshkin A, Sirotkin A, Sirotkin Y, et al.\n2013. \n<span class=\"ref-title\">Assembling single-cell genomes and mini-metagenomes from chimeric MDA products</span>. <span class=\"ref-journal\">J Comput Biol</span>\n<span class=\"ref-vol\">20</span>: 714&#x02013;737. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3791033/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24093227\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Comput+Biol&amp;title=Assembling+single-cell+genomes+and+mini-metagenomes+from+chimeric+MDA+products&amp;volume=20&amp;publication_year=2013&amp;pages=714-737&amp;pmid=24093227&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C51\"><span class=\"mixed-citation\">Peng Y, Leung HCM, Yiu SM, Chin FYL. 2011. \n<span class=\"ref-title\">Meta-IDBA: a de novo assembler for metagenomic data</span>. <span class=\"ref-journal\">Bioinformatics</span>\n<span class=\"ref-vol\">27</span>: 94&#x02013;101. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3117360/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21685107\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Meta-IDBA:+a+de+novo+assembler+for+metagenomic+data&amp;volume=27&amp;publication_year=2011&amp;pages=94-101&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C52\"><span class=\"mixed-citation\">Peng Y, Leung HCM, Yiu SM, Chin FYL. 2012. \n<span class=\"ref-title\">IDBA-UD: a de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth</span>. <span class=\"ref-journal\">Bioinformatics</span>\n<span class=\"ref-vol\">28</span>: 1420&#x02013;1428. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22495754\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=IDBA-UD:+a+de+novo+assembler+for+single-cell+and+metagenomic+sequencing+data+with+highly+uneven+depth&amp;volume=28&amp;publication_year=2012&amp;pages=1420-1428&amp;pmid=22495754&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C53\"><span class=\"mixed-citation\">Pevzner PA, Tang H, Tesler G. 2004. \n<span class=\"ref-title\">De novo repeat classification and fragment assembly</span>. <span class=\"ref-journal\">Genome Res</span>\n<span class=\"ref-vol\">14</span>: 1786&#x02013;1796. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC515325/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n15342561\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=De+novo+repeat+classification+and+fragment+assembly&amp;volume=14&amp;publication_year=2004&amp;pages=1786-1796&amp;pmid=15342561&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C54\"><span class=\"mixed-citation\">Prjibelski AD, Vasilinetc I, Bankevich A, Gurevich A, Krivosheeva T, Nurk S, Pham S, Korobeynikov A, Lapidus A, Pevzner PA. 2014. \n<span class=\"ref-title\">ExSPAnder: a universal repeat resolver for DNA fragment assembly</span>. <span class=\"ref-journal\">Bioinformatics</span>\n<span class=\"ref-vol\">30</span>: 293&#x02013;301. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4058921/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n24931996\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=ExSPAnder:+a+universal+repeat+resolver+for+DNA+fragment+assembly&amp;volume=30&amp;publication_year=2014&amp;pages=293-301&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C55\"><span class=\"mixed-citation\">Richter DC, Ott F, Auch AF, Schmid R, Huson DH. 2008. \n<span class=\"ref-title\">MetaSim: a sequencing simulator for genomics and metagenomics ed. D. Field</span>. <span class=\"ref-journal\">PLoS One</span>\n<span class=\"ref-vol\">3</span>: e3373. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2556396/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n18841204\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=MetaSim:+a+sequencing+simulator+for+genomics+and+metagenomics+ed.+D.+Field&amp;volume=3&amp;publication_year=2008&amp;pages=e3373&amp;pmid=18841204&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C56\"><span class=\"mixed-citation\">Rosen MJ, Davison M, Bhaya D, Fisher DS. 2015. \n<span class=\"ref-title\">Fine-scale diversity and extensive recombination in a quasisexual bacterial population occupying a broad niche</span>. <span class=\"ref-journal\">Science</span>\n<span class=\"ref-vol\">348</span>: 1019&#x02013;1023. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26023139\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Fine-scale+diversity+and+extensive+recombination+in+a+quasisexual+bacterial+population+occupying+a+broad+niche&amp;volume=348&amp;publication_year=2015&amp;pages=1019-1023&amp;pmid=26023139&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C57\"><span class=\"mixed-citation\">Safonova Y, Bankevich A, Pevzner P. 2015. \n<span class=\"ref-title\">dipSPAdes: assembler for highly polymorphic diploid genomes</span>. <span class=\"ref-journal\">J Comput Biol</span>\n<span class=\"ref-vol\">22</span>: 528&#x02013;545. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4449708/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25734602\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Comput+Biol&amp;title=dipSPAdes:+assembler+for+highly+polymorphic+diploid+genomes&amp;volume=22&amp;publication_year=2015&amp;pages=528-545&amp;pmid=25734602&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C58\"><span class=\"mixed-citation\">Salzberg SL, Phillippy AM, Zimin A, Puiu D, Magoc T, Koren S, Treangen TJ, Schatz MC, Delcher AL, Roberts M, et al.\n2012. \n<span class=\"ref-title\">GAGE: a critical evaluation of genome assemblies and assembly algorithms</span>. <span class=\"ref-journal\">Genome Res</span>\n<span class=\"ref-vol\">22</span>: 557&#x02013;567. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3290791/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n22147368\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=GAGE:+a+critical+evaluation+of+genome+assemblies+and+assembly+algorithms&amp;volume=22&amp;publication_year=2012&amp;pages=557-567&amp;pmid=22147368&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C59\"><span class=\"mixed-citation\">Shakya M, Quince C, Campbell JH, Yang ZK, Schadt CW, Podar M. 2013. \n<span class=\"ref-title\">Comparative metagenomic and rRNA microbial diversity characterization using archaeal and bacterial synthetic communities</span>. <span class=\"ref-journal\">Environ Microbiol</span>\n<span class=\"ref-vol\">15</span>: 1882&#x02013;1899. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3665634/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23387867\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Environ+Microbiol&amp;title=Comparative+metagenomic+and+rRNA+microbial+diversity+characterization+using+archaeal+and+bacterial+synthetic+communities&amp;volume=15&amp;publication_year=2013&amp;pages=1882-1899&amp;pmid=23387867&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C60\"><span class=\"mixed-citation\">Sharon I, Kertesz M, Hug LA, Pushkarev D, Blauwkamp TA, Castelle CJ, Amirebrahimi M, Thomas BC, Burstein D, Tringe SG, et al.\n2015. \n<span class=\"ref-title\">Accurate, multi-kb reads resolve complex populations and detect rare microorganisms</span>. <span class=\"ref-journal\">Genome Res</span>\n<span class=\"ref-vol\">25</span>: 534&#x02013;543. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4381525/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25665577\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Accurate,+multi-kb+reads+resolve+complex+populations+and+detect+rare+microorganisms&amp;volume=25&amp;publication_year=2015&amp;pages=534-543&amp;pmid=25665577&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C61\"><span class=\"mixed-citation\">Treangen TJ, Koren S, Sommer DD, Liu B, Astrovskaya I, Ondov B, Darling AE, Phillippy AM, Pop M. 2013. \n<span class=\"ref-title\">MetAMOS: a modular and open source metagenomic assembly and analysis pipeline</span>. <span class=\"ref-journal\">Genome Biol</span>\n<span class=\"ref-vol\">14</span>: R2. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4053804/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n23320958\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=MetAMOS:+a+modular+and+open+source+metagenomic+assembly+and+analysis+pipeline&amp;volume=14&amp;publication_year=2013&amp;pages=R2&amp;pmid=23320958&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C62\"><span class=\"mixed-citation\">Tsai Y-C, Conlan S, Deming C, Segre JA, Kong HH, Korlach J, Oh J. 2016. \n<span class=\"ref-title\">Resolving the complexity of human skin metagenomes using single-molecule sequencing</span>. <span class=\"ref-journal\">MBio</span>\n<span class=\"ref-vol\">7</span>: e01948&#x02013;15. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4752602/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26861018\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=MBio&amp;title=Resolving+the+complexity+of+human+skin+metagenomes+using+single-molecule+sequencing&amp;volume=7&amp;publication_year=2016&amp;pages=e01948-15&amp;pmid=26861018&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C63\"><span class=\"mixed-citation\">Turnbaugh PJ, Ley RE, Hamady M, Fraser-Liggett CM, Knight R, Gordon JI. 2007. \n<span class=\"ref-title\">The human microbiome project</span>. <span class=\"ref-journal\">Nature</span>\n<span class=\"ref-vol\">449</span>: 804&#x02013;810. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3709439/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17943116\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+human+microbiome+project&amp;volume=449&amp;publication_year=2007&amp;pages=804-810&amp;pmid=17943116&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C64\"><span class=\"mixed-citation\">Tyson GW, Chapman J, Hugenholtz P, Allen EE, Ram RJ, Richardson PM, Solovyev VV, Rubin EM, Rokhsar DS, Banfield JF. 2004. \n<span class=\"ref-title\">Community structure and metabolism through reconstruction of microbial genomes from the environment</span>. <span class=\"ref-journal\">Nature</span>\n<span class=\"ref-vol\">428</span>: 37&#x02013;43. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n14961025\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Community+structure+and+metabolism+through+reconstruction+of+microbial+genomes+from+the+environment&amp;volume=428&amp;publication_year=2004&amp;pages=37-43&amp;pmid=14961025&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C65\"><span class=\"mixed-citation\">Vasilinetc I, Prjibelski AD, Gurevich A, Korobeynikov A, Pevzner P. 2015. \n<span class=\"ref-title\">Assembling short reads from jumping libraries with large insert sizes</span>. <span class=\"ref-journal\">Bioinformatics</span>\n<span class=\"ref-vol\">31</span>: 3262&#x02013;3268. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26040456\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Assembling+short+reads+from+jumping+libraries+with+large+insert+sizes&amp;volume=31&amp;publication_year=2015&amp;pages=3262-3268&amp;pmid=26040456&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C66\"><span class=\"mixed-citation\">Venter JC, Remington K, Heidelberg JF, Halpern AL, Rusch D, Eisen JA, Wu D, Paulsen I, Nelson KE, Nelson W, et al.\n2004. \n<span class=\"ref-title\">Environmental genome shotgun sequencing of the Sargasso Sea</span>. <span class=\"ref-journal\">Science</span>\n<span class=\"ref-vol\">304</span>: 66&#x02013;74. [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n15001713\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Environmental+genome+shotgun+sequencing+of+the+Sargasso+Sea&amp;volume=304&amp;publication_year=2004&amp;pages=66-74&amp;pmid=15001713&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C67\"><span class=\"mixed-citation\">Vinson JP, Jaffe DB, O\'Neill K, Karlsson EK, Stange-Thomann N, Anderson S, Mesirov JP, Satoh N, Satou Y, Nusbaum C, et al.\n2005. \n<span class=\"ref-title\">Assembly of polymorphic genomes: algorithms and application to <em>Ciona savignyi</em></span>. <span class=\"ref-journal\">Genome Res</span>\n<span class=\"ref-vol\">15</span>: 1127&#x02013;1135. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC1182225/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n16077012\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Assembly+of+polymorphic+genomes:+algorithms+and+application+to+Ciona+savignyi&amp;volume=15&amp;publication_year=2005&amp;pages=1127-1135&amp;pmid=16077012&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C68\"><span class=\"mixed-citation\">Wu Y-W, Ye Y. 2011. \n<span class=\"ref-title\">A novel abundance-based algorithm for binning metagenomic sequences using <em>l</em> -tuples</span>. <span class=\"ref-journal\">J Comput Biol</span>\n<span class=\"ref-vol\">18</span>: 523&#x02013;534. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC3123841/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n21385052\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=J+Comput+Biol&amp;title=A+novel+abundance-based+algorithm+for+binning+metagenomic+sequences+using+l+-tuples&amp;volume=18&amp;publication_year=2011&amp;pages=523-534&amp;pmid=21385052&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C69\"><span class=\"mixed-citation\">Wu Y-W, Tang Y-H, Tringe SG, Simmons BA, Singer SW. 2014. \n<span class=\"ref-title\">MaxBin: an automated binning method to recover individual genomes from metagenomes using an expectation-maximization algorithm</span>. <span class=\"ref-journal\">Microbiome</span>\n<span class=\"ref-vol\">2</span>: 26. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4129434/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n25136443\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=MaxBin:+an+automated+binning+method+to+recover+individual+genomes+from+metagenomes+using+an+expectation-maximization+algorithm&amp;volume=2&amp;publication_year=2014&amp;pages=26&amp;pmid=25136443&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C70\"><span class=\"mixed-citation\">Xie M, Ren M, Yang C, Yi H, Li Z, Li T, Zhao J. 2016. \n<span class=\"ref-title\">Metagenomic analysis reveals symbiotic relationship among bacteria in microcystis-dominated community</span>. <span class=\"ref-journal\">Front Microbiol</span>\n<span class=\"ref-vol\">7</span>: 56. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC4735357/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n26870018\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Front+Microbiol&amp;title=Metagenomic+analysis+reveals+symbiotic+relationship+among+bacteria+in+microcystis-dominated+community&amp;volume=7&amp;publication_year=2016&amp;pages=56&amp;pmid=26870018&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C71\"><span class=\"mixed-citation\">Yooseph S, Sutton G, Rusch DB, Halpern AL, Williamson SJ, Remington K, Eisen JA, Heidelberg KB, Manning G, Li W, et al.\n2007. \n<span class=\"ref-title\">The Sorcerer II global ocean sampling expedition: expanding the universe of protein families</span>. <span class=\"ref-journal\">PLoS Biol</span>\n<span class=\"ref-vol\">5</span>: 0432&#x02013;0466. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC1821046/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n17355171\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=The+Sorcerer+II+global+ocean+sampling+expedition:+expanding+the+universe+of+protein+families&amp;volume=5&amp;publication_year=2007&amp;pages=0432-0466&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li><li id=\"NURKGR213959C72\"><span class=\"mixed-citation\">Zerbino DR, Birney E. 2008. \n<span class=\"ref-title\">Velvet: algorithms for de novo short read assembly using de Bruijn graphs</span>. <span class=\"ref-journal\">Genome Res</span>\n<span class=\"ref-vol\">18</span>: 821&#x02013;829. <span class=\"nowrap\">[<a class=\"int-reflink\" target=\"_BLANK\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles\r\n/PMC2336801/\">PMC free article</a>]</span> [<a target=\"_BLANK\" href=\"https://pubmed.ncbi.nlm.nih.gov/\r\n18349386\" ref=\"reftype=pubmed&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord\">PubMed</a>] <span class=\"nowrap\">[<a href=\"https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Velvet:+algorithms+for+de+novo+short+read+assembly+using+de+Bruijn+graphs&amp;volume=18&amp;publication_year=2008&amp;pages=821-829&amp;pmid=18349386&amp;\" target=\"_blank\" rel=\"noopener noreferrer\" ref=\"reftype=other&amp;article-id=5411777&amp;issue-id=291735&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar\">Google Scholar</a>]</span></span></li></ul></div></div><br>', NULL, 2);

-- --------------------------------------------------------

--
-- Table structure for table `authors`
--

DROP TABLE IF EXISTS `authors`;
CREATE TABLE IF NOT EXISTS `authors` (
  `id_author` int NOT NULL AUTO_INCREMENT,
  `name` varchar(255) NOT NULL,
  `firstname` varchar(255) NOT NULL,
  `position` int DEFAULT NULL,
  PRIMARY KEY (`id_author`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;

-- --------------------------------------------------------

--
-- Table structure for table `author_article`
--

DROP TABLE IF EXISTS `author_article`;
CREATE TABLE IF NOT EXISTS `author_article` (
  `id_author` int NOT NULL,
  `id_article` int NOT NULL,
  KEY `id_author` (`id_author`),
  KEY `id_article` (`id_article`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;

-- --------------------------------------------------------

--
-- Table structure for table `note`
--

DROP TABLE IF EXISTS `note`;
CREATE TABLE IF NOT EXISTS `note` (
  `id_article` int NOT NULL,
  `id_user` int NOT NULL,
  `note` enum('1','2','3','4','5') NOT NULL,
  KEY `id_article` (`id_article`),
  KEY `id_user` (`id_user`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;

--
-- Dumping data for table `note`
--

INSERT INTO `note` (`id_article`, `id_user`, `note`) VALUES
(27, 2, '3'),
(18, 2, '2'),
(81, 2, '3');

-- --------------------------------------------------------

--
-- Table structure for table `status`
--

DROP TABLE IF EXISTS `status`;
CREATE TABLE IF NOT EXISTS `status` (
  `id_status` int NOT NULL AUTO_INCREMENT,
  `name_status` varchar(15) NOT NULL,
  PRIMARY KEY (`id_status`)
) ENGINE=InnoDB AUTO_INCREMENT=5 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;

--
-- Dumping data for table `status`
--

INSERT INTO `status` (`id_status`, `name_status`) VALUES
(1, 'Undefined'),
(2, 'Tasks'),
(3, 'Processed'),
(4, 'Rejected');

-- --------------------------------------------------------

--
-- Table structure for table `user`
--

DROP TABLE IF EXISTS `user`;
CREATE TABLE IF NOT EXISTS `user` (
  `id_user` int NOT NULL AUTO_INCREMENT,
  `name_user` varchar(15) NOT NULL,
  `email` varchar(255) DEFAULT NULL,
  `password` varchar(200) DEFAULT NULL,
  `profile` enum('expert','assistant') DEFAULT NULL,
  PRIMARY KEY (`id_user`)
) ENGINE=InnoDB AUTO_INCREMENT=24 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;

--
-- Dumping data for table `user`
--

INSERT INTO `user` (`id_user`, `name_user`, `email`, `password`, `profile`) VALUES
(1, 'Nicolas', 'nicolas@mail.com', '1234', NULL),
(2, 'Eddy IKHLEF', 'eddy.ikhlef@protonmail.com', '$2y$10$HWiiDwRMwF06bRvN0k2EdO3Y1/EqosajkHcyIAzd3Ylsfp1RkJZqe', 'expert'),
(3, 'Vincent', 'vincent@mail.com', '1234', NULL),
(4, 'Marie-Line', 'marieline@mail.com', '1234', NULL),
(5, 'Pedro', 'pedro@gmail.com', '1234', NULL),
(6, 'thierry', 'thierry.galliano@etu.univ-amu.fr', '$2y$10$R0HIU0GkdXConABOnNuTZePLhGu7bgMqLGR/aZHbkMoSCdo6jG4Em', 'expert');

--
-- Constraints for dumped tables
--

--
-- Constraints for table `article`
--
ALTER TABLE `article`
  ADD CONSTRAINT `article_ibfk_1` FOREIGN KEY (`status`) REFERENCES `status` (`id_status`);

--
-- Constraints for table `author_article`
--
ALTER TABLE `author_article`
  ADD CONSTRAINT `author_article_ibfk_1` FOREIGN KEY (`id_author`) REFERENCES `authors` (`id_author`),
  ADD CONSTRAINT `author_article_ibfk_2` FOREIGN KEY (`id_article`) REFERENCES `article` (`id_article`);

--
-- Constraints for table `note`
--
ALTER TABLE `note`
  ADD CONSTRAINT `note_ibfk_1` FOREIGN KEY (`id_article`) REFERENCES `article` (`id_article`),
  ADD CONSTRAINT `note_ibfk_2` FOREIGN KEY (`id_user`) REFERENCES `user` (`id_user`);
COMMIT;

/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
